# DESCRIPTION

## STATEMENT REGARDING FEDERAL FUNDING

This invention was made with government support under Grant No. GM062449, awarded by the National Institutes of Health. The government has certain rights in this invention.

Understanding an organism's metabolism at a system level requires knowledge of the physicochemical constraints limiting its metabolic capabilities under different growth conditions, and the genetic regulatory mechanisms that ultimately allow it to adapt to a changing environment. In some cases there is an obvious connection between an environmental change and the regulatory mechanisms responding to it, an example being a switch from aerobic to anaerobic growth. However, there are constraints leading to less obvious metabolic changes, involving a complex global rearrangement of the cell's metabolism. A key aim of systems biology is to uncover the metabolic constraints determining such complex phenotypic changes, which can be understood only when the system is analyzed at a global scale. In the absence of cell-scale kinetic models, flux balance analysis (FBA) provides experimentally testable predictions on an organism's metabolic flux state, which are based on conservation principles, particularly mass conservation, and metabolic capacity constraints. The impact of local constraints, such as uptake capacities, has been investigated, and capacity constraints over full metabolic pathways have been considered as well.

## SUMMARY

The technologies described herein are useful in, without limitation: 1) calculating cell growth rates in various environments and genetic backgrounds; 2) calculating the order of substrate utilization from a defined growth medium; 3) calculating metabolic flux reorganization in various environments and at various growth rates; and 4) calculating the maximum metabolic rate and optimal metabolite concentrations and enzyme activities by applying a computational optimization method to a kinetic model of a metabolic pathway. The methods described herein, including computer-implemented methods and/or apparatus, such as computer-devices embodying/for implementing the methods supercede existing methods and capabilities in several ways. In particular, current flux balance-based modeling approaches have limited ability to predict substrate uptake from the environment. Also, the predictions they generate are based on previous knowledge of the maximum uptake rates in the corresponding medium (the actual variables one aims to predict), and, in contrast to extensive experimental evidence, FBA in itself predicts the simultaneous utilization of all carbon sources from a mixed-substrate growth medium. Using the modeling framework described herein, one can provide superior capabilities to predict cellular metabolism over the existing modeling frameworks.

In one embodiment, provided herein is a method of optimizing one or more biological activities in cells in a cell culture, comprising: calculating one or more optimal cell culture parameters for one or more biological activities in a cell in a cell culture by applying an optimization method to a list of one or more reactions representing or affecting the one or more biological activities, wherein the optimization method uses an intracellular molecular crowding parameter (e.g., employs one or more values, algorithms, formulas, etc., that represent the extent of intracellular crowding or solvent capacity of the cell in calculating an optimal value) for one or more elements of the one or more reactions to calculate a cell culture parameter for the one or more biological activities in the cells. The method further comprises initiating and/or maintaining the one or more optimal cell culture parameter in a culture of the cells. In one embodiment, the cell culture parameter is a concentration of a metabolite in the cell culture, such as a carbon or nitrogen source, including, without limitation, one or more of: glucose, galactose, maltose, lactate, glycerol, an amino acid and a nucleotide. Additional non-limiting examples of a cell culture parameter include: O2, CO2, or N2 levels, pH, buffer capacity, viscosity, degree of agitation (e.g., stirring), cell density, etc. In one non-limiting embodiment the method comprises calculating an order of substrate usage from a growth medium and controlling an order of substrate usage in the culture to achieve the one or more optimal biological activities.

The method may further comprise calculating one or more of a maximum metabolic rate, an optimal metabolite concentration and an enzyme activity by applying a computational optimization method to a kinetic model of a metabolic pathway.

In another embodiment, a method is provided for achieving an optimal function of a biochemical reaction network in a cell. The method comprises: (a) calculating optimal properties of a biochemical reaction network by applying a computational optimization method to a list of reactions representing said biochemical reaction network, wherein the optimization method uses an intracellular molecular crowding parameter for one or more elements of the one or more reactions of the list of reactions to calculate the optimal properties; (b) altering the list of reactions in the biochemical reaction network and re-computing the optimal properties; and (c) repeating (b) until a desired optimal function is reached. The method may further comprise calculating one or more of a maximum metabolic rate, an optimal metabolite concentration and an enzyme activity by applying a computational optimization method to a kinetic model of a metabolic pathway. The method also may further comprise culturing a cell under culture conditions that favor achievement of the optimal function (e.g., the cells are cultured in a manner, based on the calculations, that is calculated to result in achievement of the optimal function, such as growth rate or metabolite (e.g., starting material, intermediate or product of a cellular metabolic process), by-product, secreted product or (e.g., recombinant) protein production). The method may further comprise (d) constructing the genetic makeup of a cell to contain the biochemical reactions which result from (c); (e) placing the cell constructed under (d) in culture under a specified environment to obtain a population of cells; and (f) cultivating the cells as in step (e) for a sufficient period of time and under conditions to allow the cells to evolve to the desired optimal function determined under (c), wherein the biochemical reaction network comprises a comprehensive biochemical reaction network.

According to a further embodiment, provided herein is a computer-implemented method for achieving an optimal function of a biochemical reaction network in cells in a cell culture, including prokaryotic or eukaryotic cells, for example and without limitation, bacterial, fungal (including yeast), mammalian and cancer cells. The method comprises (a) calculating in a computer one or more optimal cell culture parameters in a cell culture using a flux balance analysis constrained by a solvent capacity of cells in the cell culture, wherein the optimal cell culture parameter is calculated by determining an optimal property of a biochemical reaction network comprising a list of biochemical reactions by applying a computational optimization method to one or more of the biochemical reactions of the biochemical reaction network, the optimization method comprising (i) altering one or more elements of the one or more biochemical reactions in the biochemical reaction network and re-computing the optimal property, and (ii) repeating (i) until an optimal function is reached; and (b) initiating or maintaining the optimal cell culture parameter in the cell culture to achieve the optimal function of the biochemical reaction network in the cells. According to one embodiment, the optimization method further comprises calculating one or more of a maximum metabolic rate, an optimal metabolite concentration and an enzyme activity by applying a computational optimization method to a kinetic model of a metabolic pathway. In an example of an implementation of the method, the method further comprises culturing a cell under culture conditions that favor achievement of the optimal function. Optionally, the method further comprises: (c) constructing the genetic makeup of a cell to contain the biochemical reactions (d) placing the cell constructed under (c) in culture under a specified environment to obtain a population of cells; and (e) cultivating the cells as in step (d) for a sufficient period of time and under conditions to allow the cells to evolve to the desired optimal function determined under (a), wherein the biochemical reaction network comprises a comprehensive biochemical reaction network. In one embodiment, the optimal function is maximizing biomass production. In another, the optimal property is maximal internal yield of ATP. In a further embodiment, the solvent capacity accounts for ribosome density in the cells in the cell culture as a measure of ribosomal-, enzyme associated-, and non-metabolic proteins in the cells or accounts for mitochondria as a subcellular compartment in the cells in the cell culture. According to one embodiment, the method comprises a model of a pathway for ATP generation that involves reactions in the serine biosynthesis, one-carbon metabolism and the glycine cleavage system.

According to another embodiment, also provided is a computer readable medium having stored thereon instructions which, when executed by a processor, cause the processor to implement a process implementing a computer model for achieving an optimal function of a biochemical reaction network in cells in a cell culture. The process comprises: (a) calculating one or more optimal cell culture parameters in a cell culture using a flux balance analysis constrained by a solvent capacity of cells in the cell culture, wherein the optimal cell culture parameter is calculated by determining an optimal property of a biochemical reaction network comprising a list of biochemical reactions by applying a computational optimization method to one or more of the biochemical reactions of the biochemical reaction network, the optimization method comprising (i) altering one or more elements of the one or more biochemical reactions in the biochemical reaction network and re-computing the optimal property, and (ii) repeating (i) until an optimal function is reached; and (b) initiating or maintaining the optimal cell culture parameter in the cell culture to achieve the optimal function of the biochemical reaction network in the cells. The computer-readable medium may include in its various embodiments instructions which, when executed by a processor, cause the processor to implement a process implementing a computer model for achieving any method described in this document, including, without limitation, the following: a computer readable medium as described above in which the optimization method further comprises calculating one or more of a maximum metabolic rate, an optimal metabolite concentration and an enzyme activity by applying a computational optimization method to a kinetic model of a metabolic pathway; a computer readable medium as described above wherein the optimal function is maximizing biomass production; a computer readable medium as described above in which the optimal property is maximal internal yield of ATP; a computer readable medium as described above in which the solvent capacity accounts for ribosome density in the cells in the cell culture as a measure of ribosomal-, enzyme associated-, and non-metabolic-proteins in the cells; a computer readable medium as described above in which the solvent capacity accounts for mitochondria as a subcellular compartment in the cells in the cell culture; a computer readable medium as described above in which the cells are human cells; a computer readable medium as described above in which the cells are cancer cells; and a computer readable medium as described above that comprises a model of a pathway for ATP generation that involves reactions in the serine biosynthesis, one-carbon metabolism and the glycine cleavage system.

According to further embodiments, a device is provided that comprises the computer-readable medium described according to any embodiment herein, and a processor for executing the instructions on the computer-readable medium for achieving an optimal function of a biochemical reaction network in cells in a cell culture. As above, the instructions, when executed by a processor, cause the processor to implement a process implementing a computer model for achieving an optimal function of a biochemical reaction network in cells in a cell culture, comprising: (a) calculating one or more optimal cell culture parameters in a cell culture using a flux balance analysis constrained by a solvent capacity of cells in the cell culture, wherein the optimal cell culture parameter is calculated by determining an optimal property of a biochemical reaction network comprising a list of biochemical reactions by applying a computational optimization method to one or more of the biochemical reactions of the biochemical reaction network, the optimization method comprising (i) altering one or more elements of the one or more biochemical reactions in the biochemical reaction network and re-computing the optimal property, and (ii) repeating (i) until an optimal function is reached; and (b) initiating or maintaining the optimal cell culture parameter in the cell culture to achieve the optimal function of the biochemical reaction network in the cells. As above. The computer-readable medium may include in its various embodiments instructions which, when executed by a processor, cause the processor to implement a process implementing a computer model for achieving any method described in this document, including, without limitation, the following: a computer readable medium as described above in which the optimization method further comprises calculating one or more of a maximum metabolic rate, an optimal metabolite concentration and an enzyme activity by applying a computational optimization method to a kinetic model of a metabolic pathway; a computer readable medium as described above wherein the optimal function is maximizing biomass production; a computer readable medium as described above in which the optimal property is maximal internal yield of ATP; a computer readable medium as described above in which the solvent capacity accounts for ribosome density in the cells in the cell culture as a measure of ribosomal-, enzyme associated-, and non-metabolic-proteins in the cells; a computer readable medium as described above in which the solvent capacity accounts for mitochondria as a subcellular compartment in the cells in the cell culture; a computer readable medium as described above in which the cells are human cells; a computer readable medium as described above in which the cells are cancer cells; and a computer readable medium as described above that comprises a model of a pathway for ATP generation that involves reactions in the serine biosynthesis, one-carbon metabolism and the glycine cleavage system. The device also may comprise additional components for implementation of the instructions, including, without limitation one or more sensors, such as pH, gas (e.g. O2 or CO2), chemical or temperature sensors, one or more culture vessels, one or more heating elements or cooling elements, reservoirs for storing storing cell culture medium, cell culturing ingredients or an analytical composition (e.g., a composition useful in testing a cell culture parameter) and one or more mechanisms for dispensing the storing cell culture medium, cell culturing ingredients or an analytical composition into a cell culture or other container, such as a cuvette, or for taking and/or analyzing samples from one or more cell culture vessels, including, mechanical, fluidic and robotic mechanisms, a display (e.g., a monitor); one or more analytical devices, such as a spectrophotometer for measuring cell density and/or the results of an analytical assay; and one or more communication subsystems for receiving or transmitting data, including, without limitation wired communication devices, such as Ethernet, modem, FIREWIRE, Universal Serial Bus (USB), and THUNDERBOLT devices, or wireless communication devices such as BLUETOOTH or 802.11g or n protocol devices.

## DETAILED DESCRIPTION

All ranges or numerical values stated herein, whether or not preceded by the term “about” unless stated otherwise are considered to be preceded by the term “about” to account for variations in precision of measurement and functionally equivalent ranges.

As used herein, the terms “comprising,” “comprise” or “comprised,” and variations thereof, are meant to be open ended. The terms “a” and “an” are intended to refer to one or more.

Methods for conducting flux balance calculations for cell cultures are provided. In one aspect, cytoplasmic molecular crowding (synonymous with solvent capacity) in cells in a cell culture are used in flux balance calculations. In another aspect, reaction kinetics (e.g., enzyme kinetics) parameters of reactions taking place in cells of the cell culture are used along with molecular crowding considerations to conduct flux balance calculations. The methods provided herein are useful in optimizing cell growth conditions and/or one or more reactions in a cell culture system. The methods are useful in research and commercial applications where it is desirable to optimize cell culture conditions with respect to growth rate of cells (e.g., to maximize biomass production), and/or optimizing certain biochemical reactions (including enzymatic and non-enzymatic biochemical reactions) or activities of cells in a culture system. As an example, for production of biomass, with or without consideration of the production of a certain reaction product in the cell, such as sugars, oils, fatty acids, alcohols, polysaccharides, polymers, etc., cell growth can be maximized by optimizing one or more biochemical reactions in the cell by maintaining cell growth conditions within certain tolerances based on the flux balance calculations described herein. Production of certain metabolites by a cell, such as a wild type or mutant cell, can be optimized by the flux balance analysis methods described herein. As an example, production of fatty acids or ethanol can be optimized for biofuel production. Production of product(s) of a recombinantly-modified cell can be optimized by the methods described herein. Metabolic engineering involves the modification by random mutation or by directed methods, such as by recombinant methods, to optimize the production of certain metabolites or products of cellular metabolism. Production of sugars, alcohols, fatty acids, triglycerides, polymers or subunits thereof, or literally any cellular component, including reaction starting materials, intermediates or products, can be optimized using the methods described herein. The methods described herein provide significantly increased accuracy in flux balance calculations as compared to prior methods that do not consider cytoplasmic molecular crowding by itself or with reaction kinetics of biochemical reactions within the cells.

In a flux balance model of cellular metabolism a cell's metabolic network is mathematically represented by the stoichiometric matrix, Smi, providing the stoichiometric coefficient of metabolite m (m=1, . . . , M) in reaction i (i=1, . . . , N) where M and N are the number of metabolites and reactions, respectively. The cell is assumed to be in a steady state, where the concentration of each intracellular metabolite (other than those that constitute the biomass) remains constant in time. Thus, the stationary reaction rates (fluxes) consuming and producing a metabolite should balance,

\(\begin{matrix}
{{{\sum\limits_{i = 1}^{N}{S_{mi}f_{i}}} = 0},} & (1)
\end{matrix}\)

where, fi denotes the flux of reaction i. The study of the solution space defined by (1) together with maximum capacity constraints for the uptake rates of extracellular substrates constitutes the basis of ‘flux balance analysis’ (FBA).

We extend this framework to consider the physical and spatial constraints resulting from the very high intracellular concentration of macromolecules. Given that the enzyme molecules have a finite molar volume vi we can only fit a finite number of them in a given volume V. Indeed, if ni is the number of moles of the ith enzyme, then

Σi=1Nvini≦V.   (2)

Equation (2) represents a constraint on the enzyme levels ni, potentially affecting their maximum attainable values and relative abundance. Dividing by cell mass M we can reformulate this constraint in terms of the enzyme concentrations Ei=ni/M (moles/unit mass), resulting in

\(\begin{matrix}
{{{\sum\limits_{i = 1}^{N}{v_{i}E_{i}}} \leq \frac{1}{C}},} & (3)
\end{matrix}\)

where C=M/V≈0.34 g/ml is the E. coli cytoplasmic density. Equation (3) imposes a constraint on the maximum attainable enzyme concentrations and, therefore, we refer to it as the enzyme concentration constraint. This constraint is reflected in the metabolic fluxes as well. Indeed, an enzyme concentration Ei results in a flux fi=biEi over reaction i, where the parameter bi is determined by the reaction mechanism, kinetic parameters, and metabolite concentrations. Therefore, the enzyme concentration constraint is reflected in the metabolic flux constraint

Σi=1Naifi≦1,   (4)

where ai=Cvi/bi [Eq. 5], affecting the maximum attainable fluxes and the flux distribution among different metabolic reactions. From here on, we refer to this mathematical framework as ‘flux balance analysis with molecular crowding’ (FBAwMC). Furthermore, since the coefficient ai quantifies the contribution to the overall crowding by reaction i we refer to it as the ‘crowding coefficient of reaction i’, or simply ‘crowding coefficient’.

Accordingly, in one non-limiting aspect, a method of optimizing one or more biological activities (for example and without limitation, a growth rate, production of one or more metabolic products or recombinant proteins) in a cell in a cell culture is provided. The method comprises calculating an optimal cell culture parameter for one or more biological activities in a cell in a cell culture by applying an optimization method to a list of one or more reactions representing or affecting the one or more biological activities, wherein the optimization method uses an intracellular molecular crowding parameter for one or more elements (e.g., enzyme, substrate, cofactor, etc.) of the one or more reactions to calculate an optimal cell culture parameter for the one or more biological activities in the cell; initiating or maintaining the optimal cell culture parameter in a cell culture. A cell culture parameter is a culture condition, for example and without limitation, culture medium composition or physical characteristic of the culture, including without limitation, concentration of, for example, one or more of a carbon source, a nitrogen source, cofactors, salts, buffers, nucleotides, amino acids, oxygen (e.g., pO2) and products and/or byproducts of cellular metabolism, as well as pH, CO2, temperature, cell density, osmolality, viscosity and/or other physical parameter of the culture.

By “initiating or maintaining an optimal cell culture parameter” it is meant that in the culture at its inception, or at any time during cell culture, including at more than one time point, continuously, substantially continuously or for any duration or multiple durations, the “optimal cell culture parameter or parameters are initiate or re-initiated. For instance, at optimal cell culture conditions, a carbon source might be at a specific concentration. The optimal concentration of the carbon source is then periodically tested and, if not at an optimal concentration, additional carbon source is added to the culture to reach the optimal concentration. It should be recognized that the optimal concentration of any given cell culture constituent may change as optimal parameters are re-calculated based on readings of one or more cell culture parameters over time. A person of skill in the art also will recognize that, depending on the culture conditions, the timing, duration and the method of initiating and/or maintaining the cell culture parameter, and/or the method of monitoring the concentration of the metabolite in the culture, the concentration of the metabolite may rise above or below the exact calculated optimal concentration, and that given the practicalities of implementation, a person of skill in the art would recognize that “initiating and maintaining an optimal concentration” means approximating that concentration, as is practicable and acceptable in terms of the desired outcome, such as maximum cell growth or production of a product by the cell.

U.S. Pat. No. 7,127,379 provides one example of how the methods described herein can be implemented (see also, Edwards J. S., et al. (1999) Journal of Biological Chemistry, 274(25):17410 16). In that document, flux calculations are used in the implementation of a method for directed evolution of a cell culture. The present methods, including considerations of cytoplasmic cell crowding and/or reaction kinetics, are not only applicable to the evolutionary methods described in that patent, but more generally to optimizing culture conditions with the goal of attaining an optimal, desired outcome.

The methods described herein and steps thereof are typically of such complexity that they are preferably implemented in a computer, and as such, a biochemical reaction network can be designed for implementation in a computer (in silico), e.g., by instructions executed by one or more processors. According to typical embodiments, the methods described herein, including all or substantially all steps thereof are performed in a computer and are therefore considered to be “computer-implemented”. Following the design of the reaction network, cell growth and/or production of one or more cellular constituents can be optimized for a genetically modified organism or a wild-type strain that corresponds to the network used for the computer simulations. Organisms may achieve the optimal behavior in a non-unique fashion—that is there may be equivalent optimal solutions.

Thus, in one aspect, methods are provided for determining optimal functions of a comprehensive biochemical reaction network in a living cell with respect to one or more functions of a cell. The method can be performed by representing a listing of the biochemical reactions in the network in a computer; using optimization methods to calculate the optimal properties of the network; altering the list of reactions in the network and re-computing the optimal properties; and repeating the altering step until the desired performance is met. A person of skill in the computer programming arts can design such as system using any of a number of available programming languages and software tools available in the computer programming field, including mathematical modeling and/or object-oriented processes as are available.

In addition to in silico steps, according to one non-limiting embodiment, the methods can further include steps involving culturing a living cell, or a population of cells and optimizing culture conditions to optimize cell growth and/or production of a cellular constituent. In an additional embodiment, these steps include constructing the genetic makeup of a cell to contain the biochemical reactions which result from repeating the altering step until the desired performance are met; placing the cell constructed thereunder in culture under the specified environment; and cultivating the cell for a sufficient period of time and under conditions to allow the cell to evolve to the determined desired performance.

A biochemical reaction network is an interrelated series of biochemical reactions that are part of a biochemical pathway or linked biochemical pathways. Many biochemical reaction networks have been identified such as metabolic reaction networks, catabolic reaction networks, polypeptide and nucleic acid synthesis reaction networks, amino acid synthesis networks, energy metabolism and so forth. Other types of biochemical reaction networks include regulatory networks including cell signaling networks, cell cycle networks, genetic networks involved in regulation of gene expression, such as operon regulatory networks, and actin polymerization networks that generate portions of the cytoskeleton. Most of the major cell functions rely on a network of interactive biochemical reactions.

To implement the methods described herein, the reaction structure of a comprehensive, preferably substantially whole, or most preferably whole biochemical reaction network in an organism to be biochemically designed can be reconstructed for computer simulations. A whole biochemical reaction network includes all of the biochemical reactions of a cell related to a certain biochemical function. For example a whole metabolic reaction network includes essentially all of the biochemical reactions that provide the metabolism of a cell. This is made possible with the advent of whole genome sequencing. Biochemical reaction networks have been worked out for a number of organisms, such as E. coli and S. cerevisiae, as shown in the Examples, below. Metabolic reaction networks exemplify a universal biochemical reaction network found in some form in all living cells. A comprehensive biochemical reaction network is an interrelated group of biochemical reactions that affect a detectable property, and that can be modified in a predictable manner with respect to the effect of such modifications on the detectable property in the context of a living cell. For example, a comprehensive biochemical reaction network can include core reactions that effect the yield of a biomolecule produced by the cell, even though the core reactions include only a portion of the reactions in the whole biochemical reaction network involved in yield of the biomolecule, provided that computational methods can be used to predict the effect of changes in the core biochemical reactions on the yield in a living cell. A substantially whole biochemical reaction network is an interrelated group of biochemical reactions that are responsible for a detectable property of a living cell. Substantially whole biochemical reaction networks include core reactions as well as secondary reactions that have an effect on the detectable property, even though this effect can be relatively minor. Changes in substantially whole biochemical reaction networks can be predicted using computational methods. The methods described herein can also utilize the majority of reactions in a whole biochemical reaction network, rather than a comprehensive, substantially whole, or whole biochemical reaction network.

Optimal properties can be determined using the methods described herein include, for example, glycerol uptake rate, oxygen uptake rate, growth rate, sporulation occurrence and/or rates, rates of scouring of rare elements under nutritionally poor conditions, biomass, and yields of biomolecules such as proteins, amino acids, carbohydrates, fatty acids, alcohols, triglycerides, antibiotics, vitamins, amino acids, and fermentation products. Optimal properties also include, for example, yields of chiral compounds and other low molecular weight compounds. Optimal properties also include, for example, the maximal internal yields of key co-factors, such as energy carrying ATP or redox carrying NADPH and NADH. Optimal properties can also be defined by a cellular engineer to include properties such as flux rates through key reactions in the biochemical reaction network. The methods described herein facilitate achievement of an optimal performance related to one or more of the properties to be achieved, such as a target growth rate or yield of a cellular constituent or other product.

For implementation of the methods described herein, biochemical reactions of a reconstructed biochemical reaction network are represented in a computer by a listing of the biochemical reactions in the reconstructed biochemical reaction network. The listing can be represented in a computer database, for example as a series of tables of a relational database, so that it can be interfaced with computer processes (e.g., functions and/or algorithms that are implemented by software and/or hardware) that represent network simulation and calculation of optimal properties.

The biochemical network reconstruction preferably is of high quality. The process of high quality biochemical reaction network, specifically metabolic reaction network, reconstruction has been established (see, e.g., M. W. Covert, et al. Trends in Biochemical Sciences, 26: 179 186 (2001); Edwards J., et al. BMC Structural Biology, 1(2) (2000a); Edwards J. S., et al., Journal of Biological Chemistry, 274(25):17410 16, (1999), Karp P. D. et al. ISMB 4:116 24, (1996); Karp P. D. et al., Nucleic. Acids Res. 28(1):56 59 (2000); Ogata et al. Nucleic Acids Res. 27(1):29 34 (1999); Schilling C. H. et al. J. Theor. Biol., 203(3): 249 83 (2000); Selkov E. Jr. et al., Nucleic Acids Res., 26(1): 43 45 (1998); and Selkov E. et al., Gene 197(1 2):GC11 26 (1997)). This process typically involves the use of annotated genome sequences, and biochemical and physiological data. These annotated genome sequences and biochemical and physiological data can be found in any useful source. Careful analysis of the reconstructed network is needed to reconcile all the data sources used. Similar methods can be used for the reconstruction of other biochemical reaction networks.

As illustrated in the Examples, below, the reconstructed comprehensive, substantially whole, or whole biochemical reaction network can then be used to determine optimal properties of the comprehensive, substantially whole, or whole biochemical reaction network, or portions thereof, under specified and varying environmental conditions. This determination allows the design of a biochemical reaction network that achieves a desired performance in a specified environment. In one embodiment, this can be combined with steps for constructing the genetic makeup of a cell and cultivating the cell to provide a method for developing a recombinant cell, or a population of cells, that achieves the desired performance.

Optimal properties of the comprehensive, substantially whole, or whole biochemical reaction network, or portion thereof, under a series of specified environments can be determined using computational methods known as optimization methods. Optimization methods are known in the art (see e.g., Edwards J. S., et al., Journal of Biological Chemistry, 274(25):17410 16, (1999)). The optimization methods used in the methods described herein utilize linear and/or no-linear optimization with linear constraints. The optimization with respect to the reaction rates is linear and we can utilize any available package for linear optimization with linear constraints to find the optimal reaction rates. The optimization with respect to the metabolite concentrations in non-linear and we can, for example, utilize simulated annealing or other non-linear optimization packages with linear constraints to find the optimal metabolite concentrations.

The reconstructed metabolic network can be used to perform quantitative simulations of the metabolic flux distribution in a steady state using established methods (see, e.g., Bonarius et al., Trends in Biotechnology, 15(8): 308 14 (1997); Edwards J. S., et al., Metabolic flux Balance Analysis, In: (Lee S. Y., Papoutsakis E T., eds.) Metabolic Engineering: Marcel Deker. P 13 57 (1999); and Varma A. et al., Bio/Technology 12:994 98 (1994a)). Computer simulations of the metabolic network can be performed under any conditions. Furthermore, any reaction list can be simulated in a computer by changing the parameters describing the environment and the contents of the reaction list.

The metabolic capabilities of a reconstructed metabolic network can be assessed using the established method of flux balance analysis (FBA) (Bonarius et al., (1997); Edwards et al., (1999); and Varma and Palsson (1994a)). FBA is based on the conservation of mass in the metabolic network in a steady state and capacity constraints (maximal fluxes through the reactions) on individual reactions in the network. Additionally, experimentally determined strain specific parameters are also required, the biomass composition (Pramanik J. et al., Biotechnology and Bioengineering, 56(4): 398 421 (1997)) and the maintenance requirements (Varma A. et al., Applied and Environmental Microbiology, 60(10): 3724 31 (1994b)). In addition to the above parameters, cytoplasmic molecular crowding and/or reaction kinetics are constraints placed on individual reactions in the reaction network. These factors are then used to calculate the flux distribution through the reconstructed metabolic network.

More specifically, the definition of these factors leads mathematically to a closed solution space to the equations in which all feasible solutions lie. There are thus many possible solutions (flux distributions) to any problem. The ‘best’ or optimal solution within the set of all allowable solutions can then be determined using optimization procedures and a stated objective. The optimization procedure used may be linear programming and the objective may be the optimal use of the biochemical reaction network to produce all or some biomass components simultaneously. Non-limiting examples of these optimization procedures are established and have been published (Varma et al. (1994a); Bonarious (1997); and Edwards et al. (1999)). The comparison of the calculated behavior based on the optimal growth objective to the experimental data is favorable in the majority of cases (Varma (1994b); Edwards J. S., et al., Nat Biotechnol., 19(2): 125 30 (2001a); and Edwards, et al. Biotechnol Bioeng. 2002; 77(1):27-36), though, as shown in the examples below, the addition of constraints relating to cytoplasmic molecular crowding and/or reaction kinetics, leads to superior tracking of in silico data to experimental data. In other words, these solution confinement and optimization procedures lead to a prediction of the optimal uses of a biochemical reaction network to support cellular growth and/or desired biological function.

Steady state metabolic flux distributions are mathematically confined to the solution space defined for a given reconstructed metabolic network, where each solution in the solution space corresponds to a particular flux distribution through the network or a particular metabolic phenotype (Edwards and Palsson (1999)). Under a single specified growth condition, the optimal metabolic flux distribution can be determined using linear programming (LP) or other related and/or useful approaches for calculating optimal solutions of such problems.

More specifically, the definition of these factors leads mathematically to a closed solution space to the equations in which all feasible solutions lie. There are thus many possible solutions (flux distributions) to any problem. The ‘best’ or optimal solution within the set of all allowable solutions can then be determined using optimization procedures and a stated objective. The optimization procedure used may be linear programming and the objective may be the use of the biochemical reaction network such as to produce biomass components at a maximal rate. Non-limiting examples of these optimization procedures are established and have been published (Varma and Palsson (1994a); Bonarious (1997); and Edwards et al. (1999)). The comparison of the calculated behavior based on the optimal growth objective to the experimental data is favorable in the majority of cases (Varma (1994b); Edwards J. S., et al. Nat Biotechnol., 19(2): 125 30 (2001a); and Edwards, et al., Biotechnol Bioeng. 2002; 77(1):27-36), though, as shown in the examples below, the addition of constraints relating to cytoplasmic molecular crowding and/or reaction kinetics, leads to superior tracking of in silico data to experimental data. In other words, these solution confinement and optimization procedures lead to a prediction of the optimal uses of a biochemical reaction network to support cellular growth and/or desired biological function.

The metabolic reconstruction is then used to predict the optimal flux distribution within the given constraints. Using the optimization procedure, the properties of the corresponding actual biochemical reaction network may not be optimal or the same as desired from a practical standpoint. The simulated reconstructed network and its synthesis in an organism may not display the optimal solution desired, also referred to herein as the desired optimal performance or desired optimal function. Lack of optimality may be due to the fact that:

The natural organism with an intact network has never experienced the environmental conditions of interest and never undergone growth competition and selection in this environment, or

The wild type network is perturbed from its optimal state by genetic manipulations, through the deletion/addition of a new reaction from/to the network.

The in silico methods described herein are designed to resolve the second cause of lack of optimality, by altering the reactions in the network until a desired performance is achieved. Culturing methods can be used to resolve the first cause of the lack of optimality related to growth competition and selection.

As mentioned above, after calculation of the optimal properties, one or more cellular parameters may be changed, in order to achieve a desired performance, growth, metabolic or production goal. In one instance, a metabolic engineer can alter the reaction list in the network, or an algorithm can be developed that automatically alters one or more reactions in the reaction list to achieve a desired performance. After alteration of one or more growth parameters and/or the biochemical list, optimal properties of this network under given environmental conditions can be calculated. For optimizing performance, this typically involves repeated monitoring of one or more culture parameters, along with repeated calculations based on the latest culture parameters and/or trends. In the context of engineering/evolving a cell, this is an iterative design procedure that may require many different versions of the reaction list until the desired performance is achieved. The desired performance is a qualitative characteristic or quantitative value for a property calculated using an optimization procedure. Many properties for which a desired performance can be achieved are known in the art. For example, a desired performance can be a desired growth rate or a desired yield of a biomolecule such as an enzyme or an antibiotic.

As mentioned above, after calculation of the optimal properties, a metabolic engineer can alter the reaction list in the network, or an algorithm can be developed that automatically alters one or more reactions in the reaction list, to achieve a desired performance. After alteration of the biochemical list, optimal properties of this network under given environmental conditions can be calculated. This is an iterative design procedure that may require many different versions of the reaction list until the desired performance is achieved. The desired performance is a qualitative characteristic or quantitative value for a property calculated using an optimization procedure. Many properties for which a desired performance can be achieved are known in the art. For example, a desired performance can be a desired growth rate or a desired yield of a biomolecule such as an enzyme or an antibiotic.

The optimization method may be carried out using a computer system. The computer system typically includes a processor for executing instructions, a database that provides information regarding one or more biochemical reaction networks of at least one organism; a user interface capable of receiving a selection of one or more biochemical reaction networks for optimization and/or comparison, and capable of receiving a selection of a desired performance; and a process (e.g., software, instructions) for carrying out the optimization method calculations and recalculations. Furthermore, the computer system may include a process for performing biochemical reaction network reconstruction.

The computer system can be a stand-alone computer, a portable computer or PDA, such as a laptop computer or a smart phone, or a conventional network system including a client/server environment and one or more database servers. A number of conventional network systems, including a local area network (LAN) or a wide area network (WAN), and including wireless and/or wired components, are known in the art. Additionally, client/server environments, database servers, and networks are well documented in the technical, trade, and patent literature. For example, the database server can run on an operating system such as UNIX, running a relational database management system, a World Wide Web application, and a World Wide Web Server.

The information in the database may include biomolecular sequence information regarding biomolecules involved in the biochemical reactions of the biochemical reaction network, for example information regarding multiple biomolecular sequences such as genomic sequences. At least some of the genomic sequences can represent open reading frames located along one or more contiguous sequences on each of the genomes of the one or more organisms. The information regarding biochemical reaction networks can include information identifying those biochemical reaction networks to which a biomolecular sequence plays a role and the specific reactions in the biochemical reaction network involving the biomolecule.

The database can include any type of biological sequence information that pertains to biochemical reactions. For example, the database can be a nucleic acid sequence database, including ESTs and/or more preferably full-length sequences, or an amino acid sequence database. The database preferably provides information about a comprehensive, substantially whole, or whole biochemical reaction network. For example, the database can provide information regarding a whole metabolic reaction network. The database can provide nucleic acid and/or amino acid sequences of an entire genome of an organism.

The database can include biochemical and sequence information from any living organism and can be divided into two parts, one for storing sequences and the other for storing information regarding the sequences. For example, the database can provide biochemical reaction information and sequence information for animals (e.g., human, primate, rodent, amphibian, insect, etc.), plants, or microbes. The database may be annotated, such as with information regarding the function, especially the biochemical function, of the biomolecules of the database. The annotations can include information obtained from published reports studying the biochemistry of the biomolecules of the database, such as specific reactions to which a biomolecule is involved, whether the biomolecule is or encodes an enzyme, whether the sequence is a wild-type sequence, etc.

The annotations and sequences of the database can provide sufficient information for a selected biochemical genotype of an organism to be identified. A biochemical genotype is a grouping of all the nucleic acid or amino acid sequences in a selected biochemical process of an organism. For example, a metabolic genotype is a grouping of all the nucleic acid and/or amino acid sequences of proteins involved in metabolism. Methods for identifying metabolic genotypes have been described in the literature (see e.g. Edwards and Palsson 1999).

The database can be a flat file database or a relational database. The database can be an internal database, or an external database that is accessible to users, for example a public biological sequence database, such as GenBank or GenPept. An internal database is a database maintained as a private database, typically maintained behind a firewall, by an enterprise. An external database is located outside an internal database, and is typically maintained by a different entity than an internal database. In one non-limiting embodiment, the methods rely on information stored both in an internal and an external database. In such a system, proprietary information can be maintained in an internal database, but the system also can obtain information from one or more external databases. A number of external public biological sequence databases are available and can be used with the methods described herein. For example, many of the biological sequence databases available from the National Center for Biological Information (NCBI), part of the National Library of Medicine, can be used with the current invention. Other examples of external databases include the Blocks database maintained by the Fred Hutchinson Cancer Research Center in Seattle, and the Swiss-Prot site maintained by the University of Geneva. Additionally, the external databases can include a database providing information regarding biochemical reactions, including databases of published literature references describing and analyzing biochemical reactions. Where a database included in the computer systems of the present invention is a public computer database that does not identify information that is relevant for a particular biochemical reaction network, the computer system either includes a function for performing biochemical reaction network reconstruction, or includes identification of the database entries that pertain to a particular biochemical reaction network. Additionally, there are several databases with biochemical pathway information, these databases include, for non-limiting example, EcoCyc, KEGG, WIT, and EMP. These databases can be used to provide the information to reconstruct the metabolic models.

In addition to the database discussed above, the computer system typically includes a user interface capable of receiving a selection of one or more biochemical reaction networks for optimization and/or comparison, and capable of receiving a selection of an optimal performance. The interface can be a graphic user interface where selections are made using a series of menus, dialog boxes, and/or selectable buttons, for example. The interface typically takes a user through a series of screens beginning with a main screen. The user interface can include links that a user may select to access additional information relating to a biochemical reaction network.

The function of the computer system that carries out the optimization methods typically includes a processing unit that executes a computer program product (process) that includes a computer-readable program code embodied on a computer-readable medium and/or present in a memory function connected to the processing unit. The memory function can be, for example, a disk drive (optical and/or magnetic), Random Access Memory, Read Only Memory, or Flash Memory.

The computer program product, including instructions, that is read and executed by a processing unit of a computer system, includes modules (sub-programs) and processes/functions, such as a computer-readable program code embodied on a computer-readable medium for implementing aspects of the computer program, including, without limitation, functions and algorithms. The program code is capable of interacting with a database, for example, as described above, and typically effects the following steps within the computing system: providing an interface for receiving a selection of a desired performance of the networks; determining the desired optimal properties, displaying the results of the determination, and altering the biochemical reaction network, before recalculating optimal properties of the biochemical reaction network, and repeating the process until a desired performance is achieved. Altering the biochemical reaction network can be performed based on an alteration identified by a user, or can be performed automatically by the program code. The computer program can further provide an identification of database entries that are part of a reconstructed biochemical network, or can perform biochemical reaction network reconstruction. Furthermore, the computer program can provide a function for comparing biochemical reaction networks to identify differences in components and properties. Lastly, the computer program may comprise a process or module for receiving data obtained from a cell culture (automatically or manually) and a process or module for controlling one or more external devices, such as a solenoid valve, for controlling a rate of input into a culture of one or more culture medium constituents, such as water, sugars, oxygen, carbon dioxide, buffers, chelating compounds, etc., or for controlling other physical or chemical parameters of the culture, such as a heating device or stirrer.

The computer-readable program code can be generated, for example, using any well-known compiler that is compatible with a programming language used to write software for carrying out the methods of the current invention. Many programming languages and software programs are known that can be used to write software to perform the computational methods described herein, such as, for example and without limitation MATLAB® (The Mathworks, Inc., Natick, Mass.), or similar software.

As mentioned above, an aspect of the methods described herein can further include steps that involve adaptive evolution of a cultured strain to achieve a desired performance. Virtually any cell can be used with the methods described herein including, for example, a prokaryotic cell, or a eukaryotic cell such as a fungal cell or an animal cell including a cell of an animal cell line. However, a biochemical reaction network of the cell, or the cell of a closely related organism, must be sufficiently characterized to allow a high quality reconstruction of the comprehensive, substantially whole, and/or whole biochemical reaction network in a computer. Preferably, essentially the entire genome of the organism has been sequenced and genes encoding biomolecules, typically proteins, involved in the biochemical reaction network have been identified.

The genetic makeup of a cell can be constructed to contain the biochemical reactions that meet the desired performance to produce a cell with a potential to meet the desired performance. This can be achieved using the indigenous list of reactions in the cell and by adding and subtracting reactions from this list using genetic manipulations to achieve the reaction list capable of achieving the desired performance criteria, identified by the steps performed in silico described above. For example, reactions can be added or subtracted from the list by adding, changing, or deleting all or portions of one or more genes encoding one or more biomolecules involved in the reaction, for example by adding, changing, or deleting protein coding regions of one or more genes or by adding, changing, or deleting regulatory regions of one or more genes. In addition, for example, reactions can be added or subtracted from the list by altering expression of regulatory components (e.g., transcription factors) that effect the expression of one or more biomolecules involved in one or more reactions of the reaction list. The resulting engineered cell may or may not display the optimal properties calculated ahead of time by the in silico methods using the iterative optimization procedure described above.

After a cell has been identified or constructed to have a potential to meet the desired performance, it is placed in culture under a specified environment. The specified environment is determined during the optimization procedure. That is, the optimization procedure calculates properties of the network under various environments, as described above, and identifies the specified environment in which the desired performance is achieved.

In one aspect of the methods described herein, a cell culture is optimized to produce a desired growth rate and/or biomass accumulation, or to produce a specific end-product or products, such as a fatty acid, methanol or ethanol for biofuel production, antibiotic, recombinant protein, sugar, polymer, etc. At the inception of a culture, the methods and computational processes described herein are used to determine optimal cultural parameters, such as optimal metabolite concentration. For instance, glucose may be optimally present in a certain concentration in the culture media in order to optimize growth rates without, for example, harmful accumulation of by-products. In another embodiment, an amino acid may be added to the culture medium, or a culture medium constituent may be limited in order to maximize production of a protein, such as a recombinant protein (for instance, too rapid production of a protein may be counter-productive, resulting in cytoplasmic crowding by the over-produced protein, preventing optimal metabolic activity of the cell and/or optimal function of the cellular processes involved in making the protein, such as limitation of inducers or necessary transcription factors). Culture conditions, such as glucose concentration, pO2, pH, etc., may be monitored occasionally, at fixed time periods, or continuously, and the data input either automatically or manually into a computer program to determine if optimal conditions for the desired outcome are present and, optionally, to determine steps, such as the addition of sugars, CO2, O2, buffers, chelating agents, water, new media, etc., necessary to bring the culture to, or back to, optimal conditions. The computer system may be configured (that is, comprises necessary software, hardware and peripheral components), to automatically bring the cell culture to optimal parameters by automatically adding or removing materials from the cell culture to optimize a desired outcome.

In one example, the cell culture is a continuous culture and a computerized system is used to monitor culture parameters, including, for example and without limitation, temperature, sugar (or other carbon and/or nitrogen source) concentration, pH, pO2, and/or any other relevant parameter or indicator of culture status. The computerized system also contains hardware and computer processes for controlling flow of new culture media into the cell culture; flow of culture media (containing biomass) out of the culture and the addition of one or more culture constituents, such as a carbon source, a nitrogen source, a buffer, O2 or CO2, and/or an amino acid(s) to the culture. A person of skill in the art can readily configure and program such as system.

According to another non-limiting embodiment of the methods described herein, in which a cell is required to adapt to culture conditions, the cells are cultured for a sufficient period of time and under conditions to allow the cells to evolve to the desired performance. That is, adaptive evolution of natural or engineered strains can be carried out as guided by the general optimization methods or procedures. Natural strains that have not experienced a particular environment or genetically altered strains can be analyzed by the network reconstruction and optimization procedures disclosed above. These strains can then be put under a selection pressure consistent with the desired function of the organisms and evolved towards the predetermined performance characteristics. The cells may achieve the desired performance without additional adaptive evolution. That is, the sufficient period of time for culturing the cells, may be immediately after the genetic makeup of the cells is constructed using the methods described herein without the need for further adaptive evolution.

In other words, extended cultivation of a non-optimal or non-evolved strain can be performed to optimize or evolve the metabolic network toward the optimal solution that is achievable under the defined environmental conditions. The practice of this evolutionary process requires on the order of weeks to years to optimize a metabolic network depending on how far it is from the optimal conditions at the beginning of the evolutionary process, and how difficult it is to achieve the necessary changes through random mutation and shifts in regulation of gene expression. This process can be accelerated by the use of chemical mutagens and/or radiation. Additionally, the process can be accelerated by genetically altering the living cell so that it contains the biochemical reactants determined by the in silico method described above, that achieve a desired performance.

Methods are known in the art for culturing cells under specified environmental conditions. For example, if the cell is E. coli, and the desired performance is a desired growth rate, the procedure set out below can be used. This procedure can be readily adapted for use with other bacterial cells and/or other performance criteria. Additionally, the procedures can be readily developed for use with other cell types such as animal cells. For example, the methods can be readily adapted for use with other culturing systems, such as large scales systems in which cells adhere to a culturing vessel. The culturing methods may be adapted for high-throughput analysis, as known in the art.

If a strain needs to be directionally evolved to achieve the desired performance, then following the construction of the metabolic reaction network in the chosen host strain, the cells are typically stored frozen at −80° C. with 30% glycerol. For each adaptive evolutionary process, frozen stocks can be plated on LB agar and grown overnight at 37° C. From the plate, individual colonies can be identified that arose from a single cell. An individual colony can be used to inoculate a liquid culture, known as a pre-culture. Pre-cultures inoculated from a single colony of the respective strain are grown overnight in the defined medium for the subsequent evolutionary process. A pre-culture sample is taken the following day, typically at mid-log phase (in the middle of logarithmic growth) of growth to inoculate the culture conditions that define the environment that the adaptive evolution is to take place. Batch bioreactors or other suitable culture vessels are then initiated. This, typically would be done at 250 mL volumes in micro-carrier spinner flasks inside a temperature controlled incubator on top of a magnetic stir plate, set at suitable, typically high, speed to ensure sufficient aeration and at the optimal growth temperature (37° C. for wild type E. coli) for any given strain. Other frequently used cultivation procedures known in the art can also be used.

After a suitable time period, typically the following day for E. coli (before the culture reaches stationary phase), an aliquot of the culture now in mid-log phase is serially transferred to a new spinner flask containing fresh medium. If the culture is being optimized for growth rate, stationary phase must be avoided to ensure that the selection criterion is growth rate. Then serial transfers are performed at fixed time intervals (typically every 24 hours depending on the growth rate) at mid-log phase and the volume of the inoculum into the new culture vessel can be adjusted accordingly based on the increase in growth rate.

Growth rate is thus monitored frequently, typically on a daily basis in order to determine the proper volume of the inoculum to use for the next serial transfer. This serial cultivation process is repeated sufficiently often to allow the cells to evolve towards its optimal achievable growth under the conditions specified through the medium composition.

The growth and metabolic behavior typically is monitored during the adaptive evolutionary process to determine how the population is evolving over time. At fixed time intervals, typically every few days, the culture is tested for metabolic and growth behavior, by measuring the oxygen uptake rate, substrate uptake rate and the growth rate. The results are then plotted as a data point on the phenotype phase plane. Movement of the so-determined data point towards the line of optimality would indicate evolution towards optimal growth behavior. These measurements of the membrane transport fluxes along with the growth rate are repeated until the cells are observed to be operating their metabolic network such that the data points lie at the optimal conditions. The evolutionary process can then be continued until there is no further increase in the optimal performance, e.g., growth rate. If no further change is observed, then maximal growth rate has been achieved for the given conditions.

Byproduct secretion can be monitored by HPLC or other suitable methods of analytical chemistry to assess changes in metabolism that are implicated in the evolution towards optimal growth behavior. For these studies it typically is imperative to determine a correlation of dry weight vs optical density for the evolved strain since this will be different from the wild type. In addition to monitoring the growth rate and steady state growth, the cultures are inspected for any signs of possible contamination or co-evolution with a mutant subpopulation. Aliquots for each day of culture are kept refrigerated as a backup in the event of any contamination, and the phenotype of the culture is ascertained by plating samples of the culture and inspecting for any differences in colony morphology or different mutants. On a daily basis, the optical density of the culture, time of inoculation, inoculum volume, growth rate, and any signs of contamination, can be logged. Samples are also frozen at −80° C. in 30% glycerol for each day of culture for any possible further use.

According to a further embodiment, provided herein is a computer-implemented method for achieving an optimal function of a biochemical reaction network in cells in a cell culture, including prokaryotic or eukaryotic cells, for example and without limitation, bacterial, fungal (including yeast), mammalian and cancer cells. The method comprises (a) calculating in a computer one or more optimal cell culture parameters in a cell culture using a flux balance analysis constrained by a solvent capacity of cells in the cell culture, wherein the optimal cell culture parameter is calculated by determining an optimal property of a biochemical reaction network comprising a list of biochemical reactions by applying a computational optimization method to one or more of the biochemical reactions of the biochemical reaction network, the optimization method comprising (i) altering one or more elements of the one or more biochemical reactions in the biochemical reaction network and re-computing the optimal property, and (ii) repeating (i) until an optimal function is reached; and (b) initiating or maintaining the optimal cell culture parameter in the cell culture to achieve the optimal function of the biochemical reaction network in the cells. According to one embodiment, the optimization method further comprises calculating one or more of a maximum metabolic rate, an optimal metabolite concentration and an enzyme activity by applying a computational optimization method to a kinetic model of a metabolic pathway. In an example of an implementation of the method, the method further comprises culturing a cell under culture conditions that favor achievement of the optimal function. Optionally, the method further comprises: (c) constructing the genetic makeup of a cell to contain the biochemical reactions (d) placing the cell constructed under (c) in culture under a specified environment to obtain a population of cells; and (e) cultivating the cells as in step (d) for a sufficient period of time and under conditions to allow the cells to evolve to the desired optimal function determined under (a), wherein the biochemical reaction network comprises a comprehensive biochemical reaction network. In one embodiment, the optimal function is maximizing biomass production. In another, the optimal property is maximal internal yield of ATP. In a further embodiment, the solvent capacity accounts for ribosome density in the cells in the cell culture as a measure of ribosomal-, enzyme associated-, and non-metabolic proteins in the cells or accounts for mitochondria as a subcellular compartment in the cells in the cell culture. According to one embodiment, the method comprises a model of a pathway for ATP generation that involves reactions in the serine biosynthesis, one-carbon metabolism and the glycine cleavage system.

According to another embodiment, also provided is a computer readable medium having stored thereon instructions which, when executed by a processor, cause the processor to implement a process implementing a computer model for achieving an optimal function of a biochemical reaction network in cells in a cell culture. The process comprises: (a) calculating one or more optimal cell culture parameters in a cell culture using a flux balance analysis constrained by a solvent capacity of cells in the cell culture, wherein the optimal cell culture parameter is calculated by determining an optimal property of a biochemical reaction network comprising a list of biochemical reactions by applying a computational optimization method to one or more of the biochemical reactions of the biochemical reaction network, the optimization method comprising (i) altering one or more elements of the one or more biochemical reactions in the biochemical reaction network and re-computing the optimal property, and (ii) repeating (i) until an optimal function is reached; and (b) initiating or maintaining the optimal cell culture parameter in the cell culture to achieve the optimal function of the biochemical reaction network in the cells. The computer-readable medium may include in its various embodiments instructions which, when executed by a processor, cause the processor to implement a process implementing a computer model for achieving any method described in this document, including, without limitation, the following: a computer readable medium as described above in which the optimization method further comprises calculating one or more of a maximum metabolic rate, an optimal metabolite concentration and an enzyme activity by applying a computational optimization method to a kinetic model of a metabolic pathway; a computer readable medium as described above wherein the optimal function is maximizing biomass production; a computer readable medium as described above in which the optimal property is maximal internal yield of ATP; a computer readable medium as described above in which the solvent capacity accounts for ribosome density in the cells in the cell culture as a measure of ribosomal-, enzyme associated-, and non-metabolic-proteins in the cells; a computer readable medium as described above in which the solvent capacity accounts for mitochondria as a subcellular compartment in the cells in the cell culture; a computer readable medium as described above in which the cells are human cells; a computer readable medium as described above in which the cells are cancer cells; and a computer readable medium as described above that comprises a model of a pathway for ATP generation that involves reactions in the serine biosynthesis, one-carbon metabolism and the glycine cleavage system.

According to further embodiments, a device is provided that comprises the computer-readable medium described according to any embodiment herein, and a processor for executing the instructions on the computer-readable medium for achieving an optimal function of a biochemical reaction network in cells in a cell culture. As above, the instructions, when executed by a processor, cause the processor to implement a process implementing a computer model for achieving an optimal function of a biochemical reaction network in cells in a cell culture, comprising: (a) calculating one or more optimal cell culture parameters in a cell culture using a flux balance analysis constrained by a solvent capacity of cells in the cell culture, wherein the optimal cell culture parameter is calculated by determining an optimal property of a biochemical reaction network comprising a list of biochemical reactions by applying a computational optimization method to one or more of the biochemical reactions of the biochemical reaction network, the optimization method comprising (i) altering one or more elements of the one or more biochemical reactions in the biochemical reaction network and re-computing the optimal property, and (ii) repeating (i) until an optimal function is reached; and (b) initiating or maintaining the optimal cell culture parameter in the cell culture to achieve the optimal function of the biochemical reaction network in the cells. As above, The computer-readable medium may include in its various embodiments instructions which, when executed by a processor, cause the processor to implement a process implementing a computer model for achieving any method described in this document, including, without limitation, the following: a computer readable medium as described above in which the optimization method further comprises calculating one or more of a maximum metabolic rate, an optimal metabolite concentration and an enzyme activity by applying a computational optimization method to a kinetic model of a metabolic pathway; a computer readable medium as described above wherein the optimal function is maximizing biomass production; a computer readable medium as described above in which the optimal property is maximal internal yield of ATP; a computer readable medium as described above in which the solvent capacity accounts for ribosome density in the cells in the cell culture as a measure of ribosomal-, enzyme associated-, and non-metabolic-proteins in the cells; a computer readable medium as described above in which the solvent capacity accounts for mitochondria as a subcellular compartment in the cells in the cell culture; a computer readable medium as described above in which the cells are human cells; a computer readable medium as described above in which the cells are cancer cells; and a computer readable medium as described above that comprises a model of a pathway for ATP generation that involves reactions in the serine biosynthesis, one-carbon metabolism and the glycine cleavage system. The device also may comprise additional components for implementation of the instructions, including, without limitation one or more sensors, such as pH, gas (e.g. O2 or CO2), chemical or temperature sensors, one or more culture vessels, one or more heating elements or cooling elements, reservoirs for storing storing cell culture medium, cell culturing ingredients or an analytical composition (e.g., a composition useful in testing a cell culture parameter) and one or more mechanisms for dispensing the storing cell culture medium, cell culturing ingredients or an analytical composition into a cell culture or other container, such as a cuvette, or for taking and/or analyzing samples from one or more cell culture vessels, including, mechanical, fluidic and robotic mechanisms, a display (e.g., a monitor); one or more analytical devices, such as a spectrophotometer for measuring cell density and/or the results of an analytical assay; and one or more communication subsystems for receiving or transmitting data, including, without limitation wired communication devices, such as Ethernet, modem, FIREWIRE, Universal Serial Bus (USB), and THUNDERBOLT devices, or wireless communication devices such as BLUETOOTH or 802.11g or n protocol devices. The computer-readable medium may be located in a device comprising the processor or in communication with a device, such as with one or more external storage media. As would be apparent to those of ordinary skill one or more aspects of the integrated system, such as reaction parameters, might be located remotely and downloaded or otherwise integrated into the system of implementing the methods described herein.

In one embodiment of the implementation of the methods described herein, the methods and computer-readable media are configured to be executable within a device which comprises a processor and a computer-readable medium as part of a computer subsystem of the device. As indicated above, the device may comprise any of the innumerable components for culturing cells, analyzing cells, and otherwise processing cells, including, without limitation, mechanical, fluidic, and robotic components useful for implementing the methods described herein—as would be readily apparent to one of ordinary skill in the art. The processor executes instructions for implementing the method, namely an in silico model as described herein. The model comprises according to one non-limiting embodiment, an intracellular space and an extracellular space. The intracellular space comprises steric (volume, spatial or molecular crowding) limitations attributable to cellular components, including without limitation: organelles, cytoskeleton, and other intracellular structures within the subcellular compartment, such as nucleus, mitochondria, microtubules, microfilaments, intermediate filaments, and further comprises enzymes and enzyme structures such as ribosomes as well as other constituents, including sugars, metabolites, and nucleic acids. Components in the intracellular space have one or more size (spatial constraint, e.g., volume) and/or kinetic parameters associated with them, and the components interact within the model according to those size and/or kinetic parameters. The extracellular compartment may include such influences as waste accumulation, temperature, pH, extracellular compositions, such as sugars, waste materials, buffers, cytokines, growth factors, extracellular matrix and/or cell growth scaffold, cell wall, cell culture vessel characteristics, etc., each of which can be assigned a kinetic relationship with the modeled cells and the constituents thereof.

The following examples are provided as non-limiting examples of the invention described herein.

## EXAMPLES

The following is a table of abbreviations used in this example and throughout.

### Example 1

Here we study the impact of the limited solvent capacity on E. coli cell metabolism at different physiological growth conditions. We demonstrate that this constraint is relevant for fast growing cells, and predict the existence of a metabolic switch between cells growing at low and high nutrient abundance, respectively. We carry out flux measurements of several reactions in the E. coli central metabolism, observing a partial agreement with the model predictions. Moreover, to uncover the regulatory mechanisms that control the changes in flux rates, we perform gene expression and enzyme activity measurements, finding that the switch is controlled predominantly at the enzyme activity level implemented by changes in the activity of a few key enzymes in the E. coli central metabolism. Finally, we discuss the potential relevance of the limited solvent capacity constraint to experimental observations in other organisms.

Estimation of crowding coefficients: The E. coli intracellular density is C=0.34 g/ml. The specific volume was estimated for several proteins using the molar volumes and masses reported in Lee B, Proc Natl Acad Sci USA 1983, 80:622-626, resulting in average of 0.73 ml/g and standard deviation of 0.02 ml/g. The enzymes' turnover rates were obtained from the BRENDAdatabase (Schomburg I, et al. Nucleic Acids Res 2002, 30:47-49) for 102 E. coli enzymes.

Metabolic flux predictions: The Flux Balance analysis with Molecular Crowding is implemented by solving the following optimization problem: maximize the biomass production rate subject to the constraints: balance in the production and consumption of each metabolite (flux balance), the maximum capacity constraint for the carbon source uptake rate and the solvent capacity constraint (Eq. 3). After expressing the reaction's stoichiometric coefficients in units of mol/dry biomass, the maximum growth rate corresponds to the biomass production rate, where biomass production is an auxiliary reaction containing as substrates the cellular components in their relative concentrations and as product the cell's biomass. The crowding coefficients were modeled as noise, assigning them randomly selected values from a list of estimated values for about 700 E. coli enzymes. The predictions for all fluxes were calculated (not shown)

Following the same procedure as for glucose, we also made predictions for the E. coli metabolic fluxes when growing on glycerol, lactate and succinate. The increase of the carbon source concentration in the growth medium was modeled as an increase of the maximum capacity of the corresponding carbon source uptake flux. For each maximum capacity we computed the fluxes that maximize the biomass production rate, obtaining a prediction for the optimal flux of all reactions and the optimal biomass production rate. Because the biomass production rate equals the growth rate, using these predictions we can analyze the behavior of metabolic fluxes as a function of the growth rate.

Bacterial strain and general growth conditions: The E. coli K12 strain MG1655 (F−λ− ilvG rfb50 rph1) was used throughout the work. In order to obtain biomass samples for flux measurements, 20-ml of the overnight grown culture (˜8-10 h) of wild-type cells in LB-medium was inoculated in 980-ml M9 minimal medium (Sigma) containing 2 g/L glucose, where 90% was natural glucose and the remaining 10% was labeled glucose [1,2-13C2] glucose (with >99% purity and 99% isotope enrichment for each position, [Cambridge Isotope Laboratories, Andover, Mass.]). Cells were grown in a continuous growth mode at 5 different dilution rates (0.1, 0.25, 0.4, 0.55, and 0.72 h−1) in a Labfors bioreactor (Infors, Switzerland). The growth of the bacterial culture was regularly monitored at A600 nm to document steady state at all dilution rates. The dissolved oxygen was set at 100% initial value, and sterile air was continuously sparged into the medium. Growth parameters, such as pO2, pH, temperature (37° C.) and agitation (˜400 rpm) were continuously monitored using microprocessor probes. The pH of the medium was constant around 7.0 and was controlled with regular adjustments by automatic supply of acid (10% H3PO4) and base (2N NaOH) using two peristaltic pumps. For determining intracellular metabolic enzyme activities and global transcriptome profiles, the bacterium was grown under similar conditions (except only natural glucose was used as source of carbon) in three separate experiments, and biomass samples were collected at all five-dilution rates. Biomass samples for intracellular enzyme activity, gene expression and flux determination were harvested at the end of each major dilution rates indicated by constant A600 nm (optical densities) and pO2 concentrations in the growth medium.

Metabolic enzyme activity assays: 30-ml samples for enzyme assays were collected for various dilution rates at the end of each dilution rate. The cell pellets for enzyme assays were harvested by centrifugation at 4,000×g at 4° C. for 10 min. These cell pellets were re-suspended and washed in 100 mM Tris-HCl (pH 7.0) sonication buffer containing 20 mM KCl, 5 mM MnSO4, 2 mM DTT and 0.1 mM EDTA. The cells were disrupted by 3 sonication cycles of 30 sec each in a sonicator (Fisher Scientific) to recover maximum possible yield of enzyme. The cell debris was removed by centrifugation and the resulting cell extract (supernatant) was immediately used for enzyme assays or stored at −20° C. All operations were carried out on ice. The supernatant of this sample was used to determine total protein concentration in enzyme samples using standard Bradford's assay (Biorad, Richmond, Calif.). This sample was also used for estimation of quantitative assay of endogenous enzymes. The method of continuous spectrophotometric rate determination with time was followed for measurement of activities of the enzymes of metabolic pathways. All the enzyme assays were done at 30° C. in a thermostatically controlled UV/Vis spectrophotometer (Cary 500) with 1-cm light path. All components the reaction mixture and respective substrates were pipetted out in a quartz cuvette (Fisher Scientific) and blanks were adjusted. Reactions were initiated by adding supernatant from the sonicated enzyme samples to give a final volume of 1 ml. The millimolar extinction coefficients for NAD+, NADH, NADP+ and NADPH was 6.22 cm−1 mM−1 at 340 nm, and those of methyl viologen and benzyl viologen at 578 nm was 9.78 and 8.65 cm−1.mM−1, respectively, whereas the millimolar extinction coefficient value for 5-mercapto-2-nitrobenzoic acid at 412 nm was 13.6 cm−1 mM−1. For all enzyme assays described below, we define 1 unit of enzyme as the amount of enzyme required to convert 1 μmol of substrate into specific products per minute per milligram of protein under defined conditions of pH and temperature. Most enzyme assay protocols used were the standard assay protocols from Sigma (St. Louis, Mo., USA), expect few, which were obtained from the published literature (Peng, L. et al. Appl Microbiol Biotechnol 61, 163-78; Van der Werf, M. J., et al., Arch Microbiol 167, 332-342; and Zhao, J., et al., (2004) Appl Microbiol Biotechnol 64, 91-98.).

The assay conditions for various enzymes were as follows (the gene names are listed in parenthesis against the enzyme name): Glucose:PEP phosphotransferase (pts): 0.1 M Tris-HCl (pH 8.4), 10 mM MgCl2, 1 mM DTT, 1 mM NADP+, 10 mM -glucose, 3 U glucose-6-phosphate dehydrogenase, 10 mM PEP. Phosphofructose kinase (pfkA): 50 mM imidazol-HCl (pH 7.0), 0.05 mM ATP, 5 mM MgCl2, 1 mM EDTA, 0.25 mM NADH, 0.25 mM fructose-6-phosphate (F6P), 0.5 U aldolase, 0.5 U glyceraldehyde phosphate dehydrogenase, 0.5 U triose phosphateisomerase. Fructose-1,6-bisphosphate aldolase (fbaA): 0.05 M Tris-HCl (pH 7.5), 0.1 mM cysteine-HCl, 0.1 M potassium acetate, 2 mM FDP, 0.7 mM CoCl2, 0.25 mM NADH, 20 U triose phosphate isomerase, 2 U glycerol-3-phosphate dehydrogenase. Glyceraldehyde-3-phosphate dehydrogenase (gapA): 0.1 mM Tricine-HCl (pH 8.1), 5 mM potassium phosphate, 20 mM neutralized sodium arsenate, 2 mM FDP, 2 mM DTT, 1 mM NAD+, 1 U aldolase. Triosephosphate isomerase (tpiA): 300 mM triethanolamine buffer (pH 7.8), 0.2 mM NADH, 1 U glycerolphosphate dehydrogenase, 5 mM glyceraldehyde-3-phosphate. Phosphoglycerate kinase (pgk): 0.1 M triethanolamine buffer (pH 7.8), 1 mM EDTA, 2 mM MgSO4.7H2O, 1 U glyceraldehyde-phosphate dehydrogenase, 1 mM ATP, 10 mM 3-phosphoglycerate. Pyruvate kinase (pykA): 0.1 M Tris-HCl (pH 7.5), 5 mM ADP, 1 mM DTT, 10 mM KCl, 15 mM MgCl2, 0.5 mM phosphoenol pyruvate, 0.25 mM NADH, 10 U lactate dehydrogenase. α-Ketoglutarate dehydrogenase (sucA): 0.2 M phosphate buffer (pH 7.2), 1 mM CoASH, 0.1 M cysteine-HCl (pH 7.2), 10 mM NAD+ (pH 7.2), 3 mM α-ketoglutarate. Fumarase (fumA): (assay based on formation of fumarate at 240 nm) 0.1 M Tris-HCl buffer (pH 7.2), 50 mM L-malate. Malate dehydrogenase (mdh): 2.5 ml 0.1 M Tris-HCl (pH 8.8), 0.1 ml 0.1 mM sodium malate, 0.1 ml 10 mM NAD+ and cell extract, and water to a final volume of 3 ml. Phosphotransacetylase (pta): 0.1 M Tris-HCl (pH 7.8), 0.2 mM CoA, 30 mM NH4Cl, 1.0 mM DTT, 1.0 mM NAD+, 5.0 mM -malate, 4 U citrate synthase, 20 U malate dehydrogenase, 2.0 mM acetyl phosphate. Acetate kinase (ackA): 0.1 M Tris-HCl (pH 7.4), 0.8 M potassium acetate, 0.2 M KCl, 4 mM ATP, 4 mM MgCl2, 1.6 mM PEP, 0.4 mM NADH, 4 U pyruvate kinase, 55 U lactate dehydrogenase; Citrate synthase (gltA): 100 mM Tris-HCl (pH 8.0), 8 mM Acetyl CoA, 10 mM sodium oxaloacetate, 10 mM 5,5′-dithiobios-2-notrobenzoate; Glucose-6-phosphate dehydrogenase (zwf): 100 mM Tris-HCl (pH 7.5), 2.5 mM MnCl2, 2 mM glucose-6-phosphate, 1 mM DTT, 1 mM NADP+; PEP carboxylase (ppc): 66 mM Tris-HCl (pH 9.0), 10 mM MgCl2, 10 mM sodium bicarbonate, 0.15 mM NADH, 2 U malate dehydrogenase, 5 mM phosphoenolpyruvate; Pyruvate dehydrogenase (aceE): 100 mM Tris-acetate (pH 7.8), 5 mM pyruvate, 0.1 mM CoA, 7 mM sodium arsenate, 2 mM methyl viologen; Enolase (eno): 100 mM Triethalomine buffer (pH 7.4), 5.6 mM phosphoglycerate, 0.35 mM β-NADH, 75 mM MgSO4, 300 mM KCl, 1 mM ADP, 0.1 ml solution of 15 mM Tris-HCl (pH 7.4) mixed with 0.2% BSA. Phosphoglucose isomerase (pgi): 100 mM Tris-HCl (pH 7.8), 10 mM MgCl2, 0.5 mM NADP+, 10 U glucose-6-phosphate dehydrogenase, 2 mM fructose-6-phosphate.

The following equation was used for calculating enzyme activities for most of the enzyme (unless specified)

\({{Units}\text{/}{ml}\mspace{14mu} {of}\mspace{14mu} {enzyme}} = {\frac{{\Delta \; {A_{340{nm}}/\min}\mspace{11mu} ({Test})} - {\Delta \; {A_{340{nm}}/\min}\mspace{11mu} ({Blank})}}{{E.C.} \times V_{e}} \times V_{a} \times {DF}}\)

where, Va: total volume (ml) of assay; DF: Dilution factor; E.C.: millimolar extinction coefficient of NAD+, NADH, NADP+ or NADPH at 340 nm was 6.22 cm−1.mM−1, and those of methyl viologen and benzyl viologen at 578 nm was 9.78 and 8.65 cm−1.mM−1, respectively, whereas it was 13.6 cm−1.mM−1 for 5-mercapto-2-nitrobenzoic acid at 412 nm. Ve: Volume of enzyme

\({{Units}\text{/}{mg}\mspace{14mu} {protein}} = \frac{{Units}\text{/}{ml}\mspace{14mu} {enzyme}}{{mg}\mspace{14mu} {protein}\text{/}{ml}}\)

Supernatants of the samples were also used for determining total protein concentrations using standard Bradford's assay (BioRad).

Flux measurements and analyses: For flux analysis, biomass (from −100 ml culture) and supernatant samples were collected at all five dilution rates. These samples were immediately flash frozen in liquid nitrogen and stored at −80° C. until further analysis. Flux rates were determined using a tracer-substrate based GC-MS and NMR metabolome mapping platform. The analyses included determining positional 13C tracer enrichment in multiple intermediary metabolites of glycolysis, glycogen synthesis, tricarboxylic acid cycle and their intracellular products from [1,2-13C2]-D-glucose, as described in detail below. The retention times and mass-to-charge (m/z) ion clusters of selected ions of bacterial and culture media metabolites were determined using mass isotopomer analysis (MIDA) (See, e.g., Lee W N, et al. Anal Biochem 1995, 226:100-112 and Lee W N et al., Dev Neurosci 1996, 18:469-477), and expressed as net fluxes by subtracting reverse fluxes from forward tracer incorporation patterns via reversible metabolic steps (Xu J, et al., Anal Biochem 2003, 315:238-246). Results were expressed as mmol/hr/g dry biomass glucose. Each experiment was carried out using triplicate cell cultures for each condition within each experiment, and the experiments were repeated once. Mass spectroscopic analyses were carried out by three independent automatic injections of 1 μl samples by the automatic sampler and accepted only if the standard sample deviation was less than 1% of the normalized peak intensity. Statistical analysis was performed using the Student's t-test for unpaired samples. Two-tailed significance at the 99% confidence interval (μ+/−2.58σ), P<0.01 indicated significant differences in glucose derived fluxes. For some reversible reactions, we measured both forward and reverse fluxes and calculated net fluxes towards product synthesis.

Glycogen glucose and RNA ribose stable isotope studies: RNA ribose and glycogen glucose were isolated by acid hydrolysis of cellular RNA after Trizol purification of cell extracts. Total RNA amounts were assessed by spectrophotometric determination, in triplicate cultures. Ribose and glycogen glucose were derivatized to their aldonitrile acetate form using hydroxylamine in pyridine with acetic anhydride (Supelco, Bellefonte, Pa.) before mass spectral analyses. We monitored the ion cluster around the m/z256 (carbons 1-5 of ribose) (chemical ionization, CI) and m/z217 (carbons 3-5 of ribose) and m/z242 (carbons 1-4 of ribose) (electron impact ionization, EI) to determine molar enrichment and the positional distribution of 13C in ribose. For glycogen glucose we monitored m/z327-332 using CI. By convention, the base mass of 12C-compounds (with their derivatization agents) is given as m0 as measured by mass spectrometry as described elsewhere (Boros L G, et al., (2002) Drug Discov Today 7: 364-372). Ribose or glucose molecules labeled with a single 13C atom on the first carbon position (m1) recovered from RNA or glycogen, respectively, were used to gauge the ribose fraction produced by direct oxidation of glucose through the G6PD pathway. Ribose molecules labeled with 13C on the first two carbon positions (m2) were used to measure the fraction produced by transketolase. Doubly labeled ribose molecules (m2 and m4) on the fourth and fifth carbon positions were used to measure molar fraction produced by triose phosphate isomerase and transketolase.

Lactate: Lactate from the cell culture media (0.2 ml) was extracted by ethylene chloride after acidification with HCl. Lactate was derivatized to its propylaminehepta-fluorobutyrate ester form and the m/z328 (carbons 1-3 of lactate) (chemical ionization, CI) was monitored for the detection of m1 (recycled lactate through the PC) and m2 (lactate produced by the Embden-Meyerhof-Parnas pathway) for the estimation of pentose cycle activity. In this study we recorded the m1/m2 ratios in lactate produced and released by bacterial cells in order to determine pentose cycle activity versus anaerobic glycolysis.

Glutamate: Glutamate label distribution from glucose is suitable for determining glucose oxidation versus anabolic glucose use within the TCA cycle, also known as anaplerotic flux. Tissue culture medium was first treated with 6% perchloric acid and the supernatant was passed through a 3 cm3 Dowex-50 (H+) column. Amino acids were eluted with 15 ml 2N ammonium hydroxide. To further separate glutamate from glutamine, the amino acid mixture was passed through a 3 cm3 Dowex-1 (acetate) column, and then collected with 15 ml 0.5 N acetic acid. The glutamate fraction from the culture medium was converted to its trifluoroacetyl butyl ester (TAB). Under EI conditions, ionization of TAB-glutamate produces two fragments, m/z198 and m/z152, corresponding to C2-C5 and C2-C4 of glutamate. Glutamate labeled on the 4-5 carbon positions indicates pyruvate dehydrogenase activity while glutamate labeled on the 2-3 carbon positions indicates pyruvate carboxylase activity for the entry of glucose carbons to the TCA cycle. TCA cycle anabolic glucose utilization is calculated based on the m1/m2 ratios of glutamate.

Fatty acids: Palmitate, stearate, cholesterol and oleate were extracted after saponification of cell pellets in 30% KOH and 100% ethanol using petroleum ether. Fatty acids were converted to their methylated derivative using 0.5N methanolic-HCL. Palmitate, stearate and oleate were monitored at m/z270, m/z 298 and m/z264, respectively, with the enrichment of 13C labeled acetyl units which reflect synthesis, elongation and desaturation of the new lipid fraction as determined by mass isotopomer distribution analysis (MIDA) of different isotopomers.

Gas Chromatography/Mass Spectrometry (GC/MS): Mass spectral data were obtained on the HP5973 mass selective detector connected to an HP6890 gas chromatograph. The settings were as follows: GC inlet 250° C., transfer line 280° C., MS source 230° C., MS Quad 150° C. An HP-5 capillary column (30 m length, 250 μm diameter, 0.25 μm film thickness) was used for glucose, ribose and lactate analyses.

13C, 1H and 31P Nuclear Magnetic Resonance studies of intracellular metabolites: Nuclear Magnetic Resonance (NMR) studies included acetate, alanine, betaine, cholines creatine, glucose, glutamate, glutamine, total glutathion (GSH), glycine, 3-hydroxybutyrate, myo-inositol, lactate, phosphocreatine, pyruvate, valine, hosphocreatine, creatine, ATP, ADP, AMP, NAD+, and total phosphomonoesters (PME) and phosphodiesters (PDE) extracted by ice-cold 0.9% NaCl, 12% PCA and 8 M KOH. The procedure included the transferring of 5 mL medium into a 15-mL tube (ice-bath), removing the rest of the medium thoroughly, washing cells with 6 mL of ice-cold NaCl, adding 2 mL ice-cold 12% PCA to the frozen cells or media and spinning at 1300 g for 20 min at 4° C. After transferring the supernatant into a new 50 ml tube and resuspending the pellet in 2 mL ice-cold 12% PCA in the old 50 mL tube we placed the tubes in an ultrasound ice bath for 5 min. We collected the supernatants, and lyophilized them in a freeze-dry system overnight. After re-suspending 13C, 1H and 31P spectra were obtained on a 9T Brucker vertical bore instrument for quantitative and 13C positional analyses.

Tricarboxylic acid cycle analysis using trimethylsilyl (TMS) derivatives: Frozen pellets (−80° C.; wet 0.5 g) were powderized and extracted with 2:1 (volume) of chloroform-methanol using Omni-TH homogenizer. The slurry was centrifuged at 670 g for 20 min and the upper methanol-water phase was collected and treated with 200 μl of methoxylamine-HCl to protect keto and aldehyde groups. The lower chloroform phase is vortexed for 5 min with 10 mL of methanol-water 3:2 by volume. After 20 min centrifugation the two upper methanol-water phases were combined. The combined methanol-water phase was adjusted to pH 8.0 and evaporated under constant flow of Nitrogen gas in an exhaust fume hood. The residue was reacted with 100 μL of bis(trimethylsilyl)trifluoroacetamide with 10% trimethylchlorosilane (Regisil) at 70° C. for 70 minutes to form the TMS and MOX-TMS derivatives of TCA cycle metabolites. GC-MS analyses were carried out on an Agilent 5975 mass spectrometer, equipped with a model 6890 gas chromatograph and a Varian VF-5MS capillary column (60 m, 0.25 mm i.d., 0.25 mm film thickness.

Flux data analysis and statistical methods: Each experiment was carried out using triplicate cell cultures for each condition within each experiment and experiments were repeated once. Mass spectroscopic analyses were carried out by three independent automatic injections of 1 μl samples by the automatic sampler and accepted only if the standard sample deviation was less than 1% of the normalized peak intensity. Statistical analysis was performed using the Student's t-test for unpaired samples. Two-tailed significance at the 99% confidence interval (μ+/−2.58σ), p<0.01 indicated significant differences in glucose-derived fluxes. For some reversible reactions, we measured both forward and reverse fluxes and calculated net fluxes

RNA preparation for microarray analysis: At all five dilution rates, 10-20 ml of the cell culture was collected, mixed with 10% (v/v) of ice cold stop-solution (5% water-saturated phenol in absolute ethanol), and cell pellets were obtained by centrifugation at 4,500×g for 5 min at 4° C., followed by flash-freezing of pellets with liquid nitrogen. Cell pellets were stored at −80° C. until further use. RNA was isolated from the frozen cell pellets using Masterpure RNA isolation kit (Epicentre Biotechnologies, Madison, Wis.) and RNA samples were processed for transcriptome analysis using E. coli Affymetrix microarray chips, as described previously (Beg Q K, et al., Proc Natl Acad Sci USA 2007, 104:12663-1266.). Dchip method was used to analyze all microarray data as described in detail below.

The culture samples for microarray analysis were collected at all five dilution rates. Approximately, 10 ml of the cell culture was obtained and rapidly mixed with 1/10th volume of the ice-cold stop-solution (5% water-saturated phenol in absolute ethanol) to inhibit any further transcriptional activity. The tubes were capped, and the sample and stop solution were mixed by inversion. The cell pellets were obtained by centrifugation at 4,500×g for 5 min at 4° C., were immediately flash frozen in liquid nitrogen, and were stored at −80° C. until further use. RNA was isolated from the frozen cell pellets using Epicenter's Masterpure RNA isolation kit (using manufacturer's product manual). The samples were also treated with DNAse for 1 hr at 37° C. to remove any DNA contamination in the RNA samples. 10 μg of all RNA samples were processed for transcriptome analysis using E. coli Affymetrix microarray chips by the Microarray Resource Centre, Department of Genetics and Genomics at Boston University School of Medicine as described previously (Beg Q K, et al., Proc Natl Acad Sci USA 104: 12663-12668). The microarray data was normalized using dChip (Li, C et al. Genome Biology 2001, 2: research0032.1-0032.11). The detailed microarray data from five dilution rates is extensive and is not shown.

STEM Clustering Analysis: We used the Short Time-series Expression Miner (STEM) (Ernst and Bar-Joseph, 2006) to identify significant patterns in gene expression profiles in response to the increasing dilution rates. The gene expression profiles were transformed so that they represent the log ratio change in expression from the first sample. The STEM clustering method identifies from a comprehensive library of distinct profiles those with a statistically significant number of genes most closely matching the shape of the profile. Significant profiles are then grouped together such that all profiles in the same group are similar within a threshold. The clusters of significant profiles are then analyzed with a Gene Ontology (GO) enrichment analysis. FIG. 7 shows the library of profiles considered and the significant profiles at a 0.05 Bonferroni corrected level in color. Significant profiles that were grouped together are in the same color.

In FIGS. 8-12, we show genes from significant profiles organized by their clustering group. We also show a GO enrichment table for the set of genes assigned to each of the clusters. The GO categories assigned to various clusters in FIGS. 8-12 reveal important results with respect to the changing gene expression profile at the five dilution rates. We found that the expression profiles for most of the genes in Cluster 1 (FIG. 8) are very similar to the expression profiles for the genes related to the TCA cycle (FIG. 5). For this cluster of genes we see that the GO categories cellular biosynthetic and metabolic processes have significant p-values. In Cluster 2 (FIG. 9) we see that genes for other cellular processes such as flagella, fimbrium development, cell adhesion, and activities in the outer membrane-bound periplasmic are down-regulated at all dilution rates until 0.55 h−1; and the genes for most of these processes were up-regulated at the highest measured dilution rate of 0.72 h−1. Analysis of Cluster 3 (FIG. 10) reveals that activities of most of the genes responsible for part of cell membrane and porin activities are up regulated at intermediate dilution rates of 0.4 h−1 followed by a down regulation at 0.55 h−1 and up-regulation again at 0.72 h−1. In Cluster 4 (FIG. 11) we see that the genes responsible for synthesis and metabolism of various kinds of polysaccharides, biopolymers and lipids, and DNA-mediated transposition were always down-regulated during the course of intermediate growth rates, except at the highest growth rates (0.72 h−1), which means that E. coli never had the requirement of synthesizing or metabolizing polysaccharides at intermediate growth rates. Analysis of Cluster 5 (FIG. 12) reveals that the genes responsible for biosynthesis and metabolism of various amino acids, (such as glutamine, histidine, arginine etc.), urea cycle, and metabolism of various carboxylic acids are always up regulated throughout the growth of the bacterium at all dilution rates.

Querying expression data to identify specific expression profiles: To assess the quality of the microarray profiles and to identify genes with expression patterns that are similar to genes encoding enzymes of the central carbon metabolism, we used TimeSearcher (Hochheiser, H., et al., Dynamic Querying for Pattern Identification in Microarray and Genomic Data. In Proceedings IEEE Multimedia Conference and Expo 2003, (IEEE, Piscataway, N.J.), Vol. 3, pp. III-453-III-456.) to identify genes having similar expression profiles to any of the 23 genes corresponding to the 18 enzymes of FIG. 4. TimeSearcher displays a set of genes satisfying constraints imposed by visual query boxes. The input is a set of known genes and a set of constraint boxes around these genes, and the output is these genes plus all the other genes that also satisfy the constraint boxes.

Querying gene expression of operons in the central carbon metabolism: We also examined the expression profile obtained at five-dilution rates growth of E. coli organized in the operons in the central carbon metabolism. In addition to the genes already listed in FIG. 5, we identified additional genes that are organized in operons (eg. frdABCD, sdhABCD, aceABK) for several more reactions in the central carbon metabolism, for which we don't have measured flux value. Plots (not shown, indicated a good agreement in the expression pattern of genes from the same operon.

**Results**

Limited solvent capacity constrains the metabolic rate of fast growing E. coli cells: The cell's cytoplasm is characterized by a high concentration of macromolecules resulting in a limited solvent capacity for the allocation of metabolic enzymes. See, Equations 2-4 (Eq. 2-4) above.

To understand the relevance of the constraint (Eq. 4) at physiological growth conditions we first estimate the crowding coefficients (where ai=Cvi/bi) using data from experimental reports. The E. coli cytoplasmic density of macromolecules is C=0.34 g/ml (Zimmerman S B, et al. J Mol Biol 1991, 222:599-620), while the molar volumes of proteins are proportional to their molar masses (Lee B, Proc Natl Acad Sci USA 1983, 80:622-626). The coefficient of proportionality represents the specific volume and it is about 0.73 ml/g. This empirical law allows us to compute the molar volumes of E. coli enzymes from their molar masses. As a first approximation we estimate bi, the coefficient of proportionality between reaction rate and enzyme concentration, from the enzyme's turnover numbers. Data obtained from the BRENDA data base for about hundred E. coli enzymes (Additional file 1) shows that the turnover numbers vary over five orders of magnitude (FIG. 1a), from 10−2 to 102 l/s. Using these parameter estimates we compute the crowding coefficients ai for about a hundred E. coli enzymes (FIG. 1b), resulting in an average and standard deviation of 0.014 and 0.009 1/[mmol/g/h], respectively. Because of the large enzyme turnover variations the crowding coefficients are distributed over a wide range as well, from 10−6 to 100 1/[mmol/g/h] (FIG. 1b).

FBAwMC predicts a change of effective metabolic efficiency objective: Having estimated the crowding coefficients we next evaluate the relevance of the solvent capacity constraint (Eq. 4) at physiological growth conditions. To this end we utilize a FBA model of E. coli MG 1655 metabolic network that takes into account this constraint referred to as “flux balance analysis with molecular crowding” (FBAwMC). Under conditions of aerobic growth in a glucose-limited medium, FBAwMC predicts a saturation of the glucose uptake rate and the growth rate (FIG. 2a,b) with increasing the glucose uptake capacity. The predicted maximum glucose uptake rate (˜15 mmol/g/h) and maximum growth rate (˜0.7 h−1 are within the range of experimentally determined values (Fischer E, et al., Anal Biochem 2004, 325:308-316), corroborating our previous report (Beg Q K, et al., Proc Natl Acad Sci USA 2007, 104:12663-12668) that the solvent capacity constraint (Eq. 4) is relevant at physiological conditions.

Associated with the predicted saturation of E. coli metabolic rates, FBAwMC predicts a metabolic switch characterized by a change in the effective criteria of metabolic efficiency. At low growth rates the ratio between the biomass production rate and the glucose uptake rate is at a maximum but decreases with increasing the growth rate. In contrast, the ratio between the biomass production rate and the average reaction rate increases with increasing the growth rate, reaching a maximum at high growth rates. In agreement with our expectations, at low growth rates nutrients are scarce and the best strategy for a cell is to maximize the biomass production rate per unit of limiting nutrient (in this case, glucose) uptake rate. In contrast, at high growth rates the nutrients are abundant, the predicted metabolic rate is limited by the solvent capacity constraint (Eq. 4) and, therefore, the maximum growth rate is achieved by maximizing the biomass production rate per average reaction rate (FIG. 2c). The predicted change in metabolic efficiency objective is accompanied by a redistribution of the metabolic fluxes, including those of exchange fluxes. Indeed, a characteristic example is the predicted excretion of acetate at high growth rates (FIG. 2d) that is well-supported by experimental observations (El-Mansi E M, et al., J Gen Microbiol 1989, 135:2875-2883; Reiling H E, et al., J Biotechnol 1985, 2:191-206; and Wolfe A J, Microbiol Mol Biol Rev 2005, 69:12-50)

FBAwMC-predicted metabolic fluxes are within the range of experimental values: FBAwMC is also able to predict internal metabolic fluxes as a function of the growth rate. A subset of the FBAwMC derived flux predictions in the central carbon metabolism are shown in FIG. 3. In most cases the FBAwMC predicted fluxes are within the range of experimentally determined values. This is a striking result given that this implementation of FBAwMC does not contain any free parameters. The only model parameters are the crowding coefficients, which were determined above using independent experimental results. We should also note that the observed wide variability around the average behavior. Further testing of our predictions will be necessary upon availability of better estimates for the crowding coefficients.

Limiting our analysis to the expected behavior, we observe a slope change for several fluxes when reaching the highest growth rates. The reactions of the glycolytic pathway, the flux towards the pentose-phosphate pathway via the reaction catalyzed by the gene product of zwf, and the acetate pathway switch at high growth rates to a faster flux increase with increasing the growth rate. The experimental values corroborate this qualitative behavior, but the change is bigger for the ptsG-catalyzed reaction and even qualitatively different for the pykA-catalyzed reaction, both being part of the glycolytic pathway. A second noticeable effect is the predicted saturation of the TCA cycle flux at high growth rates. The experimentally measured values of the TCA cycle flux exhibit, however, a stronger effect characterized by a decreasing tendency at high growth rates (FIG. 3). Taken together these results indicate that while for most reactions the FBAwMC predictions are within the range of experimental measurements, a method for a more accurate estimate of the crowding coefficients on a network scale will be required to provide more precise predictions.

Identifying the regulatory mechanism(s) that control the action of the metabolic switch: To examine if the changes in growth conditions and the corresponding adjustments in cellular metabolism can be traced by distinct molecular signatures we next measured the in vitro activity of eighteen selected enzymes that catalyze reactions in the central carbon metabolism of E. coli MG1655, and correlate their changes with those observed for the measured flux rates (FIG. 4). For several enzymes there is a good correlation between the measured enzyme- and flux activities (Pearson Correlation Coefficient, PCC, close to-or larger than 0.8). For example, with an increasing growth rate the enzyme activity of the ptsG and pfkA gene products follow the same increasing tendency as the fluxes of the corresponding metabolic reactions (PCC=0.79 and 0.85, respectively). The glycolytic flux is known to be controlled by the activity of these two enzymes while other reactions adjust their fluxes through changes in metabolite concentrations. In contrast, we found no significant correlation between the measured fluxes and enzyme activities of the TCA reactions (PCC=0.64, 0.35, −0.03 and −0.28 for enzymes associated with gltA, sucA. fumA and mdh, respectively), implying that the TCA flux is controlled by the activity of enzymes catalyzing reactions outside this pathway. A possible candidate to exert this action is the acetate pathway. Indeed, an increase of the flux on the acetate pathway towards the production of acetate can balance both the increase in the flux originating from the glycolytic pathway through aceE and a decrease in the flux from Acetyl-CoA to the TCA cycle. This hypothesis is supported by the increase in the enzyme activity of phosphotransacetylase (pta) when the growth rate increases beyond 0.4 h−1 (PCC=0.98), which is exactly the growth rate threshold where the switch is taking place.

In parallel with the enzyme activity measurements we also prepared mRNA from samples obtained at all five dilution rates and processed them for microarray analysis. In contrast to the observed overall correlation between measured fluxes and in vitro enzyme activities we do not observe a significant correlation between the measured metabolic fluxes and the relative changes in mRNA levels of enzyme-encoding genes (FIG. 5), implying that the switch and corresponding enzymatic functions are not predominantly controlled at the transcriptional level. Correspondingly, no significant correlation between the in vitro enzyme activities and the relative changes in mRNA levels of enzyme-encoding genes can be seen (FIG. 5)Taken together these results indicate that the metabolic switch is predominantly controlled by an increase in the enzyme activities of the end products of ptsG and pfkA controlling the glycolysis flux, and pta controlling the acetate pathway flux, respectively.

**Discussion**

Developing a modeling framework that can describe and predict in a quantitative manner the experimentally observed behavior of an organism is a significant challenge for systems biology. One prerequisite of this goal is to uncover the physicochemical constraints exerting the main influences on cellular metabolism (Price N D, Reed J L, Palsson B O: Genome-scale models of microbial cells: evaluating the consequences of constraints. Nat Rev Microbiol 2004, 2:886-897). Our results here and in Beg Q K, et al. (Intracellular crowding defines the mode and sequence of substrate uptake by Escherichia coli and constrains its metabolic activity. Proc Natl Acad Sci USA 2007, 104:12663-12668) indicate that the limited solvent capacity represents a physiologically relevant constraint for fast growing E. coli cells. The incorporation of this constraint to the FBA modeling framework leads to the FBAwMC model whose predictions indicate that the solvent capacity constraint results in a maximum glucose uptake rate and growth rate that are within the range of experimentally determined values. The flux predictions for several reactions of the E. coli metabolism are within the range of our measurements, as well.

From the perspective of quantitative modeling using flux balance approximations, the solvent capacity constraint forces us to consider reaction kinetics via the crowding coefficients, at least for fast growing cells. At low metabolic rates the solvent capacity constraint is less relevant and flux balance alone is sufficient to obtain satisfactory predictions. In contrast, at high metabolic rates a precise knowledge of the crowding coefficients is required to obtain accurate predictions. In the absence of kinetic information we can still obtain a good approximation by sampling the crowding coefficients from a list of estimated values and then focus on the resulting general trend.

More importantly, the solvent capacity constraint allows the interpretation of the metabolic switch taking place between slow and fast growing E. coli cells. A recent study of FBA models with different objectives demonstrates that under nutrient scarcity a FBA model with the maximization of the biomass yield objective achieve the highest predictive accuracy, while maximizing the ATP or biomass yield per average flux unit is the best objective for unlimited growth on glucose under aerobic conditions (Schuetz R, et al., Mol Syst Biol 2007, 3:119). In contrast, by considering the solvent capacity constraint we obtain the same results using the maximization of biomass production rate objective alone (FIG. 2c). This is more consistent with the expectation that cells achieving the fastest growth rates outgrow cells growing at a slower rate, but how the highest growth rate is achieved is determined by both the availability of substrates and internal metabolic constraints, such as the solvent capacity. Furthermore, the well-known acetate excretion (El-Mansi E M, et al., J Gen Microbiol 1989, 135:2875-2883. Reiling H E, et al. J Biotechnol 1985, 2:191-206; and Wolfe A J, Microbiol Mol Biol Rev 2005, 69:12-50) is explained by the solvent capacity constraint as well. We should note, however, that this does not exclude the possibility that under certain physiological conditions acetate excretion may result from a limited availability of oxygen in the culture medium (Varma A, et al., Appl Environ Microbiol 1994, 60:3724-3731).

### Example 2

Here, we develop a modified FBA model that incorporates a solvent capacity constraint for the attainable enzyme concentrations within the crowded cytoplasm. Using this model, we predict the maximum growth rate of E. coli MG1655 wild-type and mutant strains on single carbon sources and for the dynamic patterns of substrate utilization from a mixed-substrate growth medium. We test the model predictions by using growth rate measurements and microarray and substrate concentration temporal profiles, and we obtain a good agreement between model predictions and experimental measurements. Taken together, these results suggest that macromolecular crowding indeed imposes a physiologically relevant constraint on bacterial metabolic activity and that incorporating this constraint allows for improved modeling of cell metabolism from system-level principles.

Mathematical Framework: The flux balance analysis with molecular crowding (FBAwMC) modeling framework is implemented by solving the following optimization problem: maximize the growth rate subject to the constraints described above. The maximum growth rate corresponds to the biomass production rate, where biomass production is an auxiliary reaction containing as substrates the cellular components in their relative concentrations and as product the cell's biomass.

We model the crowding coefficients ai as noise. The reported results were obtained by assigning a random value to them from the gamma distribution

\(\begin{matrix}
{{P(a)} = {\frac{\beta}{\langle a\rangle}\left( {\frac{\beta}{\langle a\rangle}a} \right)^{\beta - 1}{\exp \left( {{- \frac{\beta}{\langle a\rangle}}a} \right)}}} & \lbrack 6\rbrack
\end{matrix}\)

where β>0 and <a> is the average crowding coefficient. There is no particular reason for this choice other than by changing β we can explore different scenarios. For instance, for β=1 we obtain an exponential distribution, whereas for β>>1 we obtain a distribution that is almost concentrated around a=<a>. The results reported in FIGS. 13-15 were obtained by using β=3 and running the simulations 1,000 times to test the sensitivity of the results with respect to the specific ai values. Similar results are obtained by using other P(a) distributions.

The maximum growth rate μ for each carbon source was obtained assuming an unbound uptake rate for that carbon source and zero for all other carbon sources. The average crowding coefficient <a> was fitted to obtain the minimum square deviation between the measured and model predicted growth rates, resulting <a>=0.0040±0.0005 h·g/mmol, in which g is grams dry weight. However, the maximum growth rates on glucose and glycerol are more consistent with <a>=0.0031±0.0001 h·g/mmol and <a>=0.0053±0.0001 h·g/mmol, respectively.

To model the temporal order of substrate uptake we considered an initial concentration of 0.4 g/liter for glucose, galactose, lactate, maltose, and glycerol, zero concentration for acetate, and cell's dry weight DW=0.00675 g. The progression of the dry weight and the external substrate concentrations were obtained from the integration of the differential equations where m is restricted to external metabolites, μm is the molar mass of metabolite m, and V is the working volume. The maximum growth rate μ(t) and the fluxes fi(t) are obtained by solving the FBAwMC model for the substrate concentration profile at time t. The value of <a> is smaller if glucose alone is consumed and larger if glycerol is consumed. Therefore, we solve three FBAwMC problems corresponding to the consumption of glucose alone (<a>=0.0031 h·g/mmol), consumption of all substrates except glycerol (<a>=0.004 h·g/mmol), and consumption of all substrates (<a>=0.0053 h·g/mmol), and selected the condition resulting in the maximum growth rate.

Estimation of the Crowding Coefficients: The crowding coefficients play a key role in our model. Therefore, it is important to estimate their values from experimental measurements. However, this is a challenging task, as it requires knowledge of the kinetic parameters associated with the E. coli enzymes, which are not available in all cases. Yet we can obtain an estimate for the crowding coefficients by making use of the known turnover rate of several E. coli enzymes. The crowding coefficients (units of time×mass/mole) are defined by Eq. 5 (see above), where C (units of mass/volume) is the density of the E. coli cytoplasm, vi (units of volume/mole) is the molar volume of enzyme i, and bi (units of 1/time) is the proportionality factor between the flux [units of mole/(mass×time)], and the enzyme concentration (units of mole/mass), i.e., fi=biEi. In general bi is decomposed into

bi=xiki,   [7]

where ki (units of 1/time) is the enzyme turnover number, and 0≦xi≦1 (no units) is determined by the concentration of substrates, products, and activators/inhibitors associated with the ith reaction. For example, if reaction i is a single substrate irreversible reaction and it is characterized by Michaelis-Menten kinetics, then

\(\begin{matrix}
{{x_{i} = \frac{S}{S + K_{i}}},} & \lbrack 8\rbrack
\end{matrix}\)

where S is the substrate concentration and Ki is the Michaelis-Menten or half-saturation constant. Thus, xi≈0 at small substrate concentrations (S<<Ki), whereas at saturation (S>>Ki) xi≈1. Substituting Eq. 7 into Eq. 6 and assuming that enzymes are working near saturation (xi≈1), we obtain

\(\begin{matrix}
{a_{i} \sim \frac{{Cv}_{i}}{k_{i}}} & \lbrack 9\rbrack
\end{matrix}\)

where the symbol ˜ indicates that this is an estimate and some variation is expected for enzymes that are not working at saturation. The magnitudes on the right-hand side can be estimated by using different sources of experimental data. The E. coli cytoplasmatic concentration is C≈0.34 g/ml). The enzymes' molar volumes can be estimated from the equation vi(molar)=Mivi(specific), [10], where Mi is the molar mass of enzyme i (units of mass/mole) and vi(specific) is its specific volume (units of volume/mass). We estimate the specific volume from measurements of this magnitude for globular proteins. These data indicate that the specific volume has small variations among enzymes relative to the molar mass variations. Therefore, we take the same specific volume for all enzymes and equal to the reported average specific volume for the proteins, resulting in vi(specific)≈0.73 ml/g. Finally, we consider the molar mass and the turnover number of several E. coli enzymes reported in BRENDA. Using these data and Eqs. 9 and 10, we computed ai for several E. coli enzymes, resulting in the distribution shown in FIG. 17.

Growth Experiments, Carbon Substrate, and Microarray Analyses: The E. coli K12 strain MG1655 (F− λ− ilvG rfb50 rph1) was used throughout the work. Isogenic E. coli mutants (pgk, atpC, gpmA, nuoA, gdhA, and pfkA) were obtained from F. Blattner (University of Wisconsin, Madison, Kang Y, et al., (2004) J Bacteriol 186:4921-4930.). The experimental details of the growth rate measurements, substrate concentration assays and microarray analyses are detailed below.

Bacterial Strains and Growth Conditions: The E. coli K12 strain MG1655 (F−-λ−-ilvG rfb50 rph1) was used throughout the work. Isogenic E. coli mutants (pgk, atpC, gpmA, nuoA, gdhA, and pfkA) were obtained from F. Blattner (University of Wisconsin, Madison). Chemicals and reagents used in the growth experiments were from Sigma (St. Louis, Mo.) or Fisher Scientific (Pittsburgh, Pa.). The growth experiments using M9 minimal medium containing mixture of five carbon sources (glucose, maltose, galactose, L-lactate, and glycerol) were carried out in a 2-liter Labfors bioreactor with 1.2-liter working volume (Infors, Switzerland). All the carbon sources (used in equal ratios) were filter sterilized and added to the growth medium at a final concentration equivalent to 0.2% sugar. For these growth experiments, the dissolved oxygen was set at 100% initial value, and sterile air was continuously sparged into the medium. Growth parameters, such as pH, pO2, temperature, and agitation were continuously monitored through microprocessor probes. Samples were collected at 30-min intervals to document various growth phases and extracellular substrate concentrations and to assess the transcriptome state. For assessing the exponential-phase transcriptome states in individual carbon-limited (glucose, galactose, maltose, glycerol, lactate, or acetate) M9 minimal medium, the carbon source concentration in the M9 minimal medium was adjusted to the number of carbon atoms equivalent to that present in 0.2% glucose. Fifty microliters of the overnight inoculum of E. coli MG1655 (prepared in the same medium used for the experiment) was inoculated in 50 ml of growth medium in a 250-ml Erlenmeyer flask supplemented with the appropriate carbon source. The growth profile of OD600 was documented at regular intervals.

Experimental Determination of Maximum Growth Rates: When determining maximum growth rates in different carbon sources, we used a 12- to 15-h-old E. coli culture (prepared in the same substrate to be used for the final medium) to inoculate a Sixfors bioreactor (Infors, Switzerland) containing M9 minimal medium (400 ml working volume) supplemented with various carbon sources, and monitored the growth profile at regular intervals by optical density at 600 nm. A method of continuous cultivation of bacterial cells with smooth changes in growth rate was used for determining the maximum growth rate in various substrates. This method has been shown to be more precise to determine growth rate in E. coli than batch culture (Paalme T, et al. (1997) Antonie Van Leeuwenhoek 71:217-230 and Paalme T, et al., (1995) J Microbiol Methods 24:145-153). When the cells reached stationary phase (also indicated by constant OD and sudden increase in dissolved oxygen value), the flow of fresh growth medium was started at a dilution rate slightly below their maximum growth rate calculated from their growth profile in the exponential phase. The cells were allowed to grow at this dilution rate until they reached a steady state. Once the cells were found to be growing well in the steady state, the washout of cells was started at a dilution rate that was above their maximum growth rate. The washout was done for the next 6 h, and the OD600 data obtained (OD readings were converted into log) during the washout period were used to calculate the maximum growth rate for all the cultures.

\(\begin{matrix}
{{{Growth}\mspace{14mu} {rate}} = {D - \left( \frac{\ln_{f} - \ln_{i}}{3} \right)}} & \lbrack 11\rbrack
\end{matrix}\)

where D is the dilution rate during washout of cells, ln f is the natural log of the final OD600, and ln i is the natural log of an OD600 3 h before the final OD600.

Substrate Concentration Assays: For determining the residual concentration of individual carbon sources in the growth medium, samples of cell culture were centrifuged for 2 min at 13,000 g at 4° C. The supernatant was filtered through a 0.22-μm filter and stored at −80° C. until further use. Concentrations of D-galactose, D-glucose, glycerol, L-lactate, maltose, and acetate in the cell-free supernatant were determined by using corresponding Enzymatic BioAnalysis kits (R-Biopharm, South Marshall, Mich.) according to the manufacturers' instructions. Before analysis, all the samples were placed in a water-bath at 80° C. to stop any background enzymatic activity. For maltose analysis, the samples were treated with glucose oxidase and hydrogen peroxide and passing air current for 1 hr to remove residual glucose (which might impair the precision of the maltose assay) as per the manufacturer's instructions.

Microarray Sample Collection: For the shake-flask exponential-phase experiments using individual carbon sources, at OD600≈0.2, the whole cell culture volume (50 ml) was mixed with 5 ml of ice-cold stop-solution (5% water-saturated phenol in absolute ethanol), and a cell pellet was obtained by centrifugation at 4,500 g for 5 min at 4° C., followed by flash freezing of pellets with liquid nitrogen. The pellets were stored at −80° C. until further use. For the time series experiments using a mixture of five carbon sources in M9 minimal medium, culture samples for microarray analysis were collected at 30-min interval between 2 and 8 h of growth. Approximately 5-50 ml of the cell culture (depending on the stage the cells were growing) was obtained and rapidly mixed with 1/10th vol of the ice-cold stop-solution (5% water-saturated phenol in absolute ethanol) to inhibit any further transcription. The tubes were capped, and the sample and stop-solution were mixed by inversion. The cell pellets were obtained by centrifugation at 4,500 g for 5 min at 4° C., immediately flash frozen in liquid nitrogen, and stored at −80° C. until further use. RNA was isolated from the frozen cell pellets by using Epicenter's Masterpure RNA isolation kit (using the manufacturer's product manual). The samples were also treated with DNase for 1 h at 37° C. to remove DNA contamination in the RNA samples. Ten micrograms of all RNA samples was processed for transcriptome analysis using E. coli Affymetrix microarray chips by the Microarray Resource Centre, Department of Genetics and Genomics at Boston University School of Medicine (www.gg.bu.edu/microarray/index.htm).

Microarray Analysis of Samples from Individual Carbon Source-Limited Media (Glucose, Maltose, Galactose, Glycerol, Lactate, and Acetate): The Dataset for this analysis is not shown as it is too large. Details are summarized herein. To perform the data analyses, we first identified genes for which there is no evidence of sequence-specific hybridization intensity in any of the five samples. If there was no sequence-specific hybridization, that gene was not analyzed any further. We searched for genes that vary between the five arrays much more than genes with similar hybridization intensity. The idea here is that the observed difference for a gene that varies much more than its neighbors is probably not due to technical noise and thus the variability could be due to underlying biological differences between the samples. The ratio of the variability of a particular gene to the average variability of its 50 closest neighbors was determined. When analyzing these data, we chose an arbitrary threshold of 2.5 for this variability ratio and identified the top 150 genes that had a ratio larger than 2.5 (i.e., the 150 genes that are probably the most dramatically affected by the given growth conditions).

Examination of the top 150 genes reveals that glucose-, maltose-, and galactose-limited growth results in the condition-specific up-regulation of genes involved with the transport and catalysis of the specific carbon source (e.g., of those of the maltose regulon in the maltose-limited growth condition), whereas glycerol and acetate display similarities in the expression program, but the “acetate signature” is stronger and more expanded compared to glycerol's. This observation suggests the appearance and subsequent expansion of a foraging program as the quality of the carbon source decreases, as previously suggested (Liu M, et al. (2005) J Biol Chem 280:15921-15927). As the mid-log phase growth experiment using lactate as sole carbon source was done separately then other five carbon source experiments, we compared the expression of the genes expressed under lactate-limiting conditions with those of glucose on a different Affymetrix chip (dataset not shown). Those genes that failed to exhibit sequence-specific hybridization signal in either sample were removed from consideration. As random variability due to system noise is generally inversely related to hybridization intensity, we compared the relative difference between the glucose and lactate samples for each gene to the average difference between the two samples for genes with similar hybridization intensity. The idea is that if a gene is in a hybridization intensity neighborhood of noisy signals, the observed difference between the two samples is less likely to be due to a biological cause than if other genes with similar hybridization intensity show little change. This idea was reduced to a number. Genes with a “surplus variability” were identified. Other genes were identified that were more likely to be differentially expressed than those genes with a surplus variability<4. Finally, we calculated the log 2-fold change between the lactate and glucose samples and identified those that are more than 2-fold higher in the lactate sample and in the glucose (GLC) sample. To have all the information after using two different microarray platforms, we scaled the raw values from the second glucose experiment so that the total sum of raw intensities of common genes would be the same as the first. The two glucose experiments were then averaged together. Then we scaled the lactate array by the same factor as used to make the two glucose sums the same. Then for each gene and condition we took the log 2 of the intensity value over the geometric mean of the intensity values across all six experiments. The glucose data in the correlation matrix shown in the bottom panel of FIG. 16 are the average of the two glucose values from the datasets after this normalization.

Microarray Analysis of Samples from Time Series Mixed-Substrate Experiment: To analyze the dataset, first we removed those expression measurements that were not sequence-specific in any of the samples in the dataset. This filter removes expression measurements for genes that are probably not expressed under the growth conditions. Then we calculated the fold change for each time point relative to the geometric mean of the hybridization intensity of all time points for each gene. Then we organized the expression measurements from top to bottom so that genes that show similar changes between the samples are near each other. The Principal Component Analysis (Peterson L E (2003) Comput Methods Programs Biomed 70:107-119.) (FIG. 18), which globally assesses the similarity of transcriptome profiles to each other by considering all the genes on the array, displays that there are similar expression profiles in all samples collected between 2 and 3.5 h (exclusive glucose utilization phase). This was followed by transition in the transcriptome state at the beginning of the mixed-utilization phase between 4 and 4.5 h. The 5- to 5.5-h (mixed substrate utilization phase) and 6.5-h samples (late carbon utilization phase) display similar global expression profiles, interrupted by a significant alteration in the expression profile at 6 h (which is indicating the switch from acetate secretion to acetate utilization).

Querying Expression Data to Identify Specific Expression Profiles: TimeSearcher was used to identify genes having expression profiles similar to those of genes that are known to participate in specific uptake pathways. TimeSearcher displays a set of genes satisfying constraints imposed by visual query boxes. The input is a set of known genes and a set of constraint boxes around these genes, and the output is these genes plus all the other genes that also satisfy the constraint boxes. Genes with similar expression pattern identified for each profile were identified. These individual figures for glucose, lactate, maltose, galactose, glycerol, and acetate uptake clearly showed that several other genes that do not directly take part in substrate uptake of these individual carbon sources are also up-regulated along with the substrate uptake genes. Many of these genes take part in the intermediate steps of metabolism after substrate uptake.

Hierarchical Clustering of Time-Series Gene Expression Data: To further study the clusters determined by TimeSearcher we performed hierarchical clustering with optimal leaf ordering (Bar-Joseph Z, et al. (2001) Bioinformatics 17:S22-S29 and Eisen M B, et al. (1998) Proc Natl Acad Sci USA 95:14863-14868). Next, we examined the enrichment of different clusters for different Gene Ontology (GO) category annotations by using the hypergeometric distribution to compute P values. Corrected P values for multiple hypothesis testing were computed by using a randomization procedure. We observed three major expression clusters, denoted as A, B, and C (see FIG. 16). The details of GO analysis of genes in each cluster about these expression clusters were identified.

Probabilistic Clustering of Time-Series Data: While hierarchical clustering is useful for visualizing and analyzing complete datasets, it is less appropriate for studying specific clusters because it is a greedy method that is sensitive to noise. To complement the hierarchical clustering results we have used a Hidden Markov Model (HMM) to cluster the data (Ernst J, Vainas O, et al. (2007) Mol Syst Biol 3:74) Unlike in that reference, static transcription factor-gene association data were not used as part of the clustering model. The HMM model was restricted such that each state of the model was associated with one time point, and every state had a transition to at least one state in the next time point and no more than three states. Output distributions of each state were associated with a Gaussian distribution. Genes were grouped into clusters such that each gene in the same cluster had the same most probable path through the HMM model. A total of 16 clusters were identified.

Stress Response: We also examined the expression profiles of stress-response genes during stationary phase, when the stationary phase σ factor RpoS controls the cellular physiology and complex gene regulatory network. In addition to the up-regulation of genes responsible for glycerol and acetate uptake (FIG. 15), several other known genes for the stress response were also up-regulated. For this we selected several genes which are shown to be up-regulated during stationary-phase stress response, and compared their expression profiles in our microarray data. Our results (FIG. 19) on mixed-substrate microarray analysis also showed that several known genes during various stationary-phase stress responses (osmotic stress, periplasmic shock, genes that help cell for long-term survival, universal stress) also show a high expression pattern after the cell reaches stationary phase; the expression pattern of these genes come down between 6 and 6.5 h and then go up again after 6.5 h. Thus, it can be concluded that E. coli cells display a dual stress response, a mild response prior to switching to the utilization of glycerol and acetate and a major one at the complete exhaustion of all substrates.

Biological functions of various genes shown in FIG. 19—rpoS: RNA polymerase, σ S (σ38) factor; synthesis of many growth phase related proteins. Upon entry into stationary phase, the major adjustments in cellular physiology are controlled by complex regulatory network involving stationary-phase σ factor RpoS.

The following genes are involved in survival of osmotic stress: otsA and otsB: trehalose-6-phosphate synthase and trehalose-6-phosphate phophatase; respectively; treC: trehalose-6-phosphate hydrolase; treB: PTS system enzyme II, trehalose specific; treA: trehalase, periplasmic; osmY: hyperosmotically inducible periplasmic protein; osmC: osmotically inducible protein; osmB: osmotically inducible lipoprotein; osmE: osmotically inducible protein. The following genes help cells for long-term survival: bolA: possible regulator of murein genes; dps: global regulator, starvation conditions; cbpA: curved DNA-binding protein; functions closely related to DnaJ; glgS: glycogen biosynthesis, rpoS dependent. Other genes induced during various stress responses: cspD: stress-induced DNA replication inhibitor; rpoE: RNA polymerase, σE factor; heat shock and oxidative stress; rseA: σE factor, negative regulatory protein (induced during periplasmic shock); csrA: carbon storage regulator; controls glycogen synthesis, gluconeogenesis, cell size,; and surface properties; wrbA: flavoprotein WrbA (Trp repressor binding protein); uspA: universal stress protein.

**Results**

FBA with Molecular Crowding Predicts the Relative Maximum Growth of E. coli Growing on Single Carbon Sources: FBAwMC is described above. To examine the validity of macromolecular crowding as a constraint on a cell's metabolic activity, and to test the predictive capability of the FBAwMC framework, we first examined the phenotypic consequences of extracellular substrate availability during growth in single carbon-limited medium with oxygen being in abundance, focusing on the maximum growth rate. The FBAwMC contains as a free parameter the average crowding coefficient <a>, and the model predictions for the maximum growth rate are proportional to <a>. We first assumed that <a> is a constant independent of the substrates. In this case it is possible to make predictions for the maximum growth rate in different substrates in arbitrary units. To obtain the maximum growth rates in specific units we fit <a> to minimize the mean-square deviation between the predicted and measured growth rates, resulting in <a>=0.0040±0.0005 h·g/mmol, in which g is grams of dry weight. We have obtained an independent estimate of ai for ≈100 E. coli enzymes as well, resulting in values between 10−6 and 10−1 and most probable values between 10−5 and 10−2 (in units of h·g/mmol). The obtained <a> is, therefore, within the expected range.

Using the reconstructed E. coli MG1655 metabolic network, we first tested the maximal growth rate of E. coli MG1655 cells in various single carbon-limited media and compared the results with the theoretically predicted growth rates (FIG. 13a). In most cases the line of perfect agreement falls within the standard deviation, implying an overall good agreement between the model predictions and the measured maximum growth rates. For glucose and glycerol, the line of perfect agreement is outside the standard deviation, indicating that our assumption of a substrate-independent <a> is not valid for these two substrates. E. coli is better adapted to growth on glucose, suggesting a smaller average crowding coefficient than in any of the other carbon sources. Indeed, the average crowding coefficient necessary to obtain a perfect agreement for glucose is smaller: <a>=0.0031±0.0001 h·g/mmol. In contrast, in some carbon-limited media E. coli reaches its predicted maximal growth rate only after a period of adaptive evolution (Fong S S, et al., (2004) Nat Genet 36:1056-1058; Ibarra R U, et al., (2002) Nature 420:186-189), suggesting a higher average crowding coefficient before metabolic adaptation. Indeed, the average crowding coefficient necessary to obtain a perfect agreement for glycerol is larger: <a>=0.0053±0.0001 h·g/mmol.

The FBAwMC framework also allows us to predict the maximal growth rate of microbial strains with deleted metabolic enzymes, by simply removing the corresponding metabolic reaction from the FBAwMC model and recomputing the maximal growth rate. To test the power of this predictive capability we experimentally determined the maximal growth rate of several E. coli MG1655 single gene deletion mutants grown in glucose-limited medium. As shown in FIG. 13b, the agreement between predicted and measured maximal growth rates is remarkably good for various E. coli mutants, providing further evidence for the validity of our approach. It is worth noting that, as with FBA alone, this analysis is not limited to single-enzyme mutants, but can be carried out for any combination of two or more enzyme deletions as well.

Substrate Hierarchy Utilization by E. coli Cells Growing in Mixed Substrates Extensive experimental data indicate that when grown in complex media bacterial cells use the available substrates either preferentially or simultaneously depending on the growth conditions. To further assess the role of an enzyme concentration limit on cellular metabolism we next examined the substrate utilization of E. coli cells in a mixed carbon-limited medium, and we compared the results to the FBAwMC E. coli model-predicted substrate uptake and utilization (FIG. 14). We grew E. coli MG1655 for 12 h in a batch culture containing an equal concentration (0.04% each) of five different carbon sources (galactose, glucose, maltose, glycerol, and lactate) (FIG. 14a). These substrates are taken up by E. coli through substrate-specific transport mechanisms and enter the central carbon metabolism through various substrate intermediates (FIG. 15 metabolic pathways). Note, that in single carbon-limited medium, maximum growth rates of E. coli in glucose (0.74 h−1) was higher; whereas the experimentally measured maximal growth rates in glycerol (0.41 h−1) and lactate (0.38 h−1) were lower than the model predictions (FIG. 13a). In contrast, the maximal growth rates obtained on maltose (0.61 h−1) and galactose (0.51 h−1) were in good agreement with the FBAwMC-predicted values (FIG. 13a).

As typically seen in batch culture, initially E. coli cells showed minimal growth (lag phase) followed by rapid population expansion between 2 and 8 h (exponential growth phase) with no further growth afterward (stationary phase) (FIG. 14a). Parallel with this, the growth rate rapidly increased with the start of the logarithmic growth phase, reaching its maximum between 3 and 3.5 h. Thereafter the growth rate steadily declined, becoming negligible to zero after 8 h (FIG. 14a).

Of the five supplied carbon sources, in the first 3.5 h of growth only glucose was used (phase 1); it was depleted from the medium within the first 4 h (FIG. 14a). This “exclusive glucose use” phase coincided with the initial explosive growth and the maximal attained growth rate of the culture (FIG. 14a). At 3.5-4 h E. coli cells started to use all four remaining carbon sources, albeit at different rates. Galactose, lactate, and maltose were preferentially used during the next 2 h (phase 2), all three of them being depleted from the growth medium by the sixth hour (FIG. 14b). During this “mixed carbon utilization” phase lactate was used up at the fastest rate, followed by maltose and galactose. A small amount of glycerol was also taken up during this time interval, but its predominant utilization occurred only after 6 h, and it was completely depleted from the medium by 7.5 h (FIG. 14b). The concentration of acetate, a well known byproduct of rapid E. coli aerobic growth, increased steadily, reaching its peak concentration in the growth medium at 6 h of growth. Thereafter, the process was reversed, and acetate, along with glycerol, was rapidly consumed and was depleted from the medium by 8 h (FIG. 14b), denoting a “late carbon utilization” phase in the culture (phase 3). Of note, as single carbon source, acetate provides a lower maximal growth rate (0.24 h−1) than any of the five supplied carbon sources (FIG. 13a). Taken together, the sequential order of carbon substrates' uptake in the batch culture experiment only partially correlates with the maximal growth rate they individually provide: it appears earlier for lactate and later for maltose and glycerol.

Subsequently, we tested FBAwMC E. coli model on the mixed-substrate conditions. In contrast with FBA, which predicts the simultaneous utilization of all carbon sources, we find a remarkably good correlation between the mode and sequence of FBAwMC-predicted and measured substrate uptake and consumption (FIGS. 14b and 14c). There are, however, two notable differences. First the FBAwMC predicts a lesser excretion of acetate. In turn the substrates are consumed faster in vivo (FIG. 14b) because a larger fraction of the carbon source is diverted toward the excretion of acetate. As a consequence the different phases of substrate consumption are shifted to the right (longer times) for the model predictions. The second major discrepancy is the delayed consumption of galactose in the model predictions (FIG. 14c). Yet, overall FBAwMC correctly predicts the existence of the three experimentally observed phases of substrate consumption: initial consumption of glucose, intermediate mixed-substrate consumption, and late consumption of glycerol and acetate.

As surrogate markers of cellular metabolism, during the batch culture experiments we also traced the changes in pH and oxygen concentrations in the growth medium. We observed a steady decline in pH during the first 6 h, followed by a slight increase then decrease between 6 and 7 h, and finally an increase between 7 and 8 h (FIG. 14a). There was also an accelerating decline in the dissolved oxygen concentration (pO2) in the medium during the first 7 h, followed by a rapid stepwise increase during the next 30 min. However, the decline phase (indicating aerobic respiration in an increasingly acidic environment because of acetate excretion) was consistently interrupted by rapid upswings in pO2 concentration (FIG. 14a). These spikes indicate brief pauses in aerobic metabolism likely due to switches in predominant substrate use. Indeed, the first of these spikes, at ≈4 h, correlates with the depletion of glucose and initiation of mixed-substrate utilization; the second, at ≈5 h, with the depletion of lactate and increased utilization of maltose; and the third, at ≈6 h, with the start of joint glycerol and acetate utilization. Similarly, the first rapid increase in oxygen concentration at ≈7 h correlates with the near-depletion of glycerol, followed by a final increase after 30 min corresponding with the depletion of acetate from the medium (FIG. 14b).

The Mode and Sequence of Substrate Utilization Correlate with the Expression of Genes Participating in the Uptake Modules: We also prepared mRNA from samples obtained at 30-min intervals between 2 and 8 h and processed them for microarray analysis. At the level of substrate uptake pathways (FIG. 15) it is evident that the expression of ptsG, the gene encoding the glucose transporter PtsG/Crr, was at a high level from the first time point up until the depletion of glucose from the growth medium, and ptsG expression was rapidly turned off afterward. Similarly, the expression of the gene (lldP) encoding the lactate transporter, LldP, was high during the first 4.5 h of growth and was turned off rapidly thereafter, in agreement with the earlier than predicted utilization of lactate. In contrast, the expression of gene products responsible for maltose and galactose uptake and utilization were turned on much later and peaked at 4.5-5.5 h, corresponding with the period of their maximal uptake. The expression of gene products responsible for glycerol uptake and utilization peaked in two waves, the smaller one between 4.5 and 5 h and the larger one at 6.5-7 h, the latter corresponding to maximal glycerol consumption from the medium (FIG. 14b) Finally, the expression of acs, whose gene product catalyzes acetate uptake toward the citric acid cycle, peaked between 6 and 8 h of growth, corresponding to the maximal uptake of previously secreted acetate from the growth medium. We note that all these changes are in good agreement with the FBAwMC model-predicted uptake of the corresponding substrates (FIG. 15, black tracings).

To assess the quality of the microarray profiles and to identify genes with expression patterns that are similar to those of genes encoding enzymes of the uptake pathways we used TimeSearcher. We find that most genes displaying expression patterns similar to those of the query genes are colocalized with them in the same operon. For example, for the maltose uptake module genes (malEFGK, malQ, and glk), TimeSearcher identified several other genes (lamB, malM, malP, malS, and malZ) with similar expression profiles. These genes are part of various operons within the maltose regulon, although not all of them directly participate in maltose uptake. Similarly, for glycerol metabolism several related glycerol utilization genes (glpA, glpB, glpC, glpD, glpQ, and glpT) displayed expression patterns that were similar to those of the three genes responsible for glycerol uptake (glpF, glpK, and gpsA). The products of these genes are part of the pathway for glycerol catabolism after its uptake.

Activation of Stress Programs upon Switching Metabolic Phases—To assess the global state of E. coli transcriptome during the various metabolic phases of the time course experiment, we used three different data analysis methods to analyze the full microarray data. These methods included hierarchical clustering with optimal leaf ordering (Bar-Joseph Z, et al., (1998) Proc Natl Acad Sci USA.) (FIG. 16), principal component analysis (PCA) (Peterson L E (2003) Comput Methods Programs Biomed 70:107-119.) (FIG. 18), and a probabilistic clustering method based on hidden Markov models (HMMs) (Ernst J, Vainas O, Harbison C T, Simon I, Bar-Joseph Z (2007) Mol Syst Biol 3:74.) (not shown). It was evident that during the exclusive glucose utilization phase there are similar expression profiles in all samples collected between 2 and 3.5 h, followed by transition in the transcriptome state at the beginning of mixed utilization phase after 4 h (FIG. 16). Within the latter phase, the up-regulation of, e.g., the genes of the maltose regulon are clearly evident (FIG. 16).

Samples obtained during the mixed-substrate utilization phase (5 and 5.5 h) and the late carbon utilization phase (6.5 h) display similar global expression profiles (FIG. 16), interrupted by a significant alteration in the expression profile at 6 h that denotes the switch from acetate secretion to acetate utilization (FIG. 15). Interestingly, the transcriptome at 6 h displays substantial similarity to that characterizing E. coli cells at the (near) exhaustion of all substrates from the medium (7-8 h), a phase that is characterized by generic stress response (FIG. 16 and FIG. 19). Similarly, many of the genes up-regulated at the end of the last phase are also up-regulated to a lesser extent at 3.5 h, the stage of switching from exclusive glucose utilization to a mixed-substrate utilization phase.

To further characterize the time-point-specific expression profiles, we also prepared mRNA samples from individual mid-logarithmic batch culture E. coli cells (OD600≈0.2) grown in glucose-, maltose-, glycerol-, acetate-, lactate, or galactose-limited medium, processed them for microarray analysis, and compared the obtained transcriptome profiles with those of the individual time points (FIG. 16 Lower) from mixed-substrate experiment. It is evident that the transcriptome profiles during the glucose-only and mixed-substrate utilization phases display the highest correlation to that of glucose- and maltose-limited cultures, especially at the initial time points, whereas the late carbon utilization profiles are most similar to that of glycerol- and, especially, acetate-limited cultures. Highly notable is the transcriptome profile of galactose-limited cultures, which shows some similarity to that of cells at the stage of switching from exclusive glucose utilization to a mixed-substrate-utilization phase (3.5 h), and an even higher similarity to the transcriptome profiles of cells when all carbon sources are depleted (8 h). Thus, E. coli cells display a partial adaptation/stress response at each major metabolic transition, followed by a generic stress response (FIG. 19) and implementation of a foraging program (Liu M, et al., (2005) J Biol Chem 280:15921-15927.) at complete exhaustion of all extracellular substrates that seems to be most primed for acetate and galactose catabolism.

**Discussion**

A key aim of systems biology is the identification of the organizing principles and fundamental constraints that characterize the function of molecular interaction networks, including those that define cellular metabolism. In the present work we have focused on the identification of principles that define the growth and substrate utilization mode of bacterial cells in complex environments. Our experimental results indicate the occurrence of three major metabolic phases during the growth of E. coli on one type of mixed-substrate medium. Glucose, which by itself provides the highest growth rate, is preferentially used by E. coli, followed by simultaneous utilization of maltose, l-lactate, and galactose. Glycerol and (secreted) acetate are used at a third and final stage of growth. In addition, global mRNA expression data indicate that the organism-level integration of cellular functions in part involves the appearance of partial stress response by E. coli at the boundaries of major metabolic phases, and, as previously shown (Liu M, et al., (2005) J Biol Chem 280:15921-15927.), the activation of a foraging program upon exhaustion of substrates from the growth medium (FIG. 16).

The simulation results show that the FBAwMC model introduced here successfully captures all main features of the examined metabolic activities. First, there is a significant correlation between in vivo relative maximal growth rates of E. coli in different carbon-limited media and the in silico predictions of the FBAwMC (FIG. 13). Second, the FBAwMC model predicts remarkably well the existence of three metabolic phases and hierarchical mode (i.e., single- or mixed-substrate utilization) of substrate utilization in mixed-substrate growth medium (FIGS. 14-16). In essence, our modeling approach indicates that when E. coli cells grow in conditions of substrate abundance their growth rate is determined by the solvent capacity of the cytoplasm; vice versa, the solvent capacity should be saturated at the maximal growth rate. Therefore, when growing in a mixture of abundant carbon sources E. coli cells should preferentially consume the carbon source resulting in the highest growth rate. At solvent capacity saturation, the synthesis of metabolic enzymes for the utilization of a second, less efficient, carbon source can take place only at the expenses of degrading metabolic enzymes involved in the consumption of the more efficient carbon source. However, this would result in a growth rate reduction and, therefore, cells preferentially using the more efficient carbon source would outgrow those that allow the simultaneous utilization of other carbon sources.

We observe, however, two discrepancies of the FBAwMC model predictions: (i) a higher than predicted amount of secreted acetate in the growth medium, and (ii) a somewhat earlier uptake and consumption of various substrates from the medium compared with that predicted by the model. The first discrepancy is likely rooted on the contribution of other cell components apart from metabolic enzymes. With increasing growth rate higher concentrations of ribosomal proteins, mRNA, and DNA are required in addition to metabolic enzymes (Neidhardt F C, et al., (1990) Physiology of the Bacterial Cell: A Molecular Approach (Sinauer, Sunderland, Mass.). This observation indicates that the FBAwMC model may underestimate the impact of macromolecular crowding and the resulting excretion of acetate. The second discrepancy is quite likely a consequence of the first one, as acetate secretion is generally correlated with an increased carbon source uptake rate (El-Mansi E M, et al., (1989) J Gen Microbiol 135:2875-2883).

Taken together, our results show that in silico models incorporating flux balance and other physicochemical constraints can capture increasingly well the metabolic activity of bacterial cells, and that the maximum enzyme concentration is a key constraint shaping the hierarchy of substrate utilization in mixed-substrate growth conditions. Yet, while the metabolic capabilities of a cell are limited by such constraints, in reality any change in metabolic activity is controlled by regulatory mechanisms evolved in the context of these constraints. Therefore, constrained optimization approaches are also expected to help us better understand and/or uncover regulatory mechanisms acting in E. coli and other organisms.

### Example 3

**Flux Balance in S. Cerevisiae with Molecular Crowding and Kinetic Modeling**

During cellular metabolism the concentration of enzymes and metabolites are continuously adjusted in order to achieve specific metabolic demands. It is highly likely that during evolution global metabolic regulation has evolved such as to achieve a given metabolic demand with an optimal use of intracellular resources. However, the size of enzymes and intermediate metabolites are dramatically different. Enzymes are macromolecules that occupy a relatively large amount of space within a cell's crowded cytoplasm, while metabolites are much smaller. This implies that metabolite concentrations are likely to be adjusted to minimize the overall ‘enzymatic cost’ (in terms of space cost).

Here we study the validity of this hypothesis by focusing on the glycolysis pathway of the yeast, Saccharomyces cerevisiae, for which a kinetic model is available. We show that the maximum glycolysis rate determined by the limited solvent capacity is close to the values measured in vivo. Furthermore, the measured concentration of intermediate metabolites and enzyme activities of glycolysis are in agreement with the predicted values necessary to achieve this maximum glycolysis rate. Taken together these results indicate that the limited solvent capacity constraint is relevant for S. cerevisiae at physiological conditions. From the modeling point of view, this work demonstrates that a full kinetic model together with the limited solvent capacity constraint can be used to predict not only the metabolite concentrations, but in vivo enzyme activities as well.

**Methods**

Kinetic model of glycolysis: We use the S. cerevisiae glycolysis model reported in Hynne F, Dano S, Sorensen P G (2001) Full-scale model of glycolysis in Saccharomyces cerevisiae. Biophys Chem 94: 121-163 (see below for details). The only modification is the extension of the phsophofructokinase (Pfk) kinetic model from an irreversible to a reversible model.

FIG. 21 is a schematic representation of saccharomyces cerevisiae glycolysis.

The optimization objective is the glycolysis rate

\(\begin{matrix}
{\frac{R}{1 - \varphi} = \frac{1}{\sum\limits_{i = 2}^{10}{a_{i}r_{i}}}} & \lbrack 12\rbrack
\end{matrix}\)

where Φ is the fraction of cell volume occupied by cell components other than glycolysis enzymes, ri is the rate of the i-th reaction relative to the glycolysis rate,

\(\begin{matrix}
{a_{i} = \frac{v_{spec}\mu_{i}\rho}{x_{i}k_{i}}} & \lbrack 13\rbrack
\end{matrix}\)

is the crowding coefficient associated with the i-th reaction, vspec is the specific volume, and μi and ki are the molar mass and catalytic constant of the enzyme catalyzing the i-th reaction. Note that the transport (3) and storage (11) reactions have been excluded. The former does not contribute to the cytoplasm crowding and the latter is considered and step outside glycolysis.

Given the storage rate and the concentration of ATP, the rate equation [24] below determines the concentration of G6P. Furthermore, given this G6P concentration, and the concentration of extracellular glucose, ATP and ADP, the rate equation [14] determines the concentration of intracellular glucose. The remaining metabolite concentrations are obtained such that to maximize [12].

The following are rate equation models, as reported in Hynne F, et al., (2001) Biophys Chem 94: 121-163.

Glucose transport (TRANS):

\(\begin{matrix}
{v_{1} = \begin{matrix}
{1 + \frac{\lbrack{GLCx}\rbrack}{K_{1{GLC}}} +} \\
{\frac{\frac{{P_{1}\frac{\lbrack{GLCx}\rbrack}{K_{1{GLC}}}} + 1}{{P_{1}\frac{\lbrack{GLC}\rbrack}{K_{1{GLC}}}} + 1}\left( {1 + \frac{\lbrack{GLC}\rbrack}{K_{1{GLC}}} + \frac{\left\lbrack {G\; 6P} \right\rbrack}{K_{1{IGLC}}} + \frac{\lbrack{GLC}\rbrack \left\lbrack {G\; 6P} \right\rbrack}{K_{1{GLC}}K_{1{IIGLC}}}} \right)}{\frac{\lbrack{GLCx}\rbrack - \lbrack{GLC}\rbrack}{K_{1{GLC}}}}V_{1,\max}}
\end{matrix}} & \lbrack 14\rbrack
\end{matrix}\)

Hexokinase (HK):

\(\begin{matrix}
{x_{2} = \frac{\begin{matrix}
{{K_{2{DGLC}}K_{2{ATP}}} + {K_{2{GLC}}\lbrack{ATP}\rbrack} +} \\
{{K_{2{ATP}}\lbrack{GLC}\rbrack} + {\lbrack{GLC}\rbrack \lbrack{ATP}\rbrack}}
\end{matrix}}{\lbrack{ATP}\rbrack \lbrack{GLC}\rbrack}} & \lbrack 15\rbrack
\end{matrix}\)

Phosphoglucoisomerase (PGI):

\(\begin{matrix}
{x_{3} = \frac{K_{3\; G\; 6P} + \left\lbrack {G\; 6P} \right\rbrack + {\frac{K_{3G\; 6P}}{K_{3\; F\; 6P}}\left\lbrack {F\; 6P} \right\rbrack}}{\left\lbrack {G\; 6P} \right\rbrack - \frac{\left\lbrack {F\; 6P} \right\rbrack}{K_{3{eq}}}}} & \lbrack 16\rbrack
\end{matrix}\)

Phosphofructokinase-1 (PFK):

\(\begin{matrix}
{x_{4} = \frac{{K_{5F\; 6P}\left( {1 + {\kappa_{4}\frac{\lbrack{ATP}\rbrack^{2}}{\lbrack{ADP}\rbrack^{2}}}} \right)} + \left( {\left\lbrack {F\; 6P} \right\rbrack + {\frac{K_{4F\; 6P}}{K_{4{FBP}}}\lbrack{FBP}\rbrack}} \right)^{2}}{\left( {\left\lbrack {F\; 6P} \right\rbrack + \frac{\lbrack{FBP}\rbrack}{K_{4{eq}}}} \right)\left( {\left\lbrack {F\; 6P} \right\rbrack + {\frac{K_{4F\; 6P}}{K_{4{FBP}}}\lbrack{FBP}\rbrack}} \right)}} & \lbrack 17\rbrack
\end{matrix}\)

This reaction is generally considered as irreversible. Ignoring its reversibility would result, however, in infinitely large values for FBP. Therefore, we have made the reversible extension of this model following Hofmeyr J-H et al., (1997) The reversible Hill equation: how to incorporate cooperative enzymes into metabolic models. Compt Appl Biosci 13: 377-385.

Aldolase (ALD):

\(\begin{matrix}
{x_{5} = \frac{\begin{matrix}
{K_{5{FBP}} + \lbrack{FBP}\rbrack + \frac{\lbrack{GAP}\rbrack K_{5{DHAP}}V_{5f}}{K_{5{eq}}V_{5r}} +} \\
{\frac{\lbrack{DHAP}\rbrack K_{5{GAP}}V_{5f}}{K_{5{eq}}V_{5r}} + \frac{\lbrack{FBP}\rbrack \lbrack{GAP}\rbrack}{K_{5{IGAP}}} + \frac{{\lbrack{GAP}\rbrack \lbrack{DHAP}\rbrack}V_{5f}}{K_{5{eq}}V_{5r}}}
\end{matrix}}{\lbrack{FBP}\rbrack - \frac{\lbrack{GAP}\rbrack \lbrack{DHAP}\rbrack}{K_{5{eq}}}}} & \lbrack 18\rbrack
\end{matrix}\)

Triosephosphate isomerase (TPI):

\(\begin{matrix}
{x_{6} = \frac{K_{6{DHAP}} + \lbrack{DHAP}\rbrack + {\frac{K_{6{DHAP}}}{K_{6{GAP}}}\lbrack{GAP}\rbrack}}{\lbrack{DHAP}\rbrack - \frac{\lbrack{GAP}\rbrack}{K_{6{eq}}}}} & \lbrack 19\rbrack
\end{matrix}\)

Glyceraldehyde 3-phosphate dehydrogenase (GAPDH):

\(x_{7} = {\frac{K_{7{GAP}}{K_{7{NAD}}\left( {1 + \frac{\lbrack{GAP}\rbrack}{K_{7{GAP}}} + \frac{\lbrack{BGP}\rbrack}{K_{7{BGP}}}} \right)}\left( {1 + \frac{\lbrack{NAD}\rbrack}{K_{7{NAD}}} + \frac{\lbrack{NADH}\rbrack}{K_{7{NADH}}}} \right)}{{\lbrack{GAP}\rbrack \lbrack{NAD}\rbrack} - \frac{\lbrack{BGP}\rbrack \lbrack{NADH}\rbrack}{K_{7{eq}}}}\mspace{45mu}\lbrack 20\rbrack}\)

(lpPEP):

v8=k8 f[BGP][ADP]−k8 r[PEP][ATP]  [21]

Pyruvate kinase (PK):

\(\begin{matrix}
{x_{9} = \frac{\left( {K_{9{PEP}} + \lbrack{PEP}\rbrack} \right)\left( {K_{9{ADP}} + \lbrack{ADP}\rbrack} \right)}{\lbrack{ADP}\rbrack \lbrack{PEP}\rbrack}} & \lbrack 22\rbrack
\end{matrix}\)

Glycerol 3-phosphate dehydrogenase (G3PDH):

\(\begin{matrix}
{x_{10} = \frac{\begin{matrix}
{{K_{10{DHAP}}\left( {1 + {\frac{K_{15{INADH}}}{\lbrack{NADH}\rbrack}\left( {1 + \frac{\lbrack{NAD}\rbrack}{K_{10{INAD}}}} \right)}} \right)}\lbrack{DHAP}\rbrack} \\
\left( {1 + {\frac{K_{15{NADH}}}{\lbrack{NADH}\rbrack}\left( {1 + \frac{\lbrack{NAD}\rbrack}{K_{10{INAD}}}} \right)}} \right)
\end{matrix}}{\lbrack{DHAP}\rbrack}} & \lbrack 23\rbrack
\end{matrix}\)

Storage

v11=k11[ATP][G6P]  [24]

The catalytic constants were obtained from experimental estimates for Saccharomyces carlsbergensis (Boiteux et al., Design of glycolysis, Phil Trans R Soc Lond B, 1981, 293, pgs. 5-22), except for g3pdh that was obtained from an estimate for Eidolon helvum.

III. Experimental Data used in the Comparison with the Theoretical Predictions

Catalytic constants, cell density, specific volume: The catalytic constants were obtained from experimental estimates for Saccharomyces carlsbergensis (Boiteux et al., Design of glycolysis, Phil Trans R Soc Lond B, 1981, 293, pgs. 5-22), except for glycerol 3-phosphate dehydrogenase that was obtained from an estimate for Eidolon helvum (Schomburg I, Nucleic Acids Res 30: 47-49). For the cell density we use an estimate reported for E. coli, p=0.34 g/ml (Zimmerman S B, et al. (1991) J Mol Biol 222: 599-620.). The specific volume was estimated for several proteins using the molar volumes and masses reported in (Lee B (1983) Proc Natl Acad Sci USA 80: 622-626), resulting in average of 0.73 ml/g, and a standard deviation of 0.02 ml/g.

Optimal metabolite concentrations: The optimal metabolite concentrations are obtained maximizing [15] with respect to the free metabolite concentrations. In the case of FIG. 2b,c,d, all metabolite concentrations are fixed to their experimental values, except for the metabolite indicated by the X-axis. In the case of FIG. 3a,b, all intermediate metabolite concentrations are optimized, keeping fixed the concentration of external glucose and co factors (ATP, ADP, AMP, NADH, NAD). In both cases the reaction rates relative to the glycolysis rate (ri) were taken as input parameters, using the values reported in Hynne F, et al. Biophys Chem 94: 121-163. The maximization was performed using simulated annealing (Press W H, et al. (1993) Numerical recipes in C: The art of scientific computing. Cambridge: Cambridge University Press).

Parameter sensitivity: To analyze the sensitivity of our predictions to the model parameters we have generated random sets of kinetic parameters, assuming a 10% variation of the fixed metabolite concentrations and all kinetic constants except for the catalytic activities. For the latter we assumed a larger variation of 50%, because they were estimated from a different organism. For each set of parameters we make predictions for the metabolite concentrations and enzyme activities. FIG. 24 reports the mean values and standard deviations.

**Results**

Limited solvent capacity constraint: The cell's cytoplasm is characterized by a high concentration of macromolecules resulting in a limited solvent capacity for the allocation of metabolic enzymes. More precisely, given that enzyme molecules have a finite molar volume vi only a finite number of them fit in a given cell volume V. Indeed, if ni is the number of moles of the ith enzyme, then

Σi=1Nvini+V0=V,   [25]

where Vo accounts for the volume of other cell components and the free volume necessary for cellular transport as well. Equation (14) can be also rewritten as

vspecΣi=1Nρi=1−φ,   [26]

where ρi=niμi/V is the enzyme density (enzyme mass/volume), μi is the molar mass vspec is the specific volume, and Φ=Vo/V is the fraction of cell volume occupied by cell components other than the enzymes catalyzing the reactions of the pathway under consideration, including the free volume necessary for diffusion. The specific volume has been assumed to be constant for all enzymes, an approximation that has been shown to be realistic at least for globular proteins. In this new form we can clearly identify the enzyme density (or mass, given that the volume is constant) as the enzyme associated variable contributing to the solvent capacity constraint. This choice is more appropriate than the enzyme concentration Ci,=ni/V (moles/volume) because the specific volume is approximately constant across enzymes, while the molar volume can exhibit significant variations. For example, according to experimental data for several globular proteins the molar volume exhibits a 70% variation while the specific volume is almost constant, with a small 2% variation.

The solvent capacity constraint thus imposes a limit to the amount of catalytic units (i.e., enzymes) that can be allocated in the cell cytoplasm. In the following we show that this in turn leads to a constraint for the maximum metabolic rate. The rate of the ith reaction per unit of cell dry weight (mol/time/mass) is given by

\(\begin{matrix}
{{R_{i} = {{x_{i}A_{i}} = {\frac{x_{i}k_{i}C_{i}}{M} = \frac{x_{i}k_{i}\mu_{i}\rho_{i}}{M}}}},} & \lbrack 27\rbrack
\end{matrix}\)

where Ai is the specific enzyme activity, Ci is the enzyme concentration in molar units, ki is the catalytic constant and M is the cell mass. The coefficient xi is determined by the specific kinetic model: it takes values in the range of 0≦xi≦1, and it is a function of metabolite concentrations. For example, if the ith reaction is described by Michaelis-Menten kinetics with one substrate then x,=S;I(Ki+Si), where Si is the substrate concentration and ki is the equilibrium constant. More generally, xi is a function of the concentration of substrates, products and other metabolites regulating the enzyme activity. The fact that the reaction rates are roughly proportional to the enzyme concentrations [27] suggests that the limited solvent capacity constraint [26] has an impact on the reaction rates as well. Indeed, from equations [26 and [27], we obtain

\(\begin{matrix}
{{R = \frac{1 - \varphi}{\sum\limits_{i = 1}^{N}{a_{i}r_{i}}}},} & \lbrack 28\rbrack
\end{matrix}\)

where R is the cell metabolic rate (or pathway rate), ri=Ri/lR is the rate of reaction i relative to the metabolic rate, and

\(\begin{matrix}
{{a_{i} = \frac{v_{spec}\mu_{i}\rho}{x_{i}k_{i}}},} & \lbrack 29\rbrack
\end{matrix}\)

where p=M/V is the cell density. We refer to ai as the crowding coefficients because they quantify the contribution of each reaction rate to molecular crowding. The crowding coefficient of a reaction i increases with increasing the enzyme's molar mass μi and decreases with increasing catalytic activity ki. It is also a function of the metabolite concentrations through xi.

Hypothetical three metabolites pathway: To illustrate the impact of the limited solvent capacity constraint, we first analyze a hypothetical example, in which we use the relative reaction rates as input parameters, and the metabolite concentrations are the variables to be optimized. Given the reaction rates and the “optimal” metabolite concentrations, the enzyme activities are determined by equation [27]. Finally, the maximum metabolic rate is computed using equation [28].

Consider a metabolic pathway consisting of two reversible reactions converting metabolite M1 into M2 (reaction 1) and M2 into M3 (reaction 2), catalyzed by enzymes e1 and e2, respectively (FIG. 22, inset). The reaction rates per unit of cell mass, R1 and R2, are modeled by reversible Michaelis-Menten rate equations

\(\begin{matrix}
{{R_{1} = {\frac{\left\lbrack M_{1} \right\rbrack - {\left\lbrack M_{2} \right\rbrack/K_{1\; {eq}}}}{K_{11} + \left\lbrack M_{1} \right\rbrack + {{K_{11}\left\lbrack M_{2} \right\rbrack}/K_{12}}}K_{1}C_{1}\frac{1}{M}}},} & \lbrack 30\rbrack \\
{R_{2} = {\frac{\left\lbrack M_{2} \right\rbrack - {\left\lbrack M_{3} \right\rbrack/K_{2\; {eq}}}}{K_{22} + \left\lbrack M_{2} \right\rbrack + {{K_{22}\left\lbrack M_{1} \right\rbrack}/K_{23}}}K_{2}C_{2}\frac{1}{M}}} & \lbrack 31\rbrack
\end{matrix}\)

where K1eq and K2eq are the equilibrium constants of reaction 1 and 2, respectively, Kim is the Michaelis-Menten constant of metabolite m in reaction i, M is the cell mass and Ci is the concentration of the ith enzyme. For the purpose of illustration, we assume that the kinetic parameters associated with both reactions are the same, all Michaelis constants equal to 1 mM, and fixed pathway ends metabolite concentrations [M1]=3 mM and [M2]=1 mM. Furthermore, mass conservation for M2 implies that R1=R2=R (r1=r2=1) in the steady state, where R is the pathway rate. Under these approximations the pathway rate [28] is given by

\(\begin{matrix}
{{\frac{R}{R_{0}} = \begin{bmatrix}
{\frac{K_{11} + \left\lbrack M_{1} \right\rbrack + {{K_{11}\left\lbrack M_{2} \right\rbrack}/K_{12}}}{\left\lbrack M_{1} \right\rbrack - {\left\lbrack M_{2} \right\rbrack/K_{1{eq}}}} +} \\
\frac{K_{22} + \left\lbrack M_{2} \right\rbrack + {{K_{22}\left\lbrack M_{3} \right\rbrack}/K_{23}}}{\left\lbrack M_{2} \right\rbrack - {\left\lbrack M_{3} \right\rbrack/K_{2{eq}}}}
\end{bmatrix}^{- 1}},} & \lbrack 32\rbrack
\end{matrix}\)

where Ro=(1−Φ)k,lvspecμ1p. At this point the value of Ro is not needed and we focus on the relative pathway rate R/Ro. When reaction 1 is close to equilibrium [M2]≈[M1]K1eq=3 mM, the first term in the right hand side becomes very large, resulting in a small pathway rate (FIG. 22). When reaction 2 is close to equilibrium [M2]≈[M3]/K2eq=1 mM, the second term in the right hand side becomes very large, again resulting in a small pathway rate (FIG. 22). At an intermediate [M2]* between these two extremes the pathway rate achieves its maximum. Therefore, given the solvent capacity constraint, there is an optimal metabolite concentration resulting in a maximum pathway rate.

S. cerevisiae glycolysis: Next, we investigated whether the observation of an optimal metabolite concentration for maximum pathway rate extrapolates to a more realistic scenario. For this purpose we use the glycolysis pathway of the yeast S. cerevisiae (FIG. 23A) as a case study. Glycolysis represents a universal pathway for energy production in all domains of life. In S. cerevisiae it has been studied extensively resulting in the description of a rate equation model for each of its reactions. In particular, we consider the kinetic model developed in Hynne F, et al. (2001) (see Methods). To compare our predictions with experimentally determined values we consider the glycolysis reaction rates and metabolite concentrations reported in Hynne F et al. (2001) and the enzyme activities reported in (Teusink B, et al. (2000).

In analogy with the three metabolites case study (FIG. 22), first we investigate the dependency of the glycolysis rate R on the concentration of a given metabolite. In this case we fix all other metabolite concentrations and all relative reaction rates to their experimentally determined values. By doing so the predicted glycolysis rate is an implicit function of the free metabolite concentration alone, through equation [28]. For example, FIG. 23B displays the maximum metabolic rate R as a function of the concentration of fructose-6-phosphate (F6P). R is predicted to achieve a maximum around a F6P concentration of 0.4 mM, close to its experimentally determined value of 0.5 mM (red triangle in FIG. 23B). Similar conclusions are obtained for D-glyceraldehyde-3phosphate (GAP) (FIG. 23C) and glycerone-phosphate (DHAP) (FIG. 23D). This analysis corroborates that there is an optimal metabolite concentration maximizing R and, more importantly, that this concentration is very close to the experimentally determined metabolite concentrations. In all cases the measured metabolite concentrations are within the range of 50% or more of the maximum glycolysis rate.

To further test the optimal metabolite concentration hypothesis, we perform a global optimization and simultaneously compute the optimal concentrations of the glycolysis intermediate metabolites. In this case we fix the concentrations of external glucose and co-factors and all relative reaction rates to their experimentally determined values. By doing so the predicted glycolysis rate is an implicit function of the glycolysis intermediate metabolite concentrations, through Equation [28]. The optimal intermediate metabolite concentrations are those maximizing Equation [28]. FIG. 24A displays the predicted optimal metabolite concentrations as a function of their experimentally determined values (black symbols), the line representing a perfect match. The agreement is remarkably good given the wide range of metabolite concentrations. For phospho-enolpyruvate (PEP), the predicted value is very sensitive to the model parameters, as indicated by the wide error bars. For fructose 1,6-biphosphate (FBP) the predicted value is smaller by a factor of five than the experimental value, but it is still within range. Taken together, these results indicate that the measured concentrations of intermediate metabolites in the S. cerevisiae glycolysis are close to the predicted optimal values maximizing the glycolysis rate given the limited solvent capacity constraint.

Using the optimal intermediate metabolite concentrations we can make predictions for the enzyme activities as well. Indeed, from the first equality in [27] it follows that

\(\begin{matrix}
{\frac{A_{i}}{R} = {\frac{r_{i}}{x_{i}}..}} & \lbrack 33\rbrack
\end{matrix}\)

The reaction rates relative to the glycolysis rate ri are obtained from experimental data, while xi are obtained after susbtituting the predicted optimal metabolite concentrations on the reaction's kinetic models. FIG. 24B displays the predicted enzyme activities (in units of the glycolysis rate) as a function of the experimentally determined values (black symbols), the line representing a perfect match. In most cases we obtain a relatively good agreement between experimentally measured and predicted values, with the exception of phosphofructokinase (Pfk), for which the measured enzyme activities are significantly overestimated. Of note, for pyruvate kinase (pk) the predictions are significantly affected by the model parameters, as indicated by the wide error bars.

The preceeding analysis does not exclude the possibility that other constraints could result in a good agreement as well. To address this point we consider the more general optimization objective R=(1−φ)/Σi=1N(airi)H [34], parametrized by the exponent H. Although this objective is not inspired by some biological intuition, it allows to explore other possibilites beyond the original case H=1. FIG. 24 shows our predictions for the case H=0.1 (red symbols) and H=10 (blue symbols), representing a milder and a stronger dependency with the crowding coefficient ai, respectively. For H=0.1, 1.0 and 10 the predicted metabolite concentrations are in good agreement with the experimental values. These results indicate that it is sufficient that the optimization objective is a monotonic decreasing function of the crowding coefficients. When the latter is satisfied the metabolite concentrations are up to a great extent constrained by the kinetic model. This is not, however, the case for the enzyme activities. For H=0.1 and the enzymes pfk, tpi and pk, there is a large deviation from the perfect match line. For H=0 and the enzymes (tpi and pk, there is a large deviation from the perfect match line as well. Overall, H=1 gives the better agreement between enzyme activity predictions and their measured values. In addition, it provides a clear biophysical interpretation of the solvent capacity constraint (H=1).

Finally, we use equation [28] to compute the maximum glycolysis rate as determined by the limited solvent capacity constraint. The global optimization predicts the glycolysis rate R=(1−Φ)×12.5 mmol/min/g dry weight. Taking into account that about 30% of the cell is occupied by cell components excluding water, that proteins account for ˜45% of the dry weight, and that of these glycolysic enzymes account for ˜22% of the protein mass we obtain l−Φ0.03. Therefore, given that glycolysis enzymes occupy only 3% of the cell volume, we obtain R˜0.38 mmol/min/g dry weight. This prediction is in very good agreement with the experimentally determined glycolysis rate of S. cerevisiae, ranging between 0.1 to 0.4 mmol/min/g dry weight.

The successful modeling of cell metabolism requires the understanding of the physicochemical constraints that are relevant at physiological growth conditions. In our previous work focusing on E. coli we have reported results indicating that the limited solvent capacity is an important constraint on cell metabolism, especially under nutrient rich growth conditions. Using a flux balance approach that incorporates this constraint we predicted the maximum growth rate in different carbon sources, the sequence and mode of substrate uptake and utilization from a complex medium, and the changes in intracellular flux rates upon varying E. coli cells' growth rate. More importantly, these predictions were in good agreement with experimentally determined values.

Here we have extended the study of the impact of the limited solvent capacity by (i) considering a different organism (S. cerevisiae), and (ii) a full kinetic model of glycolysis. Using the full kinetic model of S. cerevisiae glycolysis, we have demonstrated that the predicted optimal intermediate metabolite concentrations and enzyme activites are in good agreement with the corresponding experimental values. Discrepancies were only observed in association with two different steps in the glycolysis pathway, namely the reaction catalyzed by pjk and the reactions between BPG and PEP. It cannot be excluded that these discrepancies are associated with deficiencies of the kinetic model. Furthermore, the glycolysis rate achieved at the optimal metabolite concentrations is in the range of the experimentally measured values.

From the quantitative modeling point of view our results indicate that a full kinetic model together with the solvent capacity constraint can be used to make predictions for the metabolite concentrations and enzyme activities. Thus, we propose the simultaneous optimization of intermediate metabolite concentrations, maximizing the metabolic rate given the solvent capacity, as a method to computationally predict the concentrations of a metabolic pathway's intermediate metabolites and enzyme activities. We have demonstrated the applicability of this method by computing the concentration of S. cerevisiae glycolysis intermediate metabolites, resulting in a good agreement with published data.

The high concentration of macromolecules in the cell's cytoplasm imposes a global constraint on the metabolic capacity of an organism. No clear explanation has been provided to support that choice. Our starting postulate is an undeniable physical constraint, the total cell volume. Under this constraint, the enzyme molar volumes are the primary magnitude quantifying the enzymatic cost. In turn, since the enzyme-specific volumes are approximately constant, we can use the enzyme density (mass/volume) as an alternative measure of enzymatic cost. This modeling framework has advantages and disadvantages with respect to more traditional approaches based on dynamical systems modeling. As an advantage, our method does not require as input parameters the enzyme activities but rather make quantitative predictions for them. On the other hand, our method is based on a steady state approximation. Therefore, in its present form, it cannot be used to understand dynamical processes, such as the observed metabolite concentration oscillations in S. cerevisiae cells when growing at high glucose concentrations.

### Example 4

**Serine Biosynthesis with One Carbon Catabolism and the Glycine Cleavage System Represents a Novel Pathway for ATP Generation**

Previous experimental evidence indicates that some cancer cells have an alternative glycolysis pathway with net zero ATP production, implying that upregulation of glycolysis in these cells may not be related to the generation of ATP. Here we use a genome-scale model of human cell metabolism to investigate the potential metabolic alterations in cells using net zero ATP glycolysis. We uncover a novel pathway for ATP generation that involves reactions from the serine biosynthesis, one-carbon metabolism and glycine cleavage systems, and show that the pathway is transcriptionally upregulated in an inducible murine model of Myc-driven liver tumorigenesis. This pathway has a predicted two-fold higher flux rate in cells using net zero ATP glycolysis than those using standard glycolysis and generates twice as much ATP with significantly lower rate of lactate, but a higher rate of alanine secretion. Thus, in cells using the standard- or the net zero ATP glycolysis pathways a significant portion of the glycolysis flux is always associated with ATP generation, and the ratio between the flux rates of the two pathways determines the rate of ATP generation and lactate and alanine secretion during glycolysis.

Oxidative phosphorylation (OxPhos) in the mitochondria is the major pathway for ATP generation in normal cells under normal oxygen conditions (normoxia), generating 32 mole of ATP per mole of glucose. In contrast, under conditions of oxygen limitation (hypoxia), the mitochondrial activity is down-regulated and cells switch to glycolysis for ATP generation that yields only 2 mole of ATP per mole of glucose. Surprisingly, as first observed by Warburg, the metabolism of cancer cells is frequently characterized by a significant upregulation of glycolysis even under normoxic conditions, with both an increased glucose uptake and excretion of lactate (Warburg effect, aerobic glycolysis). More recently, it became evident that the Warburg effect is not unique to cancer cells alone. Indeed, both rapidly proliferating normal cells and non-proliferating cells with high metabolic activity display high levels of glycolysis with lactate excretion under normoxic conditions. Despite the importance of OxPhos and aerobic glycolysis in ATP generation, previous empirical evidence indicates that some cancer cells also utilize an alternative glycolysis pathway with net zero ATP generation. This striking observation implies a physiological role for aerobic glycolysis other than ATP generation. One such role may be the capacity of glycolysis to fulfill the need of rapidly proliferating cells for precursor metabolites. However, it has been shown previously that the need for precursor metabolites in itself is not sufficient to explain the high glycolysis rates observed in proliferating cells. Instead, molecular crowding and its resulting constraint on macromolecular concentrations is the key factor determining the Warburg effect. The high density of macromolecules in the cell imposes limits on the total mitochondrial content per unit of cell volume and the total content of ribosomes and metabolic enzymes as well. In turn, the inherent limitation in mitochondrial density results in an upper bound on the maximum achievable OxPhos capacity. We have shown previously that this maximum is achieved at physiological conditions and that it results in a metabolic switch involving an upregulation of glycolysis and lactate excretion. Yet, all these results were obtained making use of the standard glycolysis pathway, with a yield of 2 moles of ATP per mole of glucose.

Here we investigate the metabolic flux redistributions in proliferating cells that utilize the alternative glycolysis pathway with net zero ATP production. To this end we improve on our previous flux balance model of human cell metabolism by more precisely accounting for protein synthesis, including a self-consistent constraint that all ribosomal-, enzyme associated-, and non-metabolic proteins need to be accounted for by the rate of protein synthesis, which is proportional to the ribosomal density. We also make a more precise accounting of the molecular crowding constraint by considering mitochondria as a subcellular compartment independent from the cytosol. Using this model we uncover a novel pathway for ATP generation that involves reactions in the serine biosynthesis, one-carbon metabolism and the glycine cleavage system, and show that the pathway is transcriptionally upregulated in an inducible murine model of Myc driven liver tumorigenesis. The flux rate of this pathway is predicted to be two-fold higher in cells with net zero ATP glycolysis relative to cells with the standard glycolysis. Furthermore, it accounts for most of the glycolysis rate in cells with net zero ATP glycolysis.

**Results**

As a starting point, we utilize a genome-scale metabolic reconstruction of a generic human cell that includes most biochemical reactions catalyzed by enzymes encoded in the human genome essentially as described in Mo M L, et al. ((2007) A genome-scale, constraint-based approach to systems biology of human metabolism. Mol Biosyst 3: 598-60). We add auxiliary reactions to represent nutrient uptake, excretion of metabolic byproducts, basal ATP demand needed for cell maintenance, basal rate of protein degradation, synthesis of cell biomass components (proteins, lipids, RNA and DNA) and cell proliferation (biomass components→cell). We assume that the cell is in a steady state where the production and consumption of every metabolite and macromolecules balances, known as the flux balance constraint. We use Smi to denote the stoichiometric coefficient of metabolite m in reaction i. We use fi to denote the steady state reaction rate (flux) of the ith reaction in the metabolic network, where all reversible reactions are represented by a forward and backward rate, respectively. Reactions are divided into nutrient import reactions (RI), reactions taking place outside the mitochondria (RnM) and reactions taking place in the mitochondria (RM). We use φc to denote the relative cell volume fraction occupied by the cth cellular compartment, where a compartment represents the overall contribution of macromolecules of certain type (e.g., ribosomes) or of certain cell organelle (e.g., mitochondria). Specifically, here we consider proteins that do not form part of enzyme complexes or ribosomes (P0), all metabolic enzymes catalyzing reactions outside the mitochondria (EnM), all metabolic enzymes catalyzing reactions in the mitochondria (EM), ribosomes (R), and mitochondria (M). We assume the proliferation rate (μ) and the total relative volume fraction occupied by macromolecules and organelles (φmax) are known and are given as input parameters of the model Finally, we estimate the metabolic fluxes and compartment densities as the solution of the following optimization problem:

find the fi and φc that minimize the sum of nutrient import costs:

\(\begin{matrix}
{\sum\limits_{i \in {RI}}{c_{i}f_{i}}} & \lbrack 35\rbrack
\end{matrix}\)

subject to the metabolic constraints: flux balance constraints:

\(\begin{matrix}
{{\sum\limits_{i}{S_{mi}f_{i}}} = 0} & \lbrack 36\rbrack
\end{matrix}\)

minimum/maximum flux constraints:

vi,min≦fi≦vi,max   [37]

minimum/maximum volume fraction constraints:

0≦φc≦φmax   [38]

molecular crowding constraints:

\(\begin{matrix}
{{{\sum\limits_{i \in {RnM}}{a_{i}f_{i}}} \leq \varphi_{EnM}}{{\sum\limits_{i \in {RM}}{a_{i}f_{i}}} \leq \varphi_{M}}{{a_{M,{ATP}}f_{ATP\_ synthase}} \leq \varphi_{M}}{{a_{R}f_{protein\_ synthesis}} \leq \varphi_{R}}{{\varphi_{P\; 0} + \varphi_{EnM} + \varphi_{R} + \varphi_{M}} \leq \varphi_{\max}}} & \lbrack 39\rbrack
\end{matrix}\)

where ci is the nutrient import cost associated with the uptake reaction i, ai=vi/keff,i are the crowding coefficients of metabolic enzymes (enzyme molar volume/enzyme effective turnover), aR=vR/kR is the ribosome crowding coefficient (ribosome molar volume/protein synthesis rate per ribosome), and aM,ATP=vs,M/rM the crowding coefficient of mitochondria ATP generation (ATP synthesis rate per mitochondria mass/mitochondria specific volume).

The estimation of all the model parameters is presented in the Methods section. Here we discuss some of them that deserve particular attention. The cost of importing molecules is in general determined by their size, charge and hydrophilic properties. In a first approximation we assume that the cost of importing molecules is proportional to their size and use the molecular weight as a surrogate for size. Within this approximation the import cost ci is equal to the molecular weight of the molecule imported in the auxiliary uptake reaction i. The effective turnover numbers keff, i, quantify the reaction rate per enzyme molecule. For example, for an irreversible single substrate reaction satisfying Michaelis-Menten kinetics, keff=kS/(K+S), where k is the enzyme turnover number, K the half-saturation concentration and S the substrate concentration.

The turnover numbers of some human enzymes are reported in the BRENDA database. They have a typical value of 10 sec−1 and a significant variation from 1 to 100 sec−1. However, for most reactions we do not know the turnover number, the kinetic model, or the metabolite concentrations, impeding us to estimate keff. To cope with this indeterminacy we performed a sampling strategy, whereby the keff,i were sampled from a reasonable range of values, and then focused on the predicted average behavior and 90% confidence intervals (see Methods for additional details). The typical enzyme crowding coefficient is about ai˜0.00013 (mM/min)−1, which is interpreted as follows: to maintain a reaction rate of 1 mM/min we need to allocate a relative cell volume of 0.00013 (0.013%) for the corresponding enzyme. The crowding coefficients are significantly larger for ribosomes and the mitochondria: aR=3.6 (mM/min)−1 and aM,ATP=0.017 (mM/min)−1, respectively.

The flux balance equation for proteins (equation (36) with m=proteins) is formulated more generally than before. Previous models have assumed a constant protein concentration and have not taken into account the self-consistent need to synthesize all the proteins in enzyme complexes and ribosomes. In contrast, here we account for three major categories, proteins not associated with metabolism, proteins that are components of enzyme complexes, and ribosomal proteins, with their concentrations (moles/cell volume) denoted by P0, PE, and PR, respectively. In proliferating cells, these concentrations will decrease at a rate (μ+kD)(P0+PE+PR), where μ denotes the proliferation rate and kD the basal rate of protein turnover. The total concentration of proteins in enzyme complexes can be estimated as PE=nPEE=nPEΣifi/keff,i, where nPE is the average number of proteins in an enzyme complex (about 2.4) and E is the total concentration of metabolic enzymes. Similarly, PR=nPRφR/vR, where nPR is the number of proteins in a ribosome (82 for the 80S ribosomes) and φR/vR is the concentration of ribosomes. Putting all these elements together, the balance between protein turnover and synthesis implies fProtein_synthesis=(μ+kD)[P0+nPEΣifi/keff,i+(nPR/vR)φR], where the term (μ+kD)P0 is the only one considered in previous models. In an effective manner, each metabolic reaction contributes to a component of protein synthesis, with a stoichiometric coefficient (μ+kD)nPE/keff,i quantifying the amount of protein necessary to keep the concentration of the corresponding enzyme constant. Similarly, a constant ribosome volume fraction also accounts for a component of protein synthesis, with a stoichiometric coefficient (μ+kD)nPR/vR quantifying the amount of protein necessary to keep the ribosomes concentration constant. The rate of protein synthesis accounting for this effective protein dilution/degradation thus models the autocatalytic nature of cell metabolism, whereby the macromolecular complexes catalyzing the metabolic reactions ultimately are themselves a product of metabolism.

To model the alternative glycolysis pathway we replaced the pyruvate kinase catalyzed reaction: Phosphoenolpyruvate+ADP+H+→Pyruvate+ATP, by the ATP independent reaction: Phosphoenolpyruvate+H+→Pyruvate+Phosphate. In the latter reaction phosphoglycerate mutase (PGM1) is a candidate phosphate acceptor, which is then dephosphorylated by a yet unknown mechanism.

Changes in the Relative Macromolecular Densities with Increased Cell Proliferation

FIG. 25 shows the predicted relative volume fraction occupied by non-mitochondrial Enzymes (φEnM=Σi|EnMaifi), mitochondria (φM), and ribosomes (φR) as a function of the proliferation rate for cells growing in a medium containing glucose, glutamine, essential amino acids, and oxygen. The ribosomal density increases monotonically with increasing the proliferation rate in a nonlinear fashion, reaching a maximum of 10% of the cell volume at the highest proliferation rate of approximately 2.79/day (minimum doubling time of ln(2)/μmax=6 hours) (FIG. 25, circles). At low proliferation rates the mitochondrial density increases with increasing the proliferation rate from 10% to about 30% of the cell volume (FIG. 25, triangles). However, beyond a proliferation rate of about 0.8/day (doubling time 21 hours), the mitochondrial density decreases with increasing the proliferation rate. This is in turn accompanied by a dramatic increase in the density of metabolic enzymes (FIG. 25, squares). Our model thus predicts that when switching from low to high proliferation rates, the cell makes a transition from a mitochondria dominated molecular crowding regime to one dominated by enzymes+ribosomes (FIG. 25).

The impact of altering the different model parameters on the behavior of the model can also be tested. Larger values of the mitochondrial crowding coefficient am, e.g., due to a decrease in mitochondrial efficiency for ATP generation, will cause a decrease of mitochondrial density at lower proliferation rates. Larger ribosome crowding coefficient aR, e.g., due to a decrease in protein synthesis efficiency, will result in a faster increase of the ribosome density with increasing the proliferation rate, and a consequently faster decrease of the mitochondria density. Similarly, an increase in the average crowding coefficient of metabolic enzymes will cause a faster increase of the total enzyme concentration with increasing the proliferation, resulting in a faster decrease of the mitochondrial density as well. Cancer cells are characterized by partial alterations in all of these components, potentially resulting in a more dramatic effect than that depicted in FIG. 25. In particular, mutations leading to damaged mitochondria will enhance the effect, as originally hypothesized by Warburg.

Metabolic Switch from Low- to High Proliferation Rates

The predicted transition in the macromolecular composition of the cell is accompanied by a global switch in the cell's metabolic state (FIG. 26). At the proliferation rate of about 0.8/day (doubling time 21 hours) the model predicts a substantial increase in glucose uptake, sudden activation of glutamine uptake and α-ketoglutarate dehydrogenase activity, complete deactivation of pyruvate decarboxylase (PCm) and activation of pyruvate dehydrogenase (PDHm). The activity of pyruvate carboxylase in the low proliferation regime, where there is no glutamine uptake, is consistent with recent experimental data showing that pyruvate carboxylase is needed for growth without glutamine. The activation of glutamine uptake at high proliferation rates is also in agreement with what have been observed experimentally. We also observe activation of lactate excretion at high proliferation rates, the hallmark of the Warburg effect. Several notable changes take place around the pyruvate branching point (FIG. 26). Most noticeably, the glycolysis pathway (FIG. 26) is truncated at 3-phosphoglycerate and the flux over the ATP-decoupled pyruvate kinase-catalyzed reaction is zero at all proliferation rates. We emphasize that we have not imposed a zero flux over this reaction, and the zero flux is a prediction of the model itself. Phenylalanine, isoleucine, lysine and valine are the major sources of TCA cycle intermediates and pyruvate (via malate) at low proliferation rates. However, at high proliferation rates the TCA cycle intermediates and pyruvate are instead generated from glutamine. The cytosolic- (LDH-L) and mitochondrial L-lactate dehydrogenases (LDH-Lm) form a cycle between pyruvate and lactate. At low proliferation rates, LDH-L converts pyruvate to lactate and LDH-Lm converts lactate back to pyruvate, both reactions working at the same rate (FIG. 26, pyruvate-lactate loop, dashed lines). At high proliferation rates the cycle is reverted, LDH-L converting lactate to pyruvate and LDH-Lm pyruvate back to lactate (FIG. 26, pyruvate-lactate loop, solid lines). In the latter case the LDH-Lm catalyzed reaction has a higher rate, resulting in the net production of lactate, which is then excreted. Finally, at high proliferation alanine is produced from pyruvate and is then excreted. We note the amino acid selectivity for pyruvate and TCA cycle intermediates at low proliferation rates depends on the specific choice of nutrient import cost in equation (1). For example, assuming that the cost of nutrient uptake is equal for all nutrients, we obtain that tryptophan is utilized as a source of pyruvate at low proliferation rates (data not shown). However, the high fluxes of glutamine uptake, the 3-phosphoglycerate shift towards serine biosynthesis, and alanine excretion at high proliferation rates, and the results described below are independent of the choice of nutrient cost coefficients.

**Novel Pathway for ATP Generation**

When considering the molecular crowding constraint, our simulations show that at high proliferation rates most of the glycolysis rate is diverted towards the biosynthesis of serine (FIG. 26). However, this flux rate exceeds by more than 10 fold the serine requirements for protein synthesis (FIG. 26, serine to protein synthesis, top center panel). Therefore, we hypothesized that cells utilizing the net zero ATP glycolysis may overexpress some alternative pathway for ATP generation. To test this hypothesis we inspected the genome-scale reaction rate predictions, focusing on reactions producing ATP. Following this approach we identified the reactions with high rates of ATP production in cells with a net zero ATP glycolysis at different proliferation rates. At low proliferation rates (0.03/day, doubling time 24 days) ATP synthase was the dominant reaction, supplying most of the ATP required for cell maintenance (FIGS. 27A and 27B, left panels). On the other hand, at high proliferations rates (2.79/day, doubling time 6 hours) the formate-tetrahydrofolate ligase (FTHFL), working in the reverse direction to form ATP, is the dominant reaction (FIG. 27A, right panel). Formate-tetrahydrofolate ligase is also active in cells with the standard glycolysis (FIG. 27B, right panel). However, in the case of standard glycolysis phosphoglycerate kinase and pyruvate kinase are the dominant reactions at high proliferation rates (FIG. 27B, right panel). Finally, we note that the molecular crowding constraint is determinant in the differential utilization of ATP synthase and FTHFL at high proliferation rates (FIG. 27B, HIGH), because its removal from the model (which mathematically is equivalent to Setting φmax=∞ in equation (39)) results in a dramatic increase in the ATP synthase rate and a decrease of the FTHFL rate at high proliferation rates (FIG. 28, HIGH).

By tracking back the flux from the formate-tetrahydrofolate ligase-catalyzed reaction to glycolysis we uncovered a novel pathway for ATP generation (FIG. 29). The pathway is composed of three main steps. First, synthesis of L-serine from the glycolysis intermediate metabolite 3-phosphoglycerate, using NAD and L-glutamate as cofactors (FIG. 29B), with the overall reaction

3-phosphoglycerate+L-glutamate+NAD++H2O→L-Serine+α-Ketoglutarate+H++NADH+Phosphate   (40)

Second, the conversion of L-serine to glycine with a concomitant one-carbon metabolism cycle, resulting in the net generation of 1 mole of ATP per mole of serine transformed, using NADP+ as a cofactor (FIG. 4b), with the overall reaction

L-Serine+NADP++H2O+ADP+Phosphate→Glycine+NADPH+2 H++ATP

Finally, the conversion glycine to ammonium (NH4) in the mitochondria with a concomitant one-carbon metabolism cycle, using NAD+ and NADP+ as cofactors (FIG. 29D or 29E), with the overall reaction:

Glycine+NADP++NAD++H2O+ADP+Phosphate→NH4+CO2+NADPH+NADH+2 H++ATP.

This pathway has a net yield of 2 mole of ATP per mole of 3-phosphoglycerate, therefore 4 mole of ATP per mole of glucose. Furthermore, when compared to the standard glycolysis, the net zero ATP glycolysis manifests a significant decrease in lactate excretion while increasing the alanine excretion (FIGS. 27A AND 27B, right panel).

Although the reactions in the reaction cycle shown in FIG. 29C are all annotated as reversible in the human metabolic network reconstruction (Schellenberger J, et al. (2010) BMC Bioinformatics 11: 213), the cycle may not work in the direction of ATP production due to thermodynamic constraints. FTHFL can efficiently catalyze the synthesis of ATP in the bacterium Clostridium cylindrosporum. However, it remains to be elucidated whether this is also feasible in human cells, where the tri-functional enzyme C1-tetrahydrofolate synthase is responsible for the methylene-tetrahydrofolate dehydrogenase, methenyl-tetrahydrofolate cyclohydrolase and FTHFL activities. To address this issue, we have analyzed a kinetic model of the reaction cycle shown in FIG. 29C, focusing on the cytosolic enzymes alone. The model is described below and is based on a previous model of folate metabolism. We demonstrate that the kinetic model has a stable steady state in the direction of ATP production, indicating that the novel pathway is thermodynamically feasible.

Although the reactions in the reaction cycle shown in FIG. 29C are all annotated as reversible, the cycle may not work in the direction of ATP production due to thermodynamic constraints. As indicated above, to address this issue, we analyze a kinetic model of the reaction cycle shown in FIG. 4b, focusing on the cytosolic enzymes alone. The four reactions in FIG. 29C are modeled as reversible random multimolecular reactions:

\(\begin{matrix}
{f_{SHMT} = {\begin{Bmatrix}
{{k_{{SHMT},f}\frac{\left\lbrack {{th}\; f} \right\rbrack}{K_{{SHMT},{{th}\; f}} + \left\lbrack {{th}\; f} \right\rbrack}\frac{\left\lbrack {{ser} - L} \right\rbrack}{K_{{SHMT},{{ser} - L}} + \left\lbrack {{ser} - L} \right\rbrack}} -} \\
{k_{{SHMT},r}\frac{\left\lbrack {{mlth}\; f} \right\rbrack}{K_{{SHMT},{{mlth}\; f}} + \left\lbrack {{mlth}\; f} \right\rbrack}\frac{\lbrack{gly}\rbrack}{K_{{SHMT},{gly}} + \lbrack{gly}\rbrack}}
\end{Bmatrix}E_{SHMT}}} & (43) \\
{f_{MTHFD} = {\begin{Bmatrix}
{{k_{{MTHFD},f}\frac{\left\lbrack {{mlth}\; f} \right\rbrack}{K_{{MTHFD},{{mlth}\; f}} + \left\lbrack {{mlth}\; f} \right\rbrack}\frac{\lbrack{nadp}\rbrack}{K_{{MTHFD},{nadp}} + \lbrack{nadp}\rbrack}} -} \\
{k_{{MTHFD},r}\frac{\left\lbrack {{meth}\; f} \right\rbrack}{K_{{MTHFD},{{meth}\; f}} + \left\lbrack {{meth}\; f} \right\rbrack}\frac{\lbrack{nadph}\rbrack}{K_{{MTHFD},{nadph}} + \lbrack{nadph}\rbrack}}
\end{Bmatrix}E_{{MTHFD}\; 1}}} & (44) \\
{f_{MTHFC} = {\begin{Bmatrix}
{{k_{{MTHFC},f}\frac{\left\lbrack {{meth}\; f} \right\rbrack}{K_{{MTHFC},{{meth}\; f}} + \left\lbrack {{meth}\; f} \right\rbrack}} -} \\
{k_{{MTHFC},r}\frac{\left\lbrack {10{fth}\; f} \right\rbrack}{K_{{MTHFC},{10{fth}\; f}} + \left\lbrack {10{fth}\; f} \right\rbrack}}
\end{Bmatrix}E_{{MTHFD}\; 1}}} & (45) \\
{f_{FTHFL} = {\begin{Bmatrix}
{{k_{{FTHFL},f}\frac{\left\lbrack {10{fth}\; f} \right\rbrack}{K_{{FTHFL},{10{fth}\; f}} + \left\lbrack {10{fth}\; f} \right\rbrack}\frac{\lbrack{adp}\rbrack}{K_{{FTHFL},{adp}} + \lbrack{adp}\rbrack}\frac{\lbrack{pi}\rbrack}{K_{{FTHFL},{pi}} + \lbrack{pi}\rbrack}} -} \\
{k_{{FTHFL},r}\frac{\left\lbrack {{th}\; f} \right\rbrack}{K_{{FTHFL},{{th}\; f}} + \left\lbrack {{th}\; f} \right\rbrack}\frac{\lbrack{atp}\rbrack}{K_{{FTHFL},{atp}} + \lbrack{atp}\rbrack}\frac{\lbrack{for}\rbrack}{K_{{FTHFL},{for}} + \lbrack{for}\rbrack}}
\end{Bmatrix}E_{{MTHFD}\; 1}}} & (46)
\end{matrix}\)

where freaction denotes the net reaction rate, kreaction,f and kreaction,r forward and backward turnover numbers, Kreaction,metabolite the half-saturation constant, [metabolite] the corresponding metabolite concentration, and Eenzyme the concentration of the corresponding enzyme. Here the following abbreviations have been used: tetrahydrofolate (thf), 5,10-methylene hydrofolate (mlthf), 5,10-methenyltetrahydrofolate (methf), 10-formyltetrahydrofolate (10fthf), L-serine (ser-L), glycine (gly), formate (for), serine hydroxymethyltransferase (SHMT), methylenetetrahydrofolate dehydrogenase (MTHFD), methenyltetrahydrofolate cyclohydrolase (MTHFC), and 5-formethyltetrahydrofolate cycloligase (FTHFL).We note in human cells the MTHFD, MTHFC and FTHFL activities are carried on by a tri-functional enzyme encoded by the MTHD1. The kinetic parameter values are reported in Table S3, below. These parameters where obtained from Nijhout H F, et al. ((2004) J Biol Chem 279:55008-16) or through references cited in the BRENDA database (Scheer M, et al. (2011) Nucleic Acids Res 39:D670-6). In humans the cytosolic enzyme activities of MTHFD, MTHFC and FTHFL are carried by the tri-functional enzyme C1-tetrahydrofolate synthase, encoded by the SMTHFD1 gene. Therefore, the reaction cycle shown in FIG. 29C is regulated by the activity of two enzymes, serine hydroxymethyltransferase (SHMT) and C1-tetrahydrofolate synthase (SMTHFD1). The total concentration of these two enzymes ESHMT+SMTHFD1 determines how fast the system evolves to a steady state and the absolute rate at steady state, but it has no impact on the metabolite concentrations at steady state Taking this fact into consideration we focus on the system behavior as a function of the relative concentration of one of the enzymes, SHMT for example ESMTHFD1/(ESHMT+ESMTHFD1).

We focus our analysis on the concentrations of the intermediate metabolites thf, mlthf, methf and 10fthf, assuming the concentration of ser-L, glyc, formate, co-factors and enzymes constant, and given as input parameters of the model (Table 9, above). The concentration of intermediate metabolites evolve in time following the first order differential equations

\(\begin{matrix}
{\mspace{79mu} {\frac{\left\lbrack {{th}\; f} \right\rbrack}{t} = {{f_{FTHFL}\left( {\left\lbrack {{th}\; f} \right\rbrack,\left\lbrack {10\; {fthf}} \right\rbrack} \right)} - {f_{SHMT}\left( {\left\lbrack {{mlth}\; f} \right\rbrack,\left\lbrack {{th}\; f} \right\rbrack} \right)}}}} & (47) \\
{\frac{\left\lbrack {{mlth}\; f} \right\rbrack}{t} = {{f_{SHMT}\left( {\left\lbrack {{mlth}\; f} \right\rbrack,\lbrack{thf}\rbrack} \right)} - {f_{MTHFC}\left( {\left\lbrack {{mlth}\; f} \right\rbrack,\left\lbrack {{meth}\; f} \right\rbrack} \right)}}} & (48) \\
{\frac{\left\lbrack {{meth}\; f} \right\rbrack}{t} = {{f_{MTHFC}\left( {\left\lbrack {{mlth}\; f} \right\rbrack,\lbrack{methf}\rbrack} \right)} - {f_{MTHFD}\left( {\left\lbrack {{meth}\; f} \right\rbrack,\left\lbrack {10{fth}\; f} \right\rbrack} \right)}}} & (49) \\
{\frac{\left\lbrack {{th}\; f} \right\rbrack}{t} = {{f_{MTHFD}\left( {\left\lbrack {{meth}\; f} \right\rbrack,\left\lbrack {10\; {fthf}} \right\rbrack} \right)} - {f_{FTHFL}\left( {\left\lbrack {10\; {fthf}} \right\rbrack,\left\lbrack {{th}\; f} \right\rbrack} \right)}}} & (50)
\end{matrix}\)

Our aim is to determine whether this dynamical system has a steady state with in the direction of producing ATP. To this end we numerically determined the fixed point of (47)-(50) that resulted in the highest steady state flux with ATP production. This was accomplished solving the optimization problem:

Maximize fFTHFL([mlthf], [thf])   (51)

subject to the fixed point constraints

\(\begin{matrix}
{\frac{\left\lbrack {{th}\; f} \right\rbrack}{t} = {\frac{\left\lbrack {{mlth}\; f} \right\rbrack}{t} = {\frac{\left\lbrack {{meth}\; f} \right\rbrack}{t} = {\frac{\left\lbrack {{th}\; f} \right\rbrack}{t} = 0}}}} & (52)
\end{matrix}\)

and the metabolite concentration bounds

0≦[thf]≦[thf]max

0≦[mlthf]≦[mlthf]max

0≦[methf]≦[methf]max

0≦[10fthf]≦[10fthf]max   (53)

The problem was solved in Matlab using the function fmicon, with all upper bounds equal to 1 mM.

For all 0<ESMTHFD1/(ESHMT+ESMTHFD1)<1 the steady state with maximum rate is characterized by a positive rate of ATP production (FIG. 30, gray circles), indicating that the reaction cycle shown in FIG. 4b is thermodynamically feasible in the direction of ATP production. The maximum rate is achieved at a relative concentration ESMTHFD1/(ESHMT+ESMTHFD1) around 0.05. Hence, the cycle achieves higher rates when the relative concentration of SMTHFD1 is much higher than that of SHMT. We also note the maximum rate calculated from the specified kinetic parameters is much higher than the median obtained from simulations applying a two-fold change in the model kinetic parameters (FIG. 30, black squares). This could indicate that the kinetic parameters in this pathway have been selected for maximum ATP production. However, since the curve for the observed kinetic parameters is still within the 90% confidence intervals (FIG. 30, black error bars) we cannot exclude this is just coincidence.

**The Novel ATP-Producing Pathway is Regulated by Myc**

Consistent with our modeling results, the upregulation of serine and glycine biosynthesis have been observed in some tumor types before. To start to gain insight into this pathway's regulation, we next aimed to identify transcription factors regulating the novel ATP-producing pathway. We performed a search of several transcription factor signatures annotated in the Molecular Signatures Database (MSigDB, Subramanian A, et al. (2005) Proc Natl Acad Sci USA 102: 15545-15550). We identified Myc as a transcriptional regulator of six out of the 11 genes in the pathway: PHGDH, PSPH, SHMT1, MTHFD1, MTHFD2 and GCSH (Myc Target Gene Database, (Zeller K I, et al. (2003) Genome Biol 4: R69)), indicating that Myc-induced tumorigenesis may be intimately linked to the activation of this pathway. To test this hypothesis, we have analyzed data from a doxycycline-inducible murine model of Myc-driven liver cancer (Hu S, et al. (2011) Cell Metab 14: 131-142), reporting gene expression microarrays at different tumor stages. We find that the genes coding for the enzymes in the novel ATP-producing pathway are induced following Myc induction, and all, but two (Psat1 and Psph), return to their control levels upon Myc downregulation-induced tumor regression (FIG. 31). Thus, activation of the novel ATP-producing pathway is evident upon Myc-induced tumorigenesis.

**Discussion**

The existence of an alternative glycolysis pathway with net zero ATP production in rapidly proliferating cells challenges the general notion that the production of ATP is a major function of glycolysis. Instead, an alternative hypothesis suggests that the increased rate of glycolysis in rapidly proliferating cells is present to support the increased demand for precursor metabolites by anabolic processes involved in cell growth and proliferation. However, based on a partial- or full genome-scale reconstruction of human cell metabolism containing the standard glycolysis pathway, we and others have shown that the anabolic requirements can be satisfied without the need for a dramatic upregulation of glycolysis and the excretion of lactate. We have recapitulated this result here, now using the alternative glycolysis pathway with net zero ATP production, providing in silico evidence that the demand for precursor metabolites can be satisfied without upregulation of the alternative glycolysis pathway (FIG. 26).

We have shown previously that molecular crowding is a major determinant of the metabolic changes observed in highly proliferating mammalian and prokaryotic cells and in quiescent cells with high energy demands. In essence, the high density of macromolecules in the intracellular millieu results in a “competition” among mitochondria, ribosomes, metabolic enzymes and structural protein for the available intracellular space. At low metabolic rates this constraint is less pronounced, and therefore the density of the required organelles and macromolecules can increase to accommodate the increasing metabolic rate. However, just as a finite number of people can be placed in a room, only a finite amount of mitochondria can be present in the cell, resulting in an upper bound for OxPhos capacity. To satisfy its energetic needs beyond this maximum OxPhos capacity the cell need to switch to other pathways that are less costly in terms of the required cell volume fraction to allocate the corresponding enzymes, such as the classic glycolysis pathway. However, this hypothesis has been challenged by the observation that highly proliferating cells utilize an alternative glycolysis pathway with net zero ATP production. To resolve this contradiction we have improved our genome-scale metabolic model of a human cell to be able to investigate the consequences of a net zero ATP production glycolysis.

The results of our in silico analyses yield several surprising observations. The glycolysis flux is upregulated in highly proliferating cells and it is routed from 3-phosphoglycerate toward serine biosynthesis. This prediction is supported by experimental observations in both cancer cells and highly proliferating normal cells. Rapidly proliferating normal cells (including embryonic cells) and cancer cells express the M2 isoenzyme of pyruvate kinase (PKM2). The PKM2 isoform can be present either as a tetramer with high PK catalytic activity or as a dimer with low catalytic activity, in the latter case greatly reducing the rate over the last step of standard glycolysis. The upregulation of serine and glycine biosynthesis were observed both in various tumor types and mitogen-activated normal lymphocytes, and serum stimulation of Rat1A fibroblast proliferation resulted in an increased 13C-labeled glycine derived from 3-phosphoglycerate, in a myc dependent manner. Also, when found upregulated, the serine biosynthesis pathway's enzymes are individually essential both for the growth of a subset of breast cancer and melanoma cell lines and for tumorigenesis in 3-dimensional in vitro and in vivo breast cancer models. More importantly, we provided evidence that the expression of genes in this pathway correlates with Myc overexpression in a Myc-driven murine tumor model (FIG. 31). These observations support the activity of the novel ATP-producing pathway in a subset of tumors and, in particular, in Myc driven tumors.

Our in silico approach allow us to investigate the fate of the high rate of the serine biosynthesis pathway. We discover that its final endpoint is a novel pathway for ATP generation, starting from the biosynthesis of serine and involving reactions in the one carbon metabolism pathway and the serine cleavage system (FIG. 29). The reaction responsible for the net ATP generation is catalyzed by formate-tetrahydrofolate ligase (EC 6.3.4.3), working in the ATP production direction. This pathway has a yield of 2 mole of ATP per mole of 3-phosphoglycerate, or 4 moles of ATP per mole of glucose. Taken together our in silico evidence indicates that, even in the context of an alternative glycolysis pathway with net zero ATP production, glycolysis is upregulated to satisfy the high energy demand of highly proliferating cells, during conditions where molecular crowding imposes a bound or a reduction in the mitochondrial density.

The novel pathway doubles the ATP yield from 2 to 4 mole of ATP per mole of glucose (See, Table 10).

The novel pathway requires, however, the balance of several co-factors and thus it is coupled to several other reactions. Yet, it remains to be elucidated what the potential evolutionary advantage of having two alternative glycolysis pathways is (i.e., the net zero ATP and the standard pathways). As we show here, the novel pathway can generate two times more ATP, thus an energetic reason is probably likely. In contrast, the novel pathway involves 17 reactions, 7 more than the standard glycolysis, potentially contributing more to molecular crowding. Taken together with OxPhos, we obtain a hierarchy in terms of ATP yield: OxPhos>>net-zero-ATP-glycolysis>standard glycolysis, and the same hierarchy in terms of molecular crowding. Therefore, these pathways provide the cell with different alternatives to cope with competing efficiency principles, ATP yield per mole of substrate or ATP yield per occupied volume fraction. Concomittantly, other factors, such as the cellular lactate and alanine production also has several potential advantages on the population level that may enhance the invasiveness of tumor cells. Also, tumor cells frequently encounter fluctuating hypoxia levels within growing tumors requiring a capability to rapidly deploy alternative metabolic strategies. In any event, our model identifies several metabolic changes that can be subject to further theoretical and experimental investigations and delineates potential enzyme targets for treatment modalities attempting to interfere with cancer metabolism.

## Materials and Methods

Metabolic network reconstruction: The reactions annotated in H. sapiens metabolic reconstruction 1 were downloaded from the BiGG database (Schellenberger J, et al. (2010) BiGG: a Biochemical Genetic and Genomic knowledgebase of large scale metabolic reconstructions. BMC Bioinformatics 11: 213).

Crowding coefficients: Dividing the mitochondrium specific volume (3.15 mL/g in mammalian liver and 2.6 mL/g in muscle) by the rate of ATP production per mitochondrial mass (0.1-1.0 mmol ATP/min/g) we obtain aM values between 0.0026 to 0.032 min/mM. Except when specified, we use the median 0.017 min/mM. Dividing the ribosome molar volume (vR=4,000 nm3 6.02 1023/mol=2.4 L/mmol) by the rate of protein synthesis per ribosome (0.67 proteins/min) we obtain aR=3.6 min/mM. The enzyme crowding coefficients were estimated as ai=vE/ki. Multiplying the median molecular weight of human enzymes (98,750 g/mol, BRENDA by the enzymes specific volume (approximated by the specific volume of spherical proteins, 0.79 mL/g) we obtain an estimated enzymes molar volume of vE=0.078 L/mmol.

Sensitivity analysis: The turnover numbers of human enzymes k have significant variations from 1 to 100 sec−1 and the distribution of log10(k) is approximately uniform in this range (BRENDA). Based on this data we sampled the log10(keff) values from a uniform distribution in the range between log10(1) to log10(100). At each proliferation rate we run 100 simulations. On each simulation, for each reaction, a value of keff,i is extracted from the distribution described above. With this set of keff,i parameters we then solve the optimization problem (1)-(5) and obtain estimates for the reaction rates. Based on the 100 simulations we finally estimate the median and 90% confident intervals for the rate of each reaction. This data is reported in FIGS. 2-4 for selected reactions. Macromolecular composition: Proteins were divided into three pools: ribosomal-, components of metabolic enzyme complexes-, and non-metabolic proteins. Each ribosome contributes to nPR=82 proteins/ribosome (49 in the 60S and 33 in the 40S subunits). The ribosomal protein concentration was computed as PR=nPRφR/vR. Each enzyme contributes with nPE=2.4 proteins in average, estimated as median enzyme molecular weight (98,750 g/mol, reported above) divided by the median molecular weight of a human protein (40,835 g/mol). The median molecular weight of a human protein was estimated from the median protein length (355 amino acids) and the typical amino acid composition. The enzyme related protein concentration was computed as PE=ΣinPEfi/ki. The concentration of non-metabolic proteins was estimated as 85% (10% metabolic enzymes and 5% ribosomal protein) of the reported total protein content per cell dry weight (0.018 mmol/g DW), i.e. 0.015 mmol/g DW. The lipids, DNA and RNA composition were estimated by their relative abundance in a generic mammalian cell. The abundance per cell dry weight was converted to concentrations after dividing by the typical cell specific volume 4.3 mL/g. This resulted in a concentration of non-metabolic protein of P0=3.59 mM. The maximum macromolecular density of human cells in the absence of osmotic stress is around φmax=40%.

Maintenance parameters: The ATP production rate necessary for cell maintenance is 1.55 mmol ATP/g DW/h. The basal protein degradation rate was estimated as kD(P0+PE+PR), where kD=0.01/h.

Simulations: The optimization problem in equations (1)-(5) was solved in Matlab, using the linear programming function linprog. All reversible reactions were represented by an irreversible reaction on each direction with their own effective turnover number keff,i. Most flux bounds were set to vi,min=0 and vi,max=∞, unless specified otherwise.

Microarray data analysis: The Gene expression dataset reported in Hu S, et al. ((2011) (13)C-Pyruvate Imaging Reveals Alterations in Glycolysis that Precede c-Myc-Induced Tumor Formation and Regression. Cell Metab 14: 131-142) was downloaded from the Gene Expression Omnibus (GEO:GSE28198)). RMA normalized signals were calculated and mean-centered across samples. Model predicted relative cell volume fraction occupied by metabolic enzymes (squares), ribosomes (circles) and mitochondria (triangles), respectively. The model-predicted median and 90% confidence intervals are shown. Note, that the predicted metabolic switch is lost if the molecular crowding constraint is removed from the model (see FIG. S1). Abbreviations: metabolite import/export (Ex metabolite: glc=glucose, gln=glutamine, Ala-L=L-alanine, phe-L=L-phenylalanine, ile-L=isoleucine, lys-L=L-lysine, val-L=L-valine, lac-L=L-lactate), phosphoglycerate kinase (PGK), pyruvate kinase (PK), phosphoglycerate dehydrogenase (PGCD), L-alanine transaminase (ALATA-L), malic enzyme (ME), malate dehydrogenase (MDH), pyruvate carboxylase (PC), pyruvate dehydrogenase (PDH), citrate synthase (CS), α-ketoglutarate dehydrogenase (AKGD).

Whereas particular embodiments of the one or more inventions described herein have been described above for purposes of illustration, it will be evident to those skilled in the art that numerous variations of the details of those one or more inventions may be made without departing from the embodiments defined in the appended claims.

This application is a Continuation-In-Part of U.S. patent application Ser. No. 12/170,852, filed Jul. 10, 2008, which claims the benefit under 35 U.S.C. §119(e) to U.S. Provisional Patent Application No. 60/959,024, filed Jul. 10, 2007, this application also claims the benefit under 35 U.S.C. §119(e) to U.S. Provisional Patent Application No. 61/540,770, filed Sep. 29, 2011, each of which is incorporated herein by reference in its entirety.

