# DESCRIPTION

## BACKGROUND OF THE INVENTION

Cancer is generally characterized by an uncontrolled and invasive growth of cells. These cells may spread to other parts of the body (metastasis). Conventional anticancer therapies, consisting of surgical resection, radiotherapy and chemotherapy, can be effective for some cancers/patients; however, they are not effective for many cancer sufferers. Thus, further medical treatments are needed.

The role of bacteria as an anticancer agent has been recognized for over 100 years, and many genera of bacteria, including Clostridium, Bifidus, and Salmonella, have been shown to preferentially accumulate in tumor tissue and cause regression.

The use of Salmonella typhimurium to treat solid tumors began with the development of a nonpathogenic strain, VNP20009. Well-tolerated in mice and humans, this strain has been shown to preferentially accumulate (>2000-fold) in tumors over the liver, spleen, lung, heart and skin, retarding tumor growth between 38-79%, and prolonging survival of tumor-bearing mice. In initial clinical trials, S. typhimurium was found to be tolerated at high dose and able to effectively colonize human tumors.

## SUMMARY OF THE INVENTION

Engineered, non-pathogenic Salmonella selectively colonize tumors one thousand-fold more than any other organ, invade and deliver therapies cytosolically into cancer cells making the bacteria ideal delivery vehicles for cancer therapy. It is herein demonstrated that controlling the activity of flhDC and subsequent flagellar expression in engineered Salmonella enables intracellular protein delivery selectively in tumor cells in vivo and in vitro. The expression of flhDC/flagella is controlled to enable both colonization of tumors and invasion into cancer cells for the purposes of intracellular protein and therapeutic delivery. Flagella are needed for cell invasion into cancer cells in vitro and in vivo. However, flagellar expression of Salmonella in the bloodstream and/or in systemic circulation causes rapid clearance and significantly reduces tumor colonization. As a result, an inducible version of flhDC was genetically engineered into an engineered strain of Salmonella lacking a native version of the transcription factor (alternatively, the endogenous promoter for flhDC can replaced with an inducible promoter). The inducible system allowed for tight expression control of flhDC within the therapeutic strain. Salmonella lacking the ability to express flhDC colonized tumors with greater selectivity than a parental control strain. Inducing expression of flhDC by administration of ‘remote controlling’, with, for example, arabinose in an arabinose inducible system, within intratumoral, engineered Salmonella enabled intracellular invasion and protein delivery into tumor cells.

Herein is described Salmonella containing and method to control flagellar expression through external means (e.g., a small molecule inducible genetic circuit or inducible expression system) in such a way that engineered strains of Salmonella do not express flagellin systemically. Once the bacteria have colonized tumors to optimal levels, then a ‘remote control’/inducible strategy is employed where a small molecule is used to induce expression of flagella and the type 3 secretion system by activating expression of a recombinant and/or inducible version of the motility regulator, flhDC.

Another aspect provides for the deletion of the SseJ gene in a Salmonella delivery strain. This gene constricts the location of the Salmonella to the Salmonella-containing vacuole (SCV), increasing the delivery potential of the strain. This can be in combination with/without the previously described control of delivery.

One aspect provides a bacterial cell comprising: a) inducible expression of flagella; and b) a lysis gene or lysis cassette operably linked to an intracellularly induced Salmonella promoter. In one aspect, the bacterial cell is an intratumoral bacteria cell. In another aspect, the bacterial cell is a Clostridium, Bifidus, E. coli or Salmonella cell. In one aspect, bacterial cell is a Salmonella cell. In one aspect, the lysis cassette is Lysin E from phage phiX174, the lysis cassette of phage iEPS5, or the lysis cassette from lambda phage. In another aspect, intracellularly induced Salmonella promoter is a promoter for one of the genes in Salmonella pathogenicity island 2 type III secretion system (SPI2-T3SS) selected from the group SpiC/SsaB, SseF, SseG, SseI, SseJ, SseK1, SseK2, SifA, SifB, PipB, PipB2, SopD2, GogB, SseL, SteC, SspH1, SspH2, or SirP.

In one aspect, the cell does not comprise endogenous flhDC expression. In another aspect, the cell comprises an exogenous inducible promoter operably linked to an endogenous or exogenous flhDC gene. In one aspect, the exogenous inducible promoter is operably linked to the endogenous flhDC gene. In another aspect, the exogenous inducible promoter is operably linked the exogenous flhDC gene. In aspect, the exogenous inducible promoter comprises the arabinose inducible promoter PBAD (L-arabinose), LacI (IPTG), salR, or nahR (acetyl salicylic acid (ASA)).

In one aspect, the bacterial cell comprises a SseJ deletion or wherein expression of SseJ has been reduced.

One aspect provides a cell comprises a plasmid that expresses a peptide. In one aspect, the peptide is a therapeutic peptide, such as NIPP1 or activated caspase 3.

One aspect provides a composition comprising a population of cells described herein and a pharmaceutically acceptable carrier.

Another aspect provides a method to selectively colonize cancer cells, such as a tumor and/or tumor associated cells comprising administering a population of the bacterial cells described herein to a subject in need thereof. In one aspect, the tumor associated cells are tumor cells or intratumoral immune cells, cancer cells or stromal cells within tumors. Another aspect provides a method to treat cancer comprising administering to a subject in need thereof an effective amount of a population of the bacterial cells described herein to treat said cancer. A further aspect provides a method of inhibiting tumor growth/proliferation or reducing the volume/size of a tumor comprising administering to a subject in need thereof an effective amount of a population of the bacterial cells described herein, so as to suppress tumor growth or reduce the volume of the tumor. Another aspect provides a method to treat, reduce formation/number or inhibit spread of metastases comprising administering to subject in need thereof an effective amount of a population of the bacterial cells described herein, so as to treat, reduce formation/number or inhibit spread of metastases. In one aspect, the tumor, tumor associated cells, cancer, or metastases are a lung, liver, kidney, breast, prostate, pancreatic, colon, head and neck, ovarian and/or gastroenterological tumor, tumor associated cells, cancer or metastases. In one aspect, the bacterial cells deliver a therapeutic peptide to said tumor, tumor associated cells, cancer or metastases. In one aspect, the peptide is NIPP1 or activated caspase 3. In one aspect, the cells do not express endogenous flhDC. In another aspect, expression of flhDC in the bacterial cell is under the control of an inducible promoter, wherein the bacterial cells comprise an exogenous inducible promoter controlling expression of endogenous flhDC or the bacterial cells comprise an exogenous inducible promoter operably linked an exogenous flhDC gene. In one aspect, the expression of flhDC is induced after said tumor, tumor associated cells, cancer or metastases have been colonized (e.g., between 1×106 and 1×1010 CFU/g tumor) by said bacteria.

One aspect provides a bacterial cell comprising: a) a SseJ deletion or wherein expression of SseJ has been reduced; and b) a lysis gene or lysis cassette operably linked to an intracellularly induced Salmonella promoter. In one aspect, the bacterial cell is an intratumoral bacteria cell. In another aspect, the bacterial cell is a Clostridium, Bifidus or Salmonella cell. In aspect, the bacterial cell is a Salmonella cell. In one aspect, the lysis cassette is Lysin E from phage phiX174, the lysis cassette of phage iEPS5, or the lysis cassette from lambda phage. In another aspect, the intracellularly induced Salmonella promoter is a promoter for one of the genes in Salmonella pathogenicity island 2 type III secretion system (SPI2-T3SS) selected from the group SpiC/SsaB, SseF, SseG, SseI, SseJ, SseK1, SseK2, SifA, SifB, PipB, PipB2, SopD2, GogB, SseI SteC, SspH1, SspH2, or SirP.

In one aspect, the cell of any one of claims 28-33, wherein the cell does not comprise endogenous flhDC expression. In another aspect, the cell comprises an exogenous inducible promoter operably linked to an endogenous or exogenous flhDC gene. In another aspect, the exogenous inducible promoter is operably linked to the endogenous flhDC gene. In another aspect, the exogenous inducible promoter is operably linked the exogenous flhDC gene. In aspect, the exogenous inducible promoter comprises the arabinose inducible promoter PBAD (L-arabinose), LacI (IPTG), nahR (acetyl salicylic acid (ASA)), or salR acetyl salicylic acid (ASA).

In one aspect, the bacterial cell comprises a plasmid that expresses a peptide. In one aspect, the peptide is a therapeutic peptide, such as NIPP1 or activated caspase 3.

One aspect provides for a composition comprising a population of cells as described herein and a pharmaceutically acceptable carrier.

One aspect provides a method to colonize a tumor and/or tumor associated cells comprising administering a population of the bacterial cells described herein to a subject in need thereof. In one aspect, the tumor associated cells are tumor cells, intratumoral immune cells or stromal cells within tumors. In one aspect there is provided a method to treat cancer comprising administering to subject in need thereof an effective amount of a population of the bacterial cells described herein so as to treat said cancer. Another aspect provides a method of inhibiting tumor growth/proliferation or reducing the volume/size of a tumor comprising administering to subject in need thereof an effective amount of a population of the bacterial cells described herein, so as to suppress tumor growth or reduce the volume of the tumor. A further aspect provides a method to treat, reduce formation/number or inhibit spread of metastases comprising administering to subject in need thereof an effective amount of a population of the bacterial cells described herein, so as to treat, reduce formation/number or inhibit spread of metastases. In one aspect, the tumor, tumor associated cells, cancer, or metastases are a lung, liver, kidney, breast, prostate, pancreatic, colon, head and neck, ovarian and/or gastroenterological tumor, tumor associated cells, cancer or metastases. In another aspect, the bacterial cells deliver a therapeutic peptide, such as NIPP1 or activated caspase 3, to said tumor, tumor associated cells, cancer or metastases. In one embodiment, endogenous expression of flhDC is under control of an exogenous inducible promoter. In another aspect, expression of flhDC is under the control of an inducible promoter, wherein the bacterial cells comprise an exogenous inducible promoter operably linked an exogenous flhDC gene. In a further aspect, the expression of flhDC is induced after said tumor, tumor associated cells, cancer or metastases have been colonized by said bacteria.

One aspect provides a bacterial cell comprising: a) constitutive or inducible expression of a therapeutic peptide, wherein the therapeutic peptide is activated caspase-3 and wherein said activated caspase-3 is expressed as an activated protein without further processing; and b) a lysis gene or lysis cassette operably linked to an intracellularly induced Salmonella promoter. In one aspect, the bacterial cell is an intratumoral bacteria cell. In one aspect, the bacterial cell is a Clostridium, Bifidus or Salmonella cell. In another aspect, the bacterial cell is a Salmonella cell. In one aspect, the lysis cassette is Lysin E from phage phiX174, the lysis cassette of phage iEPS5, or the lysis cassette from lambda phage. In one aspect, the intracellularly induced Salmonella promoter is a promoter for one of the genes in Salmonella pathogenicity island 2 type III secretion system (SPI2-T3SS) selected from the group SpiC/SsaB, SseF, SseG, SseI, SseJ, SseK1, SseK2, SifA, SifB, PipB, PipB2, SopD2, GogB, SseL, SteC, SspH1, SspH2, or SirP.

In another aspect, the bacterial cell does not comprise endogenous flhDC expression. In one aspect, the bacterial cell comprises an exogenous inducible promoter operably linked to an endogenous or exogenous flhDC gene. In one aspect, the exogenous inducible promoter is operably linked to the endogenous flhDC gene. In another aspect, the exogenous inducible promoter is operably linked the exogenous flhDC gene. In one aspect, the exogenous inducible promoter comprises the arabinose inducible promoter PBAD (L-arabinose), LacI (IPTG), nahR (acetyl salicylic acid (ASA)) or salR acetyl salicylic acid (ASA).

In aspect, the bacterial cell comprises a SseJ deletion or wherein expression of SseJ has been reduced.

One aspect provides for cells that express at least one additional exogenous therapeutic peptide, such as NIPP1.

Another aspect provides a composition comprising a population of cells described herein and a pharmaceutically acceptable carrier.

One aspect provides a method to colonize a tumor and/or tumor associated cells comprising administering a population of the bacterial cells described herein to a subject in need thereof. In one aspect, the tumor associated cells are tumor cells, intratumoral immune cells or stromal cells within tumors. One aspect provides a method to treat cancer comprising administering to subject in need thereof an effective amount of a population of the bacterial cells described herein so as to treat said cancer. In one aspect there is provided a method of inhibiting tumor growth/proliferation or reducing the volume/size of a tumor comprising administering to subject in need thereof an effective amount of a population of the bacterial cells of any one of claims described herein, so as to suppress tumor growth or reduce the volume of the tumor. One aspect provides a method to treat, reduce formation/number or inhibit spread of metastases comprising administering to subject in need thereof an effective amount of a population of the bacterial cells described herein, so as to treat, reduce formation/number or inhibit spread of metastases. In one aspect, the tumor, tumor associated cells, cancer, or metastases are a lung, liver, kidney, breast, prostate, pancreatic, colon, head and neck, ovarian and/or gastroenterological tumor, tumor associated cells, cancer or metastases. In one aspect, the bacterial cells deliver said caspase to said tumor, tumor associated cells, cancer or metastases. In another aspect, the bacterial cells deliver at least one additional exogenous therapeutic peptide, such as NIPP1. In aspect, the endogenous expression of flhDC is under control of an exogenous inducible promoter. In another aspect, the expression of flhDC is under the control of an inducible promoter, wherein the bacterial cells comprise an exogenous inducible promoter operably linked an exogenous flhDC gene. In one aspect, the bacterial cells do not express endogenous flhDC. In one aspect, the expression of flhDC is induced after said tumor, tumor associated cells, cancer or metastases have been colonized by said bacteria.

## DETAILED DESCRIPTION OF THE INVENTION

The majority of proteins are intracellular. Specifically targeting intracellular pathways specifically in cancer cells using macromolecular therapies increases the potential treatment options for any patient. However, macromolecular therapies that target intracellular pathways face significant barriers associated with tumor targeting, distribution, internalization and endosomal release. Engineered, non-pathogenic Salmonella selectively colonize tumors one thousand-fold more than any other organ, invade and deliver therapies cytosolically into cancer cells making the bacteria ideal delivery vehicles for cancer therapy.

However, a problem with using bacteria as an anti-cancer agent is their toxicity at the dose required for therapeutic efficacy and an obstacle in cancer gene therapy is the specific targeting of therapy directly to the cancer. Another issue to be addressed is systemic clearance of Salmonella. A further issue is the activity of cytosolic Salmonella (as compared to SCV Salmonella). A novel therapeutic platform for controlled colonization and/or invasion of engineered Salmonella in cancer cells and controlled gene and protein delivery in cancer cells, and therefore treatment for cancer, is provided herein.

To address these challenges, a bacterial delivery platform was developed that harnesses mechanisms unique to Salmonella to intracellularly deliver protein-based drugs. Salmonella sense the intracellular environment and accumulate inside cells when in tumors. Genetic circuits were engineered that force entry into cancer cells and release proteins from the endosome into the cytoplasm. Intracellular lysis makes the platform self-limiting and reduces the possibility of unwanted infection. Delivered nanobodies and protein interactors (NIPP1) bind to their targets and cause cell death. Delivery of caspase-3 to mice reduces growth of breast tumors and eliminates liver tumors. Intracellular delivery of protein-based drugs to tumors opens up the entire proteome for treatment.

### Definitions

Unless defined otherwise, all technical and scientific terms used herein have the same meaning as commonly understood by one of ordinary skill in the art to which this invention belongs. Although any methods and materials similar or equivalent to those described herein can be used in the practice or testing of the present invention, several embodiments with regards to methods and materials are described herein. As used herein, each of the following terms has the meaning associated with it in this section.

For the purposes of clarity and a concise description, features can be described herein as part of the same or separate embodiments; however, it will be appreciated that the scope of the invention may include embodiments having combinations of all or some of the features described.

References in the specification to “one embodiment”, “an embodiment”, etc., indicate that the embodiment described may include a particular aspect, feature, structure, moiety, or characteristic, but not every embodiment necessarily includes that aspect, feature, structure, moiety, or characteristic. Moreover, such phrases may, but do not necessarily, refer to the same embodiment referred to in other portions of the specification. Further, when a particular aspect, feature, structure, moiety, or characteristic is described in connection with an embodiment, it is within the knowledge of one skilled in the art to affect or connect such aspect, feature, structure, moiety, or characteristic with other embodiments, whether or not explicitly described.

As used herein, the indefinite articles “a”, “an” and “the” should be understood to include plural reference unless the context clearly indicates otherwise.

The phrase “and/or,” as used herein, should be understood to mean “either or both” of the elements so conjoined, e.g., elements that are conjunctively present in some cases and disjunctively present in other cases.

As used herein, “or” should be understood to have the same meaning as “and/or” as defined above. For example, when separating a listing of items, “and/or” or “or” shall be interpreted as being inclusive, e.g., the inclusion of at least one, but also including more than one, of a number of items, and, optionally, additional unlisted items. Only terms clearly indicated to the contrary, such as “only one of” or “exactly one of,” or, when used in the claims, “consisting of,” will refer to the inclusion of exactly one element of a number or list of elements. In general, the term “or” as used herein shall only be interpreted as indicating exclusive alternatives (i.e., “one or the other but not both”) when preceded by terms of exclusivity, such as “either,” “one of,” “only one of,” or “exactly one of.”

As used herein, the terms “including,” “includes,” “having,” “has,” “with,” or variants thereof, are intended to be inclusive similar to the term “comprising.”

As used herein, the term “about” means plus or minus 10% of the indicated value. For example, about 100 means from 90 to 110. Numerical ranges recited herein by endpoints include all numbers and fractions subsumed within that range (e.g., 1 to 5 includes 1, 1.5, 2, 2.75, 3, 3.90, 4, and 5). It is also to be understood that all numbers and fractions thereof are presumed to be modified by the term “about.”

The terms “individual,” “subject,” and “patient,” are used interchangeably herein and refer to any subject for whom diagnosis, treatment, or therapy is desired, including a mammal. Mammals include, but are not limited to, humans, farm animals, sport animals and pets. A “subject” is a vertebrate, such as a mammal, including a human. Mammals include, but are not limited to, humans, farm animals, sport animals and companion animals. Included in the term “animal” is dog, cat, fish, gerbil, guinea pig, hamster, horse, rabbit, swine, mouse, monkey (e.g., ape, gorilla, chimpanzee, orangutan) rat, sheep, goat, cow and bird.

The terms “treatment”, “treating” and the like are used herein to generally mean obtaining a desired pharmacologic and/or physiologic effect, such as arresting or inhibiting, or attempting to arrest or inhibit, the development or progression of a disorder and/or causing, or attempting to cause, the reduction, suppression, regression, or remission of a disorder and/or a symptom thereof. The effect may be prophylactic in terms of completely or partially preventing a disease or symptom thereof and/or may be therapeutic in terms of a partial or complete cure for a disease and/or adverse effect attributable to the disease. As would be understood by those skilled in the art, various clinical and scientific methodologies and assays may be used to assess the development or progression of a disorder, and similarly, various clinical and scientific methodologies and assays may be used to assess the reduction, regression, or remission of a disorder or its symptoms. Additionally, treatment can be applied to a subject or to a cell culture (in vivo or in vitro).

The terms “inhibit”, “inhibiting”, and “inhibition” refer to the slowing, halting, or reversing the growth or progression of a disease, infection, condition, group of cells, protein or its expression. The inhibition can be greater than about 20%, 40%, 60%, 80%, 90%, 95%, or 99%, for example, compared to the growth or progression that occurs in the absence of the treatment or contacting.

“Expression” refers to the production of RNA from DNA and/or the production of protein directed by genetic material (e.g., RNA (mRNA)). Inducible expression, as opposed to constitutive expression (expressed all the time), is expression which only occurs under certain conditions, such as in the presence of specific molecule (e.g., arabinose) or an environmental que.

The term “exogenous” as used herein with reference to a nucleic acid (or a protein) and a host refers to a nucleic acid that does not occur in (and cannot be obtained from) a cell of that particular type as it is found in nature or a protein encoded by such a nucleic acid. Thus, a nonnaturally-occurring nucleic acid is considered to be exogenous to a host once in the host. It is important to note that non-naturally occurring nucleic acids can contain nucleic acid subsequences or fragments of nucleic acid sequences that are found in nature provided the nucleic acid as a whole does not exist in nature. For example, a nucleic acid molecule containing a genomic DNA sequence within an expression vector is non-naturally occurring nucleic acid, and thus is exogenous to a host cell once introduced into the host, since that nucleic acid molecule as a whole (genomic DNA plus vector DNA) does not exist in nature. Thus, any vector, autonomously replicating plasmid, or virus (e.g., retrovirus, adenovirus, or herpes virus) that as a whole does not exist in nature is considered to be non-naturally occurring nucleic acid. It follows that genomic DNA fragments produced by PCR or restriction endonuclease treatment as well as cDNAs are considered to be non-naturally occurring nucleic acid since they exist as separate molecules not found in nature. An exogenous sequence may therefore be integrated into the genome of the host. It also follows that any nucleic acid containing a promoter sequence and polypeptide-encoding sequence (e.g., cDNA or genomic DNA) in an arrangement not found in nature is non-naturally occurring nucleic acid. A nucleic acid that is naturally occurring can be exogenous to a particular host microorganism. For example, an entire chromosome isolated from a cell of yeast x is an exogenous nucleic acid with respect to a cell of yeast y once that chromosome is introduced into a cell of yeast y.

In contrast, the term “endogenous” as used herein with reference to a nucleic acid (e.g., a gene) (or a protein) and a host refers to a nucleic acid (or protein) that does occur in (and can be obtained from) that particular host as it is found in nature. Moreover, a cell “endogenously expressing” a nucleic acid (or protein) expresses that nucleic acid (or protein) as does a host of the same particular type as it is found in nature. Moreover, a host “endogenously producing” or that “endogenously produces” a nucleic acid, protein, or other compound produces that nucleic acid, protein, or compound as does a host of the same particular type as it is found in nature.

Flagella are filamentous protein structures found in bacteria, archaea, and eukaryotes, though they are most commonly found in bacteria. They are typically used to propel a cell through liquid (i.e., bacteria and sperm). However, flagella have many other specialized functions. Flagella are usually found in gram-negative bacilli. Gram-positive rods (e.g., Listeria species) and cocci (some Enterococcus species, Vagococcus species) also have flagella.

Engineered Salmonella could be any strain of Salmonella designed to lyse and deliver protein intracellularly.

The term “contacting” refers to the act of touching, making contact, or of bringing to immediate or close proximity, including at the cellular or molecular level, for example, to bring about a physiological reaction, a chemical reaction, or a physical change, e.g., in a solution, in a reaction mixture, in vitro, or in vivo.

An “effective amount” is an amount sufficient to effect beneficial or desired result, such as a preclinical or clinical result. An effective amount can be administered in one or more administrations. The term “effective amount,” as applied to the compound(s), biologics and pharmaceutical compositions described herein, means the quantity necessary to render the desired therapeutic result. For example, an effective amount is a level effective to treat, cure, or alleviate the symptoms of a disorder and/or disease for which the therapeutic compound, biologic or composition is being administered. Amounts effective for the particular therapeutic goal sought will depend upon a variety of factors including the disorder being treated and its severity and/or stage of development/progression; the bioavailability, and activity of the specific compound, biologic or pharmaceutical composition used; the route or method of administration and introduction site on the subject; the rate of clearance of the specific compound or biologic and other pharmacokinetic properties; the duration of treatment; inoculation regimen; drugs used in combination or coincident with the specific compound, biologic or composition; the age, body weight, sex, diet, physiology and general health of the subject being treated; and like factors well known to one of skill in the relevant scientific art. Some variation in dosage can occur depending upon the condition of the subject being treated, and the physician or other individual administering treatment will, in any event, determine the appropriate dose for an individual patient.

As used herein, “disorder” refers to a disorder, disease or condition, or other departure from healthy or normal biological activity, and the terms can be used interchangeably. The terms would refer to any condition that impairs normal function. The condition may be caused by sporadic or heritable genetic abnormalities. The condition may also be caused by non-genetic abnormalities. The condition may also be caused by injuries to a subject from environmental factors, such as, but not limited to, cutting, crushing, burning, piercing, stretching, shearing, injecting, or otherwise modifying a subject's cell(s), tissue(s), organ(s), system(s), or the like.

The terms “cell,” “cell line,” and “cell culture” as used herein may be used interchangeably. All of these terms also include their progeny, which are any and all subsequent generations. It is understood that all progeny may not be identical due to deliberate or inadvertent mutations.

A “coding region” of a gene consists of the nucleotide residues of the coding strand of the gene and the nucleotides of the non-coding strand of the gene which are homologous with or complementary to, respectively, the coding region of an mRNA molecule which is produced by transcription of the gene.

“Complementary” as used herein refers to the broad concept of subunit sequence complementarity between two nucleic acids, e.g., two DNA molecules. When a nucleotide position in both of the molecules is occupied by nucleotides normally capable of base pairing with each other, then the nucleic acids are considered to be complementary to each other at this position. Thus, two nucleic acids are complementary to each other when a substantial number (at least 50%) of corresponding positions in each of the molecules are occupied by nucleotides which normally base pair with each other (e.g., A:T and G:C nucleotide pairs). Thus, it is known that an adenine residue of a first nucleic acid region is capable of forming specific hydrogen bonds (“base pairing”) with a residue of a second nucleic acid region which is antiparallel to the first region if the residue is thymine or uracil. Similarly, it is known that a cytosine residue of a first nucleic acid strand is capable of base pairing with a residue of a second nucleic acid strand which is antiparallel to the first strand if the residue is guanine. A first region of a nucleic acid is complementary to a second region of the same or a different nucleic acid if, when the two regions are arranged in an antiparallel fashion, at least one nucleotide residue of the first region is capable of base pairing with a residue of the second region. Preferably, the first region comprises a first portion and the second region comprises a second portion, whereby, when the first and second portions are arranged in an antiparallel fashion, at least about 50%, and preferably at least about 75%, at least about 90%, or at least about 95% of the nucleotide residues of the first portion are capable of base pairing with nucleotide residues in the second portion. More preferably, all nucleotide residues of the first portion are capable of base pairing with nucleotide residues in the second portion.

“Encoding” refers to the inherent property of specific sequences of nucleotides in a polynucleotide, such as a gene, a cDNA, or an mRNA, to serve as templates for synthesis of other polymers and macromolecules in biological processes having either a defined sequence of nucleotides (i.e., rRNA, tRNA and mRNA) or a defined sequence of amino acids and the biological properties resulting therefrom. Thus, a gene encodes a protein if transcription and translation of mRNA corresponding to that gene produces the protein in a cell or other biological system. Both the coding strand, the nucleotide sequence of which is identical to the mRNA sequence and is usually provided in sequence listings, and the non-coding strand, used as the template for transcription of a gene or cDNA, can be referred to as encoding the protein or other product of that gene or cDNA.

As used herein, an “essentially pure” preparation of a particular protein or peptide is a preparation wherein at least about 95%, and preferably at least about 99%, by weight, of the protein or peptide in the preparation is the particular protein or peptide.

A “fragment” or “segment” is a portion of an amino acid sequence, comprising at least one amino acid, or a portion of a nucleic acid sequence comprising at least one nucleotide. The terms “fragment” and “segment” are used interchangeably herein.

As used herein, a “functional” biological molecule is a biological molecule in a form in which it exhibits a property by which it is characterized. A functional enzyme, for example, is one which exhibits the characteristic catalytic activity by which the enzyme is characterized.

“Homologous” as used herein, refers to the subunit sequence similarity between two polymeric molecules, e.g., between two nucleic acid molecules, e.g., two DNA molecules or two RNA molecules, or between two polypeptide molecules. When a subunit position in both of the two molecules is occupied by the same monomeric subunit, e.g., if a position in each of two DNA molecules is occupied by adenine, then they are homologous at that position. The homology between two sequences is a direct function of the number of matching or homologous positions, e.g., if half (e.g., five positions in a polymer ten subunits in length) of the positions in two compound sequences are homologous then the two sequences are 50% homologous, if 90% of the positions, e.g., 9 of 10, are matched or homologous, the two sequences share 90% homology. By way of example, the DNA sequences 3′ATTGCC5′ and 3′TATGGC share 50% homology.

As used herein, “homology” is used synonymously with “identity.”

The determination of percent identity between two nucleotide or amino acid sequences can be accomplished using a mathematical algorithm. For example, a mathematical algorithm useful for comparing two sequences is the algorithm of Karlin and Altschul (1990, Proc. Natl. Acad. Sci. USA 87:2264-2268), modified as in Karlin and Altschul (1993, Proc. Natl. Acad. Sci. USA 90:5873-5877). This algorithm is incorporated into the NBLAST and XBLAST programs of Altschul, et al. (1990, J. Mol. Biol. 215:403-410), and can be accessed, for example at the National Center for Biotechnology Information (NCBI) world wide web site having the universal resource locator using the BLAST tool at the NCBI website. BLAST nucleotide searches can be performed with the NBLAST program (designated “blastn” at the NCBI web site), using the following parameters: gap penalty=5; gap extension penalty=2; mismatch penalty=3; match reward=1; expectation value 10.0; and word size=11 to obtain nucleotide sequences homologous to a nucleic acid described herein. BLAST protein searches can be performed with the XBLAST program (designated “blastn” at the NCBI web site) or the NCBI “blastp” program, using the following parameters: expectation value 10.0, BLOSUM62 scoring matrix to obtain amino acid sequences homologous to a protein molecule described herein. To obtain gapped alignments for comparison purposes, Gapped BLAST can be utilized as described in Altschul et al. (1997, Nucleic Acids Res. 25:3389-3402). Alternatively, PSI-Blast or PHI-Blast can be used to perform an iterated search which detects distant relationships between molecules (Id.) and relationships between molecules which share a common pattern. When utilizing BLAST, Gapped BLAST, PSI-Blast, and PHI-Blast programs, the default parameters of the respective programs (e.g., XBLAST and NBLAST) can be used.

The percent identity between two sequences can be determined using techniques similar to those described above, with or without allowing gaps. In calculating percent identity, typically exact matches are counted.

As used herein, the term “hybridization” is used in reference to the pairing of complementary nucleic acids. Hybridization and the strength of hybridization (i.e., the strength of the association between the nucleic acids) is impacted by such factors as the degree of complementarity between the nucleic acids, stringency of the conditions involved, the length of the formed hybrid, and the G:C ratio within the nucleic acids.

As used herein, an “instructional material” includes a publication, a recording, a diagram, or any other medium of expression which can be used to communicate the usefulness of the peptide of the invention in the kit for effecting alleviation of the various diseases or disorders recited herein. Optionally, or alternately, the instructional material may describe one or more methods of alleviating the diseases or disorders in a cell or a tissue of a mammal. The instructional material of the kit of the invention may, for example, be affixed to a container which contains the identified compound invention or be shipped together with a container which contains the identified compound. Alternatively, the instructional material may be shipped separately from the container with the intention that the instructional material and the compound be used cooperatively by the recipient.

The term “nucleic acid” typically refers to large polynucleotides. By “nucleic acid” is meant any nucleic acid, whether composed of deoxyribonucleosides or ribonucleosides, and whether composed of phosphodiester linkages or modified linkages such as phosphotriester, phosphoramidate, siloxane, carbonate, carboxymethylester, acetamidate, carbamate, thioether, bridged phosphoramidate, bridged methylene phosphonate, bridged phosphoramidate, bridged phosphoramidate, bridged methylene phosphonate, phosphorothioate, methylphosphonate, phosphorodithioate, bridged phosphorothioate or sulfone linkages, and combinations of such linkages. The term nucleic acid also specifically includes nucleic acids composed of bases other than the five biologically occurring bases (adenine, guanine, thymine, cytosine and uracil).

As used herein, the term “nucleic acid” encompasses RNA as well as single and double stranded DNA and cDNA. Furthermore, the terms, “nucleic acid,” “DNA,” “RNA” and similar terms also include nucleic acid analogs, i.e., analogs having other than a phosphodiester backbone. For example, the so called “peptide nucleic acids,” which are known in the art and have peptide bonds instead of phosphodiester bonds in the backbone, are considered within the scope of the present invention. By “nucleic acid” is meant any nucleic acid, whether composed of deoxyribonucleosides or ribonucleosides, and whether composed of phosphodiester linkages or modified linkages such as phosphotriester, phosphoramidate, siloxane, carbonate, carboxymethylester, acetamidate, carbamate, thioether, bridged phosphoramidate, bridged methylene phosphonate, bridged phosphoramidate, bridged phosphoramidate, bridged methylene phosphonate, phosphorothioate, methylphosphonate, phosphorodithioate, bridged phosphorothioate or sulfone linkages, and combinations of such linkages. The term nucleic acid also specifically includes nucleic acids composed of bases other than the five biologically occurring bases (adenine, guanine, thymine, cytosine, and uracil). Conventional notation is used herein to describe polynucleotide sequences: the left-hand end of a single-stranded polynucleotide sequence is the 5′-end; the left-hand direction of a double-stranded polynucleotide sequence is referred to as the 5′-direction. The direction of 5′ to 3′ addition of nucleotides to nascent RNA transcripts is referred to as the transcription direction. The DNA strand having the same sequence as an mRNA is referred to as the “coding strand”; sequences on the DNA strand which are located 5′ to a reference point on the DNA are referred to as “upstream sequences”; sequences on the DNA strand which are 3′ to a reference point on the DNA are referred to as “downstream sequences.”

The term “nucleic acid construct,” as used herein, encompasses DNA and RNA sequences encoding the particular gene or gene fragment desired, whether obtained by genomic or synthetic methods.

Unless otherwise specified, a “nucleotide sequence encoding an amino acid sequence” includes all nucleotide sequences that are degenerate versions of each other and that encode the same amino acid sequence. Nucleotide sequences that encode proteins and RNA may include introns.

The term “oligonucleotide” typically refers to short polynucleotides, generally, no greater than about 50 nucleotides. It will be understood that when a nucleotide sequence is represented by a DNA sequence (i.e., A, T, G, C), this also includes an RNA sequence (i.e., A, U, G, C) in which “U” replaces “T.”

“Substantially homologous nucleic acid sequence” means a nucleic acid sequence corresponding to a reference nucleic acid sequence wherein the corresponding sequence encodes a peptide having substantially the same structure and function as the peptide encoded by the reference nucleic acid sequence; e.g., where only changes in amino acids not significantly affecting the peptide function occur. Preferably, the substantially identical nucleic acid sequence encodes the peptide encoded by the reference nucleic acid sequence. The percentage of identity between the substantially similar nucleic acid sequence and the reference nucleic acid sequence is at least about 50%, 65%, 75%, 85%, 95%, 99% or more. Substantial identity of nucleic acid sequences can be determined by comparing the sequence identity of two sequences, for example by physical/chemical methods (i.e., hybridization) or by sequence alignment via computer algorithm. Suitable nucleic acid hybridization conditions to determine if a nucleotide sequence is substantially similar to a reference nucleotide sequence are: 7% sodium dodecyl sulfate SDS, 0.5 M NaPO4, 1 mM EDTA at 50° C. with washing in 2× standard saline citrate (SSC), 0.1% SDS at 50° C.; preferably in 7% (SDS), 0.5 M NaPO4, 1 mM EDTA at 50° C. with washing in 1×SSC, 0.1% SDS at 50° C.; preferably 7% SDS, 0.5 M NaPO4, 1 mM EDTA at 50° C. with washing in 0.5×SSC, 0.1% SDS at 50° C.; and more preferably in 7% SDS, 0.5 M NaPO4, 1 mM EDTA at 50° C. with washing in 0.1×SSC, 0.1% SDS at 65° C. Suitable computer algorithms to determine substantial similarity between two nucleic acid sequences include, GCS program package (Devereux et al., 1984 Nucl. Acids Res. 12:387), and the BLASTN or FASTA programs (Altschul et al., 1990 Proc. Natl. Acad. Sci. USA. 1990 87:14:5509-13; Altschul et al., J. Mol. Biol. 1990 215:3:403-10; Altschul et al., 1997 Nucleic Acids Res. 25:3389-3402). The default settings provided with these programs are suitable for determining substantial similarity of nucleic acid sequences for purposes of the present invention.

By describing two polynucleotides as “operably linked” is meant that a single-stranded or double-stranded nucleic acid moiety comprises the two polynucleotides arranged within the nucleic acid moiety in such a manner that at least one of the two polynucleotides is able to exert a physiological effect by which it is characterized upon the other. By way of example, a promoter operably linked to the coding region of a gene is able to promote transcription of the coding region.

As used herein, the term “pharmaceutically acceptable carrier” means a chemical composition with which an appropriate compound or derivative can be combined and which, following the combination, can be used to administer the appropriate compound to a subject. “Pharmaceutically acceptable” means physiologically tolerable, for either human or veterinary application. As used herein, “pharmaceutical compositions” include formulations for human and veterinary use.

As used herein, the term “purified” and like terms relate to an enrichment of a molecule or compound relative to other components normally associated with the molecule or compound in a native environment. The term “purified” does not necessarily indicate that complete purity of the particular molecule has been achieved during the process. A “highly purified” compound as used herein refers to a compound that is greater than 90% pure. In particular, purified sperm cell DNA refers to DNA that does not produce significant detectable levels of non-sperm cell DNA upon PCR amplification of the purified sperm cell DNA and subsequent analysis of that amplified DNA. A “significant detectable level” is an amount of contaminate that would be visible in the presented data and would need to be addressed/explained during analysis of the forensic evidence.

“Recombinant polynucleotide” refers to a polynucleotide having sequences that are not naturally joined together. An amplified or assembled recombinant polynucleotide may be included in a suitable vector, and the vector can be used to transform a suitable host cell.

A recombinant polynucleotide may serve a non-coding function (e.g., promoter, origin of replication, ribosome-binding site, etc.) as well.

A host cell that comprises a recombinant polynucleotide is referred to as a “recombinant host cell.” A gene which is expressed in a recombinant host cell wherein the gene comprises a recombinant polynucleotide, produces a “recombinant polypeptide.”

A “recombinant polypeptide” is one which is produced upon expression of a recombinant polynucleotide.

A “recombinant cell” is a cell that comprises a transgene. Such a cell may be a eukaryotic or a prokaryotic cell. Also, the transgenic cell encompasses, but is not limited to, an embryonic stem cell comprising the transgene, a cell obtained from a chimeric mammal derived from a transgenic embryonic stem cell where the cell comprises the transgene, a cell obtained from a transgenic mammal, or fetal or placental tissue thereof, and a prokaryotic cell comprising the transgene.

The term “regulate” refers to either stimulating or inhibiting a function or activity of interest.

By “small interfering RNAs (siRNAs)” is meant, inter alia, an isolated dsRNA molecule comprised of both a sense and an anti-sense strand. In one aspect, it is greater than 10 nucleotides in length. siRNA also refers to a single transcript which has both the sense and complementary antisense sequences from the target gene, e.g., a hairpin. siRNA further includes any form of dsRNA (proteolytically cleaved products of larger dsRNA, partially purified RNA, essentially pure RNA, synthetic RNA, recombinantly produced RNA) as well as altered RNA that differs from naturally occurring RNA by the addition, deletion, substitution, and/or alteration of one or more nucleotides.

By the term “specifically binds to”, as used herein, is meant when a compound or ligand functions in a binding reaction or assay conditions which is determinative of the presence of the compound in a sample of heterogeneous compounds, or it means that one molecule, such as a binding moiety, e.g., an oligonucleotide or antibody, binds preferentially to another molecule, such as a target molecule, e.g., a nucleic acid or a protein, in the presence of other molecules in a sample.

The terms “specific binding” or “specifically binding” when used in reference to the interaction of a peptide (ligand) and a receptor (molecule) also refers to an interaction that is dependent upon the presence of a particular structure (i.e., an amino sequence of a ligand or a ligand binding domain within a protein); in other words the peptide comprises a structure allowing recognition and binding to a specific protein structure within a binding partner rather than to molecules in general. For example, if a ligand is specific for binding pocket “A,” in a reaction containing labeled peptide ligand “A” (such as an isolated phage displayed peptide or isolated synthetic peptide) and unlabeled “A” in the presence of a protein comprising a binding pocket A the unlabeled peptide ligand will reduce the amount of labeled peptide ligand bound to the binding partner, in other words a competitive binding assay.

The term “standard,” as used herein, refers to something used for comparison. For example, it can be a known standard agent or compound which is administered and used for comparing results when administering a test compound, or it can be a standard parameter or function which is measured to obtain a control value when measuring an effect of an agent or compound on a parameter or function. Standard can also refer to an “internal standard”, such as an agent or compound which is added at known amounts to a sample and is useful in determining such things as purification or recovery rates when a sample is processed or subjected to purification or extraction procedures before a marker of interest is measured. Internal standards are often a purified marker of interest which has been labeled, such as with a radioactive isotope, allowing it to be distinguished from an endogenous marker.

Methods involving conventional molecular biology techniques are described herein. Such techniques are generally known in the art and are described in detail in methodology treatises, such as Molecular Cloning: A Laboratory Manual, 2nd ed., vol. 1-3, ed. Sambrook et al., Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., 1989; and Current Protocols in Molecular Biology, ed. Ausubel et al., Greene Publishing and Wiley-Interscience, New York, 1992 (with periodic updates). Methods for chemical synthesis of nucleic acids are discussed, for example, in Beaucage and Carruthers, Tetra. Letts. 22: 1859-1862, 1981, and Matteucci et al., J. Am. Chem. Soc. 103:3185, 1981.

As used herein, the terms “including”, “includes”, “having”, “has”, “with”, or variants thereof, are intended to be inclusive similar to the term “comprising.”

The terms “comprises,” “comprising,” and the like can have the meaning ascribed to them in U.S. Patent Law and can mean “includes,” “including” and the like. As used herein, “including” or “includes” or the like means including, without limitation.

### I. Bacteria/Flagella

Bacteria useful in the invention include, but are not limited to, Clostridium, Bifidus, Escherichia coli or Salmonella, T3SS-dependent bacteria, such as shigella, salmonella and Yersinia Pestis. Further, E. coli can be used if the T3SS system is place in E. Coli.

**Salmonella**

Examples of Salmonella strains which can be employed in the present invention include Salmonella typhi (ATCC No. 7251) and S. typhimurium (ATCC No. 13311). Attenuated Salmonella strains include S. typhi-aroC-aroD (Hone et al. Vacc. 9:810 (1991) S. typhimurium-aroA mutant (Mastroeni et al. Micro. Pathol. 13:477 (1992)) and Salmonella typhimurium 7207. Additional attenuated Salmonella strains that can be used in the invention include one or more other attenuating mutations such as (i) auxotrophic mutations, such as aro (Hoiseth et al. Nature, 291:238-239 (1981)), gua (McFarland et al Microbiol. Path., 3:129-141 (1987)), nad (Park et al. J. Bact, 170:3725-3730 (1988), thy (Nnalue et al. Infect. Immun., 55:955-962 (1987)), and asd (Curtiss, supra) mutations; (ii) mutations that inactivate global regulatory functions, such as cya (Curtiss et al. Infect. Immun., 55:3035-3043 (1987)), crp (Curtiss et al (1987), supra), phoP/phoQ (Groisman et al. Proc. Natl. Acad. Sci., USA, 86:7077-7081 (1989); and Miller et al. Proc. Natl. Acad. Sci., USA, 86:5054-5058 (1989)), phop.sup.c (Miller et al. J. Bact, 172:2485-2490 (1990)) or ompR (Dorman et al. Infect. Immun., 57:2136-2140 (1989)) mutations; (iii) mutations that modify the stress response, such as recA (Buchmeier et al. MoI. Micro., 7:933-936 (1993)), htrA (Johnson et al. MoI. Micro., 5:401-407 (1991)), htpR (Neidhardt et al. Biochem. Biophys. Res. Com., 100:894-900 (1981)), hsp (Neidhardt et al. Ann. Rev. Genet, 18:295-329 (1984)) and groEL (Buchmeier et al. Sci., 248:730-732 (1990)) mutations; mutations in specific virulence factors, such as IsyA (Libby et al. Proc. Natl. Acad. Sci., USA, 91:489-493 (1994)), pag or prg (Miller et al (1990), supra; and Miller et al (1989), supra), iscA or virG (d'Hauteville et al. MoI. Micro., 6:833-841 (1992)), plcA (Mengaud et al. Mol. Microbiol., 5:367-72 (1991); Camilli et al. J. Exp. Med, 173:751-754 (1991)), and act (Brundage et al. Proc. Natl. Acad. Sci., USA, 90:11890-11894 (1993)) mutations; (v) mutations that affect DNA topology, such as top A (Galan et al. Infect. Immun., 58: 1879-1885 (1990)); (vi) mutations that disrupt or modify the cell cycle, such as min (de Boer et al. Cell, 56:641-649 (1989)); (vii) introduction of a gene encoding a suicide system, such as sacB (Recorbet et al. App. Environ. Micro., 59:1361-1366 (1993); Quandt et al. Gene, 127:15-21 (1993)), nuc (Ahrenholtz et al. App. Environ. Micro., 60:3746-3751 (1994)), hok, gef, kil, or phlA (Molin et al. Ann. Rev. Microbiol., 47:139-166 (1993)); (viii) mutations that alter the biogenesis of lipopolysaccharide and/or lipid A, such as rFb (Raetz in Escherichia coli and Salmonella typhimurium, Neidhardt et al, Ed., ASM Press, Washington D.C. pp 1035-1063 (1996)), galE (Hone et al. J. Infect. Dis., 156:164-167 (1987)) and htrB (Raetz, supra), msbB (Reatz, supra; and U.S. Pat. No. 7,514,089); and (ix) introduction of a bacteriophage lysis system, such as lysogens encoded by P22 (Rennell et al. Virol, 143:280-289 (1985)), lamda murein transglycosylase (Bienkowska-Szewczyk et al. Mol. Gen. Genet., 184:111-114 (1981)) or S-gene (Reader et al. Virol, 43:623-628 (1971)).

The attenuating mutations can be either constitutively expressed or under the control of inducible promoters, such as the temperature sensitive heat shock family of promoters (Neidhardt et al. supra), or the anaerobically induced nirB promoter (Harbome et al. Mol. Micro., 6:2805-2813 (1992)) or repressible promoters, such as uapA (Gorfinkiel et al. J. Biol. Chem., 268:23376-23381 (1993)) or gcv (Stauffer et al. J. Bact, 176:6159-6164 (1994)).

In one embodiment, the bacterial delivery system is safe and based on a non-toxic, attenuated Salmonella strain that has a partial deletion of the msbB gene. This deletion diminishes the TNF immune response to bacterial lipopolysaccharides and prevents septic shock. In another embodiment, it also has a partial deletion of the purI gene. This deletion makes the bacteria dependent on external sources of purines and speeds clearance from non-cancerous tissues (13). In mice, the virulence (LD50) of the therapeutic strain is 10,000-fold less than wild-type Salmonella (72, 73). In pre-clinical trials, attenuated Salmonella has been administered systemically into mice and dogs without toxic side effects (17, 27). Two FDA-approved phase I clinical trials have been performed and showed that this therapeutic strain can be safely administered to patients (20). In one embodiment, the strain of bacteria is VNP20009, a derivative strain of Salmonella typhimurium. Deletion of two of its genes—msbB and purI—resulted in its complete attenuation (by preventing toxic shock in animal hosts) and dependence on external sources of purine for survival. This dependence renders the organism incapable of replicating in normal tissue such as the liver or spleen, but still capable of growing in tumors where purine is available.

Further, insertion of a failsafe circuit into the bacterial vector prevents unwanted infection and defines the end of therapy without the need for antibiotics to remove the bacteria (e.g., salmonella).

**Flagella**

1) flhDC Sequence

In one aspect, the flhDC sequence is the bicistronic, flhDC coding region found in the Salmonella Typhimurium 14028s strain or a derivative thereof

**Accession Number—**

- - fhD-NCBI Reference Sequence: NC_016856.1
  - flhC-NCBI Reference Sequence: NC_016856.1

Other sequences can also be used to control flagella activity, these include, for example, motA, motB, flhE, cheZ, cheY cheB, cheR, cheM, cheW, cheA, fliA, fliY, fliZ, fluB, fliS, fluE, fliF, fliJ, fliL, fliM, fliN, fliO, flip, fliQ, fliR, fliG, fliH, fliI, fliT, fliD, fliC, fljB, ycrG, flgN, flgM, flgA, flgB, flgC, flgD, flgE, flgF, flgG, flgH, flgI, flgJ, flgK and/or flgL.

### II. Vectors/Plasmids

In the present compositions and/or methods, DNA, RNA (e.g., a nucleic acid-based gene interfering agent) or protein may be produced by recombinant methods. The nucleic acid is inserted into a replicable vector for expression. Many such vectors are available. The vector components generally include, but are not limited to, one or more of the following: an origin of replication, one or more marker genes, an enhancer element, a promoter, and a transcription termination sequence and coding sequence. In some embodiments, for example in the utilization of bacterial delivery agents such as Salmonella, the gene and/or promoter (a sequence of interest) may be integrated into the host cell chromosome or may be presented on, for example, a plasmid/vector.

Expression vectors usually contain a selection gene, also termed a selectable marker. This gene encodes a protein necessary for the survival or growth of transformed host cells grown in a selective culture medium. Host cells not transformed with the vector containing the selection gene will not survive in the culture medium. Typical selection genes encode proteins that (a) confer resistance to antibiotics or other toxins, e.g., ampicillin, neomycin, methotrexate, or tetracycline, (b) complement auxotrophic deficiencies, or (c) supply critical nutrients not available from complex media.

Expression vectors can contain a promoter that is recognized by the host organism and is operably linked to the nucleic acid sequence, such as a nucleic acid sequence coding for an open reading frame. Promoters are untranslated sequences located upstream (5′) to the start codon of a structural gene (generally within about 100 to 1000 bp) that control the transcription of particular nucleic acid sequence to which they are operably linked. In bacterial cells, the region controlling overall regulation can be referred to as the operator. Promoters typically fall into two classes, inducible and constitutive. Inducible promoters are promoters that initiate increased levels of transcription from DNA under their control in response to some change in culture conditions, e.g., the presence or absence of a nutrient or a change in temperature. A large number of promoters recognized by a variety of potential host cells are well known.

Promoters suitable for use with prokaryotic hosts include the β-lactamase and lactose promoter systems, alkaline phosphatase, a tryptophan (trp) promoter system, hybrid promoters such as the tac promoter, and starvation promoters (Matin, A. (1994) Recombinant DNA Technology II, Annals of New York Academy of Sciences, 722:277-291). However, other known bacterial promoters are also suitable. Such nucleotide sequences have been published, thereby enabling a skilled worker to operably ligate them to a DNA coding sequence. Promoters for use in bacterial systems also can contain a Shine-Dalgarno (S.D.) sequence operably linked to the coding sequence.

Construction of suitable vectors containing one or more of the above-listed components employs standard ligation techniques. Isolated plasmids or DNA fragments are cleaved, tailored, and re-ligated in the form desired to generate the plasmids required.

In some embodiments of the invention, the expression vector is a plasmid or bacteriophage vector suitable for use in Salmonella, and the DNA, RNA and/or protein is provided to a subject through expression by an engineered Salmonella (in one aspect attenuated) administered to the patient. The term “plasmid” as used herein refers to any nucleic acid encoding an expressible gene and includes linear or circular nucleic acids and double or single stranded nucleic acids. The nucleic acid can be DNA or RNA and may comprise modified nucleotides or ribonucleotides and may be chemically modified by such means as methylation or the inclusion of protecting groups or cap- or tail structures.

One embodiment provides a Salmonella strain comprising a lysis gene or cassette operably linked to an intracellularly induced Salmonella promoter. In one embodiment, the promoter is a promoter for one of the genes in Salmonella pathogenicity island 2 type III secretion system (SPI2-T3SS) selected from the group SpiC/SsaB (accession no. CBW17423.1), SseF (accession no. CBW17434.1), SseG (accession no. CBW17435.1), SseI (accession no. CBW17087.1), SseJ (accession no. CBW17656.1 or NC_016856.1), SseK1 (accession no. CBW20184.1), SseK2 (accession no. CBW18209.1), SifA (accession no. CBW17257.1), SifB (accession no. CBW17627.1), PipB (accession no. CBW17123.1), PipB2 (accession no. CBW18862.1), SopD2 (accession no. CBW17005.1), GogB (accession no. CBW18646.2), SseL (accession no. CBW18358.1), SteC (accession no. CBW17723.1), SspH (accession no. STM4_1483), SspH2 (accession no. CBW18313.1), or SirP (examples/an embodiment of sequences that can be used in the instant compositions/methods are provided for by accession numbers and sequences provided throughout the specification; other sequences, including those with greater than about 80%, 85%, 90%, 95%, 96%, 97%, 98%, 99% and 100% identity may also be used in the composition/methods of the invention).

In one embodiment, the Salmonella gene under the regulation of an inducible promoter is selected from ftsW (accession no. CBW16230.1), ftsA (accession no. CBW16235.1), ftsZ (accession no. CBW16236.1), murE (accession no. CBW16226.1), mukF (accession no. CBW17025.1), imp (accession no. CBW16196.1), secF (accession no. CBW16503.1), eno (accession no. CBW19030.1), hemH (accession no. CBWJ6582.1), tmk (accession no. CBW17233.1), dxs (accession no. CBW16516.1), uppS (accession no. CBW16324.1), cdsA (accession no. CBW16325.1), accA (accession no. CBWJ6335.1), pssA (accession no. CBW18718.1), msbA (accession no. CBW17017.1), tsf (accession no. CBW16320.1), trmD (accession no. CBW18749.1), cca (accession no. CBW19276.1), inJB (accession no. CBW19355.1), rpoA (accession no. CBW19477.1), rpoB (accession no. CBW20180.1), rpoC (accession no. CBW20181.1), holA (accession no. CBW16734.1), dnaC (accession no. CBW20563.1), or eng (EngA accession no. CBW18582.1; EngB accession no. CBW20039.1).

Other inducible promotors for use in the invention, including to inducibly control flagella, include, but are not limited to:

### III. Therapeutic DNA, RNA and Peptides

The present invention delivers therapeutic DNA, RNA and/or peptides to cancer cells.

Gene silencing through RNAi (RNA-interference) by use of short interfering RNA (siRNA) can be used for therapeutic gene silencing. Short hairpin RNA (shRNA) transcribed from small DNA plasmids within the target cell has also been shown to mediate stable gene silencing and achieve gene knockdown at levels comparable to those obtained by transfection with chemically synthesized siRNA.

RNAi agents are agents that modulate expression of an RNA by an RNA interference mechanism. The RNAi agents employed in one embodiment of the subject invention are small ribonucleic acid molecules (also referred to herein as interfering ribonucleic acids), i.e., oligoribonucleotides, that are present in duplex structures, e.g., two distinct oligoribonucleotides hybridized to each other (e.g., an siRNA) or a single ribooligonucleotide that assumes a small hairpin formation to produce a duplex structure (e.g, shRNA).

dsRNA can be prepared according to any of a number of methods that are available in the art, including in vitro and in vivo methods, as well as by synthetic chemistry approaches. Single-stranded RNA can also be produced using a combination of enzymatic and organic synthesis or by total organic synthesis. The use of synthetic chemical methods enables one to introduce desired modified nucleotides or nucleotide analogs into the dsRNA.

In certain embodiments, instead of the RNAi agent being an interfering ribonucleic acid, e.g., an siRNA or shRNA as described above, the RNAi agent may encode an interfering ribonucleic acid, e.g., an shRNA, as described above. In other words, the RNAi agent may be a transcriptional template of the interfering ribonucleic acid. In these embodiments, the transcriptional template is typically a DNA that encodes the interfering ribonucleic acid. The DNA may be present in a vector, where a variety of different vectors are known in the art, e.g., a plasmid vector, a viral vector, etc.

Alternative the active agent may be a ribozyme. The term “ribozyme” as used herein for the purposes of specification and claims is interchangeable with “catalytic RNA” and means an RNA molecule that is capable of catalyzing a chemical reaction.

Exemplary target genes include, but are not limited to, EZH2 (accession number for human EZH2 mRNA is NM_004456). NIPP1 (accession number for human NIPP1 mRNA is NM_002713) and PP1 (accession numbers for human PP1 mRNA are PP1α mRNA: NM_002708; PP1β mRNA: NM_206876; PP1γ mRNA: NM_002710). EZH2, NIPP1 and PP1, would disrupt cancer cell processes and eliminate and/or diminish cancer stems cells. This will stop tumors from spreading/growing and prevent metastasis formation.

In another embodiment, the epigenetic target is at least one (e.g., mRNA) of NIPP1 (accession No. NM_002713); EZH2 (accession No. NM_004456); PP1α (accession No. NM_002708); PP1β (accession No. NM_206876); PP1γ (accession No. NM_002710); Suz12 (accession No. NM_015355); EED (accession No. NM_003797); EZH1 (accession No. NM_001991); RbAp48 (accession No. NM_005610); Jarid2 (accession No. NM_004973); YY1 (accession No. NM_003403); CBX2 (accession No. NM_005189); CBX4 (accession No. NM_003655); CBX6 (accession No. NM_014292); CBX7 (accession No. NM_175709); PHC1 (accession No. NM_004426); PHC2 (accession No. NM_198040); PHC3 (accession No. NM_024947); BMI1 (accession No. NM_005180); PCGF2 (accession No. NM_007144); ZNF134 (accession No. NM_003435); RING1 (accession No. NM_002931); RNF2 (accession No. NM_0072120; PHF1 (accession No. NM_024165); MTF2 (accession No. NM_007358); PHF19 (accession No. NM_001286840); SETD1A (accession No. NM_005255723); SETD1B (accession No. NM_015048); CXXC1 (accession No. NM_001101654); ASH2L (accession No. NM_004674); DPY30 (accession No. NM_032574); RBBP5 (accession No. NM_005057); WDR5 (accession No. NM_017588); KMT2A (accession No. NM_001197104); KMT2D (accession No. XM_006719616); KMT2B (accession No. NM_014727); KMT2C (accession No. NM_170606); KAT8 (accession No. NM_032188); KDM6A (accession No. NM_001291415); NCOA6 (accession No. NM_014071); PAGR1 (accession No. NM_024516); PAXIP1 (accession No. NM_007349); ASH1L (accession No. NM_018489); SMARCA2 (accession No. NM_003070); SMARCA4 (accession No. NM_001128844); BPTF (accession No. NM_182641); or SMARCA1 (accession No. NM_001282874).

In some embodiments the therapeutic peptide to be expressed by the bacterial cell is caspase, such caspase 3 (for example, expressed in its activated form), or NIPP1.

### IV. Cancer Treatment

Bacteria such as Salmonella, Clostridium and Bifidobacterium have a natural tropism for cancers, such as solid tumors. Types of cancer that can be treated using the methods of the invention include, but are not limited to, solid tumors such as sarcomas and carcinomas (e.g., fibrosarcoma, myxosarcoma, liposarcoma, chondrosarcoma, osteogenic sarcoma, chordoma, angiosarcoma, endotheliosarcoma, lymphangiosarcoma, lymphangioendotheliosarcoma, synovioma, mesothelioma, Ewing's tumor, leiomyosarcoma, rhabdomyosarcoma, colon carcinoma, pancreatic cancer, breast cancer, ovarian cancer, prostate cancer, squamous cell carcinoma, basal cell carcinoma, adenocarcinoma, sweat gland carcinoma, sebaceous gland carcinoma, papillary carcinoma, papillary adenocarcinomas, cystadenocarcinoma, medullary carcinoma, bronchogenic carcinoma, renal cell carcinoma, hepatoma, bile duct carcinoma, choriocarcinoma, seminoma, embryonal carcinoma, Wilm's tumor, cervical cancer, uterine cancer, testicular cancer, lung carcinoma, small cell lung carcinoma, bladder carcinoma, epithelial carcinoma, glioma, astrocytoma, medulloblastoma, craniopharyngioma, ependymoma, pinealoma, hemangioblastoma, acoustic neuroma, oligodenroglioma, schwannoma, meningioma, melanoma, neuroblastoma, and retinoblastoma).

In some aspects, the subject is treated with radiation and chemotherapy before, after or during administration of the bacterial cells described herein.

### V. Administration

The invention includes administration of the attenuated Salmonella strains described herein and methods for preparing pharmaceutical compositions and administering such as well. Such methods comprise formulating a pharmaceutically acceptable carrier with one or more of the attenuated Salmonella strains described herein.

A pharmaceutical composition of the invention is formulated to be compatible with its intended route of administration. Solutions or suspensions used for parenteral, intradermal, or subcutaneous application can include the following components: a sterile diluent such as water for injection, saline solution, fixed oils, polyethylene glycols, glycerine, propylene glycol or other synthetic solvents; antioxidants such as ascorbic acid or sodium bisulfite; chelating agents such as ethylenediaminetetraacetic acid; buffers such as acetates, citrates or phosphates and agents for the adjustment of tonicity such as sodium chloride or dextrose. pH can be adjusted with acids or bases, such as hydrochloric acid or sodium hydroxide. The parenteral preparation can be enclosed in ampoules, disposable syringes or multiple dose vials made of glass or plastic.

For intravenous administration, suitable carriers include physiological saline, bacteriostatic water, Cremophor EL™ (BASF; Parsippany, N.J.) or phosphate buffered saline (PBS). It must be stable under the conditions of manufacture and storage and must be preserved against the contaminating action of other (undesired) microorganisms. The carrier can be a solvent or dispersion medium containing, for example, water, ethanol, polyol (for example, glycerol, propylene glycol, and liquid polyetheylene glycol, and the like), and suitable mixtures thereof. The proper fluidity can be maintained, for example, by the use of a coating such as lecithin, by the maintenance of the required particle size in the case of dispersion and by the use of surfactants. In many cases, it will be preferable to include isotonic agents, for example, sugars, polyalcohols such as mannitol, sorbitol, sodium chloride in the composition. Prolonged absorption of the injectable compositions can be brought about by including in the composition an agent which delays absorption, for example, aluminum monostearate and gelatin.

Injectable solutions can be prepared by incorporating the active compound in the required amount in an appropriate solvent with one or a combination of ingredients discussed above. Generally, dispersions are prepared by incorporating the active compound into a vehicle which contains a basic dispersion medium and various other ingredients discussed above. In the case of powders for the preparation of injectable solutions, the preferred methods of preparation are vacuum drying and freeze-drying which yields a powder of the active ingredient plus any additional desired ingredient from a previously.

Oral compositions generally include an inert diluent or an edible carrier. For example, they can be enclosed in gelatin capsules. For the purpose of oral therapeutic administration, the active compound can be incorporated with excipients and used in the form of tablets, troches, or capsules.

Pharmaceutically compatible binding agents, and/or adjuvant materials can be included as part of the composition. The tablets, pills, capsules, troches and the like can contain any of the following ingredients, or compounds of a similar nature: a binder such as microcrystalline cellulose, gum tragacanth or gelatin; an excipient such as starch or lactose, a disintegrating agent such as alginic acid, Primogel, or corn starch; a lubricant such as magnesium stearate or Sterotes; a glidant such as colloidal silicon dioxide; a sweetening agent such as sucrose or saccharin; or a flavoring agent such as peppermint, methyl salicylate, or orange flavoring.

For administration by inhalation, the bacteria are delivered in the form of an aerosol spray from a pressurized container or dispenser which contains a suitable propellant, e.g., a gas such as carbon dioxide, or a nebulizer.

Systemic administration can also be by transmucosal or transdermal means. For transmucosal or transdermal administration, penetrants appropriate to the barrier to be permeated are used in the formulation. Such penetrants are generally known in the art, and include, for example, for transmucosal administration, detergents, bile salts, and fusidic acid derivatives. Transmucosal administration can be accomplished through the use of nasal sprays or suppositories. For transdermal administration, the bacteria are formulated into ointments, salves, gels, or creams as generally known in the art.

It is especially advantageous to formulate compositions in dosage unit form for ease of administration and uniformity of dosage. Dosage unit form as used herein refers to physically discrete units suited as unitary dosages for the subject to be treated; each unit containing a predetermined quantity of active compound calculated to produce the desired therapeutic effect in association with the required pharmaceutical carrier. The specification for the dosage unit forms of the invention are dictated by and directly dependent on the unique characteristics of the active compound and the particular therapeutic effect to be achieved, and the limitations inherent in the art of compounding such an active compound for the treatment of individuals.

When administered to a patient the attenuated Salmonella can be used alone or may be combined with any physiological carrier. In general, the dosage ranges from about 1.0 c.f.u./kg to about 1×1012 c.f.u./kg; optionally from about 1.0 c.f.u./kg to about 1×1010 c.f.u./kg; optionally from about 1.0 c.f.u./kg to about 1×108 c.f.u./kg; optionally from about 1×102 c.f.u./kg to about 1×108 c.f.u./kg; optionally from about 1×104 c.f.u./kg to about 1×108 c.f.u./kg; optionally from about 1×105 c.f.u./kg to about 1×1012 c.f.u./kg; optionally from about 1×105 c.f.u./kg to about 1×1010 c.f.u./kg; optionally from about 1×105 c.f.u./kg to about 1×108 c.f.u./kg.

## EXAMPLES

The following examples are provided in order to demonstrate and further illustrate certain embodiments and aspects of the present invention and are not to be construed as limiting the scope thereof.

### Example I

**Introduction**

Delivering protein drugs into the cytoplasm of cancer cells would expand the number of treatable cancer targets. More than 60% of the pathways that control cellular function are intracellular (1) and almost all are difficult to access. Intracellular pathways control most of the hallmarks of cancer (2) and have been the focus of a significant fraction of cancer research. Because of their specificity, protein biologics are excellent candidates for interfering with these pathways. However, bringing functional proteins across the cell membrane is technically challenging. Effective intracellular delivery, coupled with specific protein drugs, has the potential to provide new treatments for previously incurable cancers.

**Bacterial Cultures**

All bacterial cultures (both Salmonella and DH5α) were grown in LB (10 g/L sodium chloride, 10 g/L tryptone and 5 g/L yeast extract). Resistant strains of bacteria were grown in the presence of carbenicllin (100 μg/ml), chloramphenicol (33 μg/ml), kanamycin (50 μg/ml) and/or 100 μg/ml of DAP.

**Bacterial Strains and Plasmid Construction**

Fifteen strains of Salmonella Enterica serovar Typhimurium were used throughout the experiments (Table S1). All plasmids contained a ColE1 origin and either chloramphenicol or ampicillin resistance (Table S2). All assembled DNA constructs were transformed into chemically competent DH5a E. Coli (New England Biolabs, Ipswich, MA) before electroporation into Salmonella. All cloning reagents, buffer reagents, and primers were from New England Biolabs, Fisher Scientific (Hampton, NH), and Invitrogen, (Carlsbad, CA), respectively, unless otherwise noted.

For electroporation, Salmonella cultures were grown to an optical density between 0.6 and 0.8, washed twice with 25 ml of ice-cold water, and resuspended in 400 μl ice cold water. DNA (200 ng for plasmids and 1-2 μg for linear DNA) was mixed with 50 μl of the bacterial suspension and electroporated in a 1 mm electroporation cuvette at 1,800V and 25 μF with a time constant of 5 msec.

The parental control strain (Par) was based on an attenuated therapeutic strain of Salmonella (VNP20009) that has three deletions, ΔmsbB, ΔpurI, and Δxyl that eliminate most toxicities in vivo. To enable balanced-lethal plasmid retention a strain was used (VNP200010) that has the asd gene deleted (1). A second strain (ΔflhD Par) was the basis for many strains in the study (Table S1). This strain was generated by first deleting flhD, then asd.

Genetic deletions were created using a modified lambda red recombination protocol (2). Salmonella were transformed with pkd46 (Yale CGSC E. Coli stock center) and grown from a single colony in 50 ml of LB. At an optical density of 0.1, arabinose was added to the bacterial cultures to a final concentration of 20 mM. When the optical density reached between 0.6 and 0.8, bacteria were centrifuged at 3000×g and washed twice with 25 ml ice-cold, ultrapure water (Millipore). The pelleted Salmonella were resuspended in 400 μl ice-cold water. A linear DNA segment was designed to insert an in-frame deletion into the gene (here flhD). It was generated by PCR amplification of FRT-KAN-FRT from plasmid pkd4 using primers vr121 and vr309 (Table S3). This PCR product contained kanamycin resistance flanked by FRT recombination sites and 50 base pair regions homologous to flhD. After electroporation, Salmonella recovered in LB for 2 hours at 37° C. and were left overnight at room temperature. This recovery solution was plated on kanamycin (50 μg/ml) agar plates and incubated at 37° C. until colonies formed. Colonies were screened for knockouts by colony PCR. Successful transformants were plated on kanamycin plates and grown overnight at 43° C. to eliminate pkd46 from the bacteria.

A similar process was used to delete asd. Transformants with successful deletion of flhD, were transformed with pkd46. A PCR product was created to insert an in-frame deletion into asd by PCR amplifying FRT-CHLOR-FRT from plasmid pkd3 using primers vr266 and vr268 (Table S3). This PCR product contained chloramphenicol resistance flanked by FRT recombination sites and 50 base pair regions homologous to asd. During recovery, electroporated bacteria were plated on agar containing 33 μg/ml chloramphenicol and 100 μg/ml diaminopimelic acid (DAP). Successful transformants were grown in the presence of chloramphenicol, kanamycin and DAP.

To generate the intracellular reporting strain of Salmonella, parental Salmonella strain (Par) was transformed with a plasmid containing PsseJ-GFP (plasmid P1; Table S2). The construction of this plasmid was initiated by first creating a promoter-less-GFP plasmid from pLacGFP and pQS-GFP [1]. The pQS-GFP plasmid contains chloramphenicol resistance, the ColE1 origin of replication, and the asd gene. Expression of ASD is necessary in Δasd strains and creates a balanced lethal system that maintains gene expression in vivo. The Plac-GFP gene circuit was amplified from plasmid pLacGFP with primers nd1 and nd2 (Table S4). The PCR product and the plasmid were digested with Aat2 and Pci1 and ligated with T4 DNA ligase (NEB, catalog #M0202S). The PsseJ promoter was amplified from the genome of SL1344 Salmonella using primers nd3 and nd4 (Table S4). This PCR product and the backbone plasmid were ligated after digestion with XbaI and Pci1.

A strain that re-expresses flhDC (flhDC Sal, Table S1) was created by transforming ΔflhD Salmonella with plasmid P2 (Table S2). Plasmid P2 was formed from temporary plasmid P3. Plasmid P3 was formed by amplifying flhDC from Salmonella genomic DNA using primers vr46 and vr47 (Table S4) and ligating it into plasmid PBAD-his-mycA (Invitrogen; catalog #V430-01). The PCR product was digested with NcoI, XhoI and DpnI (NEB, catalog #s R0193S, R0146S and R0176L). The PBAD-his-myc plasmid was digested with NcoI and XhoI and treated with calf intestinal phosphatase (NEB, catalog #M0290) for three hours. The PCR product was ligated into the plasmid backbone with T4 DNA ligase (NEB, catalog #M0202S).

The Plac-GFP-myc circuit was inserted into P3 by Gibson Assembly. (1) The insert (Plac-GFP-myc) was amplified from plasmid pLacGFP (1) using primers vr394 and vr395 (Table S4), which added homology regions to the backbone and added the myc tag. (2) The backbone plasmid (P3) was amplified using primers vr385 and vr386, which added homology to the insert. (3) Both PCR products were digested with DpnI for three hours, (4) and ligated by Gibson Assembly (HiFi master mix, NEB, catalog #E2621L). The gene for aspartate semialdehyde dehydrogenase (asd) gene was inserted by Gibson Assembly by amplifying asd from genomic Salmonella DNA using primers vr424 and vr425 and amplifying the plasmid backbone with primers vr426 and vr427.

A strain that re-expresses flhDC and produces GFP after invasion (flhDC reporting, Table S1) was created by transforming ΔflhD Salmonella with plasmid P4 (Table S2). The PsseJ-GFP-myc genetic circuit was amplified from P1 using primers vr269 and vr270, and the backbone of plasmid P3 was amplified using primers vr271 and vr272. The two PCR products were ligated by Gibson Assembly.

To generate the PsifA intracellular promoter-reporter strain, the PsifA promoter was cloned from Salmonella genomic DNA using primers nd5 and nd6 and inserted into P1 using XbaI and Pci1 creating plasmid P5. The PsifA reporter strain was created by transforming plasmid P5 into background Salmonella by electroporation. The generation of the PsseJ reporter strain is described above. To investigate lysis in Salmonella, lysis gene E (LysE) was put under control of PBAD. LysE was cloned using primers nd7 and nd8 and inserted into pBAD/Myc-His A (Invitrogen) using NcoI and KpnI to form plasmid P6.

Intracellular delivering (ID) Salmonella were created by cloning the Lysin E gene behind the PsseJ promoter. LysE was amplified using primers nd9 and nd10 and cloned into P1 using XbaI and Aat2. The Plac-GFP circuit was added to this plasmid by cloning it from plasmid pLacGFP using primers nd 11 and nd12 and inserting using SacI to create plasmid P7. This plasmid constitutively expresses myc-tagged GFP to identify bacteria in both live-cell and fixed-cell assays.

Genomic knockouts ΔsifA and ΔsseJ were created using the modified lambda red recombination protocol described in the creation of ΔflhD Salmonella above. Salmonella were transformed with pkd46. Linear DNA with homologous flanking regions was produced by PCR of plasmid pkd4 using primers vr432 and vr433 for ΔsseJ; and vr434 and vr435 for ΔsifA. After electroporation and recovery, colonies were screened for knockouts by colony PCR of the junction sites of the inserted PCR amplified products. Successful transformants were plated on kanamycin plates (50 μg/ml) and grown overnight at 43° C. to remove pkd46.

ID Salmonella that re-expresses flhDC (flhDC-ID Sal) was created by transforming ΔflhD with plasmids P8. Plasmid P8 was created by amplifying the Pssej-LysE gene circuit from P7 using primers vr398 and vr399 and ligating it into plasmid P2 using Gibson Assembly. The P2 backbone plasmid was amplified using primers vr396 and vr397.

A strain of ID Salmonella that constitutively expresses luciferase (ID Sal-luc; Table S1) was created by cloning Plac-luc from pMA3160 (Addgene) using primers ch1 and ch2. The P7 plasmid backbone was amplified with primers ch3 and ch4 and the pieces were ligated by Gibson Assembly to form plasmid P9 (Table S2).

To create ID Salmonella that express anti-b-actin nanobody (NB), PBAD inducible nanobody was cloned in place of flhDC in plasmid P8. The actin nanobody (Chromotek, catalog #acr) was amplified using primers vr466 and vr467. The delivery plasmid backbone was amplified using primers vr448 and vr449. The two PCR products were ligated by Gibson Assembly to create plasmid P10.

To create ID Salmonella that express the central domain of NIPP1 (NIPP1-CD), NIPP1-CD was cloned into plasmid pLacGFP. NIPP1-CD and the backbone plasmid were amplified using primers nd13-nd16 ligated by Gibson Assembly. The pLac-NIPP1-CD circuit was cloned using primers nd11 and nd17 (Table S4) and inserted into P7 using SacI to create plasmid P11.

To create ID Salmonella that intracellularly deliver CT caspase-3 (CT Casp-3), parental Salmonella were transformed with plasmid P12. This plasmid was created by PCR amplifying template DNA encoding for CT caspase-3 using primers, vr450 and vr451 from the constitutively two-chain (CT) caspase-3 encoding plasmid pC3D175CT. The pC3D175CT plasmid (Hardy Lab DNA archive Box 7, line 62) was constructed similarly to the caspase-6 CT expression construct [3] using Quikchange mutagenesis on a construct encoding full-length human caspase-3 in a pET23 expression vector (Addgene). Plasmid pC3D175CT encodes human caspase-3 residues 1-175, followed by a TAA stop codon, a ribosome binding sequence and the coding sequence for a start methionine and an inserted serine followed by the coding sequence for residues 176-286 with a six-histidine tag appended. The backbone of plasmid P8 was PCR amplified using primers vr448 and vr449 and the PCR products were ligated as previously described.

**Cell Culture**

Four cancer cell lines were used: 4T1 murine breast carcinoma cells; Hepa1-6 murine hepatocellular carcinoma cells; MCF7 human breast carcinoma cells and LS174T human colorectal carcinoma cells (ATCC, Manassas, VA). All cancer cells were grown and maintained in Dulbecco's Minimal Eagle Medium (DMEM) containing 3.7 g/L sodium bicarbonate and 10% fetal bovine serum. For microscopy studies, cells were incubated in DMEM with 20 mM HEPES buffering agent and 10% FBS. To generate tumor spheroids, single cell suspensions of LS174T cells were transferred to PMMA-coated cell culture flasks (2 g/L PMMA in 100% ethanol, dried before use).

Salmonella Invasion into Cancer Cells In Vitro

To observe invasion into cancer cells, Salmonella were administered to mouse 4T1 breast cancer cells grown on coverslips using an invasion assay. The cells and bacteria were stained with phalloidin and anti-Salmonella antibodies and imaged with 100× oil immersion microscopy. The general procedures for invasion assays, immunocytochemistry, and microscopy are detailed in the following sections.

**Invasion Assays**

For invasion assays, cancer cells were grown on coverslips for fixed-cell imaging or on well plates for live-cell imaging. For fixed imaging, glass coverslips were placed in 12-well plates and sterilized with UV light in a biosafety hood for 20 minutes. Mouse 4T1 or human MCF7 cells were seeded on the coverslips at 40% confluency and incubated overnight in DMEM. Concurrently, Salmonella were grown to an optical density (OD; at 600 nm) of 0.8. After incubation, the Salmonella were added to the 4T1 cultures at a multiplicity of infection (MOI) of 10 and allowed to infect the cells for two hours. After this invasion period, the cultures were washed five times with 1 ml of phosphate buffered saline (PBS) and resuspended in 2 ml of DMEM with 20 mM HEPES, 10% FBS and 50 μg/ml gentamycin. The added gentamycin removes extracellular bacteria. After six hours of incubation, the media was removed, and the coverslips were fixed with 10% formalin in PBS for 10 minutes.

A similar procedure was used for live-cell imaging. Cells were grown directly on well plates in DMEM (3.7 g/L sodium bicarbonate, 10% FBS) to a confluency between 30 and 50%. After growth to OD 0.8, Salmonella were added to the cell cultures at an MOI of 25 for 2 hours. After invasion, the cancer cells were washed five times with PBS, and 2 ml of DMEM with 50 μg/ml gentamycin was added to each well. Cells and bacteria were directly imaged microscopically.

**Immunocytochemistry**

Immunocytochemistry was used to obtain detailed images of Salmonella invaded into cancer cells grown on coverslips. After fixing the coverslips with formalin, they were blocked with staining buffer (PBS with 0.1% Tween 20, 1 mM EDTA, and 2% bovine serum albumin [BSA]) for 30 minutes. The Tween 20 in this buffer selectively permeabilizes mammalian cell membranes, while leaving bacterial membranes intact.

After permeabilization, coverslips were stained to identify Salmonella, released GFP, vacuolar membranes and/or intracellular f-actin with (1) rabbit anti-Salmonella polyclonal antibody (Abcam, ab35156) or FITC-conjugated rabbit anti-Salmonella polyclonal antibody (Abcam, ab69253) (2) rat anti-myc monoclonal antibody (Chromotek, catalog #9e1-100), (3) rabbit anti-LAMP1 polyclonal antibody (Abcam, catalog #ab24170), and (4) Alexaflor-568-conjugated phalloidin (ThermoFisher, catalog #A12380), respectively. Three different staining combinations were used: (1) Salmonella alone; (2) Salmonella, released GFP and actin; and (3) Salmonella, released GFP and vacuoles.

For Salmonella alone staining (combination 1), coverslips were stained with FITC-conjugated anti-Salmonella antibody at 30° C. for one hour and washed three times with staining buffer.

For Salmonella, released GFP and actin staining (combination 2), coverslips were stained with anti-Salmonella and anti-myc primary antibodies at 30° C. for one hour, and washed twice times with staining buffer. Coverslips were incubated with secondary antibodies at a 1:200 dilution for one hour at 30° C.: Alexaflor-647 chicken anti-rabbit (ThermoFisher, catalog #A21443), Alexaflor-488 donkey anti-rat (ThermoFisher, catalog #A21208), and Alexaflor-568-conjugated phalloidin to identify Salmonella, GFP and intracellular f-actin, respectively.

For Salmonella, released GFP and vacuole staining (combination 3), coverslips were stained sequentially with anti-LAMP1 primary antibodies at 30° C. for one hour, and washed three times with staining buffer. Coverslips were incubated with Alexaflor-647 chicken anti-rabbit secondary antibodies (ThermoFisher, catalog #A21443) at a 1:200 dilution for one hour at 30° C. and washed four times with staining buffer. Coverslips were then stained with FITC-conjugated anti-Salmonella antibody and anti-myc primary antibody; and washed three times with staining buffer. Coverslips were incubated with Alexaflor-568 goat anti-rat secondary antibodies (ThermoFisher, A11077) at a 1:200 dilution for one hour at 30° C. to identify GFP.

After all staining, coverslips were washed three times with staining buffer and mounted to glass slides using 20 μl mountant with DAPI (ProLong Gold Antifade Mountant, ThermoFisher, catalog #P36962). Mounted coverslips were cured overnight at room temperature.

**Microscopy**

Samples were imaged on a Zeiss Axio Observer Z.1 microscope. Fixed cells on coverslips were imaged with a 100× oil immersion objective (1.4 NA). Tumor sections were images with 10× and 20× objectives (0.3 and 0.4 NA, respectively). Time lapse fluorescence microscopy of live cells in well plates and tumor-chip devices were housed in a humidified, 37° C. environment and imaged with 5×, 10×, 63× or 100× objectives (0.2, 0.3, 1.4 and 1.4 NA, respectively). Fluorescence images were acquired with either 480/525 or 525/590 excitation/emission filters. All images were background subtracted and contrast was uniformly enhanced. Some image analysis was automated using computational code (MATLAB, Mathworks).

**Intracellular Salmonella in Tumors**

To determine the fraction of tumor-colonized Salmonella that are intracellular, BALB/c mice with 4T1 tumors were injected with 2×106 CFU of Intracellular reporting Salmonella (with PsseJ-GFP; Table S1). Ninety-six hours after bacterial injection, mice were sacrificed and tumors were excised, sectioned and stained as described in the Immunohistochemistry section below. Tumor sections were stained to identify Salmonella and GFP, which is produced by intracellular Salmonella. The fraction of intracellular Salmonella was determined by identifying Salmonella (n=1,258) in 8 images and determining the number that co-localize with GFP.

**Immunohistochemistry**

Excised tumor sections were fixed in 10% formalin for 3 days. Fixed tumor samples were then stored in 70% ethanol for 1 week. Tumor samples were embedded in paraffin and sectioned into 5 μm sections. Deparaffinization was performed by washing the sectioned tissue three times in 100% xylene, twice in 100% ethanol, once in 95% ethanol, once in 70% ethanol, once in 50% ethanol, and once in DI water. Each wash step was performed for 5 minutes. Antigen retrieval was performed by incubating the tissue sections in 95° C., 20 mM sodium citrate (pH 7.6) buffer for 20 minutes. Samples were left in sodium citrate buffer until the temperature reduced to 40° C. Samples were then rehydrated with two quick (<1 minute) rinses in DI water followed by one five-minute wash in TBS-T.

Prior to staining, tissue sections were blocked with Dako blocking buffer (Dako, catalog #X0909) for one hour. Tissue sections were stained to identify Salmonella and GFP with 1:100 dilutions of (1) FITC-conjugated rabbit anti-Salmonella polyclonal antibody (Abcam, catalog #ab69253), and (2) either rat anti-myc monoclonal antibody (Chromotek, catalog #9e1-100) or rat anti-GFP monoclonal antibody (Chromotek, catalog #3h9-100) in Tris buffered saline with 0.1% Tween 20 (TBS-T) with 2% BSA (FisherScientific, catalog #BP9704-100). Sections were washed three times in TBS-T w/ 2% BSA and incubated with Alexaflor-568 goat anti-rat secondary antibodies (ThermoFisher, catalog #A11077). After washing sections three times with TBS-T, 40 μl of mountant with DAPI (ThermoFisher, catalog #P36962) and a cover slip were added to each slide. Slides were incubated at room temperature for 24 hours until the mountant solidified.

**Flow Cytometry Analysis of Bacterial Invasion in Tumors**

Flow cytometry was used to identify cells in tumors that were invaded by Salmonella and the effect of inducing flhDC on invasion. The types of cells invaded by Salmonella was determined by isolating cells that contained invaded Salmonella and stratifying them into carcinoma, immune and other tumor-associated cells using EPCAM and anti-CD45 antibodies. The effect of inducing flhDC on cell invasion was determined by comparing mice administered flhDC-uninduced and flhDC-induced bacteria and counting the percentage of cells of the three cell types.

Two groups of mice were injected with 2×106 CFU of flhDC Salmonella (Table S1) via the tail vein. To induce production from the PBAD-flhDC gene construct in the flhDC-induced group (n=9), 100 μg of arabinose in 400 μl PBS was administered by intraperitoneal (IP) injection at 48 and 72 hours after bacterial injection. The control, flhDC-uninduced group (n=8) received IP injections at the same times. Ninety-six hours after bacterial injection, mice were sacrificed, and tumors were excised and cut in half. Tumors were processed into single cell suspensions, stained, and analyzed by flow cytometry.

To create a single cell suspension from excised tumors, they were minced with a sterile razor blade in 5 ml of RPMI with 20 mM HEPES, 10% FBS, 1 mg/ml collagenase D (Roche, catalog #11088866001), 200 units/ml of DNAse I (Roche, catalog #04716728001), and 50 μg/ml of gentamicin (ThermoFisher, catalog #BP918-1) to prevent bacterial overgrowth/invasion. Once tumor pieces were less than 5 mm long, the tumor slurry was added to a 7 ml douncer and dounced ten times. The slurry was placed in a single well of a six well plate and incubated at 37° C. for two hours. To separate the cells, the suspension was filtered through a 40 μm cell strainer (ThermoFisher, catalog #22-363-547) and centrifuged for five minutes at 300×g. Red blood cells (RBCs) were lysed by incubating the single cell suspension with RBC lysis buffer (150 mM ammonium chloride, 12 mM sodium bicarbonate and 0.1 mM EDTA) for ten minutes. The cell suspensions were added to 10 ml of D-PBS (Hyclone, catalog #SH30256001) and spun at 300×g for 5 minutes.

Single cell suspensions were fixed in PBS containing 1 mM EDTA and 5% formaldehyde for ten minutes at room temperature. Fixed cells were spun at 600×g for five minutes and resuspended in blocking buffer for one hour. Blocking buffer is TBS-T with 2% BSA and 1 mM EDTA. The 0.1% Tween 20 permeabilizes the cancer cells but not the bacteria as described in the Immunocytochemistry section above. Cell suspensions were sequentially stained with FITC-conjugated anti-Salmonella antibody (Abcam, catalog #ab69253), PE dazzle 594 anti-CD326 (EpCAM; BioLegend, catalog #118236), and APC anti-CD45 (Biolegend, catalog #103112) at concentrations of 1:2000, 1:2000 and 1:1000, respectively. First, anti-Salmonella antibodies were added to cells for 45 minutes, followed by four washed six times with staining buffer (2% BSA, 1 mM EDTA and 0.1% Tween in PBS). Then EpCAM and anti-CD45 were added for 45 minutes, followed by two washes. Fluorescence minus one (FMO) of each sample were used as gating controls for each fluorophore. Samples were analyzed on a custom-built flow cytometer (dual LSRFortessa 5-laser, BD). All fluorophores were compensated with compensation beads (BD, catalog #552845) and did not carry more than 2% bleed over into any other channel. Cells were first identified if they contained intracellular Salmonella. Non-immune cells (cancer and other associated cells) were identified by samples stained with all antibodies except CD45 (i.e. FMO gating controls). Non-cancer cells (immune and other associated cells) were identified by samples stained with all antibodies except anti-EpCAM (CD326).

Effect of flhDC Induction on Bacterial Invasion into Cells in Culture

To determine the effect of expressing flhDC in bacterial invasion, 4T1 cells were grown on glass cover slips as described in the Infection assay section above. Inducible flhDC Salmonella (Table S1) were grown in LB with 20 mM arabinose to induce flhDC expression. Control (flhDC−) bacteria were grown without arabinose. Cancer cells were infected with both induced flhDC+ and flhDC− Salmonella at an MOI of 10 (n=4 for each condition). For the induced flhDC+condition, 20 mM arabinose was added to the mammalian culture to maintain expression. Eighteen hours after invasion, the cancer cells were stained to identify intracellular Salmonella (Salmonella alone, combination 1) as described in the Immunocytochemistry section above. Three images were acquired at 20× for each coverslip, for a total of 12 images per condition. Invasion was quantified by randomly identifying 20 cancer cells from the DAPI channel of each image. Each cell defined as invaded if Salmonella staining was co-localized with the nucleus or was within 10 μm of the nucleus. Invasion fraction was defined as the number of invaded cells over the total number of cells.

Effect of flhDC on Invasion into Tumor Masses In Vitro

To quantify invasion into tumor masses, engineered Salmonella were administered to tumor-on-a-chip devices developed in our laboratory (6, 7). Microfluidic tumor-on-a-chip devices were fabricated using negative tone photoresist and PDMS based soft lithography. Master chips were constructed by spin coating a layer of SU-8 2050 onto a silicon wafer at 1250 RPM for 1 minute. This speed corresponded to an SU-8 2050 thickness of 150 μm. The silicon wafer was baked at 65° C. for 5 minutes followed by 95° C. for 30 minutes. Microfluidic designs printed on a high-resolution transparency were placed over the silicon wafer in a mask aligner. The silicon wafer with the overlaid mask was exposed to UV light (22 J/cm2) for 22 seconds. Silicon wafers were baked for 5 minutes at 65° C. followed by 95° C. for 12 minutes. Wafers were then developed in PGMEA developing solution for 10 minutes and/or until microfluidic features were microscopically distinct with sharp and defined edges.

Soft lithography was used to create the multilayer tumor on a chip device with 12 tumor chambers (two conditions with six chambers each). PDMS (Sylgard 184) at ratios of 9:1 and 15:1 were used for the channel and valve layers, respectively. The channel layer was placed on a spin coater for 1 minute at 220 rpm in order to achieve a PDMS thickness of 200 μm. The silicon wafers were degassed for 45 minutes to eliminate air bubbles in the PDMS. The silicon wafers were baked at 65 degrees for approximately one hour or until both PDMS layers were partially cured. The top valve layer of PDMS was cut and removed from the silicon wafer and aligned on top of the channel layer using a stereomicroscope. The combined layers were baked for one hour at 95° C. in order to covalently bind the two layers. The multilayered PDMS device and a glass slide was plasma treated in a plasma cleaner (Harrick) for 2.5 minutes. Valves were pneumatically actuated with a vacuum pump and the PDMS was placed on the plasma treated glass slide. Valves were actuated until the device was ready for use.

The tumor-on-a-chip was sterilized with 10% bleach followed by 70% ethanol, each for one hour. Microfluidic chips were equilibrated with media (DMEM with 20 mM HEPES, pH 7.4) for one hour. Valve actuation was used to position tumor spheroids in the tumor chambers. Valves at the rear of the chambers were opened while the efflux channel was closed. After the tumor masses were positioned, the valves were reset so that the rear valves were closed and the influx and efflux channels were open.

Prior to administration to the device, flhDC reporting Salmonella (Table S1) were grown in LB with 20 mM arabinose to induce flhDC expression. These Salmonella have inducible flhDC (PBAD-flhDC) and produce GFP when intracellular (PsseJ-GFP). Control (flhDC−) Salmonella of the same strain were grown without arabinose. The bacteria were centrifuged and resuspended in culture medium (DMEM with 20 mM HEPES) at a density of 2×107 CFU/ml. For the induced flhDC+ condition, 20 mM arabinose was added to the medium. Bacteria-containing media (flhDC+ and flhDC−; n=6 chambers each) were perfused through the tumor-on-a-chip devices for one hour at 3 μm/min for a total delivery of 2×106 CFU to each device. Bacterial administration was followed by bacteria-free media (with 20 mM HEPES) for 48 hours.

Devices were imaged at 30-minute intervals. Invasion was quantified at 31 h by measuring GFP expression by invaded bacteria in the tumor masses. Regions of interest were defined around the borders of the tumor masses. The extent of invasion was determined as the average GFP fluorescence intensity in each tumor mass. Intensities were normalized by the intensity of the average tumor mass administered control (flhDC−) Salmonella.

**Intracellular Activation of the PsifA and PsseJ Promoters**

Salmonella with GFP-reporting constructs for the PsifA and PsseJ promoters were grown in LB. These Intracellular reporting and PsifA strains contain constructs PsseJ-GFP and PsifA-GFP, respectively (Table S1). Both bacterial strains were administered to MCF7 cancer cells in six well plates at an MOI of 25 as described in the Invasion Assay section above. Live cells were imaged at 20× magnification, three hours after invasion. Images of extracellular bacteria were acquired in LB culture in six well plates at 20×. Extracellular promoter activity was determined as the average fluorescence intensity of bacteria from three wells each and normalized to the average intensity of PsseJ bacteria. The increase in promoter activity following cellular invasion was determined by averaging the fluorescence intensity of bacteria in cells in three wells and comparing it to the average intensity of extracellular bacteria.

**Bacterial Death Caused by Inducing Expression of Lysin E**

Salmonella strain PBAD-LysE (Table S1) was grown in LB in 3 ml culture tubes to an average OD of 0.25. OD was measured every 30 minutes for three hours. After 90 minutes of growth, three of the cultures were induced with 10 mM arabinose. Arabinose was not added to three control cultures. Growth and death rates were determined by fitting exponential functions to bacterial density starting at time zero (for growth) and 90 minutes (for bacterial death).

**Intracellular Lysis and GFP Delivery**

To visualize and quantify triggered intracellular lysis and GFP delivery, ID Salmonella were administered to cancer cells on coverslips and in well plates as described in the Invasion Assay section above. ID Salmonella constitutively express GFP (Plac-GFP) and express Lysin E after activation of PsseJ (PsseJ-LysE).

To quantify the extent and rate of lysis, ID Salmonella were administered to MCF7 cancer cells at an MOI of 25. Parental Salmonella that constitutively express GFP (transformed with plasmid pLacGFP) were used as controls. Transmitted-light images of cancer cells and fluorescent images of bacteria were acquired at 20× every 30 minutes for 10 hours. From three wells, 200 cancer cells were randomly selected from the first transmitted image for each condition. Over the time of the experiment, cells were scored if any bacteria invaded and when these intracellular bacteria lysed. The lysis fraction was defined as the number of cells with lysed bacteria over the total number of observed cells. The rate of intracellular lysis was determined by binning the number of cells with lysed bacteria per hour and fitting an exponential function to the cumulative fraction of cells with lysed bacteria.

The comparison of growth and death rates were (1) the growth rate of parental Salmonella in LB, (2) the growth rate of PBAD-LysE Salmonella in LB, (3) the death rate of PBAD-LysE Salmonella after induction with arabinose, (4) the growth rate of Pssei-LysE Salmonella in LB, and (5) the lysis (death) rate of Pssei-LysE Salmonella after invasion into cancer cells.

To generate images of bacterial lysis and GFP delivery, ID Salmonella were administered to 4T1 cancer cells grown on coverslips at an MOI of 10. After six hours, the coverslips were fixed and stained for Salmonella and released GFP (antibody combination #2) as described in the Immunocytochemistry section above. Images were acquired at 100× with oil immersion.

**Bacterial Protein Content**

To quantify the amount of produced GFP, ID Salmonella (Table S1) were grown in LB. The bacteria were centrifuged, washed and resuspended at four densities: 106, 107, 108, and 109 bacteria per 40 μl Laemmli buffer, which lysed the bacteria. A GFP standard was loaded at three concentrations: 1, 10 and 100 ng per 40 μl Laemmli buffer. Samples were boiled and loaded onto NuPAGE 4-12% protein gels (Invitrogen, catalog #NPO0321BOX) in MOPS buffer. Resolved gels were transferred to PVDF blotting paper. Membranes were blocked with 2% bovine serum albumin in Tris-buffered saline with 5% skim milk powder and 0.1% Tween 20 (TBST+milk) for 1 hour. Blots were incubated with rat anti-GFP monoclonal antibody (Chromotek, catalog #3h9-100) primary antibody in TBST+milk overnight. Blots were washed three times with (TBST) and incubated with HRP-conjugated goat anti-rat secondary antibody (Dako, catalog #X0909) for one hour at room temperature in TBST-milk.

**Lysis and GFP Release in Cells and SCVs**

In order to assess GFP release from vacuoles, ID Salmonella where administered to 4T1 cancer cells. A specialized staining technique was used to identify SCVs and isolate released GFP from un-released, intra-bacterial GFP. The 4T1 cells were grown on glass coverslips were infected with ID Salmonella (Table S1) at an MOI Of 10 using the methods described in the Invasion Assay section.

At two time points, 6 and 24 hours, four coverslips were fixed and permeabilized as described in the Immunocytochemistry section above. The blocking buffer used for permeabilizing the cells contained Tween 20, which selectively permeabilized mammalian, but not bacterial cell membranes. This allowed primary antibodies to bind GFP in the mammalian cytoplasm, but not inside un-lysed bacteria. After permeabilization, cells were stained for Salmonella, released GFP, and vacuoles (combination 3) in the Immunocytochemistry section) using anti-Salmonella, anti-myc, and anti-LAMP1 antibodies.

After mounting, coverslips were imaged under oil immersion at 100× magnification. Acquired images were background subtracted and borders were drawn around cells (n=24 at 6 h, and n=7 at 24 h). Released GFP was divided into two groups: vacuolar and cytosolic. Vacuolar GFP was surrounded by LAMP1-stained regions. Cytosolic GFP was all other GFP inside cells. For each cell, the vacuolar and cytosolic GFP fractions were determined as the sum of pixel intensities in the region divided by the sum of intensities in both regions (i.e. the total in the cell). To visualize the localization of released GFP in cells over time, ID Salmonella were administered to 4T1 cancer cells. The cancer cells were grown on glass coverslips were infected with ID Salmonella (Table S1) at an MOI Of 10. At two time points, 6 and 24 hours, four coverslips were fixed and permeabilized as described above. The cells were stained for Salmonella, released GFP, and β-actin (combination 2) with anti-Salmonella and anti-myc antibodies, and phalloidin. Actin staining enables visualization of structures and boundaries. Images were acquired at 100× with oil immersion.

**Dynamic Measurement of GFP Release and Diffusion**

To measure the rate of GFP dispersion through cells after lysis, MCF7 cancer cells were grown on 96-well plates with coverslip glass bottoms for imaging (ThermoFisher, catalog #160376). ID Salmonella were administered at an MOI Of 25 using the methods for live-cell imaging as described in the Invasion Assay section. After washing away extracellular bacteria and adding gentamycin, one cell with intracellular bacteria was identified, and transmitted and fluorescence images were acquired at 63× every minute for 14 hours. This process was repeated ten times. Fluorescence images were selected to start with intact bacteria and end after GFP diffusion. These images were converted into stacks in Zen (Zeiss) and intensities were measured on lines passing through bacterial centers at time zero (before lysis) until diffusion was complete. The GFP spatiotemporal intensity profiles were fit to the radial diffusion equation.

\(\begin{matrix}
{\frac{\partial C}{\partial t} = {\mathcal{D}\frac{1}{r^{2}}{\frac{\partial}{\partial r}\left( {r^{2}\frac{\partial C}{\partial r}} \right)}}} & (1)
\end{matrix}\)

In this equation, C is the GFP concentration and D is the effective diffusivity of GFP in the cytosol. When there is an instantaneous release of material at t=0 from r=0 (i.e. lysis), equation (1) has an analytical solution.

\(\begin{matrix}
{{C\left( {r,t} \right)} = {\left( {4{\pi\mathcal{D}}t} \right)^{\frac{2}{3}}\exp\left( \frac{- r^{2}}{4\mathcal{D}t} \right)}} & (2)
\end{matrix}\)

Cytosolic diffusivity of released GFP, D, was determined be fitting the GFP intensity profiles to equation (2) using least-squared fitting.

**Location of GFP Release**

To quantify the location of GFP release in cells, ID Salmonella where administered to 4T1 cancer cells on glass coverslips at an MOI Of 10 using the methods in the Invasion Assay section. At 6 hours, three coverslips were fixed, permeabilized and stained to identify Salmonella, released GFP, and vacuoles (combination 3) in the Immunocytochemistry section) using anti-Salmonella, anti-myc, and anti-LAMP1 antibodies. After mounting, coverslips were imaged under oil immersion at 100× magnification. Acquired images were background subtracted and Salmonella were identified in seven 86.7×66.0 μm regions across the three coverslips. Every bacterium within the regions was classified as un-lysed or lysed if co-localized with released GFP. The location of each lysed Salmonella was determined based on co-localization with LAMP1 staining as inside or outside SCVs. The fraction of released GFP in vacuoles was the number of lysed Salmonella in SCVs over total lysed Salmonella.

**Dependence of Protein Release on Residence in SCVs**

To determine the dependence of protein release on residence in SCVs, ID Salmonella with two gene knockouts were administered to cancer cells. 4T1 cancer cells were grown on coverslips and infected with ΔsifA, ΔsseJ, or ID Salmonella (n=3 for each condition). All three of these strains contained the PsseJ-lysE and Plac-GFP-myc gene circuits (Table S1). The ΔsifA strain predominantly accumulates in the cellular cytoplasm and the ΔsseJ strain predominantly accumulates in SCVs and does not escape into the cytoplasm. Bacteria were administered at an MOI of 10 as described in the Invasion Assay section. At 6 hours after invasion, the cancer cells were fixed, permeabilized and stained for Salmonella and released GFP as described in the Immunocytochemistry section. Nine images from three coverslips were acquired at 20× for each condition. Images were background subtracted. Lysis fraction was calculated using pixel by pixel image analysis in MATLAB. Lysis was identified as pixels that positively stained for GFP-myc. The permeabilization technique prevented staining of GFP inside un-lysed Salmonella. Un-lysed Salmonella were identified as pixels that stained for Salmonella but not GFP-myc. Total bacterial pixels is the sum of these values. Lysis fraction is the number of lysis pixels over total bacterial pixels.

**Dependence of Protein Delivery on Invasion and Intracellular Lysis**

Four strains of Salmonella were administered to cancer cells to determine the necessity of the two engineered gene circuits, PsseJ-LysE and PBAD-flhDC, on protein delivery. Two strains were used: flhDC Sal and flhDC-ID Sal (Table S1). Both of these strains have flhD deleted and only express flhDC after induction with arabinose. The flhDC-ID Sal strain also contains the PsseJ-LysE circuit which induces lysis after cell invasion. Prior to invasion, two cultures of flhDC Sal and flhDC-ID Sal bacteria were grown in LB with 20 mM arabinose to induce flhDC expression. Two cultures were grown without arabinose. For microscopy analysis, 4T1 cancer cells were grown on coverslips and infected at an MOI of 10 with one of the four strains: PsseJ-LysE−, flhDC−; PsseJ-LysE−, flhDC+; PsseJ-LysE+, flhDC−; or PsseJ-LysE+, flhDC+. For flow cytometry, 4T1 cells were grown on six well plates and infected at an MOI of 10 with the same four strains. On both coverslips and well plates, 20 mM arabinose was added to the two induced flhDC+ conditions to maintain expression.

For microscopy, coverslips were fixed, permeabilized and stained for released GFP as described in the Immunocytochemistry section. Nine images for each condition were acquired at 20× magnification and background subtracted. Protein (GFP) delivery was determined using pixel by pixel image analysis in MATLAB. A pixel was positive for delivery if it stained for GFP-myc. Total delivery was calculated as the sum of the intensities of all delivery positive pixels. Values were normalized by the PsseJ-LysE−, flhDC− condition.

For flow cytometry, cells were processed into a single cell suspension by gently pipetting after washing with PBS and adding 0.05% trypsin (ThermoFisher, catalog #25300-054). Cells were fixed with 5% formaldehyde in PBS w/ 1 mM EDTA and incubated in blocking buffer for 30 minutes. Cells were intracellularly stained with a 1:2000 dilution of FITC-conjugated anti-Salmonella antibody (Abcam, catalog #ab69253), and a 1:200 dilution of rat anti-myc monoclonal antibody (Chromotek, catalog #9e1-100) for 30 minutes. Cells were washed three times with blocking buffer. Cells were incubated with DyLight 750 anti-rat secondary antibody (ThermoFisher, catalog #SA5-10031) at a 1:200 dilution for one hour at room temperature. Samples were analyzed on a custom-built flow cytometer (dual LSRFortessa 5-laser, BD). All fluorophores were compensated with compensation beads (BD, catalog #552845) and did not carry more than 2% bleed over into any other channel. Control cells that were not infected by Salmonella were used as gating controls to identify uninfected cells in the samples, based on Salmonella staining. Cells administered non-lysing bacteria (i.e., PsseJ-LysE−) were stained with anti-Salmonella antibody, anti-rat secondary antibody, but not the anti-myc primary antibody to identify cells without GFP delivery.

Intracellular Delivery of GFP to Cells in Tumors with ID Salmonella

To identify and quantify GFP delivery to tumor cells, five BALB/c mice with 4T1 tumors were injected with 2×106 CFU of ID Salmonella (Table S1). Ninety-six hours after bacterial injection, mice were sacrificed and tumors, liver and spleens were excised. Tumors were cut in half. One half was fixed and stained for imaging and the other half was cryopreserved for protein quantification. Livers and spleens were also cryopreserved. Fixed tumors were embedded, sectioned and deparaffinized as described in the Immunohistochemistry section. Tumor sections were stained to identify GFP with a 1:50 dilution of goat anti-GFP (Abcam, ab6556) overnight, followed by incubation with a 1:50 dilution of Alexa Fluor 488-conjugated donkey anti-goat antibody (ThermoFisher, catalog #A21208) at room temperature for 1 h. After counterstaining with DAPI and mounting, sections were imaged at 20×.

To quantify the amount of delivered protein, half of the tumors as well as the livers and spleens were snap-frozen in liquid nitrogen and stored at −80° C. Lysates were made in a buffer containing 50 mM Tris-HCl at pH 7.4, 0.3% Triton-X 100, 0.1% NP-40 and 0.3 M NaCl. The buffer was supplemented with 25 mM NaF, 5 μM leupeptin, 0.5 mM phenylmethanesulfonyl fluoride, 0.5 mM benzamidine and 1 mM dithiothreitol. As with cancer cells in culture, this buffer lyses mammalian cells but not bacterial membranes, thereby separating delivered protein from protein in intact bacteria. Samples were homogenized on ice using a blender (Polytron) and a homogenizer (Potter-Elvehjem). Samples were incubated for 20 minutes on ice, centrifuged for 10 minutes at 664×g and 4° C. and the supernatant was collected. Immunoblotting was performed following 10% SDS-PAGE with anti-GPF (Abcam, catalog #ab6673) and anti-β-actin (GeneTex, catalog #GTX26276, clone AC-15). Immunoblots were visualized using eCL reagent (PerkinElmer) on a ImageQuant LAS4000 imaging system (GE Healthcare).

Effect of flhDC on Protein Delivery in Mice

To determine the effect of flhDC on protein delivery, nine BALB/c mice with 4T1 tumors were injected with 2×106 CFU of flhDC-ID Salmonella (Table S1) via the tail vein. Prior to injections, cultures of flhDC-ID Sal were grown in LB with 20 mM arabinose to induce flhDC expression. A second culture was grown without arabinose. At 48 and 72 hours after bacterial injection, 100 μg of arabinose in 400 μl of PBS was injected intraperitoneally into the flhDC+mice to maintain expression. The flhDC− mice received intraperitoneal injections of PBS at the same times. Ninety-six hours after bacterial injection, mice were sacrificed and tumors (n=4 for flhDC− and n=5 for flhDC+) were excised and sectioned as described in the Immunohistochemistry section. Tumor sections were stained to identify GFP with rat anti-GFP monoclonal antibody (Chromotek, catalog #3h9-100) and Alexaflor-568 goat anti-rat secondary antibodies (ThermoFisher, catalog #A11077). After counterstaining with DAPI, sections were imaged at 10× magnification. Images were background subtracted and were analyzed with computational code in MATLAB. Delivery was quantified at 20 random points in the transition zones of each tumor. A point was scored as positive if a cell within 20 μm contained delivered GFP. A cell was considered to have delivered protein if the GFP filled the entire cytoplasm. The delivery fraction is the number of positive points divided by the total number of random points.

**Temporal Colonization of ID Salmonella in Tumors**

To determine tumor density over time, 2×107 CFU ID Salmonella that express luciferase (ID Sal-luc, Table S1) were intravenously injected into five BALB/c mice with orthotopic 4T1 tumors in the mammary fat pad. Bacterial colonization was followed in real time by bioluminescent imaging. At 24, 48, 72, 168, 336 hours after bacterial injection, mice were injected i.p. with 100 μl of 30 mg/ml luciferin in sterile PBS, anesthetized with isoflurane, and imaged with an IVIS animal imager (PerkinElmer, SpectrumCT). Bacterial density in tumors was determined as the proton flux from the tumors. After acquiring the final image (at 14 days), tumors were excised and minced in equal volumes of sterile PBS. Homogenized tumors were cultured on agar plates. Colonies were counted after overnight growth at 37° C.

**Biodistribution and Toxicity of ID Salmonella**

To determine the biodistribution of Salmonella, five tumor-free BALB/c mice were injected with 1×107 ID Salmonella. After 14 days, six organs were excised and weighed: spleen, liver, lung, kidney, heart and brain. Organs were minced in equal volumes of sterile PBS, diluted 10 and 100 times, and cultured on agar plates. Colonies were counted after overnight growth at 37° C. To measure the toxicity of ID Salmonella, four tumor-free BALB/c mice were injected with 1×107 ID Salmonella. Four control mice were injected with sterile saline. After 14 days, whole blood was isolated from anesthetized mice by percutaneous cardiac puncture. Collected blood was divided between clot-activating serum tubes and EDTA anticoagulant tubes for chemistry and CBC analyses, respectively. Chemistry profiling and comprehensive hematology was conducted on the serum and whole blood samples by Idexx Laboratories (Grafton, MA).

Delivery of Nanobodies with ID Salmonella

To measure the delivery of nanobodies, ID Salmonella were administered to cancer cells and the extent of binding to the protein target was determined by immunoprecipitation. 4T1 cancer cells were grown to 80% confluency in T75 flasks and infected with either NB or ID Salmonella (as controls; Table S1) at an MOI of 10 as described in the Invasion assay section. The β-actin nanobody expressed by NB Salmonella is tagged with the FLAG sequence at the C terminus. Prior to administration, NB Salmonella were grown in LB with 20 mM arabinose to induce nanobody expression and 20 mM arabinose was added to the NB cultures to maintain expression. Twenty-four hours after invasion, the cancer cells were harvested using a cell lifter and centrifuged at 600×g for 10 minutes. The cell pellet was resuspended in 10 ml of lysis buffer (20 mM HEPES, 1 mM EDTA, 10% glycerol w/v, 300 mM sodium chloride and 0.1% Tween) that only lysed cancer cells but not intact bacteria. The cell suspension was homogenized in a douncer using a tight plunger. The cell lysate was clarified by centrifugation at 20,000×g for 20 minutes at 4° C. The lysate was incubated with 50 μl of anti-FLAG purification resin (Biolegend, catalog #651502) overnight at 4° C. The FLAG resin was washed three times with lysis buffer. Fifty microliters of Laemmli buffer was added directly to the bead solution and boiled for 5 minutes at 95° C. Boiled beads were loaded onto SDS-PAGE gels (15% polyacrylamide, cast in-house) in MOPS buffer for Western blotting as described in the Bacterial protein content section. Gels were transferred to nitrocellulose blotting paper. Blots were incubated with mouse anti-actin monoclonal antibody (Cell Signaling Technology, catalog #8H10D10) and HRP-conjugated goat anti-mouse secondary antibodies (ThermoFisher, catalog #31450) to identified β-actin.

**Cytotoxicity of Delivery of CT-Casp-3 and NIPP1-CD to Cells in Culture**

To measure the cytotoxicity of delivering protein drugs, ID Salmonella were administered to cancer cells in culture. Hepa 1-6 liver cancer cells were grown in six well plates to 80% confluency. NIPP1-CD, CT-Casp-3 Salmonella, and control ID Salmonella were administered at MOI of 10 as described in the Invasion assay section. Prior to invasion, cultures of CT-Casp-3 Salmonella were grown in LB with 20 mM arabinose for one hour to induce expression of CT-Casp-3. To all wells, 20 mM arabinose was added to maintain expression. Ethidium homodimer (500 ng/ml) was added to each well to stain dead cells with permeable membranes. Three mages were acquired per well (for nine images per condition) every 30 minutes for 24 hours at 20× magnification. At each time one transmitted and two fluorescent images were acquired: bacterial produced GFP (480/525 excitation/emission) and ethidium homodimer (525/590 excitation/emission). Images were background subtracted. From the fluorescent time-lapse images, cancer cells were identified that were invaded by Salmonella. Cell death was calculated as the fraction of dead Salmonella-invaded cells (co-localized with ethidium homodimer staining) over the total number of Salmonella-invaded cells.

**Delivery of CT-Casp-3 and NIPP1-CD to Tumor Masses**

To measure cell death in tumor masses after delivery of CT-Casp-3 or NIPP1-CD, ID Salmonella were administered to tumor-on-a-chip devices. Microfluidic devices were fabricated as described in the Effect of flhDC on invasion into tumor masses in vitro section. Two independent device experiments were run: (1) NIPP1-CD vs. ID control Salmonella with six chambers each; and (2) CT-Casp-3 vs. ID control Salmonella with four and three chambers, respectively. Prior to administration to the device, CT-Casp-3 Salmonella were grown in LB with 20 mM arabinose to induce expression of CT-Casp-3. NIPP1-CD and ID Salmonella were grown in LB without arabinose. All bacteria were centrifuged and resuspended in culture medium (DMEM with 20 mM HEPES) at a density of 2×107 CFU/ml. For CT-Casp-3 Salmonella, 20 mM arabinose was added to the medium. Bacteria-containing media, containing 500 ng/ml ethidium homodimer, was perfused through the tumor-on-a-chip devices for one hour at 3 μm/min for a total delivery of 2×106 CFU to each device. Bacterial administration was followed by bacteria-free media, with 20 mM HEPES and ethidium homodimer. Transmitted and fluorescence images were acquired every 30 minutes for 24 hours at 5× magnification. Death was calculated by first defined the borders of the tumor masses. Florescence images were segmented to identify regions of dead cells that stained with ethidium homodimer. The extent of death was the fraction of the tumor mass that was dead.

**The Final Fraction of Death was Determined at 24 h.**

Tumor response to delivery of CT-Casp-3 in mice Two mouse models were used to measure the effect of delivering CT-Casp-3: 4T1 murine breast cancer cells in BALB/c mice and Hepa 1-6 murine liver cancer cells in C57L/J mice. For both models, three conditions were tested by injecting saline, ID Salmonella, or CT-Casp-3 Salmonella. The saline controls establish the baseline growth rate of the tumors. The ID Salmonella (bacterial) control established the effect of colonized bacteria and intracellular lysis on the tumor growth rate. For both mouse models, three groups of six mice were subcutaneously injected with 1×105 tumor cells. Once tumors were between 50 and 75 mm3, they were injected with one of the three conditions: saline or 4×107 CFU of ID or CT-Casp-3 Salmonella. At 48 and 72 hours after injection, mice were injected i.p. with 100 mg of arabinose in 400 μl of PBS. Every five days, tumors were injected with bacteria or saline. Tumors were measured twice a week and volumes were calculated with the formula (length)*(width2)/2. Mice were sacrificed when tumors reached 1000 mm3. Tumor growth rates were determined by fitting exponential functions to tumor volumes as functions of time.

**Statistics**

For pair-wise comparisons, Student's t test was used. Statistical significance was confirmed when P<0.05. ANOVA with a Bonferroni correction was used when comparing multiple data points.


- 1. Swofford, C. A., N. Van Dessel, and N. S. Forbes, *Quorum*-*sensing
  Salmonella selectively trigger protein expression within tumors*.
  Proceedings of the National Academy of Sciences of the United States
  of America, 2015. 112(11): p. 3457-62.
- 2. Mosberg, J. A., M. J. Lajoie, and G. M. Church, *Lambda red
  recombineering in Eschenchia coli occurs through a fully
  single*-*stranded intermediate*. Genetics, 2010. 186(3): p. 791-9.
- 3. Vaidya, S., et al., *Substrate*-*induced conformational changes
  occur in all cleaved forms of caspase*-6. J Mol Biol, 2011. 406(1): p.
  75-91.
- 4. Yan, Y., et al., *Asd*-*based balanced*-*lethal system in
  attenuated Edwardsiella tarda to express a heterologous antigen for a
  multivalent bacterial vaccine*. Fish Shellfish Immunol, 2013.
  34(5): p. 1188-94.
- 5. Nakayama, K., S. M. Kelly, and R. Curtiss, *Construction of an ASD+
  expression*-*cloning vector—stable maintenance* and *high*-*level
  expression of cloned genes in a Salmonella vaccine* strain.
  Bio-Technology, 1988. 6(6): p. 693-697.
- 6. Toley, B. J. and N. S. Forbes, *Motility is critical for effective
  distribution and accumulation of bacteria in tumor tissue*. Integr
  Biol, 2012. 4(2): p. 165-76.
- 7. Walsh, C. L., et al., *A multipurpose microfluidic device designed
  to mimic microenvironment gradients and develop targeted cancer
  therapeutics*. Lab on a chip, 2009. 9: p. 545-54.

**Results and Discussion**

Herein, the creation of an intracellular protein delivery system based on the natural qualities of Salmonella is described (FIG. 1A). In the intestines, Salmonella have a partially intracellular lifestyle. To evade clearance, Salmonella invade epithelial cells using the proteins expressed by Salmonella pathogenicity island 1 (SPI1) (3,4). After invasion, Salmonella reside in early and late endosomes, which they reform into Salmonella containing vacuoles (SCVs) by expressing the genes of pathogenicity island 2 (SPI2) (5-7). SCVs enable intracellular survival (5,8) and protect the Salmonella from intracellular defense mechanisms (9,10). A step in the activation of SPI2 genes, is the sensing of the endosomal environment. These sensing mechanisms, which are unique to Salmonella, are needed for delivery of proteins into cells.

While it is well established that Salmonella invade intestinal cells (4,11), their location within tumors is more uncertain, despite extensive documentation of tumor colonization (12-16). Preferential accumulation and exponential growth in tumors are essential properties of therapeutic Salmonella (17,18). When administered in culture, Salmonella readily invade into carcinoma cells (FIG. 1B). To determine where they reside in tumors, Salmonella with a fluorescent intracellular reporter were injected into tumor-bearing BALB/c mice. In these tumors, over 70% of Salmonella were intracellular (P<0.001, n=5, FIG. 1C), demonstrating their suitability as delivery vehicles. In cells dissociated from tumors with collagenase, bacteria were present in carcinoma, immune, and other tumor associated cells (FIG. 1D).

The development of therapeutic Salmonella into an intracellular protein delivery system had three steps (FIG. 1A). The design goals were to engineer Salmonella to (1) Make a drug, (2) Invade into cells, and (3) Release the drug into cells. The use of bacteria changes what is traditionally meant by “delivery.” Unlike typical delivery vehicles, bacteria manufacture protein drugs at the disease site (19), delivering exponentially more molecules than were originally present in the injected bacteria. Chronologically, the first steps were to generate a platform strain with controlled invasion and release. The last step was to transform this platform strain with genes to synthesize different protein drugs. In the final engineered strains of Intracellular Delivering (ID) Salmonella, each of these three processes (make, invade and release) was controlled by a specialized genetic circuit.

In this system, invasion of ID Salmonella into cells is controlled with the regulation factor flhDC(FIG. 1E-G). Expression of flhDC is required for Salmonella to invade cancer cells (FIG. 1E). When flhDC is not expressed, Salmonella invaded less than 2% of cells, which was 54 times less than by Salmonella with re-expressed flhDC (84%; P<0.001; FIG. 1E). Invasion is dependent on flhDC because it regulates the production of flagella and the type III secretion system (20). In microfluidic tumor masses in vitro (21), re-expression of flhDC increased cell invasion and colonization 53 times (P<0.01, FIG. 1F). In tumors, re-expression of flhDC increased invasion into both carcinoma and immune cells (P<0.05, FIG. 1G).

The second component of ID Salmonella, release, required development of a system to trigger autonomous lysis after cell invasion (FIG. 2). This goal was achieved by identifying a Salmonella promoter that is triggered intracellularly and not extracellularly. After invasion into cells, the genes of SPI2 activate to form Salmonella containing vacuoles (SCVs) (8). When coupled to a GFP reporter, the promoters of two SPI2-associated genes, PsseJ and PsifA, both activate after invasion into cancer cells (FIG. 2A, left). However, the extracellular expression of PsseJ is 5.8 times less than PsifA (P<0.001, FIG. 2A), indicating that it is more sensitive to cell invasion.

To release a synthesized protein cargo, the bacteria must lyse after invasion. Triggered expression of Lysin gene E (LysE) from bacteriophage DX1174 causes rapid bacterial death (FIG. 2B). Salmonella, with the coupled PsseJ-LysE construct and that constitutively expresses GFP (as a model protein drug), lysed after invasion into cancer cells (FIG. 2C), and discharged GFP into the cytoplasm (FIG. 2D). Bacterial lysis occurs for 10 hours after invasion (FIG. 2E). The basal expression of Lysin E by the PsseJ-LysE circuit does not affect bacterial health and intracellular induction activated the system at near to its maximum rate (FIG. 2F). Each bacterium can deliver, on average, 163,000 GFP molecules (FIG. 2G).

After bacterial lysis, delivered protein escapes SCVs and fills the cellular cytoplasm (FIG. 2H-I). This escape is important because, immediately after invasion, most Salmonella reside within SCVs (FIG. 2H, left). When ID Salmonella lyse, clusters of released GFP protein are contained within SCVs (FIG. 2H, middle). Over time, the protein escapes the SCVs and fills the entire cytoplasm (FIG. 2I), a transition that occurs for most cells (P<0.001, FIG. 2H, right). GFP diffuses through the cytoplasm with an effective diffusivity of 0.15 μm2/min (FIG. 2J).

As designed, bacterial lysis is dependent on residence within SCVs (FIG. 3A-B). After invasion, some ID Salmonella escape into the cytoplasm and are not surrounded by a SCV membrane (FIG. 3A, left). More than 95% of GFP released from Salmonella originated inside SCVs (P<0.001; FIG. 3A, right). After invasion into cancer cells, ID Salmonella with a ΔsifA deletion, which are predominantly cytoplasmic (23), did not lyse despite containing the PsseJ-LysE construct. Comparatively, ID Salmonella with a ΔsseJ, which are predominantly vacuolar (24), almost all lysed (P<0.001, FIG. 3B). Without these deletions, most ID Salmonella localized to SCVs, lysed and delivered protein (P<0.001, FIG. 3B). This dependence indicates that the Pssej promoter only activates after SCV localization and not when in the cytoplasm. This specific sensing of the SCV environment is a feature exclusive to Salmonella.

Protein delivery was dependent on the two engineered systems, PBAD-flhDC for invasion and PsseJ-LysE for release (FIG. 3C). Salmonella without flhDC expression did not invade cells, and Salmonella without Pssei-LysE did not release the GFP cargo (FIGS. 3C&S2). Compared to controls, the presence of both systems increased protein delivery 548 times (P<0.001; FIG. 3C).

When administered systemically to tumor-bearing mice, ID Salmonella specifically deliver protein to tumor cells, and this delivery is dependent on flhDC (FIG. 3D-F). ID Salmonella invaded cells and delivered GFP that filled the cellular cytoplasm (FIG. 3D). This system delivered 60 t 12 μg GFP/g tumor (FIG. 3E), which is equivalent to 1.5×108 bacteria per gram of tumor. No GFP was detected in the livers or spleens of any mice (FIG. 3E). When tumor-bearing mice were administered ID Salmonella that did not express flhDC, little GFP was delivered (FIG. 3F). Re-expressing flhDC increased the percentage of cells that received GFP more than five times (P<0.001).

Delivery of proteins with ID Salmonella is safe and self-limiting (FIG. 3G). After intravenous administration, the tumor density of ID Salmonella reached a peak at 72 h and then dropped 97% in 11 days (FIG. 3E). The decline in density, which was caused by intracellular lysis, limits the exposure to therapy and increases safety compared to non-lysing Salmonella. After administration to healthy, tumor-free mice, ID Salmonella did not accumulate in lungs, hearts, kidneys or brains; had no effect on liver function; and caused no adverse immune responses.

To demonstrate its broad capabilities, ID Salmonella was engineered to make three different proteins (FIG. 4) that affect intracellular physiology: a nanobody (anti-actin), a protein inhibitor (NIPP1-CD), and an endogenous protein (CT casp-3). The central domain of nuclear inhibitor of protein phosphatase 1 (NIPP1-CD) removes PP1 from its holoenzymes and induces cell death (25). Constitutive two-chain active caspase-3 (CT Casp-3) is an engineered active form of caspase-3, the dominant executioner caspase that leads to apoptotic cell death (26, 27).

In one aspect, a bicistronic mRNA codes for caspase, with, for example, the large subunit followed by a ribosomal binding site and the small subunit on, for example, PBAD inducible promoter.

After bacterial delivery via invasion and lysis, the anti-actin nanobody was bound to cellular actin (FIG. 4A), demonstrating specific targeting of an intracellular protein. As potential therapeutic proteins, delivery of both NIPP1-CD and CT Casp-3 caused more cell death than controls (P<0.001; FIG. 4B, left). Induced death was dependent on invasion and protein delivery (FIG. 4B, right). When administered to microfluidic tumors devices, ID Salmonella delivering NIPP1-CD (P<0.05) and CT Casp-3 (P<0.01) caused cell death that increased with time as bacteria invaded the tumor masses (FIG. 4C).

Delivery of CT Casp-3 was effective against both liver cancer and triple-negative breast cancer in mice (FIG. 4D-E). After 14 days of treatment, delivery to BALB/c mice reduced the volume of 4T1 mammary tumors two times more than controls (P<0.05, FIG. 4D). Administration of ID Salmonella with CT Casp-3 significantly reduced the volume of liver Hepa 1-6 tumors in C57L/J mice (P<0.001; FIG. 4E, left) and reduced tumor growth rate 28 times (P<0.05; FIG. 4E, middle), which is equivalent to an increase in doubling time from 5 to 148 days. Tumor volume reduced in two mice for over 50 days, and survival increased significantly compared to bacterial controls (P<0.05, FIG. 4E right). Treatment with CT Casp-3 completely eliminated the tumor from one mouse, which was disease free for over 124 days.

**Conclusion**

Described herein is an autonomous, intracellular Salmonella vehicle that efficiently delivers properly folded and active proteins into cells. This bacterial strain is safe, eliminates tumors and increases survival. The engineered gene circuits produce protein drugs, cause hyper-invasion (flhDC) and trigger bacterial lysis after cell invasion. Because the system is autonomous, it does not require intervention and is self-timing. Protein delivery is triggered at the most opportune time for individual bacteria, ensuring that proteins are deposited inside cells and not in the extracellular environment. The accumulation of ID Salmonella in different cell types in tumors (FIGS. 1D&G), suggests that this system could be used to deliver proteins to non-cancerous tumor-associated cells, e.g., macrophages or endothelial cells.

Coupled together, two essential qualities of ID Salmonella enable the use of protein drugs that are currently not feasible. Intracellular Salmonella delivery (1) transports intact, functional proteins across the cell membrane; and preferential tumor accumulation (2) maintains safety for protein drugs that would act broadly against healthy cells. Both NIPP1-CD and CT Casp-3 have exclusively intracellular targets and would be toxic if delivered systemically. The specific accumulation of ID Salmonella eliminates these problems by focusing therapy specifically on the intracellular environment of tumors (FIGS. 1C and 3E).

The use of ID Salmonella to deliver CT Casp-3 can address the need for an effective treatment for unresectable hepatocellular carcinoma (HCC). No curative treatment currently exists for the 840,000 patients who are diagnosed with HCC annually (28, 29). Current therapies have toxic side effects and only modestly increase survival (29-31). Treatment with CT Casp-3 ID Salmonella can be curative (FIG. 4E) and is safer. Inclusion of the PsseJ-LysE circuit makes ID Salmonella self-limiting. The delivery bacteria lyse after cell invasion (FIG. 3F), reducing the possibility of unwanted infections.

Delivery with ID Salmonella enables targeting of inaccessible cancer pathways and will accelerate the generation of new cancer therapies. These therapies can be created by coding the genes for specific protein drugs into Salmonella expression cassettes. Nanobodies (FIG. 4A) can be designed that specifically inhibit pathways necessary for cancer survival and progression. Using bacteria to deliver proteins into cells will expand the number of accessible pathways, open up many targets across the soluble proteome for treatment, and increase the efficacy and safety of cancer treatment.

### Example II

**Introduction**

Intracellularly targeted, macromolecular therapies present an opportunity for treatment of cancer. The mammalian proteome consists of 60% intracellular protein while only 30% are surface associated and extracellularly exposed (1). However, macromolecules face tumor specificity, distribution, cell internalization and endosomal release barriers (2). An improved drug delivery system is needed to circumvent these delivery limitations and increase therapeutic efficacy of intracellularly active therapies. Salmonella are ideally suited for tumor selective intracellular protein delivery. Salmonella colonize tumors with high specificity, invade, and deliver protein therapies selectively inside tumor cells. Herein the discovery that flhDC expression is crucial for protein delivery into tumor cells with Salmonella has been reported. To this end, it was sought to determine the mechanisms by which flhDC expression enables intracellular therapeutic delivery in vivo. The unique mechanisms by which engineered Salmonella expressing flhDC developed resistance to intracellular therapeutic delivery was also assessed. Understanding these mechanisms help create improved tumor targeted, intracellular delivery strains of Salmonella.

Typhoidal strains of Salmonella that systemically infect humans carefully modulate flagellar expression in vivo. The typhoidal bacteria that disseminate systemically infect humans implement genetic programs to downregulate expression of the flagellar synthesis regulator (3-5), flhDC, in the blood (6, 7). One reason for this is because flagellin is a TLR5/NLRC4 agonist that strongly activates an anti-microbial immune response (8, 9). However, in tumor tissue, intracellular invasion and delivery into cancer cells requires activation of the Salmonella transcription factor, flhDC (10). Therefore, developing a method to control flhDC activity in engineered Salmonella is necessary to enable high levels of therapeutic delivery in tumors.

Modulation of flhDC activity within Salmonella has significant implications in determining tumor selectivity and reducing systemic virulence. Unlike tumors, clearance organs like the liver and spleen, have high concentrations of functional immune cells that mount strong responses upon pathogenic insult. The liver is a vital clearance organ and an essential site specific for immune mediated Salmonella clearance (11). The motility regulator, flhDC, regulates flagellar expression but is also a broad regulator of Salmonella lifestyle and virulence (10, 12, 13). Flagellar expression within Salmonella in macrophages or epithelial cells causes excessive, NLRC4 inflammasome dependent, pyroptosis. Salmonella hijack this inflammatory pathway to overexaggerate the anti-microbial response both in macrophages and within the gut, causing immune dysfunction (14, 15). Since the liver contains large quantities of Kupfer cells, flagellated Salmonella can cause significant pyroptosis in these liver specific macrophages. While pyroptosis is required in limited quantities to eliminate pathogens, flagellated Salmonella cause high levels of pyroptosis that render the anti-microbial immune response dysfunctional (14, 16). Macrophages are more effective at clearing Salmonella expressing lower levels of flhDC due to reduced flagellar expression, limited pyroptosis resulting in less immune dysfunction (16). Since tumors do not share the same level of immune function, low flagellin expression does not affect tumor colonization (17).

Upon invasion into a cell, there are two existing mechanisms by which therapy is delivered into the cytosol by Salmonella: (1) The bacteria invade, escape the intracellular vacuole, rupture, and deliver therapy into the cytosol (18-20) or (2) the bacteria are genetically engineered to lyse and deliver therapy from within the Salmonella containing vacuole into the cytosol. Several variants of cytosolic bacteria (ΔsifA Salmonella, Listeriolysin O expressing bacteria) have been used for therapeutic delivery into tumor cells (18-20). In scenario (1), therapeutic delivery would require the bacteria to reside in the cytoplasm of a cancer cell and lyse spontaneously without any control. This mechanism would depend on ubiquitin dependent degradation (21) of the bacteria and subsequent cytosolic release of the therapy. In addition, cytoplasmic pathogens are known to strongly activate NF-kB signaling and initiate innate immune responses to clear the bacteria (9, 21, 22). Therefore, a high presence of cytosolic Salmonella is detrimental for immune evasion.

Salmonella have evolved to reside within an intracellular vacuole which confers protection to the bacteria inside cells (23, 24). The bacteria modify the vacuole to confer protection against degradation and clearance (25, 26). In addition, vacuolar residence seems to be especially important for bacteria in systemic circulation as demonstrated by Salmonella Typhi. The spi-2 protein, SseJ, is required for Salmonella to escape the SCV (27). Salmonella Typhimurium, which express SseJ, are localized to the gastrointestinal tract in humans (28). Salmonella Typhi, which lacks SseJ (29), is efficient at escaping the gastrointestinal tract into systemic circulation (30). Moreover, Salmonella Typhi only expresses typhoid toxin intracellularly within the SCV (31, 32). These critical between Salmonella Typhi and Salmonella Typhimurium suggest that vacuolar residence is imperative to increase bacterial fitness in vivo.

Understanding the dynamics between vacuolar and cytosolic Salmonella expressing flhDC will aid in engineering an intracellular delivery strain of Salmonella. Herein we have shown that intracellular lysis of engineered Salmonella occurs in a vacuole. However, flagellated, intracellular Salmonella have a significant cytosolic presence (12). Intracellular Invasion is driven by flhDC and T3SS1 activity (10, 12, 33-35). Upon invading cells, Salmonella heavily modify the vacuoles in which they reside (24, 36). In doing so, some bacteria rupture the vacuole and escape (37, 38). Normally, the intravacuolar bacteria also downregulate flagellar expression through ssrB directed suppression of flhDC (39) (the ssrB protein is considered a master regulator of SPI-2 expression (33)). However, flagellated, cytosolic Salmonella have abrogated T3SS2 activity due to vacuolar escape (12). As shown herein, T3SS2 activity is needed to enable intracellular lysis and delivery of protein with therapeutic Salmonella.

Provided herein is a showing of how controlled expression of flhDC could improve tumor colonization and therapeutic delivery in vivo as compared to existing delivery strategies. Further the mechanism is eluicidated of flhDC induced resistance to therapeutic delivery and a genetic engineering strategy to rescue therapeutic delivery of the Salmonella strain. It was hypothesized that flhDC expression selectively within intratumoral bacteria is important for increasing tumor specificity, colonization and protein delivery to a spatially distributed set of tumor cells. It was further hypothesized that engineered Salmonella inducibly expressing flhDC could deliver more protein intracellularly compared to exclusively cytosolic Salmonella. It was also hypothesized that flhDC activity enabled lysis resistance in engineered Salmonella but could be rescued. To test these hypotheses, cell-based assays, tumor-on-a-chip models, and in vivo experiments were employed to quantitatively understand the mechanisms underlying intracellular therapeutic delivery with engineered Salmonella. Discovering the key mechanisms governing therapeutic delivery with Salmonella would address limitations with current delivery methods and provide a foundation to robustly improve delivery efficiency of the engineered bacteria for a wide variety of cancers.

**Bacterial Cultures**

All bacterial cultures (both Salmonella and DH5α) were grown in LB (10 g/L sodium chloride, 10 g/L tryptone and 5 g/L yeast extract). Resistant strains of bacteria were grown in the presence of carbenicllin (100 μg/ml), chloramphenicol (33 μg/ml), kanamycin (50 μg/ml) and/or 100 μg/ml of DAP.

**Cloning**

One of three plasmids were used in all experiments. The first plasmid, P1, was created by cloning the flhDC gene into the PBAD his-myc plasmid (Invitrogen; catalog #V430-01). Primers vr46 and vr47 were used to PCR the flhDC gene from VNP20009 genomic DNA. The PCR product was digested with NcoI and XhoI and DpnI (NEB, catalog #s R0193S, R0146S and R0176L). The PBAD-his-myc backbone was digested with NcoI, XhoI and calf intestinal phosphatase (NEB, catalog #M0290). A PCR cleanup column (Zymo Research) was used to clean up both products. 50 ng of digested vector backbone and 500 ng of digested PCR product were ligated together using T4 DNA ligase (NEB). The ligated product was transformed into DH5a E. Coli. Positive transformants were confirmed by sequencing (Plasmid P1a). To add the plac-GFP-myc genetic circuit to the plasmid, plasmid P1a was PCR amplified using primers vr385 and vr386. The plac-GFP-myc genetic circuit was PCR amplified from a previously generated plasmid (40) using primers vr394 and vr395. Both PCR products were DpnI digested. 50 ng of P1a PCR product and 500 ng of were ligated together using a 2×Hifi DNA assembly master mix (NEB). The resulting product was transformed into DH5a E. Coli and the complete P1b plasmid was purified from positive colonies. To create complete plasmid P1, PCR was used to amplify the P1b backbone using primers vr426 and vr427. The ASD gene was amplified from a previously generated plasmid, PCS2 (40) using primers vr424 and vr425. 50 ng of the P1b PCR product and 500 ng of the ASD PCR product were ligated together using 2×Hifi DNA assembly master mix. The resulting ligation was transformed into chemically competent DH5a E. Coli. Complete, P1 plasmid was purified from colonies screening positive for GFP, ASD and PBAD-flhDC.

To create plasmid P2, plasmid P1 was PCR amplified using primers vr396 and vr397. The psseJ-lysinE genetic circuit was amplified from synthesized DNA (Genscript) using primers vr398 and vr399. The two PCR products were DpnI digested and purified using PCR clean up columns (Zymo Research). 50 ng of backbone PCR and 500 ng of psseJ-lysinE PCR product was used in a ligation reaction with 2×Hifi assembly master mix (NEB) to create plasmid, P2a. Plasmid was purified from colonies that screened positive for plasmid assembly for downstream applications. To create complete P2, plasmid P2a was PCR amplified using primers vr426 and vr427. The ASD gene was amplified as previously described using primers vr424 and vr425. Both PCR products were DpnI digested and purified using a PCR clean up column as previously described. 50 ng of the P2a PCR product was ligated together with 500 ng of the ASD PCR product using 2×Hifi DNA assembly master mix. The resulting ligation was transformed into DH5a E. Coli and complete P2 plasmid was purified from colonies screening positive for GFP, ASD, PBAD-flhDC and sseJ-lysinE.

To create plasmid P3 (sseJ-GFP-myc+PBAD-flhDC), plasmid P1a was PCR amplified using primers vr271 and vr272. The sseJ-GFP-myc genetic circuit was PCR amplified from a previously generated plasmid (10) using primers vr269 and vr270. The resulting PCR products were DpnI digested and purified using PCR clean up columns. 50 ng of the P1a backbone and 500 ng of the psseJ-GFP-myc PCR products were ligated together using 2×Hifi DNA assembly master mix. The resulting ligations were transformed into DH5a E. Coli. Complete, P3 plasmid was purified from colonies that screened positive for psseJ-GFP-myc and PBAD-flhDC for downstream application.

**Strains**

All engineered strains were based on VNP20009 and strain details can be found in following table.

Genetic knockouts were created using a modified lambda red recombination procedure (41, 42). The master gene editing strain was created by transforming the plasmid containing the required lambda phage genes, pkd46, into VNP20009 using electroporation.

Six genomic knockout strains of Salmonella were created. Three of the knockouts were created by growing Salmonella containing pkd46 to an optical density of 0.1 at which point the bacteria were supplemented with 20 mM arabinose to induce expression of lambda genes. When the bacteria reached an optical density of 0.8, 1 microgram of DpnI digested PCR product amplified from pkd4 (vr121/vr309 for ΔflhD,vr318/vr319 for ΔfliGHI, vr432/vr433 for ΔsseJ, vr434/vr435 for ΔsifA) was transformed into the Salmonella through electroporation. Bacteria was recovered in LB for 2 hours at 37° C. and plated on agar plates containing 50 micrograms/ml of kanamycin. Resulting transformants were screened for insertion using antibiotic selection and junction PCR to confirm correct location of genomic deletion. Successful knockouts were then grown at 43° C. to cure the knockout strains of the pkd46 plasmid.

To create the ΔflhD+ΔfliGHI knockout, the above strain of ΔflhD was retransformed with pkd46 through electroporation, grown to an OD of 0.1 and induced with 20 mM arabinose until the bacteria grew to an OD of 0.8. The fliGHI knockout PCR product was amplified from pkd3 using the primers, vr266 and vr268. The PCR products were DpnI digested and 1 microgram was transformed into the lambda induced ΔflhD strain using electroporation. The bacteria were recovered in LB with 100 micrograms/ml for 2 hours at 37° C. and plated on agar plates containing 33 micrograms/ml of chloramphenicol. Successful transformants were screened as previously described and grown on LB containing 33 micrograms/ml of chloramphenicol overnight at 43° C. to cure the bacteria of pkd46.

The plasmids created were transformed into the relevant strains using electroporation. These strains are listed in Table 3.

**Mouse Models**

Six week old Balb/C mice from Jackson Laboratories were injected subcutaneously with 1×10W 4T1 tumor cells on the hind flank. Once tumors reached 500 mm3, mice were intravenously injected with either saline or bacteria. Either twenty-four or ninety-six hours after bacterial administration, mice were sacrificed, and tumors, livers and spleens were excised for downstream analysis.

**In Vivo Tumor and Liver Colonization of Salmonella**

To quantify tumor and liver colonization five groups of five Balb/C mice containing subcutaneous 4T1 tumors (˜500 mm3) were intravenously injected via the tail vein with either parental, ΔflhD, ΔfliGHI, or ΔflhD+ΔfliGHI Salmonella. Ninety-six hours after bacterial administration, tumors and livers were excised and homogenized in two volumes (w/v) of sterile PBS. Organ slurries were serially diluted 10-fold, four times for livers and eight time for tumors. 200 ul of each dilution was plated on agar containing the appropriate antibiotic. After drying, plates were incubated overnight at 37 degrees Celsius. Plates containing between 10 and 100 colonies were counted to determine bacterial colonization levels in either the tumor or liver.

**Immunohistochemistry**

Excised tumor sections were fixed in 10% formalin for 3 days. Fixed tumor samples were then stored in 70% ethanol for 1 week. Tumor samples were embedded in paraffin and sectioned into 5 μm sections. Deparaffinization was performed by washing the sectioned tissue three times in 100% xylene, twice in 100% ethanol, once in 95% ethanol, once in 70% ethanol, once in 50% ethanol, and once in DI water. Each wash step was performed for 5 minutes. Antigen retrieval was performed by incubating the tissue sections in 95° C., 20 mM sodium citrate (pH 7.6) buffer for 20 minutes. Samples were left in sodium citrate buffer until the temperature reduced to 40° C. Samples were then rehydrated with two quick (<1 minute) rinses in DI water followed by one five-minute wash in TBS-T.

Prior to staining, tissue sections were blocked with Dako blocking buffer (Dako) for one hour. Tissue sections were stained to identify Salmonella and GFP with 1:100 dilutions of (1) FITC-conjugated rabbit anti-Salmonella polyclonal antibody (Abcam), and (2) either rat anti-myc monoclonal antibody (Chromotek) or rat anti-GFP monoclonal antibody (Chromotek) in Tris buffered saline with 0.1% Tween 20 (TBS-T) with 2% BSA (FisherScientific). Sections were washed three times in TBS-T w/ 2% BSA and incubated with Alexaflor-568 goat anti-rat secondary antibodies (ThermoFisher). After washing sections three times with TBS-T, 40 μl of prolong gold mountant with DAPI (ThermoFisher) and a cover slip were added to each slide. Slides were incubated at room temperature for 24 hours until the mountant solidified. Histological Detection of Intracellular delivery of GFP to cells in tumors with FID Salmonella

To identify and quantify GFP delivery to tumor cells, two groups of ten BALB/c mice with 4T1 tumors were injected with 2×106 CFU of FID Salmonella. One group of mice was injected twice with arabinose intraperitoneally to induce flhDC expression while the other group was injected with saline as a control. Ninety-six hours after bacterial injection, mice were sacrificed and tumors, liver and spleens were excised. Tumors were cut in half. One half was fixed and stained for imaging as described in the immunohistochemistry section.

**Cell Culture**

Two cancer cell lines were used: 4T1 murine breast carcinoma cells and LS174T human colorectal carcinoma cells (ATCC, Manassas, VA). All cancer cells were grown and maintained in Dulbecco's Minimal Eagle Medium (DMEM) containing 3.7 g/L sodium bicarbonate and 10% fetal bovine serum. For microscopy studies, cells were incubated in DMEM with 20 mM HEPES buffering agent and 10% FBS. To generate tumor spheroids, single cell suspensions of LS174T cells were transferred to PMMA-coated cell culture flasks (2 g/L PMMA in 100% ethanol, dried before use).

Microfluidic System to Quantify Intracellular Invasion Distribution of flhDC Induced Salmonella

To quantify invasion into tumor masses, engineered Salmonella were administered to tumor-on-a-chip devices developed in our laboratory (43, 44). Microfluidic tumor-on-a-chip devices were fabricated using negative tone photoresist and PDMS based soft lithography. Master chips were constructed by spin coating a layer of SU-8 2050 onto a silicon wafer at 1250 RPM for 1 minute. This speed corresponded to an SU-8 2050 thickness of 150 μm. The silicon wafer was baked at 65° C. for 5 minutes followed by 95° C. for 30 minutes. Microfluidic designs printed on a high-resolution transparency were placed over the silicon wafer in a mask aligner. The silicon wafer with the overlaid mask was exposed to UV light (22 J/cm2) for 22 seconds. Silicon wafers were baked for 5 minutes at 65° C. followed by 95° C. for 12 minutes. Wafers were then developed in PGMEA developing solution for 10 minutes and/or until microfluidic features were microscopically distinct with sharp and defined edges.

Soft lithography was used to create the multilayer tumor on a chip device with 12 tumor chambers (two conditions with six chambers each). PDMS (Sylgard 184) at ratios of 9:1 and 15:1 were used for the channel and valve layers, respectively. The channel layer was placed on a spin coater for 1 minute at 220 rpm in order to achieve a PDMS thickness of 200 μm. The silicon wafers were degassed for 45 minutes to eliminate air bubbles in the PDMS. The silicon wafers were baked at 65 degrees for approximately one hour or until both PDMS layers were partially cured. The top valve layer of PDMS was cut and removed from the silicon wafer and aligned on top of the channel layer using a stereomicroscope. The combined layers were baked for one hour at 95° C. in order to covalently bind the two layers. The multilayered PDMS device and a glass slide was plasma treated in a plasma cleaner (Harrick) for 2.5 minutes. Valves were pneumatically actuated with a vacuum pump and the PDMS was placed on the plasma treated glass slide. Valves were actuated until the device was ready for use.

The tumor-on-a-chip was sterilized with 10% bleach followed by 70% ethanol, each for one hour. Microfluidic chips were equilibrated with media (DMEM with 20 mM HEPES, pH 7.4) for one hour. Valve actuation was used to position tumor spheroids in the tumor chambers. Valves at the rear of the chambers were opened while the efflux channel was closed. After the tumor masses were positioned, the valves were reset so that the rear valves were closed, and the influx and efflux channels were open.

Prior to administration to the device, flhDC reporting Salmonella were grown in LB with 20 mM arabinose to induce flhDC expression. These Salmonella have inducible flhDC (PBAD-flhDC) and produce GFP when intracellular (PsseJ-GFP). Control (flhDC−) Salmonella of the same strain were grown without arabinose. The bacteria were centrifuged and resuspended in culture medium (DMEM with 20 mM HEPES) at a density of 2×107 CFU/ml. For the induced flhDC+condition, 20 mM arabinose was added to the medium. Bacteria-containing media (flhDC+ and flhDC−; n=6 chambers each) were perfused through the tumor-on-a-chip devices for one hour at 3 μm/min for a total delivery of 2×106 CFU to each device. Bacterial administration was followed by bacteria-free media (with 20 mM HEPES) for 48 hours.

Devices were imaged at 30-minute intervals. Invasion was quantified at 31 h by measuring GFP expression by invaded bacteria in the tumor masses. Regions of interest were defined around the borders of the tumor masses. The extent of invasion was determined as the average GFP fluorescence intensity in each tumor mass. Intensities were normalized by the intensity of the average tumor mass administered control (flhDC−) Salmonella.

**Microscopy and Image Analysis**

Samples were imaged on a Zeiss Axio Observer Z.1 microscope. Fixed cells on coverslips were imaged with a 100× oil immersion objective (1.4 NA). Tumor sections were images with 10× and 20× objectives (0.3 and 0.4 NA, respectively). Time lapse fluorescence microscopy of live cells in well plates and tumor-chip devices were housed in a humidified, 37° C. environment and imaged with 5×, 10×, 63× or 100× objectives (0.2, 0.3, 1.4 and 1.4 NA, respectively). Fluorescence images were acquired with either 480/525 or 525/590 excitation/emission filters. All images were background subtracted and contrast was uniformly enhanced. All immunocytochemistry image analysis was automated using computational code (MATLAB, Mathworks). Immunohistochemical imaging of bacterial distribution in tumors was automated using MATLAB. Intracellular protein delivery within mouse tumors was visually quantified.

**Infection Assays**

For infection assays, cancer cells were grown on coverslips for fixed-cell imaging. For fixed imaging, glass coverslips were placed in 12-well plates and sterilized with UV light in a biosafety hood for 20 minutes. Mouse 4T1 were seeded on the coverslips at 40% confluency and incubated overnight in DMEM. Concurrently, Salmonella were grown to an optical density (OD; at 600 nm) of 0.8. After incubation, the Salmonella were added to the 4T1 cultures at a multiplicity of infection (MOI) of 10 and allowed to infect the cells for two hours. After this invasion period, the cultures were washed five times with 1 ml of phosphate buffered saline (PBS) and resuspended in 2 ml of DMEM with 20 mM HEPES, 10% FBS and 50 μg/ml gentamycin. The added gentamycin removes extracellular bacteria. After six hours of incubation, the media was removed, and the coverslips were fixed with 10% formalin in PBS for 10 minutes.

**Immunocytochemistry**

Immunocytochemistry was used to obtain detailed images of Salmonella invaded into cancer cells grown on coverslips. After fixing the coverslips with formalin, they were blocked with staining buffer (PBS with 0.1% Tween 20, 1 mM EDTA, and 2% bovine serum albumin (BSA)) for 30 minutes. The Tween 20 in this buffer selectively permeabilizes mammalian cell membranes, while leaving bacterial membranes intact.

After permeabilization, coverslips were stained to identify Salmonella, released GFP, and vacuolar membranes with (1) rabbit anti-Salmonella polyclonal antibody (Abcam) or FITC-conjugated rabbit anti-Salmonella polyclonal antibody (Abcam) (2) rat anti-myc monoclonal antibody (Chromotek), and (3) rabbit anti-LAMP1 polyclonal antibody (Abcam), respectively. Three different staining combinations were used: (1) Salmonella alone; (2) Salmonella, released GFP and (3) Salmonella, released GFP and vacuoles.

For Salmonella alone staining (combination 1), coverslips were stained with FITC-conjugated anti-Salmonella antibody at 30° C. for one hour and washed three times with staining buffer.

For Salmonella, released GFP and vacuole staining (combination 2), coverslips were stained sequentially with anti-LAMP1 primary antibodies at 30° C. for one hour, and washed three times with staining buffer. Coverslips were incubated with Alexaflor-647 chicken anti-rabbit secondary antibodies (ThermoFisher) at a 1:200 dilution for one hour at 30° C. and washed four times with staining buffer. Coverslips were then stained with FITC-conjugated anti-Salmonella antibody and anti-myc primary antibody; and washed three times with staining buffer. Coverslips were incubated with Alexaflor-568 goat anti-rat secondary antibodies (ThermoFisher) at a 1:200 dilution for one hour at 30° C. to identify GFP.

After all staining, coverslips were washed three times with staining buffer and mounted to glass slides using 20 μl mountant with DAPI (ProLong Gold Antifade Mountant, Thermofisher). Mounted coverslips were cured overnight at room temperature.

**Quantification of Vacuolar Fraction, Extent of Invasion, and Lysis of Engineered Salmonella**

To quantify what fraction of intracellular, flhDC expressing Salmonella were located within vacuoles, coverslips were infected with either the parental control strain of Salmonella or FID Salmonella as described in the infection assay section. Coverslips were then stained for LAMP1, Salmonella and nuclei as described in the immunocytochemistry section. Coverslips were imaged at 100× as described in the microscopy and image analysis section. Between ten and twenty cells from either the control group or FID treated group were analyzed. Salmonella either colocalized or bordered very closely by LAMP1 were defined as inside vacuoles. Salmonella that were not localized with LAMP1 closely bordering the bacteria were defined as cytosolic.

**Results**

Controlling flhD Expression Improves Tumor Targeting of Salmonella

Suppressing flhD expression of Salmonella in systemic circulation improved tumor colonization of the bacteria. While tumor colonization levels were 108 CFU/gram of tumor for both control and ΔflhD Salmonella, liver colonization of ΔflhD Salmonella was reduced ten-fold as compared to control (FIG. 5A; *, P<0.05). When flhDC was overexpressed before injection, however, tumor colonization was impaired compared to a bacterial control (FIG. 5B). These results indicated that flhDC expression before injection increased the clearance rate of Salmonella in the blood. However, suppression of flhDC before injection increased tumor colonization and specificity of Salmonella.

A lack of flhDC activity, not just a lack of flagellar expression, reduced liver colonization while maintaining similar tumor colonization levels of Salmonella. Mice were infected with three different Salmonella strains: ΔflhD, ΔfliGHI, and ΔflhD+ΔfliGHI. The AfiGHI strain lacks flagella but retains flhDC activity. The ΔflhD and ΔflhD+ΔfliGHI, which lack flhDC activity, both colonized livers 8.5 and 20-fold less than the flagellar deficient, ΔfliGHI, strain, respectively (FIG. 5C; *, P<0.05). However, tumor colonization levels of all three strains were not different (FIG. 5D). These results indicated that reduced flhDC activity, and not merely a lack of flagella, increased tumor specificity of Salmonella.

flhDC Expression Increases Intratumoral Dispersion of Salmonella

Suppressing expression of flhDC caused Salmonella to predominantly colonize and grow within tumor necrosis. flhDC uninduced were systemically administered into mice and half of the mice were administered arabinose to induce flhDC expression (FIG. 6A). Salmonella not expressing flhDC were not motile and as a result, formed spatially separated, dense colonies predominantly within tumor necrosis (yellow arrow, FIG. 6B). A small fraction of these colonies, however, were located within viable tumor tissue (green arrows, FIG. 26B). The fraction of these dense colonies present in necrosis was 75% percent as compared with 25% percent of colonies in viable tumor tissue (FIG. 6C). If it is assumed that each spatially segregated, dense colony originated from a single bacterium, the growth rate of colonies in necrosis was 0.12 hr-1 as compared to a slightly reduced rate of 0.11 hr-1 within viable tissue (*, P<0.05; FIG. 6D). These bacterial growth rates correspond to doubling times of 6 hours within necrosis and 6.5 within viable tumor tissue (*, P<0.05; FIG. 6E), which is consistent with previous estimates (45). These results indicate that Salmonella heavily favor colonization and growth within tumor necrosis as compared with viable tumor tissue.

Reexpressing flhDC within intratumoral Salmonella increased dispersion and tumor coverage of the bacteria. ΔflhD Salmonella with the PBAD-flhDC genetic circuit were injected intravenously into 4T1 tumor bearing mice and administered two doses of arabinose intraperitoneally to induce flhDC expression in Salmonella (FIG. 6A). flhDC induction of intratumoral Salmonella increased both the bacterial colony size along with bacterial coverage within the tumor (FIG. 6F). The colony size of flhDC reexpressing Salmonella increased 1.5-fold as compared with an uninduced control (*, P<0.05; FIG. 6G). While flhDC uninduced Salmonella formed dense, tightly packed colonies (top panel, FIG. 6H), a larger number of flhDC induced bacteria were located outside of these dense colonies (bottom panel, green arrows, FIG. 6H). The number of Salmonella outside of a dense, central colonies (termed satellite colonies) increased two-fold when flhDC was induced within intratumoral Salmonella (*, P<0.05; FIG. 6I). These results indicate that intratumoral induction of flhDC in Salmonella enables the bacteria to migrate away from dense colonies within tumor necrosis and towards viable cancer cells.

In Situ Expression of flhDC is Needed to Increase Intracellular Invasion of Salmonella into Spatially Distant Cells

Reexpression of flhDC increased spatial distribution of intracellular Salmonella within tumors. A tumor-on-a-chip device was used to quantify spatial distribution of intracellular Salmonella (FIG. 7A). These Salmonella expressed flhDC with arabinose supplementation and GFP after intracellular invasion (Intracellular reporting Salmonella, IR Sal). Arabinose induction of flhDC enabled broad distribution of intracellular expressing GFP Salmonella within in vitro tumor masses (+flhDC, FIG. 7B). However, uninduced ΔflhD Salmonella (−flhDC) were detected at very low concentrations throughout tumor masses (white arrow, FIG. 7B). The presence of intracellular Salmonella gradually increased deeper into tumor tissue and was enriched 140-fold in +flhDC as compared to −flhDC Salmonella for x>0.5 (**, P<0.01; ***, P<0.001; FIG. 7C). The overall amount of flhDC expressing, IR Salmonella increased exponentially over time as compared to the −flhDC control (*, P<0.05; **, P<0.01; ***, P<0.001; FIG. 7D). This indicated that flhDC induction increased the coverage of intracellular Salmonella in tumor masses.

When +flhDC IR Sal (FIG. 7E) was administered into mice and arabinose induced to express flhDC(FIG. 7F), a greater fraction of bacteria was located intracellularly (Induced, white squares; FIG. 7F). Intracellular invasion of flhDC reexpressing Salmonella increased 2.3-fold over the −flhDC IR Sal control (*, P<0.05; FIG. 7G).

Euclidean distance mapping of histoogical sections, which quantifies the proximity of every location within a tumor to the nearest bacterium, was used to quantify the distribution of intracellular bacteria. The spatial coverage of intracellular bacteria was greater after flhDC induction as indicated by Euclidean distance modeling of histological sections (FIG. 7H). Salmonella were distributed 1.6-fold more after flhDC induction (*, P<0.05; FIG. 7I). These results indicated that flhDC expression increases intracellular invasion by positioning more bacteria near a greater number of viable cancer cells. In addition, flhDC expression increases intracellular invasion in a flagella and T3SS-1 driven manner (10).

Controlling flhDC Expression Improves GFP Delivery Distribution within Tumors

Induction of flhDC within intratumoral engineered Salmonella increased protein delivery over a larger area of cells. Induced Salmonella delivered protein into a broad, spatially distributed set of cells within tumors (FIG. 8A). Euclidean distance mapping analysis of intratumoral delivery demonstrated that flhDC induction (flhDC intracellular delivering Salmonella; FID Sal) increased spatial delivery coverage 1.6-fold as compared to flhDC uninduced (Uninduced intracellular delivering Salmonella; UID Sal) Salmonella (***, P<0.001; FIG. 8B). These results demonstrate that flhDC induction increased spatial coverage in tumors (FIG. 7H, I), which, enabled the bacteria to intracellularly deliver protein into broadly distributed cells within tumors.

**Engineered Salmonella is Superior in Tumor Colonization and Protein Delivery Compared to Exclusively Cytosolic Salmonella**

Engineered Salmonella colonized tumors and delivered significantly more protein inside cancer cells compared to conventionally used, cytosolic Salmonella. As demonstrated, ΔflhD Salmonella did not colonize tumors less than a control (FIG. 5A). However, ΔsifA Salmonella colonized tumors ten-fold less than the control (*, P<0.05; FIG. 9A). Liver colonization was also reduced ten-fold between ΔsifA and control Salmonella (*, P<0.05; FIG. 9B) indicating that the ΔsifA strain exhibited overall poor fitness in vivo. Using a selective staining technique to detect bacterial lysis and protein delivery as previously described, engineered Salmonella visibly lysed more than ΔsifA Salmonella inside cells at all time points (FIG. 9C). FID Sal lysed 18-fold more than ΔsifA Salmonella (FIG. 9D; **, P<0.01). Cytosolic localization is important for protein therapies to have biological activity and anti-cancer activity. However, these results demonstrate that predominantly cytosolic Salmonella are not well suited for therapeutic delivery. This is a result of a combination of poor tumor colonization, poor systemic infectivity in vivo and poor lysis efficiency of ΔsifA compared to FID Sal. The ΔsifA strain of Salmonella fails to effectively colonize tumors and therefore, is not advantageous for intracellular protein delivery.

flhDC Expression Reduces Lysis Efficiency within Intracellular Salmonella

flhDC expression in Salmonella affects intracellular lysis and protein delivery after invasion. To understand this dynamic, cancer cells were infected with control lysing Salmonella (ID Sal) or lysing Salmonella reexpressing flhDC (FID-Sal) (FIG. 10A). As expected, FID Sal invaded cancer cells three times more than ID Salmonella (FIG. 10B, C; **, P<0.01). However, FID Sal lysed 33% less than control ID Sal (FIG. 10D; **, P<0.01). To understand why, the vacuolar/cytosolic distribution of control and flhDC expressing Salmonella was quantified after cancer cell infection (FIG. 10E). While most control Salmonella were contained in vacuoles (colocalized green and red), a larger percentage of flhDC reexpressing Salmonella were cytosolic (green only, FIG. 10F). On a population level, 90% of control were in vacuoles compared to 70% of flhDC reexpressing Salmonella (FIG. 10G). As a result, ID Salmonella were more likely to remain in vacuoles and lyse (white arrows, FIG. 10H) while a small fraction of FID Sal were more likely to escape the vacuole and remain intact (light blue arrows, FIG. 10I). In vivo, FID Sal qualitatively demonstrated a similar phenomenon (FIG. 10J). Unlysed and intracellular FID Sal were distributed throughout several cells (white arrows), likely, indicating that the bacteria were hyper-replicating in the cytoplasm of the tumor cells. These results indicate that flhDC induction increases invasion but decreases lysis efficiency of engineered Salmonella, likely because of vacuolar escape.

Vacuolar Retention of flhDC Overexpressing Salmonella Rescues Lysis and Protein Delivery Efficiency

Overexpressing flhDC in a vacuole escape impaired strain of engineered Salmonella rescued lysis efficiency and overall intracellular protein delivery. It was previously demonstrated that engineered ΔsseJ Salmonella intracellularly lysed with high efficiency. It was therefore hypothesized that overexpressing flhDC in lysing ΔsseJ Salmonella (ΔsseJ FID Sal) would rescue lysis efficiency while maintaining high levels of invasion. Cells infected with ΔsseJ FID Sal exhibited an increase in invaded, lysed bacteria (white arrow, FIG. 11A). The ΔsseJ FID Sal invaded cancer cells 1.5-fold more than FID Sal and three-fold more than ID Sal (FIG. 11B, **, P<0.01). Intracellular ΔsseJ FID Sal also lysed 25% more efficiently than FID Sal alone (FIG. 11C; **, P<0.01). The combination of these two phenomena (increased invasion and improved lysis) of the engineered strain increased overall protein delivery 2.5-fold over FID Sal (FIG. 11D; **, P<0.01). This data demonstrated that the reduced lysis efficiency resulting from flhDC activity could be rescued by overexpressing the transcription factor in Salmonella engineered to remain in vacuoles.

**Conclusions**

Modulating flhDC expression in engineered Salmonella had broad implications for intracellular therapeutic delivery within tumors (FIG. 12). Salmonella devoid of flhDC expression colonized tumors more selectively. However, overexpression of the transcription factor within systemic Salmonella decreased tumor colonization of the bacteria. Controlled expression of flhDC in tumors increased spatial distribution of extracellular and intracellular Salmonella. While flhDC expression reduced intracellular lysis efficiency of engineered Salmonella, overexpressing the transcription factor in a vacuolar resident, ΔsseJ, strain rescued lysis efficiency and improved overall protein delivery in tumor cells. Together, results demonstrate the modulating flhDC expression in therapeutic Salmonella improves several driving features of protein delivery in tumors (FIG. 12).

**Discussion**

It is shown herein that controlling flhDC expression of engineered bacteria maintains high colonization levels, improves tumor specificity and increases protein delivery distribution within tumors. Expression of flhDC also decreased intracellular lysis efficiency but was rescued by overexpressing the transcription factor in a vacuole localized strain (ΔsseJ) of Salmonella. The combination of the two genetic engineering strategies increased overall intracellular protein delivery.

The colonization pattern of flhDC uninduced Salmonella suggests that only a few hundred single bacteria infiltrate tumors and grow in situ out of the two million that are injected. These ratios are corroborated by earlier work demonstrating that one out of ten thousand bacteria adhere to tumor vasculature (46). In histological samples flhDC uninduced Salmonella form spatially separated colonies overwhelmingly localized to tumor necrosis (FIG. 6B, C). Each of these colonies could originate from clonal expansion of a single bacteria that managed to colonize the tumor. If this is the case, it would suggest that bacterial influx into tumors occurs as a rare event, is strongly assisted by extensive necrosis, and is the rate limiting step of tumor colonization. Such a rare bacterial infiltration event could explain why tumor colonization is highly variable within populations of mice or humans as described previously (47). These results could explain why extensive tumor colonization was predominantly detected predominantly in the presence of tumor necrosis in humans (47). Combining tumor vascular disrupting agents with Salmonella could therefore reduce treatment variability between patients and enable effective colonization of small, necrosis deficient primary and metastatic tumors.

Two strategies could be used to robustly initiate bacterial colonization within tumors: (1) Co-administering bacteria along with a mild TNF-alpha inducer as previously described (48) or (2) genetically modifying Salmonella to evade systemic innate immune recognition (e.g., flhDC modulation). In scenario (1) as previously demonstrated, administration with lipid A (a known TNF-alpha inducing agent) did not cause septic shock but increased vascular permeability and therefore, could have increased the probability of bacterial infiltration into tumors across a large number of mice. In scenario (2), flhDC suppression of injected Salmonella could help the bacteria evade innate immune detection of flagella in systemically circulating bacteria. This could enable bacteria to persist longer systemically without causing septic shock. Longer systemic persistence could, in turn, increase the probability of bacterial infiltration into tumors.

Wild type Salmonella are likely not optimized to deliver therapies intracellularly within tumors. One reason for this might be that necrotic tumor tissue facilitates cecile and non-motile colonization of Salmonella. The data suggests that tumors select for non-motile and likely, non-flagellated bacteria since flagellated bacteria minimally colonize tumors (FIG. 5B) likely due to innate immune mediated clearance (8, 9). The flhDC uninduced bacteria were not impaired in colonization levels as compared to the control strain (FIG. 5D). Moreover, flhDC uninduced bacteria clustered in densely packed colonies largely located within tumor necrosis (FIG. 6B). This suggests that Salmonella have a higher affinity to colonize necrosis rather than viable tissue and that external control is required to enable Salmonella to invade viable tumors cells in an flhDC dependent manner. By controllably activating flhDC expression in intratumoral Salmonella, it was demonstrated that a significant fraction of these bacteria invaded and delivered protein into a spatially distributed set of cells.

Vacuolar residence could also aid in preventing premature clearance before tumor accumulation in addition to enabling lysis of engineered Salmonella. The current paradigm for intracellular, cytosolic therapeutic delivery is to enable Salmonella to escape the vacuole and directly invade the cytosol through deletion of the sifA gene (20). Similarly, bacterial variants expressing listeriolysin O have also been used to enable vacuolar escape of therapeutic Salmonella (49-51). However, it was determined that unnatural cytosolic escape of Salmonella (ΔsifA) reduced tumor colonization 100-fold compared to the parental strain (FIG. 9A). This is likely because cytosolic pathogens elicit a strong antimicrobial and NF-kB dependent immune response that is detrimental to bacterial fitness in vivo (21, 52-55). The ΔsifA bacteria also lysed 18-fold less than FID Salmonella. These results indicate that the engineered strain significantly improved the delivery potential Salmonella as compared to existing cytosolic delivery methods.

The engineered bacterial system described herein shares similarities with strains of Salmonella Typhi that have evolved to systemically infect human hosts. Humans serve as the natural host for Salmonella Typhi and upon ingestion, the bacteria stealthily translocate from the gut into systemic circulation without attracting a significant initial immune response (30). The bacteria can circulate systemically for extended periods of time without causing septic shock (30). The typhoidal strain accomplishes this by encoding a capsular regulatory protein, TviA. The transcription factor encodes for the Vi capsule that masks bacterial LPS (56). In addition, TviA suppresses flagellar and T3SS-1 activity in systemically circulating bacteria through repression of flhDC and HiLA expression, respectively 57. Masking of the LPS and downregulation of flagellar and T3SS-I activity leads to evasion of innate immune recognition (57). The instant delivery strain of Salmonella also has a modified LPS through deletion of msbB which prevents sepsis. In addition, the expression of flhDC, which activates flagellar and to a lesser extent, T3SS-1 synthesis (10), is suppressed upon systemic administration of the engineered Salmonella. The engineered strain of Salmonella and Salmonella Typhi also share the similarity that both types of bacteria reside mostly within the intracellular vacuole. Residence within the intracellular vacuole prevents bacterial detection by cytosolic, innate immune sensors like nod-like receptors, ubiquitin and NF-kB components. These genetic modifications likely act to mask common pathogen associated molecular patterns associated with Salmonella and increase systemic persistence without causing any adverse immune responses.

### Example III

Chromosomal Integration of flhD in EBV-002.

The cell invasive capability of EBV-002 containing a single, chromosomal copy of PBAD-flhDC was assessed. Chromosomal integration of an inducible version of flhDC can create a master delivery vehicle that could be used to deliver any therapy into a tumor. Creating a single master delivery vehicle can streamline the manufacturing process of any EBV based therapy. To this end, a single copy of PBAD-flhDC was integrated in place of the endogenous flhDC gene within VNP20009 Salmonella. This chromosomally integrated strain was grown with arabinose to activate flhDC expression and used to infect cancer cells. The chromosomally integrated VNP20009 invaded cancer cells to similar levels as the bacteria containing episomal copies of flhDC (FIG. 13A). The chromosomal knockin of flhDC also was similarly inducible as compared to Salmonella with episomal PBAD-flhDC (FIG. 13B). This result indicated that the flhDC inducible genetic circuit could be genomically integrated in order to create a master EBV-002 delivery vehicle.

**Development of a Clinical Strain of EBV-002.**

A clinically compatible strain of EBV-002 was created by controlling activation of flhDC with salicylic acid, the active ingredient in aspirin (FIG. 14). Since flhDC is the main transcription factor controlling flagellar synthesis, chemotaxis and motility, bacteria are highly sensitive to even low expression levels of the protein. As a result, it was hypothesized that expression levels in the uninduced state would need to be tightly repressed in order to fully suppress uninduced cell invasion. To test this hypothesis, four different flhDC inducible EBV-002 strains were produced: salicylic induced 1) flhD, 2) flhD containing a weakly active ssrA degradation sequence, 3) flhD containing a moderately active ssrA degradation sequence and 4) flhD containing a highly active degradation tag (FIG. 14). The purpose of these degradation tags was to eliminate uninduced flhD activity that was a result of leaky expression from the pSal promoter. The intracellular invasion rates of each of these four strains were compared to the PBAD inducible version of flhDC. As expected, sample (1) was highly motile and invasive, with or without salicylic acid induction (FIG. 14B) indicating that the salicylic acid promoter was leaky. Samples (2), (3) and (4) only invaded cells after salicylic acid induction and were completely non-invasive otherwise. However, samples (2) and (3) were the most intracellularly invasive after aspirin induction (FIG. 14B). Most importantly, strains (2) and (3) were more invasive as compared to the PBAD inducible version of EBV-002 (FIG. 14B). These results demonstrate that the salicylic acid induction circuit was optimized to express flhD and regulate intracellular invasion of EBV-002 into cancer cells.

Sample (2) was characterized since this strain of EBV-002 had the highest range of activation between uninduced and induced samples. The induced bacteria swam a significantly longer distance as compared to uninduced EBV-002, which, remained stationary (FIG. 15A). Salicylate induced EBV-002 swam 12.7-fold farther than the uninduced control (***, P<0.001; FIG. 15B). This indicated that the salicylic acid inducible genetic circuit could robustly control flhDC activity in the clinical EBV-002 strain. As expected, the salicylic acid induced, clinical strain of EBV-002 invaded cancer cells 30 times more than the uninduced control (***, P<0.001; FIG. 15C, D). These results indicate that expressing flhD with a weakly active degradation tag using salicylic acid enabled the greatest control of intracellular invasion of EBV-002.

After determining which version of pSal-flhD was most effective at invading cancer cells with salicylic acid induction, the lowest amount of salicylic acid needed to enable intracellular invasion was determined next. EBV-002 was induced with either 10 nanomolar (nM), 100 nM, 500 nM, 1 micromolar (uM) or 10 uM salicylic acid and infected cancer cells with each of these strains. It was determined that a 500 nM concentration of salicylic acid was needed to enable intracellular invasion of EBV-002 (FIG. 16). This result is significant because it indicates that the induction threshold for EBV-002 is well within the concentration range of salicylic acid found in the blood stream (10-50 uM) after a person orally ingests aspirin. Together, these results indicate that EBV-002 is ready for use as an intracellular delivery vehicle within human tumors. Incorporation of the ΔsseJ mutation into EBV-002 to create EBV-003.

The ΔsseJ mutation was previously demonstrated to significantly increased lysis efficiency of the EBV strain. To this end, the EBV-002 strain containing the same salicylic acid inducible flhDC gene as well as the intracellular lysis cassette was additionally engineered with the ΔsseJ mutation in order to create EBV-003.

**In Vivo Efficacy of EBV-003.**

Biodistribution and tumor selective protein delivery were assessed in mice bearing subcutaneous 4T1 tumors. Balb/C mice with ˜750 mm3 subcutaneous tumors were intravenously injected with 1×107 CFU of EBV-003. At 72 hours p.i., mice were intraperitoneally injected with 5 mg of salicylic acid to induce flhDC expression within intratumoral bacteria. 24 hours later, mice were sacrificed and tumors, livers, and spleens were excised for analysis. Colonization and protein delivery of EBV-003 was compared to EBV-001 to assess any improvements. After colonization, EBV-003 colonized tumors 10.7-fold more than EBV-001 while keeping spleen and liver colonization unchanged (FIG. 17A, **, P<0.01). On average, EBV-003 delivered 31-fold more protein into tumor cells as compared to EBV-001 (FIG. 17B). Protein delivery was not, however, detected in the spleen or livers with either strain. These results demonstrate that EBV-003 is significantly more effective at colonizing and delivering protein selectively into tumors while sparing healthy tissue.

To determine whether EBV-003 intracellularly invaded cancer cells after salicylic acid induction in vivo, female balb/c mice were subcutaneously injected with 4T1 tumors. Once tumors were 500 mm3, the mice were injected with 1×106 CFUs via the tail vein. Seventy-two hours after bacterial administration, seven of the mice were intraperitoneally injected with 5 mg of sodium salicylate while four were given a saline injection as a control. Twenty-four hours after salicylic acid administration, the mice were sacrificed, tumors were excised, fixed and stained for Salmonella. Histological examination revealed that salicylic acid induction increased intracellular invasion of viable cancer cells within quiescent tumor tissue. More bacteria (Red Xs, FIG. 18A) were distributed across the quiescent tumor tissue after induction with salicylic acid (FIG. 18B). Salicylic acid induction resulted in a two-fold increase in cancer cells with intracellular EBV-003 as compared to the uninduced control (*, P<0.05; FIG. 18C). These results indicated that EBV-003 could be induced to invade cells using a therapeutic dose of salicylic acid.

Intracellular protein delivery with EBV-003 was also evaluated with and without salicylate induction. After salicylic acid induction, protein delivery was detected in five out of six tumors within the transition zones where tumor cells are rapidly dividing (white arrows, FIG. 19A). Whereas, delivery was only detected within the transition zone in one of the four uninduced, control mice (FIG. 19B). These results demonstrated that salicylate induction of EBV-003 enabled intracellular protein delivery in vivo.

In vivo colonization, invasion and protein delivery of EBV-003 in spontaneous breast cancer metastasis in the liver. The EBV-003 strain colonized, invaded and delivered protein selectively into metastatic breast cancer within the liver (FIG. 20). All dense bacterial colonies were only found within the metastatic breast cancer lesions within the liver (white outlined colonies, FIG. 20A). Moreover, 85% of these colonies were immediately adjacent, or within actively dividing tumor lesions (red arrows, FIG. 20A), where therapeutic delivery is most effective. On the other hand, colonies found in healthy tissue were observed far less frequently and were much smaller in size (FIG. 20A). Bacterial colonies were rarely spotted in healthy tissue and were very small (1, white arrow, FIG. 20B). However, in the metastatic lesions, the colonies appeared significantly larger in area (2, white arrows, FIG. 20B). Within the liver, 87.7% of colonies were found within the metastatic lesions while the other 12.3% were found within healthy liver tissue. Moreover, the size of the colonies within the metastatic lesions was over 118 times greater than the size of colonies in healthy tissue (***, p=2.2×10-26; FIG. 20C). This equates to an 850-fold enrichment of EBV-003 in metastatic breast cancer lesions within the liver versus the immediately adjacent healthy tissue. While we have demonstrated the ability of therapeutic Salmonella to colonize primary tumors greater than 1,000-fold more than any other organ, this is the first demonstration that Salmonella preferentially colonize metastatic tumor lesions as compared to immediately adjacent healthy tissue to a similarly high magnitude. This illustrates the exquisite selectivity of EBV-003 to colonize tumor tissue regardless of whether the tumors are primary or metastatic lesions.

The EBV-003 strain also intracellularly invaded cancer cells within liver metastases (white arrows, FIG. 21A). However, there was no difference in invasion levels between salicylate induced and uninduced EBV-003 (FIG. 21B). One reason for this could be that most of the metastatic lesions contained a higher fraction of viable tumor tissue and lower amount of necrosis. As a result, EBV-003 bacteria were more likely to be in close proximity to viable tumor cells increasing the likelihood that the bacteria could intracellularly invade the cells regardless of induction status. This is in contrast to primary tumor tissue, where salicylate induction of flhDC increased the intracellular presence of EBV-003 within the quiescent tumor tissue (FIG. 18A). This could be because the bacteria preferentially colonized necrosis and required flhDC dependent motility to swim towards and intracellularly invade the actively dividing cancer cells. Therefore, this indicates that flhDC induction is necessary for intracellular invasion within a primary tumor mass but less so within small, non-necrotic metastatic or primary lesions.

Although EBV-003 seemed to invade metastatic cancer cells in the presence or absence of flhDC activity, protein delivery was detected at higher frequencies with salicylate induction in vivo. Cytosolic delivery into cells within metastatic tumors was detected histologically (white arrow, FIG. 22A). The frequency of protein delivery into cells within metastases was three-fold higher in induced EBV-003 versus uninduced EBV-003 (**, P<0.01; FIG. 22B). Taken together, these results indicate that induction of flhDC improves protein delivery in both primary and metastatic breast tumors.

All publications, patents, and patent applications mentioned in this specification are herein incorporated by reference to the same extent as if each individual publication, patent, or patent application was specifically and individually indicated to be incorporated by reference. In the event that the definition of a term incorporated by reference conflicts with a term defined herein, this specification shall control.

