# DESCRIPTION

## FIELD OF THE INVENTION

The present invention relates to non-naturally occurring anti-bacterial peptides. More specifically the peptides can be used to treat multi-drug resistant bacterial infections. In addition, the present invention provides methods for producing anti-bacterial peptides.

Introduction to the Invention

Microorganisms resistant to multiple drugs are of major concern and the consequences can be most profound for children, who are especially susceptible to bacterial infections. The most common childhood bacterial diseases in developing countries—pneumonia, other respiratory infections, and dysentery—are no longer curable by many of the older antibiotics. The consequences are devastating: bacterial acute respiratory infections, for example, kill more than three million children every year. Many cases of these illnesses are caused by strains now resistant to common antibiotics. In wealthier countries, hospitals are reeling from an explosion of methicillin-resistant Staphylococcus aureus (MRSA). From 1974 to 2004, MRSA prevalence increased from roughly 2 percent to more than 50 percent of staph infections in many U.S. hospitals, resulting in tens of thousands of deaths. Resistance to antibiotics also has a startling impact on the cost of curing patients. In many poor countries, expenditures for drugs represent a large proportion of overall health-care costs, ranging from 20 to 60 percent of total expenditure on health. When first-line drugs fail, second-line alternative drugs are almost always far more costly and require greater medical oversight. For example, it costs as much to cure one patient of extensively drug-resistant TB as it does to cure 200 patients of susceptible TB. Besides the known pathogens, the relatively recent appearance of opportunistic organisms, intrinsically resistant to many drugs, is also complicating the advances that have been made in medical technologies. With a larger number of immunocompromised patients and longer time periods spent in an immunocompromised state, several organisms have become ‘specialized’ pathogens-typically attacking only the most vulnerable patients. Among these opportunistic pathogens are the enterococci, the coagulase-negative staphylococci, Pseudomonas aeruginosa and Acinetobacter baumanii.

In the present invention we disclose a novel designer antibiotics paradigm that exploits protein aggregation to kill bacteria by widespread proteostatic collapse. Loss of protein homeostasis or proteostasis1 is a constant threat for any living cell due to the highly crowded intracellular environment that brings into close proximity a large variety of polypeptides that need to undergo error-prone folding reactions in order to attain their native conformation2. To control this threat, cells have evolved a complex network of molecular chaperones, proteases and other specialized molecules3. Although the basic components of this machinery are conserved in all kingdoms of life, bacteria have relatively simple proteostatic networks in comparison to the more elaborate versions observed in eukaryotic cells4. In spite of these cellular response mechanisms, human protein folding pathologies have made clear that under persistent exposure to aggregating proteins, e.g. as a result of mutation, the proteostasis network eventually breaks down, which ultimately results in cell death5. On the other hand, protein aggregation turns out to be a highly ordered and specific process: aggregation is more efficient between similar than between unrelated polypeptides6-8. At a mechanistic level protein aggregation is mediated by short aggregation prone sequence segments (called APRs), which on average occur roughly once every 100 amino acids in the primary polypeptide sequence. These APRs are generally (but not always) sequence segments constituting the hydrophobic core of globular proteins or protein-protein interaction interfaces. Unfortunately the physicochemical requirement for providing these tertiary protein interaction modes also favour protein aggregation9. Indeed while forming the most stable part of the native proteins, in (partially) unfolded proteins, APRs can also self-assemble with identical APRs from another protein to form even more stable beta-structured aggregates10. The risk of aggregation is thus highest during translation, before the protein attains its native conformation11. Overall the bacterial proteomes display a higher intrinsic aggregation susceptibility than higher order organisms14. In addition, protein turnover in bacteria needs to be faster in order to support their high metabolic and cell division rates, which further increases the potential for protein aggregation in bacteria. Bacterial proteostasis is increasingly being investigated as a drug target for novel antibiotics15. For example, the antimicrobial peptide (AMP) oncocin exerts its antibacterial activity by binding to the ribosome exit channel16. Interestingly, there is a notable overlap between the amyloid forming propensity of peptides and their antibacterial activity17-18 and we observed earlier that peptides derived from aggregating protein fragments of the Gram positive Staphylococcus epidermidis, can be toxic specifically to these bacteria19.

Protein aggregation is a sequence specific process allowing a protein to template its own aggregation but not that of heterologous sequences. We previously exploited this feature for specific protein detection in Western Blots or to induce specific protein knockdown using short aggregation-prone sequence fragments (APRs) of target proteins. Using a transgenic GFP carrier fused to target APRs we generated specific loss-of-function phenotypes in both Arabidopsis thaliana and Zea mays resulting in increased plant size and increased starch content respectively21. We also found that it is possible to use synthetic peptides to the same effect and recently designed a peptide named vascin that is internalized by human cells and specifically induces VEGFR-2 inactivation by aggregation. Moreover vascin displays in vivo activity inhibiting VEGFR-2 dependent tumor growth in a syngenic murine model47. Recently we also explored whether synthetic peptides harboring endogenous APRs of Gram-positive Staphylococci can induce aggregation-associated bacterial death in vitro and found several that are active against MRSA in vivo19.

## SUMMARY OF THE INVENTION

Given the rapidly increasing problems of antibacterial resistance, especially in Gram-negative bacteria, we here set out to develop novel antimicrobial peptides against the enterohemorrhagic E. coli strain O157:H7, which produces high levels of toxins, leading to hemorrhagic diarrhea and kidney failure upon infection20. Under the assumption that the development of drug resistance is more difficult for drugs that have multiple targets, we searched for peptides that can cause misfolding or aggregation of multiple bacterial proteins. To this end we screened 125 aggregating sequences that had a high degree of redundancy in the E. coli proteome. In this manner one peptide containing this APR could potentially affect the folding of many proteins containing highly similar APRs. Using this strategy we identified several peptides that efficiently induce bactericidal protein aggregation and inclusion body formation in pathogenic E. coli. One representative of these—designated herein further as colpeptin1—showed little or no toxicity towards mammalian cells, and was also active against a number of other bacterial pathogens (see Table 2) and was effective in vivo against E. coli in a mouse bladder infection model. We demonstrate that the internalization of the non-natural peptides, such as colpeptin1, is followed by a rapid aggregation cascade of multiple proteins promoted by the primary colpeptin1 targets resulting in bacterial proteostatic collapse.

## DETAILED DESCRIPTION TO THE INVENTION

The present invention will be described with respect to particular embodiments and with reference to certain figures but the invention is not limited thereto but only by the claims. Any reference signs in the claims shall not be construed as limiting the scope. Of course, it is to be understood that not necessarily all aspects or advantages may be achieved in accordance with any particular embodiment of the invention. Thus, for example those skilled in the art will recognize that the invention may be embodied or carried out in a manner that achieves or optimizes one advantage or group of advantages as taught herein without necessarily achieving other aspects or advantages as may be taught or suggested herein.

The invention, both as to organization and method of operation, together with features and advantages thereof, may best be understood by reference to the following detailed description when read in conjunction with the accompanying figures. The aspects and advantages of the invention will be apparent from and elucidated with reference to the embodiment(s) described hereinafter. Reference throughout this specification to “one embodiment” or “an embodiment” means that a particular feature, structure or characteristic described in connection with the embodiment is included in at least one embodiment of the present invention. Thus, appearances of the phrases “in one embodiment” or “in an embodiment” in various places throughout this specification are not necessarily all referring to the same embodiment, but may. Similarly, it should be appreciated that in the description of exemplary embodiments of the invention, various features of the invention are sometimes grouped together in a single embodiment, figure, or description thereof for the purpose of streamlining the disclosure and aiding in the understanding of one or more of the various inventive aspects. This method of disclosure, however, is not to be interpreted as reflecting an intention that the claimed invention requires more features than are expressly recited in each claim. Rather, as the following claims reflect, inventive aspects lie in less than all features of a single foregoing disclosed embodiment.

Where an indefinite or definite article is used when referring to a singular noun e.g. “a” or “an”, “the”, this includes a plural of that noun unless something else is specifically stated. Where the term “comprising” is used in the present description and claims, it does not exclude other elements or steps. Furthermore, the terms first, second, third and the like in the description and in the claims, are used for distinguishing between similar elements and not necessarily for describing a sequential or chronological order. It is to be understood that the terms so used are interchangeable under appropriate circumstances and that the embodiments, of the invention described herein are capable of operation in other sequences than described or illustrated herein. The following terms or definitions are provided solely to aid in the understanding of the invention. Unless specifically defined herein, all terms used herein have the same meaning as they would to one skilled in the art of the present invention. Practitioners are particularly directed to Sambrook et al., Molecular Cloning: A Laboratory Manual, 4th ed., Cold Spring Harbor Press, Plainsview, N.Y. (2012); and Ausubel et al., Current Protocols in Molecular Biology (Supplement 114), John Wiley & Sons, New York (2016), for definitions and terms of the art. The definitions provided herein should not be construed to have a scope less than understood by a person of ordinary skill in the art.

“About” as used herein when referring to a measurable value such as an amount, a temporal duration, and the like, is meant to encompass variations of ±20% or ±10%, more preferably ±5%, even more preferably ±1%, and still more preferably ±0.1% from the specified value, as such variations are appropriate to perform the disclosed methods.

The terms “polypeptide” and “peptide” are interchangeably used further herein to refer to a polymer of amino acid residues and to variants and synthetic analogues of the same. Thus, these terms apply to amino acid polymers in which one or more amino acid residues is a synthetic non-naturally occurring amino acid, such as a chemical analogue of a corresponding naturally occurring amino acid, as well as to naturally-occurring amino acid polymers. This term also includes post-translational modifications of the polypeptide, such as glycosylation, phosphorylation, amidation, oxidation and acetylation. By “recombinant polypeptide” is meant a polypeptide made using recombinant techniques, i.e., through the expression of a recombinant or synthetic polynucleotide. The term “expression” or “gene expression” means the transcription of a specific gene or specific genes or specific genetic construct. The term “expression” or “gene expression” in particular means the transcription of a gene or genes or genetic construct into structural RNA (rRNA, tRNA) or mRNA with or without subsequent translation of the latter into a protein. The process includes transcription of DNA and processing of the resulting mRNA product. The term “recombinant host cell”, “engineered cell”, “expression host cell”, “expression host system”, “expression system” or simply “host cell”, as used herein, is intended to refer to a cell into which a recombinant vector and/or chimeric gene construct has been introduced. It should be understood that such terms are intended to refer not only to the particular subject cell but to the progeny of such a cell. Because certain modifications may occur in succeeding generations due to either mutation or environmental influences, such progeny may not, in fact, be identical to the parent cell, but are still included within the scope of the term “host cell” as used herein. The term “modulate,” “modulates,” or “modulation” refers to enhancement (e.g. an increase) or inhibition (e.g. a decrease) in the specified level or activity. The term “enhance” or “increase” refers to an increase in the specified parameter of at least about 1.25-fold, 1.5-fold, 2-fold, 3-fold, 4-fold, 5-fold, 6-fold, 8-fold, 10-fold, twelve-fold, or even fifteen-fold.

The term “inhibit” or “reduce” or grammatical variations thereof as used herein refers to a decrease or diminishment in the specified level or activity of at least about 15%, 25%, 35%, 40%, 50%, 60%, 75%, 80%, 90%, 95% or more. In particular embodiments, the inhibition or reduction results in little or essentially no detectible activity (at most, an insignificant amount, e.g., less than about 10% or even 5%).

The term “contact” or grammatical variations thereof as used with respect to a non-naturally occurring peptide or variants thereof of the invention and a bacterial isolate refers to bringing the non-naturally occurring peptide (or a variant thereof) and the bacterial isolate in sufficiently close proximity to each other for one to exert a biological effect on the other. In some embodiments, the term contact means binding of the specialized non-naturally occurring peptide to a bacterial isolate. A “therapeutically effective” amount as used herein is an amount that provides some improvement or benefit to the subject. Alternatively stated, a “therapeutically effective” amount is an amount that will provide some alleviation, mitigation, or decrease in at least one clinical symptom in the subject. Those skilled in the art will appreciate that the therapeutic effects need not be complete or curative, as long as some benefit is provided to the subject. By the terms “treat,” “treating,” or “treatment of,” it is intended that the severity of the subject's condition is reduced or at least partially improved or modified and that some alleviation, mitigation or decrease in at least one clinical symptom is achieved. As used herein, a “functional” peptide is one that substantially retains at least one biological activity normally associated with that peptide (e.g. binding to and inhibiting the growth of a bacterium (or killing a bacterium). In particular embodiments, the “functional” peptide substantially retains all of the activities possessed by the unmodified peptide. By “substantially retains” biological activity, it is meant that the peptide retains at least about 20%, 30%, 40%, 50%, 60%, 75%, 85%, 90%, 95%, 97%, 98%, 99%, or more, of the biological activity of the native polypeptide (and can even have a higher level of activity than the native peptide). Biological activities such as protein binding and bacterial inhibitory activity can be measured using assays described herein and other assays that are well known in the art.

As used herein the term “conservative amino acid substitution” refers to the substitution of an amino acid that is normally present in the peptide sequence with a different amino acid of similar size, charge, or polarity. Examples of conservative substitutions include the substitution of a non-polar (hydrophobic) residue such as isoleucine, valine and leucine for another non-polar residue. Likewise, examples of conservative substitutions include the substitution of one polar (hydrophilic) residue for another such as between arginine and lysine, between glutamine and asparagine, and between glycine and serine. Additionally, the substitution of a basic residue such as lysine, arginine or histidine for another, or the substitution of one acidic residue such as aspartic acid or glutamic acid for another acidic residue are additional examples of conservative substitutions. Examples of non-conservative substitutions include the substitution of a non-polar (hydrophobic) amino acid residue such as isoleucine, valine, leucine, alanine, methionine for a polar (hydrophilic) residue such as cysteine, glutamine, glutamic acid or lysine and/or a polar residue for a non-polar residue.

The present invention provides a novel designer antibiotics paradigm that exploits aggregation to inhibit the growth (and/or to kill) pathogenic bacteria by widespread proteostatic collapse. Specifically we identified a number of non-naturally occurring peptides, which contain a short aggregation-nucleating sequence that are redundant in the proteome of E. coli. Analysis of the mode of action of a specific representative member, designated herein further as colpeptin1, revealed that the peptides are rapidly and efficiently internalized by E. coli as 98% of the cells were positive within 15 min. Colpeptin1 uptake readily resulted in the formation of large polar inclusion bodies and bacterial cell death with about 50% cell death in 15 min and more than 80% after 1 h. The fast and lethal aggregation-associated bacterial cell death observed here is surprising as IB formation is generally a non-lethal and reversible response to acute stress in bacteria. Moreover repeated passaging of bacteria on sublethal concentrations (50% of MIC) of the active peptides for a period of 36 days did not result in the development of resistance contrary to the control antibiotic ampicillin. In addition, the peptides are active in vivo, indeed colpeptin1 effectively reduces the bacterial load in a mouse bladder infection model without adverse effects to its host. The present invention provides non-naturally occurring peptides, such as colpeptin1 and variants thereof, and methods for producing variant or similar peptides for treating pathogenic bacteria, such as for combatting difficult to treat pathogenic bacteria. Indeed, our invention shows that it is possible to exploit the sequence specificity of aggregation-prone peptide segments in proteins in bacteria to simultaneously hit several protein targets thereby resulting in fast bacterial cell death. This approach therefore represents an interesting paradigm to develop a novel class of antibiotics.

Accordingly, the invention provides in a first aspect, a non-naturally occurring anti-bacterial peptide configured to induce aggregation of one or more primary target proteins of a bacterium such as to form inclusion bodies comprising said one or more primary target proteins in said bacterium, wherein the one or more primary target proteins of said bacterium include the 3-phenylpropionate-dihydrodiol/cinnamic acid-dihydrodiol dehydrogenase (Hcab) protein.

In certain embodiments, the one or more primary target proteins of said bacterium further include:


- - chaperone protein skp (Skp), phosphate regulon sensor protein
    (PhoR), dipeptide and tripeptide permease A (Dtpa), probable
    sensor-like histidine kinase YedV (YedV), uncharacterized Na(+)/H(+)
    exchanger YjcE (YjcE), osmolarity sensor protein EnvZ (EnvZ), sensor
    protein RstB (RstB), sensor protein ZraS (ZraS), putative
    uncharacterized protein YbfO (YbfO), sensor histidine kinase DcuS
    (DcuS), signal transduction histidine-protein kinase AtoS (AtoS),
    formate hydrogenlyase subunit 4 (hycD), aromatic amino acid exporter
    YddG (YddG), UPF0226 protein YfcJ (YfcJ), and inner membrane protein
    yfeZ (YfeZ);
  - one or more proteins selected from the group consisting of the
    proteins listed in Table 8; and/or
  - one or more proteins selected from the group consisting of the
    proteins listed in Table 9.

In certain embodiments, the peptide induces aggregation of said one or more primary target proteins of said bacterium by co-aggregating with one or more aggregation-prone regions (APRs) in said one or more primary target proteins. In certain embodiments, the amino acid sequence of said one or more APRs is GLGLALV (SEQ ID NO: 128) or displays a single mismatch compared to GLGLALV (SEQ ID NO: 128).

In certain embodiments, the peptide induces aggregation of said one or more primary target proteins of said bacterium by co-aggregating with one or more APRs selected from the group consisting of: GLGLALV (SEQ ID NO: 128), GLGLALA (SEQ ID NO: 202), GLGLAIV (SEQ ID NO: 203), GLGLAMV (SEQ ID NO: 204), GLGLSLV (SEQ ID NO: 205), GLALALV (SEQ ID NO: 206), GLGLAVV (SEQ ID NO: 207), GLPLALV (SEQ ID NO: 208), GVGLALV (SEQ ID NO: 209), GLGLALS (SEQ ID NO: 210), GLLLALV (SEQ ID NO: 211), GLGLALQ (SEQ ID NO: 212), GIGLALV (SEQ ID NO: 213).

In certain embodiments, the bacterium is Gram-negative, preferably Escherichia or Acinetobacter, more preferably Escherichia coli, Acinetobacter radioresistens or Acinetobacter baumanii.

In certain embodiments, the peptide shows Minimum Inhibitory Concentration (MIC) against said bacterium of less than 32 μg/ml, such as 25 μg/mL or less, 12 μg/mL or less, or 6 μg/mL or less.

In certain embodiments, the peptide comprises sequence X1-X2-X3-X4-X5-X6-X7 configured to co-aggregate with said one or more APRs, wherein:


- - X₁ is glycine or a conservative amino acid substitution of glycine
    or a non-natural amino acid variant of glycine or a D-amino acid of
    a conservative amino acid substitution of glycine or a D-amino acid
    of a non-natural amino acid variant of glycine,
  - X₂ is leucine or a conservative amino acid substitution of leucine
    or a non-natural amino acid variant of leucine or a D-amino acid of
    leucine or a D-amino acid of a conservative amino acid substitution
    of leucine or a D-amino acid of a non-natural amino acid variant of
    leucine,
  - X₃ is glycine or a conservative amino acid substitution of glycine
    or a non-natural amino acid variant of glycine or a D-amino acid of
    a conservative amino acid substitution of glycine or a D-amino acid
    of a non-natural amino acid variant of glycine,
  - X₄ is leucine or a conservative amino acid substitution of leucine
    or a non-natural amino acid variant of leucine or a D-amino acid of
    leucine or a D-amino acid of a conservative amino acid substitution
    of leucine or a D-amino acid of a non-natural amino acid variant of
    leucine,
  - X₅ is alanine or a conservative amino acid substitution of alanine
    or a non-natural amino acid variant of alanine or a D-amino acid of
    alanine or a D-amino acid of a conservative amino acid substitution
    of alanine or a D-amino acid of a non-natural amino acid variant of
    alanine,
  - X₆ is leucine or a conservative amino acid substitution of leucine
    or a non-natural amino acid variant of leucine or a D-amino acid of
    leucine or a D-amino acid of a conservative amino acid substitution
    of leucine or a D-amino acid of a non-natural amino acid variant of
    leucine, and
  - X₇ is valine or a conservative amino acid substitution of valine or
    a non-natural amino acid variant of valine or a D-amino acid of
    valine or a D-amino acid of a conservative amino acid substitution
    of valine or a D-amino acid of a non-natural amino acid variant of
    valine.

In certain embodiments, the peptide comprises one or more D-amino acids and/or non-natural amino acids.

In certain embodiments, the peptide comprises the following structure: (A2i-1-X1-X2-X3-X4-X5-X6-X7-A2i-Zi)n, wherein:


- - n is an integer from 1 to 4 and i increases from 1 to n with each
    repeat;
  - each A_(2i-1) and A_(2i) are independently selected from 1 to 3
    contiguous gatekeepers, wherein the amino-terminal gatekeeper in the
    peptide structure is optionally acetylated and/or wherein the
    carboxy-terminal gatekeeper in the peptide sequence is optionally
    amidated,
  - X₁ is glycine or a conservative amino acid substitution of glycine
    or a non-natural amino acid variant of glycine or a D-amino acid of
    a conservative amino acid substitution of glycine or a D-amino acid
    of a non-natural amino acid variant of glycine,
  - X₂ is leucine or a conservative amino acid substitution of leucine
    or a non-natural amino acid variant of leucine or a D-amino acid of
    leucine or a D-amino acid of a conservative amino acid substitution
    of leucine or a D-amino acid of a non-natural amino acid variant of
    leucine,
  - X₃ is glycine or a conservative amino acid substitution of glycine
    or a non-natural amino acid variant of glycine or a D-amino acid of
    a conservative amino acid substitution of glycine or a D-amino acid
    of a non-natural amino acid variant of glycine,
  - X₄ is leucine or a conservative amino acid substitution of leucine
    or a non-natural amino acid variant of leucine or a D-amino acid of
    leucine or a D-amino acid of a conservative amino acid substitution
    of leucine or a D-amino acid of a non-natural amino acid variant of
    leucine,
  - X₅ is alanine or a conservative amino acid substitution of alanine
    or a non-natural amino acid variant of alanine or a D-amino acid of
    alanine or a D-amino acid of a conservative amino acid substitution
    of alanine or a D-amino acid of a non-natural amino acid variant of
    alanine,
  - X₆ is leucine or a conservative amino acid substitution of leucine
    or a non-natural amino acid variant of leucine or a D-amino acid of
    leucine or a D-amino acid of a conservative amino acid substitution
    of leucine or a D-amino acid of a non-natural amino acid variant of
    leucine,
  - X₇ is valine or a conservative amino acid substitution of valine or
    a non-natural amino acid variant of valine or a D-amino acid of
    valine or a D-amino acid of a conservative amino acid substitution
    of valine or a D-amino acid of a non-natural amino acid variant of
    valine,  
    and wherein X₁-X₂-X₃-X₄-X₅-X₆-X₇ and X₇-X₆-X₅-X₄-X₃-X₂-X₁ are used
    interchangeably in the repeats wherein n is 2 to 4,  
    and each Z_(i) is a linker.

Hence, also provided as a related aspect is a peptide comprising the following structure: (A2i-1-X1-X2-X3-X4-X5-X6-X7-A2i-Zi)n, wherein:


- - n is an integer from 1 to 4 and i increases from 1 to n with each
    repeat;
  - each A_(2i-1) and A_(2i) are independently selected from 1 to 3
    contiguous gatekeepers, wherein the amino-terminal gatekeeper in the
    peptide structure is optionally acetylated and/or wherein the
    carboxy-terminal gatekeeper in the peptide sequence is optionally
    amidated,
  - X₁ is glycine or a conservative amino acid substitution of glycine
    or a non-natural amino acid variant of glycine or a D-amino acid of
    a conservative amino acid substitution of glycine or a D-amino acid
    of a non-natural amino acid variant of glycine,
  - X₂ is leucine or a conservative amino acid substitution of leucine
    or a non-natural amino acid variant of leucine or a D-amino acid of
    leucine or a D-amino acid of a conservative amino acid substitution
    of leucine or a D-amino acid of a non-natural amino acid variant of
    leucine,
  - X₃ is glycine or a conservative amino acid substitution of glycine
    or a non-natural amino acid variant of glycine or a D-amino acid of
    a conservative amino acid substitution of glycine or a D-amino acid
    of a non-natural amino acid variant of glycine,
  - X₄ is leucine or a conservative amino acid substitution of leucine
    or a non-natural amino acid variant of leucine or a D-amino acid of
    leucine or a D-amino acid of a conservative amino acid substitution
    of leucine or a D-amino acid of a non-natural amino acid variant of
    leucine,
  - X₅ is alanine or a conservative amino acid substitution of alanine
    or a non-natural amino acid variant of alanine or a D-amino acid of
    alanine or a D-amino acid of a conservative amino acid substitution
    of alanine or a D-amino acid of a non-natural amino acid variant of
    alanine,
  - X₆ is leucine or a conservative amino acid substitution of leucine
    or a non-natural amino acid variant of leucine or a D-amino acid of
    leucine or a D-amino acid of a conservative amino acid substitution
    of leucine or a D-amino acid of a non-natural amino acid variant of
    leucine,
  - X₇ is valine or a conservative amino acid substitution of valine or
    a non-natural amino acid variant of valine or a D-amino acid of
    valine or a D-amino acid of a conservative amino acid substitution
    of valine or a D-amino acid of a non-natural amino acid variant of
    valine,  
    and wherein X₁-X₂-X₃-X₄-X₅-X₆-X₇ and X₇-X₆-X₅-X₄-X₃-X₂-X₁ are used
    interchangeably in the repeats wherein n is 2 to 4,  
    and each Z_(i) is a linker.

In certain embodiments, each A2i-1 and A2i are independently selected from 1 to 3 contiguous gatekeeper amino acids selected from R, K, D, E, P and/or 1 to 3 non-natural gatekeeper amino acids.

In certain embodiments, each Zi is independently selected from stretch of between 1 and 5 units, wherein a unit is PEG, an amino acid or a non-natural amino acid.

In certain embodiments, one or more amino acids of A2i-1, A2i, X1-X2-X3-X4-X5-X6-X7, and/or Zi is a D-amino acid and/or a non-natural amino acid.

Also provided is a peptidomimetic generated from said peptide.

In certain embodiments, the peptide or peptidomimetic further comprises a detectable label.

In certain embodiments, the peptide or peptidomimetic further comprises a molecule which increases the half-life extension.

In certain embodiments, the peptide or peptidomimetic further comprises a moiety that increases solubility of the molecule.

In certain embodiments, the peptide or peptidomimetic displays anti-bacterial effects against more than one bacterial taxon, such as more than one bacterial genus, species or strain.

Further provided are independently:


- - the peptide or peptidomimetic for use as a medicine;
  - the peptide or peptidomimetic for use as an anti-bacterial agent;
    and a corresponding method of treatment of a bacterial infection
    using the peptide or peptidomimetic;
  - the peptide or peptidomimetic for use as a diagnostic agent; and a
    corresponding method of diagnosis using the peptide or
    peptidomimetic; or
  - a pharmaceutical composition comprising the peptide or
    peptidomimetic and a pharmaceutically acceptable carrier.

The invention further provides in a second aspect, a non-naturally occurring anti-bacterial peptide configured to induce aggregation of one or more primary target proteins of a bacterium such as to form inclusion bodies comprising said one or more primary target proteins in said bacterium, wherein the one or more primary target proteins of said bacterium include a protein selected from the proteins listed in Table 4.

In certain embodiments, the peptide induces aggregation of said one or more primary target proteins of said bacterium by co-aggregating with one or more aggregation-protein segments (APRs) in said one or more primary target proteins.

In certain embodiments, the amino acid sequence of said one or more APRs is as listed in Table 4 or displays a single mismatch compared thereto.

In certain embodiments, the bacterium is Gram-negative, preferably Escherichia or Acinetobacter, more preferably Escherichia coli, Acinetobacter radioresistens or Acinetobacter baumanii.

In certain embodiments, the peptide shows Minimum Inhibitory Concentration (MIC) against said bacterium of less than 32 μg/ml, such as 25 μg/mL or less, 12 μg/mL or less, or 6 μg/mL or less.

In certain embodiments, the peptide comprises the following structure: (A2i-1-APR-A2i-Zi)n, wherein:


- - n is an integer from 1 to 4 and i increases from 1 to n with each
    repeat;
  - each A_(2i-1) and A_(2i) are independently selected from 1 to 3
    contiguous gatekeepers, wherein the amino-terminal gatekeeper in the
    peptide structure is optionally acetylated and/or wherein the
    carboxy-terminal gatekeeper in the peptide sequence is optionally
    amidated,
  - the names of the peptides comprised in APR are depicted in Table 5
    (P3, P4, P5, P12, P14, P16, P18, P23, P26, P29, P33, P39, P40, P49,
    P50, P58, P72, P76, P79, P80, P87, P88, P89, P90, P91, P92, P93,
    P99, P101, P103, P105, P111, P112, P113, P114, P115, P116, P117,
    p118, P123, P124 and P125) and the corresponding amino acid
    sequences for these peptides are depicted in Table 4 wherein APR
    comprises natural amino acids or APR comprises conservative amino
    acid substitutions of the amino acids present in APR or APR
    comprises non-natural amino acid analogues of the amino acids
    present in the peptide sequences present in APR or APR comprises
    D-amino acid substitutions in the peptide sequences of the amino
    acids present in APR,
  - and wherein the amino acids in APR can be in a direct or inverted
    repeat wherein n is 2 to 4,
  - and each Z_(i) is a linker.

Hence, also provided as a related aspect is a peptide comprising the following structure: (A2i-1-APR-A2i-Zi)n, wherein:


- - n is an integer from 1 to 4 and i increases from 1 to n with each
    repeat;
  - each A_(2i-1) and A_(2i) are independently selected from 1 to 3
    contiguous gatekeepers, wherein the amino-terminal gatekeeper in the
    peptide structure is optionally acetylated and/or wherein the
    carboxy-terminal gatekeeper in the peptide sequence is optionally
    amidated,
  - the names of the peptides comprised in APR are depicted in Table 5
    (P3, P4, P5, P12, P14, P16, P18, P23, P26, P29, P33, P39, P40, P49,
    P50, P58, P72, P76, P79, P80, P87, P88, P89, P90, P91, P92, P93,
    P99, P101, P103, P105, P111, P112, P113, P114, P115, P116, P117,
    p118, P123, P124 and P125) and the corresponding amino acid
    sequences for these peptides are depicted in Table 4 wherein APR
    comprises natural amino acids or APR comprises conservative amino
    acid substitutions of the amino acids present in APR or APR
    comprises non-natural amino acid analogues of the amino acids
    present in the peptide sequences present in APR or APR comprises
    D-amino acid substitutions in the peptide sequences of the amino
    acids present in APR,
  - and wherein the amino acids in APR can be in a direct or inverted
    repeat wherein n is 2 to 4,
  - and each Z_(i) is a linker.

In certain embodiments, each A2i-1 and A2i are independently selected from 1 to 3 contiguous gatekeeper amino acids selected from R, K, D, E, P and/or 1 to 3 non-natural gatekeeper amino acids.

In certain embodiments, each Zi is independently selected from stretch of between 1 and 5 units, wherein a unit is PEG, an amino acid or a non-natural amino acid.

In certain embodiments, one or more amino acids of A2i-1, A2i, APR, and/or Zi is a D-amino acid and/or a non-natural amino acid.

In certain embodiments, the peptide further comprises a detectable label.

In certain embodiments, the peptide further comprises a molecule which increases the half-life extension.

In certain embodiments, the peptide further comprises a moiety that increases solubility of the molecule.

Further provided are independently:


- - the peptide for use as a medicine;
  - the peptide for use as an anti-bacterial agent, and a corresponding
    method of treatment of a bacterial infection using the peptide;
  - the peptide for use as a diagnostic agent; and a corresponding
    method of diagnosis using the peptide; or
  - a pharmaceutical composition comprising the peptide and a
    pharmaceutically acceptable carrier.

The invention further provides in an embodiment a peptide comprising the following structure: (A2i-1-APR-A2i-Zi)n, wherein:


- - n is an integer from 1 to 4 and i increases from 1 to n with each
    repeat;
  - each A_(2i-1) and A_(2i) are independently selected from 1 to 3
    contiguous gatekeeper amino acids selected from R, K, D, E and P
    and/or 1 to 3 non-natural gatekeeper amino acid selected from
    3-methylproline, 3,4-dehydro-proline,
    2-\[(2S)-2-(hydrazinecarbonyl)pyrrolidin-1-yl\]-2-oxoacetic acid,
    beta-homoproline, alpha-methylproline, hydroxyproline,
    4-oxo-proline, beta,beta-dimethyl-proline, 5,5-dimethyl-proline,
    4-cyclohexyl-proline, 4-phenyl-proline, 3-phenyl-proline,
    4-aminoproline, 4-mercaptoproline, 2-amino-adipic acid (homoglutamic
    acid), 2-amino-heptanedioic acid (2-aminopimelic acid),
    2-amino-octanedioic acid (aminosuberic acid),
    2-amino-4-carboxy-pentanedioic acid (4-carboxyglutamic acid),
    glyoxal-hydroimidazolone, methylglyoxal-hydroimidazolone,
    N-alpha-methyl-arginine, omega-methyl-arginine, norarginine,
    homoarginine, N,N′-diethyl-homoarginine, beta-homoarginine,
    2-amino-3-ureido-propionic acid,
    2-amino-6-(1-carboxyethylamino)hexanoic acid,
    2-amino-6-carboxymethylamino)hexanoic acid,
    2-amino-6-(2-(furan-2-yl)-2-oxoethylamino)hexanoic acid,
    2-amino-6-(formyl-5-hydroxymethyl-pyrrol-1-yl)-hexanoic acid,
    c-alpha-methyl-lysine, beta,beta-dimethyl-lysine,
    N-epsilon-formyl-lysine, N-epsilon-methyl-lysine,
    N-epsilon-i-propyl-lysine, N-epsilon-dimethyl-lysine,
    N-epsilon-trimethylamonium-lysine, N-epsilon-nicotinyl-lysine,
    {\[5-amino-1-(hydrazinecarbonyl)pentyl\]carbamoyl}formic acid,
    N-alpha-methyl-lysine, homolysine, beta-homolysine,
    2-Amino-6-diazo-5-oxocaproic acid, norvaline,
    alpha-methyl-norvaline, Hydroxinorvaline, Ornithine,
    N-delta-methyl-ornithine, N-delta-N-delta-dimethyl-ornithine,
    N-delta-i-propyl-ornithine, c-alpha-methyl-ornithine,
    beta,beta-dimethyl-ornithine, canavanine,
    N-delta-methyl-N-delta-butyl-ornithine,
    N-delta-methyl-N-delta-phenyl-ornithine,
    delta-(2-methylpyrrolidine)-ornithine, delta-piperidyl-ornithine,
    gamma-amino-delta-piperidyl-valeric acid and
    delta-azepanyl-ornithine and wherein the amino-terminal gatekeeper
    amino acid or amino-terminal non-natural gatekeeper amino acid in
    the peptide structure is optionally acetylated and/or wherein the
    carboxy-terminal gatekeeper amino acid or carboxy-terminal
    non-natural gatekeeper amino acid in the peptide sequence is
    optionally amidated;
  - the names of the peptides comprised in APR are depicted in Table 5
    (P2, P3, P4, P5, P12, P14, P16, P18, P23, P26, P29, P33, P39, P40,
    P49, P50, P58, P72, P76, P79, P80, P87, P88, P89, P90, P91, P92,
    P93, P99, P101, P103, P105, P111, P112, P113, P114, P115, P116,
    P117, p118, P123, P124 and P125) and the corresponding amino acid
    sequences for these peptides are depicted in Table 4 wherein APR
    comprises natural amino acids or APR comprises conservative amino
    acid substitutions of the amino acids present in APR or APR
    comprises non-natural amino acid analogues of the amino acids
    present in the peptide sequences present in APR or APR comprises
    D-amino acid substitutions in the peptide sequences of the amino
    acids present in APR, and wherein the amino acids in APR can be in a
    direct or inverted repeat wherein n is 2 to 4;
  - and each Z_(i) is a linker and wherein each Z_(i) is independently
    selected from stretch of between 1 and 5 units, wherein a unit is
    PEG, an amino acid or a non-natural amino acid.

In yet another embodiment the invention provides the sequences depicted in Table 5 (P2, P3, P4, P5, P12, P14, P16, P18, P23, P26, P29, P33, P39, P40, P49, P50, P58, P72, P76, P79, P80, P87, P88, P89, P90, P91, P92, P93, P99, P101, P103, P105, P111, P112, P113, P114, P115, P116, P117, p118, P123, P124 and P125) and the corresponding amino acid sequences for these peptides are depicted in Table 4.

The invention provides in a further embodiment a peptide comprising the following structure: (A2i-1-X1-X2-X3-X4-X5-X6-X7-A2i-Zi)n, wherein:


- - n is an integer from 1 to 4 and i increases from 1 to n with each
    repeat;
  - each A_(2i-1) and A_(2i) are independently selected from 1 to 3
    contiguous gatekeeper amino acids selected from R, K, D, E and P
    and/or 1 to 3 non-natural gatekeeper amino acid selected from
    3-methylproline, 3,4-dehydro-proline,
    2-\[(2S)-2-(hydrazinecarbonyl)pyrrolidin-1-yl\]-2-oxoacetic acid,
    beta-homoproline, alpha-methylproline, hydroxyproline,
    4-oxo-proline, beta,beta-dimethyl-proline, 5,5-dimethyl-proline,
    4-cyclohexyl-proline, 4-phenyl-proline, 3-phenyl-proline,
    4-aminoproline, 4-mercaptoproline, 2-amino-adipic acid (homoglutamic
    acid), 2-amino-heptanedioic acid (2-aminopimelic acid),
    2-amino-octanedioic acid (aminosuberic acid),
    2-amino-4-carboxy-pentanedioic acid (4-carboxyglutamic acid),
    glyoxal-hydroimidazolone, methylglyoxal-hydroimidazolone,
    N-alpha-methyl-arginine, omega-methyl-arginine, norarginine,
    homoarginine, N,N′-diethyl-homoarginine, beta-homoarginine,
    2-amino-3-ureido-propionic acid,
    2-amino-6-(1-carboxyethylamino)hexanoic acid,
    2-amino-6-carboxymethylamino)hexanoic acid,
    2-amino-6-(2-(furan-2-yl)-2-oxoethylamino)hexanoic acid,
    2-amino-6-(formyl-5-hydroxymethyl-pyrrol-1-yl)-hexanoic acid,
    c-alpha-methyl-lysine, beta,beta-dimethyl-lysine,
    N-epsilon-formyl-lysine, N-epsilon-methyl-lysine,
    N-epsilon-i-propyl-lysine, N-epsilon-dimethyl-lysine,
    N-epsilon-trimethylamonium-lysine, N-epsilon-nicotinyl-lysine,
    {\[5-amino-1-(hydrazinecarbonyl)pentyl\]carbamoyl}formic acid,
    N-alpha-methyl-lysine, homolysine, beta-homolysine,
    2-Amino-6-diazo-5-oxocaproic acid, norvaline,
    alpha-methyl-norvaline, Hydroxinorvaline, Ornithine,
    N-delta-methyl-ornithine, N-delta-N-delta-dimethyl-ornithine,
    N-delta-i-propyl-ornithine,
    c-alpha-methyl-ornithine,beta,beta-dimethyl-ornithine,canavanine,N-delta-methyl-N-delta-butyl-ornithine,
    N-delta-methyl-N-delta-phenyl-ornithine,
    delta-(2-methylpyrrolidine)-ornithine, delta-piperidyl-ornithine,
    gamma-amino-delta-piperidyl-valeric acid and
    delta-azepanyl-ornithine and wherein the amino-terminal gatekeeper
    amino acid or amino-terminal non-natural gatekeeper amino acid in
    the peptide structure is optionally acetylated and/or wherein the
    carboxy-terminal gatekeeper amino acid or carboxy-terminal
    non-natural gatekeeper amino acid in the peptide sequence is
    optionally amidated,
  - X₁ is glycine or a conservative amino acid substitution of glycine
    or a non-natural amino acid variant of glycine or a D-amino acid of
    a conservative amino acid substitution of glycine or a D-amino acid
    of a non-natural amino acid variant of glycine,
  - X₂ is leucine or a conservative amino acid substitution of leucine
    or a non-natural amino acid variant of leucine or a D-amino acid of
    leucine or a D-amino acid of a conservative amino acid substitution
    of leucine or a D-amino acid of a non-natural amino acid variant of
    leucine,
  - X₃ is glycine or a conservative amino acid substitution of glycine
    or a non-natural amino acid variant of glycine or a D-amino acid of
    a conservative amino acid substitution of glycine or a D-amino acid
    of a non-natural amino acid variant of glycine,
  - X₄ is leucine or a conservative amino acid substitution of leucine
    or a non-natural amino acid variant of leucine or a D-amino acid of
    leucine or a D-amino acid of a conservative amino acid substitution
    of leucine or a D-amino acid of a non-natural amino acid variant of
    leucine,
  - X₅ is alanine or a conservative amino acid substitution of alanine
    or a non-natural amino acid variant of alanine or a D-amino acid of
    alanine or a D-amino acid of a conservative amino acid substitution
    of alanine or a D-amino acid of a non-natural amino acid variant of
    alanine,
  - X₆ is leucine or a conservative amino acid substitution of leucine
    or a non-natural amino acid variant of leucine or a D-amino acid of
    leucine or a D-amino acid of a conservative amino acid substitution
    of leucine or a D-amino acid of a non-natural amino acid variant of
    leucine,
  - X₇ is valine or a conservative amino acid substitution of valine or
    a non-natural amino acid variant of valine or a D-amino acid of
    valine or a D-amino acid of a conservative amino acid substitution
    of valine or a D-amino acid of a non-natural amino acid variant of
    valine,  
    and wherein X₁-X₂-X₃-X₄-X₅-X₆-X₇ and X₇-X₆-X₅-X₄-X₃-X₂-X₁ are used
    interchangeably in the repeats wherein n is 2 to 4,  
    and each Z_(i) is a linker and wherein each Z_(i) is independently
    selected from stretch of between 1 and 5 units, wherein a unit is
    PEG, an amino acid or a non-natural amino acid.

The nature of the linker moieties, Zi, in the peptides of the invention is not vital to the invention, although long flexible linkers are preferably not used. According to particular embodiments, each Zi is independently selected from a stretch of between 0 and 20 identical or non-identical units, wherein a unit is an amino acid, a monosaccharide, a nucleotide or a monomer. Non-identical units can be non-identical units of the same nature (e.g. different amino acids, or some copolymers). They can also be non-identical units of a different nature, e.g. a linker with amino acid and nucleotide units, or a heteropolymer (copolymer) comprising two or more different monomeric species. According to particular embodiments, the length of at least one, and particularly each Zi other than Zn, is at least 1 unit. According to other particular embodiments, Zn is 0 units. According to particular embodiments, all Zi linkers other than Zn are identical. According to further embodiments, all Zi moieties are identical.

According to specific embodiments, at least one, and particularly all, Zi are of between 0 and 10 units of the same nature, particularly between 0 and 5 units of the same nature. According to particular embodiments, at least one Zi moiety, and particularly all Zi moieties except Zn, is a peptide or polypeptide linker. Particularly envisaged sequences of such linkers include, but are not limited to, PPP, PP or GS. The linker can also be of a chemical nature. Particularly envisaged chemical linkers include PEG and Ttds (aka 4,7,10-trioxatridecan-13-succinamic acid).

Typically, long linkers are not used. However, according to the particular embodiments where the aggregation-promoting moieties correspond to aggregation-inducing regions of more than one protein, it is envisaged that long linkers may be used. Indeed, to ensure that the molecule can (e.g. simultaneously) interact with more than one protein, it may be beneficial to increase the distance between the different targeting aggregation-promoting moieties, so that the interaction is not prevented due to steric hindrance. In these instances, the Zi linker may be a stretch of between 0 and 100 identical or non-identical units, wherein a unit is an amino acid, a monosaccharide, a nucleotide or a monomer; or of between 0 and 90, 0 and 80, 0 and 70, 0 and 60, 0 and 50, 0 and 40, 0 and 30 or 0 and 20. Particularly, the minimal length of the Zi linker is at least 1 unit, at least 2 units, at least 3 units, at least 4 units, or at least 5 units.

In yet another embodiment the invention provides a peptide comprising the following structure: (A2i-1-X1-X2-X3-X4-X5-X6-X7-A2i-Zi)n, wherein:


- - n is an integer from 1 to 4 and i increases from 1 to n with each
    repeat;
  - each A_(2i-1) and A_(2i) are independently selected from 1 to 3
    contiguous gatekeeper amino acids selected from R, K, D, E and P
    and/or wherein the amino-terminal gatekeeper amino acid is
    optionally acetylated and/or wherein the carboxy-terminal gatekeeper
    amino acid in the peptide sequence is optionally amidated,
  - X₁ is glycine or a conservative amino acid substitution of glycine
    or a non-natural amino acid variant of glycine or a D-amino acid of
    a conservative amino acid substitution of glycine or a D-amino acid
    of a non-natural amino acid variant of glycine,
  - X₂ is leucine or a conservative amino acid substitution of leucine
    or a non-natural amino acid variant of leucine or a D-amino acid of
    leucine or a D-amino acid of a conservative amino acid substitution
    of leucine or a D-amino acid of a non-natural amino acid variant of
    leucine,
  - X₃ is glycine or a conservative amino acid substitution of glycine
    or a non-natural amino acid variant of glycine or a D-amino acid of
    a conservative amino acid substitution of glycine or a D-amino acid
    of a non-natural amino acid variant of glycine,
  - X₄ is leucine or a conservative amino acid substitution of leucine
    or a non-natural amino acid variant of leucine or a D-amino acid of
    leucine or a D-amino acid of a conservative amino acid substitution
    of leucine or a D-amino acid of a non-natural amino acid variant of
    leucine,
  - X₅ is alanine or a conservative amino acid substitution of alanine
    or a non-natural amino acid variant of alanine or a D-amino acid of
    alanine or a D-amino acid of a conservative amino acid substitution
    of alanine or a D-amino acid of a non-natural amino acid variant of
    alanine,
  - X₆ is leucine or a conservative amino acid substitution of leucine
    or a non-natural amino acid variant of leucine or a D-amino acid of
    leucine or a D-amino acid of a conservative amino acid substitution
    of leucine or a D-amino acid of a non-natural amino acid variant of
    leucine,
  - X₇ is valine or a conservative amino acid substitution of valine or
    a non-natural amino acid variant of valine or a D-amino acid of
    valine or a D-amino acid of a conservative amino acid substitution
    of valine or a D-amino acid of a non-natural amino acid variant of
    valine,  
    and wherein X₁-X₂-X₃-X₄-X₅-X₆-X₇ and X₇-X₆-X₅-X₄-X₃-X₂-X₁ are used
    interchangeably in the repeats wherein n is 2 to 4,  
    and each Z_(i) is a linker and wherein each Z_(i) is independently
    selected from stretch of between 1 and 5 units, wherein a unit is
    PEG, an amino acid or a non-natural amino acid.

In yet another embodiment the invention provides a peptide comprising the following structure: (A2i-1-X1-X2-X3-X4-X5-X6-X7-A2i-Zi)n, wherein:


- - n is an integer from 1 to 3 and i increases from 1 to n with each
    repeat;
  - each A_(2i-1) and A_(2i) are independently selected from 1 to 3
    contiguous gatekeeper amino acids selected from R, K, D, E and P and
    wherein the amino-terminal gatekeeper amino acid is optionally
    acetylated and/or wherein the carboxy-terminal gatekeeper amino acid
    in the peptide sequence is optionally amidated,
  - X₁ is glycine or a conservative amino acid substitution of glycine
    or a non-natural amino acid variant of glycine or a D-amino acid of
    a conservative amino acid substitution of glycine or a D-amino acid
    of a non-natural amino acid variant of glycine,
  - X₂ is leucine or a conservative amino acid substitution of leucine
    or a non-natural amino acid variant of leucine or a D-amino acid of
    leucine or a D-amino acid of a conservative amino acid substitution
    of leucine or a D-amino acid of a non-natural amino acid variant of
    leucine,
  - X₃ is glycine or a conservative amino acid substitution of glycine
    or a non-natural amino acid variant of glycine or a D-amino acid of
    a conservative amino acid substitution of glycine or a D-amino acid
    of a non-natural amino acid variant of glycine,
  - X₄ is leucine or a conservative amino acid substitution of leucine
    or a non-natural amino acid variant of leucine or a D-amino acid of
    leucine or a D-amino acid of a conservative amino acid substitution
    of leucine or a D-amino acid of a non-natural amino acid variant of
    leucine,
  - X₅ is alanine or a conservative amino acid substitution of alanine
    or a non-natural amino acid variant of alanine or a D-amino acid of
    alanine or a D-amino acid of a conservative amino acid substitution
    of alanine or a D-amino acid of a non-natural amino acid variant of
    alanine,
  - X₆ is leucine or a conservative amino acid substitution of leucine
    or a non-natural amino acid variant of leucine or a D-amino acid of
    leucine or a D-amino acid of a conservative amino acid substitution
    of leucine or a D-amino acid of a non-natural amino acid variant of
    leucine,
  - X₇ is valine or a conservative amino acid substitution of valine or
    a non-natural amino acid variant of valine or a D-amino acid of
    valine or a D-amino acid of a conservative amino acid substitution
    of valine or a D-amino acid of a non-natural amino acid variant of
    valine,  
    and wherein X₁-X₂-X₃-X₄-X₅-X₆-X₇ and X₇-X₆-X₅-X₄-X₃-X₂-X₁ are used
    interchangeably in the repeats wherein n is 2 to 4,  
    and each Z_(i) is a linker and wherein each Z_(i) is independently
    selected from stretch of between 1 and 5 units, wherein a unit is
    PEG, an amino acid or a non-natural amino acid.

In yet another embodiment the invention provides a peptide comprising the following structure: A2i-1-X1-X2-X3-X4-X5-X6-X7-A2i-Zi, wherein:


- - each A_(2i-1) and A_(2i) are independently selected from 1 to 3
    contiguous gatekeeper amino acids selected from R, K, D, E and P and
    wherein the amino-terminal gatekeeper amino acid is optionally
    acetylated and/or wherein the carboxy-terminal gatekeeper amino acid
    in the peptide sequence is optionally amidated,
  - X₁ is glycine or a conservative amino acid substitution of glycine
    or a non-natural amino acid variant of glycine or a D-amino acid of
    a conservative amino acid substitution of glycine or a D-amino acid
    of a non-natural amino acid variant of glycine,
  - X₂ is leucine or a conservative amino acid substitution of leucine
    or a non-natural amino acid variant of leucine or a D-amino acid of
    leucine or a D-amino acid of a conservative amino acid substitution
    of leucine or a D-amino acid of a non-natural amino acid variant of
    leucine,
  - X₃ is glycine or a conservative amino acid substitution of glycine
    or a non-natural amino acid variant of glycine or a D-amino acid of
    a conservative amino acid substitution of glycine or a D-amino acid
    of a non-natural amino acid variant of glycine,
  - X₄ is leucine or a conservative amino acid substitution of leucine
    or a non-natural amino acid variant of leucine or a D-amino acid of
    leucine or a D-amino acid of a conservative amino acid substitution
    of leucine or a D-amino acid of a non-natural amino acid variant of
    leucine,
  - X₅ is alanine or a conservative amino acid substitution of alanine
    or a non-natural amino acid variant of alanine or a D-amino acid of
    alanine or a D-amino acid of a conservative amino acid substitution
    of alanine or a D-amino acid of a non-natural amino acid variant of
    alanine,
  - X₆ is leucine or a conservative amino acid substitution of leucine
    or a non-natural amino acid variant of leucine or a D-amino acid of
    leucine or a D-amino acid of a conservative amino acid substitution
    of leucine or a D-amino acid of a non-natural amino acid variant of
    leucine,
  - X₇ is valine or a conservative amino acid substitution of valine or
    a non-natural amino acid variant of valine or a D-amino acid of
    valine or a D-amino acid of a conservative amino acid substitution
    of valine or a D-amino acid of a non-natural amino acid variant of
    valine,  
    and each Z_(i) is a linker and wherein each Z_(i) is independently
    selected from stretch of between 1 and 5 units, wherein a unit is
    PEG, an amino acid or a non-natural amino acid.

In yet another embodiment the invention provides a peptide comprising the following structure: (A2i-1-X1-X2-X3-X4-X5-X6-X7-A2i-Zi)n, wherein:


- - n is an integer from 1 to 2 and i increases from 1 to n with each
    repeat;
  - each A2i-1 and A2i are independently selected from 1 to 3 contiguous
    gatekeeper amino acids selected from R, K, D, E and P and wherein
    the amino-terminal gatekeeper amino acid is optionally acetylated
    and/or wherein the carboxy-terminal gatekeeper amino acid in the
    peptide sequence is optionally amidated,
  - X1 is glycine or a conservative amino acid substitution of glycine
    or a non-natural amino acid variant of glycine or a D-amino acid of
    a conservative amino acid substitution of glycine or a D-amino acid
    of a non-natural amino acid variant of glycine,
  - X2 is leucine or a conservative amino acid substitution of leucine
    or a non-natural amino acid variant of leucine or a D-amino acid of
    leucine or a D-amino acid of a conservative amino acid substitution
    of leucine or a D-amino acid of a non-natural amino acid variant of
    leucine,
  - X3 is glycine or a conservative amino acid substitution of glycine
    or a non-natural amino acid variant of glycine or a D-amino acid of
    a conservative amino acid substitution of glycine or a D-amino acid
    of a non-natural amino acid variant of glycine,
  - X4 is leucine or a conservative amino acid substitution of leucine
    or a non-natural amino acid variant of leucine or a D-amino acid of
    leucine or a D-amino acid of a conservative amino acid substitution
    of leucine or a D-amino acid of a non-natural amino acid variant of
    leucine,
  - X5 is alanine or a conservative amino acid substitution of alanine
    or a non-natural amino acid variant of alanine or a D-amino acid of
    alanine or a D-amino acid of a conservative amino acid substitution
    of alanine or a D-amino acid of a non-natural amino acid variant of
    alanine,
  - X6 is leucine or a conservative amino acid substitution of leucine
    or a non-natural amino acid variant of leucine or a D-amino acid of
    leucine or a D-amino acid of a conservative amino acid substitution
    of leucine or a D-amino acid of a non-natural amino acid variant of
    leucine,
  - X7 is valine or a conservative amino acid substitution of valine or
    a non-natural amino acid variant of valine or a D-amino acid of
    valine or a D-amino acid of a conservative amino acid substitution
    of valine or a D-amino acid of a non-natural amino acid variant of
    valine,  
    and wherein X₁-X₂-X₃-X₄-X₅-X₆-X₇ and X₇-X₆-X₅-X₄-X₃-X₂-X₁ are used
    interchangeably in the repeats wherein n is 2,  
    and each Z_(i) is a linker and wherein each Z_(i) is independently
    selected from stretch of between 1 and 5 units, wherein a unit is
    PEG, an amino acid or a non-natural amino acid.

In yet another embodiment the invention provides Colpeptin1 (P2) which sequence is depicted in SEQ ID NO: 1.

SEQ ID NO: 1: amino-RGLGLALVRRPRGLGLALVRR-carboxyl

In yet another embodiment X1 and X3 are selected from glycine or a non-natural variant of glycine selected from the list consisting of N-alpha-methyl-glycine (sarcosine), cyclopropylglycine and cyclopentylglycine, X2, X4 and X6 are selected from leucine or valine or a non-natural variant of leucine selected from the list consisting of 2-amino-3,3-dimethyl-butyric acid (t-Leucine), alpha-methylleucine, hydroxyleucine, 2,3-dehydro-leucine, N-alpha-methyl-leucine, 2-Amino-5-methyl-hexanoic acid (homoleucine), 3-Amino-5-methylhexanoic acid (beta-homoleucine), 2-Amino-4,4-dimethyl-pentanoic acid (4-methyl-leucine, neopentylglycine), 4,5-dehydro-norleucine (allylglycine), L-norleucine, N-alpha-methyl-norleucine and 6-hydroxy-norleucine, X5 is selected from alanine or a non-natural variant of alanine selected from the list consisting of 2-amino-isobutyric acid (2-Methylalanine), 2-Amino-2-methylbutanoic acid (Isovaline), N-alpha-Methyl-alanine, 2-Amino-2-methylpent-4-enoic acid (alpha-allylalanine), beta-homoalanine, 2-indanyl-glycine, Di-n-propyl-glycine, Di-n-butyl-glycine, Diethyl-glycine, (1-naphthyl)alanine, (2-naphthyl)alanine, cyclohexylglycine, adamantyl-glycine, beta-homoallylglycine and X7 is selected from valine or leucine or a non-natural variant of leucine selected from the list consisting of c-alpha-methyl-valine (2,3-dimethylbutanoic acid), 2,3-dehydro-valine, 3,4-dehydro-valine, 3-methyl-L-isovaline (methylvaline), 2-amino-3-hydroxy-3-methylbutanoic acid (hydroxyvaline), beta-homovaline and N-alpha-methyl-valine.

The term “peptides of the invention” is conveniently used herein to encompass any peptides and peptide variants embodying the principles of the invention as disclosed herein, such as without limitation peptides based on coleptin1 and variants thereof.

Specifically alanine analogues which can be used to develop variants of the peptides of the invention, such as variant peptides of SEQ ID NO: 1, by replacing at least one alanine in the structures of the peptides of the invention, such as in the structure of SEQ ID NO: 1, for an analogue of alanine are depicted in Table I:

Specifically arginine analogues which can be used to develop variants of the peptides of the invention, such as variants of SEQ ID NO: 1, by replacing at least one arginine in the structure in the peptides, such as in the structure of SEQ ID NO: 1, for an analogue of arginine are depicted in Table II:

Specifically glycine analogues which can be used to develop variants of the peptides of the invention, such as variants of SEQ ID NO: 1, by replacing at least one glycine in the structure of the peptides, such as in the structure of SEQ ID NO: 1, for an analogue of glycine are depicted in Table III:

In certain embodiments, specifically glycine analogues which can be used to develop variants of the peptides of the invention, such as variants of SEQ ID NO: 1, by replacing at least one glycine in the structure of the peptides, such as in the structure of SEQ ID NO: 1, may include beta-alanine or the analogues depicted in Table III, and may preferably be beta-alanine.

Specifically leucine analogues which can be used to develop variants of the peptides of the invention, such as variants of SEQ ID NO: 1, by replacing at least one leucine in the structure of the peptides, such as in the structure of SEQ ID NO: 1, for an analogue of leucine are depicted in Table IV:

5 In certain embodiments, specifically leucine analogues which can be used to develop variants of the peptides of the invention, such as variants of SEQ ID NO: 1, by replacing at least one leucine in the structure of the peptides, such as in the structure of SEQ ID NO: 1, for an analogue of leucine are depicted below:

In certain embodiments, specifically leucine analogues which can be used to develop variants of the peptides of the invention, such as variants of SEQ ID NO: 1, by replacing at least one leucine in the structure of the peptides, such as in the structure of SEQ ID NO: 1, for an analogue of leucine may be norvaline or alpha-methyl-norvaline.

Specifically valine analogues which can be used to develop variants of the peptides of the invention, such as variants of SEQ ID NO: 1, by replacing at least one valine in the structure of the peptides, such as in the structure of SEQ ID NO: 1, for an analogue of valine are depicted in Table V:

Specifically lysine analogues which can be used to develop peptides of the invention by replacing the gatekeeper residue lysine (K) for an analogue of lysine (K) are depicted in Table VI:

In certain embodiments, specifically lysine analogues which can be used to develop peptides of the invention by replacing the gatekeeper residue lysine (K) for an analogue of lysine (K) may be N-epsilon-formyl-lysine, N-epsilon-methyl-lysine, N-epsilon-i-propyl-lysine, N-epsilon-dimethyl-lysine, N-epsilon-trimethylamonium-lysine, N-epsilon-nicotinyl-lysine, {[5-amino-1-(hydrazinecarbonyl)pentyl]carbamoyl}formic acid, N-alpha-methyl-lysine, homolysine, beta-homolysine, 2-Amino-6-diazo-5-oxocaproic acid, Hydroxinorvaline, Ornithine, N-delta-methyl-ornithine, N-delta-N-delta-dimethyl-ornithine, N-delta-i-propyl-ornithine, c-alpha-methyl-ornithine, beta,beta-dimethyl-ornithine, canavanine, N-delta-methyl-N-delta-butyl-ornithine, N-delta-methyl-N-delta-phenyl-ornithine, delta-(2-methylpyrrolidine)-ornithine, delta-piperidyl-ornithine, gamma-amino-delta-piperidyl-valeric acid or delta-azepanyl-ornithine.

Specifically proline analogues which can be used to develop variants of the peptides of the invention, such as variants of SEQ ID NO: 1, by replacing of proline in the gatekeepers as depicted in claim 1 for an artificial proline (P) are depicted in Table VII:

In yet another embodiment the linker, Zi, in the peptides of the invention consists of proline, 4-hydroxyproline, (2R, 5S)-5-phenyl-pyrrolidine-2-carboxylic acid, 3,4-dehydro-L-proline, beta-(2-benzothiazolyl)-alanine, 3-(2-furyl)-alanine or beta-(2-thienyl)-alanine.

In yet another embodiment the invention provides a cyclic peptide which is produced by forming a ring structure between an amino-terminal and carboxy-terminal non-natural gatekeeper residue (A2i-1 and A2i).

In yet another embodiment the peptides of the invention further comprise a detectable label.

The present invention also includes isotopically labelled peptides, which are identical to those defined herein, but for the fact that one or more atoms are replaced by an atom having an atomic mass or mass number different from the atomic mass or mass number usually found in nature. Examples of isotopes that may be incorporated into peptides of the present invention include isotopes of hydrogen, carbon, nitrogen, oxygen, phosphorous, sulphur, fluorine and chlorine, such as 2H, 3H, 13C, 11C, 14C, 15N, 18O, 17O, 31P, 32P, 35S, 18F, and 36Cl, respectively. Peptides of the present invention and pharmaceutically acceptable salts of said peptides or which contain the aforementioned isotopes and/or other isotopes of other atoms are within the scope of this invention. Certain isotopically labeled peptides of the present invention, for example those into which radioactive isotopes such as 3H and 14C are incorporated, are useful in drug and/or substrate tissue distribution assays. Tritiated, i.e. 3H, and carbon-14, i.e., 14C, isotopes are particularly preferred for their ease of preparation and detectability. Further, substitution with heavier isotopes such as deuterium, i.e., 2H, may afford certain therapeutic advantages resulting from greater metabolic stability, for example increased in vivo half-life or reduced dosage requirements and, hence, may be preferred in some circumstances, Isotopically labelled peptides of formula I of this invention may generally be prepared by carrying out the procedures disclosed in the Examples below, by substituting a readily available isotopically labelled reagent for a non-isotopically labelled reagent.

In yet another embodiment the peptides of the invention further comprise a molecule which increases the half-life extension.

In yet another embodiment the peptides of the invention further comprise a moiety that increases the solubility of the peptides.

According to other particular embodiments, the molecules may be fused to other moieties, e.g. to extend their half-life in vivo. Apart from increasing stability, such moieties may also increase solubility of the molecule they are fused to. Although the presence of gatekeepers (the numbered X moieties) is in principle sufficient to prevent premature aggregation of the molecules and keep them in solution, the further addition of a moiety that increases solubility (i.e. prevents aggregation) may provide easier handling of the molecules, and particularly improve stability and shelf-life. A well-known example of such moiety is PEG (polyethylene glycol). This moiety is particularly envisaged, as it can be used as linker as well as solubilizing moiety. Other examples include peptides and proteins or protein domains, or even whole proteins (e.g. GFP). In this regard, it should be noted that, like PEG, one moiety can have different functions or effects. For instance, a flag tag (sequence DYKDDDDK) is a peptide moiety that can be used as a label, but due to its charge density, it will also enhance solubilisation. PEGylation has already often been demonstrated to increase solubility of biopharmaceuticals (e.g. Veronese and Mero, BioDrugs. 2008; 22(5):315-29). Adding a peptide, polypeptide, protein or protein domain tag to a molecule of interest has been extensively described in the art. Examples include, but are not limited to, peptides derived from synuclein (e.g. Park et al., Protein Eng. Des. Sel. 2004; 17:251-260), SET (solubility enhancing tag, Zhang et al., Protein Expr Purif 2004; 36:207-216), thioredoxin (TRX), Glutathione-S-transferase (GST), Maltose-binding protein (MBP), N-Utilization substance (NusA), small ubiquitin-like modifier (SUMO), ubiquitin (Ub), disulfide bond C (DsbC), Seventeen kilodalton protein (Skp), Phage T7 protein kinase fragment (T7PK), Protein G B1 domain, Protein A IgG ZZ repeat domain, and bacterial immunoglobulin binding domains (Hutt et al., J Biol Chem.; 287(7):4462-9, 2012). The nature of the tag will depend on the application, as can be determined by the skilled person. For instance, for transgenic expression of the molecules described herein, it might be envisaged to fuse the molecules to a larger domain to prevent premature degradation by the cellular machinery. Other applications may envisage fusion to a smaller solubilisation tag (e.g. less than 30 amino acids, or less than 20 amino acids, or even less than 10 amino acids) in order not to alter the properties of the molecules too much.

Apart from extending half-life, molecules may be fused to moieties that alter other or additional pharmacokinetic and pharmacodynamic properties. For instance, it is known that fusion with albumin (e.g. human serum albumin), albumin-binding domain or a synthetic albumin-binding peptide improves pharmacokinetics and pharmacodynamics of different therapeutic proteins (Langenheim and Chen, Endocrinol.; 203(3):375-87, 2009). Another moiety that is often used is a fragment crystallizable region (Fc) of an antibody. The nature of these moieties is not vital to the invention and can be determined by the person skilled in the art depending on the application.

In yet another embodiment the peptides of the invention further comprising at least one D-alanine at the amino-terminus and/or the carboxy-terminus of said peptides.

In yet another embodiment the invention provides the peptides of the invention for use as a medicament.

In yet another embodiment the invention provides the peptides of the invention for use as an anti-bacterial agent.

In yet another embodiment the invention provides the peptides of the invention for use to treat gram-positive bacteria.

In yet another embodiment the invention provides the peptides of the invention for use to treat gram-negative bacteria.

In yet another embodiment the invention provides the peptides of the invention for use to treat drug-resistant bacterial strains.

In yet another embodiment the invention provides the peptides of the invention for use to treat multidrug-resistant bacterial strains.

In yet another embodiment the invention provides the peptides of the invention for use to treat multidrug-resistant bacterial strains selected from the list comprising of carbapenem-resistant Enterobacteriaceae, drug-resistant Neisseria gonorrhoeae, multidrug resistant Acetinobacter, drug-resistant Campylobacter, extended spectrum bata-lactamase producing Enterobacteriaceae, multidrug-resistant Pseudomonas aeruginosa, Drug-resistant non-typhoidal Salmonella, Drug-resistant Salmonella typhi and Drug-resistant Shigella.

In yet another embodiment the invention provides the use of the peptides of the invention for use as a diagnostic agent.

In yet another embodiment the invention provides a pharmaceutical composition comprising the peptides of the invention and a pharmaceutically acceptable carrier.

In yet another embodiment the invention provides a method to produce an anti-bacterial peptide comprising the following steps:


- - i) generating an in silico list of aggregation prone regions (APRs)
    with a length of 5-14 amino acids, such as 5-12 amino acids, or 6-14
    amino acids, such as 6-12 amino acids which APRs are identified in a
    bacterial proteome,
  - ii) synthetizing a number of 20-200 different peptides comprising an
    APR based on the following structure:
    (A_(2i-1)-APR-A_(2i)-Z_(i))_(n), wherein n is an integer from 1 to 4
    and i increases from 1 to n with each repeat and each A_(2i-1) and
    A_(2i) are independently selected from 1 to 3 contiguous gatekeeper
    amino acids selected from R, K, D, E and P,
  - iii) testing said peptides for an anti-bacterial effect and
    producing an anti-bacterial peptide.

One particularly convenient way of identifying such sequences in a protein, in particular identifying APR sequences in proteins present in a proteome, is by using a beta-aggregation-predicting algorithm. Such algorithms may typically take into account biophysical parameters. Tango and Zyggregator are common examples of such algorithms, but many more have been described in the art, including, but not limited to those described by Bryan et al., PLoS Comput Biol. 5(3):e1000333, 2009; Caflish, Curr Opin Chem Biol. 10(5):437-44, 2006; Conchillo-Sole et al., BMC Bioinformatics 8:65, 2007; Galzitskaya et al., PLoS Comput Biol. 29; 2(12):e177, 2006; Goldschmidt et al., PNAS 107(8):3487-92, 2010; Maurer-Stroh et al., Nat Methods 7(3):237-42, 2010; Rojas Quijano et al., Biochemistry 45(14):4638-52, 2006; Saiki et al., Biochem Biophys Res Commun 343(4):1262-71, 2006; Sanchez de Groot et al., BMC Struct Biol 5:18, 2005; Tartaglia et al., Protein Sci. 14(10):2723-34, 2005; Tartaglia et al., J Mol Biol. 380(2):425-36, 2008; Thompson et al., PNAS 103(11):4074-8, 2006; Trovato et al., Protein Eng Des Sel. 20(10):521-3, 2007; Yoon and Welsh, Protein Sci. 13(8):2149-60, 2004; Zibaee et al., Protein Sci. 16(5):906-18, 2007. Note that many of these are primarily involved with amyloid aggregating sequences and not just with amorphous beta-aggregation. The sequence space of both forms of aggregation can overlap (Rousseau et al., Current Opinion in Structural Biology 16:118-126, 2006), and both forms of aggregation are envisaged, as long as the kinetics and conditions of the reaction favour aggregation of the protein(s) of interest.

In some embodiments, the peptides of the invention may comprise one or more additional residues at the amino- and/or carboxyl-terminal ends. In some embodiments, the one or more additional residues are D-alanines. For example, a peptide may comprise one or two D-alanines at the amino- and/or carboxyl-terminal ends.

Likewise, those skilled in the art will appreciate that the present invention also encompasses fusion polypeptides comprising a specialized non-naturally occurring peptides. As an alternative, the fusion protein can comprise a reporter molecule. In other embodiments, the fusion protein can comprise a polypeptide that provides a function or activity that is the same as or different from the activity of the peptide, e.g. a targeting, binding, or enzymatic activity or function. Likewise, it will be understood that the peptides specifically disclosed herein will typically tolerate substitutions in the amino acid sequence and substantially retain biological activity. To identify peptides of the invention other than those specifically disclosed herein, amino acid substitutions may be based on any characteristic known in the art, including the relative similarity or differences of the amino acid side-chain substituents, for example, their hydrophobicity, hydrophilicity, charge, size, and the like.

As is known in the art, a number of different programs can be used to identify whether a polypeptide has sequence identity or similarity to a known sequence. Sequence identity or similarity may be determined using standard techniques known in the art, including, but not limited to, the local sequence identity algorithm of Smith & Waterman, Adv. Appl. Math. 2:482 (1981), by the sequence identity alignment algorithm of Needleman & Wunsch, J. Mol. Biol. 48:443 (1970), by the search for similarity method of Pearson & Lipman, Proc. Natl. Acad. Sci. USA 85:2444 (1988), by computerized implementations of these algorithms (GAP, BESTFIT, FASTA, and TFASTA in the Wisconsin Genetics Software Package, Genetics Computer Group, 575 Science Drive, Madison, Wis.), the Best Fit sequence program described by Devereux et al., Nucl. Acid Res. 12:387 (1984), preferably using the default settings, or by inspection. An example of a useful algorithm is PILEUP. PILEUP creates a multiple sequence alignment from a group of related sequences using progressive, pairwise alignments. It can also plot a tree showing the clustering relationships used to create the alignment. PILEUP uses a simplification of the progressive alignment method of Feng & Doolittle, J. Mol. Evol. 35:351 (1987); the method is similar to that described by Higgins & Sharp, CABIOS 5:151 (1989). Another example of a useful algorithm is the BLAST algorithm, described in Altschul et al., J. Mol. Biol. 215:403 (1990) and Karlin et al., Proc. Natl. Acad. Sci. USA 90:5873 (1993). A particularly useful BLAST program is the WU-BLAST-2 program which was obtained from Altschul et al., Meth. Enzymol., 266:460 (1996); blast.wustl/edu/blast/README.html. WU-BLAST-2 uses several search parameters, which are preferably set to the default values. The parameters are dynamic values and are established by the program itself depending upon the composition of the particular sequence and composition of the particular database against which the sequence of interest is being searched; however, the values may be adjusted to increase sensitivity. An additional useful algorithm is gapped BLAST as reported by Altschul et al., Nucleic Acids Res. 25:3389 (1997). A percentage amino acid sequence identity value is determined by the number of matching identical residues divided by the total number of residues of the “longer” sequence in the aligned region. The “longer” sequence is the one having the most actual residues in the aligned region (gaps introduced by WU-Blast-2 to maximize the alignment score are ignored). The alignment may include the introduction of gaps in the sequences to be aligned. In addition, for sequences which contain either more or fewer amino acids than the peptides specifically disclosed herein, it is understood that in one embodiment, the percentage of sequence identity will be determined based on the number of identical amino acids in relation to the total number of amino acids. Thus, for example, sequence identity of sequences shorter than a sequence specifically disclosed herein, will be determined using the number of amino acids in the shorter sequence, in one embodiment. In percent identity calculations relative weight is not assigned to various manifestations of sequence variation, such as insertions, deletions, substitutions, etc.

In a specific embodiment peptides of the invention can be modified for in vivo use by the addition, at the amino- and/or carboxyl-terminal ends, of a blocking agent to facilitate survival of the relevant polypeptide in vivo. This can be useful in those situations in which the peptide termini tend to be degraded by proteases prior to cellular uptake. Such blocking agents can include, without limitation, additional related or unrelated peptide sequences that can be attached to the amino and/or carboxyl terminal residues of the peptide to be administered. This can be done either chemically during the synthesis of the peptide or by recombinant DNA technology by any suitable methods. For example, one or more non-naturally occurring amino acids, such as for example D-alanine, can be added to the termini. Alternatively, blocking agents such as pyroglutamic acid or other molecules known in the art can be attached to the amino and/or carboxyl terminal residues, or the amino group at the amino terminus or carboxyl group at the carboxyl terminus can be replaced with a different moiety. Additionally, the peptide terminus can be modified, e.g. by acetylation of the N-terminus and/or amidation of the C-terminus. Likewise, the peptides can be covalently or noncovalently coupled to pharmaceutically acceptable “carrier” proteins prior to administration.

### Administration of Peptides of the Invention—Pharmaceutical Compositions Comprising Peptides of the Invention

In one embodiment, the peptides of the invention are administered directly to a subject. Generally, the compounds of the invention will be suspended in a pharmaceutically-acceptable carrier (e.g., physiological saline) and administered orally or by intravenous infusion, or administered subcutaneously, intramuscularly, intrathecally, intraperitoneally, intrarectally, intravaginally, intranasally, intragastrically, intratracheally, or intrapulmonarily. In another embodiment, the intratracheal or intrapulmonary delivery can be accomplished using a standard nebulizer, jet nebulizer, wire mesh nebulizer, dry powder inhaler, or metered dose inhaler. They can be delivered directly to the site of the bacterial infection, such as for example lungs, kidney, bladder, or intestines. The dosage required depends on the choice of the route of administration; the nature of the formulation; the nature of the patient's illness; the subject's size, weight, surface area, age, and sex; other drugs being administered; and the judgment of the attending physician. Suitable dosages are in the range of 0.01-100 mg/kg. Wide variations in the needed dosage are to be expected in view of the variety of peptides and variants possible and the differing efficiencies of various routes of administration. For example, oral administration would be expected to require higher dosages than administration by i.v. injection. Variations in these dosage levels can be adjusted using standard empirical routines for optimization as is well understood in the art. Administrations can be single or multiple (e.g., 2-, 3-, 4-, 6-, 8-, 10-; 20-, 50-, 100-, 150- or more fold).

In certain embodiments, the peptides of the invention comprise at least one modified terminus, e.g., to protect the peptide against degradation. In some embodiments, the N-terminus is acetylated and/or the C-terminus is amidated. In certain embodiments, the peptides of the invention comprise at least one non-natural amino acid (e.g., 1, 2, 3, or more) or at least one terminal modification (e.g. 1 or 2). In some embodiments, the peptide comprises at least one non-natural amino acid and at least one terminal modification.

The peptides of the present invention can optionally be delivered in conjunction with other therapeutic agents. The additional therapeutic agents can be delivered concurrently with the peptides of the invention. As used herein, the word “concurrently” means sufficiently close in time to produce a combined effect (that is, concurrently can be simultaneously, or it can be two or more events occurring within a short time period before or after each other). In one embodiment of the invention, the non-naturally occurring peptide is delivered to a patient concurrently with an antibiotic that modulates the growth of bacteria where the combined activity of the non-naturally occurring peptide and the antibiotic agent have superior activity to the bacteria alone. Another aspect of the invention relates to a kit comprising a peptide from the invention and useful for carrying out the methods of the invention. The kit may further comprise additional reagents for carrying out the methods (e.g., buffers, containers, additional therapeutic agents) as well as instructions. As a further aspect, the invention provides pharmaceutical formulations and methods of administering the same to achieve any of the therapeutic effects (e.g. bacterial killing) discussed above. The pharmaceutical formulation may comprise any of the reagents discussed above in a pharmaceutically acceptable carrier, e.g. a non-naturally occurring peptide or variant thereof. By “pharmaceutically acceptable” it is meant a material that is not biologically or otherwise undesirable, i.e., the material can be administered to a subject without causing any undesirable biological effects such as toxicity. The formulations of the invention can optionally comprise medicinal agents, pharmaceutical agents, carriers, adjuvants, dispersing agents, diluents, and the like. The peptides of the invention can be formulated for administration in a pharmaceutical carrier in accordance with known techniques. See, e.g. Remington, The Science And Practice of Pharmacy (Ed. 2014). In the manufacture of a pharmaceutical formulation according to the invention, the peptide (including the physiologically acceptable salts thereof) is typically admixed with, inter alia, an acceptable carrier. The carrier can be a solid or a liquid, or both, and is preferably formulated with the peptide as a unit-dose formulation, for example, a tablet, which can contain from 0.01 or 0.5% to 95% or 99% by weight of the peptide. One or more peptides can be incorporated in the formulations of the invention, which can be prepared by any of the well-known techniques of pharmacy. A further aspect of the invention is a method of treating subjects in vivo, comprising administering to a subject a pharmaceutical composition comprising a peptide of the invention in a pharmaceutically acceptable carrier, wherein the pharmaceutical composition is administered in a therapeutically effective amount. Administration of the peptides of the present invention to a human subject or an animal in need thereof can be by any means known in the art for administering compounds. The formulations of the invention include those suitable for oral, rectal, topical, buccal (e.g., sub-lingual), vaginal, parenteral (e.g., subcutaneous, intramuscular including skeletal muscle, cardiac muscle, diaphragm muscle and smooth muscle, intradermal, intravenous, intraperitoneal), topical (i.e., both skin and mucosal surfaces, including airway surfaces), intranasal, transdermal, intraarticular, intrathecal, and inhalation administration, administration to the liver by intraportal delivery, as well as direct organ injection (e.g., into the liver, into the brain for delivery to the central nervous system, into the pancreas, or into a tumor or the tissue surrounding a tumor). The most suitable route in any given case will depend on the nature and severity of the condition being treated and on the nature of the particular peptide which is being used.

For injection, the carrier will typically be a liquid, such as sterile pyrogen-free water, sterile normal saline, hypertonic saline, pyrogen-free phosphate-buffered saline solution. For other methods of administration, the carrier can be either solid or liquid.

For oral administration, the peptide can be administered in solid dosage forms, such as capsules, tablets, and powders, or in liquid dosage forms, such as elixirs, syrups, and suspensions. Peptides can be encapsulated in gelatin capsules together with inactive ingredients and powdered carriers, such as glucose, lactose, sucrose, mannitol, starch, cellulose or cellulose derivatives, magnesium stearate, stearic acid, sodium saccharin, talcum, magnesium carbonate and the like. Examples of additional inactive ingredients that can be added to provide desirable color, taste, stability, buffering capacity, dispersion or other known desirable features are red iron oxide, silica gel, sodium lauryl sulfate, titanium dioxide, edible white ink and the like. Similar diluents can be used to make compressed tablets. Both tablets and capsules can be manufactured as sustained release products to provide for continuous release of medication over a period of hours. Compressed tablets can be sugar coated or film coated to mask any unpleasant taste and protect the tablet from the atmosphere, or enteric-coated for selective disintegration in the gastrointestinal tract. Liquid dosage forms for oral administration can contain coloring and flavoring to increase patient acceptance. Formulations suitable for buccal (sub-lingual) administration include lozenges comprising the compound in a flavored base, usually sucrose and acacia or tragacanth; and pastilles comprising the compound in an inert base such as gelatin and glycerin or sucrose and acacia. Formulations of the present invention suitable for parenteral administration comprise sterile aqueous and non-aqueous injection solutions of the peptide, which preparations are preferably isotonic with the blood of the intended recipient. These preparations can contain anti-oxidants, buffers, bacteriostats and solutes which render the formulation isotonic with the blood of the intended recipient. Aqueous and non-aqueous sterile suspensions can include suspending agents and thickening agents. The formulations can be presented in unit/dose or multi-dose containers, for example sealed ampoules and vials, and can be stored in a freeze-dried (lyophilized) condition requiring only the addition of the sterile liquid carrier, for example, saline or water-for-injection immediately prior to use.

Extemporaneous injection solutions and suspensions can be prepared from sterile powders, granules and tablets of the kind previously described. For example, in one aspect of the present invention, there is provided an injectable, stable, sterile composition comprising a peptide of the invention, in a unit dosage form in a sealed container. The peptide or salt is provided in the form of a lyophilizate which is capable of being reconstituted with a suitable pharmaceutically acceptable carrier to form a liquid composition suitable for injection thereof into a subject. The unit dosage form typically comprises from about 1 mg to about 10 grams of the peptide or salt. When the peptide or salt is substantially water-insoluble, a sufficient amount of emulsifying agent which is pharmaceutically acceptable can be employed in sufficient quantity to emulsify the peptide or salt in an aqueous carrier. One such useful emulsifying agent is phosphatidyl choline. Formulations suitable for rectal administration are preferably presented as unit dose suppositories. These can be prepared by admixing the peptide with one or more conventional solid carriers, for example, cocoa butter, and then shaping the resulting mixture. Formulations suitable for topical application to the skin preferably take the form of an ointment, cream, lotion, paste, gel, spray, aerosol, or oil. Carriers which can be used include petroleum jelly, lanoline, polyethylene glycols, alcohols, transdermal enhancers, and combinations of two or more thereof. Formulations suitable for transdermal administration can be presented as discrete patches adapted to remain in intimate contact with the epidermis of the recipient for a prolonged period of time. Formulations suitable for transdermal administration can also be delivered by iontophoresis (see, for example, Tyle, Pharm. Res. 3:318 (1986) and typically take the form of an optionally buffered aqueous solution of the peptides. Suitable formulations comprise citrate or bis/tris buffer (pH 6) or ethanol/water and contain from 0.1 to 0.2M of the compound. The peptide can alternatively be formulated for nasal administration or otherwise administered to the lungs of a subject by any suitable means, e.g., administered by an aerosol suspension of respirable particles comprising the peptide, which the subject inhales. The respirable particles can be liquid or solid. The term “aerosol” includes any gas-borne suspended phase, which is capable of being inhaled into the bronchioles or nasal passages. Specifically, aerosol includes a gas-borne suspension of droplets, as can be produced in a metered dose inhaler or nebulizer, or in a mist sprayer. Aerosol also includes a dry powder composition suspended in air or other carrier gas, which can be delivered by insufflation from an inhaler device, for example. Aerosols of liquid particles comprising the peptide can be produced by any suitable means, such as with a pressure-driven aerosol nebulizer or an ultrasonic nebulizer, as is known to those of skill in the art. Aerosols of solid particles comprising the peptide can likewise be produced with any solid particulate medicament aerosol generator, by techniques known in the pharmaceutical art. Alternatively, one can administer the peptide in a local rather than systemic manner, for example, in a depot or sustained-release formulation.

Further, the present invention provides liposomal formulations of the peptides disclosed herein and salts thereof. The technology for forming liposomal suspensions is well known in the art. When the peptide or salt thereof is an aqueous-soluble salt, using conventional liposome technology, the same can be incorporated into lipid vesicles. In such an instance, due to the water solubility of the peptide or salt, the peptide or salt will be substantially entrained within the hydrophilic center or core of the liposomes. The lipid layer employed can be of any conventional composition and can either contain cholesterol or can be cholesterol-free. When the peptide or salt of interest is water-insoluble, again employing conventional liposome formation technology, the salt can be substantially entrained within the hydrophobic lipid bilayer which forms the structure of the liposome. In either instance, the liposomes which are produced can be reduced in size, as through the use of standard sonication and homogenization techniques. The liposomal formulations containing the peptides disclosed herein or salts thereof, can be lyophilized to produce a lyophilizate which can be reconstituted with a pharmaceutically acceptable carrier, such as water, to regenerate a liposomal suspension. In the case of water-insoluble peptides, a pharmaceutical composition can be prepared containing the water-insoluble peptide, such as for example, in an aqueous base emulsion. In such an instance, the composition will contain a sufficient amount of pharmaceutically acceptable emulsifying agent to emulsify the desired amount of the peptide. Particularly useful emulsifying agents include phosphatidyl cholines and lecithin. In particular embodiments, the peptide is administered to the subject in a therapeutically effective amount, as that term is defined above. Dosages of pharmaceutically active peptides can be determined by methods known in the art, see, e.g., Remington's Pharmaceutical Sciences. The therapeutically effective dosage of any specific peptide will vary somewhat from peptide to peptide, and patient to patient, and will depend upon the condition of the patient and the route of delivery. As a general proposition, a dosage from about 0.1 to about 50 mg/kg will have therapeutic efficacy, with all weights being calculated based upon the weight of the peptide, including the cases where a salt is employed. Toxicity concerns at the higher level can restrict intravenous dosages to a lower level such as up to about 10 mg/kg, with all weights being calculated based upon the weight of the peptide, including the cases where a salt is employed. A dosage from about 10 mg/kg to about 50 mg/kg can be employed for oral administration. Typically, a dosage from about 0.5 mg/kg to 5 mg/kg can be employed for intramuscular

injection. Particular dosages are about 1 μmol/kg to 50 μmol/kg, and more particularly to about 22 μmol/kg and to 33 μmol/kg of the peptide for intravenous or oral administration, respectively. In particular embodiments of the invention, more than one administration (e.g. two, three, four, or more administrations) can be employed over a variety of time intervals (e.g., hourly, daily, weekly, monthly, etc.) to achieve therapeutic effects. The present invention finds use in veterinary and medical applications. Suitable subjects include both avians and mammals, with mammals being preferred. The term “avian” as used herein includes, but is not limited to, chickens, ducks, geese, quail, turkeys, and pheasants. The term “mammal” as used herein includes, but is not limited to, humans, bovines, ovines, caprines, equines, felines, canines, lagomorphs, etc. Human subjects include neonates, infants, juveniles, and adults.

### Synthesis of Peptides

In specific embodiments the peptides of the invention can be produced according to several peptide synthesis methods known in the art. The peptide synthesis method may be any of, for example, a solid phase synthesis process and a liquid phase synthesis process. That is, the object peptide can be produced by repeating condensation of a partial peptide or amino acid capable of constituting compound (I) and the remaining portion (which may be constituted by two or more amino acids) according to a desired sequence. When a product having the desirable sequence has a protecting group, the object peptide can be produced by eliminating a protecting group. Examples of the condensing method and eliminating method of a protecting group to be known include methods described in the following (1)-(5). (1) M. Bodanszky and M. A. Ondetti: Peptide Synthesis, Interscience Publishers, New York (1966), (2) Schroeder and Luebke: The Peptide, Academic Press, New York (1965), (3) Nobuo Izumiya, et al.: Peptide Gosei-no-Kiso to Jikken (Basics and experiments of peptide synthesis), published by Maruzen Co. (1975), (4) Haruaki Yajima and Shunpei Sakakibara: Seikagaku Jikken Koza (Biochemical Ex-periment) 1, Tanpakushitsu no Kagaku (Chemistry of Proteins) IV, 205 (1977) and (5) Haruaki Yajima, ed.: Zoku Iyakuhin no Kaihatsu (A sequel to Development of Pharmaceuticals), Vol. 14, Peptide Synthesis, published by Hirokawa Shoten. After the reaction, the peptides can be purified and isolated using conventional methods of purification, such as solvent extraction, distillation, column chromatography, liquid chromatography, recrystallization, etc., in combination thereof. When the peptide obtained by the above-mentioned method is in a free form, it can be converted to a suitable salt by a known method; conversely, when the peptide is obtained in the form of a salt, the salt can be converted to a free form or other salt by a known method. The starting compound may also be a salt. Examples of such salt include those exemplified as salts of the peptides mentioned bellow. For condensation of protected amino acid or peptide, various activation reagents usable for peptide synthesis can be used, which are particularly preferably trisphosphonium salts, tetramethyluronium salts, carbodiimides and the like. Examples of the trisphosphonium salt include benzotriazol-1-yloxytris(pyrrolizino)phosphoniumhexafluorophosphate (PyBOP), bromotris(pyrrolizino)phosphoniumhexafluorophosphate (PyBroP), 7-azabenzotriazol-1-yloxytris(pyrrolizino)phosphoniumhexafluorophosphate (PyAOP), examples of the tetramethyluronium salt include 2-(1H-benzotriazol-1-yl)-1,1,3,3-hexafluorophosphate (HBTU), 2-(7-azabenzotriazol-1-yl)-1,1,3,3-hexafluorophosphate (HATU), 2-(1H-benzotriazol-1-yl)-1,1,3,3-tetramethyluroniumtetrafluoroborate (TBTU), 2-(5-norbornane-2,3-dicarboxyimide)-1,1,3,3-tetramethyluroniumtetrafluoroborate (TNTU), O—(N-succimidyl)-1,1,3,3-tetramethyluroniumtetrafluoroborate (TSTU), and examples of the carbodiimide include DCC, N,N′-diisopropylcarbodiimide (DIPCDI), N-ethyl-N′-(3-dimethylaminopropyl)carbodiimide hydrochloride (EDCI.HCl) and the like. For condensation using these, addition of a racemization inhibitor (e.g., HONB, HOBt, HOAt, HOOBt etc.) can be used. A solvent to be used for the condensation can be appropriately selected from those known to be usable for peptide condensation reaction. For example, acid amides such as anhydrous or water-containing N,N-dimethylformamide, N,N-dimethylacetamide, N-methylpyrrolidone and the like, halogenated hydrocarbons such as methylene chloride, chloroform and the like, alcohols such as trifluoroethanol, phenol and the like, sulfoxides such as dimethyl-sulfoxide and the like, tertiary amines such as pyridine and the like, ethers such as dioxane, tetrahydrofuran and the like, nitriles such as acetonitrile, propionitrile and the like, esters such as methyl acetate, ethyl acetate and the like, an appropriate mixture of these and the like can be used. Reaction temperature is appropriately selected from the range known to be usable for peptide binding reactions, and is normally selected from the range of about −20 C (“C” represents “degrees Celsius”) to 50 degrees C. An activated amino acid derivative is normally used from 1.5 to 6 times in excess. In phase synthesis, when a test using the ninhydrin reaction reveals that the condensation is insufficient, sufficient condensation can be conducted by repeating the condensation reaction without elimination of protecting groups. If the condensation is yet insufficient even after repeating the reaction, unreacted amino acids can be acylated with acetic anhydride, acetylimidazole or the like so that an influence on the subsequent reactions can be avoided. Examples of the protecting groups for the amino groups of the starting amino acid include Z, Boc, tert-pentyloxycarbonyl, isobornyloxycarbonyl, 4-methoxybenzyloxycarbonyl, Cl—Z, Br—Z, adamantyloxycarbonyl, trifluoroacetyl, phthaloyl, formyl, 2-nitrophenylsulphenyl, diphenylphosphinothioyl, Fmoc, trityl and the like. Examples of the carboxyl-protecting group for the starting amino acid include allyl, 2-adamantyl, 4-nitrobenzyl, 4-methoxybenzyl, 4-chlorobenzyl, phenacyl and benzy-oxycarbonylhydrazide, tert-butoxycarbonylhydrazide, tritylhydrazide and the like, in addition to the above-mentioned C1-6 alkyl group, C3-10 cycloalkyl group, C7-14 aralkyl group. The hydroxyl group of serine or threonine can be protected, for example, by esterification or etherification. Examples of the group suitable for the esterification include lower (C2-4) alkanoyl groups such as an acetyl group and the like, aroyl groups such as a benzoyl group and the like, and the like, and a group derived from an organic acid and the like. In addition, examples of the group suitable for etherification include benzyl, tetrahydropyranyl, tert-butyl(Bu.sup.t), trityl (Trt) and the like. Examples of the protecting group for the phenolic hydroxyl group of tyrosine include Bzl, 2,6-dichlorobenzyl, 2-nitrobenzyl, Br—Z, tert-butyl and the like. Examples of the protecting group for the imidazole of histidine include Tos, 4-methoxy-2,3,6-trimethylbenzenesulfonyl (Mtr), DNP, Bom, Bum, Boc, Trt, Fmoc and the like.

Examples of the protecting group for the guanidino group of arginine include Tos, Z, 4-methoxy-2,3,6-trimethylbenzenesulfonyl (Mtr), p-methoxybenzenesulfonyl (MBS), 2,2,5,7,8-pentamethylchromane-6-sulfonyl (Pmc), mesitylene-2-sulfonyl (Mts), 2,2,4,6,7-pentamethyldihydrobenzofuran-5-sulfonyl (Pbf), Boc, Z, NO2 and the like. Examples of the protecting group for a side chain amino group of lysine include Z, Cl—Z, trifluoroacetyl, Boc, Fmoc, Trt, Mtr, 4,4-dimethyl-2,6-dioxocyclohexylideneyl (Dde) and the like. Examples of the protecting group for indolyl of tryptophan include formyl (For), Z, Boc, Mts, Mtr and the like. Examples of the protecting group for asparagine and glutamine include Trt, xanthyl (Xan), 4,4′-dimethoxybenzhydryl (Mbh), 2,4,6-trimethoxybenzyl (Tmob) and the like. Examples of activated carboxyl groups in the starting material include corresponding acid anhydride, azide, active esters [ester with alcohol (e.g., pentachlorophenol, 2,4,5-trichlorophenol, 2,4-dinitrophenol, cyanomethylalcohol, paranitrophenol, HONB, N-hydroxysuccimide, 1-hydroxybenzotriazole (HOBt), 1-hydroxy-7-azabenzotriazole (HOAt))] and the like. Examples of the activated amino group in the starting material include corresponding phosphorous amide. Examples of the method for removing (eliminating) a protecting group include a catalytic reduction in a hydrogen stream in the presence of a catalyst such as Pd-black or Pd-carbon; an acid treatment using anhydrous hydrogen fluoride, methanesulfonic acid, trifluoromethanesulfonic acid, trifluoroacetate, trimethylsilyl bromide (TMSBr), trimethylsilyl trifluoromethanesulfonate, tetrafluoroboric acid, tris(trifluoro)boric acid, boron tribromide, or a mixture solution thereof; a base treatment using diisopropylethylamine, triethylamine, piperidine, piperazine or the like; and reduction with sodium in liquid ammonia, and the like. The elimination reaction by the above-described acid treatment is generally carried out at a temperature of −20 C to 40 C; the acid treatment is efficiently conducted by adding a cation scavenger such as anisole, phenol, thioanisole, metacresol and paracresol; dimethylsulfide, 1,4-butanedithiol, 1,2-ethanedithiol and the like. Also, a 2,4-dinitrophenyl group used as a protecting group of the imidazole of histidine is removed by thiophenol treatment; a formyl group used as a protecting group of the indole of tryptophan is removed by deprotection by acid treatment in the presence of 1,2-ethanedithiol, 1,4-butanedithiol, or the like, as well as by alkali treatment with dilute sodium hydroxide, dilute ammonia, or the like.

In addition, the peptides of the invention may be a solvate (e.g., hydrate) or a non-solvate (e.g., non-hydrate). The peptides may be labeled with an isotope (e.g. 3H, 14C, 35S, 125I) or the like. Furthermore, the peptides may be a deuterium conversion form wherein 1H is converted to 2H(D). Peptides labeled or substituted with an isotope can be used as, for example, a tracer (PET tracer) for use in Positron Emission Tomography (PET), and is useful in the fields of medical diagnosis and the like.

For the peptides mentioned herein, the left end is the N-terminal (amino terminal) and the right end is the C-terminal (carboxyl terminal) in accordance with the conventional peptide marking. The C-terminal of peptide may be any of an amide (—CONH2), a carboxyl group (—COOH), a carboxylate (—COO−), an alkylamide (—CONHR), and an ester (—COOR). Particularly, amide (—CONH2) is preferable. The compounds may be in a salt form. Examples of such salt include metal salts, ammonium salts, salts with organic base, salts with inorganic acid, salts with organic acid, salts with basic or acidic amino acid, and the like.

In certain embodiments, the peptides may also be in a prodrug form. A prodrug means a compound which is converted to a functional peptide of the invention with a reaction due to an enzyme, gastric acid, etc. under the physiological condition in the living body, that is, a compound which is converted to a peptide of the invention with oxidation, reduction, hydrolysis, etc. according to an enzyme; a compound which is converted to a peptide of the invention by hydrolysis etc. due to gastric acid, etc. Examples of a prodrug of a peptide of the invention include a compound wherein an amino of the peptide is acylated, alkylated or phosphorylated (e.g., compound wherein amino of the peptide is eicosanoylated, alanylated, pentylaminocarbonylated, (5-methyl-2-oxo-1,3-dioxolen-4-yl)methoxycarbonylated, tetrahydrofuranylated, pyrrolidylmethylated, pivaloyloxymethylated or tert-butylated, and the like); a compound wherein a hydroxy of the peptide is acylated, alkylated, phosphorylated or borated (e.g., a compound wherein a hydroxy of the peptide is acetylated, palmytoylated, propanoylated, pivaloylated, succinylated, fumarylated, alanylated or dimethylaminomethylcarbonylated); a compound wherein a carboxy of the peptide is esterified or amidated (e.g., a compound wherein a carboxy of the peptide is C1-6 alkyl esterified, phenyl esterified, carboxymethyl esterified, dimethylaminomethyl esterified, pivaloyloxymethyl esterified, ethoxycarbonyloxyethyl esterified, phthalidyl esterified, (5-methyl-2-oxo-1,3-dioxolen-4-yl)methyl esterified, cyclohexyloxycarbonylethyl esterified or methylamidated) and the like. Among others, a compound wherein carboxy of compound (I) is esterified with C1-6 alkyl such as methyl, ethyl, tert-butyl or the like is preferably used. These compounds can be produced from a peptide by a method known per se. A prodrug of a peptide of the invention may also be one which is converted into a peptide of the invention under a physiological condition, such as those described in IYAKUHIN no KAIHATSU (Development of Pharmaceuticals), Vol. 7, Design of Molecules, p. 163-198, Published by HIROKAWA SHOTEN (1990). In the present specification, the prodrug may form a salt. Examples of such a salt include those exemplified as the salt of a peptide of the invention. A peptide of the invention may form a crystal. Crystals having a singular crystal form or a mixture of plural crystal forms are also included in a peptide of the invention. Crystals can be produced by crystallizing a peptide of the invention according to a crystallization method known per se. In addition, a peptide of the invention may be a pharmaceutically acceptable co-crystal or co-crystal salt. Here, the co-crystal or co-crystal salt means a crystalline substance consisting of two or more particular substances which are solids at room temperature, each having different physical properties (e.g. structure, melting point, heat of melting, hygroscopicity, solubility, stability etc.). The cocrystal and cocrystal salt can be produced by cocrystallization known per se. The crystal of a peptide of the invention is superior in physicochemical properties (e.g., melting point, solubility, stability) and biological properties (e.g. pharmacokinetics (absorption, distribution, metabolism, excretion), efficacy expression), and thus it is extremely useful as a medicament.

In further aspects and embodiments, the invention also provides subject-matter as set forth in any one and all of (1) to (14) below:

(1) A peptide comprising the following structure: (A2i-1-X1-X2-X3-X4-X5-X6-X7-A2i-Zi)n, wherein:


- - n is an integer from 1 to 4 and i increases from 1 to n with each
    repeat;
  - each A_(2i-1) and A_(2i) are independently selected from 1 to 3
    contiguous gatekeeper amino acids selected from R, K, D, E and P
    and/or 1 to 3 non-natural gatekeeper amino acid selected from
    3-methylproline, 3,4-dehydro-proline,
    2-\[(2S)-2-(hydrazinecarbonyl)pyrrolidin-1-yl\]-2-oxoacetic acid,
    beta-homoproline, alpha-methylproline, hydroxyproline,
    4-oxo-proline, beta,beta-dimethyl-proline, 5,5-dimethyl-proline,
    4-cyclohexyl-proline, 4-phenyl-proline, 3-phenyl-proline,
    4-aminoproline, 4-mercaptoproline, 2-amino-adipic acid (homoglutamic
    acid), 2-amino-heptanedioic acid (2-aminopimelic acid),
    2-amino-octanedioic acid (aminosuberic acid),
    2-amino-4-carboxy-pentanedioic acid (4-carboxyglutamic acid),
    glyoxal-hydroimidazolone, methylglyoxal-hydroimidazolone,
    N-alpha-methyl-arginine, omega-methyl-arginine, norarginine,
    homoarginine, N,N′-diethyl-homoarginine, beta-homoarginine,
    2-amino-3-ureido-propionic acid,
    2-amino-6-(1-carboxyethylamino)hexanoic acid,
    2-amino-6-carboxymethylamino)hexanoic acid,
    2-amino-6-(2-(furan-2-yl)-2-oxoethylamino)hexanoic acid,
    2-amino-6-(formyl-5-hydroxymethyl-pyrrol-1-yl)-hexanoic acid,
    c-alpha-methyl-lysine,
    beta,beta-dimethyl-lysine,N-epsilon-formyl-lysine,N-epsilon-methyl-lysine,N-epsilon-i-propyl-lysine,
    N-epsilon-dimethyl-lysine, N-epsilon-trimethylamonium-lysine,
    N-epsilon-nicotinyl-lysine,
    {\[5-amino-1-(hydrazinecarbonyl)pentyl\]carbamoyl}formic acid,
    N-alpha-methyl-lysine, homolysine, beta-homolysine,
    2-Amino-6-diazo-5-oxocaproic acid, norvaline,
    alpha-methyl-norvaline, Hydroxinorvaline, Ornithine,
    N-delta-methyl-ornithine, N-delta-N-delta-dimethyl-ornithine,
    N-delta-i-propyl-ornithine,
    c-alpha-methyl-ornithine,beta,beta-dimethyl-ornithine,canavanine,N-delta-methyl-N-delta-butyl-ornithine,
    N-delta-methyl-N-delta-phenyl-ornithine,
    delta-(2-methylpyrrolidine)-ornithine, delta-piperidyl-ornithine,
    gamma-amino-delta-piperidyl-valeric acid and
    delta-azepanyl-ornithine and wherein the amino-terminal gatekeeper
    amino acid or amino-terminal non-natural gatekeeper amino acid in
    the peptide structure is optionally acetylated and/or wherein the
    carboxy-terminal gatekeeper amino acid or carboxy-terminal
    non-natural gatekeeper amino acid in the peptide sequence is
    optionally amidated,
  - X₁ is glycine or a conservative amino acid substitution of glycine
    or a non-natural amino acid variant of glycine or a D-amino acid of
    a conservative amino acid substitution of glycine or a D-amino acid
    of a non-natural amino acid variant of glycine,
  - X₂ is leucine or a conservative amino acid substitution of leucine
    or a non-natural amino acid variant of leucine or a D-amino acid of
    leucine or a D-amino acid of a conservative amino acid substitution
    of leucine or a D-amino acid of a non-natural amino acid variant of
    leucine,
  - X₃ is glycine or a conservative amino acid substitution of glycine
    or a non-natural amino acid variant of glycine or a or a D-amino
    acid of a conservative amino acid substitution of glycine or a
    D-amino acid of a non-natural amino acid variant of glycine,
  - X₄ is leucine or a conservative amino acid substitution of leucine
    or a non-natural amino acid variant of leucine or a D-amino acid of
    leucine or a D-amino acid of a conservative amino acid substitution
    of leucine or a D-amino acid of a non-natural amino acid variant of
    leucine,
  - X₅ is alanine or a conservative amino acid substitution of alanine
    or a non-natural amino acid variant of alanine or a D-amino acid of
    alanine or a D-amino acid of a conservative amino acid substitution
    of alanine or a D-amino acid of a non-natural amino acid variant of
    alanine,
  - X₆ is leucine or a conservative amino acid substitution of leucine
    or a non-natural amino acid variant of leucine or a D-amino acid of
    leucine or a D-amino acid of a conservative amino acid substitution
    of leucine or a D-amino acid of a non-natural amino acid variant of
    leucine,
  - X₇ is valine or a conservative amino acid substitution of valine or
    a non-natural amino acid variant of valine or a D-amino acid of
    valine or a D-amino acid of a conservative amino acid substitution
    of valine or a D-amino acid of a non-natural amino acid variant of
    valine,  
    and wherein X₁-X₂-X₃-X₄-X₅-X₆-X₇ and X₇-X₆-X₅-X₄-X₃-X₂-X₁ are used
    interchangeably in the repeats wherein n is 2 to 4,  
    and each Z_(i) is a linker and wherein each Z_(i) is independently
    selected from stretch of between 1 and 5 units, wherein a unit is
    PEG, an amino acid or a non-natural amino acid.  
    (2) A peptide according to (1) wherein:
  - X₁ and X₃ are selected from glycine, N-alpha-methyl-glycine
    (sarcosine), cyclopropylglycine and cyclopentylglycine
  - X₂, X₄ and X₆ are selected from leucine, valine,
    2-amino-3,3-dimethyl-butyric acid (t-Leucine), alpha-methylleucine,
    hydroxyleucine, 2,3-dehydro-leucine, N-alpha-methyl-leucine,
    2-Amino-5-methyl-hexanoic acid (homoleucine),
    3-Amino-5-methylhexanoic acid (beta-homoleucine),
    2-Amino-4,4-dimethyl-pentanoic acid (4-methyl-leucine,
    neopentylglycine), 4,5-dehydro-norleucine (allylglycine),
    L-norleucine, N-alpha-methyl-norleucine and 6-hydroxy-norleucine
  - X₅ is selected from alanine, 2-amino-isobutyric acid
    (2-Methylalanine), 2-Amino-2-methylbutanoic acid (Isovaline),
    N-alpha-Methyl-alanine, 2-Amino-2-methylpent-4-enoic acid
    (alpha-allylalanine), beta-homoalanine, 2-indanyl-glycine,
    Di-n-propyl-glycine, Di-n-butyl-glycine, Diethyl-glycine,
    (1-naphthyl)alanine, (2-naphthyl)alanine, cyclohexylglycine,
    adamantyl-glycine, beta-homoallylglycine
  - X₇ is selected from valine, leucine, c-alpha-methyl-valine
    (2,3-dimethylbutanoic acid), 2,3-dehydro-valine, 3,4-dehydro-valine,
    3-methyl-L-isovaline (methylvaline),
    2-amino-3-hydroxy-3-methylbutanoic acid (hydroxyvaline),
    beta-homovaline and N-alpha-methyl-valine.  
    (3) A peptide according to (1) or (2) wherein Z_(i) consists of
    proline, 4-hydroxyproline, (2R,
    5S)-5-pentyl-pyrrolidine-2-carboxylic acid, 3,4-dehydro-L-proline,
    beta-(2-benzothiazolyl)-alanine, 3-(2-furyl)-alanine or
    beta-(2-thienyl)-alanine.  
    (4) A cyclic peptide according to any one of (1) to (3) wherein the
    aminoterminal and carboxyterminal gatekeeper aminoacids A_(2i-1) and
    A_(2i) form a ring structure.  
    (5) A peptidomimetic generated from a peptide according to (1),
    (2), (3) or (4).  
    (6) The peptide or peptidomimetic of any one of (1) to (5), further
    comprising a detectable label.  
    (7) The peptide or peptidomimetic of any one of (1) to (5), further
    comprising a molecule which increases the half-life extension.  
    (8) The peptide or peptidomimetic of any one of (1) to (5), further
    comprising a moiety that increases solubility of the molecule.  
    (9) The peptide or peptidomimetic of any one of (1) to (5), further
    comprising at least one D-alanine at the amino-terminus and/or the
    carboxy-terminus.  
    (10) A molecule according to any one of (1) to (9) for use as a
    medicine.  
    (11) A molecule according to any one of (1) to (9) for use as an
    anti-bacterial agent.  
    (12) A molecule according to any one of (1) to (9) for use as a
    diagnostic agent.  
    (13) A pharmaceutical composition, comprising according to any one
    of (1) to (9) and a pharmaceutically acceptable carrier.  
    (14) In yet another embodiment the invention provides a method to
    produce an anti-bacterial peptide comprising the following steps:
  - (i) generating an in silico list of aggregation prone regions (APRs)
    with a length of 6-12 amino acids which APRs are identified in a
    bacterial proteome,
  - (ii) synthetizing a number of 20-200 different peptides comprising
    an APR based on the following structure:
    (A_(2i-1)-APR-A_(2i)-Z_(i))_(n), wherein n is an integer from 1 to 4
    and i increases from 1 to n with each repeat and each A_(2i-1) and
    A_(2i) are independently selected from 1 to 3 contiguous gatekeeper
    amino acids selected from R, K, D, E and P,
  - (iii) testing said peptides for an anti-bacterial effect and
    producing an anti-bacterial peptide.

In further aspects and embodiments, the invention also provides subject-matter as set forth in any one and all of (1*) to (12*) below:

(1*) A peptide comprising the following structure: (A2i-1-APR-A2i-Zi)n, wherein:


- - n is an integer from 1 to 4 and i increases from 1 to n with each
    repeat;
  - each A_(2i-1) and A_(2i) are independently selected from 1 to 3
    contiguous gatekeeper amino acids selected from R, K, D, E and P or
    1 to 3 non-natural gatekeeper amino acid selected from
    3-methylproline, 3,4-dehydro-proline,
    2-\[(2S)-2-(hydrazinecarbonyl)pyrrolidin-1-yl\]-2-oxoacetic acid,
    beta-homoproline, alpha-methylproline, hydroxyproline,
    4-oxo-proline, beta,beta-dimethyl-proline, 5,5-dimethyl-proline,
    4-cyclohexyl-proline, 4-phenyl-proline, 3-phenyl-proline,
    4-aminoproline, 4-mercaptoproline, 2-amino-adipic acid (homoglutamic
    acid), 2-amino-heptanedioic acid (2-aminopimelic acid),
    2-amino-octanedioic acid (aminosuberic acid),
    2-amino-4-carboxy-pentanedioic acid (4-carboxyglutamic acid),
    glyoxal-hydroimidazolone, methylglyoxal-hydroimidazolone,
    N-alpha-methyl-arginine, omega-methyl-arginine, norarginine,
    homoarginine, N,N′-diethyl-homoarginine, beta-homoarginine,
    2-amino-3-ureido-propionic acid,
    2-amino-6-(1-carboxyethylamino)hexanoic acid,
    2-amino-6-carboxymethylamino)hexanoic acid,
    2-amino-6-(2-(furan-2-yl)-2-oxoethylamino)hexanoic acid,
    2-amino-6-(formyl-5-hydroxymethyl-pyrrol-1-yl)-hexanoic acid,
    c-alpha-methyl-lysine, beta,beta-dimethyl-lysine,
    N-epsilon-formyl-lysine, N-epsilon-methyl-lysine,
    N-epsilon-i-propyl-lysine, N-epsilon-dimethyl-lysine,
    N-epsilon-trimethylamonium-lysine, N-epsilon-nicotinyl-lysine,
    {\[5-amino-1-(hydrazinecarbonyl)pentyl\]carbamoyl}formic acid,
    N-alpha-methyl-lysine, homolysine, beta-homolysine,
    2-Amino-6-diazo-5-oxocaproic acid, norvaline,
    alpha-methyl-norvaline, Hydroxinorvaline, Ornithine,
    N-delta-methyl-ornithine, N-delta-N-delta-dimethyl-ornithine,
    N-delta-i-propyl-ornithine, c-alpha-methyl-ornithine,
    beta,beta-dimethyl-ornithine, canavanine,
    N-delta-methyl-N-delta-butyl-ornithine,
    N-delta-methyl-N-delta-phenyl-ornithine,
    delta-(2-methylpyrrolidine)-ornithine, delta-piperidyl-ornithine,
    gamma-amino-delta-piperidyl-valeric acid and
    delta-azepanyl-ornithine and wherein the amino-terminal gatekeeper
    amino acid or amino-terminal non-natural gatekeeper amino acid in
    the peptide structure is optionally acetylated and/or wherein the
    carboxy-terminal gatekeeper amino acid or carboxy-terminal
    non-natural gatekeeper amino acid in the peptide sequence is
    optionally amidated,
  - the names of the peptides comprised in APR are depicted in Table 5
    (P3, P4, P5, P12, P14, P16, P18, P23, P26, P29, P33, P39, P40, P49,
    P50, P58, P72, P76, P79, P80, P87, P88, P89, P90, P91, P92, P93,
    P99, P101, P103, P105, P111, P112, P113, P114, P115, P116, P117,
    p118, P123, P124 and P125) and the corresponding amino acid
    sequences for these peptides are depicted in Table 4 wherein APR
    comprises natural amino acids or APR comprises conservative amino
    acid substitutions of the amino acids present in APR or APR
    comprises non-natural amino acid analogues of the amino acids
    present in the peptide sequences present in APR or APR comprises
    D-amino acid substitutions in the peptide sequences of the amino
    acids present in APR,  
    and wherein the amino acids in APR can be in a direct or inverted
    repeat wherein n is 2 to 4, and each Z_(i) is a linker and wherein
    each Z_(i) is independently selected from stretch of between 1 and 5
    units, wherein a unit is PEG, an amino acid or a non-natural amino
    acid.  
    (2\*) A peptide according to (1\*) wherein Z_(i) consists of
    proline, 4-hydroxyproline, (2R,
    5S)-5-phenyl-pyrrolidine-2-carboxylic acid, 3,4-dehydro-L-proline,
    beta-(2-benzothiazolyl)-alanine, 3-(2-furyl)-alanine or
    beta-(2-thienyl)-alanine.  
    (3\*) A cyclic peptide according to any one of (1\*) to (2\*)
    wherein the aminoterminal and carboxyterminal gatekeeper aminoacids
    A_(2i-1) and A_(2i) form a ring structure.  
    (4\*) The peptide of any one of (1\*) to (3\*), further comprising a
    detectable label  
    (5\*) The peptide of any one of (1\*) to (4\*), further comprising a
    molecule which increases the half-life extension.  
    (6\*) The peptide of any one of (1\*) to (3\*), further comprising a
    moiety that increases solubility of the molecule.  
    (7\*) The peptide of any one of (1\*) to (3\*), further comprising
    at least one D-alanine at the amino-terminus and/or the
    carboxy-terminus.  
    (8\*) A molecule according to any one of (1\*) to (7\*) for use as a
    medicine.  
    (9\*) A molecule according to any one of (1\*) to (7\*) for use as
    an anti-bacterial agent.  
    (10\*) A molecule according to any one of (1\*) to (7\*) for use as
    a diagnostic agent.  
    (11\*) A pharmaceutical composition, comprising according to any one
    of (1\*) to (7\*) and a pharmaceutically acceptable carrier.  
    (12\*) A method to produce an anti-bacterial peptide comprising the
    following steps:
  - (i) generating an in silico list of aggregation prone regions (APRs)
    with a length of 6-10 amino acids which APRs are identified in a
    bacterial proteome,
  - (ii) synthesizing a number of 20-200 different peptides comprising
    an APR based on the following structure:
    (A_(2i-1)-APR-A_(2i)-Z_(i))_(n), wherein n is an integer from 1 to 4
    and i increases from 1 to n with each repeat and each A_(2i-1) and
    A_(2i) are independently selected from 1 to 3 contiguous gatekeeper
    amino acids selected from R, K, D, E and P,
  - (iii) testing said peptides for an anti-bacterial effect and
    producing an anti-bacterial peptide.

It is to be understood that although particular embodiments, specific configurations as well as materials and/or molecules, have been discussed herein for engineered peptides and methods according to the present invention, various changes or modifications in form and detail may be made without departing from the scope and spirit of this invention. The following examples are provided to better illustrate particular embodiments, and they should not be considered limiting the application. The application is limited only by the claims.

### Examples

1. Design and Screening of E. coli Specific Aggregating Peptides

In the present invention we used the statistical thermodynamics algorithm TANGO to identify aggregation prone regions (APR) in the proteome of E. coli strain O157:H7. This yielded 3535 sequences of at least 6 amino acids in length and a TANGO score of at least 20%. In order to generate efficient aggregation seeds, we employed a previously devised tandem repeat design19,21, in which the APRs are repeated once and separated by a linker. Given the length limitation on solid phase peptide synthesis with regard to yield and purity, we focused on the 1542 APRs with a length of 7 amino acids. In order to maximize the potential of these peptides to induce aggregation in E. coli, we ranked the APRs by their frequency of occurrence throughout the E. coli proteome, allowing a single amino acid mismatch, and selected the first 75 most frequently occurring sequences from this list (Table 4). In the design pattern, and in order to increase the colloidal stability of these aggregating peptides, the APRs are flanked by aggregation gatekeepers, a class of residues that was previously shown to reduce aggregation kinetics22-24. Since positively charged residues have been shown to help bacterial uptake25, we selected arginine to obtain the following peptide layout: R-APR-RR. To generate the tandem we used a single proline residue as a linker between the gatekeeper-flanked APRs. In addition to the 75 peptides generated in this fashion, we added 2 variants of each of the first 25 peptides in the list by randomly mutating one residue in the first APR repeat to arginine to further modulate the aggregation propensity of the peptides (Table 4). All the peptides were generated using solid phase synthesis at 200 nmole scale and dissolved in DMSO to a theoretical stock concentration of 2 mM (assuming 100% synthesis efficiency). Peptide activity on the growth of E. coli O157:H7 was measured at dilutions of the peptide corresponding to concentrations of 1, 6, 12 and 25 μg/mL. Although no peptide was able to inhibit bacterial growth at the highest dilution, 43 of them were active against E. coli O157 at 25 μg/mL, of which 11 were active at 12 μg/mL and 6 had an apparent Minimum Inhibitory Concentration (MIC) value of 6 μg/mL (Table 5). We used the CAMP software prediction algorithm, which is trained to identify known AMPs26, to analyse our sequences and found that 90% were predicted to be antibacterial, irrespective of whether we found them to be active or not (65% were inactive). The Matthews Correlations Coefficient (MCC) with the measured activity was 0.1 at 12 μg/mL and 0.24 at 25 μg/mL. This indicates that the key properties of known AMPs, such as hydrophobicity and charge, captured by a machine learning algorithm are not sufficient to predict the antibacterial activity of aggregating peptides and that our peptides work by a mechanism not incorporated in the current prediction algorithms.

2. Activity and Selectivity of E. coli Derived Aggregating Peptides

We selected, resynthesized and HPLC purified several peptides from the screen, i.e. P2 (RGLGLALVRRPRGLGLALVRR, SEQ ID NO: 1), P5 (RALLTTLLRRPRALLTTLLRR, SEQ ID NO: 3), P14 (RGLLALLARRPRGLLALLARR, SEQ ID NO: 6) as well as P105 (RALLRTLLRRPRALLTTLLRR, SEQ ID NO: 5) and confirmed Minimum Inhibitory Concentration (MIC) and Minimum Bactericidal Concentration (MBC) values of HPLC-grade purified peptides to be 6-12 μg/mL (see Table 1). Analysis of the rate of peptide bactericidal activity against E. coli O157:H7 (at MIC concentration) showed that the peptides exerted full bactericidal effect within 30 min to 2 h (FIG. 1A). Cross-section transmission electron microscopy of peptide-treated bacteria revealed the widespread presence of large inclusion bodies, a hallmark of protein aggregation in E. coli, suggesting that the peptides act by interfering with bacterial proteostasis (FIG. 1B, C, D). These inclusion bodies, which are also called Large Polar Aggregates, are mostly located at the poles of the bacterial cells, as expected for these structures due to nucleoid occlusion27. Interestingly, when bacteria were repeatedly passaged on sublethal concentrations (50% of MIC) of the active peptides for a period of 36 days, no development of resistance was observed, whereas this was the case for the control antibiotic ampicillin (FIG. 1E). As a first indication of specificity of the peptides, we evaluated their hemolytic activity on human erythrocytes (FIG. 1F), which revealed the P2 peptide (RGLGLALVRRPRGLGLALVRR, SEQ ID NO: 1) to have a favourable, (particularly, the most specific) toxicity profile towards E. coli strain O157:H7. This was further confirmed by CellTiter Blue (FIG. 1G) and LDH release (FIG. 1H) assays on HeLa cells. The specificity of P2 for E. coli O157:H7 was estimated by determining the concentration at which bacterial growth is 50% inhibited (IC50=1.5 μg/mL) and compared to the concentration at which the peptide induces 50% lysis of human erythrocytes (LC50=1100 μg/mL) yielding a therapeutic ratio of 730. Based on this observation, P2 was selected for further characterization and will henceforth be called colpeptin1. As a control, we generated mutants of colpeptin1 in which we introduced proline substitutions at different positions in the APRs (Table 1), which conserves the hydrophobicity but disrupts the beta-sheet propensity and hence reduces the aggregation propensity of the peptides. Due to the conservation of hydrophobicity and charge of the mutant peptides, the CAMP prediction software again classifies these controls as antimicrobial. However, when we treated bacteria with the control peptides, we obtained MIC values of more than 200 μg/mL, indicating that beta-structure formation is key to the antimicrobial activity of our peptides and confirming that the mode of action of other AMPs is not predictive for the antibacterial activity observed with our sequences. In a next step we evaluated the activity of colpeptin1 against various bacterial strains and correlated this with the sequence conservation of the HcaB protein present in the bacterial strains. The data are shown in Table 2.

3. Colpeptin1 Forms Soluble Oligomeric Beta-Structured Aggregates that Mature into Amorphous Aggregates

Analysis of colpeptin1 by electrospray ionisation-mass spectrometry linked to ion mobility spectrometry (ESI-IMS-MS)28 revealed that immediately following solubilisation of colpeptin1 in 100 mM ammonium acetate buffer the peptide is not only monomeric, but also readily forms soluble oligomers ranging from dimers up to 9-mers, and likely higher order (FIG. 1I). Consistent with this, the main species observed by Dynamic Light Scattering (DLS) upon dissolving have apparent hydrodynamic radii of approximately 1-2 nm, which quickly grow to large particles within a few hours (FIG. 1J). The composition of the colpeptin1 solution evolves over time towards larger species, which in the mass spectrometer is paralleled by a consumption of the smaller species (the larger being outside the detection range of the instrument). A study of the solubility of the peptide over time using ultracentrifugation also shows aggregation, with less than 60% of the peptide remaining in solution by ultracentrifugation 30 min after dissolving (FIG. 1K). The insoluble fraction collected in this manner was brought back into suspension (in 10% of the original volume) and the secondary structure content was analysed using Fourier Transform Infrared Spectroscopy (FTIR), which shows major peaks around 1622 and 1641, consistent with beta-structure formation (FIG. 1L). Mature aggregates were only mildly positive for the amyloid-specific dye pentameric formyl thiophene acetic acid (p-FTAA)29-31 (FIG. 1M). By transmission electron microscopy mature aggregates mainly formed amyloid-like aggregates, which occasionally assembled into ordered fibrils (FIG. 1N). Taken together, these features are consistent with colpeptin1 forming β-structured soluble oligomers that slowly convert to insoluble fiber-like aggregates. Upon exposure of Colpeptin1 to polyphosphate, a naturally abundant form of phosphate in E. coli that was previously shown to facilitate amyloid formation32, the peptide displays typical amyloid-aggregation kinetics as measured using pFTAA fluorescence (FIG. 1O). An analysis of a proline substituted and inactive control of colpeptin1 (termed P-colpeptin1) showed that upon solubilisation under identical conditions, P-colpeptin1 is almost completely soluble and does not adopt a beta-structured conformation, although sedimentation analysis still reveals aggregation on a longer timescale (FIGS. 1K&L).

4. Colpeptin1 Displays In Vivo Activity Against E. coli in a Bladder Infection Model

In order to test the in vivo potential of colpeptin1 we treated a co-culture of mammalian (HeLa) cells and E. coli O157:H7 with colpeptin1 and observed the preferential accumulation of colpeptin1 in bacteria but not in mammalian cells (FIG. 2A). Finally we found that colpeptin1 incubated in 25% or 50% human serum for 2 hours was still able to inhibit bacterial growth at 25 μg/mL and 50 μg/mL (FIG. 2D). Given these positive results, we then established the tolerance of the Swiss mice to the peptide by performing a dose escalation experiment. We observed no acute adverse effects across the entire concentration range to the basic physiological and behavioral parameters of the animals upon intraperitoneal administration of up to 30 mg/kg colpeptin1. The parameters observed included body weight, food and water consumption, home cage activity and locomotion. We subsequently treated a cohort of 6 mice with daily injections at the maximum tolerated dose for 18 days without any apparent adverse effects. The mice were allowed to recover for 3 days, at which point they were sacrificed and a survey of the major organs was undertaken, which revealed no major morphological signs of toxicity (Tables 6, 7 and FIGS. 5, 6). In addition an ELISA assay on the serum of these animals revealed no specific antibody response towards the colpeptin1 peptide (FIG. 2E). A limited biodistribution study in healthy animals following injection (IP or urinary tract) of a single dose of 10 mg/kg FITC-labelled colpeptin1 revealed a clear distribution of fluorescent material for at least 3 h after injection and a total clearance of the peptide after 24 h (FIG. 7). Based on these observations, we tested the antibacterial efficacy of the colpeptin1 in a mouse bladder infection model. In this model, an inoculum of 50 μL of a 108 CFU/mL suspension of E. coli O157:H7 was delivered via the urethra to the bladder of healthy Swiss mice. 1 h post-infection, we administered a single injection of colpeptin1 at 10 mg/kg, either via the urethra (n=15) or intraperitoneally (n=15). 24 h after treatment, the animals were sacrificed and the bacterial titer in kidney, colon, bladder and ureter was determined by plating the macerated tissue (FIG. 2F-I). These experiments revealed a significant reduction of the bacterial titer in the different organs of colpeptin1 treated animals of more than 2-log fold, both after intraperitoneal and urethral delivery (p-val <10−4 compared to buffer control & p-val <10−4 compared to non-aggregating P-colpeptin1 control, anova with Tukey post-test). The effect was comparable to that of orally dosed ampicillin (20 mg/kg) that was used as a control, indicating that the antimicrobial activity of colpeptin1 against E. coli is maintained in vivo.

**5. Colpeptin1 Uptake Results in IB Formation and Growth Inhibition**

To study its mode of action, we derivatized colpeptin1 with fluorescein isothiocyanate (FITC) and established that the conjugate retained its antibacterial activity (MIC=3 μg/mL against E. coli O157: H7) and quantified colpeptin1 uptake by flow cytometry. Analysis of colpeptin1 uptake by E. coli O157:H7 showed that after 15 minutes already, 97.7±2.9% (N=4) of the cells are positive for FITC (FIGS. 3B&F), increasing to nearly 100% after 1 h and beyond (FIG. 3C-F). In parallel, fluorescence microscopy of treated E. coli O157:H7 at MIC concentration confirmed no enrichment at the cell membrane of FITC-colpeptin1, but rather showed a clear accumulation of fluorescence in intracellular polar inclusion bodies (IBs) from 15 minutes onwards (FIG. 3G) that persisted at later time points (FIG. 3H). In addition, colpeptin1 induced IBs could be stained with p-FTAA, a dye that specifically binds to amyloid-like aggregates as well as disease-associated protein inclusion bodies31, confirming the ordered beta-sheet rich aggregated protein structure of these inclusions (FIGS. 3I&J). This demonstrates that colpeptin1 uptake and IB formation occur in close succession. Kinetics of bacterial cell death as measured by CFU determination after colpeptin1 treatment (FIG. 3K) also closely follow peptide internalization and coincide with the appearance of IBs after 15 min treatment (50% after 15 min). On the other hand, bacterial cell death as monitored by Propidium Iodide (PI) uptake as a result of membrane permeabilisation increased more slowly (2.1±1.3% after 15 min to 85±13.2% after 3 h, FIGS. 3A-E & 3L), showing that at short treatment times, significant growth inhibition is established coincidentally with IB formation but before membrane permeability can be observed. In agreement with this, morphological analysis of bacteria treated at 4×MIC for 2 h and untreated controls using Scanning Electron Microscopy (SEM, FIG. 3M, N, O) shows that colpeptin1-treated bacteria appear shrunken but no cellular leakage is apparent. Together these data suggest a chain of events in which colpeptin1 internalisation is coincidental with colpeptin1-containing IB formation and bacterial growth inhibition.

**6. Colpeptin1 Induces Lethal Bacterial Proteostatic Collapse**

Bacterial IB formation is a common event associated with cellular stress including exposure to heat and recombinant protein (over)expression. This process however is often transient and reversible and does not necessarily lead to bacterial cell death. In fact, recombinant protein production in bacteria relies to a large extent on the ability of bacteria to cope with IBs. As an example, we measured the consequences of overexpressing the highly aggregation-prone core domain of the human p53 protein (p53CD) on growth (FIG. 4A) of E. coli BL21 cells, which are routinely used for recombinant protein production. Although p53CD expression resulted in a delay of the exponential growth phase, consistent with cellular stress resulting from the overexpression, there was no effect on colony formation, showing the stress is not lethal. In order to understand why colpeptin1 induced IB formation is irreversibly toxic we compared the composition of IBs purified from E. coli O157:H7 cells treated with colpeptin1 at MIC concentration for 1 h with IBs purified from E. coli strain BL21 overnight overexpressing p53CD. Inspection of the resulting samples by TEM confirmed the successful purification of these IBs (FIG. 4C). The composition of IBs was subsequently analysed by coomassie-stained SDS-PAGE (FIG. 4D). The overall pattern of coomassie staining revealed that a large number of similar bacterial proteins are trapped in the IBs of both colpeptin1 treated E. coli O157:H7 and p53 overexpressing E. coli BL21, but not in untreated bacteria suggesting a common molecular machinery associated with IB formation. Among the IB trapped proteins, a number of molecular chaperones that are known to occur in inclusion bodies could be detected, including the bacterial Hsp70 homolog DnaK, the Hsp60 chaperonin GroEL, the ribosome associated chaperone Trigger Factor (TF) and the bacterial Hsp40 DnaJ (FIG. 4E). The polar localization of a fluorescently traceable DnaK-mCerulean3 fusion protein (the latter moiety comprising a blue fluorescent protein) in E. coli K12 MG1655 cells exposed to colpeptin1 confirms the association of DnaK with IBs (FIG. 4F). Our data show that colpeptin1 and p53CD IBs share a number of proteins, including many chaperones and ribosomal proteins which are common constituents of IBs. Second, in addition to this common core, we find that colpeptin1 IBs contain many more additional proteins than p53CD IBs. This shows that the toxic impact of colpeptin1 treatment corresponds to a more extensive proteomic impact than the effect of p53CD overexpression. This observation is in line with our initial design hypothesis aiming at inducing proteostatic collapse by aggregation of multiple proteins.

**7. Colpeptin1 Induces the Co-Translational Aggregation of Direct Sequence Targets**

In accordance with our design, the colpeptin1 APR is highly redundant in the E. coli proteome and can also be found in 18 other E. coli proteins with a single mismatch (Table 8) and 158 proteins with a double mismatch. In the case of colpeptin1, its direct APR match is the G17LGLALV24 sequence from HcaB. HcaB (3-phenylpropionate-dihydrodiol/cinnamic acid-dihydrodiol dehydrogenase) is a non-essential enzyme in the aromatic compound metabolism that occurs at very low abundance (0.015 ppm according to PaxDB33). Taguchi and co-workers have previously determined the solubility of the entire E. coli proteome including HcaB using a cell-free translation system showing that HcaB is a highly aggregation prone protein that is highly dependent on GroEL/ES for folding34. In a next step we cloned HcaB into an inducible vector for recombinant expression and chromatographically purified the protein from lysates of E. coli BL21:DE3 cells overexpressing HcaB. Using this material we performed an immunisation scheme in mouse (see materials and methods), yielding antiserum against HcaB in a Western blot, revealing a band at the right molecular weight, as well as 2 off-target bands (FIG. 4H). Using this antiserum, we compared the effect of colpeptin1 treatment (1 h at MIC concentration) on the presence of HcaB in the soluble and insoluble fraction of E. coli O157:H7, endogenously expressing HcaB, and E. coli BL21 cells overexpressing HcaB (FIG. 4H), confirming the accumulation of HcaB in the inclusion body fraction of colpeptin1 treated cells. This confirms that HcaB does aggregate upon treatment with colpeptin1. When calculating the translational efficiency of these genes based on Tuller's method35, which uses typical decoding times of individual codons, we found that the translational efficiency of the colpeptin1 target proteins was significantly higher (student t test, p<0.001, FIG. 4J) than for undetected putative targets, suggesting that a high translation rate might facilitate colpeptin1 induced aggregation. To verify this hypothesis, we measured the MBC value of colpeptin1 in the presence of the macrolide antibiotic erythromycin, which is a bacteriostatic drug that acts by blocking the polypeptide exit channel in the ribosome. Strikingly, we observed a marked desensitization of bacteria (E. coli O157) (MBC >100 μg/mL) to colpeptin1 after pretreating the cells with 100 μg/mL erythromycin for 2 h to block translation during the exposure to colpeptin1, strongly supporting a co-translational induction of protein aggregation as the mode of action.

The colpeptin1 APR is highly redundant in the Acinetobacter baumannii proteome and can also be found in 18 other A. baumannii proteins with a single mismatch and 268 proteins with a double mismatch (Table 9), supporting the observed bactericidal activity of colpeptin1 against both E. coli and A. baumanii. (FIG. 9)

**8. Colpeptin1 Induces a Multi-Target Aggregation Cascade Leading to Proteostatic Collapse**

The question then remains how the aggregation of colpeptin1 and its detected targets relates to the aggregation of the other proteins found in colpeptin1 IBs. A first possibility is that colpeptin1-induced proteostatic collapse results from saturation of bacterial chaperones. In order to test this possibility we determined the effect on colpeptin1 activity of major bacterial chaperones and proteases in the E. coli K-12 BW25113 strain (KEIO collection37) and found that from the individual knockouts of the principal proteostatic components of E. coli, only DnaK had a mild effect on colpeptin1 activity (Table 3). Furthermore, previously designed peptides that induce lethal aggregation in Staphylococcus epidermidis19, using tandem repeats of strain specific APRs flanked by positive charges are not active against Gram-negative E. coli and vice versa even though these peptides are internalized. Together these data suggest that saturation of chaperones and the proteostatic system by aggregating peptides is not the principal mechanism of action of colpeptin1. Alternatively, and in line with our hypothesis, APR redundancy drives proteostatic collapse by a sequence-specific cascade of protein aggregation. We reasoned that the primary colpeptin1 targets have additional APRs in their sequence and these form colpeptin1-induced co-aggregated proteins which are also present in these IBs, which proteins on their own can be further connected by an aggregation cascade determined by secondary APRs in primary colpeptin1 targets. Going one step further, still an additional number of proteins can be connected in a similar manner by a third layer, finally leaving few proteins unconnected in the cell.

To compare, the p53CD has one dominant APR according to TANGO (ILTIITL, SEQ ID NO: 223)38, which has no exact match in the proteome of E. coli O157:H7, but it has 3 proteins that have an APR with one mutation, and 50 proteins with an APR that is 2 mutations away. None of these proteins could be detected in the IBs (data not shown), demonstrating that p53CD aggregation is a much more isolated event in sequence space. The proteins detected in IBs of p53CD on the other hand consist mainly of chaperone clients39-46, confirming that p53CD aggregation constitutes a proteostatic stress but which contrary to colpeptin1 does not cause a proteostatic collapse and is not lethal to the bacterial cell.

**9. Determination of the MIC Values for 28 Different Peptides Against a Set of Bacteria**

In this experiment a set of 28 peptides of which the amino acid sequences are depicted in Table 4, were administered to the E. coli BL2 strain, Acinetobacter baumannii, Klebsiella pneumoniae and Pseudomonas aeruginosa. The MIC values were determined and are depicted in FIG. 8.

10. Colpeptin1 has Broad within Species Reactivity and is not Affected by the Pathogen Resistance Profile

In this experiment we assessed the potency of Colpeptin1 against a number of clinical isolates of E. coli and a number of clinical isolates of Acinetobacter baumannii (including 2 reference strains). FIG. 9 (upper panel) lists the MIC values of Colpeptin1 (indicated as P2 in the FIG. 8) against a number of clinical isolates of E. coli. FIG. 9 (lower panel) lists the MIC values of Colpeptin1 (indicated as P2 in the FIG. 9) Sensitivity (S) or resistance (R) or intermediate resistance (I) of the clinical isolates against 12 different antibiotics is depicted in the panel (data according to CLSI criteria for the specific antibiotics). Our data show that that colpeptin1 has a broad within species activity and that the MIC values are not affected by the antibiotic pathogen resistance profile, both for E. coli and Acinetobacter baumannii.

## Materials and Methods

### 1. Bio-Informatics Analysis

Protein sequences for various bacterial strains were obtained from UniProt (Nucleic Acid Res. (2008) 36, D190-5), and redundancy was removed using the cd-hit algorithm (Fu L. et al (2012) Bioinformatics 28, 3150). We used the TANGO algorithm for all APR identifications in this manuscript. We used a cutoff on the TANGO score of 5 per residue since this gives a Mathews Correlation Coefficient between prediction and experiment of 0.92 (Fernandez-Escamilla A M et al (2004) Nat. Biotechnol. 22, 1302). The settings of TANGO were Temperature=298K, pH=7.5, Ionic Strength=0.10 M.

### 2. Peptide Synthesis

During the screening stage peptides were synthesized using standard solid-phase peptide synthesis (JPT, Berlin, Germany). Peptide hits were resynthesised in-house at higher scale using an Intavis Multipep RSi synthesis robot and HPLC purified to 95% using Zorbax SB-C3 semi-preparative column (Agilent, USA) on a Prominence HPLC (Shimadzu, Japan). Peptides were lyophilized and stored at −20° C. prior to use.

### 3. Bacterial Strains and Growth Conditions

Gram negative bacterial strains were cultivated in Luria-Bertani (LB) broth (Difco) and gram positive bacteria strains were grown in a rich medium, brain heart infusion (BHI) broth (Difco, Sparks, Md.) at 37° C. Whenever required, growth media were supplemented with appropriate antibiotic to the medium or plates. Escherichia coli DH5a was used for cloning and plasmid amplification. For selection of antibiotic resistance colonies, E. coli carrying plasmid were grown in LB medium supplemented with 25 μg/mL ampicillin or 100 μg/mL Erythromycin (sigma). Bacterial CFU counting was done on blood agar plates (BD Biosciences). Species identification and antibiograms for all clinical isolates were performed using MALDI-TOF and VITEK@2 automated system (bioMerieux).

### 4. MIC Determination

The MICs of active peptides were determined via the Broth microdilution assay according to the EUCAST guideline, which were performed in 96-well polystyrene flat-bottom microtiter plates (BD Biosciences). Briefly, a single colony was inoculated into 5 mL LB medium and grown to the end-exponential growth phase in a shaking incubator at 37° C. Cultures were subsequently diluted to an OD600 of 0.002 (1×108 CFU/mL) in fresh LB medium. 100 μL of LB medium with different concentration of peptides ranging from 100 μg/mL to 1 μg/mL were serially diluted to the sterile 96-well plate (at least 3 wells in each plate). Afterward, 100 μL of the diluted bacteria were pipetted into 96-well plates containing different concentration of peptides. In each plate, the grown bacteria with maximum concentration of carrier and medium were considered as positive and negative controls, respectively. Thereafter, 96-well plates was statically incubated overnight at 37° C. to allow bacterial growth. A multipurpose UV/VIS plate reader at OD590 nm, 1 s shaking measured the absorbance of the growth bacteria using a Perkin Elmer spectrophotometer (1420 Multilabel Counter Victor 3).

### 5. Antibody and Antibiotic Product Codes

Anti-CLPB (Aviva, Catalog #ARP53790_P050), Anti-DnaK (Aviva, Catalog #OAED00201), Anti-Trigger Factor (Clontech, Catalog #M201), Anti-groEL (Abcam, Catalog #ab82592), Anti-DnaJ (Enzo Life Sciences, Catalog #ADI-SPA-410-D). Ampicillin sodium, CAS number 69-52-3, Duchefa Biochemie, catalog #A0104. Erythromycin, CAS Number 114-07-8, Sigma Aldrich, CAS Number 114-07-8, catalog #E5389.

### 6. Biophysical Characterization

Dynamic light scattering (DLS) measurements were made at room temperature with a DynaPro DLS plate reader instrument (Wyatt, Santa Barbara, Calif., USA) equipped with a 830 nm laser source. Samples (100 μL PBS buffer, 2 mM peptide) were placed into a flat-bottom 96-well microclear plate (Greiner, Frickenhausen, Germany). The autocorrelation of scattered light intensity at a 90° angle was recorded for 10 s and averaged over 40 recordings to obtain a single data point. The Wyatt Dynamics software was used to calculate the hydrodynamic radius by assuming a spherical particle shape. Attenuated Total Reflection Fourier Transform Infrared Spectroscopy (ATR FTIR) was performed using a Bruker Tensor 27 infrared spectrophotometer equipped with a Bio-ATR II accessory. Spectra were recorded in the range of 900-3500 cm−1 at a spectral resolution of 4 cm−1 by accumulating 120 data acquisitions. The spectrophotometer was continuously purged with dried air. Spectra were corrected for atmospheric interference, baseline-subtracted, and rescaled in the amide II area (1500 to 1600 cm−1). For Transmission Electron Microscopy (TEM) aliquots from peptide preparations were adsorbed to carbon-coated Formvar 400-mesh copper grids (Agar Scientific) for 1 min. The grids were blotted, washed, and stained with 1% (wt.vol−1) uranyl acetate. Samples were studied with a JEOL JEM-1400 microscope (JEOL Tokyo, Japan) at 80 kV.

### 7. Time Killing Kinetic Assay

The time-kill kinetics study of the peptides was carried out to assess the killing rate of the bacteria at enough exposure time points. This study was done according to standard guide for assessment of antimicrobial activity using time-kill kinetics procedure. Selection of agent concentrations was guided by MIC endpoints.

Briefly, 20 μL of frozen cultures of E. coli O157:H7 were inoculated into 5 mL LB and grown to the end-exponential growth phase in a shaking incubator at 37° C. Cultures were subsequently diluted to an OD600=0.002 (1×101 CFU/mL) in fresh LB medium (1 mL). To evaluate the effect of aggregators over time, bacteria were subjected to a concentration of different peptides at the MIC value for different periods of time (5 min, 10 min, 30 min, 1 h, till 6 h). After the defined contact period, 50 μl of each culture were serially diluted and plated on blood agar plates. Plates were overnight incubated at 37° c. without shaking. Positive and negative controls were bacteria treated with maximum used buffer and the LB medium, respectively. The number of viable organisms was counted as CFU/ml.

### 8. Multistep Resistance Development Study

The ability of the target strains to develop resistance to active compounds was evaluated by repeated subculturing in the presence of the half-MIC value of the active peptides over 30 days. Briefly, E. coli O157 cultures were grown in Luria Broth (LB) medium; the optical density of bacteria was then adjusted to an OD600 of 0.002 (equivalent to 1×108 CFU/mL). Then, bacteria were treated by the P2 peptide at half-MIC concentration, after a 24 h incubation period, the MIC's were tested by a micro-dilution assay according to the EUCAST guideline and the bacteria re-cultured in the presence of the half-MIC value of the respective aggregator. Ampicillin was used as the positive control in this experiment.

### 9. Scanning Electron Microscopy

For scanning electron microscopy (SEM) E. coli O157 bacteria in end-exponential growth phase were diluted to a density of 108 CFU/mL and treated with supra-MIC concentrations of peptides. After 2 h treatment, bacteria were fixed with 2% glutaraldehyde for 1 hour. 1% Osmium tetroxide (OsO4) was used as postfixation in 0.1 M sodium cacodylate buffer for 1 h. Samples were washed three times with cacodylate buffer (0.1 M sodium cacodylate) for 10 min at room temperature. The samples were dehydrated with a graded ethanol series (50%, 70%, 96%, 100% alcohol). After the dehydration step, samples were dried by Hexamethyldisilazane (HMDS) for 1 h, and mounted on the specimen stubs and sputter coated with gold. A SEM-FEG microscope (JEOL JSM 6700F) with an accelerating voltage of 30 kV was used.

**10. Cross-Section Transmission Electron Microscopy**

E. coli at the end-exponential growth phase were washed twice and diluted with physiological water and subsequently treated with either 4×MIC value of specific aggregator peptides (Colpeptin1 or P2Pro2) or buffer for 2 h (Control group) at 37° C. After 2 h, bacteria were centrifuged at 6000 rpm for 4 min and pellets were fixed by 2.5% glutaraldehyde in 0.1M Na-cacodylate buffer pH=7.2-7.4 [+2.5 mM CaCl2+1 mM MgCl2] for 1 h. Then, the pellets were washed with cacodylate buffer, resuspended in 1.5% low melting point agarose (Sigma A4018) in cacodylate buffer (40° C.) and centrifuged at 6000 rpm, 4 min. The centrifuge tubes were placed on ice for 15 minutes, after which the tips containing the pellets were cut off and the pellets removed in a drop of cacodylate buffer. Pellets were cut in 1 mm3 cubes (4° C.), post-fixed with 1% Osmium tetroxide (OsO4) in distillate water for 2 h and washed twice with distillate water. Thereupon the samples were dehydrated in a graded ethanol series (30, 50, 70, 90, 100%) 5 min each step at 4° C. while slowly rotating (ethanol 100%, 3 times repeated). Finally, cells were treated by propylene oxide twice 15 min at 4° C., infiltrated with a 1:1 mixture of epoxy resin and propylene oxide (60′ @ 4° C., slowly rotate) and subsequently left in a mixture of 2:1 epoxyresin and propylene oxide overnight under a fume hood without caps. The next morning, samples were placed in 100% fresh epoxy resin, embedded in BEEM capsules in the evening and polymerized for 2 days in an oven at 60° C. Ultrathin sections were cut with a Leica ultracut UCT ultramicrotome and observed in a JEOL JEM1400 transmission electron microscope operated at 80 kV and equipped with an Olympus Quemesa 11Mpxl camera.

**11. In Vitro Haemolytic Activity Test**

The hemolytic activities of peptides were determined by hemolysis against human erythrocytes. Pooled fresh blood was obtained and erythrocytes were collected by centrifugation 3000 rpm for 5 minutes (Anticoagulated by EDTAK) (Cristina et al. 2015). The pellet was washed three times with PBS and was diluted to a concentration of 8% in PBS. 100 μL of 8% Red blood cells solution was mixed with 100 μL of serial dilutions of peptides in PBS buffer in 96 well plates (BD Biosciences). The reaction mixtures were incubated for 1 h at 37° C. Thereupon, the plate was centrifuged for 10 minutes at 3000 rpm and 100 μL of supernatant was transferred to a sterilized 96-well plate (flat bottom). The release of hemoglobin was determined by measuring the absorbance of the supernatant at 405 nm. The hemolytic activity was determined as the minimal peptide concentration that caused hemolysis (minimal hemolytic concentration, MHC). Erythrocytes in 1% Triton and Max used concentration of vehicle were used as control of 100% and 0% hemolysis, respectively.

**12. In Vitro Mammalian Cytotoxicity**

Mammalian cytotoxicity was measured using the LDH release (Roche, Mannheim, Germany) and CellTiter Blue (Promega) methods. Briefly, HeLa cells (obtained from Bart De Strooper lab, tested to be mycoplasma free) were seeded in 96-well round bottom plates at a concentration of 3×105 cells/mL in Dulbecco's Modified Eagle's Medium and treated by different concentrations of peptides. Cells treated with 1% Triton™ X-100 and vehicle were considered as positive and negative controls, respectively. Micro-plates were incubated at 37° C., 5% CO2 and 90% humidity for 4 hours. The micro-plate was centrifuged at 1350 rpm for 10 minutes. 100 μL of supernatant was transferred into clear 96-well flat bottom microplates. In order to determine LDH activity in the supernatants, 100 μl reaction mix (catalyst and dye solution) were added to each well and incubated for 30 min at RT in the dark and the LDH reaction was stopped by adding 100 ul of the Stop solution. The absorbances of the samples were measured at 490 nm. The cell viability was calculated using the formula: (exp.value−negative control value)/(positive control value−negative control value)*100. The amount of absorbance is proportional to the number of living cells and corresponds to the cells' metabolic activity.

13. Cloning and Expression of the hcaB Gene in E. coli ATCC 25922 Strain

The coding regions of E. coli O157 HcaB were amplified using hcaB specific primers (ATGTCGACATGAGCGATCTGCATAACGA (SEQ ID NO: 224), ATGTCGACATGGAGCGATTTATCGAAGAAGGC (SEQ ID NO: 225), ATCCCGGGTTAAAGATCCAACCCAGCCG (SEQ ID NO: 226)) containing additionally a SalI and SmaI restriction site for cloning purposes. Two truncates versions of the gene, one with the targeted gene part, APR, and the other one without the target region were designed. Genomic DNA (gDNA) of bacteria E. coli O157 strain, a clinical isolate was used as a template. The amplicons were ligated into SalI/SmaI-digested pCN68 E. coli-Staphylococcus shuttle vectors yielding different truncated version pCN-hcaB. In this plasmid, PblaZ is the promoter. Ampicillin (25 μg/mL) or erythromycin (100 μg/mL) was used as the selection marker. Correctness of cloning was confirmed first by restriction enzyme digestion, PCR, and nucleotide sequence analysis of the insert and then sequencing.

**14. Macrolides and Peptides Interaction**

To evaluate the effect of peptides in the presence of Erythromycin, E. coli O157 was grown in 5 ml of LB (Luria-Bertani). Exponential-phase cultures were then diluted to 108 cell/ml. Bacteria was treated by Erythromycin at the concentration of 100 ug/ml for 3 h to stop growing at 37° c., without shaking. Different concentrations of peptides (from 100 ug/ml to 0.75 ug/ml) or buffer were plated in 96 well with at least 3 times replicate wells (50 ul). 50 ul of erythromycin treated bacteria was added to each wells and 96 well plate was incubated for 2 h at 37° C. After 2 h, bacteria was serially diluted and cultured on blood agar plates. Plates were incubated at 37° C., overnight. The number of living cells were quantified by CFU counting.

Fluorescence Microscopy of Co-Cultures of Bacteria and Mammalian Cell Conjugated with FITC Labeled Peptides

For imaging purposes, human HELA cells were grown on small cell view cellular dish with glass bottom (Greiner Bio-One/GmbH/35 mm ref: 627860) to form a confluent monolayer. Thereupon, cells were infected with 200 μL of overnight culture of E. coli O157 strain with FITC peptide (3×MIC) for 24 h. Cells were stained with CellMask Deep Red plasma membrane dye and 1 μl of NucBlue reagent for 30 min (Invitrogen), then medium was removed and 2 ml paraformaldehyde 4% was added to the plate for fixation. Plate was incubated for 6 h at RT. The co-cultured cells were washed with 3× with 1 mL saline prior to imaging.

15. Staining with Luminescent Conjugated Oligomers (LCOs)

200 μL of end-exponential culture E. coli O157 were washed with PBS for three times, the bacterial numbers were adjusted to 108 cells and afterward bacteria were treated with peptides (at MIC) or Pro2 peptides as control for 2 h. After 2 h, cells were incubated with 1 μL of the LCO dye p-FTAA for 1 h and 30 min. The absorption and emission spectra were measured at 480 to 600 nm.

**16. Flow Cytometry Analysis of Bacteria Using Labeled Peptides and Propidium Iodide (PI)**

Using a double staining technique with propidium iodide (PI) and FITC peptides, killing rate and peptides uptake were evaluated in a two dimensional analysis. Briefly, end-exponential growth phase E. coli O157 cells (108 CFU/mL) were washed with PBS and treated by peptides (Colpeptin11 or Pro2/FITC labeled) at MIC value for different time periods. Treated bacteria were again washed with PBS buffer three times. 1 ul of PI (Invitrogen) was added to the bacteria, and after incubated for 5 min, the mixture was aliquoted (500 μL) into FACs tubes. To correlate the activity of the peptides with cell death, the fluorescence intensity was measured in two channels using the Gallios™ Flow Cytometer, PI: excitation 536 nm and emission 617 nm, FITC: excitation 490 nm and emission 525 nm.

**17. Inclusion Body Purification**

20 mL of overnight culture of bacteria was centrifuged for 30 min at 6000 rpm and washed by physiological water. Bacteria were treated by peptide at MIC for half killing time, afterward the bacterial pellets were washed by 10 mL buffer A (50 mM Hepes pH 7.5, 300 mM NaCl, 5 mM Beta-Mercaptho ethanol, 1.0 mM EDTA) and centrifuged at 4° C. for 30 min at 6 k rpm. The supernatant were discard and 20 mL of Buffer B (Buffer A plus 1 μg/mL Leupeptin, 0.1 mg/mL AEBSF) was added to the bacterial pellet. In order to break the cells, a Glen Creston Cell Homogenizer with pressure set to 20000-25000 psi was used and in addition, the suspensions were sonicated (Branson Digital sonifier 50/60 HZ) on ice with alternating 2 min (15 pulse at 50% power with 30 sec pause on ice, until completing 2 min total sonication time). The lysed cells were centrifuged at 4° C. for 30 min at 10 k rpm. The precipitated fraction was afterwards re-suspended with 10 mL Buffer D (Buffer A plus: 0.8% V/V Triton X-100, 1% Sodium Deoxycholate) and the suspension was sonicated to ensure the pellet is completely dissolved. This step was repeated 3 times. Centrifugation was performed at 4° C. for 30 min at 10.000 rpm. Finally to solubilize IB, the pellet was suspended in 1 mL of buffer F (50 mM Hepes pH 7.5, 8.0 M urea) per gram of precipated fraction.

**18. Peptides Activity and Stability in Presence of Serum**

Briefly, in order to get human serum, the fresh blood was kept at RT (room temperature) for 20 min to be clot. The tube was then centrifuged at 3000 pm for 10 min and serum separated. Serum was diluted into RPMI medium (50%) and peptides with different concentration of 5 μg/mL, 25 μg/mL and 50 μg/mL were added to each well. After 2 h incubation, End-exponential E. coli O157 culture washed three times by PBS (8000 rpm, 10 min). The number of bacteria then adjusted to 9×108 cells in RPMI medium with or without Serum. After 2 h incubation, bacteria were serially diluted and were cultured on blood agar plats. Plates were then incubated at 37° C. overnight. The numbers of living bacteria were quantified as the number of CFU/ml (colony-forming units).

**19. HcaB Purification and Antibody Production**

The coding whole E. coli hcaB gene (SE2232, Taxonomy ID: 83333) was amplified using hcaB-specific primer (ATGCATCATCACCATCACCACAGCGATCTGCATAACGA (SEQ ID NO: 227) and TTAAAGATCCAACCCAGCCG (SEQ ID NO: 228)) with additionally NdeI and BamHI restriction site (underlined and italic) for cloning purposes. Anti-6×His (bold) tag used as a tag on the recombinant proteins to facilitate protein purification.

Genomic DNA (gDNA) of E. coli strain O157 was used as a template. The amplicon was ligated into NdeI/BamHI-digested pET11C plasmid as a component of a system for protein expression in E. coli yielding pET-his-hcaB. In this plasmid, T7 RNA polymerase (highly active constitutive promoter) was the promoter. Ampicillin was used as the selection marker. All recombinant plasmids were replicated in E. coli BL21 to have BL21 pET-his-hcaB. Correctness of cloning was confirmed by restriction enzyme digestion, and nucleotide sequence analysis of the insert.

Protein purified as described previously (Luminy and Cedex 2011). Briefly, E. coli BL21 pET-his-hcaB was grown in 1L Luria-Bertani (LB) broth with 100 μg/ml ampicillin and 1 mM isopropyl-β-D-thiogalactopyranoside (IPTG) in 37° C., overnight with shaking. Afterwards, by centrifugation (4,000 rpm, 10 min, 4° C.) the cells harvested and re-suspended in 25 ml lysis buffer (PBS pH 7.5, 1 mM b-mercaptoethanol plus 1 tablet protease inhibitor Mini, EDTA-freelyse), bacteria lysed by French pressed and stirred in 4° C. with adding DnaseI and whole proteins were purified. To get rid of aggregation, we kept the proteins on the ice and then filtered. HcaB protein purification performed by AKTA FPLC system, which is a fully automated liquid chromatography system, where HcaB protein purified using HiPrep™ HP 5 ml column. Before actual purification run, the AKTA were stripped, charged, and blanked. The Purification was done by program in AKTA Xpress. The purified proteins collected from different tubes and combine fractions and kept at 4° C. Purified proteins were checked by running that on SDS gel. Three Swiss mice immunized twice intraperitoneally, in 10 days. The first injection was administered with a mixture of HcaB protein (50 ug/mouse) and Complete Freund's adjuvant (CFA; Sigma) (1:1). In day tenth, a booster injection (protein and Incomplete Freund's adjuvant (IFA; Sigma) performed after titration of the antibody by ELISA. Then from the serum, the total immunoglobulin Gs (IgGs) purified by absorption to a protein G column (GE Healthcare) according to the manufacturer's instructions.

**20. Experimental Animals**

Female swiss mice from 5 to 8 weeks (20 and 23 g) were used for the experiments. The animals were used in experiments after a period of 3 days of adaptation to the experimental cages, with regular 12 h light-dark periods and ambient temperature of 20° C.

**21. In Vivo Toxicity Test**

The procedures used in the assays were approved by the local Animal Ethical Committee and conform to international standards of animal welfare (Approval P067/2015 of the Ethical Committee of KU Leuven). A safe concentration of 30 mg/kg dose achieved from escalation experiment and observed no acute adverse effects upon intraperitoneal administration. Briefly, 5-6 week-old Swiss females were divided into 3 groups (6 animals/group) and administered 30 mg/kg of Colpeptin1 (group A) or vehicle (physiological water pH 7.5) (group b) once a day for 18 days via IP injection. During the treatment period the clinical, physiological and behavioral parameters including body weight, food and water consumption, body condition score, home cage activity and locomotion were constantly monitored and recorded.

Three days after the last administration animals were anesthetized and blood was collected using a standard retro-orbital puncture from each animal. Next, the mice sacrificed and underwent complete necropsy with gross examination and organ weights. The organs (Heart, Liver, Spleen, kidney, bone marrow, Brain, Lung) were sampled and immersion fixed in 4% paraformaldehyde. Formalin-fixed tissues after dehydration were routinely processed and embedded in paraffin blocks for histopathological examination. 5 μm thick sections taken from these blocks (Thermo Scientific Microm HM355S microtome) were then stained with Hematoxylin and Eosin (Leica ST5010 Autostainer XL) and evaluated under a Leica DM 2500 light microscope by a board-certified veterinary pathologist. Hematology examination was performed using an automate high-resolution flow cytometer, Abbott Cell-Dyn 3700.

**22. The Urinary Tract Infection Model**

8 weeks old Swiss female mice were used for the urinary tract infection model as described previously4. Briefly, mice were anesthetized by intraperitoneal administration of Nembutal 10% then with fingers gently the bladder of mouse was massaged and pushed down on to expel remaining urine. Thereafter, the anesthetized mice were inoculated transurethrally with 50 μl of bacterial suspension slowly (1×108 cfu/ml) by sterile catheter in the bladder over 5 s in order to avoid vesicoureteral reflux through a surgical microscope. We estimated the sample size as follows: allowing a type I error rate of 0.05, a type II error 0.2 and estimating the maximum standard deviation of the CFU determination at 1 log CFU, we calculated that a sample size of 15 would allow us to reliably detect an effect size of 1 log CFU difference between treated and untreated. After 1 h mice were randomized and divided into 5 groups (15 mice/groups) group A and B received 10 mg/Kg Colpeptin interaperitoneally or transurethrally, respectively. Group C received P2Pro2 peptides (Proline substitutions) via transurethrally. Group D received the Ampicillin orally as the positive control and Group D received the vehicle (physiological water). The catheter was then removed immediately after inoculation. 24 h post infection, mice were sacrificed and Kidney, Bladder, Ureter, Colon were washed with PBS and homogenized. The homogenized tissues were serially diluted and were cultured on blood agar plates. The plates were kept in 37° C. overnight. The rate of bacteria was measured by CFU value. Blinding: Sample preparation and treatment of the animals was performed by L.K. and L.K., who also performed the CFU determination, however between treatment and readout the animals were randomly shuffled by P.C. and the key to the grouping was not revealed until after all the results were in.

23. Construction of E. coli MG1655 dnaK-mCer3

To construct E. coli MG1655 dnaK-mCer3, plasmid pGBKD-mCer3 was first constructed by integrating a mCer3 amplicon, generated with primers 5′-AGAATTCGGCAGCGGCAGCGGCAGCGTGAGCAAGGGCGAGGA-3′ (Fw) (SEQ ID NO: 229) and 5′-AGGATCCTTACTTGTACAGCTCGTCCA-3′ (Rev) (SEQ ID NO: 230), into pGBKDparSpMT15 using EcoRI and BamHI restriction sites. In addition to adding the respective restriction sites to the end of the amplicon, these primer pairs also add a flexible linker (encoding GSGSGS6) facilitating folding of fluorescent fusion proteins constructed with these sequences. A mCer3-frt-cat-frt cassette was subsequently PCR amplified from plasmid pGBKD-mCer3 using primer pair PGP-21DNA

The amplicon was subsequently inserted into MG1655, creating a C-terminal DnaK-mCer3 fusion. The cat cassette was subsequently flipped out by transiently equipping this strain with plasmid pCP207, resulting in the desired MG1655 dnaK-mCer3 strain.

24. Protein Purification from SDS Gel for MS Analysis

The purified inclusion bodies were loaded on SDS gel (4-15% Mini-PROTEAN@ TGX™ Precast Protein Gels, 10-well, 50 μl) and stained by coomassie blue (R250). The excised bands were cut by a sterile scalpel under a laminar flow. The gel slices washed in several cycles by incubating them in 50 mM ammonium bicarbonate/ACN (acetonitrile) (1:1) for 10 min at room temperature until the blue stain is gone and replacing the buffer by 100% ACN and incubating for 5 min. After the last cycle, the samples were dried by speedvac and digested with 250 ng of modified trypsin (Promega) in 50 mM ammonium bicarbonate buffer (pH 8.3) overnight at 37° C. Peptides were extracted by adding 5% ACN+0.1% formic acid, and followed by 10% ACN+0.1% FA and 95% ACN in 0.5% FA) and dried by Speedvac. The extracted peptides were cleaned up by using pierce C18 spin Columns (Thermo Fisher Scientific) according to the manufacturer's instructions. The samples were diluted in 10 μL with 5% ACN+0.1% FA for injection in MS machine.

**25. Statistics**

Statistical calculations were performed using Prism unless otherwise indicated.

For the ANOVA analysis of the infection model (FIG. 2F-I), the assumption that the groups have similar standard deviations was tested using Bartlett's test, which showed no significant differences between the standard deviations, except for the ureter data, which we ignored because it was caused by a reduction in the standard deviation of the untreated group.

