# DESCRIPTION

## FIELD OF THE INVENTION

The present invention relates to an isolated polynucleotide encoding click beetle luciferase variant polypeptides that have enhanced luminescence and produce longer, near-infrared wavelength signals. The present invention also relates to near-infrared bioluminescence systems that include said click beetle luciferase variant polypeptides and infrared-emitting substrates and methods of using said click beetle luciferase variant polypeptides and near-infrared bioluminescence systems.

## BACKGROUND

Bioluminescence with a longer-wavelength and lower-energy emission is of significant interest both for multiplexing applications with multiple emission colors and for in-depth tissue imaging where shorter wavelengths tend to be strongly absorbed. Many standard systems for optical imaging have limited utility in a whole-animal context due to the diminished transmission of light through biological samples. Light penetration is limited by the absorption coefficients of particular components in blood. Strong absorption by Hemoglobin (Hb) and oxygenated hemoglobin (HbO2) diminish transmission and penetration depth of light through blood and animal tissues. Luminescent systems that emit light in the far-red and near-infrared region (680-900 nm) allow for optimal imaging due to the minimum absorbance spectrums of Hb and HbO2. This region of maximum light penetration is known as the whole animal “optical window.” Bioluminescent reporter systems have been used extensively in research animals, yet still suffer from the limitations of diminished tissue penetration. Typical bioluminescent light emission wavelengths (460-620 nm) occur in a region with limited penetration depth. The ideal bioluminescent reporter systems in whole animals would benefit greatly from the bright light emission in the region of 680-900 nm. While numerous bioluminescent systems have been modified to shift visible light emission toward the red, none have achieved a strong red emission to overlap significantly with the critical “optical window” of blood transmittance.

Previous approaches for molecular imaging in vivo include quantum dot conjugates that luminesce by bioluminescence resonance energy transfer (BRET) in the absence of external excitation. These conjugates, prepared by coupling carboxylate-presenting quantum dots to a mutant of the bioluminescent protein of Renilla reniformis luciferase, emit a long-wavelength (from red to near-infrared) bioluminescent light in cells and in animals, even in deep tissues. However, this approach is limited by the signal intensity for Renilla luciferase and by the relatively low aqueous solubility of the coelenterazine substrate. Another approach, e.g., Aka Lumine (Wako), utilizes a luciferin derivative which is dimmer and less red (675 nm) (see FIG. 19A and FIG. 19B).

Therefore, there is a need in small animal optical imaging applications for longer wavelength and lower energy-emitting near-infrared bioluminescence systems that will make it possible to detect signals from deep tissue, where standard near-infrared bioluminescence systems (450-620 nm) tend to be strongly absorbed.

## SUMMARY OF THE INVENTION

The present invention is directed to an isolated polynucleotide encoding a click beetle red luciferase (CBR) variant polypeptide having at least 80% amino acid sequence identity to SEQ ID NO: 1 and comprising at least one amino acid substitution at a position corresponding to position 4, 16, 34, 47, 51, 52, 55, 72, 73, 74, 79, 82, 83, 87, 89, 104, 109, 113, 117, 119, 124, 130, 131, 133, 136, 144, 146, 156, 159, 170, 179, 186, 200, 211, 218, 224, 225, 226, 228, 229, 234, 247, 251, 252, 253, 255, 280, 281, 285, 308, 309, 310, 319, 329, 334, 335, 337, 346, 348, 349, 350, 352, 354, 355, 358, 363, 370, 377, 390, 393, 394, 400, 401, 409, 412, 420, 422, 431, 437, 439, 444, 445, 453, 455, 467, 471, 473, 479, 484, 489, 496, 501, 503, 508, 516, 528, 531, 535, 537, 539, or combination thereof, of SEQ ID NO: 1, wherein the variant CBR polypeptide has at least one of enhanced luminescence, altered light emission wavelength, altered substrate specificity, or a combination thereof, as compared to a CBR polypeptide of SEQ ID NO: 1. The CBR variant polypeptide may further comprise at least one amino acid substitution at a position corresponding to position 351, 389, 457, or combination thereof, of SEQ ID NO: 1. The CBR variant polypeptide may comprise a substitution corresponding to at least one of R4H, H16Q, H34Y, D47E, S51N, Y52C, F55L/V, K72E, I79V, M73K/T, N74S, E82G, N83H, F87S, I89V, V104D, I109N/V, L113Q, M117T, I119F/T, I124V, N130K, I131N/T, N133D, K136N, F144L, K146E, N156D, N156K, G159D, Y170C, K179S, V186A, G200G, N211N, H218L/Y, G225S, T226C/G/H/N/Q/Y, L228P, I229V, V234A, G251S, G251I, Y252C, V255D/F, E253K, R280S, S281N/Q, V285A, I309T, E319G, N329D, R334E/Q/H/S/N/K, C335S, K337E, I346N, Q348H/E, L350P, G351K/R, D352N, R355G, S358P, T363A/S, I370T, I389F/G/S/V, I390I, M393K/L, V394M, N400D, N401S, I409T, D412G, F420F, Y422C, V431A, E437G, I439V, S444C/R/T, Q445H, E453K, V455D, K457N, D471V, E473A, S479T, K484E/M/R, E489V, Y496H, E501G, V503M, Y508C, V516A, T528A, E531G, Q535H, L537W, K539R, or combinations thereof, or SEQ ID NO: 1. The CBR variant polypeptide may comprise an amino acid substitution at a position corresponding to position 389, 444, and 251 of SEQ ID NO: 1. The amino acid substitutions may comprise I389F, S444R, and G251S. The CBR variant polypeptide may comprise an amino acid substitution at positions corresponding to positions 334 and 351 of SEQ ID NO: 1. The amino acid substitutions may comprise R334S and G351R. The CBR variant polypeptide may further comprise an amino acid substitution at a position corresponding to positions 51 and 444 of SEQ ID NO: 1. The amino acid substitutions may comprise S51N and S444R. The CBR variant polypeptide may comprise an amino acid polypeptide of SEQ ID NO: 2, SEQ ID NO: 3, or SEQ ID NO: 4. The CBR variant polypeptide may have enhanced luminescence compared to a CBR polypeptide of SEQ ID NO: 1. The CBR variant polypeptide may have enhanced luminescence when a luciferin is utilized by the CBR variant polypeptide to generate luminescence. The CBR variant polypeptide may have enhanced luminescence when a luciferin derivative is utilized by the CBR variant polypeptide to generate luminescence. The luciferin derivative may comprise:

The CBR variant polypeptide may have at least 2 fold increase in luminescence compared to a CBR polypeptide of SEQ ID NO: 1. The CBR variant polypeptide may have at least 4 fold increase in luminescence compared to a CBR polypeptide of SEQ ID NO: 1. The CBR variant polypeptide may have altered light emission spectra compared to a CBR polypeptide of SEQ ID NO: 1. The CBR variant polypeptide may be able to emit light at a longer wavelength when a luciferin is utilized by the CBR variant polypeptide to generate luminescence. The CBR variant polypeptide may be able to emit light at a longer wavelength when a luciferin derivative is utilized by the CBR variant polypeptide to generate luminescence. The luciferin derivative may comprise:

If the luciferin derivative comprises

the CBR variant polypeptide may emit light having a shift in spectral maximum of at least about 1 nm to at least about 100 nm relative to the light produced by the CBR polypeptide of SEQ ID NO: 1. The CBR variant polypeptide may emit light having a spectral maximum between about 650 nm to about 800 nm. The CBR variant polypeptide may emit light having a spectral maximum between about 725 nm to about 775 nm. The CBR variant polypeptide may emit light having a spectral maximum of about 750 nm. If the luciferin derivative comprises

the CBR variant polypeptide may emit light having a shift in spectral maximum of at least about 1 nm to at least about 100 nm relative to the light produced by the CBR polypeptide of SEQ ID NO: 1. The CBR variant polypeptide may emit light having a shift in spectral maximum of at least about 75 nm relative to the light produced by the CBR polypeptide of SEQ ID NO: 1. The CBR variant polypeptide may emit light having a spectral maximum between about 650 nm to about 800 nm. The CBR variant polypeptide may emit light having a spectral maximum between about 700 nm to about 775 nm. The CBR variant polypeptide may emit light having a spectral maximum of about 725 nm. The CBR variant polypeptide may have altered substrate specificity compared to a CBR polypeptide of SEQ ID NO: 1. The CBR variant polypeptide may have a change in relative specificity relative to the CBR variant polypeptide in the presence of a luciferin compared to a luciferin derivative. The CBR variant polypeptide may have a change in relative specificity relative to the CBR variant polypeptide in the presence of a luciferin derivative compared to a different luciferin derivative. The luciferin derivative may comprise:

The variant CBR polypeptide may have luciferase activity. The variant CBR polypeptide may have a Km for PBI-4813 of at least about 0.01 μM to at least about 5.00 μM. The variant CBR polypeptide may have a Km for PBI-4813 of at least about 0.50 μM to at least about 3.00 μM. The variant CBR polypeptide may have a Km for PBI-4813 of at least about 0.82 μM or 2.41 μM. The variant CBR polypeptide may have a Km for PBI-4739 of at least about 0.01 μM to at least about 5.00 μM. The variant CBR polypeptide may have a Km for PBI-4739 of at least about 1.50 μM to at least about 4.50 μM. The variant CBR polypeptide may have a Km for PBI-4739 of at least about 2.33 μM or 3.95 μM. The variant CBR polypeptide may have a relative Vmax that is at least 2 fold higher than the relative Vmax of a CBR polypeptide of SEQ ID NO: 1 using PBI-4813 as a substrate. The variant CBR polypeptide may have a relative Vmax that is at least 2 fold higher than the relative Vmax of a CBR polypeptide of SEQ ID NO: 1 using PBI-4739 as a substrate. The sequence may have been codon-optimized. The sequence may comprise a polynucleotide of SEQ ID NOs: 6-9. The polynucleotide may further encode a polypeptide of interest linked to the CBR variant polypeptide, the polypeptide of interest and the CBR variant polypeptide capable of being expressed as a fusion protein. The polypeptide of interest may comprise HALOTAG®.

The present invention is directed to a vector comprising the polynucleotide, or a fragment thereof, as described above. The polynucleotide may be operably linked to a promoter.

The present invention is directed to a cell comprising the polynucleotide as described above or the vector as described above.

The present invention is directed to a non-human transgenic animal comprising the cell as described above.

The present invention is directed to a non-human transgenic animal comprising the polynucleotide as described above or the vector as described above.

The present invention is directed to a CBR variant polypeptide encoded by the polynucleotide as described above.

The present invention is directed to a circularly permuted luciferase comprising the polypeptide encoded by the polynucleotide as described above or a fragment thereof.

The present invention is directed to a fusion protein comprising a CBR variant polypeptide encoded by the polynucleotide as described above.

The present invention is directed to a near-infrared bioluminescence system comprising the polynucleotide as described above and a luciferin derivative. The luciferin derivative may comprise:

The present invention is directed to a method of producing a CBR variant polypeptide comprising growing the cell as described above under conditions that permit expression of the CBR variant polypeptide.

The present invention is directed to a method of producing a CBR variant polypeptide comprising introducing the vector as described above into a cell under conditions which permit expression of the CBR variant polypeptide.

The present invention is directed to a kit comprising the polynucleotide as described above or the vector as described above.

The present invention is directed to a kit comprising the CBR variant polypeptide as described above. The kit may further comprise at least one of:


- (a)

and


- (b) a buffer reagent.

The present invention is directed to a bioluminescence resonance energy transfer (BRET) system comprising: a first fusion protein including a first target protein and a bioluminescence donor molecule, wherein the bioluminescence donor molecule is a CBR variant encoded by the polynucleotide as described above; a second fusion protein including a second target protein and a fluorescent acceptor molecule; and a CBR substrate. The CBR substrate may be a luciferin or luciferin derivative. The luciferin derivative may comprise:

The present invention is directed to a method for measuring bioluminescence using at least one of the polynucleotide as described above; the vector as described above; the cell as described above; the animal as described above; the CBR variant polypeptide as described above; the circularly permuted luciferase as described above; the fusion protein as described above, or the near-infrared bioluminescence system as described above. The bioluminescence may be measured in a live, intact non-human animal.

The present invention is directed to a method of measuring the enzymatic activity of a luminogenic protein. The method comprises contacting a luminogenic protein, a deprotecting enzyme, and a protected luminophore; and detecting light produced from the composition, wherein the luminogenic protein is a CBR variant encoded by the polynucleotide as described above and the luminophore is a luciferin derivative comprising

The enzymatic activity may be measured in a live, intact non-human animal.

The present invention is directed to a method for measuring the activity of a non-luminescent enzyme of interest. The method comprises (a) providing a luminogenic molecule wherein the molecule is a substrate for the non-luminescent enzyme of interest and a pro-substrate of a CBR variant encoded by the polynucleotide as described above; (b) contacting the luminogenic molecule with at least one non-luminescent enzyme of interest and at least one CBR variant to produce a reaction mixture; and (c) determining activity of the non-luminescent enzyme of interest by measuring luminescence of the reaction mixture. The luminogenic molecule may be a modification of

The non-luminescent enzyme of interest may be a protease enzyme, a cytochrome P450 enzyme, a monoamine oxidase enzyme, or a glutathione S-transferase enzyme. The activity of the non-luminescent enzyme may be measured in a live, intact animal.

The present invention is directed to a method to detect the presence of at least two molecules in a sample or a cell. The method comprises contacting the sample or cell with a first reporter molecule comprising a CBR variant encoded by the polynucleotide as described above, wherein the first reporter molecule is operatively linked to a first component of the sample or cell; contacting the sample with a second reporter molecule, wherein the second reporter molecule is operatively linked to a second component of the sample or cell; and detecting the presence of the first and second reporter molecules to determine the presence and/or amounts of the first and second components in the sample or cell.

The present invention is directed to a method to detect an interaction between a first protein and a second protein in a sample. The method comprises contacting a sample with:

wherein the sample comprises: a first polynucleotide encoding a first fusion protein, wherein the first fusion protein comprises a first fragment of a luminescent enzyme and a first protein; and a second polynucleotide encoding a second fusion protein, wherein the second fusion protein comprises a second fragment of the luminescent enzyme and a second protein; and detecting luminescence in the sample. The detection of luminescence indicates an interaction between the first protein and the second protein, wherein the luminescent enzyme is encoded by the isolated polynucleotide as described above. When the first protein and second protein interact, the first fragment of the luminescent enzyme and the second fragment of the luminescent enzyme may reconstitute a full-length enzyme capable of stably binding the cell-permeable substrate.

The present invention is directed to a method to detect an interaction between a first protein and a second protein in a sample. The method comprises contacting a sample with:

wherein the sample comprises: a first polynucleotide encoding a first fusion protein, wherein the first fusion protein comprises a luminescent enzyme and a first protein, wherein the luminescent enzyme is encoded by the isolated polynucleotide of any one of claims 1-46; and a second polynucleotide encoding a second fusion protein, wherein the second fusion protein comprises a fluorescent acceptor molecule and a second protein; and (b) detecting bioluminescence resonance energy transfer (BRET) in the sample, indicating an interaction or close proximity of the bioluminescent donor and the fluorescence acceptor.

## DETAILED DESCRIPTION

Disclosed herein are click beetle red luciferase (CBR) variants that have enhanced performance in the near-infrared (near-IR) range using luciferin and/or novel luciferin derivatives. The disclosed CBR variants have enhanced performance in vitro and in living cells, e.g., increased brightness and longer wavelengths. Also disclosed herein are near-IR bioluminescence systems that comprise the CBR variants and novel luciferin derivatives. The disclosed near-IR bioluminescence systems have enhanced performance in the near-IR range and address sensitivity limitations that are related to tissue absorption and associated with other bioluminescent detections systems. Unlike other bioluminescence systems, such as VivoGlo™ (Promega Corp.), the disclosed near-IR bioluminescence systems provide near-IR luminescence signals that penetrate tissue well and provide enhanced detection of signals coming from deep animal tissues. The present disclosure also encompasses methods of using and kits including the CBR variants and/or bioluminescence systems.

### 1. DEFINITIONS

Unless otherwise defined, all technical and scientific terms used herein have the same meaning as commonly understood by one of ordinary skill in the art. In case of conflict, the present document, including definitions, will control. Preferred methods and materials are described below, although methods and materials similar or equivalent to those described herein can be used in practice or testing of the present invention. All publications, patent applications, patents and other references mentioned herein are incorporated by reference in their entirety. The materials, methods, and examples disclosed herein are illustrative only and not intended to be limiting.

The terms “comprise(s),” “include(s),” “having,” “has,” “can,” “contain(s),” and variants thereof, as used herein, are intended to be open-ended transitional phrases, terms, or words that do not preclude the possibility of additional acts or structures. The terminology used herein is for the purpose of describing particular embodiments only and is not intended to be limiting. As used in the specification and the appended claims, the singular forms “a,” “and” and “the” include plural references unless the context clearly dictates otherwise. The present disclosure also contemplates other embodiments “comprising,” “consisting of” and “consisting essentially of,” the embodiments or elements presented herein, whether explicitly set forth or not.

For the recitation of numeric ranges herein, each intervening number there between with the same degree of precision is explicitly contemplated. For example, for the range of 6-9, the numbers 7 and 8 are contemplated in addition to 6 and 9, and for the range 6.0-7.0, the number 6.0, 6.1, 6.2, 6.3, 6.4, 6.5, 6.6, 6.7, 6.8, 6.9, and 7.0 are explicitly contemplated.

As used herein, “bioluminescence” or “luminescence” is light produced as a result of a reaction between an enzyme and a substrate that generates light.

In general, “enhanced” means that the particular property (e.g. bioluminescence or luminescence) is increased relative to that of a reference luciferase plus luciferin combination or luciferase under consideration, where the increase is at least 1%, at least 2%, at least 3%, at least 4%, at least 5%, at least 10%, at least 20%, at least 25%, at least 50%, at least 75%, at least 90%, at least 100%, at least 200%, at least 500%, or at least 1000% greater than the reference luciferase plus luciferin combination or luciferase under consideration.

The term “luciferin substrate” as used herein refers to a molecule capable of creating light via a chemical or biochemical reaction (e.g., luciferin, luciferin derivative, or a functional analog thereof). Suitable luciferin substrates for luciferase enzymes include luciferin, luciferin derivatives, and functional analogs of luciferins. In some embodiments, functional analogs of luciferins include modified luciferins including derivatives of these compounds. Exemplary compounds include those disclosed in U.S. patent application Ser. No. 14/200,563.

The naturally-occurring substrate for beetle luciferases is firefly luciferin, a polytherocyclic organic acid, D-(−)-2-(6′-hydroxy-2′-benzoth-iazolyl)-Δ2-thiazolin-4-carboxylic acid (“D-luciferin” also referred to as “luciferin”). Luciferin may be isolated from nature (e.g., from fireflies) or synthesized. Synthetic luciferin can have the same structure as the naturally occurring luciferin or can be derivatized, so long as it functions analogously. Examples of derivatives of luciferin include D-luciferin methyl ester and other esters of luciferase that are hydrolyzed or acted upon by esterases in a sample to yield luciferin, and naphthyl- and quinolyl-luciferin (Branchini et al., 1989). There are multiple commercial sources for luciferin (e.g., Promega Corp. Madison, Wis.).

The term “luciferin derivative” as used herein refers to a type of luminogenic molecule or compound having a substantial structure of D-luciferin and is a luciferase substrate, e.g., aminoluciferin, or luciferase substrates disclosed in U.S. Patent Publication No. 2007/0015790.

A “luciferase reaction mixture” contains a luciferase enzyme and materials that will allow the luciferase enzyme to generate a light signal. The materials needed, and the particular concentrations and/or amounts, of the materials needed to generate a luminescent signal will vary depending on the luciferase enzyme used as well as the type of luciferase-based assay being performed. In general, for click beetle luciferase, including the CBR variants, these materials can include: ATP, magnesium (Mg2+) salt, such as magnesium sulfate, a click beetle luciferase enzyme, and a luciferin or novel luciferin derivative capable of generating light when the luciferin or novel luciferin derivative is used as a substrate for the click beetle luciferase. Often other materials will be added to the solution including: a buffer to maintain the reaction at the proper pH, an additive such as PRIONEX or Bovine serum albumin (BSA) to help maintain luciferase activity, reducing agents, detergents, esterases, salts, amino acids, e.g. D-cysteine, etc. An example luciferase reaction mixture would contain a click beetle luciferase, MgSO4, ATP, Tergitol NP-9, and Tricine.

A “luciferase detection mixture” contains materials that will allow for the detection of a luciferase enzyme. The materials needed, and the particular concentrations and/or amounts, of the materials needed to generate a luminescent signal will vary depending on the luciferase enzyme used as well as the type of luciferase-based assay being performed. In general, for click beetle luciferase, including the CBR variants disclosed herein, these materials can include: ATP, magnesium (Mg2+) salt, such as magnesium sulfate, and a luciferase substrate, e.g., luciferin, luciferin derivative, functional analog, or novel luciferin derivative capable of generating light when the luciferase substrate, e.g., luciferin, luciferin derivative, functional analog, or novel luciferin derivative is used as a substrate for the click beetle luciferase. Often other materials will be added to the solution including: a buffer to maintain the reaction at the proper pH, an additive such as PRIONEX or Bovine serum albumin (BSA) to help maintain luciferase activity, reducing agents, detergents, esterases, salts, amino acids, e.g. D-cysteine, etc. An example luciferase detection mixture would contain a luciferase substrate, MgSO4, ATP, Tergitol NP-9, and Tricine.

The term “luminescence” refers to the light output of the luciferase, e.g., the CBR variant, under appropriate conditions, e.g. in the presence of a suitable substrate such as a luciferin or novel luciferin derivative. The light output may be measured as an instantaneous or near-instantaneous measure of light output (which is sometimes referred to as “T=0” luminescence or “flash”) at the start of the luminescence reaction, which may be initiated upon addition of the luciferin or novel luciferin derivative substrate. The luminescence reaction in various embodiments is carried out in a solution. In other embodiments, the luminescence reaction is carried out on a solid support. The solution may contain a lysate, for example from the cells in a prokaryotic or eukaryotic expression system. In other embodiments, expression occurs in a cell-free system, or the luciferase protein is secreted into an extracellular medium, such that, in the latter case, it is not necessary to produce a lysate. In some embodiments, the reaction is started by injecting appropriate materials, e.g. luciferin substrate, buffer, etc., into a reaction chamber (e.g. a well of a multiwell plate such as a 96-well plate) containing the luminescent protein. In still other embodiments, the luciferase substrate is introduced into a host, which may be expressing a luciferase, e.g., a CBR variant, and measurements of luminescence are made on the host or a portion thereof, which can include a whole organism or cells, tissues, explants, or extracts thereof. The reaction chamber may be situated in a reading device which can measure the light output, e.g. using a luminometer or photomultiplier. The light output or luminescence may also be measured over time, for example in the same reaction chamber for a period of seconds, minutes, hours, etc. The light output or luminescence may be reported as the average over time, the half-life of decay of signal, the sum of the signal over a period of time, or the peak output. Luminescence may be measured in Relative Light Units (RLUs).

“Relative substrate specificity” is determined by dividing the luminescence of a luciferase in the presence of a test luciferin substrate by the luminescence of the luciferase in the presence of a reference luciferin substrate. For example, relative specificity may be determined by dividing the luminescence of a luciferase with a novel luciferin derivative by the luminescence of the luciferase with a different luciferin (e.g. D-luciferin or novel luciferin derivative). The test luciferin substrate and the reference luciferin substrate that are compared are considered a comparison substrate pair for determining relative substrate specificity.

A “change in relative substrate specificity” is determined by dividing the relative substrate specificity of a test luciferase using a comparison substrate pair by the relative substrate specificity of a reference luciferase using the same comparison substrate pair. For example, a change in relative specificity may be determined by dividing the relative substrate specificity of a test luciferase with a novel luciferin derivative compared to a different luciferin (e.g. D-luciferin or novel luciferin derivative), by the relative substrate specificity of a reference luciferase with the same novel luciferin derivative compared to the same different luciferin used for the test luciferase.

The term “fusion polypeptide” or “fusion protein” refers to a chimeric protein containing a reference protein (e.g., CBR variant) joined at the N- and/or C-terminus to one or more heterologous sequences (e.g., a non-CBR polypeptide).

The term “identity,” in the context of two or more nucleic acids or polypeptide sequences, refers to two or more sequences or subsequences that are the same or have a specified percentage of amino acid residues or nucleotides that are the same when compared and aligned for maximum correspondence over a comparison window or designated region as measured using any number of sequence comparison algorithms or by manual alignment and visual inspection. Methods of alignment of sequence for comparison are well-known in the art. Optimal alignment of sequences for comparison can be conducted by the algorithm of Smith et al., (J. Mol. Biol. 147:195-197 (1981)), by the homology alignment algorithm of Needleman and Wunsch, (J. Mol. Biol., 48:443-453 (1970)), by the search for similarity method of Pearson and Lipman, (Proc. Natl. Acad. Sci. USA, 85:2444-2448 (1988)), by computerized implementations of algorithms e.g., FASTA, SSEARCH, GGSEARCH (available at the University of Virginia FASTA server by William R. Pearson http://fasta.bioch.virginia.edu/fasta_www2/fasta_intro.shtml), the Clustal series of programs (Chenna et al., Nucl. Acids Res. 31(13):3497-3500 (2003); available examples at http://www.ebi.ac.uk or http://www.ch.embnet.org), or other sequence analysis software. It is known in the art that generating alignments with maximum correspondence between polypeptide sequences with significant sequence alterations (e.g. altered domain order, missing/added domains, repeated domains, shuffled domains, circular permutation) may involve the use of specialized methods, such as the ABA method (Raphael et al., Genome Res. 14(11):2336-2346 (2004)), other suitable methods, or performing the alignment with two concatenated identical copies of the polypeptide sequences.

Nucleic acids are known to contain different types of “mutations”, which refers to an alteration in the sequence of a nucleotide at a particular base position relative to the wild-type sequence. Mutations may also refer to insertion or deletion of one or more bases so that the nucleic acid sequence differs from a reference, e.g., a wild-type sequence, or replacement with a stop codon. A “substitution” refers to a change in an amino acid at a particular position in a sequence.

The term “nucleic acid molecule,” “polynucleotide” or “nucleic acid sequence” as used herein, refers to nucleic acid, including DNA or RNA, that comprises coding sequences necessary for the production of a polypeptide or protein precursor. The encoded polypeptide may be a full-length polypeptide, a fragment thereof (less than full-length), or a fusion of either the full-length polypeptide or fragment thereof with another polypeptide, yielding a fusion polypeptide.

A polynucleotide encoding a protein or polypeptide means a nucleic acid sequence comprising the coding region of a gene, or in other words, the nucleic acid sequence encoding a gene product. The coding region may be present in a cDNA, genomic DNA or RNA form. When present in a DNA form, the oligonucleotide may be single stranded (e.g., the sense strand) or double stranded. Suitable control elements such as enhancers/promoters, splice junctions, polyadenylation signals, etc. may be placed in close proximity to the coding region of the gene, if needed, to permit proper initiation of transcription and/or correct processing of the primary RNA transcript. Other control or regulatory elements include, but are not limited to, transcription factor binding sites, splicing signals, polyadenylation signals, termination signals and enhancer elements.

By “peptide,” “protein” and “polypeptide” is meant amino acid chains of varying lengths, regardless of post-translational modification (e.g., glycosylation or phosphorylation). The nucleic acid molecules of the invention encode a variant of a man-made (i.e. synthetic) variant protein or polypeptide fragment thereof, which has an amino acid sequence that is at least 60%, e.g., at least 65%, 70%, 75%, 80%, 85%, 90%, 95%, 96%, 97%, 98%, 99%, or 100%, amino acid sequence identity to the amino acid sequence of the parental protein from which it is derived, where the parental protein can be a naturally-occurring (native or wild-type) sequence or a variant sequence which is subsequently modified further.

As used herein, “pure” or “purified” means an object species is the predominant species present (i.e., on a molar and/or mass basis, it is more abundant than any other individual species, apart from water, solvents, buffers, or other common components of an aqueous system in the composition), and, in some embodiments, a purified fraction is a composition wherein the object species comprises at least about 50% (on a molar basis) of all macromolecular species present. Generally, a “substantially pure” composition will comprise more than about 80% of all macromolecular species present in the composition, in some embodiments more than about 85%, more than about 90%, more than about 95%, or more than about 99%. In some embodiments, the object species is purified to essential homogeneity (contaminant species cannot be detected in the composition by conventional detection methods) wherein the composition consists essentially of a single macromolecular species.

The term “variant” refers to a modified version of a starting polypeptide or polynucleotide sequence. The term “parental” is a relative term that refers to a starting sequence which is then modified. The parental sequence is generally used as a reference for the protein encoded by the resulting modified sequence, e.g. to compare the activity levels or other properties of the proteins encoded by the parental and the modified sequences. The starting sequence can be a naturally-occurring (i.e. native or wild-type) sequence. The starting sequence can also be a variant sequence which is then further modified. A polypeptide sequence is “modified” when one or more amino acids (which may be naturally-occurring or synthetic) are substituted, deleted, and/or added at the beginning, middle, i.e., at an internal position, and/or end of the sequence. A polynucleotide sequence is “modified” when one or more nucleotides are substituted, deleted, and/or added at the beginning, middle, i.e., at an internal position, and/or end of the sequence, but which may or may not alter the amino acid encoded by the sequence. In some embodiments, the modifications produce a variant that is a functional fragment of CBR or a particular CBR variant. A functional fragment is a fragment which is less than a full-length parental sequence which has the same functional activity as the full-length parental sequence. Functional activity is the ability to exhibit luminescence. In some embodiments, the modifications produce a variant that is a permuted sequence of the parental sequence, such as a circularly permuted sequence and permuted sequences comprising deletions and/or insertions.

Variant is also used herein to describe a protein with an amino acid sequence that is substantially identical to a referenced protein with an amino acid sequence that retains at least one biological activity. A conservative substitution of an amino acid, i.e., replacing an amino acid with a different amino acid of similar properties (e.g., hydrophilicity, degree, and distribution of charged regions) is recognized in the art as typically involving a minor change. These minor changes can be identified, in part, by considering the hydropathic index of amino acids, as understood in the art. Kyte et al., J. Mol. Biol. 157:105-132 (1982). The hydropathic index of an amino acid is based on a consideration of its hydrophobicity and charge. It is known in the art that amino acids of similar hydropathic indexes can be substituted and still retain protein function. In one aspect, amino acids having hydropathic indexes of ±2 are substituted. The hydrophilicity of amino acids can also be used to reveal substitutions that would result in proteins retaining biological function.

Substitution of amino acids having similar hydrophilicity values can result in peptides retaining biological activity as is understood in the art. Substitutions may be performed with amino acids having hydrophilicity values within ±2 of each other. Both the hydrophobicity index and the hydrophilicity value of amino acids are influenced by the particular side chain of that amino acid. Consistent with that observation, amino acid substitutions that are compatible with biological function are understood to depend on the relative similarity of the amino acids, and particularly the side chains of those amino acids, as revealed by the hydrophobicity, hydrophilicity, charge, size, and other properties. “Variant” also can be used to describe a polypeptide or a fragment thereof that has been differentially processed, such as by proteolysis, phosphorylation, or other post-translational modification, yet retains its biological activity.

The term “vector” refers to nucleic acid molecules into which fragments of DNA may be inserted or cloned and can be used to transfer DNA segment(s) into a cell and capable of replication in a cell. Vectors may be derived from plasmids, bacteriophages, viruses, cosmids, and the like.

The term “wild-type” or “native” as used herein refers to a gene or gene product that has the characteristics of that gene or gene product isolated from a naturally occurring source. A wild-type gene is that which is most frequently observed in a population and is thus arbitrarily designated the “wild-type” form of the gene. In contrast, the term “mutant” refers to a gene or gene product that displays modifications in sequence and/or functional properties (i.e., altered characteristics) when compared to the wild-type gene or gene product. It is noted that naturally occurring mutants can be isolated; these are identified by the fact that they have altered characteristics when compared to the wild-type gene or gene product.

### 2. CBR VARIANTS

The present disclosure provides variants of click beetle red luciferase (CBR) that utilize luciferin and luciferin derivatives and produce longer near-IR (NIR) wavelength signals. Various techniques described herein were used to evolve CBR to produce brighter light output with extended conjugation luciferins and to identify sites for amino acid substitution to produce an improved synthetic CBR polypeptide. It was found that making one or more amino acid substitutions, either alone or in various combinations, produced synthetic CBR-type polypeptides having enhanced luminescence (e.g., enhanced brightness) and longer wavelength emissions. Also provided is a polynucleotide encoding a CBR variant of the invention or a fusion thereof, an isolated host cell having the polynucleotide or the CBR variant or a fusion thereof, and methods of using the polynucleotide, CBR variant or a fusion thereof or host cell of the invention.

The CBR variants have at least one amino acid substitution at a position corresponding to position 4, 16, 34, 47, 51, 52, 55, 72, 73, 74, 79, 82, 83, 87, 89, 104, 109, 113, 117, 119, 124, 130, 131, 133, 136, 144, 146, 156, 159, 170, 179, 186, 200, 211, 218, 224, 225, 226, 228, 229, 234, 247, 251, 252, 253, 255, 280, 281, 285, 308, 309, 310, 319, 329, 334, 335, 337, 346, 348, 349, 350, 352, 354, 355, 358, 363, 370, 377, 390, 393, 394, 400, 401, 409, 412, 420, 422, 431, 437, 439, 444, 445, 453, 455, 467, 471, 473, 479, 484, 489, 496, 501, 503, 508, 516, 528, 531, 535, 537, 539, or a combination thereof, of SEQ ID NO: 1. The CBR variants may further include at least one amino acid substitution at a position corresponding to position 351, 389, 457, or a combination thereof, of SEQ ID NO: 1. The CBR variants may have at least one amino acid substitution of corresponding to at least one of R4H, H16Q, H34Y, D47E, S51N, Y52C, F55LN, K72E, I79V, M73K/T, N74S, E82G, N83H, F87S, I89V, V104D, I109NN, L113Q, M117T, I119F/T, I124V, N130K, I131N/T, N133D, K136N, F144L, K146E, N156D, N156K, G159D, Y170C, K179S, V186A, G200G, N211N, H218L/Y, G225S, T226C/G/H/N/Q/Y, L228P, I229V, V234A, G251S, G251I, Y252C, V255D/F, E253K, R280S, S281N/Q, V285A, I309T, E319G, N329D, R334E/Q/H/S/N/K, C335S, K337E, I346N, Q348H/E, L350P, G351K/R, D352N, R355G, S358P, T363A/S, I370T, I389F/G/SN, I390I, M393K/L, V394M, N400D, N401S, I409T, D412G, F420F, Y422C, V431A, E437G, I439V, S444C/R/T, Q445H, E453K, V455D, K457N, D471V, E473A, S479T, K484E/M/R, E489V, Y496H, E501G, V503M, Y508C, V516A, T528A, E531G, Q535H, L537W, K539R, or combinations thereof, of SEQ ID NO: 1. The CBR variant polypeptide may have an amino acid substitution at a position corresponding to positions 389, 444, and 251 of SEQ ID NO: 1. The amino acid substitutions may include I389F, S444R, and G251S. The CBR variant polypeptide may have an amino acid substitution at a position corresponding to positions 334 and 351 of SEQ ID NO: 1. The amino acid substitutions may include R334S and G351R. The CBR variant may comprise an amino acid polypeptide of SEQ ID NO: 2 or SEQ ID NO: 3.

Several of the CBR variants and parental CBR disclosed herein have been assigned shorthand names to facilitate discussion. The term “ATG 343” (also referred to as “343” and “pF4Ag-CBR”) refers to a CBR polypeptide sequence of SEQ ID NO: 1. The CBR polypeptide sequence of SEQ ID NO: 1 is encoded by the polynucleotide sequence of SEQ ID NO: 5. The term “ATG 685” (also referred to as “685” and “pF4Ag-HT7-CBR”) refers to a CBR polypeptide sequence of SEQ ID NO: 1 fused to HALOTAG®. The term “ATG 1230” (SEQ ID NO: 3) refers to a CBR variant having the amino acid substitutions I389F, S444R, and G251S relative to SEQ ID NO: 1 (where the format “x#y” indicates a parent amino acid ‘x’ at a position ‘#’ that is changed to variant amino acid ‘y’). The term “ATG 1240” (SEQ ID NO: 2) refers to a CBR variant having the amino acid substitutions R334S and G351R relative to SEQ ID NO: 1. The term “ATG 1240+S51N+S444R” (SEQ ID NO: 4) refers to a CBR variant having the amino acid substitutions of ATG 1240, i.e., R334S and G351R of SEQ ID NO: 1, as well as amino acid substitutions S51N and S444R relative to SEQ ID NO: 1.

The sequences of a CBR variant are substantially the same as the amino acid sequence of a corresponding parental CBR, e.g., SEQ ID NO: 1. A polypeptide or peptide having substantially the same sequence means that an amino acid sequence is largely, but is not entirely, the same and retains the functional activity of the sequence to which it is related. In general, two amino acid sequences are substantially the same if they are at least 60%, e.g., at least 65%, 70%, 75%, 80%, 85%, 90%, 95%, 96%, 97%, 98%, or 99%, but less than 100%, amino acid sequence identity. In some embodiments, the CBR variant is encoded by a recombinant polynucleotide.

In some embodiments, the CBR variant has at least 60%, e.g., at least 65%, 70%, 75%, 80%, 85%, 90%, 95%, 96%, 97%, 98%, or 99%, or 100%, amino acid sequence identity to SEQ ID NOs: 1, 2, 3, or 4. Ordinarily, variant fragments are at least about 50 amino acids in length, often at least about 60 amino acids in length, more often at least about 70, 80, 90, 100, 150, 200, 300, 400, 500 or 550 amino acids in length, or more and retain the ability to generate luminescence. A full length luciferase, fragment thereof, or variant thereof may be fused to heterologous amino acid sequences and still be functional in the invention.

These CBR variants display at least one of: enhanced luminescence (including increased brightness, enhanced signal stability, signal duration, and/or decreased sensitivity to substrate inhibition); altered light emission spectra; and altered substrate specificity (i.e., change in relative substrate specificity). In various embodiments, the present invention encompasses novel luciferases that are present in solution as soluble proteins, chemically linked to other molecules (e.g., fusion proteins), or attached onto a solid surface (e.g., particles, capillaries, or assay tubes or plates).

a. Enhanced Luminescence

The enhanced luminescence of a CBR variant may be due to one or more of the following characteristics: enhanced light output (i.e. brightness), enhanced substrate specificity, enhanced signal stability, enhanced signal duration, and/or decreased sensitivity to substrate inhibition. Enhanced signal stability includes an increase in how long the signal from a luciferase continues to luminesce, for example, as measured by the half-life of decay of the signal in a time-course. The term “substrate inhibition” as used herein refers to the inhibition of luciferase enzyme activity (e.g., inhibition of generating luminescence and lower RLU values) at high concentration levels of substrate. A CBR variant having decreased sensitivity to substrate inhibition does not have lower RLU values with increasing amounts of substrate. For example, PBI-4813 at concentrations greater than ten times the Km inhibits the activity of CBR and ATG1230, thus the luminescence generated, i.e., RLU values, is decreased, whereas ATG1240 does not show this inhibition with PBI-4813 and has decreased sensitivity to substrate inhibition.

Enhanced luminescence may be determined relative to the comparable property of a luciferase such as wild-type CBR, a CBR variant protein, Renilla luciferase (e.g., hRluc), or firefly luciferase (e.g., Luc2; luciferase from Photinus pyralis) combined with a native, known, or novel substrate, as shown in the Examples below. For example, the luminescence of a given CBR variant in combination with a particular luciferin (including native, known, or novel luciferin, or derivatives thereof) may be compared to the properties of CBR combined with any of a native, known, or novel luciferins or derivatives thereof disclosed herein, using one or more of the assays disclosed in the Examples below. In particular, enhanced luminescence may be determined by measuring the luminescence signal (RLU) resulting from the incubation of bacterial or mammalian lysates containing CBR variants in question with the luciferase substrate, e.g., a novel luciferin derivative such as PBI-4813 or PBI-4739. The luminescence signal may be compared to that of a reference point such as CBR with D-luciferin or a luciferin derivative such as PBI-4813 or PBI-4739, or Luc2 (firefly) luciferase with D-luciferin.

In certain embodiments, the CBR variant has increased luminescence emission, e.g., at least about 1-fold, at least about 2-fold, at least about 3-fold, at least about 4-fold, at least about 5-fold, at least about 6-fold, at least about 7-fold, at least about 8-fold, at least about 9-fold, or at least about 10-fold in a prokaryotic cell and/or a eukaryotic cell relative to the corresponding reference luciferase, such as a CBR of SEQ ID NO: 1, using the same or different substrate, such as D-luciferin, PBI-4813 or PBI-4739. In some embodiments, one or more properties of the CBR variant are compared to comparable properties of a luciferase from another species, e.g. a firefly luciferase or a Renilla luciferase.

b. Altered Light Emission Spectra

The CBR variant polypeptide may have altered light emission spectra compared to a parental CBR, e.g., SEQ ID NO: 1. The CBR variant polypeptide may be able to emit light at a longer wavelength when a luciferin is utilized by the CBR variant polypeptide to generate luminescence. The CBR variant polypeptide is able to emit light at a longer wavelength when a luciferin derivative, such as PBI-4813 or PBI-4739, is utilized by the CBR variant polypeptide to generate luminescence.

The CBR variant polypeptide may emit light having a shift in spectral maximum relative to the light produced by the CBR polypeptide of SEQ ID NO: 1 using a luciferin or novel luciferin derivative such as PBI-4813 or PBI-4739. In some embodiments, the CBR variant polypeptide may emit light having an increased or decreased shift in spectral maximum of at least about 1 nm to at least about 200 nm, at least about 2 nm to at least about 200 nm, at least about 3 nm to at least about 200 nm, at least about 4 nm to at least about 200 nm, at least about 5 nm to at least about 200 nm, at least about 6 nm to at least about 200 nm, at least about 7 nm to at least about 200 nm, at least about 8 nm to at least about 200 nm, at least about 9 nm to at least about 200 nm, at least about 10 nm to at least about 200 nm, at least about 15 nm to at least about 200 nm, at least about 20 nm to at least about 200 nm, at least about 30 nm to at least about 200 nm, at least about 40 nm to at least about 200 nm, at least about 50 nm to at least about 200 nm, at least about 60 nm to at least about 200 nm, at least about 70 nm to at least about 200 nm, at least about 80 nm to at least about 200 nm, at least about 90 nm to at least about 200 nm, at least about 100 nm to at least about 200 nm, at least about 150 nm to at least about 200 nm, at least about 1 nm to at least about 150 nm, at least about 2 nm to at least about 150 nm, at least about 3 nm to at least about 150 nm, at least about 4 nm to at least about 150 nm, at least about 5 nm to at least about 150 nm, at least about 6 nm to at least about 150 nm, at least about 7 nm to at least about 150 nm, at least about 8 nm to at least about 150 nm, at least about 9 nm to at least about 150 nm, at least about 10 nm to at least about 150 nm, at least about 15 nm to at least about 150 nm, at least about 20 nm to at least about 150 nm, at least about 30 nm to at least about 150 nm, at least about 40 nm to at least about 150 nm, at least about 50 nm to at least about 150 nm, at least about 60 nm to at least about 150 nm, at least about 70 nm to at least about 150 nm, at least about 80 nm to at least about 150 nm, at least about 90 nm to at least about 150 nm, at least about 100 nm to at least about 150 nm, at least about 1 nm to at least about 100 nm, at least about 2 nm to at least about 100 nm, at least about 3 nm to at least about 100 nm, at least about 4 nm to at least about 100 nm, at least about 5 nm to at least about 100 nm, at least about 6 nm to at least about 100 nm, at least about 7 nm to at least about 100 nm, at least about 8 nm to at least about 100 nm, at least about 9 nm to at least about 100 nm, at least about 10 nm to at least about 100 nm, at least about 15 nm to at least about 100 nm, at least about 20 nm to at least about 100 nm, at least about 30 nm to at least about 100 nm, at least about 40 nm to at least about 100 nm, at least about 50 nm to at least about 100 nm, at least about 60 nm to at least about 100 nm, at least about 70 nm to at least about 100 nm, at least about 80 nm to at least about 100 nm, at least about 90 nm to at least about 100 nm, at least about 1 nm to at least about 70 nm, at least about 2 nm to at least about 70 nm, at least about 3 nm to at least about 70 nm, at least about 4 nm to at least about 70 nm, at least about 5 nm to at least about 70 nm, at least about 6 nm to at least about 70 nm, at least about 7 nm to at least about 70 nm, at least about 8 nm to at least about 70 nm, at least about 9 nm to at least about 70 nm, at least about 10 nm to at least about 70 nm, at least about 15 nm to at least about 70 nm, at least about 20 nm to at least about 70 nm, at least about 30 nm to at least about 70 nm, at least about 40 nm to at least about 70 nm, at least about 50 nm to at least about 70 nm, at least about 60 nm to at least about 70 nm, at least about 1 nm to at least about 50 nm, at least about 2 nm to at least about 50 nm, at least about 3 nm to at least about 50 nm, at least about 4 nm to at least about 50 nm, at least about 5 nm to at least about 50 nm, at least about 6 nm to at least about 50 nm, at least about 7 nm to at least about 50 nm, at least about 8 nm to at least about 50 nm, at least about 9 nm to at least about 50 nm, at least about 10 nm to at least about 50 nm, at least about 15 nm to at least about 50 nm, at least about 20 nm to at least about 50 nm, at least about 30 nm to at least about 50 nm, at least about 40 nm to at least about 50 nm, at least about 1 nm to at least about 20 nm, at least about 2 nm to at least about 20 nm, at least about 3 nm to at least about 20 nm, at least about 4 nm to at least about 20 nm, at least about 5 nm to at least about 20 nm, at least about 6 nm to at least about 20 nm, at least about 7 nm to at least about 20 nm, at least about 8 nm to at least about 20 nm, at least about 9 nm to at least about 20 nm, at least about 10 nm to at least about 20 nm, or at least about 15 nm to at least about 20 nm, relative to the light produced by the CBR polypeptide of SEQ ID NO: 1. The CBR variant polypeptide may emit light having an increased or decreased shift in spectral maximum of at least about 1 nm, at least about 2 nm, at least about 3 nm, at least about 4 nm, at least about 5 nm, at least about 6 nm, at least about 7 nm, at least about 8 nm, at least about 9 nm, at least about 10 nm, at least about 15 nm, at least about 20 nm, at least about 30 nm, at least about 40 nm, at least about 50 nm, at least about 60 nm, at least about 70 nm, at least about 80 nm, at least about 90 nm, at least about 100 nm, at least about 150 nm, or at least about 200 nm relative to the light produced by the CBR polypeptide of SEQ ID NO: 1.

The CBR variant polypeptide may emit light having a spectral maximum between about 650 nm to about 1000 nm, between about 700 nm to about 1000 nm, between about 725 nm to about 1000 nm, between about 750 nm to about 1000 nm, between about 775 nm to about 1000 nm, between about 800 nm to about 1000 nm, between about 850 nm to about 1000 nm, between about 900 nm to about 1000 nm, between about 950 nm to about 1000 nm, about 650 nm to about 900 nm, between about 700 nm to about 900 nm, between about 725 nm to about 900 nm, between about 750 nm to about 900 nm, between about 775 nm to about 900 nm, between about 800 nm to about 900 nm, between about 850 nm to about 900 nm, about 650 nm to about 800 nm, between about 700 nm to about 800 nm, between about 725 nm to about 800 nm, between about 750 nm to about 800 nm, between about 775 nm to about 800 nm, about 650 nm to about 775 nm, between about 700 nm to about 775 nm, between about 725 nm to about 775 nm, between about 750 nm to about 775 nm, between about 650 nm to about 750 nm, between about 700 nm to about 750 nm, between about 725 nm to about 750 nm, between about 650 nm to about 725 nm, between about 700 nm to about 725 nm, or between about 725 nm to about 750 nm when a luciferin or novel luciferin derivative, such as PBI-4813 or PBI-4739, is used as a substrate. The CBR variant polypeptide may emit light having a spectral maximum of at least about 600 nm, at least about 650 nm, at least about 700 nm, at least about 725 nm, at least about 750 nm, at least about 775 nm, at least about 800 nm, at least about 900 nm, or at least about 1000 nm when a luciferin or novel luciferin derivative, such as PBI-4813 or PBI-4739, is used as a substrate.

c. Altered Substrate Specificity

The CBR variant polypeptide may have altered substrate specificity compared to a CBR polypeptide of SEQ ID NO: 1. The CBR variant polypeptide may have increased or decreased substrate specificity for a luciferin or novel luciferin derivative such as PBI-4739 or PBI-4813, compared to a CBR polypeptide of SEQ ID NO: 1. The CBR variant polypeptide may have a change, such as an increase or decrease, in relative specificity relative to the CBR variant polypeptide in the presence of a luciferin compared to a luciferin derivative, such as PBI-4739 or PBI-4813. The CBR variant polypeptide may have a change, such as an increase or decrease, in relative specificity relative to the CBR variant polypeptide in the presence of a luciferin derivative compared to a different luciferin derivative, such as PBI-4739 or PBI-4813.

The CBR variant polypeptide may have an altered Km or Vmax compared to a CBR polypeptide of SEQ ID NO: 1 using D-luciferin and/or a novel luciferin derivative. A CBR variant polypeptide having a lower Km for a particular substrate may be an advantage in in vivo imaging where it is difficult to “saturate” the substrate.

The CBR variant may have a Km for PBI-4813 of at least about 0.10 μM, at least about 0.20 μM, at least about 0.30 μM, at least about 0.40 μM, at least about 0.50 μM, at least about 0.60 μM, at least about 0.70 μM, at least about 0.80 μM, at least about 0.81 μM, at least about 0.82 μM, at least about 0.83 μM, at least about 0.84 μM, at least about 0.85 μM, at least about 0.86 μM, at least about 0.87 μM, at least about 0.88 μM, at least about 0.89 μM, at least about 0.90 μM, at least about 1.00 μM, at least about 1.50 μM, at least about 2.00 μM, at least about 2.10 μM, at least about 2.20 μM, at least about 2.30 μM, at least about 2.40 μM, at least about 2.50 μM, at least about 2.60 μM, at least about 2.70 μM, at least about 2.80 μM, at least about 2.90 μM, at least about 3.00 μM, at least about 4.00 μM, or at least about 5.00 μM, or has a Km ranging from at least about 0.01 μM to at least about 5.00 μM, at least about 0.01 μM to at least about 4.00 μM, at least about 0.01 μM to at least about 3.00 μM, at least about 0.01 μM to at least about 2.50 μM, at least about 0.01 μM to at least about 2.00 μM, at least about 0.01 μM to at least about 1.00 μM, at least about 0.01 μM to at least about 0.80 μM, at least about 0.01 μM to at least about 0.50 μM, at least about 0.05 μM to at least about 5.00 μM, at least about 0.05 μM to at least about 4.00 μM, at least about 0.05 μM to at least about 3.00 μM, at least about 0.05 μM to at least about 2.50 μM, at least about 0.05 μM to at least about 2.00 μM, at least about 0.05 μM to at least about 1.00 μM, at least about 0.05 μM to at least about 0.80 μM, at least about 0.08 μM to at least about 5.00 μM, at least about 0.08 μM to at least about 4.00 μM, at least about 0.08 μM to at least about 3.00 μM, at least about 0.08 μM to at least about 2.50 μM, at least about 0.08 μM to at least about 2.00 μM, or at least about 0.08 μM to at least about 1.00 μM.

The CBR variant may have a Km for PBI-4739 of at least about or has a Km ranging from at least about 0.10 μM, at least about 0.50 μM, at least about 1.00 μM, at least about 1.50 μM, at least about 2.00 μM, at least about 2.10 μM, at least about 2.20 μM, at least about 2.30 μM, at least about 2.40 μM, at least about 2.50 μM, at least about 2.60 μM, at least about 2.70 μM, at least about 2.80 μM, at least about 2.90 μM, at least about 3.00 μM, at least about 3.10 μM, at least about 3.20 μM, at least about 3.30 μM, at least about 3.40 μM, at least about 3.50 μM, at least about 3.60 μM, at least about 3.70 μM, at least about 3.80 μM, at least about 3.90 μM, at least about 4.00 μM, or at least about 5.00 μM, or has a Km ranging from at least about 0.01 μM to at least about 5.00 μM, at least about 0.01 μM to at least about 4.00 μM, at least about 0.01 μM to at least about 3.00 μM, at least about 0.01 μM to at least about 2.50 μM, at least about 0.01 μM to at least about 2.00 μM, at least about 0.01 μM to at least about 1.00 μM, at least about 0.01 μM to at least about 0.80 μM, at least about 0.01 μM to at least about 0.50 μM, 1.00 μM to at least about 5.00 μM, at least about 1.00 μM to at least about 4.00 μM, at least about 1.00 μM to at least about 3.00 μM, at least about 1.00 μM to at least about 2.50 μM, at least about 1.00 μM to at least about 2.00 μM, 2.00 μM to at least about 5.00 μM, at least about 2.00 μM to at least about 4.00 μM, at least about 2.00 μM to at least about 3.00 μM, at least about 2.00 μM to at least about 2.50 μM, 3.00 μM to at least about 5.00 μM, or at least about 3.00 μM to at least about 4.00 μM.

The CBR variant may have a Km for PBI-4813 that is at least about 1-fold, at least about 2-fold, at least about 3-fold, at least about 4-fold, at least about 5-fold, at least about 6-fold, at least about 7-fold, at least about 8-fold, at least about 9-fold, at least about 10-fold, at least about 15-fold, at least about 20-fold, at least about 30-fold, at least about 4-fold, at least about 50-fold, at least about 60-fold, at least about 70-fold, at least about 80-fold, at least about 90-fold, at least about 100-fold lower, at least about 150-fold, at least about 200-fold, at least about 300-fold, at least about 400-fold, at least about 500-fold, or at least about 1000-fold lower than the Km of a CBR polypeptide of SEQ ID NO: 1 for PBI-4813.

The CBR variant may have a Km for PBI-4813 that is at least about 1-fold, at least about 2-fold, at least about 3-fold, at least about 4-fold, at least about 5-fold, at least about 6-fold, at least about 7-fold, at least about 8-fold, at least about 9-fold, at least about 10-fold, at least about 15-fold, at least about 20-fold, at least about 30-fold, at least about 4-fold, at least about 50-fold, at least about 60-fold, at least about 70-fold, at least about 80-fold, at least about 90-fold, at least about 100-fold lower, at least about 150-fold, at least about 200-fold, at least about 300-fold, at least about 400-fold, at least about 500-fold, or at least about 1000-fold lower than the Km of a CBR polypeptide of SEQ ID NO: 1 for D-luciferin.

The CBR variant may have a Km for PBI-4739 that is at least about 1-fold, at least about 2-fold, at least about 3-fold, at least about 4-fold, at least about 5-fold, at least about 6-fold, at least about 7-fold, at least about 8-fold, at least about 9-fold, at least about 10-fold, at least about 15-fold, at least about 20-fold, at least about 30-fold, at least about 4-fold, at least about 50-fold, at least about 60-fold, at least about 70-fold, at least about 80-fold, at least about 90-fold, at least about 100-fold lower, at least about 150-fold, at least about 200-fold, at least about 300-fold, at least about 400-fold, at least about 500-fold, or at least about 1000-fold lower than the Km of a CBR polypeptide of SEQ ID NO: 1 for PBI-4739.

The CBR variant may have a Km for PBI-4739 that is at least about 1-fold, at least about 2-fold, at least about 3-fold, at least about 4-fold, at least about 5-fold, at least about 6-fold, at least about 7-fold, at least about 8-fold, at least about 9-fold, at least about 10-fold, at least about 15-fold, at least about 20-fold, at least about 30-fold, at least about 4-fold, at least about 50-fold, at least about 60-fold, at least about 70-fold, at least about 80-fold, at least about 90-fold, at least about 100-fold lower, at least about 150-fold, at least about 200-fold, at least about 300-fold, at least about 400-fold, at least about 500-fold, or at least about 1000-fold lower than the Km of a CBR polypeptide of SEQ ID NO: 1 for D-luciferin.

The CBR variant may have a relative Vmax for PBI-4813 that is at least about 1-fold, at least about 2-fold, at least about 3-fold, at least about 4-fold, at least about 5-fold, at least about 6-fold, at least about 7-fold, at least about 8-fold, at least about 9-fold, at least about 10-fold, at least about 15-fold, at least about 20-fold, at least about 30-fold, at least about 4-fold, at least about 50-fold, at least about 60-fold, at least about 70-fold, at least about 80-fold, at least about 90-fold, at least about 100-fold lower, at least about 150-fold, at least about 200-fold, at least about 300-fold, at least about 400-fold, at least about 500-fold, or at least about 1000-fold higher than the relative Vmax of a CBR polypeptide of SEQ ID NO: 1 for PBI-4813.

The CBR variant may have a relative Vmax for PBI-4739 that is at least about 1-fold, at least about 2-fold, at least about 3-fold, at least about 4-fold, at least about 5-fold, at least about 6-fold, at least about 7-fold, at least about 8-fold, at least about 9-fold, at least about 10-fold, at least about 15-fold, at least about 20-fold, at least about 30-fold, at least about 4-fold, at least about 50-fold, at least about 60-fold, at least about 70-fold, at least about 80-fold, at least about 90-fold, at least about 100-fold lower, at least about 150-fold, at least about 200-fold, at least about 300-fold, at least about 400-fold, at least about 500-fold, or at least about 1000-fold higher than the relative Vmax of a CBR polypeptide of SEQ ID NO: 1 for PBI-4739.

d. Fusion Proteins

In some embodiments, the CBR variant of the invention has one or more heterologous amino acid sequences at the N-terminus, C-terminus, or both (a fusion polypeptide such as one with an epitope or fusion tag), which optionally directly or indirectly interact with a molecule of interest. In some embodiments, the presence of the heterologous sequence(s) does not substantially alter the luminescence of the CBR variant either before or after the interaction with the molecule of interest. The heterologous amino acid sequence may be any protein of interest such as RNasin or RNase, and/or a channel protein, a receptor, a membrane protein, a cytosolic protein, a nuclear protein, a structural protein, a phosphoprotein, a kinase, a signaling protein, a metabolic protein, a mitochondrial protein, a receptor associated protein, a fluorescent protein, an enzyme substrate, a transcription factor, a transporter protein and/or a targeting sequence, e.g., a myristilation sequence, a mitochondrial localization sequence, or a nuclear localization sequence, that directs the hydrolase fragment, for example, a fusion protein, to a particular location.

In some embodiments, the heterologous amino acid sequence is an epitope tag. In some embodiments, the heterologous amino acid sequence is one which, during or after interaction with a molecule of interest, undergoes a conformational change, which in turn alters the activity of the CBR variant e.g., a CBR variant with such an amino acid sequence is useful to detect allosteric interactions. The CBR variant or a fusion with the CBR variant or a fragment thereof may be employed as a reporter.

The CBR variants of the present invention may be coupled to any protein of interest or molecule of interest. In some embodiments, the variants are fusion proteins, for example some variants are coupled to a HALOTAG® polypeptide (also referred to as “HT7”) attached at either the N-terminus or the C-terminus. In some embodiments, a fusion or chimeric protein contains a CBR variant joined at the N-terminus to a HALOTAG® fusion protein (Promega). In other embodiments, a fusion or chimeric protein contains a CBR variant joined at the C-terminus to a HALOTAG® fusion protein. Signal sequences, in combination with membrane anchoring sequences, may be used to position or display CBR variants on the outer surface of the cellular membrane. Other methods, known in the art may also be used to position CBR variants to the membrane or other locations within the cell.

e. Vectors and Host Cells Encoding the Modified Luciferase or Fusions Thereof

Once a desirable nucleic acid molecule encoding a CBR variant or a fragment thereof, such as one with luminescence activity or which may be complemented by another molecule to result in luminescence activity, or a fusion thereof with luminescence activity, is prepared, an expression cassette encoding the CBR variant or a fragment thereof, e.g., one for complementation, or a fusion thereof with luminescence activity, may be prepared. For example, a nucleic acid molecule comprising a nucleic acid sequence encoding a CBR variant is optionally operably linked to transcription regulatory sequences, e.g., one or more enhancers, a promoter, a transcription termination sequence or a combination thereof, to form an expression cassette. The nucleic acid molecule or expression cassette may be introduced to a vector, e.g., a plasmid or viral vector, which optionally includes a selectable marker gene, and the vector introduced to a cell of interest, for example, a prokaryotic cell such as E. coli, Streptomyces spp., Bacillus spp., Staphylococcus spp. and the like, as well as eukaryotic cells including a plant (dicot or monocot), fungus (including yeast, e.g., Pichia, Saccharomyces or Schizosaccharomyces), or a mammalian cell, lysates thereof, or to an in vitro transcription/translation mixture. Mammalian cells include but are not limited to bovine, caprine, ovine, canine, feline, non-human primate, e.g., simian, and human cells. Mammalian cell lines include, but are not limited to, CHO, COS, HEK293, HeLa, CV-1, SH-SY5Y, and NIH 3T3 cells, although numerous other cell lines can also be used as well.

The expression of an encoded CBR variant may be controlled by any promoter capable of expression in prokaryotic cells or eukaryotic cells including synthetic promoters. Prokaryotic promoters include, but are not limited to, SP6, T7, T5, tac, bla, trp, gal, lac or maltose promoters, including any fragment that has promoter activity. Eukaryotic promoters include, but are not limited to, constitutive promoters, e.g., viral promoters such as CMV, SV40 and RSV promoters, as well as regulatable promoters, e.g., an inducible or repressible promoter such as the Tet promoter, the hsp70 promoter and a synthetic promoter regulated by CRE, including any fragment that has promoter activity. The expression of an encoded CBR variant may also be controlled by post-transcriptional processes, such as by regulation of RNA processing or regulation of translation, for example by RNAi, miRNA, shRNA, siRNA, or by RNA or protein degradation. The nucleic acid molecule, expression cassette and/or vector of the invention may be introduced to a cell by any method including, but not limited to, calcium-mediated transformation, electroporation, microinjection, lipofection, and the like.

f. Optimized Sequences Encoding the CBR Variants

Also provided is an isolated nucleic acid molecule (polynucleotide) comprising a nucleic acid sequence encoding a CBR variant of the invention, a functional fragment thereof or a fusion protein thereof. In some embodiments, the isolated nucleic acid molecule comprises a nucleic acid sequence which is optimized for expression in at least one selected host. Optimized sequences include sequences which are codon optimized, i.e., codons which are employed more frequently in one organism relative to another organism, e.g., a distantly related organism, as well as modifications to add or modify Kozak sequences and/or introns, and/or to remove undesirable sequences, for instance, potential transcription factor binding sites. Such optimized sequences can provide enhanced expression, e.g. increased levels of protein expression, when introduced into a host cell. Examples of optimized sequences are disclosed in U.S. Pat. No. 7,728,118 and U.S. Pat. Appl. Publ. Nos. 2008/0070299 and 2008/0090291, each of which is incorporated by reference herein.

In some embodiments, the polynucleotide includes a nucleic acid sequence encoding a CBR variant of the invention, which nucleic acid sequence is optimized for expression in a mammalian host cell in culture (e.g., CHO cells), in a living animal (e.g., mouse), or in a tissue-specific cell in a living animal. In some embodiments, the polynucleotide may include a codon optimized sequence of any one of SEQ ID NOs: 2-4. In some embodiments, the polynucleotide may include a polynucleotide sequence of any one of SEQ ID NOs: 7-9. In some embodiments, the polynucleotide may include a polynucleotide sequence of SEQ ID NO: 10. In some embodiments, the nucleic acid sequence is optimized for expression in a bacterial cell or plant. In some embodiments, an optimized polynucleotide no longer hybridizes to the corresponding non-optimized sequence, e.g., does not hybridize to the non-optimized sequence under medium or high stringency conditions. The term “stringency” is used in reference to the conditions of temperature, ionic strength, and the presence of other compounds, under which nucleic acid hybridizations are conducted. With “high stringency” conditions, nucleic acid base pairing will occur only between nucleic acid fragments that have a high frequency of complementary base sequences. Thus, conditions of “medium” or “low” stringency are often used when it is desired that nucleic acids that are not completely complementary to one another be hybridized or annealed together. The art knows well that numerous equivalent conditions can be employed to comprise medium or low stringency conditions.

In some embodiments, the polynucleotide has less than 90%, e.g., less than 80%, nucleic acid sequence identity to the corresponding non-optimized sequence and optionally encodes a polypeptide having at least 60%, e.g., at least 65%, 70%, 75%, 80%, 85%, 90%, 95%, 96%, 97%, 98%, 99%, or 100%, amino acid sequence identity with the polypeptide encoded by the non-optimized sequence. Constructs, e.g., expression cassettes, and vectors comprising the isolated nucleic acid molecule, e.g., with optimized nucleic acid sequence, as well as kits comprising the isolated nucleic acid molecule, construct or vector are also provided.

A nucleic acid molecule comprising a nucleic acid sequence encoding a CBR variant of the invention, a fragment thereof or a fusion thereof is optionally optimized for expression in a particular host cell and also optionally operably linked to transcription regulatory sequences, e.g., one or more enhancers, a promoter, a transcription termination sequence or a combination thereof, to form an expression cassette.

In some embodiments, a nucleic acid sequence encoding a CBR variant of the invention, a fragment thereof or a fusion thereof is optimized by replacing codons, e.g., at least 25% of the codons in a parental CBR sequence with codons which are preferentially employed in a particular (selected) cell. Preferred codons have a relatively high codon usage frequency in a selected cell, and preferably their introduction results in the introduction of relatively few transcription factor binding sites for transcription factors present in the selected host cell, and relatively few other undesirable structural attributes. Examples of undesirable structural attributes include, but not limited to, restriction enzyme sites, eukaryotic sequence elements, vertebrate promoter modules and transcription factor binding sites, response elements, E. coli sequence elements, mRNA secondary structure. Thus, the optimized nucleic acid product may have an improved level of expression due to improved codon usage frequency, and a reduced risk of inappropriate transcriptional behavior due to a reduced number of undesirable transcription regulatory sequences.

An isolated and optimized nucleic acid molecule may have a codon composition that differs from that of the corresponding wild-type nucleic acid sequence at more than 30%, 35%, 40% or more than 45%, e.g., 50%, 55%, 60% or more of the codons. Exemplary codons for use in the invention are those which are employed more frequently than at least one other codon for the same amino acid in a particular organism and, in some embodiments, are also not low-usage codons in that organism and are not low-usage codons in the organism used to clone or screen for the expression of the nucleic acid molecule. Moreover, codons for certain amino acids (i.e., those amino acids that have three or more codons), may include two or more codons that are employed more frequently than the other (non-preferred) codon(s). The presence of codons in the nucleic acid molecule that are employed more frequently in one organism than in another organism results in a nucleic acid molecule which, when introduced into the cells of the organism that employs those codons more frequently, is expressed in those cells at a level that is greater than the expression of the wild-type or parent nucleic acid sequence in those cells.

In some embodiments of the invention, the codons that are different are those employed more frequently in a mammal, while in still other embodiments, the codons that are different are those employed more frequently in a plant or bacteria. Preferred codons for different organisms are known to the art, e.g., see http://www.kazusa.or.jp./codon/. A particular type of mammal, e.g., a human, may have a different set of preferred codons than another type of mammal. Likewise, a particular type of plant or bacteria may have a different set of preferred codons than another type of plant or bacteria. In some embodiments of the invention, the majority of the codons that differ are ones that are preferred codons in a desired host cell. Preferred codons for organisms including mammals (e.g., humans) and plants are known to the art (e.g., Wada et al., Nucl. Acids Res., 18:2367 (1990); Murray et al., Nucl. Acids Res., 17:477 (1989)).

### 3. NEAR-INFRARED BIOLUMINESCENCE SYSTEMS

The present disclosure also provides near-IR bioluminescence systems that include the CBR variants, described above, and a novel luciferin derivative. Certain combinations of the CBR variants and novel luciferin derivatives provide significant technical advantages for bioluminescent assays especially for in depth tissue imaging because near-IR luminescence is generated. The disclosed invention provides enhanced detection of signals from deep animal tissues, in part, because of the longer wavelength are not absorbed by the tissues, but also because there is an improvement in light emission over the parental CBR and known luciferin substrates. In addition, other reagents may be included to bioluminescence systems, including but not limited to those that inhibit or prevent inactivation of the CBR variant or otherwise extend or enhance luminescent signal.

a. Novel Luciferin Derivatives

The near-IR bioluminescence systems may include novel red-shifted luciferin derivatives, such as those disclosed in PCT application No. PCT/US2014/021678, which is incorporated herein by reference in its entirety. For example, the luciferin derivative may include a compound of one of Formulas (Ia), (Ib), (Ic), (II), (III), (IV), (V), (VI), or (VII), or pro-substrates thereof, such as Reductase Substrates, Glycosidase Substrates, Protease and Protease-Dependent Protein Modifying Substrates, Oxidase Substrates, Carboxyl-Based Pro-Substrates, Glutathione Transferase Substrate, Beta-Lactamase Substrates, and Other Pro-Substrates. In some embodiments, the novel luciferin derivative may be PBI-4739 or PBI-4813 (see FIG. 1; disclosed in PCT Application No. PCT/US2014/021678).

(1) Compound of Formula(s) (Ia), (Ib), (Ic), (II), (III), (IV), (V), (VI), or (VII)

The luciferin derivative may include a compound according to Formulas (Ia), (Ib) and (Ic):

wherein


- X is CN or

- each Y is independently halo, SO₃H, C₁₋₄ alkyl, substituted C₁₋₄
  alkyl, OR¹ or NR¹R²;
- each R¹ is H, C₁₋₁₀ alkyl or substituted C₁₋₁₀ alkyl;
- each R² is H, C₁₋₁₀ alkyl or substituted C₁₋₁₀ alkyl; or
- R¹ and R² together form a 4 to 8 membered ring;
- n is 1 to 6;
- two Y substituents may join together to form a ring containing from 5
  to 7 ring atoms; and
- wherein at least one Y is either OH or NR¹R².

The luciferin derivative may include a compound, but are not limited to:

In other embodiments, the luciferin derivative may include a compound according to Formula (II):

wherein


- X is CN or

- Y is OR¹ or NR¹R²;
- R¹ is H, C₁₋₁₀ alkyl or substituted C₁₋₁₀ alkyl; and
- R² is H, C₁₋₁₀ alkyl or substituted C₁₋₁₀ alkyl; or
- R¹ and R² together form a ring.

In a further embodiments, the luciferin derivative may include a compound according to Formula (III):

wherein


- X is CN or

- Y is OR¹ or NR¹R²;
- R¹ is H, C₁₋₁₀ alkyl or substituted C₁₋₁₀ alkyl; and
- R² is H, C₁₋₁₀ alkyl or substituted C₁₋₁₀ alkyl; or
- R¹ and R² together form a ring.

In yet other embodiments, the luciferin derivative may include a compound according to Formula (IV):

wherein


- X is CN or

- Y is OR¹ or NR¹R²;
- R¹ is H, C₁₋₁₀ alkyl or substituted C₁₋₁₀ alkyl; and
- R² is H, C₁₋₁₀ alkyl or substituted C₁₋₁₀ alkyl; or
- R¹ and R² together form a ring.

In other embodiments, the luciferin derivative may include a compound according to Formula (V):

wherein


- X is CN or

- Y is OR¹ or NR¹R²;
- R¹ is H, C₁₋₁₀ alkyl or substituted C₁₋₁₀ alkyl; and
- R² is H, C₁₋₁₀ alkyl or substituted C₁₋₁₀ alkyl; or
- R¹ and R² together form a ring.

In other embodiments, the luciferin derivative may include a compound according to Formula (VI):

wherein


- X is CN or

- Y is OR¹ or NR¹R²;
- R¹ is H, C₁₋₁₀ alkyl or substituted C₁₋₁₀ alkyl; and
- R² is H, C₁₋₁₀ alkyl or substituted C₁₋₁₀ alkyl; or
- R¹ and R² together form a ring.

In an additional embodiments, the luciferin derivative may include a compound according to Formula (VII):

wherein


- X is CN or

- each Y is independently OR¹ or NR¹R²;
- each R¹ is independently H, C₁₋₁₀ alkyl or substituted C₁₋₁₀ alkyl;
  and
- each R² is independently H, C₁₋₁₀ alkyl or substituted C₁₋₁₀ alkyl; or
- R¹ and R² together form a ring.

In certain embodiments, the emission maximum of the compounds is at least about 650 nm, about 655 nm, about 680, or about 760 nm.

(2) Pro-Substrates

The present invention also provides compounds which are substrates for various non-luciferase enzymes and are pro-substrates for luciferase enzymes. The non-luciferase enzymes include, but are not limited to, reductases, glycosidases, proteases, peptidases, oxidases, esterases, cytochrome P450s, beta-lactamases, glycosylases and glutathione transferases.

In some embodiments, these pro-substrates have a substituent that is a substrate for a non-luciferase position at the one or more Y positions which is cleaved to form

wherein


- X is CN or

- each Y is OH or NH₂, and
- n is 1 to 3.

(a) Reductase Substrates

In some embodiments, the compound is a reductase substrate. In some embodiments, a reductase substrate has the formula:

wherein


- X is CN or

- Y is OR;
- R is

- W is S, NR_(N), or O;
- Z is S, NR_(N), O or CH; and
- R_(N) is H, C₁₋₄ alkyl, or substituted C₁₋₄ alkyl.

In some embodiments, a reductase substrate has the formula:

wherein


- X is CN or

- Y is OR¹ or NR¹R²; and
- R¹ is H, C₁₋₁₀ alkyl or substituted C₁₋₁₀ alkyl;
- R² is H, C₁₋₁₀ alkyl or substituted C₁₋₁₀ alkyl; or
- R¹ and R² together form a 4 to 8 membered ring.

In some embodiments, a reductase substrate has the formula:

wherein


- X is CN or

- Y is OR; and
- R is

Reductase substrates include, but are not limited to, the following compounds:

(b) Glycosidase Substrates

In some embodiments, the compound is a glycosidase substrate. In some embodiments, the glycosidase substrate is a compound of formula:

wherein


- X is CN or

- Y is OR;
- R is

- A is OR or NHAc;
- each R₅ is independently H, a monosaccharide or a polyethylene glycol
  moiety of up to 40 units.

Glycosidase substrates include, but are not limited to, the following compounds:

(c) Protease and Protease-Dependent Protein Modifying Substrates

In some embodiments, the compound is a protease or protease-dependent protein modifying substrate. In some embodiments, the substrate is a compound of formula:

wherein


- X is CN or

- Y is NHR;
- R is

- R₇ is an amino acid side chain;
- R₆ is H, a nitrogen protecting group, or a chain of up to 20 amino
  acids.

Suitable nitrogen protecting groups include, but are not limited to, those traditionally known to those skilled in the art, such as Boc, Cbz, Ac and Fmoc.

These substrates include, but are not limited to, the following compounds:

(d) Oxidase Substrates

In some embodiments, the compound is an oxidase substrate. In some embodiments, an oxidase substrate is a compound of formula:

wherein


- X is —CH(OR₁₀)₂;
- R₁₀ is C₁₋₄ alkyl, substituted C₁₋₄ alkyl, benzyl, or substituted
  benzyl;
- Y is OR¹ or NR¹R²; and
- R¹ is H, C₁₋₁₀ alkyl or substituted C₁₋₁₀ alkyl;
- R² is H, C₁₋₁₀ alkyl or substituted C₁₋₁₀ alkyl; or
- R¹ and R² together form a 4 to 8 membered ring.

In some embodiments, an oxidase substrate is a compound of formula:

wherein


- X is CN or

- Y is H or OR; and
- R is C₁₋₁₀ alkyl, substituted C₁₋₁₀ alkyl, aryl, substituted aryl,
  aralkyl or substituted aralkyl.

In some embodiments, an oxidase substrate is a compound of formula:

wherein


- X is —CH(OR₁₀)₂;
- R₁₀ is C₁₋₄ alkyl, substituted C₁₋₄ alkyl, benzyl or substituted
  benzyl;
- Y is OR¹ or NR¹R²; and
- R¹ is H, C₁₋₁₀ alkyl or substituted C₁₋₁₀ alkyl;
- R² is H, C₁₋₁₀ alkyl or substituted C₁₋₁₀ alkyl; or
- R¹ and R² together form a 4 to 8 membered ring.

Oxidase substrates include, but are not limited to, the following compounds:

(e) Carboxyl-Based Pro-Substrates

In some embodiments, the compound is a carboxyl-based pro-substrate. In some embodiments, the carboxyl-based pro-substrate is a compound of formula:

wherein


- R₈ is CH₂OH, C(O)R₁₀ or —C(O)ZR₉;
- Z is O or NH;
- R₉ is C₁₋₇ alkyl or substituted C₁₋₇ alkyl;
- R₁₀ is a peptide;
- Y is OR¹ or NR¹R²; and
- R¹ is H, C₁₋₁₀ alkyl or substituted C₁₋₁₀ alkyl;
- R² is H, C₁₋₁₀ alkyl or substituted C₁₋₁₀ alkyl; or
- R¹ and R² together form a 4 to 8 membered ring.

Carboxyl-based pro-substrates include, but are not limited to, the following compounds:

(f) Glutathione Transferase Substrate

In some embodiments, the compound may be a glutathione transferase substrate. In some embodiments, the glutathione transferase substrate is a compound of formula:

wherein


- X is CN or

- Y is OR;
- R is

- - each R₁₁ is independently H, C₁₋₆ alkyl, substituted C₁₋₆ alkyl,
    CF₃, halogen, NO₂, CO₂R₁₂ or any two adjacent R₁₁ can form a fused
    ring provided that at least one of R₁₁ is NO₂; and
  - R₁₂ is H, C₁₋₆ alkyl or substituted C₁₋₆ alkyl.

Glutathione transferase substrates include, but are not limited to, the following compounds:

(g) Beta-Lactamase Substrates

In some embodiments, the compound is a beta-lactamase substrate. In some embodiments, the beta-lactamase substrate is a compound of formula:

wherein


- X is CN or

- Y is O, NH, N(C₁₋₇ alkyl), or N(substituted C₁₋₇ alkyl);
- Z is absent or O;
- A is C₁₋₄ alkylene or substituted C₁₋₄ alkylene; and
- R₁₄ is H, phenacetyl, or a cephalosporin side chain.

Suitable cephalosporin side chains include those known to one of ordinary skill in the art.

Beta-lactamase substrates include, but are not limited to, the following compounds:

(h) Other Pro-Substrates

The present invention also provides compounds which react with various biologically important small molecules, such as hydrogen peroxide, and are pro-substrates for luciferase enzymes. In some embodiments, these compounds are reactive to hydrogen peroxide. In some embodiments, these compounds have the formula:

wherein


- X is CN or

- Y is L-R;
- L is a linker; and
- R is a boronic acid or borate ester.

In some embodiments, R is —B(OR15)2; wherein each R15 is independently selected from H and C1-4 alkyl. In some embodiments, R is

wherein each R16 and R17 is independently selected from H, C1-4 alkyl, substituted C1-4 alkyl, CF3, phenyl or substituted phenyl; or R16 and R17 together can be an alkyl ring having from 3-7 carbons or can be replaced by a fused 6-membered aromatic ring.

In some embodiments, the linker is a direct bond. In other embodiments, the linker is

wherein

A is —C6(R20)4—, —O—C6(R20)4— or —(CR21═CR21)n— or —S—C6(R20)4— or —NR′—C6(R20)4 or a direct bond;

R′ is H, C1-4 alkyl, or substituted C1-4 alkyl;

each R23 is independently halo, H, C1-4 alkyl, substituted C1-4 alkyl, C1-4 hydroxyalkyl, substituted C1-4 hydroxyalkyl, C1-4 alkylcarboxylic acid or substituted C1-4 alkylcarboxylic acid;

each R20 is independently H, halo, CH3, OCH3, or NO2;

each R21 is independently H or CH3;

n is 1 or 2; and

X is a selected from —O—,

These compounds include, but are not limited to, the following compounds:

### 4. METHODS OF USING THE CBR VARIANTS AND/OR BIOLUMINESCENT SYSTEMS

The CBR variants and near-IR bioluminescence systems may be used in any way that luciferases and luciferase substrates, e.g., luciferin and luciferin derivatives, have been used. The CBR variants and/or near-IR bioluminescent systems may be used with a sample (including cells, tissues, animals, etc.) or with in vivo imaging and detected using various microscopy and imaging techniques. The CBR variants and/or near-IR bioluminescent systems may be used as a transcriptional reporter or as a biosensor. The CBR variants and/or near-IR bioluminescent systems may be used in dual-color assays or multiplexing. The CBR variants and/or near-IR bioluminescent systems may be used in assays to detect the presence or activity of enzymes such as non-luminescent enzymes. For example, they may be used in a bioluminogenic method which employs an analog of luciferins to detect one or more molecules in a sample, e.g., an enzyme, a cofactor for an enzymatic reaction, an enzyme substrate, an enzyme inhibitor, an enzyme activator, or OH radicals, or one or more conditions, e.g., redox conditions. In some embodiments, the CBR variants and/or near-IR bioluminescent systems may be used as an energy donor to another molecule (e.g., to a fluorophore, a chromophore, or a nanoparticle). The CBR variants and near-IR bioluminescence systems may be used in protein proximity assays or protein complementation assays. The disclosed CBR variants and/or near-IR bioluminescent systems of the invention are also useful in in situ methods of analyzing cells. Methods of performing in situ analysis of cells using a luciferase are known in the art. The disclosed CBR variants and/or near-IR bioluminescent systems of the invention may be used to distinguish between substrates and inhibitors of an enzyme. The screening may be performed either in vitro or in vivo.

The sample may include an animal (e.g., a vertebrate), a plant, a fungus, physiological fluid (e.g., blood, plasma, urine, mucous secretions and the like), a cell, a cell lysate, a cell supernatant, or a purified fraction of a cell (e.g., a subcellular fraction). The presence, amount, spectral distribution, emission kinetics, or specific activity of such a molecule may be detected or quantified. The molecule may be detected or quantified in solution, including multiphasic solutions (e.g., emulsions or suspensions), or on solid supports (e.g., particles, capillaries, or assay vessels).

The luciferase reaction may be restricted or limited by time, enzyme concentration, and/or substrate concentration. Reaction conditions may be adjusted so that the luciferase reaction is carried out under conditions that result in about, at least about, or at most about 20, 30, 40, 50, 60, 70, 80, 90, 95, 96, 97, 98, 99 or 100% completion, or any range derivable therein. For example, the reaction may be carried out at a temperature between about 20° C. and about 45° C., about 20° C. and about 40° C., about 20° C. and about 35° C., about 20° C. and about 30° C., about 20° C. and about 25° C., between about 25° C. and about 45° C., about 25° C. and about 40° C., about 25° C. and about 35° C., about 25° C. and about 30° C., between about 30° C. and about 45° C., about 30° C. and about 40° C., about 30° C. and about 35° C., between about 35° C. and about 45° C., or between about 35° C. and about 40° C. The reaction may be carried out at 20° C., 21° C., 22° C., 23° C., 24° C., 25° C., 26° C., 27° C., 28° C., 29° C., 30° C., 31° C., 32° C., 33° C., 34° C., 35° C., 36° C., 37° C., 38° C., 39° C., 40° C., 41° C., 42° C., 43° C., 44° C., or 45° C. These temperature condition and/or reaction may be maintained or measured for 1 min, 5 min, 10 min, 15 min, 20 min, 25 min, 30 min, 35 min, 40 min, 45 min, 50 min, 55 min, 60 min, 65 min, 70 min, 75 min, 80 min, 85 min, 90 min, 100 min, 110 min, 120 min, 130 min, 140 min, 150 min, 160 min, 170 min, 180 min, 190 min, 200 min, 210 min, 220 min, 230 min, 240 min, 250 min, or more.

a. In Vivo Imaging

The CBR variants and near-IR (NIR) bioluminescence systems may be used for in vivo imaging and may improve the speed, detection limit, and depth penetration of bioluminescence imaging. The CBR variants and near-IR bioluminescence systems provide a means for non-invasive animal imaging for a variety of applications, e.g. understanding tumor biology, evaluation of potential therapeutic compounds. For example, the methods described herein can be used for the rapid and inexpensive evaluation of tumor progression and response to anti-cancer therapeutics in small animals, e.g., using transgenic non-human animals, e.g., mice, that express a luciferase reporter gene linked to a promoter or gene that is expressed, e.g., selectively expressed, in the cells that are desired to be imaged. Expression of a selected protein of interest may be imaged in real time in a living cell or animal, using a cell or transgenic animal that expresses a reporter construct including a nucleic acid encoding a CBR variant linked in frame to a nucleic acid encoding the selected protein of interest, or to the promoter for the selected protein.

The methods may be performed on cells or animals (e.g., non-human mammals, e.g., experimental animals, such as rodents, e.g., rats or mice) that express a CBR variant reporter construct. Generating such cells or animals may be performed using standard molecular biological techniques. Sufficient amounts of the novel luciferin derivative described herein may be added or administered to the cells or animals, and images of the NIR bioluminescence may be obtained using standard imaging methods. The promoter activity, protein expression, protein subcellular localization, protein translocation, and protein half-life may be evaluated in real time in living cells and animals.

When an experimental animal is used, the cells containing the NIR bioluminescence can be identified and excised, and evaluated further, e.g., using assays for gene expression, protein expression, or other genetic or biochemical parameters. The bioluminescence system may be designed with another luciferin/luciferase pair (e.g., with different emission maxima) to allow for simultaneous imaging of bioluminescence from two or more luciferases.

(1) Imaging Methods

The methods described herein can be practiced with any imaging system that can detect near infrared bioluminescence. Common imaging systems are available from Xenogen (e.g., IVIS), Hamamatsu, Roper, and Kodak.

(2) Live Cell

In various embodiments, the CBR variants can be used for detecting luminescence in live cells. In some embodiments, a CBR variant can be expressed in cells (as a reporter or otherwise), and the cells treated with a luciferase substrate, e.g., luciferin, luciferin derivative, e.g., a novel luciferin derivative such as PBI-4739 or PBI-4813, or functional analog, which may permeate cells in culture, react with the CBR variant and generate luminescence. PBI-4739 or PBI-4813 show comparable non-toxicity to D-luciferin in terms of cell toxicity studies. In some embodiments, versions of PBI-4739 or PBI-4813 containing chemical modifications known to increase the stability of native luciferin derivative in media may be synthesized and used for more robust, live cell CBR variant-based reporter assays. In still other embodiments, a sample (including cells, tissues, animals, etc.) containing a CBR variant and/or a novel luciferin derivative of the present invention may be assayed using various microscopy and imaging techniques. In still other embodiments, a secretable CBR variant may be expressed in cells as part of a live-cell reporter system.

b. Use as Transcriptional Reporters

The CBR variants and near-IR bioluminescence systems may be used as genetic transcriptional reporter systems. The CBR variant or fragment thereof may be used to study transcriptional expression patterns of any promoter and/or gene, such as genes involved in development. In some embodiments, the CBR variant or fragment thereof could be operably linked to transcription regulatory sequences, e.g., one or more enhancer, a promoter, a transcription termination sequence or a combination thereof, to form an expression cassette. For example, the CBR variant could be operably linked to a minimal promoter and a cAMP-response element (CRE).

In certain embodiments, provided is a method for measuring the activity of a promoter in a sample, wherein the promoter is operably linked to a gene encoding a CBR variant enzyme. The method includes (a) contacting the sample with a luciferin or novel luciferin derivative; and (b) determining the activity of the promoter by measuring luminescence of the sample, wherein the sample comprises the promoter. The promoter may be operably linked to the gene via a translational or transcriptional fusion. A biological pathway of interest, for example, may be examined by treating a cell that comprises the promoter, which is operably linked to a gene encoding the luminescent enzyme, with an inducer agent of the pathway. This promoter activity may then be measured and monitored to study any correlation between the activity of the promoter and the pathway of interest, as well as obtain kinetic measurements relating to gene expression (e.g. inducibility, repression and activation).

c. Multiplexing

The CBR variant and bioluminescence system may be used in a multiplexed reaction with another enzyme (e.g. a luciferase) that emits light at a different wavelength. In some embodiments, the CBR variant may be used as a transcriptional reporter and paired with a luciferase that emits light at a different wavelength contained in an assay reagent. In some embodiments, the CBR variant may be used with one or more additional luciferases, where the luminescence of each luciferase may be separately measured through the use of selective enzyme inhibitors. For example, the luminescence of the CBR variant may be measured upon addition of appropriate substrates and buffers, followed by measurement of a second luciferase upon a subsequent addition of appropriate substrates and buffers and one or more inhibitors selective for the first luciferase.

In some embodiments, the CBR variant of the present invention may be used as a functional reporter for a particular gene and the second enzyme in the multiplex reaction may be used as a functional reporter for a second gene. In some embodiments, the CBR variant may be used with one or more additional luciferases, where the luminescence of each luciferase may be easily resolved using a luminometer with wavelength-discriminating filters enabling the measurement of both signals from the same sample. For example, luminescence generated from the CBR variant enzyme (between approximately 725-750 nm) and green CHROMA-LUC™ (approximately 537 nm) can be easily resolved using a luminometer with wavelength-discriminating filters, enabling the measurement of both signals from the same sample.

In some embodiments, a CBR variant may be used as transcriptional reporter and paired with either aequorin or a cAMP circularly-permuted firefly luciferase biosensor, or both simultaneously, to detect multiple pathways in a single sample. In such a system, for example, aequorin could be used for the detection and/or measurement of calcium, the biosensor for the detection and/or measurement of cAMP, and the CBR variant for monitoring of downstream gene expression.

The CBR variant may be multiplexed with a luciferase derived from: beetle luciferases, such as firefly (e.g., Photinus pyralis (e.g., Luc2; Promega Corp) or Photuris pennsylvanica luciferases) or a different click beetle luciferase (Pyrophorus plagiophthalamus or Pyrearinus termitilluminans), such as green click beetle luciferase (CHROMA-LUC™; Promega Corp.), a bioluminescent decapod, e.g. Oplophorus luciferase, such as Oplophorus gracilirostris luciferases or variants thereof, e.g., NanoLuc® luciferase (Promega Corporation), as described in U.S. Pat. Nos. 8,557,970 and 8,669,103, and U.S. Patent Publication Nos. 2014/0223590 and U.S. Patent Publication No. 2014/0227759, marine organisms such as cnidarians (e.g., Renilla reniformis luciferase), copepod luciferases, such as Gaussia luciferase, e.g., Gaussia princeps luciferase, Metridia luciferases, e.g., Metridia longa and Metridia pacifica luciferases, Vargula luciferases, e.g., Vargula hilgendorfii luciferase, Pleuromamma xiphias luciferase, a glowworm luciferase (e.g. Phrixothrix hirtus), and variants, recombinants, and mutants thereof. The CBR variants may be multiplexed with green fluorescent protein (GFP), photoproteins such as Aequorin, obelin, iPhotina, and variants, recombinants, and mutants thereof. For example, if a CBR variant of the present invention is used as a functional reporter, then the green firefly luciferase or green CHROMA-LUC™ luciferase could be used to control for non-specific effects on genetic regulation or to normalize for transfection efficiency.

d. Detection of Non-Luminescent Enzymes

The CBR variant and bioluminescence system containing the pro-substrate described herein may be used in luminescence-based assays to detect the activity of a non-luminescent enzyme. The CBR variant and pro-substrate may be contained in an assay reagent for measuring a specific aspect of cellular physiology, for example ATP to estimate cell viability, or caspase activity to estimate cellular apoptosis. The CBR variant and the pro-substrate may be added to a sample suspected of containing the non-luminescent enzyme of interest. The non-luminescent enzyme present in the sample reacts with the pro-substrate releasing the luciferin substrate for the CBR variant to use as a substrate and thus luminescence is generated and measured. In some embodiments, the non-luminescent enzyme of interest is a reductase, glycosidase, protease, serine proteases, threonine proteases, cysteine proteases, such as caspase-3 and caspase-8, aspartate proteases, glutamic acid proteases, and metalloproteases, peptidase, oxidase, such as a monoamine oxidase enzyme, esterase, cytochrome P450, beta-lactamase, glycosylase and glutathione transferase protease enzyme, such as a glutathione S-transferase enzyme. In some embodiments, the pro-substrates, as described herein may be used to detect the non-luminescent enzymes.

e. Bioluminescence Resonance Energy Transfer (BRET)

The CBR variant and bioluminescence system may be used in any method for detecting ligand-protein and/or protein-protein interactions. In various embodiments, the CBR variant enzymes may be used to transfer energy to an energy acceptor. One such method is Bioluminescence Resonance Energy Transfer (BRET). With respect to BRET, energy transfer from a bioluminescent donor to a fluorescent acceptor results in a shift in the spectral distribution of the emission of light. This energy transfer may enable real-time monitoring of protein-protein or ligand-protein interaction in vitro or in vivo.

In some embodiments, the CBR variant enzymes used in BRET analysis can be used to determine if two molecules are capable of binding to each other or co-localize in a cell. For example, a CBR variant enzyme can be used as a bioluminescence donor molecule which is combined with a molecule or protein of interest to create a first fusion protein. In various embodiments, the first fusion protein contains a CBR variant enzyme and a protein of interest. In various embodiments, the first fusion proteins containing the CBR variant enzyme can be used in BRET analysis to detect protein/protein interaction in systems including but not limited to cell lysates, intact cells, and living animals. In various embodiments, HALOTAG® can be used as a fluorescent acceptor molecule. In some embodiments, HALOTAG® can be fused to a second protein of interest or to a CBR variant enzyme. For example, a CBR variant enzyme can be fused to HALOTAG®, expressed in cells or animals, and labeled with a fluorescent HALOTAG® ligand such as HALOTAG® TMR ligand. The fusion can subsequently be excited to fluoresce in the presence of a cell-permeant CBR variant enzyme substrate. In some embodiments, BRET may be performed using CBR variant enzymes in combination with fluorescent proteins, including but not limited to Green Fluorescent Protein (GFP), Red Fluorescent Protein (RFP), far red fluorescent protein, and near infrared fluorescent proteins, such as iRFP and IFP1.4, or fluorescent labels including fluorescein, rhodamine green, Oregon green, Alexa 488, to name a few non-limiting examples. The fluorescent label may be a fluorescent BRET acceptor that absorbs at the emission of the luciferase (e.g., 650-750 nm), such as IRDye® 800CW, IRDye® 800RS, IRDye® 800 phosphoramidite, IRDye® 750, IRDye® 700DX, IRDye® 700 phosphoramidite, IRDye® 680LT, IRDye® 680RD, IRDy IRDye® e 650, 1,1′,3,3,3′,3′-Hexamethylindotricarbocyanine iodide, 1,1′-Diethyl-2,2′-dicarbocyanine iodide, 1,1′-Diethyl-4,4′-carbocyanine iodide, 1,4,8,11,15,18,22,25-Octabutoxy-29H,31H-phthalocyanine, 2,11,20,29-Tetra-tert-butyl-2,3-naphthalocyanine, 2,3,9,10,16,17,23,24-Octakis(octyloxy)-29H,31H-phthalocyanine, 2,3-Naphthalocyanine Dye content, 2,9,16,23-Tetra-tert-butyl-29H,31H-phthalocyanine, 29H,31H-Phthalocyanine β-form, 3,3′-Diethylthiadicarbocyanine iodide, 3,3′-Diethylthiatricarbocyanine iodide, 3,3′-Diethylthiatricarbocyanine perchlorate, Aluminum 1,8,15,22-tetrakis(phenylthio)-29H,31H-phthalocyanine chloride, Aluminum 2,9,16,23-tetrakis(phenylthio)-29H,31H-phthalocyanine chloride, Aluminum 2,9,16,23-tetraphenoxy-29H,31H-phthalocyanine chloride, Aluminum 2,9,16,23-tetraphenoxy-29H,31H-phthalocyanine hydroxide, Aluminum phthalocyanine chloride, Aluminum phthalocyanine hydroxide, Cobalt(II) 1,2,3,4,8,9,10,11,15,16,17,18,22,23,24,25-hexadecafluoro-29H,31H-phthalocyanine, Cobalt(II) 2,3-naphthalocyanine, Cobalt(II) phthalocyanine β-form, Copper phthalocyanine-3,4′,4?,4?′-tetrasulfonic acid tetrasodium salt, Copper(II) 1,2,3,4,8,9,10,11,15,16,17,18,22,23,24,25-hexadecafluoro-29H,31H-phthalocyanine, Copper(II) 1,4,8,11,15,18,22,25-octabutoxy-29H,31H-phthalocyanine, Copper(II) 2,3,9,10,16,17,23,24-octakis(octyloxy)-29H,31H-phthalocyanine, Copper(II) 2,3-naphthalocyanine, Copper(II) 2,9,16,23-tetra-tert-butyl-29H,31H-phthalocyanine, Copper(II) phthalocyanine-tetrasulfonic acid tetrasodium salt, Copper(II) phthalocyanine, Copper(II) phthalocyanine β-form, Copper(II) phthalocyanine sublimed grade, Copper(II) phthalocyanine, Dilithium phthalocyanine, Disodium phthalocyanine, Gallium(III)-phthalocyanine chloride, IR-775 chloride, IR-780 iodide, IR-783, IR-792 perchlorate, IR-797 chloride, Iron(II) phthalocyanine, Iron(III) phthalocyanine chloride, Iron(III) phthalocyanine-4,4′,4″,4′″-tetrasulfonic acid, compound with oxygen monosodium salt hydrate, Lead(II) phthalocyanine, Lead(II) tetrakis(4-cumylphenoxy)phthalocyanine, Magnesium phthalocyanine, Manganese(II) phthalocyanine, Manganese(III) phthalocyanine chloride, Methylsilicon(IV) phthalocyanine hydroxide, Naphthol Green B, Nickel(II) 1,4,8,11,15,18,22,25-octabutoxy-29H,31H-phthalocyanine, Nickel(II) phthalocyanine-tetrasulfonic acid tetrasodium salt, Nickel(II) phthalocyanine, Poly(copper phthalocyanine), Silicon 2,3-naphthalocyanine bis(trihexylsilyloxide), Silicon 2,3-naphthalocyanine dichloride, Silicon 2,3-naphthalocyanine dihydroxide, Silicon 2,3-naphthalocyanine dioctyloxide, Silicon 2,9,16,23-tetra-tert-butyl-29H,31H-phthalocyanine dihydroxide, Silicon phthalocyanine dichloride, Silicon phthalocyanine dihydroxide, Tin(IV) phthalocyanine oxide, Titanium(IV) phthalocyanine dichloride, Titanyl phthalocyanine, Titanyl phthalocyanine, Vanadyl 2,3-naphthalocyanine, Vanadyl 3,10,17,24-tetra-tert-butyl-1,8,15,22-tetrakis(dimethylamino)-29H,31H-phthalocyanine Dye content 80%, Zinc 1,2,3,4,8,9,10,11,15,16,17,18,22,23,24,25-hexadecafluoro-29H,31H-phthalocyanine, Zinc 1,4,8,11,15,18,22,25-octabutoxy-29H,31H-phthalocyanine, Zinc 2,11,20,29-tetra-tert-butyl-2,3-naphthalocyanine, Zinc 2,3,9,10,16,17,23,24-octakis(octyloxy)-29H,31H-phthalocyanine, Zinc 2,9,16,23-tetra-tert-butyl-29H,31H-phthalocyanine, Zinc phthalocyanine, or Zinc(II) tetranitrophthalocyanine.

f. Protein Proximity Assays for Live Cells or Lytic Formats

In some embodiments, the CBR variant may be used as a circularly permuted (CP) or straight split (SS) luminescent enzyme fusion protein to measure protein proximity. The CBR variant enzyme is permuted or split via insertion of a protease substrate amino acid sequence (e.g., TEV) to generate low bioluminescence. The CBR variant luminescent enzyme is tethered (e.g., via genetic fusion) to a monitor protein. A potential interacting protein is tethered (e.g., via genetic fusion) to a protease (e.g., TEV). When the two monitor proteins interact or are in sufficient proximity (e.g., via a constitutive interaction, a drug stimulus or a pathway response), the CBR variant enzyme is cleaved to generate increased bioluminescence activity. The example may be applied to measurements of protein proximity in cells or in biochemical assays.

g. Protein Complementation Assays

In some embodiments, the disclosed compounds may be used in other methods for detecting ligand-protein and protein-protein interactions or proximity, such as the protein complementation assay (PCA) or enzyme fragmentation assay. Protein complementation assays (PCA) provide a means to detect the interaction of two biomolecules, e.g., polypeptides. PCA utilizes two fragments of the same protein, e.g., the CBR variant, that when brought into close proximity with each other can reconstitute into a functional, active protein. In some embodiments, the PCA utilizing a CBR variant may be used to detect molecular proximity by virtue of the reconstitution of the CBR variant via the binding interaction of enzyme components or subunits. Fragments of the CBR variant are fused to proteins of interest. If the proteins of interest interact, the fragments of the CBR variant interact to reconstitute a full-length CBR variant enzyme.

For example, the CBR variant enzyme can be separated into two fragments at a site(s) tolerant to separation and each fragment of the separated CBR variant enzyme can be fused to one of a pair of polypeptides of interest believed to interact, e.g., FKBP and FRB. If the two polypeptides of interest do in fact interact, the CBR variant enzyme fragments, for example, then come into close proximity with each other to reconstitute the functional, active CBR variant enzyme. In some embodiments, the activity of the reconstituted CBR variant enzyme can then be detected and measured using the disclosed compounds and the cell-permeable substrate. In some embodiments, the split CBR variant enzyme can be used in a more general complementation system similar to lac-Z (Langley et al., PNAS 72:1254-1257 (1975)) or ribonuclease S (Levit and Berger, J. Biol. Chem. 251:1333-1339 (1976)). In some embodiments, a CBR variant enzyme fragment (designated “A”) known to complement with another CBR variant enzyme fragment (“B”) can be fused to a target protein, and the resulting fusion can be monitored via luminescence in a cell or cell lysate containing fragment B. In some embodiments, the source of fragment B could be the same cell (e.g., if the gene for fragment B is integrated into the genome of the cell or is contained on another plasmid within the cell) or it could be a lysate or purified protein derived from another cell. In some embodiments, this same fusion protein (fragment A) could be captured or immobilized using a fusion between fragment B and a polypeptide such as HALOTAG® capable of attachment to a solid support. In some embodiments, luminescence can be used to demonstrate successful capture or to quantify the amount of material captured.

h. Biosensors

The CBR variants may be used as biosensors, which, in the presence of another molecule (e.g., one or more molecules of interest), or under certain conditions, has one or more altered activities. Upon interacting with a molecule of interest or being subject to certain conditions, the biosensor undergoes a conformational change or is chemically altered which causes an alteration of the enzyme activity or luminescence, e.g., specific activity, spectral distribution, or emission kinetics. For example, the CBR variant of the present invention, for example a circularly permuted variant, can comprise an interaction domain for a molecule of interest. Alternatively, for example, the CBR variant may be coupled to an energy acceptor, for example a fluorescent protein, and comprise an interaction domain that alters the efficiency of energy transfer from the enzyme to the energy acceptor. For example, the biosensor could be generated to detect proteases, kinases, a ligand, a binding protein such as an antibody, cyclic nucleotides such as cAMP or cGMP, or a metal such as calcium, by insertion of a suitable sensor region into the CBR variant sequence. One or more sensor region can be inserted at the C-terminus, the N-terminus, and/or at one or more suitable location in the polypeptide sequence, where the sensor region comprises one or more amino acids. In the case of a circularly-permuted CBR variant, the sensor region may be inserted between the N- and C-termini of the parent CBR variant. In addition, one or all of the inserted sensor regions may include linker amino acids to couple the sensor to the remainder of the CBR variant polypeptide.

In some embodiments, full-length circularly permuted CBR variant enzymes may be fused to respective binding partners, e.g., FRB and FKBP, and used in a protein complementation-type assay with the luciferin or novel luciferin derivative. The key difference between the method disclosed herein and traditional protein complementation is that there was no complementation, but rather there was dimerization of two full length enzymes, e.g., circularly permuted CBR variant enzymes.

Briefly, the circularly permuted reporter proteins similarly configured for low activity are fused to both of the fusion protein partners. For example, each fusion partner may be linked to identically structured, permuted reporters. Interaction of the fusion partners brought the permuted reporters into close proximity, thereby allowing reconstitution of a hybrid reporter having higher activity.

### 5. SAMPLE

The disclosed CBR variants and/or near-IR bioluminescent systems may be used with samples containing biological components. The sample may comprise cells and/or tissue. The sample may comprise heterogeneous mixtures of components (including intact cells, cell extracts, cell lysates, bacteria, viruses, organelles, and mixtures thereof) or a single component or homogeneous group of components (e.g., natural or synthetic amino acid, nucleic acid or carbohydrate polymers, or lipid membrane complexes). The compounds are generally non-toxic to living cells and other biological components within the concentrations of use.

The sample may include an animal (e.g., a vertebrate), a plant, a fungus, physiological fluid (e.g., blood, plasma, urine, mucous secretions and the like), a cell, a cell lysate, a cell supernatant, or a purified fraction of a cell (e.g., a subcellular fraction). In certain embodiments, the sample may be a cell. In some embodiments, the sample may be a live cell. The cell may be a eukaryotic cell, e.g., yeast, avian, plant, insect or mammalian cells, including but not limited to human, simian, murine, canine, bovine, equine, feline, ovine, caprine or swine cells, or prokaryotic cells, or cells from two or more different organisms, or cell lysates or supernatants thereof. The cells may not have been genetically modified via recombinant techniques (nonrecombinant cells), or may be recombinant cells which are transiently transfected with recombinant DNA and/or the genome of which is stably augmented with a recombinant DNA, or which genome has been modified to disrupt a gene, e.g., disrupt a promoter, intron or open reading frame, or replace one DNA fragment with another. The recombinant DNA or replacement DNA fragment may encode a molecule to be detected by the methods of the invention, a moiety which alters the level or activity of the molecule to be detected, and/or a gene product unrelated to the molecule or moiety that alters the level or activity of the molecule. The cells may have been genetically modified via recombinant techniques.

### 6. KIT

Kits for using the CBR variant and/or the near-IR bioluminescent systems are provided herein. Such kits comprise an active CBR variant and luciferin substrate. The kits may further include a buffer and instructions. The kit components, compositions, and buffers may also be modified by the addition of suitable components. Suitable kit components, compositions and buffers that may be used in the described methods can also be obtained commercially. The different components may comprise subsets of these parts and may be combined in any way that either facilitates the application of the invention or prolongs storage life.

In some embodiments, the kit comprises a separate container comprising lyophilized luciferase. In some embodiments, the container comprising lyophilized luciferase further comprises lyophilized luciferin or a derivative thereof that is a luciferase substrate.

One or more reagents may be supplied in a solid form or liquid buffer that is suitable for inventory storage, and later for addition into the reaction medium when the method of using the reagent is performed. Suitable packaging is provided.

(1) Containers/Vessels

The reagents included in the kits can be supplied in containers of any sort such that the life of the different components are preserved and are not adsorbed or altered by the materials of the container. For example, sealed glass ampules may contain lyophilized luciferase or buffer that has been packaged under a neutral, non-reacting gas, such as nitrogen. Ampules may consist of any suitable material, such as glass, organic polymers, such as polycarbonate, polystyrene, etc., ceramic, metal or any other material typically employed to hold reagents. Other examples of suitable containers include simple bottles that may be fabricated from similar substances as ampules, and envelopes, that may consist of foil-lined interiors, such as aluminum or an alloy. Other containers include test tubes, vials, flasks, bottles, syringes, or the like. Containers may have a sterile access port such as a bottle having a stopper that can be pierced by a hypodermic injection needle. Other containers may have two compartments that are separated by a readily removable membrane that upon removal permits the components to mix. Removable membranes may be glass, plastic, rubber, etc.

(2) Instructional Materials

The kits may also be supplied with instructional materials. Instructions may be printed on paper or other substrate and/or may be supplied as an electronic-readable medium such as a floppy disc, CD-ROM, DVD-ROM, Zip disc, videotape, audio tape, etc. Detailed instructions may not be physically associated with the kit; instead, a user may be directed to an interne web site specified by the manufacturer or distributor of the kit, or supplied as electronic mail.

### 7. EXAMPLES

The present invention can be utilized as illustrated by the following non-limiting examples.

**Example 1**

**Characterization of Near-IR Substrates PBI-4739 and PBI-4813 with Ultra-Glo™ Luciferase, QuantiLum® Recombinant Luciferase, and Purified Click Beetle Red Luciferase (CBR)**

Materials: The following were used in the Examples: Ultra-Glo™ Luciferase (Promega Cat. No. E140); QuantiLum® Recombinant Luciferase (Promega Cat. No. E1701); Click Beetle Red Luciferase (CBR; 0.5 mg/mL purified; Promega); Bright-Glo™ assay buffer (Promega Cat. No. E264A); PBI-4739 (Promega—See FIG. 1); and PBI-4813 (Promega—See FIG. 1).

Experimental Details. The substrates, PBI-4813 and PBI-4739, were diluted to 1 mM (final concentration) in Bright-Glo™ Assay buffer+1 mM ATP. 50 μL of substrate solution was added to 50 μL of Click Beetle Red, UltraGlo® or QuantiLum® purified enzyme (0.5 mg/mL diluted in DMEM+0.1% Prionex) in triplicate, and then the samples assayed using a Tecan M-1000 plate reader in spectral scan mode.

FIG. 2 provides the spectral data for Click Beetle Red Luciferase with PBI-4813 (A) and PBI-4739 (B), where the spectral maximum was 655 nm for PBI-4813 and 760 nm for PBI-4739. FIG. 3 provides the spectral data for UltraGlo® luciferase with PBI-4813 (A) and PBI-4739 (B), where the spectral maximum was 670 nm for PBI-4813 and 660 nm for PBI-4739. FIG. 4 provides the spectral data for QuantiLum® luciferase (firefly) with PBI-4813 (A) and PBI-4739 (B), where the spectral maximum was 680 nm for PBI-4813 and equal to or greater than 700 nm for PBI-4739.

**Example 2**

**Library Screening**

A library of Click Beetle Red variants was prepared using the Diversify™ PCR Random Mutagenesis Kit (Clontech) according to the manufacturer's instructions using pF4Ag-HT7-CBR as a template, i.e., CBR-HALOTAG® fusion protein. The library of variant DNA was cloned into the pF4Ag-HT7 (Promega) and transformed into 50 μL KRX competent cells (Promega Corporation). The cells were grown overnight on LB-ampicillin plates at 37° C.

Colonies were picked and grown overnight at 37° C. in 200 μL of M9-minimal media (1× M9 salts, 0.1 mM CaCl2, 2 mM MgSO4, 1 mM Thiamine-HCL, 1% gelatin, 0.2% glycerol and 100 μg/mL Ampicillin) in wells of 96-well plates. 10 μL of the overnight culture was diluted into 190 μL M9-minimal media and grown overnight at 37° C. 10 μL of the second overnight culture was diluted into 190 μL M9-minimal induction media (M9 minimal media+0.05% glucose and 0.02% rhamnose) and grown overnight at 25° C.

The cells were assayed via a robot. Briefly, 25 μL of cell culture was added to 25 μL of lysis buffer (50 mM HEPES pH 7.5, 0.3× Passive Lysis Buffer (PLB; Promega Corporation), 0.006U RQ1 DNase1 (Promega Corporation)) and incubated for 3 min. 50 μL of Bright-Glo™ Assay Reagent (Promega Corporation) or PBI-4813 assay reagent (Bright-Glo™ Assay buffer+ATP (to 1 mM) and PBI-4813 (to 5 μM)) was added to lysed cells, and luminescence detected on Tecan Genios Pro (Table 1).

**Example 3**

**Click Beetle Red Mutation Combinations**

Some of the mutations identified in the library screen outlined in Example 2 were combined to identify beneficial combinations of mutations. The mutations were introduced into the Click Beetle Red (CBR) luciferase (SEQ ID NO: 1) using the QuikChange Multi Site-Directed Mutagenesis Kit (Agilent). Variants were cloned, expressed and screened as described in Example 2. Table 2 and FIG. 5 list the variants and their fold change in luminescence over CBR luciferase with the substrate PBI-4813 or PBI-4739.

**Example 4**

**Variants Tested in Mammalian Cells**

Some of the variants identified in Examples 2 and 3 were screened for their performance in mammalian cells. DNA was prepared by Plasmid.com. HEK293 and HeLa cells were plated into wells of a 24-well plate at 0.05×106 cells/mL and grown overnight at 37° C. with CO2. 2.2 μg variant DNA was mixed with 80 μL OptiMEM and 6.6 μL FuGENE® HD transfection reagent (Promega Corporation) and incubated at room temperature for 5 min. 25 μL of the DNA mixture was added to each well.

After overnight incubation, the media was removed from the cells and 1 mL DPBS (Life Technologies) was added. The cells then underwent a freeze-thaw to generate cell lysates. 50 μL of the lysate was aliquoted into wells of a 96-deep well plate. 50 μL of assay reagent (20 μM of PBI-4813 or PBI-4739 in Bright-Glo™ Assay buffer containing 1 mM ATP) was added to the lysates, and luminescence was detected on an ImageQuant. Luminescence of the variants was normalized to the average of two CBR samples (FIG. 6 and FIG. 7).

**Example 5**

**Hydrogen Acceptor Mutagenesis Design**

The amino acid differences between CBR and CBG99 contribute to color change, such as amino acid changes that have a major contribution, Y224V, H247S, and Q348H, and amino acid changes that have a minor contribution, I346N and T349S. At position 351, the amino acid is glycine in CBR and CBG99 and arginine in CBG68.

Efficient light emission from some red-shifted luciferin derivatives may require the presence of an amino acid that can serve as a hydrogen acceptor (H-acceptor) at an appropriate position in the active site of a CBR variant. For example, PBI-4739 may require an H-acceptor in the vicinity of its hydroxyl group. The computational experiments described below were performed using Discovery Studio software (Accelrys). A homology model of CBR was generated based on an X-ray structure template of the related Luciola cruciata luciferase complexed with DLSA (Protein Data Bank accession 2D1S). The PBI-4739 ligand was manually docked into this model to match the position of the DLSA ligand in 2D1S, and amino acid side chains within 5 Å of PBI-4739 were energy minimized. Molecular dynamics simulation was performed on CBR amino acid side chains within 5 Å of PBI-4739. Representative output conformations were used to examine in silico which CBR mutations (amino acid replacements) might provide the desired H-acceptor.

A first group of mutations was identified based on restructuring a network of interacting side chains. The targeted positions were in the vicinity of the PBI-4739 hydroxyl and at least partially exposed to solvent. Based on an alignment of representative beetle luciferases (firefly, click beetle, glowworm), it was determined that the side chains at these positions are conserved (E308), mostly conserved (R334 and T226) or variable (G351). Various combinations of side chains at these positions allowed formation of a stabilizing H-bond network that shields the ligand from solvent and bridges multiple secondary structure elements. The mutation combinations below were intended to provide different H-bond networks while at the same time recruited one of the side chains as an H-acceptor for interaction with hydroxyl of PBI-4739:

G351R+R334 (E, Q, D, N, H, S, T, C, Y)

G351K+R334 (E, Q, D, N, H, S, T, C, Y)

G351K+T226N+R334 (E, Q, D, N, H, S, T, C, Y)

G351E+E308R+R334 (E, Q, D, N, H, S, T, C, Y)

E308R+R334 (E, Q, D, N, H, S, T, C, Y)

T226N+R334 (E, Q, D, N, H, S, T, C, Y)

A second group of mutations was suggested based on targeting individual positions that can accommodate a variety of H-acceptors. Preference was given to substitutions that would not be accessible by a random mutagenesis approach, i.e. not accessible by single base substitutions:

T226 (N, Q, E, D, H, C, Y)

C310 (Q, E, N, D, H)

Q348 (H, E)

S281 (N, Q, H, Y)

**Example 6**

**Hydrogen Acceptor Mutagenesis Variant Screen**

A. Mutagenesis. Hydrogen acceptor mutagenesis was performed using the QuikChange Multi Site-Directed Mutagenesis Kit (Agilent) as previously described using degenerate oligos specific to the amino acids listed below. The following variants were cloned and expressed as described in Example 2:

T226 (Q,N,E,D,H,C,Y)

C310(Q,E,N,D,H)

S281 (Q,N,H,Y)

Q348 ((H,E)

G351R+R334 (E,Q,D,N,H,S,T,C,Y)

G351K+R334 (E,Q,D,N,H,S,T,C,Y)

G351+226N+R334 (E,Q,D,N,H,S,T,C,Y)

G351E+E308R+R334 (E,Q,D,N,H,S,T,C,Y)

E308R+R334 (C,Y)

T226N+R334(E,D,S,T,C,Y)

B. Primary Screen. 1. One plate of clones was picked for each of the 10 sets of amino acid combinations described. These plates were then sequenced and assayed as previously described.

2. Cell lysates were prepared from cells containing the variants listed in Table 3 by diluting 100 μL of induced culture with 100 μL of lysis buffer (0.3× PLB, 0.006U RQ DNAse1) using a Tecan liquid handling robot. 50 μL of each cell lysate was assayed with either PBI-4813 or PBI-4739 (20 μM in Bright-Glo™ assay buffer containing 1 mM ATP), and luminescence detected on an ImageQuant CCD imager. Luminescence of the variants was normalized to CBR luciferase (FIG. 8).

C. Secondary screen. 1. Hits from the primary screen were selected and then processed in a secondary screen. Each sample was assayed in quadruplicate using the same assay method described above for the primary screen.

2. The variants were assayed as described above in for the primary screen. Luminescence of the variants was normalized to CBR luciferase (Table 3).

**Example 7**

**Combinations of H-Acceptor Mutagenesis and Library Screen Mutants**

Combinations of the Hydrogen acceptor mutants identified in Example 6 were designed and prepared using the QuikChange Multi Site-Directed Mutagenesis Kit (Agilent). Variants were cloned and expressed as described in Example 2 except the template was pF4Ag-CBR. Variants were screened as described in Example 6. Luminescence of the variants using luciferin (“D-LH2”), PBI-4813, and PBI-4739 was normalized to CBR luciferase (Table 4 and FIG. 9).

**Example 8**

**Evaluation of Clone 1230 and 1240**

Variant clones identified as ATG 1230 (CBR+389F+444R+251S) and ATG 1240 (CBR+334S+351R) were further evaluated for their luminescence, spectral property, Km and live cell kinetics with either PBI-4813 or PBI-4739. These variants were cloned and expressed as described in Example 2 except the template was pF4Ag-CBR, i.e., no HALOTAG®.

A. Luminescence with PBI-4813 and PBI-4739. Cell lysates of these variant clones were prepared and assayed for luminescence as described in Example 6. Luminescence was detected on a GloMax® Discover equipped with a red-sensitive photomultiplier tube (PMT) and compared to CBR luciferase (Table 5 and FIG. 10).

B. Spectral Measurements. Purified ATG 1230 and ATG 1240 enzyme were prepared. CBR was diluted 1:10 in 0.3× PLB+0.1% Prionex. The enzymes were assayed in triplicate by adding 50 μL of enzyme to 50 μL of assay reagent (20 μM PBI-4813 or PBI-4739 in Bright-Glo™ Assay buffer containing 1 mM ATP). Luminescence was detected on a Tecan-M1000 using spectral scan mode (FIG. 11). Using PBI-4813 as substrate, ATG-1240 had a spectral max at about 725 nm while CBR and ATG-1230 had a spectral max of about 650 nm. Using PBI-4739 as substrate, ATG-1240 had a spectral max at about 750 nm while CBR and ATG-1230 had a spectral max of about 760 nm.

C. Km Titration. Cell lysates of ATG 1230 and ATG 1240 expressed in HEK293 cells were prepared as described in Example 4. The cell lysates were then diluted 1:10 in 0.3× PLB+0.1% Prionex. Dilutions (2×) of each substrate (PBI-4813 or PBI-4739) were prepared in Bright-Glo™ Assay buffer containing 1 mM ATP. 50 μL of diluted cell lysate was assayed with 50 μL of each substrate assay solution. Luminescence was detected on the GloMax® Discover (FIG. 12). FIG. 12A and FIG. 12B show the relative Vmax (RLU) and Km (μM) values for ATG 1230 and ATG 1240 lysates using PBI-4813 and PBI-4739 as substrates. Because the enzyme concentration is not known for the lysates of for live cell titrations, Vmax as expressed as a velocity is not possible. Instead, the relative Vmax calculates the maximum brightness for each lysate with different substrates. The Km for PBI-4813 with CBR in the lysates was about 30× lower than CBR with D-luciferin. The Km for PBI-4813 with ATG1240 was about 5× lower than CBR with D-luciferin.

D. Live Cell Kinetics. HEK293 cells were plated into wells of a 96-well plate at 10,000 cells/well for the kinetic read and 5,000 cells/well for luminescence detection. Cells were transfected (n=24 for each sample) using 17 μg of each DNA (ATG 1230 or ATG 1240) in a total of 776 μL of OptiMEM. 50 μL of FuGENE® HD (Promega Corporation) was then added, and the samples were incubated for 10 min at room temperature. 5 μL of DNA complex was added to 12 wells/clone to each plate. Plates were then incubated overnight at 37° C. with CO2. Cells were assayed by first removing the growth media and replacing it with CO2 independent media+0.5% FBS containing diluted substrate (100 μM of PBI-4813, 1 mM of PBI-4739, or 3 mM of D-luciferin). For the kinetic reading, the plate was incubated at room temperature for 5 min, and the kinetics were measured for 150 min at 37° C. on a GloMax® Discover (FIG. 13). For luminescence detection, the plate was incubated in a CO2 incubator for 90 min, and luminescence was detected on a GloMax® Discover (FIG. 14).

HeLa cells were plated at 20,000 cells/well into wells of 96-well white assay plates and grown overnight. A transfection mixture was made containing 13 μg of CBR DNA and 40 μL FuGENE® HD in a total volume of 620 μL OptiMEM. The mixture was incubated at room temperature for 10 min. The cells were then transfected, in triplicate, with 5 μL of transfection complex and grown overnight.

The transfected cells were then assayed. 3× serial dilutions of D-luciferin starting at 3 mM in CO2 independent media+0.5% FBS, and 3× serial dilutions of PBI-4813 starting at 0.3 mM were prepared. The growth media from the transfected cells was removed and replaced with one of the 3× serial diluted substrate containing media. The cells were incubated for 60 min at 37° C. Luminescence was detected on a GloMax®-Multi+luminometer (FIG. 21A and FIG. 21B). For CBR in live cells, the Km for PBI-4813 was about 20 times lower than the Km for D-luciferin.

HEK293T cells were plated at 15,000 cells/well into wells of 96-well white assay plates and grown overnight. A transfection mixture was made containing 13 μg of CBR DNA and 40 μL FuGENE® HD in a total volume of 620 μL OptiMEM. The mixture was incubated at room temperature for 10 min. The cells were then transfected, in triplicate, with 5 μL of transfection complex and grown overnight.

The transfected cells were then assayed. 2× serial dilutions of D-luciferin starting at 2 mM in CO2 independent media+0.5% FBS, and 2× serial dilutions of PBI-4813 and PBI-4739 starting at 1 mM were prepared. The growth media from the transfected cells was removed and replaced with one of the 3× serial diluted substrate containing media. The cells were incubated 60 min at 37° C., and the cells were imaged on a ImageQuant CCD imager (FIG. 22). For CBR, the Km of PBI-4739 was about 5 times lower relative to D-luciferin while the Km of PBI-4813 was about 23 times lower relative to D-luciferin.

HEK293T cells were plated at 15,000 cells/well into wells of 96-well white assay plates and grown overnight. A transfection mixture was made containing 6.6 μg of CBR or ATG-1240 DNA and 20 μL FuGENE® HD in a total volume of 310 μL OptiMEM. The mixture was incubated at room temperature for 10 min. The cells were then transfected, in triplicate, with 5 μL of transfection complex and grown overnight.

The transfected cells were then assayed. 2× serial dilutions of PBI-4813 and PBI-4739 starting at 2 mM in CO2 independent media+0.5% FBS were prepared. The growth media from the transfected cells was removed and replaced with one of the serial diluted substrate containing media. The cells were incubated 60 min at 37° C., and luminescence detected on a modified GloMax® Discover luminometer containing a red shifted PMT (see FIG. 23A and FIG. 23B). The Km of ATG 1240 was about 5 times higher than the Km of CBR with PBI-4813. The Km of ATG 1240 was about 3 times higher than the Km of CBR with PBI-4739.

**Example 9**

**Secondary Library Screen using ATG 1240 as a Template**

A random clone library was prepared as described in Example 2 using ATG 1240 as a template. The variants were then cloned into pF4Ag (no HT7). The library was screened as follows (50 plates total):

i. Primary Screen. Cells were grown as described in Example 2. Cell lysates were prepared by diluting 100 μL of induced culture with 100 μL of lysis buffer (0.3× PLB+0.006U RQDNAse1) using a Tecan liquid handling robot. 50 μL of each cell lysate was then assayed with either PBI-4813 or PBI-4739. (20 μM substrate in Bright-Glo™ assay buffer containing 1 mM ATP). Luminescence was detected on a GloMax® Discover.

ii. Secondary Screen. Cells from each hit in the primary screen were streaked onto new plates and grown overnight. Colonies were picked in quadruplicate and cultures were grown as described in Example 2. 50 μL of each cell lysate was assayed with 50 μL of BrightGlo™ assay buffer containing 1 mM ATP and 30 μM PBI-4813 or 50 μM PBI-4739. Luminescence was detected on a GloMax® Discover Multimode Detection System (Table 6). The variants in this library were identified as having on average between 2-3 mutations per gene.

**Example 10**

**Insertion Mutagenesis: Engineering the Click Beetle Red-Luciferin Binding Pocket**

The goal was to significantly increase the light emission from Click Beetle Red (CBR) mutant(s) with the red-shifted substrates, e.g., PBI-4739 (more red and less soluble), PBI-4813 (brighter, possible substrate inhibition). The previously described random mutagenesis and site-directed mutagenesis yielded about 3-fold improvement in light emission.

Approach 1 was to perform cassette mutagenesis of contiguous residue stretches in the vicinity of luciferin ligand binding. Approach 2 was to attempt to expand the luciferin ligand binding pocket by inserting single Ala residues at specific sites.

Suitable insertion sites were identified by visual inspection of the previously generated 3D structure model of CBR with PBI-4739, as described in Example 5. Additional information on sites that tolerate insertions/deletions was obtained from an alignment of diverse beetle luciferase protein sequences (e.g., firefly, click beetle, and glowworm) retrieved from the public GenBank and UniProt databases. Further information was obtained from superimposing 3D structures of luciferases (e.g. firefly luciferase with ligand, PDB accession 2D1S) and related non-luciferases (e.g. CoA-ligase with ligand, PDB accession 3N12) and examining how sequence changes such as insertions/deletions allow secondary structure elements move to accommodate different ligands in the binding pocket.

Based on combined analyses of the 3D structures and protein sequence alignment, several sites in CBR were identified as targets for insertion mutagenesis. Single Ala residues were inserted after these positions (bolded and underlined in FIG. 18) in CBR.

Variant constructs generated by the insertion mutagenesis were prepared by Gene Dynamics. Cells from each variant clone were grown overnight onto LB-ampicillin plates, and colonies picked in quadruplicate and grown in cultures as described in Example 6. Cell lysates were prepared as previously described, and 50 μL of cell lysate was assayed with 50 μL of Bright-Glo™ assay buffer containing 1 mM ATP and 30 μM PBI-4813 or PBI-4739. Luminescence was detected on a GloMax® Discover and normalized to CBR (Table 7).

**Example 11**

**Recombination of ATG 1240 Library Mutants**

Combinations of the ATG 1240 library mutants identified in Example 9 were designed and prepared using DNA shuffling (Stemmer (1994) PNAS USA 91:10747-10751). Briefly, a library of mutant genes was created via DNA shuffling. The mutant genes were then cloned into the pF4Ag vector and screened as described in Example 6. Luminescence was detected on a GloMax® Discover Multimode Detection System and normalized to ATG-1240 luciferase (Table 8 and FIG. 15).

**Example 12**

**Live Cell Substrate Titration**

HEK293 cells were plated into wells of a 96-well plate at 5,000 cells/well and grown overnight. Cells were then transfected (n=24 for each sample) using 17 μg of ATG 1240 DNA in a total of 776 μL of OptiMEM. 50 μL of FuGENE® HD (Promega Corporation) was then added, and the samples incubated for 10 min at room temperature. 5 μL of DNA complex was added to 18 wells/sample to each plate. Plates were then incubated overnight at 37° C. with CO2.

Cells were assayed by first removing the growth media and replacing it with CO2 independent media+0.5% FBS containing 2× diluted substrate (PBI-4813 or PBI-4739). 100 μL of each diluted substrate was added in triplicate to the transfected cells and incubated for 60 min. Luminescence was detected on a GloMax® Discover (FIG. 16).

**Example 13**

**Mimicry of Tissue Attenuation using Long Pass Filters**

This example demonstrates how much light is produced in cells transfected with Luc2 (Photinus pyralis luciferase) and assayed with D-luciferin compared to ATG 1240 assayed with D-luciferin, PBI-4813 or PBI-4739 using various long pass cut-off filters.

A transfection complex was prepared by diluting ATG 1240 DNA or Luc2 DNA (pF4Ag-Luc2) to 100 ng/μL, and then 2× serial dilutions prepared in 50 ng/μL pGEM carrier DNA. 40 μL of the diluted DNA was the mixed with 160 μL phenol red free OptiMEM and 16 μL of FuGENE® HD and incubated for 20 min. 5 μL of each transfection complex was added to wells of a 96-well black assay plates and then 100 μL of diluted HEK293T cells (200,000/mL in DMEM+10% FBS) was added. The cells were grown overnight at 37° C. with CO2.

To assay the cells, 4 mM D-luciferin, 2 mM PBI-4813 and 2 mM PBI-4739 substrate solutions were prepared in DMEM+10% FBS. 100 μL of the D-luciferin solution was added to cells expressing Luc2 and ATG 1240, and 100 μL of PBI-4813 or PBI-4739 solution was added to ATG 1240 expressing cells. Samples were incubated for10 min at 37° C., and luminescence detected on GloMax® Discover (heated to 37° C.) with no filter, 610 nm Long Pass (LP) filter, 665 nm LP filter, 695 nm LP filter, 720 nm LP filter or 760 nm LP filter.

These results in FIG. 17 suggest that ATG 1240 assayed with D-luciferin may be brighter than with PBI-4739 even when light is attenuated with long pass filters. PBI-4813 and D-luciferin are comparable at the greatest light attenuation. Note that the Km for PBI-4813 was more than 10 times less than for D-luciferin in live cell. This could be an advantage for PBI-4813 in live animals.

**Example 14**

**Codon Optimization**

600,000 cells (HEK293, 3T3, or CHO cells) in 3 mL were plated in 6 well plates. Briefly, for each cell type grown in t-75 flasks, media was removed from the t-75 flask and the cells were washed with DPBS. 3 mL of Trypsin was added to the cells and the cells were incubated for 3 min. 10 mL of growth media (DMEM+10% FBS for HEK293T and 3T3, Ham F12+10% FBS for CHO) were added. Cells were centrifuged at 500 rpm for 5 min. The cells were counted and diluted to 200,000 per mL. 3 mL of cells were added to each well of a six well plate (i.e., 600,000 cells).

Protocol for transfection of HEK293 cells grown in 3,000 μL of medium in 6-well plates using a FuGENE® HD:DNA ratio of 3.0:1. This protocol will prepare sufficient DNA/FuGENE® HD reagent to transfect 3 wells at 3000 μL/well.

Cell plating. HEK293 cells were plated the day before transfection at a density of 5×105 cells per well of a 6-well plate in 3 ml of complete growth medium (DMEM+10% Fetal Bovine Serum).

Complex preparation. 465 μL of 0.020 μg/μL plasmid solution were prepared in OptiMEM, OptiPro, or sterile deionized water. Briefly, 9.9 μg of test DNA was added in 465 μl total volume. 10 ng of Nanoluc DNA was added to each reaction (“ATG42”) for normalization. 30 μl of FuGENE® HD reagent was added and mixed carefully by pipetting (15 times) or by vortexing briefly. The mixture incubated for 5 to 10 min at room temperature. 150 μl of complex was added per well to the cells, and mixed thoroughly. The Reagent:DNA ratio may range from 2.5:1.0 to 3.5:1.0. The recommended Reagent:DNA ratio for HEK293 cells was 3.0:1 at 3.0 μg DNA per well.

Re-plate cells: Growth media was removed and the cells were washed with 1 ml of DPBS. 3 ml of growth media was added and the cells were centrifuged at 500 rpm. For each sample, the pellets were re-suspended in 1 ml of media and the cells were counted. The HEK cells had very little pellet and the cell counts were below 100,000/ml. See Table 9 for dilution of 3T3 and CHO cells. 100 μL of each sample (n=16) were plated in a 96 well plate. One plate was used for each cell type.

Assay: Bright-Glo assay buffer was reconstituted with Bright-Glo assay substrate. 500 μL of furimazine was added to 25 ml of NanoGlo Assay buffer. 100 μL of reconstituted Bright-Glo was added to each sample (n=8) and 100 μL of NanoGlo was added to the other samples (n=8) and read on GloMax®-Multi+.

Legend/background information for codon optimization experiments:

Variant 343=CBR sequence (“ATG 343”)

Variant 1240=CBR sequence+R334S/G351R (“ATG 1240”)

Variant 1929=CBR sequence+R334S/G351R (codon optimized for RNA structure and codon usage in rodents) (“ATG 1929”)

Variant 1944=CBR sequence+R334S/G351R (codon optimized for RNA structure and codon usage in CHO cells) (“ATG 1944”)

Variant 1945=CBR sequence+R334S/G351R (codon optimized for RNA structure and codon usage in mouse lung and liver cells) (“ATG 1945”)

While the present invention is described in connection with what is presently considered to be the most practical and preferred embodiments, it should be appreciated that the invention is not limited to the disclosed embodiments, and is intended to cover various modifications and equivalent arrangements included within the spirit and scope of the claims. Modifications and variations in the present invention may be made without departing from the novel aspects of the invention as defined in the claims. The appended claims should be construed broadly and in a manner consistent with the spirit and the scope of the invention herein.

For reasons of completeness, various aspects of the invention are set out in the following numbered clauses:

Clause 1. An isolated polynucleotide encoding a click beetle red luciferase (CBR) variant polypeptide having at least 80% amino acid sequence identity to SEQ ID NO: 1 and comprising at least one amino acid substitution at a position corresponding to position 4, 16, 34, 47, 51, 52, 55, 72, 73, 74, 79, 82, 83, 87, 89, 104, 109, 113, 117, 119, 124, 130, 131, 133, 136, 144, 146, 156, 159, 170, 179, 186, 200, 211, 218, 224, 225, 226, 228, 229, 234, 247, 251, 252, 253, 255, 280, 281, 285, 308, 309, 310, 319, 329, 334, 335, 337, 346, 348, 349, 350, 352, 354, 355, 358, 363, 370, 377, 390, 393, 394, 400, 401, 409, 412, 420, 422, 431, 437, 439, 444, 445, 453, 455, 467, 471, 473, 479, 484, 489, 496, 501, 503, 508, 516, 528, 531, 535, 537, 539, or combination thereof, of SEQ ID NO: 1, wherein the variant CBR polypeptide has at least one of enhanced luminescence, altered light emission wavelength, altered substrate specificity, or a combination thereof, as compared to a CBR polypeptide of SEQ ID NO: 1.

Clause 2. The isolated polynucleotide of clause 1, wherein the CBR variant polypeptide further comprises at least one amino acid substitution at a position corresponding to position 351, 389, 457, or combination thereof, of SEQ ID NO: 1.

Clause 3. The isolated polynucleotide of clause 1 or 2, wherein the CBR variant polypeptide comprises a substitution corresponding to at least one of R4H, H16Q, H34Y, D47E, S51N, Y52C, F55LN, K72E, I79V, M73K/T, N74S, E82G, N83H, F87S, I89V, V104D, I109N/V, L113Q, M117T, I119F/T, I124V, N130K, I131N/T, N133D, K136N, F144L, K146E, N156D, N156K, G159D, Y170C, K179S, V186A, G200G, N211N, H218L/Y, G225S, T226C/G/H/N/Q/Y, L228P, I229V, V234A, G251S, G251I, Y252C, V255D/F, E253K, R280S, S281N/Q, V285A, I309T, E319G, N329D, R334E/Q/H/S/N/K, C335S, K337E, I346N, Q348H/E, L350P, G351K/R, D352N, R355G, S358P, T363A/S, I370T, I389F/G/S/V, I390I, M393K/L, V394M, N400D, N401S, I409T, D412G, F420F, Y422C, V431A, E437G, I439V, S444C/R/T, Q445H, E453K, V455D, K457N, D471V, E473A, S479T, K484E/M/R, E489V, Y496H, E501G, V503M, Y508C, V516A, T528A, E531G, Q535H, L537W, K539R, or combinations thereof, or SEQ ID NO: 1.

Clause 4. The isolated polynucleotide of any one of clauses 1-3, wherein the CBR variant polypeptide comprises an amino acid substitution at a position corresponding to position 389, 444, and 251 of SEQ ID NO: 1.

Clause 5. The isolated polynucleotide of any one of clauses 1-4, wherein the amino acid substitutions comprise I389F, S444R, and G251S.

Clause 6. The isolated polynucleotide of any one of clauses 1-5, wherein the CBR variant polypeptide comprises an amino acid substitution at positions corresponding to positions 334 and 351 of SEQ ID NO: 1.

Clause 7. The isolated polynucleotide of any one of clauses 1-6, wherein the amino acid substitutions comprise R334S and G351R.

Clause 8. The isolated polynucleotide of any one of clauses 1-7, wherein the CBR variant polypeptide further comprises an amino acid substitution at a position corresponding to positions 51 and 444 of SEQ ID NO:1.

Clause 9. The isolated polynucleotide of any one of clauses 1-8, wherein the amino acid substitutions comprise S51N and S444R.

Clause 10. The isolated polynucleotide of clause 1, wherein the CBR variant polypeptide comprises an amino acid polypeptide of SEQ ID NO: 2, SEQ ID NO: 3, or SEQ ID NO: 4.

Clause 11. The isolated polynucleotide of any one of the preceding clauses, wherein the CBR variant polypeptide has enhanced luminescence compared to a CBR polypeptide of SEQ ID NO: 1.

Clause 12. The isolated polynucleotide of clause 11, wherein the CBR variant polypeptide has enhanced luminescence when a luciferin is utilized by the CBR variant polypeptide to generate luminescence.

Clause 13. The isolated polynucleotide of clause 11, wherein the CBR variant polypeptide has enhanced luminescence when a luciferin derivative is utilized by the CBR variant polypeptide to generate luminescence.

Clause 14. The isolated polynucleotide of clause 13, wherein the luciferin derivative comprises:

Clause 15. The isolated polynucleotide of clause 14, wherein the CBR variant polypeptide has at least 2 fold increase in luminescence compared to a CBR polypeptide of SEQ ID NO: 1.

Clause 16. The isolated polynucleotide of clause 14, wherein the CBR variant polypeptide has at least 4 fold increase in luminescence compared to a CBR polypeptide of SEQ ID NO: 1.

Clause 17. The isolated polynucleotide of any one of the preceding clauses, wherein the CBR variant polypeptide has altered light emission spectra compared to a CBR polypeptide of SEQ ID NO: 1.

Clause 18. The isolated polynucleotide of clause 17, wherein the CBR variant polypeptide is able to emit light at a longer wavelength when a luciferin is utilized by the CBR variant polypeptide to generate luminescence.

Clause 19. The isolated polynucleotide of clause 17, wherein the CBR variant polypeptide is able to emit light at a longer wavelength when a luciferin derivative is utilized by the CBR variant polypeptide to generate luminescence.

Clause 20. The isolated polynucleotide of clause 19, wherein the luciferin derivative comprises:

Clause 21. The isolated polynucleotide of clause 20, wherein if the luciferin derivative comprises

the CBR variant polypeptide emits light having a shift in spectral maximum of at least about 1 nm to at least about 100 nm relative to the light produced by the CBR polypeptide of SEQ ID NO: 1.

Clause 22. The isolated polynucleotide of clause 21, wherein the CBR variant polypeptide emits light having a spectral maximum between about 650 nm to about 800 nm.

Clause 23. The isolated polynucleotide of clause 21, wherein the CBR variant polypeptide emits light having a spectral maximum between about 725 nm to about 775 nm.

Clause 24. The isolated polynucleotide of clause 21, wherein the CBR variant polypeptide emits light having a spectral maximum of about 750 nm.

Clause 25. The isolated polynucleotide of clause 20, wherein if the luciferin derivative comprises

the CBR variant polypeptide emits light having a shift in spectral maximum of at least about 1 nm to at least about 100 nm relative to the light produced by the CBR polypeptide of SEQ ID NO: 1.

Clause 26. The isolated polynucleotide of clause 25, wherein the CBR variant polypeptide emits light having a shift in spectral maximum of at least about 75 nm relative to the light produced by the CBR polypeptide of SEQ ID NO: 1.

Clause 27. The isolated polynucleotide of clause 25, wherein the CBR variant polypeptide emits light having a spectral maximum between about 650 nm to about 800 nm.

Clause 28. The isolated polynucleotide of clause 25, wherein the CBR variant polypeptide emits light having a spectral maximum between about 700 nm to about 775 nm.

Clause 29. The isolated polynucleotide of clause 25, wherein the CBR variant polypeptide emits light having a spectral maximum of about 725 nm.

Clause 30. The isolated polynucleotide any one of the preceding clauses, wherein the CBR variant polypeptide has altered substrate specificity compared to a CBR polypeptide of SEQ ID NO: 1.

Clause 31. The isolated polynucleotide of clause 30, wherein the CBR variant polypeptide has a change in relative specificity relative to the CBR variant polypeptide in the presence of a luciferin compared to a luciferin derivative.

Clause 32. The isolated polynucleotide of clause 30, wherein the CBR variant polypeptide has a change in relative specificity relative to the CBR variant polypeptide in the presence of a luciferin derivative compared to a different luciferin derivative.

Clause 33. The isolated polynucleotide of clause 31 or 32, wherein the luciferin derivative comprises:

Clause 34. The isolated polynucleotide of any one of the preceding clauses, wherein the variant CBR polypeptide has luciferase activity.

Clause 35. The isolated polynucleotide of any one of the preceding clauses, wherein the variant CBR polypeptide has a Km for PBI-4813 of at least about 0.01 μM to at least about 5.00 μM.

Clause 36. The isolated polynucleotide of clause 35, wherein the variant CBR polypeptide has a Km for PBI-4813 of at least about 0.50 μM to at least about 3.00 μM.

Clause 37. The isolated polynucleotide of clause 35, wherein the variant CBR polypeptide has a Km for PBI-4813 of at least about 0.82 μM or 2.41 μM.

Clause 38. The isolated polynucleotide of any one of the preceding clauses, wherein the variant CBR polypeptide has a Km for PBI-4739 of at least about 0.01 μM to at least about 5.00 μM.

Clause 39. The isolated polynucleotide of clause 38, wherein the variant CBR polypeptide has a Km for PBI-4739 of at least about 1.50 μM to at least about 4.50 μM.

Clause 40. The isolated polynucleotide of clause 38, wherein the variant CBR polypeptide has a Km for PBI-4739 of at least about 2.33 μM or 3.95 μM.

Clause 41. The isolated polynucleotide of any one of the preceding clauses, wherein the variant CBR polypeptide has a relative Vmax that is at least 2 fold higher than the relative Vmax of a CBR polypeptide of SEQ ID NO: 1 using PBI-4813 as a substrate.

Clause 42. The isolated polynucleotide of any one of the preceding clauses, wherein the variant CBR polypeptide has a relative Vmax that is at least 2 fold higher than the relative Vmax of a CBR polypeptide of SEQ ID NO: 1 using PBI-4739 as a substrate.

Clause 43. The isolated polynucleotide of any one of the preceding clauses, wherein the sequence has been codon-optimized.

Clause 44. The isolated polynucleotide of any one of the preceding clauses, wherein the sequence comprises a polynucleotide of SEQ ID NOs: 6-9.

Clause 45. The isolated polynucleotide of any one of the preceding clauses, wherein the polynucleotide further encodes a polypeptide of interest linked to the CBR variant polypeptide, the polypeptide of interest and the CBR variant polypeptide capable of being expressed as a fusion protein.

Clause 46. The isolated polynucleotide of clause 45, wherein the polypeptide of interest comprises HALOTAG®.

Clause 47. A vector comprising the polynucleotide, or a fragment thereof, of any one of the preceding clauses.

Clause 48. The vector of clause 47, wherein the polynucleotide is operably linked to a promoter.

Clause 49. A cell comprising the polynucleotide of any one of clauses 1-46 or the vector of clause 47 or 48.

Clause 50. A non-human transgenic animal comprising the cell of clause 49.

Clause 51. A non-human transgenic animal comprising the polynucleotide of any one of clauses 1-46 or the vector of clause 47 or 48.

Clause 52. A CBR variant polypeptide encoded by the polynucleotide of any one clauses 1-46.

Clause 53. A circularly permuted luciferase comprising the polypeptide encoded by the polynucleotide of any of clauses 1-46 or a fragment thereof.

Clause 54. A fusion protein comprising a CBR variant polypeptide encoded by the polynucleotide of any one of clauses 1-46.

Clause 55. A near-infrared bioluminescence system comprising the polynucleotide of any one of clauses 1-46 and a luciferin derivative.

Clause 56. The near-infrared bioluminescence system of clause 55, wherein the luciferin derivative comprises:

Clause 57. A method of producing a CBR variant polypeptide comprising growing the cell of clause 47 under conditions that permit expression of the CBR variant polypeptide.

Clause 58. A method of producing a CBR variant polypeptide comprising introducing the vector of clause 47 or 48 into a cell under conditions which permit expression of the CBR variant polypeptide.

Clause 59. A kit comprising the polynucleotide of any one of clauses 1-46 or the vector of clause 47 or 48.

Clause 60. A kit comprising the CBR variant polypeptide of clause 46.

Clause 61. The kit of clause 59 or 60 further comprising at least one of:

(a)

and

(b) a buffer reagent.

Clause 62. A bioluminescence resonance energy transfer (BRET) system comprising: a first fusion protein including a first target protein and a bioluminescence donor molecule, wherein the bioluminescence donor molecule is a CBR variant encoded by the polynucleotide of any one of clauses 1-46; a second fusion protein including a second target protein and a fluorescent acceptor molecule; and a CBR substrate.

Clause 63. The BRET system of clause 62, wherein the CBR substrate is a luciferin or luciferin derivative.

Clause 64. The BRET system of clause 63, wherein the luciferin derivative comprises:

Clause 65. A method for measuring bioluminescence using at least one of the polynucleotide of any one of clauses 1-46; the vector of any one of clauses 47 or 48; the cell of clause 49; the animal of any of clauses 50 or 51; the CBR variant polypeptide of clause 52; the circularly permuted luciferase of clause 53; the fusion protein of clause 54, or the near-infrared bioluminescence system of clause 55 or 56.

Clause 66. The method of clause 65, wherein the bioluminescence is measured in a live, intact non-human animal.

Clause 67. A method of measuring the enzymatic activity of a luminogenic protein, the method comprising: contacting a luminogenic protein, a deprotecting enzyme, and a protected luminophore; and detecting light produced from the composition, wherein the luminogenic protein is a CBR variant encoded by the polynucleotide of any one of clauses 1-46 and the luminophore is a luciferin derivative comprising:

Clause 68. The method of clause 67, wherein the enzymatic activity is measured in a live, intact non-human animal.

Clause 69. A method for measuring the activity of a non-luminescent enzyme of interest, the method comprising: (a) providing a luminogenic molecule wherein the molecule is a substrate for the non-luminescent enzyme of interest and a pro-substrate of a CBR variant encoded by the polynucleotide of any one of clauses 1-46; (b) contacting the luminogenic molecule with at least one non-luminescent enzyme of interest and at least one CBR variant to produce a reaction mixture; and (c) determining activity of the non-luminescent enzyme of interest by measuring luminescence of the reaction mixture.

Clause 70. The method of clause 69, wherein the luminogenic molecule is a modification of

Clause 71. The method of any one of clauses 69 and 70 wherein the non-luminescent enzyme of interest is a protease enzyme, a cytochrome P450 enzyme, a monoamine oxidase enzyme, or a glutathione S-transferase enzyme.

Clause 72. The method of any one of clauses 69 to 71 wherein the activity of the non-luminescent enzyme is measured in a live, intact animal.

Clause 73. A method to detect the presence of at least two molecules in a sample or a cell, the method comprising: contacting the sample or cell with a first reporter molecule comprising a CBR variant encoded by the polynucleotide of any one of clauses 1-46, wherein the first reporter molecule is operatively linked to a first component of the sample or cell; contacting the sample with a second reporter molecule, wherein the second reporter molecule is operatively linked to a second component of the sample or cell; and detecting the presence of the first and second reporter molecules to determine the presence and/or amounts of the first and second components in the sample or cell.

Clause 74. A method to detect an interaction between a first protein and a second protein in a sample, the method comprising:

(a) contacting a sample with:

wherein the sample comprises:


- - (i) a first polynucleotide encoding a first fusion protein, wherein
    the first fusion protein comprises a first fragment of a luminescent
    enzyme and a first protein; and
  - (ii) a second polynucleotide encoding a second fusion protein,
    wherein the second fusion protein comprises a second fragment of the
    luminescent enzyme and a second protein; and

(b) detecting luminescence in the sample,

wherein the detection of luminescence indicates an interaction between the first protein and the second protein, wherein the luminescent enzyme is encoded by the isolated polynucleotide of any one of clauses 1-46.

Clause 75. The method of clause 74, wherein when the first protein and second protein interact, the first fragment of the luminescent enzyme and the second fragment of the luminescent enzyme reconstitute a full-length enzyme capable of stably binding the cell-permeable substrate.

Clause 76. A method to detect an interaction between a first protein and a second protein in a sample, the method comprising:

(a) contacting a sample with:

wherein the sample comprises:


- - (i) a first polynucleotide encoding a first fusion protein, wherein
    the first fusion protein comprises a luminescent enzyme and a first
    protein, wherein the luminescent enzyme is encoded by the isolated
    polynucleotide of any one of clauses 1-46; and
  - (ii) a second polynucleotide encoding a second fusion protein,
    wherein the second fusion protein comprises a fluorescent acceptor
    molecule and a second protein; and

(b) detecting bioluminescence resonance energy transfer (BRET) in the sample, indicating an interaction or close proximity of the bioluminescent donor and the fluorescence acceptor.

