# DESCRIPTION

## TECHNICAL FIELD

The present invention relates to the biotechnical field, and especially relates to a method for introducing an exogenous DNA by overcoming the restriction modification barrier of a target bacterium.

## BACKGROUND ART

The bacterial restriction modification system consists of restriction endonucleases (restriction enzymes) and DNA methyltransferases, and the former are able to specifically recognize and cleave DNA while the latter are able to add a methyl modification to a base of DNA to prevent the cleavage of DNA by restriction enzymes. The Restriction modification system is able to selectively degrade exogenous DNA invading into bacteria to enable self-protection of bacteria. Restriction modification systems are divided into four major types, according to their subunit constitutions, cleavage sites, sequence specificities, and co-factor characteristics. Subunits of restriction enzymes of the restriction modification systems type I, type II, and type III are able to recognize and cleave non-methylated DNA. However, if DNA is first recognized and modified by the subunit of the methyltransferase, cleavage cannot be achieved by the restriction enzyme. The restriction modification system type IV consists of only the restriction enzymes and does not contain the methyltransferases. It recognizes and cleaves DNA having an exogenous methylation pattern, and thus is a methylation-dependent restriction enzyme. Additionally, a recent study indicates that phosphorothioation-modification-enzymes for DNA backbones and corresponding restriction enzymes thereof are a new type of restriction modification systems (Nucleic Acids Research, 38, 7133-7141). Such a complex modification-cleavage mode considerably protects the safety of bacteria's own DNA and is used as a main means of bacteria for effectively preventing invasion of exogenous DNA released by bacteriophages and dead bacteria in the environment. Meanwhile, this also becomes the main barrier for introducing an exogenous DNA into bacteria and enabling genetic manipulation by using molecular biological methods. The genetic manipulation of bacteria having multiple restriction modification systems is especially difficult.

To date, investigators have invented two types of techniques for overcoming the restriction modification barriers. The first strategy is modifying exogenous DNAs, including in vitro modification and in vivo modification of E. coli. For example, in vitro modification of exogenous DNAs using crude protein extract (containing DNA methyltransferase) of the target bacterium enables transformation of Helicobacter pylori, Bacillus cereus, and Bacillus weihenstephanensis (Molecular Microbiology, 37, 1066-1074, Applied and Environmental Microbiology, 74, 7817-7820); or cloning and expression of DNA methyltransferase of the target bacterium in E. coli and in vivo modification of exogenous plasmid DNAs, for example, cloning and expression of two DNA methyltransferases of Bifidobacterium adolescentis in E. coli TOP10 and modification of shuttle plasmids, enables genetic transformation of Bifidobacterium adolescentis (Nucleic Acids Research, 37, e3). The second type, i.e., a method for inactivating restriction modification systems, includes inactivation by physical means and gene knockout. By transitorily inactivating restriction enzymes of the target bacterium using heating after transformation, the transformation efficiency of exogenous plasmids for Corynebacterium glutamicum is increased to 108 CFU/μg DNA (Microbial Biotechnology, 52, 541-545); after gene CAC1502 is knocked out, Clostridium acetobutylicum can be allowed to accept unmethylated plasmid DNA (PLoS ONE, 5, e9038); and however, there is also a report indicating that knockout of Saul restriction endonuclease is not sufficient to allow Staphylococcus aureus to accept exogenous DNAs (Applied and Environmental Microbiology, 75, 3034-3038).

Although the techniques described above may increase transformation efficiency of exogenous DNAs for target bacterium to some extent, there are still the following problems and deficiencies: Although some solutions are able to perform in vitro modification on exogenous DNA molecules using DNA methyltransferases of the target bacterium, in vitro modification efficiency using the crude protein extract is low, and a part of plasmids would be also degraded at the same time of modification; the solution of in vivo modification has not eliminated methylation of exogenous DNA molecules by E. coli's own DNA methyltransferases, and such DNAs having an E. coli methylation pattern are prone to activate the restriction system type III of the target bacterium. Restriction enzymes of a number of bacteria are not heat-sensitive and cannot be transitorily inactivated by heating; if a bacterium contains multiple restriction modification systems, a lot of time and effort are needed to knockout restriction enzyme genes one by one, and meanwhile, knockout of restriction enzymes also causes the target bacterium to be infected by bacteriophages, which is extremely adverse to the construction of strains for industrial microbial fermentation. The techniques have poor generality and may be only applicable to one or a few types of bacteria.

## SUMMARY OF THE INVENTION

An object of the present invention is to provide a method for introducing an exogenous DNA molecule into a target bacterium.

The method provided in the present invention comprises the steps of

1) co-expressing all DNA-methyltransferase-encoding genes in the genome of a target bacterium in E. coli in which the restriction modification system thereof has been deleted to obtain a recombinant bacterium A;

2) introducing an exogenous plasmid DNA molecule into said recombinant bacterium A for in vivo modification so as to obtain a methylation-modified exogenous DNA plasmid molecule;

3) introducing said methylation-modified exogenous plasmid DNA molecule into said target bacterium.

In the method described above, in step 1), said co-expression of all DNA-methyltransferase-encoding genes in the genome of the target bacterium in E. coli in which the restriction modification system thereof has been deleted, is introduction, via a recombinant vector, of all DNA-methyltransferase-encoding genes in the genome of the target bacterium into said E. coli in which the restriction modification system thereof has been deleted; and

step 2) comprises the steps of

A) introducing said exogenous plasmid DNA molecule into said recombinant bacterium A to obtain a recombinant bacterium B;

B) inducing and culturing said recombinant bacterium B to obtain an induced recombinant bacterium B;

C) extracting the plasmid DNA of said induced recombinant bacterium B to obtain a methylation-modified exogenous plasmid DNA molecule.

In the method described above, in step 1) said recombinant vector is one which coexpresses all DNA methyltransferases obtained by inserting all DNA-methyltransferase-encoding genes into an expression plasmid; and

each DNA-methyltransferase-encoding gene in the recombinant vector described above can use respective promoters or share one promoter (constituting an operon structure).

In B) of step 2), said inducing and culturing comprises a temperature induction or induction using an inducing agent such as arabinose, IPTG, xylose, or rhamnose.

In the method described above, in 13) of step 2), said inducing and culturing comprises culturing the recombinant bacterium B under inducing conditions;

the optimal inducing conditions are that said recombinant bacterium B is inducing and culturing in a liquid culture medium containing arabinose at a final concentration of 0.2% by mass;

the temperature for said inducing and culturing is 25° C.-37° C. and the time for said inducing and culturing is 3-24 hours; and

the temperature for said inducing and culturing is preferably 30° C. and the time for said inducing and culturing is preferably 12 hours.

In the method described above, said target bacterium is an Eubacterium or Archaebacterium containing a restriction modification systems, and said Eubacterium or Archaebacterium containing restriction modification systems can be, but is not limited to, Bacillus amyloliquefaciens TA208, Bacillus cereus ATCC 10987, or Nitrobacter hamburgensis X14;

said E. coli in which the restriction modification system thereof has been deleted can be, but is not limited to, Escherichia coli EC135 CGMCC NO. 5925; the genotype of the restriction modification system thereof is mcrA Δ (mrr-hsdRMS-mcrBC) Δdcm::FRT Δdam::FRT.

In the method described above, said exogenous DNA molecule can be, but is not limited to, pAD123, pAD43-25, pMK3, pMK4, pHCMC02, pHCMC04, pDG148StuI, pWYE748, or pBBR1-MCS5-PNham3450-GFP.

In the method described above, all DNA-methyltransferase-encoding genes in said Bacillus amyloliquefaciens TA208 are BAMTA20806525, BAMTA208—6715, BAMTA208—19835 and BAMTA208—16660; and nucleotide sequences of said BAMTA208—06525, BAMTA208—6715, BAMTA208—19835 and BAMTA208—16660 are, in this order, SEQ ID NO: 2, SEQ ID NO: 3, SEQ ID NO: 4, and SEQ ID NO: 5 in the sequence listing;

all DNA-methyltransferase-encoding genes in said Bacillus cereus ATCC 10987 are BCE—0393, BCE—4605, BCE—5606, BCE—5607, BCE—0365, and BCE—0392; and nucleotide sequences of said BCE—0393, BCE—4605, BCE—5606, BCE—5607, BCE—0365 and BCE—0392 are, in this order, SEQ ID NO: 6, SEQ ID NO: 7, SEQ ID NO: 8, SEQ ID NO: 9, SEQ ID NO: 10, and SEQ ID NO: 11 in the sequence listing; and

all DNA-methyltransferase-encoding genes in said Nitrobacter hamburgensis X14 are Nham—0569, Nham—0582, Nham—0803, and Nham—3225; and nucleotide sequences of said Nham—0569, Nham—0582, Nham—0803, and Nham—3225 are, in this order, SEQ ID NO: 12, SEQ ID NO: 13, SEQ ID NO: 14, and SEQ ID NO: 15 in the sequence listing.

Before the steps described above, a step of determining all DNA-methyltransferase-encoding genes in the genome of the target bacterium may be included:

(a) predicting genes encoding DNA methyltransferases in target bacterium by homologous sequence alignment and the like;

(b) introducing each gene encoding DNA methyltransferases, as predicted, to an inducible expression vector respectively, transferring it into E. coli with no restriction modification system, and then inducing and culturing it respectively;

(C) preparing genome DNAs (including chromosomal DNAs and the expression vector) of the aforementioned E. coli into which the expression vector is transferred, and detecting whether DNAs have been methylation-modified, wherein if it is determined that DNAs have been methylation-modified, it is proven that the predicted DNA methyltransferases do have the function of DNA methylation modification and the proteins thereof have activity in E. coli. Detection methods include high-performance liquid chromatography and DNA dot hybridization method.

The DNA methyltransferases described above include methyltransferases type I, type II, and type III, and a DNA methyltransferase type I should include a methyl transfer subunit and a DNA recognition subunit.

Wherein, although said inducible expression vector may be a low-copy vector or may be a high-copy vector, it should be compatible with shuttle/integration plasmids for transforming target bacterium. Promoters for methyltransferases may be promoters of themselves, or may be inducible promoters of E. coli, and include, but is not limited to, arabinose-inducible promoter, IPTG-inducible promoter, and rhamnose-inducible promoter. The induction temperature for a methyltransferase is 8° C.-43° C.

Said E. coli with no restriction modification system is an E. coli wherein all known restriction enzymes and DNA methyltransferases including, but is not limited to, Dam, Dcm, EcoKI, Mrr, McrA, and Mrr have been deleted.

Another object of the present invention is to provide an E. coli EC135.

The E. coli provided in the invention has a deposit accession number of CGMCC No. 5925.

The strain EC135 was deposited in China General Microbiological Culture Collection Center (simply referred to as CGMCC; Address: No. 1 Beichen West Road, Chaoyang District, Beijing, Institute of Microbiology Chinese Academy of Sciences, Postal Code 100101) on Mar. 21, 2012, with a Deposit No. of CGMCC No. 5925, and the category thereof is designated as Escherichia coli.

The methods for transforming target bacterium vary with different target bacterium selected, and include, but is not limited to, a chemical transformation method, conjugative transfer, an electrical transformation method, and a protoplast transformation method.

The experiments of the invention have demonstrated that the invention has the following advantages compared to prior methods for enabling genetic manipulation by overcoming the restriction modification barrier of bacteria.

1. By using a strain of E. coli whose restriction modification system has been completely deleted as a host, on one hand, the activation effect of the E. coli modification pattern (Dam, Dcm, and EcoKI) on the restriction modification system type IV of the target bacterium is overcome, and on the other hand, deletions of Mrr, McrA and, McrBC of E. coli is favorable to the expression and cloning of DNA methyltransferases of the target bacterium;

2. By co-expressing all DNA methyltransferases of the target bacterium, precise simulation of the DNA methylation pattern of the target bacterium has been achieved;

3. What is used is in vivo DNA methylation modification in E. coli, which is performed in the process of culturing bacteria without additional in vitro reactions and without methylation reaction substrates added, and is convenient and fast;

4. In the invention, Saccharomyces cerevisiae may be used to assemble several methyltransferase genes, and the construction of a co-expression plasmid may be achieved within one week, accelerating the process in which the target bacterium overcomes the restriction modification barrier;

5. The restriction modification system of the transformants of the target bacterium obtained using this method is the same as that of original strains. This method will not damage original restriction modification system of the target bacterium;

6. The invention has good generality, and has been successfully applied to genetic manipulations of two strains of Bacillus amyloliquefaciens and a strain of Gram-negative chemoautotrophic bacteria, and is expected to be expanded to more genera and species.

The invention is significant to the construction of the genetic manipulation system of intractable bacteria having multiple restriction modification systems.

## SPECIFIC EMBODIMENTS

The following examples facilitate better understanding of the invention, instead of limiting the invention. The experimental methods described below are all conventional methods, unless specified otherwise. The test materials used in the examples described below are all commercially available from conventional biochemical reagent stores, unless specified otherwise. The quantitative tests in the examples below are all set with triplicate experiments, and results show average values.

### Example 1

**Construction of E. coli EC135 Whose Restriction Modification System has been Completely Deleted**

E. coli TOP10 (Catalog No. CD101, Beijing TransGen Biotech Co., Ltd.) was used as the an original strain and, in the following order, gene dcm (Dcm methylase encoding gene) thereof was knocked out; chromosomal recA gene was mutated to wild type, and gene dam (Dam methylase encoding gene) was knocked out, respectively, to obtain E. coli EC135. Details were as follows.

Competent cells of the strain TOP10 was prepared; a plasmid pKD46 (No. 7739, purchased from Coli Genetic Stock Center, U.S., simply referred to as CGSC below) was transformed into the strain TOP10, and ampicillin-resistant transformants were screened at 30° C. to obtain TOP10/pKD46.

The chloramphenicol resistance gene was amplified with primers WB089 and WB090 using a plasmid pKD3 (No. 7631, purchased from CGSC, U.S.) as the template, and 1166 bp was recovered by cutting the gel to obtain a PCR product of the chloramphenicol resistance gene.

The strain TOP10/pKD46 was cultured in an LB medium at 30° C.; induction was performed for 1 hour by adding arabinose at a final concentration of 100 mM when OD600 was 0.2; 40 μL of competent cells were transformed using 5 μL of the recovered PCR product of the chloramphenicol resistance gene after the competent cells were prepared, and a recovered product was coated on an LB plate containing 34 μg/mL of chloramphenicol after 1 h recovery at 30° C. and was cultured at 37° C. overnight. Recombinants were picked and subject to colony PCR identification using primers WB064 and WB065. The PCR product of a positive recombinant should have a size of 1816 bp, while the original gene had a size of 1980 bp. Positive recombinants were picked and were subject to continuous culture in an LB liquid culture medium at 42° C. for three passages to eliminate plasmid pKD46 and obtain a strain TOP10 dcm::CmR.

A single colony of the strain TOP10 dcm::CmR was picked after dilution and plate coating to prepare an electrotransformation competent strain; a plasmid pCP20 (No. 7629, purchased from CGSC, U.S.) was transformed into the strain TOP10 dcm::CmR; after screening for recombinants with ampicillin-resistant recombinants, a single colony was cultured in an LB medium without antibiotics at 30° C. until OD600 0.2, and then temperature was increased to 42° C. for overnight culture. A non-resistant LB plate was coated after bacterial solution was diluted, and a single colony was picked and subject to colony PCR identification using primers WB064 and WB065. The amplification product of a positive recombinant with chloramphenicol resistance gene eliminated should have a size of 886 bp (FIG. 1), and this positive recombinant was designated as TOP10Adcm::FRT.

Since a double-mutation of dam and recA is a lethal genotype, reversion of gene recA1 was first performed before gene dam was knocked out. Wild-type gene recA (1236 bp) was amplified with primers WB253 and WB254 using chromosomal DNA of E. coli W3110 (No. 4474, purchased from CGSC, U.S.) as the template, and was ligated to sites NotI and BamHI of vector pKOV (Catalog No. 25769, purchased from Addgene, U.S.) to obtain pKOV-recA. pKOV-recA was transformed into the strain TOP10Adcm::FRT, chloramphenicol-resistant transformants were screened at 30° C., transformants were picked and subsequently cultured overnight in a liquid LB medium at 42° C., and an LB plate containing chloramphenicol was coated to allow pKOV-recA to be integrated into the site recA1 of the chromosome. After a single colony is obtained, colony PCR was performed using primers WB255 and WB256. If there is a PCR band greater than 7 Kb, it is demonstrated that pKOV-recA has been integrated into the chromosome. Correct recombinants were picked and coated on an LB plate containing 10% sucrose after culturing at 42° C., a single colony was lined on both an LB plate with chloramphenicol and an LB plate without chloramphenicol for simultaneous passage, and chloramphenicol-sensitive colonies were picked. Gene recA was amplified using primers WB255 and WB256 and sequencing was performed, and the base at site 482 of gene recA of recA+recombinant should be G, while that of the strain TOP10 is A.

The steps of knockout of gene dam were the same as those of dcm, and amplification primers for the chloramphenicol resistance gene were WB087 and WB088, and outer detection primers were WB062 and WB063. The amplification product had a size of 740 bp after genes were knocked out, while the original gene had a size of 1356 bp (FIG. 2).

Through the three steps of genetic manipulations described above, the resultant strain with dcm and dam deleted and recA reverse-mutated was EC135.

The above-mentioned strain EC135 was deposited in China General Microbiological Culture Collection Center (simply referred to as CGMCC; Address: No. 1 Beichen West Road, Chaoyang District, Beijing, Institute Of Microbiology Chinese Academy of Sciences, Postal Code 100101) on Mar. 21, 2012, with a Deposit No. of CGMCC No. 5925, and the category thereof was designated as Escherichia coli.

See Table 1 for the sequences of the primers used.

### Examples 2

**The Introduction of an Exogenous DNA Molecule into Bacillus amyloliquefaciens TA208 by Overcoming the Restriction Modification Barrier**

I. The Construction of Recombinant Bacterium Co-Expressing all DNA-methyltransferase-encoding genes of TA208

1. The Achievement of DNA-methyltransferase-Encoding Genes of Strain TA208

1) the Prediction of DNA-methyltransferase-Encoding Genes of Strain TA208

Genome-wide sequences of the strain TA208 have been determined and its GenBank No. is CP002627. There are in total 5 putative DNA-methyltransferase-encoding genes on its chromosome, locus tags of these genes are BAMTA208—06525, BAMTA208—6715, BAMTA208—14440, BAMTA208—19835, and BAMTA208—16660, respectively. PCR amplification results of five gene fragments were shown in FIG. 3.

2) the Verification of DNA-methyltransferase-Encoding Genes of Strain TA208

The five genes described above were respectively cloned to sites between NcoI and XbaI, EcoRI and XbaI, EcoRI and XbaI, EcoRI and SalI, and EcoRI and SalI of plasmid pBAD43 (Tight regulation, modulation, and high-level expression by vectors containing the arabinose PBAD promoter. L M Guzman, D Belin, M J Carson, J. Beckwith. Journal of Bacteriology, 1995 177:4121-30, publically available from Institute Of Microbiology Chinese Academy of Sciences), so as to obtain a plasmid pBAD43 containing BAMTA208—06525, a plasmid pBAD43 containing BAMTA208 6715, a plasmid pBAD43 containing BAMTA208—14440, a plasmid pBAD43 containing BAMTA208—19835, and a plasmid pBAD43 containing BAMTA208—16660; and the 5 types of plasmids were respectively transformed into E. coli EC135 to obtain recombinant bacteria 1-5. Plasmids of recombinant bacteria 1-5 were extracted and sent for sequencing in order to verify correctness, and consequently the recombinant bacteria were positive recombinant bacteria.

Dot hybridization verification: the recombinant bacteria 1-5 verified to be positive were induced to express methyltransferases (arabinose at a final concentration of 0.2% by mass was added after culturing in an LB medium until OD600 0.2, and induction was performed at 30° C. for 12 hours), and then total DNAs were extracted using DNeasy Blood and Tissue Kit (Qiagen) to obtain DNA1-DNA5. DNA1-DNA5 obtained above were respectively boiled for 3 min to be denaturized into single strands, which were subsequently inserted into an ice-water mixture for quenching. EC135/pBAD43 (pBAD43 transferred into EC135) was used as a negative control.

Total DNAs of EC135/pBAD43 and 5 samples DNA1-DNA5 were all spotted onto a Protran BA85 nitrocellulose film (Whatman) at 450 ng, 150 ng, and 50 ng, which was repeated on three films. The films were placed in 5% skim milk powder formulated with TBST (200 mM NaCl, 0.1% Tween20, 50 mM Tris-HCl, pH7.4) after 2 min UV crosslinking, and were blocked at room temperature (25° C.) for 1 hour. Three films were then placed into a hybridization bag, and 10 mL 1:10000 diluted rabbit anti-N6 mA serum (New England Biolabs), 10 mL 1:10000 diluted rabbit anti-N4mC serum (New England Biolabs), and 10 mL 1:20000 diluted mouse anti-5mC monoclonal antibody (Zymo Research, Catalog No.A3001-50) were added, respectively. The films were washed 5 times with TBST after incubation for 1 hour at room temperature (25° C.), and corresponding horse radish peroxidase labeled goat anti-rabbit second antibody (Catalog No. 111-035-003, Jackson ImmunolResearch, U.S.) or goat anti-mouse second antibody (Catalog No. 81-6520, Zymed, U.S.), with a dilution of 1:10000, was added after the films were placed into the hybridization bag. The films were washed 5 times with TBST after incubation for 1 hour at room temperature. Solutions A and B (each 0.5 mL) of an ECL reagent (Catalog No. RPN2232, Amersham) were uniformly mixed and evenly dropped on the surfaces of the films. Fluorescence signal was exposed to X-ray films in a dark room.

Results were shown in FIG. 4 (in which m6A/m4C/m5C were results of hybridizations using different antibodies). It can be seen that BAMTA208—06525, BAMTA208—6715, BAMTA208—19835, and BAMTA208—16660 have activities of methylation modification and are DNA methyltransferase genes.

2. The Achievement of Recombinant Bacterium Co-Expressing all DNA-methyltransferase-encoding Genes

1) Construction

Using a plasmid pBAD43 containing BAMTA208—06525 (SEQ ID NO: 2), a plasmid pBAD43 containing BAMTA208—6715 (SEQ ID NO: 3), a plasmid pBAD43 containing BAMTA208—19835 (SEQ ID NO: 4), and a plasmid pBAD43 containing BAMTA208—16660 (SEQ ID NO: 5) as templates respectively, with WB325 and WB475 (having a size of 839 bp, FIG. 3) as primers used for BAMTA208—06525, WB476 and WB477 (having a size of 1512 bp, FIG. 3) as primers used for BAMTA208—6715, WB478 and WB479 (having a size of 1794 bp, FIG. 3) as primers used for BAMTA208—19835, and WB480 and WB326 (having a size of 1272 bp, FIG. 3) as primers used for BAMTA208—16660, the sequences of the primers being shown in Table 1, PCR amplification was performed to obtain 4 PCR products respectively.

The 4 PCR products were respectively recovered by cutting the gel, and equal amounts of the 4 PCR products were mixed and concentrated to a total volume 50 μL to obtain PCR total products.

500 ng of plasmid pWYE724 (see FIG. 5 for the plasmid map, and the sequence thereof is SEQ ID NO: 1) was double-cleaved by EcoRI and SalI, and was mixed with the PCR total products described above after being recovered by cutting, and then by using a lithium acetate transformation method (Methods in Enzymology, 350, 87-96.), Saccharomyces cerevisiae DAY414 was transformed (Mds3 regulates morphogenesis in Candida albicans through the TOR pathway, Zacchi L F, Gomez-Raja J, Davis D A, Molecular and Cellular Biology, 2010, 30: 3695-3710, publically available from Institute Of Microbiology Chinese Academy of Sciences). Transformants were screened on a plate of a complete synthetic culture medium without tryptophan added (SC trp-, Beijing FunGenome Co., Ltd.). A single colony was picked into a YPD culture medium, and a yeast plasmid was extracted using a glass bead method (Nucleic Acids Research, 20, 3790). The plasmid was transformed into E. coli TOP10, transformants were screened using an LB plate containing 100 μg/mL spectinomycin, and EcoRV single cleavage was performed after the plasmid was extracted. A correct recombinant plasmid should generate four bands 8515 bp, 3633 bp, 1018 bp, and 571 bp (as shown in FIG. 6), which was designated as pM.Bam (the structural schematic diagram was shown in FIG. 7), and was sent for sequencing to demonstrate that the results were correct.

The plasmid pM.Bam was transformed into the strain EC135 obtained in example 1, and cleavage verification was performed according to the method described above. The correct transformant was EC135/pM.Bam, which was a host precisely simulated in a pattern of Bacillus amyloliquefaciens TA208 DNA methylation.

2) Verification

In order to detecting the effectiveness of methylation simulation, methyltransferase genes in EC135/pM.Bam were allowed to be expressed by inducing at 30° C. for 12 hours using arabinose at a final concentration of 0.2%. Total DNAs were extracted and dot hybridization detection was performed using chromosomal DNA of the strain TA208 as a control according to the method described above in 1, and the result was that there was no significant difference in the hybridization between EC135/pM.Bam and TA208, which demonstrated that methyltransferases were all expressed, and which demonstrated that EC135/pM.Bam is a recombinant bacterium co-expressing all DNA-methyltransferase-encoding genes.

II. The Introduction of an Exogenous DNA Molecule into a Target Bacterium by Overcoming Restriction Modification Barrier

1. The Introduction of Shuttle Plasmid into a Target Bacterium by Overcoming Restriction Modification Barrier

A. Introduction

1) Shuttle plasmid pAD43-25 (7262 bp, Catalog No. ECE166, purchased from Bacillus Genetic Stock Center, U.S., simply referred to as BGSC below) was transformed into EC135/pM.Bam obtained in “I” to obtain recombinant bacteria respectively.

2) The recombinant bacteria were subject to arabinose induction (in which induction was performed at 30° C. for 12 hours using arabinose at a final concentration of 0.2%) to obtain induced recombinant bacteria.

3) Plasmids of the induced recombinant bacteria were extracted (wherein the plasmids include pM.Bam and shuttle plasmids, while pWYE724 is a low-copy plasmid, pM.Bam has a copy number of about 5/cell, and the shuttle plasmid has a copy number of about 300/cell), and Bacillus amyloliquefaciens TA208 was transformed (Complete genome sequence of Bacillus amyloliquefaciens TA208, a strain for industrial production of guanosine and ribavirin, Guoqiang Zhang, Aihua Deng, Qingyang Xu, Yong Liang, Ning Chen, Tingyi Wen, Journal of Bacteriology, 193 (12):3142-3143) using a method as recorded in Analytical Biochemistry, 409, 130-137, to obtain TA208/pAD43-25 (EC135/pM.Bam).

Using the same method, shuttle plasmid pAD43-25 was respectively transferred into TOP10 and EC135, and was then transferred into TA208 after modification to obtain TA208/pAD43-25 (TOP10) and TA208/pAD43-25 (EC 135) respectively.

Using the same method, shuttle plasmids pAD123 (5952 bp, No. ECE165, purchased from BGSC, U.S.), pMK3 (7214 bp, No. ECE15, purchased from BGSC, U.S.), pMK4 (5585 bp, No. ECE16, purchased from BGSC, U.S.), pHCMC02 (6866 bp, No. ECE188, purchased from BGSC, U.S.), pHCMC04 (8089 bp, No. ECE189, purchased from BGSC, U.S.), and pDG148StuI (8272 bp No. ECE145, purchased from BGSC, U.S.) were respectively transferred into EC135/pM.Bam, TOP10, and EC135, and were then transferred into TA208 after modification to obtain the following, respectively:

TA208/pAD123 (EC135/pM.Bam), TA208/pAD123 (TOP10), and TA208/pAD123 (EC135),

TA208/pMK3 (EC135/pM.Bam), TA208/pMK3 (TOP10), and TA208/pMK3 (EC 135),

TA208/pMK4 (EC135/pM.Bam), TA208/pMK4 (TOP10), and TA208/pMK4 (EC135), TA208/pHCMC02 (EC135/pM.Bam), TA208/pHCMC02 (TOP10), and TA208/pHCMC02 (EC135),

TA208/pHCMC 04 (EC135/pM.Bam), TA208/pHCMC 04 (TOP10), and TA208/pHCMC 04 (EC135),

TA208/pDG148StuI (EC135/pM.Bam), TA208/pDG148StuI (TOP10), and TA208/pDG148StuI (EC135).

B. The Calculation of Transformation Efficiency

TA208/pAD43-25 (EC135/pM.Bam), TA208/pAD43-25 (TOP10), TA208/pAD43-25 (EC 135), TA208/pAD123 (EC135/pM.Bam), TA208/pAD123 (TOP10), TA208/pAD123 (EC 135), TA208/pMK3 (EC135/pM.Bam), TA208/pMK3 (TOP10), TA208/pMK3 (EC135), TA208/pMK4 (EC135/pM.Bam), TA208/pMK4 (TOP10), TA208/pMK4 (EC135), TA208/pHCMC02 (EC135/pM.Bam), TA208/pHCMC02 (TOP10), TA208/pHCMC02 (EC135), TA208/pHCMC 04 (EC135/pM.Bam), TA208/pHCMC 04 (TOP10), TA208/pHCMC 04 (EC135), TA208/pDG148StuI (EC135/pM.Bam), TA208/pDG148StuI (TOP10), and TA208/pDG148StuI (EC135). Using the number of single colonies on the coated plate after transformation with the above, according to the amount of DNA of shuttle plasmids used (since the copy number of pM.Bam was too low, the total amount of plasmids added was the amount of DNA of shuttle plasmids), the transformation efficiency was calculated. 4 transformants were picked randomly for DNA verification by extracting plasmids. The experiment was repeated three times and the results showed the average value ±the standard deviation.

The transformation efficiency is: the number of single colonies (CFU)×(the volume of recovered solution after transformation/the volume of coated bacterial solution) xthe dilution ratio of recovered solution/the amount of shuttle plasmids used in transformation (μg).

Results were shown in FIG. 8, and it can be seen:

the transformation efficiency for TA208/pAD43-25 (EC135/pM.Bam) is 6.4±2.6×105CFU/μg DNA;

the transformation efficiency for TA208/pAD43-25 (TOP10) was 0;

the transformation efficiency for TA208/pAD43-25 (EC135) was 2.4±1.6×103CFU/μg DNA;

the transformation efficiency for TA208/pAD123 (EC135/pM.Bam) was 9.1±4.3×105CFU/μg DNA;

the transformation efficiency for TA208/pAD123 (TOP10) was 0;

the transformation efficiency for TA208/pAD123 (EC135) was 0;

the transformation efficiency for TA208/pMK3 (EC135/pM.Bam) was 2.2±0.5×105CFU/μg DNA;

the transformation efficiency for TA208/pMK3 (TOP10) was 0;

the transformation efficiency for TA208/pMK3 (EC135) was 4.5×101CFU/μg DNA;

the transformation efficiency for TA208/pMK4 (EC135/pM.Bam) was 3.0±0.4×106CFU/μg DNA;

the transformation efficiency for TA208/pMK4 (TOP10) was 0;

the transformation efficiency for TA208/pMK4 (EC135) was 3.1×102CFU/μg DNA;

the transformation efficiency for TA208/pHCMC02 (EC135/pM.Bam) was 2.0±0.3×106CFU/μg DNA;

the transformation efficiency for TA208/pHCMC02 (TOP10) was 0;

the transformation efficiency for TA208/pHCMC02 (EC135) was 6.3×101CFU/μg DNA;

the transformation efficiency for TA208/pHCMC 04 (EC135/pM.Bam) was 1.9±0.1×106CFU/μg DNA;

the transformation efficiency for TA208/pHCMC 04 (TOP10) was 0;

the transformation efficiency for TA208/pHCMC 04 (EC135) was 1.5×101CFU/μg DNA;

the transformation efficiency for TA208/pDG148StuI (EC135/pM.Bam) was 9.7±3.2×104CFU/μg DNA;

the transformation efficiency for TA208/pDG148StuI (TOP10) was 0; and

the transformation efficiency for TA208/pDG148StuI (EC135) was 0.

Plasmids prepared by TOP10 cannot transform the strain TA208, plasmids prepared in the strain EC135 had a transformation efficiency at a level of 0-103CFU/μg DNA, and shuttle plasmids prepared in EC135/pM.Bam had a transformation efficiency at a level of 104-106CFU/μg DNA.

Plasmids of single colonies for each strain were extracted simultaneously and an electrophoretic or sequencing comparison with a plasmid before transformation was performed, and the result showed an identical size and demonstrated that these strains were positive plasmids in which an exogenous DNA molecule was transferred.

2. The Introduction of Integration Plasmid into a Target Bacterium

The method for preparing an integration plasmid pWYE748 containing a homologous arm of gene upp and chloramphenicol resistance gene was as follows: Using primers WB607 and WB608 as the primers and chromosomal DNA of TA208 as the template, a homologous arm 641 bp upstream of gene upp was subject to PCR amplification; using WB609 and WB610 as primers and a plasmid pMK4 as the template, chloramphenicol resistance gene 906 bp was subject to amplification; using WB611 and WB612 as the primers and chromosomal DNA of TA208 as the template, a homologous arm 669 bp downstream of gene upp was subject to PCR amplification; and the three stages of the PCR products were recovered by cutting the gel, and were then mixed (1 μL each) and used as templates. PCR was performed again using WB607 and WB612 as primers to obtain a fragment of 2216 bp. The PCR product was cloned to a plasmid pMD19-T (Catalog No. D106A, Taraka), and the resultant plasmid was the integration plasmid pWYE748.

Using the transformation method described above in 1, the integration plasmid pWYE748 was modified in EC135/pM.Bam, and then the strain TA208 was transformed and chloramphenicol-resistant transformants were screened.

Amplification identification was performed with primers WB605 and WB606, using DNA of the transformant and genomic DNA of wild-type TA208 as the templates. The results of amplification were shown as in FIG. 9, and the amplification size of the transformant was 2375 bp (gene upp was inserted into the chloramphenicol resistance gene), while the original size of chloramphenicol resistance gene of wild-type TA208 was 2049 bp, which demonstrated that an exogenous gene was transferred and the integration was successful, and the transformant was designated as BS043.

Gene upp encodes uracil phosphoribosyl transferase, by which 5-fluorouracil is converted into 5-fluorouracil nucleoside monophosphate and eventually metabolized into a toxic metabolite 5-fluorouracil nucleoside diphosphate, which strongly inhibits the activity of thymidylate synthase and results in death of bacteria.

It is further demonstrated whether BS043 has included an exogenous gene, and the method thereof is: Strains BS043 were grown on MM culture media (see Molecular Microbiology, 46, 25-36. for the components of the culture media, and 100 mg/L adenosine was added) with/without 10 μM 5-fluorouracil (Catalog No. F6627, Sigma) added, and a strain TA208 was used as a control. Results were shown in FIG. 10, and it can be seen: BS043 can grow on the culture medium with 10 μM 5-fluorouracil added, which demonstrated that an exogenous gene was transferred and the integration was successful.

However, in EC135 and TOP10, TA208 was transformed several times by the extracted integration plasmid pWYE748 and it failed to obtain any transformant.

### Example 3

**The Introduction of an Exogenous DNA Molecule into Bacillus cereus ATCC 10987 by Overcoming Restriction Modification Barrier**

I. The Construction of Recombinant Bacterium Co-Expressing all DNA-methyltransferase-Encoding Genes of ATCC 10987

1. The Achievement of DNA-methyltransferase-Encoding Genes of Strain ATCC 10987

1) the Prediction of DNA-methyltransferase-Encoding Genes of Strain ATCC 10987

Genome-wide sequences of the strain ATCC 10987 have been disclosed and its GenBank No. is AE017194. There are total 9 putative DNA-methyltransferase-encoding genes on its chromosome, locus tags of these genes are BCE—0841-BCE—0842, BCE—0839-BCE—0842, BCE—0365, BCE—0392, BCE—0393, BCE—4605, BCE—5606, BCE—5607, and BCE—1018, respectively.

2) the Verification DNA-methyltransferase-Encoding Genes of Strain ATCC 10987 BCE—0841-BCE—0842, BCE—0839-BCE—0842, BCE—0365, BCE—0392, BCE—0393, BCE—4605, BCE—5606, BCE—5607, and BCE—1018 were respectively cloned to sites between NheI and KpnI, NcoI and KpnI, NheI and KpnI, NheI and KpnI, NheI and KpnI, NheI and KpnI, NheI and KpnI, NheI and XbaI, and NheI and KpnI of a plasmid pBAD43, so as to obtain a plasmid pBAD43 containing BCE—0841-BCE—0842, a plasmid pBAD43 containing BCE—0839-BCE—0842, a plasmid pBAD43 containing BCE—0365, a plasmid pBAD43 containing BCE—0392, a plasmid pBAD43 containing BCE—0393, a plasmid pBAD43 containing BCE—4605, a plasmid pBAD43 containing BCE—5606, a plasmid pBAD43 containing BCE—5607, and a plasmid pBAD43 containing BCE—1018.

The aforementioned types of plasmids were respectively transformed into E. coli EC135 to obtain recombinant bacteria 1-9.

Plasmids of recombinant bacteria 1-9 were extracted and sent for sequencing in order to verify correctness, and consequently the recombinant bacteria are positive recombinant bacteria.

Dot Hybridization Verification:

After the recombinant bacteria 1-9 verified to be positive were induced to express methyltransferases (arabinose at a final concentration of 0.2% was used and induction was performed at 30° C. for 12 hours), total DNAs were extracted using DNeasy Blood and Tissue Kit (Qiagen) to obtain DNA1-DNA9. DNA1-DNA9 obtained above were boiled for 3 min to be denaturized into single strands, which were subsequently inserted into an ice-water mixture for quenching. EC135/pBAD43 was used as a negative control.

Total DNAs of EC135/pBAD43 and 9 samples DNA1-DNA9 were all spotted onto a Protran BA85 nitrocellulose film (Whatman) at 450 ng, 150 ng, and 50 ng, which was repeated on three films. The films were placed in 5% skim milk powder formulated with TBST (200 mM NaCl, 0.1% Tween20, 50 mM Tris-HCl, pH7.4) after 2 min UV crosslinking, and were blocked at room temperature for 1 hour. Three films were then placed into a hybridization bag and 10 mL 1:10000 diluted rabbit anti-N6 mA serum, 10 mL 1:10000 diluted rabbit anti-N4mC serum, and 10 mL 1:20000 diluted mouse anti-5mC monoclonal antibody, were added, respectively. The films were washed 5 times after incubation for 1 hour at room temperature, and corresponding goat anti-rabbit second antibody or goat anti-mouse second antibody, with a dilution ratio of 1:10000, was added after the films were placed into the hybridization bag. The films were washed 5 times after incubation for 1 hour at room temperature. Solutions A and B (each 0.5 mL) of an ECL reagent were uniformly mixed and evenly dropped on the surfaces of the films. Fluorescence signal was exposed to X-ray films in a dark room.

Results were shown in FIG. 11 (in which m6/m4C/m5C were results of hybridizations using different antibodies). It can be seen that BCE—0393, BCE—4605, BCE—5606, BCE—5607, BCE—0365, and BCE—0392 have activities of methylation modification and are DNA methyltransferase genes.

2. The Achievement of Recombinant Bacterium Co-Expressing all DNA-methyltransferase-encoding Genes

1) Construction

Using a plasmid pBAD43 containing BCE—0393 (SEQ ID NO: 6), a plasmid pBAD43 containing BCE—4605 (SEQ ID NO: 7), a plasmid pBAD43 containing BCE—5606 (SEQ ID NO: 8), a plasmid pBAD43 containing BCE—5607 (SEQ ID NO: 9), a plasmid pBAD43 containing BCE—0365 (SEQ ID NO: 10), and a plasmid pBAD43 containing BCE—0392 (SEQ ID NO: 11) as templates respectively, with WB325 and WB575 (having a size of 2160 bp) as primers used for BCE—0393, WB576 and WB577 (having a size of 1102 bp) for BCE—4605, WB578 and WB579 (having a size of 1345 bp) for BCE—5606, WB580 and WB581 (having a size of 1316 bp) for BCE—5607, WB582 and WB583 (having a size of 1071 bp) for BCE—0365, and WB584 and WB326 (having a size of 1257 bp) for BCE—0392, the sequences of the primers being as shown in Table 1, PCR amplification was performed to obtain 6 PCR products respectively.

The 6 PCR products were respectively recovered by cutting the gel, and equal proportions of the 6 PCR products were mixed and concentrated to a total volume 50 μL to obtain PCR total products.

500 ng of plasmid pWYE724 was double-cleaved by EcoRI and SalI, and was mixed with the PCR total products described above after being recovered by cutting, and then by using a lithium acetate transformation method (Methods in Enzymology, 350, 87-96.), Saccharomyces cerevisiae DAY414 was transformed. Transformants were screened on a plate of a complete synthetic culture medium without tryptophan added (SC trp-, Beijing FunGenome Co., Ltd.). A single colony was picked into a YPD culture medium, and a yeast plasmid was extracted using a glass bead method (Nucleic Acids Research, 20, 3790). The plasmid was transformed into E. coli TOP10, transformants were screened using an LB plate containing 100 μg/mL spectinomycin, and HindIII single cleavage was performed after the plasmid was extracted. A correct recombinant plasmid should generate five bands 6544 bp, 4237 bp, 3642 bp, 1315 bp, and 827 bp (as shown in FIG. 12), which was designated as pM.Bce (the structural schematic diagram was shown in FIG. 13), and was sent for sequencing to demonstrate that the result was correct.

The plasmid pM.Bce was transformed into the strain EC135 obtained in example 1, and cleavage verification was performed according to the method described above. The correct transformant is EC135/pM.Bce, which is a host precisely simulated in a pattern of Bacillus cereus ATCC 10987 DNA methylation.

2) Verification

In order to detecting the effectiveness of methylation simulation, methyltransferase genes were allowed to be expressed by inducing EC135/pM.Bce at 30° C. for 12 hours using arabinose induction with a final concentration of 0.2%. Total DNAs were extracted and dot hybridization detection was performed using chromosomal DNA of the strain ATCC 10987 as a control according to the aforementioned method, and the result was that there was no significant difference in the hybridization between EC135/pM.Bce and ATCC 10987, which demonstrated that methyltransferases were all expressed, and which demonstrated that EC135/pM.Bce was a recombinant bacterium co-expressing all DNA-methyltransferase-encoding genes.

II. The Introduction of Shuttle Plasmid into a Target Bacterium by Overcoming Restriction Modification Barrier

Shuttle plasmids were respectively transferred into EC135/pM.Bce, EC135, and TOP10, the plasmids were extracted after arabinose induction (see aforementioned methods), and strains ATCC 10987 were respectively transformed using the same method as in “I” of Example 2. The shuttle plasmids described above were pAD43-25, pAD123, pMK3, pMK4, and pHCMC02, and the following were respectively obtained:

ATCC 10987/pAD43-25 (EC135/pM.Bce), ATCC 10987/pAD43-25 (TOP10), ATCC 10987/pAD43-25 (EC135), ATCC 10987/pAD123 (EC135/pM.Bce), ATCC 10987/pAD123 (TOP10), ATCC 10987/pAD123 (EC135), ATCC 10987/pMK3 (EC135/pM.Bce), ATCC 10987/pMK3 (TOP10), ATCC 10987/pMK3 (EC135), ATCC 10987/pMK4 (EC135/pM.Bce), ATCC 10987/pMK4 (TOP10), ATCC 10987/pMK4 (EC135), ATCC 10987/pHCMC02 (EC135/pM.Bce), ATCC 10987/pHCMC02 (TOP10), and ATCC 10987/pHCMC02 (EC135);

Using the number of single colonies on the coated plate after transformation, according to the amount of DNA of shuttle plasmids used (since the copy number of pM.Bce was about 5/cell and the copy number of the shuttle plasmid was about 300/cell, the total amount of plasmids added was the amount of DNA of shuttle plasmids), the transformation efficiency was calculated. 4 transformants were picked randomly for DNA verification by extracting plasmids. The experiment was repeated three times and the result employs the average value±the standard deviation.

The transformation efficiency is: the number of single colonies (CFU)×(the volume of recovered solution after transformation/the volume of coated bacterial solution)×the dilution ratio of recovered solution/the amount of shuttle plasmids used in transformation (μg).

Results are shown in FIG. 14, and it can be seen: the transformation efficiency for ATCC 10987/pAD43-25 (EC135/pM.Bce) was 3.2±1.3×105CFU/μg DNA;

the transformation efficiency for ATCC 10987/pAD43-25 (TOP10) was 8.3±0.7×103CFU/μg DNA;

the transformation efficiency for ATCC 10987/pAD43-25 (EC135) was 7.1±2.9×103CFU/μg DNA;

the transformation efficiency for ATCC 10987/pAD123 (EC135/pM.Bce) was 9.0±1.4×105CFU/μg DNA;

the transformation efficiency for ATCC 10987/pAD123 (TOP10) was 8.7±0.8×103CFU/μg DNA;

the transformation efficiency for ATCC 10987/pAD123 (EC135) was 5.7±3.3×103CFU/μg DNA;

the transformation efficiency for ATCC 10987/pMK3 (EC135/pM.Bce) was 8.4±2.2×105CFU/μg DNA;

the transformation efficiency for ATCC 10987/pMK3 (TOP10) was 8.7±2.7×102CFU/μg DNA; the transformation efficiency for ATCC 10987/pMK3 (EC135) was 1.2±0.6×103CFU/μg DNA;

the transformation efficiency for ATCC 10987/pMK4 (EC135/pM.Bce) was 2.3±0.2×107CFU/μg DNA;

the transformation efficiency for ATCC 10987/pMK4 (TOP10) was 3.3±0.4×105CFU/μg DNA;

the transformation efficiency for ATCC 10987/pMK4 (EC135) was 3.5±1.4×105CFU/μg DNA;

the transformation efficiency for ATCC 10987/pHCMC02 (EC135/pM.Bce) was 2.5±1.5×105CFU/μg DNA;

the transformation efficiency for ATCC 10987/pHCMC02 (TOP10) was 0; and

the transformation efficiency for ATCC 10987/pHCMC02 (EC135) was 0.

It can be seen that the transformation efficiencies of plasmids prepared in EC135/pM.Bce was 102-103 times that of the plasmids prepared in TOP10 and EC135.

Plasmids of single colonies for each strain were extracted simultaneously and an electrophoretic or sequencing comparison with a plasmid before transformation was performed, and the result showed an identical size and demonstrated that these strains were positive plasmids in which an exogenous DNA molecule was transferred.

### Example 4

**The Introduction of an Exogenous DNA Molecule into Nitrobacter hamburgensis X14 by Overcoming Restriction Modification Barrier**

I. The Construction of Recombinant Bacterium Co-Expressing all DNA-methyltransferase-encoding Genes of X14

1. The Achievement of DNA-Methyltransferase-Encoding Genes of Strain X14

1) the Prediction of DNA-Methyltransferase-Encoding Genes of Strain X14

Genome-wide sequences of the strain X14 have been disclosed and their GenBank Nos. are CP000319, CP000320, CP000321, and CP000322. There are total 10 putative DNA-methyltransferase-encoding genes on the chromosome, locus tags of these genes are Nham—0569, Nham—0582, Nham—0803, Nham—0842, Nham—1185, Nham—1353, Nham—2515, Nham—3225, Nham—3845, and Nham—4499, respectively.

2) the Verification of DNA-methyltransferase-Encoding Genes of Strain X14

Nham—0569, Nham—0582, Nham—0803, Nham—0842, Nham—1185, Nham—1353, Nham—2515, Nham—3225, Nham—3845, and Nham—4499 were respectively cloned to sites between EcoRI and KpnI, EcoRI and KpnI, NheI and KpnI, NheI and KpnI, EcoRI and KpnI, EcoRI and KpnI, NheI and KpnI, EcoRI and KpnI, NheI and KpnI, and EcoRI and KpnI of a plasmid pBAD43, so as to obtain a plasmid pBAD43 containing Nham—0569, a plasmid pBAD43 containing Nham—0582, a plasmid pBAD43 containing Nham—0803, a plasmid pBAD43 containing Nham—0842, a plasmid pBAD43 containing Nham—1185, a plasmid pBAD43 containing Nham—1353, a plasmid pBAD43 containing Nham—2515, a plasmid pBAD43 containing Nham—3225, a plasmid pBAD43 containing Nham—3845, and a plasmid pBAD43 containing Nham—4499. The aforementioned types of plasmids were respectively transferred into E. coli EC135 to obtain recombinant bacteria 1-10.

Plasmids of recombinant bacteria 1-10 were extracted and sent for sequencing in order to verify correctness, and consequently the recombinant bacteria are positive recombinant bacteria.

Dot Hybridization Verification:

After the recombinant bacteria 1-10 verified to be positive were induced to express methyltransferases (arabinose at a final concentration of 0.2% was used and induction was performed at 30° C. for 12 hours), total DNAs were extracted using DNeasy Blood and Tissue Kit (Qiagen) to obtain DNA1-DNA10. DNA1-DNA10 obtained above were boiled for 3 min to be denaturized into single strands, which were subsequently inserted into an ice-water mixture for quenching. EC135/pBAD43 was used as a negative control.

Total DNAs of EC135/pBAD43 and 10 samples DNA1-DNA10 were all spotted onto a Protran BA85 nitrocellulose film (Whatman) at 450 ng, 150 ng, and 50 ng, which was repeated on three films. The films were placed in 5% skim milk powder formulated with TBST (200 mM NaCl, 0.1% Tween20, 50 mM Tris-HCl, pH7.4) after 2 min UV crosslinking, and were blocked at room temperature for 1 hour. Three films were then placed into a hybridization bag and 10 mL 1:10000 diluted rabbit anti-N6 mA serum, 10 mL 1:10000 diluted rabbit anti-N4mC serum, and 10 mL 1:20000 diluted mouse anti-5mC monoclonal antibody, were added, respectively. The films were washed 5 times after incubation for 1 hour at room temperature, and corresponding goat anti-rabbit second antibody or goat anti-mouse second antibody, with a dilution ratio of 1:10000, was added after the films were placed into the hybridization bag. The films were washed 5 times after incubation for 1 hour at room temperature. Solutions A and B (each 0.5 mL) of an ECL reagent were uniformly mixed and evenly dripped on the surfaces of the films. Fluorescence signal was exposed to X-ray films in a dark room.

Results were shown in FIG. 15 (in which m6A/m4C/m5C were results of hybridizations using different antibodies). It can be seen that Nham—0569, Nham—0582, Nham—0803, and Nham—3225 have activities of methylation modification and are DNA methyltransferase genes.

2. The Achievement of Recombinant Bacterium Co-Expressing all DNA-methyltransferase-encoding Genes

1) Construction

Using a plasmid pBAD43 containing Nham—0569 (SEQ ID NO: 12), containing Nham—0582 (SEQ ID NO: 13), a plasmid pBAD43 containing Nham—0803 (SEQ ID NO: 14), and a plasmid pBAD43 containing Nham—3225 (SEQ ID NO: 15), as templates respectively, with WB325 and WB585 (having a size of 648 bp) as primers used for Nham—0569, WB586 and WB587 (having a size of 1983 bp) for Nham—0582, WB588 and WB589 (having a size of 1317 bp) for Nham—0803, and WB590 and WB326 (having a size of 1131 bp) for Nham—3225, the sequences of the primers being as shown in Table 1, PCR amplification was performed to obtain 4 PCR products respectively.

The 4 PCR products were respectively recovered by cutting the gel, and equal proportions of the 4 PCR products were mixed and concentrated to a total volume 50 μL to obtain PCR total products.

500 ng of plasmid pWYE724 was double-cleaved by EcoRI and SalI, and was mixed with the PCR total products described above after being recovered by cutting, and then by using a lithium acetate transformation method (Methods in Enzymology, 350, 87-96.), Saccharomyces cerevisiae DAY414 was transformed. Transformants were screened on a plate of a complete synthetic culture medium without tryptophan added (SC trp-, Beijing FunGenome Co., Ltd.). A single colony was picked into a YPD culture medium, and a yeast plasmid was extracted using a glass bead method (Nucleic Acids Research, 20, 3790). The plasmid was transformed into E. coli TOP10, transformants were screened using an LB plate containing 100 μg/mL spectinomycin, and BamHI single cleavage was performed after the plasmid was extracted. A correct recombinant plasmid should generate four bands 8217 bp, 3848 bp, 1003 bp, and 343 bp (as shown in FIG. 16), which was designated as pM.Nham (the structural schematic diagram was shown in FIG. 17), and was sent for sequencing to demonstrate that the result was correct.

pM.Bce was transformed into the strain EC135 obtained in example 1, and cleavage verification was performed according to the method described above. The correct transformant was EC135/pM.Nham, which was a host precisely simulated in a pattern of Nitrobacter hamburgensis X14 DNA methylation.

2) Verification

In order to detecting the effectiveness of methylation simulation, methyltransferase genes in EC135 μM.Bam were allowed to be expressed by arabinose induction (inducing at 30° C. for 12 hours using arabinose induction with a final concentration of 0.2%). Total DNAs were extracted and dot hybridization detection was performed using chromosomal DNA of the strain X14 as a control according to the aforementioned method, and the result was that there was no significant difference in the hybridization between EC135/pM.Nham and the strain X14, which demonstrated that methyltransferases were all expressed, and which demonstrated that EC135/pM.Nham was a recombinant bacterium co-expressing all DNA-methyltransferase-encoding genes.

II. The Introduction of Shuttle Plasmid into a Target Bacterium by Overcoming Restriction Modification Barrier

The particular method for constructing a plasmid pBBR1-MCS5PNham3450-GFP was: Using WB654 and WB655 as primers and the genome of the strain X14 as the template, a gene Nham—3450 promoter fragment 216 bp was subject to amplification. Using WB656 and WB650 as primers and pAD123 as the template, gene GFP 717 bp was subject to amplification; the two PCR products were recovered by cutting the gel and were selected (1 μL each) as templates, and PCR was performed again using WB654 and WB650 as primers to obtain a PCR product of 933 bp, which was cloned into sites SalI and PstI of a plasmid pBBR1-MCS5 (Four new derivatives of the broad-host-range cloning vector pBBR1MCS, carrying different antibiotic-resistance cassettes. Kovach, Michael E.Elzer, Philip H.Steven Hill, D.Robertson, Gregory T. Farris, Michael A. Roop Ii, R. Martin Peterson, Kenneth M. 1995, Gene 166 (1): 175-176, publically available from Institute Of Microbiology Chinese Academy of Sciences). Sequencing verification was performed.

The plasmid pBBR1-MCS5-PNham3450-GFP was a broad-host-range plasmid containing green fluorescent protein encoding gene operated by Nham—3450, which was transferred into EC135/pM.Nham, subjected to plasmid extraction after arabinose induction, and transformed the strain X14, and the method for transforming the strain X14 was as follows: The strain was cultured in a DSM756a culture medium (1.5 g yeast extract, 1.5 g peptone, 2 g NaNO2, 0.55 g sodium pyruvate, 1 mL trace element solution (33.8 mg MnSO4.H2O, 49.4 mg H3BO3, 43.1 mg ZnSO4.7H2O, 37.1 mg (NH4)6Mo7O24, 97.3 mg FeSO4.7H2O and 25 mg CuSO4.5H2O in 1 L deionized water) and 100 mL stock solution (0.07 g CaCO3, 5 g NaCl, 0.5 g MgSO4.7H2O, 1.5 g KH2PO4 in 1 L deionized water) in 1 L deionized water, pH 7.4)) until OD600 was 0.1, was subject to ice bath 10 min, and was centrifuged at 4° C. in 8000 rpm for 10 min to collect bacteria, which were washed with 10% precooled glycerol for 4 times and re-suspended in 10% glycerol in 1/1000 of the volume of the original culture. 90 μL cells were selected and uniformly mixed with 150 ng plasmids, the mixture was added to a 1 mm electroporation cup with 1200V electric shock once (electroporator ECM399). The cells were washed into 100 mL 756a culture medium, were recovered at 28° C. for 1 day, and then were added to gentamycin with a final concentration of 20 μg/mL for further recovery for 1 day. A recovered mixture was trans-inoculated to a fresh 756a culture medium containing 20 μg/mL gentamycin at a volume ratio of 1/100. After OD600 of the culture medium was 0.1, trans-inoculation was performed once more using the same method, until the bacteria were grown to the extent that OD600 was 0.1, so as to obtain X14/pBBR1-MCS5-PNham3450-GFP (EC135/pM.Nham);

10 μL, culture broths of X14/pBBR1-MCS5-PNham3450-GFP (EC135/pM.Nham) were respectively selected, and were placed under a fluorescence microscope for observation after smearing, with X14 as a control.

Results were shown in FIG. 18, in which A was X14 and B was X14/pBBR1-MCS5-PNham3450-GFP (EC135/pM.Nham). It can be seen that bacteria X14/pBBR1-MCS5-PNham3450-GFP (EC135/pM.Nham) emits green fluorescence under 488 nm excitation light and untransformed bacteria do not have fluorescence.

It was demonstrated that an exogenous gene has been transferred.

10 mL×14/pBBR1-MCS5-PNham3450-GFP (EC135/pM.Nham) bacterial solution was selected, and total DNAs were extracted using a bacterial DNA extraction kit (DP302-02, Tiangen CO.). Then, 5 μL DNA transformed E. coli TOP10 was selected, gentamycin-resistant transformants were screened, and the extracted plasmid was verified to have an identical size (5701 bp) to that of the original plasmid pBBR1-MCS5-PNham3450-GFP, which demonstrated that the exogenous DNA molecule pBBR1-MCS5-PNham3450-GFP in X14/pBBR1-MCS5-PNham3450-GFP (EC135/pM.Nham) did not have any loss and was completely transferred into X14 by overcoming the restriction modification barrier.

In contrast, Carsiotis, M. et al. reported (Genetic engineering of enhanced microbial nitrification. Carsiotis, M. and Khanna, S. US Environmental Protection Agency, Risk Reduction Engineering Laboratory. 1989) that plasmid DNA extracted from a normal host of E. coli failed to achieve genetic transformation to the strain X14. Consequently, the method reported herein has great advantages.

