# DESCRIPTION

## TECHNICAL FIELD OF THE INVENTION

- relate to genetically engineered microorganisms

## BACKGROUND OF THE INVENTION

- motivate L-ornithine
- describe current preparation processes
- limitations of whole-cell biotransformation
- introduce thermophilic enzyme
- describe bacterial strains
- limitations of bacterial strains
- need for better microbial strategies

## SUMMARY OF THE INVENTION

- introduce eukaryotic cell factory
- genetically modify ornithine carbamyoltransferase
- enhance L-ornithine biosynthesis
- enhance α-ketoglutarate biosynthesis
- produce polyamines

## DETAILED DESCRIPTION

- motivate yeast S. cerevisiae as chassis host
- describe advantages of S. cerevisiae
- explain L-arginine biosynthesis compartmentalization
- describe challenges in constructing L-ornithine over-producing yeast strain
- introduce conventional techniques used in invention
- define "modified" host organism
- define "open reading frame (ORF)"
- define "recombinant"
- define "recombinant eukaryotic cells"
- define "recombinant prokaryotic cells"
- define "endogenous" nucleic acid or protein
- define "heterologous" nucleic acid or protein
- define "gene"
- define "genome"
- define "nucleic acid"
- define "peptide", "polypeptide", and "protein"
- define "enzyme"
- define "promoter"
- define "terminator"
- define "overexpression"
- define "overproducing"
- define "Crabtree effect"
- define "vector"
- define "sequence identity"
- describe modified eukaryotic cell for L-ornithine overproduction
- outline genetic modifications for L-ornithine production
- specify yeast genus and species
- list yeast species
- specify prokaryotic cell types
- list gram negative bacteria genera
- list gram positive bacteria genera
- describe modified yeast cell
- reduce L-ornithine consumption pathway
- reduce ornithine carbamoyltransferase activity
- attenuate ARG3 gene
- reduce L-ornithine degradation activity
- reduce L-ornithine transaminase activity
- delete CAR2 gene
- increase arginase activity
- overexpress CAR1 gene
- enhance carbon channelling from α-ketoglutarate
- describe L-ornithine biosynthetic pathway
- propose strategies to improve pathway efficiency
- enhance endogenous pathway in subcellular organelle
- overexpress enzymes/proteins in pathway
- list enzymes/proteins in pathway
- re-localize L-ornithine 'acetylated derivatives cycle'
- overexpress genes in pathway from bacteria cell
- enhance carbon channeling from substrate to α-ketoglutarate
- increase enzyme/protein activity in pathway
- overexpress pyruvate carboxylase
- overexpress citrate synthase
- increase pyruvate dehydrogenase activity
- overexpress E1 alpha subunit of pyruvate dehydrogenase
- describe eukaryotic cell
- overexpress aconitase
- overexpress isocitrate dehydrogenase
- increase pyruvate carrier activity
- overexpress enzymes/proteins
- decrease carbon substrate uptake
- increase NADH alternative oxidase activity
- convert carbon source to L-ornithine
- prepare L-ornithine
- incorporate L-ornithine into products
- genetically modify eukaryotic cells
- produce chemical compounds
- overproduce polyamines
- increase ornithine decarboxylase activity
- increase flux towards putrescine
- decrease ODC antizyme activity
- delete Methylthioadenosine phosphorylase
- down-regulate spermdine synthase
- increase flux towards spermidine
- overexpress yeast polyamine oxidase
- introduce flux towards spermine production
- increase activity of spermine synthase
- express heterologous SPS from other eukaryotic or prokaryotic sources
- combine with increase in activities of ODC, SAMDC and/or SPDS
- down-regulate ODC antizyme
- down-regulate yeast FMS1
- down-regulate polyamine acetyltransferase
- facilitate export of polyamines to media
- overexpress different export proteins
- down-regulate genes associated with polyamine uptake
- increase resistance of strains to polyamine toxicity
- down-regulate SR protein kinase
- down-regulate Putative serine/threonine protein kinase
- down-regulate BRP1 and FES1
- overexpress QDR3 and YAP1
- produce agmatine
- direct flux from ornithine to arginine and/or agmatine production
- overexpress native yeast ARG3, ARG1, ARG10 and/or ARG4 genes
- express heterologous genes encoding these activities
- delete or down-regulate native arginase (CAR1) genes
- convert arginine to agmatine
- introduce heterologous gene encoding arginine decarboxylase
- produce putrescine from agmatine
- introduce genes encoding enzyme activities of agmatinase
- convert agmatine to putrescine by introduction of two heterologous steps
- produce spermidine from agmatine
- express genes encoding enzymes for these activities
- produce spermidine via condensation of L-aspartate semialdehyde and putrescine
- express genes encoding enzyme with these activities
- delete or down-regulate reactions that compete with polyamine synthesis
- produce polyamines from bacterial sources
- modify E. coli WL3110 to increase ornithine and putrescine production
- overexpress ornithine decarboxylase converting ornithine to putrescine
- overexpress ornithine biosynthetic genes
- transform with plasmids for overexpression of endogenous or heterologous speE and speD genes
- produce spermidine in C. glutamicum
- modify C. glutamicum strain ATCC 13032 to increase ornithine production
- overexpress ornithine decarboxylase gene
- delete or disrupt carbamoyl-transferase argF gene
- genetically modify eukaryotic cell for attenuated activity of ornithine carbamyoltransferase
- genetically modify eukaryotic cell for enhanced L-ornithine biosynthesis from α-ketoglutarate
- genetically modify eukaryotic cell for overexpression of at least one gene selected from a group consisting of a gene encoding a L-ornithine

## EXAMPLES

- construct engineered yeast strains

### Strain Construction of Ornithine-Overproducing Strains

- construct plasmids using MOPE strategy and DNA assembler
- amplify promoters, terminators, and genes from genomic DNA
- assemble expression modules using one-step PCR strategy
- co-transform expression modules into yeast strains
- verify assembled pathways by PCR and restriction endonuclease digestion
- replace ARG3 promoter with weaker promoters
- integrate genes into chromosome of yeast background strains

### L-Arginine Leaky Auxotroph Enables L-Ornithine Overproduction

- describe L-ornithine biosynthesis pathway and its regulation
- replace ARG3 promoter with weaker promoters
- verify increased ornithine production
- discuss advantages of fine-tuning ARG3 expression

### Pathway Re-Localization and Subcellular Trafficking Engineering Elevates L-Ornithine Synthesis

- describe L-ornithine biosynthesis pathway and its compartmentalization
- delete CAR2 gene
- overexpress ARG5, 6, ARG7, and ARG8
- verify increased ornithine production
- overexpress ARG2
- verify further increased ornithine production
- discuss anaplerotic role of ARG2
- overexpress GCN4
- verify effect of GCN4 overexpression on ornithine production
- discuss importance of ORT1 in L-ornithine synthesis
- overexpress ORT1
- verify increased ornithine production
- overexpress AGC1
- verify increased ornithine production
- discuss importance of glutamate trafficking
- overexpress Odc1p
- verify effect of Odc1p overexpression on ornithine production
- overexpress GDH1, GDH3, GLN1, and GLT1
- verify effect of overexpressing different glutamate biosynthesis pathways on ornithine production

### ‘Crabtree Negative’ S. cerevisiae Construction Enables Efficient Carbon Channeling to L-Ornithine

- introduce Crabtree effect limitation
- motivate PDA1 overexpression
- describe PDA1 overexpression results
- motivate AOX overexpression
- describe AOX overexpression results
- motivate MTH1-ΔT overexpression
- describe MTH1-ΔT overexpression results
- motivate ODC attenuation
- describe ODC attenuation results
- describe fed-batch strategy results

### Example 5

- introduce urea cycle engineering
- describe arginase regulation
- motivate CAR1 overexpression
- describe CAR1 overexpression results
- introduce putrescine biosynthetic pathway
- describe OAZ1 deletion
- describe OAZ1 deletion procedure
- introduce ornithine decarboxylase overexpression
- describe SPE1 overexpression procedure
- describe TPO1 overexpression
- describe polyamine production measurement
- describe polyamine quantification results
- introduce spermidine biosynthetic pathway
- describe SPE2 and SPE3 overexpression
- describe SPE3 overexpression procedure
- describe SPE2 overexpression procedure
- describe spermidine production measurement
- describe spermidine quantification results
- describe spermidine production via CANSDH and CANSDC
- codon-optimize CANSDH and CANSDC genes
- clone CANSDH and CANSDC into plasmid pYX212
- transform plasmids into strain PUT-B(KanMX)
- describe overexpression of spermine biosynthetic pathway in S. cerevisiae
- amplify SPE1 gene from S. cerevisiae genome
- clone SPE1 into vector pSP-GM1
- amplify SPE2 and SPE3 genes from S. cerevisiae genome
- clone SPE2 and SPE3 into pSP-GM1
- describe increased export of polyamines in S. cerevisiae
- delete AGP2, SAM3, and DUR3 genes
- transform plasmids pSP-GM1-PUT and GO4 into strain AKYDOAZ-Trans2
- describe production of agmatine in S. cerevisiae
- delete CAR1 gene from strain CEN.PK113-11C
- amplify ARG3, ARG1, and ARG4 genes from S. cerevisiae genome
- clone ARG3, ARG1, and ARG4 into pSP-GM1
- amplify speA gene from E. coli
- clone speA into plasmid pIYC04
- co-transform plasmids pSP-GM1-ARGC and pIYC04-speA into strain AKYDCAR1
- describe production of spermidine/spermine in E. coli
- modify E. coli WL3110 to increase ornithine and putrescine production
- overexpress speE and speD genes in E. coli
- describe production of spermidine/spermine in Corynebacterium glutamicum
- modify C. glutamicum strain ATCC 13032 to increase ornithine production

