# DESCRIPTION

## TECHNICAL FIELD

This disclosure relates to methods for determining the sensitivity or resistance of retroviruses such as HIV to therapeutic treatments based on viral protease inhibitors, by the use of yeast, particularly to the use of yeast for determining the resistance or sensitivity of the viral protease to the chemical molecules used in the context of therapeutic protocols. The disclosure also relates to diagnostic kits comprising the elements for implementing the method.

## BACKGROUND

The aetiological agents of AIDS are the human immunodeficiency viruses types 1 and 2. These viruses, which share certain clinical and biological characteristics, have major differences, in particular with regard to the ways in which the host is infected. Infection by HIV-2 is more difficult than by HIV-1 (Ancelle R, O Bletry, A C Baglin, F Brun-Vezinet, M A Rey and P Godeau, 1987, Long incubation period for HIV-2 infection. Lancet. 1:688-9; Marlink R, P Kandki, I Thior, K Travers, G Eisen, T Siby, I Traore, C C Hsieh, M C Dia and E H Gueye. 1994. Reduced rate of disease development after HIV-2 infection as compared to HIV-1. Science. 265:1587-90; Adjorlolo-Johnson G, K M De Cock, E Ekpini, K M Vetter, T Sibailly, K Brattegaard, D Yavo, R Doorly, J P Whitaker and L Kestens. 1994. Prospective comparison of mother-to-child transmission of HIV-1 and HIV-2 in Abidjan, Ivory Coast. JAMA. 272:462-6; Marlink, R. 1996. Lessons from the second AIDS virus, HIV-2. AIDS. 10:689-99.), the plasma viral load of individuals infected by HIV-2 is less high than that in individuals infected by HIV-1 (Andersson S, H Norrgren, Z da Silva, A Biague, S Bamba, S Kwok, C Christopherson, G Biberfeld, and J Albert. 2000. Plasma viral load in HIV-1 and HIV-2 singly and dually infected individuals in Guinea-Bissau, West Africa: significantly lower plasma virus set point in HIV-2 infection than in HIV-1 infection. Arch. Intern. Med. 160:3286-93; Popper S J, A D Sarr, K U Travers, A Gueye-Ndiaye, S Mboup, M E Essex, and P J Kanki. 1999. Lower human immunodeficiency virus (HIV) type 2 viral load reflects the difference in pathogenicity of HIV-1 and HIV-2. J Infect Dis. 180:1116-21.), and the individuals infected by HIV-2 develop the illness more slowly (Vittinghoff E, S Scheer, P O'Malley, G Colfax, S D Holmberg and S P Buchbinder. 1999. Combination antiretroviral therapy and recent declines in AIDS incidence and mortality. J Infect Dis. 179:717-20; Blanco R, Carrasco, L, and Ventoso, I. 2003. Cell killing by HIV-1 protease. J. Biol. Chem. 278:1086-93; Liu H, Krizek J, and Bretscher A. 1992. Construction of a GAL1-regulated yeast cDNA expression library and its application to the identification of genes whose overexpression causes lethality in yeast. Genetics 132:665-673).

HIV-2 was identified for the first time in West Africa in 1986 (Clavel F, D Guetard, F Brun-Vezinet, S Chamaret, M A Rey, M O Santos-Ferreira, A G Laurent, C Dauguet, C Katlama, and C Rouzioux. 1986. Isolation of a new human retrovirus from West African patients with AIDS. Science. 233:343-6). In this region, the prevalence of HIV-2 varies between 1% and 10% (Langley C L, E Benga-De, C W Critchlow, I Ndoye, M D Mbengue-Ly, J Kuypers, G Woto-Gaye, S Mboup, C Bergeron, K K Holmes, and N B Kiviat. 1996. HIV-1, HIV-2, human papillomavirus infection and cervical neoplasia in high-risk African women. AIDS. 10:413-7, Poulsen A G, B Kvinesdal, P Aaby, K Molbak, K Frederiksen, F Dias and E Lauritzen. 1989. Prevalence of and mortality from human immunodeficiency virus type 2 in Bissau, West Africa. Lancet. 1:827-31; Wilkins A, D Ricard, J Todd, H Whittle, F Dias, and A Paulo Da Silva. 1993. The epidemiology of HIV infection in a rural area of Guinea-Bissau. AIDS. 7:1119-22). The majority of these cases of infection by HIV-2, outside West Africa, are found in European countries and especially in Portugal where the individuals infected by VIH-2 represent 13% of the population infected by human immunodeficiency viruses (Soriano V, P Gomes, W Heneine, A Holguin, M Doruana, R Antunes, K Mansinho, W M Switzer, C Araujo, V Shanmugam, H Lourenco, J Gonzalez-Lahoz and F Antunes. 2000. Human immunodeficiency virus type 2 (HIV-2) in Portugal: clinical spectrum, circulating subtypes, virus isolation, and plasma viral load. J Med. Virol. 61:111-6). In France it has been estimated that 1% of the population infected by HIV is infected by the type 2 virus.

In developed countries, the individuals infected by HIV-1 and/or by HIV-2 are treated by chemical therapy, composed of molecules having an inhibiting activity for one or other of the two viral enzymes: Reverse Transcriptase and Protease.

Although this treatment has significantly helped to reduce morbidity and mortality caused by HIV infection (Palella F J, Jr, K M Delaney, A C Moorman, M O Loveless, J Fuhrer, G A Satten, D J Aschman and S D Holmberg. 1998. Declining morbidity and mortality among patients with advanced human immunodeficiency virus infection. HIV Outpatient Study Investigators. N. Engl. J. Med. 338:853-60; Vittinghoff E, S Scheer, P O'Malley, G Colfax, S D Holmberg and S P Buchbinder. 1999. Combination antiretroviral therapy and recent declines in AIDs incidence and mortality. J. Infect. Dis. 179:717-20), some cases of therapeutic failure have been observed.

The possibility of amplifying, from the plasma RNA or cell DNA of the individuals infected by HIV-1 and in therapeutic failure, has made it possible to understand at the molecular level the spontaneous or progressive inefficacy of therapeutic treatments. The determination in particular of the nucleic sequence of the two viral enzymes Reverse Transcriptase and Protease has shown the appearance of a certain number of mutations. The results obtained during studies in vitro, in which a wild viral strain (and therefore sensitive to treatments) carried the said mutations, have clearly demonstrated the implication of these mutations in the resistance of the virus to treatment.

Researchers have therefore done a certain amount of work on these mutations and on the resistances that they generate, in order to orient and guide the choice of therapeutic treatment and to optimize its efficacy.

Unfortunately, the economic strategies of the laboratories have the majority of the time led to a general lack of interest in the scientific community with regard to the treatment of patients infected by HIV-2 (the populations most affected by HIV-2 being mainly those in developing countries) or have led to unsuitable solutions: treatments, tests and analyses that are too expensive, diagnoses that are too lengthy or impossible to implement on site, absence of competent structures in the country concerned, etc.

Thus the results obtained during the various studies carried out on HIV-2 have not been sufficiently consistent to make it possible to formulate a correlation between a particular mutation of the HIV-2 protease, and a resistance phenotype.

In addition, the progression of the illness being slower in individuals infected by HIV-2 than in those infected by HIV-1, the counting of T CD4 cells and the determination of the plasma viral load do not rapidly take account of the emergence of resistant strains in patients under treatment.

There exist at the present time three companies that provide the resistance profile of an HIV strain isolated from an infected patient. Conceptionally the three tests resemble each other and are based on the ability of each protease inhibitor to inhibit the release of an infectious recombinant virus comprising the protease of the virus infecting the patient. The companies are: Viralliance (France), which produces Phenoscript™, Virco (Belgium), which produces Antivirogram™, and Virologic (United States), which produces Phenosense™. In the three cases, performing these tests requires significant logistic organisation, personnel skilled in molecular biology and virology, and expensive infrastructures of the P3 secure laboratory type (it would appear that a complete profile would currently cost between 800 and 1,000 euros per sample). The delay existing between the time when the biological material arrives at the laboratories and the time when the resistance profile is established varies, for each strain of HIV-1, between two and three weeks.

Under these circumstances, putting on the market a reliable rapid test that is simple to implement and inexpensive would be advantageous. Such a test would assist the treating doctors to monitor the appearance of resistant strains in patients infected by retroviruses, in particular HIV 1 or 2, in particular for deprived populations. Moreover, this test could also be used for a “high speed” research for new molecules having inhibiting activity for the retrovirus protease.

## SUMMARY

This disclosure relates a method for determining sensitivity or resistance of isolates of HIV (human immunodeficiency virus) retroviruses to chemical molecules having an inhibiting activity on a viral protease or to therapeutic treatments based on inhibitors of the viral protease, including causing cell lysis of at least one yeast by expression of the retrovirus protease.

This disclosure also relates a method for determining sensitivity or resistance of isolates of HIV (human immunodeficiency virus) retroviruses to chemical molecules having an inhibiting activity on a viral protease or to therapeutic treatments based on inhibitors of the viral protease, including extracting nucleic acids (DNA or RNA) from cells (blood or other) taken from a person or animal infected by the retrovirus; amplifying sequences coding for the protease of the retrovirus to be studied, with or without the or some of the amino acid sequences situated upstream and downstream of the cleavage site of the precursor in which they are situated; recombining the fragments of DNA, the final product of the amplification, and an expression vector allowing the expression of the sequence coding for the protease of the retrovirus to be studied under the control of a known inducible promoter, and co-transformation of the recombined vector with at least one yeast cell whose cell lysis is caused by expression of the retrovirus protease; culturing the co-transformed yeast cell or cells to obtain a sufficient number of transformants to perform the sensitivity or resistance test, and recovery of the transformants issuing from the co-transformed cell, on any suitable medium; incubating the transformants in the presence of an increasing concentration of each viral protease inhibitor to be tested; counting the living cells; and deducing the resistance phenotype.

This disclosure further relates a diagnostic kit including nucleotide primers including, for a first amplification, primers: sense primer: 5′ GAAAGAAGCCCCGCAACTTC3′ and antisense primer: 5′GGGATCCATGTCACTTGCCA3′ and, for a second amplification, primers: sense primer: 5′CGAGGATCCGGAGACACCATACAGGGAGCCACCAACAGCGGCCGCG CCATGCCTCAATTC3′ and antisense primer: 5′GCGGAGCTCGCTTTAGCATTATTTTTAT TGGCTCTACTGCGGCCGCTTAA GATT3′; at least one expression vector; at least one strain of yeast with an auxotrophy marker to permit selection of a transformant expressing a viral protease; and at least one multi-well plate or any other suitable support.

## DETAILED DESCRIPTION

Our methods and kits permit the rapid and low-cost definition of the resistance phenotype of the HIV-2 protease in infected patients, by virtue of the use of yeast.

Our methods can also be implemented to define the resistance phenotype of the HIV-1 protease, or the protease of any other retrovirus.

A scientific article that appeared in 2003 demonstrated that the expression of the HIV-1 protease by the yeast Saccharomyces cerevisiae caused the death of the latter through a still unknown mechanism, the consequence of which was the cell lysis of the yeast in question (Blanco R, Carrasco L and Ventoso I. 2003. Cell killing by HIV-1 protease. J. Biol. Chem. 278:1086-93).

We demonstrated that the same phenomenon occurred when the yeasts expressed the HIV-2 protease. Consequently, by inhibiting the viral enzymatic activity by modification of its catalytic site, we succeeded in preventing the appearance of this cell event.

In addition, Blanco et al (Blanco R, Carrasco L, and Ventoso I. 2003. Cell killing by HIV-1 protease. J. Biol. Chem. 278:1086-93) also showed that the inhibition of the HIV-1 protease by one of the inhibitors used in anti-HIV therapy inhibited the cell death of the yeast caused by the expression of the viral protease. Because of this fact, it is possible to quantitatively measure the sensitivity and resistance of the protease of infected individuals to the various inhibited molecules.

Our methods, therefore, make it possible to determine in the cellular context of the yeast, the sensitivity phenotype of the viral protease of a retrovirus such as HIV-2, to drugs with an inhibiting activity.

In other words, it is possible to determine the sensitivity or resistance of isolates of retroviruses such as HIV (human immunodeficiency virus) to chemical molecules having an inhibiting activity on the viral protease or to therapeutic treatments based on inhibitors of the viral protease, characterized by the use for this purpose of at least one yeast whose cell lysis is caused by the expression of the retrovirus protease.

The method comprises an expression vector, the choice of the cell system, the method of expressing proteases of infected individuals and the test of susceptibility to drugs.

It comprises the following steps:


- - extracting the nucleic acids (DNA or RNA) from cells (blood or
    other) taken from the individual or animal infected by the
    retrovirus, by any suitable means;
  - amplifying the sequences coding for the protease of the retrovirus
    to be studied;
  - recombining the fragments of DNA, the final product of the
    amplification, and an expression vector allowing the expression of
    the sequence coding for the protease of the retrovirus to be studied
    under the control of a known inducible promoter, and
    co-transformation of the recombined vector with at least one yeast
    cell whose cell lysis is caused by the expression of the retrovirus
    protease;
  - culture of the co-transformed yeast cell or cells to obtain a
    sufficient number of transformants to perform the sensitivity or
    resistance test, and recovery of the transformants issuing from the
    co-transformed cell, on any suitable medium;
  - incubation of the transformants in the presence of an increasing
    concentration of each viral protease inhibitor to be tested;
  - counting the living cells; and
  - deducing the resistance phenotype.

The sequences coding the therapeutic targets (reverse transcriptase and protease), as well as the so-called “structure proteins” (matrix, capsid, nucleocapsid) and the Integrase enzymatic activity, are situated within a common polypeptide precursor called Gag-Pol, coded by the gag-pol viral gene (Clavel F, Guyader M, Guetard D, Salle M, Montagnier L, Alizon M. 1986. Molecular cloning and polymorphism of the human immune deficiency virus type 2. Nature, 324:691-5). It is the action of the viral protease that, by hydrolysis of specific peptide bonds referred to as cleavage sites, framing the primary sequences of the various constituents of the precursor, is responsible for the release of these proteins (Oroszlan S and Luftig R B. 1990. Retroviral proteinases. Curr Top Microbiol Immunol. 157:153-85. It has been shown that, for the HIV-1 protease, the amino acid sequences are situated upstream and downstream of the cleavage site fulfilling an important role in the recognition event of the enzyme for its substrate, and therefore determinant for its proteolytic activity (Pettit S C, Simsic J, Loeb D D, Everitt L, Hutchison C A 3rd, Swanstrom R. 1991. Analysis of retroviral protease cleavage sites reveals two types of cleavage sites and the structural requirements of the P1 amino acid. J. Biol. Chem. 266:14539-47, Pettit S C, Moody M D, Wehbie R S, Kaplan A H, Nantermet P V, Klein C A, Swanstrom R. 1994. The p2 domain of human immunodeficiency virus type 1 Gag regulates sequential proteolytic processing and is required to produce fully infectious virions. J Virol. 68:8017-27, Moody M D, Pettit S C, Shao W, Everitt L, Loeb D D, Hutchison C A 3rd, Swanstrom R. 1995. A side chain at position 48 of the human immunodeficiency virus type-i protease flap provides an additional specificity determinant. Virology. 207:475-85, Boross P, Bagossi P, Copeland T D, Oroszlan S, Louis J M, Tozser J. 1999. Effect of substrate residues on the P2′ preference of retroviral proteinases. Eur J Biochem. 264:921-9.).

The sequences of the protease amplified by the method include those coding for the isolated protein or those coding for the protein and comprising all or some of the amino acid sequences situated upstream and downstream of the cleavage site of the protein precursor in which they are situated.

The protocols used in the laboratory with a view to obtaining yeast transformants coding for an exogenous gene generally begin with a first step for obtaining the fragments of DNA coding for the gene of interest, or by gene amplification (PCR technique), or by release of the gene by virtue of the action of the restriction enzymes, which will cut the DNA containing the sequence of interest. This first step can be equivalent in time to half a day's work.

The DNA fragment released is then sub-cloned in an expression vector by the action of the DNA Ligase enzyme (an operation lasting one night) and the product of the reaction is amplified in a bacterium, after its transformation (one day to obtain bacteria having incorporated plasmid DNA, and one and a half days of obtaining and characterising the transformant sought, containing the plasmid coding for the gene of interest).

Moreover, to produce sufficient quantities of the plasmid containing the gene of interest with a view to transformation of the yeast, the bacterial clone obtained in the previous step was amplified (one night) and the plasmids were purified by conventional known methods (one day).

The purified plasmid obtained is then used to transform the selected yeast strain (½ day).

The transformed yeast strain is obtained approximately 4 days after the transformation event.

Consequently, by using conventional protocols making it possible to obtain a sufficient quantity of yeast transformants for a subsequent study of a gene of interest, for example for developing a sensitivity or resistance test, the time that elapses between the preparation of the DNA fragment coding for the gene of interest and the obtaining of the yeast strain expressing it is a minimum of 8 days. Moreover, as disclosed above, several techniques must be used.

The magnitude of these delays and the multiplicity of the techniques used necessarily give rise to high production costs, incompatible with the development of a rapid sensitivity or resistance test that is simple to implement and inexpensive.

We therefore developed of an innovative technology allowing easy management of a large number of samples, for the purpose in particular of testing, rapidly, effectively and at less cost, a large quantity of proteases issuing from different patients under treatment.

Considering the studies on the ability of yeast to repair “broken DNA” (nicked DNA) by the homologous recombination mechanism, it was found that, during this cell event, a DNA molecule is repaired at a precise point in its sequence, by putting in place homologous sequences at the “nicking” site and taken within another DNA molecule. By using this physiological phenomenon, it is possible to introduce a defined sequence within the “nicked” DNA provided that the defined sequence is framed on each side by sequences identical to those situated around the “nicking” site.

The minimum size of the homologous sequences that have to be present in the two DNA molecules for the recombination event to be able to take place is approximately 40 pairs of bases.

This technique very advantageously simplifies obtaining the transformants by reducing in particular the number of manipulations which involve a significant reduction in the experimentation time necessary (approximately half compared with the known protocols) and in the production cost. Use of this technique also enables a large number of samples to be manipulated at the same time.

### EXAMPLE

This example of implementation is used for preference when the protease expressed is the isolated protein. When the protease is expressed with all or some of the amino acid sequences situated upstream and downstream of the cleavage site of the protein precursor that contains it, the primers and nucleotide fragments disclosed above will be modified accordingly.

**1—Preparation and Modification of the Expression Vector:**

The modification enables the ability to sub-clone the gene of the protease of each individual studied and to transform the yeast with the plasma obtained by a simple and rapid procedure (a single-step procedure).

We created a modified version of the vector pRS316-Gal1/10, which comprises the inducible promoter GAL1/10 in position 5′ of the cloning site of the gene to be expressed (Liu H, Krizek J, and Bretscher A. 1992. Construction of a GAL-1 regulated yeast cDNA expression library and its application to the identification of genes whose overexpression causes lethality in yeast. Genetics 132:665-673).

Because of this, the gene is expressed when the cell is transformed with this vector or in the presence of Galactose and the gene is not expressed when the cells transformed are in the presence of glucose as a carbon source.

For the fragment of amplified viral DNA to be able to be inserted by homologous recombination in the expression vector, the vector must be modified by adding to it, just after the sequence of the inducible promoter, a primer 5′ of approximately 40 pairs of bases, followed by a single restriction site (in order to be able to linearize the vector), and a primer 3′ of approximately 40 pairs of bases. Although this modified and linearized vector is a good substrate for the homologous recombination event, the sequence introduced at position 5′ (between the gene and the promoter) must not inhibit the transcription of the genes.

To produce the modified version of the expression vector pRS316-Gal 1/10, we exchanged the BamHI-Sac1 fragment of this site with another DNA fragment also framed by the BamHI-Sac1 restriction sites that contain (from 5′ to 3′):


- - the BamHI restriction site, unique in the vector, followed by
  - the 35 to 45 nucleotides in the HIV-2 sequence situated just
    upstream of the protease, followed by
  - an Not1 restriction site, unique in the vector, followed by
  - the 35 to 45 nucleotides of the HIV-2 sequence situated just
    downstream of the protease, followed by
  - the Sac1 restriction site, unique in the vector.

Modifications to the size of this fragment of approximately 80-90 pairs of bases were carried out in order to optimize the experimental cloning, transformation and expression system. A single modified fragment amongst 10 different ones that were tested was sufficiently optimum for the homologous recombination and for the expression of the viral protease.

The sequence of this fragment is as follows:

The pRS316-Gal1/10 expression vector thus modified is called pRS316-Gal1/10M.

**2—Choice of the Cell System**

Any strain of yeast with auxotrophy markers necessary in order to allow the selection of the transformant expressing the viral protease, and preferably any strain of Saccharomyces cerevisiae.

**3—Sub-cloning of the Proteases of Individuals Affected by the Retrovirus, and Transformation of Yeasts in a Single Step**

The DNA sequence coding the HIV-2 protease is amplified by the PCR technique, twice.

The object of the first is to produce a significant quantity of DNA. This reaction takes place using DNA extracted from lymphocytes of the peripheral blood of infected individuals. The nucleotide primers used are of the type:

The object of the second amplification is to frame the protease of an initialisation codon of the transcription (ATG) and of a termination codon of the transcription (TAA) and to add on each side the sequences of approximately 40 nucleotides that we brought to the vector when it was modified. This PCR reaction takes place on the product of the first PCR with the primers of the type:

The framing of the protease by the initiation and termination codons is that of the isolated protease or of the protein comprising all or some of the amino acid sequences situated upstream and downstream of the cleavage site of the precursor in which it is situated.

The co-transformation of the pRS316-Gal1/10M vector cleaved by the Not1 restriction enzyme with the product of the second PCR makes it possible, through the homogeneous recombination event that took place in the cell, to obtain a transformed yeast cell, carrying the sequence of the HIV-2 protease under the control of the inducible promoter Gal (FIGS. 1 and 2).

**4—The Test**

A sample of peripheral blood is taken from the individual infected with HIV-2. The lymphocytes issuing from this sampling are purified or not, and their DNA extracted by known methods.

This DNA undergoes the two aforementioned PCR reactions in order to create the fragment of DNA, carrying the sequence of the protease with or without the or some of the amino-acid sequences situated upstream and downstream of the cleavage site of the precursor in which it is situated, and compatible with the expression vector in order to cause the phenomenon of homogeneous recombination in the transformed cells.

After purification of the product of the second PCR a yeast strain having a genotype ura3 is co-transformed with the pRS316Gal1/10M linear vector (by its Not1 site).

The transformants potentially producing the protease are recovered on any suitable carrier, such as for example gelose composed of agar, glucose as a source of carbon, and a synthetic environment with a deficiency of uracil.

Approximately 105 cells, issuing from a single transformant, are deposited and distributed in 12 wells with a 96-well plate, and incubated in galactose in the presence of 11 increasing concentrations of each inhibitor to be tested. The twelfth well does not contain any inhibitor.

After 36-48 hours at 30° C., the living cells are counted by a densitometric reading at 600 nm (FIG. 3).

The dose necessary for inhibiting half the cell growth under these conditions, compared with the cell growth in the absence of drugs and in the presence of glucose (IC50), defines the susceptibility or resistance of this specific strain.

The interval of time between the blood sampling and the definition of the resistance profile is only one week.

## Diagnostic Kit for Resistance to Retroviruses

A diagnostic kit for determining the sensitivity or resistance of retrovirus isolates to therapeutic retroviral treatments based on inhibitors of the viral protease, comprising:


- - nucleotide primers, and for example nucleotide primers as previously
    disclosed for implementation of the amplification of DNA coding for
    the retrovirus protease by the PCR technique, namely:
    - for the first amplification primers of the type:

- - - for the second amplification, primers of the type:

- - at least one expression vector and for example the plasmid modified
    and linearized according to the methods of the invention and as
    previously described;
  - at least one strain of yeast with the necessary auxotrophy marker in
    order to permit the selection of the transformant expressing the
    viral protease, and preferably a strain of *Saccharomyces
    cerevisiae*; and
  - at least one multi-well plate or any other suitable support.

Naturally, when the amplification is carried out on the DNA coding for the protease of the retrovirus with all or some of the amino acid sequences situated upstream and downstream of the cleavage site of the protein precursor that contains it, the primers and nucleotide fragments will be modified accordingly.

The principle of the kit, also illustrated in FIG. 4, is as follows:


- - Using the blood of the patient, the gene coding for the viral
    protease is amplified by an RT-PCR reaction with the primers
    described in the method according to the invention and supplied in
    the kit.
  - Amplification of the gene of the viral protease using cell DNA is
    also possible.

The amplified product and the modified and linearized vector (also supplied in the kit) are used for transforming a strain of yeast, also supplied in the kit, in a multi-well plate.


- - In a preferred though in no way limiting fashion, after incubation
    at 30° in a dry oven in a selective medium containing glucose, 2
    microlitres of the cell suspension are transferred to a new plate,
    supplied in the kit, containing increasing concentrations of each
    active principle used in therapy. 300 microlitres of the selected
    medium containing galactose are added to each well, before
    incubating at to 30° C. in a dry oven for 36-48 hours.

At the end of the incubation, the living cells are counted by a densitometric reading at 600 nm. The dose necessary for inhibiting half the cell growth, compared with the cell growth of the strain of yeast not expressing the viral protease, will define the I′IC50 for each inhibitor. The comparison between the I′IC50s obtained and those of a reference wild protease makes it possible to determine any resistance phenotype.

The interval of time between the taking of blood and the definition of the resistance profile is only one week.

