# DESCRIPTION

## CONTINUING APPLICATION DATA

This application claims the benefit of U.S. Provisional Application Ser. No. 61/844,128, filed Jul. 9, 2013, and U.S. Provisional Application Ser. No. 61/996,835, filed May 14, 2014, each of which is incorporated by reference herein.

## BACKGROUND

Autism spectrum disorder (ASD) is a lifelong neurodevelopmental disorder characterized by social deficits, impaired verbal and nonverbal communication and repetitive movements or circumscribed interests (see, for example, American Psychiatric Association (2013) Desk Reference to the Diagnostic Criteria from DSM-5, 5th ed. Washington, D.C.; American Psychiatric Association). About 1 in 68 children are identified with autism spectrum disorder according to estimates from CDC's Autism and Developmental Disabilities Monitoring (ADDM) Network (Centers for Disease Control and Prevention, 2014, MMWR Surveill Summn; 63:1-21). The current process for a clinical diagnosis includes establishing a developmental history and assessments of behavioral characteristics such as speech, language, intellectual abilities, and educational or vocational attainment. Patients can be reliably diagnosed through behavioral testing at age 2 years. However, for a variety of reasons, the average age of diagnosis is 4.5 years. It is increasingly recognized that detection of ASD at the earliest age possible age is important for initiating optimally effective intervention and results in better patient and family outcomes (Payakachat et al., 2012, Expert Rev Pharnmacoecon Outcomes Res; 12:485-503; and Thompson, 2013, J Appl Res Intellect Disabil; 26:81-107). Establishing personalized therapy for children with ASD at the earliest age possible improves outcomes including a higher level of cognitive and social function and improved communication as well as decreased financial and emotional burden on families (Dawson et al., 2010, Pediatrics; 125:e17-23; and Ganz, 2007, Arch Pediatr Adolesc Med; 161:343-349). Thus, the development of a biologically-based blood test to aid in the assessment of risk for a diagnosis of ASD at an early age would facilitate implementing intensive behavioral therapy at the earliest age possible and would be beneficial to patients, families and medical providers.

## SUMMARY OF THE INVENTION

The present invention includes a method for identifying a metabolomic signature characteristic for autism in a human, the method including:

a) assaying a collection of biosamples isolated from autistic subjects for one or a plurality of small molecule metabolites by gas chromatography mass spectrometry (GCMS);

b) assaying a collection of biosamples isolated from non-autistic control subjects for one or a plurality of small molecule metabolites by GCMS;

c) identifying one or a plurality of small molecule metabolites assayed by GCMS that are differentially produced in autistic subjects as compared to non-autistic control subjects;

d) assaying the collection of biosamples isolated from autistic subjects for one or a plurality of small molecule metabolites by one or more untargeted liquid chromatography-high resolution mass spectrometry methodologies (LC/HRMS);

e) assaying the collection of biosamples isolated from non-autistic control subjects for one or a plurality of small molecule metabolites by one or more untargeted LC/HRMS methodologies;

f) identifying one or a plurality of small molecule metabolites assayed by the one or more untargeted LC/HRMS methodologies that are differentially produced in autistic subjects as compared to non-autistic control subjects;

g) combining the plurality of small molecule metabolites identified by step c) and step f) to form a training set of small molecule metabolites; and

h) selecting from the training set a subset of small molecule metabolites with a statistically significant abundance difference in the collection of biosamples isolated form autistic patients as compared to the collection of biosamples isolated from control non-autistic control subjects;

wherein the subset of small molecules of step h) includes a metabolomic signature for autism in a human.

In some aspects of the methods of the present invention, assaying biosamples by one or more untargeted liquid chromatography-high resolution mass spectrometry methodologies (LC/HRMS) includes assaying the biosamples by C8 liquid chromatography coupled to electrospray ionization in positive ion polarity (C8pos), C8 liquid chromatography coupled to electrospray ionization in negative ion polarity (C8neg), hydrophilic interaction liquid chromatography coupled to electrospray ionization in positive ion polarity (HILICpos), and/or hydrophilic interaction liquid chromatography coupled to electrospray ionization in negative ion polarity (HILICneg).

The present invention includes a method for identifying a metabolomic signature characteristic for autism in a human, the method including:

assaying a collection of biosamples isolated from autistic subjects for one or a plurality of small molecule metabolites by two or more methodologies selected from gas chromatography mass spectrometry (GCMS), C8 liquid chromatography coupled to electrospray ionization in positive ion polarity (C8pos), C8 liquid chromatography coupled to electrospray ionization in negative ion polarity (C8neg), hydrophilic interaction liquid chromatography coupled to electrospray ionization in positive ion polarity (HILICpos), and/or hydrophilic interaction liquid chromatography coupled to electrospray ionization in negative ion polarity (HILICneg);

assaying a collection of biosamples isolated from non-autistic control subjects for one or a plurality of small molecule metabolites by the same two or more methodologies selected from GC-MS, C8pos, C8neg, HILICpos, and/or HILICneg; and

identifying for each of the two or methodologies one or a plurality of small molecule metabolites that are differentially produced in autistic subjects as compared to non-autistic control subjects;

combining the plurality of small molecule metabolites that are differentially produced in autistic subjects as compared to non-autistic control subjects identified by each of the two or more methodologies to form a training set of small molecule metabolites; and

selecting from the training set a subset of small molecule metabolites with a statistically significant abundance difference in the biosamples isolated from autistic subjects as compared to the biosamples isolated from control non-autistic control subjects;

wherein the subset of small molecules with a statistically significant abundance difference in the biosamples isolated from autistic subjects as compared to the biosamples isolated from control non-autistic control subjects includes a metabolomic signature for autism.

In some aspects, biosamples are assayed by three or more methodologies selected from gas chromatography mass spectrometry (GCMS), C8 liquid chromatography coupled to electrospray ionization in positive ion polarity (C8pos), C8 liquid chromatography coupled to electrospray ionization in negative ion polarity (C8neg), hydrophilic interaction liquid chromatography coupled to electrospray ionization in positive ion polarity (HILICpos), and/or hydrophilic interaction liquid chromatography coupled to electrospray ionization in negative ion polarity (HILICneg).

In some aspects, biosamples are assayed by four or more methodologies selected from gas chromatography mass spectrometry (GCMS), C8 liquid chromatography coupled to electrospray ionization in positive ion polarity (C8pos), C8 liquid chromatography coupled to electrospray ionization in negative ion polarity (C8neg), hydrophilic interaction liquid chromatography coupled to electrospray ionization in positive ion polarity (HILICpos), and/or hydrophilic interaction liquid chromatography coupled to electrospray ionization in negative ion polarity (HILICneg).

In some aspects, biosamples are assayed by gas chromatography mass spectrometry (GCMS), C8 liquid chromatography coupled to electrospray ionization in positive ion polarity (C8pos), C8 liquid chromatography coupled to electrospray ionization in negative ion polarity (C8neg), hydrophilic interaction liquid chromatography coupled to electrospray ionization in positive ion polarity (HILICpos), and hydrophilic interaction liquid chromatography coupled to electrospray ionization in negative ion polarity (HILICneg).

The present invention includes a method for identifying a metabolomic signature characteristic for autism in a human, the method including:

a) assaying a collection of biosamples isolated from autistic subjects for one or a plurality of small molecule metabolites by gas chromatography mass spectrometry (GCMS);

b) assaying a collection of biosamples isolated from non-autistic control subjects for one or a plurality of small molecule metabolites by GCMS;

c) identifying one or a plurality of small molecule metabolites assayed by GCMS that are

differentially produced in autistic subjects as compared to non-autistic control subjects;

d) assaying the collection of biosamples isolated from autistic subjects for one or a plurality of small molecule metabolites by C8 liquid chromatography coupled to electrospray ionization in positive ion polarity (C8pos);

e) assaying the collection of biosamples isolated from non-autistic control subjects for one or a plurality of small molecule metabolites by C8pos;

f) identifying one or a plurality of small molecule metabolites assayed by C8pos that are differentially produced in autistic subjects as compared to non-autistic control subjects;

g) assaying the collection of biosamples isolated from autistic subjects for one or a plurality of small molecule metabolites by C8 liquid chromatography coupled to electrospray ionization in negative ion polarity (C8neg);

h) assaying the collection of biosamples isolated from non-autistic control subjects for one or a plurality of small molecule metabolites by C8neg;

i) identifying one or a plurality of small molecule metabolites assayed by C8neg that are differentially produced in autistic subjects as compared to non-autistic control subjects;

j) assaying the collection of biosamples isolated from autistic subjects for one or a plurality of small molecule metabolites by hydrophilic interaction liquid chromatography coupled to electrospray ionization in positive ion polarity (HILICpos);

k) assaying the collection of biosamples isolated from non-autistic control subjects for one or a plurality of small molecule metabolites by HILICpos;

l) identifying one or a plurality of small molecule metabolites assayed by HILICpos that are differentially produced in autistic subjects as compared to non-autistic control subjects;

m) assaying the collection of biosamples isolated from autistic subjects for one or a plurality of small molecule metabolites by hydrophilic interaction liquid chromatography coupled to electrospray ionization in negative ion polarity (HILICneg);

n) assaying the collection of biosamples isolated from non-autistic control subjects for one or a plurality of small molecule metabolites by HILICneg;

o) identifying one or a plurality of small molecule metabolites assayed by HILICneg that are differentially produced in autistic subjects as compared to non-autistic control subjects;

p) combining the plurality of small molecule metabolites identified by step c), step f), step I), step l), and step o) to form a training set of small molecule metabolites; and

q) selecting from the training set a subset of small molecule metabolites with a statistically significant abundance difference in the collection of biosamples isolated form autistic patients as compared to the collection of biosamples isolated from control non-autistic control subjects;

wherein the subset of small molecules of step q) includes a metabolomic signature for autism in a human.

In some aspects, the training set a subset of small molecule metabolites with a statistically significant abundance difference in the collection of biosamples isolated from autistic patients as compared to the collection of biosamples isolated from control non-autistic control subjects are selecting by univariate analysis, multivariate analysis, machine learning analysis, support vector machine analysis (SVM), and/or partial least squares analysis (PLS).

With any of the methods of the present invention, a small molecule metabolite may have a molecular weight of from about 10 Daltons to about 3000 Daltons.

With any of the methods of the present invention, a biosample may be cerebrospinal fluid, brain tissue, amniotic fluid, blood, serum, plasma, amniotic fluid, or urine.

With any of the methods of the present invention, the biosample may be plasma.

With any of the methods of the present invention, the metabolomic signature for autism includes one or more of the 179 metabolites listed in Table 6.

With any of the methods of the present invention, the metabolomic signature for autism includes at least 40 of the metabolites listed in Table 6.

With any of the methods of the present invention, the metabolomic signature for autism includes about 80 to about 160 of the metabolites listed in Table 6.

With any of the methods of the present invention, the metabolomic signature for autism includes any one or more of the metabolites, any two or more metabolites, any three or more metabolites, any four or more metabolites, any five or more metabolites, any six or more metabolites, any seven or more metabolites, any eight or more metabolites, any nine or more metabolites, any ten or more metabolites, any eleven or more metabolites, any twelve or more metabolites, any thirteen or more metabolites, any fourteen or more metabolites, any fifteen or more metabolites, any sixteen or more metabolites, any seventeen or more metabolites, any eighteen or more metabolites, any nineteen or more metabolites, any twenty or more metabolites, or twenty one metabolites of homocitrulline, 2-hydroxyvaleric acid, cystine, aspartic acid, isoleucine, creatinine, serine, 4-hydroxyphenyllactic acid, citric acid, glutamic acid, lactic acid, DHEA sulfate, glutaric acid, 5-hydroxynorvaline, heptadecanoic acid, 5-aminovaleric acid lactam, succinic acid, myristic acid, 2-hydroxyvaleric acid, methylhexadecanoic acid, and/or 3-aminoisobutyric acid.

With any of the methods of the present invention, the metabolomic signature for autism includes any one or more of, any one or more of the metabolites, any two or more metabolites, any three or more metabolites, any four or more metabolites, any five or more metabolites, any six or more metabolites, any seven or more metabolites, any eight or more metabolites, any nine or more metabolites, any ten or more metabolites, any eleven or more metabolites, any twelve or more metabolites, any thirteen or more metabolites, any fourteen or more metabolites, any fifteen or more metabolites, any sixteen or more metabolites, any seventeen or more metabolites, any eighteen or more metabolites, any nineteen or more metabolites, any twenty or more metabolites, or twenty one or more metabolites, any twenty two or more metabolites, any twenty three or more metabolites, any twenty four or more metabolites, any twenty five or more metabolites, and/or twenty six metabolites of 2-aminooctanoic acid, acesulfame, ADMA, choline, CMPF, cysteine, cystinc, DHEA sulfate (DHEAS), glycine, glycocholic acid, hypoxanthine, indoleacrylic acid, indoxyl sulfate, LysoPC(16:1(9Z)), LysoPE(0:0/18:1(9Z)), LysoPE(22:6(4Z,7Z,10Z,13Z,16Z19Z)/0:0), LysoPE(22:6(4Z,7Z,10Z,13Z,16Z,19Z)/0:0), methionine, p-cresol sulfate, phenylalanine, phenyllactic acid, proline, serotonin, tryptophan, uric acid, and/or valine.

With any of the methods of the present invention, the metabolomic signature for autism includes any one or more of, any one or more of the metabolites, any two or more metabolites, any three or more metabolites, any four or more metabolites, any five or more metabolites, any six or more metabolites, any seven or more metabolites, any eight or more metabolites, any nine or more metabolites, any ten or more metabolites, any eleven or more metabolites, any twelve or more metabolites, any thirteen or more metabolites, any fourteen or more metabolites, any fifteen or more metabolites, any sixteen or more metabolites, any seventeen or more metabolites, any eighteen or more metabolites, any nineteen or more metabolites, any twenty or more metabolites, or twenty one or more metabolites, any twenty two or more metabolites, any twenty three or more metabolites, any twenty four or more metabolites, any twenty five or more metabolites, any twenty six metabolites or more metabolites, any twenty seven metabolites or more metabolites, any twenty eight metabolites or more metabolites, and/or twenty nine metabolites of homocitrulline, glutaric acid, saccharopine, 5-aminovaleric acid, lactate, succinate, isocitrate, DHEAS, DHA, androsterone sulfate, 27-norcholesterol, Lyso PE, PE, long chain Fas, LysoPC, aspartate, glutamate, acetylornithine, valine, isoleucine, ketoleucine, serine, homocysteic acid, valine, cystine, hydroxyacetone, phosphohydroxypyruvate, indole-3-lactate, and/or 3-amino isobutyrate.

With any of the methods of the present invention, a metabolic signature for autism may demonstrate decreased homocitrulline, increased glutaric acid, increased saccharopine, increased 5-aminovaleric acid, increased lactate, increased succinate, decreased isocitrate, increased DHEAS, increased DHA, increased androsterone sulfate, increased 27-norcholesterol, decreased Lyso PE, decreased PE, decreased long chain Fas, decreased LysoPC, increased asparate, increased glutamate, increased acetylornithine, decreased valine, decreased isoleucine, increased ketoleucine, increased serine, decreased homocysteic acid, decreased valine, decreased cystine, increased hydroxyacetone, increased phosphohydroxypyruvate, decreased indole-3-lactate, and/or increased 3-amino isobutyrate.

With any of the methods of the present invention, the metabolomic signature for autism includes homocitrulline.

With any of the methods of the present invention, the metabolomic signature for autism includes decreased homocitrulline.

Any of the methods of the present invention may further include a step of determining a chemical identity for one or a plurality of the cellular metabolites. In some aspects, the chemical identity of one or a plurality of the cellular metabolites is determined using molecular exact mass for the metabolite or mass spectrometry fragmentation patterns of the metabolites.

Any of the methods of the present invention may further include determining a ratio of two or more small molecule metabolites.

Any of the methods of the present invention may further include a combination assessment of the relative abundance of two or more small molecule metabolites.

With any of the methods of the present invention, the biosamples from autistic subjects autistic subjects are obtained from a phenotypic subpopulation of autism subjects and wherein the metabolomic signature for autism includes a metabolomic signature for the phenotypic subpopulation of autism subjects. In some aspects the phenotypic subpopulation of autism subjects includes low function autism (LFA) or high function autism (HFA).

The present invention includes a metabolomic signature for autism produced according to a method as described above.

The present invention includes a metabolomic signature for autism, the metabolomic signature including any one or more features, two or more features, three or more features, four or more features, five or more features, six or more features, seven or more features, eight or more features, nine or more features, ten or more features, eleven or more features, twelve or more features, thirteen or more features, fourteen or more features, fifteen or more features, sixteen or more features, seventeen or more features, eighteen or more features, nineteen or more features, twenty or more features, or twenty one features of homocitrulline, 2-hydroxyvaleric acid, cystine, aspartic acid, isoleucine, creatinine, serine, 4-hydroxyphenyllactic acid, citric acid, glutamic acid, lactic acid, DHEA sulfate, glutaric acid, 5-hydroxynorvaline, heptadecanoic acid, 5-aminovaleric acid lactam, succinic acid, myristic acid, 2-hydroxyvaleric acid, methylhexadecanoic acid, and/or 3-aminoisobutyric acid.

The present invention includes a metabolomic signature for autism, the metabolomic signature including any one or more of, any one or more of the metabolites, any two or more metabolites, any three or more metabolites, any four or more metabolites, any five or more metabolites, any six or more metabolites, any seven or more metabolites, any eight or more metabolites, any nine or more metabolites, any ten or more metabolites, any eleven or more metabolites, any twelve or more metabolites, any thirteen or more metabolites, any fourteen or more metabolites, any fifteen or more metabolites, any sixteen or more metabolites, any seventeen or more metabolites, any eighteen or more metabolites, any nineteen or more metabolites, any twenty or more metabolites, or twenty one or more metabolites, any twenty two or more metabolites, any twenty three or more metabolites, any twenty four or more metabolites, any twenty five or more metabolites, and/or twenty six metabolites of 2-aminooctanoic acid, acesulfame, ADMA, choline, CMPF, cysteine, cystine, DHEA sulfate (DHEAS), glycine, glycocholic acid, hypoxanthine, indoleacrylic acid, indoxyl sulfate, LysoPC(16:1(9Z)), LysoPE(0:0/18:1(9Z)). LysoPE(22:6(4Z,7Z,10Z,13Z,16Z,19Z)/0:0), LysoPE(22:6(4Z,7Z,10Z,13Z,16Z,19Z)/0:0), methionine, p-cresol sulfate, phenylalanine, phenyllactic acid, proline, serotonin, tryptophan, uric acid, and/or valine.

The present invention includes a metabolomic signature for autism, the metabolomic signature including one or more of the features set forth in Table 6.

The present invention includes a metabolomic signature for autism including at least 40 of the metabolites listed in Table 6.

The present invention includes a metabolomic signature for autism including about 80 to about 160 of the metabolites listed in Table 6.

In some aspects of a metabolomic signature for autism of the present invention, a signature may include homocitrulline. In some aspects of the metabolic signature, homocitrulline is decreased.

In some aspects of a metabolomic signature for autism of the present invention, the metabolic signature is indicative of high functioning autism (HFA) and/or low functioning autism (LFA).

In some aspects of a metabolomic signature for autism of the present invention, the metabolomic signature for autism includes any one or more of, any one or more of the metabolites, any two or more metabolites, any three or more metabolites, any four or more metabolites, any five or more metabolites, any six or more metabolites, any seven or more metabolites, any eight or more metabolites, any nine or more metabolites, any ten or more metabolites, any eleven or more metabolites, any twelve or more metabolites, any thirteen or more metabolites, any fourteen or more metabolites, any fifteen or more metabolites, any sixteen or more metabolites, any seventeen or more metabolites, any eighteen or more metabolites, any nineteen or more metabolites, any twenty or more metabolites, or twenty one or more metabolites, any twenty two or more metabolites, any twenty three or more metabolites, any twenty four or more metabolites, any twenty five or more metabolites, any twenty six metabolites or more metabolites, any twenty seven metabolites or more metabolites, any twenty eight metabolites or more metabolites, and/or twenty nine metabolites of homocitrulline, glutaric acid, saccharopine, 5-aminovaleric acid, lactate, succinate, isocitrate, DHEAS, DHA, androsterone sulfate, 27-norcholesterol, Lyso PE, PE, long chain Fas, LysoPC, asparate, glutamate, acetylornithine, valine, isoleucine, ketoleucine, serine, homocysteic acid, valine, cystine, hydroxyacetone, phosphohydroxypyruvate, indole-3-lactate, and/or 3-amino isobutyrate.

In some aspects of a metabolomic signature for autism of the present invention, the metabolomic signature for autism includes decreased homocitrulline, increased glutaric acid, increased saccharopine, increased 5-aminovaleric acid, increased lactate, increased succinate, decreased isocitrate, increased DHEAS, increased DHA, increased androsterone sulfate, increased 27-norcholesterol, decreased Lyso PE, decreased PE, decreased long chain Fas, decreased LysoPC, increased asparate, increased glutamate, increased acetylornithine, decreased valine, decreased isoleucine, increased ketoleucine, increased serine, decreased homocysteic acid, decreased valine, decreased cystine, increased hydroxyacetone, increased phosphohydroxypyruvate, decreased indole-3-lactate, and/or increased 3-amino isobutyrate.

The present invention includes a method for assessing a subjects risk for autism, the method including:

assaying a biosample from the subject for one or a plurality of small molecule metabolites by one or more methodologies selected from gas chromatography mass spectrometry (GCMS), C8 liquid chromatography coupled to electrospray ionization in positive ion polarity (C8pos), C8 liquid chromatography coupled to electrospray ionization in negative ion polarity (C8neg), hydrophilic interaction liquid chromatography coupled to electrospray ionization in positive ion polarity (HILICpos), and/or hydrophilic interaction liquid chromatography coupled to electrospray ionization in negative ion polarity (HILICneg);

quantifying the amount of one or more of the 179 small molecule metabolites listed in Table 6;

wherein a statistically significant abundance difference as compared to non-autistic controls in one or more of the 179 small molecule metabolites listed in Table 6 indicates an increased risk of autism.

The present invention includes a method for assessing a subjects risk for autism, the method including assaying a biosample from the subject for one or a plurality of small molecule metabolites; and quantifying the amount of one or more of the 179 small molecule metabolites listed in Table 6; wherein a statistically significant abundance difference as compared to non-autistic controls in one or more of the 179 small molecule metabolites listed in Table 6 indicates an increased risk of autism. In some aspects, the biosample is assayed by one or more methodologies selected from gas chromatography mass spectrometry (GCMS), C8 liquid chromatography coupled to electrospray ionization in positive ion polarity (C8pos), C8 liquid chromatography coupled to electrospray ionization in negative ion polarity (C8neg), hydrophilic interaction liquid chromatography coupled to electrospray ionization in positive ion polarity (HILICpos), and/or hydrophilic interaction liquid chromatography coupled to electrospray ionization in negative ion polarity (HILICneg).

In some aspects of a method for assessing a subjects risk for autism of the present invention, a statistically significant abundance difference as compared to non-autistic controls of at least 40 of the metabolites listed in Table 6 indicates an increased risk of autism.

In some aspects of a method for assessing a subjects risk for autism of the present invention, a statistically significant abundance difference as compared to non-autistic controls of about 80 to about 160 of the metabolites listed in Table 6 indicates an increased risk of autism.

In some aspects of a method for assessing a subjects risk for autism of the present invention, a statistically significant abundance difference as compared to non-autistic controls of any one or more any one or more of the metabolites, any two or more metabolites, any three or more metabolites, any four or more metabolites, any five or more metabolites, any six or more metabolites, any seven or more metabolites, any eight or more metabolites, any nine or more metabolites, any ten or more metabolites, any eleven or more metabolites, any twelve or more metabolites, any thirteen or more metabolites, any fourteen or more metabolites, any fifteen or more metabolites, any sixteen or more metabolites, any seventeen or more metabolites, any eighteen or more metabolites, any nineteen or more metabolites, any twenty or more metabolites, or twenty one metabolites of homocitrulline, 2-hydroxyvaleric acid, cystine, aspartic acid, isoleucine, creatinine, serine, 4-hydroxyphenyllactic acid, citric acid, glutamic acid, lactic acid, DHEA sulfate, glutaric acid, 5-hydroxynorvaline, heptadecanoic acid, 5-aminovaleric acid lactam, succinic acid, myristic acid, 2-hydroxyvaleric acid, methylhexadecanoic acid, and/or 3-aminoisobutyric acid indicates an increased risk of autism.

In some aspects of a method for assessing a subjects risk for autism of the present invention, a statistically significant abundance difference as compared to non-autistic controls of any one or more of, any one or more of the metabolites, any two or more metabolites, any three or more metabolites, any four or more metabolites, any five or more metabolites, any six or more metabolites, any seven or more metabolites, any eight or more metabolites, any nine or more metabolites, any ten or more metabolites, any eleven or more metabolites, any twelve or more metabolites, any thirteen or more metabolites, any fourteen or more metabolites, any fifteen or more metabolites, any sixteen or more metabolites, any seventeen or more metabolites, any eighteen or more metabolites, any nineteen or more metabolites, any twenty or more metabolites, or twenty one or more metabolites, any twenty two or more metabolites, any twenty three or more metabolites, any twenty four or more metabolites, any twenty five or more metabolites, and/or twenty six metabolites of 2-aminooctanoic acid, acesulfarne, ADMA, choline, CMPF, cysteine, cystine, DHEA sulfate (DHEAS), glycine, glycocholic acid, hypoxanthine, indoleacrylic acid, indoxyl sulfate, LysoPC(16:1(9Z)), LysoPE(0:0/18:1 (9Z)), LysoPE(22:6(4Z,7Z,10Z,13Z,16Z,19Z)/0:0), LysoPE(22:6(4Z,7Z,10Z,13Z,16Z,19Z)/0:0), methionine, p-cresol sulfate, phenylalanine, phenyllactic acid, proline, serotonin, tryptophan, uric acid, and/or valine indicates an increased risk of autism.

In some aspects of a method for assessing a subjects risk for autism of the present invention, a statistically significant abundance difference as compared to non-autistic controls of any one or more of, any one or more of the metabolites, any two or more metabolites, any three or more metabolites, any four or more metabolites, any five or more metabolites, any six or more metabolites, any seven or more metabolites, any eight or more metabolites, any nine or more metabolites, any ten or more metabolites, any eleven or more metabolites, any twelve or more metabolites, any thirteen or more metabolites, any fourteen or more metabolites, any fifteen or more metabolites, any sixteen or more metabolites, any seventeen or more metabolites, any eighteen or more metabolites, any nineteen or more metabolites, any twenty or more metabolites, or twenty one or more metabolites, any twenty two or more metabolites, any twenty three or more metabolites, any twenty four or more metabolites, any twenty five or more metabolites, any twenty six metabolites or more metabolites, any twenty seven metabolites or more metabolites, any twenty eight metabolites or more metabolites, and/or twenty nine metabolites of homocitrulline, glutaric acid, saccharopine, 5-aminovaleric acid, lactate, succinate, isocitrate, DHEAS, DHA, androsterone sulfate, 27-norcholesterol, Lyso PE, PE, long chain Fas, LysoPC, asparate, glutamate, acetylornithine, valine, isoleucine, ketoleucine, serine, homocysteic acid, valine, cystine, hydroxyacetone, phosphohydroxypyruvate, indole-3-lactate, and/or 3-amino isobutyrate indicates an increased risk of autism.

In some aspects of a method for assessing a subjects risk for autism of the present invention, decreased homocitrulline, increased glutaric acid, increased saccharopine, increased 5-aminovaleric acid, increased lactate, increased succinate, decreased isocitrate, increased DHEAS, increased DHA, increased androsterone sulfate, increased 27-norcholesterol, decreased Lyso PE, decreased PE, decreased long chain Fas, decreased LysoPC, increased asparate, increased glutamate, increased acetylornithine, decreased valine, decreased isoleucine, increased ketoleucine, increased serine, decreased homocysteic acid, decreased valine, decreased cystine, increased hydroxyacetone, increased phosphohydroxypyruvate, decreased indole-3-lactate, and/or increased 3-amino isobutyrate is indicative of autism.

In some aspects of a method for assessing a subjects risk for autism of the present invention, a statistically significant abundance difference as compared to non-autistic controls of homocitrulline indicates an increased risk of autism.

In some aspects of a method for assessing a subjects risk for autism of the present invention, the method further includes determining a ratio of two or more small molecule metabolites.

In some aspects of a method for assessing a subjects risk for autism of the present invention, the method further includes a combination assessment of the relative abundance of two or more small molecule metabolites.

In some aspects of a method for assessing a subjects risk for autism of the present invention, a biosample may be cerebrospinal fluid, brain tissue, amniotic fluid, blood, serum, plasma, anmiotic fluid, or urine.

In some aspects of a method for assessing a subjects risk for autism of the present invention, a biosample may be plasma.

In some aspects of a method for assessing a subjects risk for autism of the present invention, the subject is less than two years of age.

In some aspects of a method for assessing a subjects risk for autism of the present invention, the metabolic signature is indicative of a phenotypic subpopulation of autism subjects.

In some aspects of a method for assessing a subjects risk for autism of the present invention, the metabolic signature is indicative of high functioning autism (HFA) and/or low functioning autism (LFA).

The terms used in the specification generally have their ordinary meanings in the art, within the context of the invention, and in the specific context where each term is used. Some terms have been more specifically defined below to provide additional guidance to the practitioner regarding the description of the invention.

The term “and/or” means one or all of the listed elements or a combination of any two or more of the listed elements.

The words “preferred” and “preferably” refer to embodiments of the invention that may afford certain benefits, under certain circumstances. However, other embodiments may also be preferred, under the same or other circumstances. Furthermore, the recitation of one or more preferred embodiments does not imply that other embodiments are not useful, and is not intended to exclude other embodiments from the scope of the invention.

The terms “comprises” and variations thereof do not have a limiting meaning where these terms appear in the description and claims.

Unless otherwise specified, “a,” “an,” “the,” and “at least one” are used interchangeably and mean one or more than one.

Also herein, the recitations of numerical ranges by endpoints include all numbers subsumed within that range (e.g., 1 to 5 includes 1, 1.5, 2, 2.75, 3, 3.80, 4, 5, etc.).

For any method disclosed herein that includes discrete steps, the steps may be conducted in any feasible order. And, as appropriate, any combination of two or more steps may be conducted simultaneously.

The above summary of the present invention is not intended to describe each disclosed embodiment or every implementation of the present invention. The description that follows more particularly exemplifies illustrative embodiments. In several places throughout the application, guidance is provided through lists of examples, which examples can be used in various combinations. In each instance, the recited list serves only as a representative group and should not be interpreted as an exclusive list.

## DETAILED DESCRIPTION OF ILLUSTRATIVE EMBODIMENTS

The present invention includes methods for the identification of metabolic biomarkers characteristic of autism spectrum disorder (ASD) in humans. A metabolomics-based approach was used to identify a plurality of metabolic biomarkers that are differentially produced in autistic patients relative to typically developing individuals. Samples are analyzed using multiple high resolution mass spectrometry-based techniques to orthogonally measure a broad range of small molecular weight metabolites differentially produced in autistic patient samples versus non-autistic control samples. These individual metabolites or a panel of such metabolites serve as metabolic signatures of autism. Such metabolic signatures are used in diagnostic methods to accurately identify individuals with autism spectrum disorder (ASD).

As there is not one universal chromatographic mass spectrometric technique capable of detecting all of the metabolites in a biosample, with the present invention multiple high resolution mass spectrometry-based techniques are used, each independently measuring a broad range of small molecular weight metabolites differentially produced in autistic patient samples versus non-autistic control samples. Any of a number of known high resolution mass spectrometry-based techniques may be used to independently measure a broad range of small molecular weight metabolites differentially produced in autistic patient samples versus non-autistic control samples. For example, samples may be assayed using at least two, at least three, at least four, at least five, or at least six different high resolution mass spectrometry-based techniques.

In some aspect, any combination of one or more gas chromatography-mass spectrometry (GC-MS) methodologies and/or one or more liquid chromatography-high resolution mass spectrometry (LC-HRMS) methodologies may be used. In some aspects, a GC-MS method may be targeted. In some aspects, a LC-HRMS method may be untargeted. Subsequently, in some embodiments, tandem mass spectrometry (MS-MS) methods may be employed for the structural confirmation of metabolites. LC-HRMS methodologies may include C8 chromatography and/or Hydrophilic Interaction Liquid Chromatography (HILIC) chromatography. Either of C8 chromatography or HILIC chromatography may be coupled to electrospray ionization in both positive and negative ion polarities, resulting in multiple data acquisitions per sample.

In some embodiments, samples may be analyzed using five different chromatographic-mass spectrometry-based methods, GC-MS and four untargeted LC-HRMS methods. The four untargeted LC-HRMS methods may include C8 chromatography and HILIC chromatography, both coupled to electrospray ionization in both positive and negative ion polarities, resulting in 4 separate data acquisitions per sample, to orthogonally measure a broad range of metabolites in blood plasma. Univariate, multivariate, and machine learning methods may be used to develop models in which the importance of features used for the determination of biomarkers to distinguish samples from the children with ASD from samples from the TD children were ranked. A training set of samples may be used for univariate and multivariate analysis to build the classification models. Additional samples may be used as an independent validation test set.

Statistical models were created using different combinations of the significant mass features. In one embodiment, these models generated a set of 179 features that were altered in abundance in the ASD samples and a subset of these features could properly classify the ASD and TD samples in the independent validation set with a maximum accuracy of 81%.

As used herein, a “training set” is a set of data used in various areas of information science to discover potentially predictive relationships. Training sets are used in artificial intelligence, machine learning, genetic programming, intelligent systems, and statistics. In all these fields, a training set has much the same role and is often used in conjunction with a test set.

As used herein, a “test set” is a set of data used in various areas of information science to assess the strength and utility of a predictive relationship. Test sets are used in artificial intelligence, machine learning, genetic programming, intelligent systems, and statistics. In all these fields, a test set has much the same role.

Data collected during analysis may be quantified for one or more than one metabolite. Quantifying data may be obtained by measuring the levels or intensities of specific metabolites present in a sample. The quantifying data may be compared to corresponding data from one or more than one reference sample. A “reference sample” is any suitable reference sample for the particular disease state. For example, a reference sample may be a sample from a control individual, i.e., a person not suffering from ASD with or without a family history of ASD (also referred to herein as a “typically developing individual,” or “normal” counterpart. A reference sample may also be a sample obtained from a patient clinically diagnosed with ASD. As would be understood by a person of skill in the art, more than one reference sample may be used for comparison to the quantifying data.

As used herein, the term “metabolite” or “cellular metabolite” refers to specific small molecules, the levels or intensities of which are measured in a sample, and that may be used as markers to diagnose a disease state. As used herein, the term “feature” refers to a single small metabolite, or a fragment of a metabolite. Metabolites include, but are not limited to, sugars, organic acids, amino acids, fatty acids, hormones, vitamins, acids, bases, lipids, glycosides, amines, oximes, esters, dipeptides, tripeptides, cholesterols, oxysterols, glycerols, steroids, oligopeptides (less than about 100 amino acids in length), as well as ionic fragments thereof. In some aspects, metabolites are less than about 3000 Daltons in molecular weight. In some aspects, metabolites are less than about 1500 Daltons in molecular weight. In some aspects, metabolites are from about 10 to about 3000 Daltons in molecular weight. In some aspects, metabolites are from about 50 to about 3000 Daltons in molecular weight. In some aspects, metabolites are from about 10 Daltons to about 1500 Dalton in molecular weight. In some aspects, metabolites are from about 50 Daltons to about 1500 Dalton in molecular weight.

As used herein, the term “biomarker” or “metabolic biomarker” refers to metabolites that exhibit statistically significant alterations between diseased and controls.

The terms “metabolic signature” and “biomarker profile” as used herein refer to one or a plurality of metabolites identified by the inventive methods. A metabolic signature of autism is a population of cellular metabolites that are significantly altered in autistic patient biofluids, providing a molecular fingerprint of autism spectral disorders. Such a metabolic signature of autism may be used to diagnose autism in an individual.

The invention provides methods for identifying metabolites in biofluids of individuals with autism. Said metabolites are found using the methods described herein to be differentially secreted in patient tissues or biofluids. These metabolites may be found in either greater or lesser amounts in autistic as compared to non-autistic individuals. Thus, the present invention includes a blood test for the diagnosis of ASD. ASD is a lifelong neurodevelopmental disorder characterized by deficits in social interaction, communication and repetitive or stereotypical behaviors which has recently seen a dramatic increase in prevalence, reaching an estimate of 1 in 50 school-aged children. Earlier diagnosis and treatment is important for optimal therapeutic outcomes. The blood test of the present invention can be performed at an earlier age will have a dramatic impact on earlier therapeutic interventions and better outcomes for ASD children.

Metabolic biomarkers may be identified by their unique molecular mass and consistency, thus the actual identity of the underlying compound that corresponds to the biomarker is not required for the practice of this invention. Biomarkers may be identified using, for example, Mass Spectrometry such as MALDI/TOF (time-of-flight), SELDL/TOF, liquid chromatography-mass spectrometry (LC-MS), gas chromatography-mass spectrometry (GC-MS), high performance liquid chromatography-mass spectrometry (HPLC-MS), capillary electrophoresis-mass spectrometry, nuclear magnetic resonance spectrometry, tandem mass spectrometry (e.g., MS/MS, MS/MS/MS, ESI-MS/MS etc.), secondary ion mass spectrometry (SIMS), and/or ion mobility spectrometry (e.g. GC-IMS, IMS-MS, LC-IMS, LC-IMS-MS etc.). Alternatively, certain biomarkers can be identified by, for example, gene expression analysis, including real-time PCR, RT-PCR, Northern analysis, and in situ hybridization.

In some aspects, a method for identifying a metabolomic signature characteristic for autism in a human may include one or more of the steps:

assaying a collection of biosamples isolated from autistic subjects for one or a plurality of small molecule metabolites by two or more methodologies selected from gas chromatography mass spectrometry (GCMS), C8 liquid chromatography coupled to electrospray ionization in positive ion polarity (C8pos), C8 liquid chromatography coupled to electrospray ionization in negative ion polarity (C8neg), hydrophilic interaction liquid chromatography coupled to electrospray ionization in positive ion polarity (HILICpos), and/or hydrophilic interaction liquid chromatography coupled to electrospray ionization in negative ion polarity (HILICneg);

assaying a collection of biosamples isolated from non-autistic control subjects for one or a plurality of small molecule metabolites by the same two or more methodologies selected from GC-MS, C8pos, C8neg, HILICpos, and/or HILICneg;

identifying for each of the two or methodologies one or a plurality of small molecule metabolites that are differentially produced in autistic subjects as compared to non-autistic control subjects;

combining the plurality of small molecule metabolites that are differentially produced in autistic subjects as compared to non-autistic control subjects identified by each of the two or more methodologies to form a training set of small molecule metabolites; and

selecting from the training set a subset of small molecule metabolites with a statistically significant abundance difference in the biosamples isolated from autistic subjects as compared to the biosamples isolated from control non-autistic control subjects;

wherein the subset of small molecules with a statistically significant abundance difference in the biosamples isolated from autistic subjects as compared to the biosamples isolated from control non-autistic control subjects comprises a metabolomic signature for autism.

In some aspects, biosamples are assayed by three or more, four or more, or all five of the methodologies of gas chromatography mass spectrometry (GCMS), C8 liquid chromatography coupled to electrospray ionization in positive ion polarity (C8pos), C8 liquid chromatography coupled to electrospray ionization in negative ion polarity (C8neg), hydrophilic interaction liquid chromatography coupled to electrospray ionization in positive ion polarity (HILICpos), and hydrophilic interaction liquid chromatography coupled to electrospray ionization in negative ion polarity (HILICneg).

In some aspects, a method for identifying a metabolomic signature characteristic for autism in a human may include one or more of the steps:

a) assaying a collection of biosamples isolated from autistic subjects for one or a plurality of small molecule metabolites by gas chromatography mass spectrometry (GCMS);

b) assaying a collection of biosamples isolated from non-autistic control subjects for one or a plurality of small molecule metabolites by GCMS;

c) identifying one or a plurality of small molecule metabolites assayed by GCMS that are differentially produced in autistic subjects as compared to non-autistic control subjects;

d) assaying the collection of biosamples isolated from autistic subjects for one or a plurality of small molecule metabolites by one or more untargeted liquid chromatography-high resolution mass spectrometry methodologies (LC/HRMS);

e) assaying the collection of biosamples isolated from non-autistic control subjects for one or a plurality of small molecule metabolites by one or more untargeted LC/HRMS methodologies;

f) identifying one or a plurality of small molecule metabolites assayed by the one or more untargeted LC/HRMS methodologies that are differentially produced in autistic subjects as compared to non-autistic control subjects;

g) combining the plurality of small molecule metabolites identified by step c) and step f) to form a training set of small molecule metabolites; and

h) selecting from the training set a subset of small molecule metabolites with a statistically significant abundance difference in the collection of biosamples isolated form autistic patients as compared to the collection of biosamples isolated from control non-autistic control subjects;

wherein the subset of small molecules of step h) comprises a metabolomic signature for autism in a human.

In some aspects, assaying biosamples by one or more untargeted liquid chromatography-high resolution mass spectrometry methodologies (LC/HRMS) includes assaying the biosamples by C8 liquid chromatography coupled to electrospray ionization in positive ion polarity (C8pos), C8 liquid chromatography coupled to electrospray ionization in negative ion polarity (C8neg), hydrophilic interaction liquid chromatography coupled to electrospray ionization in positive ion polarity (HILICpos), and/or hydrophilic interaction liquid chromatography coupled to electrospray ionization in negative ion polarity (HILICneg).

The present invention includes methods for identifying a metabolomic signature characteristic for autism in a human including the steps of:

a) assaying a collection of biosamples isolated from autistic subjects for one or a plurality of small molecule metabolites by gas chromatography mass spectrometry (GCMS);

b) assaying a collection of biosamples isolated from non-autistic control subjects for one or a plurality of small molecule metabolites by GCMS;

c) identifying one or a plurality of small molecule metabolites assayed by GCMS that are differentially produced in autistic subjects as compared to non-autistic control subjects;

d) assaying the collection of biosamples isolated from autistic subjects for one or a plurality of small molecule metabolites by C8 liquid chromatography coupled to electrospray ionization in positive ion polarity (C8pos);

e) assaying the collection of biosamples isolated from non-autistic control subjects for one or a plurality of small molecule metabolites by C8pos;

f) identifying one or a plurality of small molecule metabolites assayed by C8pos that are differentially produced in autistic subjects as compared to non-autistic control subjects;

g) assaying the collection of biosamples isolated from autistic subjects for one or a plurality of small molecule metabolites by C8 liquid chromatography coupled to electrospray ionization in negative ion polarity (C8neg);

h) assaying the collection of biosamples isolated from non-autistic control subjects for one or a plurality of small molecule metabolites by C8neg;

i) identifying one or a plurality of small molecule metabolites assayed by C8neg that are differentially produced in autistic subjects as compared to non-autistic control subjects;

j) assaying the collection of biosamples isolated from autistic subjects for one or a plurality of small molecule metabolites by hydrophilic interaction liquid chromatography coupled to electrospray ionization in positive ion polarity (HILICpos);

k) assaying the collection of biosamples isolated from non-autistic control subjects for one or a plurality of small molecule metabolites by HILICpos;

l) identifying one or a plurality of small molecule metabolites assayed by HILICpos that are differentially produced in autistic subjects as compared to non-autistic control subjects;

m) assaying the collection of biosamples isolated from autistic subjects for one or a plurality of small molecule metabolites by hydrophilic interaction liquid chromatography coupled to electrospray ionization in negative ion polarity (HILICneg);

n) assaying the collection of biosamples isolated from non-autistic control subjects for one or a plurality of small molecule metabolites by HILICneg;

o) identifying one or a plurality of small molecule metabolites assayed by HILICneg that are differentially produced in autistic subjects as compared to non-autistic control subjects;

p) combining the plurality of small molecule metabolites identified by step c), step f), step I), step l), and step o) to form a training set of small molecule metabolites; and

q) selecting from the training set a subset of small molecule metabolites with a statistically significant abundance difference in the collection of biosamples isolated form autistic patients as compared to the collection of biosamples isolated from control non-autistic control subjects; wherein the subset of small molecules of step q) comprises a metabolomic signature for autism in a human.

Metabolites, as set forth herein, can be detected using alternative spectrometry methods or other methods known in the art, in addition to any of those described herein.

In some aspects of the methods for identifying a metabolomic signature characteristic for autism in a human of the present invention, a training set a subset of small molecule metabolites with a statistically significant abundance difference in the collection of biosamples isolated from autistic patients as compared to the collection of biosamples isolated from control non-autistic control subjects may be identified by univariate analysis, multivariate analysis, machine learning analysis, support vector machine analysis (SVM), and/or partial least squares analysis (PLS).

The present invention provides for metabolomic signatures for autism produced according to the methods described above. Such a signature may include any of the metabolites described herein, taken alone, as a population, or in any informative combination, as biomarkers of autism.

For example, in some aspects, a metabolic signature of autism may include any one or more of the 179 metabolites listed in Table 6. For example, at least about 5 or more of the metabolites, at least about 10 or more of the metabolites, at least about 20 or more of the metabolites, at least about 30 or more of the metabolites, at least about 40 or more of the metabolites, at least about 50 or more of the metabolites, at least about 60 or more of the metabolites, at least about 70 or more of the metabolites, at least about 80 or more of the metabolites, at least about 90 or more of the metabolites, at least about 100 or more of the metabolites, at least about 110 or more of the metabolites, at least about 120 or more of the metabolites, at least about 130 or more of the metabolites, at least about 140 or more of the metabolites, at least about 150 or more of the metabolites, at least about 160 or more of the metabolites, or at least about 170 or more of the metabolites listed in Table 6.

In some aspects, for example, a metabolic signature of autism may include about 10 of the metabolites, about 20 of the metabolites, about 30 of the metabolites, about 40 of the metabolites, about 50 of the metabolites, about 60 of the metabolites, about 70 of the metabolites, about 80 of the metabolites, about 90 of the metabolites, about 100 of the metabolites, about 110 of the metabolites, about 120 of the metabolites, about 130 of the metabolites, about 140 of the metabolites, about 150 of the metabolites, about 160 of the metabolites, or about 170 of the metabolites listed in Table 6.

In some aspects, a metabolic signature of autism may include a range of the metabolites listed in Table 6, including, for example, about 10 to about 20 of the metabolites, about 10 to about 30 of the metabolites, about 10 to about 40 of the metabolites, about 10 to about 50 of the metabolites, about 10 to about 60 of the metabolites, about 10 to about 70 of the metabolites, about 10 to about 80 of the metabolites, about 10 to about 90 of the metabolites, about 10 to about 100 of the metabolites, about 10 to about 110 of the metabolites, about 10 to about 120 of the metabolites, about 10 to about 130 of the metabolites, about 10 to about 140 of the metabolites, about 10 to about 150 of the metabolites, about 10 to about 160 of the metabolites, about 10 to about 170 of the metabolites, about 20 to about 30 of the metabolites, about 20 to about 40 of the metabolites, about 20 to about 50 of the metabolites, about 20 to about 60 of the metabolites, about 20 to about 70 of the metabolites, about 20 to about 80 of the metabolites, about 20 to about 90 of the metabolites, about 20 to about 100 of the metabolites, about 20 to about 110 of the metabolites, about 20 to about 120 of the metabolites, about 20 to about 130 of the metabolites, about 20 to about 140 of the metabolites, about 20 to about 150 of the metabolites, about 20 to about 160 of the metabolites, about 20 to about 170 of the metabolites, about 30 to about 40 of the metabolites, about 30 to about 50 of the metabolites, about 30 to about 60 of the metabolites, about 30 to about 70 of the metabolites, about 30 to about 80 of the metabolites, about 30 to about 90 of the metabolites, about 30 to about 100 of the metabolites, about 30 to about 110 of the metabolites, about 30 to about 120 of the metabolites, about 30 to about 130 of the metabolites, about 30 to about 140 of the metabolites, about 30 to about 150 of the metabolites, about 30 to about 160 of the metabolites, about 30 to about 170 of the metabolites, about 40 to about 50 of the metabolites, about 40 to about 60 of the metabolites, about 40 to about 70 of the metabolites, about 40 to about 80 of the metabolites, about 40 to about 90 of the metabolites, about 40 to about 100 of the metabolites, about 40 to about 110 of the metabolites, about 40 to about 120 of the metabolites, about 40 to about 130 of the metabolites, about 40 to about 140 of the metabolites, about 40 to about 150 of the metabolites, about 40 to about 160 of the metabolites, about 40 to about 170 of the metabolites, about 50 to about 60 of the metabolites, about 50 to about 70 of the metabolites, about 50 to about 80 of the metabolites, about 50 to about 90 of the metabolites, about 50 to about 100 of the metabolites, about 50 to about 110 of the metabolites, about 50 to about 120 of the metabolites, about 50 to about 130 of the metabolites, about 50 to about 140 of the metabolites, about 50 to about 150 of the metabolites, about 50 to about 160 of the metabolites, about 50 to about 170 of the metabolites, about 60 to about 60 of the metabolites, about 60 to about 70 of the metabolites, about 60 to about 80 of the metabolites, about 60 to about 90 of the metabolites, about 60 to about 100 of the metabolites, about 60 to about 110 of the metabolites, about 60 to about 120 of the metabolites, about 60 to about 130 of the metabolites, about 60 to about 140 of the metabolites, about 60 to about 150 of the metabolites, about 60 to about 160 of the metabolites, about 60 to about 170 of the metabolites, about 70 to about 80 of the metabolites, about 70 to about 90 of the metabolites, about 70 to about 100 of the metabolites, about 70 to about 110 of the metabolites, about 70 to about 120 of the metabolites, about 70 to about 130 of the metabolites, about 70 to about 140 of the metabolites, about 70 to about 150 of the metabolites, about 70 to about 160 of the metabolites, about 70 to about 170 of the metabolites, about 80 to about 90 of the metabolites, about 80 to about 100 of the metabolites, about 80 to about 110 of the metabolites listed, about 80 to about 120 of the metabolites, about 80 to about 130 of the metabolites, about 80 to about 140 of the metabolites, about 80 to about 150 of the metabolites, about 80 to about 160 of the metabolites, about 80 to about 170 of the metabolites, about 90 to about 100 of the metabolites, about 90 to about 110 of the metabolites, about 90 to about 120 of the metabolites, about 90 to about 130 of the metabolites, about 90 to about 140 of the metabolites, about 90 to about 150 of the metabolites, about 90 to about 160 of the metabolites, about 90 to about 170 of the metabolites, about 100 to about 110 of the metabolites, about 100 to about 120 of the metabolites, about 100 to about 130 of the metabolites, about 100 to about 140 of the metabolites, about 100 to about 150 of the metabolites, about 100 to about 160 of the metabolites, about 100 to about 170 of the metabolites, about 110 to about 120 of the metabolites, about 110 to about 130 of the metabolites, about 110 to about 140 of the metabolites, about 110 to about 150 of the metabolites, about 110 to about 160 of the metabolites, about 110 to about 170 of the metabolites, about 120 to about 130 of the metabolites, about 120 to about 140 of the metabolites, about 120 to about 150 of the metabolites, about 120 to about 160 of the metabolites, about 120 to about 170 of the metabolites, about 130 to about 140 of the metabolites, about 130 to about 150 of the metabolites, about 130 to about 160 of the metabolites, about 130 to about 170 of the metabolites, about 130 to about 150 of the metabolites, about 130 to about 160 of the metabolites, about 130 to about 170 of the metabolites, about 140 to about 150 of the metabolites, about 140 to about 160 of the metabolites, about 140 to about 170 of the metabolites, about 150 to about 160 of the metabolites, about 150 to about 170 of the metabolites, or about 160 to about 170 of the metabolites listed in Table 6.

For example, a metabolic signature of autism may include one or more of the metabolites listed in Table 5. For example, a metabolic signature of autism may include any one or more of the metabolites, any two or more metabolites, any three or more metabolites, any four or more metabolites, any five or more metabolites, any six or more metabolites, any seven or more metabolites, any eight or more metabolites, any nine or more metabolites, any ten or more metabolites, any eleven or more metabolites, any twelve or more metabolites, any thirteen or more metabolites, any fourteen or more metabolites, any fifteen or more metabolites, any sixteen or more metabolites, any seventeen or more metabolites, any eighteen or more metabolites, any nineteen or more metabolites, any twenty or more metabolites, or twenty one metabolites selected from homocitrulline, 2-hydroxyvaleric acid, cystine, aspartic acid, isoleucine, creatinine, serine, 4-hydroxyphenyllactic acid, citric acid, glutamic acid, lactic acid, DHEA sulfate, glutaric acid, 5-hydroxynorvaline, heptadecanoic acid, 5-aminovaleric acid lactam, succinic acid, myristic acid, 2-hydroxyvaleric acid, methylhexadecanoic acid, and/or 3-aminoisobutyric acid.

For example, a metabolic signature of autism may include one or more of the metabolites listed in Table 9; including, for example, any one or more of, any one or more of the metabolites, any two or more metabolites, any three or more metabolites, any four or more metabolites, any five or more metabolites, any six or more metabolites, any seven or more metabolites, any eight or more metabolites, any nine or more metabolites, any ten or more metabolites, any eleven or more metabolites, any twelve or more metabolites, any thirteen or more metabolites, any fourteen or more metabolites, any fifteen or more metabolites, any sixteen or more metabolites, any seventeen or more metabolites, any eighteen or more metabolites, any nineteen or more metabolites, any twenty or more metabolites, any twenty one or more metabolites, any twenty two or more metabolites, any twenty three or more metabolites, any twenty four or more metabolites, any twenty five or more metabolites, or twenty six metabolites selected from 2-aminooctanoic acid, acesulfame, ADMA, choline, CMPF, cysteine, cystine, DHEA sulfate (DHEAS), glycine, glycocholic acid, hypoxanthine, indoleacrylic acid, indoxyl sulfate, LysoPC(16:1(9Z)), LysoPE(0:0/18:1(9Z)), LysoPE(22:6(4Z,7Z,10Z,13Z,16Z,19Z)/0:0), LysoPE(22:6(4Z,7Z,10Z,13Z,16Z,19Z)/0:0), methionine, p-cresol sulfate, phenylalanine, phenyllactic acid, proline, serotonin, tryptophan, uric acid or valine.

For example, a metabolic signature of autism may include one or more of the metabolites listed in FIG. 13; including, for example, any one or more of, any one or more of the metabolites, any two or more metabolites, any three or more metabolites, any four or more metabolites, any five or more metabolites, any six or more metabolites, any seven or more metabolites, any eight or more metabolites, any nine or more metabolites, any ten or more metabolites, any eleven or more metabolites, any twelve or more metabolites, any thirteen or more metabolites, any fourteen or more metabolites, any fifteen or more metabolites, any sixteen or more metabolites, any seventeen or more metabolites, any eighteen or more metabolites, any nineteen or more metabolites, any twenty or more metabolites, any twenty one or more metabolites, any twenty two or more metabolites, any twenty three or more metabolites, any twenty four or more metabolites, any twenty five or more metabolites, any twenty six metabolites, any twenty seven or more of the metabolites, any twenty eight or more of the metabolites, or twenty nine of the metabolites selected from homocitrulline, glutaric acid, saccharopine, 5-aminovaleric acid, lactate, succinate, isocitrate, DHEAS, DHA, androsterone sulfate, 27-norcholesterol, Lyso PE, PE, long chain Fas, LysoPC, asparate, glutamate, acetylornithine, valine, isoleucine, ketoleucine, serine, homocysteic acid, valine, cystine, hydroxyacetone, phosphohydroxypyruvate, indole-3-lactate, and/or 3-amino isobutyrate.

Any one or more of such metabolites may be quantified gas chromatography mass spectrometry (GCMS), C8 liquid chromatography coupled to electrospray ionization in positive ion polarity (C8pos), C8 liquid chromatography coupled to electrospray ionization in negative ion polarity (C8neg), hydrophilic interaction liquid chromatography coupled to electrospray ionization in positive ion polarity (HILICpos), or hydrophilic interaction liquid chromatography coupled to electrospray ionization in negative ion polarity (HILICneg). In some aspects, any one or more of such metabolites may be quantified by the methodology indicated in Table 5, Table 6, or Table 9.

In some aspects of the methods of the present invention, the chemical identity of small molecules that exhibit statistically significant differences between autistic and non-autistic individuals are confirmed. The chemical structures of metabolites identified as statistically significantly different between autistic and non-autistic subjects may be confirmed using HRMS methods, using chromatographic conditions identical to those used for their discovery. HRMS-MS analyses may be performed on Agilent QTOF mass spectrometers for patient samples, reference compounds and samples spiked with reference compounds. Ionization and collision energy conditions may be optimized to obtain the highest quality MS-MS spectra. The resulting HRMS or HR-MS-MS ion fragmentation spectra may be compared to confirm annotated identities for each small molecule metabolite to establish a panel of validated candidate diagnostic biomarkers. The data may be compared to spectra available in several locations, including public databases database. If an MS-MS spectrum does not match available database spectra, a reference compound may be obtained for the putatively annotated compounds and MS-MS spectra will be obtained for the reference compound then compared with that of the sample.

In some aspects, a metabolic signature of autism is demonstrated by an increase or a decrease in abundance when compared to typical/normal controls. Including, for example, decreased homocitrulline, increased glutaric acid, increased saccharopine, increased 5-aminovaleric acid, increased lactate, increased succinate, decreased isocitrate, increased DHEAS, increased DHA, increased androsterone sulfate, increased 27-norcholesterol, decreased Lyso PE, decreased PE, decreased long chain Fas, decreased LysoPC, increased asparate, increased glutamate, increased acetylornithine, decreased valine, decreased isoleucine, increased ketoleucine, increased serine, decreased homocysteic acid, decreased valine, decreased cystine, increased hydroxyacetone, increased phosphohydroxypyruvate, decreased indole-3-lactate, and/or increased 3-amino isobutyrate in comparison to normal controls.

This may be measured as an average abundance ratio relative to a normal control. In some aspects, an average abundance ratio of other than about 1 may be indicative of autism. For example, an average abundance ratio of greater than about 1 (for example, including, but not limited to, about 1.01, about 1.02, about 1.03, about 1.04, about 1.05, about 1.06, about 1.07, about 1.08, about 1.09, about 1.1, about 1.11, about 1.12, about 1.13, about 1.14, about 1.15, about 1.16, about 1.17, about 1.18, about 1.19, about 1.2, about 1.21, about 1.22, about 1.23, about 1.24, about 1.25, about 1.26, about 1.27, about 1.28, about 1.29, about 1.3, about 1.31, about 1.32, about 1.33, about 1.34, about 1.35, about 1.36, about 1.37, about 1.38, about 1.39, about 1.4, about 1.41, about 1.42, about 1.43, about 1.44, about 1.45, about 1.46, about 1.47, about 1.48, about 1.49, or about 1.5) may be indicative of autism. In some aspects, an average abundance ratio of less than about 1 (for example, including, but not limited to, about 0.99, about 0.98, about 0.97, about 0.96, about 0.95, about 0.94, about 0.93, about 0.92, about 0.91, about 0.9, about 0.89, about 0.88, about 0.87, about 0.86, about 0.85, about 0.84, about 0.83, about 0.82, about 0.81, about 0.8, about 0.79, about 0.78, about 0.77, about 0.76, about 0.75, about 0.74, about 0.73, about 0.72, about 0.71, about 0.7, about 0.69, about 0.68, about 0.67, about 0.66, about 0.65, about 0.64, about 0.63, about 0.62, about 0.61, about 0.6, about 0.59, about 0.58, about 0.57, about 0.56, about 0.55, about 0.54, about 0.53, about 0.52, about 0.51, or about 0.5) may be indicative of autism.

The present invention relates to small molecules or metabolites found to have significantly different abundances or intensities between plasma samples from autistic children and typically developing, normal children. And, the present invention includes methods of assessing a subject's risk for developing autism and/or for the diagnosis of autism. A subject may be determined to be at risk for ASD or diagnosed with ASD based on a statistically significant (p<0.05) increase or decrease relative to the corresponding data of a reference sample from a non-ASD subject in the level of one or more of the small molecule metabolites of a metabolic signature identified by the methods described herein.

In some aspects, the quantification of one or more small molecule metabolites of a metabolic signature of autism may be assayed using a physical separation method, such as, for example, one or more methodologies selected from gas chromatography mass spectrometry (GCMS), C8 liquid chromatography coupled to electrospray ionization in positive ion polarity (C8pos), C8 liquid chromatography coupled to electrospray ionization in negative ion polarity (C8neg), hydrophilic interaction liquid chromatography coupled to electrospray ionization in positive ion polarity (HILICpos), and/or hydrophilic interaction liquid chromatography coupled to electrospray ionization in negative ion polarity (HILICneg). In some aspects, the determination of a metabolite may be by a methodology other than a physical separation method, such as for example, a colorimetric, enzymatic, immunological methodology.

In some aspects, a method of assessing a subject's risk for autism and for the diagnosis of autism may include assaying a biosample from the subject for one or a plurality of small molecule metabolites and quantifying the amount of one or more of the 179 small molecule metabolites listed in Table 6, wherein a statistically significant abundance difference as compared to non-autistic controls in one or more of the 179 small molecule metabolites listed in Table 6 indicates an increased risk of autism.

In some aspects, a method of assessing a subject's risk for autism and for the diagnosis of autism may include a step of assaying a biosample from the subject for one or a plurality of small molecule metabolites by one or more methodologies selected from gas chromatography mass spectrometry (GCMS), C8 liquid chromatography coupled to electrospray ionization in positive ion polarity (C8pos), C8 liquid chromatography coupled to electrospray ionization in negative ion polarity (C8neg), hydrophilic interaction liquid chromatography coupled to electrospray ionization in positive ion polarity (HILICpos), and/or hydrophilic interaction liquid chromatography coupled to electrospray ionization in negative ion polarity (HILICneg) and quantifying the amount of one or more of the 179 small molecule metabolites listed in Table 6, wherein a statistically significant abundance difference as compared to non-autistic controls in one or more of the 179 small molecule metabolites listed in Table 6 indicates an increased risk of autism.

In some aspects, one or more of the 179 metabolites listed in Table 6 may include, for example, at least about 5 or more of the metabolites, at least about 10 or more of the metabolites, at least about 20 or more of the metabolites, at least about 30 or more of the metabolites, at least about 40 or more of the metabolites, at least about 50 or more of the metabolites, at least about 60 or more of the metabolites, at least about 70 or more of the metabolites, at least about 80 or more of the metabolites, at least about 90 or more of the metabolites, at least about 100 or more of the metabolites, at least about 110 or more of the metabolites, at least about 120 or more of the metabolites, at least about 130 or more of the metabolites, at least about 140 or more of the metabolites, at least about 150 or more of the metabolites, at least about 160 or more of the metabolites, or at least about 170 or more of the metabolites listed in Table 6.

In some aspects, one or more of the 179 metabolites listed in Table 6 may include, for example, about 10 of the metabolites, about 20 of the metabolites, about 30 of the metabolites, about 40 of the metabolites, about 50 of the metabolites, about 60 of the metabolites, about 70 of the metabolites, about 80 of the metabolites, about 90 of the metabolites, about 100 of the metabolites, about 110 of the metabolites, about 120 of the metabolites, about 130 of the metabolites, about 140 of the metabolites, about 150 of the metabolites, about 160 of the metabolites, or about 170 of the metabolites listed in Table 6.

In some aspects, one or more of the 179 metabolites listed in Table 6 may include a range of the metabolites, including, for example, about 10 to about 20 of the metabolites, about 10 to about 30 of the metabolites, about 10 to about 40 of the metabolites, about 10 to about 50 of the metabolites, about 10 to about 60 of the metabolites, about 10 to about 70 of the metabolites, about 10 to about 80 of the metabolites, about 10 to about 90 of the metabolites, about 10 to about 100 of the metabolites, about 10 to about 110 of the metabolites, about 10 to about 120 of the metabolites, about 10 to about 130 of the metabolites, about 10 to about 140 of the metabolites, about 10 to about 150 of the metabolites, about 10 to about 160 of the metabolites, about 10 to about 170 of the metabolites, about 20 to about 30 of the metabolites, about 20 to about 40 of the metabolites, about 20 to about 50 of the metabolites, about 20 to about 60 of the metabolites, about 20 to about 70 of the metabolites, about 20 to about 80 of the metabolites, about 20 to about 90 of the metabolites, about 20 to about 100 of the metabolites, about 20 to about 110 of the metabolites, about 20 to about 120 of the metabolites, about 20 to about 130 of the metabolites, about 20 to about 140 of the metabolites, about 20 to about 150 of the metabolites, about 20 to about 160 of the metabolites, about 20 to about 170 of the metabolites, about 30 to about 40 of the metabolites, about 30 to about 50 of the metabolites, about 30 to about 60 of the metabolites, about 30 to about 70 of the metabolites, about 30 to about 80 of the metabolites, about 30 to about 90 of the metabolites, about 30 to about 100 of the metabolites, about 30 to about 110 of the metabolites, about 30 to about 120 of the metabolites, about 30 to about 130 of the metabolites, about 30 to about 140 of the metabolites, about 30 to about 150 of the metabolites, about 30 to about 160 of the metabolites, about 30 to about 170 of the metabolites, about 40 to about 50 of the metabolites, about 40 to about 60 of the metabolites, about 40 to about 70 of the metabolites, about 40 to about 80 of the metabolites, about 40 to about 90 of the metabolites, about 40 to about 100 of the metabolites, about 40 to about 110 of the metabolites, about 40 to about 120 of the metabolites, about 40 to about 130 of the metabolites, about 40 to about 140 of the metabolites, about 40 to about 150 of the metabolites, about 40 to about 160 of the metabolites, about 40 to about 170 of the metabolites, about 50 to about 60 of the metabolites, about 50 to about 70 of the metabolites, about 50 to about 80 of the metabolites, about 50 to about 90 of the metabolites, about 50 to about 100 of the metabolites, about 50 to about 110 of the metabolites, about 50 to about 120 of the metabolites, about 50 to about 130 of the metabolites, about 50 to about 140 of the metabolites, about 50 to about 150 of the metabolites, about 50 to about 160 of the metabolites, about 50 to about 170 of the metabolites, about 60 to about 60 of the metabolites, about 60 to about 70 of the metabolites, about 60 to about 80 of the metabolites, about 60 to about 90 of the metabolites, about 60 to about 100 of the metabolites, about 60 to about 110 of the metabolites, about 60 to about 120 of the metabolites, about 60 to about 130 of the metabolites, about 60 to about 140 of the metabolites, about 60 to about 150 of the metabolites, about 60 to about 160 of the metabolites, about 60 to about 170 of the metabolites, about 70 to about 80 of the metabolites, about 70 to about 90 of the metabolites, about 70 to about 100 of the metabolites, about 70 to about 110 of the metabolites, about 70 to about 120 of the metabolites, about 70 to about 130 of the metabolites, about 70 to about 140 of the metabolites, about 70 to about 150 of the metabolites, about 70 to about 160 of the metabolites, about 70 to about 170 of the metabolites, about 80 to about 90 of the metabolites, about 80 to about 100 of the metabolites, about 80 to about 110 of the metabolites listed, about 80 to about 120 of the metabolites, about 80 to about 130 of the metabolites, about 80 to about 140 of the metabolites, about 80 to about 150 of the metabolites, about 80 to about 160 of the metabolites, about 80 to about 170 of the metabolites, about 90 to about 100 of the metabolites, about 90 to about 110 of the metabolites, about 90 to about 120 of the metabolites, about 90 to about 130 of the metabolites, about 90 to about 140 of the metabolites, about 90 to about 150 of the metabolites, about 90 to about 160 of the metabolites, about 90 to about 170 of the metabolites, about 100 to about 110 of the metabolites, about 100 to about 120 of the metabolites, about 100 to about 130 of the metabolites, about 100 to about 140 of the metabolites, about 100 to about 150 of the metabolites, about 100 to about 160 of the metabolites, about 100 to about 170 of the metabolites, about 110 to about 120 of the metabolites, about 110 to about 130 of the metabolites, about 110 to about 140 of the metabolites, about 110 to about 150 of the metabolites, about 110 to about 160 of the metabolites, about 110 to about 170 of the metabolites, about 120 to about 130 of the metabolites, about 120 to about 140 of the metabolites, about 120 to about 150 of the metabolites, about 120 to about 160 of the metabolites, about 120 to about 170 of the metabolites, about 130 to about 140 of the metabolites, about 130 to about 150 of the metabolites, about 130 to about 160 of the metabolites, about 130 to about 170 of the metabolites, about 130 to about 150 of the metabolites, about 130 to about 160 of the metabolites, about 130 to about 170 of the metabolites, about 140 to about 150 of the metabolites, about 140 to about 160 of the metabolites, about 140 to about 170 of the metabolites, about 150 to about 160 of the metabolites, about 150 to about 170 of the metabolites, or about 160 to about 170 of the metabolites listed in Table 6.

In some aspects, a method of assessing a subject's risk for autism and/or for the diagnosis of autism may include assaying a biosample from the subject for one or a plurality of small molecule metabolites and quantifying the amount of one or more of the 21 small molecule metabolites listed in Table 5, wherein a statistically significant abundance difference as compared to non-autistic controls in one or more of the 21 small molecule metabolites listed in Table 5 indicates an increased risk of autism. For example, a statistically significant abundance difference as compared to non-autistic controls of any one or more any one or more of the metabolites, any two or more metabolites, any three or more metabolites, any four or more metabolites, any five or more metabolites, any six or more metabolites, any seven or more metabolites, any eight or more metabolites, any nine or more metabolites, any ten or more metabolites, any eleven or more metabolites, any twelve or more metabolites, any thirteen or more metabolites, any fourteen or more metabolites, any fifteen or more metabolites, any sixteen or more metabolites, any seventeen or more metabolites, any eighteen or more metabolites, any nineteen or more metabolites, any twenty or more metabolites, or twenty one metabolites of homocitrulline, 2-hydroxyvaleric acid, cystine, aspartic acid, isoleucine, creatinine, serine, 4-hydroxyphenyllactic acid, citric acid, glutamic acid, lactic acid, DHEA sulfate, glutaric acid, 5-hydroxynorvaline, heptadecanoic acid, 5-aminovaleric acid lactam, succinic acid, myristic acid, 2-hydroxyvaleric acid, methylhexadecanoic acid, and/or 3-aminoisobutyric acid indicates an increased risk of autism.

In some aspects, a method of assessing a subject's risk for autism and for the diagnosis of autism may include assaying a biosample from the subject for one or a plurality of small molecule metabolites and quantifying the amount of one or more of the 26 small molecule metabolites listed in Table 9, wherein a statistically significant abundance difference as compared to non-autistic controls in one or more of the 26 small molecule metabolites listed in Table 9 indicates an increased risk of autism. For example, a statistically significant abundance difference as compared to non-autistic controls of any one or more of, any one or more of the metabolites, any two or more metabolites, any three or more metabolites, any four or more metabolites, any five or more metabolites, any six or more metabolites, any seven or more metabolites, any eight or more metabolites, any nine or more metabolites, any ten or more metabolites, any eleven or more metabolites, any twelve or more metabolites, any thirteen or more metabolites, any fourteen or more metabolites, any fifteen or more metabolites, any sixteen or more metabolites, any seventeen or more metabolites, any eighteen or more metabolites, any nineteen or more metabolites, any twenty or more metabolites, or twenty one or more metabolites, any twenty two or more metabolites, any twenty three or more metabolites, any twenty four or more metabolites, any twenty five or more metabolites, and/or twenty six metabolites of 2-aminooctanoic acid, acesulfame, ADMA, choline, CMPF, cysteine, cystine, DHEA sulfate (DHEAS), glycine, glycocholic acid, hypoxanthine, indoleacrylic acid, indoxyl sulfate, LysoPC(16:1(9Z)), LysoPE(0:0/18:1(9Z)), LysoPE(22:6(4Z,7Z,10Z,13Z,16Z,19Z)/0:0), LysoPE(22:6(4Z,7Z,10Z,13Z,16Z,19Z)/0:0), methionine, p-cresol sulfate, phenylalanine, phenyllactic acid, proline, serotonin, tryptophan, uric acid, and/or valine indicates an increased risk of autism.

In some aspects, a method of assessing a subject's risk for autism and for the diagnosis of autism may include assaying a biosample from the subject for one or a plurality of small molecule metabolites and quantifying the amount of one or more of the 29 small molecule metabolites listed in FIG. 13, wherein a statistically significant abundance difference as compared to non-autistic controls in one or more of the 29 small molecule metabolites listed in FIG. 13 indicates an increased risk of autism. For example, a statistically significant abundance difference as compared to non-autistic controls of any one or more of, any one or more of the metabolites, any two or more metabolites, any three or more metabolites, any four or more metabolites, any five or more metabolites, any six or more metabolites, any seven or more metabolites, any eight or more metabolites, any nine or more metabolites, any ten or more metabolites, any eleven or more metabolites, any twelve or more metabolites, any thirteen or more metabolites, any fourteen or more metabolites, any fifteen or more metabolites, any sixteen or more metabolites, any seventeen or more metabolites, any eighteen or more metabolites, any nineteen or more metabolites, any twenty or more metabolites, or twenty one or more metabolites, any twenty two or more metabolites, any twenty three or more metabolites, any twenty four or more metabolites, any twenty five or more metabolites, any twenty six metabolites or more metabolites, any twenty seven metabolites or more metabolites, any twenty eight metabolites or more metabolites, and/or twenty nine metabolites of homocitrulline, glutaric acid, saccharopine, 5-aminovaleric acid, lactate, succinate, isocitrate, DHEAS, DHA, androsterone sulfate, 27-norcholesterol, Lyso PE, PE, long chain Fas, LysoPC, asparate, glutamate, acetylornithine, valine, isoleucine, ketoleucine, serine, homocysteic acid, valine, cystine, hydroxyacetone, phosphohydroxypyruvate, indole-3-lactate, and/or 3-amino isobutyrate indicates an increased risk of autism.

In some aspects, a method of assessing a subject's risk for autism and for the diagnosis of autism may include assaying a biosample from the subject for decreased homocitrulline, increased glutaric acid, increased saccharopine, increased 5-aminovaleric acid, increased lactate, increased succinate, decreased isocitrate, increased DHEAS, increased DHA, increased androsterone sulfate, increased 27-norcholesterol, decreased Lyso PE, decreased PE, decreased long chain Fas, decreased LysoPC, increased asparate, increased glutamate, increased acetylornithine, decreased valine, decreased isoleucine, increased ketoleucine, increased serine, decreased homocysteic acid, decreased valine, decreased cystine, increased hydroxyacetone, increased phosphohydroxypyruvate, decreased indole-3-lactate, and/or increased 3-amino isobutyrate.

In some aspects, a method of assessing a subject's risk for autism and for the diagnosis of autism may include assaying a biosample from the subject for decreased glycine, serine, threonine, alanine, histidine, glutamyl amino acids, taurine, and/or carnosine.

In some aspects, a method of assessing a subject's risk for autism and for the diagnosis of autism may include assaying a biosample from the subject for decreased homocitrulline.

Biosamples may be from any of a variety of mammalian subjects. In preferred embodiments, a biosample is from a human subject. A biosample may be from an individual clinically diagnosed with ASD. ASD may be diagnosed by any of a variety of well-known clinical criteria. For example, diagnosis of autism spectrum disorder may be based on the DSM-IV criteria determined by an experienced neuropsychologist and/or the Autism Diagnostic Observation Schedule-Generic (ADOS-G) which provides observation of a child's communication, reciprocal social interaction, and stereotyped behavior including an algorithm with cutoffs for autism and autism spectrum disorders.

A biosample may be from an individual determined to be at some risk for ASD (for example by family history) with little or no current ASD symptoms. A biosample may be a suitable reference or control sample from an individual not suffering from ASD with or without a family history of ASD. In some aspects, a plurality of samples is obtained from a population, for example, a population of individuals with ASD, at risk for ASD, or normal, typically developing individuals. A biosample may be from an adult subject. A biosample may be from a child, for example, a child that is under about 6 years of age, under about 4 years of age, under about 2 years of age, or under about 1 year of age, about 1 to about 6 years of age, about 1 to about 5 years of age, about 1 to about 4 years of age, about 1 to about 2 years of age, about 2 to about 6 years of age, about 2 to about 4 years of age, or about 4 to about 6 years of age. A biosample may be from a phenotypic subpopulation of autism subjects, such as, for example, high functioning autism (HFA) or low functioning autism (LFA).

In accordance with the methods disclosed herein, any type of biological sample that originates from anywhere within the body of a subject may be tested, including, but not limited to, blood (including, but no limited to serum or plasma), cerebrospinal fluid (CSF), pleural fluid, urine, stool, sweat, tears, breath, saliva, a tissue sample, amniotic fluid, a chorionic villus sampling, brain tissue, a biopsy of any solid tissue including tumor, adjacent normal, smooth and skeletal muscle, adipose tissue, liver, skin, hair, brain, kidney, pancreas, lung, colon, stomach, or the like may be used. A blood sample may include, for example, a whole blood sample, a blood serum sample, a blood plasma sample, or other blood components, such as, for example, a subfraction of whole blood. A sample may be from a live subject. In some applications, samples may be collected post mortem.

When a blood sample is drawn from a subject, it can be processed in any of many known ways. The range of processing can be from little to none (such as, for example, frozen whole blood) or as complex as the isolation of a particular cell type. Common and routine procedures include the preparation of either serum or plasma from whole blood. All blood sample processing methods, including spotting of blood samples onto solid-phase supports, such as filter paper or other immobile materials, are contemplated by the present invention.

With the preparation of samples for analysis, metabolites may be extracted from their biological source using any number of extraction/clean-up procedures that are typically used in quantitative analytical chemistry.

A computer may be used for statistical analysis. Data for statistical analysis can be extracted from chromatograms (spectra of mass signals) using softwares for statistical methods known in the art. “Statistics” is the science of making effective use of numerical data relating to groups of individuals or experiments. Methods for statistical analysis are well-known in the art. In one embodiment a computer is used for statistical analysis. In one embodiment, the Agilent MassProfiler or MassProfilerProfessional software is used for statistical analysis. In another embodiment, the Agilent MassHunter software Qual software is used for statistical analysis. In other embodiments, alternative statistical analysis methods can be used. Such other statistical methods include the Analysis of Variance (ANOVA) test, Chi-square test, Correlation test, Factor analysis test, Mann-Whitney U test, Mean square weighted derivation (MSWD), Pearson product-moment correlation coefficient, Regression analysis, Spearman's rank correlation coefficient, Student's T test, Welch's T-test, Tukey's test, and Time series analysis. In different embodiments signals from mass spectrometry can be transformed in different ways to improve the performance of the method. Either individual signals or summaries of the distributions of signals (such as mean, median or variance) can be so transformed. Possible transformations include taking the logarithm, taking some positive or negative power, for example the square root or inverse, or taking the arcsin. In different embodiments, statistical classification algorithms are used to create a classification model in order to predict autism and non-autism. Machine learning-based classifiers have been applied in various fields such as machine perception, medical diagnosis, bioinformatics, brain-machine interfaces, classifying DNA sequences, and object recognition in computer vision. Learning-based classifiers have proven to be highly efficient in solving some biological problems.

“Sensitivity” and “specificity” are statistical measures of the performance of a binary classification test. Sensitivity measures the proportion of actual positives which are correctly identified as such (e.g. the percentage of sick people who are correctly identified as having the condition). Specificity measures the proportion of negatives which are correctly identified (e.g. the percentage of healthy people who are correctly identified as not having the condition). These two measures are closely related to the concepts of type I and type II errors. A theoretical, optimal prediction can achieve 100% sensitivity (i.e. predict all people from the sick group as sick) and 100% specificity (i.e. not predict anyone from the healthy group as sick). A specificity of 100% means that the test recognizes all actual negatives—for example, in a test for a certain disease, all disease free people will be recognized as disease free. A sensitivity of 100% means that the test recognizes all actual positives—for example, all sick people are recognized as being ill. Thus, in contrast to a high specificity test, negative results in a high sensitivity test are used to rule out the disease. A positive result in a high specificity test can confirm the presence of disease. However, from a theoretical point of view, a 100%-specific test standard can also be ascribed to a ‘bogus’ test kit whereby the test simply always indicates negative. Therefore the specificity alone does not tell us how well the test recognizes positive cases. Knowledge of sensitivity is also required. For any test, there is usually a trade-off between the measures. For example, in a diagnostic assay in which one is testing for people who have a certain condition, the assay may be set to overlook a certain percentage of sick people who are correctly identified as having the condition (low specificity), in order to reduce the risk of missing the percentage of healthy people who are correctly identified as not having the condition (high sensitivity). Eliminating the systematic error improves accuracy but does not change precision. This trade-off can be represented graphically using a receiver operating characteristic (ROC) curve.

The “accuracy” of a measurement system is the degree of closeness of measurements of a quantity to its actual (true) value. The “precision” of a measurement system, also called reproducibility or repeatability, is the degree to which repeated measurements under unchanged conditions show the same results. Although the two words can be synonymous in colloquial use, they are deliberately contrasted in the context of the scientific method. A measurement system can be accurate but not precise, precise but not accurate, neither, or both. For example, if an experiment contains a systematic error, then increasing the sample size generally increases precision but does not improve accuracy.

The term “predictability” (also called banality) is the degree to which a correct prediction or forecast of a system's state can be made either qualitatively or quantitatively. Perfect predictability implies strict determinism, but lack of predictability does not necessarily imply lack of determinism. Limitations on predictability could be caused by factors such as a lack of information or excessive complexity.

In some embodiments, the invention discloses a method for diagnosing autism with at least about 80% accuracy, at least about 81% accuracy, at least about 82% accuracy, at least about 83% accuracy, at least about 84% accuracy, at least about 85% accuracy, at least about 86% accuracy, at least about 87% accuracy, at least about 88% accuracy, at least about 89% accuracy, at least about 90% accuracy, at least about 91% accuracy, at least about 92% accuracy, at least about 93% accuracy, at least about 94% accuracy, at least about 95% accuracy, at least about 96% accuracy, at least about 97% accuracy, at least about 98% accuracy, or at least about 99% accuracy.

In some embodiments, the invention discloses a method for diagnosing autism with at least about 80% sensitivity, at least about 81% sensitivity, at least about 82% sensitivity, at least about 83% sensitivity, at least about 84% sensitivity, at least about 85% sensitivity, at least about 86% sensitivity, at least about 87% sensitivity, at least about 88% sensitivity, at least about 89% sensitivity, at least about 90% sensitivity, at least about 91% sensitivity, at least about 92% sensitivity, at least about 93% sensitivity, at least about 94% sensitivity, at least about 95% sensitivity, at least about 96% sensitivity, at least about 97% sensitivity, at least about 98% sensitivity, or at least about 99% sensitivity.

In some embodiments, the invention discloses a method for diagnosing autism with at least about 75% specificity, at least about 80% specificity, at least about 81% specificity, at least about 82% specificity, at least about 83% specificity, at least about 84% specificity, at least about 85% specificity, at least about 86% specificity, at least about 87% specificity, at least about 88% specificity, at least about 89% specificity, at least about 90% specificity, at least about 91% specificity, at least about 92% specificity, at least about 93% specificity, at least about 94% specificity, at least about 95% specificity, at least about 96% specificity, at least about 97% specificity, at least about 98% specificity, or at least about 99% specificity,

In some embodiments, the invention discloses a method for diagnosing autism with any combination of accuracy, sensitivity, and specificity selected from those described above.

In some embodiments, the invention discloses a method for diagnosing autism with accuracy, sensitivity, and/or specificity as described in the example included herewith.

In some aspects, an average abundance ratio of the concentration of a signature metabolite indicative of autism in an autism sample in comparison to typically developing sample may be determined. Such an average abundance ratio may be utilized in the diagnosis of autism. Further, such an average abundance ratio may be indicative of a phenotypic subpopulation of autism. The average abundance ratio of any number of signature metabolites indicative of autism may be utilized in the determination of autism and/or a phenotypic subpopulation of autism. For example, an average abundance ratio may be determined for any one or any plurality of the metabolites described in Table 5, Table 6, and/or Table 9, as previously described herein. In some aspects, an average abundance of other than about 1 may be indicative of autism and/or a phenotypic subpopulation of autism. For example, a fold change ratio of greater than about 1 (for example, including, but not limited to, about 1.01, about 1.02, about 1.03, about 1.04, about 1.05, about 1.06, about 1.07, about 1.08, about 1.09, about 1.1, about 1.11, about 1.12, about 1.13, about 1.14, about 1.15, about 1.16, about 1.17, about 1.18, about 1.19, about 1.2, about 1.21, about 1.22, about 1.23, about 1.24, about 1.25, about 1.26, about 1.27, about 1.28, about 1.29, about 1.3, about 1.31, about 1.32, about 1.33, about 1.34, about 1.35, about 1.36, about 1.37, about 1.38, about 1.39, about 1.4, about 1.41, about 1.42, about 1.43, about 1.44, about 1.45, about 1.46, about 1.47, about 1.48, about 1.49, or about 1.5) may be indicative of autism and/or a phenotypic subpopulation of autism. For example, a fold change ratio of less than about 1 (for example, including, but not limited to, about 0.99, about 0.98, about 0.97, about 0.96, about 0.95, about 0.94, about 0.93, about 0.92, about 0.91, about 0.9, about 0.89, about 0.88, about 0.87, about 0.86, about 0.85, about 0.84, about 0.83, about 0.82, about 0.81, about 0.8, about 0.79, about 0.78, about 0.77, about 0.76, about 0.75, about 0.74, about 0.73, about 0.72, about 0.71, about 0.7, about 0.69, about 0.68, about 0.67, about 0.66, about 0.65, about 0.64, about 0.63, about 0.62, about 0.61, about 0.6, about 0.59, about 0.58, about 0.57, about 0.56, about 0.55, about 0.54, about 0.53, about 0.52, about 0.51, or about 0.5) may be indicative of autism and/or a phenotypic subpopulation of autism.

In some aspects, a ratio of the concentration in the same sample of one signature metabolite indicative of autism relative to the concentration of a second signature metabolite indicative of autism may be determined. Such a ratio may be utilized in the diagnosis of autism. Further, such a ratio may be indicative of a phenotypic subpopulation of autism. A ratio of any one signature metabolite described herein relative to any second signature metabolite described herein may be determined to indicative of autism and/or a phenotypic subpopulation of autism. Such a signature metabolite described herein includes, but is not limited to, any of those described in Table 5, Table 6, and/or Table 9.

In some aspects, a ratio of the concentration in the same sample of a signature metabolite indicative of autism as described herein relative to the concentration of another metabolite may be determined. Such a ratio may be utilized in the diagnosis of autism. Further, such a ratio may be indicative of a phenotypic subpopulation of autism. Such a signature metabolite described herein includes, but is not limited to, any of those described in Table 5, Table 6, and/or Table 9.

In some aspects, a method for diagnosing autism based on identification and/or quantification of one or more signature metabolites indicative of autism as described herein may further include the identification and/or quantification of one or more additional known markers of autism. For example, one or more of the markers and/or methodologies for their identification and/or quantification as described in US Patent Application 20120190055 (“Molecule Biomarkers of Autism”), which is hereby incorporated by reference in its entirety, may be used. One or more of the markers and/or the methodologies for their identification and/or quantification as described in U.S. Pat. No. 8,273,575 (“Methods for the diagnosis, risk assessment, and monitoring of autism spectrum disorders”, which is hereby incorporated by reference in its entirety, may be used. In some aspects, the nucleic acids from a biological sample may be analyzed to determine the genotype and/or expression of genes associated with or relevant to autism.

The metabolic markers and signatures described herein may be utilized in tests, assays, methods, kits for diagnosing, predicting, modulating, or monitoring ASD, including ongoing assessment, monitoring, susceptibility assessment, carrier testing and prenatal diagnosis.

The present invention includes a kit for identifying and/or measuring one or more metabolites associated with the assessment of a risk for ASD. In some aspects, the kit may be for the determination of a metabolite by a physical separation method. In some aspects, the kit may be for the determination of a metabolite by a methodology other than a physical separation method, such as for example, a colorimetric, enzymatic, immunological methodology. In some aspects an assay kit may also include one or more appropriate negative controls and/or positive controls. Kits of the present invention may include other reagents such as buffers and solutions needed to practice the invention are also included. Optionally associated with such container(s) can be a notice or printed instructions. As used herein, the phrase “packaging material” refers to one or more physical structures used to house the contents of the kit. The packaging material is constructed by well-known methods, preferably to provide a sterile, contaminant-free environment. As used herein, the term “package” refers to a solid matrix or material such as glass, plastic, paper, foil, and the like. Kits of the present invention may also include instructions for use. Instructions for use typically include a tangible expression describing the reagent concentration or at least one assay method parameter, such as the relative amounts of reagent and sample to be admixed, maintenance time periods for reagent/sample admixtures, temperature, buffer conditions, and the like. In some aspects, a kit may be a packaged combination comprising the basic elements of a first container comprising, in solid form, a specific set of one or more purified metabolites, as described herein, and a second container comprising a physiologically suitable buffer for resuspending the specific subset of purified metabolites. Such a kit may be used by a medical specialist to determine whether or not a subject is at risk for ASD. Appropriate therapeutic intervention may be prescribed or initiated upon the determination of a risk of ASD. One or more of the metabolites described herein may be present in a kit.

The present invention is illustrated by the following examples. It is to be understood that the particular examples, materials, amounts, and procedures are to be interpreted broadly in accordance with the scope and spirit of the invention as set forth herein.

### EXAMPLES

**Example 1**

**Biomarkers of Autism Spectrum Disorder in the Blood Plasma of Children**

The diagnosis of autism spectrum disorder (ASD) at the earliest age possible is important for initiating optimally effective intervention. Patients can be reliably diagnosed through behavioral testing at approximately two years of age. However, in the United States the average age of diagnosis is around four years. Increasing evidence indicates that ASD has many causes and a variety of genetic risk factors. Identifying metabolic biomarker signatures of ASD from blood samples offers an opportunity for developing early diagnostic tests.

With the present example, a study was undertaken to discover metabolic features from plasma samples that may be able to discriminate children with ASD from typically developing (TD) children. The ultimate goal of this research is to develop blood-based ASD biomarkers.

The etiology of the vast majority of cases of ASD are unknown and their genetics have proven to be incredibly complex (State and Sestan, 2012, Science; 337:1301-1303; and Berg and Geschwind, 2012, Genome Biol; 13:247). There is now widespread appreciation that there will be many causes of ASD with varying combinations of genetic and environmental risk factors at play. Numerous studies have attempted to identify the causes of the disorder by studying transcriptomics and genomics, leading to the identification of multiple genes associated with ASD (Berg and Geschwind, 2012, Genome Biol; 13:247; and Huguet et al., 2013, Annu Rev Genomics Hum Genet; 14:191-213). There are currently hundreds of observable genetic variants that account for about 20% of the cases of autism. These data are currently most useful in understanding the intra-familial genetics of autism. For this reason, clinical tests based on genomic measures often include genetic counseling to assess the chance of disease occurrence or recurrence within a family (Bucan et al., 2009, PLoS Genet; 5:e1000536; and Wang et al., 2009, Nature; 459:528-533). Prediction accuracies of ASD risk based on genomic approaches range from 56% to 70% depending largely on the population of patients assessed. Separate analyses of at least one of the genomic studies by Skafidas et al. has questioned whether the results have been confounded by biases due to ancestral origins (Belgard et al., 2014, Mol Psychiatry; 19(4):405-7; and Skafidas et al., 2014, Mol Psychiatry; 19(4):504-10). An additional limitation of genomic studies is that the results of environmental influences on the child and/or mother are not discernible. Metabolomics is more sensitive to biochemical changes caused by even subtle environmental influences and therefore can complement genomic approaches by addressing some of these factors that are closer to phenotype.

Given the complexities of the genetic environment of ASD, metabolomic profiling may provide an alternative path to developing early diagnostic tests. Previous metabolic studies of ASD have used biological matrices such as cells, organelles, urine and blood, and have implicated a wide number of metabolites including fatty acids, sterols, intermediary metabolites, phospholipids, and molecules associated with oxidative stress (El-Ansary et al., 2011, Lipids Health Dis; 10:62; James et al., 2009, Am J Clin Nutr; 89:425-430; Lee and Tierney, 2011, Autism Res Treat; 2011:653570; Damodaran and Arumugam, 2011, Redox Rep; 16:216-222; and Yap et al., 2010, J Proteome Res; 9:2996-3004). Two recent reports highlight the potential use of metabolomic analysis of urine to identify signatures of ASD. One study used 1H-NMR methods and showed changes in metabolites associated with the tryptophan/nicotinic acid metabolic pathway, sulphur and amino acid pathways, as well as microbial metabolites implicating the involvement of microbial metabolism in the etiology of ASD (Yap et al., 2010, J Proteome Res; 9:2996-3004). Ming et al. used a combination of liquid- and gas-chromatography based mass spectrometry methods to identify changes in a number of amino acids and antioxidants such as carnosine, as well as confirming the changes associated with altered gut microbiomes (Ming et al., 2012, J Proteome Res; 11:5856-5862).

Measurement of metabolites offers an excellent opportunity to identify differences in small molecule abundance that may have the ability to characterize some forms of ASD. High resolution mass spectrometry (HRMS) is not only a very sensitive detection method for small molecule metabolites, it also provides accurate mass data that aids in metabolite identification through molecular formulae determination (Dunn et al., 2005, Analyst; 130:606-625). HRMS offers an additional distinct advantage in the ability to distinguish between compounds with the same nominal mass (isobaric compounds), providing enhanced chemical formula and structure information (Gross, 1994, J Am Soc Mass Spectrom; 5:57).

Unfortunately there is not one universal chromatographic mass spectrometric technique capable of detecting all of the metabolites in blood. To identify novel potential biomarkers associated with ASD, it is necessary to facilitate broad metabolite detection coverage. Toward this goal, we applied an orthogonal approach to chromatographic separation, mass spectral ionization and detection (Bruce et al., 2008, Anal Biochem; 372:237-249). The current study employed multiple chromatographic mass spectrometric metabolomic methods including gas chromatography-mass spectrometry (GC-MS) and liquid chromatography-high resolution mass spectrometry (LC-HRMS) to discover a wide range of metabolites in blood plasma samples that were able to differentiate TD individuals from those with ASD. Subsequently, tandem mass spectrometry (MS-MS) experiments were employed to aid in structural confirmation of the metabolites discovered by LC-HRMS.

This example performed a broad evaluation of small molecules in blood plasma to discover metabolites that may lead to biomarkers associated with ASD. Univariate, multivariate and machine learning methods were employed to determine if metabolites or groups of metabolites exhibiting statistically significant abundance differences can be used as biomarkers to distinguish children with ASD from TD individuals.

**Methods**

**Subject Samples**

The experimental subjects were initially recruited through the UC Davis M.I.N.D. Institute Clinic, Regional Centers, referrals from clinicians, area school districts and community support groups such as Families for Early Autism Treatment (FEAT), and were limited to a narrow age range of 4-6 years (see Table 1). Typically developing participants (N=30) were recruited from area school districts and community centers. All facets of the original study were approved by the University of California at Davis Institutional Review Board (IRB). Written informed consent was obtained from the parent or guardian of each participant and data were analyzed without personal information identifiers. Following informed consent, subjects completed diagnostic and psychological measures. Study participants with ASD (N=52) were enrolled under inclusion criteria consisting of a diagnosis of autism spectrum disorder based on the DSM-IV criteria determined by an experienced neuropsychologist (BAC), which was further corroborated by the following measures using research reliable clinicians: the Autism Diagnostic Observation Schedule-Generic (ADOS-G) provides observation of a child's communication, reciprocal social interaction, and stereotyped behavior including an algorithm with cutoffs for autism and autism spectrum disorders.

The Autism Diagnostic Interview-Research (ADI-R) is a comprehensive, semi-structured parent interview that assesses a child's developmental history and relevant behaviors characteristic of ASD and generates a diagnostic algorithm for children with ASD. Based on the DSM-IV criteria (American Psychiatric Association (2013) Desk Reference to the Diagnostic Criteria from DSM-5, 5th ed. Washington, D.C.: American Psychiatric Association), only children with strictly defined autistic disorder were enrolled whereas children with pervasive developmental disorder—not otherwise specified (PDD-NOS) or Asperger Syndrome were excluded from the study. The Social Communication Questionnaire (SCQ) was used as a screening tool to ensure the absence of symptoms of ASD in the TD control children. The patients recruited for this study were primarily Caucasian and the ages were similar between groups. However, the participants with autism had lower IQ scores than the typically developing subjects (Corbett et al., 2007, Mol Psychiatry; 12:292-306; and Ashwood et al., 2011, PLoS One; 6:e19299).

The exclusion criteria for all subjects included the presence of Fragile X or other serious neurological (for example, seizures), psychiatric (for example, bipolar disorder) or known medical conditions such as autoimmune disease and inflammatory bowel diseases/celiac disease. All subjects were screened via parental interview for current and past physical illness. Children with known endocrine, cardiovascular, pulmonary, and liver or kidney disease were excluded from enrollment in the study. Dietary restriction for participation in the study was not required with the exception of an overnight fast. Participation in the study required two clinical visits for behavioral assessment and blood draws.

Regarding patient medication, 18 out of 52 of the subjects with ASD in this study were taking medications which included risperidone (5 subjects), sertraline (3 subjects), aripiprazole (2 subjects), antihistamines (2 subjects), antivirals (2 subjects), antifungals (2 subjects), and various other less frequent drugs. Three of the 30 typical subjects were taking medications, which included methylphenidate (1 subject), albuterol (1 subject) and loratadine (1 subject). Ten of the 52 ASD subjects were on a gluten and/or casein-free (GFCF) diet. Importantly, blood draws were administered prior to morning administration of any medication.

Samples were collected on Thursday morning visits to the M.I.N.D. Institute over a period of 13 months. Blood was drawn into a 9.6 mL EDTA vaccutainer tube by an experienced pediatric phlebotomist between the hours of 8 and 10 AM following an overnight fast. Tubes were immediately inverted 6 to 8 times to assure mixing with the anticoagulant and placed on ice. Immediately after serum separation and aliquoting, samples were sent on the morning of the draw via courier with a barcode label, wrapped tube cap with a strip of parafilm; bubble wrapped then set in a biohazard bag which was placed inside a carrier between coolant packs. Samples were stored at −80° C. This original sample set was derived from 87 children. Upon review, 5 samples were removed after visual inspection and observation of overt hemolysis. The final 82 samples used in these studies originated from 52 children with ASD and 30 children in the TD group. The children were chosen so that the age and gender distributions were similar across the groups. There was no statistical difference in age between ASD cases and the typical developing children for the current study (Welch's t-test P=0.25).

A training set of 61 of the 82 samples was used for univariate and multivariate analysis to build the classification models. The remaining 21 samples were designated as an independent, validation test set. These 21 samples were not utilized in the selection of features or the development of the classification models and represent an independent set of samples to assess the robustness of the classification model.

**Sample Preparation for LC-MS**

Plasma samples were split into 50 μl aliquots and stored at −80° C. prior to metabolite extraction. Samples were kept on ice during these procedures. Samples were randomized into three batches for the LC-HRMS analysis such that diagnosis, IQ, age and ethnicity were equally distributed in each batch. Small molecules were extracted from 50 μL plasma aliquots using 450 μL of 8:1 methanol:water solution at −20° C. (Jiye et al., 2005, Anal Chem; 77:8086-8094). The extraction solution also contained internal standards. The samples were agitated for 10 minutes at 2 to 8° C. then centrifuged at 18,400×G for 20 minutes at 4° C. to remove the precipitant. The supernatant was transferred to a fresh tube and the centrifugation step was repeated to remove any residual precipitate. After the final centrifugation, 450 μL of supernatant was transferred to a fresh tube then evaporated to dryness in a SpeedVac, then resolublized in 45 L of a 50:50 mixture of 0.1% formic acid in acetonitrile: 0.1% formic acid, also containing internal standards. This solution was then transferred to a high performance liquid chromatograph (HPLC) autosampler injection vial for LC-HRMS analysis.

**Mass Spectrometry**

Both targeted GC-MS as well as untargeted LC-HRMS were employed for better metabolome coverage. Four untargeted LC-HRMS methods were used including C8 or HILIC chromatography coupled to electrospray ionization in both positive and negative ion polarities, resulting in 4 separate data acquisitions per sample. LC-HRMS methods were developed and tested prior to the evaluation of the clinical patient samples to optimize the breadth of coverage of small molecule metabolites.

**Liquid Chromatography High Resolution Mass Spectrometry**

LC-HRMS was performed using an Agilent G6540 Quadrupole Time of Flight (QTOF) LC-HRMS system consisting of an Agilent 1290 HPLC coupled to a high resolution (QTOF) mass spectrometer. Electrospray ionization (ESI) in both positive and negative ion modes was employed using a dual ESI source under high-resolution exact mass conditions. For Hydrophilic Interaction Liquid Chromatography (HILIC), a Waters Acquity ultra high performance liquid chromatography (UPLC) BEH Amide column with dimensions 2.1×150 mm, 1.7 μM particle size was used and maintained at 40° C. Data was acquired for each sample for 29 minutes at a flow rate of 0.5 mL/minute using a solvent gradient with 0.1% formic acid in water and 0.1% formic acid in acetonitrile. A 2 μL aliquot of sample was injected. For C8 chromatography, data was acquired for each sample for 50 minutes at a flow rate of 0.5 ml/minute using a gradient with 0.1% formic acid in water and 0.1% formic acid in acetonitrile. An Agilent Zorbax Eclipse Plus C8 2.1×100 mm, 1.8 μM particle size column was used and maintained at 40° C. A 2 μL aliquot of sample was injected.

**Gas Chromatography—Mass Spectrometry**

GC-MS analyses were performed as described in Fiehn et al. (Fiehn et al., 2008, Plant J; 53:691-704). GC-MS data was acquired using an Agilent 6890 gas chromatograph coupled to a LECO Pegasus IV TOF mass spectrometer. Metabolite identification was done by comparing sample data to a database of over 1,000 compounds identified by GC-MS that includes mass spectra, retention indices, structures and links to external metabolic databases.

**Metabolite Chemical Structure Confirmation by LC-HRMS-MS**

The chemical structures of key metabolites were further confirmed using tandem mass spectrometry (LC-HRMS-MS) methods with chromatographic conditions identical to those used for their discovery. LC-HRMS-MS analyses were performed on an Agilent QTOF mass spectrometer for patient samples and/or, reference blood samples with collision energy conditions optimized to obtain the highest quality product ion spectra. The resulting product ion spectra were then compared to MS-MS spectra available in public spectral databases such as METLIN (Smith et al., 2005 Ther Drug Monit; 27:747-751), MassBank (Horai et al., 2010, J Mass Spectrom; 45:703-714) and Stemina's own SteminaMetDB database.

**Data Analysis**

**LC-HRMS Data Preprocessing**

Raw mass spectral data total ion chromatograms and internal standard extracted ion chromatograms were initially examined for quality criteria established during method development such as abundance thresholds, retention time peak shape consistency. Data files exhibiting chromatograms that met outlier criteria were removed from further analysis. Raw data were converted to open source mzData files (Orchard et al., 2007, Proteomics; 7:3436-3440). Peak picking and feature creation were performed using open source software library XCMS (Smith et al., 2006, Anal Chem; 78:779-787) then deviations in retention times were corrected using the obiwarp algorithm (Prince and Marcotte, 2006, Anal Chem; 78:6140-6152) based on a non-linear clustering approach to align the LC-HRMS data. Mass features were generated using the XCMS density based grouping algorithm then, missing features were integrated based on retention time and mass range of a feature bin using iterative peak filling. A “mass feature” (also abbreviated here as “feature”) is a moiety detected by the mass spectrometer that is defined by the two properties of 1) the detected mass-to-charge ratio (n/z) and 2) the chromatographic retention time.

A series of data filters were then employed to remove features exhibiting low abundance levels and those resulting from background noise, fragments and contaminants from subsequent data analyses. To reduce LC-HRMS batch variations in feature detection, the abundance values were then normalized by sample to the experiment-wide median area of spiked-in internal reference standards. The integrated areas of the normalized mass features from the GC-MS and LC-HRMS platforms were combined into a single dataset. The 4572 features for the training set of samples that passed preprocessing filters.

**Training and Independent Validation Sets**

The 82 patient samples (52 ASD and 30 TD samples) were split into two sets, (1) a training set of 61 samples (39 ASD and 22 TD) for identification of statistically significant features and classification modeling and (2) a 21-sample independent validation set (13 ASD and 8 TD) used to evaluate performance of the classification models. This was accomplished by randomizing the samples using the diagnosis, patient IQ, and gender these training and validations sets so that each set contained a similar proportion of factors used in randomization. The validation sample set was withheld from the univariate filtering and model development process to act as an independent external sample set to evaluate model performance.

**Univariate Filtering of Mass Features**

T-tests were used to reduce the overall feature set, the potential for over-fitting, and increase the biological interpretability of the predictive signature (Haury et al., 2011, PLoS One; 6:e28210). The integrated areas of mass features normalized to internal standards (IS) from the GC-MS and LC-HRMS platforms were combined into a single dataset. The 4572 features passing the preprocessing filters for the training set of samples were further filtered using Welch T-tests under the null hypothesis that no difference in mean integrated areas of a mass feature is present between the experimental classes, and the alternative hypothesis that there is a difference in mean integrated areas between ASD and TD training set samples to identify differential features. For each feature that exhibited a statistically significant change with an uncorrected p value<0.05, its extracted ion chromatogram (EIC) of was reviewed for consistency of integration across samples, peak shape, and a minimum peak height requirement of >3000. Features passing this EIC quality review process were then utilized in the classification modeling. False discovery rates (FDRs) were calculated using the Benjamin-Hochberg method of p-value correction (Benjamini and Hochberg, 1995, J R Stat Soc Ser B; 57:289-300).

**Classification Modeling**

Model development was performed with two primary goals: to robustly rank the importance of metabolites in discriminating ASD using a VIP (Variable Importance in the Projection) score index and to identify the minimum set of predictive metabolites needed to reach the highest levels of differentiation of the ASD and TD experimental classes. Models were created by training a Partial Least Squares Discriminant Analysis (PLS-DA) or Support Vector Machine (SVM) classifier using the entire 61-sample training set. The modeling techniques PLS-DA as well as SVM with a linear kernel (Wold, 1985, “Partial least squares,” In: Kotz S, Johnson N L, editors. Encyclopedia of statistical sciences. New York: Wiley, Vol. 6. pp. 581-591; and Cortes and Vapnik, 1995, Mach Learn; 20:273-297) were both utilized to demonstrate that the molecular signature can be predictive using multiple approaches. Partial Least Squares (PLS) and SVM classification models were created using the R package Classification and Regression Training “caret” version 5.17-7 (Kuhn, 2008, J Stat Softw; 28:1-26). Receiver operator Curve (ROC) analysis was performed using the R package ROCR version 1.0-5 (Sing et al., 2005, Bioinformatics; 21:3940-3941).

A nested cross validation (CV) approach (FIG. 1) was used to meet the first objective of model development—a robust measure of feature VIP scores. Feature robustness was measured by resampling the training set 100 times using an 80:20 split into 49-sample CV training and 12-sample CV test sets. VIP scores were calculated for each of the 100 resamples and the most informative features at each resample was identified by backwards recursive feature elimination (in 20-feature steps) using on Area Under the ROC Curve (AUC). The most informative set of features was then used to predict each CV test set. The VIP scores were averaged across the 100 resamples to create the VIP index for each feature. The classification performance metrics of the CV test sets were averaged across resamples to understand potential future performance.

The second objective of the classification modeling approach was to identify the minimum number of features with the highest level of classification accuracy. This objective was met using feature subsets based on the VIP score index and evaluating the subset performance in validation test set of samples. The classification models were created using the entire 61 sample training set and by stepping through features. The feature stepping process utilized the 20 top VIP features then added the next 20 highest weighted features until all 179 features were evaluated. Performance metrics (Accuracy, Sensitivity, Specificity, and ROC analysis) based on the prediction of the 21 sample independent validation set for assessment of the molecular signature at each feature subset bin size (see Table 4).

Feature annotation (assignment of putative chemical structures) was carried out for each of the features contained within the feature set(s) that performed best in the models(s). Annotation was accomplished by comparing n/z value of each mass feature to the m/z value of common ESI adducts contained in public chemical databases and/or Stemina's internal metabolite database. The molecular formulae of the mass features with putative annotations were then input into the “Find by Formula” (FBF) algorithm in the Agilent MassHunter Qualitative Analysis software which tests whether the mass spectra for a given feature is a reasonable match with the proposed formula. In most cases, the annotations for any feature with a median FBF score of less than 70, a retention time difference greater than 35 seconds or which was present in less than 50% of the data files were not included for further analysis due to lack of confidence in the annotation.

All mass features that were annotated with chemical identities in that the measured exact mass was consistent (within 20 ppm relative mass error) with one or more chemical structures. These annotations were considered to be putative until the chemical structure of the feature was further confirmed by LC-HRMS-MS.

Features from the GC-MS analysis were identified as described by (Fiehn et al., 2008, Plant J; 53:691-704). This procedure uses comparison of the sample data to spectra of metabolite reference standards that had been previously acquired by the same identical GC-MS method. Therefore, the data analysis and confirmation of the metabolite chemical structures was performed by a simple comparison of the acquired patient sample data to the database. GC-MS data also contained peaks that were unidentified that showed statistically significant changes depending on sample class.

**Results**

The use of multiple analytical methods provided a broad coverage of the metabolome and each method contributed mass features to the model for classification of the children with ASD from the TD controls. Each method was assessed for the unique features it provided. Initially, 10187 mass features were detected by the 5 analytical platforms together. The HILIC LC-HRMS method resulted in the highest number of distinctive mass features in the models, followed by C8 LC-HRMS then GC-MS. Univariate analysis filtering was performed on 4572 features that passed the previous filters. About 60% of the LC-HRMS features were putatively annotated with a chemical structure and 8% (503) of the annotated features passed the FBF procedural criteria. Approximately 36% (142) of the targeted GC-MS features were confirmed metabolites. A breakdown of these results is contained in Table 2.

Data across the 61-sample training set from all analytical platforms were used to identify and robustly rank the features that could be utilized to discriminate plasma samples from children with ASD from samples from typically developing (TD) children. The univariate analysis filtering, as described above, resulted in 389 statistically significant features. An additional 210 features were removed from the analysis after EIC review, leaving 179 features that were moved forward for inclusion in classification modeling. The 179 features comprised 3% of the LC-HRMS and 8% of the GC-MS preprocessed set of features and are shown in Table 6.

**Training Set Model Performance**

SVM and PLS classification methods were used to discriminate between samples from children with ASD and TD children using the 179 selected features as variables and each feature's contribution toward classification was evaluated for future biomarker development efforts. Using the optimal scores from all of the 100 modeling iterations performed for each modeling method (CV Training Set), ROC plots were generated from both the training set and the independent validation test sets to understand model performance. The 100 models generated were averaged and plotted as a function of true response rate versus false positive rate. Both SVM and PLS modeling methods indicated that a metabolic signature could be detected that could classify children with ASD from TD individuals. The SVM model provided AUC values of 0.95 (95% confidence interval (CI) 0.94-0.96) and the PLS model gave AUC values of 0.92 (95% CI 0.91-0.94). To confirm that the model classification accuracies were not random results, the features were also modeled with random permutations of the group diagnosis class labels. These results showed near random classification, with AUC values between 0.52 (95% CI 0.48-0.57) and 0.52 (95% CI 0.49-0.56) for SVM and PLS, respectively, indicating that the features could not discriminate the classes using a randomized data set (FIG. 2).

Anticipating that blood tests for ASD may be more efficient and less expensive if they measure an optimally lower number of metabolites, the classification modeling paradigm also included a feature number optimization in each model, based on the highest resulting AUC. The feature sets were evaluated with the VIP scores of individual features based on their contribution to the most predictive models (Table 4). These data together indicate that not all of the features contributed equally to the models and that the number of features could be reduced by removing those that contributed less while still retaining model accuracy and robustness. As a result, the entire set of 179 features was not required for optimal model performance for either of the modeling methods (FIG. 3). The SVM models that were trained using an 80 feature set exhibited the best combined classification performance metrics (when compared to PLS and other SVM results) with an average accuracy of 90%, an average sensitivity of 92%, an average specificity of 87%, and an average AUC of 0.95 (Table 3).

**Validation Set Model Performance**

Different subsets of features, created based on the weighted VIP scores, were evaluated independently of the outer cross-validation loop using the 21-sample independent validation set. The 80-feature SVM model described above had a classification prediction accuracy of 81%, a sensitivity of 85%, a specificity of 75% and an AUC of 0.84 (FIG. 2, thin, black line). The best performing PLS model, comprised of 140 variables, had an accuracy of 81%, a sensitivity of 85%, a specificity of 75% and an AUC of 0.79 (FIG. 2; thin, gray line; Table 4). The results suggest that at least 40 features are needed to reach an accuracy of 70% and that a range of 80 to 160 features perform well with this independent validation data set.

**Confirmation of Metabolite Chemical Structures**

The chemical identities of the 7 LC-MS mass features that were confirmed by LC-HRMS-MS are shown in Table 5. Included in the metabolites confirmed by LC-HRMS-MS or targeted GC-MS was homocitrulline, which had the greatest statistical significance and the highest rank of all features in both SVM and PLS classification models in this study. Other metabolites showing significant up or down regulation include: aspartate, glutamate, dehydroepiandrosterone sulfate (DHEAS), citric acid, succinic acid, methylhexa-decanoic acid, tetra-decanoic acid, hepta-decanoic acid, isoleucine, glutaric acid, 3 aminoisobutyric acid, and creatinine. These are listed in Table 5 and represent a variety of molecular classes including amino acids, organic acids, sterols, and fatty acids.

Table 6 provides supplementary information of all 179 model features.

Table 7 is a table of the results from the cross-validation (CV) training sets. N is the number of times the bin size performed the best in the training set with the corresponding number of features. Accuracy, sensitivity, specificity, and AUC are the averaged value of the feature bin size.

Table 8 is a table showing classifier performance metrics based on predictions on the independent 21-sample validation set. Classifier performance metrics based on predictions on the independent 21-sample validation set. Feature No. corresponds to the number of the ordered, ranked VIP features that were evaluated.

**Discussion**

The untargeted metabolomic approach described in this example did not possess bias toward possible pathways other than the separation and detection limits of the analytical methods used. This approach has resulted in the discovery of a biochemically diverse set of metabolites that might be useful in distinguishing individuals at risk for ASD.

Identification of Metabolites Previously Associated with ASD

Examples of metabolites showing significant up or down regulation in our study that have been previously associated with autism include:

Tricarboxylic acid cycle associated molecules including citric acid (decreased) and succinic acid (increased) were found to be significantly altered in the ASD participants. Elevations in urinary succinate (Yap et al., 2010, J Proteome Res; 9:2996-3004; and Ming et al., 2012, J Proteome Res; 11:5856-5862) and decreased urinary citrate (Frye et al., 2013, Transl Psychiatry; 3:e220) in children with autism have been reported by others;

Fatty acids have previously been observed to be decreased in the plasma of children with ASD, similar to our observations for methylhexa-, tetra- and hepta-decanoic acids (El-Ansary et al., 2011, Lipids Health Dis; 10:62). Links between saturated fatty acid metabolism and oxidative stress have been reported in erythrocytes in children with ASD (Ghezzo et al., 2013, PLoS One; 8:e66418);

3 aminoisobutyric acid was increased in samples from participants with ASD. This is also consistent with previous findings (Adams et al., 2011, Nutr Metab (Lond); 8:34); and

Creatinine was decreased in children with ASD and is consistent with the findings of Whitely et al., observing similar changes in urinary creatinine in children diagnosed with PDD (Whiteley et al., 2006, Pediatr Int; 48:292-297).

**Evidence for a Role in Mitochondrial Dysfunction in ASD**

Many of the confirmed metabolites are directly associated both with ASD and with aspects of mitochondrial biology. Mitochondrial disease or dysfunction has been proposed to be potentially involved in autism (Marazziti et al., 2012, Eur Rev Med Pharnmacol Sci; 16:270-275). In addition, several metabolites are associated with other processes already proposed to be involved in ASD including oxidative stress (Rossignol and Frye, 2012, Mol Psychiatry; 17:389-401) and energy production (Blaylock, 2009, Altern Ther Health Med; 15:60-67).

Aspartate and glutamate levels in blood were significantly elevated, as has been observed in previous ASD studies (Shinohe et al., 2006, Prog Neuropsychopharmacol Biol Psychiatry; 30:1472-1477; and Moreno-Fuenmayor et al., 1996, Invest Clin; 37:113-128). Mutations in the aspartate/glutamate mitochondrial transporter, SLC25A12, have been previously associated with ASD. This transporter is an important component of the malate/aspartate shuttle, a crucial system supporting oxidative phosphorylation, adenosine triphosphate production, and key metabolites for the urea cycle (Napolioni et al., 2011, Mol Neurobiol; 44:83-92).

DHEAS, the predominant plasma sterol, was found to be increased in children with ASD. DHEA is known to affect mitochondrial energy production through inhibition of enzymes associated with the respiratory chain (Safiulina et al., 2006. Toxicol Sci; 93:348-356) with variable findings in children with ASD (Strous et al., 2005, Eur Neuropsychopharmacol; 15:305-309; and Tordjman et al., 1995, J Autism Dev Disord; 25:295-304).

The branched chain amino acid isoleucine was reduced in samples from children with ASD versus TD children. This has also been observed by others (Arnold et al., 2003, J Autism Dev Disord; 33:449-454). Possible molecular mechanisms would include mutation in the branched chain amino acid kinase dehydrogenase (BCKD-kinase), a mitochondrial enzyme (Novarino et al., 2012, Science; 338:394-397) as well as a role for these amino acids in energy metabolism (Valerio et al., 2011, Aging (Albany N.Y.); 3:464-478).

Glutaric acid levels were elevated. Increased urinary glutaric acid occurs in a variety of neuronal deficiencies such as glutaryl-CoA dehydrogenase (GCDH) deficiency. A significant portion of the glutaric acid metabolism takes place in the mitochondria (Miller and Kölker, 2004, J Inherit Metab Dis; 27:903-910).

The Potential Relationship of the Gut Microbiome with ASD

This potential connection between the gut microbiome and ASD is also receiving considerable attention (Mulle et al., 2013. Curr Psychiatry Rep; 15:337). Metabolomic studies of urine from individuals with ASD have identified molecules such as dimethylamine, hippurate or phenylacetylglutamine that have been associated with the microbiome (Yap et al., 2010, J Proteome Res; 9:2996-3004; and Ming et al., 2012, J Proteome Res; 11:5856-5862). In this study, decreased plasma levels of p-hydroxyphenyllactate were observed. p-hydroxyphenyllactate is a metabolite associated with bifidobacteria and lactobacilli that is known to serve as an antioxidant both in the circulation and tissues (Beloborodova et al., 2012, J Biomed Sci; 19:89).

In addition, levels of aspartate, citrate, creatinine, DHEA-S, hydroxyphenyllactate, indoleacetate, isoleucine glutamate and glutarate were all found to have significant changes distinguishing between ASD and TD individuals, whereas in previous studies of urine metabolites, changes in these compounds were not significant (Ming et al., 2012, J Proteome Res; 11:5856-5862).

**Identification of Previously Unidentified Metabolic Alterations in ASD**

This study has also identified new, previously undescribed potential ASD biomarkers such as homocitrulline, which had the greatest statistical significance and the highest rank of all features in both SVM and PLS classification models. Homocitrulline is a poorly understood molecule which is known to be formed inside the mitochondria from lysine and carbamoyl phosphate. Homocitrullinuria (HHH) syndrome patients, with a urea cycle deficiency related to ornithine translocase (SLC25A15) deficiency, have higher urinary homocitrulline levels, and can exhibit behavioral abnormalities similar to ASD such as developmental delay, ataxia, spasticity, learning disabilities, cognitive deficits and/or unexplained seizures (Palmieri, 2004, Pflugers Arch; 447:689-709). From these data it is plausible to suggest that changes in the urea cycle function may be related to the decreases in homocitrulline we observed in plasma.

Physicians and clinicians with specialized training are currently able to diagnose children with ASD by two years of age using behavioral characteristics. It is increasingly recognized, however, that detection of ASD at an earlier age results in better patient and family outcomes (Payakachat et al., 2012, Expert Rev Pharmacoecon Outcomes Res; 12:485-503; and Thompson, 2013, J Appl Res Intellect Disabil; 26:81-107). Therefore, a biologically-based blood test for ASD that can be administered at an early age would be highly beneficial to patients, families and medical providers. The current study profiled metabolites in blood plasma to evaluate the possibility that differences in the abundance of identified metabolites might provide a signature that could prove useful in distinguishing individuals at high risk for developing ASD. The cohort of subjects enrolled in this study was carefully assembled to reflect a diagnosis of ASD by strict research criteria. Beyond careful clinical diagnosis, great pains were taken to insure that fasting blood collection was obtained at the same time for all study participants and that complicating factors such as illness were minimized.

Metabolomics determines changes in small molecule metabolites that are reactants and products of endogenous biochemical processes as well as small molecules derived from diet, the gut microbiome and contact with the environment. Perturbations in their abundance can result not only from genomic and proteomic influences, but environmental and epigenetic influences as well. A metabolomic approach may therefore provide enhanced predictive results by keying in on common, end stage metabolites rather than on specific genomic or proteomic determinants. Since no single analytical method is capable of assessing all metabolites, we optimized and employed chromatographic methods linked to multiple mass spectrometric ionization methods that separate and detect molecules based on different chemical properties. Each of these methods provided features used by the classification models in our study.

Two independent statistical classification methods (PLS and SVM) were employed to determine the most influential metabolites and mass features that could be used to discriminate between ASD and TD individuals. Both classification modeling methods yielded relatively similar results with respect to maximum prediction accuracy of about 81% as evaluated by an independent validation sample set. Having established that predictive classification models could be obtained, we then used the recursive feature elimination approach to establish the minimal numbers of features needed for a predictive model. Interestingly, several of the key features for classification were common between the two methods indicating their importance in the development of future blood based diagnostics.

**CONCLUSIONS**

This example demonstrates that a profile of altered metabolites in the blood plasma of children can be detected by the combination of several MS-based metabolomic analyses. Statistical models developed from the derived metabolic data distinguished children with ASD from TD individuals with accuracy better than 80%. The study used a well curated set of samples from clinically diagnosed children with ASD and typically developing individuals between 4 and 6 years of age. Further research is being carried out to confirm the chemical structures of more of the discovered metabolites and to determine which are the most robust for determining ASD risk by evaluating them in larger and younger patient populations.

**Example 2**

**Additional Confirmed Metabolites**

Using the procedures described in more detail in Example 1, a second set of ASD samples (the MIND2 study) was assayed. This study population included samples from 180 typical (69% male; average age 3.1 years; developmental status 106) and 93 autistic subjects (83% male; average age 3 years; developmental status 62). The dietary status of all subjects when samples were taken was fed. Citrate was used as an anticoagulant.

The additional metabolites listed in Table 9 below exhibit a statistically significant difference between autistic and non-autistic individuals have been confirmed. Briefly, for sample preparation and mass spectrometry: small molecules were extracted using 8:1 methanol:water solution at −20° C.; samples were centrifuged to remove precipitate, evaporated to dryness then solubilized for LC-HRMS analysis; targeted GC-MS and untargeted LC-HRMS (C8 or HILIC chromatography) methods were optimized for metabolome coverage. LC-HRMS was performed using an Agilent G6540 QTOF LC-HRMS system; and electrospray ionization (ESI) in both positive and negative ion modes under high resolution exact mass conditions; and GC-MS data was acquired using an Agilent 6890 gas chromatograph coupled to a LECO Pegasus IV TOF MS.

A comparison of the metabolic features identified in the present example with those identified in Example 1 shows the identification of DHEAS, lysophospholipids, oxidized fatty acids, isoleucine, succinic acid, and cysteine as associated with ASD in both studies.

Using the non-targeted, MS-based metabolomic analysis of blood plasma, as described in more detail Example 1, a larger set of patients will be studied to identify and validate biomarkers for diagnostic tests to detect ASD earlier and improve patient outcomes. The biomarkers will be used to gain new insight into biochemical mechanisms involved in metabolic subtypes of ASD.

The biomarkers described herein will be used to as biomolecular targets will for the identification of new modes of therapy, and will be used to obtain insights into personalized treatment recommendations.

**Example 3**

**Metabolic Signatures for High Functioning Autism and Low Functioning Autism**

As described in more detail in Example 1, using a number of supervised and unsupervised statistical methods, a metabolic signature that was highly predictive of ASD was identified. In samples from a population of 70 patients with ASD and 30 typically developing age-matched controls, the samples Model Accuracy Sensitivity Specificity were divided into High Functioning Autism (HFA) (IQ>70; n=33), Low Functioning Autism (LFA) (IQ<70, n=36), and Typically developing (TD) children (n=34) with an age range of 4-6 years (average 5.4 years).

Briefly, for this our analysis, 80% of the samples were included as a model training set, with the remaining 20% reserved for the blinded test set. Samples were analyzed using 5 different chromatographic-mass spectrometry based methods designed to orthogonally measure a broad range of small molecules that can ultimately be associated with metabolites and biomarkers. The top 266 statistically significant unique metabolic features were used to develop classification models that were evaluated relative to the test set. The models evaluate the predictive capacity of metabolic signatures to discriminate between individuals with autism and typical individuals, LFA and typical individuals, and HFA and typical individuals (Table 10).

FIG. 5 shows the overlap of biometabolic signatures between High Functioning Autism (HFA) and Low Functioning Autism (LFA) populations, Autism (Aut) and HFA populations, and Autism and LFA populations.

For 11 of the 39 features of the overlap of LFA with Aut shown in FIG. 5, additional putative identifications (PAMs) include:

HILICneg_M526T303: LysoPE(18:0/0:0), GPEtn(18:0/0:0), and LysoPE(0:0/18:0). HILICneg_MI51T65: 2-Hydroxyethyl methacrylate, HEMA 3-Oxohexanoic acid; 3-Oxohexanoate, 3-Oxohexanoic acid, 2-Ketohexanoic acid, 3-keto-n-caproic acid, (R)-3-methyl-2-oxo-Pentanoic acid, 2-Oxohexanoic acid; 2-Oxohexanoate, 2-Methyl-3-ketovaleric acid, Adipate semialdehyde, Hexan-1-one-6-carboxylate; 6-Oxohexanoate, Ketoleucine, 2-oxo-3-methylvaleric acid, 5-Oxohexanoic acid, 5-Oxohexanoate, 4-Acetylbutyric acid, 3-Methyl-2-oxovaleric acid, 6-Hydroxyhexan-6-olide, 6-Hydroxy-6-hexanolactone, 1-Oxa-2-oxo-3-hydroxycycloheptane, 5-keto-n-caproic acid, 3-oxo-4-methyl-pentanoic acid, 4-keto-n-caproic acid, Ethyl 3-oxobutanoate, Ethyl acetoacetate, Mevalonolactone, 2oxo-3R-methyl-pentanoic acid, (R)-Pantolactone, (R)-Pantoyl lactone, (3R)-Dihydro-3-hydroxy-4,4-di methyl-2(3H)-Furanone, and 2-oxoisocaproic acid.

C8neg_M117T127: Butanone, Butanal, Tetrahydrofuran, beta-hydroxybutyrate, 2-Hydroxyvaleric acid, b-Hydroxyisovaleric acid, 3-Hydroxy-2-methyl-[R—(R,R)]-butanoic acid, 3-Hydroxy-2-methyl-[R—(R,S)]-butanoic acid, DL-a-Hydroxyvaleric acid, L-alpha-Hydroxyisovaleric acid, (S)-2-Ethyl-3-hydroxypropionic acid, a-hydroxyisovalerate, 2-Ethylhydracrylic acid, 2-Methyl-3-hydroxybutyric, acid 4-hydroxy-valeric acid, 5-Hydroxypentanoate, and 5-hydroxy valeric acid.

HILICneg_M117T61: Tetrahydrofuran, Butanone, Butanal, 5-hydroxy valeric acid, 5-Hydroxypentanoate, 2-Methyl-3-hydroxybutyric acid, 2-Ethylhydracrylic acid, 2-Hydroxyvaleric acid, DL-a-Hydroxyvaleric acid, L-alpha-Hydroxyisovaleric acid, 4-hydroxy-valeric acid, b-Hydroxyisovaleric acid, beta-hydroxybutyrate, 3-Hydroxy-2-methyl-[R—(R,S)]-butanoic acid, a-hydroxyisovalerate, 3-Hydroxy-2-methyl-[R—(R,R)]-butanoic acid, and (S)-2-Ethyl-3-hydroxypropionic acid.

HILICneg_M117T67: Pyruvaldehyde, Acrylic acid, Malondialdehyde, Propenoate, Acrylic acid, Acrylate, 2-Propenoic acid, Vinylformic acid, Erythrono-1,4-lactone, Methyl oxalate, Methylmalonic acid, 2(3H)-Furanone, dihydro-3,4-dihydroxy, Threonolactone, and Succinic acid.

FIG. 6 shows the abundance in both autistic (A) and typical (T) subjects of the five biometabolic features in common between HFA, LFA, and Aut populations for use in diagnosis of autism.

FIG. 7 shows the abundance in autistic (A) and typical (T) subjects of eleven of the thirty-nine biometabolic features in common between LFA and Aut populations for use in diagnosis of autism.

FIG. 8 shows the abundance in autistic (A) and typical (T) subjects of the thirteen biometabolic features in common between HFA and Aut populations for use in diagnosis of autism. FIG. 9 shows the abundance of additional biometabolic features in High Functioning Autism (HFA), Low Functioning Autism (LFA), Autism (Aut), and typical populations. And, FIG. 10 shows combined features from all analytical methods. FIG. 11 shows the distribution for citrulline (the HILIC(+) feature M190T512) in HFA versus typical populations, LFA versus typical populations, and LFA+LFA versus typical populations. FIG. 12 shows the GCMS distribution for feature S123 in HFA versus typical populations, LFA versus typical populations, and LFA+LFA versus typical populations.

The increase in classification accuracy observed in LFA versus TD which was 16% greater the HFA versus TD model suggests that more severe forms of the disorder have a marked impact on metabolism. The overall classification accuracy is a global measure of the model's performance toward accurate diagnoses. Sensitivity is the percentage of individuals correctly classified as diagnosed with ASD and higher values indicate the probability that an individual with ASD will be correctly diagnosed, leading to fewer false negative diagnosis. The measure of specificity indicates the probability that a typical individual will be correctly classified as typical and not as having ASD. Putative annotation of the mass features shows a broad variety of metabolites are represented in the models including fatty acids, phospholipids, amino acids, intermediary, and others. For example, isoleucine was observed at significantly lower levels in the ASD patients, showing an average abundance ratio of 0.55 for LFA/TD and 0.70 for HFA/TD. This is consistent with the identification of a point mutation in a gene encoding the branched-chain amino acid dehydrogenase kinase (BCKDK), which causes degradation and depletion of the branched chain amino acids leucine, isoleucine and valine, leading to a form of autism with epilepsy (Novarino et al., 2012, Science; 338:394-397).

This example has identified a metabolic signature in blood plasma able to classify high and/or low functioning autistic individuals from typical individuals through a comprehensive metabolomic analysis.

Additional blood samples from 295 additional patients obtained as part of the Autism Phenome Project (APP) will be evaluated (⅔ are diagnosed with ASD and the remaining third are typically developing children). These samples are from children aged 2 to 3.5 years. Evaluating patient samples from these younger children will allow the identification of biomarkers which will diagnose patients at an earlier age providing potentially greater impact on patient outcomes. The APP is a longitudinal study, plasma samples have been collected from these children when they reached 5 years of age. These samples will provide a valuable resource for future studies to investigate the stability of metabolomics signatures of ASD over early childhood. Inclusion criteria for APP subjects are ambulatory, no suspected vision or hearing problems, motor milestones not significantly delayed, and body weight greater than 20 pounds. Exclusion criteria included presence of a fragile health condition preventing valid participation in the assessment, any family disorders or diseases that might complicate the comparison group (for example, a parent with bipolar disease, cousin or sibling with autism), and typically developing children with abnormal MSEL scores.

**Example 4**

**Additional Cohorts**

This example will continue the work of the previous examples, which successfully discovered 179 metabolites (or groups of metabolites) in blood that can identify patients with ASD with over 80% accuracy. Biomarkers that can be measured in the blood of patients may allow a metabolic understanding of the disorder and earlier diagnosis than behavioral analysis which is the primary method of diagnosis today. This example will directly measure hundreds to thousands of metabolites in the plasma of individuals with ASD and compare these measurements to those obtained from non-autistic individuals of a similar age. A non-targeted metabolomic analysis approach will be used to study banked blood samples from a very well characterized set of samples at the MIND Institute at UC-Davis. Ultimately, this example will inform whether abnormal levels of some metabolites are present in the plasma of individuals with ASD compared to typical patients. The metabolites will be identified and will be mapped to metabolic pathways that will simultaneously help develop a better understanding of the mechanisms of ASD and provide potential targets for future therapeutic development. Ultimately, the identified metabolites can be transferred to other types of platforms such as a clinical diagnostic kit.

As shown in the previous examples, samples from these cohorts demonstrated that combinations of metabolites found in plasma samples form signatures which can identify individuals with ASD. With this example additional samples from several cohorts of well-characterized subjects with ASD and age-matched typically developing control children will be assayed.

The complete disclosure of all patents, patent applications, and publications, and electronically available material (including, for instance, nucleotide sequence submissions in, e.g., GenBank and RefSeq, and amino acid sequence submissions in. e.g., SwissProt, PIR, PRF, PDB, and translations from annotated coding regions in GenBank and RefSeq) cited herein are incorporated by reference. In the event that any inconsistency exists between the disclosure of the present application and the disclosure(s) of any document incorporated herein by reference, the disclosure of the present application shall govern. The foregoing detailed description and examples have been given for clarity of understanding only. No unnecessary limitations are to be understood therefrom. The invention is not limited to the exact details shown and described, for variations obvious to one skilled in the art will be included within the invention defined by the claims. All headings are for the convenience of the reader and should not be used to limit the meaning of the text that follows the heading, unless so specified.

