# DESCRIPTION

## FIELD OF THE INVENTION

Cumulus cell (CC) gene expression is being explored as an additional method to morphological scoring to choose the embryo with the highest chance to pregnancy. The present invention relates to a novel method of identifying biomarker genes for evaluating the competence of a mammalian oocyte in giving rise to a viable pregnancy after fertilization, based on the use of live birth and embryonic development as endpoint criteria for the oocytes to be used in an exon level analysis of potential biomarker genes. The invention further provides CC-expressed biomarker genes thus identified, as well as prognostic models based on the biomarker genes identified using the methods of the present invention.

## BACKGROUND TO THE INVENTION

Single embryo transfer (SET) is the preferred treatment to limit multiple pregnancies after ART. In order not to compromise the carry home baby rate, the selection of the embryo for transfer in the first cycle becomes even more important. Next to the existing criterion based on morphology, other methods are currently under investigation. The use of quantitative gene expression measurements in cumulus cells (CC), which are in close contact with the oocyte during growth and maturation, seems a promising method (Huang and Wells 2010). Since the first published study on the subject where CC expression could be related to embryo development (McKenzie et al. 2004), several other studies have investigated this possibility and try to relate CC expression to different endpoints. Examples of endpoints investigated are: embryo development (Anderson et al. 2009; Cillo et al. 2007; Feuerstein et al. 2007; Hasegawa et al. 2005; van Montfoort et al. 2008; Zhang et al. 2005), aneuploidy stage of the oocyte (Fragouli et al. 2012b), oocyte nuclear maturity stage (Ouandaogo et al. 2011) and probably the most important from a patient perspective: pregnancy outcome (Assidi et al. 2011; Assou et al. 2008; Gebhardt et al. 2011; Wathlet et al. 2012; Wathlet et al. 2011). Confirmation of results between different studies does not seem obvious in the analysis of CC gene expression. In the current literature not many genes were found in common in different studies. For example, hyaluronan synthase 2 (HAS2) was higher expressed in good quality embryos compared to low embryo morphology in two studies (Cillo et al. 2007; McKenzie et al. 2004), but could not be related to embryo morphology in two other studies (Anderson et al. 2009; Gebhardt et al. 2011). Divergences can be due to a different experimental design, with different endpoints, but gene expression can be influenced by known factors such as the stimulation protocol of the patients (Adriaenssens et al. 2009; Adriaenssens et al. 2010; Grondahl et al. 2009) or not yet assessed factors such as culture media used in the different IVF laboratories.

In this study, 47 individual cumulus complexes from 47 Intra-cytoplasmic sperm injection (ICSI) patients were retrospectively analyzed with quantitative real-time polymerase chain reaction (QPCR). Using the current sample set, a pregnancy prediction model from a previous study (Wathlet et al. 2012) was validated for its predictive power. In a next step, in an attempt to search for new genes with a stronger predictive power, new multiparametric models were built considering the 3 genes (ephrin-B2 (EFNB2), calcium/calmodulin-dependent protein kinase ID (CAMK1D), stanniocalcin 1 (STC1)) described earlier and multiple novel genes and splice variants of said genes in mathematical models to predict the oocyte competence (see Tables).

Our patient sample set allowed for an analysis never reported before in literature: CC from oocytes that did not result in pregnancy in the fresh transfer cycle and the CC from their sibling oocytes that resulted in pregnancy after a frozen embryo transfer (FRET) cycle were analyzed (Intra-patient analysis). To our knowledge this is the first study to compare pregnant and non-pregnant CC from the same retrieval cycle in a SET setting as would be done in a final clinical application.

## SUMMARY OF THE INVENTION

The present invention is directed to a method for detecting a biomarker gene or its splice variant for evaluating the competence of a mammalian oocyte to lead to childbirth, to implantation, to form a pre-implantation blastocyst or embryo, and/or to lead to fertilization; said method comprising

using live birth or embryo development as a selection endpoint for the oocytes to be used;

making exon level analysis of gene expression in a micro array experiment of a sample comprising at least one granulosa or cumulus cell associated with the oocyte;

making intra-patient based comparison of said exon level analysis of gene expression; and

consider only p-values <0.05 in a paired t-test as significant for a biomarker gene in being capable for evaluating the competence of a mammalian oocyte to lead to childbirth, to implantation, to form a pre-implantation blastocyst or embryo, and/or to lead to fertilization.

In an alternative embodiment a gene is considered a biomarker gene in evaluating the competence of an oocyte in case of differential expression of said exon level gene analysis, wherein said differential expression is at least 20% different from the exon level expression of said gene in control or reference standard.

Compared to existing methods the method of the present invention differs in the selection endpoint for the oocytes, in the exon level analysis of the genes, and the intra-patient based comparison. Using live birth and embryo development as the selection endpoint for the oocytes to be used in the identification of biomarker genes does indeed differentiate the results of the present finding over the art in which other endpoints, also referred to as ‘intermediate endpoints’, like fertilisation, morphology based embryo quality, or blastocyst development have been used. The relevance of the intermediate endpoints for full oocyte competence is limited as for example oocytes with a good (morphology based) embryo development capacity on day 3 will only result in a pregnancy in 33% of the cases. Choice of the proper endpoint has accordingly been key to come to the present set of biomarkers genes shown to give Positive Predictive Values (PPV's) and Negative Predictive Values (NPV's) of at least 60%.

A further differentiating feature is based on the fact that the gene selection was based on an intra-patient comparison. Doing for example a retrospective cumulus cell analysis of patients that became pregnant or not, the data will be biased by the inter-patient variance influencing the expression of the genes. Cumulus cell gene expression levels are influenced by: patient specific characteristics (e.g.: age, BMI, pretreatment), oocyte quality, and by expression of other genes (Adriaenssens et al. 2010). The competence of an oocyte is also determined by the ability to succeed in different processes. By using intra-patient samples instead, said inter-patient variabilty is suppressed, adding to the identification of the biomarker genes of the present invention shown to be applicable genes that can be used for live birth prediction for patients with different pretreatments.

Finally, the oocyte gene expression analysis is performed as an exon level analysis. As demonstrated in the examples hereinafter, this added to the resolving power of the predictive models. Where the overall expression for some of the genes was shown not to be predictive, looking at the expression level of exons within these genes showed to have a predictive value. When looking at the overall expression of a gene some of the signals associated at exon level will be leveled out with possible loss of interesting markers.

Using the foregoing method a minimal list of predictive biomarker genes has been obtained, and is presented in Table 13 herein below. Accordingly, in a further aspect the present invention provides the foregoing method wherein the exon level analysis of gene expression is performed on the genes of Table 13. This list of genes is to be seen as a reservoir of oocyte competence marker genes that can be used in multiparametric analyses to select the combinations of independent genes with the strongest prediction capacity for live birth. As demonstrated in the examples herein below, the present inventors successfully applied combinations of these genes in predictive models of oocyte competence after different types of Assisted Reproduction Technology (ART) treatment.

Thus in another embodiment the present invention provides a method to detect a biomarker gene combination model of genes or splice variants using cumulus cell gene expression from oocytes in a screening experiment comprising:

a. Detecting at least 2 biomarker genes identified using the foregoing method;

b. Using a two-tailed t-test to assess the correlation of the biomarker gene expression combinations of said at least 2 biomarker genes with the oocytes capacity for live birth, and retaining combinations of said at least 2 biomarker genes with a cut off value of Type I p<0.05 as a model to predict the competence of a mammalian oocyte in being capable of live birth or for evaluating embryo or blastocyte in vitro-development;

c. Making a stepwise multiparametric regression analysis to establish whether splice variants of said at least 2 biomarkers have a type III p value <0.3 and adding said splice variant to the model retained in step b; and

d. Determining the overall p value of the model comprising the splice variants of step c, and retaining said splice variants in said model when the overall p value is further reduced.

As evident from the examples hereinafter, in one embodiment the foregoing screening experiment is a micro array experiment or a QPCR experiment.

In an alternative approach one departs from combinations of at least 2 biomarker genes of the genes as presented in Table 13 instead. In a particular embodiment the genes used to establish a biomarker gene expression model are selected from the 11 genes listed in Table 8 below. Using the thus identified biomarker genes and combination of biomarkers genes, the present invention further provides the use of said genes and gene combinations in an in vitro method of evaluating the competence of a mammalian oocyte to to lead to childbirth, to implantation, to form a pre-implantation blastocyst or embryo, and/or to lead to fertilization. Thus, in a further embodiment the present invention provides an in vitro method for evaluating the competence of a mammalian oocyte to lead to childbirth, to implantation, to form a pre-implantation blastocyst or embryo, and/or to lead to fertilization; said method comprising the steps of:


- - determining the level of a biomarker gene expression identified
    using the above mentioned method; or the level of biomarker gene
    expression of a combination of biomarkers identified using the
    foregoing method to detect a biomarker gene combination model of
    genes or splice variants, in a sample comprising at least one
    granulosa or cumulus cell associated with the oocyte; and
  - evaluating the competence of the oocyte to lead to childbirth, to
    implantation, to form a pre-implantation blastocyst or embryo,
    and/or to lead to fertilization; based on said expression level(s).

In the foregoing method, evaluating the competence of the oocyte is done using a biomarker gene combination model determined using the method of the present invention. In particular using one of the biomarker gene combination models as described herein. In said models the expression levels of the genes will be used in a mathematical formula (infra) providing the probability of pregnancy (P) of said oocyte. In one embodiment, the expression levels used in evaluating the competence of the oocyte are normalized expression levels. Thus in one embodiment the in vitro methods for evaluating the competence of a mammalian oocyte as provided herein, further comprise the step of normalizing the expression levels. In another embodiment it comprises the step of normalizing the expression levels, wherein expression levels of the biomarker genes are normalized by correcting the absolute expression level of a said marker by comparing its expression to the expression of a gene that is not a marker, e.g., a housekeeping gene that is constitutively expressed. In one embodiment the housekeeping gene used in normalizing the biomarker gene expression levels is selected from the group consisting UBC, B2M, actin, GAPDH, HPRT, CPB, G6PD, histone H2A, and mitochondrial ribosomal protein S 18C gene (also known as RNA18S5); in particular UBC or B2M.

In an alternative method the competence of a mammalian oocyte is evaluated by comparing the level of marker gene expression with a control of which the competence is known. Differential expression of said gene is indicative for the competence of the oocyte when there is at least 20% difference in expression level. Thus in one embodiment the competence of a mammalian oocyte is evaluated using one of the biomarker gene combination models as described herein. In said embodiment the in vitro methods may further comprise the step of normalizing the exon level gene expression of said genes; and evaluating the competence of the oocyte to lead to childbirth, to implantation, to form a pre-implantation blastocyst or embryo, and/or to lead to fertilization; based on said normalized expression levels. In another embodiment the oocyte is evaluated by comparing the level of marker gene expression with a control of which the competence is known, optionally using normalized expression levels, and wherein said oocyte is capable to lead to childbirth, to implantation, to form a pre-implantation blastocyst or embryo, and/or to lead to fertilization, when there is at least 20% difference in expression level for said gene.

The skilled artisan is well aware of the methodologies available to determine the level of marker gene expression of said one or more biomarker genes. In one embodiment it comprises measuring polynucleotide levels of said genes by means of biological assays using primers and/or probes capable of specifically hybridizing to said polynucleotides or to one or more regions within said polynucleotides. In another embodiment it comprises measuring protein levels of related gene products by means of biological assays using binders, antibodies or fragments thereof, for said proteins, their pro-forms, their substrates, or their metabolisation products.

Although the foregoing methods are configured to determine the competence of a mammalian oocyte and in particular a human oocyte, this thus not imply the sample to be limited thereto. Samples to be used in the present invention evidently include granulosa or cumulus originating from an oocytes, but may as well be based on follicular fluid, or from culture medium, comprising at least one granulosa or cumulus cell associated with an oocyte. In the control or reference sample the competence of the oocyte is known, either samples with known competent or known non-competent oocytes can be used, wherein said samples could be obtained from the same or a different subject than the sample to be tested. In a particular embodiment the control or reference sample comprises at least one granulosa or cumulus cell associated with a known non-competent oocyte, either or not obtained from the same subject as the sample to be tested.

As follows from the examples hereinafter, it has been found by that the exon level analysis of the following genes, i.e. CAMK1D, PTGS2, EFNB2, VCAN, STC1, STC2, PGR and GPX3 allows to establish biomarker gene expression models to predict the competence of an oocyte in a sample. In particular the biomarker genes to be used in the in vitro methods according to the invention, are selected from the group comprising SASH1, MROH9, NCOA7, DNAH3, HSPH1 exon 2, HSPH1exon 6, GALNTL6, SPTBN5, CAMK1D exon 1, CAMK1D exon 9, and EFNB2. The present invention thus provides an in vitro method for evaluating the competence of a mammalian oocyte to lead to childbirth, to implantation, to form a pre-implantation blastocyst or embryo, and/or to lead to fertilization; said method comprising the steps of:


- - determining exon level gene expression of one or more biomarker
    genes selected from the group consisting of CAMK1D, PTGS2, EFNB2,
    VCAN, STC1, STC2, PGR and GPX3 in a sample comprising at least one
    granulosa or cumulus cell associated with the oocyte;
  - comparing the level of biomarker gene expression with a control of
    which the competence is known; and
  - evaluating the competence of the oocyte to lead to childbirth, to
    implantation, to form a pre-implantation blastocyst or embryo,
    and/or to lead to fertilization; based on said comparison.

In a particular embodiment the exon level analysis of gene expression is performed on the splice variant of one or more genes selected from SASH1, MROH9, NCOA7, DNAH3, HSPH1 exon 2, HSPH1exon 6, GALNTL6, SPTBN5, CAMK1D exon 1, CAMK1D exon 9, and EFNB2. Preferred combinations of biomarkers within the foregoing set of genes are selected from the list comprising;


- - EFNB2 and NCOA7;
  - CAMK1D exon 9 and HSPH1 exon2 and NCOA7
  - CAMK1D exon 9 and HSPH1 exon6 and NCOA7
  - CAMK1D exon 1 and SASH1; and
  - EFBN2 and SASH1; eventually in combination with one or more
    additional genes selected from the genes enlisted in Table 13; in
    particular with one ore more additional more genes selected from the
    genes enlisted in Table 8; in particular with one or more additional
    genes selected from the group consisting of CAMK1D, PTGS2, EFNB2,
    VCAN, STC1, STC2, PGR and GPX3; more particular with a gene selected
    from EFNB2, CAMK1D exon 1, CAMK1D exon 9, HSPH1 exon 2, HSPH1 exon
    6, NCOA7 and SASH1; even more in particular with a gene selected
    from of HSPH1 exon 2, HSPH1 exon 6, NCOA7 and SASH1

Thus in a further embodiment the present invention provides an in vitro method for evaluating the competence of a mammalian oocyte to lead to childbirth, to implantation, to form a pre-implantation blastocyst or embryo, and/or to lead to fertilization; said method comprising the steps of:


- - determining exon level gene expression of 2, 3, 4, 5, 6, 7, or 8
    biomarker genes selected from the group consisting of CAMK1D, PTGS2,
    EFNB2, VCAN, STC1, STC2, PGR and GPX3 in a sample comprising at
    least one granulosa or cumulus cell associated with the oocyte;
  - normalizing the exon level gene expression of said genes; and
  - evaluating the competence of the oocyte to lead to childbirth, to
    implantation, to form a pre-implantation blastocyst or embryo,
    and/or to lead to fertilization; based on said normalized expression
    levels.

In another embodiment the present invention provides an in vitro method for evaluating the competence of a mammalian oocyte to lead to childbirth, to implantation, to form a pre-implantation blastocyst or embryo, and/or to lead to fertilization; said method comprising the steps of:


- - determining exon level gene expression of either CAMK1D exon 1,
    CAMK1D exon 9 or EFNB2, in combination with one or more genes
    selected from the genes enlisted in Table 13; in particular with one
    ore more additional genes selected from the genes enlisted in Table
    8; in particular with one or more additional genes selected from the
    group consisting of HSPH1 exon 2, HSPH1 exon 6, NCOA7 and SASH1 in a
    sample comprising at least one granulosa or cumulus cell associated
    with the oocyte;
  - normalizing the exon level gene expression of said genes; and
  - evaluating the competence of the oocyte to lead to childbirth, to
    implantation, to form a pre-implantation blastocyst or embryo,
    and/or to lead to fertilization; based on said normalized expression
    levels.

In a further aspect the present invention provides the use of the in vitro methods according to invention for identifying oocytes that are capable of giving rise to a viable pregnancy after fertilization, optionally in combination with another in vitro oocyte, sperm or embryo evaluation method.

In a further aspect the present invention provides an oocyte bank comprising mammalian oocytes scored according to an in vitro method according the invention, as well as a method of preparing an embryo comprising the use of an oocyte scored according to an in vitro method according the invention.

Further numbered embodiments of the present invention include;

1. An in vitro method for evaluating the competence of a mammalian oocyte to lead to childbirth, to implantation, to form a pre-implantation blastocyst or embryo, and/or to lead to fertilization; said method comprising the steps of:


- - determining the level of marker gene expression of HSPH1 in a sample
    comprising at least one granulosa or cumulus cell associated with
    the oocyte
  - comparing the level of marker gene expression with a control of
    which the competence is known, or normalizing the exon level gene
    expression of said genes; and
  - evaluating the competence of the oocyte to lead to childbirth, to
    implantation, to form a pre-implantation blastocyst or embryo,
    and/or to lead to fertilization; based on said comparison or using
    said normalized expression level.

2. An in vitro method according to claim 1 further comprising: determining the level of marker gene expression of 1 or more additional genes selected from the list comprising PGR, GSTA4, GSTA3, GPX3, EFNB2, CAMK1D, PTGS2, STC1, STC2, SASH1, MROH9, SPTBN5, GALNTL6, NCOA7, DNAH3 or VCAN; in particular 1 or more additional genes selected from the list comprising EFNB2, CAMK1D exon 1, CAMK1D exon 9, HSPH1 exon 2, HSPH1 exon 6, NCOA7 and SASH1 in a sample comprising at least one granulosa or cumulus cell associated with the oocyte.

3. An in vitro method according to claim 2 comprising: determining the level of marker gene expression of HSPH1 and 1, 2, 3, 4, 5, 6, or 7 genes selected from the list comprising CAMK1D, PTGS2, EFNB2, GSTA4, STC1, STC2 and VCAN in a sample comprising at least one granulosa or cumulus cell associated with the oocyte; in particular HSPH1 and 1, 2, 3, 4, or 5 genes selected from CAMK1 D exon 1, CAMK1 D exon 9, EFNB2, NCOA7 and SASH1.

4. An in vitro method according to claim 3 comprising: determining the level of marker gene expression of HSPH1, CAMK1D, PTGS2, EFNB2, GSTA4, STC1, STC2 and VCAN in a sample comprising at least one granulosa or cumulus cell associated with the oocyte; in particular determining the level of marker gene expression of HSPH1, CAMK1D exon 1, CAMK1D exon 9, EFNB2, NCOA7 and SASH1.

5. An in vitro method according to claim 2 comprising: determining the level of marker gene expression of HSPH1 and 1, 2, 3, 4, 5, 6, or 7 genes selected from the list comprising SASH1, CAMK1D, MROH9, SPTBN5, GALNTL6, NCOA7, DNAH3 and EFBN2 in a sample comprising at least one granulosa or cumulus cell associated with the oocyte.

6. An in vitro method according to claim 5 comprising: determining the level of marker gene expression of HSPH1, SASH1, CAMK1D, MROH9, SPTBN5, GALNTL6, NCOA7, DNAH3 and EFBN2 in a sample comprising at least one granulosa or cumulus cell associated with the oocyte.

7. An in vitro method according to anyone of claims 1 to 6 further comprising: determining the level of marker gene expression of one or more additional genes selected from the list comprising RABGAP1L, SLC7A11, ALDH1L2, ASNS, BTNL3, TICRR, CHTOP, CDC42EP3, CEBPG, DOCK9, GATS, GOT1, HINT3, KLF10, MBD3, MOCOS, MSR1, NPHP4, NPR1, PAK7, PHGDH, RNF166, ROBO2, SLC6A9, SLIT2, TSC22D3, TUB1, and UNC80; in a sample comprising at least one granulosa or cumulus cell associated with the oocyte.

8. An in vitro method according to anyone of claims 1 to 7 wherein the level of marker gene expression of HSPH1 is determined by determining the expression level of exon 2 and/or exon 6 of HSPH1.

9. An in vitro method for evaluating the competence of a mammalian oocyte to lead to childbirth, to implantation, to form a pre-implantation blastocyst or embryo, and/or to lead to fertilization; said method comprising the steps of:


- - determining the level of marker gene expression of Exon 1 and/or
    exon 9 of CAMK1D in a sample comprising at least one granulosa or
    cumulus cell associated with the oocyte
  - comparing the level of marker gene expression with a control of
    which the competence is known; and
  - evaluating the competence of the oocyte to lead to childbirth, to
    implantation, to form a pre-implantation blastocyst or embryo,
    and/or to lead to fertilization; based on said comparison.
  - Alternatively, determining the level of marker gene expression of
    Exon 1 and/or exon 9 of CAMK1D in a sample comprising at least one
    granulosa or cumulus cell associated with the oocyte
    - normalizing the exon level gene expression of said genes; and
    - evaluating the competence of the oocyte to lead to childbirth, to
      implantation, to form a pre-implantation blastocyst or embryo,
      and/or to lead to fertilization; based on said normalized
      expression levels.

10. An in vitro method according to claim 9 further comprising: determining the level of marker gene expression of 1 or more additional genes selected from the list comprising PGR, GSTA4, GSTA3, GPX3, EFNB2, HSPH1, PTGS2, STC1, STC2, SASH1, MROH9, SPTBN5, GALNTL6, NCOA7, DNAH3 or VCAN in a sample comprising at least one granulosa or cumulus cell associated with the oocyte; in particular 1, 2, 3, 4, or 5 additional genes selected from HSPH1 exon 2, HSPH1 exon 6, EFNB2, NCOA7 and SASH1.

11. An in vitro method according to claim 10 comprising: determining the level of marker gene expression of Exon 1 and/or exon 9 of CAMK1D and 1, 2, 3, 4, 5, 6, or 7 genes selected from the list comprising HSPH1, PTGS2, EFNB2, GSTA4, STC1, STC2 and VCAN in a sample comprising at least one granulosa or cumulus cell associated with the oocyte; in particular Exon 1 and/or exon 9 of CAMK1D and 1, 2, 3, 4, or 5 genes selected from HSPH1 exon 2, HSPH1 exon 6, EFNB2, NCOA7 and SASH1.

12. An in vitro method according to claim 11 comprising: determining the level of marker gene expression of HSPH1, CAMK1D, PTGS2, EFNB2, GSTA4, STC1, STC2 and VCAN in a sample comprising at least one granulosa or cumulus cell associated with the oocyte; in particular the expression of Exon 1 and/or exon 9 of CAMK1 D and HSPH1 exon 2, HSPH1 exon 6, EFNB2, NCOA7 and SASH1.

13. An in vitro method according to anyone of claims 9 to 12 further comprising: determining the level of marker gene expression of one or more additional genes selected from the list comprising RABGAP1L, SLC7A11, ALDH1L2, ASNS, BTNL3, TICRR, CHTOP, CDC42EP3, CEBPG, DOCK9, GATS, GOT1, HINT3, KLF10, MBD3, MOCOS, MSR1, NPHP4, NPR1, PAK7, PHGDH, RNF166, ROBO2, SLC6A9, SLIT2, TSC22D3, TUB1, and UNC80; in a sample comprising at least one granulosa or cumulus cell associated with the oocyte.

14. An in vitro method for evaluating the competence of a mammalian oocyte to lead to childbirth, to implantation, to form a pre-implantation blastocyst or embryo, and/or to lead to fertilization; said method comprising the steps of:


- - determining the level of marker gene expression of SASH1 in a sample
    comprising at least one granulosa or cumulus cell associated with
    the oocyte
  - comparing the level of marker gene expression with a control of
    which the competence is known, or normalizing the exon level gene
    expression of said genes; and
  - evaluating the competence of the oocyte to lead to childbirth, to
    implantation, to form a pre-implantation blastocyst or embryo,
    and/or to lead to fertilization; based on said comparison or using
    said normalized expression level.

15. An in vitro method according to claim 14 further comprising: determining the level of marker gene expression of 1 or more additional genes selected from the list comprising PGR, GSTA4, GSTA3, GPX3, EFNB2, CAMK1D, PTGS2, STC1, STC2, HSPH1, MROH9, SPTBN5, GALNTL6, NCOA7, DNAH3 or VCAN; in particular 1 or more additional genes selected from the list comprising EFNB2, CAMK1D exon 1, CAMK1D exon 9, HSPH1 exon 2, HSPH1 exon 6, NCOA7 and SASH1 in a sample comprising at least one granulosa or cumulus cell associated with the oocyte.

16. An in vitro method according to claim 15 comprising: determining the level of marker gene expression of HSPH1 and 1, 2, 3, 4, 5, 6, or 7 genes selected from the list comprising CAMK1D, PTGS2, EFNB2, GSTA4, STC1, STC2 and VCAN in a sample comprising at least one granulosa or cumulus cell associated with the oocyte; in particular HSPH1 and 1, 2, 3, 4, or 5 genes selected from CAMK1 D exon 1, CAMK1 D exon 9, EFNB2, NCOA7, HSPH1 exon 2, and HSPH1 exon 6.

17. An in vitro method according to claim 16 comprising: determining the level of marker gene expression of HSPH1, CAMK1D, PTGS2, EFNB2, GSTA4, STC1, STC2 and VCAN in a sample comprising at least one granulosa or cumulus cell associated with the oocyte; in particular determining the level of marker gene expression of HSPH1, CAMK1D exon 1, CAMK1D exon 9, EFNB2, NCOA7 and SASH1.

18. An in vitro method according to claim 15 comprising: determining the level of marker gene expression of SASH1 and 1, 2, 3, 4, 5, 6, or 7 genes selected from the list comprising HSPH1, CAMK1D, MROH9, SPTBN5, GALNTL6, NCOA7, DNAH3 and EFBN2 in a sample comprising at least one granulosa or cumulus cell associated with the oocyte.

19. An in vitro method according to claim 18 comprising: determining the level of marker gene expression of HSPH1, SASH1, CAMK1D, MROH9, SPTBN5, GALNTL6, NCOA7, DNAH3 and EFBN2 in a sample comprising at least one granulosa or cumulus cell associated with the oocyte.

20. An in vitro method according to anyone of claims 14 to 19 further comprising: determining the level of marker gene expression of one or more additional genes selected from the list comprising RABGAP1L, SLC7A11, ALDH1L2, ASNS, BTNL3, TICRR, CHTOP, CDC42EP3, CEBPG, DOCK9, GATS, GOT1, HINT3, KLF10, MBD3, MOCOS, MSR1, NPHP4, NPR1, PAK7, PHGDH, RNF166, ROBO2, SLC6A9, SLIT2, TSC22D3, TUB1, and UNC80; in a sample comprising at least one granulosa or cumulus cell associated with the oocyte.

21. An in vitro method according to anyone of claims 14 to 20 wherein the level of marker gene expression of SASH1 is determined by determining the expression level of exon 12 of SASH 1.

## DETAILED DESCRIPTION OF THE INVENTION

In the context of the present invention the term ‘oocyte competence’ or ‘competence’ is meant to be the ability of an oocyte to resume meiosis, cleave after fertilization, help promote embryonic development and pregnancy establishment, and bring a pregnancy to term in good health. In other words, it represents the ability of a mammalian oocyte to lead to childbirth, to implantation, to form a pre-implantation blastocyst or embryo, and/or to lead to fertilization.

According to the invention, the oocyte may result from a natural cycle, a modified natural cycle or a stimulated cycle for ART (Assisted reproduction techniques) comprising e.g. In Vitro Fertilization (IVF) or intracytoplasmic sperm injection (ICSI). The oocytes used in the methods of the present invention are typically obtained between 20 to 44 hours after ovulation induction, wherein the ovulation induction could be obtained in a natural or modified natural cycle. The term “natural cycle” refers to the natural cycle by which the female or woman produces an oocyte. The term “modified natural cycle”, also referred to as a “stimulated cycle” refers to the process by which, multiple follicle growth and/or ovulation of the oocytes is induced by treatment of the female or woman. Ovulation triggers that can be used in such stimulated cycle include for example Luteinizing hormone or analogs, Chorionic Gonadotrophins and analogs, FSH and agonists, GnRH and analogs, Epidermal growth factor (EGF) and analogs, EGF-like proteins (peptides) amphiregulin, epiregulin, betacellulin and their analogs, Interleukin-6, Interleukin-1, Leukemia Inhibitory Factor (LIF) Phosphodiesterase type 4 Inhibitors, Low Molecular weight compounds activating any of the foregoing, and Any combinations of the foregoing. Particular treatments include ovarian stimulation with GnRH analogs (agonist or antagonists) associated with recombinant FSH and/or hMG; or with clomiphene citrate, tamoxifen, letrozole and the like.

In one embodiment the oocytes used in the methods of the present invention are obtained and/or collected between 20 to 40 hours after the ovulatory trigger. In another embodiment the oocytes used in the methods of the present invention are obtained and/or collected between 34 to 38 hours after ovulation induction; in particular 36 hours after ovulation induction. As will be evident to the skilled artisan the ovulatory trigger to the oocytes can be performed in vitro or in vivo.

A ‘granulosa cell’ also referred to as a ‘follicular cell’, is a somatic cell of the sex cord that is closely associated with the developing female gamete (oocyte) in the ovary of mammals.

A ‘cumulus cell’ is a cell as present in the discus proligerus (cluster of cells) that surrounds the oocyte both in the ovarian follicle and after ovulation.

Within the methods of the present invention, reference to ‘an oocyte’ is meant to include granulosa cells and/or cumulus cells associated with an oocyte matured and/or ovulation induced oocyte. Thus in one embodiment the present invention the oocytes used refer to cumulus cells, wherein said cumulus cells can be analyzed using the methods of the present invention prior or after the ovulatory trigger.

By the phrase “determining the level of marker expression” is meant an assessment of the degree of expression of a marker in a sample at the nucleic acid or protein level, using technology available to the skilled artisan to detect a sufficient portion of any marker expression product (including nucleic acids and proteins). The ‘marker gene expression level’ may for example be determined using nucleic acid microarray, Northern blot, reverse transcription PCR, Western blot, enzyme-linked immunosorbent assay, protein microarray or FACS analysis. The term ‘marker gene expression’ is meant to include expression of the full-length gene or variants thereof, in particular splice variants containing specific exons. Particularly interesting splice variants in the context of the present invention are any of the following splice variants or combinations thereof comprising:


- - Exon 2 and/or exon 6 of HSPH1,
  - Exon 1 and/or exon 9 of CAMK1D,
  - Exon 1 and/or exon 2 of NCOA7,
  - Exon 12 of SASH1,
  - Exon 14 of MROH9,
  - Exon 8 of SPTBN5,
  - Exon 16 of GALNTL6, and/or
  - Exon 21 of DNAH3

In a particular embodiment the combination of splice-variants used in the methods of the present invention is selected from the list comprising: exon 2 of HSPH1 and exon 6 of HSPH1, exon 2 of HSPH1 and exon 1 or exon 9 of CAMK1D, exon 2 of HSPH1 and exon 1 or exon 2 of NCOA7; exon 2 of HSPH1 and exon 12 of SASH1, exon 2 of HSPH1 and exon 14 or MROH9, exon 2 of HSPH1 and exon 8 of SPTBN5, exon 2 of HSPH1 and GALNTL6, exon 2 of HSPH1 and exon 21 of DNAH3, exon 6 of HSPH1 and exon 1 or exon 9 of CAMK1D, exon 6 of HSPH1 and exon 1 or exon 2 of NCOA7; exon 6 of HSPH1 and exon 12 of SASH1, exon 6 of HSPH1 and exon 14 or MROH9, exon 6 of HSPH1 and exon 8 of SPTBN5, exon 6 of HSPH1 and GALNTL6, exon 6 of HSPH1 and exon 21 of DNAH3, exon 2 and exon 6 of HSPH1 and exon 1 of CAMK1D, exon 2 and exon 6 of HSPH1 and exon 9 of CAMK1D, exon 2 and exon 6 of HSPH1 and exon 1 of NCOA7, exon 2 and exon 6 of HSPH1 and exon 2 of NCOA7; exon 2 and exon 6 of HSPH1 and exon 12 of SASH1, exon 2 and exon 6 of HSPH1 and exon 14 or MROH9, exon 2 and exon 6 of HSPH1 and exon 8 of SPTBN5, exon 2 and exon 6 of HSPH1 and GALNTL6, exon 2 and exon 6 of HSPH1 and exon 21 of DNAH3, exon 2 and exon 6 of HSPH1 and exon 1 and exon 9 of CAMK1D, exon 2 and exon 6 of HSPH1 and exon 1 and exon 9 of CAMK1D and exon 1 and exon 2 of NCOA7, exon 2 and exon 6 of HSPH1 and exon 1 and exon 9 of CAMK1D and exon 1 and exon 2 of NCOA7 and exon 12 of SASH1, exon 2 and exon 6 of HSPH1 and exon 1 and exon 9 of CAMK1D and exon 1 and exon 2 of NCOA7 and exon 12 of SASH1 and exon 14 of MROH9, exon 2 and exon 6 of HSPH1 and exon 1 and exon 9 of CAMK1D and exon 1 and exon 2 of NCOA7 and exon 12 of SASH1 and exon 14 of MROH9 and exon 8 of SPTBN5, exon 2 and exon 6 of HSPH1 and exon 1 and exon 9 of CAMK1D and exon 1 and exon 2 of NCOA7 and exon 12 of SASH1 and exon 14 of MROH9 and exon 8 of SPTBN5 and exon 16 of GALNTL6, exon 2 and exon 6 of HSPH1 and exon 1 and exon 9 of CAMK1D and exon 1 and exon 2 of NCOA7 and exon 12 of SASH1 and exon 14 of MROH9 and exon 8 of SPTBN5 and exon 16 of GALNTL6 and exon 21 of DNAH3.

In a further particular embodiment the combination of splice-variants used in the methods of the present invention is selected from the list comprising: exon 1 of CAMK1D and exon 9 of CAMK1D, exon 1 of CAMK1D and exon 2 or exon 6 of HSPH1, exon 1 of CAMK1D and exon 1 or exon 2 of NCOA7, exon 1 of CAMK1D and exon 12 of SASH1, exon 1 of CAMK1D and exon 14 of MROH9, exon 1 of CAMK1D and exon 8 of SPTBN5, exon 1 of CAMK1D and exon 16 of GALNTL6, exon 1 of CAMK1D and exon 21 of DNAH3, exon 9 of CAMK1D and exon 2 or exon 6 of HSPH1, exon 9 of CAMK1D and exon 1 or exon 2 of NCOA7, exon 9 of CAMK1D and exon 12 of SASH1, exon 9 of CAMK1D and exon 14 of MROH9, exon 9 of CAMK1D and exon 8 of SPTBN5, exon 9 of CAMK1D and GALNTL6, exon 9 of CAMK1D and exon 21 of DNAH3, exon 1 and 9 of CAMK1D and exon 2 of HSPH1, exon 1 and 9 of CAMK1D and exon 6 of HSPH1, exon 1 and 9 of CAMK1D and exon 1 of NCOA7, exon 1 and 9 of CAMK1D and exon 2 of NCOA7, exon 1 and 9 of CAMK1D and exon 12 of SASH1, exon 1 and 9 of CAMK1D and exon 14 of MROH9, exon 1 and 9 of CAMK1D and exon 8 of SPTBN5, exon 1 and 9 of CAMK1D and exon 16 of GALNTL6, exon 1 and 9 of CAMK1D and exon 21 of DNAH3, exon 1 and 9 of CAMK1D and exon 2 and exon 6 of HSPH1, exon 1 and 9 of CAMK1D and exon 2 and exon 6 of HSPH1 and exon 1 of NCOA7, exon 1 and 9 of CAMK1D and exon 2 and exon 6 of HSPH1 and exon 1 and exon 2 of NCOA7, exon 1 and 9 of CAMK1D and exon 2 and exon 6 of HSPH1 and exon 1 and exon 2 of NCOA7 and exon 12 of SASH1, exon 1 and 9 of CAMK1D and exon 2 and exon 6 of HSPH1 and exon 1 and exon 2 of NCOA7 and exon 12 of SASH1 and exon 14 of MROH9, exon 1 and 9 of CAMK1D and exon 2 and exon 6 of HSPH1 and exon 1 and exon 2 of NCOA7 and exon 12 of SASH1 and exon 14 of MROH9 and SPTBN5, exon 1 and 9 of CAMK1D and exon 2 and exon 6 of HSPH1 and exon 1 and exon 2 of NCOA7 and exon 12 of SASH1 and exon 14 of MROH9 and SPTBN5 and exon 16 of GALNTL6, exon 1 and 9 of CAMK1D and exon 2 and exon 6 of HSPH1 and exon 1 and exon 2 of NCOA7 and exon 12 of SASH1 and exon 14 of MROH9 and SPTBN5 and exon 16 of GALNTL6 and exon 21 of DNAH3, exon 1 of CAMK1D and SASH1, exon 9 of CAMK1D and exon 2 of HSPH1 and NCOA7, exon 9 of CAMK1D and exon 2 of HSPH1 and exon 1 or exon 2 of NCOA7, exon 1 of CAMK1D and exon 12 of SASH1, exon 9 of CAMK1D and exon 6 of HSPH1 and exon 12 of SASH1, exon 9 of CAMK1D and exon 6 of HSPH1 and SASH1

In another embodiment the combination of splice-variants used in the methods of the present invention is selected from the list comprising: EFNB2 and exon 9 or exon 1 of CAMK1D, EFNB2 and exon 2 or exon 6 of HSPH1, EFNB2 and exon 1 or exon 2 of NCOA7, EFNB2 and exon 12 of SASH1, EFNB2 and exon 14 of MROH9, EFNB2 and exon 8 of SPTBN5, EFNB2 and exon 16 of GALNTL6, EFNB2 and exon 21 of DNAH3, EFNB2 and exon 9 or exon 1 of CAMK1D and exon 2 or exon 6 of HSPH1, EFNB2 and exon 9 or exon 1 of CAMK1D and exon 1 or exon 2 of NCOA7, EFNB2 and exon 9 or exon 1 of CAMK1D and exon 12 of SASH1, EFNB2 and exon 2 or exon 6 of HSPH1 and exon 1 or exon 2 of NCOA7, EFNB2 and exon 2 or exon 6 of HSPH1 and exon 12 of SASH1, EFNB2 and exon 1 or exon 2 of NCOA7 and exon 9 or exon 1 of CAMK1D, EFNB2 and exon 1 or exon 2 of NCOA7 and exon 2 or exon 6 of HSPH1, EFNB2 and exon 1 or exon 2 of NCOA7 and exon 12 of SASH1, EFNB2 and exon 12 of SASH1 and exon 1 or exon 2 of NCOA7 and exon 2 or exon 6 of HSPH1, EFNB2 and exon 9 or exon 1 of CAMK1D and exon 2 or exon 6 of HSPH1 and exon 1 or exon 2 of NCOA7, EFNB2 and NCOA7, EFNB2 and SASH1, EFNB2 and exon 1 or exon 2 of NCOA7, EFNB2 and exon 12 of SASH1.

In another embodiment the combinations of splice variants used in the methods of the present invention is selected from the list comprising exon 12 of SASH1 and HSPH1 or exon 2 or exon 6 of HSPH1; exon 12 of SASH1 and MROH9 or exon 14 of MROH9; exon 12 of SASH1 and SPTBN5 or exon 8 of SPTBN5; exon 12 of SASH1 and CAMK1D or exon 1 or exon 9 of CAMK1D; exon 12 of SASH1 and GALNTL6 or exon 16 of GALNTL6; exon 12 of SASH1 and NCOA7 or exon 1 or exon 2 of NCOA7; exon 12 of SASH1 and DNAH3 or exon 21 of DNAH3; exon 12 of SASH1 and EFNB2; exon 12 of SASH1 and HSPH1 or exon 2 or exon 6 of HSPH1 and SPTBN5 or exon 8 of SPTBN5; exon 12 of SASH1 and HSPH1 or exon 2 or exon 6 of HSPH1 and CAMK1D or exon 1 or exon 9 of CAMK1D; exon 12 of SASH1 and HSPH1 or exon 2 or exon 6 of HSPH1 and GALNTL6 or exon 16 of GALNTL6; exon 12 of SASH1 and HSPH1 or exon 2 or exon 6 of HSPH1 and NCOA7 or exon 1 or exon 2 of NCOA7; exon 12 of SASH1 and HSPH1 or exon 2 or exon 6 of HSPH1 and DNAH3 or exon 21 of DNAH3; exon 12 of SASH1 and HSPH1 or exon 2 or exon 6 of HSPH1 and EFNB2; exon 12 of SASH1 and HSPH1 or exon 2 or exon 6 of HSPH1 and MROH9 or exon 14 of MROH9; exon 12 of SASH1 and HSPH1 or exon 2 or exon 6 of HSPH1 and MROH9 or exon 14 of MROH9 and SPTBN5 or exon 8 of SPTBN5; exon 12 of SASH1 and HSPH1 or exon 2 or exon 6 of HSPH1 and MROH9 or exon 14 of MROH9 and CAMK1D or exon 1 or exon 9 of CAMK1D; exon 12 of SASH1 and HSPH1 or exon 2 or exon 6 of HSPH1 and MROH9 or exon 14 of MROH9 and GALNTL6 or exon 16 of GALNTL6; exon 12 of SASH1 and HSPH1 or exon 2 or exon 6 of HSPH1 and MROH9 or exon 14 of MROH9 and NCOA7 or exon 1 or exon 2 of NCOA7; exon 12 of SASH1 and HSPH1 or exon 2 or exon 6 of HSPH1 and MROH9 or exon 14 of MROH9 and DNAH3 or exon 21 of DNAH3; exon 12 of SASH1 and HSPH1 or exon 2 or exon 6 of HSPH1 and MROH9 or exon 14 of MROH9 and EFNB2; exon 12 of SASH1 and HSPH1 or exon 2 or exon 6 of HSPH1 and MROH9 or exon 14 of MROH9 and SPTBN5 or exon 8 of SPTBN5 and CAMK1D or exon 1 or exon 9 of CAMK1D; exon 12 of SASH1 and HSPH1 or exon 2 or exon 6 of HSPH1 and MROH9 or exon 14 of MROH9 and SPTBN5 or exon 8 of SPTBN5 and GALNTL6 or exon 16 of GALNTL6; exon 12 of SASH1 and HSPH1 or exon 2 or exon 6 of HSPH1 and MROH9 or exon 14 of MROH9 and SPTBN5 or exon 8 of SPTBN5 and NCOA7 or exon 1 or exon 2 of NCOA7; exon 12 of SASH1 and HSPH1 or exon 2 or exon 6 of HSPH1 and MROH9 or exon 14 of MROH9 and SPTBN5 or exon 8 of SPTBN5 and DNAH3 or exon 21 of DNAH3; exon 12 of SASH1 and HSPH1 or exon 2 or exon 6 of HSPH1 and MROH9 or exon 14 of MROH9 and SPTBN5 or exon 8 of SPTBN5 and EFNB2; exon 12 of SASH1 and HSPH1 or exon 2 or exon 6 of HSPH1 and MROH9 or exon 14 of MROH9 and SPTBN5 or exon 8 of SPTBN5 and CAMK1D or exon 1 or exon 9 of CAMK1D and GALNTL6 or exon 16 of GALNTL6; exon 12 of SASH1 and HSPH1 or exon 2 or exon 6 of HSPH1 and MROH9 or exon 14 of MROH9 and SPTBN5 or exon 8 of SPTBN5 and CAMK1D or exon 1 or exon 9 of CAMK1D and NCOA7 or exon 1 or exon 2 of NCOA7; exon 12 of SASH1 and HSPH1 or exon 2 or exon 6 of HSPH1 and MROH9 or exon 14 of MROH9 and SPTBN5 or exon 8 of SPTBN5 and CAMK1D or exon 1 or exon 9 of CAMK1D and DNAH3 or exon 21 of DNAH3; exon 12 of SASH1 and HSPH1 or exon 2 or exon 6 of HSPH1 and MROH9 or exon 14 of MROH9 and SPTBN5 or exon 8 of SPTBN5 and CAMK1D or exon 1 or exon 9 of CAMK1D and EFNB2; exon 12 of SASH1 and HSPH1 or exon 2 or exon 6 of HSPH1 and MROH9 or exon 14 of MROH9 and SPTBN5 or exon 8 of SPTBN5 and CAMK1D or exon 1 or exon 9 of CAMK1D and GALNTL6 or exon 16 of GALNTL6 and NCOA7 or exon 1 or exon 2 of NCOA7; exon 12 of SASH1 and HSPH1 or exon 2 or exon 6 of HSPH1 and MROH9 or exon 14 of MROH9 and SPTBN5 or exon 8 of SPTBN5 and CAMK1D or exon 1 or exon 9 of CAMK1D and GALNTL6 or exon 16 of GALNTL6 and DNAH3 or exon 21 of DNAH3; exon 12 of SASH1 and HSPH1 or exon 2 or exon 6 of HSPH1 and MROH9 or exon 14 of MROH9 and SPTBN5 or exon 8 of SPTBN5 and CAMK1D or exon 1 or exon 9 of CAMK1D and GALNTL6 or exon 16 of GALNTL6 and EFNB2; exon 12 of SASH1 and HSPH1 or exon 2 or exon 6 of HSPH1 and MROH9 or exon 14 of MROH9 and SPTBN5 or exon 8 of SPTBN5 and CAMK1D or exon 1 or exon 9 of CAMK1D and GALNTL6 or exon 16 of GALNTL6 and NCOA7 or exon 1 or exon 2 of NCOA7 and DNAH3 or exon 21 of DNAH3; exon 12 of SASH1 and HSPH1 or exon 2 or exon 6 of HSPH1 and MROH9 or exon 14 of MROH9 and SPTBN5 or exon 8 of SPTBN5 and CAMK1D or exon 1 or exon 9 of CAMK1D and GALNTL6 or exon 16 of GALNTL6 and NCOA7 or exon 1 or exon 2 of NCOA7 and EFNB2; exon 12 of SASH1 and HSPH1 or exon 2 or exon 6 of HSPH1 and MROH9 or exon 14 of MROH9 and SPTBN5 or exon 8 of SPTBN5 and CAMK1D or exon 1 or exon 9 of CAMK1D and GALNTL6 or exon 16 of GALNTL6 and NCOA7 or exon 1 or exon 2 of NCOA7 and DNAH3 or exon 21 of DNAH3 and EFNB2.

In another embodiment the models used in the in vitro models according to the invention is based on the determination of either EFNB2, or exon 1 or exon 9 of CAMK1D in combination with one or more marker genes selected from STC1, SASH1, PGR, GSTA4, GSTA3, GPX, NCOA7, HSPH1, MROH9, DNAH3, GALNTL6 and SPTBN5; in particular with one or more genes selected from STC1, SASH1, PGR, GSTA4, GSTA3, GPX, NCOA7, and HSPH1. In a further embodiment the models used are based on the determination of either EFNB2, or exon 1 or exon 9 of CAMK1D in combination with one or more splice variants selected from exon 12 of SASH1, exon 1 of NCOA7, exon 2 of NCOA7, exon 2 of HSPH1, exon 6 of HSPH1, exon 14 of MROH9, exon 21 of DNAH3, exon 16 of GALNTL6 and exon 8 of SPTBN5; in particular with one or more splice variants selected from exon 12 of SASH1, exon 1 of NCOA7, exon 2 of NCOA7, exon 2 of HSPH1, and exon 6 of HSPH1.

The control is at least one granulosa or cumulus cell associated with an oocyte, of which the competence is known. In this respect a positive control is at least one granulosa or cumulus cell associated with an oocyte competent to lead to childbirth, to implantation, to form a pre-implantation blastocyst or embryo, and/or to lead to fertilization. In another embodiment, a positive control may be a chromosomally normal oocyte.

In certain embodiments the level of expression of at least 2 marker genes of Table 13 is determined. In a particular embodiment the level of biomarker gene expression is determined for one of the following combinations;

EFNB2, GSTA4 and PGR;

EFNB2, GSTA4, PGR, GPX3 and GSTA3;

EFNB2, and NCOA7;

EFNB2, and SASH1;

EFNB2, CAMK1D exon 1, CAMK1D exon 9, SASH1, MROH9, NCOA7, DNAH3, HSPH1 exon 2, HSPH1 exon 6, GALNTL6, and SPTBN5;

EFNB2, CAMK1D exon 1, CAMK1D exon 9, HSPH1 exon 2, HSPH1 exon 6, NCOA7, and SASH1;

CAMK1D exon 9, HSPH1 exon 2 and NCOA7;

CAMK1D exon 9, HSPH1 exon 6 and NCOA7;

CAMK1D exon 1, and SASH1;

These new models proved to be able to rank oocyte for their potential for pregnancy, and live birth. As is evident from the foregoing models CAMK1D exon 1 or exon 9 and EFNB2 are confirmed as positive control genes in both intra- and inter-patient analysis, combining these markers with further genes selected from GSTA4, GSTA3, PGR, GPX3, NCOA7, SASH1, MROH9, DNAH3, HSPH1 exon 2, HSPH1 exon 6, GALNTL6 and SPTBN5, results in models that are predictive in determining an oocyte competence to pregnancy and live-birth. As is evident form the examples hereinafter, using these combinations of genes the PPV, NPV and accuracy of the prediction exceeded 60%. In another embodiment the in vitro models of the present invention, see for example model 3 in Table 6 below, may further comprise patient and/or cycle parameters such as for example age, days of stimulation and relative estradiol (E2), Anti-Mullerian hormone levels (AMH), day 3 follicle stimulating hormone levels (Day 3 FSH) and the like; in a particular embodiment the patient and cycle parameters used in the in vitro models of the present invention are days of stimulation, relative E2 and age.

In one embodiment and departing from the foregoing set of genes, the present invention provides an in vitro model for predicting the competence of a mammalian oocyte to lead to pregnancy and live birth; said method comprising the steps of:


- - determining the level of marker gene expression of EFNB2, GSTA4 and
    PGR in a sample comprising at least one granulosa or cumulus cell
    associated with the oocyte
  - comparing the level of marker gene expression with a control of
    which the competence is known; or normalizing the exon level gene
    expression of said genes and

evaluating the competence of the oocyte to lead to pregnancy and live birth based on said comparison or on said normalized expression levels.

In one embodiment of the foregoing method the competence of the oocyte is based on determining the probability of pregnancy (P) given by the equation P=−a+b*EFNB2+c*GSTA4−d*PGR; wherein a is a number from and between 1.00 to 4.00; b is a number from and between 0.00 and 2.00; c is a number from and between 0.00 and 2.00; d is a number from between 0.00 and 2.00; EFNB2 is the normalized expression level of EFNB2; GSTA4 is the normalized expression level of GSTA4; and PGR is the normalized expression level of PGR. In a particular embodiment a is a number from and between 2.00 to 3.00; b is a number from and between 0.00 and 1.00; c is a number from and between 0.00 and 1.00; d is a number from between 0.00 and 1.00, in the foregoing equation. In an even further embodiment a is 2.26; b is 0.79; c is 0.095; and d is 0.096 in the foregoing equation.

In one embodiment of the foregoing model based on EFNB2, GSTA4 and PGR, the in vitro method further includes determining the level of marker gene expression of GPX3 and GSTA3 in a sample comprising at least one granulosa or cumulus cell associated with the oocyte, and using said further marker gene expression in evaluating the competence of the oocyte. In a further embodiment of the foregoing model based on EFNB2, GSTA4 and PGR, the competence of the oocyte is based on determining the probability of pregnancy (P) given by the equation P=−a+b*EFNB2+c*GSTA4−d*PGR−e*GPX3−f*GSTA3; wherein a is a number from and between 1.00 to 4.00; b is a number from and between 0.00 and 2.00; c is a number from and between 0.00 and 2.00; d is a number from between 0.00 and 2.00; e is a number from and between 0.00 and 2.00; f is a number from and between 0.00 and 2.00; EFNB2 is the normalized expression level of EFNB2; GSTA4 is the normalized expression level of GSTA4; PGR is the normalized expression level of PGR; GPX3 is the normalized expression of GPX3; and GSTA3 is the normalized expression level of GSTA3. In particular embodiment a is a number from and between 1.00 to 2.00; b is a number from and between 0.00 and 1.00; c is a number from and between 0.00 and 1.00; d is a number from between 0.00 and 1.00; e is a number from and between 0.00 and 1.00; and f is a number from between 0.00 and 1.00 in the foregoing equation. More in particular a is 1.02, b is 0.63, c is 0.27, d is 0.11, e is 0.43 and f is 0.51 in the foregoing equation.

In one embodiment of the foregoing models based on EFNB2, GSTA4 and PGR, the in vitro method further includes determining the patient and cycle characteristics age, days of stimulation and Rel E2 (Relative E2), and using said further patient and cycle characteristics in evaluating the competence of the oocyte. In a further embodiment of the foregoing model based on EFNB2, GSTA4 and PGR, the competence of the oocyte is based on determining the probability of pregnancy (P) given by the equation P=−a+b*EFNB2+c*GSTA4−d*PGR−e*GPX3−f*GSTA3+g*days of stimulation+h*Rel E2+i*age; wherein a is a number from and between 5.00 to 15.00; b is a number from and between 0.00 and 2.00; c is a number from and between 0.00 and 2.00; d is a number from between 0.00 and 2.00; e is a number from and between 0.00 and 2.00; f is a number from and between 0.00 and 2.00; g is a number from and between 0.00 and 2.00; h is a number from and between 0.00 and 2.00; i is a number from and between 0.00 and 2.00; EFNB2 is the normalized expression level of EFNB2; GSTA4 is the normalized expression level of GSTA4; PGR is the normalized expression level of PGR; GPX3 is the normalized expression of GPX3; and GSTA3 is the normalized expression level of GSTA3. In particular embodiment a is a number from and between 9.00 to 13.00; b is a number from and between 1.00 and 2.00; c is a number from and between 0.00 and 1.00; d is a number from between 0.00 and 1.00; e is a number from and between 0.00 and 1.00; and f is a number from between 0.00 and 1.00; g is a number from and between 0.00 and 1.00; h is a number from and between 0.00 and 1.00; i is a number from and between 0.00 and 1.00 in the foregoing equation. More in particular a is 11.27, b is 1.35, c is 0.46, d is 0.24, e is 0.66, f is 0.86, g is 0.0.50, h is 0.009 and i is 0.14 in the foregoing equation.

In one embodiment and departing from the foregoing set of genes, the present invention provides an in vitro model for predicting the competence of a mammalian oocyte to lead to pregnancy and live birth; said method comprising the steps of:


- - determining the level of marker gene expression of EFNB2 and NCOA7
    in a sample comprising at least one granulosa or cumulus cell
    associated with the oocyte
  - comparing the level of marker gene expression with a control of
    which the competence is known; or normalizing the exon level gene
    expression of said genes and

evaluating the competence of the oocyte to lead to pregnancy and live birth based on said comparison or on said normalized expression levels.

In one embodiment of the foregoing method the competence of the oocyte is based on determining the probability of pregnancy (P) given by the equation P=a+b*EFNB2+c*NCOA7; wherein a is a number from and between 2.00 to 5.00; b is a number from and between 2.00 and 4.00; c is a number from and between 1.00 and 3.00; EFNB2 is the normalized expression level of EFNB2; and NCOA7 is the normalized expression level of NCOA7. In a particular embodiment the probability of pregnancy (P) is given by the equation P=a+b*EFNB2+c*NCOA7; wherein a is a number from and between 3.00 to 4.00; b is a number from and between 2.00 and 3.00; c is a number from and between 1.00 and 2.00. In an even further embodiment a is 3.23; b is 2.57 and c is 1.87 in the foregoing equation.

In one embodiment and departing from the foregoing set of genes, the present invention provides an in vitro model for predicting the competence of a mammalian oocyte to lead to pregnancy and live birth in a subject pretreated with a GnRH (Gonadotropin-releasing Hormone) antagonist and rFSH (recombinant Follicle Stimulating Hormone); said method comprising the steps of:


- - determining the level of marker gene expression of CAMK1D exon 9,
    HSPH1 exon 2, and NCOA7 in a sample comprising at least one
    granulosa or cumulus cell associated with the oocyte
  - comparing the level of marker gene expression with a control of
    which the competence is known; or normalizing the exon level gene
    expression of said genes; and

evaluating the competence of the oocyte to lead to pregnancy based on said comparison or on said normalized expression levels.

In one embodiment of the foregoing method the competence of the oocyte is based on determining the probability of pregnancy (P) given by the equation P=a+b*CAMK1D exon 9+c*HSPH1 exon 2+d*NCOA7; wherein a is a number from and between −3.00 to −1.00; b is a number from and between 2.00 and 4.00; c is a number from and between 0.00 and 3.00; d is a number from and between 0.00 and 2.00; CAMK1D exon 9 is the normalized expression level of CAMK1D exon 9; HSPH1 exon 2 is the normalized expression level of HSPH1 exon 2; NCOA7 is the normalized expression level of NCOA7; and wherein a positive value refers to pregnancy and a negative value to no pregnancy. In a particular embodiment the probability of pregnancy (P) is given by the equation P=a+b*CAMK1D exon 9+c*HSPH1 exon 2+d*NCOA7; wherein a is a number from and between −2.00 to −1.00; b is a number from and between 1.00 and 2.00; c is a number from and between 0.00 and 1.00; d is a number from and between 0.00 and 1.00. In an even further embodiment a is −1.37; b is 1.79; c is 0.89 and d is 0.74.

In one embodiment and departing from the foregoing set of genes, the present invention provides an in vitro model for predicting the competence of a mammalian oocyte to lead to pregnancy and live birth in a subject pretreated with a GnRH (Gonadotropin-releasing Hormone) antagonist and HP-hMG (highly purified human menopausal gonadotropin); said method comprising the steps of:


- - determining the level of marker gene expression of CAMK1D exon 1 and
    SASH1 in a sample comprising at least one granulosa or cumulus cell
    associated with the oocyte
  - comparing the level of marker gene expression with a control of
    which the competence is known; or normalizing the exon level gene
    expression of said genes and

evaluating the competence of the oocyte to lead to pregnancy based on said comparison or on said normalized expression levels.

In one embodiment of the foregoing method the competence of the oocyte is based on determining the probability of pregnancy (P) given by the equation P=a+b*SASH1−c*CAMK1D exon 1; wherein a is a number from and between 0.00 to 2.00; b is a number from and between 0.00 and 2.00; c is a number from and between 0.00 and 2.00; CAMK1D exon 1 is the normalized expression level of CAMK1D exon 1; SASH1 is the normalized expression level of SASH1; and wherein a positive value refers to pregnancy and a negative value to no pregnancy. In a particular embodiment the probability of pregnancy (P) is given by the equation P=a+b*SASH1−c*CAMK1D exon 1; wherein a is a number from and between 0.00 to 1.00; b is a number from and between 0.00 and 1.00; c is a number from and between 0.00 and 1.00. In an even further embodiment a is 0.47; b is 0.91; and c is 0.0.23.

Comparison of gene expression levels according to the methods of the present invention is preferably performed after the gene expression levels obtained have been corrected for both differences in the amount of sample assayed and variability in the quality of the sample used (e.g., amount and quality of mRNA tested). Normalizing the levels against reference genes in the same sample may carry out correction. Typically “housekeeping genes”, such as UBC, B2M, actin, GAPDH, HPRT, CPB, G6PD, histone H2A, or mitochondrial ribosomal protein S 18C gene, in particular UBC or B2M are used for this normalization. Expression levels of a marker gene may be normalized by correcting the absolute expression level of a marker by comparing its expression to the expression of a gene that is not a marker, e.g., a housekeeping gene that is constitutively expressed. Suitable genes for normalization include housekeeping genes such as UBC, B2M, actin, GAPDH, HPRT, CPB, G6PD, histone H2A, or mitochondrial ribosomal protein S 18C gene. This normalization allows amongst others the comparison of the expression level in one sample, e.g., a test sample to a control sample.

Differences in marker gene expression in cumulus cells or granulosa cells associated with a given oocyte, compared to the expression level of cumulus or granulosa cells associated with other oocytes in a group, permits ranking of oocytes in a group of oocytes according to relative quality. Thus, in one embodiment of the invention, a plurality of oocytes from an individual, or group of individuals, are grouped and screened in a single assay; the oocytes characterized with the highest quality probability scores are then selected for fertilization and/or implantation.

The identification of the markers and the determination of the association between change in marker gene expression in cumulus cells and attendant oocyte quality provides the basis for clinical diagnostic kits based on either mRNA or protein expression. Basic materials and reagents required for determination of oocyte quality according to the invention may be assembled in a kit. In certain embodiments, the kit comprises at least one reagent that specifically detects expression levels of at least one gene as disclosed herein, and instructions for using the kit according to one or more methods of the invention. Each kit necessarily comprises reagents which render the procedure specific. Thus, for detecting mRNA expressed by at least one marker gene of the set, the reagent will comprise a nucleic acid probe complementary to mRNA, such as, for example, a cDNA or an oligonucleotide. The nucleic acid probe may or may not be immobilized on a substrate surface (e.g., a microarray). For detecting a polypeptide product encoded by at least one gene of the set, the reagent will comprise an antibody that specifically binds to the polypeptide. Depending on the procedure, the kit may further comprise one or more of: extraction buffer and/or reagents, amplification buffer and/or reagents, hybridization buffer and/or reagents, immunodetection buffer and/or reagents, labeling buffer and/or reagents, and detection means. Protocols for using these buffers and reagents for performing different steps of the procedure may also be included in the kit. Reagents may be supplied in a solid (e.g., lyophilized) or liquid form. Kits of the present invention may optionally comprise different containers (e.g., vial, ampoule, test tube, flask or bottle) for each individual buffer and/or reagent. Other containers suitable for conducting certain steps for the disclosed methods may also be provided. In certain embodiments, the kits of the present invention further comprise control samples. Instructions for using the kit according to one or more methods of the invention may comprise instructions for processing the cumulus cell or granulosa cells samples, and/or performing the test, and instructions for interpreting the results, i.e. to decide on the competence of the oocyte to lead to childbirth, to implantation, to form a pre-implantation blastocyst or embryo, and/or to lead to fertilization. In one embodiment, a kit for determining mRNA expression of marker genes by quantitative RT-PCR would include standard primers and other reagents for performing quantitative RT-PCR, with reaction components formulated for optimized success in detection for each gene to be assayed. The amplification primers may be selected based on the nucleotide sequence of the relevant marker gene(s), depending on the species of mammalian oocyte being assessed. Any single or combination of the marker genes could be incorporated into the kit. To provide users options to choose balance between coverage and cost, different kits could provide different collections of primers for the marker genes targeted for analysis.

Possible Applications of the Method According to this Invention

In preferred embodiments, the inventive methods may also be used to identify women subjects who produce or do not produce pregnancy competent oocytes based on the levels of expression of a set of differentially expressed genes. However, the inventive methods are applicable to non-human animals as well, e.g., other mammals, avians, amphibians, reptiles, et al. For example, the subject invention may be used to derive animal models for the study of putative female fertility treatments.

Additionally, the present invention may be used to identify female subjects who have an abnormality that precludes or inhibits their ability to produce pregnancy competent oocytes, e.g., exposure to medication, exposure to toxicants, environmental factors, ovarian dysfunction, ovarian cyst, pre-menopausal or menopausal condition, cancer, autoimmune disorder, hormonal dysfunction, cell proliferation disorder, or another health condition that inhibits or precludes the development of pregnancy competent oocytes. For example, subjects who do not express specific pregnancy signature genes at characteristic expression levels may be screened to assess whether they have an underlying health condition that precludes them from producing pregnancy competent oocytes. Particularly, such subjects may be screened to assess whether they are exhibiting signs of menopause, whether they have a cancer, autoimmune disease or ovarian abnormality, e.g., ovarian cyst, or whether they have another health condition, e.g., hormonal disorder, allergic disorder, etc., that may preclude the development of “pregnancy competent” oocytes.

Additionally, the subject methods may be used to assess the efficacy of putative female fertility treatments in humans or non-human female subjects. Essentially, such methods may comprise treating a female subject, preferably a woman, with a putative fertility enhancing treatment, isolating at least one oocyte and associated surrounding follicular or granulosa cells from said woman after treatment, optionally further isolating at least one oocyte and associated surrounding cells prior to treatment, isolating at least one cumulus cell from each of said isolated oocytes; detecting the levels of expression of at least one gene that is expressed or not expressed at characteristic levels by cumulus cells that are associated with (surround) pregnancy competent oocytes; and assessing the efficacy of said putative fertility treatment based on whether it results in cumulus cells that express at least one pregnancy signature gene at levels more characteristic of cumulus cells that surround pregnancy competent oocytes (than without treatment). As noted, while female human subjects are preferred, the subject methods may be used to assess the efficacy of putative fertility treatments in non-human female animals, e.g., female non-human primates or other suitable animal models for the evaluation of putative human fertility treatments.

Still further, the present invention may be used to enhance the efficacy of in vitro or in vivo fertility treatments. Particularly, oocytes that are found to be “pregnancy incompetent”, or are immature, may be cultured in a medium containing one or more gene products that are encoded by genes identified as being “pregnancy signature” genes, e.g., hormones, growth factors, differentiation factors, and the like, prior to, during, or after in vivo, or in vitro fertilization. Essentially, the presence of these gene products should supplement for a deficiency in nutritional gene products that are ordinarily expressed by cumulus and/or granulosa cells that surround “pregnancy competent” oocytes, and which normally nurture oocytes and thereby facilitate the capability of these oocytes to yield viable pregnancies upon fertilization.

Alternatively, one or more gene products encoded by said pregnancy signature genes may be administered to a subject who is discovered not to produce pregnancy competent oocytes according to the methods of the invention. Such administration may be parenteral, e.g., by intravenous, intramuscular, intravaginal, subcutaneous injection or by oral or transdermal administration. Alternatively, these gene products may be administered locally to a target site, e.g., a female ovarian or uterine environment. For example, a female subject may have her uterus or ovary implanted with a drug delivery device that provides for the sustained delivery of one or more gene products encoded by “pregnancy signature” genes.

As detailed above, differences in marker gene expression in cumulus cells or granulosa cells associated with a given oocyte, compared to the expression level of cumulus or granulosa cells associated with other oocytes in a group, permits ranking of oocytes in a group of oocytes according to relative quality. Thus, in one embodiment of the invention, a plurality of oocytes from an individual, or group of human or non-human female subjects, are grouped and screened in a single assay; the oocytes characterized with the highest quality probability scores are then selected for fertilization and/or implantation.

## EXAMPLES

### Example 1

**QPCR Analysis**

Material and Methods

Patient Population

This study was approved by the Ethical Committee of the UZBrussel and the patients written consent was obtained. Forty seven ICSI patients were selected based on the embryo transfer policy (single embryo transfer) and ovarian stimulation protocol prescribed: GnRH antagonist in combination with recombinant follicle stimulating hormone (FSH) (Gonal-f, Merck-Serono, Geneva, Switzerland; n=4 or Puregon, MSD, Oss, The Netherlands; n=43). Causes of infertility were: male factor only (n=19), female factor only (ovulation disorder n=3 and tubal infertility n=2), combination male and female factor (OAT and endometriosis n=3) and idiopathic (n=20). Twenty patients had single embryo transfer on day 3 of culture, from these 10 became pregnant. Twenty seven patients had transfer on day 5, from these 9 became pregnant.

Collection of Human Cumulus Cells and Embryo Culture

Vaginal ultrasound was used to monitor follicular development. The endocrine profile was monitored by analysis of serum 17β-estradiol (E2), progesterone, FSH and LH by electrochemiluminescence on a COBAS 6001 immunoanalyser (Roche, Roche Diagnostics, Mannheim, Germany) using validated assays with respectively sensitivities of 5 ng/l, 0.03 μg/l, <0.1 IU/l, 0.1 IU/l and total imprecisions (% CV) of respectively <6, <7, <6 and <6. Final follicular maturation was induced with a dose of 10 000 IU hCG when at least three follicles of 17 mm in diameter were observed by transvaginal ultrasound. Oocyte retrieval was done 36 h later. CC collection was done as described in (Wathlet et al. 2011). Briefly, individual oocyte denudation was performed in 40 μl droplets of HTF-SSS containing 80 IU/ml Cumulase (MediCult, Lyon, France) for not longer than 30 s and washed sequentially in droplets without enzyme. At any time, oocytes were handled individually from this point onwards in order to allow retrospective analysis of the CC per oocyte. After denudation, the CC were plunged directly in liquid nitrogen. ICSI was performed as described previously (Van Landuyt et al. 2005) and embryos were cultured in sequential media of SAGE (CooperSurgical, Leisegang Medical, Berlin). Embryos were vitrified on day 3 or day 5/6 of embryo culture as was described earlier (Van Landuyt et al. 2011) and used in a subsequent FRET cycle.

For all 47 patients, only the CC related to those oocytes resulting in embryos selected for transfer were analyzed exception made for 7 of the 28 non-pregnant patients, 1 extra CC sample (except for 1 patient, 2 CC samples) related to a vitrified embryo giving pregnancy after a frozen single-embryo transfer cycle, was analyzed (8 extra CC in total from 7 patients). FIG. 1 shows the different samples used for each analysis.

Gene Selection

This study is the 3rd one in a row to evaluate the predictive value of CC gene expression for oocyte quality using QPCR. Over the 3 studies, we followed a precise strategy to choose which genes to analyze regarding to oocyte quality in ICSI patients. The first study identified 4 top genes, 2 predictive for embryo morphology (inositol-trisphosphate 3-kinase A (ITPKA) and transient receptor potential cation channel, subfamily M, member 7 (TRPM7)) and 2 for pregnancy outcome (syndecan 4 (SDC4) and versican (VCAN)). It was chosen to include those 4 genes in the next study (Wathlet et al. 2012). ITPKA and TRPM7 were again related to embryo development, but SDC4 and VCAN were not retained this time in the pregnancy models and were replaced by EFNB2, CAMK1D and STC1 when using stepwise multiple regression analysis. As several studies showed no relation between the genes predictive for embryo development and pregnancy outcome (Gebhardt et al. 2011; Wathlet et al. 2012), it was decided to repeat only the 3 pregnancy related genes in this third study. Our hypothesis is that by this ‘cascade’ testing strategy, the strongest pregnancy predictive genes may be filtered out through the consecutive studies (Table 2). The other 9 genes analyzed in this study were: GSTA3, GSTA4, GPX3, GSR, ITPR1, SLC2A1, THBS1, TGFB1 and PGR (Table 1).

RNA Extraction and cDNA Synthesis

Total RNA was extracted as described earlier (Adriaenssens et al. 2010) using the RNeasy Micro Kit (Qiagen, Westburg, Leusden, The Netherlands) including a DNase step and addition of 5 ng/μl poly(dA) (Roche Applied Science, Mannheim, Germany) prior to extraction. Extraction was followed by a second DNase treatment (RQ1 RNase-Free DNase, Promega, Leiden, The Netherlands).

Reverse transcriptase (RT) was done as previously described (Adriaenssens et al. 2010) with the iScript cDNA Synthesis Kit (Bio-Rad Laboratories, Ghent, Belgium). Negative controls were generated by omitting the enzyme or the RNA in the RT reaction.

Real-Time PCR

Primer sequences for CAMK1D, STC1 and EFNB2 are listed in Wathlet et al 2012. Primers for GPX3, GSTA3, GSTA4, PGR, THB1, ITPRA, SLC2A1, GSR and TGFb1 can be found in Table 3. Both beta-2-microglobulin (B2M) and ubiquitin C (UBC) were validated and used before as normalization factor (Wathlet et al. 2011). Cycling conditions, negative controls, standard curves and normalization (with B2M and UBC) are as described earlier (Wathlet et al. 2011), but all PCR reactions were adapted to 10 μl reactions. All values mentioned hereafter are the normalized values to the mean of both B2M and UBC for each sample.

Statistics

Inter-Patient Analysis

In a first analysis, a two-tailed t-test (GraphPad Prism version 4.01 for Windows, GraphPad Software, San Diego, Calif. USA) was used to compare cumulus complexes of oocytes resulting in pregnancy or not (19 live birth, 28 non-pregnant). All data were LOG transformed to obtain normal distribution. Since multiple genes were tested, the Bonferroni correction allowed us to consider only P-values <0.0042 were considered significant after Bonferroni correction.

In a second analysis, a model was built using stepwise multiple regression analysis as described earlier (Wathlet et al. 2011). Briefly, a linear regression model, with an equation as output ‘y=a+bx+cz+ds+et+fu+gv+hw’, was built with as response variables (x, z, s, t, u, v, w) gene expression and/or patient and cycle characteristics (all are listed in Table 3). ‘b-h’ are the respective indexes of the included variables and ‘a’ is the intercept of the equation. A variable was added to the model if the type III P-value of the variable was <0.3 and if the P-value of the model was improved. At the end of the model a backwards regression step was performed to exclude redundant variables. Four different models were built to predict pregnancy. In two models only gene expression values were allowed (first the model was restricted to 3 genes, next all genes were allowed into the models as long as they improved the P-value of the model). In two other models, the need for correction by patient and cycle characteristics was assessed by allowing all patient and cycle characteristics to the models only composed of genes, when those extra variables could improve the model. By introducing those extra factors, possible inter-patient variability on gene expression could be leveled out and increase the differences related to oocyte quality.

For all models, accuracy ((True positive+true negative)/(true positive+false positive+false negative+true negative)) and positive and negative predictive values (PPV=True positive/(true positive+false negative) and NPV=True negative/(true negative+false positive) were calculated. The power of all models was represented by Receiver Operating Characteristic (ROC) and the area under curve (AUC) was calculated.

Intra-Patient Analysis

Finally, the study allowed comparing 2 (or 3) oocytes originating from one oocyte retrieval cycle with known pregnancy outcome per oocyte, as all embryos were transferred individually in consecutive cycles. For this purpose, from 7 of the 28 patients that were not pregnant in the fresh cycle, the CC of the embryos that were replaced in a subsequent frozen single embryo transfer cycle resulting in pregnancy were compared to those transferred in the fresh cycle. For one patient, two consecutive frozen embryo replacement cycles were analyzed as the first embryo did not end in a pregnancy. Seven genes were chosen for this analysis based on their presence in one of the above mentioned models or their P-value of addition, when first added to a pregnancy model. A paired t-test was performed for each gene and the chance to pregnancy was calculated with the earlier defined models from the inter-patient analysis, containing only genes and built on the 47 CC samples (19 live birth), but excluding the 8 CC samples from the frozen cycles.

Results

Patient Population

No statistical differences were found between the patient and cycle characteristics of the pregnant and the non-pregnant groups (Table 3). Mean age and BMI for both groups of patients was low but comparable. Progesterone levels were low in both groups. Mean cycle number attempt was not different in both groups: respectively in the pregnant and non-pregnant group 12 and 17 patients underwent their first cycle, 6 and 10 their second cycle and 1 for both groups their third cycle. Percentages of oocyte maturity and fertilization were normal in both groups, and the percentage of good quality embryos on day 3 was more than 50%. Embryo quality score on moment of transfer was not different for both groups. Nineteen fresh SET cycles resulted in live birth.

Pregnancy Prediction

Validation of Predictive Genes (CAMK1D, EFNB2 and STC1) Using a Model Built on a Previous Sample Set

A model predicting pregnancy composed of 3 genes, considered in a previous study (Wathlet et al. 2012), was tested in the current, independent patient series. The gene expression values for CAMK1D, EFNB2 and STC1 of the CC of the 47 patients were introduced in the equation obtained before and gave a value predicting the chance to pregnancy for each of the 47 oocytes. The obtained PPV and NPV calculated with the 47 samples of this study were 62% and 86%, with an accuracy of 72%.

Inter-Patient Analysis: t-Test of All 12 Genes in the Current Sample Set

The cumulus complexes of 28 oocytes not resulting in pregnancy were compared to 19 CC of oocytes that resulted in a live birth. Only EFNB2 was statistically higher in the pregnant group. CAMK1D, GSTA4 and GSR only showed a trend of higher expression in the pregnant group (respective P-values: 0.0068, 0.0123 and 0.0507). Graphs for all genes can be found in FIG. 2.

Inter-Patient Analysis: Stepwise Multiple Regression Analysis

In a first step to build a pregnancy model, the P-value of addition when added as first variable was calculated for all genes and can be found in Table 4.

First, the model was restricted to the inclusion of 3 of the 12 genes (Model 1). EFNB2, GSTA4 and PGR were retained and gave a model with a P-value of 0.0015, a PPV of 68%, a NPV of 79% an accuracy of 73% and an AUC of 0.82. When trying to improve this model by also allowing patient and cycle characteristics (from Table 3), no improvement on the previous model was found (Model 1 bis).

In a next step, more than 3 genes were allowed into the model if improving the P-value (Model 2). Five of the 12 genes were retained in this model (i.e. EFNB2, GSTA4, PGR, GPX3 and GSTA3) which yielded a P-value <0.0001 with a PPV of 78%, a NPV of 83%, an accuracy of 81% and an AUC of 0.93. Adding patient and cycle characteristics improved the model (Model 3). The retained parameters were: days of stimulation, relative E2 and age. The PPV, NPV and accuracy of the extended model all increased to 93% and the AUC to 0.95 (Table 5). Full mathematical models can be found in Table 6. ROC curves are shown in FIG. 3.

Intra-Patient Pregnancy Prediction

For seven patients, cryostored CC samples related to cryopreserved embryos which had led to a clinical pregnancy after transfer in a subsequent transfer cycle were analyzed. This material was used to analyze the genes present in the above obtained multiparametric models (Table 6) and/or the 5 genes with the smallest P-value of addition (Table 4) i.e. CAMK1D, EFNB2, GPX3, GSR, GSAT4, GSTA3 and PGR. As a first explorative method a paired t-test for those seven genes was performed and can be found in FIG. 4. Five genes had an upwards trend in the CC of an oocyte resulting in pregnancy (P-value <0.05; only P-values <0.07 are significant after Bonferroni correction for multiple comparisons): EFNB2 (only significant difference between pregnant and non-pregnant), CAMK1D, GSR and PGR. GSTA4, GPX3 and GSTA3 had all P-values >0.05. Fold changes between the CC from the same patient were calculated and an average per gene was made (from 1.1 to 2.5) (Table 7). For the five genes with a P-value <0.1 in the paired t-test, the percentage of correctly estimated CC was 71% to 88%, based on the level expected (e.g. higher expression is expected in the pregnant compared to the non-pregnant related CC) from the paired t-test. In a next step the predictive power of the multiparametric models (the 3 gene model (=Model 1) and the 5 gene model (=Model 2)) obtained in the inter-patient analysis (47 CC samples) was used to rank the CC of each patient (Table 6) for their chance to pregnancy. As all patient and cycle characteristics are identical for oocytes within a single retrieval cycle it makes obviously no sense to use models containing those variables. All CC, except for patient 4, were correctly ranked (Table 7) for their chance to pregnancy.

Discussion

The expression of 12 genes in the CC and their capacity to predict the pregnancy potential of the oocyte they surrounded were analyzed in the present study. Three genes (CAMK1D, EFNB2 and STC1) were included from a previous study (Wathlet et al. 2012), where they were coming out as the most predictive ones for pregnancy prediction. EFNB2 was also significantly up-regulated in the current study in the CC from the oocytes that gave pregnancy and CAMK1D showed the same trend (P=0.0068). The model composed of genes only from the previous patient dataset, using CAMK1D, EFNB2 and STC1, was validated in this independent patient group and yielded a similar overall performance of the model, with accuracies of 72% in the current study and 79% in the previous study.

Besides the EFNB2 and CAMK1D coming from our previous study, two newly studied genes, GSTA4 and GSR, also showed an up-regulated trend in the CC of the pregnant group.

A multiparametric approach tested whether it would be possible to predict live birth by using only 3 of the 12 tested genes in the current sample set (inter-patient analysis). The 3 most predictive genes were GSTA4, PGR and EFNB2 and resulted in a similar accuracy as the previous model with EFNB2, CAMK1D and STC1 (73% versus 72%). Using three genes, none of the patient or cycle characteristics could improve the model, suggesting that the expression of the three genes was minimally influenced by patient and cycle factors. The live birth model could be improved by including two more genes (GPX3 and GSTA3), which increased the accuracy up to 81% and resulted in an AUC of 0.93. This 5 gene model was improved with three patient and cycle characteristics (age, relative E2 and days of stimulation) and resulted in an optimized model with a PPV, NPV and accuracy of 93%, but similar AUC as the model only containing genes. Ideally, pregnancy prediction based on gene expression should be possible with a limited set of genes to reduce analysis time and cost. Of the 12 genes tested in this patient population, the 2 most recurrent genes are GSTA4 and EFNB2 (see Table 5). These 2 genes are present in all 3 pregnancy models and have respectively in 3 out of 3 and in 2 out of 3 predictive models a type-III P-value <0.01. The models also showed that the gene expression values are always more important than the patient and cycle characteristics, as the type-III P-values in the models are only significant for the genes and not for the patient and cycle characteristics. Furthermore, the AUC results are comparable between the model containing only genes and the model combining genes and patient and cycle characteristics.

For 7 patients, 2 (or 3) CC from oocytes giving embryos with a good morphology and consecutively transferred, were analyzed (Intra-patient analysis). For all oocytes, pregnancy outcome was known as all embryo transfers were done in subsequent SET cycles. Having assessed the expression of 7 genes (EFNB2, CAMK1D, GSR, PGR, GSTA4, GSTA3 and GPX3), it was easy to estimate, based on single genes or on a combination of them in a model, which oocyte of the two considered ones would have the highest chance to pregnancy. The best predictions, when looking at each gene individually, were found for CAMK1D, GSR, EFNB2, PGR and GSTA4 with 71% to 86% correct estimations. Combining 3 or 5 genes, according to the earlier models, 6 of the 7 patients had the cumulus complexes correctly predicting the chance to pregnancy. This prediction was the same when using the 3 or 5 gene model. Surprisingly, the multiparametric model failed on a different patient than the single gene analysis. The multiparametric approach seems stronger than the individual genes as all genes wrongly predicted the pregnancy outcome for the second comparison of patient 6, but the multiparametric model predicted successfully, especially as the CC of the oocytes that resulted in pregnancy were not considered for building the predictive model. The failed prediction of the multiparametric approach for patient 4, where the individual genes were correct in 4 out of 5, could be due to other factors (e.g. the endometrium status at the moment of transfer). Although this analysis is still limited in patient numbers, these results are encouraging. This is the first study that confirms that some genes predicting pregnancy between patients might also be capable of ranking the quality of oocytes within patients using a multiparametric approach and providing a chance to pregnancy for each oocyte. CC gene expression analysis might become a valuable tool in the ART lab but does obviously not take into account the eventual influence of poor sperm quality and out-of-phase endometrium. This study could confirm the use of earlier found predictive genes in a new patient population, with the same stimulation protocol, but with different culture media. This data suggests that EFNB2 and CAMK1D, the only genes that we analyzed in two studies using two different culture media, were not affected by culture media. To test the validity of the current models, a future analyze should involve patients with different stimulation protocols and different culture media. Reasons why the specific genes of this study are important for pregnancy prediction remains speculative. As mentioned previously (Wathlet et al. 2012), CAMK1D may, among other pathways, be related to steroidogenesis by its strong correlation with steroid related genes (CYP11A1, STC2 and HSD3B1). In this data set CAMK1D also strongly correlated with a steroid related gene i.e. PGR next to EFNB2 and GSTA4 (all P<0.0001 with Pearson correlation analysis) still leaving the possibilities open for more than one pathway to which CAMK1D could be associated with. The presence of PGR in the pregnancy models and its predictive power within patients might indicate that steroid-related genes could be helpful in pregnancy prediction. PGR has been described before, but no difference between pregnant and non-pregnant could be confirmed with quantitative PCR (Hamel et al. 2010b). GSTA3 that was present in some of the pregnancy models also has a link with progesterone production (Raffalli-Mathieu et al. 2008) reiterating the importance of the steroidogenesis pathway. The function of EFNB2 in the ovary is not yet known, but B-class ephrins were proposed to be related to the luteinization process (Egawa et al. 2003) and the ephrin B2 receptor was found differently expressed between CC from normal oocytes compared to aneuploid oocytes (Fragouli et al. 2012b). Out of the other genes tested, only members of the glutathione family were retained in the models or were significant in the t-test. Glutathione enzymes are important for detoxification actions (of free radicals) in the cells through the use of glutathione. Hypoxia leads to the formation of reactive oxygen species (ROS) which can cause lipid peroxidation, enzyme inactivation and cell damage, resulting in apoptosis (Buttke and Sandstrom 1994) not only in CC, but also in the oocyte (Tatemoto et al. 2000). Oxidative stress has already been reported by other groups as a possible target to assess oocyte quality (Lee et al. 2010; van Montfoort et al. 2008). One reason that some transcripts of those pathways are higher in the CC from oocytes giving pregnancy might be that those oocytes are better protected against a stressful environment if needed.

Conclusion

By testing presumably important genes batch wise in three consecutive studies on different patient groups for which cumulus complexes were frozen individually per oocyte, we retained the most predictive genes for pregnancy and opposed these every time to new candidate genes. This ‘cascade’ strategy attempted to increase the power of pregnancy prediction using CC gene expression as quality marker for oocytes in ART. The strategy proved effective as the model with EFNB2, CAMK1D and STC1 from the second study (Wathlet et al. 2012) could be confirmed on an independent patient sample set. In an attempt to further improve prediction models for live birth, models were built (inter-patient analysis), still retaining EFNB2 together with PGR and genes related to the glutathione metabolism. The new models proved to be able to rank oocyte for their potential for pregnancy (Intra-patient analysis). The validity of our current models, for routine application, still need prospective assessment in a larger and more diverse patient population allowing Intra-patient analysis.

### Example 2

**Exon Level Analysis**

Three micro array experiments were performed and their resulting data led to 4 micro array analyses. Below respectively named micro array analysis 1, 2, 3 or 4 (see table 14 below). Array experiment 1 and 2 were used for data-mining, (for finding genes or exons of genes that are predictive for oocyte competence) and array experiment 3 was used for filtering the recurrent predictive genes (finding genes or exons that are more general applicable as they are predictive in different fertility centers and after slightly different treatments). The most novel analysis is based on micro array experiment 2 which comprises an intra patient and intra-cycle (oocytes from the same stimulation cohort) analysis for live birth.

Material:

For the three micro array experiments cumulus cells were collected as described before (Wathlet et al. 2011). The RNA was extracted using the Qiagen Rneasy micro kit as described in Wathlet et al. 2011. The quality and quantity of the RNA was verified using the Bioanalyzer (Agilent) and the Nanodrop (Thermo Scientific).

For the micro arrays in the first set amplification was performed by starting from 100 ng total-RNA and a 1 round T7 based amplification (Message Amp II-Biotin Enhanced, Ambion). Good Quality aRNA (size distribution, 3′/5′ ratio & % present calls) was verified and Affymetrix Human Genome U133 Plus 2.0 Arrays were performed at the Salk Institute (USA, San Diego). Further analysis of the gene readouts was performed using Genesifter (Perkin Elmer). For Array experiment 2 and 3 RNA was extracted, measured, and evaluated as mentioned above but amplification was performed using the Nugen V2 kit. Arrays were, both for array experiment 2 and 3, the Affymetrix Human Gene ST arrays. Data were mainly analyzed in Microsoft Excel.

Method:

Intra-patient comparison comprises that both the positive sample (CC of an oocyte resulting in a pregnancy in a single embryo transfer) and the negative sample (CC of an oocyte resulting in no pregnancy after single embryo transfer) originate from the same patient and the same stimulation and pick up cycle.

Comparison or analysis was performed one parametric with regular or paired t-test and multiparametric using stepwise multiregression analysis. The multiparametric approach resulted in gene or exon combinations which increases the predictive power.

Aim:

Find genes or splice variants of genes (exons) predictive for live birth after ART treatment. This is done in an intra-patient comparison,

Set Up and First Results (Array Analysis 1 and 2):

Array experiment 1, performed in 2006, comprised an inter patient comparison for live birth and an intra-patient comparison for embryo development. Patient pretreatment was GnRH Agonist combined with HP-hMG. This resulted in 2 quality lists: comprising over 500 genes, (“Q1”=genes predictive for successful oocyte and embryo development, “Q3”=genes predictive for live birth)

Array analysis 2 was performed in January 2012 and consisted of an intra-patient analysis for live birth and delayed development for 5 patients and 15 arrays and resulted in several lists with differentially regulated genes. Patients were in this case pretreated with GnRH Antagonist and recombinant FSH.

Array experiment 3 was only used as a confirmation dataset in micro array analysis 4 and is further described below.

Array Analysis 4: Reanalysis of the Micro Array Data of Array Experiment 2+3 on an Exon Level

In searching for genes expressed in the cumulus cells predictive for pregnancy we surprisingly found, that for some genes, specific combinations of splice variants for these genes were predictive. This finding is not obvious with the 2 most standard analysis methods, micro arrays and Q RT-PCR, currently available in the lab, as is explained below.

In this field of research the most commonly applied micro array analysis (Affymetrix Human Genome U133 Plus 2 and Affymetrix Human Gene ST arrays) evaluate/consider one generalized(average) expression value obtained per gene, based on 4 or more probes recognizing some part of the coding and non coding RNA related to a specific gene. This number reflects the abundance or absence of a bulk of RNA molecules related to a specific gene but gives no information on the regulation of specific splice variants.

The alternative “standard” analysis technique, quantitative PCR after reversed transcription (RT-QPCR), on the other hand normally only analyzes the abundance of 1 exon, which corresponds for most of the genes to evaluating only a specific fraction of the splice variants of this specific gene.

Specifically investigating more than 1 splice variant is relevant as specific splice variants could be coding for tissue specific (e.g. cumulus cell specific) or secreted or more active forms of the protein. Alternative splice variants (often shorter forms) can also have a regulatory function and actually down-regulate the amount or the effect of the protein. Therefore combining the information of different splice variants for a specific gene holds more information but requires also more time and resources to investigate and is not commonly done.

Notwithstanding, in the present study we reanalysed the foregoing array experiments on an exon level. In this experiment genes predictive for successful fertilization, embryo development and implantation on an exon level were selected in array experiment 2 as described below.

The Samples:

Fifteen Affymetrix micro arrays were performed on 15 cumulus samples from five ICSI patients. CC originated from oocytes that resulted in 3 outcome categories: 1. an embryo with a high morphological score (Embryo quality 1) that was transferred and resulted in a pregnancy (P-CC) 2. an embryo with a high morphological score (Embryo quality 1) that was transferred but did not result in a pregnancy (NP-CC) and 3. an embryo that developed too slow to be considered for transfer (NT-CC).

The patients considered for this study were stimulated with GnRH antagonist and rFSH and were not pregnant from the first single embryo transfer and came back for the transfer of a single frozen embryo which did result in a pregnancy. As such the CC samples from category 1, 2 or 3 originated from 1 pick up cycle and this thus allowed an intra patient analysis.

The Analysis:

This analysis was performed on an exon level. Four comparisons were performed, 3 times an ANOVA comparing the 3 conditions, 1 time 2 t-tests were performed comparing the best condition, the P-CC to the NP-CC and P-CC to the NT-CC. The variable patient was considered as a random variable in the t-test comparison, and was once random and once fixed in the ANOVA analysis on 5 patients and was considered as random in the ANOVA comparison of 3 of the 5 patients (considering only the patients that got transfer on day 5 and not the 2 patients that got transfer on day 3).

Genes were considered potentially interesting if they were: a) differentially expressed between P-CC and both NP-CC and NT-CC, b) in the same direction different (e.g. down in the 2 negative control samples), c) at least one of the difference was >1.5 with a q<0.1 (False discovery rate) and d) more than one exon of that gene was retained.

The Results:

This resulted in a 34 gene list that was the subject of this patent application. These were than supplemented with the up till now eleven strongest genes from the earlier lists (Q1+2+1.246—table 14 above) which results in the 45 list (Table 13). These were on an exon level filtered in the array experiment 3 using stepwise multiple regression analysis. In brief, in array experiment 3, 112 micro arrays were available from individual CC of 112 patients that were pregnant or not after a single embryo transfer cycle (SET-cycle). All exon values from the table 13 genes in these 112 patients were considered for prediction model building using stepwise multiple regression analysis. This was done in multiple rounds to compensate for too high numbers of variables and was performed using cross-validation to prevent over fitting. This led to the most powerful combinations of genes to be retained, which gave rise to the “11 gene list” (SASH1, MROH9, NCOA7, DNAH3, HSPH1 exon 2, HSPH1exon 6, GALNTL6, SPTBN5, CAMK1D exon 1, CAMK1D exon 9,EFNB2). (Table 8)

The “11 genes list” consists of the most predictive exon level genes combinations and are therefore the core of the current application. The original “45 gene list” contains other genes that are related to them and could eventually replace one or more of the 11 genes if needed. An example of this is given below and discussed in more detail in Table 11. In brief an strong oocyte competence prediction model for GnRH antagonist and rFSH stimulated patients was found to contain CAMK1D exon 9, HSPH1 exon 2 and NCOA7 (Accuracy=73% ROC-AUC=0.733) but for the same patients a model with CAMK1D exon 9, HSPH1 exon 6 and NCOA7 was also prediction oocyte competence, even if it was slightly less powerful, (Accuracy=68% ROC-AUC=0.714).

Conclusions:

Several gene and specific exons of genes have been successfully related to live birth and embryo development in array experiments comprising patients undergoing different pretreatments and these were also confirmed in an independent array dataset (array experiment 3). The strength of the 3the array experiment lays in the fact that it was performed on samples from three different European sites. As such the final subset of genes retained should be more universally applicable, than any known gene combination up till now. To test this, two confirmation experiments were performed departing from this subset of 11 genes, to demonstrate the ability of the foregoing genes in predicting oocyte competence. The first confirmation experiment has been a retrospective study, including an intra- and inter-patient analysis and the second study has been a prospective study in 3 groups of patients.

### Example 3

**Retrospective Study**

The retrospective study was performed with samples collected from 107 patients in two clinical sites. The QPCR was performed exon specific for 7 oocyte quality markers, from the 11 gene list (HSPH1 exon 2, HSPH1 exon 6, CAMK1D exon 1, CAMK1D exon 9, EFNB2, NCOA7, SASH1) and 2 endogenous controls (UBC and B2M) in three different batches (Table 9).

Cumulus Cell Collection:

The cumulus cell samples were collected in one European and one American Fertility Centre (Table 9) and originate from patients that got one embryo per transfer (SET=Single Embryo Transfer) and proved to be pregnant or not. This allowed the retrospective comparison of CC of competent and less competent oocytes. This is the most obvious approach but oocytes originate from different patients (=inter-patient analysis) and this results in some inter-patient variance that is not controlled for. There was also a subset of patients that got 2 or more consecutive single embryo transfers from a single oocyte harvest procedure (pick up), that resulted in a pregnancy or not and thus allowed also in the retrospective study an intra-patient comparison of the CC.

Three types of cumulus cell samples have been collected: samples from oocytes of which the embryo was transferred and led to a pregnancy, samples from good quality embryos that were transferred but did not result in a pregnancy, and cumulus samples from delayed embryos. Delayed embryos are embryos that are developing slower, which is considered indicative for poor oocyte competence, and these embryos are therefore not considered for transfer. The cumulus cells from delayed embryos were used as an intra-patient, negative control for those patients who had an oocyte giving a pregnancy. The distinction between the CC of morphologic identical embryos resulting in a pregnant and not (later referred to pregnant versus not pregnant) is considered more important than the distinction between the CC of oocytes leading to a pregnancy or to an embryo delayed during in vitro development (later also referred to as “pregnant versus delayed embryo development”) as the latter information is already available to labs performing extended embryo culture.

All patients underwent ovarian stimulation with GnRH antagonist and rFSH or HP-hMG and were initially analyzed per treatment group.

Statistics:

During the statistical analysis, first a one-parametric approach was used with Student t-tests (paired and unpaired). Secondly, a cross-validated multi-parametric model was built with the use of stepwise multiple regression analysis for a binary response.

Results 1: Intra-Patient Analysis

a) One Parametric: Pregnant Versus Delayed Embryo Development

In the one-parametric analysis (paired t-test) between CC of oocytes resulting in a pregnancy or a delayed embryo growth (n=8), we studied 7 genes and found a consistent upregulation in 4 of the 7 genes (Table 10). CAMK1D exon 1 and NCOA7 were upregulated in 88% of the cases, while CAMK1D exon 9 and SASH1 were upregulated in 75% of the patients.

However, no significant differences were reached (paired t-test p-values ranged from 0.14 to 0.24), because of the large inter-patient variance.

b) One Parametric: Pregnant Versus Not Pregnant

In the comparison between the CC of oocytes resulting in a pregnancy or not of the same patient (n=7), 3 of the 7 genes that were studied, showed a consistent upregulation in 71 to 86% of the patients: EFNB2, CAMK1D exon 1, and CAMK1D exon 9. Significance was reached with CAMK1D exon 1 (p 0.021). One gene, HSPH1 exon 6 showed a consistent downregulation in 71% of the patients.

This first intra-patient analysis thus confirms an upregulation of a known oocyte quality marker gene, CAMK1D, in the CC of oocytes resulting in a pregnancy compared to the CC of oocytes that will result in good morphology embryos that do not lead to a pregnancy or to delayed embryos of the same patient. These results also confirm the relevance of three new genes: NCOA7, SASH1 and HSPH1.

Earlier results in a similar setting indicated that a multiparametric approach increases the number of correctly predicted samples (Wathlet et al., 2013). Therefore, we also performed a multiparametric intra-patient analysis on the same samples.

c) Multiparametric: Pregnant Versus Delayed Embryo Development or Not Pregnant

The more sophisticated multiparametric analysis resulted in a strong pregnancy predictive model with EFNB2 and NCOA7, showing an accuracy of 73% and an ROC-AUC of 0.772. More detailed results are provided below (Table 11 and FIG. 5). This further confirmed the predictive quality of known and novel CC genes as oocyte quality markers.

Results 2: Inter-Patient Analysis

To increase the sample size inter-patient analyses were also performed on samples from 2 sites. This extra analysis should provide more generally applicable models.

While the same genes are predictive for oocyte competence in the patients stimulated with GnRH antagonist and rFSH or HP-hMG, the relation is for some genes inverse (e.g. CAMK1D in FIGS. 6 and 7). For the ease of interpretation, pretreatment specific models were built and described here. These will later be combined into one mathematical model with treatment specific indices (earlier applied in Adriaenssens et al. 2010).

a) GnRH Antagonist and rFSH Stimulated Patients, Pregnant Versus Delayed Embryo Development

The intra-patient analysis indicated that four of the seven genes analyzed here could be used to distinguish the CC of an oocyte leading to a pregnancy or to an embryo delayed in development.

In the current GnRH antagonist and rFSH stimulated patients the inter-patient variance seems however to prevent the construction of a strong mathematical model. The addition of an extra gene of the 45 list could circumvent this.

b) GnRH Antagonist and rFSH Stimulated Patients, Pregnant Versus Not Pregnant

For the GnRH antagonist and rFSH stimulated patients in batch 1, 2 and 3 (n=34 pregnant and 29 non-pregnant patients), a strong oocyte quality predictive model containing 3 genes was obtained. Two genes, CAMK1D exon 9 and HSPH1 exon 2, were positively correlated with pregnancy and NCOA7 was negatively correlated. More detailed results are provided below (Table 11 and FIG. 6). The accuracy of this model was 73% and AUC of 0.733. This model is currently used in the prospective study for GnRH antagonist and rFSH stimulated patients.

c) GnRH Antagonist and HP-hMG Stimulated Patients, Pregnant Versus Delayed Embryo Development

It was found that the multi-parametric model for antagonist HP-hMG patients distinguishing the CC of oocytes leading to a pregnancy from the CC of oocytes leading to an embryo with a delayed development (n=20), had a high accuracy (80%) and AUC (0.8081). SASH1 was positively correlated with pregnancy, and CAMK1D exon 1 was negatively correlated with pregnancy. More detailed results are provided below (Table 11 and FIG. 7). SASH1 expression was also correlated with CAMK1D exon 9, NCOA7 expression. CAMK1D exon 1 expression was correlated with CAMK1D exon 9 expression.

d) GnRH Antagonist and HP-hMG Stimulated Patients, Pregnant Versus Not Pregnant

The pregnant versus non-pregnant multi-parametric model in GnRH agonist and HP-HMG patients (n=24) was also strong (Accuracy=75% and AUC=0.8392) and was also largely depending on the positive relation of SASH1 expression with oocyte developmental competence (detailed figure not shown).

Conclusions of the Retrospective Study

CAMK1D exon 9 and EFNB2, both chosen as positive control genes are in the current dataset confirmed both in the intra- and inter-patient analysis.

The relation between oocyte competence and expression of patentable genes was also confirmed by their implication in predictive models after different types of ART pretreatment. HSPH1 exon 2 and NCOA7 are retained in the final model for antagonist/rFSH stimulated patients. SASH1 is a powerful predictive gene for antagonist/HP-hMG stimulated patients. The fact that HSPH1exon 6 and EFNB2 are not retained in the final model does not imply that these markers are not related to oocyte competence; as they are correlated with other markers (HSPH1 exon 6 is correlated with HSPH1 exon 2, and EFNB2 is correlated with CAMK1D expression) and hence are rejected from the regression analysis.

### Example 4

**Prospective Study**

A case-control assessor blinded prospective study was performed by FOBI using 2 of the oocyte competence predictive models described higher (FIGS. 6 and 7). The study comprised 3 groups of patients: the experimental group with morphological embryo grading (as described in Wathlet et al 2012) and CC gene analysis and transfer of a day 3 embryo and the 2 control groups with morphological embryo grading alone and transfer of a day 3 or a day 5 embryo. The 2 controls are included as pregnancy rates differ between the two transfer regimes. Day 5 transfer slightly increases clinical pregnancy rates in the first cycle but cumulative pregnancy rates are higher in in day 3 transfer cycles (Glujovsky et al. Cochrane review 2012).

The Aim:

The aim was to identify within the pool of oocytes obtained from each patient the oocyte with the highest competence by comparing the expression of specific genes in the CC that surrounded each oocyte. The embryo originating from this oocyte was transferred back to the patient. The CC gene expression analysis combined with morphological grading is expected to increase the chance on pregnancy for these patients compared to matched patients without the CC evaluation (i.e. choice of embryo to transfer only decided on routine morphological embryo grading).

Patient Population and Genes Analyzed:

This study comprised 17 patients. The patients included are patients from the ART clinic scheduled for fertility treatment with intra-cytoplasmatic sperm injection (ICSI) and single embryo transfer after 3 days of embryo culture (generally an 8 cell stage embryo). Patient pretreatments allowed are GnRH antagonist plus rFSH or HP-hMG and the gene expression models described in FIGS. 5 and 6 are used for these patients respectively.

The genes analyzed (CAMK1D exon 9, CAMK1D exon 1, HSPH1 exon 2, NCOA7, SASH1) originated from the 11 and 45 gene list submitted in this patent application.

The Different Steps in the Procedure:

This study comprised the following steps:


- - 1. Patients were informed and included when they signed informed
    consent,
  - 2. Oocytes were collected from these patients as usual
  - 3. Oocytes were individually freed from their cumulus cells using
    cumulase as usual
  - 4. Cumulus cells were processed or frozen immediately
  - 5. The oocytes were further used for the infertility treatment (in
    the current study: ICSI and embryo culture)\*
  - 6. The expression of specific genes in the cumulus cells was
    assessed
  - 7. The obtained CC score potentially indicated the most competent
    oocyte (with the highest chance on live birth) and was used to
    select the most promising embryo between the embryos with the best
    morphological grade available for each patient
  - 8. This single embryo was transferred back to the patient \*\*
  - 9. For each patient that got an embryo transfer based on the CC gene
    expression, a matching patient (same age category, similar
    pretreatment and having the same amount of good quality embryos
    available for transfer) with embryo transfer on day 3 (generally an
    8 cell embryo) and day 5 (generally an expanded blastocyst) without
    CC gene expression analysis was selected in an assessor blinded
    manner.
  - 10. For the 3 groups of patients (the experimental group and the 2
    control groups) the implantation and pregnancy rates are evaluated.

Remarks: * this step could comprise oocyte freezing for later use, eventually in specific media supplemented with nutrients to compensate for the deficiency that was observed in the CC gene expression result.

** the CC score could be used independently or combined with morphologic or other oocyte, sperm or embryo evaluation methods

The Cumulus Cell Gene Expression Evaluation:

The expression of specific genes was quantified in the CC of each oocyte. The absolute expression of each gene was normalized to the expression of 2 endogenous genes in the same CC sample to compensate for cell number differences and technical variance between the samples. The normalized expression values were than used in the mathematical formula described in FIGS. 6 and 7 and result in a score for each oocyte. The higher this score, the higher the chance that this oocyte would result in a live birth.

Results: Pregnancy Rates when Using CC Gene Expression Based Oocyte Evaluation

From the 17 patients 8 had a GnRH antagonist rFSH and 9 had a GnRH antagonist HP-hMG pretreatment. A beneficial effect of the CC evaluation is apparent for the day 3 transfer regimes (FIG. 7). The chance on a pregnancy was 2× higher for the patients with a day 3 embryo transfer and CC-Test compared to those patients with a day 3 embryo transfer without CC-Test. The number of patients is currently too limited to confirm the significance of an increase when compared to the patients with a day 5 embryo transfer.

Conclusions of the Prospective Study

Implementing the CC-Test analysis in the clinic as described above significantly increased the chance on a pregnancy for the patients involved in the study and thus proves the relevance of the genes which are the subject of the current patent application.

The models used in this proof of principle experiment contained 2 and 3 genes. From earlier micro array experiments (with more genes and more patients analyzed) we know that adding more genes (up to at least 5 or 6) will further improve the predictive power of the models.

This work is currently ongoing with more genes from the list in the current patent.

General Conclusion:

The 45 oocyte quality marker gene list originates from 2 array experiments using UZBrussel patients and was cross validated using data from a 3th independent array experiment containing more than 100 patients from 3 different European fertility centers. This approach should provide solid marker genes and gene combinations/models to predict treatment outcome.

As a proof of concept a subset of 7 genes, containing 4 novel genes, was tested using an alternative technique (QPCR) on CC samples from new patients collected at 2 fertility centers. The 4 genes were confirmed as valuable oocyte quality marker. This confirms that the proposed gene list contains valid oocyte quality predictive genes.

Finally the gene models were tested in a prospective study where pregnancy rates were increased in the patients where embryos were selected based on the expression of the above described oocyte quality marker genes in the CC.

The latter study was performed in patients of the ART clinic undergoing standard treatment by ICSI and thus confirms the applicability of the oocyte quality markers in a clinical setting.

