# DESCRIPTION

## STATEMENT AS TO FEDERALLY SPONSORED RESEARCH

This invention was made with the support of the United States government under Grant No: UO1AI056493, awarded by the National Institutes of Health and by the Department of Defense Grant DAMD17-98-C-8030. The government has certain rights in the invention.

## FIELD OF THE INVENTION

This disclosure relates antibodies that neutralize botulinum neurotoxins (e.g., BoNT/A) and their use in the treatment of botulism.

## INTRODUCTION

Botulism is caused by botulinum neurotoxin secreted by members of the genus Clostridium and is characterized by flaccid paralysis, which if not immediately fatal requires prolonged hospitalization in an intensive care unit and mechanical ventilation. Naturally occurring botulism is found in infants or adults whose gastrointestinal tracts become colonized by Clostridial bacteria (infant or intestinal botulism), after ingestion of contaminated food products (food botulism), or in anaerobic wound infections (wound botulism) (Center for Disease Control (1998) Botulism in the United States, 1899-1998, Handbook for epidemiologists, clinicians, and laboratory workers. Atlanta, Ga. U.S. Department of Health and Human Services, Public Health Service: downloadable at “bt.cdc.gov/agent/botulism/index.asp”). Botulism neurotoxins (BoNTs) are also classified by the Centers for Disease Control (CDC) as one of the six highest-risk threat agents for bioterrorism (the “Category A agents”), due to their extreme potency and lethality, ease of production and transport, and need for prolonged intensive care (Arnon et al. (2001) JAMA 285: 1059-1070). As a result of these threats, specific pharmaceutical agents are needed for prevention and treatment of intoxication.

No specific small molecule drugs exist for prevention or treatment of botulism, but an investigational pentavalent toxoid vaccine is available from the CDC (Siegel (1988) J. Clin. Microbiol. 26: 2351-2356) and a recombinant vaccine is under development (Smith (1998) Toxicon 36: 1539-1548). Regardless, mass civilian or military vaccination is unlikely due to the rarity of disease or exposure and the tact that vaccination would prevent subsequent medicinal use of BoNT. Post-exposure vaccination is useless, due to the rapid onset of disease. Toxin neutralizing antibody (Ab) can be used for pre- or post-exposure prophylaxis or for treatment (Franz et al. (1993) Pp. 473-476 In B. R. DasGupta (ed.), Botulinum and Tetanus Neurotoxins: Neurotransmission and Biomedical Aspects. Plenum Press, New York). Small quantities of both equine antitoxin and human botulinum immune globulin exist and are currently used to treat adult (Black and Gunn. (1980) Am. J. Med., 69: 567-570; Hibbs et al. (1996) Clin. Infect. Dis., 23: 337-340) and infant botulism (Arnon (1993). Clinical trial of human botulism immune globulin. p. 477-482. In B. R. DasGupta (ed.), Botulinum and Tetanus Neurotoxins: Neurotransmission and Biomedical Aspects. Plenum Press, New York) respectively.

The development of mAb therapy for botulism is complicated by the fact that there are at least seven BoNT serotypes (A-G) (Hatheway (1995) Curr. Top. Microbio. Immunol. 195: 55-75) that show little, if any, antibody cross-reactivity. While only four of the BoNT serotypes routinely cause human disease (A, B, E, and F), there has been one reported case of infant botulism caused by BoNT C (Oguma et al. (1990) Lancet 336: 1449-1450), one outbreak of foodborne botulism linked to BoNT D (Demarchi, et al. (1958) Bull. Acad. Nat. Med., 142: 580-582), and several cases of suspicious deaths where BoNT G was isolated (Sonnabend et al. (1981) J. Infect. Dis., 143: 22-27). Aerosolized BoNT/C, D, and G have also been shown to produce botulism in primates by the inhalation route (Middlebrook and Franz (1997) Botulinum Toxins, chapter 33. In F. R. Sidell, E. T. Takafuji, D. R. Franz (eds.), Medical Aspects of Chemical and Biological Warfare. TMM publications, Washington, D.C.), and would most likely also affect humans. Thus, it is likely that any one of the seven BoNT serotypes can be used as a biothreat agent.

Variability of the BoNT gene and protein sequence within serotypes has also been reported and there is evidence that such variability can affect the binding of monoclonal antibodies to BoNT/A (Kozaki et al. (1998) Infect. Immun., 66: 4811-4816; Kozaki et al. (1995) Microbiol. Immunol., 39: 767-774).

## SUMMARY

Antibodies that bind to and neutralize and/or otherwise clear botulinum neurotoxin(s) are disclosed herein. Particularly effective neutralization of a Botulism neurotoxin (BoNT) serotype can be achieved by the use of neutralizing antibodies that bind two or more subtypes of the particular neurotoxin serotype with particularly high affinity and/or by combinations of such antibodies. The present disclosure provides antibodies that bind BoNT subtypes BoNT/A, BoNT/B, and BoNT/E. Compositions comprising neutralizing antibodies that bind two or more BoNT subtypes (e.g., BoNT/B1, BoNT/B2, BoNT/B3, etc.) with high affinity are also provided herein.

A neutralizing antibody for Botulinum neurotoxin (BoNT) is provided herein. The antibody typically comprises at least one VH complementarily determining region (CDR) selected from the group consisting of a 2A10 VH CDR, a 3E1VH CDR, a 3E2VH CDR, a 3E3VH CDR, a 3E4VH CDR, a 3E4.1VH CDR, 3E5VH CDR, a 3E6VH CDR, a 3E6.1VH CDR, a 4E1 IVH CDR, a 4E13VH CDR, a 4E16VH CDR, a 4E16.1VH CDR, a 4E17VH CDR, a 4E17.1VH CDR, an A12 VH CDR, a 6A12 VH CDR, a B1.1 VH CDR, a B6 VH CDR, a B6.1 VH CDR, a B8 VH CDR, a B8.1 VHCDR, a B11 VH CDR, a B11C3 VH CDR, a B11E8 VH CDR, a B12 VH CDR, a B12.1 VH CDR, a B12.2 VH CDR, a 1B18 VH CDR, a 2B18.1 VH CDR, a 4B19 VH CDR, and a 1B22 VH CDR, a 1B10 VH CDR, a 1B10.1 VH CDR, a 2B18.2 VH CDR, a 5 2B18.3 VH CDR, a 1B22.4 VH CDR, a 2B23 VH CDR, a 2B24 VH CDR, a 2B25 VH CDR, a 2B25.1 VH CDR, a 2B26 VH CDR, a 2B27 VH CDR, a 2B28 VH CDR, a 2B29 VH CDR, a 2B30 VH CDR, a 4B17.1 VH CDR, a 4B17.1C VH CDR, a 4B17.1D VH CDR, a 4B17.1F VH CDR, a 4B17.1G VH CDR, a 3E6.2 VH CDR, a 4E17.4 VH CDR, a 4E17.6 VH CDR, B11.H12 VH CDR, B11.E9 VH CDR, 4B1 VH CDR, 4B3 VH CDR, 4B5 VH CDR, 4B6 VH CDR, 4B7 VH CDR, 1B14 VH CDR, 4A1 VH CDR, 4A1.1 VH CDR, 5A20.4 VH CDR, ING1.1C1 VH CDR, ING1.5B1 VH CDR, ING1.2B10 VH CDR, and ING1.3C2 VH CDR, and/or at least one VL complementarity determining region selected from the group consisting of a 2A10 VL CDR, a 3E1VL CDR, a 3E2 VL CDR, a 3E3 VL CDR, a 3E4 VL CDR, a 3E4.1VL CDR, a 3E5 VL CDR, a 3E6 VL CDR, a 3E6.1 VL CDR, a 4E11 VL CDR, a 4E13 VL CDR, a 4E16 VL CDR, a 4E16.1 VL CDR, a 4E17 VL CDR, a 4E17.1 VL CDR, an A12 VL CDR, a 6A12 VL CDR, a B1.1 VL CDR, a B6 VL CDR, a B6.1 VL CDR, a B8 VL CDR, a B8.1 VL CDR, a B11 VLCDR, a B11C3 VL CDR, a B11E8 VL CDR, a B12 VL CDR, a B12.1 VL CDR, a B12.2 VL CDR, a 1B18 VL CDR, a 2B18.1 VL CDR, a 4B19 VL CDR, and a 1B22 VL CDR, a 1B10 VL CDR, a 1B10.1 VL CDR, a 2B18.2 VL CDR, a 2B18.3 VL CDR, a 1B22.4 VL CDR, a 2B23 VL CDR, a 2B24 VL CDR, a 2B25 VL CDR, a 2B25.1 VL CDR, a 2B26 VL CDR, a 2B27 VL CDR, a 2B28 VL CDR, a 2B29 VL CDR, a 2B30 VL CDR, a 4B17.1 VL CDR, a 4B17.1C VL CDR, a 4B17.1D VL CDR, a 4B17.1F VL CDR, a 4B17.1G VL CDR, a 3E6.2 VL CDR, a 4E17.4 VL CDR, a 4E17.6 VL CDR, B11.H12 VL CDR, B11.E9 VL CDR, 4B1 VL CDR, 4B3 VL CDR, 4B5 VL CDR, 4B6 VL CDR, 4B7 VL CDR, 1B14 VL CDR, 4A1 VL CDR, 4A1.1 VL CDR, 5A20.4 VL CDR, ING1.1C1 VL CDR, ING1.5B1 VL CDR, ING1.2B10 VL CDR, and ING1.3C2 VL CDR.

The antibody may contain the VH CDRs of an antibody selected from the group consisting of 1B10, 1B10.1, 2B18.2, 2B18.3, 1B22.4, 2B23, 2B24, 2B25, 2B25.1, 2B26, 2B27, 2B28, 2B29, 2B30, 4B17.1, 4B17.1C, 4B17.1D, 4B17.1F, 4B17.1G, 3E6.2, 4E17.4, 4E17.6 B11.H12, B11.E9, 4B1, 4B3, 4B5, 4B6, 4B7, 1B14, 4A1, 4A1.1, 5A20.4, ING1.1C1, ING1.5B1, ING1.2B10, and ING1.3C2; and/or the VL CDRs of an antibody selected from the group consisting of 1B10, 1B10.1, 2B18.2, 2B18.3, 1B22.4, 2B23, 2B24, 2B25, 2B25.1, 2B26, 2B27, 2B28, 2B29, 2B30, 4B17.1, 4B17.1C, 4B17.1D, 4B17.1F, 4B17.1G, 3E6.2, 4E17.4, 4E17.6 B11.H12, B11.E9, 4B1, 4B3, 4B5, 4B6, 4B7, 1B14, 4A1, 4A1.1, 5A20.4, ING1.1C1, ING1.5B1, ING1.2B10, and ING1.3C2.

The antibody may contain the VH and VL CDRs of an antibody selected from the group consisting of 1B10, 1B10.1, 2B18.2, 2B18.3, 1B22.4, 2B23, 2B24, 2B25, 2B25.1, 2B26, 2B27, 2B28, 2B29, 2B30, 4B17.1, 4B17.1C, 4B17.1D, 4B17.1F, 4B17.1G, 3E6.2, 4E17.4, 4E17.6 B11.H12, B11.E9, 4B1, 4B3, 4B5, 4B6, 4B7, 1B14, 4A1, 4A1.1, 5A20.4, ING1.1C1, ING1.5B1, ING1.2B10, and ING1.3C2

The antibody may contain the VH and VL domains of an antibody selected from the group consisting of 1B10, 1B10.1, 2B18.2, 2B18.3, 1B22.4, 2B23, 2B24, 2B25, 2B25.1, 2B26, 2B27, 2B28, 2B29, 2B30, 4B17.1, 4B17.1C, 4B17.1D, 4B17.1F, 4B17.1G, 3E6.2, 4E17.4, 4E17.6 B11.H12, B11.E9, 4B1, 4B3, 4B5, 4B6, 4B7, 1B14, 4A1, 4A1.1, 5A20.4, ING1.1C1, ING1.5B1, ING1.2B10, and ING1.3C2.

The antibody may be a single chain Fv (scFv), a Fab, a (Fab′)2, an (ScFv)2, and the like. The antibody may be an IgG. The antibody may be selected from the group consisting of 1B10, 1B10.1, 2B18.2, 2B18.3, 1B22.4, 2B23, 2B24, 2B25, 2B25.1, 2B26, 2B27, 2B28, 2B29, 2B30, 4B17.1, 4B17.1C, 4B17.1D, 4B17.1F, 4B17.1G, 3E6.2, B6.1, B11E8, 4E17.1, 4E16.1, 3E6.1, B12.1, 4E17.4, 4E17.6 B11.H12, B11.E9, 4B1, 4B3, 4B5, 4B6, 4B7, 1B14, 4A1, 4A1.1, 5A20.4, ING1.1C1, ING1.5B1, ING1.2B10, and ING1.3C2. The antibody may also be in a pharmaceutically acceptable excipient (e.g., in a unit dosage formulation).

Methods of inhibiting the activity of Botulinum neurotoxin in a mammal may involve administering to a mammal in need thereof a composition comprising at least one neutralizing anti-BoNT antibody as described herein. The composition may include at least two different antibodies, each of which binds to different BoNT subtypes. The composition may also include at least three, at least four, or more different antibodies, each of which may bind to different BoNT epitopes.

Compositions provided herein may partially or fully neutralize a Botulinum neurotoxin (BoNT). The compositions typically include a first antibody that binds a BoNT/B or a BoNT/E serotype, e.g., one or more antibodies as described above, and a second antibody that binds a BoNT serotype selected from the group consisting of BoNT/A, BoNT/B, BoNT/C, BoNT/D, BoNT/E, and BoNT/F.

Nucleic acids provided herein encode one or more antibodies that are described herein. Cells containing such antibodies are also provided herein. Kits provided for neutralizing a Botulinum neurotoxin may include a composition containing one or more antibodies as described herein. The kits optionally also include instructional materials teaching the use of the composition to neutralize a Botulinum neurotoxin. The composition may be stored in a disposable syringe.

## DEFINITIONS

A “BoNT polypeptide” refers to a Botulinum neurotoxin polypeptide (e.g., a BoNT/A polypeptide, a BoNT/B polypeptide, a BoNT/C polypeptide, and so forth). The BoNT polypeptide can refer to a full-length polypeptide or to a fragment thereof. Thus, for example, the term “BoNT/A polypeptide” refers to either a full-length BoNT/A (a neurotoxin produced by Clostridium botulinum of the type A serotype) or a fragment thereof (e.g. the HC fragment). The HC fragment of BoNT/A is an approximately 50 kDa C-terminal fragment (residues 873-1296) of BoNT/A (Lacy and Stevens (1999) J. Mol. Biol., 291: 1091-1104).

A “BoNT” serotype refers one of the standard known BoNT serotypes (e.g. BoNT/A, BoNT/B, BoNT/C, BoNT/D, BoNT/E, BoNT/F, BoNT/G etc.). BoNT serotypes differ from each other by as little as about 35% at the amino acid level (e.g., between BoNT/E and BoNT/F) up to about 66% at the amino acid level, (e.g., for BoNT/A vs BoNT/C or D). Thus, BoNT serotypes differ from each other by about 35-66% at the amino acid level.

The term “BoNT subtype” (e.g., a BoNT/A1 subtype) refers to botulinum neurotoxin gene sequences of a particular serotype (e.g., A, B, C, D, E, F, etc.) that differ from each other sufficiently to produce differential antibody binding. The subtypes may differ from each other by at least 2.5%, by at least 5%, by at least 10%, by at least 15% or up to about at least 20% at the amino acid level. The subtypes differ from each other by no more than 35%, by no more than 31.6%, by no more than 30%, or 25%, by less than about 20% or 16% at the amino acid level. BoNT subtypes may differ from each other by at least 2.6%, by at least 3%, and by at least 3,6% at the amino acid level. BoNT subtypes typically differ from each other by less than about 31.6%, by less than about 16%, at the amino acid level, other by less than about 31.6%, by less than about 16%, at the amino acid level.

An “anti-BoNT antibody” refers to an antibody that binds a BoNT polypeptide, specifically binds a BoNT polypeptide with a KD less than 10−7, less than 10−8, less than 10−9, less than 10−10, less than 10−11, or less than 10−12 or less.

“Neutralization” refers to a measurable decrease in the toxicity and/or circulating level of a Botulinum neurotoxin (e.g., BoNT/A).

“Potency” refers to the degree of protection from challenge with BoNT. This can be measured/quantified for example, as an increase in the LD50 of a Botulinum neurotoxin (BoNT). In toxicology, the median lethal dose, LD50 (abbreviation for “Lethal Dose, 50%”), or LCt50 (Lethal Concentration & Time) of a toxic substance or radiation is the dose required to kill half the members of a tested population. The LD50 usually expressed as the mass of substance administered per unit mass of test subject, such as grams of substance per kilogram of body mass. Stating it this way allows the relative toxicity of different substances to be compared, and normalizes for the variation in the size of the animals exposed (although toxicity does not always scale simply with body mass). Typically, the LD50 of a substance is given in milligrams per kilogram of body weight. In the case of some toxins, the LD50 may be more conveniently expressed as micrograms per kilogram (μg/kg) of body mass.

The term “high affinity” when used with respect to an antibody refers to an antibody that specifically binds to its target(s) with an affinity (KD) of at least about 10−8 M, preferably at least about 10−9 M, at least about 10−10 M, and at least about 10−11 M. “High affinity” antibodies may have a KD that ranges horn about 1 nM to about 5 pM.

The following abbreviations are used herein: AMP, ampicillin; BIG, botulinum immune globulin; BoNT, botulinum neurotoxin; BoNT/A, BoNT type A; BoNT/B: Botulinum neurotoxin serotype B, BoNT/A HC: Botulinum neurotoxin serotype A binding domain, C-terminal domain of the BoNT/A heavy chain; BoNT/A HN: Botulinum neurotoxin serotype A translocation domain, N-terminal domain of the BoNT/A heavy chain; BoNT/A LC: Botulinum neurotoxin serotype A catalytic domain; CDR, complementarity determining region; ELISA, enzyme-linked immunosorbent assay; GLU, glucose; HBS, HEPES-buffered saline (10 mM HEPES, 150 mM NaCl [pH 7.4]); IgG, immunoglobulin G; IMAC, immobilized-metal affinity chromatography; IPTG, isopropyl-β-D-thiogalactopyranoside; KAN, kanamycin; KD, equilibrium constant; koff, dissociation rate constant; kon, association rate constant; MPBS, skim milk powder in PBS; NTA, nitrilotriacetic acid; PBS, phosphate-buffered saline (25 mM NaH2PO4, 125 mM NaCl [pH 7.0]; RU, resonance units; scFv, single-chain Fv antibody fragment; TPBS, 0.05% (vol/vol) Tween 20 in PBS; TMPBS, 0.05% (vol/vol) Tween 20 in MPBS; TU, transducing units; VH, immunoglobulin heavy-chain variable region; VK, immunoglobulin kappa light-chain variable region; VL immunoglobulin light-chain variable region; wt, wild type; CDC: Centers for Disease Control, CMV: cytomegalovirus; Fab: antigen binding fragment of immunoglobulin with variable domain and first constant domain, Fc: fragment crystalizable, Fab′2; fragment, antigen binding; mAb; monoclonal antibody, FACS: fluorescence activated cell sorting, LD50; lethal dose 50%; MFI: mean fluorescent intensity, MLA: mouse lethality assay; PCR: polymerase chain reaction, SD-CAA: selective growth dextrose casamino acids media, SG-CAA media: selective growth galactose casamino acids media; AgaII; yeast agglutinin receptor, pM; picomolar, fM; femtomolar, IU; International Unit, CHO; Chinese hamster ovary cells.

The terms “polypeptide”, “peptide”, or “protein” are used interchangeably herein to designate a linear series of amino acid residues connected one to the other by peptide bonds between the alpha-amino and carboxy groups of adjacent residues. The amino acid residues are usually in the natural “L” isomeric form. However, residues in the “D” isomeric form can be substituted for any L-amino acid residue, as long as the desired functional property is retained by the polypeptide. In addition, the amino acids, in addition to the 20 “standard” amino acids, include modified and unusual amino acids, which include, but are not limited to those listed in 37 CFR (§ 1.822(b)(4)). Furthermore, it should be noted that a dash at the beginning or end of an amino acid residue sequence indicates either a peptide bond to a further sequence of one or more amino acid residues or a covalent bond to a carboxyl or hydroxyl end group. However, the absence of a dash should not be taken to mean that such peptide bonds or covalent bond to a carboxyl or hydroxyl end group is not present, as it is conventional in representation of amino acid sequences to omit such.

The term “antibody” (also used interchangeably with “immunoglobulin”) encompasses polyclonal and monoclonal antibody preparations where the antibody may be of any class of interest (e.g., IgM, IgG, and subclasses thereof), as well as preparations including hybrid antibodies, altered antibodies, F(ab′)2 fragments, F(ab) molecules, Fv fragments, single chain fragment variable displayed on phage (scFv), single chain antibodies, single domain antibodies, chimeric antibodies, humanized antibodies, and functional fragments thereof which exhibit immunological binding properties of the parent antibody molecule. The antibodies may be conjugated to other moieties, and/or may be bound to a support (e.g., a solid support), such as a polystyrene plate or bead, test strip, and the like.

Immunoglobulin polypeptides include the kappa and lambda light chains and the alpha, gamma (IgG1, IgG2, IgG3, IgG4), delta, epsilon and mu heavy chains or equivalents in other species. Full-length immunoglobulin “light chains” (usually of about 25 kDa or about 214 amino acids) comprise a variable region of about 110 amino acids at the NH2-terminus and a kappa or lambda constant region at the COOH-terminus. Full-length immunoglobulin “heavy chains” (of about 150 kDa or about 446 amino acids), similarly comprise a variable region (of about 116 amino acids) and one of the aforementioned heavy chain constant regions, e.g., gamma (of about 330 amino acids).

An immunoglobulin light or heavy chain variable region is composed of a “framework” region (FR) interrupted by three hypervariable regions, also called “complementarily determining regions” or “CDRs”. The extent of the framework region and CDRs have been defined (see, “Sequences of Proteins of Immunological Interest,” E. Kabat et al., U.S. Department of Health and Human Services, (1991 and Lefranc et al. IMGT, the international ImMunoGeneTics information system®. Nucl. Acids Res., 2005, 33, D593-D597)). A detailed discussion of the IMGTS system, including how the IMGTS system was formulated and how it compares to other systems, is provided on the World Wide Web at imgt.cines.fr/textes/IMGTScientificChart/Numbering/IMGTnumberingsTable.html. The sequences of the framework regions of different light or heavy chains are relatively conserved within a species. The framework region of an antibody, that is the combined framework regions of the constituent light and heavy chains, serves to position and align the CDRs. The CDRs are primarily responsible for binding to an epitope of an antigen. All CDRs and framework provided by the present disclosure are defined according to Kabat et al. supra, unless otherwise indicated.

An “antibody” thus encompasses a protein having one or more polypeptides that can be genetically encodable, e.g., by immunoglobulin genes or fragments of immunoglobulin genes. The recognized immunoglobulin genes include the kappa, lambda, alpha, gamma, delta, epsilon and mu constant region genes, as well as myriad immunoglobulin variable region genes. Light chains are classified as either kappa or lambda. Heavy chains are classified as gamma, mu, alpha, delta, or epsilon, which in turn define the immunoglobulin classes, IgG, IgM, IgA, IgD and IgE, respectively.

A typical immunoglobulin (antibody) structural unit is known to comprise a tetramer. Each tetramer is composed of two identical pairs of polypeptide chains, each pair having one “light” (about 25 kD) and one “heavy” chain (about 50-70 kD). The N-terminus of each chain defines a variable region of about 100 to 110 or more amino acids primarily responsible for antigen recognition. The terms variable light chain (VL) and variable heavy chain (VH) refer to these light and heavy chains respectively.

Antibodies encompass intact immunoglobulins as well as a number of well characterized fragments produced by digestion with various peptidases. Thus, for example, pepsin digests an antibody below the disulfide linkages in the hinge region to produce F(ab)′2, a dimer of Fab which itself is a light chain joined to VH-CH1 by a disulfide bond. The F(ab)′2 may be reduced under mild conditions to break the disulfide linkage in the hinge region thereby converting the (Fab′)2 dimer into an Fab′ monomer. The Fab′ monomer is essentially an Fab with part of the hinge region (see, Fundamental Immunology, W. E. Paul, ed., Raven Press, N.Y. (1993), for a more detailed description of other antibody fragments). While various antibody fragments are defined in terms of the digestion of an intact antibody, one of skill will appreciate that such Fab′ fragments may be synthesized de novo either chemically or by utilizing recombinant DNA methodology. Thus, the term antibody, as used herein also includes antibody fragments either produced by the modification of whole antibodies or synthesized de nova using recombinant DNA methodologies, including, but are not limited to, Fab′2, IgG, IgM, IgA, scFv, dAb, nanobodies, unibodies, and diabodies.

Antibodies and fragments of the present disclosure encompass those that are bispecific. Bispecific antibodies or fragments can be of several configurations. For example, bispecific antibodies may resemble single antibodies (or antibody fragments) but have two different antigen binding sites (variable regions). Bispecific antibodies may be produced by chemical techniques (Kranz et al. (1981) Proc. Natl. Acad. Sci., USA, 78: 5807), by “polydoma” techniques (see, e.g., U.S. Pat. No. 4,474,893), or by recombinant DNA techniques. Bispecific antibodies may have binding specificities for at least two different epitopes, at least one of which is an epitope of BoNT. The BoNT binding antibodies and fragments can also be heteroantibodies. Heteroantibodies are two or more antibodies, or antibody binding fragments (e.g., Fab) linked together, each antibody or fragment having a different specificity.

An “antigen-binding site” or “binding portion” refers to the part of an immunoglobulin molecule that participates in antigen binding. The antigen binding site is formed by amino acid residues of the N-terminal variable (“V”) regions of the heavy (“H”) and light (“L”) chains. Three highly divergent stretches within the V regions of the heavy and light chains are referred to as “hypervariable regions” which are interposed between more conserved flanking stretches known as “framework regions” or “FRs”. Thus, the term “FR” refers to amino acid sequences that are naturally found between and adjacent to hypervariable regions in immunoglobulins. In an antibody molecule, the three hypervariable regions of a light chain and the three hypervariable regions of a heavy chain are disposed relative to each other in three dimensional space to form an antigen binding “surface”. This surface mediates recognition and binding of the target antigen. The three hypervariable regions of each of the heavy and light chains are referred to as “complementarily determining regions” or “CDRs” and are characterized, for example by Kabat et al. Sequences of proteins of immunological interest, 4th ed. U.S. Dept. Health and Human Services, Public Health Services, Bethesda, Md. (1987).

A 1B10 antibody refers to an antibody expressed by clone 1B10 or to an antibody synthesized in other manners, but having the same CDRs and optionally, the same framework regions as the antibody expressed by clone 1B10. Similarly, antibodies 1B10.1, 2B18.2, 2B18.3, 1B22.4, 2B23, 2B24, 2B25, 2B25.1, 2B26, 2B27, 2B28, 2B29, 2B30, 4B17.1, 4B17.1C, 4B17.1D, 4B17.1G, 3E6.2, 4E17.4, 4E17.6, 4B19.1, B6.C12, B6.D2, B11.A5, B11.E9, B11.F7, B6.1, B11E8, 4E17.1, 4E16.1, 3E6.1, B12.1, B11.H12 B11.E9, 4B1, 4B3, 4B5, 4B6, 4B7, 1B14, 4A1, 4A1.1, 5A20.4, ING1.1C1, ING1.5B1, ING1.2B10, ING1.3C2, and the like refer to antibodies expressed by the corresponding clone(s) and/or to antibodies synthesized in other manners, but having the same CDRs and optionally, the same framework regions as the referenced antibodies.

As used herein, the terms “immunological binding” and “immunological binding properties” refer to the non-covalent interactions of the type which occur between an immunoglobulin molecule and an antigen for which the immunoglobulin is specific. The strength or affinity of immunological binding interactions can be expressed in terms of the dissociation constant (KD) of the interaction, wherein a smaller KD represents a greater affinity. Immunological binding properties of selected polypeptides can be quantified using methods well known in the art. One such method entails measuring the rates of antigen binding site/antigen complex formation and dissociation, wherein those rates depend on the concentrations of the complex partners, the affinity of the interaction, and on geometric parameters that equally influence the rate in both directions. Thus, both the “on rate constant” (kon) and the “off rate constant” (koff) can be determined by calculation of the concentrations and the actual rates of association and dissociation. The ratio of koff/kon enables cancellation of all parameters not related to affinity and is thus equal to the equilibrium dissociation constant KD (see, generally, Davies et al. Ann. Rev. Biochem. 1990, 59: 439-15 473).

A “BoNT-neutralizing antibody” refers to an antibody that binds to one or more Botulinum neurotoxin(s) (e.g., BoNT/B1, BoNT/B2, etc.) and that by so-binding reduces the toxicity and/or circulating level of that BoNT neurotoxin. Thus, for example the term “BoNT/B-neutralizing antibody”, as used herein refers to an antibody that specifically binds to a BoNT/B polypeptide (e.g, a BoNT/B1 polypeptide). An exemplary antibody may bind to an HC domain of a BoNT/B polypeptide and reduces the toxicity and/or circulating level of the BoNT/B polypeptide. Reduced toxicity can be measured as an increase in the time that paralysis developed and/or as a lethal dosage (e.g., LD50) as described herein. Antibodies derived from BoNT-neutralizing antibodies include, but are not limited to, the antibodies whose sequence is expressly provided herein.

Antibodies derived from BoNT-neutralizing antibodies have a binding affinity of about 1.6×10−8 or better and can be derived by screening libraries of single chain Fv fragments displayed on phage or yeast constructed from heavy (VH) and light (VL) chain variable region genes obtained from mammals, including mice and humans, immunized with botulinum toxoid, toxin, or BoNT fragments. Antibodies can also be derived by screening phage or yeast display libraries in which a known BoNT-neutralizing variable heavy (VH) chain is expressed in combination with a multiplicity of variable light (VL) chains or conversely a known BoNT-neutralizing variable light chain is expressed in combination with a multiplicity of variable heavy (VH) chains. BoNT-neutralizing antibodies also include those antibodies produced by the introduction of mutations into the variable heavy or variable light complementarily determining regions (CDR1, CDR2 or CDR3) as described herein. Finally BoNT-neutralizing antibodies include those antibodies produced by any combination of these modification methods as applied to the BoNT-neutralizing antibodies described herein and their derivatives.

An “epitope” is a site on an antigen (e.g. BoNT) to which an antibody binds. Epitopes can be formed both from contiguous amino acids or noncontiguous amino acids juxtaposed by tertiary folding of a protein. Epitopes formed from contiguous amino acids are typically retained on exposure to denaturing solvents whereas epitopes formed by tertiary folding are typically lost on treatment with denaturing solvents. An epitope typically includes at least 3, and more usually, at least 5 or 8-10 amino acids in a spatial conformation. Methods of detemining spatial conformation of epitopes include, for example, x-ray crystallography and 2-dimensional nuclear magnetic resonance. See, e.g., Epitope Mapping Protocols in Methods in Molecular Biology, Vol. 66, Glenn E. Morris, Ed (1996).

A neutralizing epitope refers to the epitope specifically bound by a neutralizing antibody.

“Isolated” refers to an entity of interest that is in an environment different from that in which the compound may naturally occur. An “isolated” compound is separated from all or some of the components that accompany it in nature and may be substantially enriched. “Isolated” also refers to the state of a compound separated from all or some of the components that accompany it during manufacture (e.g., chemical synthesis, recombinant expression, culture medium, and the like).

A single chain Fv (“scFv”) polypeptide is a covalently linked VH:VL heterodimer which may be expressed from a nucleic acid including VH- and VL-encoding sequences either joined directly or joined by a peptide-encoding linker (Huston, et al. (1988) Proc. Nat. Acad. Sci. USA, 85: 5879-5883). A number of structures are available for converting the light and heavy polypeptide chains from an antibody V region into an scFv molecule which will fold into a three dimensional structure substantially similar to the structure of an antigen-binding site, See, e.g. U.S. Pat. Nos. 5,091,513 and 5,132,405 and 4,956,778.

Recombinant design methods may be used to develop suitable chemical structures (linkers) for converting two heavy and light polypeptide chains from an antibody variable region into a scFv molecule which will fold into a three-dimensional structure that is substantially similar to native antibody structure.

Design criteria include determination of the appropriate length to span the distance between the C-terminal of one chain and the N-terminal of the other, wherein the linker is generally formed from small hydrophilic amino acid residues that do not tend to coil or form secondary structures. Such methods have been described in the art. See, e.g., U.S. Pat. Nos. 5,091,513 and 5,132,405 to Huston et al.; and U.S. Pat. No. 4,946,778 to Ladner et al.

In this regard, the first general step of linker design involves identification of plausible sites to be linked. Appropriate linkage sites on each of the VH and VL polypeptide domains include those which will result in the minimum loss of residues from the polypeptide domains, and which will necessitate a linker comprising a minimum number of residues consistent with the need for molecule stability. A pair of sites defines a “gap” to be linked. Linkers connecting the C-terminus of one domain to the N-terminus of the next generally comprise hydrophilic amino acids which assume an unstructured configuration in physiological solutions and may be free of residues having large side groups which might interfere with proper folding of the VH and VL chains. Thus, suitable linkers generally comprise polypeptide chains of alternating sets of glycine and serine residues, and may include glutamic acid and lysine residues inserted to enhance solubility. One particular linker has the amino acid sequence (Gly4Ser)3 (SEQ ID NO:109). Another particularly preferred linker has the amino acid sequence comprising 2 or 3 repeats of [(Ser)4Gly] (SEQ ID NO:110), such as [(Ser)4Gly]3 (SEQ ID NO:111), and the like. Nucleotide sequences encoding such linker moieties can be readily provided using various oligonucleotide synthesis techniques known in the art (see, e.g., Sambrook, supra.).

The phrase “specifically binds to” or “specifically immunoreactive with”, when referring to an antibody refers to a binding reaction which is determinative of the presence of the protein in the presence of a heterogeneous population of proteins and other biologics. Thus, under designated immunoassay conditions, the specified antibodies bind to a particular protein and do not bind in a significant amount to other proteins present in the sample. Specific binding to a protein under such conditions may require an antibody that is selected for its specificity for a particular protein. For example, BoNT/B-neutralizing antibodies can be raised to BoNT/B protein(s) that specifically bind to BoNT/B protein(s), and not to other proteins present in a tissue sample. A variety of immunoassay formats may be used to select antibodies specifically immunoreactive with a particular protein. For example, solid-phase ELISA immunoassays are routinely used to select monoclonal antibodies specifically immunoreactive with a protein. See Harlow and Lane (1988) Antibodies, A Laboratory Manual, Cold Spring Harbor Publications, New York, for a description of immunoassay formats and conditions that can be used to determine specific immunoreactivity.

The term “conservative substitution” is used in reference to proteins or peptides to reflect amino acid substitutions that do not substantially alter the activity (specificity or binding affinity) of the molecule. Typically conservative amino acid substitutions involve substituting one amino acid for another amino acid with similar chemical properties (e.g. charge or hydrophobicity). The following six groups each contain amino acids that are typical conservative substitutions for one another: 1) Alanine (A), Serine (S), Threonine (T); 2) Aspartic acid (D), Glutamic acid (E); 3) Asparagine (N), Glutamine (Q); 4) Arginine (R), Lysine (K); 5) Isoleucine (I), Leucine (L), Methionine (M), Valine (V); and 6) Phenylalanine (F), Tyrosine (Y), Tryptophan (W).

## DETAILED DESCRIPTION

This disclosure provides antibodies that specifically bind to botulinum neurotoxin, with those that bind to botulinum neurotoxin serotype B and E being of particular interest, as well as antibodies that bind other botulinum neurotoxin serotypes (e.g., A). Botulinum neurotoxin is produced by the anaerobic bacterium Clostridium botulinum. Botulinum neurotoxin poisoning (botulism) arises in a number of contexts including, but not limited to food poisoning (food borne botulism), infected wounds (wound botulism), “infant botulism” from ingestion of spores and production of toxin in the intestine of infants, and as a chemical/biological warfare agent. Botulism is a paralytic disease that typically begins with cranial nerve involvement and progresses caudally to involve the extremities. In acute cases, botulism can prove fatal.

There are multiple subtypes of various BoNT serotypes. There are also many antibodies that bind, for example the BoNT/A1 subtype but will not bind the BoNT/A2 subtype, and so forth.

The present disclosure is related to the discovery that particularly efficient neutralization of a botulism neurotoxin (BoNT) subtype is achieved by the use of neutralizing antibodies that bind two, three, or more subtypes of the particular BoNT serotype with high affinity. This may be accomplished by using two, three, four, or more different antibodies directed against each of the subtypes, or alternatively, by the use of antibodies that are cross-reactive for different BoNT subtypes, or by bispecific or polyspecific antibodies with specificities for two, three, or four or more BoNT epitopes, and/or serotypes, and/or subtypes.

It was discovered that combining neutralizing antibodies increases the potency of the antibody composition dramatically. This increase makes it possible to generate a multi-antibody, and/or multi-specific antibodies of the required potency for therapeutic use. The high potency of the antibody combinations can allow vialling of an extremely low dose of total antibody as a therapeutic dose, resulting in lower manufacturing costs. As one combines two, three or four monoclonal antibodies, the particular BoNT epitope that is recognized becomes less important. Thus, compositions containing at least two, or at least three high affinity antibodies that bind overlapping or non-overlapping epitopes on the BoNT are contemplated herein.

Thus, compositions contemplated herein may include two or more, three or more, four or more, five or more different antibodies selected from the antibodies described herein (see, e.g., FIGS. 1, 2, and 11) and/or antibodies comprising one or more CDRs from these antibodies, and/or one or more antibodies comprising mutants or derivatives of these antibodies. The composition may include antibodies selected from the group consisting of 1B10.1, 2b18.1, B11.E8, and optionally B6.1 and B8.1. The compositions may also contain antibodies 4E17.1 together with two of the following: 3E2, 3E6.1, 3E6.2, 4E16.1.

Compositions contemplated herein may include monovalent BoNT/B antitoxins (e.g. comprising B12.1, B11.E8, B6.1, 2B18.1 and 1B10.1). Compositions may also include monovalent BoNT/E antitoxins (e.g. comprising 3E2, 3E6.1, 3E6.2, 4E16.1, and 4E17.1). Compositions containing trivalent BoNT/A, BoNT/B and BoNT/E toxins (e.g. comprising antibodies selected from those described in PCT Pub. Nos. WO 07/094754, WO 05/016232, and 09/008916, and B12.1, B11.E8, B6.1, 2B18.1, 1B10.1, 3E2, 3E6.1, 3E6.2, 4E16.1, and 4E17.1) are also contemplated.

As indicated above, the antibodies provided by the present disclosure bind to one or more botulinum neurotoxin type B, E, and in certain instances Bont/A subtypes, and, in some embodiments, can neutralize the neurotoxin. Neutralization, in this context, refers to a measurable decrease in the toxicity and/for circulating level of the target neurotoxin. Such a decrease in toxicity can be measured in vitro by a number of methods well known to those of skill in the art. One such assay involves measuring the time to a given percentage (e.g., 50%) twitch tension reduction in a hemidiaphragm preparation. Toxicity can be determined in vivo, e.g. as an LD50 in a test animal (e.g. mouse) botulinum neurotoxin type A in the presence of one or more putative neutralizing antibodies. The neutralizing antibody or antibody combination can be combined with the botulinum neurotoxin prior to administration, or the animal can be administered the antibody prior to, simultaneous with, or after administration of the neurotoxin. The rate of clearance of BoNT mediated by a test antibody, or combination of test antibodies, can be measured (e.g. in mice) by administering labeled BoNT (e.g. radiolabeled BoNT/A) and measuring the levels of BoNT in the serum and the liver over time in the presence or absence of test antibody or antibodies (see, e.g., Ravichandran et al. (2006) J Pharmacol Exp Ther 318: 1343-1351 (2006).

As the antibodies of the present disclosure act to neutralize botulinum neurotoxins, they are useful in the treatment of pathologies associated with botulinum neurotoxin poisoning. The treatments essentially comprise administering to the poisoned organism (e.g. human or non-human mammal) a quantity of one or more neutralizing antibodies sufficient to neutralize (e.g. mitigate or eliminate) symptoms of BoNT poisoning.

Such treatments are most desired and efficacious in acute cases (e.g. where vital capacity is less than 30-40 percent of predicted and/or paralysis is progressing rapidly and/or hypoxemia with absolute or relative hypercarbia is present. These antibodies can also be used to treat early cases with symptoms milder than indicated (to prevent progression) or even prophylactically (a use the military envisions for soldiers going in harm's way). Treatment with the neutralizing antibody can be provided as an adjunct to other therapies (e.g. antibiotic treatment).

The antibodies provided by this disclosure can also be used for the rapid detection/diagnosis of botulism (type B, E, or A toxin(s)) and thereby supplement and/or replace previous laboratory diagnostics.

This disclosure also provides the epitopes specifically bound by botulinum neurotoxin antibodies described herein. These epitopes can be used to isolate, and/or identify and/or screen for other antibodies BoNT neutralizing antibodies as described herein.

### I. Potency of Botulinum Neurotoxin (BoNT)-Neutralizing Antibodies.

Without being bound to a particular theory, it is believed that the current antitoxins used to treat botulism (horse and human) have a potency of about 5000 mouse LD50s/mg (human) and 55,000 mouse LD50s/mg (horse).

Based on calculation, a commercially desirable antitoxin may generally have a potency greater than about 10,000 to 100,000 LD50s/mg. Combinations of the antibodies described herein (e.g., two or three antibodies) can meet this potency. Thus, this disclosure provides antibodies and/or antibody combinations that neutralize at least about 10,000 mouse LD50s/mg of antibody, preferably at least about 15,000 mouse LD50s/mg of antibody, more preferably at least about 20,000 mouse LD50s/mg of antibody, and most preferably at least about 25,000 mouse LD50s/mg of antibody.

### II. Botulinum Neurotoxin (BoNT)-Neutralizing Antibodies.

BoNT neutralizing antibodies may be selected based on their affinity to one or more BoNT subtypes. A number of subtypes are known for each BoNT serotype. Thus, for example, BoNT/A subtypes include, but are not limited to, BoNT/A1, BoNT/A2, BoNT/A3, and the like (see, e.g., FIG. 3). It is also noted, for example, that the BoNT/A1 subtype includes, but is not limited to 62A, NCTC 2916, ATCC 3502, and Hall hyper (Hall Allergan) and are identical (99.9-100% identity at the amino acid level.) and have been classified as subtype A1. The BoNT/A2 sequences (Kyoto-F and FRI-A2H) (Willems, et al. (1993) Res. Microbiol. 144:547-556) are 100% identical at the amino acid level. Another BoNT/A subtype, (that we are calling A3) is produced by a strain called Loch Maree that killed a marcher of people in an outbreak in Scotland.

Similarly, as shown in FIG. 3, a number of subtypes are also known for serotypes B, C, E, and F. Using, the methods described herein, it was discovered that high affinity antibodies that are cross-reactive with two or more subtypes within a serotype can also he produced (e.g., selected/engineered). Moreover, without being bound to a particular theory, it appears that these cross-reactive antibodies can be more efficient in neutralizing Botulinum neurotoxin, particularly when used in combination one or more different neutralizing antibodies.

The sequences of the variable heavy (VH) and variable light (VL) domains for a number of prototypical BoNTB and BoNT/E antibodies are illustrated in Tables 1-5, and in FIGS. 1-2 and 11. The relationship of certain antibodies specific for BoNT/E and BoNT/B disclosed herein with respect to the clonal lineage of the first lead selected is provided in FIG. 10.

The antibodies of the present disclosure can be used individually, and/or in combination with each other, and/or in combination with other known anti-BoNT antibodies (see, e.g., Application Pub. No: 20080124328, filed on Jan. 26, 2006, application Ser. No. 09/144,886, filed on Aug. 31, 1998, Application Pub. No. 20040175385, filed on Aug. 1, 2003, 60/912,173, filed on Jun. 5, 2007 and PCT Pub. Nos. WO 07/094751, WO 05/016232, and WO 09/008916, which are incorporated herein by reference for all purposes). These antibodies can be used individually, and/or in combination with each other, and/or in combination with other know anti-BoNT antibodies to form bispecific or polyspecific antibodies.

It will be appreciated that the amino acid sequence of a CDR can also be defined using alternative systems, which will be readily apparent to and applied by the ordinarily skilled artisan (see, “Sequences of Proteins of Immunological Interest,” E. Kabat et al., U.S. Department of Health and Human Services, (1991 and Lefranc et al. IMGT, the international ImMunoGeneTics information system. Nucl. Acids Res., 2005, 33. D593-D597)). A detailed discussion of the IMGTS system, including how the IMGTS system was formulated and how it compares to other systems, is provided on the World Wide Web at imgt.cines.fr/textes/IMGTScientificChart/Numbering/IMGTnumberingsTable.html. An example of a definition of CDRs of an exemplary antibody, 1B10.1, is provided in FIG. 12. Amino acids indicated in bold are CDR regions according to IMGT boundary definitions, with underlined residues defined according to The Kabat system that resulted in the CDRs described above. All amino acid sequences of CDR in the present disclosure are defined according to Kabat et al., supra, unless otherwise indicated.

Using the teachings and the sequence information provided herein, the variable light and variable heavy chains can be joined directly or through a linker (e.g., (Gly4Ser)3, SEQ ID NO:1) to form a single-chain Fv antibody. The various CDRs and/or framework regions can be used to form human antibodies, chimeric antibodies, antibody fragments, polyvalent antibodies, and the like.

Anti-BoNT antibodies of the present disclosure have a binding affinity (KD) for a BoNT protein of at least 10−8, at least 10−9, at least 10−10, and most preferably at least 10−11, 10−12 M or less. Some exemplary KDs (M−1) for BoNT/B or BoNT/E fall in the following ranges: between 5×10−11 to 3×10−10, between 4×10−10 to 2×10−10, between 7×10−10 to 1×10−9, between 8×10−10 to 5×10−9, between 1×10−9 to 3×10−9, between 4×10−9 to 2×10−8.

The antibody of the present disclosure may be defined by the epitope of BoNT bound by the antibody. The antibodies provided here may encompass those that bind to one or more epitopes of BoNT to which an antibody containing one or more of the CDRs set forth in Tables 1-5 bind. Epitopes bound by an antibody may be described by a specific BoNT domain and/or the residues therein that contribute to the interaction between the antibody and a BoNT protein. Certain residues of the epitopes bound by the exemplary antibodies are provided in the table below.

Numbering system used for the toxin in the table above is based on Lacy et al. (1999) J. Mol. Biol. 291:1091-1104. Residues that are bolded and underlined have important contributions energetically and eliminations of these residues often lead to total loss of detectable binding by the corresponding antibodies to a particular BoNT. Based on the table above, an antibody such as 4E17.1, may be described by its affinity to the HN domain of BoNT/A. The antibody having the same epitope as 4E17.1 may also be characterized as an antibody with an epitope(s) containing one or more of the residues: Y753, E756, E757 in BoNT/A. In another example, an antibody such as 3E6.1, may be described by its affinity to the LC domain of BoNT/E. The antibody having the same epitope as 3E6.1 may be characterized as an antibody with an epitope(s) containing one or more of the residues: S604, Q607, Q608 in BoNT/E

The ability of a particular antibody to recognize the same epitope as another antibody can be determined by the ability of one antibody to competitively inhibit binding of the second antibody to the antigen. Competitive inhibition of binding may also be referred to as cross-reactivity of antibodies. Any of a number of competitive binding assays can be used to measure competition between two antibodies to the same antigen. For example, a sandwich ELISA assay can be used for this purpose. Additional methods for assaying for cross-reactivity are described later below.

A first antibody is considered to competitively inhibit binding of a second antibody, if binding of the second antibody to the antigen is reduced by at least 30%, usually at least about 40%, 50%, 60% or 75%, and often by at least about 90%, in the presence of the first antibody using any of the assays used to assess competitive binding.

Accordingly, antibodies provided by the present disclosure encompass those that compete for binding to a BoNT/E with an antibody that includes one or more of the VH CDRs set forth in Table 1 and/or one or more of the VL CDRs set forth in Table 2. Antibodies provided by the present disclosure also encompass those that compete for binding to a BoNT/B with an antibody that includes one or more of the VH CDRs set forth in Table 3 and/or one or more of the VL CDRs set forth in Table 4. Additional antibodies may encompass those that compete for binding to a BoNT/A with an antibody with one or more CDRs set forth in Table 5.

For example, an antibody may have the binding specificity (i.e., in this context, the same CDRs, or substantially the same CDRs) of an antibody having VH and VL CDRs or full length VH and VL as set forth in Tables 1-5. An antibody of the present disclosure may therefore contain a CDR as set forth in a VH or VL sequence shown in Tables 1-5 and, additionally, may have at least 80% identity, preferably, 85%, 90%, or 95% identity to a full-length VH or VL sequence. For example, an antibody may contain the CDRs of a VH and a VL sequence and human framework sequences set forth in Tables 1-5.

### III. Preparation of BoNT Neutralizing Antibodies.

A) Recombinant Expression of BoNT-Neutralizing Antibodies.

Using the information provided herein, the botulinum neurotoxin-neutralizing antibodies of the present disclosure are prepared using standard techniques well known to those of skill in the art.

For example, the polypeptide sequences provided herein (see, e.g., Tables 1-5, and/or FIGS. 1-2 and 11) can be used to determine appropriate nucleic acid sequences encoding the BoNT-neutralizing antibodies and the nucleic acids sequences then used to express one or more BoNT-neutralizing antibodies. The nucleic acid sequence(s) can be optimized to reflect particular codon “preferences” for various expression systems according to standard methods well known to those of skill in the art.

Using the sequence information provided, the nucleic acids may be synthesized according to a number of standard methods known to those of skill in the art. Oligonucleotide synthesis, is preferably carried out on commercially available solid phase oligonucleotide synthesis machines (Needham-VanDevanter et al. (1984) Nucleic Acids Res. 12:6159-6168) or manually synthesized using, for example, the solid phase phosphoramidite triester method described by Beaucage et. al. (1981) Tetrahedron Letts. 22(20): 1859-1862.

Once a nucleic acid encoding an anti-BoNT antibody is synthesized it can be amplified and/or cloned according to standard methods. Molecular cloning techniques to achieve these ends are known in the art. A wide variety of cloning and in vitro amplification methods suitable for the construction of recombinant nucleic acids are known to persons of skill. Examples of these techniques and instructions sufficient to direct persons of skill through many cloning exercises are found in Berger and Kimmel, Guide to Molecular Cloning Techniques, Methods in Enzymology volume 152 Academic Press, Inc., San Diego, Calif. (Berger); Sambrook et al. (1989) Molecular Cloning—A Laboratory Manual (2nd ed.) Vol. 1-3, Cold Spring Harbor Laboratory, Cold Spring Harbor Press, NY, (Sambrook); and Current Protocols in Molecular Biology, F. M. Ausubel et al., eds., Current Protocols, a joint venture between Greene Publishing Associates, Inc. and John Wiley & Sons, Inc., (1994 Supplement) (Ausubel). Methods of producing recombinant immunoglobulins are also known in the art. See, Cabilly, U.S. Pat. No. 4,816,567; and Queen et al. (1989) Proc. Nat'l Acad. Sci. USA 86: 10029-10033.

Examples of techniques sufficient to direct persons of skill through in vitro amplification methods, including the polymerase chain reaction (PCR) the ligase chain reaction (LCR), Qβ-replicase amplification and other RNA polymerase mediated techniques are found in Berger, Sambrook, and Ausubel, as well as Mullis et al., (1987) U.S. Pat. No. 4,683,202; PCR Protocols A Guide to Methods and Applications (Innis et al. eds) Academic Press Inc. San Diego, Calif. (1990) (Innis); Arnheim & Levinson (Oct. 1, 1990) C&EN 36-47; The Journal Of NIH Research (1991) 3, 81-94; (Kwoh et al. (1989) Proc. Natl. Acad. Sci. USA 86, 1173; Guatelli et al. (1990) Proc. Natl. Acad. Sci. USA 87, 1874; Lomell et al. (1989) J. Clin. Chem 35, 1826; Landegren et al., (1988) Science 241, 1077-1080; Van Brunt (1990) Biotechnology 8, 291-294; Wu and Wallace, (1989) Gene 4, 560; and Barringer et al. (1990) Gene 89, 117. Improved methods of cloning in vitro amplified nucleic acids are described in Wallace et al., U.S. Pat. No. 5,426,039.

Once the nucleic acid for an anti-BoNT antibody is isolated and cloned, one can express the gene in a variety of recombinantly engineered cells known to those of skill in the art. Examples of such cells include bacteria, yeast, filamentous fungi, insect (especially employing baculoviral vectors), and mammalian cells. It is expected that those of skill in the art are knowledgeable in the numerous expression systems available for expression of antibodies.

In brief summary, the expression of natural or synthetic nucleic acids encoding anti-BoNT antibodies will typically be achieved by operably linking a nucleic acid encoding the antibody to a promoter (which is either constitutive or inducible), and incorporating the construct into an expression vector. The vectors can be suitable for replication and integration in prokaryotes, eukaryotes, or both. Typical cloning vectors contain transcription and translation terminators, initiation sequences, and promoters useful for regulation of the expression of the nucleic acid encoding the anti-BoNT antibody. The vectors optionally comprise generic expression cassettes containing at least one independent terminator sequence, sequences permitting replication of the cassette in both eukaryotes and prokaryotes, i.e., shuttle vectors, and selection markers for both prokaryotic and eukaryotic systems. See Sambrook et al (1989) supra.

To obtain high levels of expression of a cloned nucleic acid it is common to construct expression plasmids which typically contain a strong promoter to direct transcription, a ribosome binding site for translational initiation, and a transcription/translation terminator. Examples of regulatory regions suitable for this purpose in E. coli are the promoter and operator region of the E. coli tryptophan biosynthetic pathway as described by Yanofsky (1984) J. Bacteriol., 158:1018-1024, and the leftward promoter of phage lambda (PL) as described by Herskowitz and Hagen (1980) Ann. Rev. Genet., 14:399-445 and the L-arabinose (araBAD) operon (Better (1999) Gene Exp Systems pp 95-107 Academic Press, Inc., San Diego, Calif.). The inclusion of selection markers in DNA vectors transformed in E. coli is also useful. Examples of such markers include genes specifying resistance to ampicillin, tetracycline, or chloramphenicol. See Sambrook et al (1989) supra for details concerning selection markers, e.g., for use in E. coli.

Expression systems for expressing anti-BoNT antibodies are available using, for example, E. coli, Bacillus sp. (see, e.g., Palva, et al. (1983) Gene 22:229-235; Mosbach et al. (1983) Nature, 302: 543-545), and Salmonella. E. coli systems may also be used.

The anti-BoNT antibodies produced by prokaryotic cells may require exposure to chaotropic agents for proper folding. During purification from, e.g., E. coli, the expressed protein is optionally denatured and then remained. This can be accomplished, e.g., by solubilizing the bacterially produced antibodies in a chaotropic agent such as guanidine HCl. The antibody is then renatured, either by slow dialysis or by gel filtration (see, e.g., U.S. Pat. No. 4,511,503). Alternatively, nucleic acid encoding the anti-BoNT antibodies may be operably linked to a secretion signal sequence such as pelB so that the anti-BoNT antibodies are secreted into the medium in correctly-folded form (Better et al (1988) Science 240: 1011-1043).

Methods of transfecting and expressing genes in mammalian cells are known in the art (see e.g. Birch and Racher Adv. Drug Deliv. Rev. 2006, 58: 671-685). Transducing cells with nucleic acids can involve, for example, incubating viral vectors containing anti-BoNT nucleic acids with cells within the host range of the vector (see, e.g., Goeddel (1990) Methods in Enzymology, vol. 185, Academic Press, Inc., San Diego, Calif. or Krieger (1990) Gene Transfer and Expression—A Laboratory Manual, Stockton Press, New York, N.Y. and the references cited therein).

The culture of cells used in the present disclosure, including cell lines and cultured cells from tissue or blood samples is well known in the art (see, e.g., Freshney (1994) Culture of Animal Cells, a Manual of Basic Technique, third edition, Wiley-Liss, N. Y. and the references cited therein).

Techniques for using and manipulating antibodies are found in Coligan (1991) Current Protocols in Immunology Wiley/Greene, NY; Harlow and Lane (1989) Antibodies: A Laboratory Manual Cold Spring Harbor Press, NY; Stites et al. (eds.) Basic and Clinical Immunology (4th ed.) Lange Medical Publications, Los Altos, Calif., and references cited therein; Goding (1986) Monoclonal Antibodies: Principles and Practice (2d ed.) Academic Press, New York, N.Y.; and Kohler and Milstein (1975) Nature 256: 495-497.

The BoNT-neutralizing antibody gene(s) (e.g. BoNT-neutralizing scFv gene) may be subcloned into the expression vector pUC119mycHis (Tomlinson et al. (1996) J. Mol. Biol., 256: 813-817) or pSYN3, resulting in the addition of a hexahistidine tag at the C-terminal end of the scFv to facilitate purification. Detailed protocols for the cloning and purification of certain BoNT-neutralizing antibodies are found, for example, in Amersdorfer et al. (1997) Infect. Immunity, 65(9): 3743-3752, and the like.

B) Preparation of Whole Polyclonal or Monoclonal Antibodies.

The anti-BoNT antibodies of the present disclosure include individual, allelic, strain, or species variants, and fragments thereof, both in their naturally occurring (full-length) forms and in recombinant forms. Certain antibodies may be selected to bind one or more epitopes bound by the antibodies described herein (e.g 2A10, 3E1, 3E2, 3E3, 3E4, 3E4,1, 3E5, 3E6, 3E6.1, 3E6.2, 4E11, 4E13, 4E16, 4E16.1, 4E17, 4E17.1, A12, 6A12, B1.1, B6, B6.1, B8, B8.1, B11, B11C3, B11E8, B12, B12.1, B12.2, 1B18, 2B18.1, 4B19, 4B19.1, 1B22, 1B10, 1B10.1, 2B18.2, 2B18.3, 1B22.4, 2B23, 2B24, 2B25, 2B25.1, 2B26, 2B27, 2B28, 2B29, 2B30, 4B17.1, 4B17.1C, 4B17.1D, 4B17.1F, 4B17.1G, 3E6.2, 4E17.4, 4E17.6, 4B19.1, B6.C12, B6.D2, B11.A5, B11.F7, B11.H12, B11.E9, 4B1, 4B3, 4B5, 4B6, 4B7, 1B14, 4A1, 4A1.1, 5A20.4, ING1.1C1, ING1.5B1, ING1.2B10, and ING1.3C2). The antibodies can be raised in their native configurations or in non-native configurations. Anti-idiotypic antibodies can also be generated. Many methods of making antibodies that specifically bind to a particular epitope are known to persons of skill. The following discussion is presented as a general overview of the techniques available; however, one of skill will recognize that many variations upon the following methods are known.

1) Polyclonal Antibody Production.

Methods of producing polyclonal antibodies are known to those of skill in the art. In brief, an immunogen (e.g., BoNT/A, BoNT/B, BoNT/E, etc.), subsequences including, but not limited to subsequences comprising epitopes specifically bound by antibodies expressed by clones clones 2A10, 3E1, 3E2, 3E3, 3E4, 3B4.1, 3E5, 3E6, 3E6.1, 3E6.2, 4E11, 4E13, 4E16, 4E16.1, 4E17, 4E17.1, n A12, 6A12, B1.1, B6, B6.1, B8, B8.1, B11, B11C3, B11E8, B12, B12.1, B12.2, 1B18, 2B18.1, 4B19, 4B19.1, 1B22, 1B10, 1B10.1, 2B18.2, 2B18.3, 1B22.4, 2B23, 2B24, 2B25, 2B25.1, 2B26, 2B27, 2B28, 2B29, 2B30, 4B17.1, 4B17.1C, 4B17.1D, 4B17.1F, 4B17.1G, 3E6.2, 4E17.4, 4E17.6, 4B19.1, B6.C12, B6.D2, B11.A5, B11.F7, B11.H12, B11.E9, 4B1, 4B3, 4B5, 4B6, 4B7, 1B14, 4A1, 4A1.1, 5A20.4, ING1.1C1, ING1.5B1, ING1.2B10, and ING1.3C2 disclosed herein, preferably a purified polypeptide, a polypeptide coupled to an appropriate carrier (e.g., GST, keyhole limpet hemanocyanin, etc.), or a polypeptide incorporated into an immunization vector such as a recombinant vaccinia virus (see, U.S. Pat. No. 4,722,848) is mixed with an adjuvant and animals are immunized with the mixture. The animal's immune response to the immunogen preparation is monitored by taking test bleeds and determining the titer of reactivity to the polypeptide of interest. When appropriately high titers of antibody to the immunogen are obtained, blood is collected from the animal and antisera are prepared. Further fractionation of the antisera to enrich for antibodies reactive to the BoNT polypeptide is performed where desired (see, e.g., Coligan (1991) Current Protocols in Immunology Wiley/Greene, NY; and Harlow and Lane (1989) Antibodies: A Laboratory Manual Cold Spring Harbor Press, NY).

Antibodies that specifically bind to the neutralizing epitopes described herein can be selected from polyclonal sera using the selection techniques described herein.

2) Monoclonal Antibody Production.

In some instances, it is desirable to prepare monoclonal antibodies from various mammalian hosts, such as mice, rodents, primates, humans, etc. Descriptions of techniques for preparing such monoclonal antibodies are found in, e.g., Stites et al. (eds.) Basic and Clinical Immunology (4th ed.) Lange Medical Publications, Los Altos, Calif., and references cited therein; Harlow and Lane, supra; Goding (1986) Monoclonal Antibodies: Principles and Practice (2d ed.) Academic Press, New York, N.Y.; and Kohler and Milstein (1975) Nature 256: 495-497.

Summarized briefly, monoclonal antibody production using hybridomas may proceed by injecting an animal with an (e.g., BoNT/A, BoNT/B, BoNT/E, etc.) subsequences including, but not linnted to subsequences comprising epitopes specifically bound by antibodies expressed by clones 2A10, 3E1, 3E2, 3E3, 3E4, 3E4.1, 3E5, 3E6, 3E6.1, 3E6.2, 4E11, 4E13, 4E16, 4E16.1, 4E17, 4E17.1, A12, 6A12, B1.1, B6, B6.1, B8, B8.1, B11, B11C3, B11.E8, B12, B12.1, B12.2, 1B18, 2B18.1, 4B19, 4B19.1, 1B22, 1B10, 1B10.1, 2B18.2, 2B18.3, 1B22.4, 2B23, 2B24, 2B25, 2B25.1, 2B26, 2B27, 2B28, 2B29, 2B30, 4B17.1, 4B17.1C, 4B17.1D, 4B17.1F, 4B17.1G, 3E6.2, 4E17.4, 4E17.6, 4B19.1, B6.C12, B6.D2, B11.A5, B11.F7, B11.H12, B11E9, 4B1, 4B3, 4B5, 4B6, 4B7, 1B14, 4A1, 4A1.1, 5A20.4, ING1.1C1, ING1.5B1, ING1.2B10, and ING1.3C2 disclosed herein. The animal is then sacrificed and cells taken from its spleen, which are fused with myeloma cells. The result is a hybrid cell or “hybridoma” that is capable of reproducing antibodies in vitro. The population of hybridomas is then screened to isolate individual clones, each of which secretes a single antibody species to the immunogen. In this manner, the individual antibody species obtained are the products of immortalized and cloned single B cells from the immune animal generated in response to a specific site recognized on the immunogenic substance.

Alternative methods of immortalization include transformation with Epstein Barr Virus, oncogenes, or retroviruses, or other methods known in the art. Colonies arising from single immortalized cells are screened for production of antibodies of the desired specificity and affinity for the BoNT antigen, and yield of the monoclonal antibodies produced by such cells is enhanced by various techniques, including injection into the peritoneal cavity of a vertebrate (preferably mammalian) host. The antibodies of the present disclosure are used with or without modification, and include chimeric antibodies such as humanized murine antibodies.

Techniques for creating recombinant DNA versions of the antigen-binding regions of antibody molecules which bypass the generation of hybridomas are contemplated for the present BoNT (e.g., BoNT/B) binding antibodies and fragments. DNA is cloned into a bacterial expression system. One example of a suitable technique uses a bacteriophage lambda vector system having a leader sequence that causes the expressed Fab protein to migrate to the periplasmic space (between the bacterial cell membrane and the cell wall) or to be secreted. One can rapidly generate and screen great numbers of functional Fab fragments for those which bind BoNT. Such BoNT binding agents (Fab fragments with specificity for a BoNT polypeptide) are specifically encompassed within the BoNT binding antibodies and fragments of the present disclosure. Other methods for screening and production of antibodies may employ one or more of display systems such as phage display, yeast display, ribosome, etc., and an antibody production system such as that derived front transgenic mice.

### IV. Modification of BoNT Neutralizing Antibodies.

A) Display Techniques can be Used to Increase Antibody Affinity.

To create higher affinity antibodies, mutant scFv gene repertories, based on the sequence of a binding scFv (see, e.g., Tables 1-5, and/or FIGS. 1, 2, and/or 11), can be created and expressed on the surface of phage. Display of antibody fragments on the surface of viruses which infect bacteria (bacteriophage or phage) makes it possible to produce human or other mammalian antibodies (e.g., scFvs) with a wide range of affinities and kinetic characteristics. To display antibody fragments on the surface of phage (phage display), an antibody fragment gene is inserted into the gene encoding a phage surface protein (e.g., pIII) and the antibody fragment-pIII fusion protein is expressed on the phage surface (McCafferty et al. (1990) Nature, 348: 552-554; Hoogenboom et al. (1991) Nucleic Acids Res., 19: 4133-4137).

Since the antibody fragments on the surface of the phage are functional, those phage bearing antigen binding antibody fragments can be separated from non-binding or lower affinity phage by antigen affinity chromatography (McCafferty et al. (1990) Nature, 348: 552-554). Mixtures of phage are allowed to bind to the affinity matrix, non-binding or lower affinity phage are removed by washing, and bound phage are eluted by treatment with acid or alkali.

By infecting bacteria with the eluted phage or modified variants of the eluted phage as described below, more phage can be grown and subjected to another round of selection. In this way, an enrichment of 1000 fold in one round may become 1,000,000 fold in two rounds of selection (see, e.g., McCafferty et al. (1990) Nature, 348: 552-554). Thus, even when enrichments in each round are low, multiple rounds of affinity selection leads to the isolation of rare phage and the genetic material contained within which encodes the sequence of the binding antibody (see, e.g., Marks et al. (1991) J. Mol. Biol., 222: 581-597). The physical link between genotype and phenotype provided by phage display makes it possible to test every member of an antibody fragment library for binding to antigen, even with libraries as large as 100,000.000 clones. For example, after multiple rounds of selection on antigen, a binding scFv that occurred with a frequency of only 1/30,000,000 clones was recovered (Marks et al. (1991) J. Mol. Biol., 222: 581-597.).

Yeast display may also be utilized to increase antibody affinity and has the ability to finely discriminate between mutants of close affinity. Antibody variable region genes (V-genes) may be diversified either randomly or using spiked oligonucleotides, and higher affinity mutants selected using various types of affinity chromatography or flow cytometry.

1) Chain Shuffling.

One approach for creating mutant scFv gene repertoires involves replacing either the VH or VL gene from a binding scFv with a repertoire of VH or VL genes (chain shuffling) (see, e.g., Clackson et al. (1991) Nature, 352: 624-628). Such gene repertoires contain numerous variable genes derived from the same germline gene as the binding scFv, but with point mutations (see, e.g., Marks et al. (1992) Bio/Technology, 10: 779-783). Using light or heavy chain shuffling and phage display, the binding avidities of, e.g., BoNT/E or BoNT/B-neutralizing antibody fragment can be dramatically increased (see, e.g., Marks et al. (1992) Bio/Technology, 10: 779-785 in which the affinity of a human scFv antibody fragment which bound the hapten phenyloxazolone (phox) was increased from 300 nM to 15 nM (20 fold)).

Thus, to alter the affinity of BoNT-neutralizing antibody a mutant scFv gene repertoire may be created containing the VH gene of a known BoNT-neutralizing antibody (e.g., 2A10, 3E1, 3E2, 3E3, 3E4, 3E4.1, 3E5, 3E6, 3E6.1, 3E6.2, 4E11, 4E13, 4E16, 4E16.1, 4E17, 4E17.1, n A12, 6A12, B1.1, B6, B6.1, B8, B8.1, B11, B11C3, B11E8, B12, B12.1, B12.2, 1B18, 2B18.1, 4B19, 4B19.1, 1B22, 1B10, 1B10.1, 2B18.2, 2B18.3, 1B22.4, 2B23, 2B24, 2B25, 2B25.1, 2B26, 2B27, 2B28, 2B29, 2B30, 4B17.1, 4B17.1C, 4B17.1D, 4B17.1F, 4B17.1G, 3E6.2, 4E17.4, 4E17.6, 4B19.1, B6.C12, B6.D2, B11.A5, B11.F7, B11.H12, B11.E9, 4B1, 4B3, 4B5, 4B6, 4B7, 1B14, 4A1, 1A1.1, 5A20.4, ING1.1C1, ING1.5B1, ING1.2B10, and ING1.3C2) and a VL gene repertoire (light chain shuffling). Alternatively, an scFv gene repertoire is created containing the VL gene of a known BoNT-neutralizing antibody (e.g 2A10, 3E1, 3E2, 3E3, 3E4, 3E4.1, 3E5, 3E6, 3E6.1, 3E6.2, 4E11, 4E13, 4E16, 4E16.1, 4E17, 4E17.1, n A12, 6A12, B1.1, B6, B6.1, B8, B8.1, B11, B11C3, B11E8, B12, B12.1, 1B12.2, 1B18, 2B18.1, 4B19, 4B19.1, 1B22, 1B10, 1B10.1, 2B18.2, 2B18.3, 1B22.4, 2B23, 2B24, 2B25, 2B25.1, 2B26, 2B27, 2B28, 2B29, 2B30, 4B17.1, 4B17.10, 4B17.1D, 4B17.1F, 4B17.1G, 3E6.2, 4E17.4, 4E17.6, 4B19.1, B6.C12, B6.D2, B11.A5, B11.F7, B11.H12, B11.E9, 4B1, 4B3, 4B5, 4B6, 4B7, 1B14, 4A1, 4A1.1, 5A20.4, ING1.1C1, ING1.5B1, ING1.2B10, and ING1.3C2) and a VH gene repertoire (heavy chain shuffling). The scFv gene repertoire may be cloned into a phage display vector (e.g., pHEN-1, Hoogenboom et al. (1991) Nucleic Acids Res., 19: 4133-4137) and after transformation a library of transformants is obtained. Phage are prepared and concentrated and selections are performed, in addition to chain shuffling, it is also possible to shuffle individual complementarily determining regions (CDRs).

The antigen concentration may be decreased in each round of selection, reaching a concentration less than the desired Kd by the final rounds of selection. This results in the selection of phage on the basis of affinity (Hawkins et al. (1992) J. Mol. Biol. 226; 889-896).

Chain shuffling may be combined with the stringent selections made possible by yeast display and flow cytometry. This novel approach was found to be particularly powerful for increasing antibody affinity (see example 1).

2) Increasing the Affinity of Anti-BoNT Antibodies by Site Directed Mutagenesis.

The majority of antigen contacting amino acid side chains are located in the complementarity determining regions (CDRs), three in the VH (CDR1, CDR2, and CDR3) and three in the VL (CDR1, CDR2, and CDR3) (see, e.g., Chothia et al. (1987) J. Mol. Biol., 196: 901-917; Chothia et al. (1986) Science, 233: 755-8; Nhan et al. (1991) J. Mol. Biol., 217: 133-151). Without being bound to a theory, it is believed that these residues contribute the majority of binding energetics responsible for antibody affinity for antigen. In other molecules, mutating amino acids that contact ligand has been shown to be an effective means of increasing the affinity of one protein molecule for its binding partner (Lowman et al. (1993) J. Mol. Biol., 234: 564-578; Wells (1990) Biochemistry, 29: 8509-8516). Thus mutation (randomization) of the CDRs and screening against, for example, BoNT/A, BoNT/E, BoNT/B, or the epitopes thereof, can be used to generate anti-BoNT antibodies having improved binding affinity.

Each CDR is randomized in a separate library, using, for example, A12 as a template. To simplify affinity measurement, A12, or other lower affinity anti-BoNT antibodies, are used as a template, rather than a higher affinity scFv. The CDR sequences of the highest affinity mutants from each CDR library are combined to obtain an additive increase in affinity. A similar approach has been used to increase the affinity of human growth hormone (hGH) for the growth hormone receptor over 1500 fold from 3.4×10−10 to 9.0×10−13 M (see, e.g., Lowman et al. (1993) J. Mol. Biol., 234: 564-578).

To increase the affinity of BoNT-neutralizing antibodies, amino acid residues located in one or more CDRs (e.g., 9 amino acid residues located in VL CDR3) are partially randomized by synthesizing a “doped” oligonucleotide in which the wild type nucleotide occurred with a frequency of, e.g. 49%. The oligonucleotide is used to amplify the remainder of the BoNT-neutralizing scFv gene(s) using PCR.

For example, to create a library in which VH CDR3 is randomized, an oligonucleotide is synthesized which anneals to the BoNT-neutralizing antibody VH framework 3 and encodes VH CDR3 and a portion of framework 4. At the four positions to be randomized, the sequence NNS can be used, where N is any of the 4 nucleotides, and S is “C” or “T”. The oligonucleotide is used to amplify the BoNT-neutralizing antibody VH gene using PCR, creating a mutant BoNT-neutralizing antibody VH gene repertoire. PCR is used to splice the VH gene repertoire with the BoNT-neutralizing antibody light chain gene, and the resulting scFv gene repertoire cloned into a phage display vector (e.g., pHEN-1 or pCANTAB5E). Ligated vector DNA is used to transform electrocompetent E. coli to produce a phage antibody library.

To select higher affinity mutant scFv, each round of selection of the phage antibody libraries is conducted on decreasing amounts of one or more BoNT subtypes. Clones from the third and fourth round of selection can screened for binding to the desired antigen(s) (e.g., BoNT/B BoNT/E, etc.) by ELISA on 96 well plates. scFv from, e.g., twenty to forty ELISA positive clones can be expressed, e.g. in 10 ml cultures, the periplasm harvested, and the scFv koff determined by BIAcore. Clones with the slowest koff are sequenced, and each unique scFv subcloned into an appropriate vector (e.g., pUC119 mycHis). The scFv are expressed in culture, and purified. Affinities of purified scFv can be determined by BIAcore.

By way of illustration, FIG. 5 shows a scheme used for affinity maturation of HuC25 (FIG. 5A) and 3D12 (FIG. 5B) scFv using yeast display (see, e.g., Ser. No. 11/342,271, filed on Jan. 26, 2006, Ser. No. 09/144,886, filed on Aug. 31, 1998, Ser. No. 10/632,706, filed on Aug. 1, 2003, U.S. Ser. No. 60/942,173 filed on Jun. 5, 2008, and PCT application Nos: PCT/US2006/003070 and PCT/US03/24371, which are incorporated herein by reference for all purposes).

3) Creation of Anti-BoNT (scFv′)2 Homodimers.

To create anti-BoNT (e.g., BoNT-neutralizing) (scFv′)2 antibodies, two anti-BoNT scFvs are joined, either through a linker (e.g., a carbon linker, a peptide, etc.) or through a disulfide bond between, for example, two cysteines. Thus, for example, to create disulfide linked scFv, a cysteine residue can be introduced by site directed mutagenesis between a myc tag and a hexahistidine tag at the carboxy-terminus of an anti-BoNT/B. Introduction of the correct sequence can be verified by DNA sequencing. The construct may be in pUC119, so that the pelB leader directs expressed scFv to the periplasm and cloning sites (NcoI and NotI) exist to introduce anti-BoNT mutant scFv. Expressed scFv has the myc tag at the C-terminus, followed by two glycines, a cysteine, and then 6 histidines to facilitate purification by IMAC. After disulfide bond formation between the two cysteine residues, the two scFv can be separated from each other by 26 amino acids (two 11 amino acid myc tags and 4 glycines). An scFv expressed from this construct, purified by IMAC may predominantly comprise monomeric scFv. To produce (scFv′)2 dimers, the cysteine can be reduced by incubation with 1 MM beta-mercaptoethanol, and half of the scFv blocked by the addition of DTNB. Blocked and unblocked scFvs can be incubated together to form (scFv′)2 and the resulting material can optionally be analyzed by gel filtration. The affinity of the anti-BoNT scFv′ monomer and (scFv′)2 dimer can optionally be determined by BIAcore.

The (scFv′)2 dimer may be created by joining the scFv fragments through a linker, more preferably through a peptide linker. This can be accomplished by a wide variety of means well known to those of skill in the art. For example, one preferred approach is described by Holliger et al. (1993) Proc. Natl. Acad. Sci. USA, 90: 6444-6448 (see also WO 94/13804).

Typically, linkers are introduced by PCR cloning. For example, synthetic oligonucleotides encoding the 5 amino acid linker (Gly4Ser, SEQ ID NO:1) can be used to PCR amplify the BoNT-neutralizing antibody VH and VL genes which are then spliced together to create the BoNT-neutralizing diabody gene. The gene can then be cloned into an appropriate vector, expressed, and purified according to standard methods well known to those of skill in the art.

4) Preparation of Anti-BoNT (scFv)2, Fab, and (Fab′)2 Molecules.

Anti-BoNT antibodies such as anti-BoNT/E or anti-BoNT/B scFv, or variant(s) with higher affinity, are suitable templates for creating size and valency variants. For example, an anti-BoNT (scFv′)2 can be created from the parent scFv (e.g., 2A10, 3E1, 3E2, 3E3, 3E4, 3E4.1, 3E5, 3E6, 3E6.1, 3E6.2, 4E11, 4E13, 4E16, 4E16.1, 4E17, 4E17.1, n A12, 6A12, B1.1, B6, B6.1, B8, B8.1, B11, B11C3, B11E8, B12, B12.1, B12.2, 1B18, 2B18.1, 4B19, 4B19.1, 1B22, 1B10, 1B10.1, 2B18.2, 2B18.3, 1B22.4, 2B23, 2B24, 2B25, 2B25.1, 2B26, 2B27, 2B28, 2B29, 2B30, 4B17.1, 4B17.1C, 4B17.1D, 4B17.1F, 4B17.1G, 3E6.2, 4E17.4, 4E17.6, 4B19.1, B6.C12, B6.D2, B11.A5, B11.F7, B11.H12, B11.E9, 4B1, 4B3, 4B5, 4B6, 4B7, 1B14, 4A1, 4A1.1, 5A20.4, ING1.1C1, ING1.5B1, ING1.2B10, and ING1.3C2, etc.) as described above. An scFv gene can be excised using appropriate restriction enzymes and cloned into another vector as described herein.

Expressed scFv may include a myc tag at the C-terminus, followed by two glycines, a cysteine, and six histidines to facilitate purification. After disulfide bond formation between the two cystine residues, the two scFv may be separated from each other by 26 amino acids (e.g., two eleven amino acid myc tags and four glycines). Single-chain Fv (scFv) can be expressed from this construct and purified.

To produce (scFv′)2 dimers, the cysteine is reduced by incubation with 1 mM β-mercaptoethanol, and half of the scFv blocked by the addition of DTNB. Blocked and unblocked scFv are incubated together to form (scFv′)2, which is purified. As higher affinity scFv are isolated, their genes are similarly used to construct (scFv′)2.

Anti-BoNT Fab may also be expressed in E. coli using an expression vector similar to the one described by Better et. al. (1988) Science, 240: 1041-1043. For example, to create a BoNT/B or BoNT/E-neutralizing Fab, the VH and VL genes are amplified from the scFv using PCR. The VH gene is cloned into an expression vector (e.g., a pUC119 based bacterial expression vector) that provides an IgG CH1 domain downstream from, and in frame with, the VH gene. The vector also contains the lac promoter, a pelB leader sequence to direct expressed VH-CH1 domain into the periplasm, a gene 3 leader sequence to direct expressed light chain into the periplasm, and cloning sites for the light chain gene. Clones containing the correct VH gene are identified, e.g., by PCR fingerprinting. The VL gene is spliced to the CL gene using PCR and cloned into the vector containing the VH CH1 gene.

B) Selection of Neutralizing Antibodies.

Selection of anti-BoNT antibodies (whether produced by phage display, yeast display, immunization methods, hybridoma technology, etc.) involves screening the resulting antibodies for specific binding to an appropriate antigen(s). In the instant case, suitable antigens can include, but are not limited to BoNT/E1, BoNT/E2, BoNT/E3, BoNT/B1, BoNT/B2, BoNT/B3, BoNT/B4, BoNT/A1, BoNT/A2, BoNT/A3, a C-terminal domain of BoNT heavy chain (binding domain), BoNT holotoxins, recombinant BoNT domains such as HC (binding domain), HN (translocation domain), or LC (light chain), and the like. The neutralizing antibodies may be selected for specific binding of an epitope recognized by one or more of the antibodies described herein.

Selection can be by any of a number of methods well known to those of skill in the art. In one example, selection is by immunochromatography (e.g., using immunotubes, Maxisorp, Nunc) against the desired target, e.g., BoNT/E, BoNT/B, etc. In a related example, selection is against a BoNT protein in a surface plasmon resonance system (e.g., BIAcore, Pharmacia) either alone or in combination with an antibody that binds to an epitope specifically bound by one or more of the antibodies described herein. Selection can also be done using flow cytometry for yeast display libraries. Yeast display libraries are sequentially selected, first on BoNT/B1, then on other BoNT/B subtypes (BoNT/B2, B3 and B4) to obtain antibodies that bind with high affinity to all subtypes of BoNT/B. This can be repeated for other subtypes.

For phage display, analysis of binding can be simplified by including an amber codon between the antibody fragment gene and gene III. This makes it possible to easily switch between displayed and soluble antibody fragments simply by changing the host bacterial strain. When phage are grown in a supE suppresser strain of E. coli, the amber stop codon between the antibody gene and gene III is read as glutamine and the antibody fragment is displayed on the surface of the phage. When eluted phage are used to infect a non-suppressor strain, the amber codon is read as a stop codon and soluble antibody is secreted from the bacteria into the periplasm and culture media (Hoogenboom et al. (1991) Nucleic Acids Res., 9: 4133-4137). Binding of soluble scFv to antigen can be detected, e.g., by ELISA using a murine IgG monoclonal antibody (e.g., 9E10) which recognizes a C-terminal myc peptide tag on the scFv (Evan et al. (1985) Mol. Cell Biol., 5: 3610-3616; Munro et al. (1986) Cell, 46: 291-300), e.g., followed by incubation with polyclonal anti-mouse Fc conjugated to a detectable label (e.g., horseradish peroxidase).

As indicated above, purification of the anti-BoNT antibody can be facilitated by cloning of the scFv gene into an expression vector (e.g., expression vector pUC119mycHIS) that results in the addition of the myc peptide tag followed by a hexahistidine tag at the C-terminal end of the scFv. The vector also preferably encodes the pectate lyase leader sequence that directs expression of the scFv into the bacterial periplasm where the leader sequence is cleaved. This makes it possible to harvest native properly folded scFv directly from the bacterial periplasm. The BoNT-neutralizing antibody is then expressed and purified from the bacterial supernatant using immobilized metal affinity chromatography.

**C) Measurement of Anti-BoNT Antibody Affinity for One or More BoNT Subtypes.**

As explained above, selection for increased avidity involves measuring the affinity of an anti-BoNT (e.g., a BoNT-neutralizing) antibody (or a modified BoNT-neutralizing antibody) for one or more targets of interest (e.g. BoNT/E subtype(s) or domains thereof. For example, the KD of a BoNT/E-neutralizing antibody and the kinetics of binding to BoNT/E are determined in a BIAcore, a biosensor based on surface plasmon resonance. For this technique, antigen is coupled to a derivatized sensor chip capable of detecting changes in mass. When antibody is passed over the sensor chip, antibody binds to the antigen resulting in an increase in mass that is quantifiable. Measurement of the rate of association as a function of antibody concentration can be used to calculate the association rate constant (kon). After the association phase, buffer is passed over the chip and the rate of dissociation of antibody (koff) determined. Kon is typically measured in the range 1.0×102 to 5.0×106 M and koff in the range 1.0×10−1 to 1.0×10−6M. The equilibrium constant Kd is then calculated as koff/kon and thus is typically measured in the range 10−5 to 10−12 M. Affinities measured in this manner correlate well with affinities measured in solution by fluorescence quench titration.

Phage display and selection generally results in the selection of higher affinity mutant scFvs (Marks et al. (1992) Bio/Technology, 10: 779-783; Hawkins et al. (1992) J. Mol. Biol. 226: 889-896; Riechmann et al. (1993) Biochemistry, 32: 8848-8855; Clackson et al. (1991) Nature, 352: 624-628), but probably does not result in the separation of mutants with less than a 6 fold difference in affinity (Riechmann et al. (1993) Biochemistry, 32: 8848-8855). Thus a rapid method is needed to estimate the relative affinities of mutant scFvs isolated after selection. Since increased affinity results primarily from a reduction in the koff, measurement of koff should identify higher affinity scFv. koff can be measured in the BIAcore on unpurified scFv in bacterial periplasm, since expression levels are high enough to give an adequate binding signal and koff is independent of concentration. The value of koff for periplasmic and purified scFv is typically in close agreement.

### V. Humanized, Human Engineered or Human Antibody Production.

The present BoNT (e.g., BoNT/B) binding antibodies and fragments can be humanized or human engineered antibodies. As used herein, a humanized antibody, or antigen binding fragment thereof, is a recombinant polypeptide that comprises a pardon of an antigen binding site from a non-human antibody and a portion of the framework and/or constant regions of a human antibody. A human engineered antibody or antibody fragment may be derived from a human or non-human (e.g., mouse) source that has been engineered by modifying (e.g., deleting, inserting, or substituting) amino acids at specific positions so as to alter certain biophysical properties or to reduce any detectable immunogenicity of the modified antibody in a human.

Humanized antibodies also encompass chimeric antibodies and CDR-grafted antibodies in which various regions may be derived from different species. Chimeric antibodies may be antibodies that include a non-human antibody variable region linked to a human constant region. Thus, in chimeric antibodies, the variable region is mostly non-human, and the constant region is human. Chimeric antibodies and methods for making them are described in Morrison, et al., Proc. Natl. Acad. Sci. USA, 81: 6841-6855 (1984), Boulianne, et al., Nature, 312: 643-646 (1984), and PCT Application Publication WO 86/01533. Although, they can be less immunogenic than a mouse monoclonal antibody, administrations of chimeric antibodies have been associated with human anti-mouse antibody responses (HAMA) to the non-human portion of the antibodies. Chimeric antibodies can also be produced by splicing the genes from a mouse antibody molecule of appropriate antigen-binding specificity together with genes from a human antibody molecule of appropriate biological activity, such as the ability to activate human complement and mediate ADCC. Morrison et al. (1984), Proc. Natl. Acad. Sci., 81: 6851; Neuberger et al. (1984), Nature, 312: 604. One example is the replacement of an Fc region with that of a different isotype.

CDR-grafted antibodies are antibodies that include the CDRs from a non-human “donor” antibody linked to the framework region from a human “recipient” antibody. Generally, CDR-grafted antibodies include more human antibody sequences than chimeric antibodies because they include both constant region sequences and variable region (framework) sequences from human antibodies. Thus, for example, a CDR-grafted humanized antibody may comprise a heavy chain that comprises a contiguous amino acid sequence (e.g., about 5 or more, 10 or more, or even 15 or more contiguous amino acid residues) from the framework region of a human antibody (e.g., FR-1, FR-2, or FR-3 of a human antibody) or, optionally, most or all of the entire framework region of a human antibody. CDR-grafted antibodies and methods for making them are described in, Jones et al., Nature, 321: 522-525 (1986), Riechmann et al., Nature, 332: 323-327 (1988), and Verhoeyen et al., Science, 239: 1534-1536 (1988)). Methods that can be used to produce humanized antibodies also are described in U.S. Pat. Nos. 4,816,567, 5,721,367, 5,837,243, and 6,180,377. CDR-grafted antibodies are considered less likely than chimeric antibodies to induce an immune reaction against non-human antibody portions. However, it has been reported that framework sequences from the donor antibodies are required for the binding affinity and/or specificity of the donor antibody, presumably because these framework sequences affect the folding of the antigen-binding portion of the donor antibody. Therefore, when donor, non-human CDR sequences are grafted onto unaltered human framework sequences, the resulting CDR-grafted antibody can exhibit, in some cases, loss of binding avidity relative to the original non-human donor antibody. See, e.g., Riechmann et al., Nature, 332: 323-327 (1988), and Verhoeyen et al., Science, 239: 1534-1536 (1988).

Human engineered antibodies include for example “veneered” antibodies and antibodies prepared using HE™ technology (U.S. Pat. 5,869,619). HE™ technology is commercially available, and involves altering an non-human antibody or antibody fragment, such as a mouse or chimeric antibody or antibody fragment, by making specific changes to the amino acid sequence of the antibody so as to produce a modified antibody with reduced immunogenicity in a human that nonetheless retains the desirable binding properties of the original non-human antibodies. Techniques for making human engineered proteins are described in Studnicka et al., Protein Engineering, 7: 805-814 (1994), U.S. Pat. Nos. 5,766,886, 5,770,196, 5,821,123, and 5,869,619, and PCT Application Publication WO 93/11794.

“Veneered” antibodies are non-human or humanized (e.g., chimeric or CDR-grafted antibodies) antibodies that have been engineered to replace certain solvent-exposed amino acid residues so as to further reduce their immunogenicity or enhance their function. As surface residues of a chimeric antibody are presumed to be less likely to affect proper antibody folding and more likely to elicit an immune reaction, veneering of a chimeric antibody can include, for instance, identifying solvent-exposed residues in the non-human framework region of a chimeric antibody and replacing at least one of them with the corresponding surface residues from a human framework region. Veneering can be accomplished by any suitable engineering technique, including the use of the above-described HE™ technology.

In a different approach, a recovery of binding avidity can be achieved by “de-humanizing” a CDR-grafted antibody. De-humanizing can include restoring residues from the donor antibody's framework regions to the CDR grafted antibody, thereby restoring proper folding. Similar “de-humanization” can be achieved by (i) including portions of the “donor” framework region in the “recipient” antibody or (ii) grafting portions of the “donor” antibody framework region into the recipient antibody (along with the grafted donor CDRs).

For a further discussion of antibodies, humanized antibodies, human engineered, and methods for their preparation, see Kontermann and Dubel, eds., Antibody Engineering, Springer, New York, N.Y., 2001.

The present antibodies and fragments encompass human antibodies, such as antibodies which bind BoNT polypeptides and are encoded by nucleic acid sequences which are naturally occurring somatic variants of human germline immunoglobulin nucleic acid sequence, and fragments, synthetic variants, derivatives and fusions thereof. Such antibodies may be produced by any method known in the art, such as through the use of transgenic mammals (such as transgenic mice) in which the native immunoglobulin repertoire has been replaced with human V-genes in the mammal chromosome. Such mammals appear to carry out VDJ recombination and somatic hypermutation of the human germline antibody penes in a normal fashion, thus producing high affinity antibodies with completely human sequences.

Human antibodies to target protein can also be produced using transgenic animals that have no endogenous immunoglobulin production and are engineered to contain human immunoglobulin loci. For example, WO 98/24893 discloses transgenic animals having a human Ig locus wherein the animals do not produce functional endogenous immunoglobulins due to the inactivation of endogenous heavy and light chain loci. WO 91/00906 also discloses transgenic non-primate mammalian hosts capable of mounting an immune response to an immunogen, wherein the antibodies have primate constant and/or variable regions, and wherein the endogenous immunoglobulin encoding loci are substituted or inactivated. WO 96/30498 and U.S. Pat. No. 6,091,001 disclose the use of the Cre/Lox system to modify the immunoglobulin locus in a mammal, such as to replace all or a portion of the constant or variable region to form a modified antibody molecule. WO 94/02602 discloses non-human mammalian hosts having inactivated endogenous Ig loci and functional human Ig loci. U.S. Pat. No. 5,939,598 discloses methods of making transgenic mice in which the mice lack endogenous heavy chains, and express an exogenous immunoglobulin locus comprising one or more xenogeneic constant regions. See also, U.S. Pat. Nos. 6,114,598, 6,657,103 and 6,833,268.

Using a transgenic animal described above, an immune response can be produced to a selected antigenic molecule, and antibody producing cells can be removed from the animal and used to produce hybridomas that secrete human monoclonal antibodies. Immunization protocols, adjuvants, and the like are known in the art, and are used in immunization of, for example, a transgenic mouse as described in WO 96/33735. The monoclonal antibodies can be tested for the ability to inhibit or neutralize the biological activity or physiological effect of the corresponding protein. Human monoclonal antibodies with specificity for the antigen used to immunize transgenic animals are also disclosed in WO 96/34096 and U.S. patent application no. 20030194404; and U.S. patent application no. 20030031667.

Additional transgenic animals useful to make monoclonal antibodies include the Medarex HuMAb-MOUSE®, described in U.S. Pat. No. 5,770,429 and Fishwild, et al. (Nat. Biotechnol. 14:845-851, 1996), which contains gene sequences from unrearranged human antibody genes that code for the heavy and light chains of human antibodies. Immunization of a HuMAb-MOUSE® enables the production of fully human monoclonal antibodies to the target protein.

Also, Ishida et al. (Cloning Stem Cells. 4:91-102, 2002) describes the TransChromo Mouse (TCMOUSE™) which comprises megabase-sized segments of human DNA and which incorporates the entire human immunoglobulin (hIg) loci. The TCMOUSE™ has a fully diverse repertoire of hIgs, including all the subclasses of IgGs (IgG1-G4 ). Immunization of the TC MOUSE™ with various human antigens produces antibody responses comprising human antibodies. See also Jakobovits et al., Proc. Natl. Acad. Sci. USA, 90:2551 (1993); Jakobovits et al., Nature, 362:255-258 (1993); Bruggermann et al., Year in Immunol., 7:33 (1993); and U.S. Pat. Nos. 5,591,669; 5,589,369; 5,545,807; and U.S Patent Publication Nos. 20020199213 and 20030092125, which describe methods for biasing the immune response of an animal to the desired epitope. Human antibodies may also be generated by in vitro activated B cells (see U.S. Pat. Nos. 5,567,610 and 5,229,275).

Human antibodies can also be generated through the in vitro screening of antibody display libraries. See Hoogenboom et al. (1991), J. Mol. Biol. 227: 381; and Marks et al. (1991), J. Mol. Biol. 222: 581. Various antibody-containing phage display libraries have been described and may be readily prepared. Libraries may contain a diversity of human antibody sequences, such as human Fab, Fv, and scFv fragments that may be screened against an appropriate target. Phage display libraries may comprise peptides or proteins other than antibodies which may be screened to identify selective binding agents of BoNT.

The development of technologies for making repertoires of recombinant human antibody genes, and the display of the encoded antibody fragments on the surface of filamentous bacteriophage, has provided a means for making human antibodies directly. The antibodies produced by phage technology are produced as antigen binding fragments—usually Fv or Fab fragments—in bacteria and thus lack effector functions. Effector functions can be introduced by one of two strategies: The fragments can be engineered either into complete antibodies for expression in mammalian cells, or into bispecific antibody fragments with a second binding site capable of triggering an effector function.

Methods for display of peptides on the surface of yeast and microbial cells have also been used to identify antigen specific antibodies. See, for example, U.S. Pat. No. 6,699,658. Antibody libraries may be attached to yeast proteins, such as agglutinin, effectively mimicking the cell surface display of antibodies by B cells in the immune system.

In addition to phage display methods, antibodies may be isolated using ribosome mRNA display methods and microbial cell display methods. Selection of polypeptide using ribosome display is described in Hanes et al., (Proc. Nat.l Acad. Sc.i USA, 94:4937-4942, 1997) and U.S. Pat. Nos. 5,643,768 and 5,658,754 issued to Kawasaki. Ribosome display is also useful for rapid large scale mutational analysis of antibodies. The selective mutagenesis approach also provides a method of producing antibodies with improved activities that can be selected using ribosomal display techniques.

Human BoNT-neutralizing antibodies of the present disclosure are may be produced in trioma cells. Genes encoding the antibodies are then cloned and expressed in other cells, particularly, nonhuman mammalian cells.

The general approach for producing human antibodies by trioma technology has been described by Ostberg et al. (1983) Hybridoma 2: 361-367, Ostberg, U.S. Pat. No. 4,634,664, and Engelman et al., U.S. Pat. No. 4,634,666. The antibody-producing cell lines obtained by this method are called triomas because they are descended from three cells; two human and one mouse. Thomas have been found to produce antibody more stably than ordinary hybridomas made from human cells.

Other approaches to antibody production include in vitro immunization of human blood. In this approach, human blood lymphocytes capable of producing human antibodies are produced. Human peripheral blood is collected from the patient and is treated to recover mononuclear cells. The suppressor T-cells then are removed and remaining cells are suspended in a tissue culture medium to which is added the antigen and autologous serum and, preferably, a nonspecific lymphocyte activator. The cells then are incubated for a period of time so that they produce the specific antibody desired. The cells then can be fused to human myeloma cells to immortalize the cell line, thereby to permit continuous production of antibody (see U.S. Pat. No. 4,716,111).

In another approach, mouse-human hybridomas which produce human BoNT-neutralizing antibodies are prepared (see, e.g., U.S. Pat. No. 5,506,132). Other approaches include immunization of murines transformed to express human immunoglobulin genes, and phage display screening (Vaughan et al. supra.).

### VI. Other Antibody Forms.

Sequence provided herein can be used to generate other antibody forms, including but not limited to nanobodies, UniBodies, and/or affibodies.

VHH and/or Nanobodies.

The Camelidae heavy chain antibodies are found as homodimers of a single heavy chain, dimerized via their constant regions. The variable domains of these camelidae heavy chain antibodies are referred to as VHH domains or VHH, and can be either used per se as nanobodies and/or as a starting point for obtaining nanobodies. Isolated VHH retain the ability to bind antigen with high specificity (see, e.g., Hamers-Casterman et al. (1993) Nature 363: 446 448). VHH domains, or nucleotide sequences encoding them, can be derived from antibodies raised in Camelidae species, for example in camel, dromedary, llama, alpaca and guanaco. Other species besides Camelidae (e.g, shark, pufferfish) can produce functional antigen-binding heavy chain antibodies, from which (nucleotide sequences encoding) such naturally occurring VHH can be obtained, e.g. using the methods described in U.S. Patent Publication US 2006/0211088.

Human proteins may be used in therapy primarily because they are not as likely to provoke an immune response when administered to a patient. Comparisons of camelid VHH with the VH domains of human antibodies reveals several key differences in the framework regions of the camelid VHH domain corresponding to the VH/VL interface of the human VH domains. Mutation of these human residues to VHH resembling residues has been performed to produce “camelized” human VH domains that retain antigen binding activity, yet have improved expression and solubility.

Libraries of single VH domains have also been derived for example from VH genes amplified from genomic DNA or from mRNA from the spleens of immunized mice and expressed in E. coli (Ward et al. (1989) Nature 341: 544-546) and similar approaches can be performed using the VH domains and/or the VL domains described herein. The isolated single VH domains are called “dAbs” or domain antibodies. A “dAb” is an antibody single variable domain (VH or VL) polypeptide that specifically binds antigen. A “dAbV binds antigen independently of other V domains; however, as the term is used herein, a “dAbU can be present in a homo- or heteromultimer with other VH or VL domains where the other domains are not required for antigen binding by the dAb, i.e., where the dAb binds antigen independently of the additional VH or VL domains.

As described in U.S. Patent Publication No. 2006/0111088 methods are known for the cloning and direct screening of immunoglobulin sequences (including but not limited to multivalent polypeptides comprising: two or more variable domains—or antigen binding domains—and in particular VH domains or VHH domains; fragments of VL, VH or VHH domains, such as CDR regions, for example CDR3 regions; antigen-binding fragments of conventional 4-chain antibodies such as Fab fragments and scFv's, heavy chain antibodies and domain antibodies; and in particular of VH sequences, and more in particular of VHH sequences) that can be used as part of and/or to construct such nanobodies.

Methods and procedures for the production of VHH/nanobodies can also be found for example in WO 94/04678, WO 96/34103, WO 97149805, WO 97/49805 WO 94/25591, WO 00/43507 WO 01/90190, WO 03/025020, WO 04/062551, WO 04/041863, WO 04/041865, WO 04/041862, WO 04/041867, PCT/BE2004/000159, Hamers-Casterman et al. (1993) Nature 363: 446; Riechmann and Muyldermans (1999) J. Immunological Meth., 231: 25-38; Vu et al. (1997) Molecular Immunology, 34(16-17): 1121-1131; Nguyen et al. (2000) EMBO J., 19(5): 921-930; Arbabi Ghahroudi et al. (19997) FEBS Letters 414: 521-526; van der Linden et al. (2000) J. Immunological Meth., 240: 185-195; Muyldermans (2001) Rev. Molecular Biotechnology 74: 277-302; Nguyen et al. (2001) Adv. Immunol. 79:261, and the like, which are all incorporated herein by reference.

UniBodies.

UniBodies are generated by an antibody technology that produces a stable, smaller antibody format with an anticipated longer therapeutic window than certain small antibody formats. UniBodies may be produced from IgG4 antibodies by eliminating the hinge region of the antibody. Unlike the full size IgG4 antibody, the half molecule fragment is very stable and is termed a UniBody. Halving the IgG4 molecule left only one area on the UniBody that can bind to a target. Methods of producing UniBodies are described in detail in PCT Publication WO2007/059782, which is incorporated herein by reference in its entirety (see, also, Kolfschoten et al. (2007) Science 317: 1554-1557).

Affibodies.

Affibody molecules are class of affinity proteins based on a 58-amino acid residue protein domain, derived from one of the IgG-binding domains of staphylococcal protein A. This three helix bundle domain has been used as a scaffold for the construction of combinatorial phagemid libraries, from which affibody variants that target the desired molecules can be selected using phage display technology (see, e.g., Nord et al. (1997) Nat. Biotechnol. 15: 772-777; Ronmark et al. (2002) Eur. J. Biochem., 269: 2647-2655.). Details of affibodies and methods of production are known to those of skill (see, e.g., U.S. Pat. No. 5,831,012 which is incorporated herein by reference in its entirety).

**VI. Assaying for Cross-Reactivity at a Neutralizing Epitope.**

The antibodies of the present disclosure encompass those that specifically bind to one or more epitopes recognized by antibodies described herein (e.g., 2A10, 3E1, 3E2, 3E3, 3E4, 3E4.1, 3E5, 3E6, 3E6.1, 3E6.2, 4E11, 4E13, 4E16, 4E16.1, 4E17, 4E17.1, n A12, 6A12, B1.1, B6, B6.1, B8, B8.1, B11, B11C3, B11E8, B12, B12.1, B12.2, 1B18, 2B18.1, 4B19, 4B19.1, 1B22, 1B10, 1B10.1, 2B18.2, 2B18.3, 1B22.4, 2B23, 2B24, 2B25, 2B25.1, 2B26, 2B27, 2B28, 2B29, 2B30, 4B17.1, 4B17.1C, 4B17.1D, 4B17.1F, 4B17.1G, 3E6.2, 4E17.4, 4E17.6, 4B19.1, B6.C12, B6.D2, B11.A5, B11.F7, B11.H12, B11.E9, 4B1, 4B3, 4B5, 4B6, 4B7, 1B14, 4A1, 4A1.1, 5A20.4, ING1.1C1, ING1.5B1, ING1.2B10, and ING1.3C2, etc.). In other words, antibodies are cross-reactive with one of more of these antibodies. Means of assaying for cross-reactivity are well known to those of skill in the art (see, e.g., Dowbenko et al. (1988) J. Virol. 62: 4703-4711).

This can be ascertained by providing one or more isolated target BoNT polypeptide(s) (e.g. BoNT/B1 and/or BoNT/B2, or recombinant domains of said toxin, such as HC) attached to a solid support and assaying the ability of a test antibody to compete with, an antibody described herein for binding to the target BoNT peptide. Thus, immunoassays in a competitive binding format are preferably used for cross-reactivity determinations. For example, a BoNT/E and/or BoNT/B polypeptide may be immobilized to a solid support. Antibodies to be tested (e.g. generated by selection from a phage-display library) added to the assay compete with 2A10, 3E1, 3E2, 3E3, 3E4, 3E4.1, 3E5, 3E6, 3E6.1, 3E6.2, 4E11, 4E13, 4E16, 4E16.1, 4E17, 4E17.1, n A12, 6A12, B1.1, B6, B6.1, B8, B8.1, B11, B11C3, B11E8, B12, B12.1, B12.2, 1B18, 2B18.1, 4B19, 4B19.1, 1B22, 1B10, 1B10.1, 2B18.2, 2B18.3, 1B22.4, 2B23, 2B24, 2B25, 2B25.1, 2B26, 2B27, 2B28, 2B29, 2B30, 4B17.1, 4B17.1C, 4B17.1D, 4B17.1F, 4B17.1G, 3E6.2, 4E17.4, 4E17.6, 4B19.1, B6.C12, B6.D2, B11.A5, B11.F7, B11.H12, B11.E9, 4B1, 4B3, 4B5, 4B6, 4B7, 1B14, 4A1, 4A1.1, 5A20.4, ING1.10, ING1.5B1, ING1.2b10, and ING1.3C2, etc antibodies binding to the immobilized BoNT polypeptide(s). The ability of test antibodies to compete with the binding of the 2A10, 3E1, 3E2, 3E3, 3E4, 3E4.1, 3E5, 3E6, 3E6.1, 3E6.2, 4E11, 4E13, 4E16, 4E16.1, 4E17, 4E17.1, A12, 6A12, B1.1, B6, B6.1, B8, B8.1, B11, B11C3, B11E8, B12, B12.1, B12.2, 1B18, 2B18.1, 4B19, 4B19.1, 1B22, 1B10, 1B10.1, 2B18.2, 2B18.3, 1B22.4, 2B23, 2B24, 2B25, 2B25.1, 2B26, 2B27, 2B28, 2B29, 2B30, 4B17.1, 4B17.1C, 4B17.1D, 4B17.1F, 4B17.1G, 3E6.2, 4E17.4, 4E17.6, 4B19.1, B6.C12, B6.D2, B11.A5, B11.F7, B11.H12, B11.E9, 4B1, 4B3, 4B5, 4B6, 4B7, 1B14, 4A1, 4A1.1, 5A20.4, ING1.1C1, ING1.5b1, ING1.2B10, and ING1.3C2 etc antibodies to the immobilized protein(s) are compared. The percent cross-reactivity above proteins is then calculated, using standard calculations.

If the test antibody competes with one or more of the 2A10, 3E1, 3E2, 3E3, 3E4, 3E4.1, 3E5, 3E6, 3E6.1, 3E6.2, 4E11, 4E13, 4E16, 4E16.1, 4E17, 4E17.1, n A12, 6A12, B1.1, B6, B6.1, B8, B8.1, B11, B11C3, B11E8, B12, B12.1, B12.2, 1B18, 2B18.1, 4B19, 4B19.1, 1B22, 1B10, 1B10.1, 2B18.2, 2B18.3, 1B22.4, 2B23, 2B24, 2B25, 2B25.1, 2B26, 2B27, 2B28, 2B29, 2B30, 4B17.1, 4B17.1C, 4B17.1D, 4B17.1F, 4B17.1G, 3E6.2, 4E17.4, 4E17.6, 4B19.1, B6.C12, B6.D2, B11.A5, B11.F7, B11.H12, B11.E9, 4B1, 4B3, 4B5, 4B6, 4B7, 1B14, 4A1, 4A1.1, 5A20.4, ING1.1C1, ING1.5B1, ING1.2B10, and ING1.3C2, etc antibodies and has a binding affinity comparable to or greater than about 1×10−8 M with the same target then the test antibody is expected to be a BoNT-neutralizing antibody.

Cross-reactivity may performed by using surface plasmon resonance in a BIAcore. In a BIAcore flow cell, the BoNT polypeptide(s) (e.g., BoNT/B and/or BoNT/E) are coupled to a sensor chip (e.g. CM5) as described in copending application No. 60/942,173, disclosure of which is incorporated herein by reference. With a flow rate of 5 μl/min, a titration of 100 nM to 1 μM antibody is injected over the flow cell surface for about 5 minutes to determine an antibody concentration that results in near saturation of the surface. Epitope mapping or cross-reactivity is then evaluated using pairs of antibodies at concentrations resulting in near saturation and at least 100 RU of antibody bound. The amount of antibody bound is determined for each member of a pair, and then the two antibodies are mixed together to give a final concentration equal to the concentration used for measurements of the individual antibodies. Antibodies recognizing different epitopes show an essentially additive increase in the RU bound when injected together, while antibodies recognizing identical epitopes show only a minimal increase in RU. Antibodies may be said to be cross-reactive if, when “injected” together they show an essentially additive increase (preferably an increase by at least a factor of about 1.4, more preferably an increase by at least a factor of about 1.6, and most preferably an increase by at least a factor of about 1.8 or 2.

Cross-reactivity may also be determined by incubating a yeast displayed scFv with a BoNT domain polypeptide followed by incubation with an epitope-tagged scFv. Bound scFv is detected with an antibody recognizing the epitope tag and the level of BoNT domain display quantitated by incubation with anti-SV5 (see example 1).

Cross-reactivity at the desired epitopes can ascertained by a number of other standard techniques (see, e.g., Geysen et al (1987) J. Immunol. Meth. 102, 259-274). This technique involves the synthesis of large numbers of overlapping BoNT peptides. The synthesized peptides are then screened against one or more of the prototypical antibodies (e.g., 1B10, 3E6.2, etc.) and the characteristic epitopes specifically bound by these antibodies can be identified by binding specificity and affinity. The epitopes thus identified can be conveniently used for competitive assays as described herein to identify cross-reacting antibodies.

The peptides for epitope mapping can be conveniently prepared using “Multipin” peptide synthesis techniques (see, e.g., Geysen et al (1987) Science, 235: 1184-1190). Using the known sequence of one or more BoNT subtypes (see, e.g., Atassi et al. (1996) J. Prot. Chem., 7: 691-700 and references cited therein), overlapping BoNT polypeptide sequences can be synthesized individually in a sequential manner on plastic pins in an array of one or more 96-well microtest plate(s).

The procedure for epitope mapping using this multipin peptide system is described in U.S. Pat. No. 5,739,306. Briefly, the pins are first treated with a pre-coat buffer containing 2% bovine serum albumin and 0.1% Tween 20 in PBS for 1 hour at room temperature. Then the pins are then inserted into the individual wells of 96-well microtest plate containing the antibodies in the pre-coat buffer, e.g. at 2 μg/ml. The incubation is preferably for about 1 hour at room temperature. The pins are washed in PBST (e.g., 3 rinses for every 10 minutes), and then incubated in the wells of a 96-well microtest plate containing 100 mu 1 of HRP-conjugated goat anti-mouse IgG (Fc) (Jackson ImmunoResearch Laboratories) at a 1:4,000 dilution for 1 hour at room temperature. After the pins are washed as before, the pins are put into wells containing peroxidase substrate solution of diammonium 2,2′-azino-bis [3-ethylbenzthiazoline-b-sulfonate] (ABTS) and H2O2 (Kirkegaard & Perry Laboratories Inc., Gaithersburg, Md.) for 30 minutes at room temperature for color reaction. The plate is read at 405 nm by a plate reader (e.g., BioTek ELISA plate reader) against a background absorption wavelength of 492 nm. Wells showing color development indicate reactivity of the BoNT peptides in such wells with the test antibodies.

### VII. Assaying for Neutralizing Activity of Anti-BoNT Antibodies

Preferred antibodies of the present disclosure act, individually or in combination, to neutralize (reduce or eliminate) the toxicity of botulinum neurotoxin type. Neutralization can be evaluated in vivo or in vitro. In vivo neutralization measurements simply involve measuring changes in the lethality (e.g., LD50 or other standard metric) due to a BoNT neurotoxin administration due to the presence of one or more antibodies being tested for neutralizing activity. The neurotoxin can be directly administered to the test organism (e.g. mouse) or the organism can harbor a botulism infection (e.g., be infected with Clostridium botulinum). The antibody can be administered before, during, or after the injection of BoNT neurotoxin or infection of the test animal. A decrease in the rate of progression, or mortality rate indicates that the antibody(s) have neutralizing activity.

One suitable in vitro assay for neutralizing activity uses a hemidiaphragm preparation (Deshpande et al. (1995) Toxicon, 33: 551-557). Briefly, left and right phrenic nerve hemidiaphragm preparations are suspended in physiological solution and maintained at a constant temperature (e.g. 36° C.). The phrenic nerves are stimulated supramaximally (e.g. at 0.05 Hz with square waves of 0.2 ms duration). Isometric twitch tension is measured with a force displacement transducer (e.g., GrassModel FT03) connected to a chart recorder.

Purified antibodies are incubated with purified BoNT (e.g. BoNT/A1, BoNT/A2, BoNT/B1, etc.) for 30 min at room temperature and then added to the tissue bath, resulting in a final antibody concentration of about 2.0×10−8 M and a final BoNT concentration of about 2.0×10−11 M. For each antibody studied, time to 50% twitch tension reduction is determined (e.g., three times for BoNT alone and three times for antibody plus BoNT). Differences between times to a given (arbitrary) percentage (e.g. 50%) twitch reduction are determined by standard statistical analyses (e.g. two-tailed t test) at standard levels of significance (e.g., a P value of <0.05 considered significant).

### VIII. Diagnostic Assays.

As explained above, the anti-BoNT antibodies of the present disclosure can be used for the in vivo or in vitro detection of BoNT toxin (e.g. BoNT/E toxin) and thus, are useful in the diagnosis (e.g. confirmatory diagnosis) of botulism. The detection and/or quantification of BoNT in a biological sample obtained from an organism is indicative of a Clostridium botulinum infection of that organism.

The BoNT antigen can be quantified in a biological sample derived from a patient such as a cell, or a tissue sample derived from a patient. As used herein, a biological sample is a sample of biological tissue or fluid that contains a BoNT concentration that may be correlated with and indicative of a Clostridium botulinum infection. Preferred biological samples include blood, urine, saliva, and tissue biopsies.

Although the sample is typically taken from a human patient, the assays can be used to detect BoNT antigen in samples from mammals in general, such as dogs, cats, sheep, cattle and pigs, and most particularly primates such as humans, chimpanzees, gorillas, macaques, and baboons, and rodents such as mice, rats, and guinea pigs.

Tissue or fluid samples are isolated from a patient according to standard methods well known to those of skill in the art, most typically by biopsy or venipuncture. The sample is optionally pretreated as necessary by dilution in an appropriate buffer solution or concentrated, if desired. Any of a number of standard aqueous buffer solutions, employing one of a variety of buffers, such as phosphate, Tris, or the like, at physiological pH can be used.

A) Immunological Binding Assays

The BoNT polypeptide (e.g., BoNT/E, BoNT/B, etc.) can be detected in an immunoassay utilizing one or more of the anti-BoNT antibodies of the present disclosure as a capture agent that specifically binds to the BoNT polypeptide.

As used herein, an immunoassay is an assay that utilizes an antibody (e.g. a anti-BoNT/E antibody) to specifically bind an analyte (e.g., BoNT/E). The immunoassay is characterized by the binding of one or more anti-BoNT antibodies to a target (e.g. one or more BoNT/E subtypes) as opposed to other physical or chemical properties to isolate, target, and quantify the BoNT analyte.

The BoNT marker can be detected and quantified using any of a number of well recognized immunological binding assays (see, e.g., U.S. Pat. Nos. 4,366,241; 4,376,110; 4,517,288; and 4,837,168, and the like). For a review of the general immunoassays, see also Methods in Cell Biology Volume 37: Antibodies in Cell Biology, Asai, ed. Academic Press, Inc. New York (1993); Basic and Clinical Immunology 7th Edition, Stites & Terr, eds. (1991)).

The immunoassays of the present disclosure can be performed in any of a number of configurations (see, e.g., those reviewed in Maggio (ed.) (1980) Enzyme Immunoassay CRC Press, Boca Raton, Fla.; Tijan (1985) “Practice and Theory of Enzyme Immunoassays,” Laboratory Techniques in Biochemistry and Molecular Biology, Elsevier Science Publishers B.V., Amsterdam; Harlow and Lane, supra; Chan (ed.) (1987) Immunoassay: A Practical Guide Academic Press, Orlando, Fla.; Price and Newman (eds.) (1991) Principles and Practice of Immunoassays Stockton Press, NY; and Ngo (ed.) (1988) Non isotopic Immunoassays Plenum Press, NY).

Immunoassays often utilize a labeling agent to specifically bind to and label the binding complex formed by the capture agent and the analyte (e.g., an anti-BoNT/E antibody/BoNT/E complex). The labeling agent can itself be one of the moieties comprising the antibody/analyte complex. Thus, for example, the labeling agent can be a labeled BoNT/E polypeptide or a labeled anti-BoNT/E antibody. Alternatively, the labeling agent is optionally a third moiety, such as another antibody, that specifically binds to the BoNT antibody, the BoNT peptide(s), the antibody/polypeptide complex, or to a modified capture group (e.g., biotin) which is covalently linked to BoNT polypeptide or to the anti-BoNT antibody.

The labeling agent encompasses an antibody that specifically binds to the anti-BoNT antibody. Such agents are well known to those of skill in the art, and most typically comprise labeled antibodies that specifically bind antibodies of the particular animal species from which the anti-BoNT antibody is derived (e.g., an anti-species antibody). Thus, for example, where the capture agent is a human derived BoNT/E antibody, the label agent may be a mouse anti-human IgG, i.e., an antibody specific to the constant region of the human antibody.

Other proteins capable of specifically binding immunoglobulin constant regions, such as streptococcal protein A or protein G are also used as the labeling agent. These proteins are normal constituents of the cell walls of streptococcal bacteria. They exhibit a strong non immunogenic reactivity with immunoglobulin constant regions from a variety of species (see generally Kronval, et al., (1973) J. Immunol., 111:1401-1406, and Akerstrom, et al., (1985) J. Immunol., 135:2589-2542, and the like).

Throughout the assays, incubation and/or washing steps may be required after each combination of reagents. Incubation steps can vary from about 5 seconds to several hours, preferably from about 5 minutes to about 24 hours. However, the incubation time will depend upon the assay format, analyte, volume of solution, concentrations, and the like. Usually, the assays are carried out at ambient temperature, although they can be conducted over a range of temperatures, such as 5° C. to 45° C.

1) Non Competitive Assay Formats.

Immunoassays for detecting BoNT neurotoxins (e.g. BoNT serotypes and/or subtypes) may be either competitive or noncompetitive. Noncompetitive immunoassays are assays in which the amount of captured analyte (in this case, BoNT polypeptide) is directly measured. In one preferred “sandwich” assay, for example, the capture agent (e.g., an anti-BoNT antibody) is bound, directly or indirectly to a solid substrate where it is immobilized. These immobilized anti-BoNT antibodies capture BoNT polypeptide(s) present in a test sample (e.g., a blood sample). The BoNT polypeptide(s) thus immobilized are then bound by a labeling agent, e.g., an anti-BoNT/E antibody bearing a label. Alternatively, the second antibody may lack a label, but it may, in turn, be bound by a labeled third antibody specific to antibodies of the species from which the second antibody is derived. Free labeled antibody is washed away and the remaining bound labeled antibody is detected (e.g., using a gamma detector where the label is radioactive).

2) Competitive Assay Formats.

In competitive assays, the amount of analyte (e.g., BoNT/E) present in the sample is measured indirectly by measuring the amount of an added (exogenous) analyte displaced (or competed away) from a capture agent (e.g., anti-BoNT/E antibody) by the analyte present in the sample. For example, in one competitive assay, a known amount of BoNT/E is added to a test sample with an unquantified amount of BoNT/E, and the sample is contacted with a capture agent, e.g., an anti-BoNT/E antibody that specifically binds BoNT/E. The amount of added BoNT/E that binds to the anti-BoNT/E-neutralizing antibody is inversely proportional to the concentration of BoNT/E present in the test sample.

The anti-BoNT/E antibody can be immobilized on a solid substrate. The amount of BoNT/E bound to the anti-BoNT/E antibody is determined either by measuring the amount of BoNT/E present in a BoNT/E-anti-BoNT/E antibody complex, or alternatively by measuring the amount of remaining uncomplexed BoNT/E.

B) Reduction of Non Specific Binding.

One of skill will appreciate that it is often desirable to reduce non specific binding in immunoassays and during analyte purification. Where the assay involves, for example BoNT/E polypeptide(s), BoNT/E-neutralizing antibody, or other capture agent(s) immobilized on a solid substrate, it is desirable to minimize the amount of non specific binding to the substrate. Means of reducing such non specific binding are well known to those of skill in the art. Typically, this involves coating the substrate with a proteinaceous composition. In particular, protein compositions such as bovine serum albumin (BSA), nonfat powdered milk, and gelatin are widely used.

C) Substrates.

As mentioned above, depending upon the assay, various components, including the BoNT polypeptide(s), anti-BoNT antibodies, etc., are optionally bound to a solid surface. Many methods for immobilizing biomolecules to a variety of solid surfaces are known in the art. For instance, the solid surface may be a membrane (e.g., nitrocellulose), a microtiter dish (e.g., PVC, polypropylene, or polystyrene), a test tube (glass or plastic), a dipstick (e.g., glass, PVC, polypropylene, polystyrene, latex, and the like), a microcentrifuge tube, or a glass, silica, plastic, metallic or polymer bead. The desired component may be covalently bound, or noncovalently attached through nonspecific bonding.

A wide variety of organic and inorganic polymers, both natural and synthetic may be employed as the material for the solid surface. Illustrative polymers include polyethylene, polypropylene, poly(4-methylbutene), polystyrene, polymethacrylate, poly(ethylene terephthalate), rayon, nylon, poly(vinyl butyrate), polyvinylidene difluoride (PVDF), silicones, polyformaldehyde, cellulose, cellulose acetate, nitrocellulose, and the like. Other materials which may be employed include paper, glasses, ceramics, metals, metalloids, semiconductive materials, cements or the like. In addition, substances that form gels, such as proteins (e.g., gelatins), lipopolysaccharides, silicates, agarose and polyacrylamides can be used. Polymers which form several aqueous phases, such as dextrans, polyalkylene glycols or surfactants, such as phospholipids, long chain (12-24 carbon atoms) alkyl ammonium salts and the like are also suitable. Where the solid surface is porous, various pore sizes may be employed depending upon the nature of the system.

In preparing the surface, a plurality of different materials may be employed, e.g., as laminates, to obtain various properties. For example, protein coatings, such as gelatin can be used to avoid non specific binding, simplify covalent conjugation, enhance signal detection or the like.

If covalent bonding between a compound and the surface is desired, the surface will usually be polyfunctional or be capable of being polyfunctionalized. Functional groups which may be present on the surface and used for linking can include carboxylic acids, aldehydes, amino groups, cyano groups, ethylenic groups, hydroxyl groups, mercapto groups and the like. The manner of linking a wide variety of compounds to various surfaces is well known and is amply illustrated in the literature. See, for example, Immobilized Enzymes, Ichiro Chibata, Halsted Press, New York, 1978, and Cuatrecasas, (1970) J. Biol. Chem. 245 3059.

In addition to covalent bonding, various methods for noncovalently binding an assay component can be used. Noncovalent binding is typically nonspecific absorption of a compound to the surface. Typically, the surface is blocked with a second compound to prevent nonspecific binding of labeled assay components. Alternatively, the surface is designed such that it nonspecifically hinds one component but does not significantly bind another. For example, a surface bearing a lectin such as concanavalin A will bind a carbohydrate containing compound but not a labeled protein that lacks glycosylation. Various solid surfaces for use in noncovalent attachment of assay components are reviewed in U.S. Pat. Nos. 4,447,576 and 4,254,082.

D) Other Assay Formats

BoNT polypeptides or anti-BoNT antibodies (e.g. BoNT/E neutralizing antibodies) can also be detected and quantified by any of a number of other means well known to those of skill in the art. These include analytic biochemical methods such as spectrophotometry, radiography, electrophoresis, capillary electrophoresis, high performance liquid chromatography (HPLC), thin layer chromatography (TLC), hyperdiffusion chromatography, and the like, and various immunological methods such as fluid or gel precipitin reactions, immunodiffusion (single or double), immunoelectrophoresis, radioimmunoassays (RIAs), enzyme-linked immunosorbent assays (ELISAs), immunofluorescent assays, and the like.

Western blot analysis and related methods can also be used to detect and quantify the presence of BoNT polypeptides in a sample. The technique generally composes separating sample products by gel electrophoresis on the basis of molecular weight, transferring the separated products to a suitable solid support, (such as a nitrocellulose filter, a nylon filter, or derivatized nylon filter), and incubating the sample with the antibodies that specifically bind either the BoNT polypeptide. The antibodies specifically bind to the biological agent of interest on the solid support. These antibodies are directly labeled or alternatively are subsequently detected using labeled antibodies (e.g., labeled sheep anti-human antibodies where the antibody to a marker gene is a human antibody) which specifically bind to the antibody which binds the BoNT polypeptide.

Other assay formats include liposome immunoassays (LIAs), which use liposomes designed to bind specific molecules (e.g., antibodies) and release encapsulated reagents or markers. The released chemicals are then detected according to standard techniques (see, Monroe et al., (1986) Amer. Clin. Prod. Rev. 5:34-41).

E) Labeling of Anti-BoNT (e.g., Anti-BoNT/E) Antibodies.

Anti-BoNT antibodies can be labeled by any of a number of methods known to those of skill in the art. Thus, for example, the labeling agent can be, e.g., a monoclonal antibody, a polyclonal antibody, a protein or complex such as those described herein, or a polymer such as an affinity matrix, carbohydrate or lipid. Detection proceeds by any known method, including immunoblotting, western analysis, gel-mobility shift assays, tracking of radioactive or bioluminescent markers, nuclear magnetic resonance, electron paramagnetic resonance, stopped-flow spectroscopy, column chromatography, capillary electrophoresis, or other methods which track a molecule based upon an alteration in size and/or charge. The detectable group can be any material having a detectable physical or chemical property. Such detectable labels have been well-developed in the field of immunoassays and, in general, any label useful in such methods can be applied in the various embodiments of the present disclosure. Thus, a label is any composition detectable by spectroscopic, photochemical, biochemical, immunochemical, electrical, optical or chemical means. Useful labels in the present disclosure include magnetic beads (e.g. Dynabeads™), fluorescent dyes (e.g., fluorescein isothiocyanate, Texas red, rhodamine, Alexa fluor dyes and the like), radiolabels (e.g., 3H, 125I, 35S, 14C, or 32P), enzymes (e.g., LacZ, CAT, horse radish peroxidase, alkaline phosphatase and others, commonly used as detectable enzymes, either as marker gene products or in an ELISA), and colorimetric labels such as colloidal gold or colored glass or plastic (e.g. polystyrene, polypropylene, latex, etc.) beads.

The label may be coupled directly or indirectly to the desired component of the assay according to methods well known in the art. As indicated above, a wide variety of labels may be used, with the choice of label depending on the sensitivity required, ease of conjugation of the compound, stability requirements, available instrumentation, and disposal provisions.

Non radioactive labels are often attached by indirect means. Generally, a ligand molecule (e.g., biotin) is covalently bound to the molecule. The ligand then binds to an anti-ligand (e.g., streptavidin) molecule which is either inherently detectable or covalently bound to a signal system, such as a detectable enzyme, a fluorescent compound, or a chemiluminescent compound. A number of ligands and anti-ligands can be used. Where a ligand has a natural anti-ligand, for example, biotin, thyroxine, and cortisol, it can be used in conjunction with the labeled, naturally occurring anti-ligands. Alternatively, any haptenic or antigenic compound can be used in combination with an antibody.

The molecules can also be conjugated directly to signal generating compounds, e.g., by conjugation with an enzyme or fluorophore. Enzymes of interest as labels will primarily be hydrolases, particularly phosphatases, esterases and glycosidases, or oxidoreductases, particularly peroxidases. Fluorescent compounds include fluorescein and its derivatives, rhodamine and its derivatives, dansyl, umbelliferone, etc. Chemiluminescent compounds include luciferin, and 2,3-dihydrophthalazinediones, e.g., luminol. For a review of various labeling or signal producing systems which may be used, see, U.S. Pat. No. 4,391,904, which is incorporated herein by reference.

Means of detecting labels are well known to those of skill in the art. Thus, for example, where the label is a radioactive label, means for detection include a scintillation counter or photographic film as in autoradiography. Where the label is a fluorescent label, it may be detected by exciting the fluorochrome with the appropriate wavelength of light and detecting the resulting fluorescence, e.g., by microscopy, visual inspection, via photographic film, by the use of electronic detectors such as charge coupled devices (CCDs) or photomultipliers and the like. Similarly, enzymatic labels may be detected by providing appropriate substrates for the enzyme and detecting the resulting reaction product. Finally, simple colorimetric labels may be detected simply by observing the color associated with the label. Thus, in various dipstick assays, conjugated gold often appears pink, while various conjugated beads appear the color of the bead.

Some assay formats do not require the use of labeled components. For instance, agglutination assays can be used to detect the presence of BoNT peptides. In this case, antigen-coated particles are agglutinated by samples comprising the target antibodies. In this format, none of the components need be labeled and the presence of the target antibody is detected by simple visual inspection.

### IX. Compositions.

The BoNT-neutralizing antibodies of this disclosure are useful in mitigating the progression of botulism produced, e.g., by endogenous disease processes or by chemical/biological warfare agents. Typically compositions comprising one, two, or more different antibodies can be provided as a pharmaceutical composition and administered to a mammal (e.g., to a human) in need thereof.

As disclosed herein, particularly efficient neutralization of a botulism neurotoxin (BoNT) subtype is achieved by the use of neutralizing antibodies that bind two or more subtypes of the particular BoNT serotype with high affinity. This can be accomplished by using two or more different antibodies directed against each of the subtypes and/or neutralizing antibodies that bind two or more BoNT subtypes (e.g., BoNT/B1, BoNT/B2, BoNT/B3, etc.) with high affinity.

Different neutralizing antibodies when combined, exhibit a potency that is increased dramatically. This increase makes it possible to generate a botulinum antibody composition of the required potency for therapeutic use. It was also surprising that as one begins combining two and three monoclonal antibodies, the particular BoNT epitope that is recognized becomes less important. Compositions comprising at least two, at least three, or more high affinity antibodies that bind overlapping or non-overlapping epitopes on the BoNT are contemplated herein.

Compositions contemplated herein may contain two, three, or more different antibodies selected from the following: 2A10, 3E1, 3E2, 3E3, 3E4, 3E4.1, 3E5, 3E6, 3E6.1, 3E6.2, 4E11, 4E13, 4E16, 4E16.1, 4E17, 4E17.1, A12, 6A12, B1.1, B6, B6.1, B6.C12, B6,D2, B8, B8.1, B11, B11.C3, B11E8, B11.A5, B11.F7, B11.H12, B12, B12.1, B12.2, 1B18, 2B18.1, 4B19, 4B19.1, 1B22, 1B10, 1B10.1, 2B18.2, 2B18.3, 1B22.4, 2B23, 2B24, 2B25, 2B25.1, 2B26, 2B27, 2B28, 2B29, 2b30, 4B17.1, 4B17.1C, 4B17.1D, 4B17.1F, 4B17.1G, 3E6.2, 4E17.4, 4E17.6, 4B19.1, B6.C12, B6.D2, B11.A5, B11.F7, B.H12, 4B1, 4B3, 4B5, 4B6, 4B7, 1B14, 4A1, 4A1.1, 5A20.4, 1C1, 5B1, 2B10, B11.E9, and 3C2. The composition may optionally further include antibodies comprising one or more CDRs from these antibodies, and/or one or more antibodies comprising mutants or derivatives of these antibodies. Exemplary compositions of the present disclosure contains at least 2, at least 3, or at least 4 of the antibodies B10.1, 2B18.1 B11E8, B6.1, B12.1 or of the antibodies 3E6.2, 4E17.1, 4E16.1, 3E2, which antibodies can be provided in combination with a pharmaceutical carrier.

An exemplary composition for binding BoNT/B includes 1B10.1, 2B18.1, and B11E8, 2B18.3 may be optionally included in addition to this exemplary composition. 2B18.1 may also be optionally substituted with 2B18.3 and vice versa. B11E8 may also be substituted with B12.1, 4B19.1, 2B23, or 1B22. This composition may further include B6.1 and/or B8.1. Such compositions can find use in neutralizing BoNT/B.

An exemplary composition for binding BoNT/E includes 4E17.1 and two or more antibodies from the following: 4E16.1, 3E2, 3E6.2, and 3E6.1. 3E6.1 may optionally be substituted with 3E6.2 and vice versa. Such compositions can find use in neutralizing BoNT/E.

The subject composition encompasses compositions that specifically neutralize one serotype, such as serotype BoNT/A, BoNT/B, or BoNT/E. The composition may also contain any first combination of antibodies described above that specifically neutralizes one serotype together with a second combination of antibodies that specifically neutralizes a different serotype. The subject composition may contain multiple combinations such that that composition may neutralize two, three, or more serotypes (e.g. BoNT/A, BoNT/B, and/or BoNT/E).

An exemplary composition that neutralizes multiple serotypes may include any of the combinations described above or one or more of the antibodies disclosed in Tables 1-5, together with one or more of the antibodies that bind to BoNT/A. Antibodies that bind to BoNT/A that may be included in the subject composition encompass those that have one or more CDRs or full-length VH or VL from clone 3D12, RAZ1, CR2, or 2G11, listed in the table below.

For example, a combination of antibodies that neutralize BoNT/A that may be in the subject composition may include RAZ1, CR2, and 2G11. Such a combination that neutralizes BoNT/A may be included in a composition containing the various exemplary combinations described above to neutralize BoNT/B or BoNT/E. As an example of a composition that neutralizes both BoNT/A and BoNT/B, the composition may include RAZ1, CR2, and 2G11 together with 1B10.1, 2B18.1, and B11E8, with any optional addition or substitutions mentioned above. One optional substitution of any of RAZ1, CR2, and 2G11 may be 4E17.1. 4E17.1 may also be added to a composition used to neutralize BoNT/A. Where a composition neutralizes all BoNT/A, BoNT/B, and BoNT/E, the composition may contain RAZ1, CR2, 2G11, 1B10.1, 2B18.1, B11E8, 4E17.1, together with two or more of 4E16.1, 3E2, 3E6.2 and/or 3E6.1. Similarly, any composition described herein may modified to include additional antibodies or to have an antibody substituted with another (e.g. derivative thereof).

Where combinations of antibodies are disclosed herein, such combinations can he provided in a single formulation or can be provided as separate formulations in a kit, where the separate formulations may contain a single antibody or two antibodies. Such separate formulations of a kit may be combined prior to administration or administered by separate injection.

The BoNT-neutralizing antibodies provided by the present disclosure are useful for parenteral, topical, oral, or local administration, such as by aerosol or transdermally, for prophylactic and/or therapeutic treatment. The pharmaceutical compositions call be administered in a variety of unit dosage forms depending upon the method of administration. For example, unit dosage forms suitable for oral administration include powder, tablets, pills, capsules and lozenges. The antibodies comprising the pharmaceutical compositions of the present disclosure, when administered orally, are preferably protected from digestion. This is typically accomplished either by complexing the antibodies with a composition to render them resistant to acidic and enzymatic hydrolysis or by packaging the antibodies in an appropriately resistant carrier such as a liposome. Means of protecting proteins from digestion are well known in the art.

The pharmaceutical compositions of the present disclosure are particularly useful for parenteral administration, such as intravenous administration or administration into a body cavity or lumen of an organ. The compositions for administration will commonly comprise a solution of one or more BoNT-neutralizing antibody dissolved in a pharmaceutically acceptable carrier, which may be an aqueous carrier. A variety of aqueous carriers can be used, e.g., buffered saline and the like.

Non-aqueous pharmaceutically acceptable carriers (excipients) are known to those of skill in the art. Such excipients, can comprise any substance that is biocompatible and liquid or soft enough at the subject's body temperature to release the active agent(s) (e.g., antibodies) somatotropin into the subject's bloodstream at a desired rate. Non-aqueous carriers are usually hydrophobic and commonly organic, e. g., an oil or fat of vegetable, animal, mineral or synthetic origin or derivation. The carrier may include at least one chemical moiety of the kind that typifies “fatty” compounds, e. g., fatty acids, alcohols, esters, etc., i. e., a hydrocarbon chain, an ester linkage, or both. “Fatty” acids in this context include, but are not limited to, acetic, propionic and butyric acids through straight- or branched-chain organic acids containing up to 30 or more carbon atoms. The non-aqueous carrier may be immiscible in water and/or soluble in the substances commonly known as fat solvents. The non-aqueous carrier can correspond to a reaction product of a “fatty” compound or compounds with a hydroxy compound, e. g., a mono-hydric, di-hydric, trihydric or other polyhydric alcohol, e.g., glycerol, propanediol, lauryl alcohol, polyethylene or-propylene glycol, etc. These compounds include, but are not limited to, the fat-soluble vitamins, e.g., tocopherols and their esters, e. g., acetates sometimes produced to stabilize tocopherols. Sometimes, for economic reasons, the carrier can comprise a natural, unmodified vegetable oil such as sesame oil, soybean oil, peanut oil, palm oil, or an unmodified fat. Alternatively the vegetable oil or fat may be modified by hydrogenation or other chemical means which is compatible with the present disclosure. The appropriate use of hydrophobic substances prepared by synthetic means is also envisioned. Non-aqueous excipient compositions can also comprise, in addition to a biocompatible oil, an “antihydration agent” which term as used herein means a substance that retards hydration of the active agent(s) and/or the biocompatible oil or fat and thereby further decreases and/or stabilizes the rate of release of the active agent(s) from that composition following administration to an animal. A great variety of non-toxic antihydration agents are known. By way of example there are “gelling” agents that, when dispersed, and in some cases heated to dissolve them in the oil, give the body of oil greater visco-elasticity (and therefore greater structural stability) and thereby slow down penetration of the oil by body fluids.

Illustrative antihydration agents include various polyvalent metal salts or complexes of organic acids, for instance fatty acids having from about 8 or 10 to about 20 or22 carbon atoms, e. g. aluminum, zinc, magnesium or calcium salts of lauric acid, palmitic acid, stearic acid and the like. Such salts can be mono-, di- or tri-substituted, depending on the valence of the metal and the degree of oxidation of the metal by the acid. Of common usage are the aluminum salts of such fatty acids. Aluminum monostearate and distearate are frequently used anti-hydration agents. Others that are useful include aluminum tristearate, calcium mono- and distearate, magnesium mono- and distearate and the corresponding palmitates, laurates and the like. The concentration of such an antihydration agent, based on the weight of the oil plus that agent, may be between about 1% and about 10% (most typically between about 2% and about 5%), although other concentrations may be suitable in some cases.

The various solutions are sterile and generally free of undesirable matter. These compositions may be sterilized by conventional, well known sterilization techniques. The compositions may contain pharmaceutically acceptable auxiliary substances as required to approximate physiological conditions such as pH adjusting and buffering agents, toxicity adjusting agents and the like, for example, sodium acetate, sodium chloride, potassium chloride, calcium chloride, sodium lactate and the like. The concentration of BoNT-neutralizing antibody in these formulations can vary widely, and will be selected primarily based on fluid volumes, viscosities, body weight and the like in accordance with the particular mode of administration selected and the patient's needs.

Thus, a typical pharmaceutical composition for intravenous administration would be about 0.1 to 10 mg per patient per day. Dosages from about 1 mg up to about 200 mg per patient per day can be used. Methods for preparing parenterally administrable compositions will be known or apparent to those skilled in the art and are described in more detail in such publications as Remington's Pharmaceutical Science, 15th ed., Mack Publishing Company, Easton, Pa. (1980).

The compositions containing the BoNT-neutralizing antibodies of the present disclosure or a cocktail thereof are generally administered for therapeutic treatments. Preferred pharmaceutical compositions are administered in a dosage sufficient to neutralize (mitigate or eliminate) the BoNT toxin(s) (i.e., reduce or eliminate a symptom of BoNT poisoning (botulism)). An amount adequate to accomplish this is defined as a “therapeutically effective dose.” Amounts effective for this use will depend upon the severity of the disease and the general state of the patient's health.

Single or multiple administrations of the compositions may be administered depending on the dosage and frequency as required and tolerated by the patient. In any event, the composition should provide a sufficient quantity of the antibodies of the present disclosure to effectively treat the patient.

**X. Kits for Diagnosis or Treatment.**

Kits for the treatment of botulism or for the detection/confirmation of a Clostridium botulinum infection are also provided. Kits will typically comprise one or more anti-BoNT antibodies (e.g., BoNT-neutralizing antibodies for pharmaceutical use). For diagnostic purposes, the antibody(s) can optionally be labeled. In addition the kits will typically include instructional materials disclosing means of use BoNT-neutralizing antibodies in the treatment of symptoms of botulism. The kits may also include additional components to facilitate the particular application for which the kit is designed. Thus, for example, where a kit contains one or more anti-BoNT antibodies for detection of diagnosis of BoNT subtype, the antibody can be labeled, and the kit can additionally contain means of detecting the label (e.g. enzyme substrates for enzymatic labels, filter sets to detect fluorescent labels, appropriate secondary labels such as a sheep anti-human antibodies, or the like). The kits may additionally include buffers and other reagents routinely used for the practice of a particular method. Such kits and appropriate contents are well known to those of skill in the art.

Kits provided for the treatment of botulism may contain one or more BoNT neutralizing antibodies. The antibodies can be provided separately or mixed together. Typically the antibodies will be provided in a sterile pharmacologically acceptable excipient. The antibodies can also be provided pre-loaded into a delivery device (e.g., a disposable syringe).

The kits can optionally include instructional materials teaching the use of the antibodies, recommended dosages, contraindications, and the like.

## EXAMPLES

The following examples are offered to illustrate, but not to limit any embodiments provided by the present disclosure.

### Example 1

**Human Monoclonal Antibodies to Botulinum Neurotoxin Types A and B from Immune Yeast Displayed Antibody Libraries**

The following methods and materials were used in the present example.

**Methods and Materials**

Oligonucleotide Primers

Primary Library Construction

Light Chain Shuffled Library Construction

VH Gene Amplification from pYD2

The underlined segment is the reverse complement of the JHFor primers

Other Primers

Other primers include the family of primers that used to create the VL-rep library (should anneal to the different Vk and Vl genes are add the scFv linker, VHFR4, NcoI-XhoI site. Also used is a family of primers to amplify the VH gene to clone into this vector, with 5′ overhang and 3′ overhang for gap repair into VL-rep library.

Strains, Media Antibodies, and Toxins

Yeast strain Saccharomyces cerevisiae EBY100 (GAL1-AGA1TURA3 ura3-52 trp1 leu2Δ1 his3Δ200 pep4::HIS3 prb1Δ1.6R can1) was maintained in YPD medium (1% yeast extract, 2% peptone, 2% dextrose) (Current Protocols in Molecular Biology, John Wiley and Sons, Chapter 13.1.2). EBY100 transformed with expression vector pYD2 (Razai et al., J Mol Biol. 2005, 351:158-69) was selected on SD-CAA medium (0.7% yeast nitrogen base, 0.1M Sodium phosphate, 0.5% casamino acids, 2% dextrose, 0.006% Leucine). ScFv yeast surface display was induced by transferring yeast cultures from SD-CAA to SG-CAA medium (identical to SD-CAA medium except the glucose was replaced by galactose) and growing at 18° C. for 24˜48 hr as described previously (Feldhaus et al. Nat Biotechnol. 2003, 21:163-70). Bacteria strain E. coli DH5α, (K12, A(lac-pro), supE, thi, hsdD5/F′ traD36, proA+B+, lacIq, lacZAM15) was used for cloning and preparation of plasmid DNA. Pure BoNT/A1 (Hall hyper), BoNT/A2 (FR1-H1A2), BoNT/B1, BoNT/B2, BoNT/B3, and BoNT/B4 were purified from their respective strains or purchased from Metabiologics (Madison, Wis.). Complex BoNT/A3 was expressed and purified from strains A254 and Ba207 (Hill et al. J. Bacteriol. 2007, 189:818-32). Mouse anti-SV5 and anti-myc 9E10 antibody and BoNT/A antibodies 7C1 and 9D8 were purified from hybridoma supernatant using Protein G and directly labeled with Alexa-488 or Alexa-647 using a kit provided by the manufacturer (Molecular Probes). Recombinant human BoNT/A antibodies 3D12 and AR2 and BoNT/B antibodies A12 and 6A12 were purified from Chinese hamster ovary cells (CHO) supernatants (Nowakowski et al. Proc Natl Acad Sci USA. 2002, 99:11346-50; Razai et al., J Mol Biol. 2005, 351:158-69). For flow cytometry (FACS), purified human or mouse IgGs were directly labeled with either Alexa-647 or Alexa-488 using a kit provided by the manufacturer (Molecular Probes).

Immune scFv Yeast Antibody Library Generation

scFv yeast libraries were constructed from human volunteers immunized with pentavalent toxoid. Briefly approximately 25 ml of blood was drawn 7 days after immunization and PBLs isolated using Lymphoprep tubes (Axis-Shield PoC AS, Norway). Blood was drawn after obtaining informed consent and under a protocol reviewed and approved by the USAMRIID and UCSF IRBs. Total RNA was isolated from PBLs using an RNAgents® kit (Promega). cDNA was synthesized from total RNA by using AMV reverse transcriptase (Invitrogen) and HuIgG1-4-C1FOR, HuCK1FOR primers (Amersdorfer et al. Vaccine 2002, 20:1640-8; Marks et al. Eur J Immunol. 1991, 21:985-91). VH and VK gene fragments were amplified by PCR from cDNA by using Pfu polymerase (Stratagene) and a mixture of HuVH1-6BACK, HuVH2bBACK, HuVH5bBACK, HuVH7aBACK and HuJH1-5FOR mix primers for the VH gene and HuVK1-6BACK, HuVK2aBACK, HuVK2bBACK and HuJK1-5FOR primers or the Vk gene (Marks et al. 1991, supra). scFv linker DNA template was prepared as previously described (Marks et al. 1991, supra). PCR fragments were gel purified, isolated from the gel using GENECLEAN® Turbo (Q.BIOgene). scFv gene repertoires were constructed by using PCR to splice together the VH and VK gene repertoires with scFv linker DNA (Marks et al. 1991, supra). The scFv genes were gel purified, isolated from the gel and reamplified using HuVHBACK and HuJkFOR primer mixes which appended NcoI and NotI site restriction sites (Marks et al. 1991, supra). scFv genes were digested with NcoI and NotI and ligated into NcoI-NotI digested pYD2 vector (Razai et al. 2005, supra). scFv gene repertoires were amplified from the ligation mixtures using primers GAP5 and GAP3 (Razai et al. 2005, supra) to append homologous overlaps with pYD vector DNA. Appended scFv PCR products were ethanol precipitated, combined with NcoI-NotI digested pYD2 and used to transform EBY100. Transformed yeast were cultured and subcultured in SDCAA and library size calculated by serially diluting and plating the transformed culture on SDCAA plates. Final library size was calculated as the product of the number of transformants and the percentage of clones with full length scFv insert as determined by PCR (Razai et al., 2005, supra).

Selection and Characterization of Lead BoNT/B and BoNT/A scFv Antibodies

For antibody selection, forty times the library size was grown and scFv display induced. For sorting, libraries were incubated with BoNT/A or BoNT/B in five times excess antigen, assuming each yeast expresses 1×105 scFv on the surface, at room temperature (RT) for 1 hr. For BoNT/B sorting, the first two rounds were done by staining with 50-100 nM BoNT/B1 and the third and fourth round of staining were done by staining with 1-10 nM BoNT/B1. Libraries were washed once with ice cold FACS buffer (PBS (1.9 mM NaH2PO4, 8.1 mM Na2HPO4, 154 mM NaCl2, pH7.4), 0.5% BSA, 1 mM MgCl2, 0.5 mM CaCl2). BoNT/B binding was detected by staining with using 2.5 μg/ml of an equimolar mixture of mAbs A12 and 6A12 labeled with Alexa-647 for 1 hr. at 4° C. Simultaneously, scFv display level was quantitated by staining with 2.5 μg/ml anti-SV5 mAb labeled with Alexa-488. Yeast libraries were washed as described above, resuspended in 200 to 700 μl FACS buffer and were sorted on a FACS Aria (Becton-Dickinson) with sort gates set to collect all SV5 positive BoNT/B binding yeast. After the last round of sorting, yeast were plated on SD-CAA plates and individual clones grown and induced. Individual clones were screened to identify BoNT/B binding scFv and unique clones identified by PCR fingerprinting and DNA sequencing (Amersdorfer et al. 2002, supra). For each unique clone, the affinity of the yeast displayed scFv for BoNT/B1, B2, B3, and B4 was determined exactly as previously described (Razai et al. 2005, supra).

For one set of BoNT/A sortings, the first two rounds of sorting were done by staining with 100 nM BoNT/A1 and 20 nM BoNT/A2 respectively. Libraries were washed once with ice cold FACS buffer (PBS (1.9 mM NaH2PO4, 8.1 mM Na2HPO4, 154 mM NaCl2, pH7.4), 0.5% BSA, 1 mM MgCl2, 0.5 mM CaCl2). BoNT/A binding was detected by staining with using 2.5 μg/ml of an equimolar mixture of mAbs 7C1 and 9D8 labeled with Alexa-647 for 1 hr. at 4° C. Simultaneously, scFv display level was quantitated by staining with 2.5 μg/ml anti-SV5 mAb labeled with Alexa-488. Yeast libraries were washed as described above, resuspended in 200 to 700 μl FACS buffer and were sorted on a FACS Aria (Becton-Dickinson) with sort gates set as in FIG. 8. For the last two rounds of sorting, yeast libraries were incubated with either 20 nM BoNT/A1 or 20 nM BoNT/A2 (see FIG. 8), washed, then stained with a combination of 2.5 μg/ml of AR1-Alexa-647 and 3D12-Alexa-488. Sort gates were set to capture only yeast binding BoNT/A which was bound by both 3D12 and AR1, thus ensuring that any resulting scFv would bind BoNT/A epitopes that did not overlap with these two mAbs. For a second set of BoNT/A sortings, the first three rounds of soiling were done by staining with 100 nM BoNT/A1, the third and fourth round of sorting were done by staining with 50 nM BoNT/A1, and the final round of sorting was done by staining with 50 nM recombinant BoNT/A1 LC or BoNT/A1 HN. For the first four rounds of sorting, BoNT/A binding was detected by staining with using 2.5 μg/ml of an equimolar mixture of mAbs 7C1 and 9D8 labeled with Alexa-647 for 1 hr at 4° C. For the final round of sorting, BoNT/A binding was detected by staining with 2.5 μg/ml of either mAbs 7C1 (for LC incubation) or 9D8 (for HN incubation) labeled with Alexa-647 for 1 hr. at 4° C.

After the last round of sorting, yeast were plated on SD-CAA plates and individual clones grown and induced. Individual clones were screened to identify BoNT/A binding scFv and unique clones identified by PCR fingerprinting and DNA sequencing (Amersdorfer et al. 2002, supra). For each unique clone, the affinity of the yeast displayed scFv for BoNT/A1 and BoNT/A2 was determined exactly as previously described (Razai et al. 2005, supra).

Construction and Expression of Yeast Displayed BoNT/B Domains and Mapping of Antibody Binding

Primers HCFor, HCRev, HNFor, HNRev, and LCFor and LCRev were designed to PCR amplify the BoNT/B HC, HN, or LC gene fragment respectively from the vectors, adding the restriction sites NcoI and NotI. Following digestion of both pYD2 and the resulting PCR amplification product with NcoI and NotI, the BoNT/B gene fragments were gel-purified and ligated into pYD2. Ligation mixtures were used to transform E. coli DH5α and correct clones identified by DNA sequencing. Plasmid DNA was used to transform LiAc-treated EBY100 cells. Yeast cultures were then grown and induced, as described above. For mapping scFv binding to yeast displayed BoNT/B domains, scFv genes in pYD2 were excised by digestion with NcoI and NotI, ligated into pSYN1, and the ligation mixture used to transform E. coli TG1. scFv expression was induced and the periplasmic fraction containing the say prepared as previously described. Periplasmic fractions containing scFv were incubated for 1 hour at RT with yeast displaying BoNT/B domains. After washing with PBS, yeast were incubated with 1 ug/ml of mAb 9E10 which recognizes an epitope tag at the scFv C-terminus. After washing, 9E10 binding was detected using 1 ug/ml anti-mouse gamma1 specific antibody and the level of BoNT domain display quantitated by incubation with 1 μg/ml of anti-SV5-Alexa-488. For some antibodies, mapping was performed by staining with 1 μg/ml of the respective IgG conjugated to Alexa-647.

Mapping the Overlap of BoNT Antibodies

Each yeast displayed scFv was grown and induced and incubated with 100 nM BoNT/B for 1 hour at RT followed by washing with PBS. After resuspension, yeast were incubated with scFv containing periplasmic preparations (see above), followed by washing and then incubation with 1 μg/ml 9E10. After washing, 9E10 binding was detected using 1 μg/ml anti-mouse gamma1 specific antibody and the level of BoNT domain display quantitated by incubation with 1 μg/ml of anti-SV5-Alexa-488. For some antibodies, mapping was performed by staining with 1 μg/ml of the respective IgG conjugated to Alexa-647.

Construction and Sorting of Chain Shuffled Yeast Antibody Libraries

To facilitate light chain shuffling, a light chain library was created in the yeast display vector pYD2 by cloning in VL gene repertoires from donors six, nine, and ten. cDNA was synthesized from total RNA prepared from donor PBLs by using AMV reverse transcriptase (Invitrogen) and HuCK1FOR primers (Amersdorfer et al. 2002, supra; Marks et al. 1991, supra). VK gene fragments were amplified by PCR from cDNA by using Pfu polymerase (Stratagene) and a mixture of HuVK1-6BACK, HuVK2aBACK, HuVK2bBACK and HuJK1-5FOR primers (Marks et al. 1991, supra). To further increase light chain diversity, the light chain repertoire from a large non-immune scFv phage antibody library was also utilized (Sheets et al. Proc Natl Acad Sci USA. 1998, 95:6157-62). Phage antibody library plasmid DNA was prepared and the VL genes PCR amplified using primers HuVK1-6BACK, HuVK2aBACK, HuVK2bBACK and HuJK1-5FOR for Vk genes and primers Vl genes. PCR fragments were gel purified and isolated from the gel using GENECLEAN® Turbo (Q.BIOgene). VL genes were reamplified by using PCR primers to append the 3′ end of the VH framework 4, containing a XhoI restriction site, and the (Gly4Ser)3 scFv linker to the 5′ end of the VL genes (see FIG. 9). VL repertoire DNA was gel purified, digested with NcoI and NotI and ligated into NcoI-NotI digested pYD2 DNA. The ligation mixture was used to transform E. coli DH5α, creating a library that was determined to be diverse by PCR fingerprinting and DNA sequencing. To create light chain shuffled scFv libraries, light chain library DNA was prepared and digested with either NcoI or HindIII and XhoI. It was determined that when cutting with NcoI-NotI, recombination could occur between the scFv linker DNA and the Gly-Ser linker after the AgaII protein, resulting in approximately 25% of transformants having no light chain. Digesting with HindIII cuts in AgaII, eliminating this problem. The VH gene was amplified from its respective scFv gene in pYD2 using primers that had 25 nucleotide sequences complementary to the 5′ and 3′ ends of the digested library vector DNA depending on whether the vector was digested with NcoI or HindIII, respectively. Gel purified VH gene was mixed with digested vector DNA and used to transform LiAc-treated EBY100 cells. Chain shuffled libraries were created for scFv 4A1.1, 5A20.4, B12.1, 2B18.1, 1B10.1, 4B19.1, and 2b25.1.

To select higher affinity scFv, light chain shuffled libraries were grown and scFv display induced. Yeast were stained with BoNT/A1 or BoNT/B1 at a concentration 10 times greater then the KD and equal to the KD for the first two rounds of sorting respectively with the majority of BoNT binding yeast collected. Subsequent rounds of sorting were increasingly stringent with the antigen concentration decreased and less than 1% of the yeast collected. A total of four to six rounds of sorting were performed for each chain shuffled library, after which the sort output was plated to allow for characterization of individual yeast displayed scFv. Twelve individual clones were characterized by DNA sequencing or the scFv gene and the affinity for BoNT determined as previously described.

Construction and Purification of IgG

The VH and VK genes of scFv ING1, ING2 were amplified with primers annealing to the 5′ and 3′ ends of the full length VH and VK genes and containing in frame MluI and NheI or DraIII and BsiWI restriction sites respectively for cloning into N5KG1Val-Lark or N5LG1Val-Lark (see Nowakowski et al. 2002, supra and Razai et al. 2005, supra for details of primer design). These vector results in expression of IgG of the γ1/kappa or γ1/lambda isotype. Amplified VH DNA was digested with MluI and NheI, and ligated into N5KG1Val-Lark or N5LG1Val-Lark. Clones containing the correct VH were identified by DNA sequencing. Amplified VK genes were cloned into pCR-TOPO (Invitrogen) and clones containing the correct VK identified by DNA sequencing. VK genes were excised from pCR-TOPO by digestion with DraIII and BsiWI and ligated into DraIII- and BsiWI-digested N5KG1Val-Lark or N5LG1Val-Lark DNA containing the appropriate VH gene. Clones containing the correct VH and VK genes were identified by DNA sequencing, and vector DNA was used to transfect CHO DG44 cells by electroporation. Stable cell lines were established by selection in G418 and expanded into 1 L spinner flasks. Supernatant containing IgG were collected, concentrated by ultra filtration, and purified on Protein G (Pharmacia) column.

Measurement of Solution Phase Affinity at Equilibrium

Equilibrium binding studies were conducted at room temperature (˜25° C.) using a KinExA 3000 flow fluorimeter to quantify the free BoNT/A or BoNT/B at equilibrium using varying concentrations of antibody as previously described (Razai et al. 2005, supra). Studies of reaction mixtures were performed m PBS (pH 7.4), with 1 mg/ml BSA and 0.02% (w/v) sodium azide as a preservative. Antibody was serially diluted into a constant concentration of pure BoNT/A or BoNT/B sufficient to produce a reasonable signal, where the antibody concentration was varied from less than 0.1 to greater than 10-fold above the value of the apparent KD. The BoNT/A and BoNT/B concentrations were no more than 4-fold above the KD to ensure a KD controlled experiment. Samples were allowed to reach equilibrium for as long as two days, then each of the 12 dilutions were passed over a flow cell with a 4 mm column of Azlactone beads (Sapidyne Instruments) eovalently coated with the corresponding antibody to capture the free BoNT/A or free BoNT/B. Passing an Alexa-647 labeled BoNT/A or BoNT/B antibody binding a non-overlapping epitope over the beads produced a signal relative to the amount of free BoNT/A or BoNT/B bound to the beads. All data points were run in duplicate and sample volume varied from 4 to 25 ml depending on antibody affinity. The equilibrium titration data were fit to a 1:1 reversible binding model using KinExA Pro Software (version 1.0.2; Sapidyne Instruments) to determine the KD (Drake et al. 2004 Anal. Biochem. 325:35-43).

**Results**

Generation of Human Yeast Displayed scFv Antibody Libraries

Immune yeast displayed scFv antibody libraries were constructed from human volunteers immunized with pentavalent botulinum toxoid (serotypes BoNT/A1, BoNT/B1, BoNT/C, BoNT/D, and BoNT/E). RNA was prepared from the peripheral blood lymphocytes of six different donors and the immunoglobulin heavy (VH) and kappa light (VK) chain variable regions amplified by using the polymerase chain reaction (PCR) as previously described (Marks 1991, supra; Marks et al. J. Mol Biol. 1991, 222:581-97). VH and VK gene repertoires from each donor were spliced together to create scFv gene repertoires which were cloned for display as N-terminal fusions to the agglutinin receptor (AgaII) protein on tile surface of yeast. A total of six yeast displayed say libraries was generated, ranging in size from 4.1 to 25.7×106 members. Each library was diverse as determined by PCR fingerprinting and DNA sequencing of 10 randomly selected clones. After induction of scFv display, the percentage of yeast displaying scFv ranged from 45-55% as determined by staining with SV5 antibody binding the C-terminal SV5 tag fused to each scFv.

Generation of High Affinity Human Antibodies to Type B Botulinum Neurotoxins

To generate a panel of human antibodies to type B botulinum neurotoxins, six different yeast displayed scFv libraries were sorted separately on BoNT/B1. Sorts were performed using relatively high concentrations of BoNT/B1 holotoxin in the initial rounds (50-100×10−9 M, nM) to ensure collection of all antigen binding scFv. In later rounds, the antigen concentration was decreased to between 1-10 nM to select for the higher affinity antibodies and sorts were performed on other BoNT/B subtypes (BoNT/B2, B3, and B4). Libraries were sorted a total of three to six rounds and yeast displayed scFv from individual colonies were screened for binding to BoNT/B1. Antigen binding clones were further characterized with respect to the diversity of scFv present using colony PCR, BstN1 fingerprinting (Marks et al. 1991, supra), and DNA sequencing. In this manner, 18 scFv were isolated, each with a different VH CDR3 (Table 8). The equilibrium binding constant for BoNT/B1 was measured for each of the yeast displayed scFv (Table 8). Affinities ranged from 60 to 0.08×10−9 M, with a mean KD of 8.6×10−9 M. For many of these scFv, a number of additional clonally related scFv were also isolated that were of lower affinity.

To determine the BoNT/B subtype specificity, the KD of the yeast displayed scFv were measured for BoNT/B2, B3, and B4 (Table 8). A number of scFv only bound the immunizing BoNT/B1 subtypes, e.g. mAb 4B1. To determine which BoNT/B functional domain the scFv bound, the BoNT/B HC, HN, and LC genes were cloned into pYD2 and displayed on the surface of Saccharomyces cerevisiae (FIG. 6), as previously reported for BoNT/A domains (Levy et al. J Mol Biol. 2007, 365:196-210). Each domain was well displayed on the yeast surface, as quantitated using a C-terminal SV5 tag fused to each domain (FIG. 6). The domain recognized by each of the scFv was determined by incubating yeast displayed BoNT/B domains with either native scFv expressed in E. coli or IgG constructed from scFv genes (see below). Native scFv was generated by subcloning the say genes into the bacterial secretion vector pSYN1. To determine how many non-overlapping BoNT/B epitopes were recognized by the lead antibodies, yeast displayed scFv were incubated with BoNT/B holotoxin, followed by incubation with purified native scFv. scFv recognizing overlapping epitopes showed no yeast staining while scFv binding non-overlapping epitopes stained the yeast surface. For some of these assays, purified IgG constructed from the scFv V-genes was used for yeast staining instead of the scFv. Using these assays, it was determined that of the 18 scFv, five bound the BoNT/B HC, three bound the HN, and ten bound the LC (FIG. 7). A total of nine non-overlapping epitopes were recognized by the scFv antibodies, three on the HC, two on the HN, and four on the LC. Seven of the ten LC antibodies clustered within one antibody footprint of each other (FIG. 7). Of note, of the 10 scFv that bound to all four BoNT/B subtypes, 9 bound to LC, 1 bound to HN, and none bound to HC. This is consistent with the relative percent homology of the three domains between the four BoNT/B subtypes, the LC is the most conserved and the HC is the least conserved.

Generation and Characterization of High Affinity Human Antibodies to Type A Botulinum Neurotoxins

The sortings of yeast displayed scFv libraries were designed to generate antibodies that bound multiple BoNT/A subtypes and/or bound to epitopes not recognized by previously generated antibodies. To generate antibodies binding multiple BoNT/A subtypes, a library constructed from donor six was sequentially sorted on BoNT/A1 and BoNT/A2 (FIG. 8). To generate antibodies binding epitopes not recognized by existing mAbs, yeast libraries were incubated with BoNT/A then simultaneously stained with mAbs 3D12 and AR2 labeled with different fluorophores and sorted (FIG. 8). Using this approach, two yeast displayed scFv were identified, ING1 and ING2. A number of additional scFv were also isolated that were of lower affinity but which were clonally related to ING1 or ING2, having the same VH CDR3 but point mutations in the gene and/or different light chain genes. ING1 and ING2 both bound BoNT/A1 and A2 with high affinity as yeast displayed scFv, but only ING1 bound BoNT/A3 (Table 9). Additional mAbs to BoNT/A epitopes were generated by sorting a yeast displayed library constructed from donor ten on BoNT/A1. From these sorts, two new yeast displayed scFv were identified, 4A1 and 5A20. Both of these yeast displayed scFv only bound BoNT/A1. A number of additional scFv were also isolated that were of lower affinity but which were clonally related to 4A1 or 5A20, having the same VH CDR3 but point mutations in the VH gene and/or different light chain genes.

Yeast displayed BoNT/A scFv were further characterized to determine which BoNT domain they bound by staining with recombinant BoNT/A HC, HN, or LC. 4A1 was determined to bind the HN domain and both ING2, and 5A20 bound the LC (Table 9). We previously determined using yeast displayed BoNT/A domains that ING1 recognized a complex epitope requiring the presence of both the N-terminal domain of the BoNT/A heavy chain translocation domain (HN) and the BoNT/A light chain by (LC) (Levy et al. 2007, supra). To determine whether the epitopes recognized by the antibodies overlapped, yeast displayed ING1, ING2, 4A1, or 5A20 scFv was incubated with BoNT/A and then stained with IgG constructed from each of these scFv. Each of these mAbs was found to recognize non-overlapping epitopes.

Affinity Maturation of Antibodies to Type A and Type B Botulinum Neurotoxins

Since it is statistically improbable that the original immune cognate VH-VL pair is recreated in the primary library, scFv affinity maturation was performed by recombining the scFv VH gene with a library of VL genes, so called light chain shuffling (Clackson et al. Nature 1991, 352:624-8; Marks et al. Biotechnology (N Y). 1992, 10:779-83; Schier et al. J Mol Biol. 1996, 263:551-67). To facilitate light chain shuffling, a light chain library was created in the yeast display vector pYD2 by subcloning the VL genes from the donor six, nine, and ten scFv gene repertoires in pYD2. To further increase light chain diversity, especially since lambda light chain V-genes had not been amplified from the immune donors, the light chain repertoire from a large non-immune scFv phage antibody library (Sheets et al. 1998, supra) that had been subcloned into pYD2 was also utilized. VL genes were amplified from the scFv gene repertoires in pYD2 by using PCR and then reamplified to append the end of the VH framework 4 (FR4), containing a XhoI restriction site, and the (Gly4Ser)3 scFv linker to the 5′ end of the VL genes (FIG. 9). After cloning into pYD2, a library of size 4.2×107 was created that was determined to be diverse by PCR fingerprinting and DNA sequencing.

To create light chain shuffled scFv libraries, the light chain library was digested with NcoI or HindIII and XhoI. The VH gene was amplified from its respective scFv gene in pYD2 using primers that had 25 nucleotide sequences complementary to the 5’ and 3′ ends of the digested library vector DNA and was cloned into Saccharomyces cerevisiae by gap repair. Alternatively, scFv chain shuffled libraries were created by amplifying the VH FR4-scFv linker-VL gene repertoire from pYD2 and splicing it to a specific VH gene by overlap extension. The chain shuffled scFv gene repertoire was then cloned into pYD2. A total of seven chain shuffled libraries were created from the VH genes of scFv 4A1, 5A20, 1B18, 1B10, 1B22, 2B25, and 4B19. Their sizes ranged from 2.0×107 to 4.0×107. An additional chain shuffled library was created from the VH genes of scFv B12 (see table 3) using splicing by overlap extension as previously described (Marks et al. 1992, supra). Each of these libraries was sorted separately using BoNT/A1 or BoNT/B1 at decreasing concentrations for four to six rounds until a single clone became dominant. The highest affinity clone was identified by DNA sequencing and measuring the KD of yeast displayed scFv. The KD of the light chain shuffled scFv increased 2.2 to 74 fold from the KD of the parental scFv to a KD between 1.0 and 6.38×10−10 M. In general, the largest increase in affinities was observed for the lowest affinity parental scFv. The average KD increased 16 fold from 4.98×10−9 M to 0.31×10−9 M (Table 10).

Impact of Conversion of Yeast Displayed scFv to IgG on Affinity

For many applications, such as diagnostic ELISA based assays as well as in vivo BoNT neutralization studies, it is desirable to utilize IgG. To determine the success rate of converting scFv to IgG, we converted twelve yeast displayed scFv, including five lead antibodies and seven affinity matured chain shuffled scFv, to full length IgG consisting of the human gamma 1 constant region and the human kappa or lambda constant region (Nowakowski et al. 2002, supra). Stable CHO DG44 cell lines were established for each of the twelve antibodies and IgG was purified from cell culture supernatant. The monovalent was determined for each IgG using kinetic exclusion analysis (KinExA).

Each of the twelve scFv was successfully cloned, stable cell lines established, and IgG purified at yields of 5 to 50 mg/L of cell culture supernatant. The solution KD of each of these IgG was lower (higher affinity) than the parental scFv (Tables 10, and 11). The affinity of the five IgG constructed from the lead scFv increased 1.5 to 25 fold from the affinity of the parental scFv, with the average KD decreasing 8.8 fold from 2.68×10−9 M to 0.31×10−9 M (Table 11). Similarly, the affinity of the seven IgG constructed from the affinity matured scFv increased 2.25 to 712 fold from the affinity of the parental scFv, with the average KD decreasing 6.43 fold from 3.14×10−10 M to 0.49×10−10 M (Table 10). Six of the seven IgG constructed from the affinity matured scFv had KD's less than 1.0×10−10 M.

**Discussion**

These studies indicated that the scFv were diverse with respect to epitope recognized and were of an affinity (average KD=9.2×10−9 M) expected from an immune library. On average, three lead scFv were generated from each library constructed from a human donor, a relatively small number.

The magnitude of the increase in affinity from chain shuffling did not appear to be dependent on whether the source of the VL genes was from the same donor as the initial VH-VL gene pairing or not. Presumably, light chain diversity across donors was similar enough that shuffling recapitulated the high affinity scFv binding site. The average affinity of the seven IgG constructed from the chain shuffled scFv V-genes was 45 pM, a value that would be at the very high end of antibodies generated by the humoral immune system. Three of these antibodies had KD<1.5 pM. The very high average affinity suggests that these scFv do not represent the cognate VH-VL pairing, which should not have such a high affinity. Rather it is likely that a VL pair was found for the VH that generated an antibody of higher affinity than that of the cognate VH-VL pairing.

### Example 2

**Affinity Maturation of Human Botulinum Neurotoxin Antibodies by Light Chain Shuffling via Yeast Mating**

The following methods and materials were used in the present example.

**Methods and Materials**

Oligonucleotide Primers

Primers for VL Library Construction

Strains, Media Antibodies, and Toxins:

Yeast strains Saccharomyces cerevisiae JAR300 ((GAL1-AGA1TURA3 ura3-52 trp1 leu2Δ1 his3Δ200 pep4::HIS3 prb1Δ1.6R can1 MATa) and YVH10 (BJ5464 Ura-Trp-MAT alpha) were maintained in synthetic dextrose plus casein amino acids (SD-CAA, 0.7% yeast nitrogen base, 0.1M Sodium phosphate, 0.5% casamino acids, 2% dextrose, 0.006% leucine) media with tryptophan and uracil added. After transformation with pPNL20s, JAR300 transformants were selected in SD-CAA media with uracil. After transformation with pPNL30s, YVH10 transformants were selected in SDCAA media with tryptophan. Yeast mating was performed on YPD plates (Current Protocols in Molecular Biology, John Wiley and Sons, Chapter 13.1.2). After mating, diploid yeast were selected on SD-CAA media. Fab surface display was induced by transferring yeast cultures from SD-CAA to SG-CAA medium (identical to SD-CAA medium except the glucose was replaced by galactose) and growing at 18° C. for 24-48 hours. Bacteria strain E. coli DH5α, (K12, Δ(lac-pro), supE, thi, hsdD5/F′ traD36, proA+B+, lacIq, lacZΔM15) was used for all cloning and preparation of plasmid DNA. Chinese Hamster Ovary cell line CHO DG44 was maintained in CHO-SFM-II media (Invitrogen). Pure BoNT/A1 (Hall hyper), BoNT/A2 (FR1-H1A2), BoNT/B1, BoNT/B2, BoNT/B3, and BoNT/B4 were purified from their respective strains or purchased from Metabiologics (Madison, Wis.). Complex BoNT/A3 was expressed and purified from strains A254 and Ba207. Mouse anti-SV5 antibody was purified from hybridoma supernatant using Protein G and directly labeled with Alexa-488 or Alexa-647 using a kit provided by the manufacturer (Molecular Probes). Chinese ovary cells (CHO) supernatants (Nowakowski et al. 2002, supra; Razai et al. 2005, supra). For flow cytometry (FACS), purified human or mouse IgGs were directly labeled with either Alexa-647 or Alexa-488 using a kit provided by the manufacturer (Molecular Probes).

Construction of a Light Chain Repertoire in YVH10 for Light Chain Shuffling (Donor 9)

A single light chain repertoire in YVH10 was constructed containing kappa and lambda light chains. First, vector pPNL30 was modified to insert a 90 base pair stuffer between the XhoI and BsiWI sites.

Two different methods were used to construct the three Fab libraries for affinity and specificity maturation of three scFv lead clones: ING1, B6 and B11. For ING1 and B6 Fab library construction before yeast mating, a single VH gene from the lead scFv was PCR amplified using the specific primers mentioned above. But for B11Fab library construction, error-prone PCR was applied when preparing a mutated VH repertoire from the lead scFv B11 VH gene. The single VH gene of ING1 or B6, or the whole repertoire of randomly mutated B11 VH gene were then cloned into pPNL20S vector. Cloning was achieved by co-transforming JAR300 yeast (by the method of Gietz and Schiestl) with BamH I/Nhe I linearized vector and gap-tailed amplicon utilizing yeast gap repair. The transformants were selected on SD-CAA+uracil before mating with VL library yeast.

The VL libraries were constructed using conventional methods. Briefly, the light chain vector pPNL30S was digested with XhoI/BsiWI and gel-purified. YVH10 yeast was used for pPNL30S+VL transformation, and the transformants selected on SD-CAA+tryptophan.

For yeast mating, fresh cultures of VL library in YVH10/pPNL30-LC (MAT α strain) after selection were grown in the selective media at 30° C. with shaking, and a single VH clone or a repertoire of VH (for B11) library in JAR300/pPNL20-HC (MAT “a” strain) were grown in another selectable media similarly. 1 OD600/ml (2×107 yeast) of each culture were mixed together, pelleted, and resuspended in 200 μl YPD media before placing in the center of a pre-warmed 30° C. YPD plate without spreading. The plates were incubated at 30° C. for 6 h for yeast mating and the yeast were resuspended in SD-CAA medium from the plate after mating. To check the mating efficiency and the final Fab library size, appropriate dilutions of mated yeast were plated on SD-CAA agar plates and the Fab library was selectively grown in SD-CAA media without tryptophan or uracil. The OD600 reading at the start of growth was 0.025 to allow growth of the diploids to outcompete the non-growing haploids.

For diversity analysis, ten randomly picked clones from each library were checked by BstNI fingerprinting and/or sequencing for both VH and VL genes.

For Fab expression induction, freshly saturated SD-CAA cultures of the yeast libraries were induced in SG-CAA liquid media at 18° C. for 24-48 h with shaking. Anti-SV5/Alexa647 were used to check the Fab expression after induction.

Selection of Mutant Clones from Fab Libraries by FACS

The optimally expressed diploid yeast were FACS sorted two rounds with one of the subtypes of BoNT/A or B, then using different subtypes of BoNT/A or B for the following rounds of sorting. An amount of diploid yeast at least ten times larger than the library size or the sort output from the previous round were washed and resuspended in FACS buffer, and a desired concentration of one subtype of pure BoNT/A or B was added. Sorting and selections were generally performed after allowing the reaction mixture to come to equilibrium. The volume for incubation of yeast with toxin was chosen to ensure that toxin was in at least a five fold excess over the number of expressed Fab (assuming 5×105 Fab/yeast). Incubation times were chosen to ensure that the reaction had come to at least 90% of equilibrium, using the formula shown in the Discussion section below. BoNT/A1 concentration for the first round of sorting of the ING1 Fab library was 10 nM. The 1st round sort output were divided into two parts after yeast growth and induction for the second round sorting: one with 10 BoNT/A2, the other using 5 nM BoNT/A1. Since both BoNT/A1 and A2 binders are clearly enriched after two rounds of sorting, for better cross reactive and higher affinity clone selection, decreased concentrations of BoNT/A2 were used for the following 3 rounds of sorting: 5.1 and 0.5 nM before individual clones were picked out for sequencing and affinity measurement. BoNT/B1 concentrations used for the 5 rounds of sorting of the B6 Fab library were: 100, 25, 10, 5, or 2 nM separately, while those for the B11 Fab library were: 25, 10, 2, 1, or 1 nM. After incubation, cells were washed with ice-cold FACS buffer and resuspended in a 1:400 dilution of Alexa 488 labeled anti-SV5 antibody and one of the following: Alexa 647 labeled 7C1 or 3D12 antibody (for the ING1 Fab library), or two Alexa-647 labeled anti-BoNT/B mAb (A12 and 6A12), which bound the catalytic domain of the toxin, an epitope that did not overlap with B6 or B11 (for the B6 Fab library and B11 Fab library). Cells were incubated for 30 minutes with secondary antibodies at 4-8° C., washed once with FACS buffer, resuspended in 0.5-1 ml of FACS buffer and sorted on a FACSAria. Typically 0.5-5% of the expressing and toxin binding population was gated for collection in the first 2 round sorting, and 0.1-0.5% of those populations was gated in the following rounds sorting. Collected cells were grown in SD-CAA media and used for the next round of sorting after induction in SG-CAA as described above.

Over a period of 2-3 weeks, each library was sorted 4-6 rounds with sequentially diluted different subtypes of BoNT A or B. Ninety-six individual clones were picked out after the last round of sorting, and 500 pM Alexa 647 or 488 labeled toxin were used to screen the best candidates in terms of expression and binding by FACS analysis. Using MFI as a single criterion after fixed concentration toxin staining, six to twelve clones were chosen from the screened ones for further KD measurement and sequence analysis.

Quantitative equilibrium binding was determined using flow cytometry. In general, six different concentrations of one subtype of pure BoNT/A or B were utilized spanning a range of concentrations from ten times above to ten times below the KD. Incubation volumes and number of yeast stained were chosen to keep the number of antigen molecules in tenfold excess above the number of expressed Fab, assuming 5.0×105 Fab/yeast. Incubation times were chosen based on anticipated times to equilibrium calculated using approximations of the anticipated kon and koff (see above). This was usually performed by overnight incubation. Binding of either subtype of BoNT/A to yeast-displayed Fab was detected using a 1:200 dilution of 1 mg/ml monoclonal antibody binding a non-overlapping BoNT/A epitope (3D12 for the ING1 Fab library) labeled with Alexa-647. To quantify the protein-ligand affinity constant (KD) within the surface display context, only the Fab displaying yeast (SV5 binding positive) were included in the analysis by co-staining with SV5-Alexa-488. Each KD was determined in triplicate, three separate inductions and measurements. The best Fab clone from the ING1 Fab library was designated as 2G11, the highest affinity clone front B6 Fab library was named as B6.1, and that from the B11 Fab library was called B11E8 (Table 13).

Generation of IgG from Fab

IgG were generated from Fab genes. Briefly, the VH genes were amplified using PCR from their respective pPNL20 vectors with the primer pairs. DNA was digested with MluI and NheI, ligated into N5KG1Val-Lark (a gift from Mitch Reff, IDEC Pharmaceuticals, San Diego) and clones containing the correct VH identified by DNA sequencing. Vk genes were amplified from the same diploid yeast containing pPNL30 vectors and cloned into pCR-2.1 vector (Invitrogen) and clones containing the correct Vk identified by DNA sequencing. Vk genes were excised front pCR-2.1 vector with DraIII and BsiWI and ligated into DraIII and BSiWI-digested N5KG1Val-Lark DNA containing the appropriate VH gene. Clones containing the correct VH and Vk gene were identified by DNA sequencing, and vector DNA was used to transfect CHO DG44 cells by electroporation. Stable cell lines were established by selection in G418 and expanded into one liter spinner flasks. Supernatant containing IgG was collected and purified on Protein G (Pharmacia). IgG purity was assessed by native and denaturing SDS PAGE and concentration determined by A280 nm.

Measurement of IgG Solution Equilibrium Binding Constants by Flow Fluorimetry

Equilibrium binding studies were conducted at room temperature (˜25° C.) using a KinExA 3000 flow fluorimeter to quantify the free. BoNT/A or B at equilibrium using varying concentrations of antibody. Studies of sensitive detection of BoNT/A or B reaction mixtures were performed in PBS (pH 7.4), with 1 mg/ml BSA and 0.02% (w/v) sodium azide as a preservative. Antibody was serially diluted into a constant concentration of BoNT/A or B sufficient to produce a reasonable signal, where the antibody concentration was varied from less than 0.1 to greater than tenfold above the value of the apparent KD. The BoNT/A or B concentrations were no more than fourfold above the KD to ensure a KD controlled experiment. Samples were allowed to reach equilibrium for as long as two days, then each of the 12 dilutions were passed over a flow cell with a 4 mm column of Azlactone beads (Sapidyne Instruments) covalently coated with the corresponding antibody to capture the free BoNT/A or B. Passing an Alexa-647 labeled BoNT/A or B antibody binding a non-overlapping epitope over the beads produced a signal relative to the amount of free BoNT/A or B bound to the beads. All data points were run in duplicate and sample volume varied from 4 to 25 ml depending on antibody affinity. The equilibrium titration data were fitted to a 1:1 reversible binding model using KinExA Pro Software (version 1.0.2; Sapidyne Instruments) to determine the KD.

Antibodies Selected for Affinity Maturation by Light Chain Shuffling

One BoNT/A and two BoNT/B scFv antibodies were chosen for affinity maturation. The BoNT/A scFv antibody ING1 was selected from an immune scFv yeast display library constructed from a human donor (donor 6) immunized with pentavalent botulinum toxoid. ING1 binds the BoNT/A1 translocation domain (HN) with an equilibrium dissociation constant (KD) of 5.28×10−9 M as a yeast displayed scFv (Table 12). Two BoNT/B scFv antibodies (B6 and B11) were also selected from an immune scFv yeast display library constructed from a human donor immunized with pentavalent botulinum toxoid. B6 binds the BoNT/B light chain (LC) with a KD of 2.71×10−9 M and B11 binds the BoNT/B HN with a KD of 2.62×10−9 M (Table 12).

Generation of BoNT Fab from the V-Genes of scFv by Yeast Mating

To determine the feasibility of affinity maturing ING-1, B6, and B11 by yeast mating, yeast displayed Fab were constructed and the display level and KD for BoNT determined. The VH gene from each of the three scFv was PCR amplified and cloned directly into S. cerevisiae strain JAR300 by gap repair into BamH I and Nhe I digested pPNL20s. This fuses the VH gene to the CH1 gene which is fused to the N-terminus of the Aga2 yeast surface protein, resulting in display of the VH-CH1 on the yeast surface. The Vk gene from each of the three scFv was PCR amplified and cloned directly into S. cerevisiae strain YVH10 by gap repair into XhoI and BsiWII digested pPNL30s. This fuses the Vk gene to the Ck and results in secretion of the light chain. To create Fab, S. cervisiae JAR300 (a-mating type) containing the relevant VH-CH1 in pPNL20s was mixed with S. cervisiae YVH10 (a-mating type) containing the relevant light chain in pPNL30s and the resulting diploid yeast selected on uracil−, tryptophan− plates. Fab display was induced and the display level and affinity for BoNT measured by flow cytometry. All three Fab were well displayed on the yeast surface (data not shown). The affinity of the ING1 Fab was 3.9 and 4.4 fold lower than the scFv for BoNT/A1 and BoNT/A2 respectively, while the affinity of the B6 Fab was essentially identical to the affinity of the scFv for BoNT/B1 and BoNT/B2 (Table 12). Interestingly, the affinity of the B11 Fab was 2.5 lower than the scFv for BoNT/B1 but 1.46 higher than the scFv for BoNT/B2 (Table 12). The results indicate that Fab can be constructed from the V-genes of scFv using yeast mating and that the KD of yeast displayed Fab and scFv are comparable.

Generation and Sorting of Fab Chain Shuffled Libraries

The following strategy was used to create ING1, B6, and B11 light chain shuffled libraries. A library of human kappa and lambda light chains from donor 10 was cloned directly into vector pPNL30s in S. cerevisiae strain YVH10 by gap repair. The resulting light chain library contained transformants containing a light chain insert and was diverse as determined by DNA sequencing. To create Fab light chain shuffled libraries, S. cervisiae JAR300 (a-mating type) containing the relevant VH-CH1 (ING1, B6, or B11) in pPNL20s was mixed with S. cervisiae YVH10 (a-mating type) containing the light chain library in pPNL30s and the resulting diploid yeast selected on uracil−, tryptophan− plates. The number of diploid yeast colonies was at least 100 times greater than the size of the light chain shuffled library, suggesting that the light chain library diversity was captured in the chain shuffled library. Analysis of 10 colonies from each mating indicated that each had the expected wild-type VH gene and a different VL gene. Each of the 10 colonies also expressed a Fab on the yeast surface, as determined by staining with anti-SV5 antibody.

Higher affinity chain shuffled Fab were isolated by FACS. For each of the three chain shuffled libraries, Fab expression was induced and yeast stained with 1˜10×10−8 (10 nM˜100 nM) BoNT/A1 or BoNT/B1 for the first two rounds of sorting with the majority of BoNT binding yeast collected. Subsequent rounds of sorting were increasingly stringent with the antigen concentration decreased and less than 1% of the yeast collected. A total of six rounds of sorting were performed for each Fab library, after which the sort output was plated to allow for characterization of individual yeast displayed Fab. Ninety-six clones were randomly picked into 96 well microtiter plates from each of the three Fab sortings and Fab expression and BoNT binding quantitated using a single antigen concentration (5.0×10−10 M, 500 pM). While almost 100% of the picked clones showed a positive BoNT binding signal, the mean fluorescence intensity (MFI) varied significantly among them. Twelve individual clones with the highest MFI were chosen for further analysis.

Characterization of Chain Shuffled Fab

The VH and VL genes were PCR amplified from the three sets of twelve yeast colonies and the VH and VL genes sequenced. For each of the three chain shuffled libraries, sequence analysis revealed a single VH gene (the same as the original VH gene) paired with different VL genes (Table 13). Affinities (KD) of the yeast displayed Fab were measured by flow cytometry for BoNT/A1 or BoNT/B1 and compared to the affinities of the parental scFv (Table 13). Overall the affinities of the best ING1, B6, and B11 chain shuffled Fab increased 3 to 43 fold compared to their parental scFv, with yeast displayed KD ranging from 32 to 1645 pM (Table 13). The complete amino acid sequences of the VH and VL of the antibodies described in Table 13 are provided in FIG. 11.

In the case of the BoNT/A antibody ING1, seven Fab were identified which had affinities 3 to 25 fold higher than the parental ING1 scFv (Table 13). All seven had VL genes derived from the same Vk1 germline gene family and 5 of the 7 were derived from the same IGKV1-39*01 germline gene. Compared to the parental Vk gene, the Vk genes from the affinity matured Fab had 6 to 16 amino acid substitutions (Table 13). In the case of the BoNT/B antibody B6, three different Fab were identified which had affinities higher than the parental B6 scFv (Table 13). All three had VL genes derived from exactly the same Vk1 germline gene family as the parental Vk gene. The Vk genes from the affinity matured B6 Fab were more mutated than those of the ING1 affinity matured Fab, having 26 to 39 amino acid substitutions compared to the parental Vk gene (Table 13). In the case of the BoNT/B antibody B11, different Fab were identified which had affinities 61 to 177 fold higher than the parental B11 scFv (Table 13). Two had VL genes derived from the same Vk1 germline gene family as the parental Vk gene, however three had Vk genes derived from the Vk3 family. The Vk genes from the affinity matured Fab had 16 to 33 amino acid substitutions compared to the parental Vk gene (Table 13).

Generation and Characterization of IgG Constructed from the V-Genes of Fab

To determine whether the affinities observed for the Fab were recapitulated in IgG, and to generate antibodies for the diagnosis and treatment of botulism, we converted 2G11, B6.1 and B11E8 to human IgG1/kappa antibodies. IgG were expressed from stable CHO cell lines, purified using protein G and solution KD for BoNT/A or BoNT/B subtypes measured by flow fluorimetry. The affinities of the three IgG ranged from 25.1 to 6.59 pM for BoNT/A1 or BoNT/B1 and were increased from 4.85 fold to 42 fold compared to the affinities measured for the yeast displayed Fab (Table 14). We have previously observed comparable increases in affinities for yeast displayed scFv converted to IgG. 2G11 bound three BoNT/A subtypes (Bont/A1, A2, and A3 with comparable and high affinity. B6.1 bound four BoNT/B subtypes (BoNT/B1, B2, B3, and B4) with comparable and high affinity. In contrast, B11E8 only bound BoNT/B1, B2, and B3 (Table 14).

### Example 3

**Neutralization of BoNT/B In Vivo by BoNT/B Antibody Combinations**

In vivo, as found for BoNT/A, single mAbs protected mice against only relatively low BoNT/B challenge doses (up to 100 mouse LD50s per 50 μg of mAb). But for BoNT/A and BoNT/E, mAb combinations showed much more potent neutralization than for BoNT/B neutralization. The most potent three mAb combination (2B18.1:1B10.1:B12.1), however, only protected mice up to a challenge dose of 40,000 LD50s/50 μg of antibody, about 10 fold less than seen for three mAb combinations against BoNT/A and BoNT/E (Table 15). Addition of a fourth mAb (B11E8) increased potency of about 50 fold, such that all mice receiving 2 μg total of antibody were completely protected against challenge with 40,000 mouse LD50s of BoNT/B1. Since 1 IU for BoNT/B is the amount of antibody neutralizing 10,000 LD50s of BoNT/B (50% of mice survive), this would translate into a potency of about 2000-4000 IU/mg. Thus a total human therapeutic dose equivalent to the 5500 IU of licensed equine antitoxin used to treat type B botulism would be less than about 2.5 mg. Other combinations of four mAbs had comparable potency against BoNT/B1. Protection against challenge by BoNT/B2 by mAbs 2B18.1:1B10.1:B12.1:B11E8 was about 4 to 5 fold less than the protection seen for BoNT/B1, similar to the reduced potency seen against this subtype for the equine antitoxin. The reason for the reduced potency is not clear, since the mAbs bind BoNT/B2 with affinities comparable to the affinities for BoNT/B1. Regardless, the four mAb combination is still highly potent against BoNT/B2. Based on the significantly higher potency observed for the four mAb combinations, a combination of four mAbs rather than three may be preferable. BoNT/B antitoxin compositions.

An equimolar combination of the indicated amount of mAb was mixed with the indicated BoNT/B subtype and injected into mice. The amount of mAb given is the total dose of the four mAb combination. The number of mice surviving of 10 studied is reported.

### Example 4

**Neutralization of BoNT/E In Vivo by BoNT/E Antibody Combinations**

In vivo, as found for BoNT/A, single mAbs protected mice against only relatively low BoNT/E challenge doses (up to 200 mouse LD50s per 50 μg of mAb). Similar as for BoNT/A, mAb combinations showed highly potent BoNT/E neutralization. In vivo, 5-10 ug of the most potent combination of three mAbs (3E2, 3E6.1, and 4E16.1) protected mice challenged with 40,000 mouse LD50s of BoNT/E1 and BoNT/E3 (Table 16). Since 1 IU of BoNT/E antibody neutralizes 1,000 mouse LD50s, about 1 mg of this mAb combination would be the equivalent of 8.000 IU of antitoxin for BoNT/E1 or E3. This is equivalent to the human therapeutic dose of investigational BoNT/E equine antitoxin given to treat type E botulism. The potency of some of the different three mAb combinations of the four different BoNT/F mAbs was also studied, and found to be approximately equipotent in the three mAb combination.

An equimolar combination of the indicated amount of three mAbs was mixed with the indicated BoNT/E subtype and injected into mice. The amount of mAb given is the total dose of the three mAb combination. The number of mice surviving of 10 dosed is reported.

### Example 5

**Methods and Materials**

Oligonucleotide Primers

Standard PYD2 primers: pair Gap 5 and Gap 3 or pair PYDFor and PYDRev are used for per for sequencing and/or cloning. Gap 5 and Gap3-1 are used for DNA sequencing.

Primers for amplifying and cloning the target antigen gene by gap repair:

The 5′ sequence indicated anneals to NcoI-NotI digested pYD2 DNA for cloning by gap repair, the bolded sequences are the NcoI and NotI sites. The dashed sequence should include approximately 24 nucleotides that anneal to the 5′ and 3′ ends of the target antigen DNA. Primers for amplifying and introducing alanine on the target antigen gene: These primers are designed specifically for each toxin domain and mutation to construct

Strains, Media, and Reagents

Pfu DNA Polymerase (Stratagene); Taq Polymerase (New England Biolabs (NEB)); Geneclean Turbo (MP Biomedicals, LLC, Cat. 1102-600); Custom DNA primers (many vendors); deoxynucleotide triphosphates (NEB); Nco I, Not I restriction enzymes (NEB); SV5 Antibody (Invitrogen); APC conjugated Fab-specific goat-anti-human (Fab)2 (Jackson Immuno Research); Fab preparation kit (Pierce product number 44985); saccharomyces cerevisiae EBY100 (Invitrogen); 1 M lithium acetate (LiAc); 0.1 M LiAc; 2 mg/mL single stranded DNA (ssDNA); polyethylene glycol (PEG) 3350 (50%); plasmid vector pYD2 as describe above; YPD medium (recipe: to 900 mL deionized H2O add: 10 g Yeast extract and 20 g Peptone. Optional: Add 17 g agar/L for plates, autoclave, cool to 55-60° C. Add 100 ml of 20% dextrose (filter sterilized). For liquid media preparation filter sterilize (0.22 um filter) after all components have been added and dissolved.) SD-CAA medium (recipe: to 900 mL deionized H2O add: 7 g Yeast Nitrogen base w/o amino acid, 10.19 g Na2HPO4.7H2O or 5.4 g Na2HPO4, 8.56 g NaH2PO4.H2O or 7.4 g NaH2PO4, and 5 g CAA (DIFCO) w/o Tryptophan or Ura. After all components dissolve, add 100 ml of 20% dextrose and 10 ml of 0.6% (100×) Leucine. Sterilize by filtering through 0.22 μm filter; SG-CAA medium (recipe: to 900 mL deionized H2O add: 7 g Yeast Nitrogen base w/o amino acid, 10.19 g Na2HPO4.7H2O or 5.4 g Na2HPO4, 8.56 g NaH2PO4.H2O or 7.4 g NaH2PO4, and 5 g CAA (DIFCO) w/o Tryptophan or Ura. After all components dissolve, add 100 ml of 20% galactose and 10 ml of 0.6% (100×) Leucine. Sterilize by filtering through 0.22 μm filter; Alexa-647 labeling kit (Molecular Probes); Alexa-488 labeling kit (Molecular Probes); PE conjugated anti-human Fc specific antibody (Jackson Immunoresearch); FACS Buffer: to 1 L 1× PBS add 5 g BSA (final 0.5%), 1 mL 1M MgCl2 (final 1 mM) and 0.5 mL 1 M CaCl2 (final 0.5 mM), sterile filter; FACS-ARIA Cell Sorter (BD Biosciences); Pure BoNToxins (USAMIIRID, Metabiologics); recombinant BoNToxins Fragments (UASAMIIRD)

Specificity and Crossreactivity Test.

5.0×105 scFv/yeast of an induced culture, from the scFv to test, were incubated with 100 uL of a 10 nM solution of the toxin to test. After 2 hours of incubation at 4° C., cells were washed with 500 uL of cold FACS buffer and resuspended in 200 uL of the detecting antibodies. Binding of BoNToxins to yeast-displayed scFv was detected using a 1:500 dilution of 1 mg/ml mAb binding a non-overlapping epitope labeled with Alexa-647. Expression of yeast cells was detected using and anti-SV5 tag IgG labeled with Alexa-488, 1:300 dilution.

This test was repeated for each available subtype of the corresponding BoNToxin. To test for nonspecific binding, cells were also incubated with the detection antibodies only. No toxin was used for this test. Recognition of each toxin was estimated by measuring yeast fluorescence in a FACS ARIA cell sorter, using the FITC and APC channels.

Measurement of Yeast-Displayed scFv Affinity for the Corresponding BoNToxin

Quantitative equilibrium binding was determined using yeast-displayed scFv and flow cytometry as described previously (Boder et al. Proc. Natl. Acad. Sci. USA 2000, 97:10701-10705) In general, six to eight different concentrations of pure BoNT/A were used spanning a range of concentrations from ten times above to ten times below the KD. Incubation volumes and number of yeast stained were chosen to keep the number of antigen molecules in fivefold excess above the number of scFv, assuming 5.0×105 scFv/yeast. Incubation times were chosen based on anticipated times to equilibrium calculated using approximations of the anticipated association rate constant (kon) and dissociation rate constant (koff). For the higher affinity scFv, this was as long as 18-24 h. Binding of BoNToxins to yeast-displayed scFv was detected as for specificity test. Each KD was determined in triplicate, three separate inductions and measurements. To measure the antibody-toxin affinity constant (KD) within the surface display context, only the scFv displaying yeast (SV5 binding) were included in the analysis by co-staining with SV5-Alexa-488. Affinity determination was performed for each one of the toxin subtypes recognized by the scFv being analyzed.

Binding Domain Test

Binding to recombinant purified toxin domains was performed using the same protocol as for the whole toxin, on the yeast-displayed scFv. When recombinant toxin was not available the desired toxin domains were cloned and displayed in EBY 100 yeast cells (Levy et al. supra). Only the IgG format, or free scFv can be used in this test. Binding to the yeast displayed toxin domain was performed by incubating 5.0×105 induced cells with 100 uL of the IgG-Alexa-647 to test. 1:500 dilution of the labeled IgG plus 1:300 dilution of SV5-488 were used. Cells were incubated at 4° C. for 30 min and after washing with cold FACS buffer, fluorescence was measured.

Construction of Yeast-Displayed BoNToxins Fragments, Wild Type or Alanine Mutants.

Genes encoding the desired botulinum neurotoxin fragments (Lc, Ha or HC) in pYD2, wt fragments or alanine mutated, were constructed by using PCR and gap repair. Primers For and Rev were designed to amplify the synthetic gene fragment, adding the restriction sites NcoI and NotI to the 5′ and 3′ ends, respectively. The pYD2 vector was digested with NcoI and NotI, and together with the gel purified per fragment used to transform EBY100 by gap repair (Orr-Weaver et al. Proc Natl Acad Sci USA 1983, 80: 4417-21; Gietz et al. Yeast 1983, 7: 253-63). Clones containing the correct insert were confirmed by DNA sequencing to yield pYD2/Toxin fragment.

For each Alanine mutant constructed a pair of specific oligos were designed and used in combination with the pYD2 standard oligos pYDFor and pYDRev to construct the mutated fragment by per. Transformation was performed, by gap repair, using the double digested plasmid and the two per products, all gel purified. After induction, mutated fragments had display levels resulting in at least a 1.5 log shift when stained with SV5-Alexa-488, comparable to the levels of wild-type fragment.

Generation of IgG from scFv

Each scFv of interest was converted into full length IgG. Briefly, genes encoding scFv VH and VL were amplified using PCR from their respective pYD2 vectors with a designed primer pair and cloned into plasmid N5KG1. Clones containing the correct genes were identified by DNA sequencing and vector DNA was used to transfect CHO DG44 cells by electroporation. Stable cell lines were established by selection in G418 and expanded into 1-liter spinner flasks. Supernatant containing IgG was collected, concentrated by ultrafiltration and purified on Protein G (Pharmacia). IgG purity was assessed by native and denaturing SDS-PAGE and concentration determined by absorbance at 280 nm.

Epitope Sharing Test

Each scFv to test was stained with the corresponding toxin, as described above for the specificity test. Detection was done, separately, by each existing IgGs that binds to the toxin used. Fluorescence measurements were used as indicator of the spatial proximity of both epitopes, for the scFv and for the IgG. No signal meant that epitopes were too close and both antibodies could not bind simultaneously. The contrary was assumed for a strong signal, epitopes were well separated permitting binding of both antibodies.

After full characterization of each scFv an analysis of the toxin sequences was performed but only for the determined binding fragment. For an antibody that had been shown to bind the Hn domain, or translocation domain, toxin alignment was done only for the sequences regions corresponding to the Hn domains of the toxin subtypes.

Protein Sequence Alignment

The protein analysis and multiple sequence alignment were performed using the ClustalW application from the Mac Vector Software (Mac Vector, INC.). All known sequences for the different subtypes of the BoNT serotype were used to find the areas that presented antigen differences. The differential binding to each subtype was used as an indicator of a probable binding site, normally several regions were found. Each region was located in the 3D structure of the whole toxin assessing the level of exposure and conservation of the amino acids in the area.

The aim was to locate areas that could explain the binding pattern of the antibody and then test by alanine scanning the energetic importance of each site. Mutated amino acids that led to loss of binding were considered as energetically favorable contacts. Further single yeast displayed alanine mutants of the putative epitope site are constructed and antibody binding determined in order to identify the epitope.

Measurement of the Affinity of Fab Fragments for Yeast Displayed Wild Type and Alanine Mutants Toxin Domains.

By measuring the affinity of antibody for the alanine mutants compared to the wild type yeast displayed antigen, it is possible to determine the energetic contribution of the amino acid side chain to antibody binding.

The dissociation equilibrium constants (KD) of the Fab fragment of the antibody to test for wild type and alanine mutants of yeast displayed toxin fragments were measured by flow cytometry on a FACS ARIA cell sorter. First, EBY100 yeast cultures harboring the pYD2/toxin domain, wild type or the pYD2/alanine mutant plasmids were grown and induced as described above for yeast-displayed scFv. Aliquots of 1×105 induced yeast cells (0.005 OD600 ml−1) were washed in FACS buffer and incubated with dilutions (according to the IgG KD) of the Fab fragments such that the KD would be spanned by at least fivefold, where possible. Incubation volumes were chosen to ensure that a tenfold molar excess of the antibody (ligand) over the displayed moiety (HC, HN or LC) would be maintained. For this purpose, it was assumed that 105 copies of the toxin domain were displayed on the yeast surface. Incubation with the Fab was allowed to proceed for 4 h at 23° C. Cells were then washed in FACS buffer and resuspended in allophycocyanin-conjugated Fab-specific goat-anti-human F(ab)¢c2 at 1:1000 dilution in FACS buffer. To accurately determine the KD of the Fab fragments within the surface display context, we included only protein displaying yeast in the analysis by staining with SV5 (Alexa-488) mAb (VanAntwerp et al. Biotechnol. Prog. 2000, 16:31-37)

Calculation of the Change in Free Energy of Binding to the Wild Type and Alanine Mutants.

For each alanine mutation, the change of free energy (DDGmut-wt) between the alanine (Ala) mutant relative to that of the wild type (wt) was calculated using the following standard formula and using the previously measured KD constants:

ΔΔGmut-wt=RT ln(KDmut/KDwt)

These calculations provide a measure of the energetic contribution of each one of the alanine substituted amino acid residues, for the mAb, therefore indicating the position of its functional epitope. Fab was used as it is monovalent, compared to the bivalent IgG. This eliminates the avidity effect that can occur with bivalent binding which can lead to falsely high (and inaccurate) affinities (Levy et al. supra).

The numbering system used for the residues of toxin in the table above is based on Lacy et al. (1999) J. Mol. Biol. 291:1091-1104. Residues that are bolded and underlined were found to have important contributions energetically and eliminations of the residues often lead to total loss of binding the corresponding antibodies.

It is understood that the examples and embodiments described herein are for illustrative purposes only and that various modifications or changes in light thereof will be suggested to persons skilled in the art and are to be included within the spirit and purview of this application and scope of the appended claims. All publications, patents, and patent applications cited herein are hereby incorporated by reference in their entirety for all purposes.

While the subject antibody, method, and composition have been particularly shown and described with references to preferred embodiments thereof, it will be understood by those skilled in the art that various changes in form and details may be made therein without departing from the scope of the invention encompassed by the appended claims.

