# DESCRIPTION

## TECHNICAL FIELD

The present invention relates to a technology of amplifying a gene created by gene rearrangement from a biological sample without applying a bias, a system for analyzing the resulting genetic information, and therapy and diagnosis thereof.

## BACKGROUND ART

A biological defense mechanism utilizing an immune system is heavily dependent on specific immunity provided mainly by T cells and B cells. T cells and B cells do not react to their own cells or molecules and are capable of specifically recognizing and attacking exogenous pathogens such as viruses or bacteria. For this reason, T cells and B cells have a mechanism capable of recognizing and distinguishing autoantigens as well as various antigens derived from other organisms by a receptor molecule expressed on a cell surface. T cell receptor (TCR) and B cell receptor (BCR) act as an antigen receptor in T cells and B cells, respectively. An intracellular signal is transmitted by a stimulation from such antigen receptors. Production of inflammatory cytokines, chemokines or the like are enhanced, cell proliferation increased, and various immune responses are initiated. TCRs recognize a peptide bound to a peptide binding groove of a major histocompatibility complex (MHC) expressed on an antigen-presenting cell (peptide-MHC complex, pMHC) to distinguish self from non-self and recognize an antigen peptide (Non Patent Literature 1). TCRs are heterodimer receptor molecules consisting of two TCR polypeptide chains. There are αβ TCRs expressed by normal T cells and γδ TCRs with a special function. α and β chain TCR molecules form a complex with a plurality of CD3 molecules (CD3ζ chain, CD3ε chain, CD3γ chain, and CD3δ chain), transmit an intracellular signal after antigen recognition, and initiate various immune responses. With a viral infection, an endogenous antigen such as a cancer antigen derived from a cancer cell or a viral antigen proliferated in a cell is presented as an antigen peptide on an MHC class I molecule. Further, an antigen derived from an exogenous microorganism is taken up and processed by an antigen-presenting cell by endocytosis, and then presented on an MHC class II molecule. Such antigens are recognized by TCRs expressed by each of CD8+ T cell and CD4+ T cell. It is also known that a costimulatory molecule such as a CD28, ICOS, or OX40 molecule is important for stimulation via a TCR molecule.

A TCR gene consists of numerous V regions (variable region, V), J regions (joining region, J), D regions (diversity region, D), and constant regions, C regions (C) encoded by different regions in the genome. In T cell differentiation process, such gene fragments are genetically rearranged in various combinations. α chain and γ chain TCRs express genes consisting of V-J-C and β chain and δ chain TCRs express genes consisting of V-D-J-C. Currently, database of the IMGT (the International ImMuno GeneTics project) has 43 types of functional α chain TCR V gene fragments (TRAV), 50 types of TCR J gene fragments (TRAJ), 40-42 types of functional β chain TCR V gene fragments (TRBV), 2 types of TCR D gene fragments (TRBD), types of TCR J gene fragments (TRBJ), 4-6 types of functional γ chain V gene fragments (TRGV), 5 types of TCR J gene fragments (TRGJ), 3 types of functional 5 chain V gene fragments (TRDV), 3 types of TCR D gene fragments (TRDD), and 4 types of TCR J gene fragments (TRDJ) (Non Patent Literature 2). Diversity is created by rearrangement of such gene fragments. In addition, insertion or deletion of one or more bases between V and D or D and J gene fragments leads to the formation of a random amino acid sequence to create a more diverse TCR gene sequence.

A region where a TCR molecule directly binds to a pMHC complex surface (TCR footprint) is composed of three diverse complementarity determining regions (CDR) within the V region, CDR1, CDR2, and CDR3 regions. The CDR3 region in particular comprises a part of a V region, a part of J region and a V-D-J region formed by a random sequence, forming the most diverse antigen recognition site. Meanwhile, the other regions are called FRs (framework region) serving the role of forming a backbone structure of a TCR molecule. In a differentiation and maturation process of a T cell in the thymus gland, a β chain TCR is genetically rearranged initially, and conjugates with a pTα molecule to form a pre-TCR complex molecule. An α chain TCR is then rearranged to form an αβ, TCR molecule, and when a functional αβ, TCR is not formed, rearrangement occurs in the other α chain TCR gene allele. It is known that after undergoing positive/negative selection in the thymus gland, a TCR with a suitable affinity is selected to acquire antigen specificity (Non Patent Literature 3).

A BCR is known as an immunoglobulin (Ig). A membrane-bound Ig acts as an antigen receptor molecule as a BCR. A secretory protein thereof is secreted to the outside of a cell as an antibody. A large amount of antibodies is secreted from a terminally differentiated plasma cell and has functions to eliminate pathogens by binding to a pathogenic molecule such as a virus or bacteria or by a subsequent immune reaction such as a complement binding reaction. A BCR is expressed on a B cell surface. After binding to an antigen, the BCR transmits an intracellular signal to initiate various immune responses or cell proliferation. Diversity of amino acid sequences at an antigen-binding site is responsible for the specificity of a BCR. Sequences at an antigen-binding site greatly vary among BCR molecules and are called variable sections (V regions). Meanwhile, a sequence of a constant region (C region) is highly conserved among BCR molecules or antibody molecules. Such a region has an effector function of an antibody or a signaling function of a receptor.

A BCR and an antibody are the same except for the presence or absence of a membrane-binding domain. An Ig molecule consists of polypeptide chains, two heavy chains (H chains) and two light chains (L chains). In one Ig molecule, two H chains, or one H chain and one L chain, are bound by a disulfide bond. There are 5 different H chain classes (isotypes) called μ chain, α chain, γ chain, δ chain, and ε chain in Ig, which are called IgM, IgA, IgG, IgD, and IgE, respectively. It is known that functions and roles generally vary depending on the isotype, e.g., an antibody with a high level of specificity which is functional in biological defense is an IgG antibody, an IgA antibody is involved in mucosal immunity, and an IgE antibody is important in allergy, asthma, and atopic dermatitis. Furthermore, it is known that there are several types of subclasses in isotypes, such as IgG1, IgG2, IgG3, and IgG4. It is understood that there are two types of L chains, A chain (IgL) and K chain (IgK), which can bind to an H chain of any class, and there is no functional difference therebetween (Non Patent Literature 4).

As in TCR genes, BCR genes are formed by gene rearrangement that occurs in a somatic cell. A variable section is encoded in a few separate gene fragments in the genome, which induce somatic cell genetic recombination in the differentiation process of a cell. A genetic sequence of a variable section of an H chain consists of a C region (constant region, C) defining an isotype that is different from a D region, a J region, and a V region. Each gene fragment is separated in the genome, but is expressed as a series of V-D-J-C genes by gene rearrangement. The database of the IMGT has 38-44 types of functional IgH chain V gene fragments (IGHV), 23 types of D gene fragments (IGHD), 6 types of J gene fragments (IGHJ), 34 types of functional IgK chain V gene fragments (IGKV), 5 types of J gene fragments (IGKJ), 29-30 types of functional IgL chain V gene fragments (IGLV), and 5 types of J gene fragments (IGLJ). These gene fragments undergo gene rearrangement to ensure diversity of BCRs. Furthermore, highly diverse CDR3 regions are formed by a random insertion or deletion in an amino acid sequence as in TCRs (Non Patent Literature 2).

In a differentiation and maturation process of a B cell, IgM is initially produced by an immature B cell. A naive B cell that has not been exposed to an antigen co-expresses IgM and IgD. After being stimulated and activated by a stimulation of an antigen, a class switch (isotype switch) that converts a C region of IgM, Cμ, with a C region sequence of IgG, Cγ, occurs while the sequence of a variable section remains the same. Similarly, Cμ is converted to C region of IgA (Ca) or C region of IgE (Cε) to produce IgA or IgG. With such a class switch recombination, the type of antibody required for eliminating a pathogen is produced where it is required. Furthermore, in the proliferation process of a B cell that has undergone a class switch, a mutation occurs at a high frequency in the variable section of an IgG, IgA or IgE region (somatic hypermutation). As a result, a B cell that has acquired a higher level of specificity to an antigen is further stimulated and proliferated, such that an antibody producing B cell with a higher level of specificity is selected through this process (affinity maturation) (Non Patent Literature 5).

A T cell or a B cell produces one type of TCR or BCR with a high level of specificity to a specific antigen. With numerous antigen specific T cells and B cells in a living organism, a diverse TCR repertoire or BCR repertoire can be formed to effectively function as a defense mechanism against various pathogens. Thus, analysis of a TCR or BCR repertoire, which is an important indicator of specificity or diversity of immune cells, is a useful analytical tool for analyzing monoclonality or immune disorder. If T cells or B cells proliferate in response to an antigen, the ratio of a specific TCR or BCR gene is observed to increase in a diverse repertoire (increased clonality). An attempt has been made to detect development of tumor in lymphoid cells expressing TCRs or BCRs in terms of increase in clonality by TCR or BCR repertoire analysis (Non Patent Literature 6). Further, it is reported that the usage frequency of a specific Vβ, chain increases when exposed to a molecule that selectively stimulates a TCR having a specific Vβ, chain such as super antigens (Non Patent Literature 7). In order to investigate antigen specific immune responses, it is frequently used in analysis of refractory autoimmune diseases that are induced by immune disorders such as rheumatoid arthritis, systemic lupus erythematosus, Sjögren's syndrome, and idiopathic thrombocytopenic purpura, and the usefulness thereof has been demonstrated.

Conventional TCR repertoire analysis is an analytic method for examining how many individual V chains are used by T cells in a sample. One of the methods is a method of analyzing the ratio of T cells expressing individual Vβ chains by using a specific Vβ, chain specific antibody with flow cytometry (FACS analysis). Since a relatively large number of cells are required, this technology is useful for analyzing peripheral blood comprising many lymphocytes, but cannot be adapted to a tissue material sample. Further, since an antibody that is compatible with all of the V chains is still not available today, comprehensive analysis is not possible.

In addition thereto, TCR repertoire analysis using molecular biological technology has been designed based on information on TCR genes obtained from the human genomic sequence. This is a method of extracting an RNA from a cell sample to synthesize a complementary DNA and then amplifying and quantifying a TCR gene by PCR. It has been conventional to use a method of designing numerous individual TCR V chain specific primers to separately quantify by real-time PCR or the like, or a method of simultaneously amplifying such specific primers (Multiple PCR). However, even in quantification using an endogenous control for each V chain, accurate analysis is not possible when a large number of primers are used. Furthermore, multiple PCR has a disadvantage in that a difference in efficiency of amplification among primers results in a bias in PCR amplification. In order to overcome such a disadvantage of multiple PCR, Tsuruta et al reported Adaptor-ligation PCR, which adds an adaptor to the 5′ terminal of a double stranded complementary DNA of a TCR gene and then amplifies all γδ TCR genes with a common adaptor primer and a C region specific primer (Non Patent Literature 8). Furthermore, methods applied to amplification of αβ TCR genes for quantification with oligoprobes specific to individual V chains were developed, i.e., Reverse dot blot (Non Patent Literature 9) and Microplate hybridization assay (Non Patent Literature 10). They are excellent methods for amplifying a TCR gene without introducing a bias. However, hardly any information other than the usage frequency of V chains can be obtained. Base sequence information or the like of a CDR3 region, the J chain, D chain or antigen recognition site, required subsequent cloning of a complement chain DNA of a TCR gene and determination of the base sequence.

In recent years, rapidly advancing next generation sequence analysis techniques have enabled large scale base sequence determination of genes. By amplifying a TCR gene from a human sample by PCR and using a next generation sequence analysis technique, it is possible to materialize a next generation TCR repertoire analysis method for obtaining and analyzing more detailed clone level genetic information from TCR repertoire analysis for obtaining information that has been small scale and limited to V chain usage frequency or the like. In this context, few next generation TCR repertoire analysis methods have been developed (Patent Literatures 1 and 2) while other attempts have also been made (Patent Literatures 3-11).

## SUMMARY OF INVENTION

### Solution to Problem

The present invention is an invention related to an analysis method and analysis system applied to (1) a technique for amplifying a TCR or BCR genetic sequence produced by gene rearrangement from multiple gene fragments in a genome without applying a bias (unbiased gene amplification technique), and (2) a technology for determining the base sequence of a TCR or BCR gene amplified by the unbiased gene amplification technique in a large scale by a next generation sequencing method, assigning V, D, J, and C regions, and analyzing a TCR repertoire or BCR repertoire.

Diverse genetic sequences are created by gene rearrangement of multiple gene fragments of V, D, J, and C regions on a genome for TCRs or BCRs. A technology of producing numerous primers specific to numerous V or J regions that are present and amplifying in the same or separate reaction solution is widely utilized to determine a base sequence of a TCR or BCR gene by a next generation sequencing technique. However, the difference in amplification efficiency among primers would be a critical issue in PCR amplification which exponentially amplifies a small amount of gene. Further, it is necessary that primers set to V and J regions are compatible with all known allelic sequences. A point mutation is introduced at a high frequency (up to about 20%) in a variable section region of IgG, IgA or IgE by a somatic hypermutation mechanism for a BCR gene. Thus, if a 20 base primer is set, about 4 bases would have a mismatch. Hence, it is difficult to materialize uniform gene amplification with a conventional method. That is, known methods of designing a V chain specific primer based on a genomic sequence cannot avoid a mismatch with the actual BCR genetic sequence such that quantitative gene amplification is not guaranteed. Furthermore, a BCR has an isotype and a subclass defined by a C region sequence. It is necessary to develop a quantification method for each isotype or subclass utilizing a difference in the base sequence among isotypes or subclasses. In order to overcome the disadvantage of a technology using a V chain specific primer currently in use, the inventors have completed a method of amplifying a TCR or BCR gene including all isotype and subtype genes with a set of primers consisting of one type of forward primer and one type of reverse primer without changing the frequency of presence and determining a base sequence in a large scale by using a next generation sequencing.

Focus was placed on the genetic structure of a TCR or BCR gene. An adaptor sequence is added, without setting a primer to highly diverse V regions, to a 5′ terminal thereof to amplify a gene comprising all V regions.

Such an adaptor can have any length or sequence in a base sequence. About 20 base pairs are optimal, but a sequence from 10 bases to 100 bases can be used.

An adaptor added to the 3′ terminal is removed with a restriction enzyme. In addition, all TCR or BCR genes are amplified by amplifying with a reverse primer specific to a C region which has a common sequence with an adaptor primer with the same sequence as a 20 base pair adaptor.

A complementary strand DNA is synthesized with a reverse transcriptase from a TCR or BCR gene messenger RNA and then a double stranded complementary DNA is synthesized. A double stranded complementary DNA comprising V regions with different lengths is synthesized by a reverse transcription reaction or a double strand synthesizing reaction. Adaptors consisting of 20 base pairs and 10 base pairs are added to the 5′ terminal section of such genes by a DNA ligase reaction.

The genes can be amplified by setting a reverse primer in a C region of a heavy chain of μ chain, α chain, δ chain, γ chain or ε chain or a light chain of κ chain or λ chain for BCRs and α chain, β chain, γ chain or δ chain for TCRs.

As a reverse primer set in a C region, a primer is set which matches the sequence of each of Cβ, Cα, Cγ and Cδ for TCRs and the sequence of each of Cμ, Cα, Cδ, Cγ, Cε, Cκ, and Cλ for BCRs and has a mismatch to an extent where other C region sequences are not primed.

A reverse primer of a C region is optimally made while considering the base sequence, base composition, DNA melting temperature (Tm), or presence of a self-complementary sequence, such that amplification with an adaptor primer is possible.

Each BCR gene IgG subtype (γ1, γ2, γ3, and γ4) and IgA subtype (α1 and α2) can be amplified with the same primer to determine the subtype by determining the base sequence.

A primer can be set in a region other than the base sequence that is different among allelic sequences in a C region sequence to uniformly amplify all alleles.

A plurality of stages of nested PCR are performed in order to enhance the specificity of an amplification reaction.

The length (number of bases) of a primer candidate sequence is not particularly limited for a sequence not comprising a sequence that is different among allelic sequences for each primer. However, the number of bases is 10-100, preferably 15-50, and more preferably 20-30. Thus, the present invention also provides the following.

### <In Silico>

In one aspect, the present invention relates to a technology for analyzing a TCR or BCR repertoire based on a group of expressed TCRs or BCR genetic sequences derived from a biological sample.

The present invention is not dependent on the model of sequencer for any V(-D)-J-C series nucleic acid sequence. Classification itself is possible even without unbiasness.

The input can be either a plus strand or complementary strand.

For classification of nucleic acid sequences, it is common to set a reference database that has accumulated standard sequences serving as the baseline of classification (hereinafter, referred to as reference sequence) and assign each nucleic acid sequence to one of the reference sequences by a technology for homology search. However, it is necessary in this case to prepare an enormous number of reference sequences from combining each region of V, D, and J, which is not practical. A technology of setting a reference database for each of V, D, and J is conceivable. However, the difference from a reference sequence would be large due to a random mutation in V. Further, D and J have a short region. Thus, the possibility of oversight cannot be ignored for a common homology search technology. A technology of translating the entire nucleic acid sequence of a subject of analysis into an amino acid sequence and classifying the sequence by materials is conceivable. However, such a technology would be vulnerable especially to sequencing error from insertion/deletion and the relationship with previously-reported gene names and alleles would be unknown, such that it would be difficult to use known information.

The reference database used in the present invention is prepared for each of V, D, and J (and C for BCR) gene regions. Typically, a nucleic acid sequence data set is used for each allele or each region published by the IMGT, but is not limited thereto. Any data set with a unique ID assigned to each sequence can be used.

For the input sequence set used in the present invention, an adaptor sequence or low quality region is generally trimmed in advance and only a sequence with an sufficient length for analysis is extracted to construct a high quality set. This step is not necessarily required, but is used in a preferred embodiment. This is because, even without such processing, an LQ sequence would simply be “unclassifiable”.

The input sequence set used in the present invention searches for homology with a reference database for each gene region and records an alignment with the closest reference allele and the sequence thereof. In this regard, an algorithm with high tolerance for a mismatch except for C is used for homology search. For instance, when a common homology search program such as BLAST is used, setting such as shortening of the window size, reduction in mismatch penalty, or reduction in gap penalty is set for each region.

The closest reference allele is selected by using a homology score, alignment length, kernel length (length of consecutively matching base sequence) and number of matching bases as indicators applied in accordance with a defined order or priority.

For the input sequence with determined V and J used in the present invention, a CDR3 sequence is extracted with the front of CDR3 on reference V and end of CDR3 on reference J as guides. This is translated into an amino acid sequence for use in classification of a D region. When a reference database of a D region is prepared, a combination of results of homology search and results of amino acid sequence translation is used as a classification result.

In view of the above, each allele of V, D and J (and C for BCR) is assigned for each sequence in an input set. The frequency of appearance by each of V, D and J (and C for BCR) or frequency of appearance of a combination thereof in the entire input set is subsequently calculated to derive a TCR or BCR repertoire. The frequency of appearance is calculated in a unit of allele or unit of gene name depending on the precision required in classification. The latter is possible by translating each allele to a gene name. Thus, the present invention also provides the following.

<1> A method of analyzing a TCR or BCR repertoire, comprising the following steps:

(1) providing a reference database for each gene region comprising at least one of a V region, a D region, a J region and optionally a C region;

(2) providing an input sequence set which is optionally trimmed and optionally extracted to have a suitable length;

(3) searching for homology of the input sequence set with the reference database for the each gene region and recording an alignment with an approximate reference allele and/or a sequence of the reference allele;

(4) assigning the V region and the J region for the input sequence set and extracting a nucleic acid sequence of the D region based on a result of assigning;

(5) translating the nucleic acid sequence of the D region into an amino acid sequence and classifying the D region by utilizing the amino acid sequence; and

(6) calculating a frequency of appearance for each of the V region, the D region, and the J region and optionally the C region or a frequency of appearance of a combination thereof based on the classifying in (5) to derive the TCR or BCR repertoire.

<2> The method of item <1>, wherein the gene region comprises all of the V region, the D region, the J region and optionally the C region.

<3> The method of any one of items <1>-<2>, wherein the reference database is a database with a unique ID assigned to each sequence.

<4> The method of any one of items <1>-<3>, wherein the input sequence set is an unbiased sequence set.

<5> The method of any one of items <1>-<4>, wherein the sequence set is trimmed.

<6> The method of any one of items <1>-<5>, wherein the trimming is accomplished by the steps of: deleting low quality regions from both ends of a read; deleting a region matching 10 bp or more with an adaptor sequence from the both ends of the read; and using the read as a high quality read in analysis when a remaining length is 200 bp or more (TCR) or 300 bp or more (BCR).

<7> The method of item <6>, wherein the low quality refers to a 7 bp moving average of QV value less than 30.

<8> The method of any one of items <1>-<7>, wherein the approximate sequence is the closest sequence.

<9> The method of any one of items <1>-<8>, wherein the approximate sequence is determined by a ranking of 1. number of matching bases, 2. kernel length, 3. score, and 4. alignment length.

<10> The method of any one of items <1>-<9>, wherein the homology search is conducted under a condition tolerating random mutations to be scattered throughout.

<11> The method of any one of items <1>-<10>, wherein the homology search comprises at least one condition from (1) shortening of a window size, (2) reduction in a mismatch penalty, (3) reduction in a gap penalty, and (4) a top priority ranking of an indicator is a number of matching bases, compared to a default condition.

<12> The method of any one of items <1>-<11>, wherein the homology search is carried out under the following conditions in BLAST or FASTA:

V mismatch penalty=−1, shortest alignment length=30, and shortest kernel length=15;

D word length=7 (for BLAST) or K-tup=3 (for FASTA), mismatch penalty=−1, gap penalty=0, shortest alignment length=11, and shortest kernel length=8;

J mismatch penalty=−1, shortest hit length=18, and shortest kernel length=10; and

C shortest hit length=30 and shortest kernel length=15.

<13> The method of any one of items <1>-<12>, wherein the D region is classified by a frequency of appearance of the amino acid sequence.

<14> The method of any one of items <1>-<13>, wherein a combination of a result of search for homology with the nucleic acid sequence of CDR3 and a result of amino acid sequence translation is used as a classification result when there is a reference database for the D region in the step (5).

<15> The method of any one of items <1>-<14>, wherein only the frequency of appearance of the amino acid sequence is used for classification when there is no reference database for the D region in the step (5).

<16> The method of any one of items <1>-<15>, wherein the frequency of appearance is counted in a unit of a gene name and/or a unit of an allele.

<17> The method of any one of items <1>-<16>, wherein the step (4) comprises the step of assigning the V region and the J region for the input sequence set and extracting a CDR3 sequence, with the front of CDR3 on a reference V region and end of CDR3 on reference J as guides.

<18> The method of any one of items <1>-<17>, wherein the step (5) comprises translating the nucleic acid sequence of the CDR3 into an amino acid sequence and classifying a D region by using the amino acid sequence.

<19> A system for analyzing a TCR or BCR repertoire, wherein the system comprises:

(1) means for providing a reference database for each gene region comprising at least one of a V region, a D region, a J region and optionally a C region;

(2) means for providing an input sequence set which is optionally trimmed and optionally extracted to have a suitable length;

(3) means for searching for homology of the input sequence set with the reference database for the each gene region and recording an alignment with an approximate reference allele and/or a sequence of the reference allele;

(4) means for assigning the V region and the J region for the input sequence set and extracting a nucleic acid sequence of the D region based on a result of assigning;

(5) means for translating the nucleic acid sequence of the D region into an amino acid sequence and classifying the D region by utilizing the amino acid sequence; and

(6) means for calculating a frequency of appearance for each of the V region, the D region, and the J region and optionally the C region or a frequency of appearance of a combination thereof in the input sequence set to derive the TCR or BCR repertoire.

<19A> The system of item <19> having one or more features of any one of items <1>-<18>.

<20> A computer program for having a computer execute processing of a method of analyzing a TCR or BCR repertoire, the method comprising the following steps:

(1) providing a reference database for each gene region comprising at least one of a V region, a D region, a J region and optionally a C region;

(2) providing an input sequence set which is optionally trimmed and optionally extracted to have a suitable length;

(3) searching for homology of the input sequence set with the reference database for the each gene region and recording an alignment with an approximate reference allele and/or a sequence of the reference allele;

(4) assigning the V region and the J region for the input sequence set and extracting a nucleic acid sequence of the D region based on a result of assigning;

(5) translating the nucleic acid sequence of the D region into an amino acid sequence and classifying the D region by utilizing the amino acid sequence; and

(6) calculating a frequency of appearance for each of the V region, the D region, and the J region and optionally the C region or a frequency of appearance of a combination thereof in the input sequence set to derive the TCR or BCR repertoire.

<20A> The program of item <20> having one or more features of any one of items <1>-<18>.

<21> A recording medium for storing a computer program for having a computer execute processing of a method of analyzing a TCR or BCR repertoire, the method comprising the following steps:

(1) providing a reference database for each gene region comprising at least one of a V region, a D region, a J region and optionally a C region;

(2) providing an input sequence set which is optionally trimmed and optionally extracted to have a suitable length;

(3) searching for homology of the input sequence set with the reference database for the each gene region and recording an alignment with an approximate reference allele and/or a sequence of the reference allele;

(4) assigning the V region and the J region for the input sequence set and extracting a nucleic acid sequence of the D region based on a result of assigning;

(5) translating the nucleic acid sequence of the D region into an amino acid sequence and classifying the D region by utilizing the amino acid sequence; and

(6) calculating a frequency of appearance for each of the V region, the D region, and the J region and optionally the C region or a frequency of appearance of a combination thereof in the input sequence set to derive the TCR or BCR repertoire.

<21A> The recording medium of item <21> having one or more features of any one of items <1>-<18>.

<Wet>

In another aspect, the present invention is (1) a technology for uniformly amplifying a TCR or BCR genetic sequence produced by gene rearrangement from a plurality of gene fragments in a genome without applying a bias (unbiased gene amplification technology), and (2) a technology for determining the base sequence of a TCR or BCR gene amplified by the unbiased gene amplification technology in a large scale by a next generation sequencing method, assigning V, D, J, and C regions, and analyzing a TCR repertoire or BCR repertoire.

Diverse genetic sequences are created by gene rearrangement of a plurality of gene fragments of V, D, J, and C regions on a genome for TCRs or BCRs. A technology of producing a large number of primers specific to many V or J regions that are present and amplifying in the same reaction solution or separate reaction solutions is widely utilized to determine a base sequence of a TCR or BCR gene by a next generation sequencing technique. However, a difference in amplification efficiency among primers would be a critical issue in PCR amplification which exponentially amplifies a small amount of gene. Further, it is necessary that primers set to V and J regions are compatible with all known allelic sequences. A point mutation is introduced at a high frequency (up to about 20%) in a variable section region of IgG, IgA or IgE by a somatic hypermutation mechanism for a BCR gene. Thus, if a 20 base primer is set, about 4 bases would have a mismatch. Hence, it is difficult to materialize uniform gene amplification with a conventional method. That is, known methods of designing a V chain specific primer based on a genomic sequence cannot avoid a mismatch with the actual BCR genetic sequence such that quantitative gene amplification is not guaranteed. Furthermore, a BCR has an isotype and a subclass defined by a C region sequence. It is necessary to develop a quantification method for each isotype or subclass utilizing a difference in the base sequence among isotypes or subclasses. In order to overcome the disadvantage of a technology using a V chain specific primer current in use, the inventors have completed a method of amplifying a TCR or BCR gene including all isotype and subtype genes with a set of primers consisting of one type of forward primer and one type of reverse primer without changing the frequency of presence and determining a base sequence in a large scale by using a next generation sequencing.

Focus was placed on the genetic structure of a TCR or BCR gene. An adaptor sequence is added, without setting a primer to highly diverse V regions, to a 5′ terminal thereof to amplify a gene comprising all V regions.

Such an adaptor can have any length or sequence in a base sequence. About 20 base pairs are optimal, but a sequence from 10 bases to 100 bases can be used.

An adaptor added to the 3′ terminal is removed with a restriction enzyme. In addition, all TCR or BCR genes are amplified by amplifying with a reverse primer specific to a C region which has a common sequence with an adaptor primer with the same sequence as a 20 base pair adaptor.

A complementary strand DNA is synthesized with a reverse transcriptase from a TCR or BCR gene messenger RNA and then a double stranded complementary DNA is synthesized. A double stranded complementary DNA comprising V regions with different lengths is synthesized by a reverse transcription reaction or a double strand synthesizing reaction. Adaptors consisting of 20 base pairs and 10 base pairs are added to the 5′ terminal section of such genes by a DNA ligase reaction.

The genes can be amplified by setting a reverse primer in a C region of a heavy chain of μ chain, α chain, δ chain, γ chain or ε chain or a light chain of κ chain or λ chain for BCRs and α chain, β chain, γ chain or δ chain for TCRs.

As a reverse primer set in a C region, a primer is set which matches the sequence of each of Cβ, Cα, Cγ and Cδ for TCRs and the sequence of each of Cμ, Cα, Cδ, Cγ, Cε, Cκ, and Cλ for BCRs and has a mismatch to an extent where other C region sequences are not primed.

A reverse primer of a C region is optimally made while considering the base sequence, base composition, DNA melting temperature (Tm), or presence of a self-complementary sequence, such that amplification with an adaptor primer is possible.

Each BCR gene IgG subtype (γ1, γ2, γ3, and γ4) and IgA subtype (α1 and α2) can be amplified with the same primer to determine the subtype by determining the base sequence.

A primer can be set in a region other than the base sequence that is different among allelic sequences in a C region sequence to uniformly amplify all alleles.

A plurality of stages of nested PCR are performed in order to enhance the specificity of an amplification reaction.

The length (number of bases) of a primer candidate sequence is not particularly limited for a sequence not comprising a sequence that is different among allelic sequences for each primer. However, the number of bases is 10-100, preferably 15-50, and more preferably 20-30. Thus, the present invention also provides the following.

<A1> A method of preparing a sample for quantitative analysis of a repertoire of a variable region of a T cell receptor (TCR) or B cell receptor (BCR) by genetic sequence analysis using a database, comprising the steps of:

(1) synthesizing a complementary DNA by using an RNA sample derived from a target cell as a template;

(2) synthesizing a double stranded complementary DNA by using the complementary DNA as a template;

(3) synthesizing an adaptor-added double stranded complementary DNA by adding a common adaptor primer sequence to the double stranded complementary DNA;

(4) performing a first PCR amplification reaction by using the adaptor-added double stranded complementary DNA, a common adaptor primer consisting of the common adaptor primer sequence, and a first TCR or BCR C region specific primer,

wherein the first TCR or BCR C region specific primer is designed to comprise a sequence that is sufficiently specific to a C region of interest of the TCR or BCR and not homologous with other genetic sequences, and comprise a mismatching base between subtypes downstream when amplified;

(5) performing a second PCR amplification reaction by using a PCR amplicon of (4), the common adaptor primer, and a second TCR or BCR C region specific primer, wherein the second TCR or BCR C region specific primer is designed to have a sequence that is a complete match with the TCR or BCR C region in a sequence downstream the sequence of the first TCR or BCR C region specific primer, but comprise a sequence that is not homologous with other genetic sequences, and comprise a mismatching base between subtypes downstream when amplified; and

(6) performing a third PCR amplification reaction by using a PCR amplicon of (5), an added common adaptor primer in which a nucleic acid sequence of the common adaptor primer comprises a first additional adaptor nucleic acid sequence, and an adaptor-added third TCR or BCR C region specific primer in which a second additional adaptor nucleic acid sequence and a molecule identification (MID Tag) sequence are added to a third TCR or BCR C region specific sequence; wherein

the third TCR or BCR C region specific primer is designed to have a sequence that is a complete match with the TCR or BCR C region in a sequence downstream to the sequence of the second TCR or BCR C region specific primer, but comprise a sequence that is not homologous with other genetic sequences, and comprise a mismatching base between subtypes downstream when amplified,

the first additional adaptor nucleic acid sequence is a sequence suitable for binding to a DNA capturing bead and for an emPCR reaction,

the second additional adaptor nucleic acid sequence is a sequence suitable for an emPCR reaction, and

the molecule identification (MID Tag) sequence is a sequence for imparting uniqueness such that an amplicon can be identified.

<A2> The method of item <A1>, wherein, for a BCR, the C region specific primer comprises a sequence that is a complete match with an isotype C region of interest selected from the group consisting of IgM, IgA, IgG, IgE and IgD and is not homologous with other C regions, and is a sequence that is a complete match with one of the subtypes IgG1, IgG2, IgG3 and IgG4 or one of IgA1 or IgA2 for IgA or IgG, or, for a TCR, the C region specific primer is a sequence that is a complete match with a C region of α chain of interest selected from the group consisting of α chain, β chain, γ chain and δ chain and is not homologous with other C regions.

<A3> The method of item <1> or <A2>, wherein a portion of a sequence that is a complete match with all C region allelic sequences of the same isotype in the database is selected for the C region specific primer.

<A4> The method of any one of items <A1>-<A3>, wherein the common adaptor primer is designed such that the primer is unlikely to have homodimer and intramolecular hairpin structures and can stably form a double strand, and designed not to be highly homologous with all TCR genetic sequences in the database and to have the same level of melting temperature (Tm) as the C region specific primer.

<A5> The method of item <A4>, wherein the common adaptor primer designed not to have homodimer and intramolecular hairpin structures and to have homology with other genes comprising a BCR or TCR is selected.

<A6> The method of item <A5>, wherein the common adaptor primer is P20EA (SEQ ID NO: 2) and/or P10EA (SEQ ID NO: 3).

<A7> The method of any one of items <A1>-<A6>, wherein the first, second and third TCR or BCR C region specific primers are each independently a primer for BCR repertoire analysis, the primer being selected to be a sequence that is a complete match with each isotype C region of IgM, IgG, IgA, IgD or IgE, and a complete match with subtypes for IgG and IgA, and not homologous with other sequences comprised in the database, and comprise a mismatching base between subtypes downstream of the primer, and

wherein the common adaptor primer sequence is designed such that the sequence has a base length suitable for amplification, is unlikely to have homodimer and intramolecular hairpin structures, and is able to stably form a double strand, and designed not to be highly homologous with all TCR genetic sequences in the database and to have the same level of Tm as the C region specific primer.

<A8> The method of any one of items <A1>-<A7>, wherein the first, second and third TCR or BCR C region specific primers are each independently a primer for TCR or BCR repertoire analysis, each primer being selected to be a sequence that is a complete match with 1 type of α chain (TRAC), 2 types of β chains (TRBCO1 and TRBCO2), 2 types of γ chains (TRGC1 and TRGC2), and one type of δ chain (TRDC1) and is not homologous with other sequences comprised in the database, and to comprise a mismatching base between subtypes downstream of the primer,

wherein the common adaptor primer sequence is designed such that the sequence has a base length suitable for amplification, is unlikely to have homodimer and intramolecular hairpin structures, and is able to stably form a double strand, and designed not to be highly homologous with all TCR genetic sequences in the database and to have the same level of Tm as the C region specific primer.

<A9> The method of any one of items <A1>-<A8>, wherein the third TCR or BCR C region specific primer is set in a region that is up to about 150 bases from the 5′ terminal side of a C region, and the first TCR or BCR C region specific primer and the second TCR or BCR C region specific primer are set between the 5′ terminal side of a C region to about 300 bases.

<A10> The method of any one of items <A1>-<A9>, wherein the first, second and third TCR or BCR C region specific primers are each independently for BCR quantitative analysis,

wherein separate specific primers are set to 5 types of isotype sequences, and the primers are designed to completely match a target sequence and ensure a mismatch of 5 bases or more for other isotypes and are designed to be a complete match with all subtypes such that one type of primer is compatible with each similar IgG subtype (IgG1, IgG2, IgG3 and IgG4) or IgA subtype (IgA1 and IgA2).

<A11> The method of any one of items <A1>-<A10>, wherein parameters in primer design are set to: a base sequence length of 18-22 bases; a melting temperature of 54-66° C.; and % GC (% guanine.cytosine content) of 40-65%.

<A12> The method of any one of items <A1>-<A11>, wherein parameters in primer design are set to: a base sequence length of 18-22 bases; a melting temperature of 54-66° C.; and % GC (% guanine.cytosine content) of 40-65%; a self-annealing score of 26; a self-end annealing score of 10; and a secondary structure score of 28.

<A13> The method of any one of items <A1>-<A12>, wherein sequences of the first, second and third TCR or BCR C region specific primers are determined under the following conditions:

1. a plurality of subtype sequences and/or allelic sequences are uploaded into a base sequence analysis software and aligned;

2. a primer designing software is used to search for a plurality of primers satisfying a parametric condition in a C region;

3. a primer in a region without a mismatching base in the aligned sequences in 1 is selected; and

4. the presence of a plurality of mismatching sequences for each subtype and/or allele downstream of the primer determined in 3 is confirmed, and if there is no such sequence, a primer is searched further upstream, which is further repeated as needed.

<A14> The method of any one of items <A1>-<A13>, wherein the first TCR or BCR C region specific primer is set in a position at bases 41-300 with a first base of a first codon of a C region sequence produced by splicing as a baseline, the second TCR or BCR C region specific primer is set in a position at bases 21-300 with said first base as the baseline, and the third TCR or BCR C region specific primer is set in a position within 150 bases or less with said first base as the baseline, and the positions comprise a mismatching site in a subtype and/or allele.

<A15> The method of any one of items <A1>-<A14>, wherein the first TCR or BCR C region specific primer has the following structure: CM1 (SEQ ID NO: 5), CA1 (SEQ ID NO: 8), CG1 (SEQ ID NO: 11), CD1 (SEQ ID NO: 14), CE1 (SEQ ID NO: 17), CA1 (SEQ ID NO: 35) or CB1 (SEQ ID NO: 37).

<A16> The method of any one of items <A1>-<A15>, wherein the second TCR or BCR C region specific primer has the following structure: CM2 (SEQ ID NO: 6), CA2 (SEQ ID NO: 9), CG2 (SEQ ID NO: 12), CD2 (SEQ ID NO: 15), CE2 (SEQ ID NO: 18), CA2 (SEQ ID NO: 35), or CB2 (SEQ ID NO: 37).

<A17> The method of any one of items <A1>-<A16>, wherein the third TCR or BCR C region specific primer has the following structure: CM3-GS (SEQ ID NO: 7), CA3-GS (SEQ ID NO: 10), CG3-GS (SEQ ID NO: 13), CD3-GS (SEQ ID NO: 16) or CE3-GS (SEQ ID NO: 19).

<A18> The method of any one of items <A1>-<A17>, wherein each of the TCR or BCR C region specific primers is provided in a set compatible with all TCR or BCR subclasses.

<A19> A method of performing gene analysis using a sample manufactured by the method of any one of items <A1>-<A18>.

<A20> The method of item <A19>, wherein the gene analysis is the quantitative analysis of a repertoire of a variable region of a T cell receptor (TCR) or a B cell receptor (BCR).

### <Analysis System>

<B1> A method of quantitatively analyzing a repertoire of a variable region of a T cell receptor (TCR) or a B cell receptor (BCR) of a subject by using a database, wherein the method comprises:

(1) providing a nucleic acid sample comprising a nucleic acid sequence of the T cell receptor (TCR) or the B cell receptor (BCR) which is amplified from the subject in an unbiased manner;

(2) determining the nucleic acid sequence comprised in the nucleic acid sample; and

(3) calculating a frequency of appearance of each gene or a combination thereof based on the determined nucleic acid sequence to derive a TCR or BCR repertoire of the subject.

<B2> The method of item <B1>, wherein the nucleic acid sample comprises nucleic acid sequences of a plurality of types of T cell receptors (TCR) or B cell receptors (BCR) and the step (2) determines the nucleic acid sequence by a single sequencing.

<B3> The method of item <B2>, wherein the single sequencing is characterized in that at least one of the sequences used as a primer in amplification from the nucleic acid sample into a sample for sequencing has the same sequence as a nucleic acid sequence encoding a C region or a complementary strand thereof.

<B4> The method of item <B2> or <B3>, wherein the single sequencing is characterized in being performed with a common adaptor primer.

<B5> The method of any one of items <B1>-<B4>, wherein the unbiased amplification is not V region specific amplification.

<B6> The method of any one of <B1>-<B5>, wherein the repertoire is the repertoire of a variable region of a BCR, and the nucleic acid sequence is a BCR nucleic acid sequence.

<B7> A method of analyzing a disease, disorder or condition of the subject based on the TCR or BCR repertoire derived based on any one of <B1>-<B6>,

<B8> The method of item <B7>, wherein the disease, disorder or condition of the subject is selected from the group consisting of hematological tumor and colorectal cancer.

<B9> A method of treating or preventing the disease, disorder or condition of the subject determined by the method of item <B7> or <B8>, comprising: quantitatively associating the disease, disorder or condition of the subject with the TCR or BCR repertoire; and selecting means for suitable treatment or prevention from the quantitative association.

<B10> The method of item <B9>, wherein the disease, disorder or condition of the subject is selected from the group consisting of hematological tumor and colorectal cancer.

<B11> A system for quantitatively analyzing a repertoire of a variable region of a T cell receptor (TCR) or a B cell receptor (BCR) of a subject by using a database, wherein the system comprises:

(1) a kit for providing a nucleic acid sample comprising a nucleic acid sequence of the T cell receptor (TCR) or the B cell receptor (BCR) which is amplified from the subject in an unbiased manner;

(2) an apparatus for determining the nucleic acid sequence comprised in the nucleic acid sample; and

(3) an apparatus for calculating a frequency of appearance of each gene or a combination thereof based on the determined nucleic acid sequence to derive a TCR or BCR repertoire of the subject.

<B12> The system of item <B11>, wherein the nucleic acid sample comprises nucleic acid sequences of a plurality of types of T cell receptors (TCR) or B cell receptors (BCR) and the step (2) determines the nucleic acid sequence by a single sequencing.

<B13> The system of item <B12>, wherein the single sequencing is characterized in that at least one of the sequences used as a primer in amplification from the nucleic acid sample to a sample for sequencing has the same sequence as a C region.

<B14> The system of item <B12> or <B13>, wherein the single sequencing is characterized in being performed with a common adaptor primer.

<B15> The system of any one of items <B11>-<B14>, wherein the unbiased amplification is not V region specific amplification.

<B16> The system of any one of items <B11>-<B15>, wherein the repertoire is the repertoire of a variable region of a BCR, and the nucleic acid sequence is a BCR nucleic acid sequence.

<B17> A system of analyzing a disease, disorder or condition of the subject, comprising the system of any one of items <B11>-<B16> and means for analyzing the disease, disorder or condition of the subject based on the TCR or BCR repertoire derived based the system.

<B18> The system of item <B17>, wherein the disease disorder or condition of the subject is selected from the group consisting of hematological tumor and colorectal cancer.

<B19> A system of treating or preventing the disease, disorder or condition of the subject determined by the system of item <B17> or <B18>, comprising: means for quantitatively associating the disease, disorder or condition of the subject with the TCR or BCR repertoire; and means for selecting means for suitable treatment or prevention from the quantitative association.

<B20> The system of item <B19>, wherein the disease, disorder or condition of the subject is selected from the group consisting of hematological tumor and colorectal cancer.

<B21> A monoclonal T cell related to T cell large granular lymphocytic leukemia (T-LGL) expressing TCRα comprising TRAV10/TRAJ15/CVVRATGTALIFG (SEQ ID NO: 1450) or a nucleic acid encoding the same and/or TCR comprising TRBV29-1/TRBJ2-7/CSVERGGSLGEQYFG (SEQ ID NO: 1500) or a nucleic acid encoding the same.

<B22> Use of TRAV10/TRAJ15/CVVRATGTALIFG (SEQ ID NO: 1450) or a nucleic acid encoding the same in TCRα and/or TRBV29-1/TRBJ2-7/CSVERGGSLGEQYFG (SEQ ID NO: 1500) or a nucleic acid encoding the same in TCRβ as a diagnostic indicator of T cell large granular lymphocytic leukemia (T-LGL).

<B23> A method of detecting T cell large granular lymphocytic leukemia (T-LGL), comprising detecting TRAV10/TRAJ15/CVVRATGTALIFG (SEQ ID NO: 1450) or a nucleic acid encoding the same in TCRα and/or TRBV29-1/TRBJ2-7/CSVERGGSLGEQYFG (SEQ ID NO: 1500) or a nucleic acid encoding the same in TCRβ.

<B24> An detecting agent for TRAV10/TRAJ15/CVVRATGTALIFG (SEQ ID NO: 1450) or a nucleic acid encoding the same in TCRα and/or a detecting agent for TRBV29-1/TRBJ2-7/CSVERGGSLGEQYFG (SEQ ID NO: 1500) or a nucleic acid encoding the same in TCRβ.

<B25> A diagnostic agent for T cell large granular lymphocytic leukemia (T-LGL) comprising a detecting agent for TRAV10/TRAJ15/CVVRATGTALIFG (SEQ ID NO: 1450) or a nucleic acid encoding the same in TCRα and/or a detecting agent for TRBV29-1/TRBJ2-7/CSVERGGSLGEQYFG (SEQ ID NO: 1500) or a nucleic acid encoding the same in TCRβ.

<B26> A peptide which is a novel invariant TCR, comprising any one of the sequences set forth in SEQ ID NOs: 1627-1647.

<B27> An indicator peptide of a mucosal-associated invariant T (MAIT) cell, comprising a sequence selected from the group consisting of SEQ ID NOs 1648-1651, 1653-1654, 1666-1667, 1844-1848, and 1851.

<B28> A nucleic acid encoding the peptide of item <B27>.

<B29> Use of the peptide of item <B27> or <B28> or a nucleic acid encoding said peptide as a diagnostic indicator of colorectal cancer.

<B30> An indicator peptide of a natural killer T cell (NKT), comprising the sequence set forth in SEQ ID NO: 1668.

<B31> A nucleic acid encoding the peptide of item <B30>.

<B32> Use of the peptide of item <B30> or <B31> or a nucleic acid encoding said peptide as a diagnostic indicator of colorectal cancer.

<B33> A colorectal cancer-specific peptide, comprising a sequence selected from the group consisting of SEQ ID NOs: 1652, 1655-1665, 1669-1843, 1849-1850, and 1852-1860.

<B34> A nucleic acid encoding the peptide of item <B33>.

<B35> Use of the peptide of item <B33> or <B34> or a nucleic acid encoding said peptide as a diagnostic indicator of colorectal cancer.

<B36> A colorectal cancer specific peptide, comprising a sequence selected from the group consisting of SEQ ID NOs: 1861-1865 and 1867-1909.

<B37> A nucleic acid encoding the peptide of item <B36>.

<B38> Use of the peptide of item <B36> or <B37> or a nucleic acid encoding said peptide as a diagnostic indicator of colorectal cancer.

<B39> A cell population inducing a T cell at a high frequency, T cell line, or recombinantly expressed T cell having the peptide of item <B33>, <B34>, <B36> or <B37> or a nucleic acid sequence encoding said peptide.

<B40> A therapeutic agent for colorectal cancer, comprising the cell population, T cell line, or T cell of item <B39>.

<B41> A method of treating or preventing colorectal cancer by using the cell population, T cell line, or T cell of item <B39>.

<B42> A method of detecting a usage frequency of a V gene by using the method of any one of items <B1>-<B10> or the system of any one of items <B11>-<B20>.

<B43> A method of detecting a usage frequency of a J gene by using the method of any one of items <B1>-<B10> or the system of any one of items <B11>-<B20>.

<B44> A method of detecting a usage frequency of subtype frequency analysis (BCR) by using the method of any one of items <B1>-<B10> or the system of any one of items <B11>-<B20>.

<B45> A method of analyzing a pattern of CDR3 sequence lengths by using the method of any one of items <B1>-<B10> or the system of any one of items <B11>-<B20>.

<B46> A method of analyzing clonality of a TCR or a BCR by using the method of any one of items <B1>-<B10> or the system of any one of items <B11>-<B20>.

<B47> A method of extracting an overlapping read by using the method of any one of items <B1>-<B10> or the system of any one of items <B11>-<B20>.

<B48> A method of searching for a disease specific TCR or BCR clone by using the method of any one of items <B1>-<B10> or the system of any one of <B11>-<B20>.

<B49> A method of analyzing a subject with a diversity index by using the method of any one of items <B1>-<B10> or the system of any one of items <B11>-<B20>.

<B50> A method of assisting analysis on a subject with a diversity index by using the method of any one of items <B1>-<B10> or the system of any one of items <B11>-<B20>.

<B51> The method of item <B49> or <B50>, wherein the diversity index is used as an indicator for measuring a degree of recovery of an immune system after bone marrow transplantation or as an indicator for detecting abnormality in an immune system cell accompanied by hematopoietic tumor.

<B52> The method of item <B49> or <B50>, wherein the diversity index is selected from the group consisting of a Shannon-Wiener's diversity index (H′), Simpson's diversity index (λ, 1−λ, or 1/λ), Pielou's evenness index (J′) and Chaol index.

<B53> A method of analyzing a subject with a similarity index by using the method of any one of items <B1>-<B10> or the system of any one of items <B11>-<B20>.

<B54> A method of assisting analysis on a subject with a similarly index by using the method of any one of items <B1>-<B10> or the system of any one of items <B11>-<B20>.

<B55> The method of item <B53> or <B54>, wherein the similarity index is used as assessment of a degree of similarity of repertoires between matching and mismatching HLA types, or assessment of a degree of similarly of repertoires between a recipient and a donor after bone marrow transplantation.

<B56> The method of item <B53> or <B54>, wherein the similarity index is selected from the group consisting of a Morisita-Horn index, Kimoto's Cu index, and Pianka's a index.

<B57> The method of item <B1>, wherein the (1) comprises the following steps:

(1-1) synthesizing a complementary DNA by using an RNA sample derived from a target cell as a template;

(1-2) synthesizing a double stranded complementary DNA by using the complementary DNA as a template;

(1-3) synthesizing an adaptor-added double stranded complementary DNA by adding a common adaptor primer sequence to the double stranded complementary DNA;

(1-4) performing a first PCR amplification reaction by using the adaptor-added double stranded complementary DNA, a common adaptor primer consisting of the common adaptor primer sequence, and a first TCR or BCR C region specific primer,

wherein the first TCR or BCR C region specific primer is designed to comprise a sequence that is sufficiently specific to a C region of interest of the TCR or BCR and not homologous with other genetic sequences, and comprise a mismatching base between subtypes downstream when amplified;

(1-5) performing a second PCR amplification reaction by using a PCR amplicon of (1-4), the common adaptor primer, and a second TCR or BCR C region specific primer, wherein the second TCR or BCR C region specific primer is designed to have a sequence that is a complete match with the TCR or BCR C region in a sequence downstream the sequence of the first TCR or BCR C region specific primer, but comprise a sequence that is not homologous with other genetic sequences, and comprise a mismatching base between subtypes downstream when amplified; and

(1-6) performing a third PCR amplification reaction by using a PCR amplicon of (1-5), an added common adaptor primer in which a nucleic acid sequence of the common adaptor primer comprises a first additional adaptor nucleic acid sequence, and an adaptor-added third TCR or BCR C region specific primer in which a second additional adaptor nucleic acid sequence and a molecule identification (MID Tag) sequence are added to a third TCR or BCR C region specific sequence; wherein

the third TCR or BCR C region specific primer is designed to have a sequence that is a complete match with the TCR or BCR C region in a sequence downstream to the sequence of the second TCR or BCR C region specific primer, but comprise a sequence that is not homologous with other genetic sequences, and comprise a mismatching base between subtypes downstream when amplified,

the first additional adaptor nucleic acid sequence is a sequence suitable for binding to a DNA capturing bead and for an emPCR reaction,

the second additional adaptor nucleic acid sequence is a sequence suitable for an emPCR reaction, and

the molecule identification (MID Tag) sequence is a sequence for imparting uniqueness such that an amplicon can be identified.

<B58> The system of item <B11>, wherein the (1) kit comprises the following:

(1-1) means for synthesizing a complementary DNA by using an RNA sample derived from a target cell as a template;

(1-2) means for synthesizing a double stranded complementary DNA by using the complementary DNA as a template;

(1-3) means for synthesizing an adaptor-added double stranded complementary DNA by adding a common adaptor primer sequence to the double stranded complementary DNA;

(1-4) means for performing a first PCR amplification reaction by using the adaptor-added double stranded complementary DNA, a common adaptor primer consisting of the common adaptor primer sequence, and a first TCR or BCR

C region specific primer,

wherein the first TCR or BCR C region specific primer is designed to comprise a sequence that is sufficiently specific to a C region of interest of the TCR or BCR and not homologous with other genetic sequences, and comprise a mismatching base between subtypes downstream when amplified;

(1-5) means for performing a second PCR amplification reaction by using a PCR amplicon of (1-4), the common adaptor primer, and a second TCR or BCR C region specific primer, wherein the second TCR or BCR C region specific primer is designed to have a sequence that is a complete match with the TCR or BCR C region in a sequence downstream the sequence of the first TCR or BCR C region specific primer, but comprise a sequence that is not homologous with other genetic sequences, and comprise a mismatching base between subtypes downstream when amplified; and

(1-6) means for performing a third PCR amplification reaction by using a PCR amplicon of (1-5), an added common adaptor primer in which a nucleic acid sequence of the common adaptor primer comprises a first additional adaptor nucleic acid sequence, and an adaptor-added third TCR or BCR C region specific primer in which a second additional adaptor nucleic acid sequence and a molecule identification (MID Tag) sequence are added to a third TCR or BCR C region specific sequence; wherein

the third TCR or BCR C region specific primer is designed to have a sequence that is a complete match with the TCR or BCR C region in a sequence downstream to the sequence of the second TCR or BCR C region specific primer, but comprise a sequence that is not homologous with other genetic sequences, and comprise a mismatching base between subtypes downstream when amplified,

the first additional adaptor nucleic sequence is a sequence suitable for binding to a DNA capturing bead and for an emPCR reaction,

the second additional adaptor nucleic sequence is a sequence suitable for an emPCR reaction, and

the molecule identification (MID Tag) sequence is a sequence for imparting uniqueness such that an amplicon can be identified.

<B58-2> The method of item <B57> or the system of item <58>, wherein, for a BCR, the C region specific primer comprises a sequence that is a complete match with an isotype C region of interest selected from the group consisting of IgM, IgA, IgG, IgE and IgD and is not homologous with other C regions, and is a sequence that is a complete match with one of the subtypes IgG1, IgG2, IgG3 and IgG4 or one of IgA1 or IgA2 for IgA or IgG, or, for a TCR, the C region specific primer is a sequence that is a complete match with a C region of α chain of interest selected from the group consisting of α chain, β chain, γ chain and δ chain and is not homologous with other C regions.

<B58-3> The method of item <B57> or <B58-2> or the system of <B58> or <B58-2>, wherein a portion of a sequence that is a complete match with all C region allelic sequences of the same isotype in the database is selected for the C region specific primer.

<B58-4> The method of any one of items <B57> and <B58-2>-<B58-3> or the system of any one of <B58>-<B58-3>, wherein the common adaptor primer is designed such that the primer is unlikely to have homodimer and intramolecular hairpin structures and can stably form a double strand, and designed not to be highly homologous with all TCR genetic sequences in the database and to have the same level of melting temperature (Tm) as the C region specific primer.

<B58-5> The method of any one of items <B57> and <B58-2>-<B58-4> or the system of any one of <B58>-<B58-4>, wherein the common adaptor primer designed not to have homodimer and intramolecular hairpin structures and to have homology with other genes comprising a BCR or TCR is selected.

<B58-6> The method of any one of items <B57> and <B58-2>-<B58-5> or the system of any one of <B58>-<B58-5>, wherein the common adaptor primer is P20EA (SEQ ID NO: 2) and/or P10EA (SEQ ID NO: 3).

<B58-7> The method of any one of items <B57> and <B58-2>-<B58-6> or the system of any one of <B58>-<B58-6>, wherein the first, second and third TCR or BCR C region specific primers are each independently a primer for BCR repertoire analysis, the primer being selected to be a sequence that is a complete match with each isotype C region of IgM, IgG, IgA, IgD or IgE, and a complete match with subtypes for IgG and IgA, and not homologous with other sequences comprised in the database, and comprise a mismatching base between subtypes downstream of the primer, and

wherein the common adaptor primer sequence is designed such that the sequence has a base length suitable for amplification, is unlikely to have homodimer and intramolecular hairpin structures, and is able to stably form a double strand, and designed not to be highly homologous with all TCR genetic sequences in the database and to have the same level of Tm as the C region specific primer.

<B58-8> The method of any one of items <B57> and <B58-2>-<B58-7> or the system of any one of <B58>-<B58-7>, wherein the first, second and third TCR or BCR C region specific primers are each independently a primer for TCR or BCR repertoire analysis, each primer being selected to be a sequence that is a complete match with 1 type of α chain (TRAC), 2 types of β chains (TRBCO1 and TRBCO2), 2 types of γ chains (TRGC1 and TRGC2), and one type of δ chain (TRDC1) and is not homologous with other sequences comprised in the database, and comprise a mismatching base between subtypes downstream of the primer,

wherein the common adaptor primer sequence is designed such that the sequence has a base length suitable for amplification, is unlikely to have homodimer and intramolecular hairpin structures, and is able to stably form a double strand, and designed not to be highly homologous with all TCR genetic sequences in the database and to have the same level of Tm as the C region specific primer.

<B58-9> The method of any one of items <B57> and <B58-2>-<B58-8> or the system of any one of <B58>-<B58-8>, wherein the third TCR or BCR C region specific primer is set in a region that is up to about 150 bases from the 5′ terminal side of a C region, and the first TCR or BCR C region specific primer and the second TCR or BCR C region specific primer are set between the 5′ terminal side of a C region to about 300 bases.

<B58-10> The method of any one of items <B57> and <B58-2>-<B58-9> or the system of any one of <B58>-<B58-9>, wherein the first, second and third TCR or BCR C region specific primers are each independently for BCR quantitative analysis,

wherein separate specific primers are set to 5 types of isotype sequences, and the primers are designed to completely match a target sequence and ensure a mismatch of 5 bases or more for other isotypes and are designed to be a complete match with all subtypes such that one type of primer is compatible with each similar IgG subtype (IgG1, IgG2, IgG3 and IgG4) or IgA subtype (IgA1 and IgA2).

<B58-11> The method of any one of items <B57> and <B58-2>-<B58-10> or the system of any one of <B58>-<B58-10>, wherein parameters in primer design are set to: a base sequence length of 18-22 bases; a melting temperature of 54-66° C.; and % GC (% guanine.cytosine content) of 40-65%.

<B58-12> The method of any one of items <B57> and <B58-2>-<B58-11> or the system of any one of <B58>-<B58-11>, wherein parameters in primer design are set to: a base sequence length of 18-22 bases; a melting temperature of 54-66° C.; and % GC (% guanine.cytosine content) of 40-65%; a self-annealing score of 26; a self-end annealing score of 10; and a secondary structure score of 28.

<B58-13> The method of any one of items <B57> and <B58-2>-<B58-12> or the system of any one of <B58>-<B58-12>, wherein sequences of the first, second and third TCR or BCR C region specific primers are determined under the following conditions:

1. a plurality of subtype sequences and/or allelic sequences are uploaded into a base sequence analysis software and aligned;

2. a primer designing software is used to search for a plurality of primers satisfying a parametric condition in a C region;

3. a primer in a region without a mismatching base in the aligned sequences in 1 is selected; and

4. the presence of a plurality of mismatching sequences for each subtype and/or allele downstream of the primer determined in 3 is confirmed, and if there is no such sequence, a primer is searched further upstream, which is further repeated as needed.

<B58-14> The method of any one of items <B57> and <B58-2>-<B58-13> or the system of any one of <B58>-<B58-13>, wherein the first TCR or BCR C region specific primer is set in a position at bases 41-300 with a first base of a first codon of a C region sequence produced by splicing as a baseline, the second TCR or BCR C region specific primer is set in a position at bases 21-300 with said first base as the baseline, and the third TCR or BCR C region specific primer is set in a position within 150 bases or less with said first base as the baseline, and the positions comprise a mismatching site in a subtype and/or allele.

<B58-15> The method of any one of items <B57> and <B58-2>-<B58-14> or the system of any one of <B58>-<B58-14>, wherein the first TCR or BCR C region specific primer has the following structure: CM1 (SEQ ID NO: 5), CA1 (SEQ ID NO: 8), CG1 (SEQ ID NO: 11), CD1 (SEQ ID NO: 14), CE1 (SEQ ID NO: 17), CA1 (SEQ ID NO: 35) or CB1 (SEQ ID NO: 37).

<B58-16> The method of any one of items <B57> and <B58-2>-<B58-15> or the system of any one of <B58>-<B58-15>, wherein the second TCR or BCR C region specific primer has the following structure: CM2 (SEQ ID NO: 6), CA2 (SEQ ID NO: 9), CG2 (SEQ ID NO: 12), CD2 (SEQ ID NO: 15), CE2 (SEQ ID NO: 18), CA2 (SEQ ID NO: 35), or CB2 (SEQ ID NO: 37).

<B58-17> The method of any one of items <B57> and <B58-2>-<B58-16> or the system of any one of <B58>-<B58-16>, wherein the third TCR or BCR C region specific primer has the following structure: CM3-GS (SEQ ID NO: 7), CA3-GS (SEQ ID NO: 10), CG3-GS (SEQ ID NO: 13), CD3-GS (SEQ ID NO: 16) or CE3-GS (SEQ ID NO: 19).

<B58-18> The method of any one of items <B57> and <B58-2>-<B58-17> or the system of any one of <B58>-<B58-17>, wherein each of the TCR or BCR C region specific primers is provided in a set compatible with all TCR or BCR subclasses.

<B58-19> A method or system of performing gene analysis using a sample manufactured by the method of any one of items <B57> and <B58-2>-<B58-18> or the system of any one of <B58>-<B58-18>.

<B58-20> The method or system of item <B58-19>, wherein the gene analysis is the quantitative analysis of a repertoire of a variable region of a T cell receptor (TCR) or a B cell receptor (BCR).

<B59> The method of any one of items <B57> and <B58-2>-<B58-20> or the system of any one of <B58>-<B58-20>, wherein (3) derivation of the TCR or BCR repertoire is accomplished by a method comprising the following steps:

(3-1) providing a reference database for each gene region comprising at least one of a V region, a D region, a J region and optionally a C region;

(3-2) providing an input sequence set which is optionally trimmed and optionally extracted to have a suitable length;

(3-3) searching for homology of the input sequence set with the reference database for the each gene region and recording an alignment with an approximate reference allele and/or a sequence of the reference allele;

(3-4) assigning the V region and the J region for the input sequence set and extracting a nucleic acid sequence of the D region based on a result of assigning;

(3-5) translating the nucleic acid sequence of the D region into an amino acid sequence and classifying the D region by utilizing the amino acid sequence; and

(3-6) calculating a frequency of appearance for each of the V region, the D region, and the J region and optionally the C region or a frequency of appearance of a combination thereof based on the classifying in (3-5) to derive the TCR or BCR repertoire.

<B60> The system of any one of items <B11>-<B20>, and <B58>-<B58-20> and <B59>, wherein (3) an apparatus for deriving the TCR or BCR repertoire comprises the following:

(3-1) means for providing a reference database for each gene region comprising at least one of a V region, a D region, a J region and optionally a C region;

(3-2) means for providing an input sequence set which is optionally trimmed and optionally extracted to have a suitable length;

(3-3) means for searching for homology of the input sequence set with the reference database for the each gene region and recording an alignment with an approximate reference allele and/or a sequence of the reference allele;

(3-4) means for assigning the V region and the J region for the input sequence set and extracting a nucleic acid sequence of the D region based on a result of assigning;

(3-5) means for translating the nucleic acid sequence of the D region into an amino acid sequence and classifying the D region by utilizing the amino acid sequence; and

(3-6) means for calculating a frequency of appearance for each of the V region, the D region, and the J region and optionally the C region or a frequency of appearance of a combination thereof based on the classifying in (3-5) to derive the TCR or BCR repertoire.

<B60-2> The method of any one of items <B57>, <B58-2>-<B58-20>, and <B59> or the system of any one of <B58>-<B58-20>, <B59> and <B60>, wherein the gene region comprises all of the V region, the D region, the J region and optionally the C region.

<B60-3> The method of any one of items <B57>, <B58-2>-<B58-20>, <B59>, and <B60-2> or the system of any one of <B58>-<B58-20>, <B59> and <B60>-<B60-2>, wherein the reference database is a database with a unique ID assigned to each sequence.

<B60-4> The method of any one of items <B57>, <B58-2>-<B58-20>, <B59>, and <B60-2>-<B60-3> or the system of any one of <B58>-<B58-20>, <B59> and <B60>-<B60-3>, wherein the input sequence set is an unbiased sequence set.

<B60-5> The method of any one of items <B57>, <B58-2>-<B58-20>, <B59>, and <B60-2>-<B60-4> or the system of any one of <B58>-<B58-20>, <B59> and <B60>-<B60-4>, wherein the sequence set is trimmed.

<B60-6> The method of any one of items <B57>, <B58-2>-<B58-20>, <B59>, and <B60-2>-<B60-5> or the system of any one of <B58>-<B58-20>, <B59> and <B60>-<B60-5>, wherein the trimming is accomplished by the steps of: deleting low quality regions from both ends of a read; deleting a region matching 10 bp or more with an adaptor sequence from the both ends of the read; and using the read as a high quality read in analysis when a remaining length is 200 bp or more (TCR) or 300 bp or more (BCR).

<B60-7> The method of any one of items <B57>, <B58-2>-<B58-20>, <B59>, and <B60-2>-<B60-6> or the system of any one of <B58>-<B58-20>, <B59> and <B60>-<B60-6>, wherein the low quality refers to a 7 bp moving average of QV value less than 30.

<B60-8> The method of any one of items <B57>, <B58-2>-<B58-20>, <B59>, and <B60-2>-<B60-7> or the system of any one of <B58>-<B58-20>, <B59> and <B60>-<B60-7>, wherein the approximate sequence is the closest sequence.

<B60-9> The method of any one of items <B57>, <B58-2>-<B58-20>, <B59>, and <B60-2>-<B60-8> or the system of any one of <B58>-<B58-20>, <B59> and <B60>-<B60-8>, wherein the approximate sequence is determined by a ranking of 1. number of matching bases, 2. kernel length, 3. score, and 4. alignment length.

<B60-10> The method of any one of items <B57>, <B58-2>-<B58-20>, <B59>, and <B60-2>-<B60-9> or the system of any one of <B58>-<B58-20>, <B59> and <B60>-<B60-9>, wherein the homology search is conducted under a condition tolerating random mutations to be scattered throughout.

<B60-11> The method of any one of items <B57>, <B58-2>-<B58-20>, <B59>, and <B60-2>-<B60-10> or the system of any one of <B58>-<B58-20>, <B59> and <B60>-<B60-10>, wherein the homology search comprises at least one condition from (1) shortening of a window size, (2) reduction in a mismatch penalty, (3) reduction in a gap penalty, and (4) a top priority ranking of an indicator is a number of matching bases, compared to a default condition.

<B60-12> The method of any one of items <B57>, <B58-2>-<B58-20>, <B59>, and <B60-2>-<B60-11> or the system of any one of <B58>-<B58-20>, <B59> and <B60>-<B60-11>, wherein the homology search is carried out under the following conditions in BLAST or FASTA:

V mismatch penalty=−1, shortest alignment length=30, and shortest kernel length=15;

D word length=7 (for BLAST) or K-tup=3 (for FASTA), mismatch penalty=−1, gap penalty=0, shortest alignment length=11, and shortest kernel length=8;

J mismatch penalty=−1, shortest hit length=18, and shortest kernel length=10; and

C shortest hit length=30 and shortest kernel length=15.

<B60-13> The method of any one of items <B57>, <B58-2>-<B58-20>, <B59>, and <B60-2>-<B60-12> or the system of any one of <B58>-<B58-20>, <B59> and <B60>-<B60-12>, wherein the D region is classified by a frequency of appearance of the amino acid sequence.

<B60-14> The method of any one of items <B57>, <B58-2>-<B58-20>, <B59>, and <B60-2>-<B60-13> or the system of any one of <B58>-<B58-20>, <B59> and <B60>-<B60-13>, wherein a combination of a result of search for homology with the nucleic acid sequence of CDR3 and a result of amino acid sequence translation is used as a classification result when there is a reference database for the D region in the step (5).

<B60-15> The method of any one of items <B57>, <B58-2>-<B58-20>, <B59>, and <B60-2>-<B60-14> or the system of any one of <B58>-<B58-20>, <B59> and <B60>-<B60-14>, wherein only the frequency of appearance of the amino acid sequence is used for classification when there is no reference database for the D region in the step (5).

<B60-16> The method of any one of items <B57>, <B58-2>-<B58-20>, <B59>, and <B60-2>-<B60-15> or the system of any one of <B58>-<B58-20>, <B59> and <B60>-<B60-15>, wherein the frequency of appearance is counted in a unit of a gene name and/or a unit of an allele

<B60-17> The method of any one of items <B57>, <B58-2>-<B58-20>, <B59>, and <B60-2>-<B60-16> or the system of any one of <B58>-<B58-20>, <B59> and <B60>-<B60-16>, wherein the step (4) comprises the step of assigning the V region and the J region for the input sequence set and extracting a CDR3 sequence, with the front of CDR3 on a reference V region and end of CDR3 on reference J as guides.

<B60-18> The method of any one of items <B57>, <B58-2>-<B58-20>, <B59>, and <B60-2>-<B60-17> or the system of any one of <B58>-<B58-20>, <B59> and <B60>-<B60-17>, wherein the step (5) comprises translating the nucleic acid sequence of the CDR3 into an amino acid sequence and classifying a D region by using the amino acid sequence.

<B60-19> The system of any one of items <B11>-<B20>, <B58>-<B58-20>, <B59> and <B60>-<B60-18>, wherein (3) an apparatus for deriving the TCR or BCR repertoire comprises:

(3-1) means for providing a reference database for each gene region comprising at least one of a V region, a D region, a J region and optionally a C region;

(3-2) means for providing an input sequence set which is optionally trimmed and optionally extracted to have a suitable length;

(3-3) means for searching for homology of the input sequence set with the reference database for the each gene region and recording an alignment with an approximate reference allele and/or a sequence of the reference allele;

(3-4) means for assigning the V region and the J region for the input sequence set and extracting a nucleic acid sequence of the D region based on a result of assigning;

(3-5) means for translating the nucleic acid sequence of the D region into an amino acid sequence and classifying the D region by utilizing the amino acid sequence; and

(3-6) means for calculating a frequency of appearance for each of the V region, the D region, and the J region and optionally the C region or a frequency of appearance of a combination thereof in the input sequence set to derive the TCR or BCR repertoire.

<B60-20> The system of any one of items <B11>-<B20>, <B58>-<B58-20>, <B59> and <B60>-<B60-19>, wherein processing of a method of analyzing the TCR or BCR repertoire is materialized by a computer program for having a computer execute the processing comprising the following steps:

(1) providing a reference database for each gene region comprising at least one of a V region, a D region, a J region and optionally a C region;

(2) providing an input sequence set which is optionally trimmed and optionally extracted to have a suitable length;

(3) searching for homology of the input sequence set with the reference database for the each gene region and recording an alignment with an approximate reference allele and/or a sequence of the reference allele;

(4) assigning the V region and the J region for the input sequence set and extracting a nucleic acid sequence of the D region based on a result of assigning;

(5) translating the nucleic acid sequence of the D region into an amino acid sequence and classifying the D region by utilizing the amino acid sequence; and

(6) calculating a frequency of appearance for each of the V region, the D region, and the J region and optionally the C region or a frequency of appearance of a combination thereof in the input sequence set to derive the TCR or BCR repertoire.

<B60-21> The system of any one of items <B11>-<B20>, <B58>-<B58-20>, <B59> and <B60>-<B60-20> for having a computer execute processing of a method of analyzing a TCR or BCR repertoire, the method comprising the following steps:

(1) providing a reference database for each gene region comprising at least one of a V region, a D region, a J region and optionally a C region;

(2) providing an input sequence set which is optionally trimmed and optionally extracted to have a suitable length;

(3) searching for homology of the input sequence set with the reference database for the each gene region and recording an alignment with an approximate reference allele and/or a sequence of the reference allele;

(4) assigning the V region and the J region for the input sequence set and extracting a nucleic acid sequence of the D region based on a result of assigning;

(5) translating the nucleic acid sequence of the D region into an amino acid sequence and classifying the D region by utilizing the amino acid sequence; and

(6) calculating a frequency of appearance for each of the V region, the D region, and the J region and optionally the C region or a frequency of appearance of a combination thereof in the input sequence set to derive the TCR or BCR repertoire.

<Examples of Application in Analysis>

### <C1>

A method of applying a cancer idiotype peptide sensitization immune cell therapeutic method to a subject, the method comprising:

(1) analyzing a T cell receptor (TCR) or B cell receptor (BCR) repertoire of the subject by the method of any one of items <B1>-<B10>, <B57>, <B58-2>-<B58-20>, <B59>, and <B60>-<B60-21> or the system of any one of items <B11>-<B20>, <B58>-<B58-20>, <B59> and <B60>-<B60-21>;

(2) determining a TCR or BCR derived from a cancer cell of the subject based on a result of the analysis, wherein the determining is done by selecting a high ranking sequence in a frequency of presence ranking of a TCR or BCR gene derived from the cancer cell of the subject as the TCR or BCR derived from the cancer cell;

(3) determining an amino acid sequence of a candidate HLA test peptide based on the determined TCR or BCR derived from cancer, wherein the determining is performed based on a score calculated by using an HLA binding peptide prediction algorithm;

(4) synthesizing the determined peptide; and optionally

(5) administering therapy by using the synthesized peptide.

### <C2>

The method of item <C1>, wherein the candidate HLA test peptide of the step (3) is determined by using BIMAS, SYFPEITHI, RANKPEP or NetMHC.

### <C3> <Improved CTL Method>

The method of item <C1> or <C2>, wherein the method comprises, after the step (4), the steps of: mixing the peptide, an antigen presenting cell or a dendritic cell derived from the subject, and a CD8+ T cell derived from the subject and culturing the mixture; and administering the mixture after culturing to a patient.

### <C4> <DC Vaccination Therapeutic Method>

The method of any one of items <C1>-<C3> comprising, after the step (4), the steps of: mixing the peptide with the dendritic cell derived from the subject and culturing the mixture; and administering the cultured mixture to a patient.

### <C5> <Patient Autoimmune Cell Therapeutic Method>

The method of any one of items <C1>-<C4>, wherein the method comprises, after the step (4), the steps of: mixing the peptide, the antigen presenting cell or the dendritic cell derived from the subject and a CD8+ T cell derived from the subject and culturing the mixture to produce a CD8+ T cell-dendritic cell/antigen presenting cell-peptide mixture; mixing the peptide with the dendritic cell derived from the subject and culturing the mixture to produce a dendritic cell-peptide mixture; and administering the CD8+ T cell-dendritic cell/antigen presenting cell-peptide mixture and the dendritic cell-peptide mixture to a patient.

### <D1> <Isolation of Tailor-Made Cancer Specific T Cell Receptor Gene, Isolation of Cancer Specific TCR Gene by In Vitro Antigen Stimulation>

A method of isolating a cancer specific TCR gene by an in vitro antigen stimulation, comprising:

(A) mixing an antigen peptide or antigen protein derived from a subject or the determined peptide of any one of items <C1>-<C5> or a lymphocyte derived from the subject, an inactivated cancer cell derived from the subject, and a T lymphocyte derived from the subject and culturing the mixture to produce a tumor specific T cell;

(B) analyzing a TCR of the tumor specific T cell by the method of any one of items <B1>-<B10>, <B57>, <B58>-<B58-20>, <B59>, and <B60>-<B60-21> and/or the system of any one of items <B11>-<B20>, <B58>-<B58-20>, <B59> and <B60>-<B60-21>; and

(C) isolating a desired tumor specific T cell based on a result of the analyzing.

**<D1-1>**

The method of item <D1>, wherein step (A) is a step of mixing the inactivated cancer cell derived from the subject and the antigen peptide or antigen protein derived from the subject with the T lymphocyte derived from the subject and culturing the mixture to produce a tumor specific T cell.

**<D1-2>**

The method of any one of items <D1>-<D1-1>, wherein the step (A) is a step of mixing the lymphocyte derived from the subject, the inactivated cancer cell derived from the subject, and the T lymphocyte derived from the subject and culturing the mixture to produce a tumor specific T cell.

**<D1-3>**

The method of any one of items <D1>-<D1-2>, wherein the step (A) is a step of mixing the determined peptide of item C1, the inactivated cancer cell derived from the subject, and the T lymphocyte derived from the subject and culturing the mixture to produce a tumor specific T cell.

**<D2>**

**<Isolation of Tailor-Made Cancer Specific T Cell Receptor Gene, Isolation of Cancer Specific TCR Gene by Searching for a Common Sequence>**

A method of isolating a cancer specific TCR gene by searching for a common sequence, comprising:

(A) isolating a lymphocyte or cancer tissue from subjects having a common HLA;

(B) analyzing a TCR of the tumor specific T cell by the method of item B1 for the lymphocyte or cancer tissue; and

(C) isolating a T cell having a sequence in common with the tumor specific T cell.

**<E1> <CPC>**

A cell processing therapeutic method, comprising:

A) collecting a T lymphocyte from a patient;

B) analyzing TCRs based on the method of any one of items <B1>-<B10>, <B57>, <B58>-<B58-20>, <B59>, and <B60>-<B60-21> and/or the system of any one of items <B11>-<B20>, <B58>-<B58-20>, <B59> and <B60>-<B60-21> after applying antigen stimulation to the T lymphocyte, wherein the antigen stimulation is applied by an antigen peptide or antigen protein derived from the subject, an inactivated cancer cell derived from the subject, or an idiotype peptide derived from tumor;

C) selecting an optimal TCR and an optimal antigen in the analyzed TCRs;

D) producing a tumor specific α and β TCR expression viral vector of a TCR gene of the optimal TCR; and

E) introducing the T lymphocyte introduced with a tumor specific TCR gene into the patient.

**<E1-1>**

The cell processing therapeutic method of item <E1>, wherein the antigen stimulation is applied with the antigen peptide or antigen protein derived from the subject.

**<E1-2>**

The cell processing therapeutic method of item <E1> or <E1-1>, wherein the antigen stimulation is applied with the inactivated cancer cell derived from the subject.

**<E1-3>**

The cell processing therapeutic method of any one of items <E1> and <E1-1>-<E1-2>, wherein the antigen stimulation is applied with the idiotype peptide derived from tumor.

**<E1-4>**

The method of any one of items <E1> and <E1-1>-<E1-3>, wherein the step C) comprises selecting an antigen that is highly expressed in cancer tissue of the subject.

**<E1-5>**

The method of any one of items <E1> and <E1-1>-<E1-4>, wherein the step C) comprises selecting an antigen which most strongly activates a T cell in an antigen specific lymphocyte stimulation test.

**<E1-6>**

The method of any one of items <E1> and <E1-1>-<E1-5>, wherein the step C) comprises selecting an antigen that increases a frequency of a specific TCR the most from repertoire analysis conducted based on the method of any one of items <B1>-<B10>, <B57>, <B58>-<B58-20>, <B59>, and <B60>-<B60-21> and/or the system of any one of items <B11>-<B20>, <B58>-<B58-20>, <B59> and <B60>-<B60-21> before and after applying the antigen stimulation.

**<E2> <RAC of CPC>**

A method of assessing efficacy and/or safety by a stimulation test in vitro by using a cancer specific TCR gene isolated by the method of item <D2>.

**<CC1>**

A method of preparing a composition for use in a cancer idiotype peptide sensitization immune cell therapeutic method to a subject, the method comprising:

(1) analyzing a T cell receptor (TCR) or B cell receptor (BCR) repertoire of the subject by the method of any one of items <B1>-<B10>, <B57>, <B58>-<B58-20>, <B59>, and <B60>-<B60-21> and/or the system of any one of items <B11>-<B20>, <B58>-<B58-20>, <B59> and <B60>-<B60-21>;

(2) determining a TCR or BCR derived from a cancer cell of the subject based on a result of the analysis, wherein the determining is done by selecting a high ranking sequence in a frequency of presence ranking of a TCR or BCR gene derived from the cancer cell of the subject as the TCR or BCR derived from the cancer cell;

(3) determining an amino acid sequence of a candidate HLA test peptide based on the determined TCR or BCR derived from cancer, wherein the determining is performed based on a score calculated by using an HLA binding peptide prediction algorithm; and

(4) synthesizing the determined peptide.

**<CC2>**

The method of item <CC1>, wherein the candidate HLA test peptide of the step (3) is determined by using BIMAS, SYFPEITHI, RANKPEP or NetMHC.

**<CC3> <Improved CTL Method>**

The method of item <CC1> or <CC2>, wherein the method comprises, after the step (4), the step of: mixing the peptide, an antigen presenting cell or a dendritic cell derived from the subject, and a CD8+ T cell derived from the subject and culturing the mixture.

**<CC4> <DC Vaccination Therapeutic Method>**

The method of any one of items <CC1>-<CC2> comprising, after the step (4), the step of: mixing the peptide with a dendritic cell derived from the subject and culturing the mixture.

**<CC5> <Patient Autoimmune Cell Therapeutic Method>**

The method of any one of items <CC1>-<CC4>, wherein the method comprises, after the step (4), the steps of: mixing the peptide, the antigen presenting cell or the dendritic cell derived from the subject and a CD8+ T cell derived from the subject and culturing the mixture to produce a CD8+ T cell-dendritic cell/antigen presenting cell-peptide mixture; and mixing the peptide with the dendritic cell derived from the subject and culturing the mixture to produce a dendritic cell-peptide mixture.

**<DD1> <Isolation of Tailor-Made Cancer Specific T Cell Receptor Gene, Isolation of Cancer Specific TCR Gene by In Vitro Antigen Stimulation>**

A method of preparing an isolated cancer specific TCR gene by an in vitro antigen stimulation, comprising:

(A) mixing an antigen peptide or antigen protein derived from a subject or the determined peptide of any one of items <CC1>-<CC5> or a lymphocyte derived from the subject, an inactivated cancer cell derived from the subject, and a T lymphocyte derived from the subject and culturing the mixture to produce a tumor specific T cell;

(B) analyzing a TCR of the tumor specific T cell by the method of any one of items <B1>-<B10>, <B57>, <B58>-<B58-20>, <B59>, and <B60>-<B60-21> and/or the system of any one of items <B11>-<B20>, <B58>-<B58-20>, <B59> and <B60>-<B60-21>; and

(CC) isolating a desired tumor specific T cell based on a result of the analyzing.

**<DD1-1>**

The method of item <DD1>, wherein the step (A) is a step of mixing the inactivated cancer cell derived from the subject and the antigen peptide or antigen protein derived from the subject with the T lymphocyte derived from the subject and culturing the mixture to produce a tumor specific T cell.

**<DD1-2>**

The method of item <DD1> or <DD1-1>, wherein the step (A) is a step of mixing the lymphocyte derived from the subject, the inactivated cancer cell derived from the subject, and the T lymphocyte derived from the subject and culturing the mixture to produce a tumor specific T cell.

**<DD1-3>**

The method of any one of items <DD1>-<DD1-2>, wherein the step (A) is a step of mixing the determined peptide of item CC1, the inactivated cancer cell derived from the subject, and the T lymphocyte derived from the subject and culturing the mixture to produce a tumor specific T cell.

**<DD2>**

**<Isolation of Tailor-Made Cancer Specific T Cell Receptor Gene, Isolation of Cancer Specific TCR Gene by Searching for a Common Sequence>**

A method of preparing an isolated cancer specific TCR gene by searching for a common sequence, comprising:

(A) providing a lymphocyte or cancer tissue isolated from subjects having a common HLA;

(B) analyzing a TCR of the tumor specific T cell by the method of any one of items <B1>-<B10>, <B57>, <B58>-<B58-20>, <B59>, and <B60>-<B60-21> and/or the system of any one of items <B11>-<B20>, <B58>-<B58-20>, <B59> and <B60>-<B60-21> for the lymphocyte or cancer tissue; and

(C) isolating a T cell having a sequence in common with the tumor specific T cell.

**<EE1> <CCPCC>**

A method of preparing a T lymphocyte introduced with a tumor specific TCR gene for use in a cell processing therapeutic method, comprising:

A) providing a T lymphocyte collected from a patient;

B) analyzing TCCRs based on the method of any one of items <B1>-<B10>, <B57>, <B58>-<B58-20>, <B59>, and <B60>-<B60-21> and/or the system of any one of items <B11>-<B20>, <B58>-<B58-20>, <B59> and <B60>-<B60-21> after applying an antigen stimulation to the T lymphocyte, wherein the antigen stimulation is applied by an antigen peptide or antigen protein derived from the subject, an inactivated cancer cell derived from the subject, or an idiotype peptide derived from tumor;

CC) selecting an optimal TCR and an optimal antigen in the analyzed TCRs; and

DD) producing a tumor specific α and β TCR expression viral vector of a TCCR gene of the optimal TCR.

**<EE1-1>**

The method of item <EE1>, wherein the antigen stimulation is applied with the antigen peptide or antigen protein derived from the subject.

**<EE1-2>**

The method of item <EE1> or <EE1-1>, wherein the antigen stimulation is applied with the inactivated cancer cell derived from the subject.

**<EE1-3>**

The method of any one of items <EE1>-<EE1-2>, wherein the antigen stimulation is applied with the idiotype peptide derived from tumor.

**<EE1-4>**

The method of any one of items <EE1>-<EE1-3>, wherein the step C) comprises selecting an antigen that is highly expressed in cancer tissue of the subject.

**<EE1-5>**

The method of any one of items <EE1>-<EE1-4>, wherein the step C) comprises selecting an antigen which most strongly activates a T cell in an antigen specific lymphocyte stimulation test.

**<EE1-6>**

The method of any one of items <EE1>-<EE1-5>, wherein the step C) comprises selecting an antigen that increases a frequency of a specific TCCR the most from repertoire analysis conducted based on item <B1> before and after applying the antigen stimulation.

**<EE2> <RACC of CCPCC>**

A method of assessing efficacy and/or safety by a stimulation test in vitro by using a cancer specific TCCR gene isolated by the method of item <DD2>.

The specific steps of the efficacy and/or safety assessment is exemplified below.

<Efficacy> For instance, efficacy can be assessed, after culturing a T cell introduced with a cancer specific TCR gene with the antigen peptide or antigen protein derived from the subject of <EE1-1>, inactivated cancer cell derived from the subject of <EE1-2>, or idiotype peptide derived from tumor of <EE1-3>, by measuring the amount of cytokines (interferon γ or the like) secreted to the outside of a cell in response to T cell activation, by measuring the amount of expression of a specific gene that is elevated in response to T cell activation, or by measuring a cell surface molecule that is expressed or undergoes increased expression in response to T cell activation.

<Safety> For instance, safety can be assessed, when a T cell derived from the subject introduced with a cancer specific TCR gene is mixed with a normal cell derived from the subject, by measuring the above-described cytokines secreted, gene expression, or expression of a cell surface molecule in response to T cell activation and confirming that the T cell transgenically introduced with a TCR is not activated by a normal cell.

It is understood that the present invention can further be provided as a combination of one or more of the aforementioned features in addition to the explicitly shown combinations. Further embodiments and advantages of the present invention are recognized by those skilled in the art by reading and understanding the following Detailed Description as needed.

## Advantageous Effects of Invention

The present invention has an effect of being capable of handling a “large scale” sequence relative to conventional techniques. The present invention is considered to have an especially advantageous effect in terms of being able to, regardless of a mutation, amplify in an “unbiased” manner and make an accurate determination for especially BCRs, as numerous mutations are observed. The present invention is considered 1. unbiased, and 2. therefore has excellent quantifiability with respect to amplification methods and sequencing methods utilizing a V chain specific primer among conventional systems. The present invention is also advantageous with respect to techniques such as SMART PCR in terms of 1. significantly improved “level of unbiasedness” and 2. lack of the unique disadvantages of each technique. For instance, an issue of Repeated Template Switching is reported for SMART. However, the present system does not have such an issue. Further, other advantageous effects include 3. the capability of comprehensive analysis, including identification of isotypes and subtypes.

The system and method of the present invention can derive TCR and BCR repertoires of α, β, γ, and δ chains for TCRs and IgM, IgD, IgA, IgG, and IgE heavy chains and IgK and IgL light chains for BCRs and detect a change in the repertoires from various aspects. A C region primer for a sequence is arranged at a suitable position in order to accurately determine a CDR3 region base sequence that is important in identifying a disease specific TCR or BCR. Furthermore, a primer position is devised such that the type of isotype or subtype can be identified and a gene associated with a disease is readily identified.

All conventional techniques employed A plurality of PCR using numerous V chain specific primers and had a significant issue in quantification or precision. However, such an issue was resolved. Further, use of the analysis system of the present invention also accomplishes the following. For instance, the analysis system can screen for invariant TCRs. It was discovered that invariant TCRs can be screened because a read overlapping in numerous samples is searched regardless of HLA in a TCRα chain in TCR repertoire analysis for a large scale base sequence. In fact, it was possible to detect numerous TCRs derived from MAIT recognizing MR1, which is a non-classical MHC. It is known that NKT, MAIT or the like expressing an invariant TCR serves an important role in immune responses such as infection immunity, antitumor or inflammation. It is expected that a novel invariant TCR can be screened for in various tissue samples and utilized to find a cell with a unique function.

Furthermore, a TCRα and TCRβ gene pair of an antigen specific TCR can be estimated. TCRα and TCRβ are receptor molecules forming a heterodimer. An antigen specific T cell that proliferates in response to an antigen consists of specific unique TCRα and TCRβ chains. However, since TCR repertoire analysis amplifies TCRα and TCRβ genes separately, it is not possible to known which TCRα and which TCRβ form a pair. In this regard, it is possible to estimate paired TCRα and TCRβ chain genes by examining whether a combination of individuals with an overlap in a specific TCRβ chain read matches with individuals with an overlap in a TCRα chain (FIG. 44). It was possible to estimate a matching TCRα chain by using individuals with an overlap in a specific TCR chain as an indicator (Table 3-11). Although there are cases where this is assigned to a plurality of reads, it is considered to be a searching method that is useful in identifying paired TCR genes.

It is especially useful in clinical applications where a sample for highly precise, unbiased, large scale gene analysis is provided and quantitative analysis is especially required. Further, the present invention can identify a “low frequency” ( 1/10,000- 1/100,000 or lower) gene, leading to a more accurate diagnosis or therapy of leukemia or the like. This was not possible with conventional techniques (method of combining plating with an adaptor or method of combining plating with the SMART method) due to the detection limit (about 1%).

Further, a V specific technology has low quantifiability due to varying amplification efficiency among V specific primers. However, this technology performs amplification with one set of primers, thus enabling highly precise quantification in the truest sense.

Further, since all TCRs or BCRs can be amplified with one set of primers, primers and containers required for amplification can be reduced to cut expenses.

Further, BCRs are characterized by having a mutation. Thus, a method using a V chain specific primer has disadvantages such as essentially being unable to perform amplification, or producing a gene with reduced amplification efficiency or the like. Meanwhile, the method of the present invention can also solve problems in BCRs.

Further, the analysis method using the present invention is advantageous in that it can complete the method in several minutes while conventional techniques complete overnight.

### <Wet Associated Effect>

The present invention is especially useful in clinical applications where quantitative analysis is especially required and a sample is provided for highly precise, unbiased, large scale gene analysis. Further, the present invention can identify a “low frequency” ( 1/10000- 1/100000 or lower) gene, leading to a more accurate diagnosis or therapy of leukemia or the like. This was not possible with conventional techniques (method of combining plating with an adaptor or method of combining plating with the SMART method) due to the detection limit (about 1%).

Further, a V specific technology has low quantifiability due to varying amplification efficiency among V specific primers. However, this technology performs amplification with one set of primers, thus enabling highly precise quantification in the truest sense.

Further, since all TCRs or BCRs can be amplified with one set of primers, primers and containers required for amplification can be reduced to cut expenses.

Further, BCRs are characterized by having a mutation. Thus, a method using a V chain specific primer has disadvantages such as essentially being unable to perform amplification, or producing a gene with reduced amplification efficiency or the like. Meanwhile, the method of the present invention can also solve problems in BCRs.

Further, the analysis method using the present invention is advantageous in that it can complete the method in several minutes while conventional techniques complete overnight.

### <In Silico Associated Effect>

Significant differences from conventional and commonly used IMGT/High-V-QUEST include the following: IMGT/High-V-QUEST does not have a function for classifying a C region, and repertoire classification is either “unit of gene name” or “unit of allele” (i.e., (*) V (gene name)—D (gene name)—J (gene name) or V (allele)—D (allele)—J (allele)). Further, CDR3 classification is possible when performed separately from the above-described repertoire, but has no degree of freedom. On the other hand, the analysis method of the present invention can classify a C region and select “unit of gene name” or “unit of allele” for each region in repertoire classification. Further, CDR3 can also be used instead of D.

Further, in addition to the classification method of IMGT/High-V-QUEST, the present invention can also use combinations such as V (gene name)—D (allele)—J (allele), V (allele)—CDR3-J (allele) or the like. CDR3 can be used as a part of the above-described repertoire classification, or can also be classified individually. Further, the maximum number of sequences that can be processed in one batch is 150,000 in IMGT/High-V-Quest, while it is unlimited in the analysis method of the present invention. The time required for processing the same data is approximately 1/10 in the present system.

### <Effects Regarding Therapy>

The cancer idiotype peptide therapeutic method of the present invention is effective for patients when there is no specific marker (molecular target) effective for therapy in a target cancer cell or when there is no effect with therapy by an existing specific molecule targeting agent. That is, since a peptide is made based on genetic information of a cancer cell derived from an individual patient, an effect is exhibited on many tumors expressing a TCR or BCR. Lymphoma cells and leukemia cells, depending on their origins, have T cell based tumors and B cell based tumors. The present technique is applicable to each tumor form and useful in therapy of many patients. Further, when a B cell subpopulation developed into tumor is targeted, an antibody drug is used which targets a cell surface molecule expressed on a majority of B cells such as anti-CD20 antibodies. Such antibody drugs also act on normal B cells. Thus, such drugs act not only on cancer cells, but also on normal cells to induce a side effect such as decrease of immunological capability. Meanwhile, therapy targeting only cancer cells as in the present invention is highly safe. When a cancer peptide is used, highly safe therapy can be materialized by using a more highly specific peptide against cancer cells. Further, existing therapy using a cancer peptide is limited to patients with a specific HLA to which the peptide binds. Meanwhile, a peptide is designed based on genetic information of a patient as in the present invention. Thus, such a peptide is advantageous in being not limited by an HLA type and adaptable to a wide range of patients.

An existing CTL therapeutic method cocultures a lymphocyte of a patient with a tumor cell of the patient and an existing DC therapeutic method cocultures a DC cell of a patient with a tumor cell of the patient to induce a tumor specific killer T cell or a tumor specific DC. In addition, there is therapy using an artificial cancer antigen to stimulate a lymphocyte or DC cell and introduce the antigen into a patient to get an antitumor effect. As an antigen imparting specificity, use of a cancer antigen protein in comparison to the entire tumor cell, and a peptide in comparison to a protein is considered to be more effective and have fewer side effects. Unlike a protein, a peptide is advantageous in that a peptide can be chemically synthesized readily and directly based on genetic sequence information. Safety can be ensured since a peptide does not use biomaterials such as cell, medium, or infectious substance in the manufacturing process thereof. Safe therapy adapted to a wide range of patients can be materialized by designing individual peptides compatible with an HLA of a patient based on the genetic information of a cancer cell.

A synergistic effect is expected from introducing a tumor specific DC cell and CTL cell in a patient autoimmune therapeutic method. A CTL cell is expected to act as a cell already stimulated and activated by an antigen and exert an early therapeutic effect. Since a tumor specific DC cell induces a CTL cell in a patient introduced therewith, there is a sustained antitumor effect. Thus, a synergistic antitumor effect is expected from combined use of such different cells.

In cancer specific TCR gene therapy, it is important that expression of a target antigen is limited to cancer cells. An antigen localized in limited tissue such as cancer cells and testicular tissue, as in cancer-testicular antigens, is selected in therapy. However, it is known that such antigens are also expressed in some normal cells, which may be a safety related issue in therapy in some cases. Tailor-made cancer TCR gene therapy of the technique of the present invention identifies a T cell that infiltrates a patient's tumor tissue and utilizes a genetic sequence of a TCR thereof. Thus, a functional TCR considered to actually have antitumor action in a patient's body is utilized. Hence, a higher level of effect is expected. Further, since it is a T cell in a patient's body, it is highly likely that the action on normal cells is limited. Existing TCR gene therapy is limited to patients having a specific HLA and expressing a target cancer antigen. On the other hand, tailor-made therapy can make a TCR individually that is specific to a cancer antigen derived from a patient and compatible with a patient's HLA, such that therapy targeting a wider range of patients would be possible. Isolation of a cancer specific TCR gene with an in vitro stimulation is performed by stimulating a lymphocyte of a patient with an antigen protein, antigen peptide, inactivated cancer cell, idiotype peptide or the like. A TCR gene isolated via an experimental process for each patient is a TCR adapted to the HLA type of a patient, cancer cell form, cancer antigen species or other genetic background and is considered to be more effective in therapy.

## DESCRIPTION OF EMBODIMENTS

The present invention is described hereinafter. Throughout the entire specification, a singular expression should be understood as encompassing the concept thereof in the plural form unless specifically noted otherwise. Thus, singular articles (e.g., “a”, “an”, “the” and the like in case of English) should also be understood as encompassing the concept thereof in the plural form unless specifically noted otherwise. Further, the terms used herein should be understood as being used in the meaning that is commonly used in the art, unless specifically noted otherwise. Thus, unless defined otherwise, all terminologies and scientific technical terms that are used herein have the same meaning as the terms commonly understood by those skilled in the art pertaining to the present invention. In case of a contradiction, the present specification (including the definitions) takes precedence.

As used herein, “database” refers to any database related to genes and especially to a database comprising T cell receptor and B cell receptor repertoires in the present invention. Examples of such a database include, but are not limited to, IMGT (the international ImMunoGeneTics information system, www dot imgt dot org) database, DNA Data Bank of Japan (DDBJ, DNA Data Bank of Japan, www dot ddbj dot nig dot ac dot jp) database, GenBank (National Center for Biotechnology Information, www dot ncbi dot nlm dot nih dot gov/genbank/) database, ENA (EMBL (European Molecular Biology Laboratory), www dot ebi dot ac dot uk/ena) and the like.

As used herein, “genetic sequence analysis” or “gene sequencing” refers to analysis of a constituent nucleic acid sequence and/or amino acid sequence of a gene. “Genetic sequence analysis” or “gene sequencing” includes any analysis associated with a gene such as determination of a base or residue, determination of homology, determination of a domain, or determination of a latent function.

As used herein, “T cell receptor (TCR)” refers to a T cell receptor or a T cell antigen receptor, or a receptor expressed on a cell membrane of a T cell that regulates an immune system, and recognizes an antigen. There are α chain, β chain, γ chain and δ chain, constituting an αβ or γδ dimer. A TCR consisting of the former combination is called an αβ TCR and a TCR consisting of the latter combination is called a γδ TCR. T cells having such TCRs are called αβ T cell or γδ T cell. The structure is very similar to a Fab fragment of an antibody produced by a B cell, and recognizes an antigen molecule bound to an MHC molecule. Since a TCR gene of a mature T cell has undergone gene rearrangement, an individual has a diverse TCR and is able to recognize various antigens. A TCR further binds to an invariable CD3 molecule present in a cell membrane to form a complex. CD3 has an amino acid sequence called the ITAM (immunoreceptor tyrosine-based activation motif) in an intracellular region. This motif is considered to be involved in intracellular signaling. Each TCR chain is composed of a variable section (V) and a constant section (C). The constant section penetrates through the cell membrane and has a short cytoplasm portion. The variable section is present extracellularly and binds to an antigen-MHC complex. The variable section has three regions called a hypervariable section or a complementarity determining region (CDR), which binds to an antigen-MHC complex. The three CDRs are each called CDR1, CDR2, and CDR3. For a TCR, CDR1 and CDR2 are considered to bind to an MHC, while CDR3 is considered to bind to an antigen. Gene rearrangement of a TCR is similar to the process for a B cell receptor known as an immunoglobulin. In gene rearrangement of an αβ TCR, VDJ rearrangement of a β chain is first performed and then VJ rearrangement of an α chain is performed. Since a gene of a δ chain is deleted from a chromosome in rearrangement of an α chain, a T cell having an α3 TCR would not simultaneously have a γδ TCR. In contrast, in a T cell having a γδ TCR, a signal mediated by this TCR suppresses expression of a 13, chain. Thus, a T cell having a γδ TCR would not simultaneously have an αβ TCR.

As used herein, “B cell receptor (BCR)” is also called a B cell receptor or B cell antigen receptor and refers to those composed of an Igα/Igβ (CD79a/CD79b) heterodimer (α/β) conjugated with a membrane-bound immunoglobulin (mIg). An mIg subunit binds to an antigen to induce aggregation of the receptors, while an α/β subunit transmits a signal to the inside of a cell. BCRs, when aggregated, are understood to quickly activate Lyn, Blk, and Fyn of Src family kinases as in Syk and Btk of tyrosine kinases. Results greatly differ depending on the complexity of BCR signaling, the results including survival, resistance (allergy; lack of hypersensitivity reaction to antigen) or apoptosis, cell division, differentiation into antibody-producing cell or memory B cell and the like. Several hundred million types of T cells with a different TCR variable region sequence are produced and several hundred million types of B cells with a different BCR (or antibody) variable region sequence are produced. Individual sequences of TCRs and BCRs vary due to an introduced mutation or rearrangement of the genomic sequence. Thus, it is possible to obtain a clue for antigen specificity of a T cell or a B cell by determining a genomic sequence of TCR/BCR or a sequence of an mRNA (cDNA).

As used herein, “V region” refers to a variable section (V) of a variable region of a TCR chain or a BCR chain.

As used herein, “D region” refers to a D region of a variable region of a TCR chain or a BCR chain.

As used herein, “J region” refers to a J region of a variable region of a TCR chain or a BCR chain.

As used herein, “C region” refers to a constant section (C) region of a TCR chain or a BCR chain.

As used herein, “repertoire of a variable region” refers to a collection of V(D)J regions created in any manner by gene rearrangement in a TCR or BCR. The terms such as TCR repertoire and BCR repertoire are used, which are also called, for example, T cell repertoire, B cell repertoire or the like in some cases. For instance, “T cell repertoire” refers to a collection of lymphocytes characterized by expression of a T cell receptor (TCR) serving an important role in antigen recognition. A change in a T cell repertoire provides a significant indicator of an immune status in a physiological condition and disease condition. Thus, a T cell repertoire has been analyzed to identify an antigen specific T cell involved in the pathology of a disease and diagnosis of abnormality in T lymphocytes. Comparison of the variable region usage by fluorescence activated cell sorter analysis which uses a larger panel of a TCR variable region specific antibody (van den Beemd R et al. (2000) Cytometry 40: 336-345; MacIsaac C et al. (2003) J Immunol Methods 283: 9-15; Tembhare P et al. (2011) Am J Clin Pathol 135: 890-900; Langerak A W et al. (2001) Blood 98: 165-173), by polymerase chain reaction (PCR) using multiple primers (Rebai N et al. (1994) Proc Natl Acad Sci USA 91: 1529-1533), or by enzyme-linked immunosorbent assay based on PCR (Matsutani T et al. (1997) Hum Immunol 56: 57-69; Matsutani T et al. (2000) Br J Haematol 109: 759-769) have been extensively used to detect a change in a T cell repertoire. Analysis of a chain length distribution known as CDR3 spectratyping is based on the addition of a nontemplate nucleotide in a V-(D)-J region and has been used to assess clonality and diversity of T cells (Matsutani T et al. (2007) Mol Immunol 44: 2378-2387; Matsutani T et al. (2011) Mol Immunol 48: 623-629). To further identify the antigen specificity of a T cell, PCR cloning of a TCR clone type and subsequence sequencing of an antigen recognition region and CDR3 were required. Such conventional approach is commonly used. However, this is a time and labor intensive method for researching a TCR repertoire.

As used herein, “quantitative analysis” refers to analysis that is quantitative in nature. In the present invention, “quantitative analysis” refers to analysis in a form reflecting the amount of each clone that was originally present in repertoire analysis.

As used herein, “sample” includes, but is not limited to, components derived from a subject (body fluid such as blood or the like).

As used herein, “complementary DNA” refers to a DNA forming a complementary strand with respect to a target nucleic acid molecule, e.g., RNA included in an RNA sample or the like derived from a target cell.

As used herein, “double stranded complementary DNA” refers to DNAs that are complementary to each other and form a double strand. In the present invention, this can be produced, for example, with a complementary DNA forming a complementary strand with respect to an RNA included in an

RNA sample or the like derived from a target cell as a template.

As used herein, “common adaptor primer sequence” refers to a sequence of a portion added in common to all sequences in an adaptor-added double stranded complementary DNA used as a primer in the first PCR amplification reaction and the second PCR amplification reaction of the present invention.

As used herein, “adaptor-added double stranded complementary DNA” refers to a DNA used as a primer in the first PCR in the present invention, wherein a common adaptor primer sequence is added to various double stranded complementary DNAs in a sample. This is used as a template in the first primer amplification reaction.

As used herein, “common adaptor primer” refers to a DNA used as a primer in the first PCR reaction and the second PCR amplification reaction of the present invention, wherein a single common sequence is used in each reaction.

As used herein, “first TCR or BCR C region specific primer” refers to a primer used in the first PCR amplification reaction of the present invention, comprising a sequence specific to a C region of a TCR or a BCR.

FIG. 18 shows an example of a possible primer setting region of TRAC on the top row (target sequence is an artificially spliced functional TRAC exon region sequence, consisting of exons EX1, EX2, and EX3; and a primer can be set throughout the entire length). The bottom row shows an example of a possible primer setting region of TRBC (target sequence is an artificially spliced functional TRBC exon region sequence, consisting of exons EX1, EX2, EX3 and EX4; and a primer can be set throughout the entire length). In addition, a first TCR or BCR C region specific primer can be set on the most 5′ terminal side of a complementary DNA. Once a first TCR or BCR C region specific primer is set, a second TCR or BCR C region specific primer can be set downstream thereof. Furthermore, once the second TCR or BCR C region specific primer is set, a third TCR or BCR C region specific primer can be set. That is, when the first is set, the second is downstream thereof, and the third is further downstream. Theoretically, it is understood that the primers only need to be downstream by the length of the primer.

FIG. 19 shows an example of a possible primer setting region of TRGC on the top row (target sequence is an artificially spliced functional TRGC exon region sequence, consisting of exons EX1, EX2, and EX3; and a primer can be set throughout the entire length). The bottom row shows an example of a possible primer setting region of TRDC (target sequence is an artificially spliced functional TRDC exon region sequence, consisting of exons EX1, EX2, and EX3; and a primer can be set throughout the entire length). In addition, a first TCR or BCR C region specific primer can be set on the most 5′ terminal side of a complementary DNA. Once a first TCR or BCR C region specific primer is set, a second TCR or BCR C region specific primer can be set downstream thereof. Furthermore, once the second TCR or BCR C region specific primer is set, a third TCR or BCR C region specific primer can be set. That is, when the first is set, the second is downstream thereof, and the third is further downstream. Theoretically, it is understood that the primers only need to be downstream by the length of the primer.

FIG. 20 shows an example of a possible primer setting region of IGHM (target sequence is an artificially spliced functional IGHM exon region sequence, consisting of exons CH1, CH2, CH3, and CH4; and a primer can be set throughout the entire length). In addition, a first TCR or BCR C region specific primer can be set on the most 5′ terminal side of a complementary DNA. Once a first TCR or BCR C region specific primer is set, a second TCR or BCR C region specific primer can be set downstream thereof. Furthermore, once the second TCR or BCR C region specific primer is set, a third TCR or BCR C region specific primer can be set. That is, when the first is set, the second is downstream thereof, and the third is further downstream. Theoretically, it is understood that the primers only need to be downstream by the length of the primer.

FIG. 21 shows an example of a possible primer setting region of IGHA (target sequence is an artificially spliced functional IGHA exon region sequence. A secreted form consists of exons CH1, H, CH2, CH3, and CH-S, and a membrane bound form consists of CH1, H, CH2, CH3, M1 and M2. A primer can be set throughout the entire length). In addition, a first TCR or BCR C region specific primer can be set on the most 5′ terminal side of a complementary DNA. Once a first TCR or BCR C region specific primer is set, a second TCR or BCR C region specific primer can be set downstream thereof. Furthermore, once the second TCR or BCR C region specific primer is set, a third TCR or BCR C region specific primer can be set. That is, when the first is set, the second is downstream thereof, and the third is further downstream. Theoretically, it is understood that the primers only need to be downstream by the length of the primer.

FIG. 22 shows an example of a possible primer setting region of IGHG (target sequence is an artificially spliced functional IGHG exon region sequence. A secreted form consists of exons CH1, H (H1, H2, H3, H4), CH2, CH3, and CH-S, and a membrane bound form consists of CH1, H (H1, H2, H3, H4), CH2, CH3, M1 and M2. A primer can be set throughout the entire length). In addition, a first TCR or BCR C region specific primer can be set on the most 5′ terminal side of a complementary DNA. Once a first TCR or BCR C region specific primer is set, a second TCR or BCR C region specific primer can be set downstream thereof. Furthermore, once the second TCR or BCR C region specific primer is set, a third TCR or BCR C region specific primer can be set. That is, when the first is set, the second is downstream thereof, and the third is further downstream. Theoretically, it is understood that the primers only need to be downstream by the length of the primer.

FIG. 23 shows an example of a possible primer setting region of IGHD (target sequence is an artificially spliced functional IGHD exon region sequence. A secreted form consists of exons CH1, H1, H2, CH2, CH3, and CH-S, and a membrane bound form consists of CH1, H1, H2, CH2, CH3, M1 and M2. A primer can be set throughout the entire length). In addition, a first TCR or BCR C region specific primer can be set on the most 5′ terminal side of a complementary DNA. Once a first TCR or BCR C region specific primer is set, a second TCR or BCR C region specific primer can be set downstream thereof. Furthermore, once the second TCR or BCR C region specific primer is set, a third TCR or BCR C region specific primer can be set. That is, when the first is set, the second is downstream thereof, and the third is further downstream. Theoretically, it is understood that the primers only need to be downstream by the length of the primer.

FIG. 24 shows an example of a possible primer setting region of IGHE (target sequence is an artificially spliced functional IGHE exon region sequence. A secreted form consists of exon CH1, exon CH2, exon CH3, and CH-S, and a membrane bound form consists of CH1, exon CH2, exon CH3, M1 and M2. A primer can be set throughout the entire length). In addition, a first TCR or BCR C region specific primer can be set on the most 5′ terminal side of a complementary DNA. Once a first TCR or BCR C region specific primer is set, a second TCR or BCR C region specific primer can be set downstream thereof. Furthermore, once the second TCR or BCR C region specific primer is set, a third TCR or BCR C region specific primer can be set. That is, when the first is set, the second is downstream thereof, and the third is further downstream. Theoretically, it is understood that the primers only need to be downstream by the length of the primer.

FIG. 25 shows an example of a possible primer setting region of IGKC on the top row (target sequence is a functional IGKC CL sequence. A primer can be set throughout the entire length). The bottom row shows an example of a possible primer setting region of IGLC (target sequence is a functional IGLC CL sequence A primer can be set throughout the entire length). In addition, a first TCR or BCR C region specific primer can be set on the most 5′ terminal side of a complementary DNA. Once a first TCR or BCR C region specific primer is set, a second TCR or BCR C region specific primer can be set downstream thereof. Furthermore, once the second TCR or BCR C region specific primer is set, a third TCR or BCR C region specific primer can be set. That is, when the first is set, the second is downstream thereof, and the third is further downstream. Theoretically, it is understood that the primers only need to be downstream by the length of the primer.

Specifically, a first TCR or BCR C region specific primer has the following structure: for BCRs, CM1 (SEQ ID NO: 5), CA1 (SEQ ID NO: 8), CG1 (SEQ ID NO: 11), CD1 (SEQ ID NO: 14), or CE1 (SEQ ID NO: 17), and for TCRs, CA1 (SEQ ID NO: 35) or CB1 (SEQ ID NO: 37) or the like. However, the structure is not limited thereto. Such a primer sequence can be set in, but is not limited to, the following specific ranges. The first, second and third ranges can be set in the entire range, but can be mutually determined.

α sequence of TCR: base number 213 to base number 235 of SEQ ID NO: 1376 (FIG. 18)

β sequence of TCR: base number 278 to base number 300 of SEQ ID NO: 1377 (FIG. 18)

γ sequence of TCR: base number 184 to base number 201 of SEQ ID NO: 1378 (FIG. 19)

δ sequence of TCR: base number 231 to base number 249 of SEQ ID NO: 1379 (FIG. 19)

IgM heavy chain sequence of BCR: base number 77 to base number 95 of SEQ ID NO: 1380 (FIG. 20)

IgA heavy chain sequence of BCR: base number 189 to base number 208 of SEQ ID NO: 1381 (FIG. 21)

IgG heavy chain sequence of BCR: base number 262 to base number 282 of SEQ ID NO: 1382 (FIG. 22)

IgD heavy chain sequence of BCR: base number 164 to base number 183 of SEQ ID NO: 1383 (FIG. 23)

IgE heavy chain sequence of BCR: base number 182 to base number 199 of SEQ ID NO: 1384 (FIG. 24)

Igκ chain constant region sequence of BCR: base number 230 to base number 248 of SEQ ID NO: 1385 (FIG. 25)

Igλ chain sequence of BCR: base number 273 to base number 291 of SEQ ID NO: 1386 (FIG. 25)

As used herein, “specific” refers to binding to a target sequence, but binding poorly and preferably not binding to other sequences at least in a pool of target TCRs or BCRs and preferably in all sequences of TCRs or BCRs that are present. A specific sequence would be advantageously and preferably, but not necessarily limited to, fully complementary to a target sequence.

As used herein, “sufficiently specific (to a C region of interest)” refers to having sufficient specificity for a gene amplification reaction. The same sequence as a target C region would be advantageous and preferable, but it is not necessarily limited thereto.

As used herein, “first PCR amplification reaction” is a PCR amplification reaction performed in the first stage of the method of preparing a sample of the present invention.

As used herein, “not homologous with other genetic sequences” refers to having low homology to such an extent that a gene amplification reaction does not occur with sequences other than the sequence of interest (e.g., C region of interest of a TCR or BCR).

As used herein, “comprising a mismatching base (between subtypes) downstream” refers to comprising a mismatching base between subtypes, downstream of a sequence that is used when set as a primer. An amplicon would have a different sequence for each subtype by setting such a sequence. Thus, a subtype can be identified by determining a sequence.

As used herein, “second TCR or BCR C region specific primer” refers to a primer used in the second PCR amplification reaction of the present invention, comprising a sequence specific to a C region of a TCR or BCR. A second TCR or BCR C region specific primer is designed to have a sequence that is a complete match with the TCR or BCR C region in a sequence downstream the sequence of the first TCR or BCR C region specific primer, but comprise a sequence that is not homologous with other genetic sequences, and comprise a mismatching base between subtypes downstream when amplified. Examples of such a sequence include, but are not limited to, CM2 (SEQ ID NO: 6), CA2 (SEQ ID NO: 9), CG2 (SEQ ID NO: 12), CD2 (SEQ ID NO: 15), and CE2 (SEQ ID NO: 18) for BCRs, and CA2 (SEQ ID NO: 36) and CB2 (SEQ ID NO: 38) for TCRs and the like. Such a primer sequence can be set in, but is not limited to, the following specific ranges. The first, second and third ranges can be set in the entire range, but can be mutually determined. That is, when the first is set, the second is downstream thereof, and the third is further downstream. Theoretically, it is understood that the primers only need to be downstream by the length of the primer.

α sequence of TCR: base number 146 to base number 168 of SEQ ID NO: 1376 (FIG. 18), β sequence of TCR: base number 205 to base number 227 of SEQ ID NO: 1377 (FIG. 18), γ sequence of TCR: base number 141 to base number 160 of SEQ ID NO: 1378 (FIG. 19), δ sequence of TCR: base number 135 to base number 155 of SEQ ID NO: 1379 (FIG. 19), IgM heavy chain sequence of BCR: base number 43 to base number 62 of SEQ ID NO: 1380 (FIG. 20), IgA heavy chain sequence of BCR: base number 141 to base number 161 of SEQ ID NO: 1381 (FIG. 21), IgG heavy chain sequence of BCR: base number 163 to base number 183 of SEQ ID NO: 1382 (FIG. 22), IgD heavy chain sequence of BCR: base number 125 to base number 142 of SEQ ID NO: 1383 (FIG. 23), IgE heavy chain sequence of BCR: base number 155 to base number 173 of SEQ ID NO: 1384 (FIG. 24), Igκ chain constant region sequence of BCR: base number 103 to base number 120 of SEQ ID NO: 1385 (FIG. 25), Igλ chain sequence of BCR: base number 85 to base number 100 of SEQ ID NO: 1386 (FIG. 25).

As used herein, “second PCR amplification reaction” refers to a PCR amplification reaction performed in a nested form after the first PCR reaction by using a product of the first PCR reaction as a template in the sample production for analysis of the present invention. In the present invention, the amplification reaction is performed by using a common adaptor primer and a second TCR or BCR C region specific primer. In this regard, the second TCR or BCR C region specific primer is designed to have a sequence that is a complete match with the TCR or BCR C region in a sequence downstream the sequence of the first TCR or BCR C region specific primer, but comprise a sequence that is not homologous with other genetic sequences, and comprise a mismatching base between subtypes downstream when amplified.

As used herein “third PCR amplification reaction” is a PCR amplification reaction performed after a second nested PCR reaction by using a product of the second nested PCR reaction as a template in the sample production for analysis of the present invention, where a product thereof is used in the sample production for analysis of the present invention. A third PCR amplification reaction is performed, after the second nested PCR by using a product of a second nested PCR reaction as a template, by using an added common adaptor primer in which a nucleic acid sequence of the common adaptor primer comprises a first additional adaptor nucleic acid sequence, and an adaptor-added third TCR or BCR C region specific primer in which a second additional adaptor nucleic acid sequence and a molecule identification sequence (MID Tag sequence) are added to a third TCR or BCR C region specific sequence. An adaptor-added third TCR or BCR C region specific primer may comprise a sequence for verifying a nucleic acid sequence position called a key sequence. Specific example of the added common adaptor primer that can be used includes Adaptor B (SEQ ID NO: 1375)-TAATACGACTCCGAATTCCC, and specific examples of an adaptor-added third TCR or BCR C region specific primer that are used include Adaptor A (SEQ ID NO: 39)-key(TCAG)-MID1-(SEQ ID NO: 40)AAAGGGTTGGGGCGGATGC (SEQ ID NO: 1387) (entire primer is SEQ ID NO: 7), Adaptor A (SEQ ID NO: 39)-key(TCAG)-MID2 (SEQ ID NO: 41)-CCGCTTTCGCTCCAGGTCAC (SEQ ID NO: 1388) (entire primer is SEQ ID NO: 10), Adaptor A (SEQ ID NO: 39)-key(TCAG)-MID3 (SEQ ID NO: 42)-TGAGTTCCACGACACCGTCAC (SEQ ID NO: 1389) (entire primer is SEQ ID NO: 13), Adaptor A (SEQ ID NO: 39)-key(TCAG)-MID4 (SEQ ID NO: 43)-CCCAGTTATCAAGCATGCC (SEQ ID NO: 1390) (entire sequence is SEQ ID NO: 16), Adaptor A (SEQ ID NO: 39)-key(TCAG)-MID5 (SEQ ID NO: 44)-CATTGGAGGGAATGTTTTTG (SEQ ID NO: 1391) (entire primer is SEQ ID NO: 19) and the like.

As used herein, “first additional adaptor nucleic acid sequence” is a sequence added to a primer used in the third PCR amplification reaction of the present invention, which is added to a nucleic acid sequence of a common adaptor primer for use. A first additional adaptor nucleic acid sequence may be different from or the same as a second additional adaptor nucleic acid sequence. As for the characteristic of such a sequence, such a nucleic acid sequence is a sequence suitable for bonding to a DNA capturing bead and an emPCR reaction (for example, see Chee-Seng, Ku; En Yun, Loy; Yudi, Pawitan; and Kee-Seng, Chia. Next Generation Sequencing Technologies and Their Applications. In: Encyclopedia of Life Sciences (ELS). John Wiley & Sons, Ltd: Chichester. April 2010; Metzker M L. Sequencing technologies—the next generation. Nat Rev Genet. 2010 January; 11(1): 31-46). Any sequence may be used as long as the sequence has such a characteristic. Specifically, CCTATCCCCTGTGTGCCTTGGCAGTC (SEQ ID NO: 1375) is used, but the sequence is not limited thereto.

As used herein, “second addition adaptor nucleic acid sequence” is a sequence added to a primer used in the third PCR amplification reaction of the present invention, wherein the sequence is optionally used with a molecule identifying sequence (e.g., (MID Tag sequence)) and/or a key sequence and is added to a third TCR or BCR C region specific sequence to constitute an adaptor-added third TCR or BCR C region specific primer. A second additional adaptor nucleic acid sequence may be different from or the same as a second additional adaptor nucleic acid sequence. As for the characteristic of such a sequence, such a nucleic acid sequence is a sequence suitable for an emPCR reaction, and any sequence may be used as long as the sequence has such a characteristic. Specifically, CCATCTCATCCCTGCGTGTCTCCGAC (SEQ ID NO: 39) is used, but the sequence is not limited thereto.

A “key sequence” as used herein is a sequence added to a primer used in the third PCR amplification reaction of the present invention, wherein the sequence is optionally used with a molecule identifying sequence (e.g., (MID Tag sequence)) and is added to a third TCR or BCR C region specific sequence to constitute an adaptor-added third TCR or BCR C region specific primer. Such a key sequence may be any sequence as long as a nucleic acid sequence position can be verified. A key sequence with 4 bases (TCAG) is used, but a key sequence is not limited thereto.

As used herein, “molecule identifying (MID Tag) sequence” is a sequence for imparting uniqueness such that an amplicon can be identified. Thus, it is preferably different from a sequence of interest. Further, it is preferably a sequence that does not affect amplification. Examples of such a sequence include, but are not limited to, sequences of SEQ ID NOs: 1325-1374. The baseline of determination for an identification sequence (tag sequence) and representative examples thereof are the following. Specifically, the baseline of determination of a tag sequence is explained as follows. A tag sequence is a base sequence added to distinguish each sample when a plurality of samples are mixed and simultaneously sequenced. A read from one sample corresponds to one tag sequence. Thus, it is possible to identify which sample is derived from an acquired read sequence. A tag sequence is any sequence of 4 types of bases A, C, G, and T. Theoretically, about a million types of sequences can be created with 10 bases and about a trillion types of sequences with 20 bases. The length of a base sequence is preferably between 2 and 40 bases, and more preferably between 6 and 10 bases. At the same time, it is desirable to use a sequence that is free of a consecutive sequence (AA, CC, GG, or TT). Representative tags that can be used herein are, but not limited to, the following: ACGAGTGCGT (SEQ ID NO: 1325), ACGCTCGACA (SEQ ID NO: 1326), AGACGCACTC (SEQ ID NO: 1327), AGCACTGTAG (SEQ ID NO: 1328), ATCAGACACG (SEQ ID NO: 1329), ATATCGCGAG (SEQ ID NO: 1330), CGTGTCTCTA (SEQ ID NO: 1331), CTCGCGTGTC (SEQ ID NO: 1332), TAGTATCAGC (SEQ ID NO: 1333), TCTCTATGCG (SEQ ID NO: 1334), TGATACGTCT (SEQ ID NO: 1335), TACTGAGCTA (SEQ ID NO: 1336), CATAGTAGTG (SEQ ID NO: 1337), CGAGAGATAC (SEQ ID NO: 1338), ATACGACGTA (SEQ ID NO: 1339), TCACGTACTA (SEQ ID NO: 1340), CGTCTAGTAC (SEQ ID NO: 1341), TCTACGTAGC (SEQ ID NO: 1342), TGTACTACTC (SEQ ID NO: 1343), ACGACTACAG (SEQ ID NO: 1344), CGTAGACTAG (SEQ ID NO: 1345), TACGAGTATG (SEQ ID NO: 1346), TACTCTCGTG (SEQ ID NO: 1347), TAGAGACGAG (SEQ ID NO: 1348), TCGTCGCTCG (SEQ ID NO: 1349), ACATACGCGT (SEQ ID NO: 1350), ACACGACGACT (SEQ ID NO: 1351), ACACGTAGTAT (SEQ ID NO: 1352), ACACTACTCGT (SEQ ID NO: 1353), ACGACACGTAT (SEQ ID NO: 1354), ACGAGTAGACT (SEQ ID NO: 1355), ACGCGTCTAGT (SEQ ID NO: 1356), ACGTACACACT (SEQ ID NO: 1357), ACGTACTGTGT (SEQ ID NO: 1358), ACGTAGATCGT (SEQ ID NO: 1359), ACTACGTCTCT (SEQ ID NO: 1360), ACTATACGAGT (SEQ ID NO: 1361), ACTCGCGTCGT (SEQ ID NO: 1362), AGTCGTGGTGT (SEQ ID NO: 1363), ATACTAGGTGT (SEQ ID NO: 1364), ACGAGTGGTGT (SEQ ID NO: 1365), ATACGTGGCGT (SEQ ID NO: 1366), AGTCTACGCGT (SEQ ID NO: 1367), ACTAGAGGCGT (SEQ ID NO: 1368), AGTGTGTGCGT (SEQ ID NO: 1369), ACACAGTGCGT (SEQ ID NO: 1370), ACGATCTGCGT (SEQ ID NO: 1371), AGAGACGGAGT (SEQ ID NO: 1372), ACTCGTAGAGT (SEQ ID NO: 1373), and ACGACGGGAGT (SEQ ID NO: 1374).

As used herein, “third TCR or BCR C region specific sequence” is a sequence specific to a C region of a TCR or a BCR, wherein the sequence is present more downstream of a first TCR or BCR C region specific sequence and a second TCR or BCR C region specific sequence. It is a sequence used for constituting a third TCR or BCR C region specific primer. Specific examples thereof include the sequence of a specific portion in CM3-GS (SEQ ID NO: 1387), sequence of a specific portion in CA3-GS (SEQ ID NO: 1388), sequence of a specific portion in CG3-GS (SEQ ID NO: 1389), sequence of a specific portion in CD3-GS (SEQ ID NO: 1390), and sequence in a specific portion in CE3-GS (SEQ ID NO: 1391) for BCRs, specific sequences in HuVaF or HuVbF in Table 6 (SEQ ID NOs: 40-60) for TCRs and the like (corresponding to base number 51 to base number 73 of SEQ ID NO: 1376 (FIG. 18); base number 69 to base number 91 of SEQ ID NO: 1377 (FIG. 18)). More specifically, such a primer sequence can be set in a specific range such as the following, but the sequence is not limited thereto. The first, second and third ranges can be set in the entire range, but can be mutually determined. That is, when the first is set, the second is downstream thereof, and the third is further downstream. Theoretically, it is understood that the primers only need to be downstream by the length of the primer.

α sequence of TCR: base number 51 to base number 73 of SEQ ID NO: 1376 (FIG. 18)

β sequence of TCR: base number 69 to base number 91 of SEQ ID NO: 1377 (FIG. 18)

γ sequence of TCR: base number 34 to base number 53 of SEQ ID NO: 1378 (FIG. 19)

δ sequence of TCR: base number 61 to base number 78 of SEQ ID NO: 1379 (FIG. 19)

IgM heavy chain sequence of BCR: base number 7 to base number 25 of SEQ ID NO: 1380 (FIG. 20)

IgA heavy chain sequence of BCR: base number 115 to base number 134 of SEQ ID NO: 1381 (FIG. 21)

IgG heavy chain sequence of BCR: base number 109 to base number 129 of SEQ ID NO: 1382 (FIG. 22)

IgD heavy chain sequence of BCR: base number 78 to base number 96 of SEQ ID NO: 1383 (FIG. 23)

IgE heavy chain sequence of BCR: base number 45 to base number 64 of SEQ ID NO: 1384 (FIG. 24)

Igκ chain constant region sequence of BCR: base number 75 to base number 92 of SEQ ID NO: 1385 (FIG. 25)

Igλ chain sequence of BCR: base number 52 to base number 69 of SEQ ID NO: 1386 (FIG. 25) (this SEQ ID NO is also used for CM).

As used herein, “third TCR or BCR C region specific primer” is a primer used in the third PCR amplification reaction of the present invention, which is designed to have a sequence that is a complete match with the TCR or BCR C region in a sequence downstream to the sequence of the second TCR or BCR C region specific primer, but comprise a sequence that is not homologous with other genetic sequences, and comprise a mismatching base between subtypes downstream when amplified. The primer further comprises an adaptor sequence, a key sequence, and an identification sequence. Specific examples thereof include, but are not limited to, CM3-GS (SEQ ID NO: 7), CA3-GS (SEQ ID NO: 10), CG3-GS (SEQ ID NO: 13), CD3-GS (SEQ ID NO: 16) and CE3-GS (SEQ ID NO: 19). Any sequence that can be set as a third TCR or BCR C region specific sequence mentioned above can be used.

As used herein, “isotype” refers to IgM, IgA, IgG, IgE, IgD or the like that belongs to the same type but have a difference sequence from one another. Isotypes can be denoted by using various abbreviations or symbols of genes.

As used herein, “subtype” is a type within a type present in IgA and IgG for BCRs. There are IgG1, IgG2, IgG3, and IgG4 for IgG and IgA1 and IgA2 for IgA. It is also known to be present in β and γ chains for TCRs, which are TRBC1 and TRBC2 and TRGC1 and TRGC2, respectively.

As used herein, “complete match” refers to 100% identity when sequences are compared to each other.

As used herein, “complete match with all C region allelic sequences of the same isotype” refers to a match with all sequences for C region allelic sequences of the same isotype when aligned. Since all sequences in a C region would never be identical even in the same isotype, use of a sequence that is a complete match with all C region allelic sequences of the same isotype would be advantageous for immediately determining an isotype when a sequence of an amplicon is determined.

As used herein, “unlikely to have a homodimer and intramolecular hairpin structures” refers to a state of a nucleic acid molecule, especially a common adaptor primer, where a sequence is unlikely to form a dimer due to pairing with a complementary strand or the like or is unlikely to form a hairpin structure or the like due to pairing with a complementary strand in a molecule. “Unlikely” allows for a degree of homodimer or hairpin that does not substantially affect the subsequent analysis, referring to, for example, tolerance of about 10% or less, 5% or less, 1% or less, 0.5% or less, 0.1% or less, 0.05% or less, or 0.01% or less of the whole. Such a sequence can be determined by using a known technology in the art (Santa Lucia, J. Proc Natl Acad Sci USA, 95 (4): 1460-1465. (1998), Bommarito et al., Nucleic Acids Res, 28 (9): 1929-1934. (2000), Santa Lucia, J. Proc Natl Acad Sci USA, 95 (4): 1460-1465. (1998), and von Ahsen et al., ClinChem, 47 (11): 1956-1961. (2001)), for example, by a commercially available computer program or the like used in the Examples (CLC Main Workbench or Primer3).

As used herein, “not . . . have homodimer and intramolecular hairpin structures” refers to a state of a nucleic acid molecule, especially a common adaptor primer, where a sequence does not form a dimer due to pairing with a complementary strand or the like nor form a hairpin structure or the like due to pairing with a complementary strand in a molecule. Such a sequence can be determined by using a known technology in the art (Santa Lucia, J. Proc Natl Acad Sci USA, 95(4): 1460-1465. (1998), Bommarito et al., Nucleic Acids Res, 28 (9): 1929-1934. (2000), Santa Lucia, J. Proc Natl Acad Sci USA, 95 (4): 1460-1465. (1998), and von Ahsen et al., ClinChem, 47 (11): 1956-1961. (2001)), for example, by a commercially available computer program or the like used in the Examples (CLC Main Workbench or Primer3).

As used herein, a structure that “can stably form a double strand” refers to a nucleic acid molecule, especially a common adaptor primer, where a double strand, when formed with another nucleic acid molecule such as a template, forms the strand stably. Such stability can be assessed mainly by temperature, pH, melting temperature (Tm) calculated from base composition, pHm or structure stabilizing energy (−ΔG37° C.). Such a sequence can be determined by using a known technology in the art (Santa Lucia, J. Proc Natl Acad Sci USA, 95(4): 1460-1465. (1998), Bommarito et al., Nucleic Acids Res, 28(9): 1929-1934. (2000), Santa Lucia, J. Proc Natl Acad Sci USA, 95(4): 1460-1465. (1998), and von Ahsen et al., Clin Chem, 47(11): 1956-1961. (2001)), for example, by a commercially available computer program or the like used in the Examples (CLC Main Workbench or Primer3).

As used herein, “not highly homologous” refers to a nucleic acid molecule, especially a common adaptor primer, with a feature of having homology that is not high with all the TCR genetic sequences in a database in order to enhance identifiability. For sufficient analysis, the level of homology is preferably, for example, 80% or less, 70% or less, 60% or less, 50% or less, 40% or less, 30% or less, 25% or less, 20% or less, 15% or less or 10% or less.

As used herein “same level of melting temperature (Tm)” refers to a DNA melting temperature (Tm) of a sequence or a primer to be used being substantially the same, which is a preferred condition for a suitable PCR amplification reaction. “Same level” may refer to Tm being ±15° C. or less, ±14° C. or less, ±13° C. or less, ±12° C. or less, ±11° C. or less, ±10° C. or less, ±9° C. or less, ±8° C. or less, ±7° C. or less, ±6° C. or less, ±5° C. or less, ±4° C. or less, ±3° C. or less, ±2° C. or less, ±1° C. or less, or ±0.5° C. or less. The Examples are able to carry out the present invention with a difference of 10.9° C. Thus, it is understood that about 15° C. or less is acceptable as the same level. Tm is a temperature at which 50% of a DNA molecule denatures to be a single strand. Tm can be identified with a known technology in the art. For example, Tm can be found as follows (a) for an oligonucleotide shorter than 18b: Tm=(A+T)×2° C.+(G+C)×4° C., (b) for an oligonucleotide with a length of 18 b or more: Tm=81.5+16.6 (log 10[Na+])+0.41 (% G+C)—(600/N), (*A: number of As in oligonucleotide, C: number of Cs in oligonucleotide, G: number of Gs in oligonucleotide, T: number of T in oligonucleotide, % G+C: % of G+C in oligonucleotide, N: length (mer) of oligonucleotide, [Na+]: Na+ concentration in solution (M)).

As used herein, “base length suitable for amplification” refers to a length of a primer or a sequence that is used, which is suitable for an amplification reaction. Such a length can be found, for example, by a commercially available computer program or the like used in the Examples (CLC Main Workbench or Primer3). The documents such as the following can also be referred: Santa Lucia, J. Proc Natl Acad Sci USA, 95 (4): 1460-1465. (1998), Bommarito et al., Nucleic Acids Res, 28 (9): 1929-1934. (2000), Santa Lucia, J. Proc Natl Acad Sci USA, 95 (4): 1460-1465. (1998), and von Ahsen et al., Clin Chem, 47 (11): 1956-1961. (2001).

As used herein, “mismatch” refers to the presence of bases that are not identical with each other when genetic sequences are aligned.

As used herein, “% GC (% guanine.cytosine content)” refers to the percentage of G (guanine), C (cytosine) in a nucleic acid sequence with respect to the entire base (including A (adenosine), T (thymine) and U (uracil)). High percentage thereof results in a higher melting temperature and is also related to gene density or band structure of chromosomes.

As used herein, “set compatible with all TCR or BCR subclasses” refers to primers prepared in accordance with the descriptions herein for all known subclasses (refers to TRBC1, TRBC2, or TRGC1, TRGC2 or the like for TCRs, or IgG1, IgG2, IgG3 or IgG4 for IgG, IgA1 or IgA2 for IgA or the like for BCRs) of a target TCR or BCR.

As used herein, “protein”, “polypeptide”, “oligopeptide” and “peptide” are used to have the same meaning and refer to an amino acid polymer of any length. Such a polymer may be a branched or straight chain or annular. An amino may be a natural or non-natural or altered amino acid. The term may also encompass those assembled into a complex of a plurality of polypeptide chains. The term also encompasses natural or artificially altered amino acid polymers. Examples of such an alteration include disulfide bond formation, glycosylation, lipidation, acetylation, phosphorylation or any other manipulation or alteration (e.g., conjugation with a labeling component). The definition also encompasses, for example, a polypeptide comprising one or more analogs of an amino acid (e.g., including a non-natural amino acid, etc.), peptide-like compound (e.g., peptoid), and other known alterations in the art.

As used herein, “amino acid” may be natural or non-natural as long as the objective of the present invention is met.

As used herein, “polynucleotide”, “oligonucleotide” and “nucleic acid” are used in the same meaning and refer to a nucleotide polymer with any length. The term also encompasses “oligonucleotide derivative” and “polynucleotide derivative”. “Oligonucleotide derivative” or “polynucleotide derivative” refers to an oligonucleotide or a polynucleotide, which has a bond between nucleotides that is not normal or includes a derivative of a nucleotide. They are interchangeably used. Specific examples of such an oligonucleotide include 2′-O-methyl-ribonucleotide, oligonucleotide derivative with a phosphodiester bond in an oligonucleotide converted to a phosphorothioate bond, oligonucleotide derivative with a phosphodiester bond in an oligonucleotide converted to an N3′-P5′ phosphoroamidate bond, oligonucleotide derivative with a ribose and phosphodiester bond in an oligonucleotide converted to a peptide nucleic acid bond, oligonucleotide derivative with uracil in an oligonucleotide substituted with C-5 propynyl uracil, oligonucleotide derivative with uracil in an oligonucleotide substituted with C-5 thiazole uracil, oligonucleotide derivative with cytosine in an oligonucleotide substituted with C-5 propynyl cytosine, oligonucleotide derivative with cytosine in an oligonucleotide substituted with phenoxazine-modified cytosine, oligonucleotide derivative with ribose in an DNA substituted with 2′-O-propylribose, oligonucleotide derivative with ribose in an oligonucleotide substituted with 2′-methoxyethoxyribose and the like. Unless specifically noted otherwise, a specific nucleic acid sequence is further intended to encompass conservatively altered variants (e.g., degenerate codon substituted form) and complementary sequences thereof in addition to the explicitly shown sequences. Specifically, a degenerate codon substituted form can be obtained by creating a sequence in which the third position of one or more selected (or all) codons is substituted with a mixed base and/or deoxyinosine residue (Batzer et al., Nucleic Acid Res. 19: 5081 (1991); Ohtsuka et al., J. Biol. Chem. 260: 2605-2608 (1985); Rossolini et al., Mol. Cell. Probes 8: 91-98 (1994)). As used herein, “nucleic acid” is interchangeably used with gene, cDNA, mRNA, oligonucleotide, and polynucleotide. As used herein, “nucleotide” may be natural or non-natural.

As used herein, “gene” refers to an agent defining a genotype. A gene is generally arranged in a certain order in a chromosome. A gene defining the primary structure of a protein is referred to as a structural gene, and a gene affecting the expression thereof is referred to as a regulator gene. As used herein, “gene” may refer to a “polynucleotide”, “oligonucleotide” or “nucleic acid”. “Gene product” is a substance produced based on a gene and refers to a protein, mRNA or the like.

As used herein, “homology” of genes refers to the level of identity of two or more genetic sequences to one another. In general, having “homology” refers to having a high level of identity or similarity. Thus, a higher level of homology of two genes results in a higher level of identity or similarity of sequences thereof. It is possible to examine whether two types of genes are homologous by direct comparison of sequences or by hybridization under stringent conditions for nucleic acids. When directly comparing two genetic sequences, the genes are homologous typically when DNA sequences between the genetic sequences are at least 50% identical, preferably at least 70% identical, and more preferably at least 80%, 90%, 95%, 96%, 97%, 98% or 99% identical. Thus, as used herein, “homolog” or “homologous gene product” refers to a protein in another species, preferably a mammal, which exerts the same biological function as a protein constituent element of a complex further described herein.

An amino acid may be mentioned herein by a commonly known three letter symbol thereof or a one letter symbol recommended by IUPAC-IUB Biochemical Nomenclature Commission. Similarly, a nucleotide may be mentioned by a commonly recognized one letter code. Herein, comparison of similarity, identity and homology of amino acid sequences and base sequences is calculated by using a default parameter with a sequence analysis tool BLAST. For instance, identity can be searched by using NCBI's BLAST 2.2.9 (published on 5 Dec. 2004). The value of identity herein generally refers to a value from using the above-described BLAST to align sequences under default conditions. However, when a higher value is output by changing a parameter, the highest value is considered the value of identity. When identity is assessed in a plurality of regions, the highest value thereamong is considered the value of identity. Similarity is a numerical value that uses a similar amino acid for the calculation in addition to identity.

As used herein, “polynucleotide that hybridizes under stringent conditions” refers to a conventional, well-known condition in the art. Such a polynucleotide can be obtained by using colony hybridization, plaque hybridization, southern blot hybridization or the like while using a polynucleotide selected from the polynucleotides of the present invention as a probe. Specifically, such a polynucleotide refers to a polynucleotide which can be identified by using a filter with immobilized DNA derived from a colony or a plaque for hybridization at 65° C. in the presence of 0.7-1.0 M NaCl, and then using a 0.1 to 2-fold concentration SSC (saline-sodium citrate) solution (composition of an SSC solution with 1-fold concentration is 150 mM sodium chloride and 15 mM sodium citrate) to wash the filter under the condition of 65° C. Hybridization can be performed in accordance with the method described in experimental publications such as Molecular Cloning 2nd ed., Current Protocols in Molecular Biology, Supplement 1-38, DNA Cloning 1: Core Techniques, A Practical Approach, Second Edition, Oxford University Press (1995). In this regard, a sequence comprising only an A sequence or only a T sequence is preferably excluded from a sequence that hybridizes under stringent conditions. Thus, the polypeptide used in the present invention (e.g., transthyretin and the like) encompasses polypeptides encoded by a nucleic acid molecule that hybridizes under stringent conditions to a nucleic acid molecule encoding a polypeptide particularly described in the present invention. The low stringency conditions include hybridization for 18-hours at 40° C. in a buffer solution comprising 35% formamide, 5×SSC, 50 mM Tris-HCl (pH 7.5), 5 mM EDTA, 0.02% polyvinylpyrrolidone (PVP), 0.02% BSA, 100 μg/ml denatured salmon sperm DNA, and 10% (w/v) dextran sulfate, washing for 1-5 hours at 55° C. in a buffer solution consisting of 2×SSC, 25 mM Tris-HCl (pH 7.4), 5 mM EDTA, and 0.1% SDS, and washing for 1.5 hours at 60° C. in a buffer solution consisting of 2×SSC, 25 mM Tris-HCl (pH 7.4), 5 mM EDTA, and 0.1% SDS.

As used herein, “purified” substance or biological agent (e.g., nucleic acid, protein or the like) refers to a biological agent having at least some of the agents that are naturally accompanied therewith removed. Thus, purity of a biological agent in a purified biological agent is generally higher than that of a normal condition of the biological agent (i.e., concentrated). The term “purified” as used herein preferably refers to the presence of at least 75 wt. %, more preferably at least 85 wt. %, still more preferably at least 95 wt. %, and the most preferably at least 98 wt. % of biological agents of the same type. A substance used in the present invention is preferably a “purified” substance.

As used herein, “corresponding” amino acid or nucleic acid” refers to an amino acid or nucleotide which has, or is expected to have, action similar to a determined amino acid or nucleotide in a polypeptide or polynucleotide that is a baseline of comparison in a certain polypeptide molecule or polynucleotide molecule, and particularly for an enzyme molecule refers to an amino acid that is at the same position in an active site and provides the same contribution to catalytic activity. For instance, for antisense molecules, this may be a similar portion in an ortholog corresponding to a specific portion of the antisense molecule. A corresponding amino acid may be a specific amino acid that has undergone cysteinylation, glutathionylation S-S bond formation, oxidation (e.g., oxidation of methionine side chain), formylation, acetylation, phosphorylation, glycosylation, myristylation or the like. Alternatively, a corresponding amino acid may be an amino acid responsible for dimerization. Such a “corresponding” amino acid or nucleic acid may be a region or domain (e.g., V region, D region or the like) over a certain range. Thus, such a region or domain is called a “corresponding” region or domain herein.

As used herein, “fragment” refers to a polypeptide or polynucleotide with a sequence length of 1 to n−1 with respect to the full length polypeptide or polynucleotide (with length n). The length of a fragment can be appropriately changed in accordance with the objective. Examples of the lower limit of such a length include 3, 4, 5, 6, 7, 8, 9, 10, 15, 20, 25, 30, 40, 50 and more amino acids for a polypeptide. Lengths represented by an integer that is not specifically listed herein (e.g., 11 and the like) also can be suitable as a lower limit. Further, examples of length include 5, 6, 7, 8, 9, 10, 15, 20, 25, 30, 40, 50, 75, 100, and more nucleotides for a polynucleotide. Lengths represented by an integer that is not specifically listed herein (e.g., 11 and the like) also can be suitable as a lower limit. As used herein, such a fragment is understood to be within the scope of the present invention when a full length version functions as a marker, as along as the fragment itself also functions as a marker.

The term “activity” according to the present invention refers to a function of a molecule in the broadest sense herein. Activity, although not intended to be limiting, generally includes a biological function, biochemical function, physical function, therapeutic activity, diagnostic activity and chemical function of a molecule. Examples of activity include enzymatic activity, an ability to interact with another molecule, an ability to activate, promote, stabilize, inhibit, suppress, or destabilize a function of another molecule, stability, and ability to localize at a specific position in a cell. When applicable, the term also relates to a function of a protein complex in the broadest sense.

As used herein “expression” of a gene, polynucleotide, polypeptide or the like refers to the gene or the like being affected by a certain action in vivo to have another form. Preferably, expression refers to a gene, polynucleotide or the like being transcribed and translated to be in a form of a polypeptide, but being transcribed to make an mRNA can also be one form of expression. More preferably, such a polypeptide form can be those processed after translation (derivative as referred to herein).

A functional equivalent such as an isotype of a molecule such as IgG used in the present invention can be found by searching a database or the like. As used herein, “search” refers to utilizing a certain nucleic acid base sequence electronically, biologically, or by another method, preferably electronically, to find another nucleic acid base sequence having a specific function and/or property. Examples of electronic search include, but are not limited to, BLAST (Altschul et al., J.Mol.Biol. 215: 403-410 (1990)), FASTA (Pearson & Lipman, Proc. Natl. Acad. Sci., USA 85: 2444-2448 (1988)), Smith and Waterman method (Smith and Waterman, J. Mol. Biol. 147: 195-197 (1981)), Needleman and Wunsch method (Needleman and Wunsch, J. Mol. Biol. 48: 443-453 (1970)) and the like. BLAST is typically used. Examples of biological search include, but are not limited to, stringent hybridization, a macroarray with a genomic DNA applied to a nylon membrane or the like or a microarray with a genomic DNA applied to a glass plate (microarray assay), PCR, in situ hybridization and the like. Herein, a gene used in the present invention is intended to include corresponding genes identified by such electronic search or biological search.

As a functional equivalent of the present invention, it is possible to use an amino acid sequence with one or more amino acid insertions, substitutions or deletions, or addition to one or both ends. As used herein, “one or more amino acid insertions, substitutions or deletions, or addition to one or both ends” in an amino acid sequence refers to an alteration with a substitution of multiple amino acids or the like to the extent that can occur naturally by a well-known technical method such as site-directed mutagenesis or a natural mutation. An altered amino acid sequence of a molecule can have, for example, 1-30, preferably 1-20, more preferably 1-9, still more preferably 1-5, and especially preferably 1-2 amino acid insertion, substitution or deletion or addition to one or both ends. An altered amino acid sequence may be an amino acid sequence having one or more (preferably 1 or several, or 1, 2, 3 or 4) conservative substitutions in an amino acid sequence of a molecule such as CD98. “Conservative substitution” refers to a substitution of one or more amino acid residues with another chemically similar amino acid residue so as not to substantially alter a function of a protein. Examples thereof include cases where a hydrophobic residue is substituted with another hydrophobic residue, cases where a polar residue is substituted with another polar residue having the same charge and the like. Functionally similar amino acids that can be substituted in this manner are known in the art for each amino acid. Specific examples include alanine, valine, isoleucine, leucine, proline, tryptophan, phenylalanine, methionine and the like for nonpolar (hydrophobic) amino acids, glycine, serine, threonine, tyrosine, glutamine, asparagine, cysteine and the like for polar (neutral) amino acids. Examples of positively charged (basic) amino acid include arginine, histidine, lysine and the like. Further, examples of a negatively-charged (acidic) amino acid include aspartic acid, glutamic acid and the like.

As used herein, “marker (substance, protein or gene (nucleic acid))” refers to a substance that can be an indicator for tracking whether a target is in or in risk of being in a certain condition (e.g., normal cell state, transformed state, diseased state, disorder state, level of or presence of proliferation capability or differentiated state or the like). Examples of such a marker include genes (nucleic acid=DNA level), gene products (mRNA, protein, and the like), metabolites, enzymes and the like. In the present invention, detection, diagnosis, preliminary detection, prediction or prediagnosis of a certain state (e.g., disease such as differentiation disorder) can be materialized by using an agent or means specific to a marker associated with such a state, or a composition, kit or system comprising the same or the like. As used herein, “gene product” refers to a protein or mRNA encoded by a gene.

As used herein, “subject” refers to a target subjected to diagnosis, detection or the like of the present invention (e.g., an organism such as a human or an organ or cell extracted from an organism or the like).

As used herein, “sample” refers to any substance obtained from a subject or the like. For example, an eye cell and the like are encompassed. Those skilled in the art can appropriately select a preferred sample based on the descriptions herein.

As used herein, “agent” is used broadly and may be any substance or other elements (e.g., energy, radiation, heat, electricity and other forms of energy) as long as the intended objective can be achieved. Examples of such a substance include, but are not limited to, protein, polypeptide, oligopeptide, peptide, polynucleotide, oligonucleotide, nucleotide, nucleic acid (including for example DNAs such as cDNA and genomic DNA, RNAs such as mRNA), polysaccharide, oligosaccharide, lipid, organic small molecule (e.g., hormone, ligand, information transmitting substance, organic small molecule, molecule synthesized by combinatorial chemistry, small molecule that can be used as medicine (e.g., small molecule ligand and the like) and a composite molecule thereof). Typical examples of an agent specific to a polynucleotide include, but are not limited to, a polynucleotide having complementarity with a certain sequence homology (e.g., 70% or greater sequence identity) to a sequence of the polynucleotide, polypeptide such as a transcription factor that binds to a promoter region and the like. Typical examples of an agent specific to a polypeptide include, but are not limited to, an antibody directed specifically to the polypeptide or a derivative or analog thereof (e.g., single strand antibody), a specific ligand or receptor when the polypeptide is a receptor or ligand, a substrate when the polypeptide is an enzyme and the like.

As used herein, “detecting agent” broadly refers to all agents capable of detecting a target of interest.

As used herein, “diagnostic agent” broadly refers to all agents capable of diagnosing a condition of interest (e.g., disease or the like).

The detecting agent of the present invention may be a complex or composite molecule in which another substance (e.g., label or the like) is bound to a portion enabling detection (e.g., antibody or the like). As used herein, “complex” or “composite molecule” refers to any construct comprising two or more portions. For instance, when one portion is a polypeptide, the other portion may be a polypeptide or other substances (e.g., sugar, lipid, nucleic acid, other carbohydrate or the like). As used herein, two or more constituent portions of a complex may be bound by a covalent bond or any other bond (e.g., hydrogen bond, ionic bond, hydrophobic interaction, Van der Waals force or the like). When two or more portions are polypeptides, the complex may be called a chimeric polypeptide. Thus, “complex” as used herein includes molecules formed by linking a plurality of types of molecules such as a polypeptide, polynucleotide, lipid, sugar, or small molecule.

As used herein, “interaction” refers, for two substances, to applying a force (e.g., intermolecular force (Van der Waals force), hydrogen bond, hydrophobic interaction, or the like) between one substance and the other substance. Generally, two substances that have interacted are in a conjugated or bound state.

As used herein, the term “bond” refers to a physical or chemical interaction between two substances or between combinations thereof. A bond includes an ionic bond, non-ionic bond, hydrogen bond, Van der Waals bond, hydrophobic interaction and the like. A physical interaction (bond) may be direct or indirect. Indirect physical interaction (bond) is mediated by or is due to an effect of another protein or compound. A direct bond refers to an interaction, which does not occur through or due to an effect of another protein or compound and does not substantially involve another intermediate. The degree of expression of the marker of the present invention or the like can be measured by measuring a bond or interaction.

Thus, an “agent” (or detecting agent or the like) that “specifically” interacts (or binds) to a biological agent such as a polynucleotide or a polypeptide as used herein encompasses agents with affinity to the biological agent such as a polynucleotide or polypeptide that is typically similar or higher, preferably significantly (e.g., statistically significantly) higher, than affinity to other unrelated polynucleotide or polypeptide (especially those with less than 30% identity). Such affinity can be measured by, for example, hybridization assay, binding assay or the like.

As used herein, “specific” interaction (or bond) of a first substance or agent with a second substance or agent refers to the first substance or agent interacting with (or binding to) the second substance or agent at a higher level of affinity than to substances or agents other than the second substance or agent (especially other substances or agents in a sample comprising the second substance or agent). Examples of an interaction (or bond) specific to a substance or agent include, but are not limited to, a ligand-receptor reaction, hybridization in a nucleic acid, antigen-antibody reaction in a protein, enzyme-substrate reaction and the like, and when both a nucleic acid and a protein are involved, a reaction between a transcription factor and a binding site of the transcription factor and the like, protein-lipid interaction, nucleic acid-lipid interaction and the like. Thus, when substances or agents are both nucleic acids, a first substance or agent “specifically interacting” with a second substance or agent encompasses the first substance or agent having at least partial complementarity to the second substance or agent. Further, examples of a first substance or agent “specifically” interacting with (or binding to) a second substance or agent when substances or agents are both proteins includes, but are not limited to, interaction by an antigen-antibody reaction, interaction by a receptor-ligand reaction, enzyme-substrate interaction and the like. When two types of substances or agents include a protein and a nucleic acid, a first substance or agent “specifically” interacting with (or binding to) a second substance or factor encompasses an interaction (or a bond) between a transcription factor and a binding region of a nucleic acid molecule targeted by the transcription factor.

As used herein, “detection” or “quantification” of polynucleotide or polypeptide expression can be accomplished by using a suitable method including, for example, an immunological measuring method and measurement of mRNAs, including a bond or interaction to a marker detecting agent. However, measurement can be performed with the amount of PCR product in the present invention. Examples of a molecular biological measuring method include northern blot, dot blot, PCR and the like. Examples of an immunological measurement method include ELISA using a microtiter plate, RIA, fluorescent antibody method, luminescence immunoassay (LIA), immunoprecipitation (IP), single radial immunodiffusion (SRID), turbidimetric immunoassay (TIA), western blot, immunohistochemical staining and the like. Further, examples of a quantification method include ELISA, RIA and the like. Quantification may also be performed by a gene analysis method using an array (e.g., DNA array, protein array). DNA arrays are outlined extensively in (Ed. by Shujunsha, Saibo Kogaku Bessatsu “DNA Maikuroarei to Saishin PCR ho” [Cellular engineering, Extra issue, “DNA Microarrays and Latest PCR Methods”]. Protein arrays are discussed in detail in Nat Genet. 2002 December; 32 Suppl: 526-32. Examples of a method of analyzing gene expression include, but are not limited to, RT-PCR, RACE, SSCP, immunoprecipitation, two-hybrid system, in vitro translation and the like, in addition to the methods discussed above. Such additional analysis methods are described in, for example, Genomu Kaiseki Jikkenho Nakamura Yusuke Labo Manyuaru [Genome analysis experimental method Yusuke Nakamura Lab Manual], Ed. by Yusuke Nakamura, Yodosha (2002) and the like. The entirety of the descriptions therein is incorporated herein by reference.

As used herein, “amount of expression” refers to the amount of polypeptide, mRNA or the like expressed in a cell, tissue or the like of interest. Examples of such an amount of expression include amount of expression of polypeptide of the present invention at a protein level assessed by any suitable method including an immunological measurement method such as ELISA, RIA, fluorescent antibody method, western blot, and immunohistochemical staining by using the antibody of the present invention, and the amount of expression of the polypeptide used in the present invention at an mRNA level assessed by any suitable method including a molecular biological measuring method such as northern blot, dot blot, and PCR. “Change in amount of expression” refers to an increase or decrease in the amount of expression of the polypeptide used in the present invention at a protein level or mRNA level assessed by any suitable method including the above-described immunological measuring method or molecular biological measuring method. A variety of detection or diagnosis based on a marker can be performed by measuring the amount of expression of a certain marker.

As used herein, “decrease” or “suppression” of activity or expression product (e.g., protein, transcript (RNA or the like)) or synonyms thereof refers to: a decrease in the amount, quality or effect of a specific activity, transcript or protein; or activity that decreases the same.

As used herein, “increase” or “activation” of activity or expression product (e.g., protein, transcript (RNA or the like)) or synonyms thereof refers to: an increase in the amount, quality or effect of a specific activity, transcript or protein; or activity that increases the same.

Thus, it is understood that activity of an immune system can be detected or screened by using a regulatory ability such as decrease, suppression, increase or activation of the marker of the present invention as an indicator.

As used herein, “means” refers to anything that can be a tool for accomplishing an objective (e.g., detection, diagnosis, therapy). In particular, “means for selective recognition (detection)” as used herein refers to means capable of recognizing (detecting) a certain subject differently from others.

The present invention is useful as an indicator of an immune system condition. Thus, the present invention can be used to identify an indicator of an immune system condition and know the condition of a disease.

As used herein, “(nucleic acid) primer” refers to a substance required for initiating a reaction of a polymeric compound to be synthesized in a polymer synthesizing enzyme reaction. A synthetic reaction of a nucleic acid molecule can use a nucleic acid molecule (e.g., DNA, RNA or the like) complementary to a portion of a sequence of a polymeric compound to be synthesized. A primer can be used herein as a marker detecting means.

Examples of a nucleic acid molecule generally used as a primer include those having a nucleic acid sequence with a length of at least 8 contiguous nucleotides, which is complementary to a nucleic acid sequence of a gene of interest (e.g., marker of the present invention). Such a nucleic acid sequence may be a nucleic acid sequence with a length of preferably at least 9 contiguous nucleotides, more preferably at least 10 contiguous nucleotides, still more preferably at least 11 contiguous nucleotides, at least 12 contiguous nucleotides, at least 13 contiguous nucleotides, at least 14 contiguous nucleotides, at least contiguous nucleotides, at least 16 contiguous nucleotides, at least 17 contiguous nucleotides, at least contiguous nucleotides, at least 19 contiguous nucleotides, at least 20 contiguous nucleotides, at least contiguous nucleotides, at least 30 contiguous nucleotides, at least 40 contiguous nucleotides, or at least 50 contiguous nucleotides. A nucleic acid sequence used as a probe comprises a nucleic acid sequence that is at least 70% homologous, more preferably at least 80% homologous, still more preferably at least 90% homologous, or at least 95% homologous to the aforementioned sequence. A sequence suitable as a primer may vary depending on the property of a sequence intended for synthesis (amplification). However, those skilled in the art are capable of designing an appropriately primer in accordance with an intended sequence. Design of such a primer is well known in the art, which may be performed manually or by using a computer program (e.g., LASERGENE, PrimerSelect, or DNAStar).

The primers according to the present invention can be used as a primer set consisting of two or more types of the primers.

The primers and primer set according to the present invention can be used as primers and primer set in accordance with a common method in a known method of detecting a gene of interest by utilizing a nucleic acid amplification method such as PCR, RT-PCR, real-time PCR, in situ PCR, or LAMP.

The primer set according to the present invention can be selected such that a nucleotide sequence of a protein of interest such as a molecule of a T cell receptor can be amplified by a nucleic acid amplification method such as PCR. Nucleic acid amplification methods are well known. Selection of a primer pair in a nucleic acid amplification method is evident to those skilled in the art. For instance, primers can be selected in PCR such that one of the two primers (primer pair) is paired with the plus strand of a double-stranded DNA of a protein of interest such as a T cell receptor molecule while the other primer is paired with the minus strand of the double-stranded DNA and the chain extended by one of the primers is paired with the other primer. The primer of the present invention can be chemically synthesized based on the nucleotide sequence disclosed herein. Preparation of a primer is well known and can be carried out in accordance with, for example, “Molecular Cloning, A Laboratory Manual 2nd ed.” (Cold Spring Harbor Press (1989)), “Current Protocols in Molecular Biology” (John Wiley & Sons (1987-1997)).

As used herein, “probe” refers to a substance that can be means for search, which is used in a biological experiment such as in vitro and/or in vivo screening. Examples thereof include, but are not limited to, a nucleic acid molecule comprising a specific base sequence, a peptide comprising a specific amino acid sequence, a specific antibody, a fragment thereof and the like. As used herein, a probe can be used as marker detecting means.

A nucleic acid molecule generally used as a probe includes those having a nucleic acid sequence with a length of at least 8 contiguous nucleotides, which is homologous or complementary to a nucleic acid sequence of a gene of interest. Such a nucleic acid sequence may be a nucleic acid sequence with a length of preferably at least 9 contiguous nucleotides, more preferably at least 10 contiguous nucleotides, still more preferably at least 11 contiguous nucleotides, at least 12 contiguous nucleotides, at least 13 contiguous nucleotides, at least 14 contiguous nucleotides, at least 15 contiguous nucleotides, at least contiguous nucleotides, at least 25 contiguous nucleotides, at least 30 contiguous nucleotides, at least contiguous nucleotides, or at least 50 contiguous nucleotides. A nucleic acid sequence used as a probe comprises a nucleic acid sequence that is at least about 70% homologous, more preferably at least about 80% homologous, still more preferably at least about 90% homologous, or at least about 95% homologous with the aforementioned sequence.

In one embodiment, the detecting agent of the present invention may be labeled. Alternatively, the detecting agent of the present invention may be bound to a tag.

As used herein, “label” refers to an entity (e.g., substance, energy, electromagnetic wave or the like) for distinguishing a molecule or substance of interest from others. Such a method of labeling includes RI (radioisotope) method, fluorescence method, biotin method, chemiluminescent method and the like. When a plurality of markers of the present invention or agents or means for capturing the same are labeled by a fluorescence method, labeling is performed with labeling substances having different fluorescent emission maximum wavelengths. It is preferable that the difference in fluorescent emission maximum wavelengths is 10 nm or greater. When labeling a ligand, any label that does not affect the function can be used. However, Alexa™Fluor is desirable as a fluorescent substance. Alexa™Fluor is a water-soluble fluorescent dye obtained by modifying coumarin, rhodamine, fluorescein, cyanine or the like. This is a series compatible with a wide range of fluorescence wavelengths. Relative to other fluorescent dyes for the corresponding wavelength, Alexa™Fluor is very stable, bright and has a low level of pH sensitivity. Combinations of fluorescent dyes with fluorescence maximum wavelength of 10 nm or greater include a combination of Alexa™555 and Alexa™633, combination of Alexa™488 and Alexa™555 and the like. When a nucleic acid is labeled, any substance can be used that can bind to a base portion thereof. However, it is preferable to use a cyanine dye (e.g., Cy3, Cy5 or the like of the CyDye™ series), rhodamine 6G reagent, N-acetoxy-N2-acetylaminofluorene (AAF), AAIF (iodine derivative of AAF) or the like. Examples of a fluorescent substance with a difference in fluorescent emission maximum wavelengths of nm or greater include a combination of Cy5 and a rhodamine 6G reagent, a combination of Cy3 and fluorescein, a combination of a rhodamine 6G reagent and fluorescein and the like. The present invention can utilize such a label to alter a subject of interest to be detectable by the detecting means to be used. Such alteration is known in the art. Those skilled in the art can appropriately carry out such a method in accordance with the label and subject of interest.

As used herein, “tag” refers to a substance for distinguishing a molecule by a specific recognition mechanism such as receptor-ligand, or more specifically, a substance serving the role of a binding partner for binding a specific substance (e.g., having a relationship such as biotin-avidin or biotin-streptavidin). A tag can be encompassed in the scope of “label”. Accordingly, a specific substance to which a tag is bound can distinguish the specific substance by a contact with a substrate, to which a binding partner of a tag sequence is bound. Such a tag or label is well known in the art. Typical tag sequences include, but are not limited to, myc tag, His tag, HA, Avi tag and the like. Such a tag may be bound to the marker or marker detecting agent of the present invention.

In this regard, “test sample” only needs to be a cell of interest or a substance derived therefrom, which is considered to comprise an element enabling gene expression.

As used herein, “diagnosis” refers to identifying various parameters associated with a disease, disorder, condition or the like in a subject to determine the current or future state of such a disease, disorder, or condition. The condition in the body can be examined by using the method, apparatus, or system of the present invention. Such information can be used to select and determine various parameters of a formulation or method for treatment or prevention to be administered, disease, disorder, or condition in a subject or the like. As used herein, “diagnosis” when narrowly defined refers to diagnosis of the current state, but when broadly defined includes “early diagnosis”, “predictive diagnosis”, “prediagnosis” and the like. Since the diagnostic method of the present invention in principle can utilize what comes out from a body and can be conducted away from a medical practitioner such as a physician, the present invention is industrially useful. In order to clarify that the method can be conducted away from a medical practitioner such as a physician, the term as used herein may be particularly called “assisting” “predictive diagnosis, prediagnosis or diagnosis”.

The formulation procedure for a diagnostic agent or the like of the present invention as a medicament or the like is known in the art. The procedure is described, for example, in Japanese Pharmacopoeia, the United States Pharmacopeia, pharmacopeia of other countries, or the like. Thus, those skilled in the art can determine the amount to be used without undue experimentation from the descriptions herein.

As used herein, “complete match with all C region allelic sequences of the same isotype” refers to a match with all sequences for C region allelic sequences of the same isotype when aligned. Since all sequences in a C region would never be identical even in the same isotype, use of a sequence that is a complete match with all C region allelic sequences of the same isotype would be advantageous for immediately determining an isotype when a sequence of an amplicon is determined.

As used herein, “trimming” refers to removal of an unsuitable portion in gene analysis. Trimming is performed by removing low quality regions from both ends of a read, partial sequence of an artificial nucleic acid sequence imparted in an experimental procedure, or both. Trimming can be performed with a software known in the art or by referring to references (for example, cutadapt hypertext transfer protocol colon journal dot embnet dot org/index dot php/embnetjournal/article/view/200/(EMBnet dot journal, 2011); fastq-mcf Aronesty E., The Open Bioinformatics Journal (2013) 7, 1-8 (DOI: 10.2174/1875036201307010001); and fastx-toolkit hypertext transfer protocol colon hannonlab dot cshl dot edu/fastx_toolkit/(2009)). For an adaptor sequence or an artificial nucleic acid sequence, trimming is preferably accomplished by the steps of: deleting low quality regions from both ends of a read; deleting a region matching 10 bp or more with an adaptor sequence from both ends of the read; and using the read as a high quality read in analysis when the remaining length is 200 bp or more (TCR) or 300 bp or more (BCR).

As used herein, “suitable length” refers to a length that is adapted to analysis when analysis of an alignment or the like is performed in the gene analysis of the present invention. For example, such a length can be determined to be a length including 100 bases toward a D region on a V region from a sequencing initiation position on a C region. In the present invention, examples of a suitable length include, but are not limited to, 200 nucleotides or longer, preferably 250 nucleotides or longer for TCRs and 300 nucleotides or longer and preferably 350 nucleotides or longer for BCRs.

As used herein, “input sequence set” refers to a set of target sequences of TCR or BCR repertoire analysis in the gene analysis of the present invention.

As used herein, “gene region” refers to each of V region, D region, J region, C region and the like. Such gene regions are known in the art and can be appropriately determined by referring to a database or the like. As used herein, “homology” of genes refers to the level of identity of 2 or more genetic sequences to one another. In general, having “homology” refers to having a high level of identity or similarity. Thus, a higher level of homology of two genes results in a higher level of identity or similarity of sequences thereof. It is possible to examine whether two types of genes are homologous by direct comparison of sequences or by hybridization under stringent conditions for nucleic acids. As used herein, “homology search” refers to search for homology. Preferably, homology can be searched in silico by using a computer.

As used herein, “approximate” refers to having a high level of homology when homology search is performed. A software for homology search (BLAST, FASTA or the like), when executed, generally lists results in order of high level of homology. Thus, approximation is possible by appropriately selecting a result that is highly ranked.

As used herein, “closest” refers to the highest level of homology when homology search is performed. When homology is searched with a software, the result displayed as ranking number one is selected.

As used herein, “reference allele” refers to a reference allele that results in a match in a reference database when homology search is performed.

As used herein, “alignment” (or align) in bioinformatics refers to similar regions of a primary structure of a biomolecule such as DNA, RNA, or protein arranged in alignment to be identifiable or the act of arranging. Alignment can provide a clue for understanding the functional, structural, or evolutionary relationship of sequences.

As used herein, “assign” refers to allocating specific information such as a gene name, function, characteristic region (e.g., V region, J region or the like) to a certain sequence (e.g., nucleic acid sequence, protein sequence or the like). Specifically, this is accomplished by inputting or linking specific information to a certain sequence or the like.

As used herein, “CDR3” refers to the third complementarity-determining region (CDR). In this regard, CDR is a region that directly contacts an antigen and undergoes a particularly large change among variable regions, and is referred to as a hypervariable region. Each variable region of a light chain and a heavy chain has three CDRs (CDR1-CDR3) and 4 FRs (FR1-FR4) surrounding the three CDRs. Since a CDR3 region is considered to be present across V region, D region and J region, it is considered as an important key for a variable region, and is thus used as a subject of analysis.

As used herein, “front of CDR3 on a reference V region” refers to a sequence corresponding to the front of CDR3 in a V region targeted by the present invention.

As used herein, “end of CDR3 on a reference J” refers to a sequence corresponding to the end of CDR3 in a J region targeted by the present invention.

As used herein, “condition tolerating random mutations to be scattered throughout” refers to any condition which results in random mutations being scattered around. For example, such a condition is often expressed by the following condition for BLAST/FASTA optimal parameters: tolerates a maximum mismatch of 33% across the full length of an alignment; and tolerates a maximum nonconsecutive mismatch of 60% for any 30 bp therein. A functional equivalent such as an isotype of a molecule, e.g. IgG, used in the present invention can be found by searching a database or the like. As used herein, “search” refers to utilizing a certain nucleic acid base sequence electronically, biologically, or by another method, preferably electronically, to find another nucleic acid base sequence having a specific function and/or property. Examples of electronic search include, but are not limited to, BLAST (Altschul et al., J. Mol. Biol. 215: 403-410 (1990)), FASTA (Pearson & Lipman, Proc. Natl. Acad. Sci., USA 85: 2444-2448 (1988)), Smith and Waterman method (Smith and Waterman, J. Mol. Biol. 147: 195-197 (1981)), Needleman and Wunsch method (Needleman and Wunsch, J. Mol. Biol. 48: 443-453 (1970)) and the like. BLAST is typically used. Examples of biological search include, but are not limited to, stringent hybridization, a macroarray with a genomic DNA applied to a nylon membrane or the like or a microarray with a genomic DNA applied to a glass plate (microarray assay), PCR, in situ hybridization and the like. Herein, a gene used in the present invention is intended to include corresponding genes identified by such electronic search or biological search.

### Preferred Embodiments

Preferred embodiments of the present invention are described below. The embodiments are provided for better understanding of the present invention. It is understood that the scope of the present invention should not be limited to the following descriptions. Further, it is apparent that those skilled in the art can readily make modifications within the scope of the present invention while referring to the descriptions herein. For such embodiments, those skilled in the art can appropriately combine any embodiments.

(Unbiased Sample Amplification)

The present invention can use next generation sequencing techniques to prepare a sample for quantitative analysis of a repertoire of a variable region of a T cell receptor (TCR) or a B cell receptor (BCR). Such sequencing techniques can obtain a million or more reads from a sample at a reasonable cost. Even a genotype that exists at a low frequency of 1/1000000 or less can be detected by using these techniques in a specific and unbiased manner. An unbiased amplification method for amplifying all different types of sequences of a specific portion of a gene or a transcript from a sample derived from a DNA of blood, bone marrow or the like is achieved.

In one aspect, the present invention provides a method of preparing a sample for quantitative analysis of a repertoire of a variable region of a T cell receptor (TCR) or B cell receptor (BCR) by genetic sequence analysis using a database, the method comprising the steps of: (1) synthesizing a complementary DNA by using an RNA sample derived from a target cell as a template; (2) synthesizing a double stranded complementary DNA by using the complementary DNA as a template; (3) synthesizing an adaptor-added double stranded complementary DNA by adding a common adaptor primer sequence to the double stranded complementary DNA; (4) performing a first PCR amplification reaction by using the adaptor-added double stranded complementary DNA, a common adaptor primer consisting of the common adaptor primer sequence, and a first TCR or BCR C region specific primer, wherein the first TCR or BCR C region specific primer is designed to comprise a sequence that is sufficiently specific to a C region of interest of the TCR or BCR and not homologous with other genetic sequences, and comprise a mismatching base between subtypes downstream when amplified; (5) performing a second PCR amplification reaction by using a PCR amplicon of (4), the common adaptor primer, and a second TCR or BCR C region specific primer, wherein the second TCR or BCR C region specific primer is designed to have a sequence that is a complete match with the TCR or BCR C region in a sequence downstream the sequence of the first TCR or BCR C region specific primer, but comprise a sequence that is not homologous with other genetic sequences, and comprise a mismatching base between subtypes downstream when amplified; and (6) performing a third PCR amplification reaction by using a PCR amplicon of (5), an added common adaptor primer in which a nucleic acid sequence of the common adaptor primer comprises a first additional adaptor nucleic acid sequence, and an adaptor-added third TCR or BCR C region specific primer in which a second additional adaptor nucleic acid sequence and a molecule identification (MID Tag) sequence are added to a third TCR or BCR C region specific sequence; wherein the third TCR or BCR C region specific primer is designed to have a sequence that is a complete match with the TCR or BCR C region in a sequence downstream to the sequence of the second TCR or BCR C region specific primer, but comprise a sequence that is not homologous with other genetic sequences, and comprise a mismatching base between subtypes downstream when amplified, the first additional adaptor nucleic acid sequence is a sequence suitable for binding to a DNA capturing bead and for an emPCR reaction, the second additional adaptor nucleic acid sequence is a sequence suitable for an emPCR reaction, and the molecule identification (MID Tag) sequence is a sequence for imparting uniqueness such that an amplicon can be identified.

Conventional methods could not achieve unbiasedness in the truest sense. However, the present invention can achieve unbiased amplification and perform accurate analysis. In regards to unbiasedness, SMART PCR or the like is used in some cases. However, this method cannot achieve precise unbiasedness. The reason therefor is the following. SMART PCR is a method utilizing terminal transferase activity of a reverse transcriptase derived from Moloney Murine Leukemia Virus (MMLV). That is, when a reverse transcriptase reaches the 5′ terminal of an mRNA that is a template in a complementary strand DNA synthesis reaction, a secondary reaction that mainly adds a C base to the 3′ terminal of the newly synthesized complementary DNA is utilized. A primer (TS oligo) having a base sequence (GGG) complementary to the added base (CCC) at the 3′ terminal is used to change a template upon a reverse transcription reaction, resulting in synthesis of a double strand. Thus, the method is known to have a disadvantage wherein addition reactions of TS oligo continuously occur to form a TS oligo concatemer (Villanyi Z, Mai, A, Szabad J. Repeated template switching: Obstacles in cDNA libraries and ways to avoid them. The open genomics journal, 2012, 5, 1-6). Further, the method is known to have a disadvantage wherein progression of polymerase is inhibited by TS oligo in a gene with a sequence that is identical or similar to the 3′ side sequence of the TS oligo, resulting in producing a bias (Tang D T, Plessy C, Salimullah M, Suzuki A M, Calligaris R, Gustincich S, Carninci P. Suppression of artifacts and barcode bias in high-throughput transcriptome analyses utilizing template switching. Nucleic Acids Res. 2013 Feb. 1; 41 (3): e44). In fact, it is reported that the correlation is low between a standard reverse transcription reaction or an in vitro transcription and SMART PCR using microarray analysis (Puskas L G, Zvara A, Hackler L Jr, Van Hummelen P.RNA amplification results in reproducible microarray data with slight ratio bias. Biotechniques. 2002 June; 32(6): 1330-4, 1336, 1338, 1340). Further, it is reported in repeated tests of each detection method that SMART PCR exhibits lower reproducibility than the other two methods (Puskas L G, et al., Biotechniques. 2002 June; 32(6): 1330-4, 1336, 1338, 1340.).

In one embodiment where quantitative analysis is performed on a repertoire of a variable region of a BCR, the C region specific primer comprises a sequence that is a complete match with an isotype C region of interest selected from the group consisting of IgM, IgA, IgG, IgE and IgD and has a sequence that is not homologous with other C regions. Preferably, the C region specific primer is a sequence that is a complete match with one of the subtypes IgG1, IgG2, IgG3 and IgG4 or one of IgA1 and IgA2 for IgA or IgG. In another embodiment where quantitative analysis is performed on a repertoire of a variable region of a TCR, the C region specific primer is a sequence that is a complete match with a C region of a chain of interest selected from the group consisting of α chain, β chain, γ chain and δ chain and is not homologous with other C regions.

In another embodiment, it is preferable that a portion of a sequence that is a complete match with all C region allelic sequences of the same isotype in the database is selected for the C region specific primer. Such election of a complete match enables highly precise analysis.

In a preferred embodiment, the common adaptor primer is designed such that the primer is unlikely to have homodimer and intramolecular hairpin structures and can stably form a double strand, and designed not to be highly homologous with all TCR genetic sequences in the database and to have the same level of Tm as the C region specific primer. Examples of such a common adaptor primer sequence include TAATACGACTCCGAATTCCC (SEQ ID NO: 2), GGGAATTCGG (P10EA; SEQ ID NO: 3) and the like.

In a preferred embodiment, the common adaptor primer designed not to have homodimer and intramolecular hairpin structures and to have homology with other genes comprising a BCR or TCR is selected. Examples of such a common adaptor primer sequence include P20EA, P10EA and the like.

In a specific embodiment, the common adaptor primer is P20EA and/or P10EA, and the sequence thereof is TAATACGACTCCGAATTCCC (P20EA; SEQ ID NO: 2), GGGAATTCGG (P10EA; SEQ ID NO: 3).

In a preferred embodiment, the first, second and third TCR or BCR C region specific primers are each independently a primer for BCR repertoire analysis, the primer being selected to be a sequence that is a complete match with each isotype C region of IgM, IgG, IgA, IgD or IgE, and a complete match with subtypes for IgG and IgA, and not homologous with other sequences comprised in the database, and comprise a mismatching base between subtypes downstream of the primer, and wherein the common adaptor primer sequence is designed such that the sequence has a base length suitable for amplification, is unlikely to have homodimer and intramolecular hairpin structures, and is able to stably form a double strand, and designed not to be homologous with genetic sequences other than a target sequence in the database (or not homologous with other genes comprising a BCR or TCR other than a target sequence) and designed to have the same level of Tm as the C region specific primer. Examples of such a sequence include, are not limited to, P20EA (TAATACGACTCCGAATTCCC (SEQ ID NO: 2)) and P10EA (GGGAATTCGG (SEQ ID NO: 3)).

In a preferred embodiment, the first, second and third TCR C region specific primers are each independently a primer for TCR or BCR repertoire analysis, each primer being selected to be a sequence that is a complete match with 1 type of α chain (TRAC), 2 types of β chains (TRBCO1 and TRBCO2), 2 types of γ chains (TRGC1 and TRGC2), and one type of δ chain (TRDC1) and is not homologous with other sequences comprised in the database, and to comprise a mismatching base between subtypes downstream of the primer, wherein the common adaptor primer sequence is designed such that the sequence has a base length suitable for amplification, is unlikely to have homodimer and intramolecular hairpin structures, and is able to stably form a double strand, and designed not to be highly homologous with all TCR genetic sequences in the database and to have the same level of Tm as the C region specific primer. Examples of such a sequence include, but are not limited to, P20EA (TAATACGACTCCGAATTCCC (SEQ ID NO: 2)), P10EA (GGGAATTCGG (SEQ ID NO: 3)).

In a preferred embodiment, the third TCR or BCR C region specific primer is set in a region that is up to about 150 bases from the 5′ terminal side of a C region, and the first TCR or BCR C region specific primer and the second TCR or BCR C region specific primer are set between the 5′ terminal side of a C region to about 300 bases.

In a preferred embodiment, the first, second and third TCR or BCR C region specific primers are each independently for BCR quantitative analysis, wherein separate specific primers are set to 5 types of isotype sequences, and the primers are designed to completely match a target sequence and ensure a mismatch of 5 bases or more for other isotypes and are designed to be a complete match with all subtypes such that one type of primer is compatible with each similar IgG subtype (IgG1, IgG2, IgG3 and IgG4) or IgA subtype (IgA1 and IgA2). Examples of such a sequence include the following that are used in the Examples, but are not limited to: CM1 (SEQ ID NO: 5), CA1 (SEQ ID NO: 8), CG1 (SEQ ID NO: 11), CD1 (SEQ ID NO: 14), CE1 (SEQ ID NO: 17), CM2 (SEQ ID NO: 6), CA2 (SEQ ID NO: 9), CG2 (SEQ ID NO: 12), CD2 (SEQ ID NO: 15), CE2 (SEQ ID NO: 18), CM3-GS (SEQ ID NO: 7), CA3-GS (SEQ ID NO: 10), CG3-GS (SEQ ID NO: 13), CD3-GS (SEQ ID NO: 16) and CE3-GS (SEQ ID NO: 19).

In a preferred embodiment, parameters in primer design are set to: a base sequence length of 18-22 bases; a melting temperature of 54-66° C.; and % GC (% guanine.cytosine content) of 40-65%. Preferably, in addition to such parameters, parameters are set to: a base sequence length of 18-22 bases; a melting temperature of 54-66° C.; and % GC (% guanine.cytosine content) of 40-65%; a self-annealing score of 26; a self-end annealing score of 10; and a secondary structure score of 28 (for a Roche sequencer used in the Examples). Although these preferred values of base sequence length and the like may vary depending on the sequencer model, those skilled in the art can appropriately set parameters in accordance with the sequencer model.

In a preferred embodiment, conditions for a method of determining sequences of the first, second and third TCR or BCR C region specific primers include the following: 1. a plurality of subtype sequences and/or allelic sequences are uploaded into a base sequence analysis software and aligned; 2. a primer designing software is used to search for a plurality of primers satisfying a parametric condition in a C region; 3. a primer in a region without a mismatching base in the aligned sequences in 1 is selected; and 4. the presence of a plurality of mismatching sequences for each subtype and/or allele downstream of the primer determined in 3 is confirmed, and if there is no such sequence, a primer is searched further upstream, which is optionally further repeated.

In a preferred embodiment, the first TCR or BCR C region specific primer is set in a position at bases 41-300 with a first base of a first codon of a C region sequence produced by splicing as a baseline, the second TCR or BCR C region specific primer is set in a position at bases 21-300 with said first base as the baseline, and the third TCR or BCR C region specific primer is set in a position within 150 bases or less with said first base as the baseline, and the positions comprise a mismatching site in a subtype and/or allele.

In a preferred embodiment, the first TCR or BCR C region specific primer can have, but is not limited to, the following structure: CM1 (SEQ ID NO: 5), CA1 (SEQ ID NO: 8), CG1 (SEQ ID NO: 11), CD1 (SEQ ID NO: 14), CE1 (SEQ ID NO: 17), CA1 (SEQ ID NO: 35), CB1 (SEQ ID NO: 37) or the like.

In a preferred embodiment, the second TCR or BCR C region specific primer can have, but is not limited to, the following structure: CM2 (SEQ ID NO: 6), CA2 (SEQ ID NO: 9), CG2 (SEQ ID NO: 12), CD2 (SEQ ID NO: 15), CE2 (SEQ ID NO: 18), CA2 (SEQ ID NO: 35), and CB2 (SEQ ID NO: 37) or the like.

In a preferred embodiment, the third TCR or BCR C region specific primer can have, but is not limited to, the following structure: CM3-GS (SEQ ID NO: 7), CA3-GS (SEQ ID NO: 10), CG3-GS (SEQ ID NO: 13), CD3-GS (SEQ ID NO: 16) CE3-GS (SEQ ID NO: 19) or the like.

In a preferred embodiment, each of the TCR or BCR C region specific primers is provided in a set compatible with all TCR or BCR subclasses. The specific sequence thereof includes the following: CM1 (SEQ ID NO: 5), CA1 (SEQ ID NO: 8), CG1 (SEQ ID NO: 11), CD1 (SEQ ID NO: 14), CE1 (SEQ ID NO: 17), CM2 (SEQ ID NO: 6), CA2 (SEQ ID NO: 9), CG2 (SEQ ID NO: 12), CD2 (SEQ ID NO: 15), CE2 (SEQ ID NO: 18), CM3-GS (SEQ ID NO: 7), CA3-GS (SEQ ID NO: 10), CG3-GS (SEQ ID NO: 13), CD3-GS (SEQ ID NO: 16), CE3-GS (SEQ ID NO: 19), CA1 (SEQ ID NO: 35), CB1 (SEQ ID NO: 37), CA2 (SEQ ID NO: 35), CB2 (SEQ ID NO: 37) and the like.

(Large-Scale Analysis)

In another aspect, the present invention provides a method of performing gene analysis using a sample manufactured by the method of the present invention.

Gene analysis can be performed by using any analysis technology. For example, it is possible to use a technology of assigning V, D, J, and C sequences of each read sequence by using V, D, J, and C sequences obtained from a known IMGT (the international ImMunoGeneTics information system, http colon//www dot imgt dot org) database as a reference sequence and utilizing HighV-Quest of the IMGT or a novel software (Repertoire Genesis) developed by the Applicant, which was filed concurrently and described herein as a preferred example of an analysis system.

In a preferred embodiment, the gene analysis is the quantitative analysis of a repertoire of a variable region of a T cell receptor (TCR) or a B cell receptor (BCR).

Different sequences can be distinguished by sequencing individual amplification molecules. Thus, sequencing has sensitivity to detect a quantitative change in clone proliferation. In summary, one provided embodiment of the present invention provides a method of determining a profile of a recombinant DNA sequence in a T cell and/or B cell. The present method may comprise the steps of: isolating a sample from a subject; performing one or more rounds of nucleic acid amplification and spatially isolation of individual nucleic acids; and sequencing the nucleic acids.

One aspect provides a method of determining a correlation of one or more repertoires in a subject or an individual. Another aspect provides a method of developing an algorithm capable of predicting a correlation of one or more repertoires in any sample derived from a subject having a disease. Another aspect provides a method of using an algorithm capable of predicting a correlation of one or more repertoires in any sample derived from a subject to find a correlation of one repertoire of an individual or correlation of a plurality of repertoires. Another aspect provides a method of creating an algorithm that calculates a disease activity score. Another aspect provides a method of monitoring a condition of a disease of an individual.

(Analysis System)

The present invention provides bioinformatics for performing quantitative analysis of a repertoire of a variable region of a T cell receptor (TCR) or a B cell receptor (BCR) by using a next generation sequencing technique.

In one aspect, the present invention is a method of analyzing a TCR or BCR repertoire, comprising the following steps: (1) providing a reference database for each gene region comprising at least one of a V region, a D region, a J region and optionally a C region; (2) providing an input sequence set which is optionally trimmed and optionally extracted to have a suitable length; (3) searching for homology of the input sequence set with the reference database for the each gene region and recording an alignment with an approximate reference allele and/or a sequence of the reference allele; (4) assigning the V region and the J region for the input sequence set and extracting a nucleic acid sequence of the D region based on a result of assigning (preferably, assigning the V region and the J region for the input sequence set and extracting a CDR3 sequence, with the front of CDR3 on a reference V region and end of CDR3 on reference J as guides); (5) translating the nucleic acid sequence of the D region into an amino acid sequence and classifying the D region by utilizing the amino acid sequence (preferably translating the nucleic acid sequence of the CDR3 into an amino acid sequence and classifying the D region by utilizing the amino acid sequence); and (6) calculating a frequency of appearance for each of the V region, the D region, and the J region and optionally the C region or a frequency of appearance of a combination thereof based on the classifying in (5) to derive the TCR or BCR repertoire.

Each step of the present invention is explained with specific operation of each system or apparatus while referring to the flow chart in FIG. 43.

FIG. 43 is a flow chart showing a processing flow demonstrating a method of analyzing a TCR or BCR repertoire in the gene analysis system of the present invention. Further, each of the symbols S1-S6 in the Figure corresponds to each of steps (1)-(6) in the following explanation.

In the method of the present invention, (1) providing a reference database for each gene region comprising at least one of a V region, a D region, a J region and optionally a C region can be accomplished, for example for the V region, by appropriately selecting and providing a database comprising and providing information on the V region.

In the method of the present invention, (2) providing an input sequence set which is optionally trimmed and optionally extracted to have a suitable length is accomplished by optional trimming with a function of an appropriate software or the like and optionally providing an extracted input sequence set after appropriately selecting a length. An input sequence may be, for example, a set of amplicons amplified by a known method or a set of amplicons amplified by PCR with an unbiased method as described in the application submitted concurrently with the present application.

In the method of the present invention, (3) searching for homology of the input sequence set with the reference database for the each gene region and recording an alignment with an approximate reference allele and/or a sequence of the reference allele is performed by appropriately using a software for performing homology search, for each gene range (for example, the V region and the like), performing homology search with a reference database on the input sequence set, and recording alignment with an approximate reference allele and/or a sequence of the reference allele obtained as a result. In FIGS. 29 and 30, the box of “BLAST” or “BLAST analysis”, the IMGT database therebelow, and the vertical double line connecting the two correspond thereto.

In the method of the present invention, (4) assigning the V region and the J region for the input sequence set and extracting a nucleic acid sequence of the D region based on a result of assigning can be accomplished by determining a V region and/or J region based on known information from a sequence alignment. Such extraction can be accomplished by assigning the V region and the J region for the input sequence set and extracting a CDR3 sequence, with the front of CDR3 on a reference V region and end of CDR3 on reference J as guides. In FIGS. 29 and 30, defining both ends of a region as in the horizontal arrow under Dno, with the horizontal arrow under V and the horizontal arrow under J as guides, corresponds to extraction of a CDR3 sequence.

In the method of the present invention, (5) translating the nucleic acid sequence of the D region into an amino acid sequence and classifying the D region by utilizing the amino acid sequence can be accomplished by translating into an amino acid using a known method in the art and picking out a sequence corresponding to the D region by homology search or the like on the amino acid sequence. Preferably, the nucleic acid sequence of the CDR3 can be translated into an amino acid sequence and the D region is classified by utilizing the amino acid sequence.

In the method of the present invention, (6) calculating a frequency of appearance for each of the V region, the D region, and the J region and optionally the C region or a frequency of appearance of a combination thereof based on the classifying in (5) to derive the TCR or BCR repertoire can be accomplished by calculating a frequency of appearance of the V region, D region, J region and/or the C region calculated in the above-described step, for example, by organizing the frequencies into a list. A TCR or BCR repertoire can be derived thereby.

The following steps are further explained while referring to FIG. 42.

In S1 (step (1)), a reference database is provided. This may be stored in an external storage apparatus 1405, but can generally be obtained as a publically disclosed database through a communication device 1411. Alternatively, an input apparatus 1409 may be used to input and optionally record a database in a RAM 1403 or the external storage apparatus 1405. In this regard, a database comprising a region of interest such as a V region is provided.

In S2 (step (2)), an input sequence set is provided. For example, a set of sequence information obtained from a set of amplicons amplified in a PCR amplification reaction is inputted by using the input apparatus 1409 or through the communication device 1411. In this regard, an apparatus that receives an amplicon of a PCR amplification reaction and performs genetic sequence analysis thereon may be connected. Such a connection is made through a system bus 1420 or through the communication device 1411. Trimming and/or extraction of a suitable length can be optionally performed at this stage. Such processing is performed with a CPU 1401. A program for trimming and/or extraction may be provided via each of the external storage apparatus, communication device, or input apparatus.

In S3 (step (3)), alignment is performed. At this stage, homology search is performed on the input sequence set with the reference database for each of the gene regions. For the homology search, the reference database obtained via the communication device 1411 or the like is processed with a homology search program. The CPU 1401 performs the processing. Further, results obtained as a result thereof are analyzed for alignment with an approximate reference allele and/or a sequence the reference allele. This is also processed by the CPU 1401. A program for the execution thereof may be provided via each of the external storage apparatus, communication device, or input apparatus.

In S4 (step (4)), nucleic acid sequence information on D is detected. This is also processed by the CPU 1401. A program for the execution thereof may be provided via each of the external storage apparatus, communication device, or input apparatus. This step assigns a V region and a J region for the input sequence set. Assignment is also processed by the CPU 1401. Further, the CPU 1401 also extracts a nucleic acid sequence of the D region based on a result of assigning. A program for the assigning and extracting process may also be provided via each of the external storage apparatus, communication device, or input apparatus. Preferably, this step can be accomplished by determining a V region and/or J region based on known information from sequence alignment. Results can be stored in the RAM 1403 or external storage apparatus 1405.

Preferably, such extraction can be accomplished by assigning the V region and the J region for the input sequence set and extracting a CDR3 sequence, with the front of CDR3 on a reference V region and end of CDR3 on reference J as guides. Such processing can also be performed by the CPU 1401. A program therefor may also be provided via each of the external storage apparatus, communication device, or input apparatus.

In S5 (step (5)), a D region is classified. A nucleic acid sequence of the D region is translated into an amino acid sequence and the D region is classified by utilizing the amino acid sequence. This is also processed by the CPU 1401. A program for this processing may also be provided via each of the external storage apparatus, communication device, or input apparatus. A sequence corresponding to the D region may be picked up by homology search or the like on the obtained amino acid sequence. This is also processed by the CPU 1401. A program for this processing may also be provided via each of the external storage apparatus, communication device, or input apparatus. Preferably, a nucleic acid sequence of the CDR3 can be translated into an amino acid sequence to classify the D region by utilizing the amino acid sequence. This is also processed by the CPU 1401. A program for this processing may also be provided via each of the external storage apparatus, communication device, or input apparatus.

In S6 (step (6)), a frequency of appearance for each of the V region, the D region, and the J region and optionally the C region or a frequency of appearance of a combination thereof is calculated based on the above-described classifying to derive a TCR or BCR repertoire. The calculating and deriving are also processed by the CPU 1401. A program for this processing may also be provided via each of the external storage apparatus, communication device, or input apparatus.

In one preferred embodiment, the gene region used in the present invention comprises all of the V region, the D region, the J region and optionally the C region.

In one embodiment, the reference database is a database with a unique ID assigned to each sequence. A sequence of a gene can be analyzed based on a simple indicator, i.e., ID, by uniquely assigning an ID.

In one embodiment, the input sequence set is an unbiased sequence set. An unbiased sequence set can be implemented by PCR amplification with an unbiased method as described herein. When precision is not required for an unbiased method, “pseudo-unbiased method” with a relatively low quality such as the Smart method may be used. Thus, “unbiased” as used herein refers to unbiasedness with precision as accomplished by the method of the present invention. When such a level is not attained, a method is referred to as a “pseudo-unbiased method”. When an unbiased method as described herein is to be particularly distinguished, the term “precisely unbiased” may be used. However, it is understood that even without a specific description of “precisely”, unbiasedness herein is at a level accomplished by using the method as described herein.

In another embodiment, the sequence set is trimmed. An unnecessary or unsuitable nucleic acid sequence can be removed by trimming, such that efficiently of analysis can be enhanced.

In a preferred embodiment, trimming is accomplished by the steps of: deleting low quality regions from both ends of a read; deleting a region matching 10 bp or more with an adaptor sequence from the both ends of the read; and using the read as a high quality read in analysis when a remaining length is 200 bp or more (TCR) or 300 bp or more (BCR). Preferably, the low quality refers to a 7 bp moving average of QV value less than 30.

In a preferred embodiment, the approximate sequence is the closest sequence. In a specific embodiment, the approximate sequence is determined by a ranking of 1. number of matching bases, 2. kernel length, 3. score, and 4. alignment length.

In another embodiment, the homology search is conducted under a condition tolerating random mutations to be scattered throughout. Such a condition is often expressed by the following condition for BLAST/FASTA optimal parameters: tolerates a maximum mismatch of 33% across the full length of an alignment; and tolerates a maximum nonconsecutive mismatch of 60% for any 30 bp therein. In one embodiment, the homology search comprises at least one condition from (1) shortening of a window size, (2) reduction in a mismatch penalty, (3) reduction in a gap penalty, and (4) a top priority ranking of an indicator is a number of matching bases, compared to a default condition.

In another embodiment, the homology search is carried out under the following conditions in BLAST or FASTA:

V mismatch penalty=−1, shortest alignment length=30, and shortest kernel length=15;

D word length=7 (for BLAST) or K-tup=3 (for FASTA), mismatch penalty=−1, gap penalty=0, shortest alignment length=11, and shortest kernel length=8;

J mismatch penalty=−1, shortest hit length=18, and shortest kernel length=10; and

C shortest hit length=30 and shortest kernel length=15.

This condition can also be used, for example, as long as it is a situation where a shorter (about 200 bp) sequence is used to classify only part of a region (situation that does not fall under the “preferred example”). In addition, this condition can also be used in a situation where an Illumina sequencer is used. In this case, possibility of using bwa or bowtie for homology search is considered.

In a specific embodiment, the D region is classified by a frequency of appearance of the amino acid sequence.

In a further embodiment, a combination of a result of search for homology with the nucleic acid sequence of CDR3 and a result of amino acid sequence translation is used as a classification result when there is a reference database for the D region in the step (5).

In another embodiment, only the frequency of appearance of the amino acid sequence is used for classification when there is no reference database for the D region in the step (5).

In a specific embodiment, the frequency of appearance is counted in a unit of a gene name and/or a unit of an allele.

In another embodiment, the step (4) comprises the step of assigning the V region and the J region for the input sequence set and extracting a CDR3 sequence, with the front of CDR3 on a reference V region and end of CDR3 on reference J as guides.

In a further embodiment, the step (5) comprises translating the nucleic acid sequence of the CDR3 into an amino acid sequence and classifying a D region by using the amino acid sequence.

In one aspect, the present invention provides a system for analyzing a TCR or BCR repertoire, wherein the system comprises: (1) means for providing a reference database for each gene region comprising at least one of a V region, a D region, a J region and optionally a C region; (2) means for providing an input sequence set which is optionally trimmed and optionally extracted to have a suitable length; (3) means for searching for homology of the input sequence set with the reference database for the each gene region and recording an alignment with an approximate reference allele and/or a sequence of the reference allele; (4) means for assigning the V region and the J region for the input sequence set and extracting a nucleic acid sequence of the D region based on a result of assigning; (5) means for translating the nucleic acid sequence of the D region into an amino acid sequence and classifying the D region by utilizing the amino acid sequence; and (6) means for calculating a frequency of appearance for each of the V region, the D region, and the J region and optionally the C region or a frequency of appearance of a combination thereof based on the classifying in (5) to derive the TCR or BCR repertoire.

In another aspect, the present invention provides a computer program for having a computer execute processing of a method of analyzing a TCR or BCR repertoire, the method comprising the following steps: (1) providing a reference database for each gene region comprising at least one of a V region, a D region, a J region and optionally a C region; (2) providing an input sequence set which is optionally trimmed and optionally extracted to have a suitable length; (3) searching for homology of the input sequence set with the reference database for the each gene region and recording an alignment with an approximate reference allele and/or a sequence of the reference allele; (4) assigning the V region and the J region for the input sequence set and extracting a nucleic acid sequence of the D region based on a result of assigning; (5) translating the nucleic acid sequence of the D region into an amino acid sequence and classifying the D region by utilizing the amino acid sequence; and (6) calculating a frequency of appearance for each of the V region, the D region, and the J region and optionally the C region or a frequency of appearance of a combination thereof based on the classifying in (5) to derive the TCR or BCR repertoire.

In still another aspect, the present invention provides a recording medium for storing a computer program for having a computer execute processing of a method of analyzing a TCR or BCR repertoire, the method comprising the following steps: (1) providing a reference database for each gene region comprising at least one of a V region, a D region, a J region and optionally a C region; (2) providing an input sequence set which is optionally trimmed and optionally extracted to have a suitable length; (3) searching for homology of the input sequence set with the reference database for the each gene region and recording an alignment with an approximate reference allele and/or a sequence of the reference allele; (4) assigning the V region and the J region for the input sequence set and extracting a nucleic acid sequence of the D region based on a result of assigning; (5) translating the nucleic acid sequence of the D region into an amino acid sequence and classifying the D region by utilizing the amino acid sequence; and (6) calculating a frequency of appearance for each of the V region, the D region, and the J region and optionally the C region or a frequency of appearance of a combination thereof based on the classifying in (5) to derive the TCR or BCR repertoire.

(System Configuration)

The configuration of a system 1 of the present invention is explained while referring to the functional block diagram in FIG. 42. The Figure shows a case that is materialized with a single system.

The gene analysis system 1 of the present invention is configured by connecting a RAM 1403, external storage apparatus 1405 such as ROM, HDD, magnetic disk, or flash memory such as USB memory and an input output interface (I/F) 1425 via a system bus 1420 to the CPU 1401 installed in a computer system. An input apparatus 1409 such as a keyboard or a mouse, an output apparatus 1407 such as a display, and a communication device 1411 such as a modem are each connected to the input output I/F 1425. The external storage apparatus 1405 comprises an information database storing section 1430 and a program storing section 1440, which are both constant storage regions reserved in the external storage apparatus 1405.

Such a hardware configuration is designed to achieve a function of the present invention in cooperation with an OS (operating system) by the CPU 1401 calling out, deploying, and executing a software program installed on the storage apparatus 1405 on the RAM 1403 from having various instructions (commands) being input via the input apparatus 1409 or from receiving a command via the communication I/F, communication device 1411 or the like.

A reference database, input sequence set, created classification data, data of a TCR or BCR repertoire or the like, or information obtained via the communication device 1411 or the like is constantly written and updated into the database storage section 1430. Information on each sequence in each input sequence set and information such as information ID of each gene in a reference database are managed with each master table to allow information from a sample that is subjected to accumulation to be managed by IDs defined in each master table.

As input sequence set entry information, a sample provider ID, sample information, result of nucleic acid analysis, known individual/physiological information and result of TCR or BCR repertoire analysis are associated with an ID and stored in the database storage section 1430. In this regard, the result of TCR or BCR repertoire analysis is information obtained via the processing of the nucleic acid analysis result by the processing of the present invention.

Further, a computer program stored in the program storing section 1440 configures a computer as a system for implementing the above-described processing system, e.g., a system for implementing processing such as trimming, extraction, alignment, assignment, classification, or translation. Each of the features is an independent computer program, module or routine thereof or the like, which is executed by the above-described CPU 1401 to configure a computer as each system or apparatus. Hereinafter, each system is constructed by cooperation of each function in each system.

(Repertoire Analysis System/Analysis Method)

In one aspect, the present invention provides a method of quantitatively analyzing a repertoire of a variable region of a T cell receptor (TCR) or a B cell receptor (BCR) of a subject by using a database. The method comprises: (1) providing a nucleic acid sample comprising a nucleic acid sequence of the T cell receptor (TCR) or the B cell receptor (BCR) which is amplified from the subject in an unbiased manner; (2) determining the nucleic acid sequence comprised in the nucleic acid sample; and (3) calculating a frequency of appearance of each gene or a combination thereof based on the determined nucleic acid sequence to derive a TCR or BCR repertoire of the subject. This method and methods comprising one or more additional features explained herein are called “repertoire analysis method of the present invention” herein. In addition, a system materializing the repertoire analysis method of the present invention is referred to as the “repertoire analysis system of the present invention”.

The (1) providing a nucleic acid sample comprising a nucleic acid sequence of the T cell receptor (TCR) or the B cell receptor (BCR) which is amplified from the subject in an unbiased manner in the method of the present invention may provide any sample as long as the sample is suitable for determining a nucleic acid sequence. As such a technology, it is possible to use the above-described preferred amplification methods of the present invention as well as Reverse transcriptase-PCR, real-time PCR, digital PCR, emulsion PCR, amplified fragment length polymorphism (AFLP) PCR, allele specific PCR, assembly PCR, asymmetrical PCR, colony PCR, helicase-dependent amplification, hot start PCR, inverse PCR, in situ PCR, nested PCR, Touchdown PCR, loop-mediated isothermal PCR (LAMP), Nucleic acid sequence based amplification (NASBA), Ligase Chain Reaction, Branch DNA Amplification, Rolling Circle Amplification, Circle to circle Amplification, SPIA amplification, Target Amplification by Capture and Ligation (TACL), 5′-Rapid amplification of cDNA end (5′-RACE), 3′-Rapid amplification of cDNA end (3′-RACE), Switching Mechanism at 5′-end of the RNA Transcript (SMART).

The (2) determining the nucleic acid sequence comprised in the nucleic acid sample in the method of the present invention may use any method, as long as a nucleic acid sequence can be determined. Generally, a large quantity of sequencing is required. Thus, it is preferable to use an automated large-scale sequencing method. Examples of such a sequencing method include sequencing using a Roche 454 sequencer (GS FLX+, GS Junior), sequencing using the technology of an Ion Torrent sequencer (Ion PGM™ Sequencer), and sequencing using the technology of Illumina (GenomeAnalyzer IIx, Hiseq, Miseq). Other sequencing methods include Heliscope™ Sequencer, Helicos True Single Molecule Sequencing (tSMA) (Harris. T. D. et. al Science 2008, 320-160-109), SoliD™ Sequencing(Life Technologies, Inc.), Single Molecule Real Time (SMRT™) PacBio system (Pacific Biosciences, CA), Nanopore Sequencing (Oxford Nanopore Technologies, UK), LaserGen™ (LaserGen, Inc. CA) (reference: Litosh V A et al., Nucleic Acids Res. 2011 March; 39(6): e39), Lightspeed Genomics™ (Lightspeed Genomics, CA), GnuBIO (GnuBIO Inc., MA), Polonator sequencing (M. Danaher/Dover, Azco Biotec. Inc., CA), Mebious Biosystem's single molecule sequencing (Mebious Biosystems Limited), Millikan sequencing (Caerus Molecular Diagnostics, Inc), Intelligent Bio-Systems, Inc. (reference: Hutter D, et al Nucleosides Nucleotides Nucleic Acid 2010; 29(11):879-95.), Hybridization-Assisted Nanopore Sequencing (Nabsys Inc., RI), Nanopore sequencing (Noblegen Biosciences, Inc.), Nanopore sequencing (Electronic Biosciences, CA), Thermosequencing (GENIUS™ technology) (Genapsys, Inc., CA), CAERUS MOLECULAR DIAGNOTICS, INC, CA, Individual Molecule Placement Rapid Nanotransfer (IMPRNT) (Halcyon Molecular, Inc), Monochromatic aberration-corrected dual-beam low energy electron microscopy (Electron Optica, Inc., CA), ZS Genesis DNA Sequencing (ZS Genetyics, Inc) and the like. A Roche 454 sequencer creates a single stranded DNA bound to two types of adaptors which specifically bind to the 3′ terminal and the 5′ terminal. The single stranded DNA is bound to a bead via an adaptor and wrapped in a water-in-oil emulsion to form a microreactor having a bead and a DNA fragment. A gene of interest is then amplified by emulsion PCR in the water-in-oil emulsion. The bead is applied to a picotiter plate and sequenced. ATP is generated by sulfrylase, with pyrophosphoric acid generated when dNTP is taken up into a DNA by DNA polymerase as a substrate (Pyrosequencing). With the ATP and Luciferin as the substrate, luciferase emits fluorescence, which is detected with a CCD camera to determine a base sequence. For the technology of Ion Torrent, emulsion PCR is performed by the same method as Roche, and then a bead is transferred to a microchip, where a sequencing reaction is performed. For detection, the hydrogen ion concentration released when a DNA is extended by polymerase is detected on a semiconductor chip and converted to a base sequence. The sequencing of Illumia is a method of sequencing while amplifying and synthesizing a DNA of interest on a flow cell by the technology of bridge PCR and sequencing-by-synthesis. Bridge PCR creates a single stranded DNA, to which different adaptor sequences are added to both ends. An adaptor sequence is immobilized on the 5′ terminal side in advance on a flow cell, where it is immobilized on to the flow cell by an extension reaction. Similarly, an adaptor is immobilized on the 3′ terminal side at an adjacent position and binds to the 3′ terminal of a synthesized DNA to synthesize a double stranded DNA while forming a so-called bridge. Bridge binding→extension→denaturation are then repeated such that numerous single stranded DNA fragments are locally amplified to form an accumulated cluster. With such a single stranded DNA as a template, sequencing is performed. For sequencing-by-synthesis, after a sequencing primer is added, a single base synthesizing reaction is performed with 3′ terminal block fluorescent dNTP using DNA polymerase. A fluorescent substance bound to a base is excited by a laser light, and light emission is recorded as a picture by a fluorescence microscope. The base sequence is then determined by proceeding with a step of removing the fluorescent substance and the block to perform the next extension reaction and detecting fluorescence. Preferably, it is advantageous to sequence a plurality of sequence by a single sequencing. It is also advantageous in that a longer sequence length can be sequenced at once.

For the (3) calculating a frequency of appearance of each gene or a combination thereof based on the determined nucleic acid sequence to derive a TCR or BCR repertoire of the subject in the present invention, any technique can be used as long as a frequency of appearance of genes and a combination thereof can be calculated and a TCR repertoire and/or BCR repertoire can be derived. For example, the analysis tool HighV-Quest provided by the IMGT can be used, in addition to the preferred examples of the aforementioned analysis methods. It is also possible to use other technologies by using a software implemented with an alignment feature or a mapping feature, i.e., AbMapper, ALLPATHS, Arachne, BACCardl, Bfast, BLAT, Bowtie, BWA-MEM, BWA-SW, BWA, CCRa VAT & QuTie, CLC workstation, CNV-seq, Elvira, ERNE-map (rNA), GSMapper, Glimmer, gnumap, Goseq, ICAtools, LOCAS, MapSplice, Maq, MEMS, Mosaik, NGSView, Novoalign, OSLay, Partek, Perm, Projector, Qpalma, RazerS, SHARCGS, SHRiMP2, SNP-o-matic, Splicemap, SSAHA2, Stampy, Tablet, TMAP, Tophat, or Velve.

In one embodiment, the nucleic acid sample comprises nucleic acid sequences of a plurality of types of T cell receptors (TCR) or B cell receptors (BCR) and the step (2) for determining a sequence determines the nucleic acid sequence by a single sequencing. The method of the present invention can reduce or eliminate bias that can occur by determining a plurality of types of sequences by a single sequencing. Thus, the present invention is useful especially in accurately detecting a TCR or BCR read that occurs at a low frequency.

In another embodiment, the single sequencing is characterized in that at least one of the sequences used as a primer in amplification from the nucleic acid sample into a sample for sequencing has the same sequence as a nucleic acid sequence encoding a C region or a complementary strand thereof. Any TCR or BCR can be amplified in the same manner to achieve unbiasedness by using a primer having the same sequence as a nucleic acid sequence encoding a C region or a complementary strand thereof.

In another embodiment, the single sequencing is characterized in being performed with a common adaptor primer. In a preferred embodiment, the common adaptor primer is designed such that the primer has a base length suitable for amplification, is unlikely to have homodimer and intramolecular hairpin structures, and is able to stably form a double strand, and designed not to be highly homologous with all TCR genetic sequences in the database and/or to have the same level of melting temperature (Tm) as the C region specific primer. More preferably, the common adaptor primer designed not to have homodimer and intramolecular hairpin structures and to have homology with other genes comprising a BCR or TCR is selected. In a specific embodiment, the common adaptor primer is P20EA (SEQ ID NO: 2) and/or P10EA (SEQ ID NO: 3).

In one embodiment, the unbiased amplification includes being non-V region specific amplification. Bias can be further reduced or eliminated compared to a case of performing unbiased amplification by devising a multiplex or the like using a V specific primer.

In one embodiment, a repertoire targeted by the present invention is the repertoire of a variable region of a BCR, and the nucleic acid sequence is a BCR nucleic acid sequence. BCRs are considered to be prone to having a mutation, especially in a V region. Thus, accurate analysis of a BCR repertoire is difficult with a technology using V region specific amplification.

In one aspect, the present invention provides a method of analyzing a disease, disorder or condition of the subject based on the TCR or BCR repertoire derived based on the repertoire analysis method of the present invention.

In the method of analyzing a disease, disorder or condition of the present invention, the technology of analyzing a disease, disorder or condition of a subject based on a TCR or BCR repertoire derived based on the repertoire analysis method of the present invention starts from linking derived read data consisting of read types, number of reads, read frequency, V region, J region, C region, CDR3 sequence or the like with clinical information such as disease, disorder, or condition to form a database by using a spreadsheet such as EXCEL. First, for a derived individual read sequence: 1. a TCR having a known function such as NKT or MAIT is search; 2. existing public database is searched for collation with a TCR or BCR with a known function such as antigen specificity; 3. the constructed database or an existing public database is searched to associate a common sample origin, property or function with a disease, disorder or condition. Next, for a read sequence in a sample: 1. it is clarified whether a specific read frequency increases (clonality increases); 2. examination is carried out to find out whether a specific V chain or J chain usage frequency increases or decreases depending on the onset of a disease or condition of a disorder; 3. examination is carried out to find out whether the length of a CDR3 sequence in a specific V chain increases or decreases depending on the onset of a disease or condition of a disorder; 4. the composition or sequence of a CDR3 region that changes depending on the onset of a disease or condition of a disorder is examined. 5. a read that appears or disappears depending on the onset of a disease or condition of a disorder is searched; 6. a read that increases or decreases depending on the onset of a disease or condition of a disorder is searched; 7. a read that appears or increase/decrease depending on the onset of a disease or condition of a disorder is searched in another sample and associated with a disease, disorder or condition; 8. a diversity index or similarly index is calculated with a statistical analysis software such as ESTIMATES or R (vegan) by using data such as number of samples, read type, or the number of reads; and 9. a change in the diversity index or similarity index can be associated with the onset of a disease or condition of a disorder.

In one embodiment, the disease, disorder or condition of the subject in the analysis method of the present invention includes, but is not limited to, hematological tumor, colorectal cancer, immune status, rheumatoid arthritis, adult T-cell leukemia, T-cell large granular lymphocyte leukemia, idiopathic thrombocytopenic purpura, and the like.

In another embodiment, the present invention provides a method of treating or preventing the disease, disorder or condition of the subject determined by the method of the present invention, comprising: quantitatively associating the disease, disorder or condition of the subject with the TCR or BCR repertoire; and selecting means for suitable treatment or prevention from the quantitative association.

In one embodiment, diseases, disorders or conditions of a subject targeted in the method of treating or preventing in the present invention include, but are not limited to, hematological tumor, colorectal cancer, immune status, rheumatoid arthritis, adult T-cell leukemia, T-cell large granular lymphocyte leukemia, idiopathic thrombocytopenic purpura, and the like.

In another aspect, the present invention provides a system (analysis system) for quantitatively analyzing a repertoire of a variable region of a T cell receptor (TCR) or a B cell receptor (BCR) of a subject by using a database. The system comprises (1) a kit for providing a nucleic acid sample comprising a nucleic acid sequence of the T cell receptor (TCR) or the B cell receptor (BCR) which is amplified from the subject in an unbiased manner; (2) an apparatus for determining the nucleic acid sequence comprised in the nucleic acid sample; and (3) an apparatus for calculating a frequency of appearance of each gene or a combination thereof based on the determined nucleic acid sequence to derive a TCR or BCR repertoire of the subject. Such a system and systems comprising one or more additional features explained herein are referred to as “repertoire analysis system of the present invention”. The repertoire analysis system of the present invention materializes the “repertoire analysis method of the present invention”.

In another embodiment, the nucleic acid sample comprises nucleic acid sequences of a plurality of types of T cell receptors (TCR) or B cell receptors (BCR) and the apparatus of (2) is configured to be able to determine the nucleic acid sequences by a single sequencing.

In another embodiment, the single sequencing is characterized in that at least one of the sequences used as a primer in amplification from the nucleic acid sample to a sample for sequencing has the same sequence as a C region. The method of the present invention can reduce or eliminate bias that can occur by determining a plurality of types of sequence by a single sequencing. Thus, the present invention is useful especially in accurately detecting a TCR or BCR read that occurs at a low frequency.

In another embodiment, the single sequencing is characterized in that at least one of the sequences used as a primer in amplification from the nucleic acid sample to a sample for sequencing has the same sequence as a nucleic acid sequence encoding a C region or a complementary strand thereof. Such a primer may be furnished in the apparatus, comprised in a kit, or provided separately. Any TCR or BCR can be amplified in the same manner to achieve unbiasedness by using a primer having the same sequence as a nucleic acid sequence encoding a C region or a complementary strand thereof.

In another embodiment, the single sequencing is characterized in being performed with a common adaptor primer. Such a common adaptor primer may be furnished with the apparatus, comprised in a kit or provided separately. In a preferred embodiment, the common adaptor primer is designed such that the primer has a base length suitable for amplification, is unlikely to have homodimer and intramolecular hairpin structures, and is able to stably form a double strand, and designed not to be highly homologous with all TCR genetic sequences in the database and/or to have the same level of melting temperature (Tm) as the C region specific primer. Further preferably, the common adaptor primer designed not to have homodimer and intramolecular hairpin structures and to have homology with other genes comprising a BCR or TCR is selected. In a specific embodiment, the common adaptor primer is P20EA (SEQ ID NO: 2) and/or P10EA (SEQ ID NO: 3).

In one embodiment, a nucleic acid sequence comprised in a nucleic acid sample provided by the kit of the present invention is unbiasedly amplified, where the amplification is not V region specific amplification. Bias can be further reduced or eliminated compared to a case of performing unbiased amplification by devising a multiplex or the like using a V specific primer.

In one embodiment, the repertoire subjected to analysis of the system of the present invention is the repertoire of a variable region of a BCR, and the nucleic acid sequence is a BCR nucleic acid sequence. BCRs are considered to be prone to having a mutation, especially in a V region. Thus, accurate analysis of a BCR repertoire is difficult with a technology using V region specific amplification. Use of the system of the present invention allows accurate analysis of a BCR repertoire.

In another aspect, the present invention provides a system (analysis system) of analyzing a disease, disorder or condition of the subject, comprising the analysis system of the present invention and means for analyzing the disease, disorder or condition of the subject based on the TCR or BCR repertoire derived based the system. The means of analyzing a disease, disorder or condition of a subject based on a TCR or BCR repertoire derived based on the system of the analysis system of the present invention starts from linking derived read data consisting of read types, number of reads, read frequency, V region, J region, C region CDR3 sequence or the like with clinical information such as disease, disorder, or condition to form a database by using a spreadsheet such as EXCEL. First, for a derived individual read sequence: 1. a TCR having a known function such as NKT or MAIT is searched; 2. existing public database is searched for collation with a TCR or BCR with a known function such as antigen specificity; 3. the constructed database or an existing public database is searched to associate a common sample origin, property or function with a disease, disorder or condition. Next, for a read sequence in a sample: 1. it is clarified whether a specific read frequency increases (clonality increases); 2. examination is carried out to find out whether a specific V chain or J chain usage frequency increases or decreases depending on the onset of a disease or condition of a disorder; 3. examination is carried out to find out whether the length of a CDR3 sequence in a specific V chain increases or decreases depending on the onset of a disease or condition of a disorder; 4. the composition or sequence of a CDR3 region that changes depending on the onset of a disease or condition of a disorder is examined. 5. a read that appears or disappears depending on the onset of a disease or condition of a disorder is searched; 6. a read that increases or decreases depending on the onset of a disease or condition of a disorder is searched; 7. a read that appears or increase/decrease depending on the onset of a disease or condition of a disorder is searched in another sample and associated with a disease, disorder or condition; 8. a diversity index or similarly index is calculated with a statistical analysis software such as ESTIMATES or R (vegan) by using data such as number of samples, read type, or number of reads; and 9. a change in the diversity index or similarity index can be associated with the onset of a disease or condition of a disorder.

In one embodiment, the disease, disorder or condition of the subject that can be analyzed by the analysis system of the present invention includes, but is not limited to, hematological tumor, colorectal cancer, immune status, rheumatoid arthritis, adult T-cell leukemia, T-cell large granular lymphocyte leukemia, idiopathic thrombocytopenic purpura, and the like.

In another aspect, the present invention provides a system (treatment system or prevention system) of treating or preventing the disease, disorder or condition of the subject determined by the analysis system of the present invention, comprising: means for quantitatively associating the disease, disorder or condition of the subject with the TCR or BCR repertoire; and means for selecting means for suitable treatment or prevention from the quantitative association.

The means for quantitatively associating the disease, disorder or condition of the subject with the TCR or BCR repertoire in the system of the present invention can be materialized by the following configuration or the like. That is, this can be materialized by reading out information of a repertoire derived out by the analysis system of the present invention and reading out information related to a disease, disorder or condition of a subject and associating them. From the aggregated read data that is derived out, a V region, a J region, and a C region is assigned from collation with an existing reference sequence and a CDR3 sequence is determined. Matching reads are added up based on the V region, J region and CDR3 sequence. For each unique read (read with no other same sequence), the number of reads detected and the ratio with respect to the total number of reads (frequency) in a sample are calculated. The information (read sequence, number of reads, read frequency, V region, J region, C region, or CDR3 sequence) is linked with the clinical information of a subject (medical history, disease name, disease type, degree of progression, severity, HLA type, immune status or the like) to form a database by using a spreadsheet such as EXCEL or a software having a database formation feature. Read sequences in a sample are sorted by the number of reads or frequency and ranked. Further, the number of reads is added up by each V region or J region to calculate usage frequency of a V region or usage frequency of a J region. Base on such information: 1. it is clarified whether a specific read frequency increases (clonality increases); 2. examination is carried out to find out whether a specific V chain or J chain usage frequency increases or decreases depending on the onset of a disease or condition of a disorder; 3. examination is carried out to find out whether the length of a CDR3 sequence in a specific V chain increases or decreases depending on the onset of a disease or condition of a disorder; 4. the composition or sequence of a CDR3 region that changes depending on the onset of a disease or condition of a disorder is examined. 5. a read that appears or disappears depending on the onset of a disease or condition of a disorder is searched; 6. a read that increases or decreases depending on the onset of a disease or condition of a disorder is searched; 7. a read that appears or increase/decrease depending on the onset of a disease or condition of a disorder is searched in another sample and associated with a disease, disorder or condition; 8. a diversity index or similarly index is calculated with a statistical analysis software such as ESTIMATES or R (vegan) by using data such as number of samples, read type, or number of reads; and 9. a change in the diversity index or similarity index can be associated with the onset of a disease or condition of a disorder. Means for selecting means for suitable treatment or prevention from the quantitative association can have the following configuration or the like. Specifically, selection can be materialized for this selection means by associating quantitatively presented data with the information from the past or currently available information related to treatment, therapy or prevention to materialize selection of means that improves prognosis.

In one embodiment, the disease, disorder or condition of the subject includes, but is not limited to, hematological tumor, colorectal cancer, immune status, rheumatoid arthritis, adult T-cell leukemia, T-cell large granular lymphocyte leukemia, idiopathic thrombocytopenic purpura, and the like.

(Useful Cell, Peptide, and the Like)

In one aspect, the present invention provides a monoclonal T cell related to T cell large granular lymphocytic leukemia (T-LGL) expressing TCRα comprising TRAV10/TRAJ15/CVVRATGTALIFG (SEQ ID NO: 1450) or a nucleic acid encoding the same and/or TCR comprising TRBV29-1/TRBJ2-7/CSVERGGSLGEQYFG (SEQ ID NO: 1500) or a nucleic acid encoding the same.

As shown in the Examples or the like, this specific T cell has a variety of usefulness. For instance, it is demonstrated that TRAV10/TRAJ15/CVVRATGTALIFG (SEQ ID NO: 1450) or a nucleic acid encoding the same in TCRα and/or TRBV29-1/TRBJ2-7/CSVERGGSLGEQYFG (SEQ ID NO: 1500) or a nucleic acid encoding the same in TCR can be used as a diagnostic indicator for T cell large granular lymphocytic leukemia (T-LGL). Such a peptide and a nucleic acid encoding the same can be detected by using any known technology in the art. As used herein, “detecting agent” broadly refers to all agents capable of detecting a target of interest (e.g., a peptide, nucleic acid, cell or the like). For instance, for such a method, “detection” or “quantification” of polynucleotide or polypeptide expression can be accomplished by using a suitable method including, for example, an immunological measuring method and measurement of mRNAs, including bond or interaction with a marker detecting agent. Examples of a molecular biological measuring method include northern blot, dot blot, PCR and the like. Examples of an immunological measuring method include ELISA using a microtiter plate, RIA, fluorescent antibody method, luminescence immunoassay (LIA), immunoprecipitation (IP), single radial immunodiffusion (SRID), turbidimetric immunoassay (TIA), western blot, immunohistochemical staining and the like. Further, a quantification method includes ELISA, RIA and the like. Quantification may also be performed by a genetic analysis method using an array (e.g., DNA array, protein array). DNA arrays are outlined extensively in (Ed. by Shujunsha, Saibo Kogaku Bessatsu “DNA Maikuroarei to Saishin PCR ho” [Cellular engineering, Extra issue, “DNA Microarrays and Latest PCR Methods”]). Protein arrays are discussed in detail in Nat Genet. 2002 December; 32 Suppl: 526-32. Examples of a method of analyzing gene expression include, but are not limited to, RT-PCR, RACE, SSCP, immunoprecipitation, two-hybrid system, in vitro translation and the like, in addition to the methods discussed above. Such additional analysis methods are described in, for example, Genomu Kaiseki Jikkenho Nakamura Yusuke Labo Manyuaru [Genome analysis experimental method Yusuke Nakamura Lab Manual], Ed. by Yusuke Nakamura, Yodosha (2002) and the like. The entire descriptions therein is incorporated herein by reference. As used herein, “amount of expression” refers to the amount of polypeptide, mRNA or the like expressed in a cell, tissue or the like of interest. Examples of such an amount of expression include amount of expression of polypeptide of the present invention at a protein level assessed by any suitable method including an immunological measurement method such as ELISA, RIA, fluorescent antibody method, western blot, and immunohistochemical staining by using the antibody of the present invention, and the amount of expression of the polypeptide used in the present invention at an mRNA level assessed by any suitable method including molecular biological measuring method such as northern blot, dot blot, and PCR. “Change in the amount of expression” refers to an increase or decrease in the amount of expression of the polypeptide used in the present invention at a protein level or mRNA level assessed by any suitable method including the above-described immunological measuring method or molecular biological measuring method. A variety of detection or diagnosis based on a marker can be performed by measuring the amount of expression of a certain marker.

The present invention also provides a diagnostic agent for T cell large granular lymphocytic leukemia (T-cell LGL) comprising a detecting agent for TRAV10/TRAJ15/CVVRATGTALIFG (SEQ ID NO: 1450) or a nucleic acid encoding the same in TCRα and/or a detecting agent for TRBV29-1/TRBJ2-7/CSVERGGSLGEQYFG (SEQ ID NO: 1500) or a nucleic acid encoding the same in TCRβ.

As used herein, “decrease” or “suppression” of activity or expression product (e.g., protein, transcript (RNA or the like)) or synonyms thereof refers to: a decrease in the amount, quality or effect of a specific activity, transcript or protein; or activity that decreases the same.

As used herein, “increase” or “activation” of activity or expression product (e.g., protein, transcript (RNA or the like)) or synonyms thereof refers to: an increase in the amount, quality or effect of a specific activity, transcript or protein; or activity that increases the same.

Thus, it is understood that various agents with activity can be detected or screened by using a regulatory ability such as decrease, suppression, increase or activation of the marker of the present invention as an indicator.

As used herein, “agent” is used broadly and may be any substance or other elements (e.g., energy, radiation, heat, electricity and other forms of energy) as long as the intended objective can be achieved. Examples of such a substance include, but are not limited to, protein, polypeptide, oligopeptide, peptide, polynucleotide, oligonucleotide, nucleotide, nucleic acid (including for example DNAs such as cDNA and genomic DNA, RNAs such as mRNA), polysaccharide, oligosaccharide, lipid, organic small molecule (e.g., hormone, ligand, information transmitting substance, organic small molecule, molecule synthesized by combinatorial chemistry, small molecule that can be used as a medicine (e.g., small molecule ligand and the like) and a composite molecule thereof. Typical examples of an agent specific to a polynucleotide include, but are not limited to, a polynucleotide having complementarity with a certain sequence homology (e.g., 70% or greater sequence identity) to a sequence of the polynucleotide, polypeptide such as a transcription factor that binds to a promoter region and the like. Typical examples of an agent specific to a polypeptide include, but are not limited to, an antibody directed specifically to the polypeptide or a derivative or analog thereof (e.g., single strand antibody), a specific ligand or receptor when the polypeptide is a receptor or ligand, a substrate when the polypeptide is an enzyme and the like.

As used herein, “detecting agent” broadly refers to all agents capable of detecting a target of interest (e.g., normal cells (normal corneal endothelial cells) or transformed cells (e.g., transformed corneal endothelial cells)).

As used herein, “diagnostic agent” broadly refers to all agents capable of diagnosing a condition of interest (e.g., disease or the like).

The detecting agent of the present invention may be a complex or composite molecule in which another substance (e.g., label or the like) is bound to a portion enabling detection (e.g., antibody or the like). As used herein, “complex” or “composite molecule” refers to any construct comprising two or more portions. For instance, when one portion is a polypeptide, the other portion may be a polypeptide or other substances (e.g., sugar, lipid, nucleic acid, other carbohydrate or the like). As used herein, two or more constituent portions of a complex may be bound by a covalent bond or any other bond (e.g., hydrogen bond, ionic bond, hydrophobic interaction, Van der Waals force or the like). When two or more portions are polypeptides, the complex may be called a chimeric polypeptide. Thus, “complex” as used herein includes molecules formed by linking a plurality of types of molecules such as a polypeptide, polynucleotide, lipid, sugar, or small molecule.

As used herein, “interaction” refers, for two substances, to applying a force (e.g., intermolecular force (Van der Waals force), hydrogen bond, hydrophobic interaction, or the like) between one substance and the other substance. Generally, two substances that have interacted are in a conjugated or bound state.

As used herein, the term “bond” refers to a physical or chemical interaction between two substances or between combinations thereof. A bond includes an ionic bond, non-ionic bond, hydrogen bond, Van der Waals bond, hydrophobic interaction and the like. A physical interaction (bond) may be direct or indirect. Indirect physical interaction (bond) is mediated by or is due to an effect of another protein or compound. A direct bond refers to an interaction, which does not occur through or due to an effect of another protein or compound and does not substantially involve another intermediate. The degree of marker expression of the present invention or the like can be measured by measuring a bond or interaction.

Thus, an “agent” (such as detecting agent or the like) that “specifically” interacts (or binds) to a biological agent such as a polynucleotide or a polypeptide as used herein encompasses agents with affinity to the biological agent such as a polynucleotide or polypeptide that is typically similar or higher, preferably significantly (e.g., statistically significantly) higher, than affinity to other unrelated polynucleotide or polypeptide (especially those with less than 30% identity). Such affinity can be measured by, for example, hybridization assay, binding assay or the like.

As used herein, “specific” interaction (or bond) of a first substance or agent with a second substance or agent refers to the first substance or agent interacting with (or binding to) the second substance or agent at a higher level of affinity than to substances or agents other than the second substance or agent (especially other substances or agents in a sample comprising the second substance or agent). Examples of an interaction (or bond) specific to a substance or agent include, but are not limited to, a ligand-receptor reaction, hybridization in a nucleic acid, antigen-antibody reaction in a protein, enzyme-substrate reaction and the like, and when both a nucleic acid and a protein are involved, a reaction between a transcription factor and a binding site of the transcription factor and the like, protein-lipid interaction, nucleic acid-lipid interaction and the like. Thus, when substances or agents are both nucleic acids, a first substance or agent “specifically interacting” with a second substance or agent encompasses the first substance or agent having at least partial complementarity to the second substance or agent. Further, examples of a first substance or agent “specifically” interacting with (or binding to) a second substance or agent when substances or agents are both proteins includes, but are not limited to, interaction by an antigen-antibody reaction, interaction by a receptor-ligand reaction, enzyme-substrate interaction and the like. When two types of substances or agents include a protein and a nucleic acid, a first substance or agent “specifically” interacting with (or binding to) a second substance or factor encompasses an interaction (or a bond) between a transcription factor and a binding region of a nucleic acid molecule targeted by the transcription factor.

As used herein, “antibody” broadly encompasses a polyclonal antibody, monoclonal antibody, multispecific antibody, chimeric antibody, and anti-idiotype antibody, and a fragment thereof such as Fv fragment, Fab′ fragment, F(ab′)2 and Fab fragment, and other conjugates or functional equivalents produced by recombination (e.g., chimeric antibody, humanized antibody, multifunctional antibody, bispecific or oligospecific antibody, single chain antibody, scFv, diabody, and sc(Fv)2 (single chain (Fv)2), and scFv-Fc). Furthermore, such an antibody may be covalently bonded or recombinantly fused to an enzyme, such as alkaline phosphatase, horseradish peroxidase, a galactosidase or the like. The antibodies to various reads used in the present invention may be of any origin, type, shape or the like which bind to their respective specific read. Specifically, known antibodies such as a non-human animal antibody (e.g., mouse antibody, rat antibody, and camel antibody), human antibody, chimeric antibody and humanized antibody can be used. The present invention can use a monoclonal or polyclonal antibody, but a monoclonal antibody is preferred. It is preferred that a bond of an antibody to a specific read is a specific bond.

As used herein, “antigen” refers to any substrate that can be specifically bound by an antibody molecule. As used herein, “immunogen” refers to an antigen that can initiate lymphocyte activation which leads to an antigen specific immune response. As used herein, “epitope” or “antigen determinant” refers to a site in an antigen molecule to which an antibody or a lymphocyte receptor binds. A method of determining an epitope is well known in the art. Such an epitope can be determined by those skilled in the art by using a well-known and conventional technique when a primary sequence of an amino acid or a nucleic acid is provided.

As used herein, “means” refers to anything that can be a tool for accomplishing an objective (e.g., detection, diagnosis, therapy).

For an antibody used herein, it is understood that an antibody with any specificity may be used as long as false positive reactions are reduced. Thus, an antibody used in the present invention may be a polyclonal antibody or a monoclonal antibody.

The detecting agent, diagnostic agent or other medicines of the present invention can be in a form of a probe and a primer. The probe and the primer of the present invention can specifically hybridize to a specific read. As described herein, expression of a specific read is, for example, an indicator of whether there is a colorectal cancer and is useful as an indicator for the severity of a disease.

As used herein, “(nucleic acid) primer” refers to a substance required for initiating a reaction of a polymeric compound to be synthesized in a polymer synthesizing enzyme reaction. A synthetic reaction of a nucleic acid molecule can use a nucleic acid molecule (e.g., DNA, RNA or the like) complementary to a portion of a sequence of a polymeric compound to be synthesized. A primer can be used herein as a marker detecting means.

Examples of a nucleic acid molecule generally used as a primer include those having a nucleic acid sequence with a length of at least 8 contiguous nucleotides, which is complementary to a nucleic acid sequence of a gene of interest (e.g., marker of the present invention). Such a nucleic acid sequence may be a nucleic acid sequence with a length of preferably at least 9 contiguous nucleotides, more preferably at least 10 contiguous nucleotides, still more preferably at least 11 contiguous nucleotides, at least 12 contiguous nucleotides, at least 13 contiguous nucleotides, at least 14 contiguous nucleotides, at least 15 contiguous nucleotides, at least 16 contiguous nucleotides, at least 17 contiguous nucleotides, at least contiguous nucleotides, at least 19 contiguous nucleotides, at least 20 contiguous nucleotides, at least contiguous nucleotides, at least 30 contiguous nucleotides, at least 40 contiguous nucleotides, or at least 50 contiguous nucleotides. A nucleic acid sequence used as a probe comprises a nucleic acid sequence that is at least 70% homologous, more preferably at least 80% homologous, still more preferably at least 90% homologous, or at least 95% homologous to the aforementioned sequence. A sequence suitable as a primer may vary depending on the property of a sequence intended for synthesis (amplification). However, those skilled in the art are capable of designing an appropriately primer in accordance with an intended sequence. Design of such a primer is well known in the art, which may be performed manually or by using a computer program (e.g., LASERGENE, PrimerSelect, or DNAStar).

The primers according to the present invention can be used as a primer set consisting of two or more types of the primers.

The primers and primer set according to the present invention can be used as primers and primer set in accordance with a common method in a known method of detecting a gene of interest by utilizing a nucleic acid amplification method such as PCR, RT-PCR, real-time PCR, in situ PCR, or LAMP.

As used herein, “probe” refers to a substance that can be means for search, which is used in a biological experiment such as in vitro and/or in vivo screening. Examples thereof include, but are not limited to, a nucleic acid molecule comprising a specific base sequence, a peptide comprising a specific amino acid sequence, a specific antibody, a fragment thereof and the like. As used herein, a probe can be used as marker detecting means.

A nucleic acid molecule generally used as a probe includes those having a nucleic acid sequence with a length of at least 8 contiguous nucleotides, which is homologous or complementary to a nucleic acid sequence of a gene of interest. Such a nucleic acid sequence may be a nucleic acid sequence with a length of preferably at least 9 contiguous nucleotides, more preferably at least 10 contiguous nucleotides, still more preferably at least 11 contiguous nucleotides, at least 12 contiguous nucleotides, at least 13 contiguous nucleotides, at least 14 contiguous nucleotides, at least 15 contiguous nucleotides, at least contiguous nucleotides, at least 25 contiguous nucleotides, at least 30 contiguous nucleotides, at least contiguous nucleotides, or at least 50 contiguous nucleotides. A nucleic acid sequence used as a probe comprises a nucleic acid sequence that is at least about 70% homologous, more preferably at least about 80% homologous, still more preferably at least about 90% homologous, or at least about 95% homologous with the aforementioned sequence.

In one embodiment, the detecting agent of the present invention may be labeled. Alternatively, the detecting agent of the present invention may be bound to a tag.

As used herein, “label” refers to an entity (e.g., substance, energy, electromagnetic wave or the like) for distinguishing a molecule or substance of interest from others. Such a method of labeling includes RI (radioisotope) method, fluorescence method, biotin method, chemiluminescent method and the like. When a plurality of markers of the present invention or agents or means for capturing the same are labeled by a fluorescence method, labeling is performed with labeling substances having different fluorescent emission maximum wavelengths. It is preferable that the difference in fluorescent emission maximum wavelengths is 10 nm or greater. When labeling a ligand, any label that does not affect the function can be used. However, Alexa™Fluor is desirable as a fluorescent substance. Alexa™Fluor is a water-soluble fluorescent dye obtained by modifying coumarin, rhodamine, fluorescein, cyanine or the like. This is a series compatible with a wide range of fluorescence wavelengths. Relative to other fluorescent dyes for the corresponding wavelength, Alexa™Fluor is very stable, bright and has a low level of pH sensitivity. Combinations of fluorescent dyes with fluorescence maximum wavelength of 10 nm or greater include a combination of Alexa™555 and Alexa™633, combination of Alexa™488 and Alexa™555 and the like. When a nucleic acid is labeled, any substance can be used that can bind to a base portion thereof. However, it is preferable to use a cyanine dye (e.g., Cy3, Cy5 or the like of the CyDye™ series), rhodamine 6G reagent, N-acetoxy-N2-acetylaminofluorene (AAF), AAIF (iodine derivative of AAF) or the like. Examples of a fluorescent substance with a difference in fluorescent emission maximum wavelengths of nm or greater include a combination of Cy5 and a rhodamine 6G reagent, a combination of Cy3 and fluorescein, a combination of a rhodamine 6G reagent and fluorescein and the like. The present invention can utilize such a label to alter a subject of interest to be detectable by the detecting means to be used. Such alteration is known in the art. Those skilled in the art can appropriately carry out such a method in accordance with the label and subject of interest.

As used herein, “tag” refers to a substance for distinguishing a molecule by a specific recognition mechanism such as receptor-ligand, or more specifically, a substance serving the role of a binding partner to bind a specific substance (e.g., having a relationship such as biotin-avidin or biotin-streptavidin). A tag can be encompassed in the scope of “label”. Accordingly, a specific substance to which a tag is bound can distinguish the specific substance by a contact with a substrate, to which a binding partner of a tag sequence is bound. Such a tag or label is known in the art. Typical tag sequences include, but are not limited to, myc tag, His tag, HA, Avi tag and the like. Such a tag may be bound to the marker or marker detecting agent of the present invention.

The method of the present invention can be carried out by contacting the detecting agent or diagnostic agent of the present invention with a sample of interest to measure whether there is a target read of interest or a gene of the lead in the sample, or the level or amount thereof.

As used herein, “contact(ed)” refers to physically adjoin, either directly or indirectly, a substance to a polypeptide or a polynucleotide that can function as the marker, detecting agent, diagnostic agent, ligand or the like of the present invention. A polypeptide or polynucleotide can be included in many buffer solutions, salts, solutions and the like. Contact includes placing a compound in, for example, a beaker, microtiter plate, cell culture flask, microarray (e.g., gene chip) or the like comprising a polypeptide encoding a nucleic acid molecule or a fragment thereof.

In another aspect, the present invention provides a peptide which is a novel invariant TCR, comprising any one of the sequences set forth in SEQ ID NOs: 1627-1647. Such a peptide can be used as an invariant and applied as various indicators (e.g., indicator of a disease or the like).

In still another aspect, the present invention provides a TCR peptide having a mucosal-associated invariant T (MAIT) cell, comprising a sequence selected from the group consisting of SEQ ID NOs 1648-1651, 1653-1654, 1666-1667, 1844-1848, and 1851 or a nucleic acid encoding such a peptide. Such a peptide and nucleic acid can be used as a mucosal-associated invariant T (MAIT) and applied as various indicators (e.g., indicator of a disease or the like). In one specific embodiment, a peptide which is a TCR having a mucosal-associated invariant T (MAIT) of the present invention or a nucleic acid encoding such a peptide can be used as a diagnostic indicator of colorectal cancer.

In another aspect, the present invention provides a peptide, which is a TCR having a natural killer T (NKT) cell comprising the sequence set forth in SEQ ID NO: 1668 and a nucleic acid encoding the peptide. In one specific embodiment, the peptide, which is a TCR having a natural killer T (NKT) cell and a nucleic acid encoding the peptide can be used as a diagnostic indicator of colorectal cancer.

In another aspect, the present invention provides a colorectal cancer-specific peptide comprising a sequence selected from the group consisting of SEQ ID NOs: 1652, 1655-1665, 1669-1843, 1849-1850, and 1852-1860 and a nucleic acid encoding the same. In one specific embodiment, such a peptide and nucleic acid encoding the same can be used as a diagnostic indicator of colorectal cancer.

In a further aspect, the present invention provides a colorectal cancer specific peptide, comprising a sequence selected from the group consisting of SEQ ID NOs: 1861-1865 and 1867-1909 and a nucleic acid encoding the same. In one specific embodiment, such a peptide and nucleic acid encoding the same can be used as a diagnostic indicator of colorectal cancer.

In another aspect, the present invention provides a cell population inducing a T cell at a high frequency, T cell cell line, or recombinantly expressed T cell having a peptide comprising a sequence selected from the group consisting of SEQ ID NOs: 1652, 1655-1665, 1669-1843, 1849-1850, and 1852-1860 and SEQ ID NOs: 1861-1865 and 1867-1909 or a nucleic acid sequence encoding the peptide. A cell population, cell line, cell, colorectal cancer specific TCR peptide or nucleic acid encoding said peptide is useful in diagnosis or therapy. For diagnosis, colorectal cancer can be discovered, or pathological condition or prognosis can be predicted, by examining if the sequence is only in colorectal cancer patients, the sequence is observed more in colorectal cancer patients, or the sequence accumulates in cancer tissue of the same patient. For therapy of colorectal cancer, it is possible to utilize a cell population inducing a T cell at a high frequency with a colorectal cancer specific sequence, a T cell cell line with a colorectal cancer specific sequence, or a T cell (lymphocyte) artificially made to express a colorectal cancer specific sequence (As reference documents, see 1: Uttenthal B J, Chua I, Morris E C, Stauss H J. Challenges in T cell receptor gene therapy. J Gene Med. 2012 June; 14(6): 386-99. doi: 10.1002/jgm.2637. Review. PubMed PMID: 22610778.; 2: Linnemann C, Schumacher T N, Bendle G M. T-cell receptor gene therapy: critical parameters for clinical success. J Invest Dermatol. 2011 September; 131(9): 1806-16.doi: 10.1038/jid.2011.160. Epub 2011 Jun. 16. Review. PubMed PMID: 21677669.; 3: Lagisetty K H, Morgan R A. Cancer therapy with genetically-modified T cells for the treatment of melanoma. J Gene Med. 2012 June; 14(6): 400-4. doi: 10.1002/jgm.2636. Review. PubMed PMID: 22610729). Thus, the present invention provides a therapeutic agent or a prophylactic agent for colorectal cancer, comprising the above-described cell population, T cell cell line, or T cell.

(Application) The present invention can be used to calculate a base sequence (read) of a TCR or BCR gene identified by a large-scale sequencing and a frequency of appearance thereof with a software to draw a list, a distribution or a graph. Based on such information, a change in a repertoire is detected by using the following various indicators. Association with a disease or disorder can be found based on such a change.

In one aspect, the present invention provides a method of detecting a usage frequency of a V gene by using the analysis method or the analysis system of the present invention. A V gene of each read can be identified to calculate the percentage of each V gene with the respect to the entire TCR or BCR genes. It is possible to find an increase or decrease in usage frequency of V associated with a disease or pathological condition.

In another aspect, the present invention provides a method of detecting a usage frequency of a J gene by using the analysis method or the analysis system of the present invention. A J gene of each read can be identified to calculate the percentage of each J gene with the respect to the entire TCR or BCR genes. It is possible to find an increase or decrease in usage frequency of J associated with a disease or pathological condition.

In another aspect, the present invention provides a method of detecting a usage frequency of subtype frequency analysis (BCR) by using the analysis method or the analysis system of the present invention. It is possible to calculate the frequency of presence of subtypes IgA1, IgA2, IgG1, IgG2, IgG3, and IgG4 based on sequencing of a C region. It is possible to find an increase or decrease in a specific subtype associated with a disease or pathological condition.

In another aspect, the present invention provides a method of analyzing a pattern of CDR3 sequence lengths by using the analysis method or the analysis system of the present invention. A CDR3 base sequence length of each read can be calculated to find the distribution thereof. A normal distribution-like peak pattern is exhibited from normal TCRs or BCRs. It is possible to find the association with a disease or pathological condition by detecting a peak deviating away from a normal distribution.

In another aspect, the present invention provides a method of analyzing clonality of a TCR or a BCR by using the analysis method or the analysis system of the present invention. Reads having the same sequence are classified based on V sequence, J sequence, and CDR3 sequence of each read to calculate the number of copies thereof. It is possible to find a lead present at a high frequency by calculating the percentage of the number of copies of each lead relative to the number of all reads. The degree of clonality is assessed by sorting the reads in descending order by the frequency of appearance and comparing the percentage or number of reads that are present at a high frequency with a normal sample. A change in TCR or BCR clonality associated with a disease or pathological condition is examined therewith. The degree of clonality can be used particularly in detecting a leukemic cell or the like.

In another aspect, the present invention provides a method of extracting an overlapping read by using the analysis method or the analysis system of the present invention. A read of a sample classified by a specific disease, disease type, pathological condition, tissue, genotype (HLA or the like) is searched to extract overlapping TCR or BCR reads between samples. It is possible to find a TCR or BCR gene associated with a condition of a disease or disorder therewith. It is possible to identify a disease specific T cell involved in the pathology of an autoimmune disease, a B cell producing a disease associated antibody, a cancer specific T cell attacking a cancer cell or the like.

In another aspect, the present invention provides a method of searching for a disease specific TCR or BCR clone by using the analysis method or the analysis system of the present invention. It is possible to predict the progression or amelioration in a pathological condition or the onset of a disease by searching for a TCR or BCR read associated with a specific condition of a disorder or disease in a test sample and revealing the appearance or disappearance, or increase or decrease thereof.

In another aspect, the present invention provides a method of analyzing a subject with a diversity index by using the analysis method or the analysis system of the present invention. Alternatively, the present invention provides a method of assisting analysis on a subject with a diversity index by using the analysis method or the analysis system of the present invention. A read sequence identified based on a CDR3 sequence is counted and the number of read types and number of individuals are calculated to form an index for diversity of a TCR or BCR repertoire. The Shannon-Wiener's diversity index (H′), Simpson's diversity index (λ, 1−λ, or 1/λ), Pielou's evenness index (J′), Chaol index or the like is used to assess diversity by comparison with a normal sample. The index can be utilized as an indicator for measuring a degree of recovery of an immune system after bone marrow transplantation. Further, the index can be utilized as an indicator for detecting abnormality in an immune system cell accompanied by hematopoietic tumor.

In one embodiment, a method of analyzing a subject with a diversity index uses the diversity index as an indicator for measuring a degree of recovery of an immune system after bone marrow transplantation or as an indicator for detecting abnormality in a cell of the immune system accompanied by hematopoietic tumor. Such analysis using a diversity index was difficult with a conventional system.

Various diversity indices can be calculated by using an EXCEL spreadsheet or a software such as ESTIMATES (Colwell, R. K. et al. Journal of Plant Ecology 5: 3-21.) or R package (vegan) from data for the number of samples, read types, or the number of reads. The Shannon-Wiener's diversity index (H′), Simpson's diversity index (λ, 1−λ, or 1/λ), Pielou's evenness index (J′) and Chaol index are found by the mathematical equations shown below. N: total number of reads, ni: number of reads in read i Shannon-Weaver index H′

\(\begin{matrix}
{H^{\prime} = {- {\sum\limits_{i = 1}^{S}{\frac{n_{i}}{N}\ln\frac{n_{i}}{N}}}}} & \left\lbrack {{Numeral}\mspace{14mu} 1} \right\rbrack
\end{matrix}\)

Simpson's index λ

\(\begin{matrix}
{{1 - \lambda} = {1 - {\sum\limits_{i = 1}^{S}\left( \frac{n_{i}\left( {n_{i} - 1} \right)}{N\left( {N - 1} \right)} \right)}}} & \left\lbrack {{Numeral}\mspace{14mu} 2} \right\rbrack
\end{matrix}\)

Inverse Simpson's index

\(\begin{matrix}
\frac{1}{\lambda} & \left\lbrack {{Numeral}\mspace{14mu} 3} \right\rbrack
\end{matrix}\)

Pielou's J

\(\begin{matrix}
{J = \frac{H^{\prime}}{\log\; S}} & \left\lbrack {{Numeral}\mspace{14mu} 4} \right\rbrack
\end{matrix}\)

Schaol Sobs: total number of read types, F1: singleton read, F2: doubleton read

\(\begin{matrix}
{S_{{chao}\; 1} = {S_{obs} - {\left( \frac{n - 1}{n} \right)\frac{F_{1}\left( {F_{1} - 1} \right)}{2\left( {F_{2} + 1} \right)}}}} & \left\lbrack {{Numeral}\mspace{14mu} 5} \right\rbrack
\end{matrix}\)

In another aspect, the present invention is a method of analyzing a subject with a similarity index by using the analysis method or the analysis system of the present invention. Alternatively, the present invention provides a method of assisting analysis on a subject with a similarly index by using the analysis method or the analysis system of the present invention. The number of individuals and the number of types of read sequences identified based on a CDR3 sequence are calculated to find the degree of similarly of a TCR or BCR repertoire between samples to be compared. The Morisita-Horn index, Kimoto's Cn index, or Pianka's α index is used to find a degree of similarly between samples. Such an index can be utilized in the assessment of a degree of similarity of repertoires between matching and mismatching HLA types, assessment of a degree of similarly of repertoires between a recipient and a donor after bone marrow transplantation.

In one embodiment, the similarity index is used as assessment of a degree of similarity of repertoires between matching and mismatching HLA types, or as assessment of a degree of similarly of repertoires between a recipient and a donor after bone marrow transplantation. Such analysis using a similarity index was difficult with a conventional system. Various similarity indices can be calculated with ESTIMATES (Colwell, R. K. et al. Journal of Plant Ecology 5: 3-21.) or R package (vegan) by using the following mathematical equations. The Morisita-Horn index, Kimoto's Cn index, and Pianka's a index are found by the mathematical equations shown below.

Morisita-Horn index, Xi: number of times read i appear in all X reads from one of the samples, y1: number of times read i appear in all Y reads from the other sample, S: number of unique reads.

\(\begin{matrix}
{C_{MH} = \frac{2{\sum_{i = 1}^{S}{x_{i}y_{i}}}}{\left( {\frac{\sum_{i = 1}^{S}x_{i}^{2}}{X^{2}} + \frac{\sum_{i = 1}^{S}y_{i}^{2}}{Y^{2}}} \right){XY}}} & \left\lbrack {{Numeral}\mspace{14mu} 6} \right\rbrack
\end{matrix}\)

Kimoto's Cπ index

\(\begin{matrix}
{C_{\pi} = \frac{2{\sum_{i = 1}^{S}{x_{i}y_{i}}}}{\left( {{\sum_{i = 1}^{S}p_{xi}^{2}} + {\sum_{i = 1}^{S}p_{yi}^{2}}} \right){XY}}} & \left\lbrack {{Numeral}\mspace{14mu} 7} \right\rbrack \\
{{p_{xi} = \frac{x_{i}}{X}},{p_{yi} = \frac{y_{i}}{Y}}} & \left\lbrack {{Numeral}\mspace{14mu} 7\text{-}1} \right\rbrack
\end{matrix}\)

Pianka's α index

\(\begin{matrix}
{\alpha = \frac{\sum_{i = 1}^{S}{p_{xi}p_{yi}}}{\sqrt{\sum_{i = 1}^{S}{p_{xi}^{2}{\sum_{i = 1}^{S}p_{yi}^{2}}}}}} & \left\lbrack {{Numeral}\mspace{14mu} 8} \right\rbrack
\end{matrix}\)

The present invention can use next generation sequencing techniques to prepare a sample for quantitative analysis of a repertoire of a variable region of a T receptor (TCR) or a B cell receptor (BCR). Such sequencing techniques can obtain a million or more reads from a sample at a reasonable cost. Even a genotype that exists at a low frequency of 1/1,000,000 or less can be detected by using these techniques in a specific and unbiased manner. An unbiased amplification method for amplifying all different types of sequences of a specific portion of a gene or a transcript from a sample derived from a DNA of blood, bone marrow or the like is achieved.

**<Cancer Idiotype Peptide Sensitization Immune Cell Therapeutic Method>**

In one aspect, the present invention provides a method of preparing a composition for use in a cancer idiotype peptide sensitization immune cell therapeutic method to a subject. The method comprises (1) analyzing a T cell receptor (TCR) or B cell receptor (BCR) repertoire of the subject by the repertoire analysis method of the present invention or the repertoire analysis system of the present invention; (2) determining a TCR or BCR derived from a cancer cell of the subject based on a result of the analysis, wherein the determining is done by selecting a high ranking sequence in a frequency of presence ranking of a TCR or BCR gene derived from the cancer cell of the subject as the TCR or BCR derived from the cancer cell; (3) determining an amino acid sequence of a candidate HLA test peptide based on the determined TCR or BCR derived from cancer, wherein the determining is performed based on a score calculated by using an HLA binding peptide prediction algorithm; and (4) synthesizing the determined peptide. In this regard, a synthesized peptide can be used in a cancer idiotype peptide sensitization immune cell therapeutic method. In some cases, this method is called a “cancer idiotype peptide sensitization immune cell therapeutic method” herein.

A cancer idiotype peptide sensitization immune cell therapeutic method can be implemented in clinical practice by using the following specific procedures. In short, for example, (1) a peripheral blood cell of a cancer patient suffering from hematological tumor can be collected and lymphocyte cells can be separated to subsequently implement the repertoire analysis method of the present invention, and a cancer idiotype peptide sensitization immune cell therapeutic method can be performed with the use thereof.

In another embodiment, the repertoire analysis method of the present invention can be implemented for a TCR in case of T cell based tumor or for a BCR in case of B cell based tumor. Subsequently, a high ranking sequence in a frequency of presence ranking of a TCR or BCR gene is selected as the TCR or BCR derived from the cancer cell. A peptide that binds to a human leukocyte antigen (HLA) of the cancer patient determined separately from a sequence comprising a CDR3 region of the TCR or BCR gene is predicted by using an HLA binding peptide prediction program (any known program can be used as further explained herein). In addition, an HLA binding peptide is synthesized by a peptide synthesizer and the following is subsequently performed. For a tailor-made peptide sensitization CTL therapeutic method, it is possible to collect peripheral blood mononuclear cells from a patient and culture a mixture of the mononuclear cells or antigen presenting cells from the patient and a CD8+ T cell added with the peptide to apply a stimulation with an antigen peptide.

For a tailor-made peptide sensitization CTL therapeutic method, a CTL therapeutic method can be administered by introducing the peptide stimulated lymphocyte cell into the patient.

Alternatively, another method of a tailor-made peptide sensitization DC vaccine therapeutic method can be materialized by collecting a peripheral blood mononuclear cell of a patient, separating a mononuclear cell, inducing differentiation into a dendritic cell (DC) in the presence of a differentiation inducing factor, adding the peptide and culturing the mixture, and introducing the peptide sensitization dendritic cell into the patient to administer dendritic cell therapy.

A cancer idiotype peptide sensitization immune cell therapeutic method can be used in patients with hematologic cancer such as acute myeloid leukemia and related precursor cell neoplasm, lymphoblastic leukemia/lymphoma, T lymphoblastic leukemia/lymphoma, chronic lymphocytic leukemia/small lymphocytic lymphoma, B-cell prolymphocytic leukemia, hairy cell leukemia, T-cell prolymphocytic leukemia, T-cell large granular lymphocyte leukemia, and adult T cell leukemia/lymphoma, diseases similar to leukemia such as multiple myeloma and myelodysplastic syndrome, autoimmune diseases such as rheumatoid arthritis, systemic lupus erythematosus, and type I diabetes, and various infections, as well as for patients with terminal cancer, refractory autoimmune disease or severe infection. In particular, it is problematic for an antibody therapeutic method targeting a tumor cell or the like when a target antigen is not expressed on a tumor cell or a target antigen is also expressed on a normal cell. In comparison thereto, a therapeutic method with higher specificity and fewer side effects is expected because a sequence specific to a tumor cell is selected and utilized.

In one embodiment, the candidate HLA test peptide of the step (3) in the present invention is determined by using BIMAS, SYFPEITHI, RANKPEP or NetMHC.

In another embodiment, the present invention comprises, after the step (4) in the present invention, the step of: mixing the peptide, an antigen presenting cell or a dendritic cell derived from the subject, and a CD8+ T cell derived from the subject and culturing the mixture. This is also called an improved CTL method.

For example, unlike the existing broad T cell activation by an anti-CD3 antibody or IL-2, antigen specificity is imparted to a CD8+ T cell utilizing an antigen peptide such that therapy with higher specificity and fewer side effect can be expected in the improved CTL method. Further, the method is characterized in that a higher therapeutic effect can be expected because an individualized peptide created based on the information obtained from a tumor cell of the patient is used.

An improved CTL method can be used in, for example, patients with hematologic cancer such as acute myeloid leukemia and related precursor cell neoplasms, lymphoblastic leukemia/lymphoma, T lymphoblastic leukemia/lymphoma, chronic lymphocytic leukemia/small lymphocytic lymphoma, B-cell prolymphocytic leukemia, hairy cell leukemia, T-cell prolymphocytic leukemia, T-cell large granular lymphocyte leukemia, and adult T cell leukemia/lymphoma, diseases similar to leukemia such as multiple myeloma and myelodysplastic syndrome, autoimmune diseases such as rheumatoid arthritis, systemic lupus erythematosus, and type I diabetes, and various infections, as well as for terminal cancer patients and patients with a refractory autoimmune disease or severe infection.

In another embodiment, the present invention comprises, after the step (4) of the present invention, the step of: mixing the peptide with a dendritic cell derived from the subject and culturing the mixture. This is also called a DC vaccine therapy.

For example, since an individualized peptide is created based on the sequence information obtained from a tumor cell derived from the patient in DC vaccine therapy, such therapy does not act on a normal cell but act more specifically to a tumor cell such that a high therapeutic effect can be expected. Since a peptide is used as an antigen, unlike proteins, there is an advantage in being able to readily perform chemical synthesis.

DC vaccine therapy can be used in, for example, hematologic cancer such as acute myeloid leukemia and related precursor hematologic neoplasms, lymphoblastic leukemia/lymphoma, T lymphoblastic leukemia/lymphoma, chronic lymphocytic leukemia/small lymphocytic lymphoma, B-cell prolymphocytic leukemia, hairy cell leukemia, T-cell prolymphocytic leukemia, T-cell large granular lymphocyte leukemia, and adult T cell leukemia/lymphoma, diseases similar to leukemia such as multiple myeloma and myelodysplastic syndrome, autoimmune diseases such as rheumatoid arthritis, systemic lupus erythematosus, and type I diabetes, and patients with various infections, as well as for patients with terminal cancer, a refractory autoimmune disease or severe infection.

In another embodiment, the present invention comprises, after the step (4) of the present invention, the steps of: mixing the peptide, the antigen presenting cell or the dendritic cell derived from the subject and a CD8+ T cell derived from the subject and culturing the mixture to produce a CD8+ T cell-dendritic cell/antigen presenting cell-peptide mixture; and mixing the peptide with the dendritic cell derived from the subject and culturing the mixture to produce a dendritic cell-peptide mixture. This is also called a patient autoimmune cell therapeutic method.

For example, CD8+ T cell is stimulated and activated with a peptide derived from the patient as in a CTL therapeutic method and peptide sensitization of a dendritic cell is performed in a patient autoimmune cell therapeutic method. Such a therapeutic method is characterized in that a synergistic effect of a sustained effect due to the dendritic cell utilized as the antigen presenting cell and an acute effect due to CTL imparting specificity can be expected by introducing both the dendritic cell and the CD8+ cell derived from the patient into the patient.

A patient autoimmune cell therapeutic method can be used in, for example, patients with hematologic cancer (leukemia etc) such as acute myeloid leukemia and related precursor cell neoplasms, lymphoblastic leukemia/lymphoma, T lymphoblastic leukemia/lymphoma, chronic lymphocytic leukemia/small lymphocytic lymphoma, B-cell prolymphocytic leukemia, hairy cell leukemia, T-cell prolymphocytic leukemia, T-cell large granular lymphocyte leukemia, and adult T cell leukemia/lymphoma, diseases similar to leukemia such as multiple myeloma and myelodysplastic syndrome, autoimmune diseases such as rheumatoid arthritis, systemic lupus erythematosus, and type I diabetes, and various infections, as well as for patients with terminal cancer, a refractory autoimmune disease or severe infection.

In another aspect, the present invention provides a method of applying a cancer idiotype peptide sensitization immune cell therapeutic method to a subject. The method comprises (1) analyzing a T cell receptor (TCR) or B cell receptor (BCR) repertoire of the subject by the repertoire analysis method of the present invention or the repertoire analysis system of the present invention; (2) determining a TCR or BCR derived from a cancer cell of the subject based on a result of the analysis, wherein the determining is done by selecting a high ranking sequence in a frequency of presence ranking of a TCR or BCR gene derived from the cancer cell of the subject as the TCR or BCR derived from the cancer cell; (3) determining an amino acid sequence of a candidate HLA test peptide based on the determined TCR or BCR derived from cancer, wherein the determining is performed based on a score calculated by using an HLA binding peptide prediction algorithm; (4) synthesizing the determined peptide; and optionally (5) administering therapy by using the synthesized peptide. The method encompasses both a method of manufacturing a therapeutic agent and a method of therapy itself. When excluding a medical act, the method can be completed before step (5).

In a preferred embodiment, the candidate HLA test peptide of the step (3) is determined by using BIMAS, SYFPEITHI, RANKPEP or NetMHC in the present invention.

BIMAS is a program for estimating HLA peptide bond provided at www dot-bimas dot cit dot nih dot gov/.

SYFPEITHI is a search engine and a database for MHC ligands and peptide motifs provided at www dotsyfpeithi dot de/.

RANKPEP is a program for predicting a peptide bond to class I and class II MHC molecules, provided at http colon//imed dot med dot ucm dot es/Tools/rankpep dot html.

NetMHC is a program server for predicting binding of a peptide to numerous HLA alleles, provided at www dot cbs dot dtu dot dk/services/NetMHC/.

In a preferred embodiment, the present invention comprises, after the step (4), the steps of: mixing the peptide, an antigen presenting cell or a dendritic cell derived from the subject, and a CD8+ T cell derived from the subject and culturing the mixture; and administering the mixture after culturing to a patient as an improved CTL method.

In a preferred embodiment, the present invention comprises, after the step (4), the steps of: mixing the peptide with the dendritic cell derived from the subject and culturing the mixture; and administering the cultured mixture to a patient as a DC vaccination therapeutic method.

In a preferred embodiment, the present invention comprises, after the step (4), the steps of: mixing the peptide, the antigen presenting cell or the dendritic cell derived from the subject and a CD8+ T cell derived from the subject and culturing the mixture to produce a CD8+ T cell-dendritic cell/antigen presenting cell-peptide mixture; mixing the peptide with the dendritic cell derived from the subject and culturing the mixture to produce a dendritic cell-peptide mixture; and administering the CD8+ T cell-dendritic cell/antigen presenting cell-peptide mixture and the dendritic cell-peptide mixture to a patient as a patient autoimmune cell therapeutic method.

<Isolation of Tailor-Made Cancer Specific T Cell Receptor Gene, Isolation of Cancer Specific TCR Gene by In Vitro Antigen Stimulation>

In another aspect, the present invention provides a technique for isolation of a tailor-made cancer specific T cell receptor gene or isolation of a cancer specific TCR gene by in vitro antigen stimulation. Thus, the present invention provides a method of preparing an isolated cancer specific TCR gene by an in vitro antigen stimulation, comprising: (A) mixing an antigen peptide or antigen protein derived from a subject or the peptide determined in the “Cancer idiotype peptide sensitization immune cell therapeutic method” of the present invention or a lymphocyte derived from the subject, an inactivated cancer cell derived from the subject, and a T lymphocyte derived from the subject and culturing the mixture to produce a tumor specific T cell; (B) analyzing a TCR of the tumor specific T cell by the repertoire analysis method of the present invention or the repertoire analysis system of the present invention; and (C) isolating a desired tumor specific T cell based on a result of the analyzing. Such preparation of an isolated cancer specific TCR gene by an in vitro antigen stimulation can be implemented by using any well-known technology in the art once the gene information is obtained. Such an isolated tailor-made cancer specific T cell receptor gene and a cancer specific TCR gene can be used to treat or prevent a variety of cancers.

Such an isolated tailor-made cancer specific T cell receptor gene and cancer specific TCR gene can be implemented in clinical practice by using the following specific procedures.

In one embodiment, therapy using an isolated tailor-made cancer specific T cell receptor gene and cancer specific TCR gene can be materialized, for example, as follows: (1) tumor cells are extracted from a cancer patient; (2) after crushing the tumor cells from the patient, cells are separated into single cells and inactivated by radiation irradiation or chemical treatment with mitomycin C or the like; (3) peripheral blood cells are separated from whole blood of the cancer patient; (4) an RNA is extracted from cells, with some of the peripheral blood cells as an untreated control sample; (5) the inactivated tumor cells and the peripheral blood cells are mixed and cultured to activate and proliferate the tumor specific T cells; (6) after activation, an RNA is extracted from the cells by collecting the peripheral blood cells as a sample after stimulation; (7) the repertoire analysis method of the present invention is implemented on the RNA samples extracted in (4) and (6); (8) TCR genes that have greatly increased with a stimulation sample relative to a control sample are extracted and ranked, and then high ranking TCRα and TCRβ genes are selected; (9) the full-length TCRα and TCRβ genes are cloned and introduced into a retroviral vector for gene expression; (10) a gene introducing virus is created from the TCRα and TCRβ gene expression retroviral vector; (11) lymphocytes collected from the patient are infected independently and successively with TCRα and TCRβ for transfection, or a gene expression retroviral vector comprising both TCRα and TCRβ genes is created to transform both genes at once; (12) expression of TCRα/TCRβ heterodimers on a cell surface is confirmed; and (13) a tumor specific patient lymphocyte expressing TCRα/TCRβ of interest is introduced into the cells of the patient.

Specifically, the TCR or BCR determined by the method described in the “Cancer idiotype peptide sensitization immune cell therapeutic method” can be used as an antigen or peptide, for example, for hematological tumor in the embodiments of the present invention. In this regard, any cancer antigen or inactivated cancer tissue from a patient is presumed, where the following can be utilized as a typical method: a method of mixing any antigen protein or any antigen peptide, T lymphocyte, and antigen presenting cell; a method of mixing a lymphocyte from a subject and an inactivated cancer cell from the subject; and a method of mixing an antigen-resenting cell, T lymphocyte, and peptide derived from a TCR or BCR determined by the repertoire analysis provided in “Cancer idiotype peptide sensitization immune cell therapeutic method”.

Thus, in one embodiment, the step (A) in the present invention is a step of mixing the inactivated cancer cell derived from the subject and the antigen peptide or antigen protein derived from the subject with the T lymphocyte derived from the subject and culturing the mixture to produce a tumor specific T cell.

In a further embodiment, the step (A) in the present invention is a step of mixing the lymphocyte derived from the subject, the inactivated cancer cell derived from the subject, and the T lymphocyte derived from the subject and culturing the mixture to produce a tumor specific T cell.

In a further embodiment, the step (A) in the present invention is a step of mixing the peptide determined in “Cancer idiotype peptide sensitization immune cell therapeutic method”, the inactivated cancer cell derived from the subject, and the T lymphocyte derived from the subject and culturing the mixture to produce a tumor specific T cell.

Such therapy of an isolated tailor-made cancer specific T cell receptor gene and a cancer specific TCR gene can be used in patients with a wide range of cancer, including, but not limited to, adrenocortical carcinoma, anal cancer, bile duct cancer, bladder cancer, breast cancer, cervical cancer, chronic lymphocytic leukemia, chronic myelogenous leukemia, colorectal cancer, endometrial cancer, esophageal cancer, Ewing tumor, gallbladder cancer, Hodgkin's disease, hypopharyngeal cancer, laryngeal cancer, lip and oral cavity cancer, liver cancer, non-small-cell lung cancer, non-Hodgkin's lymphoma, melanoma, mesothelioma, multiple myeloma, ovarian cancer, pancreatic cancer, prostate cancer, gastric cancer, testicular cancer, thyroid cancer and the like.

In a further aspect, the present invention provides isolation of a tailor-made cancer specific T cell receptor gene, and isolation of a cancer specific TCR gene by searching for a common sequence. Thus, the present invention provides a method of preparing an isolated cancer specific TCR gene by searching for a common sequence, comprising: (A) providing a lymphocyte or cancer tissue isolated from subjects having a common HLA; (B) analyzing a TCR of the tumor specific T cell by the repertoire analysis method of the present invention or the repertoire analysis system of the present invention for the lymphocyte or cancer tissue; and (C) isolating a T cell having a sequence in common with the tumor specific T cell. Once genetic information is obtained, preparation of an isolated cancer specific TCR gene by searching for a common sequence can be performed by using any well-known technology in the art. A gene obtained by such isolation of tailor-made cancer specific T cell receptor gene or isolation of a cancer specific TCR gene by searching for a common sequence can be used in therapy and prevention of a variety of cancers. The method is also called “method of isolation of tailor-made cancer specific T cell receptor gene or isolation of cancer specific TCR gene by searching for a common sequence of the present invention”.

A gene obtained by such isolation of a tailor-made cancer specific T cell receptor gene or isolation of a cancer specific TCR gene by searching for a common sequence can be implemented in clinical practice by using the following specific procedures. In one embodiment, it is possible to materialize therapy using a gene obtained by isolation of a tailor-made cancer specific T cell receptor gene or isolation of a cancer specific TCR gene by searching for a common sequence from the following: first (1) tumor cells are extracted or peripheral blood is separated from cancer patients with the same HLA; (2) repertoire analysis is performed by using a lymphocyte cell or tumor tissue comprising a tumor cell infiltrated T cell; (3) a ranking is produced for each sample based on a frequency of presence thereof, and a tumor specific T cell exhibiting a higher frequency of presence in a tumor cell relative to a peripheral blood cell is selected; (4) a common sequence in a plurality of HLA matching cancer patients is searched for the tumor specific T cell; (5) a tumor specific TCR gene shared by the most cancer patients is selected as a tumor specific TCR for therapy; (6) the full length TCRα and TCRβ genes are cloned and introduced into a retroviral vector for gene expression; (7) a gene introducing virus is created from the TCRα and TCRβ gene expression retroviral vector; (8) lymphocytes collected from the patient are infected independently and successively with TCRα and TCRβ for transfection, or a gene expression retroviral vector comprising both TCRα and TCRβ genes is created to transform both genes at once; (9) expression of TCRα/TCRβ heterodimers on a cell surface is confirmed; and (10) a tumor specific patient lymphocyte expressing TCRα/TCRβ of interest is introduced into the cells of the patient.

Therapy using a gene obtained by such isolation of a tailor-made cancer specific T cell receptor gene or isolation of a cancer specific TCR gene by searching for a common sequence can be used in patients with a wide range of cancer, for example including, but not limited to, adrenocortical carcinoma, anal cancer, bile duct cancer, bladder cancer, breast cancer, cervical cancer, chronic lymphocytic leukemia, chronic myelogenous leukemia, colorectal cancer, endometrial cancer, esophageal cancer, Ewing tumor, gallbladder cancer, Hodgkin's disease, hypopharyngeal cancer, laryngeal cancer, lip and oral cavity cancer, liver cancer, non-small-cell lung cancer, non-Hodgkin's lymphoma, melanoma, mesothelioma, multiple myeloma, ovarian cancer, pancreatic cancer, prostate cancer, gastric cancer, testicular cancer, thyroid cancer and the like.

Thus, in another aspect, the present invention provides a method of isolating a cancer specific TCR gene by an in vitro antigen stimulation, comprising: (A) mixing an antigen peptide or antigen protein derived from a subject or the peptide determined in a cancer idiotype peptide sensitization immune cell therapeutic method or a lymphocyte derived from the subject, an inactivated cancer cell derived from the subject, and a T lymphocyte derived from the subject and culturing the mixture to produce a tumor specific T cell; (B) analyzing a TCR of the tumor specific T cell by the repertoire analysis method of the present invention or the repertoire analysis system of the present invention; and (C) isolating a desired tumor specific T cell based on a result of the analyzing. Once genetic information is obtained, preparation of a cancer specific TCR gene isolated by such an in vitro antigen stimulation can be performed by using any well-known technology in the art. Such an isolated tailor-made cancer specific T cell receptor gene or cancer specific TCR gene can be used in therapy and prevention of a variety of cancers.

Thus, in one embodiment of the method of isolating a cancer specific TCR gene by an in vitro antigen stimulation, the step (A) in the present invention comprises a step of mixing the inactivated cancer cell derived from the subject and the antigen peptide or antigen protein derived from the subject with the T lymphocyte derived from the subject and culturing the mixture to produce a tumor specific T cell.

In a further embodiment, the step (A) in the present invention is a step of mixing the lymphocyte derived from the subject, the inactivated cancer cell derived from the subject, and the T lymphocyte derived from the subject and culturing the mixture to produce a tumor specific T cell.

In a further embodiment, the step (A) in the present invention is a step of mixing the peptide determined in “Cancer idiotype peptide sensitization immune cell therapeutic method”, the inactivated cancer cell derived from the subject, and the T lymphocyte derived from the subject and culturing the mixture to produce a tumor specific T cell.

In still another aspect, the present invention provides a technique of isolating a cancer specific TCR gene by searching for a common sequence or isolating a tailor-made cancer specific T cell receptor gene, comprising: (A) isolating a lymphocyte or cancer tissue from subjects having a common HLA; (B) analyzing a TCR of the tumor specific T cell by the repertoire analysis method of the present invention for the lymphocyte or cancer tissue; and (C) isolating a T cell having a sequence in common with the tumor specific T cell. Such an isolated tailor-made cancer specific T cell receptor gene or cancer specific TCR gene can be used in therapy and prevention of a variety of cancers.

<Cell Processing Therapeutic Method>

In a further aspect, the present invention provides a cell processing therapeutic method. Specifically, the present invention provides a method of preparing a T lymphocyte introduced with a tumor specific TCR gene for use in cell processing therapeutic method, comprising: A) providing a T lymphocyte collected from a patient; B) analyzing TCRs based on the repertoire analysis method of the present invention or the repertoire analysis system of the present invention after applying an antigen stimulation to the T lymphocyte, wherein the antigen stimulation is applied by an antigen peptide or antigen protein derived from the subject, an inactivated cancer cell derived from the subject, or an idiotype peptide derived from tumor; C) selecting an optimal TCR and an optimal antigen in the analyzed TCRs; and D) producing a tumor specific α and β TCR expression viral vector of a TCR gene of the optimal TCR. The cell processing therapeutic method using the T lymphocyte introduced with a tumor specific TCR gene can be used for the therapy and prevention of a variety of cancers.

Such a cell processing therapeutic method using a T lymphocyte introduced with a tumor specific TCR gene can be implemented in clinical practice by using the following specific procedures. For example, a lymphocyte introduced with a tumor specific TCR gene can be used by the method described in <Isolation of tailor-made cancer specific T cell receptor gene, isolation of cancer specific TCR gene by in vitro antigen stimulation> or <Isolation of tailor-made cancer specific T cell receptor gene, isolation of cancer specific TCR gene by searching for a common sequence>.

Thus, any cancer antigen or cancer peptide can be manufactured or produced by synthesis as an antigen to utilize a collected inactivated patient cancer cell or to utilize an idiotype peptide derived from tumor in the cell processing therapeutic method of the present invention. As a selection method, it is possible to select an antigen highly expressed in cancer tissue or select a peptide that binds to the HLA type of a patient as an antigen.

In a preferred embodiment of the cell processing therapeutic method of the present invention, examples of conceivable optimal antigen that can be selected include, but are not limited to, (1) an antigen highly expressed in the patient's cancer tissue, (2) an antigen that most strongly activates a T cell in an antigen specific lymphocyte stimulation test, and (3) an antigen that increases the frequency of a specific TCR the most from repertoire analysis before and after an antigen stimulation. Further, it is also possible to conceive a method of selecting, as an optimal TCR, a TCR that has increased the most in an example (3), where the frequency of a specific TCR increased the most from repertoire analysis before and after an antigen stimulation. Further, it is possible to select, as the optimal TCR, a candidate optimal TCR which is artificially transgenically introduced into a lymphocyte of a patient and exhibits the highest reactivity in actual cancer tissue of the patient as a typical example.

Such a cell processing therapeutic method using a T lymphocyte introduced with a tumor specific TCR gene can be used in patients with a wide range of cancer including, but not limited to, for example adrenocortical carcinoma, anal cancer, bile duct cancer, bladder cancer, breast cancer, cervical cancer, chronic lymphocytic leukemia, chronic myelogenous leukemia, colorectal cancer, endometrial cancer, esophageal cancer, Ewing tumor, gallbladder cancer, Hodgkin's disease, hypopharyngeal cancer, laryngeal cancer, lip and oral cavity cancer, liver cancer, non-small-cell lung cancer, non-Hodgkin's lymphoma, melanoma, mesothelioma, multiple myeloma, ovarian cancer, pancreatic cancer, prostate cancer, gastric cancer, testicular cancer, thyroid cancer and the like.

Thus, in one embodiment, the antigen stimulation of the method of the present invention is applied with the antigen peptide or antigen protein derived from the subject.

In another embodiment, the antigen stimulation of the method of the present invention is applied with the inactivated cancer cell derived from the subject.

In another embodiment, the antigen stimulation of the method of the present invention is applied with the idiotype peptide derived from tumor.

In another embodiment, the step C) of the present invention comprises selecting an antigen that is highly expressed in cancer tissue of the subject.

In another embodiment, the step C) of the present invention comprises selecting an antigen which most strongly activates a T cell in an antigen specific lymphocyte stimulation test.

In another embodiment, the step C) of the present invention comprises selecting an antigen that increases a frequency of a specific TCR the most from repertoire analysis conducted based on the repertoire analysis method of the present invention or the repertoire analysis system of the present invention before and after applying the antigen stimulation.

In one specific embodiment, the present invention provides a method of assessing efficacy and/or safety by a stimulation test in vitro by using a cancer specific TCR gene isolated by <Isolation of tailor-made cancer specific T cell receptor gene, isolation of cancer specific TCR gene by searching for a common sequence the present invention>.

Efficacy can be assessed, for example, by culturing an antigen protein or antigen peptide derived from a subject who received an antigen stimulation with an antigen protein or antigen peptide derived from the subject and a T cell introduced with a cancer specific TCR gene, an inactivated cancer cell derived from the subject who has received an antigen stimulation with an inactivated cancer cell derived from the subject, and an idiotype peptide derived from tumor which has received an antigen stimulation with an idiotype peptide derived from tumor, and then measuring the amount of cytokines (interferon γ or the like) secreted out of a cell in response to T cell activation, measuring the amount of expression of a specific gene that is elevated in response to T cell activation, or measuring a cell surface molecule that is expressed or increases expression in response to T cell activation.

<Safety> When a T cell derived from the subject introduced with a cancer specific TCR gene is mixed with a normal cell derived from the subject, safety can be assessed, for example, by measuring secreted cytokines, gene expression, or expression of a cell surface molecule in response to activation of the above-described T and verifying that the TCR gene introduced T cell is not activated by a normal cell.

In one embodiment, the specific steps of efficacy and/or safety assessment can be materialized as follows. For example: (1) a retroviral gene expression system is used to create a tumor specific TCRα and TCRβ gene introduced T lymphocyte cell; (2) when assessing efficacy, a cancer cell derived from a patient is extracted, separated, and immortalized, and then subjected to mixing and culturing with a T lymphocyte introduced with a tumor specific TCR gene; (3) reactivity to a tumor cell can be quantitatively assessed to select a TCR gene reacting more strongly to a tumor cell by using the above-described culture cell and performing a cell proliferation test (thymidine uptake test, MTT test, IL-2 production test or the like); (4) when assessing safety, a control, which is an existing cell line, normal tissue free of patient's cancer cells (part of the normal tissue collected in the process of extracting tumor), or patient's peripheral blood cells in the case where solid tumor is used and immortalized, and then subjected to mixing and culturing with a T lymphocyte introduced with a tumor specific TCR gene; and (5) reactivity to a tumor cell can be quantitatively assessed to select a TCR gene that exhibits no reactivity to a normal cell by using the above-described culture cell and performing a cell proliferation test (thymidine uptake test, MTT test, IL-2 production test or the like).

Thus, in another aspect, the present invention provides a cell processing therapeutic method, comprising: A) collecting a T lymphocyte from a patient; B) analyzing TCRs based on the repertoire analysis method or the repertoire analysis system of the present invention after applying antigen stimulation to the T lymphocyte, wherein the antigen stimulation is applied by an antigen peptide or antigen protein derived from the subject, an inactivated cancer cell derived from the subject, or an idiotype peptide derived from tumor; C) selecting an optimal TCR and an optimal antigen in the analyzed TCRs; D) producing a tumor specific α and β TCR expression viral vector of a TCR gene of the optimal TCR; and E) introducing the T lymphocyte introduced with a tumor specific TCR gene into the patient.

A method of implementing the steps of introducing a resulting T lymphocyte introduced with a tumor specific TCR gene into the patient comprises the following: A) manufacturing a T lymphocyte introduced with the tumor specific TCR gene; B) confirming expression of tumor specific TCRα and TCRβ; and C) intravenously introducing the T lymphocyte introduced with a tumor specific TCR gene by intravenous drip.

Thus, in one embodiment, the antigen stimulation in the cell processing therapeutic method of the present invention is applied with the antigen peptide or antigen protein derived from the subject.

In another embodiment, the antigen stimulation in the cell processing therapeutic method of the present invention is applied with the inactivated cancer cell derived from the subject.

In another embodiment, the antigen stimulation in the cell processing therapeutic method of the present invention is applied with the idiotype peptide derived from tumor.

In another embodiment, the step C) in the cell processing therapeutic method of the present invention comprises selecting an antigen that is highly expressed in cancer tissue of the subject.

In another embodiment, the step C) in the cell processing therapeutic method of the present invention comprises selecting an antigen which most strongly activates a T cell in an antigen specific lymphocyte stimulation test.

In another embodiment, the step C) in the cell processing therapeutic method of the present invention comprises selecting an antigen that increases a frequency of a specific TCR the most from repertoire analysis conducted based on the repertoire analysis method of the present invention before and after applying the antigen stimulation.

**<Isolation of Human Form Antibody Utilizing BCR Repertoire Analysis>**

As one embodiment, the repertoire analysis method of the present invention can be used to perform BCR gene repertoire analysis to quickly obtain a human form antibody specific to a target antigen by the methods described below. (A) a method of immunizing a mouse with a target antigen protein or antigen peptide and separating a cell population (e.g., spleen, lymph node, or peripheral blood cells) comprising an antibody producing B cell from the mouse to analyze immunoglobulin heavy chain and light chain genes by the repertoire analysis method of the present invention

(A1) the method of A, wherein the immunized mouse is a KM mouse capable of producing a complete human antibody while maintaining antibody diversity

(A2) the method of A, wherein the immunized mouse is a humanized mouse created by transplanting a human stem cell into an NOG (NOD/Shi-scid, IL-2Rγnull) mouse exhibiting severe combined immunodeficiency made by mating an IL-2 receptor γ chain knockout mouse with a NOD/scid mouse

(B) comparing immunoglobulin heavy chain and light chain genetic sequences obtained from samples derived from a control mouse and an immunized mouse or mice before and after antigen immunization and frequencies thereof

(C) identifying immunoglobulin heavy chain and light chain genes that are strongly expressed or increase after immunization in the immunized mouse

(D) a method of selecting immunoglobulin heavy chain and light chain genes selected from step C and inserting the genes to match one type of antibody expression vector or inserting the genes separately into two types of antibody expression vectors

(E) introducing the immunoglobulin heavy chain and light chain gene expression vector made in step D into a eukaryotic cell such as CHO (Chinese Hamster Ovary) and culturing the cell

(F) separating/purifying an antibody molecule produced or secreted by a genetically modified cell to inspect specificity to a target antibody protein or peptide.

The above-described steps A-F are methods of directly and quickly obtaining an antigen specific human form antibody without altering an antibody gene derived from an animal after obtainment thereof into a chimeric antibody or humanized antibody of a human antibody. The methods can be used in the development and manufacture of an antibody medicine consisting of a human form antibody.

For KM mice used in this embodiment, the following can be referred: Ishida I, Tomizuka K, Yoshida H, Tahara T, Takahashi N, Ohguma A, Tanaka S, Umehashi M, Maeda H, Nozaki C, Halk E, Lonberg N. Production of human monoclonal and polyclonal antibodies in TransChromo animals. Cloning Stem Cells. 2002; 4(1): 91-102. Review. For NOG mice, the following can be referred: Ito M, Hiramatsu H, Kobayashi K, Suzue K, Kawahata M, Hioki K, Ueyama Y, Koyanagi Y, Sugamura K, Tsuji K, Heike T, Nakahata T. NOD/SCID/gamma(c) (null) mouse: an excellent recipient mouse model for engraftment of human cells. Blood. 2002 Nov. 1; 100(9): 3175-82. For CHO cells/antibody production, the following can be referred: Jayapal K P, Wlaschin K F, Hu W-S, Yap MGS. Recombinant protein therapeutics from CHO cells-20 years and counting. Chem Eng Prog. 2007; 103: 40?47.; Chusainow J, Yang Y S, Yeo J H, Toh P C, Asvadi P, Wong N S, Yap M G. A study of monoclonal antibody-producing CHO cell lines: what makes a stable high producer? Biotechnol Bioeng. 2009 Mar. 1; 102(4): 1182-96.

**<Isolation of Human Form Antibody Utilizing BCR Repertoire Analysis>**

As one embodiment, the BCR gene repertoire analysis method can be utilized to quickly obtain a human form antibody specific to a target antigen by the methods described below.

(A) a method of immunizing a mouse with a target antigen protein or an antigen peptide and separating a cell population (e.g., spleen, lymph node, or peripheral blood cells) comprising an antibody producing B cell from the mouse to analyze immunoglobulin heavy chain and light chain genes by a BCR repertoire analysis method

(A1) the method of A, wherein the immunized mouse is a KM mouse capable of producing a complete human antibody while maintaining antibody diversity

(A2) the method of A, wherein the immunized mouse is a humanized mouse created by transplanting a human stem cell into an NOG (NOD/Shi-scid, IL-2Rynull) mouse exhibiting severe combined immunodeficiency made by mating an IL-2 receptor γ chain knockout mouse with an NOD/scid mouse

(B) comparing immunoglobulin heavy chain and light chain genetic sequences obtained from samples derived from a control mouse and an immunized mouse or mice before and after antigen immunization and frequencies thereof

(C) identifying immunoglobulin heavy chain and light chain genes that are strongly expressed or increase after immunization in the immunized mouse

(D) a method of selecting immunoglobulin heavy chain and light chain genes selected from step C and inserting the genes to match one type of antibody expression vector or inserting the genes separately into two types of antibody expression vectors

(E) introducing the immunoglobulin heavy chain and light chain gene expression vector made in step D into a eukaryotic cell such as CHO (Chinese Hamster Ovary) and culturing the cell

(F) separating/purifying an antibody molecule produced or secreted by a genetically modified cell to inspect specificity to a target antibody protein or peptide.

The above-described steps A-F are methods of directly and quickly obtaining an antigen specific human form antibody without altering an antibody gene derived from an animal after obtainment thereof into a chimeric antibody or humanized antibody of a human antibody. The methods can be used in the development and manufacture of an antibody medicine consisting of a human form antibody.

Embodiments of such methods include the following. As one example thereof,

1. A KM mouse is immunized with a Myelin Oligodendrocyte Glycoprotein (MOG35-55, MOG), which is an antigen peptide of experimental autoimmune encephalomyelitis. The same quantity of 2 mg/mL MOG peptide and complete Freund's adjuvant are mixed to create an emulsion. The mouse is subcutaneously immunized with 200 μg of MOG and simultaneously immunized in the peritoneal cavity with 400 ng of pertussis toxin. A control mouse is immunized with PBS and complete Freund's adjuvant.

2. On day 2 after the first immunization, the mouse is immunized with 400 ng of pertussis toxin. After confirming an outbreak on day 10 after the immunization, the spleen is extracted from the mouse with an episode of encephalomyelitis.

3. The spleens of the outbreak mouse and control mouse are used to carry out next generation BCR repertoire analysis. Frequencies of appearance of individual BCR sequences are counted and ranked for immunoglobulin heavy chain and immunoglobulin light chains.

4. BCR sequences with a large increase in the frequency of appearance in the outbreak mouse relative to the control mouse are extracted and ranked. A combination of high ranking BCR sequences induced by the antibody administration is identified as a MOG specific antibody gene.

5. A full length human immunoglobulin sequence is cloned by PCR-cloning from a BCR gene amplicon amplified from the outbreak mouse. Each of the IgG immunoglobulin heavy chain and the immunoglobulin light chain is cloned in an antibody expression vector. There is a method of inserting the genes to match one type of antibody expression vector or inserting the genes separately into two types of antibody expression vectors.

6. A CHO (Chinese Hamster Ovary) cell is transformed by using Lipofectamine 3000 (Life Science) and IgG immunoglobulin heavy chain and immunoglobulin light chain are introduced with the constructed expression vector.

7. A CHO cell culture solution is collected. Secreted antibody proteins are collected by purification with a protein A affinity column and concentration with gel filtration.

8. binding activity to M0G35-55 or MOG protein is measured by an ELISA assay using the collected antibody to investigate the specificity of the antibody.

9. When sufficient specificity is obtained, a cell line stably expressing an antibody is acquired and a human form anti-MOG antibody is manufactured with a large-scale culturing system.

(Peptide and Therapy of the Present Invention)

The peptide of the present invention or a nucleic acid encoding the same can be used in immunotherapy. Description thereof is provided below.

A peptide provided by the present invention is derived from an antigen associated with tumorigenesis and can have the ability to bind sufficiently to an MHC (HLA) class II molecule to trigger an immune response in a human, especially a lymphocyte, especially a T lymphocyte, especially a CD4 positive T lymphocyte, and especially a TH1 type immune response induced by a CD4 positive T lymphocyte.

As used herein, “protein”, “polypeptide”, “oligopeptide” and “peptide” are used to have the same meaning and refer to an amino acid polymer of any length. Such a polymer may be a branched or straight chain or annular. An amino may be a natural or non-natural or altered amino acid. The term may also encompass those assembled into a complex of a plurality of polypeptide chains. The term also encompasses natural or artificially altered amino acid polymers. Examples of such an alteration include disulfide bond formation, glycosylation, lipidation, acetylation, phosphorylation or any other manipulation or alteration (e.g., conjugation with a labeling component). The definition also encompasses, for example, a polypeptide comprising one or more analogs of an amino acid (e.g., including a non-natural amino acid, etc.), peptide-like compound (e.g., peptoid), and other known alterations in the art. As used herein, “amino acid” may be natural or non-natural as long as the objective of the present invention is met.

As used herein, “polynucleotide”, “oligonucleotide” and “nucleic acid” are used in the same meaning and refer to a nucleotide polymer with any length. The term also encompasses “oligonucleotide derivative” and “polynucleotide derivative”. “Oligonucleotide derivative” or “polynucleotide derivative” refers to an oligonucleotide or a polynucleotide, which has a bond between nucleotides that is not normal or includes a derivative of a nucleotide. They are interchangeably used. Specific examples of such an oligonucleotide include 2′-O-methyl-ribonucleotide, oligonucleotide derivative with a phosphodiester bond in an oligonucleotide converted to a phosphorothioate bond, oligonucleotide derivative with a phosphodiester bond in an oligonucleotide converted to an N3′-P5′ phosphoroamidate bond, oligonucleotide derivative with a ribose and phosphodiester bond in an oligonucleotide converted to a peptide nucleic acid bond, oligonucleotide derivative with uracil in an oligonucleotide substituted with C-5 propynyl uracil, oligonucleotide derivative with uracil in an oligonucleotide substituted with C-5 thiazole uracil, oligonucleotide derivative with cytosine in an oligonucleotide substituted with C-5 propynyl cytosine, oligonucleotide derivative with cytosine in an oligonucleotide substituted with phenoxazine-modified cytosine, oligonucleotide derivative with ribose in a DNA substituted with 2′-O-propylribose, oligonucleotide derivative with ribose in an oligonucleotide substituted with 2′-methoxyethoxyribose and the like. Unless specifically noted otherwise, a specific nucleic acid sequence is further intended to encompass conservatively altered variants (e.g., degenerate codon substituted form) and complementary sequences thereof in addition to the explicitly shown sequences. Specifically, a degenerate codon substituted form can be obtained by creating a sequence in which the third position of one or more selected (or all) codons is substituted with a mixed base and/or deoxyinosine residue (Batzer et al., Nucleic Acid Res. 19: 5081 (1991); Ohtsuka et al., J. Biol. Chem. 260: 2605-2608 (1985); Rossolini et al., Mol. Cell. Probes 8: 91-(1994)). As used herein, “nucleic acid” is interchangeably used with gene, cDNA, mRNA, oligonucleotide, and polynucleotide. As used herein, “nucleotide” may be natural or non-natural.

As used herein, “gene” refers to an agent defining a genotype. A “gene” may refer to a “polynucleotide”, “oligonucleotide” or “nucleic acid”.

In addition to the identified peptides, a variant thereof may also be used in the present invention. Examples of such a variant include, but are not limited to, those that are homologous to the identified peptide.

As used herein, “homology” of genes refers to the level of identity of 2 or more genetic sequences to one another. In general, having “homology” refers to having a high level of identity or similarity. Thus, a higher level of homology of two genes results in a higher level of identity or similarity of sequences thereof. It is possible to examine whether two types of genes are homologous by direct comparison of sequences or by hybridization under stringent conditions for nucleic acids. When directly comparing two genetic sequences, the genes are homologous typically when DNA sequences between the genetic sequences are at least 50% identical, preferably at least 70% identical, and more preferably at least 80%, 90%, 95%, 96%, 97%, 98% or 99% identical. Thus, as used herein, “homolog” or “homologous gene product” refers to a protein in another species, preferably a mammal, which exerts the same biological function as a protein constituent element of a complex further described herein. Such a homolog is also called “ortholog gene product”. It is understood that such a homolog, homolog gene product, ortholog gene product and the like can also be used as long as they align with the objective of the present invention.

An amino acid may be mentioned herein by a commonly known three letter symbol thereof or a one letter symbol recommended by IUPAC-IUB Biochemical Nomenclature Commission. Similarly, a nucleotide may be mentioned by a commonly recognized one letter code. Herein, comparison of similarity, identity and homology of amino acid sequences and base sequences is calculated by using a default parameter with a sequence analysis tool BLAST. For instance, identity can be searched by using NCBI's BLAST 2.2.28 (published on 2013 Apr. 2). The value of identity herein generally refers to a value from using the above-described BLAST to align sequences under default conditions. However, when a higher value is output by changing a parameter, the highest value is considered the value of identity. When identity is assessed in a plurality of regions, the highest value thereamong is considered the value of identity. Similarity is a numerical value that uses a similar amino acid into the calculation in addition to identity.

In one embodiment of the present invention, “several” may be, for example, 10, 8, 6, 5, 4, 3 or 2, or a value less than any one of the value. It is known that a polypeptide with a deletion, addition, insertion or other amino acid substitutions of one to several amino acid residues maintains its biological activity (Mark et al., Proc Natl Acad Sci USA. 1984 September; 81(18): 5662-5666., Zoller et al., Nucleic Acids Res. 1982 Oct. 25; 10(20): 6487-6500, Wang et al., Science. 1984 Jun. 29; 224 (4656): 1431-1433.). An antibody with a deletion or the like can be made, for example, by site-directed mutagenesis, random mutagenesis, biopanning using an antibody phage library or the like. For example, KOD-Plus-Mutagenesis Kit (TOYOBO CO., LTD.) can be used for site-directed mutagenesis. An antibody with the same activity as the wild-type can be selected from mutant antibodies introduced with a deletion or the like by performing various characterizations such as FACS analysis or ELISA.

In one embodiment of the present invention, “90% or greater” may be, for example, 90, 95, 96, 97, 98, 99 or 100% or greater or within the range of any of the two values. For the above-described “homology”, the percentage of the number of homologous amino acids in two or a plurality of amino acid sequences may be calculated in accordance with a known method in the art. Before calculating the percentage, amino acid sequences in a group of amino acid sequences to be compared are aligned. A space is introduced in a portion of amino acid sequences when necessary to maximize the percentage of the same amino acids. An alignment method, method of calculating the percentage, comparison method, and computer programs associated therewith have been well known in the art (e.g., BLAST, GENETYX and the like). As used herein, “homology” can represent a value measured by BLAST of NCBI unless specifically noted otherwise. Blastp can be used in the default setting for an algorithm for comparing amino acid sequences with BLAST. Results of measurement are expressed in a numerical form as Positives or Identities.

As used herein, “polynucleotide that hybridizes under stringent conditions” refers to a conventional, well-known condition in the art. Such a polynucleotide can be obtained by using colony hybridization, plaque hybridization, southern blot hybridization or the like while using a polynucleotide selected from the polynucleotides of the present invention as a probe. Specifically, such a polynucleotide refers to a polynucleotide which can be identified by using a filter with immobilized DNA derived from a colony or a plaque for hybridization at 65° C. in the presence of 0.7-1.0 M NaCl, and then using a 0.1 to 2-fold concentration SSC (saline-sodium citrate) solution (composition of an SSC solution with 1-fold concentration is 150 mM sodium chloride and 15 mM sodium citrate) to wash the filter under the condition of 65° C. For “stringent condition”, the following are examples of conditions that can be used. (1) low ionic strength and a high temperature are used for washing (e.g., 0.015 M sodium chloride/0.0015 M sodium citrate/0.1% sodium dodecyl sulfate at 50° C.), (2) a denaturing agent such as formamide is used in hybridization (e.g., 50% (v/v) formamide, 0.1% bovine serum albumin/0.1% ficoll/0.1% polyvinyl pyrrolidone/50 mM sodium phosphate buffer with a pH of 6.5, 750 mM sodium chloride, and 75 mM sodium citrate at 42° C.), or (3) a solution comprising 20% formamide, 5×SSC, 50 mM sodium phosphate (pH 7.6), 5×Denhardt's solution, 10% dextran sulfate, and 20 mg/ml denatured sheared salmon sperm DNA, is incubated overnight at 37° C. and then a filter is washed with 1×SSC at about 37-50° C. The formamide concentration may be 50% or greater. Washing time may be 5, 15, 30, 60, 120 minutes, or greater. A plurality of elements are considered to affect stringency in a hybridization reaction such as temperature, salt concentration and the like. Ausubel et al., Current Protocols in Molecular Biology, Wiley Interscience Publishers, (1995) can be referred for details. “Highly stringent condition”, for example, is 0.0015 M sodium chloride, 0.0015 M sodium citrate, and 65-68° C. or 0.015 M sodium chloride, 0.0015 M sodium citrate, 50% formamide and 42° C. Hybridization is performed in accordance with the method described in experimental publications such as Molecular Cloning 2nd ed., Current Protocols in Molecular Biology, Supplement 1-38, DNA Cloning 1: Core Techniques, A Practical Approach, Second Edition, Oxford University Press (1995). In this regard, a sequence comprising only an A sequence or only a T sequence is preferably excluded from a sequence that hybridizes under stringent conditions. A moderately stringent condition can be readily determined by those skilled in the art based on, for example, the length of a DNA and is shown in Sambrook et al., Molecular Cloning: A Laboratory Manual, Third Ed., Vol.′, 7.42-7.45 Cold Spring Harbor Laboratory Press, 2001, including, for a nitrocellulose filters, hybridization conditions of a pre-wash solution of 1.0 mM EDTA (pH 8.0), 5×SSC, 0.5% SDS, and about 50% formamide and 2×SSC—6×SSC at about 40-50° C. (or other similar hybridization solutions such as a Stark's solution in about 50% formamide at about 42° C.) and washing conditions of 0.5×SSC, 0.1% SDS at about 60° C. Thus, the polypeptides used in the present invention encompass polypeptides encoded by a nucleic acid molecule that hybridizes under highly or moderately stringent conditions to a nucleic acid molecule encoding a polypeptide particularly described in the present invention.

As used herein, “purified” substance or biological agent (e.g., nucleic acid, protein or the like) refers to a substance or biological agent having at least some of the agents that are naturally accompanied therewith removed. Thus, purity of a biological agent in a purified biological agent is higher than that of a normal condition of the biological agent (i.e., concentrated). The term “purified” as used herein preferably refers to the presence of at least 75 wt. %, more preferably at least 85 wt. %, still more preferably at least 95 wt. %, and the most preferably at least 98 wt. % of biological agents of the same type. A substance or biological agent used in the present invention is preferably a “purified” substance. An “isolated” substance or biological agent (e.g., nucleic acid, protein, or the like) as used herein refers to a substance or biological agent having agents that are naturally accompanied therewith substantially removed. The term “isolated” as used herein varies depending on the objective. Thus, the term does not necessarily need to be represented by purity. However, when necessarily, the term refers to preferably the presence of at least 75 wt. %, more preferably at least 85 wt. %, still more preferably at least wt. % and the most preferably at least 98 wt. % of biological agents of the same type. A substance used in the present invention is preferably an “isolated” substance or biological agent.

As used herein, “fragment” refers to a polypeptide or polynucleotide with a sequence length of 1 to n−1 with respect to the full length polypeptide or polynucleotide (with length n). The length of a fragment can be appropriately changed in accordance with the objective. Examples of the lower limit of such a length include 3, 4, 5, 6, 7, 8, 9, 10, 15, 20, 25, 30, 40, 50 and more amino acids for a polypeptide. Lengths represented by an integer that is not specifically listed herein (e.g., 11 and the like) also can be suitable as a lower limit. Further, examples of length include 5, 6, 7, 8, 9, 10, 15, 20, 25, 30, 40, 50, 75, 100, and more nucleotides for a polynucleotide. Lengths represented by an integer that is not specifically listed herein (e.g., 11 and the like) also can be suitable as a lower limit. As used herein, such a fragment is understood to be within the scope of the present invention, for example, when a full length version functions as a marker or a target molecule, as along as the fragment itself also functions as a marker or a target molecule.

As used herein, “functional equivalent” refers to any entity having the same function of interest but a different structure relative to the original target entity. A functional equivalent can be found by searching a database or the like. As used herein, “search” refers to utilizing a certain nucleic acid base sequence electronically, biologically, or by another method to find another nucleic acid base sequence having a specific function and/or property. Examples of electronic search include, but are not limited to, BLAST (Altschul et al., J. Mol. Biol. 215:403-410 (1990)), FASTA (Pearson & Lipman, Proc. Natl. Acad. Sci., USA 85: 2444-2448 (1988)), Smith and Waterman method (Smith and Waterman, J. Mol. Biol. 147: 195-197 (1981)), Needleman and Wunsch method (Needleman and Wunsch, J. Mol. Biol. 48: 443-453 (1970)) and the like. Examples of biological search include, but are not limited to, stringent hybridization, a macroarray with a genomic DNA applied to a nylon membrane or the like or a microarray with a genomic DNA applied to a glass plate (microarray assay), PCR, in situ hybridization and the like. Herein, a gene used in the present invention is intended to include corresponding genes identified by such electronic search or biological search.

As a functional equivalent of the present invention, it is possible to use an amino acid sequence with one or more amino acid insertions, substitutions or deletions, or addition to one or both ends. As used herein, “one or more amino acid insertions, substitutions or deletions, or addition to one or both ends” in an amino acid sequence refers to an alteration with a substitution of a plurality of amino acids or the like to the extent that can occur naturally by a well-known technical method such as site-directed mutagenesis or natural mutation. An altered amino acid sequence of a molecule can have, for example, 1-30, preferably 1-20, more preferably 1-9, still more preferably 1-5, and especially preferably 1-2 amino acid insertion, substitution or deletion or addition to one or both ends. An altered amino acid sequence may be an amino acid sequence having one or more (preferably 1 or several, or 1, 2, 3 or 4) conservative substitutions in an amino acid sequence such as CCL21, CXCR3, or CCR7. “Conservative substitution” refers herein to a substitution of one or more amino acid residues with another chemically similar amino acid residue so as not to substantially alter a function of a protein. Examples thereof include cases where a hydrophobic residue is substituted with another hydrophobic residue, cases where a polar residue is substituted with another polar residue having the same charge and the like. Functionally similar amino acids that can be substituted in this manner are known in the art for each amino acid. Specific examples include alanine, valine, isoleucine, leucine, proline, tryptophan, phenylalanine, methionine and the like for nonpolar (hydrophobic) amino acids, glycine, serine, threonine, tyrosine, glutamine, asparagine, cysteine and the like for polar (neutral) amino acids. Examples of positively charged (basic) amino acid include arginine, histidine, lysine and the like. Further, examples of a negatively-charged (acidic) amino acid include aspartic acid, glutamic acid and the like.

As used herein, “subject” refers to a target subjected to diagnosis, detection or the like of the present invention.

As used herein, “agent” is used broadly and may be any substance or other elements (e.g., energy, radiation, heat, electricity and other forms of energy) as long as the intended objective can be achieved. Examples of such a substance include, but are not limited to, protein, polypeptide, oligopeptide, peptide, polynucleotide, oligonucleotide, nucleotide, nucleic acid (including for example DNAs such as cDNA and genomic DNA, RNAs such as mRNA), polysaccharide, oligosaccharide, lipid, organic small molecule (e.g., hormone, ligand, information transmitting substance, organic small molecule, molecule synthesized by combinatorial chemistry, small molecule that can be used as medicine (e.g., small molecule ligand and the like) and a composite molecule thereof). Typical examples of an agent specific to a polynucleotide include, but are not limited to, a polynucleotide having complementarity with a certain sequence homology (e.g., 70% or greater sequence identity) to a sequence of the polynucleotide, polypeptide such as a transcription factor that binds to a promoter region and the like. Typical examples of an agent specific to a polypeptide include, but are not limited to, an antibody directed specifically to the polypeptide or a derivative or analog thereof (e.g., single strand antibody), a specific ligand or receptor when the polypeptide is a receptor or ligand, a substrate when the polypeptide is an enzyme and the like.

As used herein, “therapy” refers to the prevention of amelioration, preferably maintaining the current condition, more preferably alleviation, and still more preferably disappearance of a disease or disorder (e.g., cerebral malaria) in case of such a condition, including being able to exert a prophylactic effect or an effect of improving one or more symptoms accompanying the disease. Preliminary diagnosis with suitable therapy may be referred to as “companion therapy” and a diagnostic agent therefor as “companion diagnostic agent”.

As used herein, “therapeutic agent” broadly refers to all agents capable of treating a condition of interest (e.g., diseases such as cerebral malaria or the like) and refers to an inhibiter (e.g., antibody) such as those provided by the present invention. In one embodiment of the present invention, “therapeutic agent” may be a pharmaceutical composition comprising an effective ingredient and one or more pharmacologically acceptable carriers. A pharmaceutical composition can be manufactured, for example, by mixing an effective ingredient and the above-described carriers by any method known in the technical field of pharmaceuticals. Further, usage form of a therapeutic agent is not limited as long as it is used for therapy. A therapeutic agent may be an effective ingredient alone or a mixture of an effective ingredient and any ingredient. Further, the shape of the above-described carriers is not particularly limited. For example, the carrier may be a solid or liquid (e.g., buffer solution).

As used herein, “prevention” refers to the action of taking a measure against a disease or disorder (e.g., cerebral malaria) from being in such a condition prior to being in such a condition. For example, it is possible to use the agent of the present invention to perform diagnosis, and optionally use the agent of the present invention to prevent or take measures to prevent cerebral malaria or the like

As used herein, “prophylactic agent” broadly refers to all agents capable of preventing a condition of interest (e.g., diseases such as cerebral malaria or the like).

The present invention provides a peptide derived from an antigen associated with tumorigenesis and have the ability to bind sufficiently to an MHC (HLA) class I molecule to trigger an immune response of a human leukocyte, especially a lymphocyte, especially a T lymphocyte, and especially a CD8 positive cytotoxic T lymphocyte, as well as combinations of two peptides especially useful for a vaccination of a cancer patient.

The peptide of the present invention may be derived from a tumor associated antigen, especially, for example, tumor associated antigens with a function in proteolysis, angiogenesis, cell growth, cell cycle regulation, cell division, transcriptional regulation, tissue infiltration or the like.

A peptide can be chemically synthesized and is usable as an effective pharmaceutical ingredient in manufacture of medicine. Thus, the peptide provided by the present invention can be used in immunotherapy or preferably cancer immunotherapy.

The pharmaceutical composition of the present invention further comprises an additional peptide and/or an excipient to increase the effect. This is further explained below.

The pharmaceutical composition of the present invention can comprise a peptide identified in the present invention, the peptide having a full length of 8-100 amino acids, preferably 8-30 amino acids, and most preferably 8-16 amino acids.

In addition, the peptide or variant can be further modified to improve the stability and/or binding to an MHC molecule in order to induce a more potent immune response. Methods of optimizing such a peptide sequence are well known to those skilled in the art, including, for example, introduction of a non-peptide bond or reversed peptide bond. Thus, another embodiment of the present invention provides a pharmaceutical composition wherein at least one peptide or a variant thereof comprises a non-peptide bond.

An amino acid residue in a reversed peptide bond is not bound by a peptide (—CO—NH—) where the peptide bond is reversed. Such a retro-inverso peptidomimetic can be made by using a method well known to those skilled in the art. Examples thereof include the method described in Meziere et al (1997) J. Immunol. 159, 3230-3237 incorporated herein as a reference. This approach involves creating a pseudo peptide comprising a change involving the backbone but not involving orientation of a side chain. Meziere et al (1997) show that such pseudo peptides are useful in an MHC and T helper cell response. A retro-inverso peptide comprising an NH—CO bond instead of a CO—NH peptide bond has much stronger resistance to proteolysis.

A non peptide bond is, for example, —CH2—NH, —CH2S—, —CH2CH2—, —CH═CH—, —COCH2—, —CH(OH)CH2—, or —CH2SO—. U.S. Pat. No. 4,897,445 provides a method of solid phase synthesis of a non-peptide bond (—CH2—NH) in a polypeptide chain, which involves a non-peptide bond synthesized by reacting amino aldehyde and amino acid in the presence of NaCNBH3, and polypeptide synthesized by a standard procedure.

A peptide having a sequence of the present invention can be synthesized with an additional chemical group at the amino end and/or carboxy end thereof in order to strengthen, for example, the stability, bioavailability and/or affinity of the peptide. For example, it is possible to add a hydrophobic group such as a t-butyloxylcarbonyl group, dansyl group, or carbobenzoxy group at the amino end of the peptide. Similarly, an acetyl group or 9-fluorenylmethoxy-carbonyl group can be placed at the amino end of the peptide. In addition, it is possible to add, for example, the hydrophobic group, i.e., t-butyloxylcarbonyl group or amino group to the carboxy end of the peptide.

Furthermore, the peptide used in the present invention can be synthesized to change the steric configuration thereof. For example, a D-isomer of one or more amino acid residues of the peptide can be used instead of a common L-isomer. Furthermore, at least one amino acid residue of the peptide of the present invention can be substituted with a well-known non-natural amino acid residue. Such alteration can act to increase stability, bioavailability and/or binding action of the peptide of the present invention.

Similarly, the peptide or a variant of the present invention can be chemically modified by a reaction with a specific amino acid before or after synthesis of the peptide used in the present invention. Examples of such modification are well known in the art. Examples thereof are summarized in R. Lundblad, Chemical Reagents for Protein Modification, 3rd ed. CRC Press, 2005 incorporated herein by reference. Examples of chemical modification of an amino acid include, but are not limited to, modifications by acylation, amidination, pyridoxylation of lysine, reductive alkylation, trinitrobenzylation of an amino group by 2,4,6-trinitrobenzenesulfonic acid (TNBS), sulfhydryl modification by performic acid and amide modification of a carboxyl group, oxidation from cysteine to cystic acid, generation of mercury derivatives, generation of mixed disulfide with another thiol compound, reaction with maleimide, carboxymethylation with iodoacetic acid or iodoacetamide, and carbamoylation with cyanate at an alkaline pH. With regard to the above, those skilled in the art can reference a broader methodology related to chemical modification of a protein with Current Protocols In Protein Science, Eds. Coligan et al. (John Wiley& Sons NY 1995-2000). For example, modification of an arginine residue of a protein is often the formation of an additive based on a reaction of adjacent dicarbonyl compounds such as 1,2-cyclohexanedione, 2,3-butanedione, and phenylglyoxal. Another example is a reaction of an arginine residue with methylglyoxal. Cysteine can be modified without simultaneous modification of another nucleophilic site such as lysine or histidine. For this reason, numerous reagents can be utilized in cysteine modification. Information on specific reagents is provided by Pierce Chemical Company, Sigma-Aldrich, and other websites.

A disulfide bond in a protein used in the present invention is often selectively reduced. A disulfide bond can be formed and oxidized during heat treatment of a biomedicine. A specific glutamic acid residue can be modified by using Woodward's Reagent K. An intermolecular crosslink can be formed between a lysine residue and a glutamic acid residue by using N-(3-(dimethylamino)propyl)-N′-ethylcarbodiimide. For example, diethylpyrocarbonate is a reagent for modifying a histidine residue in a protein. Histidine can also be modified by using 4-hydroxy-2-nonenal. A reaction between a lysine residue and another a amino acid group is useful, for example, in a bond between a peptide and a surface or protein/peptide crosslink. Lysine is a site where poly(ethylene)glycol attaches and is a main site of modification in glycation of a protein. A methionine residue of a protein can be modified, for example, by iodoacetamide, bromoethylamine, or chloramine-T. Tetranitromethane and N-acetylimidazole can be used in modification of a tyrosyl residue. A crosslink by formation of dityrosine can be accomplished with hydrogen peroxide/copper ions. N-bromosuccinimide, 2-hydroxy-5-nitrobenzyl bromide and 3-bromo-3-methyl-2-(2-nitrophenyl mercapto)-3H-indole(BPNS-skatole) were used in a recent study related to tryptophan modification. Suitable modification of a therapeutic protein and peptide with PEG often involves prolonging the circulation half-life. In addition, protein crosslink by glutaraldehyde, polyethylene glycol diacrylate and formaldehyde is used for preparation of a hydrogel. Chemical modification of an allergen for immunotherapy is often accomplished by carbamylation with potassium cyanate.

In general, the peptides and variants used in the present invention (at least those comprising a peptide link between amino acid residues) can be synthesized, for example, by using a Fmoc-polyamide form of solid phase peptide synthesis, as disclosed in Lu et al (1981) J. Org. Chem. 46, 3433 and a reference thereof. Purification can be performed by a combination of one or more technologies such as recrystallization, size exclusion chromatography, ion exchange chromatography, hydrophobic interaction chromatography, and reversed phase high-performance liquid chromatography that (generally) uses, for example, acetonitrile/water gradient separation. A peptide can be analyzed by using thin-layer chromatography, electrophoresis, especially caterpillar electrophoresis, solid phase extraction (CSPE), reversed phase high performance liquid chromatography, amino acid analysis after acid hydrolysis, fast atom bombardment (FAB) mass spectrometry, MALDI and ESI-Q-TOF mass spectrometry.

In still another aspect of the present invention, a nucleic acid (e.g., polynucleotide) encoding the peptide of the present invention or a variant thereof is provided. For example, DNA, cDNA, PNA, CNA, RNA, single strand and/or double strand, or natural or stable form of a polynucleotide such as a polynucleotide having phosphorothioate backbone or a combination thereof can be such a polynucleotide. It is not essential to contain intron as long as a polynucleotide encodes the peptide. Naturally, only peptides comprising a naturally occurring amino acid residue bound by a naturally occurring peptide bond is encoded by a polynucleotide. In yet another embodiment of the present invention, an expression vector with an ability to express the polypeptide according to the present invention is provided. Expression vectors of different cell types are well known in the art and can be selected without any special experimentation.

In general, a DNA is inserted into an expression vector such as a plasmid in a correct orientation with a correction reading frame for expression. If necessary, a DNA can be linked to a suitable transcription/translation regulating/managing nucleotide sequence which is recognized by a desired host. However, such a management function is generally in an expression vector. The vector is then introduced into a host by a standard technology. In regards to this, Sambrook et al (1989) Molecular Cloning, A Laboratory Manual, Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y. can be referred.

A regimen with the optimal dose and optimal amount of the nucleic peptide contained in a vaccine can be determined by those skilled in the art without any special experimentation. For example, the peptide or a mutant form thereof can be prepared as an intravenous (i.v.) injection, subcutaneous (s.c.) injection, intradermal (i.d.) injection, intraperitoneal (i.p.) injection, or intramuscular (i.m.) injection. Preferred routes of administration for a peptide injection are s.c., i.d., i.p., i.m., and i.v. Preferred routes of administration for a DNA injection are i.d., i.m., s.c., i.p., and i.v. For example, 1-500 mg, 50 μg-1.5 mg, preferably 125 μg-500 μg of peptide or DNA can be administered. The dose is dependent on each peptide or DNA. The dose in this range has been successfully used in clinical trials (Brunsvig P F, Aamdal S, Gjertsen M K, Kvalheim G, Markowski-Grimsrud C J, Sve I, Dyrhaug M, Trachsel S, Muller M, Eriksen J A, Gaudernack G; Telomerase peptide vaccination: a phase I/II study in patients with non-small cell lung cancer; Cancer ImmunolImmunother. 2006; 55(12): 1553-1564; M. Staehler, A. Stenzl, P. Y. Dietrich, T. Eisen, A. Haferkamp, J. Beck, A. Mayer, S. Walter, H. Singh, J. Frisch, C. G. Stief; An open label study to evaluate the safety and immunogenicity of the peptide based cancer vaccine IMA901, ASCO meeting 2007; Abstract No 3017).

The selection, number, and/or amount of peptides in the pharmaceutical composition of the present invention can be made specific to tissue, cancer and/or patient in preparing the composition. For example, a side effect can be avoided by deriving a correction selection of peptide by the expression pattern of a protein of tissue of a given patient. The selection may be dependent on the cancer type and condition of the disease specific to a patient receiving therapy, therapeutic regimen up to this point, immune status of the patient and naturally, the HLA haplotype of the patient. Furthermore, the vaccine according to the present invention can comprise an individualized constituent element depending on individual needs of a specific patient. Examples thereof include expression of related TAA, personal side effects due to allergy or other therapy of the individual, and different amounts of peptide in accordance with adjustments for secondary therapy after a series of initial therapeutic plans for a specific patient.

A peptide in which a parent protein is highly expressed in normal tissue is avoided or is present at a low amount in the composition of the present invention. Meanwhile, when the tumor of a patient is known to highly express a specific protein, each pharmaceutical composition for the treatment of the cancer can be present at a high amount and/or comprise a plurality of peptides specific to the specific protein or route. Those skilled in the art can select a preferred combination of immunogenic peptides by testing T cell functionality, expansion, affinity and proliferation of a specific T cell against a specific peptide, overall presentation, in vitro T cell formation and efficacy thereof by, for example, analysis of IFN-γ (also see the following examples). Generally, the most efficient peptides are then combined as a vaccine for the aforementioned objective.

A suitable vaccine preferably contains 1-20 peptides, more preferably 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19 or 20 different peptides, still more preferably 6, 7, 8, 9, 10, 11, 12, 13 or 14 different peptides, and most preferably 14 different peptides. The length of the peptide used as a cancer vaccine may be any suitable peptides. Specifically, the length can be a suitable 9-mer peptide or a suitable 7-mer or 8-mer or 10-mer or 11-mer peptide or a 12-mer, 13-mer, 14-mer, or 15-mer peptide. A longer peptide is also suitable in some cases. As described in the appended Tables 1 and 2, a 9-mer or 10-mer peptide is preferred for an MHC class I peptide and a 12- to 15-mer is preferred for an MHC class II peptide.

The peptide of the present invention constitutes a tumor or cancer vaccine. The tumor or cancer vaccine can be administered directly to the organ with the disease or systemically into a patient, administered to the patient with a vaccine applied in vitro to a human cell line or a cell from the patient, or used in vitro to select a subpopulation from immune cells of a patient and re-administered into the patient.

The peptide can be substantially pure, combined with an immunostimulatory adjuvant (see below), used in combination with an immunostimulatory cytokine, or coadministered with a suitable delivery system (e.g., liposome). The peptide can also be conjugated with a suitable carrier such as keyhole limpet hemocyanin (KLH) or mannan (see WO 95/18145 and Longenecker et al (1993) Ann. NY Acad. Sci. 690, 276-291). The peptide can also be tagged, or formed into a fusion protein or a hybrid molecule. A peptide given the sequence in the present invention is expected to stimulate CD4 or CD8CTL. However, efficiency of stimulation is increased more by a positive T cell providing assistance to the opposite CD. Thus, for an MHC class II epitope stimulating CD4CTL, a section of a hybrid molecule or a fusion partner thereof suitably provides an epitope stimulating a CD8 positive T cell. Meanwhile, for an MHC class I epitope stimulating CD8CTL, a section of a hybrid molecule or a fusion partner thereof suitably provides an epitope stimulating a CD4 positive T cell. CD4 and CD8 stimulating epitopes are well known in the art, including those specified in the present invention.

To elicit an immune response, it is generally necessary to include an excipient for enhancing immunogenicity of the composition. Thus, a pharmaceutical composition of a preferred embodiment of the present invention further has at least one suitable adjuvant. Since an adjuvant used in the present invention is a substance that non-specifically enhances or promotes an immune response to an antigen (e.g., immune response mediated by CTL and helper T (TH) cell), the adjuvant is understood to be useful for the agent of the present invention. Suitable adjuvants are, but not limited to, 1018 ISS, aluminum salt, Amplivac, AS15, BCG, CP-870, 893, CpG7909, CyaA, dSLIM, GM-CSF, IC30, IC31, imiquimod, ImuFact IMP321, IS Patch, ISS, ISCOMATRIX, Juvlmmune, LipoVac, MF59, monophosphoryl lipid A, Montanide IMS 1312, Montanide ISA 206, Montanide ISA 50V, Montanide ISA-51, OK-432, OM-174, OM-197-MP-EC, ONTAK, PepTel(R) vector system, PLG microparticles, resiquimod, SRL172, virosome and other virus-like particles, YF-17D, VEGF trap, R848, β, glucan, Pam3Cys, Aquila'S QS21 Stimulon (Aquila Biotech, Worcester, MA, USA) (saponin derived substance), mycobacteria extract, and synthetic bacterial cell wall mimetics, and other exclusive adjuvants (Ribi's Detox, Quil, Superfos or the like). Adjuvants such as Freund's incomplete or GM-CSF are preferred. Several immunogenic adjuvants specific to dendritic cells (e.g., MF59) and formulations thereof have already been described (Dupuis M, Murphy T J, Higgins D, Ugozzoli M, van Nest G, Ott G, McDonald D M; Dendritic cells internalize vaccine adjuvant after intramuscular injection; Cell Immunol. 1998; 186(1): 18-27; Allison A C; The mode of action of immunological adjuvants; Dev Biol Stand. 1998; 92: 3-11). Further, cytokines can also be used. Some cytokines are directly linked to an effect on migration of dendritic cells to lymphocyte tissue (e.g., TNF-α) as accelerating the process of a dendritic cell maturing into an effective antigen-presenting cell against a T lymphocyte (e.g. GM-CSF, IL-1, IL-4) (U.S. Pat. No. 5,849,589 (the entirety thereof is incorporated by reference)), or as acting as an immunoadjuvant (e.g., IL-12) (Gabrilovich D I, Cunningham H T, Carbone D P; IL-12 and mutant P53 peptide-pulsed dendritic cells for the specific immunotherapy of cancer; J Immunother Emphasis Tumor Immunol. 1996 (6): 414-418).

It is reported that a CpG immunostimulatory oligonucleotide also enhances the effect of an adjuvant in a vaccine setting. Although not wishing to be bound by any theory, a CpG oligonucleotide has the action of activating an inherent (non-adaptive) immune system via a toll-like receptor (TLR) (mainly TLR9). TLR9 activity triggered by CpG enhances a humoral and cellular antigen specific response to various antigens including a peptide or protein antigen, live or killed virus, dendritic cell vaccine, autologous cell vaccine, and polysaccharide conjugate in both a prophylactic vaccine and therapeutic vaccine. More significantly, this increases maturation and differentiation of a dendritic cell and increases generation of cytotoxic T lymphocytes (CTL) and TH1 cell activity without assistance of a CD4 T cell. TH1 bias induced by a TLR9 stimulation is maintained, even in the presence of a vaccine adjuvant such as alum or Freund's incomplete adjuvant (IFA) that generally promotes TH2 bias. A CpG oligonucleotide is coformulated or coadministered with another adjuvant or is formed into a microparticle, nanoparticle, lipid emulsion or a similar formulation to exhibit a higher adjuvant activity. A CpG oligonucleotide is especially required for inducing a strong response in case of a relatively weak antigen. They also accelerate an immune response. In some experiments, an antibody response comparable with a full dose vaccine without CpG was obtained with an antigen dose that has been reduced by about double digits (Arthur M. Krieg, Therapeutic potential of Toll-like receptor 9 activation, NatureReviews, Drug Discovery, 2006, 5, 471-484). U.S. Pat. No. 6,406,705 B1 describes that an antigen-specific immune response is elicited by a combination of a CpG oligonucleotide, non-nucleic acid adjuvant, and an antigen. Other TLR bound molecules such as RNA bound TLR7, TLR8, and/or TLR9 can also be used.

Examples of other useful adjuvants in the present invention include, but are not limited to, chemically modified CpGs (e.g., CpR, Idera), poly (I:C) (e.g., polyI: C12U), non-CpG bacterial DNA or RNA, imidazoquinoline, cyclophosphamide, sunitinib, bevacizumab, Celebrex, NCX-4016, sildenafil, tadalafil, vardenafil, sorafenib, XL-999, CP-547632, pazopanib, ZD2171, AZD2171, ipilimumab, tremelimumab, and SC58175, which can act as a therapeutic agent and/or adjuvant. In the context of the present invention, the amount and concentration of a useful adjuvant and additive can be readily determined by those skilled in the art without any special experimentation. Preferred adjuvants are dSLIM, BCG, OK432, imiquimod, PeviTer, and JuvImmune. In a preferred embodiment of the pharmaceutical composition of the present invention, the adjuvant is selected from the group comprising colony stimulating factors such as granulocyte macrophage colony stimulating factors (GM-CSF, sargramostim). In a preferred embodiment of the pharmaceutical composition of the present invention, the adjuvant is imiquimod.

The composition of the present invention is used in parenteral administration such as subcutaneous, intradermal, or intramuscular administration or oral administration. For this reason, the peptide and other selective molecules are dissolved or suspended in a pharmaceutically acceptable, preferably water-soluble carrier. In addition, the composition can comprise an excipient such as buffer, binding agent, blasting agent, diluent, flavoring agent, or lubricant. Further, the peptide can be co-administered with an immunostimulatory substance such as a cytokine. An extensive list of excipients that can be used in such a composition can be obtained from, for example, A. Kibbe, Handbook of Pharmaceutical Excipients, 3. Ed. 2000, American Pharmaceutical Association and pharmaceutical press. The composition can be used in tumor or cancer, preferably in CRC prevention and/or therapeutic method.

A cytotoxic T cell (CTL) recognizes a peptide shaped antigen bound to an MHC molecule, but not an original exogenous antigen itself. The MHC molecule itself is on a cell surface of an antigen-presenting cell. Thus, CTL activation is only possible in the presence of APC, MHC molecule, and trimer complex of a peptide antigen. Thus, CTL is not activated by using only a peptide. An immune response is increased by further adding APC with each MHC molecule. Thus, in a preferred embodiment, the pharmaceutical composition of the present invention additionally comprises at least one antigen-presenting cell.

An antigen-presenting cell (or a stimulator cell) generally has an MHC class I or II molecule on the surface thereof. In one embodiment, the MHC class I or II molecule having the selected antigen substantially cannot be loaded onto itself. As discussed below in detail, the selected antigen thereof can be readily loaded onto the MHC class I or II molecule in vitro.

In general, the pharmaceutical composition of the present invention comprising the nucleic acid of the present invention can be administered by the same method as a pharmaceutical composition comprising the peptide of the present invention, i.e., by intravenous, intraarterial, peritoneal, intramuscular, intradermal, intratumoral, oral, transdermal, trans-nasal cavity, trans-oral cavity, transrectal, or transvaginal administration, inhalation or topical administration.

Tumor often acquires resistance to a therapeutic agent by the mechanism of avoidance. The drug resistance occurs during therapy and in some cases appears as metastatic or recurrent tumor. To avoid such drug resistance, tumor therapy is generally administered by a combination of drugs. In many cases, a different combination is required for metastasis and tumor recurrence after an event-free period. Thus, in one embodiment of the present invention, the pharmaceutical composition is administered with a second anticancer agent. A second agent used in the present invention can be administered before, after or simultaneously with the pharmaceutical composition of the present invention. For example, if chemical properties are compatible, simultaneous administration can be performed by mixing the pharmaceutical composition of the present invention with the second anticancer agent. Another method of simultaneous administration is, for example, to inject the pharmaceutical composition of the present invention and orally administer a second anticancer agent to administer the composition and anticancer agent on the same day from independent routes of administration. The pharmaceutical composition and the second anticancer agent may also be administered through separate therapeutic courses and/or administered through the same therapeutic course on different days.

In another aspect of the present invention, a method of treating or preventing cancer in a patient is provided. The method has a step of administering any one of the pharmaceutical compositions of the present invention at a therapeutically effective amount to the patient. A therapeutically effective amount is an amount sufficient to elicit an immune response, especially to activate a subpopulation of CTL. Those skilled in the art can readily determine an effective amount by using a standard immunological method as provided in the Examples of the present specification. Another method of monitoring the effect of a specific amount of the pharmaceutical composition of the present invention is to observe the growth and/or recurrence of treated tumor.

In an especially preferred embodiment of the present invention, the pharmaceutical composition is used as an anticancer vaccine.

The composition comprising a peptide or a nucleic acid encoding the peptide of the present invention can constitute a tumor or cancer vaccine. The tumor or cancer vaccine can be administered directly to the organ with the disease or systemically into a patient, administered to the patient with a vaccine applied in vitro to a human cell line or a cell from the patient, or used in vitro to select a subpopulation from immune cells of a patient and re-administered into the patient

The composition of the present invention can be used as a vaccine or as a method of treating cancer. The cancer is oral cavity or pharyngeal cancer, gastrointestinal cancer, colon, rectal, or anal cancer, airway cancer, breast cancer, uterine, vaginal, or vulvar cancer, endometrial or ovarian cancer, male genital tract cancer, urethral cancer, bone and soft tissue cancer, Kaposi's sarcoma, skin melanoma, eye melanoma, nonmelanoma eye cancer, brain or central nervous system cancer, thyroid and other endocrine gland cancer, Hodgkin's lymphoma, non-Hodgkin's lymphoma, or myeloma, and preferably renal cancer, colorectal carcinoma, lung cancer, breast cancer, pancreatic cancer, prostate cancer, gastric cancer, brain cancer, GIST or glioblastoma. According to the present invention, the preferred amount of peptide can vary between about 0.1-100 mg in 500 μl solution, preferably about 0.1-1 mg, and most preferably about 300 μg-800 μg. In this regard, the term “about” refers to +/−10 percentage of a given value unless specifically noted otherwise. Those skilled in the art would be able to adjust the actual amount of peptide to be used based on several factors such as the immune status of an individual patient and/or the amount of TUMAP presented in a specific type of cancer. The peptide of the present invention may be provided in a suitable shape (sterilized solution or the like) other than a freeze-thaw peptide.

The pharmaceutical composition of the present invention having the peptide and/or nucleic acid according to the present invention is administered to a patient with adenoma or cancerous disease associated with each corresponding peptide or antigen. An immune response mediated by a T cell is triggered thereby. Preferably, the amount of expression vector in the pharmaceutical composition of the present invention, in the peptide (especially peptide associated with tumor) or nucleic acid of the pharmaceutical composition of the present invention, or the composition of the present invention is specific to the tissue, cancer, and/or patient.

In another embodiment of the present invention, the vaccine of the present invention is a nucleic acid vaccine. It is well known that a T cell response is elicited by inoculation of a nucleic acid vaccine such as a DNA vaccine encoding a polypeptide. The tumor or cancer vaccine can be administered directly to the organ with the disease or systemically into the patient, administered to the patient with a vaccine applied in vitro to a human cell line or a cell from the patient, or used in vitro to select a subpopulation from immune cells of a patient and re-administered into the patient. When the nucleic acid is administered into a cell in vitro, it is useful in some cases to introduce cells such that an immunostimulatory cytokine such as interleukin-2 or GM-CSF is co-expressed. The nucleic acid can be substantially pure, combined with an immunostimulatory adjuvant, used in combination with an immunostimulatory cytokine, or coadministered with a suitable delivery system (e.g., liposome). The nucleic acid vaccine may be administered with an adjuvant described in relation to the above-described peptide vaccine. Preferably, the nucleic acid vaccine is administered without an adjuvant.

The polynucleotide of the present invention, in some cases, is substantially pure or comprised in a suitable vector or a delivery system. Suitable vectors and delivery systems included are viral, such as adenovirus, vaccinia virus, retrovirus, herpes virus, adeno-associated virus, or a plurality of virus element-containing hybrid based system. Non-viral delivery systems included are cationic lipids and cationic polymers well known in the technical field of DNA delivery. Physical delivery such as a “gene gun” can also be used. The peptide or a peptide encoded by the nucleic acid is in some cases a fusion protein, such as a fusion protein with an epitope from tetanus toxoid that simulates a CD4 position T cell.

Suitably, all peptides administered to the patient are sterilized and free of pyogenic substance. A naked DNA can be administered by an intramuscular, intradermal, or subcutaneous injection. Conveniently, the nucleic acid vaccine can have any nucleic acid delivery means. Preferably, the nucleic acid, which is a DNA, can be delivered in a liposome or as a part of a viral vector delivery system.

It is preferable that a nucleic acid vaccine such as a DNA vaccine is administered into a muscle. A peptide vaccine is preferably administered s.c. or i.d. It is also preferable to administer the vaccine intradermally.

Expression of an encoded polypeptide and uptake of a nucleic acid by a professional antigen-presenting cell such as a dendritic cell are possibly a mechanism of priming of an immune response. Although there is a possibility that a dendritic cell is not introduced, it is still important as an expression peptide can be taken in from a cell introduced into the tissue (“cross-priming”. Example: Thomas A M, Santarsiero L M, Lutz E R, Armstrong T D, Chen Y C, Huang L Q, Laheru D A, Goggins M, Hruban R H, Jaffee E M. Mesothelin-specific CD8(+) T cell responses provide evidence of in vivo cross-priming by antigen-presenting cells in vaccinated pancreatic cancer patients. J Exp Med. 2004 Aug. 2; 200(3): 297-306).

A cancer immune therapeutic method mediated by a polynucleotide is described in Conry et al (1996) Seminars in Oncology 23, 135-147; Condon et al (1996) Nature Medicine 2, 1122-1127; Gong et al (1997) Nature Medicine 3, 558-561; Zhai et al (1996) J. Immunol. 156, 700-710; Graham et al (1996) Int J. Cancer 65, 664-670; and Burchell et al (1996) 309-313 In: Breast Cancer, Advances in biology and therapeutics, Calvo et al (eds), John Libbey Eurotext, the entirety of all of which is incorporated herein by reference.

It is potentially useful in the present invention to administer the peptide or nucleic acid ex vivo and to have the vaccine of the present invention target a specific cell population such as antigen-presenting cells by selective purification of a specific cell population from a patient or use of a delivery system, targeting vector and injection site (for example, as described in Zhou et al (1995) Blood 86, 3295-3301; Roth et al (1996) Scand. J. Immunology 43, 646-651, dendritic cells can be sorted). For example, a targeting vector can have a tissue or tumor specific promoter for directing expression of an antigen at a suitable location.

The vaccine of the present invention may be dependent on the cancer type and condition of the disease specific to a patient receiving therapy, therapeutic regimen up to this point, immune status of the patient and naturally the HLA haplotype of the patient. Furthermore, the vaccine according to the present invention can comprise an individualized constituent element depending on individual needs of a specific patient. Examples thereof include expression of related TAA, personal side effect due to allergy or other therapy, and different amounts of peptide in accordance with adjustment for secondary therapy after a series of initial therapeutic plans for a specific patient.

The peptide of the present invention is not only useful in cancer therapy, but also in diagnosis. The peptide is generated from glioblastoma, and these peptides are identified to be absent from normal tissue. Thus, these peptides can be used to diagnose the presence of cancer.

A pathologist can use the presence of the peptide of the present invention in tissue biopsy to assist in the diagnosis of cancer. A pathologist can know whether the tissue is malignant, inflammatory or mostly affected by mass spectrometry or detection of a specific peptide of the present invention using an antibody, or another method well known in the art. The presence of a group of peptides of the present invention enables classification or sub-classification of affected tissue.

Detection of the peptide of the present invention in an affected tissue sample enables decision with respect to the benefits of a therapeutic method related to the immune system especially when it is known or anticipated that a T lymphocyte is associated with the action mechanism. Loss of MHC expression is a mechanism that is well understood, by which a malignant cell evades immune surveillance. Thus, the presence of the peptide of the present invention indicates that this mechanism is not utilized by the analyzed cell.

The peptide of the present invention can be used in the analysis of a lymphocyte response to the peptide of the present invention. For example, it is possible to analyze an antibody response or T cell response to the peptide of the present invention or the peptide of the present invention which is a complex with an MHC molecule. These lymphocyte responses can be used as a prognosis marker for determining a further therapeutic step. These responses can also be used as a surrogate marker in an immunotherapeutic approach attempting to elicit a lymphocyte response by different means such as a protein, nucleic acid, endogenous substance, or lymphocyte immune transfer vaccination. Under the setting of gene therapy, a lymphocyte response to the peptide of the present invention can be considered in assessing a side effect. Monitoring a lymphocyte response is possibly useful in a follow-up test after transplantation therapy, such as detection of graft-versus-host and host-versus-graft diseases.

The peptides of the preset invention can be used in generation and growth of antibodies specific to an MHC/peptide complex. They can be used in a therapeutic method to apply a toxin or a radioactive substance while targeting affected tissue. As another method of using such antibodies, the antibodies can be applied while targeting a radionuclide to affected tissue for use in an imaging method such as PET. This method of use can assist in detecting small metastasis or determining an accurate position and size of affected tissue. In addition, the peptides can be used in verification of diagnosis of cancer performed by a pathologist based on biopsy sample.

The present invention can be provided as a kit. As used herein, “kit” refers to a unit generally providing portions to be provided (e.g., testing agent, diagnostic agent, therapeutic agent, antibody, label, manual and the like) into two or more separate sections. This form of a kit is preferred when a composition that should not be provided in a mixed state and is preferably mixed immediately before use for safety or the like is intended to be provided. Such a kit advantageously comprises an instruction or manual describing how the provided portions (e.g., testing agent, diagnostic agent, or therapeutic agent) are used or how a reagent should be handled. When the kit is used herein as a reagent kit, the kit generally comprises an instruction describing how to use a testing agent, diagnostic agent, therapeutic agent, antibody and the like.

In this manner, in a further aspect of the present invention, the present invention is directed to a kit, wherein the kit has (a) a container comprising the pharmaceutical composition of the present invention in a solution form or a freeze-dried form, (b) selectively a second container comprising a diluent or a reconstitution solution for the freeze-dried formulation, and (c) selectively, a manual directed to (i) use of the solution or (ii) reconstitution and/or use of the freeze-dried formulation. The kit further has one or more of (iii) buffer, (iv) diluent, (v) filter, (vi) needle, or (v) syringe. The container is preferably a bottle, vial, syringe or a test tube or a multi-purpose container. The pharmaceutical composition is preferably freeze-dried.

The kit of the present invention preferably has a manual directed to the reconstitution and/or use of the freeze-dried formulation of the present invention in a suitable container. Examples of a suitable container include bottles, vials (e.g., dual chamber vials), syringe (dual chamber syringe and the like) and test tubes. The container can be formed from various materials such as glass or plastic. Preferably, the kit and/or container comprises a manual, on the container or accompanying the container, showing the method of reconstitution and/or use. For example, the label can explain that the freeze-dried formulation is reconstituted to the above-described peptide concentration. The label can further explain that the formulation is for subcutaneous injection or is useful for a subcutaneous injection.

The container of the formulation may be a multi-purpose vial that can be used in repeated administration (e.g., 2 to 6 administrations). The kit can further have a second container with a suitable diluent (e.g., sodium bicarbonate).

The final peptide concentration of a reconstituted formulation made by mixing the diluent and the freeze-dried formulation is preferably at least 0.15 mg/mL/peptide (=75 μg) and preferably 3 mg/mL/peptide (=1500 μg) or less. The kit can further comprise other materials that are desirable for commercial purpose or for users (including other buffer, diluent, filter, needle, syringe, and package insert).

The kit of the present invention can have a single container comprising a formulation of the pharmaceutical composition of the present invention with or without other constituent elements (e.g., other compounds or pharmaceutical composition of the other compounds). Alternatively, the kit can have separate containers for each constituent element.

Preferably, the kit of the present invention comprises a formulation of the present invention packaged for use as a combination with combined administration of a second compound (adjuvant (e.g., GM-CSF), chemotherapeutic agent, natural product, hormone or antagonist, anti-angiogenic agent or angiogenesis inhibitor, apoptosis inducer, chelating agent or the like) or a pharmaceutical composition thereof. The constituent element of the kit can be a complex made in advance or each constituent element can be placed in separate containers until administration to a patient. The constituent element of the kit can be provided as one or more liquid solutions, preferably as aqueous solutions, and more preferably sterilized aqueous solutions. The constituent elements of the kit can also be provided as a solid, which preferably can be converted into a liquid by adding a suitable solvent provided in another separate container thereto.

The container of a therapeutic kit can be a vial, test tube, flask, bottle, syringe or any other means for sealing a solid or a liquid. When there are a plurality of constituent elements, the kit generally comprises a second vial or another container such that the constituent elements can be administered separately. The kit can also comprise another container for a pharmaceutically acceptable liquid. Preferably, a therapeutic kit comprises an instrument (e.g., one or more needles, syringes, instillator, pipettes or the like) that enables administration of an agent of the present invention, which is the constituent element of the kit.

The pharmaceutical composition of the present invention is suitable for administering the peptide by any acceptable route such as oral (enteral), trans-nasal cavity, transocular, subcutaneous, intradermal, intramuscular, intravenous or transdermal route. The administration is preferably subcutaneous administration and most preferably intradermal administration. Administration can be performed with an infusion pump.

As used herein, “instruction” is a document with an explanation of the method of use of the present invention for a physician or other users. The instruction describes detection method of the present invention, method of use of a diagnostic agent, or sentences instructing administration of a medicament or the like. Further, an instruction may describe a sentence instructing oral administration or administration to the esophagus (e.g., by injection or the like) as a site of administration. The instruction is prepared in accordance with a format defined by an authority of the country in which the present invention is practiced (e.g., Health, Labor and Welfare Ministry in Japan or Food and Drug Administration (FDA) in the U.S. or the like), with an explicit description showing approval by the authority. The instruction is a so-called package insert and is typically provided in, but not limited to, paper media. The instructions may also be provided in a form such as electronic media (e.g., web sites provided on the Internet or emails).

(General Techniques)

Molecular biological technology, biochemical technology, and microbiological technology used herein are well known and conventional technologies in the art that are described in, for example, Sambrook J. et al. (1989). Molecular Cloning: A Laboratory Manual, Cold Spring Harbor and 3rd Ed. thereof (2001); Ausubel, F. M. (1987). Current Protocols in Molecular Biology, Greene Pub. Associates and Wiley-Interscience; Ausubel, F. M. (1989). Short Protocols in Molecular Biology: A Compendium of Methods from Current Protocols in Molecular Biology, Greene Pub. Associates and Wiley-Interscience; Innis, M. A. (1990). PCR Protocols: A Guide to Methods and Applications, Academic Press; Ausubel, F. M. (1992). Short Protocols in Molecular Biology: A Compendium of Methods from Current Protocols in Molecular Biology, Greene Pub. Associates; Ausubel, F. M. (1995). Short Protocols in Molecular Biology: A Compendium of Methods from Current Protocols in Molecular Biology, Greene Pub. Associates; Innis, M. A. et al. (1995). PCR Strategies, Academic Press; Ausubel, F. M. (1999). Short Protocols in Molecular Biology: A Compendium of Methods from Current Protocols in Molecular Biology, Wiley, and annual updates; Sninsky, J. J. et al. (1999). PCR Applications: Protocols for Functional Genomics, Academic Press, Gait, M. J. (1985). Oligonucleotide Synthesis: A Practical Approach, IRL Press; Gait, M. J. (1990). Oligonucleotide Synthesis: A Practical Approach, IRL Press; Eckstein, F. (1991). Oligonucleotides and Analogues: A Practical Approach, IRL Press; Adams, R. L. et al. (1992). The Biochemistry of the Nucleic Acids, Chapman & Hall; Shabarova, Z. et al. (1994). Advanced Organic Chemistry of Nucleic Acids, Weinheim; Blackburn, G. M. et al. (1996). Nucleic Acids in Chemistry and Biology, Oxford University Press; Hermanson, G. T. (1996). Bioconjugate Techniques, Academic Press, Bessatsu Jikken Igaku [Experimental Medicine, Supplemental Volume], Idenshi Donyu Oyobi Hatsugen Kaiseki Jikken Ho [Experimental Methods for Transgenesis & Expression Analysis], Yodosha, 1997, and the like, the relevant portions (which can be the entire document) of which are incorporated herein by reference.

Reference literatures such as scientific literatures, patents, and patent applications cited herein are incorporated herein by reference to the same extent that the entirety of each document is specifically described.

As described above, the present invention has been described while showing preferred embodiments to facilitate understanding. The present invention is described below based on Examples. The aforementioned description and the following Examples are not provided to limit the present invention, but for the sole purpose of exemplification. Thus, the scope of the present invention is not limited to the embodiments and Examples specifically described herein and is limited only by the scope of claims.

## EXAMPLES

(Examples of Preparation of Unbiasedly Amplified Sample)

(Preparation Example 1: Analysis of BCR repertoire in peripheral blood of healthy individuals) The present Example performed BCR repertoire analysis on peripheral blood of healthy individuals.

(Materials and Methods)

Sample: Peripheral blood mononuclear cells of healthy individuals

Method:

### (1. RNA Extraction)

5 mL of whole blood was collected from said healthy individuals in a heparin-containing blood collection tube. Peripheral blood mononuclear cells (PBMC) were separated by ficoll density gradient centrifugation. Total RNA was extracted/purified from the isolated 5×106 PBMCs by using RNeasy Lipid Tissue Mini Kit (QIAGEN, Germany). The resulting RNA was quantified by absorbance of A260 by using an absorption spectrometer. The concentration was 232 ng/μL in 30 μL of eluate.

### (2. Synthesis of Complementary DNA and Double Stranded Complementary DNA)

The extracted RNA sample was used to carry out adaptor-ligation PCR. First, in order to synthesize a complementary DNA, a BSL-18E primer (Table 1-1) and 3.5 μL (812 ng) of RNA were admixed and annealed for 8 minutes at 70° C. After cooling on ice, a reverse transcription reaction was performed in the presence of an RNase inhibitor (RNAsin) to synthesize a complementary DNA with the following composition.

The complementary DNA was subsequently incubated for hours at 16° C. in the following double-stranded DNA synthesis buffer in the presence of E. coli DNA polymerase I, E. coli DNA Ligase, and RNase H to synthesize a double stranded complementary DNA. Furthermore, T4 DNA polymerase was reacted for 5 minutes at 16° C. to perform a 5′ terminal blunting reaction.

A double stranded DNA, after column purification by a High Pure PCR Cleanup Micro Kit (Roche), was incubated all night at 16° C. in the presence of a P20EA/10EA adaptor (Table 1-1) and T4 ligase in the following T4 ligase buffer for a ligation reaction.

An adaptor added double stranded DNA purified by a column as discussed above was digested by a NotI restriction enzyme (50 U/μL, Takara) with the following composition in order to remove an adaptor added to the 3′ terminal.

(3. PCR)

For the first PCR amplification from a double stranded complementary DNA (1st PCR), a common adaptor primer P20EA and each of the immunoglobulin isotype C region specific primer (CM1, CA1, CG1, CD1, and CE1) were used for 20 cycles, where each cycle consisted of 30 seconds at 95° C., 30 seconds at 55° C., and one minute at 72° C. with the following reaction composition. The primer sequences used are shown in Table 1-1.

A 1st PCR amplicon, which is a product of the first PCR amplification reaction, was then used to perform nested PCR with the reaction composition shown below between a P20EA primer and each of the immunoglobulin isotype C region specific primers. 20 cycles of PCR were performed, where each cycle consisted of 30 seconds at 95° C., 30 seconds at 55° C., and one minute at 72° C. The primer sequences used are shown in Table 1-1.

Column purification was performed with a High Pure PCR Cleanup Micro Kit (Roche) to remove a primer from a 2nd PCR amplicon, which is a product obtained from the second

PCR amplification reaction. Subsequently, PCR was performed with the following reaction composition by using a B-P20EA primer, which is a P20EA primer added with an adaptor B sequence, and a GS-PCR primer, which is C region specific primer of each immunoglobulin added with an adaptor A sequence and identification sequence MID Tag sequence, with the 2nd PCR amplicon as a template. 10 cycles of PCR were performed, where each cycle consisted of 30 seconds at 95° C., seconds at 55° C., and one minute at 72° C. The primer sequences used are shown in Table 1-1.

(4. Next Generation Sequencing)

After GS-PCR amplification under the optimal condition, 2% agarose gel electrophoresis was performed. When visualized, a band was cut out in a size of interest (500 bp-700 bp) and purified by using a DNA purification kit (QIAEX II Gel Extraction Kit, QIAGEN). The amount of collected DNA was measured by using a Quant-iT™ PicoGreen® dsDNA Assay Kit (Invitrogen). The amounts of DNA collected in an amplicon derived from each isotype were IgM (1611 ng/mL), IgG (955 ng/mL), IgA (796 ng/mL), IgD (258 ng/mL), and IgE (871 ng/mL). They were mixed so that the amount of DNA of isotype amplicons would be equal. 10 million DNAs were used in emulsion PCR for sequence analysis with Roche's next generation sequence analyzer (GS Junior Bench Top system).

(5. Data Analysis)

Sequence read analysis assigned V, D, J, and C sequences of each read sequence by using V, D, J, and C sequences obtained from the IMGT (the international ImMunoGeneTics information system, http colon//www dot imgt dot org) database as reference sequences. IMGT's HighV-Quest and newly developed repertoire analysis software (Repertoire Genesis, see the patent application concurrently filed. The content thereof is incorporated herein by reference) were used for the assignment.

FIG. 1 shows cross-reactivity of isotype specific primers. In order to assess the specificity of the immunoglobulin isotype specific primers that were used, amplification was performed with an immunoglobulin isotype specific primer of interest as well as another isotype specific primer to verify the presence of cross-reactivity. 10 μL of GS-PCR amplicon was subjected to electrophoresis with 2% agarose gel in a TAE buffer, and then assessed by ethidium bromide staining. A 2nd PCR amplicon amplified with each isotype specific primer was not amplified with another isotype specific GS-PCR primer, verifying the high level of specificity of the primers.

FIG. 2 shows results of studying the optimal dilution concentration. A GS-PCR optimal condition for each isotype was studied. A 2-fold serial dilution system of 2nd PCR amplicons was created to perform 20 cycles of GS-PCR. Excellent results were obtained for 16-fold dilution.

FIG. 3 shows results of studying the optimal number of cycles. 16-fold dilution 2nd PCR amplicons were used for 10, 15, and 20 cycles PCR. For IgM, IgG, IgA, and IgD, excellent amplification was confirmed with 10 cycles. Further, it was confirmed that 20 cycles were appropriate for IgE.

FIG. 4 shows the read length from next generation sequencing. The read length from next generation sequencing of a BCR gene is shown in FIG. 4. The number of reads in Raw data is 130,000, and more than 90,000 reads that has gone through Filter pass were obtained. Table 1-2 shows the number of reads from each isotype that was labeled with a Tag.

FIG. 5 shows the read length of each MID. The read length and number of reads divided for each MID were evenly distributed. When counted while setting the read length sufficient for analyzing a V region as 400 bp or greater, half of the reads, about 10000 reads, were considered effective for BCR repertoire analysis.

FIG. 6 shows usage frequency of a C region sequence for each isotype. Search for homology with a C region sequence of immunoglobulin isotype including subclasses was performed on the obtained reads for each isotype. The frequency of number of reads for each subclass was 73% for IgA1 and 27% for IgA2 in the IgA subclass, 62% for IgG1 and 36% for IgG2 in the IgG subclass, while hardly any reads for IgG3 or IgG4 were obtained in the IgG subclass. Further, since obtained reads for each subclass were rarely classified into other classes, primer specificity was reconfirmed at the sequence level.

HighV-Quest of IMGT was used to assign V, D, and J regions (FIGS. 6A, 7A, 8A, and RA). Further, the results of assignment of V, D, and J regions by using a newly developed repertoire analysis software (Repertoire Genesis, patent pending) are shown in Table 1-1H. The data for number of reads was used to find the frequency of a V region and J region (FIGS. 6B, 7B, 8B, and 9B). The data thereof is shown below (Table 1-1H).

FIGS. 7 (A-D) show a V region repertoire for each isotype. A repertoire of a V region sequence for each isotype (BCR V repertoire) is shown. BCR V repertoires were very similar among IgM, IgG, IgA, and IgD, but only a read having IGHV3-30 was obtained for IgE. A reason therefor is suggested to be the possibility that there are much fewer number of IgE positive cells in peripheral blood relative to other classes and therefore a biased repertoire was detected.

FIGS. 8 (A-D) show a V region repertoire for each subtype. A BCR V repertoire is shown for each IgA and IgG subclass. The IgA subclass had different frequencies in several types of V chains between IgA1 and IgA2. The frequency of presence of IGHV1-18 and IGHV4-39 was higher in IgA1 compared to that in IgA2, while the frequency of presence of IGHV3-23 and IGHV3-74 was higher in IgA2 than that in IgA1. For the IgG subclass, the frequency of IGHV3-23 and IGHV3-74, which were found to be increased in IgA2, was higher in IgG2 compared to that in IgG1. There were few reads for IgG3 and IgG4 (10 reads). The frequency of clones with IGHV4-59-1GHJ4-IGHD1-7 was 3/10 in IgG3, thus having high clonality. Reads with IGHV3-23-IGHJ4-IGHD3-10 accounted for 5/10 for IgG4 (Table 1-3).

FIG. 9 shows a BCRJ repertoire for each subclass. IGHJ4 was used in about half of the reads in IgM, IgG, IgA and IgD, while IGHJ2 was hardly used. Only IGHJ1 was used in IgE. An IGHJ repertoire in subclasses of IgM and IgA was also studied. A significant difference among subclasses was not observed unlike an IGHV repertoire.

The above results demonstrated that unbiased quantitative analysis is possible with the sample providing method of the present invention.

### Preparation Example 2: Analysis of TCR Repertoire in Peripheral Blood of Healthy Individuals

The present Example performed TCR repertoire analysis on peripheral blood of healthy individuals.

(Materials and Methods)

(Sample)

Peripheral blood mononuclear cells of 10 healthy individuals

(Method)

(1. RNA Extraction)

5 mL of whole blood was collected from 10 healthy individuals in a heparin-containing blood collection tube. Peripheral blood mononuclear cells (PBMC) were separated by ficoll density gradient centrifugation. Total RNA was extracted/purified from the isolated PBMCs by using an RNeasy Lipid Tissue Mini Kit (Qiagen, Germany). The resulting RNA was quantified by using an Agilent 2100 bioanalyzer (Agilent). The amount of acquired RNA is shown in the following Table 1-4.

(2. Synthesis of Complementary DNA and Double Stranded Complementary DNA)

The extracted RNA sample was used to carry out adaptor-ligation PCR. The method was carried out in accordance with the method shown in Preparation Example 1. Specifically, a BSL-18E primer (Table 1-5) and RNA were admixed and annealed, and then a reverse transcriptase was used to synthesize a complementary strand DNA. A double-stranded DNA was subsequently synthesized. Furthermore, T4 DNA polymerase was used to perform a 5′ terminal blunting reaction. After column purification by a High Pure PCR Cleanup Micro Kit (Roche), a P20EA/P10EA adaptor was added in a ligation reaction. An adaptor added double stranded complementary DNA purified by a column was digested by a NotI restriction enzyme.

(3. PCR)

1st PCR amplification was performed for a first PCR amplification reaction product from a double stranded complementary DNA by using a common adaptor primer P20EA and a TCRα chain or β chain C region specific primer (CA1 or CB1) shown in Table 1-1. PCR was performed for 20 cycles of a cycle of 30 seconds at 95° C., 30 seconds at 55° C., and one minute at 72° C. with the following reaction composition.

A 1st PCR amplicon was then used to perform 2nd PCR with the reaction composition shown below by using a P20EA primer and a TCRα chain or β chain C region specific primer (CA2 or CB2). 20 cycles of PCR were performed, where a cycle was 30 seconds at 95° C., 30 seconds at 55° C., and one minute at 72° C.

A primer was removed with a High Pure PCR Cleanup Micro Kit (Roche) from a 2nd PCR amplicon, which is a product obtained from the second PCR amplification reaction. Furthermore, analysis was carried out with Roche's next generation sequence analyzer (GS Junior Bench Top system), with the 2nd PCR amplicon diluted 10 fold as a template. Amplification utilized a B-P20EA primer, which is a P20EA adaptor primer added with an adaptor B sequence, and HuVaF-01-HuVaF10 (α chain) and HuVbF-01-HuVbF-10 chain), which are TCRα chain or β chain C region specific primers added with an adaptor A sequence and each MID Tag sequence (MID-1 to 26) shown in FIG. 10. The primer sequences used are shown in Table 1-6. 10 cycles of PCR were performed, where a cycle was 30 seconds at 95° C., 30 seconds at 55° C., and one minute at 72° C. To confirm PCR amplification, 10 μL of amplicon was amplified by 2% agarose gel electrophoresis (FIG. 11).

After PCR amplification by agarose gel electrophoresis shown in FIG. 11, a band comprising about 600 bp of amplicon was cut out, when visualized, and purified by using a DNA purification kit (QIAEX II Gel Extraction Kit, Qiagen). The amount of DNA from the collected PCR amplicon was measured by using a Quant-T™ PicoGreen® dsDNA Assay Kit (Invitrogen). The collected amounts of DNA from each 10 healthy individual are shown in Table 1-7.

(4. Next Generation Sequencing)

Next generation sequencing was carried out by Roche's GS Junior sequence analyzer. Specifically, a GS Junior Titanium emPCR Kit (Lib-L) was used to carry out emPCR in accordance with the protocol of the manufacturer at the ratio of beads to DNA (copy per beads:cpb) of 0.5. After emPCR, a sequence run was carried out for the beads collected with beads enrichment by using sequence run reagents, GS Junior Titanium Sequencing Kit and PicoTiterPlate Kit, in accordance with the protocol of the manufacturer.

(5. Data Analysis)

The resulting sequence data (SFF file) was classified into read sequences for each MID Tag to create a sequence file in a Fasta format by a software that comes with the GS Junior (sfffile or sffinfo). The resulting mean number of reads was TRA: 17840 reads, TRB: 5122 reads, and the percentage of Raw data of 200 bp or greater was TRA: 34.9-63.7% (mean 42.2%) and TRB: 68.8-78.7% (mean 73.1%) (Table 1-8). The newly developed repertoire analysis software (Repertoire Genesis, patent pending) was then used for collation with reference sequences in the IMGT (the international ImMunoGeneTics information system, www dot imgt dot org) database to assign a V region, D region and J region of each read and determine the CDR3 sequence. The number of assigned reads is shown in Table 1-8. Further, the frequency of the same read was analyzed and usage frequency of V, D, and J chains was studied. FIGS. 13 (A-D), (A-D), 15 (A-D), and 16 show TRV and TRJ repertoires generated by using reads obtained with Repertoire Genesis.

FIG. 10 shows an amplification method of a TCR gene. Amplification was performed with a B-P20EA primer which is a P20EA adaptor primer added with a B-adaptor, and a primer which is a 3rd nested primer added with an A-adaptor and an MID Tag sequence (MID-1 to 26).

FIG. 11 shows results of examining GS-PCR amplicons. Electrophoresis was performed on 10 μL of GS-PCR amplicons derived from 10 healthy individuals with 2% agarose gel. The top row shows GS-PCR (TRA) (TCRα chain amplicon), and bottom row shows GS-PCR (TRB) (TCRβ chain amplicon).

FIG. 12 shows a parameter setting of a TCR/BCR repertoire analysis software (Repertoire genesis).

FIG. 13 shows a TRAV repertoire in health individuals. A TRAV repertoire for 10 healthy individuals and the mean value thereof are shown. The frequency of presence of TRAV9-2, 12 and 13 are high. TRAV20 in #1 and TRAV21 in #5 are higher than other healthy individuals, exhibiting variations among individuals.

FIG. 14 shows a TRBV repertoire in healthy individuals. A TRBV repertoire for 10 healthy individuals and the mean value thereof are shown. The frequency of presence of TRBV20-1, 28 and 29-1 are high. TRBV3-1 in #8 was higher than other healthy individuals, exhibiting variations among individuals.

FIG. 15 shows a TRAJ repertoire in healthy individuals. A TRAJ repertoire for 10 healthy individuals and the mean value thereof are shown. A TRAJ repertoire of healthy individuals showed about 5% or less in any AJ family. TRAJ12 in #1, TRAJ27 in #4, TRAJ37 in #5, and TRAJ45 in #8 were higher than other healthy individuals, exhibiting variations among individuals.

FIG. 16 shows a TRBJ repertoire in healthy individuals. A TRBJ repertoire for 10 healthy individuals and the mean value thereof are shown. The frequency of presence of TRBJ2-1, 2-3, and 2-7 was high and TRBJ2-2 was high in #8 in TRBJ repertoires of healthy individuals, exhibiting variations among individuals.

Thus, it was proven that unbiased quantitative analysis is also possible in TCRs by using a sample prepared by the preparation method of the present invention.

### Preparation Example 3: Amplification of TCR and BCR Genes by Unbiased Adaptor-Ligation PCR

In the present Example, TCR and BCR genes are amplified by unbiased adaptor-ligation PCR>

(Materials and Methods)

(Sample)

Peripheral blood mononuclear cells of a healthy individual

(Method)

(1. RNA Extraction)

5 mL of whole blood was collected from one healthy individual in a heparin-containing blood collection tube. Peripheral blood mononuclear cells (PBMC) were separated by ficoll density gradient centrifugation. Total RNA was extracted/purified from the isolated 5×106 PBMCs by using an RNeasy Lipid Tissue Mini Kit (QIAGEN, Germany).

(2. Synthesis of Complementary DNA and Double Stranded Complementary DNA)

The extracted RNA sample was used to carry out an adaptor-ligation PCR. First, in order to synthesize a complementary DNA, a BSL-18E primer (Table 1-1) and 3.5 μL (812 ng) of RNA were admixed and annealed for 8 minutes at 70° C. After cooling on ice, a reverse transcription reaction was performed in the presence of an RNase inhibitor (RNAsin) to synthesize a complementary DNA with the following composition.

The complementary DNA was subsequently incubated for hours at 16° C. in the following double-stranded DNA synthesis buffer in the presence of E. coli DNA polymerase I, E. coli DNA Ligase, and RNase H to synthesize a double stranded complementary DNA. Furthermore, T4 DNA polymerase was reacted for 5 minutes at 16° C. to perform a 5′ terminal blunting reaction.

A double stranded DNA, after column purification by a High Pure PCR Cleanup Micro Kit (Roche), was incubated all night at 16° C. in the presence of a P20EA/10EA adaptor (Table 1-1) and T4 ligase in the following T4 ligase buffer for a ligation reaction.

An adaptor added double stranded DNA purified by a column as discussed above was digested by a NotI restriction enzyme (50 U/μL, Takara) with the following composition in order to remove an adaptor added to the 3′ terminal.

The 1st PCR from a double stranded complementary DNA was performed by using a common adaptor primer P20EA and TCR C region specific primer (CA1, CB1, CG1, CD1), or immunoglobulin isotype C region specific primer (CM1, CA1, CG1, CD1, CE1, CK1, CL1). A primer was set at the 3′ terminal side, middle portion or 5′ side of a C region such that a sequence comprising the full length of the C region can be amplified. 20 cycles were performed, where a cycle was 30 seconds at 95° C., 30 seconds at 55° C., and one minute at 72° C. with the following reaction composition. The primer sequences used are shown in Table 1-1.

The 1st PCR amplicon, which is a product of the first PCR amplification reaction, was then used to perform nested PCR with the reaction composition shown below between a P20EA primer and each of the immunoglobulin isotype C region specific primers. 20 cycles of PCR were performed, where a cycle was 30 seconds at 95° C., 30 seconds at 55° C., and one minute at 72° C. The primer sequences used are shown in Table 1-1.

Amplicons with a size of interest, when visualized, were observed as a result of electrophoresis of each 2nd PCR amplicon synthesized by the second PCR amplification reaction with 2% agarose gel (FIG. 17).

FIGS. 18-25 show a primer position with respect to a template. The figures show that a significant range of regions is suitable as a PCR primer of interest of the present invention. It is also understood that a specific sequence can be appropriately determined based on the principles of the present invention.

(Preparation Example 4: Detection of tumor cells using human acute lymphoblastic leukemia cell line) Tumor cells were detected using a human acute lymphoblastic leukemia cell line in the present Example.

(Materials and Methods)

(Sample)

Peripheral blood mononuclear cells of a healthy individual, MOLT-4 human acute lymphoblastic leukemia cell line

(Method)

(1. Culture of T Cell Based Leukemia Cell Line)

A human acute lymphoblastic leukemia cell line Molt-4 was used as a T cell based cell line expressing a T cell receptor (TCR). Cells were cultured in an RPMI-1640 medium comprising 10% fetal bovine serum, 100 IU/ml penicillin, 100 μg/ml streptomycin, and 2 mM L-glutamine under the condition of 5% CO2 at 37° C. A total of 1×107 cells were collected. The cells were washed and suspended in the RPMI-1640 medium such that 1×106 cells/mL.

(2. Separation of Peripheral Blood Mononuclear Cells of Healthy Individual)

5 mL of whole blood was collected from one healthy individual in a heparin-containing blood collection tube. Peripheral blood mononuclear cells (PBMC) were separated by ficoll density gradient centrifugation. The cells were washed, counted, and suspended in the RPMI 1640 medium such that 1×106 cells/mL.

(3. Preparation of Serial Diluent of Cells)

The resulting 1×106 cells/mL PBMCs and 1×106 cells/mL Molt-4 cells were mixed so as to have the following number of cells to prepare a MOlt-4 serially diluted cell suspension.

(4. RNA Extraction and Measurement of Amount of RNA)

Total RNA was extracted/purified from the serially diluted cell suspension by using an RNeasy Lipid Tissue Mini Kit (QIAGEN, Germany). The RNA was eluted in 20 μL eluate. The amount of RNA was quantified by absorbance of A260 by using an Agilent 2100 bioanalyzer (Agilent). FIG. 26 shows an image of RNA electrophoresis. The amount of RNA obtained from each sample is shown in Table 1-4B.

(5. Synthesis of Complementary DNA and Double Stranded Complementary DNA)

The extracted RNA sample was used to carry out adaptor-ligation PCR. First, in order to synthesize a complementary DNA, a BSL-18E primer and 3.5 μL of RNA were admixed and annealed for 8 minutes at 70° C. After cooling on ice, a reverse transcription reaction was performed in the presence of an RNase inhibitor (RNAsin) to synthesize a complementary DNA with the following composition.

The complementary DNA was subsequently incubated for 2 hours at 16° C. in the following double-stranded DNA synthesis buffer in the presence of E. coli DNA polymerase I, E. coli DNA Ligase, and RNase H to synthesize a double stranded complementary DNA. Furthermore, T4 DNA polymerase was reacted for 5 minutes at 16° C. to perform a 5′ terminal blunting reaction.

A double stranded DNA, after column purification by a High Pure PCR Cleanup Micro Kit (Roche), was incubated overnight at 16° C. in the presence of a P20EA/10EA adaptor (Table 1-4E) and T4 ligase in the following T4 ligase buffer for a ligation reaction.

An adaptor added double stranded DNA purified by a column as discussed above was digested by a NotI restriction enzyme (50 U/μL, Takara) with the following composition in order to remove an adaptor added to the 3′ terminal.

(6. PCR)

The 1st PCR amplification was performed by using a common adaptor primer P20EA shown in Table 1-1 and a TCRα chain or β chain C region specific primer (CB1) from a double stranded complementary DNA. 20 cycles of PCR were performed, where a cycle was 30 seconds at 95° C., 30 seconds at 55° C., and one minute at 72° C. with the composition shown below.

A 1st PCR amplicon was then used to perform nested PCR with the reaction composition shown below by using a P20EA primer and each immunoglobulin isotype C region specific primer. 20 cycles of PCR were performed, where a cycle was 30 seconds at 95° C., 30 seconds at 55° C., and one minute at 72° C.

A primer was removed with a High Pure PCR Cleanup Micro Kit (Roche) from the obtained 2nd PCR amplicon. Furthermore, analysis was carried out with Roche's next generation sequence analyzer (GS Junior Bench Top system), with the 2nd PCR amplicon diluted 10 fold as a template. Amplification utilized a B-P20EA primer, which is a P20EA adaptor primer added with an adaptor B sequence, and HuVbF primer, which is a TCR chain C region specific prime added with an adaptor A sequence and each MID Tag sequence. 10 cycles of PCR were performed, where a cycle was 30 seconds at 95° C., 30 seconds at 55° C., and one minute at 72° C.

(7. Next Generation Sequencing)

Next generation sequencing was carried out by Roche's GS Junior sequence analyzer. Specifically, a GS Junior Titanium emPCR Kit (Lib-L) was used to carry out emPCR in accordance with the protocol of the manufacturer at the ratio of beads to DNA (copy per beads:cpb) of 2. After emPCR, a sequence run was carried out for the beads collected with beads enrichment by using sequence run reagents, GS Junior Titanium Sequencing Kit and PicoTiterPlate Kit, in accordance with the protocol of the manufacturer.

(8. Data Analysis)

The resulting sequence data (SFF file) was classified into read sequences for each MID Tag to create a sequence file in a Fasta format by a software that comes with GS Junior (sfffile or sffinfo). The resulting number of effective reads was 11651. A repertoire analysis software (Repertoire Genesis) was used for collation with reference sequences in the IMGT database to assign a BV region and BJ region of each read and to determine the CDR3 sequence. An in-frame TCR read (Read 1) having a functional sequence and a TCR read (Read 2) causing a frame shift were observed from the Molt-4 cells (Table 1-4J). Each was detected at about the same frequency and estimated to be a TCR gene derived from a Molt-4 cell. Gene rearrangement in two TCR loci has been already reported in Molt-4 cells (Cited Reference 1: Tunnacliffe A, Kefford R, Milstein C, Forster A, Rabbitts T H. Sequence and evolution of the human T-cell antigen receptor beta-chain genes. Proc Natl Acad Sci USA. 1985 August; 82(15): 5068-72.) The sequence of a functional TCR gene (Read 1) matched the already reported sequence (Cited Reference 2: Assaf C, Hummel M, Dippel E, Goerdt S, Muller H H, Anagnostopoulos I, Orfanos C E, Stein H. High detection rate of T-cell receptor beta chain rearrangements in T-cell lymphoproliferations by family specific polymerase chain reaction in combination with the GeneScan technique and DNA sequencing. Blood. 2000 Jul. 15; 96(2):640-6., GenBank Accession number: M12886.1).

In order to find out the detection limit of Molt-4 cells by the next generation TCR repertoire analysis method, two TCR reads derived from Molt-4 cells were searched and collated in TCR reads acquired from serially diluted samples (FIGS. 27 (A-D)). As a result, Read 1 and Read 2 were detected in accordance with the number of cells in the serially diluted samples. It was confirmed that 61 Reads (3.1%) were present in the 0.1 sample for Read 1 and 1 Read (0.01%) was present in the 0.01% sample for Read 2 (Table 1-4K). The functional TCR, Read 1, was not detected in the 0.01% sample, while Read 2 predicted to lack functionality was detected. This suggests that certainty of tumor cell detection is elevated by searching a plurality of TCR genes derived from one T cell. The results show that the present method can detect tumor cells at high sensitivity.

(Results)

FIG. 26 shows an image of RNA electrophoresis by an Agilent 2100 bioanalyzer. Total RNA was extracted from a serially diluted cell solution and the amount of RNA was measured with an Agilent bioanalyzer. An RNA was separated with a microchip electrophoretic apparatus to check the quality of the RNA. 28S (top band) and 18S rRNA (bottom band) were detected in each sample, demonstrating that an RNA which has not been degraded was obtained.

FIGS. 27 (A-D) show TCR reads in serially diluted Molt-4 cell samples (SEQ ID NOs: 1165-1324). TCR reads acquired from each of 10%, 1%, 0.1%, and 0.01% serially diluted Molt-4 sample are described. The reads were ranked in the order of having a greater number of reads and the top 40 positions are shown. Ranking 365 to 404 are shown for the 0.01% sample. TRBV, TRBJ, and CDR3 amino acid sequences for each read, and number of reads are shown. Functional TCR reads (TRBV20-1/TRBJ2-1/CSARESTSDPKNEQFFG (SEQ ID NO: 1166)) derived from Molt-4 are shown in bold with a gray background. The other TCR reads estimated to have a functional deficiency (TRBV10-3/TRBJ2-5/CAISEPTGIRRDPVLR (SEQ ID NO: 1165)) are shown in bold.

FIG. 28 shows detection sensitivity and number of TCR reads in a serially diluted Molt-4 cell sample. Two TCR reads were detected from a Molt-4 cell (▴: TRBV20-1/TRBJ2-1/CSARESTSDPKNEQFFG (SEQ ID NO: 1166), ∘: TRBV10-3/TRBJ2-5/CAISEPTGIRRDPVLR (SEQ ID NO: 1165)). The figure shows the percentage of TCR reads derived from Molt-4 detected in TCR reads acquired from each of 10%, 1%, 0.1% and 0.01% serially diluted Molt-4 samples. The detection limit for each read was 0.1% (▴) and 0.01% (∘).

## ANALYTICAL TEST EXAMPLE

### Analytical Test Example 1 BCR Repertoire Analysis on Healthy Individuals

The present Example compared BCR repertoires of healthy individuals.

(Materials and Methods)

(Materials)

A read set was used, which was from sequencing, with a Roche GS-Junior, a cDNA of a BCR unbiasedly obtained from an RNA obtained from one specimen of healthy individual peripheral blood mononuclear cell. The read set is for each class of IgM, IgG, IgA, IgD, and IgE.

(Method)

FIG. 30 shows the overall picture of the method (FIG. 29 shows a TCR analysis scheme).

Previously reported allelic nucleic acid sequences from IMGT were obtained for use as a reference data base. BLASTN was used for homology search while setting the following parameters for each region.

V mismatch penalty=−1, shortest alignment length=30, and shortest kernel length=15;

D word length=7, mismatch penalty=−1, gap penalty=0, shortest alignment length=11, and shortest kernel length=8;

J mismatch penalty=−1, shortest hit length=18, and shortest kernel length=10; and

C shortest hit length=30 and shortest kernel length=15.

An indicator used for selecting the closest reference allele was applied in the following priority order.

1. number of matching bases, 2. kernel length, 3. score, and 4. alignment length, and then for each class, the frequency of appearance of gene name for each region was calculated and compared with one another. Further, IgG and IgA have subclasses. Thus, comparison was also performed between subclasses.

(Results)

FIG. 31 shows results of deriving out a frequency of appearance of C gene name for each read set of IgM, IgG, IgA, IgD, and IgE. Only the gene name corresponding to each class appeared and hardly any no-hit is observed, suggesting sufficient quality of the read set subjected to analysis.

The results of calculating a D repertoire for each class are shown in Tables 2-3 and 2-4. Table 2-3 and Table 2-4 show a comparison of D repertoires among classes. The number of reads appeared is described for each gene name and CDR3 amino acid sequence. The gene name and amino acid sequence with number of reads of 1 were omitted. Further, FIGS. 32 (A and B) show a V repertoire and FIG. 33 shows a J repertoire. FIGS. 34 (A and B) show a comparison of V repertoires among subclasses. FIG. 35 shows a comparison of J repertoires among subclasses. For D, a frequency is derived out for a combination of a D gene name and CDR3 amino acid sequence.

(Table 2-3) Comparison of D repertoire among classes, vertical axis: frequency (%), horizontal axis: gene name

(Table 2-4) Comparison of D repertoires among classes, vertical axis: frequency (%), horizontal axis: gene name

The above results show that the analysis technique of the present invention can calculate and materialize quick analysis in several minutes.

### Analytical Test Example 2 Comparison of BCR Repertoire Among Specimens

The present Example compared BCR repertoires among specimens.

(Materials and Methods)

(Materials)

For a read set of 5 specimens obtained by the same technology as Analysis Example 1, 4 specimens (No. 1-4) are healthy individuals and one specimen (No. 5) is a leukemia patient.

(Method)

A repertoire was derived out for each class and each region for each sample by the same method as Analysis Example 1 and compared among specimens.

(Results)

As an example of results, FIGS. 36 (A and B) show results of comparing V repertoires in IgM and FIG. 37 shows results of comparing J repertoires. It is demonstrated that only specimen No. 5 is significantly different.

### Analytical Test Example 3: Comparison of TCR Repertoires of Healthy Individuals

The present Example compared TCR repertoires of healthy individuals.

(Materials and Methods)

(Materials)

For a read set of 10 specimens obtained by the same technology as Example 1, 10 specimens (No. 1-10) are all healthy individuals.

(Methods)

A repertoire was derived out for each class and each region for each sample by the same method as Example 1 and compared among specimens.

(Results)

Results are shown in FIGS. 38-41. FIGS. 38 (A-D) show results of comparing TRAV repertoires among specimens. FIGS. 39 (A-D) show results of comparing TRBV repertoires among specimens. FIGS. 40 (A-D) show results of comparing TRAJ repertoires among specimens. FIG. 41 shows result of comparing TRBJ repertoires among specimens.

Each of the results was able to be obtained in about several minutes.

The present analysis method can materialize analysis of a C region, which was not provided by High-V-QUEST that has been commonly used. The advantage of the present system includes that “unit of gene name” or “unit of allele” can be selected for each region. Although not wishing to be bound by any theory, this is because such a selection is not materialized with High-V-QUEST that has been commonly used. The (current) issue in High-V-QUEST method is in insufficient classification of D regions, while the present system can be considered as solving such an issue. Specifically, database content for a D region is insufficient in High-V-QUEST, and therefore D region sequences that are not similar to a DB record would all be swept under “no hit”. In contrast, the system of the present system can utilize a CDR3 sequence instead of a D gene name/allele as a classification category. Thus, classification as far as currently possible can be performed. The present system can be used without any limit in the number of sequences. Although not wishing to be bound by any theory, this is due to consideration such that further deep sequencing to search for a rare clone, when performed, can be analyzed without any change. Instead, a feature of limiting the number of analysis jobs processed simultaneously (when full, automatically processed later), i.e., a job queue style management function, is introduced to prevent depletion of computational resources. The disadvantage of High-V-QUEST, which has a limited number of maximum sequences, is overcome thereby.

(Examples of Analysis System)

(Example 1 of analysis system: Diagnostic application in T-cell large granular lymphocyte leukemia (T-LGL))

The present invention performed an experiment to confirm application of the system of the present invention in diagnosis of T-cell large granular lymphocyte leukemia (T-LGL).

Sample: Peripheral blood mononuclear cells derived from T-cell large granular lymphocyte leukemia Method

(RNA Extraction)

7 mL of whole blood was collected from one patient suffering from T-cell large granular lymphocyte leukemia in a heparin-containing blood collection tube. Peripheral blood mononuclear cells (PBMC) were separated by ficoll density gradient centrifugation. Total RNA was extracted/purified from the isolated 1.66×107 PBMCs by using an RNeasy Lipid Tissue Mini Kit (QIAGEN, Germany). The resulting RNA was quantified by absorbance of A260 by using an absorption spectrometer. The amount of total RNA was 15 μg.

(Synthesis of Complementary DNA and Double Stranded Complementary DNA)

The extracted RNA sample was used to carry out an adaptor-ligation PCR. First, in order to synthesize a complementary DNA, a BSL-18E primer (Table 3-1A) and 3.5 μL of RNA were admixed and annealed for 8 minutes at 70° C. After cooling on ice, a reverse transcription reaction was performed in the presence of an RNase inhibitor (RNAsin) to synthesize a complementary DNA with the following composition.

The complementary DNA was subsequently incubated for hours at 16° C. in the following double-stranded DNA synthesis buffer in the presence of E. coli DNA polymerase I, E. coli DNA Ligase, and RNase H to synthesize a double stranded complementary DNA. Furthermore, T4 DNA polymerase was reacted for 5 minutes at 16° C. to perform a 5′ terminal blunting reaction.

A double stranded DNA, after column purification by a High Pure PCR Cleanup Micro Kit (Roche), was incubated all night at 16° C. in the presence of a P20EA/10EA adaptor (Table 3-1A) and T4 ligase in the following T4 ligase buffer for a ligation reaction.

An adaptor added double stranded DNA purified similarly by a column as discussed above was digested by a NotI restriction enzyme (50 U/μL, Takara) with the following composition in order to remove an adaptor added to the 3′ terminal.

3. PCR

The 1st PCR amplification was performed by using a common adaptor primer P20EA and a TCRα chain or β chain C region specific primer (CA1 or CB1) from a double stranded complementary DNA. 20 cycles of PCR were performed, where a cycle was 30 seconds at 95° C., 30 seconds at 55° C., and one minute at 72° C. with the composition shown below.

A 1st PCR amplicon was then used to perform 2nd PCR with the reaction composition shown below by using a P20EA primer and a TCRα chain or β chain C region specific primer (CA2 or CB2). 20 cycles of PCR were performed, where a cycle was 30 seconds at 95° C., 30 seconds at 55° C., and one minute at 72° C.

A primer was removed from the obtained 2nd PCR amplicon by a High Pure PCR Cleanup Micro Kit (Roche). Furthermore, analysis was carried out with Roche's next generation sequence analyzer (GS Junior Bench Top system), with the 2nd PCR amplicon diluted 10 fold as a template. Amplification utilized a B-P20EA primer, which is a P20EA adaptor primer added with an adaptor B sequence, and HuVaF-01-HuVaF10 (α chain) and HuVbF-01-HuVbF-10 chain), which are a TCRα chain or β chain C region specific primer added with an adaptor A sequence and each MID Tag sequence (MID-1 to 26). 10 cycles of PCR were performed, where a cycle was 30 seconds at 95° C., 30 seconds at 55° C., and one minute at 72° C.

In agarose gel electrophoresis, a band comprising about 600 bp of amplicon was cut out, when visualized, and purified by using a DNA purification kit (QIAEX II Gel Extraction Kit, Qiagen). The amount of DNA from the collected PCR amplicon was measured by using a Quant-T™ PicoGreen® dsDNA Assay Kit (Invitrogen).

### 4. Next Generation Sequencing

Next generation sequencing was carried out with Roche's GS Junior sequence analyzer. Specifically, a GS Junior Titanium emPCR Kit (Lib-L) was used to carry out emPCR in accordance with the protocol of the manufacturer at the ratio of beads to DNA (copy per beads:cpb) of 0.5. After emPCR, a sequence run was carried out for the beads collected with beads enrichment by using sequence run reagents GS Junior Titanium Sequencing Kit and PicoTiterPlate Kit in accordance with the protocol of the manufacturer.

### 5. Data Analysis

The resulting sequence data (SFF file) was classified into read sequences for each MID Tag to create a sequence file in a Fasta format by a software that comes with GS Junior (sfffile or sffinfo). For sequence read analysis, V, D, J, and C sequences of each read sequence were assigned by using V, D, J, and C sequences obtained from the IMGT (the international ImMunoGeneTics information system, www dot imgt dot org) database as reference sequences. The newly developed software (Repertoire Genesis) was used for the assignment. 22,833 reads were obtained and 16,407 reads (71.9%) were assigned for TCR.alpha. The number of unique reads was 1705 reads. 121,080 reads were obtained and 81,542 reads (67.3%) were assigned for TCR.beta. The number of unique reads was 9,224. The frequency of the obtained reads was studied, with a read having the same TRAV gene, TRAJ gene and CDR3 sequence as a unique read (Table 3-1). Similarly, the frequency was studied for a read with the same TRBV gene TRBJ gene, and CDR3 sequence (Table 3-2). As a result, 1971 reads (12.53%) accounted for a read with TRAV10, TRAJ15 and CVVRATGTALIFG (SEQ ID NO: 1450) for a TRA repertoire, suggesting the possibility that a cell expressing a specific TCR has increased clonality. Further, 22568 reads (28.57%) accounted for reads with TRBV29-1, TRBJ2-7, and CSVERGGSLGEQYFG (SEQ ID NO: 1500) in a TRB repertoire. The results suggest the possibility of monoclonal increase in T cells expressing a TCR molecule consisting of TCR.alpha. having TRAV10 and TRAJ15 and TCR having TRBV29-1 and TRBJ2-7. Various diversity indices were compared among 10 healthy individuals and LGL patients (Table 3-3). The Shannon-Weaver's index (H′), Simpson's index (.lamda.), Inverse Simpson's index (1/.lamda.), and Pielou's index (J′) indicating diversity were exhibiting a lower value compared to healthy individuals, demonstrating reduced diversity.

6. Utility in Diagnosis

It is expected that a minor residual lesion can be detected with a sequence read having TRAV10/TRAJ15/CVVRATGTALIFG (SEQ ID NO: 1450) or TRBV29-1/TRBJ2-7/CSVERGGSLGEQYFG (SEQ ID NO: 1500) as an indicator after applying therapy such as drug therapy in the LGL patients. Further, it is understood that a therapeutic effect on leukemia cells can be measured from quantitative analysis using read frequencies. Further, the possibility of being able to predict the presence of a clonality increasing disease by using various diversity indices was suggested.

(Table 3-1) TRA read (top 50) (SEQ ID NOs: 1450-1499)

**Table 3-3 Diversity Index**

FIG. 44 shows a distribution of the number of unique reads in TCRα and TCRβ chain repertoire analysis. The distribution was examined for unique reads (base sequence without commonality with other reads) of all sequence reads, with number of copies in the horizontal axis. A read that was only detected once (single) was 73.3% (1250 reads) of the whole for TCRα, and 70.5% (6502 reads) for a TCRβ chain.

FIG. 45 shows TRAV and TRAJ repertoires. The usage frequency of each of TRAV and TRAJ in all reads is shown. The horizontal axis indicates TRAV genes (top graph) and TRAJ genes (bottom graph). The vertical axis indicates the percentage (% Usage) accounted for among all reads.

FIG. 46 shows a 3D plot of a TRA repertoire. Usage frequency of each combination of TRAV and TRAJ in all reads is shown in a three-dimensional plot. The horizontal axis indicates a TRAJ gene, the depth axis indicates a TRAV gene, and the vertical axis indicates usage frequency (% Usage). The combination of TRAV10 and TRAJ15 exhibited the highest usage frequency (12.53%). FIG. 47 shows TRBV and TRBJ repertoires. The usage frequency of each of TRBV and TRBJ in all reads is shown. The horizontal axis indicates TRBV genes (top graph) and TRBJ genes (bottom graph). The vertical axis indicates the percentage (% Usage) accounted for among all reads.

FIG. 48 shows a 3D plot of a TRB repertoire. Usage frequency of each combination of TRBV and TRBJ in all reads is shown in a three-dimensional plot. The horizontal axis indicates a TRBV gene, the depth axis indicates a TRBJ gene, and the vertical axis indicates usage frequency (% Usage). The combination of TRBV29-1 and TRBJ2-7 exhibited the highest usage frequency (28.57%).

(Example 2 of analysis system: Analysis of T cells infiltrating large intestine tissue of HLA-A2402 colorectal cancer patients) The present Example analyzed T cells infiltrating large intestine tissue of HLA-A2402 colorectal cancer patients by using the analysis system of the present invention.

(Materials and Methods)

Sample: Tumor tissue of colorectal cancer patients extracted by surgical operation, peripheral blood of healthy individuals Method

(Collection and Storage of Colorectal Cancer Tissue)

Tumor tissue was collected by a tumor extraction surgery in 60 large intestine patients. 100 mg of tissue corresponding to the size of a soy bean was collected from a cancer lesion of the extracted organ. The tissue was cut into a square with a 5 mm side and immediately immersed in an RNA stabilizing agent (RNAlater®, Ambion). After storing at 4° overnight, RNAlater® was removed and then the tissue was stored at −80° C.

(Isolation of Peripheral Blood of Healthy Individuals)

As a control, peripheral blood cells of a healthy individual were used. 5 mL of whole blood was collected from 10 healthy individuals in a heparin-containing blood collection tube. Peripheral blood mononuclear cells (PBMC) were separated by ficoll density gradient centrifugation. Total RNA was extracted/purified from the isolated 5×106 PBMCs by using an RNeasy Lipid Tissue Mini Kit (QIAGEN, Germany). The resulting RNA was quantified by absorbance of A260 by using an absorption spectrometer (Table 3-3A). Table 3-3A Amount of total RNA in peripheral blood cells of healthy individuals

(Examination of HLA Haplotype)

HLA-A typing was carried out in order to identify the expression of HLA and HLA haplotype in cancer tissue. A part of cancer tissue immersed in RNA Later® was taken out, and a genomic DNA was extracted by using a QIAampDNA Mini Kit (Qiagen, Germany). The DNA was then amplified and labeled by using a WAKFlow HLA typing reagent HLA-A (Wakunaga) and analyzed with Luminex (Luminex Corp.) As a result, HLA-A2402 gene was expressed homo or hetero in 25 specimens out of 60 specimens (Tables 3-4).

**Table 3-4 List of Colorectal Cancer Tissue Expressing HLA-A2402**

(RNA Extraction and Measurement of Amount of RNA)

In order to analyze a TCR repertoire in 25 specimens expressing a HLA-A2402 gene, a portion of tissue immersed in RNAlater® was taken out, and total RNA was extracted/purified by using an RNeasyLipidTissue Mini Kit (QIAGEN, Germany). Elution from a column was carried out with 50 μL of RNAase free-sterilized water. The amount of RNA obtained from each sample is shown in Table 3-5.

**Table 3-5 Amount of Total RNA of Colorectal Cancer Sample**

(Synthesis of Complementary DNA and Double Stranded Complementary DNA)

The extracted RNA sample was used to carry out an adaptor-ligation PCR. First, in order to synthesize a complementary DNA, a BSL-18E primer and 3.5 μL of RNA were admixed and annealed for 8 minutes at 70° C. After cooling on ice, a reverse transcription reaction was performed in the presence of an RNase inhibitor (RNAsin) to synthesize a complementary DNA with the following composition.

The complementary DNA was subsequently incubated for hours at 16° C. in the following double-stranded DNA synthesis buffer in the presence of E. coli DNA polymerase I, E. coli DNA Ligase, and RNase H to synthesize a double stranded complementary DNA. Furthermore, T4 DNA polymerase was reacted for 5 minutes at 16° C. to perform a 5′ terminal blunting reaction.

A double stranded DNA, after column purification by a High Pure PCR Cleanup Micro Kit (Roche), was incubated all night at 16° C. in the presence of a P20EA/10EA adaptor and T4 ligase in the following T4 ligase buffer for a ligation reaction.

An adaptor added double stranded DNA purified by a column as discussed above was digested by a NotI restriction enzyme (50 U/μL, Takara) with the following composition in order to remove an adaptor added to the 3′ terminal.

### 5. PCR

The 1st PCR amplification was performed by using a common adaptor primer P20EA and a TCRα chain or β chain C region specific primer (CB1) from a double stranded complementary DNA from a double stranded complementary DNA. 20 cycles of PCR were performed, where a cycle was 30 seconds at 95° C., 30 seconds at 55° C., and one minute at 72° C. with the composition shown below.

A 1st PCR amplicon was then used to perform nested PCR with the reaction composition shown below between a P20EA primer and each of the immunoglobulin isotype C region specific primers. 20 cycles of PCR were performed, where a cycle was 30 seconds at 95° C., 30 seconds at 55° C., and one minute at 72° C.

A primer was removed from the obtained 2nd PCR amplicon by a High Pure PCR Cleanup Micro Kit (Roche). Furthermore, analysis was carried out with Roche's next generation sequence analyzer (GS Junior Bench Top system), with the 2nd PCR amplicon diluted 10 fold as a template. Amplification utilized a B-P20EA primer, which is a P20EA adaptor primer added with an adaptor B sequence, and HuVaF- and HuVbF primers, which are TCRα chain and β chain C region specific primers added with an adaptor A sequence and each MID Tag sequence. 10 cycles of PCR were performed, where a cycle was 30 seconds at 95° C., 30 seconds at 55° C., and one minute at 72° C.

### 6. Next Generation Sequencing

Next generation sequencing was carried out by Roche's GS Junior sequence analyzer. Specifically, a GS Junior Titanium emPCR Kit (Lib-L) was used to carry out emPCR in accordance with the protocol of the manufacturer at the ratio of beads to DNA (copy per beads:cpb) of 2. After emPCR, a sequence run was carried out for the beads collected with beads enrichment by using sequence run reagents GS Junior Titanium Sequencing Kit and PicoTiterPlate Kit in accordance with the protocol of the manufacturer.

### 7. Data Analysis

The resulting sequence data (SFF file) was classified into read sequences for each MID Tag to create a sequence file in a Fasta format by a software that comes with GS Junior (sfffile or sffinfo). A repertoire analysis software (Repertoire Genesis) was used for collation with reference sequences in the IMGT database to assign an AV region, BV region, AJ region, and BJ region of each read and determine the CDR3 sequence.

### 8. Extraction of Overlapping Unique Reads in Analysis of 10 Healthy Individuals

As a normal control, TCR sequences of peripheral blood mononuclear cells of 10 healthy individuals were examined. An overlapping read was searched among individuals by using V, J and CDR3 sequences as an indicator for TCRα and TCRβ sequence reads obtained from each healthy individual. The number of overlapping unique reads and the number of individuals with such overlapping unique reads were examined between TCRα and TCRβ chains (Table 3-6). Relative to TCR chains, the number of overlapping unique reads was significantly more in TCRα chains (809 vs. 39) and the ratio thereof was also higher (2.37% vs. 0.19%). Further, there were overlapping reads in out of 10 individuals at the maximum for TCRα chains, while all overlapping reads for TCR chains were overlapping in only 2 individuals. The results suggest that there is more similarity among individuals in a TCRα repertoire.

### 9. Analysis of Overlapping Reads in TCRα Chain

The base sequence of an overlapping read was examined in detail for a TCRα chain exhibiting a level of overlap among individuals that is high relative to a TCRβ, chain. As a result, it was discovered that many of the TCR reads exhibiting a high level of overlap are TCRα genes derived from a mucosal-associated invariant T cell (MAIT) or a natural killer T cell (NKT) known as expressing an invariant chain (Table 3-7). NKT cells mainly express TRAV10 (Vα24)-TRAJ18 and MAITs mainly express a TCR consisting of TRAV1-2 (Vα7.2)-TRAJ33. It has been reported recently that a TCR of a MAIT recognizes a vitamin B metabolite of bacteria presented by an MR1 molecule and has drawn attention for the role in immunosurveillance function (Nature. 2012 Nov. 29; 491 (7426): 717-23; JExp Med. 2013 Oct. 21; 210(11): 2305-20). When an overlapping read with a number of overlapping individuals of 4 or more was collated with an already reported invariant TCR, it was found that 45% thereof was accounted for by invariant TCRs (Table 3-7). In contrast to TCRα chains where there is a highly frequent overlapping read, the number of overlapping individuals for a TCRβ, chain was at maximum 2 (Table 3-8). Thus, the high level of overlap in TCRα is estimated to be due to the presence of an invariant TCR. 21 types of TCRα reads, which could not be collated with an already reported invariant TCR, were identified among 38 types of highly frequently overlapping reads that overlap in 4 or more individuals (Table 3-9). The possibility is suggested or them being novel invariant TCRs.

### 10. Analysis of Overlapping Read in Colorectal Cancer Patient Tissue

It is known that a cancer antigen specific T cells is present in cancer tissue of a cancer patient and has an important role in an antitumor effect. In order to identify a cancer antigen specific TCR gene, a TCR repertoire is analyzed while targeting a patient with a specific HLA to identify a TCR gene that grows in response to a specific antigen. In this experiment, cancer tissue of 25 colorectal cancer patients having a common HLA-A2402 was used to carry out TCR repertoire analysis to search for a unique read that is present while overlapping among cancer patient samples (Table 3-10). As a result, it was found that 213 reads (1.65%) were present while overlapping in a plurality of patients for a TCRα chain and 49 reads (0.11%) for a TCRβ chain. As in healthy individuals, there is a highly frequently overlapping read in maximum of 12 out of 25 cases for a TCRα chain, while only a maximum of 2 for a TCRβ chain. For a TCRα chain, there is a common read among a maximum of 12 specimens, and sequences of 7 reads overlapping in cancer tissue of 4 or more specimens were TCRα chains having TRAV1-2/TRAJ33 derived from MAITs except for one case (Table 3-11). (Table 3-10 Number of cancer specific reads and number of overlapping unique reads in cancer tissue)

### 11. Extraction of Cancer Specific TCR Sequence

TCRα reads that overlap at a high frequency comprise many invariant TCRs. These sequences are also present in healthy individuals, who are normal controls, and are not TCRs that react to a tumor antigen. In order to extract a cancer specific TCR, overlapping reads in cancer tissue which are not detected in a sample of a healthy individual were classified as a cancer specific TCR (Table 3-12). There were 56 overlapping leads that are also present in healthy individuals in a TCRα chain, while there was only one such read in a TCR chain. Reads with a number of overlapping individuals of 4 or more were reads that were also present in a healthy individual or an invariant TCR. A cancer specific read overlapping in 3 or less individuals was detected in 157 reads (1.22%) in a TCRα chain and 48 reads (0.11%) in a TCR chain (also see FIG. 49 for method of estimating TCRαβ pair reads).

**Table 3-13 Estimation of Paired TCRαβ by Combination of Overlapping Individuals**

(Example 3 of analysis system: Sequencing using Ion PGM system (Ion Torrent)) (1. RNA extraction) 5 mL of whole blood was collected from a healthy individual in a heparin-containing blood collection tube. Peripheral blood mononuclear cells (PBMC) were separated by ficoll density gradient centrifugation. Total RNA was extracted/purified from the isolated PBMCs by using an RNeasy Lipid Tissue Mini Kit (Qiagen, Germany). The resulting RNA was quantified by using an Agilent 2100 bioanalyzer (Agilent).

### (2. Synthesis of Complementary DNA and Double Stranded Complementary DNA)

The extracted RNA sample was used to carry out an adaptor-ligation PCR. The method was carried out in accordance with the method shown in Example 1. Specifically, a BSL-18E primer (Table 3-14) and RNA were admixed and annealed, and then a reverse transcriptase was used to synthesize a complementary strand DNA. A double-stranded DNA was subsequently synthesized. Furthermore, T4 DNA polymerase was used to perform a 5′ terminal blunting reaction. After column purification by a High Pure PCR Cleanup Micro Kit (Roche), a P20EA/P10EA adaptor was added in a ligation reaction. An adaptor added double stranded complementary DNA purified by a column was digested by a NotI restriction enzyme.

### (3. PCR)

The 1st PCR amplification was performed for a first PCR amplification reaction product from a double stranded complementary DNA by using a common adaptor primer P20EA and a TCRα chain or β chain C region specific primer (CA1 or CB1) shown in Table 3-14. 20 cycles of PCR were performed, where a cycle was 30 seconds at 95° C., 30 seconds at 55° C., and one minute at 72° C. with the following reaction composition.

A 1st PCR amplicon was then used to perform 2nd PCR with the reaction composition shown below by using a P20EA primer and a TCRα chain or β chain C region specific primer (CA2 or CB2). 20 cycles of PCR were performed, where a cycle was 30 seconds at 95° C., 30 seconds at 55° C., and one minute at 72° C.

PCR was performed with the 2nd PCR amplicon diluted 10 fold as a template by utilizing a B-P20EA primer shown in FIG. 10, which is a P20EA adaptor primer added with an adaptor B sequence, and HuVaF-Ol-HuVaF10 (α chain) and HuVbF-01-HuVbF-10 (β chain), which are a TCRα chain or β chain C region specific primer added with an adaptor A sequence and each MID Tag sequence (MID-1 to 26). The primer sequences used are shown in Table 6. 10 cycles of PCR were performed, where a cycle was 30 seconds at 95° C., 30 seconds at 55° C., and one minute at 72° C. To confirm PCR amplification, 10 μL of amplicon was amplified with 2% agarose gel electrophoresis.

An Ion OneTouch 2 system (Ion Torrent) was then used to perform emulsion PCR and adjust a template. An Ion OneTouch 2 (Ion Torrent) kit was used to mix the following solution.

An Ion Sphere Particle (ISP) bead was stirred and then 100 μL of ISP was added and mixed as described below.

The above-described 1,000 μL is sufficiently mixed and then stirred for 5 minutes. After setting up an Ion OneTouch Plus Reaction Filter Assembly, the total amount described above is loaded. Furthermore, 500 μL of Ion OneTouch Reaction Oil is added and then a run is initiated. After about 5.5 hours of reaction, a sample is collected. After centrifugation to remove an excessive solution, ISP is collected.

### Enrichment

An Ion OneTouch ES (Ion Torrent) is used to enrich a sample. A new tube is set in a chip loader, and a chip arm is installed. The following melt-off solution is then prepared.

Melt-Off Solution

### Dispensing the Following Solution to Each Well of 8 Strip Tubes

After setting up a reagent, an apparatus for Ion OneTouch ES is initiated for enrichment. After completion, tubes containing ISP are collected and gently inverted and mixed 5 times. An Ion PGM Sequencing 200 Kit v2 (Ion Torrent) is then used for sequencing.

It can be understood that the system of the present invention can use an apparatus other than Roche apparatuses in this manner.

(Example 4 of analysis system: TCR sequencing using Illumina MiSeq system)

The present Example demonstrates whether the system of the present invention can be implemented in TCR sequencing using an Illumina MiSeq system.

(1. RNA Extraction)

5 mL of whole blood was collected from a healthy individual in a heparin-containing blood collection tube. Peripheral blood mononuclear cells (PBMC) were separated by ficoll density gradient centrifugation. Total RNA was extracted/purified from the isolated PBMCs by using an RNeasy Lipid Tissue Mini Kit (Qiagen, Germany). The resulting RNA was quantified by using an Agilent 2100 bioanalyzer (Agilent).

(2. Synthesis of Complementary DNA and Double Stranded Complementary DNA)

The extracted RNA sample was used to carry out an adaptor-ligation PCR. The method was carried out in accordance with the method shown in Example 1. Specifically, a BSL-18E primer (Table 3-21) and RNA were admixed and annealed, and then a reverse transcriptase was used to synthesize a complementary strand DNA. A double-stranded complementary DNA was subsequently synthesized, and T4 DNA polymerase was used to perform a 5′ terminal blunting reaction. After column purification by a High Pure PCR Cleanup Micro Kit (Roche), a P20EA/P10EA adaptor was added in a ligation reaction. An adaptor added double stranded complementary DNA purified by a column was digested by a NotI restriction enzyme.

(3. PCR)

The 1st PCR amplification was performed for a first PCR amplification reaction product from a double stranded complementary DNA by using a common adaptor primer P20EA shown in Table 1 and a TCRα chain or β chain C region specific primer (CA1 or CB1). 20 cycles of PCR were performed, where a cycle was 30 seconds at 95° C., 30 seconds at 55° C., and one minute at 72° C. with the reaction composition in the following Table 3-22.

A 1st PCR amplicon was then used to perform 2nd PCR with the reaction composition shown in the following Table 3-23 by using a P20EA primer and a TCRα chain or β chain C region specific primer (CA2 or CB2). 20 cycles of PCR were performed, where a cycle was 30 seconds at 95° C., 30 seconds at 55° C., and one minute at 72° C.

(4. MiSeq Dual-Indexed Paired-End Sequencing)

A PCR amplification reaction is performed with the 2nd PCR amplicon diluted 10 fold as a template by utilizing a P5-P20EA primer, which is a P20EA adaptor primer added with a P5 sequence, R1 Seq Primer sequence, and Index2 sequence, and P7-CA3 or P7-CB3, which is a TCRα chain or β chain C region specific primer added with a P7 sequence, R2 Seq Primer sequence and Index1 sequence as shown in FIG. 50. Different Index1 and Index2 sequences are used to label an amplification primer to identify an amplified TCR gene amplicon from a plurality of samples. The primer sequences used are shown in Table 3-24. 10 cycles of PCR were performed, where a cycle was 30 seconds at 95° C., 30 seconds at 55° C., and one minute at 72° C.

(5. Purification of PCR Product by Electrophoresis)

An E-Gel agarose gel electrophoresis system is used for electrophoresis of an amplified PCR product. A precast gel containing a highly sensitive fluorescent staining agent is set in an electrophoresis apparatus, and 20 μL of sample per well is added to 2% agarose gel for electrophoresis. The amplicon is collected when a band of interest corresponding to 500-600 bp is eluted. The amount of DNA is measured by using a Quant-T™ PicoGreen® dsDNA Assay Kit (Invitrogen) for the collected PCR amplicon. Based on the resulting amount of DNA, a plurality of samples of equal molecular weight are mixed to perform a sequencing reaction.

(6. MiSeq Sequencing)

A MiSeq sample sheet is created. PhiX control is added in the range of 5-50% and sequencing is started with an MiSEQ sequencer in which a MiSeq Reagent Kit v.3 (600 cycle, Illumina) is installed. After about 65 hours, sequencing data can be obtained.

(Example 5 of analysis system: High throughput sequence analysis method for determining diversity and similarity of TCRα and TCRβ, repertoires to determine a potentially novel invariant TCRα chain—invariant TCR expressed by NKT cells and MAIT cells as example)

A comprehensive example is described below as an example summarizing the technologies described in Examples 1-4 of analysis system.

(Introduction)

As discussed above, high throughput sequencing techniques known as next generation sequencing (NGS) have undergone rapid advances, enabling large-scale sequence data analysis (Shendure J et al. (2008) Nat Biotechnol 26: 1135-1145; Metzker M L et al. (2010) Nat Rev Genet 11: 31-46). Several TCR repertoire analysis systems based on NGS have been developed by other researchers. However, many of the amplification techniques are based on Multiple PCR comprising a different specific primer in each variable region. For this reason, bias during PCR amplification cannot be avoided as bias is very common due to differential hybridization dynamics among variable region specific primers for different target genes. Thus, when using a Multiple PCR assay, a correction and an additional computational standardizing method are considered necessary in order to minimize PCR bias (Carlson C S et al. (2013) Nat Commun 4: 2680). Use of a single set of primers is a preferred method for accomplishing unbiased quantitative amplification of all TCR genes including unknown mutants with a highly diverse 5′ terminal of a sequence. Single strand oligonucleotide anchor ligation to the 3′ terminal of a cDNA including T4 RNA ligase (Troutt A B et al. (1992) Proc Natl Acad Sci USA 89: 9823-9825), cDNA homopolymer tailing, 5′ rapid amplification of cDNA ends (RACE) (Frohman M A et al. (1988) Proc Natl Acad Sci USA 85: 8998-9002), and template switching PCR (TS-PCR or SMART PCR) (Zhu Y Y et al. (2001) Biotechniques 30: 892-897) have been used to analyze TCR repertoires (Freeman J D et al. (2009) Genome Res 19: 1817-1824; Warren R L et al. (2011) Genome Res 21: 790-797). TS-PCR is simple and convenient, but a TS primer is either non-specifically annealed to a random region of RNA or repeated added. Thus, a high level of background amplification occurs (Alon S et al. (2011) Genome Res 21: 1506-1511; Kapteyn J (2010) BMC Genomics 11: 413). In this regard, present specification describes an adaptor-ligation mediated PCR (first reported by Tsuruta et al (Tsuruta Y et al. (1993) J Immunol Methods 161: 7-21; Tsuruta Y et al. (1994) J Immunol Methods 169: 17-23)) developed by an addition of an adaptor to the 5′ terminal of a double stranded complementary DNA derived from a subsequent PCR amplicon by a constant region specific primer, adaptor primer, and TCR transcript. Adaptor-ligation to a blunted double stranded complementary DNA is barely affected by the specific sequence of a cDNA, while the efficiency of 5′ adaptor-ligation using a T4 RNA ligase is sequence dependent (Jayaprakash A D et al. (2011) Nucleic Acids Res 39: e141). Furthermore, ligation of double stranded DNAs using a T4 ligase is more efficient than ssDNA ligation using a T4 RNA ligase in ligation anchored PCR (LA-PCR). Thus, such an unbiased AL-PCR allows accurate analysis of a TCR repertoire without requiring correction or standardization.

Various sequencing techniques have been developed such as Roche 454 (San Francisco, Calif.), Illumina (San Diego, Calif.), Ion-Torrent (Life Technologies, Grand Island, N.Y.), SOLiD (Life Technologies), Helicos (Cambridge, Mass.) and PacBio (Menlo Park, Calif.). Among these NGS platforms, 454 DNA sequencing creates sequence reads in the range of 50-600 base pairs (bp) or more and sufficient read output, but the number of reads per one sequencing is less than Illumina. Long read sequencing enables determination of the full length or mostly full-length of a TCR gene comprising V, D, J, and C regions. Furthermore, a recombinant TCR protein is readily generated by subsequent PCR cloning of a TCR gene. Thus, the inventors applied the adaptor-ligation mediated PCR to NGS using 454 DNA sequencing.

Natural killer T (NKT) cell are a population of separate T cells having an important role in natural immunity and acquired immunity. NKT cells regulate a wide range of immune responses such as autoimmune diseases, tumor surveillance, and host defense against a pathogenic infection. NKT cells express invariant TCRα consisting of Vα24 and Jα18 recognizing glycolipids presented by CD1d and nonclassical major histocompatibility complex class I associated protein (Godfrey D I et al. (2004) J Clin Invest 114: 1379-1388). Recently, mucosal-associated invariant T (MAIT) cells preferentially present in mucous membrane tissue have been shown to be the only T cell population expressing semi-invariant TCRα consisting of Vα7.2 and Jα33. MAIT cells recognize a microorganism vitamin B metabolite presented by MHC associated protein 1 (MR1) and nonclassical MHC class I molecules (Kjer-Nielsen L et al. (2012) Nature 491: 717-723). These T cell populations having invariant TCRα serve a central role in immunomodulation. However, it is still unidentified whether all invariant TCRα is expressed by the only T cell populations.

In the present study, the inventors used TCR repertoire analysis based on NGS that has been newly developed to perform NGS sequencing of a TCR transcript from 20 healthy individuals. First, use of a variable region and joining region was tested based on the number of sequence reads, and then clonality and diversity in TCRα and TCRβ, genes were analyzed. Unique read sequences identified by using an independently developed gene analysis program were compared at the clone level among healthy individuals. The results showed diversity in T cells to a similar extent and similar use of TRV and TRJ among individuals. Interestingly, a TCRβ, read was not shared among individuals, while a TCRα read contained a public sequence overlapping in 2 or more individuals at a high frequency. A public TCRα read contained a high percentage of invariant TCRα, indicating the presence of iNKT or MAIT cells.

In the present Example, the inventors show from NGS data that analysis of a TCR gene shared among a plurality of individuals can provide significant information in invariant TCRs expressed by NKT cells and MAIT cells.

(Demonstration in the Present Example)

High throughput sequencing of T cell receptor (TCR) genes can be a potent tool for analyzing clonality and diversity of T lymphocytes and antigen specificity. In this regard, the inventors have developed a novel TCR repertoire analysis method using 454 DNA sequencing technique combined with adaptor-ligation mediated polymerase chain reaction (PCR). This method enables amplification of all TCR genes in a truly unbiased manner, contrary to a level of pseudo-bias that can be accomplished with SMART PCR, without any bias that generally occurs in PCR.

In the present Example, the inventors have performed next generation sequencing (NGS) on TCRα and TCRβ genes in peripheral blood mononuclear cells from 20 health individuals to compare diversity and similarity of expressed TCR repertoires and use of genes among individuals. 149,216 unique reads were identified from a total of 267,037 sequence reads from the 20 healthy individuals. Preferential use of some V genes and J genes was observed, while some recombination in TRAV and TRAJ appeared to be limited. The level of observed TCR diversity differed significantly between TCRα and TCRβ, while a TCRα repertoire was more similar among individuals than a TCR repertoire. The similarity among individuals of TCRα was greatly dependent on the presence of a public TCR shared among 2 or more individuals at a high frequency. Publicly available TCRα has a TCR near a germ line having a shorter CDR3. A public TCRα sequence, especially a sequence shared among many individuals, often contained invariant TCRα derived from a mucosal-associated invariant T cell and an invariant natural killer T cell. The results suggest that search for a public TCR by NGS is useful in identifying a potentially novel invariant TCRα chain. This NGS method was found to be capable of highly precise comprehensive analysis of a TCR repertoire at a clone level.

(Materials and Methods)

Isolation of RNA extract and peripheral blood mononuclear cells

Whole blood was collected form 20 healthy individuals after obtaining informed consent. The present study was approved by the ethics committee of Clinical Research Center for Allergy and Rheumatology, National Hospital Organization, Sagamihara National Hospital. 10 mL of whole blood was collected in a heparin treated tube. Peripheral blood mononuclear cells (PBMC) were separated by Ficoll-Paque PLUS™ (GE Healthcare Health Sciences, Uppsala, Sweden) density gradient centrifugation, and washed with phosphate buffered saline (PBS). The number of cells was counted and 1×106 cells were used in RNA extraction. Total RNA was isolated and purified by using an RNeasy Lipid Tissue Mini Kit (Qiagen, Hilden, Germany) in accordance with the manufacture's manual. The amount of RNA and purity were measured by using an Agilent 2100 bioanalyzer (Agilent Technologies, Palo Alto, Calif.).

Unbiased Amplification of TCR Gene

1 μg of total RNA was converted to a complementary DNA (cDNA) by using a Superscript III reverse transcriptase (Invitrogen, Carlsbad, Calif.). A BSL-18E primer comprising poly18 and NotI site was used for cDNA synthesis. After cDNA synthesis, a double strand (ds)-cDNA was synthesized by using E. coli DNA polymerase I (Invitrogen), E. coli DNA ligase (Invitrogen), and RNase H (Invitrogen). A ds-cDNA was blunted by using T4 DNA polymerase (Invitrogen). A P10EA/P20EA adaptor was linked to the 5′ terminal of the ds-cDNA and then cleaved with a NotI restriction enzyme. After removing the adaptor and primer by using a MinElute Reaction Cleanup kit (Qiagen), PCR was performed by using either a TCRα chain constant region specific primer (CA1) or TCR chain constant region specific primer (CB1) and P20EA (Table 4-1). The PCR conditions were as follows: 20 cycles of 95° C. (30 seconds), 55° C. (30 seconds), and 72° C. (one minute). A 2nd PCR was performed by using the same PCR conditions with either CA2 or CB2 and a P20EA primer.

The following MID Tag sequences were used for identifying a sample:

MID1 (SEQ ID NO: 1325), MID2 (SEQ ID NO: 1326), MID3 (SEQ ID NO: 1327), MID4 (SEQ ID NO: 1328), MIDS (SEQ ID NO: 1329), MID6 (SEQ ID NO: 1330), MID7 (SEQ ID NO: 1331), MID8 (SEQ ID NO: 1332), MID10 (SEQ ID NO: 1334), MID11 (SEQ ID NO: 1335), MID15 (SEQ ID NO: 1339), MID16 (SEQ ID NO: 1340), MID17 (SEQ ID NO: 1341), MID18 (SEQ ID NO: 1342), MID19 (SEQ ID NO: 1343), MID20 (SEQ ID NO: 1344), MID21 (SEQ ID NO: 1345), MID22 (SEQ ID NO: 1346), MID23 (SEQ ID NO: 1347), MID24 (SEQ ID NO: 1348)

Sequencing of Amplicon by Roche 454 Sequencing System

An amplicon for NGS was prepared by amplification of a 2nd PCR product using a P20EA primer and a fusion tag primer (Table 4-1). A fusion tag primer comprising an adaptor A sequence (CCATCTCATCCCTGCGTGTCTCCGAC) (SEQ ID NO: 39), key of 4 base sequences (TCAG), molecule identification (MID) tag sequence (10 nucleotides), and TCR constant region specific sequence were designed in accordance with the manufacturer's manual. After PCR amplification, an amplicon was separated and assessed by agarose gel electrophoresis. A resulting fragment (about 600 bp) was removed from the gel and purified by using a QIAEX II gel extraction kit (Qiagen). The amount of purified amplicon was quantified by a Quant-iT™ PicoGreen® dsDNA Assay Kit (Life Technologies, Carlsbad, Calif.). Each amplicon obtained by using a different fusion tag primer from 10 healthy individuals was mixed at equimolar concentration. Emulsion PCR (emPCR) was performed in accordance with the manufacturer's manual by using an amplicon mixture with a GS Junior Titanium emPCR Lib-L kit (Roche 454 Life Sciences, Branford, Conn.).

Assignment of TRV and TRJ Segments

All read sequences were classified with a MID Tag sequence. Artificially added sequences (tag, adaptor, and key) and sequences with a low quality score were removed from both ends of the read sequences by using a software installed on the 454 sequencing system. The remaining sequences were used in assignment of TRAV and TRAJ for a TCR.alpha. sequence and assignment of TRBV and TRBJ for a TCR.beta. sequence. The sequences were assigned by sequencing using the highest identity in a data set of reference sequences for 54 TRAV, 61 TRAJ, 65 TRBV and 14 TRBJ genes including pseudogenes and a data set of Open Reading Frame (ORF) reference sequences available from ImMunoGeneTics information System® (IMGT) database (http colon//www dot imgt dot org). Data processing, assignment, and data accumulation were automatically performed by using a repertoire analysis software (Repertoire Genesis, RG) developed independently by the inventors. RG executed BLATN, automatic accumulation program, graphic program for use of TRV and TRJ, and program for sequence homology search using a CDR3 chain length distribution. Sequence homology between a query sequence and entry sequence at a nucleotide level was automatically calculated. Parameters that increased sensitivity and precision (E value threshold, minimum kernel, high score segment pair (HSP) score) were carefully optimized for each repertoire analysis.

Data Analysis

A nucleotide sequence of CDR3 in the range from the conserved cysteine at position 104 (Cys 104) (named by IMGT) to conserved phenylalanine at position 118 (Phe118) and the subsequent glycine (Gly119) were translated into an estimated amino acid sequence. A unique sequence read (USR) was defined as a sequence read without identity in an estimated amino acid sequence of CDR3 comprising TRV, TRJ and other sequence reads. The number of copies of the same USR was automatically counted by the RG software in each sample, and ranking was then assigned in the order of number of copies. The percentage of frequency of appearance of sequence reads comprising TRAV, TRAJ, TRBV and TRBJ genes in all sequence reads was calculated.

Search for USR Shared Among Samples

In order to search for a sequence shared among samples, a character string of “TRV gene name”_“estimated amino acid sequence of CDR3 region”_“TRJ gene name” (e.g., TRBV1_CASTRVVJFG_TRBJ2-5) of a USR of an individual was used as a TCR identifier. A TCR identifier in a sample was searched in a read data set from all other samples.

Diversity Index and Similarity Index

In order to estimate TCR diversity in a deep sequence data, several diversity indices, Simpson's indices and Shannon-Weaver's indices were calculated by using the function “diversity” of a vegan package in the R program. The indices were calculated based on the number of types per sample and number of individuals per sample as the scale of ecological biological diversity. USR and the number of copies were used for types and individuals, respectively, in the deep sequence data. The Simpson's index (1−λ) was defined as the following:

\(\begin{matrix}
{{1 - \lambda} = {1 - {\sum\limits_{i = 1}^{S}\left( \frac{n_{i}\left( {n_{i} - 1} \right)}{N\left( {N - 1} \right)} \right)}}} & \left\lbrack {{Numeral}\mspace{14mu} 4\text{-}1} \right\rbrack
\end{matrix}\)

(wherein N is the total number of sequence reads, ni is the number of copies of the ith USR, and S is the number of types of USR). The value ranges from 0-1, where the maximum number 1 means a high level of diversity and 0 means low diversity. The inverse Simpson's index (1/λ) was also calculated as the inverse of λ. The Shannon-Weaver's index (H′) was used as a diversity index and defined as follows:

\(\begin{matrix}
{H^{\prime} = {- {\sum\limits_{i = 1}^{S}{\frac{n_{i}}{N}\ln\frac{n_{i}}{N}}}}} & \left\lbrack {{Numeral}\mspace{14mu} 4\text{-}2} \right\rbrack
\end{matrix}\)

(wherein N is the total number of sequence reads, ni is the number of copies of the ith USR, and S is the number of types of USR). These diversity indices should be biased due to the difference in the number of reads among samples. Thus, the number of sequence reads was standardized for each sample to the minimum number of sequence reads (Venturi V et al. (2007) J Immunol Methods 321: 182-195). To standardize the sample size, random sampling was repeated 1000 times without replacement to calculate a diversity index by using an R program. The median value of the indices was used to determine the diversity index for a sample.

To estimate similarity of TCR repertoires among healthy individuals, a Morisita-Horn index (CH) was defined as follows:

\(\begin{matrix}
{C_{H} = \frac{2{\sum_{i = 1}^{S}{x_{i}y_{i}}}}{\left( {\frac{\sum_{i = 1}^{S}x_{i}^{2}}{X^{2}} + \frac{\sum_{i = 1}^{S}y_{i}^{2}}{Y^{2}}} \right){XY}}} & \left\lbrack {{Numeral}\mspace{14mu} 4\text{-}3} \right\rbrack
\end{matrix}\)

(wherein xi is the number of the ith USR in all X reads of a single sample, yi is the number of the ith USR in all Y reads in another sample, S is the number of USRs). To standardize the sample size, random sampling was repeated 1000 times without replacement to calculate a similarity index by using an R program (Venturi V et al. (2008) J Immunol Methods 329: 67-80). The median value was used for a similarity index between a pair of samples.

Statistics

Statistical significance was tested by a nonparametric Mann-Whitney U test by using the GraphPad Prism software (version 4.0, San Diego, Calif.). A value of p<0.05 is considered statistically significant.

(Results)

Repertoire Analysis Software

The cloud-based software platform RG that was developed in the present study is a high-speed, accurate and convenient computational system for TCR repertoire analysis. RG provides a consolidated software package for (1) assignment of V, D, and J segments, (2) calculation of sequence identity, (3) extraction of a CDR3 sequence, (4) counting of identical reads, (5) amino acid translation, (6) frame analysis (stop and frame shift), and (7) analysis of CDR3 length. After uploading sequencing data from an NGS sequencer, V, D, and J segments can be identified based on sequence similarity thereof by using optimized parameters. The number of reads is automatically aggregated, and subsequently processed data, tabulation chart, and graphs can be readily downloaded.

Number of Reads, Error Rates, and Nonproductive Reads

The present inventors performed high throughput sequencing on TCRα and TCRβ genes in PBMCs derived from 20 healthy individuals. A total of 172,109 and 91,234 sequence reads were assigned to TCRα and TCRβ repertoire analysis, respectively, by using the RG program (Tables 4-2 and 4-3).

Total of 94,928 and 57,982 unique sequence reads (USR) were identified in TCRα and TCRβ, respectively. The number of nucleotide sequences per read obtained by Roche 454 sequencing was a length of about 400 bp (mean bp length±SD, TCRα: 407.4±35.4, TCRβ: 409.4±37.8), showing that the sequences are of sufficient length to identify a TCR gene in the range from V to J regions. To assess the precision and quality of NGS sequencing, the inventors have calculated the frequency of mismatching nucleotides between a query sequence and a reference sequence as the error rate. The error rate was 0.72±0.18% for TRAV, 0.54±0.08% for TRAJ, 0.70±0.15% for TRBV, and 0.50±0.12% for TRBJ (Table 4-4).

The error rates were slightly lower than the mean error rate of 1.07% for the 454-sequence reported in a previous study (Gilles A et al. (2011) BMC Genomics 12: 245). The error rate was significantly higher in a V region than in a J region (AV vs. AJ: p<0.05, BV vs. BJ: p<0.0001). Higher sequence reliability was exhibited in a region closed to a sequencing primer. The frequency of read frame shift (out-of-frame) in a CDR3 region or a nonproductive read having a stop codon was calculated (Table 4-5).

There was no significant difference in the percentage of frequency of a nonproductive unique sequence read between TCRα and TCRβ (31.2±7.0% vs. 29.3±7.9%, P=0.31).

Expression of TCR Gene Comprising ORF and Pseudogene

To determine the use of TRV and TRJ genes in a TCR sequencing read, the number of copies of USR having TRV or TRJ (number of reads) was each counted. Individual USRs were ranked in the order of number of copies. The frequency percentage of each of TRV and TRJ was calculated (Figures and FIG. 52). For a TCRα repertoire, 8 pseudogenes (AV8-5, AV11, AV15, AV28, AV31, AV32, AV33 and AV37) were not expressed in healthy individuals. AV8-7 classified as an ORF (defined based on a change in regulatory element, recombinant signal and/or splicing site by IMGT) was hardly expressed (43 reads in 11 out of 20 individuals). Expression of AV18 and AV36 (classified as a functional gene) was not observed in healthy individuals. Furthermore, functional genes AV7 and AV9-1 were not sufficiently expressed in one individual (9 reads) and 2 individuals (3 reads). Expression of AJ35 and AJ58 among 8 AJ genes classified as ORF genes (AJ1, AJ2, AJ19, AJ25, AJ35, AJ58 and AJ61) was observed in all 20 individuals. AJ25 and AJ61 thereamong were expressed slightly in 3 individuals (21 reads) and 7 individuals (35 reads), respectively. AJ1, AJ2, AJ19 and AJ59 were not present in any individual. Expression of three pseudogenes AJ51, AJ55 and AJ60 was not present in any individual. Only 3 reads of the functional gene AJ14 were detected from 3 individuals.

For a TCR gene, there was no expression of 11 pseudogenes (BV1, BV3-2, BV5-2, BV7-5, BV8-1, BV8-2, BV12-1, BV12-2, BV21-1, BV22-1, and BV26) in healthy individuals. Among 5 ORF genes, BV5-7 (32 reads in 13 individuals), BV6-7 (13 reads in 8 individuals), and BV17 (3 reads in 1 individuals) were not sufficiently expressed. A BV7-1 ORF gene was not observed in any individual, while BV23-1 was expressed in all 20 individuals. For a BJ gene, there was no expression of a BJ2-2P pseudogene.

Recombination of TRAV and TRAJ at a Low Frequency

Genetic recombination with 41 TRAV and 50 TRAJ (excluding pseudogenes, ORF and genes that are not sufficiently expressed) can generate a total of 2050 AV-AJ recombinations (FIG. 53). Among them, 1969 AV-AJ recombinations (96.0%) were detected in 20 individuals. This indicates that almost all AV-AJ recombinations were used in TCR transcripts without limitation. In particular, AV1-1 to AV6 genes could not be preferentially recombined with AJ50 to AJ58. Similarly, recombination of AV35 to AV41 genes with AJ3 to AJ16 was hardly observed. Considering the position of these gene segments on a chromosome, the results indicate that an AV-AJ recombination hardly occurs between a proximal AV gene and a distal AJ gene and between a distal AV gene and a proximal AJ gene.

For TCRβ, 650 gene recombinations are generated by 50 BV (excluding 11 pseudogenes and 5 ORF) and 13 BJ genes (excluding pseudogenes). 605 BV-BJ (93.1%) thereamong were used in 30 individuals. There was no limit for combination of TRBV with TRBJ.

Preferential Use of TRV and TRJ Repertoires in Healthy Individuals

To elucidate the use of TRV and TRJ in all TRC transcripts, the frequency of USR having TRV or TRJ was each calculated (FIG. 51 and FIG. 52). Preferential use in some TRAV genes was similar to previous results (6) obtained by using a quantitative assay based on hybridization. Some TRBV genes were used more in a TRBV repertoire. The top 3, TRAV9-2 (BV4S1 by Arden), TRBV20-1 (BV2S1) and TRBV28 (BV3S1), accounted for 1/3 of all sequence reads. This was similar to the result (6) obtained in a previous study by the inventors using a microplate hybridization assay. Use of a gene significantly varied among TRBJ genes. TRBJ2-1 and TRBJ2-7 were very highly expressed, while expression of TRBJ1-3, TRBJ1-4, TRBJ1-6, TRBJ2-4 and TRBJ2-6 was low.

3 Dimensional (3D) View of the Use of TCR Repertoire

To visualize the use of TCRs having a combination of a TRV gene and a TRJ gene, the inventors made a 3D portrayal of a TCR repertoire (FIG. 54 and FIG. 55). The advantage of a 3D image is that the level of diversity of TCRs and predominance of a specific combination of a TRV gene and a TRJ gene can be readily observed. For TCRβ, there was hardly any preferential use of recombination between a TRVB gene and a TRBJ gene. The frequency of each recombination was dependent on the use of TRBV or TRBJ. BV29-1/BJ2-7, BV29-1/BJ2-1, BV29-1/BJ2-3 and BV20-1/BJ2-7 were used at a high frequency in all combinations, while others were expressed at a low frequency. In contrast, 3D imaging of a TCRα repertoire showed expression of TRAV and TRAJ at a low level in a wide distribution. The share was less than 1% in all combinations. Notably, TCR reads having AV1-2 and AJ33 were highly expressed in all healthy individuals (mean±SD: 0.99±0.85).

Digital CDR3 Chain Length Distribution

CDR3 chain length distribution analysis called CDR3 size spectratyping (Yassai M et al. (2000) J Immunol 165: 3706-3712; Yassai M et al. (2002) J Immunol 168: 3801-3807) or immunoscope analysis (Pannetier C et al. (1993) Proc Natl Acad Sci USA 90: 4319-4323; Pannetier C et al. (1995) Immunol Today 16: 176-181) was efficiently used to estimate the diversity of a TCR repertoire. The technique is based on actual peak distribution of PCR amplicons comprising a CDR3 sequence with gel electrophoresis. In the present study, a determined nucleotide sequence length of a TCR in the range from the conserved Cys 104 (named by IMGT) to conserved phenylalanine at position 118 (Phe118) was automatically calculated. This provides a visibly simple method for estimating the diversity and clonality of a TCR by using NGS data. RG can generate a diagram representing a digital CDR3 chain length distribution for each V region. The CDR3 chain length distribution of both TCRα and TCRβ was similar to a common distribution, but was not necessarily completely symmetric (FIG. 56). The CDR3 chain length is shorter in TCRα than in TCR (mean±SD: 41.2±8.3 vs. 42.8±6.1). TCRα has a positive skewness relative to TCR (skewness index: 11.1 vs 5.41), indicating that the distribution in TCRα was concentrated on the left side. Furthermore, TCRα had a positive kurtosis relative to TCRβ, indicating high kurtosis in TCRα (kurtosis index: 282.4 to 176.7).

Diversity of TCRα and TCRβ Repertoires

To show the diversity of a TCR repertoire, the inventors calculated the diversity index (Simpson's index, Shannon-Weaver's index or the like) and the average number of copies of USRs (FIG. 57). The average number of copies of USRs significantly differed between TCRα and TCRβ (2.0±0.72 to 1.70±0.57). Furthermore, there was no significant difference in the inverse Simpson's index (D) or Shannon-Weaver's index (H) between TCRα and TCRβ (D: 710.3±433.0 to 729.7±493.9, H: 7.02±0.33 vs 6.97±0.43). The results show that there is no difference in immunodiversity for TCRα and TCRβ in healthy individuals.

Similarity of TCRα and TCRβ repertoires among healthy individuals

To elucidate the correlation of use of genes among individuals, the frequency percentages of each of TRV and TRJ were plotted for all individual pairs by a scatter plot (FIG. 60). The Spearman's correlation coefficient between each pair was calculated. A matching correlation coefficient was lower in TRAV than in TRBV (mean±SD, 0.86±0.059 for TRAV, 0.89±0.038 for TRBV, p<0.001), and lower in TRAJ than in TRBJ (0.74±0.095 for TRAJ, 0.91±0.063 for TRBJ, p<0.001). The results show that the expression level of TRV and TRJ among healthy individuals was more similar among individuals in TCR relative to TCRα.

To assess the potential similarity of TCR repertoires at a clone level among healthy individuals, the inventors searched for a TCR sequence read shared among individuals. The number of TCR reads shared was counted for all pairs of individuals to calculate the frequency thereof (Table 4-6 and Table 4-7).

The mean frequency was significantly higher in TCRα relative to TCRβ, (0.76±0.52 vs 0.040±0.057, n=380, p<0.001) (FIG. 58), indicating that a TCRα repertoire comprises more shared TCR reads among individuals than TCRβ. The Morisita-Horn index, which is a similar index, was significantly larger for TCRα than for TCRβ (0.0058±0.0069 vs 0.000096±0.00029, n=190, P<0.001). The results clearly show that a TCRα repertoire was more similar among healthy individuals relative to a TCRβ repertoire.

TCR Sequence Shared Among Healthy Individuals

A small number of TCR sequences are shared among different healthy individuals. The shared TCR thereof are called public TCRs. In contrast, most TCRs were specific to each healthy individual (private TCR). To identify a public TCR sequence in 20 healthy individuals, the inventors searched for a TCRα read and a TCRβ read shared among two or more healthy individuals. 3,041 public TCRα and 206 public TCRβ sequences were obtained from 90,643 and 57,982 USRs, respectively, in 20 healthy individuals (Table 4-8).

Public TCRα was higher in frequency than TCR in peripheral blood lymphocytes (PBL) derived from a healthy individual. A public TCR sequence was obtained from 2-4 individuals, while a public TCRα sequence was observed in 16 individuals. The results shows that a TCRα public sequence is used more commonly in individuals, but a TCR repertoire was more specific to each individual. Furthermore, the frequency per individual of a TCR sequence shared between a pair of individuals was significantly higher for TCRα (7.9%) than for TCR (0.7%). To characterize a public TCRα sequence, the inventors compared the length of CDR3 between public and private TCRα sequences and observed that public TCRα had CDR3 with a shorter chain length than private TCRα (median value: 39 vs 42) (FIG. 59).

A TCR shared by a plurality of individuals comprises an invariant TCRα chain at a high frequency.

Public TCRα was observed at a high frequency in PBLs derived from a healthy individual. To determine the origin of public TCRα, the inventors examined the CDR3 sequence of public TCRα that was previous reported. Interestingly, a public TCRα sequence shared by a plurality of individuals comprised invariant TCRα at a high percentage, indicating an iNKT cell or MAIT cell (Table 4-9).

It is reported that MAIT cells express TRAV1-2 and TRAJ33, while iNKTs express TRAV10 and TRAJ18. Many public TCRα used TRAV1-2 and TRAJ33 comprising a different CDR3 sequence. The total frequency percentages of MAIT TCRα having TRAV1-2 and TRAJ33 and iNKT TCRα having TRAV10 and TRAJ18 was 0.82±0.72% and 0.15±0.41% per individual, respectively. Among 55 public TCRα sequences, 17 (31%) MAIT and 1 (1.8%) iNKT sequences were observed in 6 or more individuals (FIG. 53). The percentage increased with the number of overlapping individuals. A germ line-like CDR3 sequence without an amino acid sequence modified from a germ line sequence was observed in 27 out of 38 public TCRα (71%) excluding MAIT (TRAV1-2-TRAJ33) and NKT (TRAV10-TRAJ18).

(Discussion)

A high throughput sequencing technique has made a great leap by the development of a wide variety of NGS platforms. Although NGS promotes acquisition of an enormous amount of sequence data, it still requires PCR amplification or gene enrichment of a sequence gene of interest instead of the entire genome or gene library. Multiplex PCR with many gene specific primers have been widely used for non-homogeneous TCR or BCR genes generated by rearrangement of many gene segments. However, amplification bias between each gene occurs from using a plurality of primers to prevent accurate estimation of gene frequency. In this regard, the inventors used adaptor-ligation mediated PCR, which is an unbiased PCR technique, for TCR repertoire analysis based on NGS. This method uses a single set of primers and theoretically enables amplification of all TCR genes without applying PCR bias. Thus, this method is optimal for accurately estimating the amount of each TCR gene present from a wide range of samples.

The inventors comprehensively investigated TCRα and TCRβ repertoires derived from many individuals (n=20) at a clone level to assess a large amount of sequence data (total of 149216 unique sequence reads from 267037 sequence reads). Thus, the present study elucidated the level of diversity and similarity of a TCR repertoire in a healthy individual and gene use in a normal range. Compared to an Illumina NGS platform (Freeman J D et al. (2009) Genome Res 19: 1817-1824; Warren R L et al. (2011) Genome Res 21: 790-797; Robins H S et al. (2009) Blood 114: 4099-4107), there are not as many sample sequence reads, but the reads are longer and higher quality. The depth of different sequences in a CDR3 contig generated from many shotgun reads by using an Illumina platform can make it difficult to determine a frequency of a TCR clone type. However, it had a long sequence (mean of about 400 bp, Table 4-2 and Table 4-3) that covers all regions of CDR3, V and J and determines all TCR sequences from a single read. Direct analysis from a read sequence that does not use conjugation is highly likely to reflect the actual frequency of a TCR clone type accurately. The error rate in a TCR sequence was slightly lower than a previous report exhibiting a mean error rate of 1.07% for a 454-sequence and a high level of precision and quality was exhibited regardless of nested PCR. Furthermore, RG, the assignment and aggregation software, can quickly aggregate the use of TRV and TRJ and use of recombinations. The integrated analysis facilitates the detection of preferential use of a predetermined TRV and/or TRJ and is thus useful for researching an immune response by an antigen-specific T cell.

Unlike the widely-used multiplex PCR that typically requires compensation for PCR bias (Carlson C S et al. (2013) Nat Commun 4: 2680), AL-PCR accurately estimates a TCR repertoire without bias. A high level of expression of TRBV18 (BV18S1, named by Arden), TRBV19 (BV17S1) and TRBV7-9 (BV6S5) and a low level of expression of TRBV20-1 (BV2S1), TRBV28 (BV3S1), and TRBV29-1 (BV4S1) are reported by multiplex PCR in CD4+ and CD8+ cells (Emerson R et al. (2013) J Immunol Methods 391: 14-21). However, flow cytometry analysis showed that a large amount of TRBV20 and TRBV29 was expressed in PBL (van den Beemd R et al. (2000) Cytometry 40: 336-345; Pilch H et al. (2002) Clin Diagn LabImmunol 9: 257-266; Tzifi F (2013) BMC Immunol 14: 33). The results by the researchers for a TCR repertoire are similar to previous reports (Li S et al. (2013) Nat Commun 4: 2333). Thus, this method provides a direct, accurate, and reliable TCR repertoire result.

Use of recombination showed recombination of AJ-proximal 3′ AV segment with AV-distal 3′ AJ segment at a low frequency and recombination of AJ-proximal 5′ AV segment with AV-distal 5′ AJ segment at a low frequency. In gene rearrangement of a TCRαδ gene locus, activation of TCRα enhancer (Eα) and T early alpha (TEA) promoter initiates the first rearrangement of proximal TRAV and TRAJ segments. Subsequent second rearrangement occurs by using 5′ proximal TRAV and distal 3′ TRAJ genes (Huang C et al. (2001) J Immunol 166: 2597-2601; Krangel M S et al. (2004) Immunol Rev 200:224-232; Pasqual N et al. (2002) J Exp Med 196: 1163-1173; Aude-Garcia C et al. (2001) Immunogenetics 52: 224-230), resulting in restricted use of a TCRα repertoire (continuous bidirectional recombination model) (Chaumeil J et al. (2012) Embo J 31: 1627-1629). However, all TRAV genes can be recombined with a TRAJ gene in the second rearrangement by a gene locus contraction and DNA loop formation model (Genolet R et al. (2012) Embo J 31: 4247-4248). There was inefficient distal-proximal and proximal-distal recombination of TRAV-TRAJ genes, but the use of TRAJ was not limited across all TRAV and was rather equally distributed. This indicates that recombination frequency varies depending on the position of TRAV and is likely dependent on the loop forming ability between a TRAV gene locus and TRAJ gene locus.

Potential TCR diversity generated by addition/deletion of a nucleotide and recombination was estimated to be at most 1015 (Davis M M et al. (1988) Nature 334: 395-402). The diversity of TCR was estimated to be about 3-4×106 (Robins H S et al. (2009) Blood 114: 4099-4107) or about 1×106 in humans (Warren R L et al. (2011) Genome Res 21: 790-797) based on NGS. Furthermore, the diversity of TCRα is 50% of TCR in humans (Arstila T P et al. (1999) Science 286: 958-961). For mice, TCRα diversity is 0.79×104 (Pasqual N et al. (2002) J Exp Med 196: 1163-1173) or 1.18×104 (Cabaniols J P et al. (2001) J Exp Med 194: 1385-1390), indicating that it is 10 times lower than the TCRβ diversity. The low diversity of TCRα may be due to a difference in the recombination process between TCRα and TCRβ. However, the results of the inventors indicated that the level of diversity between TCRα and TCRβ is similar as assessed by the Simpson's index and Shannon-Weaver's index. Similarly, Wang et al have reported that TCR diversity was estimated to be equal between TCRα and TCRβ (0.47×106 vs 0.35 x×106) (Wang C et al. (2010) Proc Natl Acad Sci USA 107: 1518-1523; Dash P et al. (2011) J Clin Invest 121: 288-295). It is shown that in contrast to the previous report obtained by using a limited number of sequences, for large scale sequencing, the repertoire size for TCRα generated by V-J recombination is comparable to the repertoire size for TCRβ by V-D-J recombination.

Surprisingly, the inventors have found that TCRα repertoires are similar among individuals. This is mainly due to the presence of a TCR sequence shared among 2 or more individuals (public TCR). An addition and deletion of a random nucleotide mediated by terminal deoxynucleotidyl transferase occurs during TCR rearrangement, resulting in significant increase in diversity of a CDR3 region. However, a public TCR appears to have a germ line like CDR3 sequence that does not undergo such an alteration (Table 4-9). Furthermore, a public TCR comprises many TCR clone types having CDR3 with a shorter chain length. The results indicate that high frequency of public TCRα occurs possibly due to a difference in the intrinsic recombination mechanism from TCRβ (V-J vs V-D-J).

It is notable that public TCRα is present in many individuals. The inventors have unexpectedly found that public TCRα comprises invariant TCRα derived from MAIT cells or iNKT cells at a high ratio. These functionally important T cells have homogeneous TCRα and diverse TCRβ. MAIT cells express classical TCRα including TRAV1-2 (Vα7.2)-TRAJ33 (Jα33) and are preferentially located in the intestinal lamina propria (Tilloy F et al. (1999) J Exp Med 189: 1907-1921; Treiner E et al. (2003) Nature 422: 164-169). MAIT cells recognize vitamin B2 metabolites presented by a nonclassical MHC class I molecule, MR1. Furthermore, CD1d-restricted iNKT cells express an invariant TRAV10 (Vα24)-TRAJ18 (Jα18) chain and semi-invariant TRBV25-1 (Vβ11) (Godfrey D I et al. (2004) Nat Rev Immunol 4: 231-237) and recognize glycolipids such as α-galactosylceramide, self-glycolipid, or isoglobo-trihexosylceramide (Tupin E et al. (2007) Nat Rev Microbiol 5: 405-417). Both cell types play an important role in regulating immune responses to infection, tumor, autoimmune disease and tolerance induction (Godfrey D I et al. (2004) J Clin Invest 114: 1379-1388). The frequency of MAIT cells and iNKT cells obtained in this study is consistent with previous reports (showing that MAIT cells expanded to 1-4% of peripheral blood T cells (Martin E et al. (2009) PLoS Biol 7: e54) and iNKT cells accounted for 0.2% of the entire PBMCs (Lee P T et al. (2002) J Clin Invest 110: 793-800)). Interestingly, there are different types of public sequences having TRAV1-2 (e.g., TRAV1-2-TRAJ12, TRAV1-2-TRAJ20) and some public TCRα sequences other than MAIT and iNKT sequences that are well known. Thus, repertoire analysis based on NGS is useful in both estimating the frequency of MAIT cells or iNKT cells and identifying a potentially new invariant TCRα chain. Further identification and substantiation is required to identify potentially new invariant TCRα.

As discussed above, the inventors have developed a novel TCR repertoire analysis method based on NGS to find the similarity among different individuals between TCRα and TCRβ and comparable diversity therewith from the present Example. A public TCRα sequence comprises functionally significant T cell subpopulation, MAIT and iNKT cells at a high frequency. In addition, an approach to find a public TCR by NGS is useful in identifying a potentially new invariant TCRα chain. This technique with very high precision for TCR repertoire analysis was demonstrated to reveal antigen specific T cells associated with onset of a human disease and contribute to research, diagnosis and therapy of natural and acquired immunity.

### Applied Example 1: Example of Antibody Isolation: Example of Isolation of Human Form Antibody Utilizing BCR Repertoire Analysis

In this Example, an example of isolating a human form antibody utilizing BCR repertoire analysis is provided as a specific embodiment in actual application.

(Where a Reagent and the Like is Obtained)

Obtaining human form anti-idiotype antibody using humanized NOG mice

1. A monoclonal BCR derived from a tumor cell is observed to be highly expressed in a B cell based leukemia or malignant lymphoma patient.

2. A peripheral blood mononuclear cell is collected from the B cell based leukemia or malignant lymphoma patient to carry out the BCR repertoire analysis described in this section. An immunoglobulin H chain gene derived from a tumor cell is identified, which has the highest ranking and is significantly present from determined genetic sequences of several tens of thousands of reads.

3. The determined immunoglobulin H chain genetic sequence is used to estimate an amino acid sequence of a CDR3 region that is highly diverse, and a peptide that is identical with the sequence is synthesized.

4. 200 μg of synthetic peptide is mixed well with a complete Freund's adjuvant (CFA, Sigma Aldrich) and subcutaneously administered in a humanized NOG mouse with a syringe (first immunization). Similarly, PBS is administered to a control mouse. Furthermore, the same amount of antigen peptide is readministered after 2 weeks from the first immunization.

5. The lymph node or spleen is extracted from the mouse after 4 weeks from the first immunization. Tissue is thinly cut in a phosphate buffered saline (PBS, Invitrogen) and filtered with a cell strainer (0.75 μm, BD) to prepare a single cell.

6. The resulting cell is dissolved in a Trizol solution (Invitrogen). The genetic sequence is determined by the BCR repertoire analysis method described herein.

7. The resulting BCR genetic sequences of several tens of thousands of reads are sorted in the order of the frequency of presence (number of reads) to determine immunoglobulin H chain and L chain genetic sequences having a high ranking. Immunoglobulin H chain and L chain genetic sequences with a significantly high frequency of presence, relative to read ranking of the mouse administered with PBS as a control, are selected.

8. For the resulting immunoglobulin H chain and L chain genetic sequences, a P20EA adaptor primer and a C terminal primer are used for PCR amplification of full length immunoglobulin H chain and full length L chain genes. Each full length gene is inserted into a multi-cloning site in antibody expression vectors pEHX1.1 (for antibody H chain, TOYOBO) and pELX2.2 (for antibody L chain, TOYOBO) with a ligation reaction by using a Ligation Kit (TAKARA). An E. coli TOP10 cell line (One Shot′ TOP10 Chemically Competent E. coli, Invitrogen) is transformed to obtain an H chain expression plasmid and L chain expression plasmid.

9. Both plasmids are digested twice with BglII and EcoRI restriction enzymes. A BglII-EcoRI fragment of the L chain plasmid is then inserted into a BglII-EcoRI cleavage site of the H chain plasmid to obtain an antibody expressing plasmid that coexpresses the H chain and the L chain.

10. An antibody expressingn plasmid is extracted/purified from E. coli by using a QIAGEN Plasmid Mini kit and introduced into a CHO cell by using TransIT®-CHO Transfection Kit (TAKARA).

11. The transformed antibody expressing CHO cell line is cultured for expansion. The culture supernatant is collected and purified by using a Protein A agarose affinity column (HiTrap Protein A HP Columns, GE Healthcare) in accordance with the method of application.

12. After measuring the amount of obtained antibody protein with an absorption spectrometer, binding reactivity with an antigen peptide is examined by ELISA.

Ishida I, Tomizuka K, Yoshida H, Tahara T, Takahashi N, Ohguma A, Tanaka S, Umehashi M, Maeda H, Nozaki C, Halk E, Lonberg N. Production of human monoclonal and polyclonal antibodies in TransChromo animals. Cloning Stem Cells. 2002; 4(1): 91-102. Review. can be referred with regard to KM mice used in this Example. Ito M, Hiramatsu H, Kobayashi K, Suzue K, Kawahata M, Hioki K, Ueyama Y, Koyanagi Y, Sugamura K, Tsuji K, Heike T, Nakahata T. NOD/SCID/gamma(c) (null) mouse: an excellent recipient mouse model for engraftment of human cells. Blood. 2002 Nov. 1; 100(9): 3175-82. can be referred with regard to NOG mice. Jayapal K P, Wlaschin K F, Hu W-S, Yap MGS. Recombinant protein therapeutics from CHO cells-20 years and counting. Chem Eng Prog. 2007; 103:40?47.; Chusainow J, Yang Y S, Yeo J H, Toh P C, Asvadi P, Wong N S, Yap M G. A study of monoclonal antibody-producing CHO cell lines: what makes a stable high producer? Biotechnol Bioeng. 2009 Mar. 1; 102(4): 1182-96 can be referred with regard to CHO cell/antibody production.

### Applied Example 2: Cancer Idiotype Peptide Sensitization Immune Cell Therapeutic Method

The present Example provides a demonstration of an example for a cancer idiotype peptide sensitization immune cell therapeutic method using the repertoire analysis of the present invention. The procedure thereof is explained below (see FIG. 62).

(1) 10 mL of whole blood is collected from a malignant lymphoma patient. Peripheral blood mononuclear cells (PBMC) are separated by Ficoll-Paque gradient centrifugation (GE Healthcare Bioscience, 17-1440-02)

(2) Total RNA is extracted from the patient PBMCs by using a Trizol Reagent (Invitrogen).

(3) A cDNA is synthesized from RNA with a reverse transcriptase (Superscript II, Invitrogen, 18064-014), and then a dsDNA is synthesized with DNA Polymerase (Invitrogen, 18010-017), E. coli Ligase (Invitrogen, 18052-019), and RNase H (Invitrogen, 18021071). Furthermore, the terminal is blunted by T4 DNA Polymerase (Invitrogen, 18005-025). After a ligation reaction of a P20EA/P10EA adaptor with T4 ligase (Invitrogen, 15224-025) (see Preparation Example 2 and the like), the product was digested with NotI (TaKaRa, 1166A).

(4) The 1st PCR is carried out by using a P20EA adaptor (SEQ ID NO: 2 and a C region specific primer of IgM of a BCR gene (CM1 (SEQ ID NO: 5)), and the 2nd PCR is carried out by using CM2 (SEQ ID NO: 6) and a P20EA primer (SEQ ID NO: 2). 20 cycles of PCR reaction were each performed, where a cycle was 30 seconds at 95° C., 30 seconds at 55° C., and one minute at 72° C.

(5) Column purification is performed with a High Pure PCR Cleanup Micro Kit (Roche) to remove a primer from a 2nd PCR amplicon. Subsequently, PCR is performed by using a B-P20EA primer (SEQ ID NO: 4), which is a P20EA primer (SEQ ID NO: 2) added with an adaptor B sequence (SEQ ID NO: 1375), and a GS-PCR primer (see Table 1-1 for sequence information), which is an IgM C region specific primer (CM3) added with an adaptor A sequence (SEQ ID NO: 39) and identification sequence MID Tag sequence (see Table 1-6).

(6) After GS-PCR amplification, 2% agarose gel electrophoresis was carried out. A band was cut out in a size of interest (500 bp-700 bp), when visualized, and purified by using a QIAEX II Gel Extraction Kit (QIAGEN). The amount of the collected DNA was measured by using a Quant-iT™PicoGreen® dsDNA Assay Kit (Invitrogen). 10 million DNAs are used in emulsion PCR for sequence analysis by Roche's next generation sequence analyzer (GS Junior Bench Top system).

(7) A TCR/BCR repertoire analysis software that was newly developed in the present invention (Repertoire Genesis, see Analytical test examples, Analysis Examples 1-5 and the like herein) is used to assign V and J sequences and determine an estimated amino acid sequence of a CDR3 region with the obtained sequence data. At the same time, the number of copies for the same base sequence is counted to provide a ranking by frequency of appearance.

(8) The highest ranking BCR gene is determined. The number of reads of BCR thereof accounting for 10% or more of the total is confirmed to be notably high to identify said BCR gene as a BCR gene derived from tumor.

(9) An HLA binding peptide prediction program BIMAS (www dot bimas dot cit dot nih dot gov/) is used to predict an HLA-binding peptide for an estimated amino acid sequence of the tumor derived BCR gene. The default condition is used unless a particular condition is specified. The BCR amino acid sequence and patient HLA type are input into BIMAS to determine an estimated HLA binding peptide exhibiting the highest score among peptides in a CDR3 amino acid sequence or peptides comprising a part of the CDR3 amino acid sequence.

(10) A cytotoxic T cell (CTL) therapeutic method or dendritic cell (DC) vaccine therapeutic method is carried out by using an HLA-binding peptide with a high score as an individualized cancer peptide. Here, a DC vaccine therapeutic method is implemented.

(11) The individualized cancer peptide sequence is chemically synthesized by using a fully automatic peptide synthesizer (Protein Technologies, Inc.) Peptides with a yield of 1 mg or greater and purity of 95% or greater are acquired. The peptides are dissolved in 50% DMSO and stored at −20° C.

(12) A blood component collecting apparatus (Terumo apheresis apparatus AC-555) is used to separate monocytes from a cancer patient. After washing cells including the monocytes in an AIM-V medium (Invitrogen, 12055091), the number of cells is counted.

(13) After removing cells that did not adhere to a plastic plate, the cells were cultured for about 1 week in an AIM-V medium comprising 2000 U/mL of granulocyte macrophage colony stimulating factors (GM-CSF, Wako Pure Chemical) and 400 U/mL of interleukin-4 (IL-4, Petrotech) and are induced to differentiate into dendritic cells (DC).

(14) Differentiation into DCs is confirmed by examining expression of MHC class I & II molecules, CD40, CD80 or CD86 by using FACS analysis. 2×106 cells are then added with 20 μg/mL of individualized cancer peptide and further cultured for a day with stimulating factors (Picibanil (OK-432), Picibanil Injection 0.5KE, Chugai Pharmaceutical) in an AIM-V medium (same as above).

(15) Peptide-stimulated DC cells are collected and washed with saline, and then intravenously injected into the cancer patient by intravenous drip.

(Results)

The following is accomplished by the present Example.

(1) Next generation BCR repertoire analysis on peripheral blood of a malignant lymphoma patient identifies one type of IgM immunoglobulin heavy chain and one type of IgM immunoglobulin light chain accounting for 50% or more of all BCR reads.

(2) A CDR3 region of these immunoglobulin genes is identified by a Repertoire Genesis program.

(3) The patient HLA type (e.g., HLA-A #02) and IgM immunoglobulin heavy chain CDR3 amino acid sequence are input into a BIMAS program. A peptide sequence exhibiting the highest bond score is selected.

(4) This peptide is chemically synthesized as an individualized cancer peptide with a fully automatic peptide synthesizer, and the peptide stimulates and activates DCs from a patient in vitro.

(5) Individualized peptide-stimulated DC cells are intravenously introduced into the patient, whereby reduction in the number of tumor cells and improvement in clinical symptom can be observed.

### (Discussion)

It is understood from the present Example that the present invention accomplishes the following effect.

(1) A BCR sequence derived from the patient's cancer cells can be used to make an individualized cancer peptide for therapy. A DC therapeutic method or CTL therapeutic method can be administered to a wide range of patients, regardless of HLA type or antigen expression.

(2) Since a peptide adapted to the HLA of the patient is used, it is possible to materialize an effective DC therapeutic method of CTL therapeutic method that is more adapted to the patient and highly specific to cancer cells.

(3) Since an antigen peptide can be chemically synthesized directly from a genetic sequence obtained from BCR analysis, it is highly safe and requires no antigen identification or the like.

### Applied Example 3: Improved CTL Method

The present Example demonstrates an example with an improved CTL method using repertoire analysis of the present invention. The procedure thereof is explained below (see FIG. 63).

(1) A cancer idiotype peptide is identified by the method of (1)-(9) in Applied Example 2.

(2) An existing cancer peptide (NY-ESO-1 peptide) or a cancer idiotype peptide (peptide identified in (1)) is chemically synthesized by using a fully automatic peptide synthesizer (Protein Technologies, Inc.) Peptides with a yield of 1 mg or greater and purity of 95% or greater are acquired. The peptides are dissolved in 50% DMSO and stored at −20° C.

(3) 20 mL of peripheral blood is collected from the cancer patient. Peripheral blood mononuclear cells (PBMC) are separated by Ficoll-Paque gradient centrifugation (see Applied Example 2).

(4) CD8+ T cells are separated by using a CD8+ T cell separation magnetic bead (Miltenyi Biotech) or flow cytometry apparatus (FACS Aria II, Beckton Dickinson).

(5) Monocytes separated by a blood component collecting apparatus (Terumo apheresis apparatus AC-555) or PBMCs are cultured in a culture plate (100 mm dish, Corning, 353003) and non-adhering cells are removed.

(6) The adhering monocytes are cultured for about 1 week in an AIM-V medium (same as Applied Example 2) comprising 2000 U/mL of granulocyte macrophage colony stimulating factors (GM-CSF, Wako Pure Chemical) and 400 U/mL of interleukin-4 (IL-4, Petrotech) and are induced to differentiate into dendritic cells (DC).

(7) After differentiation into DCs is confirmed, 2×106 cells are added with 20 μg/mL of peptide (“estimated HLA binding peptide exhibiting the highest score” in Applied

Example 2) and further cultured for a day with stimulating factors (Picibanil (OK-432), Picibanil Injection 0.5KE, Chugai Pharmaceutical) in an AIM-V medium.

(8) Furthermore, the DC culture solution is stimulated and cultured with 20 μg/mL of synthetic peptide (“estimated HLA binding peptide exhibiting the highest score” in Applied Example 2) and 2×106/mL of CD8+ T cells separated in the above-described (3) and AIM-V medium (see Applied Example 2 and the like).

(9) After CD8+ T cells proliferated by antigen stimulation are separated from DCs adhering to a plastic culture plate (100 mm dish, Corning, 353003) (same as the plate in (5)), the cells are expanded and cultured in the presence of 5 μg/mL of antiCD3 antibody (OKT3, Orthoclone OKT3, Janssen Pharmaceutical) and 200 U/mL of interleukin 2 (IL-2) (Roche Applied Science, 10799068001).

(9) After the activated CD8+ T cells are collected as CTL cells and washed with saline, they are intravenously injected into the cancer patient by intravenous drip.

(Results)

The present Example accomplishes the following.

(1) An HLA binding peptide is identified from a CDR3 region of a BCR gene derived from a tumor cell of a malignant lymphoma patient.

(2) 2×106 CD8 positive cells were collected from the peripheral blood of the patient with a CD8+ T cell separation magnetic bead. The purity is 98%.

(3) Antigen stimulation is applied in mixed culture of a peptide, CD8+ cell, and DC cell derived from monocytes of the patient. Furthermore, CD8+ CTL cells can proliferate up to 50 fold in expansion and culture in the presence of anti-CD3 antibodies and IL-2.

(4) The cultured CTL cells are intravenously introduced into the patient, whereby reduction in the number of tumor cells and improvement in clinical symptom can be observed.

(Discussion)

It is understood from the present Example that the present invention accomplishes the following effect.

(1) A BCR sequence derived from the patient's cancer cells can be used to make an individualized cancer peptide for therapy. A CTL therapeutic method can be administered to a wide range of patients, regardless of HLA type or antigen expression.

(2) Since a peptide adapted to the HLA of the patient is used, it is possible to materialize an effective CTL therapeutic method that is more adapted to the patient and highly specific to cancer cells.

(3) Since an antigen peptide can be chemically synthesized directly from a genetic sequence obtained from BCR analysis, it is highly safe and requires no antigen identification or the like.

### Applied Example 4: DC Vaccine Therapeutic Method

The present Example demonstrates an example of a DC vaccine therapeutic method using repertoire analysis of the present invention. The procedure thereof is explained below (see FIG. 64).

(1) A cancer idiotype peptide is identified by the method of (1)-(9) in Applied Example 2.

(2) An existing cancer peptide (NY-ESO-1 peptide) or a cancer idiotype peptide (peptide identified in (1)) is chemically synthesized by using a fully automatic peptide synthesizer (Protein Technologies, Inc.) Peptides with a yield of 1 mg or greater and purity of 95% or greater are acquired. The peptides are dissolved in 50% DMSO and stored at −20° C. Monocytes are separated by component collection (apheresis) from a cancer patient.

(3) Monocytes are separated by a blood component collecting apparatus (Terumo apheresis apparatus AC-555) from the patient. Cells including monocytes are washed in an AIM-V medium (see Applied Example 2 and the like), and the number of cells was counted.

(4) After removing cells that did not adhere to a plastic plate (100 mm dish, Corning, 353003), the cells were cultured for about 1 week in an AIM-V medium (see Applied Example 2) comprising 2000 U/mL of granulocyte macrophage colony stimulating factors (GM-CSF, Wako Pure Chemical) and 400 U/mL of interleukin-4 (IL-4, Petrotech) and are induced to differentiate into dendritic cells (DC).

(5) Differentiation into DCs is confirmed by examining expression of MHC class I & II molecules, CD40, CD80 or CD86 by using FACS. 2×106 cells are added with 20 μg/mL of peptide (peptide synthesized in (2)) and further cultured for a day with stimulating factors (Picibanil (OK-432), Picibanil Injection 0.5KE, Chugai Pharmaceutical) in an AIM-V medium (see Applied Example 2 and the like).

(6) Peptide-stimulated DC cells are collected and washed with saline, and then intravenously injected (Terufusion Infusion System, Terumo) into the cancer patient by intravenous drip.

(Results)

The present Example accomplishes the following.

(1) An HLA binding peptide is identified from a CDR3 region of a BCR gene derived from a tumor cell of a malignant lymphoma patient.

(2) Monocytes are separated from the peripheral blood of the patient and cultured in a differentiation culture medium to detect MHC DR+, CD40+, or CD80/CD86+ cells for confirming differentiation from monocytes to DCs.

(3) The peptide-stimulated DC cells are intravenously introduced into the patient, whereby reduction in the number of tumor cells and improvement in clinical symptom can be observed.

(Discussion)

It is understood from the present Example that the present invention accomplishes the following effect.

(1) A BCR sequence derived from the patient's cancer cells can be used to make an individualized cancer peptide for therapy. A DC therapeutic method can be administered to a wide range of patients, regardless of HLA type or antigen expression.

(2) Since a peptide adapted to the HLA of the patient is used, it is possible to materialize an effective DC therapeutic method that is more adapted to the patient and highly specific to cancer cells.

(3) Since an antigen peptide can be chemically synthesized directly from a genetic sequence obtained from BCR analysis, it is highly safe and requires no antigen identification or the like.

### Applied Example 5: Patient Autoimmune Cell Therapeutic Method

The present Example demonstrates an example of a patient autoimmune cell therapeutic method using repertoire analysis of the present invention. The procedure thereof is explained below (see FIG. 65).

(1) A cancer idiotype peptide is identified by the method of (1)-(9) in Applied Example 2.

(2) An existing cancer peptide or a cancer idiotype peptide (peptide identified in (1)) is chemically synthesized by using a fully automatic peptide synthesizer (Protein Technologies, Inc.) Peptides with a yield of 1 mg or greater and purity of 95% or greater are acquired. The peptides are dissolved in 50% DMSO and stored at −20° C.

(3) 20 mL of peripheral blood is collected from a cancer patient. Peripheral blood mononuclear cells (PBMC) are separated by Ficoll-Paque gradient centrifugation.

(4) CD8+ T cells are separated by using a CD8+ T cell separation magnetic bead (Miltenyi Biotech) or flow cytometry apparatus (FACS Aria II, Beckton Dickinson).

(5) Monocytes separated by a blood component collecting apparatus (Terumo apheresis apparatus AC-555) or PBMCs are cultured in a culture plate (100 mm dish, Corning, 353003) and non-adhering cells are removed.

(6) The adhering monocytes are cultured for about 1 week in an AIM-V medium (same as Applied Example 2) comprising 2000 U/mL of granulocyte macrophage colony stimulating factors (GM-CSF, Wako Pure Chemical) and 400 U/mL of interleukin-4 (IL-4, Petrotech) and are induced to differentiate into dendritic cells (DC).

(7) After differentiation into DCs is confirmed, 2×106 cells are added with 20 μg/mL of peptide (peptide synthesized in (2)) and further cultured for a day with stimulating factors in an AIM-V medium.

(8) Furthermore, the DC culture solution is stimulated and cultured with 20 μg/mL of synthetic peptide (peptide synthesized in (2)) and 2×106/mL of CD8+ T cells isolated in the above-described (3) and AIM-V medium (same as Applied Example 2 and the like).

(9) After the activated CD8+ T cells are collected with peptide-stimulated DC cells and washed with saline, they are intravenously injected into the cancer patient by intravenous drip.

(Results)

The present Example accomplishes the following.

(1) An HLA binding peptide is identified from a CDR3 region of a BCR gene derived from a tumor cell of a malignant lymphoma patient.

(2) 2×106 CD8 positive cells were collected from the peripheral blood of the patient with a CD8+ T cell separation magnetic bead. The purity is 98% or higher.

(3) Monocytes are separated from the peripheral blood of the patient and cultured in a differentiation culture medium to confirm differentiation into DCs, MHC DR+, CD40+, or CD80/CD86+.

(4) Tumor-specific CTLs and DCs can proliferate by a mixed culture of a peptide, CD8+ cell, and DC derived from the patient's monocyte.

(5) Peptide-stimulated CD8+ cells and DC cells are both intravenously introduced into the patient, whereby reduction in the number of tumor cells and improvement in clinical symptom can be observed.

(Discussion)

It is understood from the present Example that the present invention accomplishes the following effect.

(1) A BCR sequence derived from the patient's cancer cells can be used to make an individualized cancer peptide for therapy. A patient autoimmune cell therapeutic method can be administered to a wide range of patients, regardless of HLA type or antigen expression.

(2) Since a peptide adapted to the HLA of the patient is used, it is possible to materialize an effective patient autoimmune cell therapeutic method that is more adapted to the patient and highly specific to cancer cells.

(3) Since an antigen peptide can be chemically synthesized directly from a genetic sequence obtained from BCR analysis, it is highly safe and requires no antigen identification or the like.

(4) A synergistic effect of DC cells and CTL cells can be expected, and a high therapeutic effect is anticipated.

### Applied Example 6: Isolation of Tailor-Made Cancer Specific T Cell Receptor Gene, Isolation of Cancer Specific TCR Gene by In Vitro Antigen Stimulation

The present Example demonstrates an example of isolation of a tailor-made cancer specific T cell receptor gene and isolation of a cancer specific TCR gene by in vitro antigen stimulation using repertoire analysis of the present invention. The procedure thereof is explained below (see FIG. 66).

(1) Tumor cells are extracted from a cancer patient by a conventional method.

(2) After finely cutting the tumor cell derived from the patient in a culture medium (RPMI 1640, 11875-093, Invitrogen, hereinafter also referred to as “culture solution”) and filtering with a 0.70 μm filter (Falcon cell strainer, Corning), cells are separated into single cells and inactivated with 10 μg/ml of mitomycin C (Mitomycin C for injection, Kyowa Hakko Kirin)) for 2 hours at 37° C. in the culture solution.

(3) Peripheral blood mononuclear cells (PBMC) are separated from 10 mL of whole blood of the cancer patient by Ficoll-Paque gradient centrifugation. The PBMCs are washed and then suspended in a culture medium (RPMI 1640) at a concentration of 2×106/mL.

(4) An RNA is extracted by a Trizol RNA extraction kit (Invitrogen) with some (1×106) of the PBMCs as an untreated control sample.

(5) Inactivated tumor cells and the peripheral blood cells are cultured for one week in an RPMI 1640 medium (RPMI 1640, 11875-093, Invitrogen) comprising 10% FCS (16000-044, Invitrogen) in the presence of a low concentration of IL-2 to stimulate and grow tumor specific T cells with an antigen.

(6) After activation of the T cells, live cells are collected from the culture medium and washed with PBS (045-29795, Wako Pure Chemical), and an RNA is extracted from the cells.

(7) The repertoire analysis method of the present invention is implemented by using the RNA samples extracted in (4) and (6) (for the condition thereof, the condition described in Analytical test examples and Analysis Examples 1-5 herein can be used).

(8) From the TCR genetic sequence data obtained by the next generation repertoire analysis of the present invention, a TCR gene that has greatly increased with a stimulation sample relative to a control sample is extracted and ranked, and then high ranking TCRα and TCRβ genes are selected;

(9) Each of the full-length TCRα and TCRβ genes are cloned and introduced into a retroviral vector for gene expression (Retro-X Vectors and Systems, Clonetech).

(10) A gene introducing retrovirus is created from transformation of a packaging cell GP2-293 cell line

(631458, Clonetech) by using the TCRα and TCRβ recombinant plasmid vectors prepared in (9).

(11) Lymphocyte cells separated by a blood component collecting apparatus (Terumo apheresis apparatus AC-555) are used to independently and successively infect gene recombinant TCRα and TCRβ retroviruses to obtain a population of lymphocytes expressing a functional αβ, TCR.

(12) Expression of TCRα/TCRβ heterodimers on a cell surface and the percentage of positive cells thereof are confirmed by FACS (see Applied Example 5, the same condition can be used).

(13) A tumor specific patient lymphocyte expressing TCRα/TCRβ of interest is introduced into the cells of the patient.

(Results)

The present Example accomplishes the following.

(1) When a TCR gene that increases in tumor tissue is selected and ranked from comparing a sample stimulated in the tumor tissue of a patient with a control sample, TCRs that are present in a large number in peripheral blood cells are excluded. Thus, numerous tumor specific TCR genes are extracted.

(2) TCRα and TCRβ genes at about the same level of ranking are selected from the extracted genes and utilized in making lymphocytes introduced with a tumor specific TCR gene.

(3) Full length TCRα and β chain genes can be cloned in a retrovirus expression vector. A TCRα retrovirus and TCR retrovirus with a high titer can be made by packaging.

(4) The patient lymphocyte is infected with a mixed retrovirus to verify expression of recombinant TCRα/TCRβ by FACS

(5) Tumor specific TCR gene recombinant lymphocytes manufactured by the series of steps are introduced into the patient, whereby reduction in the number of tumor cells and improvement in clinical symptom can be observed.

(Discussion)

It is understood from the present Example that the present invention accomplishes the following effect.

(1) Patient's own cancer cell and T cell can be used to make a therapeutic lymphocyte introduced with a tumor specific TCR gene. TCR gene therapy can be administered to a wide range of patients, regardless of HLA type or antigen expression.

(2) Since a TCR sequence present in a patient sample is utilized, it is possible to utilize a TCR gene matching the HLA of the patient and materialize an effective therapy that is highly specific to cancer cells.

(3) Since a TCR sequence present in a patient sample is utilized, it is possible to materialize highly safe TCR gene therapy that does not react with normal cells of the patient.

(3) Since a genetic sequence obtained from TCR analysis is directly utilized, there is no need for identifying an antigen or obtaining a TCR gene using a specific antigen.

### Applied Example 7: Preparation of Isolated Cancer Specific TCR Gene by In Vitro Antigen Stimulation

The present Example demonstrates an example of preparation of an isolated cancer specific TCR gene by in vitro antigen stimulation using repertoire analysis of the present invention. The procedure thereof is explained below (see FIG. 67).

(1) Tumor cells are extracted and peripheral blood is simultaneously separated from each of the cancer patients with the same HLA.

(2) An RNA is extracted by using a Trizol Reagent (Invitrogen) from a lymphocyte cell or tumor tissue comprising tumor cell infiltrated T cell.

(3) A TCR gene (same as the Preparation Examples and the like) is amplified by Adaptor-ligation PCR explained in the Preparation Examples and the like herein from the RNA to perform repertoire analysis by next generation sequencing with a GS Junior Bench Top system (Roche) or the like.

(4) A newly developed TCR/BCR repertoire analysis software (Repertoire Genesis, see Analysis Examples 1-5 herein) is used on the TCR genetic sequences obtained by the use thereof to determine sequences of V, D, and CDR3 regions and to create a ranking based on the frequency of presence of the same sequence.

(5) A TCR gene exhibiting a high frequency of presence in a tumor cell relative to a peripheral cell in each patient (a specific example herein is those with frequency of presence >10-fold and high ranking in tumor tissue) is searched to identify the gene as tumor specific.

(6) A TCR genetic sequence shared among a plurality of cancer patients having the same HLA is searched for such tumor specific TCR genes.

(7) A tumor specific TCR gene shared among the most cancer patients is selected as a tumor specific TCR for therapy.

(8) The full length TCRα and TCRβ genes are cloned and introduced into a retroviral vector for gene expression (the same one as in Applied Example 6 can be used).

(9) A gene introducing virus is created from the TCRα and TCRβ gene expression retroviral vector in accordance with the method of (10) in the above-described Applied Example 6.

(10) Lymphocytes collected from the patient are mixed with a culture solution containing the TCRα retrovirus made by the above-described (9) and culture solution containing a TCRβ retrovirus in equal amounts and cultured for 4 hours at 37° C. The cells are then washed with PBS and further cultured for 24 hours at 37° C.

(11) Expression of a genetically recombinant TCRαβ molecule on a cell surface is verified. The percentage of TCRβ chain positive cells to undergo transgenesis in CD8 positive cells is verified by FACS analysis using antihuman CD8 antibodies (CD8c, 6602385, Beckman Courter) and an IOTest Beta Mark TCR Vβ, repertoire analysis kit (Multi-analysis TCR Vβ, antibodies, IM3497, Beckman Courter).

(12) A cell with confirmed expression of TCRαβ of interest in (11) is cultured in an RPMI 1640 medium under the condition of 37° C. at a concentration of 0.5×106 cells. After washing lymphocytes introduced with a tumor specific TCR gene with PBS, cells are introduced into the cancer patient by intravenously injection by intravenous drip (Terufusion Infusion System, Terumo).

(Results)

The present Example accomplishes the following.

(1) When a TCR gene shared among tumor tissues of patients is selected and ranked, TCRs that are present in a large number in peripheral blood cells are excluded. Thus, numerous tumor specific TCR genes are extracted.

(2) A pair of TCRα and TCRβ genes at about the same level of ranking and present in the same patient is selected from the extracted genes and utilized in making lymphocytes introduced with a tumor specific TCR gene.

(3) Full length TCRα and β chain genes can be cloned in a retrovirus expression vector. A TCRα retrovirus and TCR retrovirus with a high titer can be made by packaging.

(4) The patient lymphocyte is infected with a mixed retrovirus to verify expression of recombinant TCRα/TCRβ by FACS.

(5) Tumor specific TCR gene recombinant lymphocytes manufactured by the series of steps are introduced into the patient, whereby reduction in the number of tumor cells and improvement in clinical symptom can be observed.

(Discussion)

It is understood from the present Example that the present invention accomplishes the following effect.

(1) Patient's own cancer cell and T cell can be used to make a therapeutic lymphocyte introduced with a tumor specific TCR gene. TCR gene therapy can be administered to a wide range of patients, regardless of HLA type or antigen expression.

(2) Since a TCR sequence present in a patient sample is utilized, it is possible to utilize a TCR gene matching the HLA of the patient and materialize an effective therapy that is highly specific to cancer cells.

(3) Since a TCR sequence present in a patient sample is utilized, it is possible to materialize highly safe TCR gene therapy that does not react with normal cells of the patient.

(3) Since a genetic sequence obtained from TCR analysis is directly utilized, there is no need for identifying an antigen or obtaining a TCR gene using a specific antigen.

### Applied Example 8: Cell Processing Therapeutic Method

The present Example demonstrates an example of a cell processing therapeutic method using repertoire analysis of the present invention. The procedure thereof is explained below (see FIG. 68).

(1) A retrovirus for transgenesis is made in accordance with Applied Example 6 to create a lymphocyte population expressing functional αβ TCRs.

(2) A tumor cell derived from a patient that has been separated and inactivated in accordance with the procedures of Applied Example 6 (1)-(2) is diluted with an RPMI 1640 medium (11875-093, Invitrogen).

(3) The lymphocyte introduced with a tumor specific TCR genes made in (1) and inactivated tumor cells of the patient were mixed at a cell concentration of 1×106/mL at a lymphocyte-tumor cell ratio (E:T ratio) of 2:1, 1:1, and 0.5:1 and cultured for 24 hours at 37° C. by using an ELISPOT kit (IFN-γ, Human, ELISpot Kit, EL285, R&D Systems).

(4) After 24 hours, the cells are removed. Production of INFγ on a PVFD membrane is detected by a coloring method, and the number of IFNγ producing cells is counted to assess the tumor specificity of lymphocytes introduced with tumor specific TCR genes.

(5) When IFNγ production is not observed in 5% of cells or less, a pair is selected with a high ranking and exhibiting a ratio of presence of TCRα and TCRβ at about the same level among TCR genes other than TCRs employed in (8) of Applied Example 6. After going through steps (9)-(11) in Applied Example 6, a new lymphocyte introduced with a tumor specific TCR gene is made.

(6) The above-described steps (1)-(4) are carried out for the TCRα and TCRβ to assess tumor specificity of lymphocytes introduced with a tumor specific TCR gene.

(Results)

The present Example accomplishes the following.

(1) A lymphocyte introduced with a tumor specific TCR gene is made to examine the reactivity to inactivated tumor cells. It can be understood that the lymphocyte introduced with a TCR gene produces IFNγ in response to tumor.

(2) The lymphocyte introduced with a tumor specific TCR gene is introduced into a patient, and an antitumor effect and improvement in clinical symptom are observed.

(Discussion)

It is understood from the present Example that the present invention accomplishes the following effect.

(1) Patient's own cancer cell and T cell can be used to make a therapeutic lymphocyte introduced with a tumor specific TCR gene. TCR gene therapy can be administered to a wide range of patients, regardless of HLA type or antigen expression.

(2) Since a TCR sequence present in a patient sample is utilized, it is possible to utilize a TCR gene matching the HLA of the patient and materialize an effective therapy that is highly specific to cancer cells.

(3) Since a TCR sequence present in a patient sample is utilized, it is possible to materialize highly safe TCR gene therapy that does not react with normal cells of the patient.

(3) Since a genetic sequence obtained from TCR analysis is directly utilized, there is no need for identifying an antigen or obtaining a TCR gene using a specific antigen.

### Applied Example 9: Method of Assessing Efficacy and/or Safety by In Vitro Stimulation Test

The present Example provides an example demonstrating assessment of efficacy and/or safety by in vitro stimulation test using repertoire analysis of the present invention. The procedure thereof is explained below (see FIG. 69).

(1) A retrovirus for transgenesis is made in accordance with Applied Example 6 to create a lymphocyte population expressing a tumor specific αβ TCR.

<Efficacy Assessment>

(1) When assessing efficacy, cancer cells derived from a patient is extracted/separated and thinly cut in a culture solution (RPMI 1640, 11875-093, Invitrogen) and then filtered with 0.70 μm filter (Falcon cell strainer, Corning) to separate a single cell. The cells are subjected to inactivation treatment for 2 hours at 37° C. with 10 μg/ml mitomycin C (Mitomycin C for injection, Kyowa Hakko Kirin) in the culture solution. After the inactivation treatment, the cells are mixed and cultured with T lymphocytes introduced with a tumor specific TCR gene made as described in Applied Example 6.

(2) Reactivity to a tumor cell is assessed by ELISPOT shown in Applied Example 8. That is, the lymphocytes introduced with a tumor specific TCR gene made in accordance with Applied Example 6 are mixed at a cell concentration of 1×106/mL at a lymphocyte-tumor cell ratio (E:T ratio) of 2:1, 1:1, and 0.5:1 are cultured for 24 hours at 37° C. by using an ELISPOT kit (IFN-γ, Human, ELISpot Kit, EL285, R&D Systems).

(3) After 24 hours, cells are removed. Production of INFy on a PVFD membrane is detected by a coloring method, and the number of IFNγ producing cells is counted to assess the tumor specificity of lymophocytes introduced with a tumor specific TCR gene. Besides ELISPOT, assessment can be performed by a cell proliferation test such as MTT assay (Cell Proliferation Kit I, MTT assay, 11465007001, Roche Diagnostics) or IL-2 production test (Human IL-2 ELISA system, GE Healthcare, RPN5965).

(Results)

The present Example accomplishes the following.

(1) When reactivity to an inactivated tumor cell of a tumor specific TCR gene recombinant lymphocyte is examined, production of IFNγ at a high frequency is recognized.

(2) The number of IFNγ positive cells increases over time during culturing, and reaches a plateau after 24 hours.

(Discussion)

It is understood from the present Example that the present invention accomplishes the following effect.

(1) Prior to administering gene therapy using a lymphocyte introduced with a tumor specific TCR gene, efficacy of using patient's own cell can be assessed. The efficacy can be predicted prior to therapy.

(2) A TCR gene can be selected and utilized by assessing efficacy. Thus, a more effective TCR gene therapy is possible.

<Safety assessment>

(1′) When assessing safety, the same test as (1) and (2) is carried out by using a control that is an existing cell line, normal tissue considered to be free of cancer cells of a patient (part of normal tissue collected in the process of tumor extraction) or peripheral blood cell of the patient in case of solid tumor.

(2′) Reactivity of a T lymphocyte introduced with a tumor specific TCR gene to normal tissue is quantified and assessed by ELISPOT.

(3′) A T lymphocyte introduced with a tumor specific TCR gene with low reactivity to normal cells and high reactivity to tumor cells is selected for use in therapy of a patient.

(Results)

The present Example accomplishes the following.

(1) When lymphocytes introduced with a tumor specific TCR gene are made to examine reactivity to inactivated normal cells, it can be understood that IFNγ is not produced and reactivity to normal cells is hardly exhibited.

(Discussion)

It is understood from the present Example that the present invention accomplishes the following effect.

(1) Prior to administering a high-risk gene therapy using a lymphocyte introduced with a tumor specific TCR gene, safety of using the patient's own cell can be assessed to materialize safer therapy.

(2) High risk TCR genes can be excluded to administer therapy using a safer TCR gene by assessing safety.

As described above, the present invention is exemplified by the use of its preferred Embodiments. However, it is understood that the scope of the present invention should be interpreted solely based on the claims. It is also understood that any patent, any patent application, and any references cited herein should be incorporated by reference in the present specification in the same manner as the contents are specifically described herein. The present application claims priority to Japanese Patent Application Nos. 2013-241403, 2013-241404, and 2013-241405, the entire content of which is incorporated herein by reference.

### INDUSTRIAL APPLICABILITY

The present invention is especially useful in clinical applications where quantitative analysis is especially required and a sample is provided for highly precise, unbiased, large scale gene analysis.

