# DESCRIPTION

## FIELD OF THE INVENTION

Certain embodiments of the present invention relate to methods and products for use in the determination of treatment of rheumatoid arthritis (RA). Particularly, although not exclusively, embodiments of the present invention relate to predicting the likelihood of a patient suffering from rheumatoid arthritis maintaining remission following cessation of disease-modifying anti-rheumatic drugs (DMARDs). Certain embodiments of the present invention are based on the determination of certain biomarkers of drug-free remission in RA following cessation of DMARD therapy.

## BACKGROUND TO THE INVENTION

Rheumatoid arthritis (RA) is a common autoimmune disease with an estimated worldwide prevalence of 0.24% that is characterised by joint inflammation and systemic manifestations. Chronic synovitis causes joint pain, stiffness, swelling and ultimately erosions leading to irreversible joint deformity. In addition to physical disability, additional extra-articular features such as accelerated atherosclerosis add to the excess morbidity and mortality of the disease.

With an increasingly ageing population, the cumulative lifetime risk of RA can be expected to increase over the next few decades. RA inflicts a substantial burden of illness—in 2010 alone, it was the cause of an estimated 3.7 million years lived with disability worldwide. The best-recognised morbidities associated with RA are physical disability and pain resulting from the progressive joint destruction associated with uncontrolled synovitis.

There is no single investigation that is diagnostic of RA, which remains a clinical diagnosis based largely upon medical history and examination findings. Nevertheless, there are several biochemical, serological and radiological investigations that can provide evidence to support the diagnostic process.

The diagnosis of RA has been formalised for research purposes by the creation of classification criteria endorsed by leading rheumatology societies. Until recently, the most widely used of these was the 1987 American College of Rheumatology (ACR) RA classification criteria. Whilst providing a sensitivity and specificity of around 90% for the diagnosis of established RA versus non-RA rheumatic disease controls, the 1987 criteria relied heavily upon features of advanced disease such as radiographic joint damage. Consequently, the 1987 criteria were criticised for their insensitivity to early disease, which made them out-dated with a later evolution towards intervention at earlier stages of clinical presentation. To address this shortfall, the ACR in collaboration with the European League Against Rheumatism (EULAR) released updated RA classification criteria in 2010. The 2010 criteria use a point-based system to define RA across four domains: joint involvement, serological status, biochemical inflammatory markers and symptom duration.

There are two principal aims in the management of RA, namely the alleviation of joint symptoms, and a reduction (ideally prevention) of joint damage and systemic disease manifestations. Long-term outcomes are more favourable when treatment is started early, and when there is comprehensive suppression of inflammation. Indeed, disease remission is now a realistic target of treatment for RA in the modern era.

One class of compounds used for the treatment of RA are so-called disease modifying anti-rheumatic drugs (DMARDs). Many of the early DMARDs including methotrexate, sulphasalazine and hydroxychloroquine are still in widespread use today.

Over the past two decades, increasing knowledge of the molecular basis of inflammation in RA, coupled with technological advances in therapeutic antibody production, have led to the development of novel biopharmaceutical agents. The mechanisms of action of these so-called “biologic” disease modifying anti-rheumatic drugs (bDMARDs) are typified by the potent but selective blockade of an inflammatory mediator, usually by means of a specific antibody or receptor fusion protein. The first of these agents to be licensed for use in RA was infliximab, a chimeric monoclonal antibody directed against tumour necrosis factor α (TNF-α). The following years have seen an explosion in the number of biologic agents, and their potent action has revolutionised the treatment of RA resistant to conventional synthetic DMARD therapy.

Remission is achievable with DMARDs prescribed in modern treat-to-target strategies, albeit with potential side effects and the requirement for safety monitoring. Approximately half of patients with RA who achieve clinical remission with DMARD therapy can expect to achieve sustained drug-free remission (DFR) following DMARD withdrawal. However, there are a lack of biomarkers that, when measured prior to DMARD withdrawal, can predict which patients can achieve DFR.

It is an aim of the present invention to provide biomarkers of drug-free remission in RA to determine when it is appropriate to stop DMARD therapy once remission has been achieved.

## SUMMARY OF CERTAIN EMBODIMENTS OF THE INVENTION

The inventors have identified novel biomarkers of drug-free remission in RA, offering insights to the pathophysiology of arthritis flare. The identification of these biomarkers allows methods to guide DMARD withdrawal, with consequent minimisation of medication side effects and healthcare costs. The identification of these biomarkers allows the development of personalised treatment strategies for RA, and related assays and kits. Accordingly, the invention provides:


- - A method of determining the likelihood of a patient maintaining
    remission of rheumatoid arthritis (RA) following cessation of
    treatment with one or more disease-modifying anti-rheumatic drugs
    (DMARDs), the method comprising:
    - (i) detecting levels of one or more biomarkers in a sample of CD4⁺
      T cells obtained from the patient; and
    - (ii) comparing the levels obtained in (i) with one or more
      reference levels;

wherein a difference in levels of the one or more biomarkers compared to the one or more reference levels is indicative of an increased likelihood of maintaining remission following the cessation of treatment with the one or more DMARDs, and wherein no difference in levels of the one or more biomarkers compared to the one or more reference levels is indicative of a decreased likelihood of maintaining remission following the cessation of treatment with the one or more DMARDs.


- - A method of treating or preventing RA in a patient undergoing
    treatment with one or more DMARDs, wherein:
    - (i) the patient has been identified as having an increased
      likelihood of maintaining remission of RA following cessation of
      treatment with one or more DMARDs according to any method as
      described herein, and treatment with the one or more DMARDs is
      reduced or ceased; or
    - (ii) the patient has been identified as having a decreased
      likelihood of maintaining remission of RA following cessation of
      treatment with one or more DMARDs according to any method
      described herein, and treatment with the one or more DMARDs is
      maintained or increased.
  - A therapeutic agent for use in a method of treating or preventing RA
    in a patient, wherein the patient has been identified as having a
    decreased likelihood of maintaining remission of RA following
    cessation of treatment with the one or more DMARDs according to any
    method as described herein, and the therapeutic agent is the one or
    more DMARDs.
  - An assay comprising:
    - (i) purifying or labelling CD4+ T cells from a sample obtained
      from a patient having or suspected of having RA;
    - (ii) detecting levels of one or more biomarkers in the purified or
      labelled CD4+ T cells; and
    - (iii) comparing the levels obtained in (ii) with one or more
      reference levels:

wherein the levels of one or more biomarkers comprise:


- - (a) expression levels of at least one, two, three or more genes
    selected from any one of SEQ ID NOs: 1 to 19; and/or
  - (b) expression levels of at least one, two, three or more gene
    variants comprising a sequence having at least 95% homology to any
    one of SEQ ID Nos 1 to 19 based on nucleic acid identity over the
    entire length of the sequence.
  - A kit comprising reagents to carry out any method or assay described
    herein, wherein the kit comprises one or more agents capable of
    specifically binding to the one or more biomarkers within CD4⁺ T
    cells in a sample obtained from the patient.

## Example 1—Study Design

The inclusion, exclusion and DMARD-cessation criteria for the Biomarkers of Remission in Rheumatoid Arthritis (BioRRA) Study are summarised in the Table below:

Inclusion and Remission Criteria

Patients were recruited to the study if they had a clinical diagnosis of RA, with fulfilment of diagnostic criteria assessed retrospectively by the research team after study enrolment.

The composite scoring system of DAS28-CRP was used to define clinical remission in the present study, as extensively used in current clinical practice (Table below):

The final clinical remission definition used in the BioRRA study. CRP, C-reactive protein in mg/L; DAS28, disease activity score in 28 joints; SJC28, swollen joint count in 28 joints; TJC28, tender joint count in 28 joints; VASpatients, patient visual analogue score in millimetres (on 0-100 mm scale).

The use of CRP was used owing to its specificity for inflammation. A threshold of DAS28-CRP<2.4 was used as the definition of clinical remission—this is lower than the remission threshold for DAS28-ESR (<2.6) and is in line with recently published data that recommends this lower remission threshold (Fleischmann et al., 2015). Values of CRP below the detectable threshold of the local clinical laboratory (<5 mg/L) were recorded as zero for the purposes of DAS28-CRP calculation.

Study Design and Sample Size Estimation

The design of the study together with sample size estimations are summarised in FIG. 1.

Patients who satisfied both DAS28-CRP<2.4 and an absence of PD signal on ultrasound assessment (see below) at the baseline visit were eligible for DMARD cessation. Further administrations of methotrexate, sulfasalazine and/or hydroxychloroquine were immediately stopped without tapering. Non-DMARDs, including folic acid, remain unaltered and non-steroidal anti-inflammatory drug (NSAID) use was permitted. Research blood tests including serum, plasma, peripheral blood mononuclear cells (PBMCs), CD4+ T cell isolation and whole blood RNA TEMPUS™ tubes (Applied Biosystems, Foster City, Calif., USA) were collected at baseline, month 3 and month 6 following DMARD cessation. An US scan was performed again at the month 6 visit. If a patient experienced a flare of their arthritis (defined as DAS28≥2.4) or if they developed PD synovitis on the month 6 scan, then they could receive rescue glucocorticoids and were discharged from the study with their DMARD therapy promptly recommenced by their referring rheumatologist. Those patients who remained in remission at 6 months with no PD synovitis were discharged back to their referring rheumatologist without DMARD therapy. 74 patients were recruited to the study, of which 44 stopped DMARDs.

Prospective Clinical Variable Assessment

Pre-specified clinical variables were recorded prospectively at the baseline study visit, as listed in the Table below:

Included in these variables is the Health Assessment Questionnaire Disability Index (HAQ-DI), a self-completed questionnaire that quantifies physical disability that has been extensively validated in the setting of RA and other chronic diseases.

Retrospective Clinical Variable Assessment

Data were obtained for a range of pre-specified variables, as listed in the Table below. These data were obtained both by patient interview at baseline visit, and from clinical notes review. Historical clinical records were available for all patients and were assessed by a single reviewer. It was acknowledged that the most reliable source of information might differ depending on the variable of interest. A systematic methodology of recording clinical variables was therefore implemented, as shown in the Table below:

Assessment of RA Criteria

Patients were recruited on the grounds of a clinical diagnosis of RA made by a consultant rheumatologist, with satisfaction of formal classification criteria assessed retrospectively after patient recruitment.

Healthy Control Participants

In order to provide a control group for subsequent transcriptomic analyses, four healthy participants were recruited. Each participant donated blood at four time points to mirror those of the patient participants—i.e. baseline, month 1, month 3 and month 6. Healthy participants donated blood between 9 am and 1 pm where possible to minimise circadian variation in laboratory samples. Blood was left to stand at room temperature to mimic the equivalent period of transit for patient samples, before being processed for CD4+ T cell extraction and subsequent downstream RNA/DNA applications using identical protocols to those of the patient participants.

CD4+ T Cell Isolation

CD4+ T cells were isolated from 27 ml of whole blood drawn into three 9 ml ethylenediaminetetraacetic acid (EDTA) tubes following the protocol of Pratt (2011). Briefly, monocytes were first depleted by the use of an anti-CD36/anti-glycophorim A cross-linking reagent (Rosettesep human monocyte depletion cocktail) that crosslinks monocytes to erythrocytes forming immunorosettes, which were separated by centrifugation after the addition of the erythrocyte aggregation agent HetaSep®. The supernatant was then collected and the remaining CD4+ T cells were positively selected by automated anti-CD4 antibody-based magnetic isolation (Easisep® whole blood CD4+ selection kit and Robosep® automated cell separator). By the prior removal of CD4′° expressing monocytes this extraction method is able to obtain CD4+ T cell purities of over 98%, compared to approximately 90% for single step column-based immunoprecipitation methods (Pratt et al., 2012). After cell counting (see below), 2×105 cells were transferred to a single well of a 96-well V-bottom plate for flow cytometry processing (see below).

CD4+ T Cell Lysis

Purified CD4+ T cell isolates were transferred to 30 ml universal tube and washed by addition of cold (4° C.) calcium- and magnesium-free Hanks balanced salt solution (HBSS)+1% foetal calf serum (FCS) to a total volume of 25 ml. The tube was then centrifuged at 400 g for 7 minutes at 4° C., after which the supernatant was removed by aspiration using a vacuum-driven glass pipette. In an RNase-free open workbench area, the pellet was then resuspended in cold (4° C.) Qiagen Buffer RLT Plus+1% β-mercaptoethanol dependent on the number of T cells to be lysed; if <5 million cells then 350 μL of Buffer/β-mercaptoethanol was added, whereas if 25 million cells then 600 μL of Buffer/β-mercaptoethanol was added (in line with the manufacturer's protocol). The suspension was thoroughly mixed by pipetting and vortexing before transfer to a QIA-shredder column. Lysis of the T cells was completed by centrifugation of the column at 13,000 g for 2 minutes, and the lysate was then stored at −80° C.

Peripheral Blood Mononuclear Cell Isolation

PBMCs were isolated from 18 ml of whole blood drawn into two 9 ml EDTA tubes. The blood was diluted in HBSS+2 mM EDTA to a total volume of 40 ml. The diluted blood was then split to two 20 ml volumes that were layered by slow pipetting above 15 ml of Lymphoprep® within a 50 ml centrifuge tube. The layered tubes were then centrifuged at 895 g for 30 minutes with minimal acceleration and deceleration speeds to maintain layering of the sample. After centrifugation, PBMCs were removed from the interface layer using a Pasteur pipette and transferred to a fresh 50 ml centrifuge tube. The PBMCs were then immediately washed in cold (4° C.) HBSS+1% FCS to a total volume of 50 ml and centrifuged at 600 g for 7 minutes at 4° C. to remove any residual Lymphoprep®. The supernatant was discarded, and the pellet resuspended in 50 ml of cold (4° C.) HBSS+1% FCS before centrifuging at 250 g for 7 minutes at 4° C. to remove any platelets. The supernatant was discarded, and the pellet resuspended in 7 ml of cold (4° C.) HBSS+1% FCS and strained through a 70 μm nylon filter to a new 50 ml centrifuge tube (to exclude clumped cells), which was kept on ice for immediate cell counting. After cell counting, 2×105 cells were transferred to a well of a 96-well V-bottom plate and stored at 4° C.

Serum Separation

Blood drawn in to a single 8 ml serum separator clot activator tube was centrifuged at 1,800 g for 12 minutes. The supernatant (i.e. serum) was then carefully removed by pipetting and divided to 1 ml aliquots, and frozen at −80° C.

Cell Counting

PBMCs and CD4+ T cells were counted within 10 μL of diluted sample placed underneath a cover slip mounted upon a Bürker haemocytometer chamber. The number of cells within 25 squares was then manually counted using alight microscope. The total number of cells present in the sample was calculated as per the Formula below:

calculation of cell number by Bürker haemocytometer chamber counting: Total cells=[volume (ml)]×[dilution factor]×[number of cells in 25 squares]×104

CD4+ Purity Check by Flow Cytometry

Unfixed samples of PBMCs and CD4+ T cells stored in 96-well V-bottom plates at 4° C. were stained and analysed by flow cytometry within 18 hours of sample isolation. Firstly, the plate was centrifuged at 400 g for 3 minutes and the supernatant removed by flicking. Cells were then resuspended in 50 μL of flow cytometry antibody mixture by thorough pipetting and incubated in the dark for 30 minutes at 4° C. After this, 100 μL of flow cytometry buffer was added to each well and the plate then centrifuged at 400 g for 3 minutes. After removal of the supernatant by flicking, the stained cells were then resuspended in 150 μL of flow cytometry buffer and the plate centrifuged at 400 g for 3 minutes. After removal of the supernatant by flicking, the cells were resuspended in 200 μL of flow cytometry buffer and transferred to individual cytometry tubes. Flow cytometry data was recorded using a FACSCanto-II cytometer and FACSDiva software (Becton Dickinson Biosciences, San Jose, Calif., USA). Analysis of flow cytometry data was performed using FlowJo software (FlowJo LLC Data Analysis Software, Ashland, Oreg., USA). PBMC samples were first gated on side-scatter area (SSC-A) and width (SSC-W) to identify singlets, which were then gated on SSC-A and forward-scatter area (FSC-A) to exclude debris. The resulting population was then gated on compensated CD3 and compensated CD4 to identify CD3+CD4+ T cells, SSC-A and compensated CD14 to identify CD14+ monocytes, and SSC-A and compensated CD19 to identify CD19′ B-cells. These gates were then applied to the CD4+ T cell isolate matched to the individual patient where available to assess purity.

CD4+ T Cell RNA Extraction

Frozen CD4+ T cell lysates (see above) were thawed at room temperature in an RNase-free open workbench area. Thawed lysates were mixed by pipetting and transferred to RNase-free conical-bottom 2 ml microcentrifuge tubes. In order to remove residual magnetic nanoparticles remaining from the CD4+ T cell isolation procedure, the lysates were centrifuged at maximum speed (13000 rpm) for 2 minutes. The supernatant was then carefully removed using a P1000 pipette (taking care not to disturb the pellet of magnetic nanoparticles at the bottom of the tube) and transferred to a Qiagen AllPrep DNA Mini spin column placed within a 2 ml collection tube.

The sample was then centrifuged at maximum speed (13000 rpm) for 30 seconds

The collection tube containing the flow-through from the DNA column was then processed according to the manufacturer's instructions, including on-column DNase digestion and all steps designated as ‘optional’ according to the manufacturer's instructions. Final purified RNA was eluted from the RNeasy Mini spin column using a volume of RNase-free water determined by total number of CD4+ T cells present in the initial sample (see above) as follows: 30 μL for <5×106 cells, or 40 μL for 25×108 cells. The amount of RNA present in each sample was then quantified using a NanoDrop™ ND1000 ultraviolet (UV) spectrophotometer (Thermo Fisher Scientific), with 2 μg of each sample (or the total sample if <2 μg) stored at −80° C. for RNA sequencing processing (see below).

Next-Generation RNA Sequencing (RNAseq)

The quantity and estimated RNA integrity (RINe) of RNA samples was measured by gel electrophoresis using a Tapestation™ 4200 machine (Agilent). Following quantification, 1.5 μg of total RNA per sample was used for RNAseq processing; where total RNA<1.5 μg, the entire sample was used. Total RNA was processed using the TruSeq™ Stranded mRNA Library Prep Kit (Illumina), according to the ‘High Sample Protocol’ section of the manufacturer's instructions. First, messenger RNA (mRNA) was enriched from the purified total RNA by poly-A selection using poly-T oligo attached magnetic beads in two rounds of purification. Enriched mRNA was then fragmented by heating with magnesium cations, and then incubated with reverse transcriptase to synthesise first strand cDNA for each sample. This step was performed in the presence of Actinomycin D to prevent DNA-dependent synthesis of a second strand. The mRNA was then degraded with RNase and second strand cDNA was then synthesised by incubation with DNA Polymerase I. Double-stranded cDNA was then isolated using solid phase reversible immobilisation (SPRI) paramagnetic beads (Agencourt™ AMPure™ XP beads, Beckman Coulter Genomics), after which the 3′ ends were adenylated to facilitate sequencing adaptor binding. The Illumina sequencing adaptors contained three key functional elements: an amplification element required for amplification of the cDNA by polymerase chain reaction, a sequencing element required for the sequencing reaction, and a unique index element to allow identification of each individual patient sample. cDNA that had successfully ligated with adaptors was selectively amplified by poymerase chain reaction. Amplified cDNA was then diluted to equimolar concentrations and pooled before sequencing.

RNA sequencing was performed using an Illumina NextSeg™ 500 in high-output mode. This configuration delivered 400 million reads over 75 cycles for 40 samples loaded across 4 lanes per flow cell. Sequencing was performed in batches across 4 separate flow cell sequencing runs. Samples were allocated to sequencing batches such that computational correction for any batch-to-batch variation at the level of either the RNA extraction (6 batches) or RNA sequencing (4 batches) could be achieved, according to a predetermined experimental design using the duplicate correlation command of the ‘limma’ Bioconductor/R package (v3.32.5) (Ritchie et al., 2015). Using this approach, it was possible to sequence all 136 CD4+ T cell samples to a depth of 10 million reads per sample, with 75 bp single-end reads.

Mutiplex Cytokne/Chemokine Electrochemiluminescence Assays

Previously separated serum samples stored at −80° C. (see above) were thawed at 37° C., mixed by vortexing and then centrifuged for 2 minutes at maximum speed (13000 rpm) to separate any contaminating debris. Serum was then transferred to 96-well V-PLEX™ plates (MesoScale Discovery) and processed according to the manufacturer's instructions. The volumes of sample loaded per well, together with the fold dilution and dilution method for each plate is detailed in the Table below, as specified by the manufacturer:

The assays included on each plate are detailed in the Table below:

Owing to their large number, samples were processed across pairs of each plate type. Each pair of plates was processed on the same day to minimise variation between plates for each set of analytes. Although V-PLEX™ plates are specifically designed and certified to have minimal variation in assay performance between separate plates of the same lot number, additional steps were taken to minimise any potential for plate-to-plate variation to affect data analysis. First, for each plate pair, all baseline samples were included together on the same individual plate. This allowed for comparison of baseline samples between patients independent of any effect of plate-to-plate variation. Secondly, it was ensured that all samples from each individual patient were included together on the same individual plate. This allowed for comparison between study time points at the level of each individual patient independent of any effect of plate-to-plate variation in absolute quantification.

Following processing, plates were immediately analysed by electrochemiluminescence (ECL) using a MESO™ QuickPlex SQ120 (Meso Scale Diagnostics, LLC.) according to the manufacturer's instructions. Signal—concentration curves based on serial dilutions of a known supplied standard were generated for each assay to calculate the concentration of analytes in each sample using the Discovery Workbench™ software version 4.0 (Meso Scale Diagnostics, LLC.). For each assay, the upper limit of detection (ULOD) for was defined as the concentration of the highest calibrator, and the lower limit of detection (LLOD) was defined as 2.5×standard deviation above the lowest calibrator concentration. Analytes with an ECL signal corresponding to a calculated concentration above the ULOD were assigned a calculated concentration equal to the ULOD. Analytes with an ECL signal corresponding to calculated concentration below the LLOD were assigned a calculated concentration equal to the LLOD. Where a sample was duplicated on the same plate (for internal quality control purposes), the mean calculated concentration value was used for analysis.

Data Analysis

Analyses were performed according to the structure detailed below. All Cox regression models used Breslow approximation for handling of tied survival times (Themeau, 2015).


- - 1. The quality of data was assessed, and where necessary, low
    quality data was excluded.
  - 2. The distributions of data were summarised by descriptive
    statistics and, for continuous data, were visualised by standard
    methods including boxplots, Q-Q plots and histograms.
  - 3. Where the distributions of data were skewed, appropriate
    transformations of data groups was performed (e.g. natural
    logarithmic transformation).
  - 4. The association of each individual variable at baseline with
    time-to-flare was assessed by univariate Cox regression using the
    ‘survival’ package.
  - 5. Baseline variables were selected based on their univariate
    p-value to be taken forward to a multivariate Cox regression model.
    For clinical and cytokine data, an elevated significance threshold
    (p\<0.2) was used in order to reduce the risk of type II error at
    this preliminary stage, in keeping with established precedent (Dales
    and Ury, 1978: Mickey and Greenland, 1989). In comparison, a more
    stringent significance threshold (p\<0.001) was utilised for RNAseq
    univariate analysis in reflection of the several log-fold greater
    number of variables analysed.
  - 6. Backward stepwise variable selection based on the Akaike
    information criterion (AIC) using the ‘stepAIC’ function of the
    ‘MASS’ package was used to create a stepwise multivariate Cox
    regression model. The regression coefficient (equivalent to the
    natural logarithm of the hazard ratio) and its corresponding 95%
    confidence interval for each variable was visualised in a forest
    plot format using the package ‘forestplot’ (and its dependent
    packages ‘checkmate’ and ‘magrittr’).
  - 7. Patients were dichotomised by the levels of variables measured at
    baseline. In the case of binary variables, patients were
    dichotomised simply by presence or absence of the variable at
    baseline. For continuous variables, thresholds were determined by
    receiver operating characteristic (ROC) analysis (using the ‘pROC’
    package). For each continuous variable, two optimum thresholds were
    set to optimise negative predictive value (NPV) and positive
    predictive value (PPV) for flare, corresponding to biomarker
    thresholds for remission and flare respectively. Confidence
    intervals for these metrics and the area under the ROC curve
    (ROC_(AUC)) were calculated by bootstrapping (2000 replicates) and
    the DeLong procedure (DeLong et al., 1988) respectively.
  - 8. Variables that were significantly associated with time-to-flare
    in the stepwise multivariate Cox regression model at the p\<0.05
    threshold were then combined in composite scores, weighted by their
    respective coefficients in multivariate Cox regression analysis.
  - 9. Survival curves were compared between the dichotomised groups
    (using the ‘survminer’, and its dependent ‘ggpubr’ and ‘ggplot2’,
    packages) by the log-rank test as a measure of their utility in
    predicting time-to-flare after DMARD cessation.
  - 10. Finally, where data allowed, the relationship between
    longitudinal changes in variables at the individual patient level
    and time-to-flare were explored.

A key assumption of Cox regression is that the hazard function for each stratum of the dataset is proportional over time—in other words, the proportional change in hazard attributable to a given variable must be constant across all time points. In a seminal paper, Schoenfeld defined residuals of the proportional hazards model that, if proportionality of hazards is true, show no significant correlation with time (Schoenfeld, 1982). Following this approach, proportionality of hazards was tested for each individual variable in univariate and stepwise multivariate Cox regression models by plotting scaled Schoenfeld residuals against time, and fitting a smoothed spline curve with four degrees of freedom using the survival package (Themeau, 2015). Proportionality of hazards was assumed if no significant association at the 0.05 threshold was observed between scaled Schoenfeld residuals versus time.

The Akaike information criterion (AIC) was used to drive stepwise backwards selection of variables in the multivariate Cox regression models. The AIC is a well-established method of nested model selection, which penalises goodness of model fit by the number of variables contained within the model (Bozdogan, 1987). The best model, displaying the optimum balance between predictive utility and number of variables, is distinguished by the lowest AIC score. In backwards stepwise selection, a set of reduced models is created by dropping each of variables in turn from the starting model. The model with the lowest AIC is then taken forward to the next round of variable selection, and the process repeated until no further reduction in AIC is possible—the model with the lowest AIC in the final step is accepted as the final stepwise model. Stepwise backwards variable selection of Cox regression models based on AIC has been previously described in analyses of differential gene expression in cancer studies, using both microarray technology (Wozniak et al., 2013) and next-generation sequencing data (Falco et al., 2016; Fan and Uu, 2016).

### Example 2—Clinical Data Results

Baseline clinical variables that are predictive of sustained DFR versus flare following DMARD cessation are identified herein.

Patient Recruitment and Outcomes

Of the 44 patients who discontinued DMARD therapy, 23 (52%) were classified as experiencing an arthritis flare, 20 (45%) maintained DFR and 1 (2%) was effectively lost to follow-up and was analysed as being censored in remission after 69 days follow-up (FIG. 2).

Survival Analysis

The association between baseline clinical variables and time to flare following DMARD cessation was analysed by univariate Cox regression for all 44 patients who stopped DMARDs (see Table below).

Proportionality of hazards was assessed for each univariate variable by correlation of scaled Schoenfeld residuals with transformed flare-free survival time (see methods above). No significant departure from proportional hazards was observed for any of the variables.

The 15 variables with a univariate value <0.2 were then entered simultaneously in a multivariate Cox regression model. Stepwise backward selection based on AIC was then performed using the same variables to fit a stepwise Cox regression model. After 6 rounds of selection, variables remained in this stepwise model (see Table below).

Proportionality of hazards was again assessed for each variable in the final stepwise multivariate Cox regression model. A significant departure from proportional hazards was observed only for current methotrexate use (p=0.04), though this was only notable for a single outlier with no discernible trend in the remainder of the data. The global Schoenfeld test was non-significant (p=0.49), indicating proportionality of hazards for the model as a whole.

Four variables were associated with shorter time-to-flare at the 5% significance level, namely: ACPA positivity. RhF positivity, current methotrexate and time from diagnosis to commencement of first DMARD. In contrast, fulfilment of ACR/EULAR Boolean remission criteria and time since last change in DMARD therapy were associated with a longer time-to-flare.

Composite Clinical Biomarker Score

Values of the six variables that were significantly (p<0.05) associated with time-to-flare at in the multivariate stepwise Cox regression model were multiplied by their respective coefficients in the model and then summed to create composite scores. The predictive value of all 63 possible combinations of these variables to predict flare and remission following DMARD cessation was then compared by area under the receiver-operating characteristic curve (ROCAUC). The ten composite scores with the highest ROCAUC are listed in the Table below.

Optimal performance was observed for a four-variable composite score incorporating ACPA positivity. ACR/EULAR Boolean remission, time since last change in DMARD therapy, and current use of methotrexate (Formula below).


- - Formula—Composite clinical biomarker score. Values for binary
    variables, as indicated by the square brackets, are only added to
    the equation if the variable is present.

Clinical score=0.654[ACPA positive]+1.130[current methotrexate]-0.521[ACR/EULAR Boolean remission]−0.012(months since last DMARD change)

The composite clinical score performed well its ability to discriminate flare versus remission following DMARD cessation, with a total area under the ROC curve (ROCAUC) of 0.85 (95% Cl 0.73-0.97). Optimal thresholds were determined to maximise detection of flare (1.82) and remission (0.51). Both thresholds performed well in the study population, with a positive predictive value (PPV) of 0.90 (95% Cl 0.78-1.00) for the flare threshold, and a negative predictive value (NPV) of 0.87 (95% Cl 0.69-1.00) for the remission threshold (Table below).

Baseline Predictors of Drug-Free Remission

Of the six variables significantly associated with time-to-flare in the multivariate stepwise Cox regression model, four had discriminatory value in a composite score for predicting flare and remission in the six months after DMARD cessation. ACPA and RhF positivity were predictive of increased likelihood of flare; and ACR/EULAR Boolean remission and months since last DMARD change were predictive of increased likelihood of remission.

ACR/EULAR Boolean Remission

A notable finding of this study is that fulfilment of ACR/EULAR Boolean remission at baseline is associated with a 3-fold lower risk of arthritis flare in the six months following DMARD cessation. Given the relatively new introduction of this remission definition, data surrounding its utility in the prediction of DFR following DMARD cessation remains scarce. The only study to have previously addressed this issue is the RETRO study, which demonstrated only a non-significant trend towards lower flare rate in those satisfying ACR/EULAR Boolean remission (ORflare 0.673, 95% Cl 0.211-2.144, p=0.503) (Haschka et al., 2016).

Any of the individual components that constitute ACR/EULAR Boolean remission failed to demonstrate a significant association with time-to-flare in the multivariate analysis, suggesting that it is the combination of variables within the definition that is of importance. Although not designed specifically for clinical use, a major limitation of the use of ACR/EULAR Boolean remission in clinical practice lies in its stringent VASpatient≤ 1/10 threshold. Indeed, 10/74 (15%) of patients recruited to this study satisfied DAS28-CRP remission at baseline yet failed to achieve ACR/EULAR remission solely on grounds of VASpatient alone (median 22/100, range 12-34). A modified Boolean remission criteria incorporating a higher VAS patient threshold (for example ≤20/100) may have greater specificity for flare without losing sensitivity.

## SUMMARY

ACR/EULAR Boolean remission is negatively associated with flare, suggesting a robust and stable clinical remission phenotype at baseline in those patients who subsequently achieved sustained drug-free remission.

## Example 3—Cytokine Results

The measurement of circulating chemokines and cytokines provides an attractive approach to the development of biomarkers of DFR in RA. Suitable biological samples (serum or plasma) can be obtained with simple venepuncture and require only a single centrifugation step before suitable for long-term freezer storage. Furthermore, many laboratory assays for the sensitive and specific detection of a wide range of human cytokines are already in widespread research and clinical use. A robust serum-based circulating cytokine/chemokine biomarker of DFR would thus be ideally placed for translation to a high-throughput assay suitable for use in clinical practice.

Surprisingly little research has been conducted to date that explores circulating serum predictors of DFR in RA beyond acute-phase markers. Nevertheless, available tools such as the multi-biomarker disease activity (MBDA) assay have been developed for the measurement of disease activity. It is possible that such measures could overlook more nuanced mediators that are not directly involved in the acute phase response, but nonetheless play crucial roles maintaining the balance between sustained DFR and arthritis flare.

Cytokine/chemokine biomarkers are identified herein that, when measured immediately prior to DMARD cessation, differentiate those patients who subsequently developed arthritis flare versus those who remained in sustained DFR. The longitudinal change in circulating cytokine/chemokine profile in both flare and remission groups are also explored, providing insights to the underlying immunobiology of flare and DFR.

Quality Control

In this study, commercially available multiplex electrochemiluminescence assays (MesoScale Discovery) were used for the detection of 39 chemokines and cytokines (see methods above). These kits benefit from both a highly specific antibody-based capture mechanism, together with robust quality control mechanisms to ensure reliable assay performance.

Sample Collection

Of the 44 patients who stopped DMARDs, baseline samples from 43 patients were available for analysis. Serum was available for 29/42 month one visits and for 15/19 patient-requested unscheduled visits. Serum was collected for all month 3 and month 6 visits. Serum was available at the time of flare for 22/23 patients who experienced an arthritis flare.

Time from blood draw to centrifugation was recorded for 133/136 serum samples collected, with a median value of 50 (IQR: 43-87, range: 30-210) minutes.

Calibrator Coefficient of Variation

Seven manufacturer-supplied calibrator solutions containing known concentrations of analytes were prepared by serial dilution for each plate. This allowed calculation of the percentage coefficient of variation (% CV) of calculated concentration for each calibrator on each plate (Appendix G). The manufacturer states that the % CV is typically less than 20% for repeat measurements. Over the 10 individual plates there were 656 pairs of calibrator measurements, of which % CV>20 in 60 (9%) of measurement pairs. Of these measurement pairs, 37/60 (62%) were within the three lowest concentration calibrators.

Limits of Detection

A total of 6324 assay measurements (excluding calibrators and blanks) were recorded over the 10 plates. Upper and lower limits of detection (ULOD and LLOD respectively) were calculated for each assay as previously described (see methods above). No samples exceeded the ULOD, whereas 3391 (54%) had a calculated concentration below the LLOD.

Samples where the calculated concentration fell below the LLOD were assigned a calculated concentration equal to the LLOD for each respective assay.

Two IL-8 assays were included with differing ranges of detection: standard IL-8 assay and an alternative IL-8 assay (IL-8 (HA)) with a higher detection range. Concentration measurements frequently fell below the limits of detection for the IL-8 (HA) assay, and thus results from the standard IL-8 assay were used for analysis. Similarly, two IL-17A assays were included based on different isotypes of detection antibody. A greater proportion of assays fell below the LLOD for the IL-17A assay on the Th17 panel plates in comparison to the IL-17A assay on the cytokine panel plates, and thus the former was excluded from analysis.

For analyses based solely on baseline samples, 13 assays where <20% of measurements were above the LLOD were excluded. For longitudinal analysis of flare patients, six assays where all measurements were below the LLOD at both baseline and time of flare were excluded. For longitudinal analysis of remission patients, eight assays where all measurements were below the LLOD at both baseline and month six visit were excluded. The assays included within each analysis following the above quality control steps are summarised in the Table below.

Logarithmic Data Transformation

The distribution of calculated concentrations for each assay was typically positively skewed. To create a more even distribution of data, the natural logarithm of each analyte concentration measurement was calculated before further data analysis. A constant value (+1) was added to each analyte concentration before logarithmic transformation to allow for zero values.

Plate Equilibration

It was not possible to analyse all serum samples together on the same plate, as the total number of samples (136) exceeded the number of available wells (80) on a single plate. It was therefore necessary to process samples across pairs of each type of MSD plate. Each pair of plates was processed on the same day to minimise variation. Furthermore, all baseline samples were run together on the same plate within each plate pair, thus permitting analysis of baseline samples without any effect of plate-to-plate variation.

By replicating baseline samples where space allowed, it was attempted to keep all samples from the same participant together on the same single plate for the purposes of longitudinal analysis—however, this was not possible in all cases. Despite the manufacturer's claims of excellent reproducibility of assay performance between plates of identical lots, it was deemed necessary to apply a data equilibration procedure to minimise any potential effect upon longitudinal analyses. First, the relationship between the concentrations of each individual analyte for duplicated samples (i.e. present on both plates within a pair) was modelled by linear regression as detailed in the formula below:

linear regression equation for plate equilibration procedure: Plate2 concentration=m*(plate1 concentration)+c

The regression formula was then applied to each individual concentration measurement for each assay within each plate2, thus equilibrating plate2 with plate1 measurements in each plate pair. Finally, to avoid any bias introduced by differing LLODs between plate pairs, the highest LLOD of either plate within each pair was applied for the purposes of longitudinal data analyses. Where a sample had been analysed on both plates, the sample measurement on the same plate as the majority of other samples for that patient was used for analysis to reduce further any bias introduced by plate-to-plate variation.

There were 11 assays where only two or fewer samples measured above the LLOD on both plates—these assays were thus excluded from plate equilibration, namely: GM-CSF, IL-1α, TNFβ, IL-12p70, IL-13, IL-1β, IL-2, IL-4, IL-21, IL-23 and IL-31. In addition, MIP1α concentration demonstrated a poor correlation between plates and was also excluded. Plate equilibration was not performed for these samples, but rather only concentrations from a single plate (i.e. excluding values from the other plate) at the level of each patient were used for analysis.

Baseline Survival Analysis

The association between baseline cytokine/chemokine levels and time-to-flare following DMARD cessation were analysed by univariate Cox regression (see Table below) for all 26 assays that passed quality control checks (see above). Proportionality of hazards was observed for all variables with the exception of ln(ICAM1+1).

There was no statistical association between ln(ICAM1+1) and flare by univariate Cox regression (p=0.95). To confirm that this lack of association was not a spurious effect of poor survival modelling, the relationship between ln(ICAM1+1) and occurrence of flare was assessed by univariate binomial logistic regression, which also demonstrated a similar lack of association (ORflare 0.66, 95% Cl 0.12-3.72, p=0.64).

Following univariate modelling, the 10 variables with a univariate p-value <0.2 were then entered to a multivariate Cox regression model, and stepwise backward variable selection based on AIC was performed. After seven rounds of selection, three variables remained in this stepwise model (Table below). Proportionality of hazards was confirmed for all three variables by the Schoenfeld residual test.

The distributions of baseline ln(MCP1+1), ln(IL27+1) and ln(CRP+1) by flare status are summarised in FIG. 3. Baseline MCP1 levels were significantly higher at baseline in those patients who experienced an arthritis flare (p=0.012, unpaired Student's T-test). There was a trend towards higher baseline CRP and IL-27 in the flare group, though this was not statistically significant (p=0.121 and p=0.224 respectively).

ROC Analysis and Biomarker Thresholds

Patients were dichotomised based on their baseline levels of MCP1, IL-27 and CRP using two thresholds determined by ROC analysis optimised for the prediction of flare and remission, as discussed above. Variables were also combined to form composite scores, weighted by their respective coefficients from the stepwise multivariate Cox regression model (see Table below).

Whereas MCP1 performed reasonably well in isolation, the discriminative value of IL-27 and CRP when used on their own was relatively poor. The best results were observed for composite measures, namely MCP1+CRP and MCP1+IL27, which performed best for the identification of future remission (FIG. 4) and flare (FIG. 5) respectively. A composite score encompassing all three variables did not yield any additional predictive value (see Table below).

Sensitivity Analysis for Time to Centrifugation

Cytokines and chemokines are known to degrade in whole blood that has been left to stand for a prolonged period before centrifugation (Jackman et al., 2011). To assess whether this may have influenced the biomarker analysis, the three variables within the composite scores (i.e. ln(MCP1+1), ln(IL27+1) and ln(CRP+1)), and time from blood draw to centrifugation, were entered to a multivariate Cox regression model. In the three visits where centrifugation delay was not recorded, the median centrifugation delay across all visits was imputed. No significant association was observed between centrifugation delay (minutes) and arthritis flare (HRflare 1.00, 95% Cl 0.99-1.00, p=0.465). Similar coefficients and statistical significance were observed for the three cytokine/chemokine variables as per the main analysis, demonstrating that their utility for predicting arthritis flare was not affected by inclusion of centrifugation delay within the model (see Table below). Proportionality of hazards was observed for all individual variables and the model as a whole.

Baseline to Flare Visit

Cytokine and chemokine levels were compared at baseline and flare visits in the 22 patients who both experienced a flare and had serum available at the time of flare. The statistical significance of differences between log-transformed cytokine/chemokine concentrations at baseline and flare visits was assessed by Student's paired T-test (see Table below). Four analytes demonstrated a >1.5 fold change in concentration at the 5% significance level after multiple-test correction using the Benjamini-Hochberg procedure: CRP, SAA, IL-6 and IP-10 (FIG. 6).

Baseline to Month Six Remission Visit

Cytokine and chemokine levels were compared at baseline and month six visits in the 19 patients who maintained DFR and had serum available at baseline and month six. The statistical significance of differences between log-transformed cytokine/chemokine concentrations at baseline and month six visits was assessed by Student's paired T-test (Table below). No analytes exceeded a >1.5 fold change in concentration at the 5% significance level (FIG. 6).

Longitudinal Change in Selected Cytokines and Chemokines

The longitudinal change in circulating concentrations of selected cytokines and chemokines was explored across all study visits where serum samples were available for analysis. Descriptive analysis with the aid of longitudinal ‘spaghetti plots’ is presented herein (FIG. 7).

Both CRP and SAA concentrations rose in the approach towards arthritis flare, and remained essentially static for patients in remission, with the exception of two remission patients who had a concomitant urinary tract and lower respiratory tract infections respectively at the time of their month 3 study visit. IL6 concentrations abruptly rose at the time of flare, and remained static for patients in remission, although many samples fell below the plate-merged LLOD. Overall, IP10 concentrations rose higher in the flare versus remission group, though there was greater heterogeneity in the individual patient trends.

There was a modest overall increase in IL-27 from baseline to flare which was not observed in patients who remained in remission, although the individual patient trends were somewhat mixed. There were no clear longitudinal trends in MCP1, in contrast to its utility for predicting flare when measured at baseline.

Monocyte Chemoattractant Protein 1 (MCP-1/CCL2)

In this study, MCP-1 is predictive of outcome following DMARD cessation, with low levels at baseline demonstrating particular discrimination for those patients who maintained DFR. This observation is biologically plausible—low circulating levels of MCP-1 could conceivably reflect lower production by stromal and immune cell mediators within the synovium, thus reducing recruitment of monocytes, T cells and NK cells to the joints.

Interestingly, no significant increase in MCP-1 concentration was observed in longitudinal analysis of patients who experienced an arthritis flare. This observation suggests a possible modulatory effect of MCP-1 at baseline, as opposed to representing a surrogate measure of disease activity. However, whether the observed low levels of circulating MCP-1 reflect a qualitative difference in immune homeostasis in those patients who maintain DFR, rather than simply a quantitative difference in subclinical synovitis, remains open to speculation and would require the study of matched synovial biopsy tissue in these patients.

Interleukin-27

In this study, circulating levels of IL-27 at baseline is predictive of outcome following DMARD cessation, with high levels at baseline demonstrating particular discrimination for those patients who experienced an arthritis flare. Furthermore, there is a significant though small increase in circulating levels of IL-27 at the time of flare (mean fold change 1.07, unadjusted p=0.021). Taken together, these results imply a positive association between circulating IL-27 and arthritis flare. This would be in keeping with pro-Th1 actions of IL-27, though it is impossible to infer causality based on these data. Indeed, higher levels of IL-27 at baseline may serve to modulate the response to upstream perturbations of the immune response following DMARD cessation rather than play a causative role per se, for example by altering the signalling of other STAT-pathway cytokines such as IL-6. The robust increase in IL-6 observed at the time of flare in this study (discussed further below) lends some support to this hypothesis. Alternatively, it is even conceivable that high IL-27 may reflect a regulatory response to pro-inflammatory mechanisms, but that this regulatory response is somehow deficient in those patients who subsequently develop an arthritis flare.

Longitudinal Cytokine Data

Longitudinal observation of trends in circulating cytokine levels provides a tantalising insight in to the mechanisms underlying the emergence of RA flare. The most striking observation was that significant changes in cytokine levels were only observed in the flare group, whereas those patients who maintained DFR demonstrated very little, if any, longitudinal change in cytokine levels. This presumably reflects the substantial dysregulation of immunity that occurs at the time of arthritis flare, which is readily detected in the peripheral circulation. In contrast, the continued immune homeostasis underlying those who maintain DFR is likely to generate more subtle signals, which may furthermore be detectable only at the individual joint level.

The acute-phase proteins CRP and serum amyloid A (SAA) demonstrated the greatest increase from baseline to time of flare (mean fold change 2.16 [adjusted p=0.008] and 2.10 [adjusted p=0.035] respectively). Taken together, the observed increased concentrations of CRP and SAA at the time of flare in this study provide biochemical evidence of a robust systemic inflammatory response in patients who experienced an arthritis flare.

Further analysis of longitudinal cytokine trends in this study yields additional insights beyond only an increase in acute-phase response. IL-6 concentration was higher at the time of flare than baseline (mean fold change 1.50, corrected p=0.031) despite many results falling below, and thus being assigned the value of, the lower limit of detection for the assay. Given this technological limitation in assay performance, it is likely that the true difference in IL-6 concentration between baseline and flare is actually greater than that observed.

The observation that IL-6 levels rise in the approach to arthritis flare in this study is in keeping with the well-established role of IL-6 in the pathogenesis of the disease. Additionally, co-stimulation of Toll-like receptor 3 (TLR3) and TLR7 pathways has been observed to lead to transcriptional synergy of IL-6 and IL-12 in mouse macrophages, an effect mediated by interaction of the transcription factors Jun B. CCAAT/enhancer-binding protein beta (C/EBPβ) and IRF1 at the IL6 and IL12b promoters (Liu et al., 2015). This effect may be of relevance given the statistically significant though small increase in circulating IL-12 at the time of flare in this study (fold change 1.15, adjusted p<0.008).

Circulating concentration of the chemokine interferon-γ inducible protein 10 (IP10, also known as CXCL10) was significantly higher at the time of flare (fold change 1.36, adjusted p=0.043). Circulating concentrations of IP-10 are higher in RA patients versus healthy controls and correlate with disease activity (Kuan et al., 2010; Pandya et al., 2017).

The observation of higher circulating levels of IP-10 at the time of flare in this study would therefore be in keeping with an active flare phenotype where activated lymphocytes are recruited to the inflamed synovium. Furthermore, the postulated role of IP-10 in amplifying the recruitment of Th1 cells may be of significance given the increased levels of the pro-Th1 cytokine IL-12 at the time of flare.

Summary

Circulating concentrations of IL-27, MCP1 and CRP at baseline were predictive of flare and remission following DMARD cessation. Longitudinal analysis demonstrated increased levels of acute phase reactants (CRP and SAA) at the time of flare, together with the pro-Th17 cytokine IL-6, and the pro-Th1 mediators IL-12 and IP-10. Taken together, one can hypothesise a potent pro-inflammatory cytokine and chemokine milieu at the time of arthritis flare, characterised potentially by dysregulated Th17 and Th1 responses. The conservative approach of this study is likely to have artificially reduced the magnitude of observed increases in cytokine concentrations, such as that observed for IL-6 for example.

### Example 4—CD4+ T Cell RNAseq Results

Circulating CD4+ T cells provide a potential cellular compartment that is both important in RA pathobiology as well as easily obtainable without the need for invasive sample collection procedures. In previous studies, a protocol was developed to isolate CD4+ T cells from whole blood with a high cell yield and purity (Pratt, 2011). Herein, the transcriptional profiling of circulating CD4+ T cells unexpectedly represents a target to identify cell-specific biomarkers of drug-free remission.

Patient Samples

CD4+ T cell samples were available at all baseline, month three and month six study visits. In total. CD4+ T cell RNA was available for sequencing for 120/154 (78%) of study visits. CD4+ T cell RNA was available for sequencing at the time of flare for 18/23 (78%) patients who experienced an arthritis flare and was available at month six for 15/20 (75%) patients who maintained drug-free remission.

Time from blood draw to start of CD4+ T cell processing, and time from start of T cell processing to freezing of T cell lysate, were recorded for 117/120 of the sequenced patient samples. The median (IQR, range) time from blood draw to start of processing was 95 (80-150, 40-388) minutes. The median (IQR, range) time from start of processing to freezing of lysate was 242 (217-285, 166-333) minutes.

The number of cells in every T cell isolate was recorded to calculate the cell yield, with a median (IQR, range) value of 2.3×105 cells per ml in whole blood (1.8-3.2, 0.5-7.2). The purity of every CD4+ T cell isolate was confirmed by flow-cytometric analysis as described (see methods above). The median (IQR, range) percentage of CD3+CD4+ cells in patient T cell isolates was 98.9 (98.3-99.3, 95.3-99.8). The percentage of contaminating monocytes and B-cells, defined as CD14+ or CD19+ respectively, were generally low. To account for small variations in CD4+ T cell purity, the percentage of CD3+CD4+ cells was included as a covariate in gene expression models together with sequencing batch.

Healthy Controls

Four healthy volunteers were recruited to donate blood as a control arm for comparison with the study population. The volunteers were aged 50 years (male), 40 years (male), 31 years (female) and 31 years (female) at the time of first donation. Sampling was performed at four time points (baseline, month 1, month 3 and month 6) from all participants giving 16 healthy CD4+ T cell samples in total. Blood was drawn where possible between 9 am-1 pm and left to stand before processing to mimic the collection and transport of the patient samples. Time from blood draw to start of CD4 T cell processing, and time from start of T cell processing to freezing of T cell lysate, were recorded for all samples. The median (IQR, range) time from blood draw to start of processing was 67 (56-90, 28-198) minutes. The median (IQR, range) time from start of processing to freezing of lysate was 254 (231-305, 170-333) minutes. There was no significant difference in mean blood draw or processing times between healthy controls and patients (p=0.141 and p=0.538 respectively, Mann-Whitney U test).

The median (IQR, range) cell yield for healthy control T cell isolates was 3.1 (2.3-3.7, 1.2-5.0)×105 cells per ml whole blood. The mean cellular yield was not significantly different between healthy controls and patients (p=0.267, Mann-Whitney U test). The median (IQR, range) percentage of CD3+CD4+ cells in healthy control T cell isolates was 98.1 (97.6-98.9, 96.8-99.2). The mean percentage CD3+CD4+ cells was not significantly different between healthy controls and patients (p=0.156, Mann-Whitney U test). The percentage of contaminating monocytes and B-cells, defined as CD14+ or CD19+ respectively, was also determined. Significantly greater mean CD19+ contamination was seen for healthy controls versus patients (p=0.022). The explanation for this is not immediately apparent. Where data were available, the mean yield of CD19+ cells in parallel PBMC isolations was not significantly different between patients and healthy controls (p=0.843, Mann-Whitney U test). This suggests that the higher CD19′ contamination in healthy T cell isolates was not due to a greater number of CD19+ cells in healthy control versus patient blood—however, an absolute CD19+ cell count in whole blood, as opposed to the CD19+ cell yield following density centrifugation, would be required to confirm this.

RNA Yield and Integrity

The quantity and quality of RNA in each T cell lysate was measured by gel electrophoresis using a Tapestation™ 4200 machine (Agilent). The median (IQR, range) RNA yield was 870 (660-884, 277-2275) ng per million cells lysed. Prior to sequencing, the 28S/18S ribosomal RNA (rRNA) ratio and estimated RNA integrity number (RINe) were calculated for each sample. Lower RINe and 28S/18S ratios suggest RNA degradation; the RNAseq platform manufacturer (llumina) recommends a RINe>8 and a 28S/18S rRNA ratio >2.0 for high quality RNAseq analysis. The median (IQR, range) RINe was 9.4 (9.2-9.6, 8.1-10.0). The median (IQR, range) 28S/18S rRNA ratio was 2.6 (2.5-2.7, 2.1-2.9). There was no correlation between RINe and total time from venepuncture to freezing of T cell lysate (repeated measures R2=−0.023, p=0.833), suggesting that the RNA integrity was not affected by laboratory processing time during the T cell isolation procedure. There was no significant difference in mean RINe between patients and healthy controls (p=0.104, Mann-Whitney U test).

Sequencing Quality

Samples were sequenced to a mean (range) depth of 12.3 (9.2-18.4) reads per sample. Sequencing read length was tightly clustered around the expected 75 bp, and no samples were found with adaptor contamination >0.1%. Quality of the sequencing was excellent with a mean Phred score >30 across all read positions and no read trimming was necessary.

Baseline Analyses

All analyses were performed with adjustment for RNA sequencing batch and sample CD4+ T cell purity (see above). Change in gene expression was considered significant if there was >1.5 fold-difference in expression between groups at the 0.05 significance level according to the moderated t-test, after false-discovery rate (FDR) correction using the Benjamini-Hochberg procedure. Where changes in gene expression failed to meet statistical significance, unadjusted p-values were used as in a secondary exploratory analysis with a post hoc significance threshold of p<0.001.

Flare Versus Healthy Control

CD4+ T cell gene expression was compared between baseline samples of patients who flared following DMARD cessation versus healthy controls. Three genes were differentially expressed after FDR adjustment (FIG. 9A), and a further 55 were differentially expressed using an unadjusted significance threshold of p<0.001 (FIG. 9B). The entire 58-gene list is detailed below:

In embodiments of the invention, the levels of the one or more biomarkers may comprise expression levels of at least 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20 or more genes selected from any one or more of the genes in the Table above.

In certain embodiments, the levels of the one or more biomarkers may comprise expression levels of at least 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19 or all of ENSG00000226029, ENSG00000112139, ENSG00000247311, ENSG00000198538, ENSG00000163631, ENSG00000171560, ENSG00000171564, ENSG00000256913, ENSG00000259657, ENSG00000265293, ENSG00000204380, ENSG00000228382, ENSG00000279148, ENSG00000214081, ENSG00000165259, ENSG00000251411, ENSG00000154099, ENSG00000261487 and ENSG00000253676.

In certain embodiments, the levels of the one or more biomarkers may comprise expression levels of at least 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19 or all of ENSG00000106927, ENSG00000226029, ENSG00000112139, ENSG00000247311, ENSG00000198538, ENSG00000163631, ENSG00000171560, ENSG00000171564, ENSG00000256913, ENSG00000259657, ENSG00000265293, ENSG00000204380, ENSG00000228382, ENSG00000279148, ENSG00000214081, ENSG00000165259, ENSG00000251411, ENSG00000154099, ENSG00000261487 and ENSG00000253676.

Univariate Cox Regression

Standard bioinformatics pipelines analyse differential gene expression dependent on the presence or absence of a binary outcome measure. Whilst this approach benefits from many years of accumulated knowledge and refined computer packages, it is inherently underpowered in comparison to survival analysis when analysing time-to-event data. Therefore, in a further analysis, the association between gene expression at baseline and time-to-flare following DMARD cessation was analysed across all sequenced genes using univariate Cox regression. Using this approach, 19 genes were identified that were significantly associated with time-to-flare by a post hoc unadjusted p-value threshold of <0.001 (FIG. 14).

None of the genes were robust to multiple test correction, although the Benjamini-Hochberg procedure appeared particularly conservative in its correction relative to the two-way analyses (Table below). No significant departure from proportional hazards was observed for any of these 19 genes.

In preferred embodiments of the invention, the levels of the one or more biomarkers may comprise expression levels of at least 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18 or 19 of these 19 genes.

Multivariate Cox Regression

The 19 genes that were significantly associated with time-to-flare at unadjusted significance level of <0.001 by univariate Cox regression were entered into multivariate Cox regression model. Stepwise backward selection based on Akaike Information Criterion (AIC) was then performed in order to reduce the number of variables to a practical size for the purposes of a biomarker signature, where a lower AIC score indicates a better model (see above). After five selection steps, 14 variables remained in the preliminary backwards stepwise multivariate Cox model (Table below), with a reduction in AIC from 86.97 to 79.42. The 13 variables in this model with a p value <0.2 were then taken forward to a second round of stepwise backward selection. Although this 13 variable model had an AIC slightly greater AIC (79.81) than the 14 variable model, this could be reduced further by an additional two selection steps. This generated a final stepwise multivariate Cox regression model with 11 variables and an AIC (77.69) that was lower than the 14 variable model (Table below). Proportionality of hazards was demonstrated for all variables in both the 14-variable and 11-variable stepwise models, and for both models as a whole.

In preferred embodiments of the invention, the levels of the one or more biomarkers may comprise expression levels of at least 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13 or 14 of these 14 genes.

In preferred embodiments of the invention, the levels of the one or more biomarkers may comprise expression levels of at least 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 or 11 of these 11 genes.

ROC Analysis

Based on a p value threshold of <0.001, three variables were selected from this final stepwise multivariate model, namely: ENSG00000228010, ENSG00000162636 and ENSG00000227070. Candidate genes were explored for their biomarker utility either alone, or in combination with each other—when combined, expression values for each gene were weighted by the respective coefficient in the final stepwise Cox regression model. Patients were dichotomised for each gene score using two thresholds determined by ROC analysis optimised for the prediction of flare and remission, as previously discussed above (see Table below).

The ROC curve for each variable, together with the survival curves for dichotomised groups based on flare and remission biomarker thresholds are presented in FIG. 15. Based on overall AUC, itis clear to see that the composite scores provided greater predictive utility than any of the single gene expression variables alone. The prognostic performance of the composite scores are detailed further in the Table below.

Overall, the 3-gene composite score performed the best for prediction of both flare and remission and discriminated patients with significantly different flare-free survival times (Formula below and FIG. 16).

The 3-gene composite biomarker score, based on log-transformed gene expression values:

Composite score=6.97(ENSG00000162636)+1.78(ENSG00000227070)−4.15(ENSG00000228010)−1.22(ACR/EULAR Boolean remission)

In even more preferred embodiments of the invention, the levels of the one or more biomarkers may therefore comprise expression levels of at least 1, 2 or 3 of the genes selected from ENSG0000022810, ENSG00000162636 or ENSG00000227070.

Sensitivity analysis Incorporating processing time During the laboratory isolation procedure, CD4+ T cells are alive until the final cell lysis step. The transcriptional profiles of the cells may therefore be influenced by the isolation procedure itself. In order to ascertain the effect of this upon the composite biomarker score, a sensitivity analysis was performed whereby the three genes from the final composite score were entered in to a multivariate Cox regression model with the addition of total time from venepuncture to freezing of CD4+ T cell lysate. Total processing time showed no significant association with time-to-flare, whereas the three genes remained strongly associated independent of processing time (Table below). Proportionality of hazards was demonstrated for all four variables and the model a whole.

Baseline comparison of flare versus drug-free remission patients—discussion In the primary analysis of the gene expression data of this study, CD4+ T cell gene expression was compared at baseline between those patients who experienced an arthritis flare versus those who remained in DFR following DMARD cessation. Although substantial number of differentially expressed genes (DEGs) were observed, none of these were robust to multiple test correction. Using post hoc unadjusted p-value thresholds of <0.001 and <0.01, 11 and 118 genes respectively were identified as differentially expressed at the pre-specified >1.5 fold-change level. Pathway analysis of the all 118 DEGs identified 12 as functioning within a network of genes involved in cell cycle, cell death and inflammatory response processes, though with no connecting edges between these identified nodes.

Flare patients—flare versus baseline visits The comparison of CD4+ T cell gene expression between flare and baseline visits revealed two DEGs that were robust to multiple test correction, and 81 DEGs at the unadjusted p<0.001 threshold. These DEGs included up-regulation of genes encoding microtubular and centrosomal proteins, together with topoisomerase-IIα, all of which are known to play crucial roles in the cell cycle. The most significantly up-regulated gene by p-value was cell division cycle associated 7 (CDCA7). MKI67, which encodes the cell-surface molecule Ki-67, was also up-regulated at the time of flare in these cells.

Several genes not directly involved in the machinery of the cell cycle were also observed to be up-regulated at the time of flare, such as low-density lipoprotein receptor (LDLR), thyroid stimulating hormone receptor (TSHR), basic leucine zipper ATF-like transcription factor (BATF), and CD109. The observation of increased LDLR expression by CD4+ T cells at the time of flare may represent a regulatory response that could be perturbed in the patients who experience an arthritis flare.

DFR Patients—Month Six Versus Baseline Visits

Nineteen DEGs were observed at an unadjusted p threshold of <0.001, with substantially lower mean expression values than the 81 DEGs observed at the same significance threshold in the flare vs. baseline analysis. Furthermore, many of DEGs observed in this remission analysis were pseudogenes of unknown function. Functional analysis suggested downregulation of genes within a network of cell signalling and injury, though with very few edges between the identified nodes. Perplexingly, 3/9 of the protein-coding DEGs were immunoglobulin genes, which was unexpected in this CD4+ T cell analysis. This may represent genuine low-level CD4+ T cell expression of immunoglobulin-related genes.

Another unexpected observation was the downregulation of IL10 at month six versus baseline in DFR patients (log2FC −0.89, unadjusted p 5.14×10−4). This was mirrored by a trend towards higher IL10 expression at the time of flare vs. baseline in those patients who experienced an arthritis flare (log2FC 0.68, unadjusted p=0.004). Given the well-established role of IL-10 as a predominantly immunoregulatory cytokine of crucial importance to the function of Tr cells (Pot et al., 2011), it is surprising that its expression by CD4+ T cells would decrease with time in those patients who maintain DFR. Nevertheless, there are two conceivable explanations for this observation. First, IL-10 is not exclusively immunoregulatory, and its role in promoting B cell proliferation is thought to be of importance in the pathogenesis of some autoimmune diseases such as SLE (Peng et al., 2013). Thus, reducing IL-10 production by CD4+ T cells could help maintain remission in certain settings. Alternatively, lower expression of IL-10 could represent a gradual shift away from a regulatory Tr1-like phenotype and towards an effector state. However, the absence of any of the DEGs identified in the flare vs. baseline comparison would suggest that if this were occurring, then the CD4+ T cells are unlikely to be following a similar differentiation pathway to that observed in flare patients. Indeed, the functional relevance of this observed reduction in CD4+ T cell IL-10 production with time in the DFR patient population is hard to interpret in the absence of measures of inflammation and gene expression within the synovial compartment.

Predictive Biomarker Survival Analysis

Using a similar approach as previously, univariate followed by multivariate Cox regression was performed using the RNAseq data in order to develop a predictive biomarker of DFR and flare following DMARD cessation. Cox regression was selected as the analysis model of choice, as this allowed for use of a time-to-event outcome measure, rather than the binary grouping to flare versus DFR described above. This was anticipated to yield greater statistical power —indeed, unadjusted p values were on average 10-fold smaller for the Cox regression analysis compared to the baseline flare vs. DFR comparison. However, standard bioinformatics pipelines for survival analysis using RNAseq data do not currently exist, and adjustment for false-discovery rate using the standard method of Benjamini-Hochberg appeared particularly conservative when applied to the univariate Cox regression results. The issue of apportioning statistical significance in Cox regression analysis when using high dimensionality data has been highlighted in the published literature (Witten and Tibshirani, 2010).

Univariate Cox regression identified a similar set of genes associated with time-to-flare at the unadjusted p<0.001 threshold as compared to the standard analysis pipeline of DEGs between flare vs. DFR groups, thus providing a degree of internal validation of this approach. After stepwise backward multivariate Cox regression modelling, three genes were identified whose expression significantly associated with time-to-flare at the p<0.001 threshold: ENSG00000228010, ENSG00000162636 and ENSG00000227070. One of these genes (ENSG00000162636) encodes the protein family with sequence similarity 102 member B (FAM102B). Whilst no publications exist concerning the function of FAM102B, the paralogous FAM102A (also known as Early Oestrogen-Induced Gene 1) is known to be involved in oestrogen signalling (Wang et al., 2004) as well as osteoclast differentiation (Choi et al., 2013), and is implicated in cell membrane trafficking (Zhang and Aravind, 2010). Both ENSG00000228010 and ENSG00000227070 are predicted to be novel antisense genes, though no published data relates to their putative targets or physiological function. In a composite score, these three genes predicted flare and sustained DFR following DMARD cessation, with an ROCAUC of 0.90.

In certain embodiments of the invention, the levels of the one or more may comprise expression levels of at least 1, 2 or all 3 of the genes selected from ENSG00000228010, ENSG00000162636 and ENSG00000227070.

Summary

The most illuminative results from the analysis of CD4+ T cell gene expression have come from the longitudinal comparison of flare visit with baseline. This analysis revealed a strong signature of upregulation of genes involved in cellular proliferation, as well as the pro-Th17 transcription factor BATF. These results provide evidence of activation of CD4+ T cells at the time of arthritis flare and are in keeping with a phenotype of systemic inflammation in keeping with clinical measures of increased disease activity, and the observed increase in levels of acute-phase and pro-inflammatory cytokines such as IL-6 (see above). Furthermore, comparison of baseline CD4+ T cell gene expression in flare patients versus healthy controls demonstrated upregulation of several genes encoding proteins known to be correlated with disease activity in RA, such as IL-15 and ORM1. This may suggest greater subclinical levels of inflammation in these patients at baseline, thus predisposing to a greater risk of arthritis flare upon DMARD cessation.

Unexpectedly, baseline expression of as few as three genes performed well in discriminating flare vs. DFR when re-applied to the same test cohort. The predictive performance of these genes in combination with clinical and cytokine/chemokine parameters will be addressed in the next Example.

### Example 5—Integrative Analysis

Results are analysed from all variable domains in order to synthesise a global predictive biomarker score to predict RA flare and sustained drug-free remission following DMARD cessation. The motivations for this integrative analysis are two-fold. First, by combining variables from different domains it is aimed to create a global biomarker score that outperforms any of the single-domain composite scores. Second, the process of variable combination can be expected to lead to variable redundancy, thus allowing for the final variable set to be smaller than the sum of the individual domain variable sets.

In performing this integrative analysis, a two-step variable reduction incorporating univariate approach is followed by stepwise backward multivariate Cox regression modelling. Receiver-operating characteristic (ROC) curve analysis is then used to further refine the variable set and select an optimum model for use as predictive biomarkers of flare and sustained remission following DMARD cessation.

Variable Selection

Before starting the process of integrative analysis, a reduced variable set was defined for exploration. The number of variables was reduced not only to avoid unnecessary and laborious downstream model reduction, but also to minimise over-fitting of the expansive data set to the relatively small study population.

Baseline variables were selected based upon their statistical significance in the domain-specific backward stepwise multivariate Cox regression models described in the previous Examples. Only those variables which were associated with time-to-flare in their respective multivariate models at an unadjusted significance threshold of p<0.05 (or p<0.001 in the case of gene expression data) were selected for integrative analysis. Thresholds were not set for the Cox regression coefficients, as these were expected to change with the merging of variable domains in the integrative analyses.

In total, 11 variables were selected for integrative analysis as detailed in the Table below.

Cox Regression

The association between baseline variables and time-to-flare following DMARD cessation was analysed by univariate Cox regression for the 43 patients where complete data was available (Table below). No significant deviation from proportionality of hazards was observed for any of the univariate variables. Given the univariate nature of this analysis, the coefficients and statistical significance of these variables mirror those already detailed in the previous Examples—minor discrepancies reflect the exclusion of the single patient without cytokine data.

All 11 baseline variables were then entered in to a multivariate Cox regression model. Stepwise backward selection based on Akaike information criterion (AIC) was then performed to fit a stepwise Cox regression model (see above). After four selection steps, 7 variables remained in this stepwise model (Table below).

Proportionality of hazards was again assessed for each variable in the final stepwise multivariate Cox regression model. A significant departure from proportional hazards was observed only for current methotrexate use (p=0.006), though as noted before in Example 3, this was only notable for a single outlier with no discernible trend in the remainder of the data. The global Schoenfeld test was non-significant (p=0.087), indicating proportionality of hazards for the model as a whole.

Receiver-Operating Characteristic (ROC) Analysis

Five variables were significantly associated with time-to-flare a tan unadjusted significance threshold of <0.05 in the multivariate stepwise Cox regression model: ENSG00000227070, ENSG00000228010, ENSG00000162636 ln(IL27+1) and baseline ACR/EULAR Boolean remission. Values of these five variables were multiplied by their respective stepwise multivariate Cox regression coefficient and then summed to create composite scores. The predictive performance of all 31 potential combinations of these variables to predict flare and remission following DMARD cessation was then compared by area under the receiver-operating characteristic curve (ROCAUC). The ten composite scores with the highest ROCAUC are shown in the Table below.

The composite score with the highest ROCAUC included all variables; notably, the removal of In(IL27+1) resulted in only a small drop in ROCAUC (Formulae below and FIG. 17).

Composite score=1.14(ENSG00000227070)+2.82(ENSG00000162636)+1.92(ln[IL27+1])−1.98(ENSG00000228010)−1.22(ACR/EULAR Boolean remission)  Five-variable composite score

Composite score=1.14(ENSG00000227070)+2.82(ENSG00000162636)−1.98(ENSG00000228010)−1.22(ACR/EULAR Boolean remission)  Four-variable composite score

Composite Score Predictive Performance

The sensitivity, specificity, positive predictive value (PPV) and negative predictive value of each of the three composite scores detailed above are presented in the Table below and FIG. 18, using the same threshold values as in the main analysis. A single optimum threshold for each composite score was selected manually based on minimising the ROC coordinate distance from the top left corner of the ROC curve plot.

Discussion

In this integrative analysis, variables from the composite biomarker scores of clinical, cytokine and RNAseq domains were combined together in a multivariate Cox regression model, with backwards stepwise selection used to create a final model with five variables: three gene expression (ENSG00000228010, ENSG00000162636 and ENSG00000227070), one cytokine (IL-27), and one clinical (ACR/EULAR Boolean remission). Whereas the role of the three gene variables remains unknown, the relevance of IL-27 and ACR/EULAR Boolean remission in the context of DFR has been identified previously.

When re-applied to the study population, this 5-variable composite score demonstrated a high predictive utility for outcome following DMARD cessation: ROCAUC 0.96 (95% Cl 0.92-1.00), sensitivity 0.91 (0.78-1.00), specificity 0.95 (0.84-1.00), PPV 0.96 (0.87-1.00). NPV 0.90 (0.78-1.00). Thus, in this study population, patients with a negative test score had a 90% chance of remaining in DFR at the end of the six-month follow-up period, versus only 4% for those with a positive score. Such a score would undoubtedly be of great utility in helping guide DMARD withdrawal in the clinic and would represent a quantum leap beyond the approximately even chance of flare that would otherwise be predicted in the absence of a predictive biomarker based on outcome of the entire non-stratified study population.

Throughout the description and claims of this specification, the words “comprise” and “contain” and variations of them mean “including but not limited to” and they are not intended to (and do not) exclude other moieties, additives, components, integers or steps. Throughout the description and claims of this specification, the singular encompasses the plural unless the context otherwise requires. In particular, where the indefinite article is used, the specification is to be understood as contemplating plurality as well as singularity, unless the context requires otherwise.

Features, integers, characteristics or groups described in conjunction with a particular aspect, embodiment or example of the invention are to be understood to be applicable to any other aspect, embodiment or example described herein unless incompatible therewith. All of the features disclosed in this specification (including any accompanying claims, abstract and drawings), and/or all of the steps of any method or process so disclosed, may be combined in any combination, except combinations where at least some of the features and/or steps are mutually exclusive. The invention is not restricted to any details of any foregoing embodiments. The invention extends to any novel one, or novel combination, of the features disclosed in this specification (including any accompanying claims, abstract and drawings), or to any novel one, or any novel combination, of the steps of any method or process so disclosed.

The reader's attention is directed to all papers and documents which are filed concurrently with or previous to this specification in connection with this application and which are open to public inspection with this specification, and the contents of all such papers and documents are incorporated herein by reference.

