# Introduction

In the last two decades, significant progress has been made in diagnosing and treating human cancers. However, cancers remain one of the most frequent causes of death in the United States. In 2017, 1,735,350 new cancer cases were diagnosed in the United States1. More than 600,000 cancer deaths are projected to occur in the United States in 2018: a death rate second only to cardiovascular diseases. Among human cancers, lung, prostate, breast, liver and colorectal cancers appear the most frequently, accounting for approximately 49% of all human cancers. These five types of cancers are projected to account for 305710 deaths in 2018 or over 50% of all cancer-related deaths in the US. Thus, understanding the mechanisms underlying the development of these cancers is crucial to reduce cancer mortality in the country.

Genome abnormalities are widely present in human cancers2. These abnormalities include single nucleotide mutations, copy number changes, chromosomal rearrangement, etc. Indeed, cancer genome abnormalities precede the development of cancer phenotypes3–6. Non-malignant tissues adjacent to cancers have been shown to contain similar genomic and transcriptomic changes as neighboring cancer tissues3–11. One of the salient abnormalities in the cancer genome is chromosomal rearrangements, which may result in the joining of 2 unrelated genes in the chromosome to produce a fusion gene. The most well-characterized example of fusion gene is the Philadelphia chromosome12 that joins the N-terminus of BCR with the tyrosine kinase domain of ABL13. The resulting chimeric protein has constitutively activated tyrosine kinase activity and transforms benign tissue into malignant one14. Several cancer-specific fusion genes have been discovered in prostate cancer samples15–17. Some of these fusion genes appear to be transforming18,19. Interestingly, one of the fusion genes, MAN2A1-FER, was found in 5 other types of human malignancies, and has been shown to transform normal livers into hepatocellular carcinomas in a short period of time19. SLC45A2-AMACR was found independently in bladder cancer20 and lung cancer cell lines21. These findings suggest that fusion genes may have wider implications than initially anticipated. To investigate whether fusion genes play a role in other human malignancies, we analyzed six fusion genes, including TRMT11-GRIK2, MTOR-TP53BP1, CCNH-C5orf30, KDM4-AC011523.2, TMEM135-CCDC67, and LRRC59-FLJ60017, in primary cancer samples from 7 different types of human malignancies and 20 cancer cell lines originating from 6 human cancers. These fusion genes are present in human cancers with variable frequencies, suggesting a much wider role for these cancer-specific fusion genes in the development of human malignancies.

# Materials And Methods

## Tissue Samples

The 536 tissue specimens used in the study consist of 101 non-small cell lung cancers, 61 ovarian cancers, 60 colon cancers, 70 liver cancers, 150 glioblastoma, 60 breast cancers, and 34 esophageal adenocarcinomas. These samples were obtained from the University of Pittsburgh Tissue Bank in compliance with institutional regulatory guidelines (Supplemental Table 1 through 7). The informed consent exemptions and protocol were approved by the Institution Review Board of University of Pittsburgh. Cancer cells were obtained by macro-dissection. Esophageal cancer specimens were from a prospective IRB approved protocol from University of Pittsburgh and were frozen tissue samples. Sixteen non-small cell lung cancer samples were obtained from the University of Kansas. Twenty-eight non-small cell lung cancer samples were obtained from the University of Iowa. All informed consent exemptions and protocols were approved by the Institution Review Board of the University of Kansas or University of Iowa. All 20 cell lines used in the study were purchased from the American Type Cell Culture (ATCC, Inc., Manassas, VA, USA) and were cultured and maintained following the manufacturer’s recommendations.

## Rna Extraction, Cdna Synthesis And Detection Of Fusion Genes

Formalin-fixed paraffin-embedded (FFPE) tissue blocks of each sample were cut for multiple unstained slides. One of these slides was stained with hematoxylin and eosin. The cancer regions were circled by pathologists and macrodissected. Total RNA was extracted using trizol to lyse the cancer tissues (Invitrogen, CA). First strand cDNA was synthesized using ~2 µg of RNA from each sample, random hexamers and Superscript IITM (Invitrogen, Inc, CA) at 42 °C for 2 hours. One microliter each cDNA sample was used for TaqMan PCR reactions with 50 heat cycles at 94 °C for 30 seconds, 61 °C for 30 seconds, and 72 °C for 30 seconds using primers and probes specific for CCNH-C5orf30 (AAAGTTATTTATCAGAGAGTCTGATGCTG/CTGTTCTACTCCAGGTATTTTCATTATATC; TaqMan probe, 5′-/56-FAM/ACAGGCAAG/ZEN/TTCTGTTCTCTTTCAGCA/3IABkFQ/-3′), mTOR-TP53BP1 (TGATAGACCAGTCCCGGGATG/CCACTGACATTCCCAGAACAAG; TaqMan probe, 5′-/56-FAM/TGTCAGCCT/ZEN/GTCAGAATCCAAGTCAAG/3IABkFQ/-3′), TRMT11-GRIK2 (GCGCTGTCGTGTACCCTTAAC/GAATGCAAGTTCCTCAGCTCC; TaqMan probe, 5′-/56-FAM/CGGAACTCC/ZEN/AGATGCTCCTGCG/3IABkFQ/-3′), LRRC59-FLJ60017 (GTGACTGCTTGGATGAGAAGC/CCCTCCTCTGGTTTGTTGTTG; TaqMan probe, 5′-/56-FAM/CAGTGTGCA/ZEN/AACAAGGTGACTGGAAG/3IABkFQ/-3′), TMEM135-CCDC67 (CAGCTGTCATGGAAGTTCAGAC/CCTCATTCTTTCCTGCTCAGAG; TaqMan probe, 5′-/56-FAM/AGTTCCTTT/ZEN/TAAGACTCACCAAGGGCAA/3IABkFQ/-3′), KDM4- AC011523.2 (AGACCACCTTCGCCTGGCAC/TCTCTCTCAGATCCAGGCTTG; TaqMan probe, 5′-/56-FAM/ACAGCATCA/ZEN/ACTACCTGCACTTTGGG/3IABkFQ/-3′), and β-actin (ACCCCACTTCTCTCTAAGGAG/GCAATGCTATCACCTCCCCTG; TaqMan probe, 5′-/56-FAM/CCAGTCCTC/ZEN/TCCCAAGTCCACAC/3IABkFQ/-3′) in a thermocycler (Eppendorf RealplexTM thermocycler). A negative control and synthetic positive control were included in each batch of reactions. The PCR products were gel purified and Sanger-sequenced on 20% positive samples.

# Results

In our previous studies16,17, we have characterized eight fusion genes identified in aggressive prostate cancer samples. Additional analyses showed that one of the fusion genes, MAN2A1-FER, is frequently present in 5 other types of human malignancies19. To expand our analyses of other fusion genes in the panel, we analyzed TRMT11-GRIK2, MTOR-TP53BP1, CCNH-C5orf30, KDM4-AC011523.2, TMEM135-CCDC67, and LRRC59-FLJ60017 in 20 cancer cell lines from six different human malignancies (Fig. 1). The TRMT11-GRIK2 fusion transcript was identified in the MD-MB-231breast cancer cell line and H1299 lung cancer cell line, while CCNH-C5orf30 was positive in 14 of 20 cancer cell lines, including all the prostate cancer cell lines tested (PC3, DU145, LNCaP and VCaP), 3 of 4 breast cancer cell lines (MCF7, VACC-3133 and MDA-MB330), 2 of 4 lung cancer cell lines (H358 and H522), 1 of 3 liver cancer cell lines (HepG2), 1 of 2 colon cancer cell lines (HCT8) and 3 of 3 GBM cell lines (LN229, U138 and A-172). LRRC59-FLJ60017 was also present in the HepG2 liver cancer cell line. These results suggest that these fusion genes are not specific for prostate cancer and may be present in primary cancer samples from a variety of human malignancies.Figure 1Fusion transcripts are present in human cancer cell lines. Representative cancer cell lines from the lung, liver, prostate, breast, colon and brain were examined for the presence of the TRMT11-GRIK2, CCNH-C5orf30, mTOR-TP53BP1, TMEM135-CCDC67, KDM4-AC011523.2 and LRRC59-FLJ60017 fusion transcripts using quantitative TaqMan RT-PCR. Red denotes positive detection of the fusion transcript, while blank indicates negative detection of the fusion transcript. All positive fusion samples were verified by Sanger sequencing (see Supplemental Figs 1–6).

To investigate whether any of the above-mentioned fusion genes has a role in human cancers, quantitative TaqMan qRT-PCR reactions using primers and probes specific for each fusion gene were performed on primary human cancer samples representing seven different types of human malignancies. Our results showed that TRMT11-GRIK2 is present in all seven types of human malignancies (Table 1, Fig. 2 and Supplemental Figs 1–6)), including breast cancer (41/60, 68.33%), colon cancer (25/60, 41.7%), esophageal adenocarcinoma (9/34, 26.5%), hepatocellular carcinoma (9/70, 12.9%), ovarian adenocarcinoma (28/61, 45.9%), glioblastoma multiforme (26/150, 17.3%) and non-small cell lung cancer (39/141, 27.7%). The CCNH-C5orf30 transcript was also detected in 7 different types of human cancers with the following relatively high frequencies: 85% (51/60) in breast cancer, 43% (26/60) in colon cancer, 50.8% (31/61) in ovarian cancer, 67.6% (23/34) in esophageal adenocarcinoma, 41.8% (59/141) in non-small cell lung cancer, 37% (26/70) in liver cancer and 53% (80/150) in glioblastoma multiforme. In contrast, mTOR-TP53BP1was only detected in five different types of human malignancies and with significantly lower frequencies, including breast cancer (10/60, 16.7%), colon cancer (4/60, 6.7%), ovarian adenocarcinoma (4/61, 6.6%), glioblastoma multiforme (7/150, 4.7%) and lung cancer (8/141, 5.7%). LRRC59-FLJ60017 was present in four different types of human cancers, esophageal adenocarcinoma (3/34, 8.8%), ovarian adenocarcinoma (4/61, 6.6%), glioblastoma multiforme (16/150, 10.7%), and non-small cell lung cancer (33/141, 23.4%). Only two glioblastoma multiforme and one lung cancer samples were positive for the TMEM135-CCDC67 fusion gene. The KDM4-AC011523.2 fusion was only found in 3 breast cancer and 3 lung cancer samples.Table 1Distribution of fusion genes in colon cancer, breast cancer, esophageal adenocarcinoma, liver cancer, ovarian adenocarcinoma, glioblastoma multiforme and non-small cell lung cancer.CancerTRMT11-GRIK2 MTOR-TP53BP1 CCNH-C5orf30 KDM4-AC011523.2 TMEM135-CCDC67 LRRC59-FLJ60017 Breast cancer Ductal Type21/304/3026/300/300/300/30  With LN met7/150/1511/150/150/150/15  Without LN met14/154/1515/150/150/150/15Lobular20/306/3025/303/300/300/30  With LN met12/150/1510/150/150/150/15  Without LN met8/156/1515/153/150/150/15Non-recurrence36/5310/5347/533/530/530/53Recurrence5/70/74/70/70/70/7Stage 121/338/3330/333/330/330/33Stage 211/162/1612/160/160/160/16Stage 36/90/97/90/90/90/9Stage 41/20/22/20/20/20/2Grade 15/82/88/80/80/80/8Grade 226/388/3833/383/380/380/38Grade 310/140/1410/140/140/140/14ER positive34/519/5145/513/510/510/51ER negative7/91/96/90/90/90/9PR positive29/447/4439/443/440/440/44PR negative12/163/1612/160/160/160/16HER2 amp+2/30/33/30/30/30/3HER2 amp−39/5710/5748/573/570/570/57Triple negative7/91/96/90/90/90/9 Colon cancer With LN met14/303/3015/300/300/300/30Without LN met11/301/3011/300/300/300/30Recurrence 5-yr3/51/54/50/50/50/5Non-recurrence22/553/5522/550/550/550/55Stage 11/30/31/30/30/30/3Stage 23/130/132/130/130/130/13Stage 320/423/4221/420/420/420/42Stage 41/21/22/20/20/20/2Grade I1/20/21/20/20/20/2Grade II19/513/5121/510/510/510/51Grade III5/71/74/70/70/70/7 Ovarian cancer With LN met14/303/3018/300/300/302/30Without LN met14/311/3113/310/310/312/31Non-recurrence16/331/3315/330/330/332/33Recurrence 5-yrs12/283/2816/280/280/282/28Survive >5 yrs16/341/3418/340/340/342/34Death from Ca12/273/2713/270/270/272/27Stage 111/221/229/220/220/222/22Stage 22/40/41/40/40/40/4Stage 315/353/3521/350/350/352/35Grade I5/71/72/70/70/70/7Grade II2/80/84/80/80/80/8Grade III16/342/3416/340/340/343/34Grade IV5/121/129/120/120/121/12 Esophageal Adenocarcinoma 9/340/3423/340/340/343/34 Non-small cell lung cancer UPMC cohort33/1017/10131/1010/1011/10112/101Stage 110/243/248/240/240/243/24Stage 219/624/6221/620/621/627/62Stage 32/50/50/50/50/50/5Stage 42/100/102/100/100/102/10With LN met10/392/3912/390/390/395/39Without LN met23/625/6219/620/621/627/62Survival 5-yr9/342/3410/340/340/346/34Death 5 yrs22/675/6721/670/671/676/67Ad Ca0/00/00/00/00/00/0Sq Ca33/1017/10131/1010/1011/10112/101Kansas cohort1/160/1615/160/162/1612/16Stage I0/50/55/50/50/54/5Stage II1/110/1110/110/112/118/11Ad Ca1/80/88/80/80/86/8Sq Ca0/80/87/80/82/86/8Iowa cohort5/241/2413/243/240/249/24Stage I2/80/84/80/80/83/8Stage II2/71/75/72/70/75/7Stage III0/60/63/61/60/61/6Stage IV1/30/31/30/30/30/3With met3/161/167/162/160/166/16Without met2/80/86/81/80/83/8Ad Ca5/231/2313/233/230/239/23Sq Ca0/00/00/00/00/00/0Sarcomatoid0/10/10/10/10/10/1 Liver cancer Total9/700/7026/700/700/700/70Survive 5 yrs5/120/127/120/120/120/12Death4/580/5819/580/580/580/58Non-recurrence5/340/3413/340/340/340/34Recurrence4/360/3613/360/360/360/36 Glioblastoma multiforme 26/1507/15080/1500/1502/15016/150Figure 2Frequency of gene fusion in primary human malignant samples. Real-time TaqMan qRT-PCR reactions were performed on formalin-fixed paraffin-embedded samples from breast cancer, colon cancer, esophageal adenocarcinoma, ovarian cancer, hepatocellular carcinoma, non-small cell lung cancer and glioblastoma multiforme. The number of samples from each cohort are indicated. Sanger sequencing was performed on 20–100% of samples that were found to be positive for the fusion genes (see Supplemental Figs 1–6).

Interestingly, ductal type breast cancers positive for TRMT11-GRIK2 were associated with a lower likelihood of developing local lymph node metastasis (46.7% versus 93.3%, p = 0.014). Liver cancers positive for TRMT11-GRIK2 were also associated with a higher rate of overall survival (41.7% versus 6.9%, p = 0.006). KDM4-AC011523.2 was only detected in lobular type breast cancer and adenocarcinoma of the lung. Patients with lobular breast cancers positive for mTOR-TP53BP1 were also less likely to have lymph node metastasis (0% versus 40%, p = 0.017). CCNH-C5orf30 was more frequent in lung cancer adenocarcinomas versus squamous type (67.7% versus 34.8%, p = 0.001) and colon cancer at advanced stages at the time of diagnosis (52.3% versus 18.8%, p = 0.037).

To investigate whether fusion genes are also present in human cancer metastatic lesions, breast cancer, colon cancer and ovarian cancer samples with matched lymph node metastases were analyzed (Fig. 3). Twenty-six of 30 metastatic breast cancers in lymph nodes were positive for TRMT11-GRIK2, including seven metastatic cancers whose matched primary breast cancers were negative for the fusion. For colon cancers, the matched status of TRMT11-GRIK2 between primary cancer samples and lymph node metastases was 78.5% (11/14Eleven lymph node metastases were exactly matched with the status of the primary colon cancer samples, while two samples of lymph node metastases were found negative for TRMT11-GRIK2 fusion. One lymph node metastasis was found positive for the fusion gene while the matched primary sample was negative (Fig. 3). For ovarian adenocarcinomas, nine metastatic lesions were found to contain the TRMT11-GRIK2 fusion gene, matching all the primary samples. However, four lymph node metastases contained no TRMT11-GRIK2 fusion gene while the matched primary cancer samples were positive. One lymph node metastasis gained the TRMT11-GRIK2 fusion over the primary cancer sample. For CCNH-C5orf30, the matching rate of primary breast cancer with lymph node metastases was 62%, while the matching rates for ovarian cancer and colon cancer with their corresponding lymph node metastases were 72% and 73%, respectively. For mTOR-TP53BP1, two of 3 lymph node metastases retained the fusion in colon and ovarian cancers. Additionally, two of 2 ovarian cancer lymph node metastases retained the LRRC59-FLJ60017 fusion. These results suggest significant heterogeneity among the cancer samples. However, most fusion genes were retained in metastatic lesions.Figure 3Fusion transcripts in metastatic lymph node samples. Real-time TaqMan qRT-PCR reactions were performed on formalin-fixed paraffin-embedded samples from breast cancer (top), ovarian cancer (middle) and colon cancer (bottom) and their corresponding lymph node metastases. Both primary (top) and matched lymph node (bottom) cancer samples are shown. Red indicates positive detection of the fusion transcript, while blank indicates negative detection of the fusion transcript. Sanger sequencing was performed on 20–100% of samples that were found to be positive for the fusion genes (see Supplemental Figs 1–6).

# Discussion

Gene fusions are the result of recombination of two unrelated genes. Fusion events can also involve genes that have similar biological roles (for example, between two genes that both have roles in transcription, such as the ESR1-YAP1 driver fusion in breast cancer)22. Almost all cancer-specific gene fusions are the result of chromosomal rearrangements or translocations17. There is increasing evidence suggesting that gene fusions are some of the key drivers of human cancer development. New gene fusion events have been discovered in prostate cancer15–17, breast cancer23–25, NSCLC19,26,27, colon cancer28–30, glioblastoma multiforme19,31,32, liver cancer19,33–35 and ovarian cancer19,36,37. Some of these fusion genes appear to play driver roles in the aggressive behaviors of these cancers19,23,26,31. Gene fusions produce two possible outcomes for the genes involved. One outcome is a gain of function due to the loss of the regulatory domain in the protein, so the enzymatic domain of the same protein becomes hyper-activated. BCR-ABL and MAN2A1-FER are examples of gain of function fusion genes, leading to hyper-activation of the fusion partner. In-frame fusions can also contribute to disease pathogenesis by creating a fusion protein that contains not only complementary functions encoded by each partner gene, but also has neomorphic properties. ESR1-YAP122, a driver fusion found in advanced breast cancer, generates a hyperactive transcription factor through the combination of the ESR1 part that provides domains necessary for DNA binding, dimerization, and nuclear localization and the YAP1 part that provides components for transcriptional activation. ESR1-YAP1 is able to drive expression of genes that promote metastatic biology, a function that full length wild-type ESR1 lacks. The other outcome is a loss of function due to the truncation of the head gene and/or complete elimination of the open reading frame of the tail gene, such as with TRMT11-GRIK2 and mTOR-TP53BP1. For both these instances, tumor suppressor activities of GRIK238 and TP53BP139 are lost due to the loss of protein translation.

Among the prostate cancer fusion genes identified in our previous study16, two of these fusion genes (SLC45A2-AMACR and MAN2A1-FER) were also found in other types of human cancers19–21, suggesting that these gene fusions are not specific to prostate cancer but may be widely present in human cancers. MAN2A1-FER was found to be the driver for liver cancer in mouse19. The expression of this fusion gene induced spontaneous liver cancer in mice by ectopically phosphorylating the EGFR extracellular domain and activating its signaling pathways19. Although the biological roles of the other fusion genes remain unknown, five of these gene fusion events (TMEM135-CCDC67, mTOR-TP53BP1, LRRC59-FLJ10067, KDM4-AC011523.2 and TRMT11-GRIK2) eliminate the open-reading frames in the tail genes and thus produce gene knockouts. CCDC6740, TP53BP139,41,42 and GRIK238 are tail genes that contain tumor suppressor activity. The TMEM135-CCDC67, mTOR-TP53BP1 and TRMT11-GRIK2 gene fusions are equivalent to the functional deletion of CCDC67, TP53BP1 and GRIK2, respectively. Deletions of these genes have been shown to promote the aggressive behaviors of cancers38–42. These gene fusion events may have significant biological implications in the development of cancer.

Based on the current analyses, TRMT11-GRIK2 and CCNH-C5orf30 are probably some of the most widely distributed gene fusions in human malignancies, being present in at least eight types of human malignancies. TRMT11-GRIK2 has frequencies ranging from 12.9% in liver cancer to 68.3% in breast cancer. This fusion gene is also found in a breast cancer cell line and a lung cancer cell line. CCNH-C5orf30 is also very frequent among different types of cancers and their corresponding cancer cell lines. In contrast, the positive rates of the other four fusion genes are much less frequent. The mechanism underlying the disparity of frequencies of these fusion genes is not clear. Although there is some correlation between the distance between the partner genes and the frequencies (both mTOR-TP53BP1 and LRRC59-FLJ60017 gene fusions have their gene partners located in different chromosomes, while both TRMT11-GRIK2 and CCNH-C5orf30 are located in the same chromosome with distances less than 24 MB), TMEM135 and CCDC67 are only separated by 6 MB, and the TMEM135-CCDC67 fusion is exceeding rare in cancers. Nevertheless, the presence of these gene fusions suggests that chromosomal recombination and translocation are probably some of the most frequent events in human cancers.

TRMT11 is a tRNA methyltransferase. The protein is essential for m2G formation at position 10 in tRNA43. This methylation event is required for tRNA stability and translation activity44. In contrast, GRIK2 encodes a glutamate receptor45 and was shown to possess tumor suppressor activity38. The process of chromosomal recombination between TRMT11 and GRIK2 to create the TRMT11-GRIK2 gene fusion destroys the open-reading frames of both genes and produces functional knockouts of these two proteins. The absence of TRMT11 may produce less efficient and unstable translation of mRNA into protein in cancer cells due to tRNA defects. Alternatively, protein translation may be repressed. The lack of GRIK2 may accelerate cell cycle progression and promote cell migration. Thus, cells with the TRMT11-GRIK2 gene fusion may be unstable and tumorigenic.

CCNH is an important member of the cyclin family. It complexes with cdk7-MAT1 and is a component in the TFIIH and RNA polymerase complexes46,47. Thus, CCNH is a critical regulator for the processes of transcription and cell cycle progression. The CCNH-C5orf30 gene fusion produces a truncated cyclin H protein with the deletion of its H5′ and HC domains. A study showed that CCNH mutants lacking the HC domain do not activate cdk747. As a result, the truncated CCNH from the gene fusion may have a negative impact on the functions of RNA polymerase and TFIIH. C5orf30 was also shown to inhibit the generation of cytokines involved in inflammation, such as TNF and IL1, and promote the expression of anti-inflammatory cytokines, such as IL10, in rheumatoid arthritis synovial fibroblasts48,49. The CCNH-C5orf30 fusion transcript contains an intact C5orf30 opening reading frame. The CCNH-C5orf30 gene fusion places C5orf30 expression under the CCNH promoter and may promote its expression. Over-expression of C5orf30 in cancer cells may help to fend off immune responses targeting the cancers.

The wide presence of the 6 fusion genes in a variety of human malignancies may provide significant utility for clinical cancer diagnosis and therapeutic targeting. The presence of these fusion genes in metastatic cancer samples can be used in clinical follow-up studies to analyze the recurrence of human cancers. If these fusion genes are present in grey zone biopsy samples, it may also help to confirm or to make a correct diagnosis. Furthermore, the chromosomal breakpoints of significant numbers of these fusion genes have been identified. These chromosome breakpoints not only serve as cancer markers but also provide unique opportunities to treat human cancers using genome editing technologies50. When multiple fusion gene breakpoints are present in the same cancer cells, multi-targeting at these chromosomal breakpoints may significantly enhance the efficiency of genome editing treatments targeting the cancer cells.

# Supplementary Information

## 



