# RESULTS

## Design of FLP-FRT-Mediated RMCE

The target QC288A and donor QC329 DNA each contained a FRT1 site and a FRT87 site in the same orientation (Fig. 1, A and B; Tao et al., 2007). The circular QC329 DNA could be integrated into the linear QC288A DNA previously placed in the soybean genome by FLP recombinase-mediated DNA recombination, for example, at the FRT1 site to form a collinear intermediate containing two FRT1 sites and two FRT87 sites. FLP recombinase could then excise the DNA segment between the two FRT1 sites to restore the original target QC288A or excise the DNA segment between the two FRT87 sites to form the recombined RMCE DNA QC288A329 (Fig. 1D). Since the FRT1 and FRT87 sites are not completely incompatible, all DNA between the outmost FRT1 and FRT87 sites could also be excised, resulting in an excision event retaining only the scp1 promoter and a recombined FRT site (Fig. 1E). Depending on the DNA strand crossover position, excision between FRT1 and FRT87 sites could restore either the FRT87 or the FRT1 site (McLeod et al., 1986). If no excision occurred, the intermediate could remain as a simple integration containing all of the QC288A and QC329 components. The FLP recombinase was provided transiently from the expression of the flp construct QC292 (Fig. 1C).

The QC288A DNA contained a selection gene, hpt, driven by the constitutive promoter scp1, and trans-genic events were selected with hygromycin. The QC329 DNA contained a promoterless selection gene, als, that would not be expressed unless a promoter was placed upstream of it. FLP-mediated DNA recombination could bring the promoterless als gene of QC329 downstream of the scp1 promoter of QC288A to form QC288A329 to enable retransformation events being selected with chlorsulfuron. The random integration events of QC329 would not survive the selection unless the promoterless als gene inserted by chance downstream of a native promoter. The fluorescent protein genes yfp in QC288A and cfp in QC329 and QC288A329 were used to screen transgenic events (Fig. 1, A, B, andD).

## Creation and Characterization of Target Events

A total of 82 target events were produced with QC288A DNA. Somatic embryo samples of the events were analyzed by quantitative PCR (qPCR) specific to the scp1, hpt, and yfp genes, regular PCR specific to the 5# end, 3# end, and full-length QC288A, and Southern hybridization with hpt and yfp probes to identify single complete-copy events. A total of 33 such events regenerated into T0 plants, and their leaf samples were analyzed by similar qPCR, PCR, and Southern analyses. Sixteen seeds from one self-pollinated T0 plant of each of 10 selected events (lines) were planted to get T1 plants, which were then analyzed by scp1-, ubiq10-, and yfp-specific qPCR assays to check for segregation. Eight lines produced homozygous T1 plants. Three lines, A, B, and C, were used in this report (Table I).

Genomic DNA fragments bordering the QC288A transgene in six target lines were cloned and sequenced. Genomic DNA sequences 601, 984, and 496 bp bordering the 5# ends of lines A, B, and C, respectively, and sequences 2,588, 1,305, and 543 bp bordering their 3# ends were obtained. The alignment of the border sequences with the QC288A sequence revealed that line A lost 5 bp from the 5# end and none from the 3# end, line B lost 17 bp from the 5# end and 49 bp from the 3# end, while line C lost 22 bp from the 5# end and 11 bp from the 3# end. None of the transgene end losses was long enough to affect the FRT recognition sites. BLASTN search of the border sequences did not yield any significant homology with any known sequences in the National Center for Biotechnology Information database (www.ncbi.nlm.nih.gov).

## Creation and Characterization of RMCE Events

Suspension cultures initiated from the developing embryos of target lines A, B, and C homozygous T1 plants were retransformed with the donor DNA QC329 cobombarded with the flp DNA QC292. Three putative retransformation events resistant to chlorsulfuron from target A, six from target B, and three from target C were screened at the callus stage for the reporter gene cfp expression followed by a common PCR to check DNA recombination at the FRT1 site (Table II). Events B5 and B6 were derived from the retransformation of the original hemizygous target B callus that had never gone through plant regeneration. All events were then evaluated by four constructspecific qPCR analyses (Fig. 1) to check for DNA recombination at the FRT1 site and the presence of the target, donor, and flp DNA (Table II), followed by five border-specific PCR analyses specific to each target line using the 5# border, 3# border, and transgenespecific primers (Fig. 1, A, D, andE) to confirm DNA recombination at and between the FRT1 and FRT87 sites (Fig. 2).

For example, event A1 was positive for both CFP expression and the FRT1 site-specific PCR. The construct-specific qPCR analyses revealed that event A1 had one copy of RMCE, contained two copies of the donor, and was free of either the target or flp (Table II). The border-specific PCR analyses revealed that event A1 was positive for both the 5# end and 3# end assays specific to RMCE (Fig. 2, A andB), negative for either the 5# end or the 3# end assays specific to the target (Fig. 2, C andD), and positive for a small excisionspecific band amplified by the full-length PCR (Fig. 2E). Since one copy of RMCE was simultaneously detected with the excision in the homozygous targetderived event A1, the event had to be an RMCEexcision, with one target converted to RMCE and the other converted to excision. The expected large RMCEspecific band (6,652 bp) of event A1 failed to be amplified by the same full-length PCR due to its competitive disadvantage to the small excisionspecific band (1,307 bp).

Based on similar qPCR (Table II) and border-specific PCR analyses (Fig. 2), event A2 was an RMCE-excision event containing a copy of the donor and flp DNA. Event A3 was a homozygous target escape containing five copies of the donor. The target-specific band of A3 was amplified by the full-length PCR (Fig. 2E). Events B1, B2, and B4 were clean RMCE-excision events containing no additional donor or flp DNA insertions. Event B3 was an RMCE-excision containing a donor. Event B5 was an RMCE-wt (for wild type) event, since it was derived from the retransformation of the original hemizygous target B callus. Accordingly, no excision band was detected in event B5 by the full-length PCR even though the large RMCE-specific band was amplified (Fig. 2E). The 1.0 copy of the target in event B5 probably was a partial copy, since the target borderspecific PCR did not detect it (Fig. 2, C and D). Event B6 was a hemizygous target escape, also containing the partial target, since 1.8 copies of the target were detected. The detection of the partial target in events B5 and B6 suggested that the original target B callus was chimeric. Event C1 was an RMCE-excision event with some cells still containing the target detected as 0.01 copy, which was confirmed by the border-specific PCR (Fig. 2, C andD). Events C2 and C3 were homozygous RMCE-RMCE events containing two copies of RMCE and one copy of the donor. Accordingly, the border-specific PCR failed to detect any target-or excision-specific bands but amplified the large RMCE-specific band (Fig. 2, C-E). a A total of 82 target events were produced. The three events were selected by qPCR, PCR, Southern hybridization, and transgene border sequence analyses for RMCE retransformation. b T0 and T1 plant leaf DNA was analyzed by qPCR specific to the scp1, hpt, ubiq10, or yfp gene of QC288A (Fig. 1A). The heat shock protein gene hsp was used as an endogenous control in all duplex qPCRs. A genomic DNA sample containing one copy of the respective transgene was used as the calibrator for each qPCR assay to calculate relative transgene copies in other samples. A value of less than 0.3 or between 0.4 and 1.3 was considered as zero or one copy. A value between 1.4 and 2.3 was considered as two copies.

c The intactness of the FRT1 site was checked by PCR with primers Scp1-S/Hygro-A to amplify a 657-bp fragment across the FRT1 site. The FRT87 site was checked with primers Yfp-3/Frt87-A to amplify a 441-bp fragment across the FRT87 site. The nearly full-length QC288A transgene was checked with primers Scp1-S/Frt87-A to amplify a 4,393-bp fragment (Fig. 1A).

d T0 plant leaf DNA was digested with EcoRV and checked with two probes specific to the hpt and yfp genes. EcoRV cuts QC288A twice in the middle at positions 2,078 and 3,246. The hpt probe hybridized to a 2,078-bp or larger band, and the yfp probe hybridized to a 1,299-bp or larger band to each copy of the QC288A transgene. a Putative RMCE callus events were selected by chlorsulfuron resistance from the retransformation of the three target lines A, B, and C and screened by CFP expression. CFP-negative events A3 and B6 were included as controls for subsequent analyses. Events A2, B5, C2, and C3 each regenerated viable T0 plants.

b Events were first analyzed by an RMCE-specific PCR with primers 35S-277F/Als-3 specific to the recombined FRT1 region common to all three target lines.

c Events were analyzed by qPCR specific to the RMCE, donor, target, or flp (Fig. 1) to confirm RMCE and to check donor and flp integration. The qPCR was done as described in Table 1, although with different primers, probes, and calibrators. nd, Not detectable.

d Retransformation frequency was calculated as the average number of confirmed RMCE events produced from a bombarded petri plate of target line callus. e Events B5 and B6 were obtained from the direct retransformation of target line B callus prior to plant regeneration during the target line creation. Callus at this stage was hemizygous and could be chimeric.

To summarize, two, five, and three RMCE events were obtained from the retransformation of five, five, and six plates of target A, B, and C cultures, respectively, and the RMCE retransformation frequencies were thus calculated as 0.4, 1, or 0.5 event per plate (Table II). The average of these frequencies is approx-imately 10 times lower than the average five events per plate frequency for standard soybean biolistic transformation but high enough for routine RMCE event production.

## Characterization of RMCE T0 Plants

Viable T0 plants regenerated from events A2, B5, C2, and C3 were analyzed by the same construct-specific qPCR analyses (Table II). Four A2 plants, A2-1, A2-2, A2-3, and A2-4, three C2 plants, C2-1, C2-2, and C2-3, and two C3 plants, C3-1 and C3-2, all retained the same molecular signatures of their respective callus parents A2, C2, and C3. The 1.0 copy of target detected in B5 callus was no longer observed in T0 plants B5-1, B5-2, and B5-3. The same border-specific PCR analyses also confirmed that the T0 plants were the same as their respective callus parents (Fig. 3).

Since the target QC288A and the RMCE QC288A329 sequences diverge downstream of the FRT1 site with hpt in QC288A and als in QC288A329 and upstream of the FRT87 site with yfp:nos in QC288A and cfp:nos in QC288A329 (Fig. 1, A andD), the alignment of the target and RMCE transgene sequences with their map sequences should confirm RMCE at the sequence level. The 30 bands, marked "x" in Figure 3, amplified from the target samples A, B, and C, RMCE callus samples A2, B5, and C2, and representative RMCE T0 plant samples A2-1, B5-3, C2-1, and C3-1, were cloned and sequenced. The alignment of the transgene sequences with predicted target, RMCE, and excision sequences confirmed accurate DNA recombination for all of the RMCE and excision events (data not shown). The sequences of the A2 and A2-1 excision-specific fragments (Fig. 3E) matched a predicted excision sequence containing an FRT1 site.

## Characterization of RMCE T1 Plants

T0 plants A2-1, A2-2, A2-3, A2-4, B5-1, B5-2, C3-1, and C3-2 produced seeds. T1 plants germinated from these seeds were analyzed by the same constructspecific qPCR analyses. Since the T0 plants of each A2, B5, or C3 event were identical (Table II), T1 plants from different T0 plants of the same event were treated as one population for segregation analysis. The RMCEspecific qPCR would detect two copies, one copy, or zero copies of RMCE for RMCE-RMCE, RMCE-excision, and excision-excision. The target, donor, and flpspecific qPCR would detect two copies, one copy, or zero copies of the respective genes for homozygous, hemizygous, or null target, donor, and flp.

Since the A2 T0 plants were hemizygous RMCEexcision containing a donor and a flp (Table II), the excision would segregate away from RMCE and the donor and flp would segregate independently if they were not linked to the RMCE-excision locus. The RMCE-excision locus of 42 A2 T1 plants segregated as 12 RMCE-RMCE, 18 RMCE-excision, and 12 excision-Figure 2. Border-specific PCR of RMCE callus events. PCR analyses specific to the 5# border, 3# border, and full length of RMCE, target, and excision were done using various combinations of the 5# border, 3# border, and transgene-specific primers (see Fig. 1). Expected PCR fragments of the RMCE 5# and 3# borders, target 5# and 3# borders, fulllength RMCE, target, and excision are 1,117, 1,351, 1,036, 732, 6,652, 5,063, and 1,307 bp for target line A events; 967, 1,180, 886, 561, 6,331, 4,742, and 986 bp for line B events; and 1,018, 1,294, 937, 675, 6,496, 4,907, and 1,151 bp for line C events. Wild-type DNA (wt) controls were included. A, RMCE 5# border specific. B, RMCE 3# border specific. C, Target 5# border specific. D, Target 3# border specific. E, Full length. The full-length PCR failed to amplify the expected 6,652-, 6,331-, or 6,496-bp RMCE band in the presence of the 1,307-, 986-, and 1,151-bp excision bands "e" for the RMCE-excision events A1, A2, B1, B2, B3, B4, and C1. The 6,331-and 6,496-bp RMCE bands "r" were amplified for the RMCE-wt event B5 and the RMCE-RMCE events C2 and C3. The 5,063-, 4,742-, and 4,907-bp target bands "t" were amplified for target escape events A3 and B6 and target controls B and C. The 1-kb markers are 10, 8, 6, 5, 4, 3, 2.5, 2, 1.5, 1, 0.75, 0.5, and 0.25 kb. Only the first band of a group of bands is marked.

Site-Specific Gene Integration and Soybean Transformation Plant Physiol. Vol. 151, 2009 excision. The donor and flp segregated together but independently from the RMCE as 15 homozygous, 16 hemizygous, and 11 null. One plant was RMCE-RMCE and seven plants were RMCE-excision, all free of any target, donor, or flp DNA. Since the B5 T0 plants, already free of any target, donor, or flp, were derived from a hemizygous target, they were hemizygous RMCE-wt (Table II). The 36 B5 T1 plants segregated as 11 RMCE-RMCE, 17 RMCE-wt, and eight wild type. Since the C3 T0 plants were already homozygous RMCE-RMCE but contained a donor (Table II), all 48 C3 T1 plants remained as RMCE-RMCE and the donor segregated independently as 12 homozygous, 24 hemizygous, and 12 null.

Target homozygous plants A, B, and C, RMCE T0 plants A2-3, A2-4, B5-1, B5-2, C3-1, and C3-2, RMCEexcision T1 plants A2-3-1 and A2-3-2, excision-excision T1 plants A2-3-3 and A2-3-4, and RMCE-RMCE T1 plants B5-1-1, B5-2-1, C3-1-1, and C3-1-2 were selected for Southern hybridization analysis. NdeI digestion and hpt, scp1, ubq, and flp probes were used. NdeI cuts QC288A only once at position 1,119 and has to cut another NdeI site in the genomic DNA bordering the 5# end of the QC288A transgene to produce a fragment, consisting of the 5# end 1,119 bp of QC288A and the genomic DNA segment (Fig. 1A), that would hybridize to both the hpt and scp1 probes (Fig. 4, A andB). NdeI cuts QC288A329 only once at position 4,395 and also the same NdeI site in the 5# genomic DNA border to produce a fragment, consisting of the 5# end 4,395 bp of QC288A329 and the same genomic DNA segment (Fig. 1D), that would hybridize to both the scp1 and ubq probes (Fig. 4, B andC). Thus, the RMCE QC288A329-specific band would be 4,395 2 1,119 = 3,276 bp larger than the corresponding target QC288Aspecific band. The ubq probe was derived from a soybean endogenous gene and hybridized to an approximately 6-kb wild-type band. Additional ubq-only bands were specific to randomly integrated donor QC329 that contained only one NdeI site at position 3,715 (Fig. 1B). scp1-only bands were specific to excision that contained only the scp1 promoter. The excision-specific fragment was produced by digestion at the same NdeI site in the 5# genomic DNA border and another NdeI site in the 3# genomic DNA border (Fig. 1E). NdeI cuts the flp DNA QC292 only once at position 4,039 (Fig. 1C). Bands hybridized to both the flp and scp1 probes were specific to randomly integrated QC292 (Fig. 4, B andD).

The Southern hybridization results were consistent with previous qPCR and PCR results except for a large scp1 band detected in C3-1 and C3-2 and extra ubq bands detected in target A-derived plants (Fig. 4, B andC). The large scp1 band of C3-1 and C3-2 also hybridized to the ubq probe and disappeared with the donor in C3-1-1 and C3-1-2, and was considered a scp1 promoter mingled with the donor at an unlinked random insertion site. Four of the five ubq-only bands below the approximately 6-kb wild-type band detected in A2-3 and A2-4 (Fig. 4D) were likely partial copies of the donor, since qPCR detected only one donor insert (Table II). One of the ubq-only bands, not detected by the donor-specific qPCR, remained in RMCE-excision plants A2-3-1 and A2-3-2, and two remained in excision-excision plants A2-3-3 and A2-3-4.

Figure 3. Border-specific PCR of RMCE T0 plants. T0 plants A2-1, A2-2, A2-3, A2-4, B5-1, B5-2, B5-3, C2-1, C2-2, C2-3, C3-1, and C3-2 regenerated from callus events A2, B5, C2, and C3 were analyzed by the same border-specific PCR analyses (see Fig. 2). Target parents A, B, and C and callus parents A2, B5, and C2 were included as controls. A, RMCE 5# border specific. B, RMCE 3# border specific. C, Target 5# border specific. D, Target 3# border specific. E, Full length. The fulllength PCR failed to amplify the expected 6,652-bp RMCE band for the RMCE-excision event A2 and T0 plants A2-1, A2-2, A2-3, and A2-4 in the presence of the 1,307-bp excision-specific bands "e." The 6,331-bp RMCE band "r" was amplified for the RMCE-wt event B5 and T0 plants B5-1, B5-2, and B5-3. The 6,496-bp RMCE band "r" was amplified for RMCE-RMCE events C2, C3 (C3 is not shown), and T0 plants C2-1, C2-2, C2-3, C3-1, and C3-2. Band "n" is nonspecific, since it is smaller than the expected 1,151-bp excision band of line C. The 5,063-, 4,742-, and 4,907-bp target bands "t" were amplified for targets A, B, and C. Only the first band of a group of bands is marked. The 30 bands marked with "x" were cloned and sequenced.

# DISCUSSION

Single-site SSI creates two directly oriented recognition sites vulnerable to excision that makes the recombination events unstable. Mutant lox sites (Albert et al., 1995;Srivastava and Ow, 2001), cre gene displacement, and transient cre expression (Albert et al., 1995;Vergunst et al., 1998) can be used to prevent the excision. A donor DNA can be circularized prior to integration by a recombinase to remove any unwanted components such as the vector backbone to prevent them from being integrated (Vergunst et al., 1998;Srivastava and Ow, 2001). To achieve marker-free SSI, a two-step approach was proposed to combine gene integration using one recombinase system such as Crelox with gene excision using another system such as FLP-FRT conditionally controlled by an inducible promoter (Srivastava and Ow, 2004).

RMCE using two recognition sites provides a flexible way for gene targeting. If two identical sites are used, they must be in opposite orientations to prevent excision, although the DNA segment between the two sites can flip (Nanto et al., 2005). Two incompatible sites such as loxP and lox5171 can also be arranged in opposite orientations for successful RMCE (Louwerse et al., 2007). Preferably, RMCE using two directly oriented incompatible sites can avoid the excision or flipping of the flanked DNA segment and has succeeded to some extent in animal systems with a loxP and a mutant loxP511 (Trinh and Morrison, 2000), a FRT and a mutant FRT3 (Horn and Handler, 2005), or a FRT and a loxP (Lauth et al., 2002). Since DNA cassette exchange is reversible in RMCE, the original target gene can be exchanged out with a donor cassette containing a third recognition site. The resulting RMCE product can be used as a new target for another round of RMCE using the third recognition site to stack genes. The process can be repeated to successively stack more genes if more incompatible recognition sites are available. In both RMCE and single-site SSI, the DNA recombination tends to select recombination events containing one complete copy of the donor gene cassette.

When using a homozygous target, an RMCE event can be chimeric, with the target on one chromosome being converted to an RMCE and the other target on the homologous chromosome being converted to an excision or unchanged. The frequent occurrence of RMCE-excision and the lack of RMCE-target in our experiments indicate that the FRT1 and FRT87 sites are not completely incompatible and that the FLP-mediated DNA recombination is highly effective. RMCE can even occur simultaneously on two homologous chromosomes, as in the case of events C2 and C3. More likely, a homozygous RMCE has to be obtained at the T1 generation via segregation. Any donor or flp DNA integrated randomly at a separate genomic site in an RMCE event can be removed by segregation.

The effective RMCE described here opens new ways for transgenic product development and transgene Genomic DNA was digested with NdeI and hybridized sequentially with probes hpt (A), scp1 (B), ubq (C), and flp (D). Target-specific bands "t," hybridizing to both the scp1 and hpt probes, were detected in target homozygous plants A, B, and C. RMCE-specific bands "r1" for hemizygous and "r2" for homozygous samples, hybridizing to both the scp1 and ubq probes, were detected in RMCE-excision T0 plants A2-3, A2-4, B5-1, and B5-2, RMCE-RMCE T0 plants C3-1 and C3-2, RMCE-excision T1 plants A2-3-1 and A2-3-1, and RMCE-RMCE T1 plants B5-1-1, B5-2-1, C3-1-1, and C3-1-2. The RMCE-specific bands are 3,276 bp larger than the corresponding target-specific bands. Excision-specific bands "e1" for hemizygous and "e2" for homozygous samples, hybridizing to only the scp1 probe, were detected in RMCE-excision T0 plants A2-3 and A2-4, RMCE-excision T1 plants A2-3-1 and A2-3-2, and excisionexcision T1 plants A2-3-3 and A2-3-4. The ubq probe hybridized to an approximately 6-kb wild-type band "w" in all samples. Donor-specific bands "d," hybridizing to only the ubq probe, were detected in as many as five copies in RMCE-excision T0 plants A2-3 and A2-4, one copy in RMCE-excision T1 plants A2-3-1 and A2-3-2, and two copies in excision-excision T1 plants A2-3-3 and A2-3-4. The top ubq bands "d-f" of RMCE-RMCE T0 plants C3-1 and C3-2 were also hybridized by the scp1 probe. They might represent mingled flp DNA QC292 containing the scp1 promoter and the donor DNA QC329 containing the ubq gene. flp-specific bands "f," hybridizing to both the flp and scp1 probes, were detected in RMCE-excision T0 plants A2-3 and A2-4. The DIGVII markers are 8,576,7,427,6,106,4,899,3,639,2,799,1,953, and 1,882 bp. Only the first band of a group of bands is marked.

expression studies. Large DNA fragments can be integrated via RMCE, which seems to rely only on FLP-catalyzed interactions between FRT sites. Various target lines can be produced in advance and maintained as production lines to accept genes with various preferences for gene silencing, tissue-specific expression, agronomic performance, etc. Multiple genes can be stacked reversibly at the same genetic locus by multiple rounds of RMCE using different recombination sites.

# MATERIALS AND METHODS

## DNA Construction

The target construct QC288 containing scp1-FRT1:hpt:nos+ubiq10:yfp:nos-FRT87, the donor construct QC329 containing FRT1-als:pinII+ubq:cfp:nos-FRT87, and the FLP expression construct QC292 containing scp1:flp:pinII were made following standard molecular cloning procedures through multiple steps using components from existing DNA constructs (Li et al., 2007).

# ACKNOWLEDGMENTS

We are grateful to DuPont/Pioneer colleagues Dr. Alex Lyznik for comments on the manuscript, Russ Booth for managing transgenic plants, and the soybean transformation group for producing transgenic plants.

## Plant Transformation

The target DNA scp1-FRT1:hpt:nos+ubiq10:yfp:nos-FRT87 was released as a 4,544-bp fragment QC288A with AscI digestion from QC288, resolved by agarose gel electrophoresis, and purified using a gel extraction kit (Qiagen). Soybean (Glycine max) embryogenic cultures were transformed with QC288A following the biolistic transformation protocol using 30 mg mL 21 hygromycin for selection (Li et al., 2007). Transgenic plants (T0) were regenerated from single-copy events identified by qPCR and Southern blot. The original target line hemizygous cultures or cultures initiated from the developing embryos of target homozygous T1 plants were cobombarded with the donor QC329 and flp QC292 plasmid DNA at a 9:3 ratio following the same biolistic transformation protocol except using 90 ng mL 21 chlorsulfuron (DuPont) to select retransformation events.

## Southern Hybridization Analysis

Soybean genomic DNA was prepared from somatic embryo or leaf samples and analyzed by Southern hybridization with digoxigenin-labeled probes (Li et al., 2007). Target event DNA was digested with EcoRV and hybridized with a 794-bp hpt probe made with primers Hpt-1 (5#-TTCAGCTTCGATGTAG-GAGGGCG-3#) and Hygro-2 (5#-GCTCCGGATCGGACGATTGC-3#) and a 693-bp yfp probe made with primers Yfp-1 (5#-TGGCCCACAGCAAG-CACGGCCTG-3#) and Yfp-2 (5#-AGGCCAGGGCGCTGGGGAAGGCG-3#). RMCE event DNA was digested with NdeI and hybridized with a 581-bp scp1 probe made with primers Scp1-S (5#-GAGATCCGTCAACATGGTGG-AGC-3#) and Scp1-A5 (5#-GGAACTTCTGTTGGGTCGACTGTAAT-3#), a 456-bp hpt 5# end probe made with primers Hygro-1 (5#-GAAAAAGCCTGAACT-CACCG-3#) and Hygro-A1 (5#-GCGCATATGAAATCACGCCATG-3#), a 696-bp ubq probe made with primers Ubq-S1 (5#-GTCGCCTTCAAGTTG-CATCATTTAAC-3#) and Ubq-A1 (5#-ATATGCGGTGGAACCGTTCTCC-3#), and a 586-bp flp probe made with primers Flp-1 (5#-GGCAGTTCGTG-GAGAGGTTCGAG-3#) and Flp-A1 (5#-CCGAGGTACTTGTTCTGCACG-AGC-3#).

## Transgene Cloning and Sequencing

Genomic DNA bordering the QC288A transgene was acquired using the GenomeWalker kit (Clontech). Genomic DNA digested with EcoRV, DraI, HpaI, or StuI was ligated to adaptors and amplified by two rounds of PCR. The first PCR used adaptor-specific primer AP1 (5#-GTAATACGACTC-ACTATAGGGCACG-3#) and QC288A-specific primers Scp1-A (5#-CTACT-GTCCTTTTGATGAAGTGACAG-3#) for the 5# end border and Vec-S1 (5#-GATCGGGAATTCTAGTGGCCGG-3#) for the 3# border. The second PCR used adaptor-specific primer AP2 (5#-CTATAGGGCACGCGTGGTC-GAC-3#) and QC288A-specific primers Scp1-A4 (5#-CTGGGCAATGGAATCCGA-GGAG-3#) for the 5# end border and Vec-S2 (5#-GCTGATGATCCCGGT-GAAGTTCC-3#) for the 3# border. Specific PCR fragments were cloned in pCR2.1-TOPO vector with the TOPO TA cloning kit (Invitrogen). Plasmid DNA was prepared with the Qiaprep plasmid DNA kit (Qiagen) and sequenced using a capillary DNA analyzer and the dye terminator cycle DNA sequencing kit (Applied Biosystems). Sequence assembly and alignment were done using Vector NTI programs (Invitrogen). Sequence searches were done using the National Center for Biotechnology Information (www.ncbi. nlm.nih.gov) advanced BLAST algorithm.

## PCR Analysis

PCR was done on leaf or somatic embryo DNA samples following the same protocol (Li et al., 2007). The 5# end intactness of the QC288A transgene was checked with primers Scp1-S and Hygro-A (5#-CGTCGCGGTGAGTT-CAGGCTT-3#) for a 657-bp band, while the 3# end intactness was checked with primers Yfp-3 (5#-GGAGCGACGCCAAGAACCAGAA-3#) and Frt87-A (5#-GGCCGCAAGCTCTAGTGAAGTTC-3#) for a 441-bp band. The fulllength QC288A was checked with primers Scp1-S and Frt87-A for a 4,393-bp band.

Putative RMCE events were screened by CFP expression with a MZFLIII stereo microscope (Leica Microsystems) and identified by PCR with primers 35S-277F (5#-GACAGTGGTCCCAAAGATGGA-3#) and Als-3 (5#-GTGGATC-TAGTAATGCGTTTGGG-3#) to amplify a 497-bp band. RMCE events were then confirmed with five PCR analyses specific to the RMCE 5# border, RMCE 3# border, target 5# border, target 3# border, and the 5# border to 3# border full lengths of RMCE, target, and excision. The RMCE 5# border PCR used the common RMCE-specific primer Als-3 and a target line 5# border-specific primer, 53-1S1 (5#-TGTTTGTTGTTTCCAAGATTGACTGC-3#), for line A, 70-1S (5#-TCTTTCCCTCCCAGAGAGTAACAAGC-3#) for line B, and 8H-ScaS1 (5#-ATAGAGGATTGGGACTCGTCGTGC-3#) for line C. The RMCE 3# border PCR used the common RMCE-specific primer Cyan-1 (5#-ATGGCCCTGTC-CAACAAGTTCATC-3#) and a target line 3# border-specific primer, 53-1A (5#-CACCAAACTAATCATATCTCACTAAATCAATCC-3#) for line A, 70-1A (5#-GCAGCGACAGGGGATTCCTCTAC-3#) for line B, and 8H-VecA (5#-AGATGCTAGAAATTCAACAACGGAAGC-3#) for line C. The target 5# border PCR used the same target line 5# border-specific primers and the common target-specific primer Hygro-A. The target 3# border PCR used the same target line 3# border-specific primers and the common target-specific primer Yfp-3. The full-length PCR used the same target line 5# and 3# borderspecific primers to simultaneously amplify the full-length RMCE, target, and excision. The expected sizes of all PCR bands are described in the Figure 2 legend. qPCR Analysis qPCR analyses were done on genomic DNA samples using the Taqman DNA polymerase kit with a 7500 real-time PCR system (Applied Biosystems). The relative quantification methodology and single-tube duplex PCRs, one for a target gene and the other for an endogenous control gene to normalize reactions, were used. After 2 min of incubation at 50°C to activate the Taq DNA polymerase and 10 min of incubation at 95°C to denature the DNA templates, 40 cycles of 15 s at 95°C and 1 min at 60°C were used. A soybean heat shock protein (hsp) gene was used as the endogenous control for all assays. Primers Hsp-F1 (5#-CAAACTTGACAAAGCCACAACTCT-3#), Hsp-R1 (5#-GGA-GAAATTGGTGTCGTGGAA-3#), and probe Hsp-T1 (5#-VIC-CTCTCATCT-CATATAAATAC-MGB-3#; Applied Biosystems) were used for the hsp control. A DNA sample known to contain one copy of the transgene to be analyzed was included as the calibrator for each qPCR assay.

The scp1, hpt, ubiq10, and yfp components of QC288A were analyzed to screen for single-copy target events and to identify homozygous T1 plants. Primers/probe sets used were 35S-277F, 35S-345R (5#-CGTGGTTGGAACGTCTTCTTTT-3#), and 35S-399T (5#-FAM-CCCCACCCACGAGGAGCATCG-BHQ1-3#; Sigma-Genosis) for the scp1 assay, Hygro-591F (5#-GGATTTCGGCTCCAACAATG-3#), Hygro-659R (5#-GCCTCGCTCCAGTCAATGA-3#), and Hygro-612T (5#-FAM-CCTGACGGACAATGGCCGCATAAC-BHQ1-3#) for the hpt assay, Ubq10-693F (5#-TGTGGTTGTCGACGAGTCAGTAAT-3#), Ubq10-769R (5#-GAGTTGA-TAAACACGACTCGTGTGT-3#), and Ubq10-719T (5#-FAM-CGGCGTCAAA-GTGGTTGCAGCC-BHQ1-3#) for the ubiq10 assay, and Yfp-67F (5#-AACG-GCCACAAGTTCGTGAT-3#), Yfp-130R (5#-TGGTCTGCTTGCCCTTGAAG-3#), and Yfp-88T (5#-FAM-ACCGGCGAGGGCATCGGCTA-BHQ1-3#) for the yfp assay.

RMCE QC288A329-, target QC288A-, and donor QC329-specific qPCR assays were all designed around the FRT1 site. Primers/probe sets used were

