# Background

Chronic kidney disease (CKD) represents a significant and increasing healthcare issue throughout the world. Although whole kidney transplantation remains the standard of care, the CKD patient population advancing to end-stage renal disease (ESRD) requiring dialysis or transplant is much greater than the number of acceptable donor kidneys [1]. Regenerative medicine and tissue engineering methodologies may provide additional therapeutic possibilities for CKD patients. From a manufacturing and product development perspective, primary cell types are preferable when compared to stem and progenitor cell populations, owing to considerable streamlining of isolation, expansion, maintenance, and characterization conditions (reviewed by [2, 3]).

Several studies on regeneration of renal architecture and function following acute kidney injury point to tubular epithelial cells as central in restoration of renal functionality [4, 5]. Starting with an autologously sourced renal tissue biopsy, tubular cell-enriched populations can be separated from other kidney cell types on the basis of differential buoyant density using density gradient centrifugation [6]. Direct injection of these bioactive renal cell sub-populations in the rat 5/6Nx model of CKD resulted in extended survival and enhancement of renal function for six months post-treatment [7]. The mechanism(s) of action by which this observed regenerative bioactivity is mediated may include secreted protein factors and miRNAs delivered via exosomes [8]. Other documented mechanisms include neo-nephrogenesis at the site of SRC delivery [9]. However, elucidation of the mechanisms of action of this and other cell-based therapeutics is not a straightforward problem and is best approached with multiple methodologies.

In the current manuscript, the specific aim is to apply bioinformatics methodologies to compare SRC component cellular sub-fractions against each other and against pre-gradient, unfractionated renal cells. Transcriptional networks and signaling pathways that are functionally relevant to the observed regenerative bioactivity associated with each of the post-gradient subpopulations are identified. These sub-populations have been nominally designated as B1–B5 in preclinical functional evaluations of SRC therapeutic potential in rodent models of CKD. KEGG pathways and Gene Ontology (GO) categories that are significantly up- or down-regulated in each of sub-fractions B1–B5 in comparison with pre-gradient fractionation (PreG) are reported. The significance of these changes in gene expression in understanding the mechanism of action of the observed reparative, restorative, and regenerative bioactivity of SRC is discussed.

# Materials And Methods

## Isolation And Preparation Of Rodent Renal Cell Populations For Genomic Analysis

The preparation of selected bioactive primary renal cells from whole rat kidney has been previously described in detail [6, 7, 10]. Briefly, whole kidneys were harvested from 5-week-old male Lewis rats (Hilltop Labs, Scottsdale, PA, USA) and kidney tissue was dissociated enzymatically in a buffer containing 4.0 units/mL dispase (Stem Cell Technologies, Inc., Vancouver, BC, Canada) and 300 units/mL collagenase IV (Worthington Biochemical, Lakewood, NJ, USA). Red blood cells and debris were removed by centrifugation through 15% iodixanol (OptiPrep®, Axis Shield, Norton, MA, USA). Primary renal cells were seeded onto tissue culture-treated polystyrene plates (NUNC, Rochester, NY, USA) and cultured in 50:50 media, a 1:1 mixture of high-glucose Dulbecco’s modified Eagle medium (DMEM):Keratinocyte Serum Free Medium (KSFM) containing 5% Fetal Bovine Serum (FBS (Hyclone, Logan, UT, USA), 2.5 µg EGF, 25 mg Bovine Pituitary Extract (BPE), 1X ITS (insulin/transferrin/sodium selenite medium supplement), and antibiotic/antimycotic (all from Invitrogen, Carlsbad, CA, USA). Prior to post-culture cell separation, primary renal cell cultures were transferred from atmospheric oxygen conditions (21%) to a more physiologically relevant low-oxygen (2%) environment for 24 h, to improve cell separation efficiency.

Separation of primary renal cell cultures, prepared as 75 × 106 cells in 2 mL unsupplemented KSFM (uKSFM), was performed by centrifugation through a four-step iodixanol (OptiPrep; 60% w/v in uKSFM) density gradient layered specifically for rodent (16%, 13%, 11%, and 7%) in 15-mL conical polypropylene tubes and centrifuged at 800 × g for 20 min at room temperature (without brake). After centrifugation, cellular sub-fractions were extracted from the gradient via pipette and collected as 4 distinct bands (B1–B4) and a pellet (B5). All bands were washed 3 times with sterile phosphate-buffered saline (PBS) prior to use. A pre-gradient sample (“PreG”) and whole kidney tissue sample (“Macro”) were collected from each rodent for comparative purposes. Culture conditions used for each rodent cell population are summarized in Table 1. RNA for genome-wide transcript analysis was prepared using Qiagen’s RNA isolation kit (Germantown, MD, USA) according to the manufacturer’s instructions. Two micrograms of RNA from each sample normalized as per Table 2 was used for genome-wide transcript analysis on the Affymetrix GeneChip Rat Genome 230 2.0 Array (Wake Forest University Health Sciences Microarray Core Facility, Winston-Salem, NC, USA).Table 1Culture conditions and gradient loadCell prepSeeding densityCulture timeFinal confluency (%)Gradient loadRK08617.5 e6/flask3d 21% O21d 2% O210072.8 e6RK08715 e6/flask2d 21% O21d 2% O28591 e6RK09719.3 e6/flask2d 21% O21d 2% O28592.5 e6Gradient load refers to the number of PreG cells loaded onto the iodixanol gradient for separation into SRC component sub-populations B1–B5Table 2RNA concentration and normalizationRNA normalizationFractionSymbolng/ulVol, 2 µgNorm 20 µl1RK0863812PreG412.194.85215.14823813B1511.623.90916.09133814B2460.284.34515.65543815B3284.087.04012.96053816B4163.6412.2227.77863817Pellet354.385.64414.3567RK0873821Macro213.059.38710.61383825PreG301.086.64313.35793826B1363.745.49814.502103827B2351.535.68914.311113828B3370.355.40014.600123829B4387.135.16614.834133830Pellet136.6714.6345.36614RK0974692Macro125.7615.9034.097154697PreG379.675.26814.732164698B1366.565.45614.544174699B2420.824.75315.247184700B3439.34.55315.447194701B4350.435.70714.293204702Pellet167.9411.9098.091RNA has been normalized by resuspending a total of 2 µg of RNA in a total volume of 20 µl. In Table 2, the column “vol, 2 µg” lists the volume of the RNA preparation representing a total of 2 µg of RNA. The column “Norm 20 µl” lists the volume of additional buffer to be added to make up a final volume of 20 µl

## Data Analysis

Affymetrix GeneChip data were normalized using RMA [11]. The gene expression profile from each fraction (B1, B2, B3, B4, B5) was compared to the gene expression profile in PreG by using paired t test. Differentially expressed genes were detected by paired t test at P < 0.05 and were then analyzed by DAVID (http://david.abcc.ncifcrf.gov) to detect Gene Ontology (GO) categories and KEGG Pathways that were significantly different between each fraction and PreG. For GO analysis, we used GO-BP-FAT, which is GO biological process categories provided by DAVID to minimize redundancy and increase specificity in GO terms. Bonferroni adjustment was applied to P values generated from GO and KEGG pathway analysis for multiple testing correction. KEGG pathway plots were generated using Pathview in Bioconductor [12]. Gene networks for GO categories were generated using QIAGEN’s Ingenuity Pathway Analysis (IPA®, QIAGEN Redwood City, USA, www.qiagen.com/ingenuity).

# Results

The manufacture of SRC is a linear process, with PreG cells isolated directly from source kidney tissue biopsies being used as starting material. Gradient sub-fractionation represents the processing stage that results in multiple sub-populations of cells (B1–B5). These sub-populations have clinically relevant functional properties in rodent models of chronic kidney disease that differ markedly from the initial PreG population [6, 7, 10]. This manuscript specifically aims to define how the final manufacturing sub-populations B1–B5 differ from the source material (PreG) by the use of genome-wide transcriptional profiling.

## Sub-Fraction B1: Go/Kegg Categories Cell Cycle, Transcriptional Control, And Inflammation Altered Relative To Preg

GO analysis of the B1 sub-population relative to PreG shows that the GO categories associated with regulation of the cell cycle (P = 9.40E−04), cell division (P = 9.81E−05), and cell cycle phase (P = 0.016) were significantly down-regulated in B1 (Table 3). Similar outcomes were also generated by KEGG pathway analysis (Table 3; Additional file 1: Figure S1). As shown in the pathway map, the down-regulated genes are distributed in each phase of the cell cycle, including cell growth (G1, G2), DNA replication (S), and mitosis (M) (Additional file 1: Figure S1). Key cell cycle regulatory proteins subject to negative regulation in B1/PreG include CDK2, CDK7, CYCE, CDC45, ORC, CHK1, CHK2, MAD2, CDC20, and APC. Taken together, these proteins are instrumental in regulating each step of the mitotic sequence [13–15].Table 3GO categories and KEGG pathways that were significantly up- or down-regulated in B1 by comparing to PreGCategoryTermRaw_PBonferroni adjusted PUp-regulatedGO_BP_FATGO-0019221: cytokine-mediated signaling pathway7.12E−060.022732GO-0022610: biological adhesion1.42E−050.044785GO-0007155: cell adhesion1.42E−050.044785KEGGrno04142:Lysosome2.57E−064.44E−04Down-regulatedGO_BP_FATGO-0051301: cell division3.24E−089.81E−05GO-0007049: cell cycle3.11E−079.40E−04GO-0045449: regulation of transcription4.95E−070.001496GO-0022403: cell cycle phase5.32E−060.015942GO-0051252: regulation of RNA metabolic process1.51E−050.04449KEGGrno03040:Spliceosome1.35E−050.00234rno04110:Cell cycle2.45E−040.041792

GO/KEGG categories significantly up-regulated in B1/PreG include cytokine-mediated signaling pathway (P = 0.023) (Additional file 1: Figure S2) which is involved in the immune response, and lysosome (P = 4.44E−04) which is involved in the digestion of foreign material and cell waste. Cell adhesion (P = 0.045) was also up-regulated (Table 3). Up-regulated pro-inflammatory cytokines identified in Additional file 1: Figure S2 include TNF, IL1A, CX3CL1, CCL2 [16]. Other pro-inflammatory or pro-adipogenic markers up-regulated in B1/PreG include LEPR and GAB1 [17].

## Sub-Fraction B2: Ecm–Receptor Interaction Is Sole Go/Kegg Classifier Altered Relative To Preg

The transcriptomic profile of B2 was found to be similar to PreG at the pathway level. No GO/KEGG categories were identified as significantly up-regulated. Only one pathway, ECM–receptor interaction (rno04512), was found to be significantly down-regulated (P = 0.039). Numerous ECM proteins, including collagen, laminin, reelin, tenascin, vitronectin, and thrombospondin (THBS), were significantly down-regulated in B2 (Additional file 1: Figure S3) relative to PreG. In contrast to the negatively regulated transcriptomic profile observed with ECM-associated genes, RHAMM expression was specifically up-regulated in B2/PreG (fold change = 1.114, P = 0.030) (Additional file 1: Figure S3).

## Sub-Fraction B3: Go/Kegg Categories Immune Response And Regulation Of Actin Polymerization Altered Relative To Preg

KEGG pathway analysis found categories related to complement and coagulation cascades are significantly up-regulated in B3/PreG (P = 0.008) (Table 4; Additional file 1: Figure S4). Up-regulated genes include complement component genes, such as C2, C3, and C4, and coagulation-related genes, such as fibrinogen (FG) and coagulation factor XIII (F13) (Additional file 1: Figure S4). Consistent with KEGG analysis, GO analysis also found a significant increase in the humoral immune response (P = 0.025) (Table 4; Additional file 1: Figure S5). Several GO categories related to the regulation of actin polymerization and actin filament length were significantly down-regulated in B3 (Table 4; Additional file 1: Figures S6, S7).Table 4GO categories and KEGG pathways that were significantly up- or down-regulated in B3 by comparing to PreGCategoryTermRaw_PBonferroni adjusted PUp-regulatedGO_BP_FATGO:0043112 ~ receptor metabolic process2.14E−060.005782GO:0007242 ~ intracellular signaling cascade3.59E−060.009661GO:0006959 ~ humoral immune response9.21E−060.024598GO:0007167 ~ enzyme linked receptor protein signaling pathway1.89E−050.049871KEGGrno04610:Complement and coagulation cascades6.60E−050.008351Down-regulatedGO_BP_FATGO:0008064 ~ regulation of actin polymerization or depolymerization2.42E−060.00522GO:0030832 ~ regulation of actin filament length2.42E−060.00522GO:0030833 ~ regulation of actin filament polymerization8.68E−060.018633

## Sub-Fraction B4: Go/Kegg Categories Regeneration And Cell Adhesion Altered Relative To Preg

Consistent with established therapeutic bioactivity in vivo, multiple regeneration-related GO categories, including vasculature development (P = 7.76E−07), blood vessel development (P = 1.36E−06), blood vessel morphogenesis (P = 1.76E−06), angiogenesis (P = 1.82E−06) and response to wounding (P = 1.35E−05) were found to be significantly up-regulated in B4 over PreG (Table 5).Table 5GO categories that were significantly up-regulated in B4 by comparing to PreGGO-BP-FAT termRaw PBonferroni adjusted PGO:0001944 ~ vasculature development2.61E−107.76E−07GO:0001568 ~ blood vessel development4.59E−101.36E−06GO:0048514 ~ blood vessel morphogenesis5.91E−101.76E−06GO:0001525 ~ angiogenesis6.13E−101.82E−06GO:0009611 ~ response to wounding4.55E−091.35E−05GO:0007155 ~ cell adhesion9.76E−092.90E−05GO:0022610 ~ biological adhesion9.76E−092.90E−05GO:0007242 ~ intracellular signaling cascade6.29E−070.001869GO:0019932 ~ second-messenger-mediated signaling1.40E−060.004152GO:0010033 ~ response to organic substance1.41E−060.004182GO:0055066 ~ di-, tri-valent inorganic cation homeostasis1.13E−050.033116

As shown in Additional file 1: Figure S8, key up-regulated receptors including CXCR4, TEK, FGFR1, and KDR are essential to angiogenesis and other regeneration-associated signaling pathways. In the GO category response to wounding (Additional file 1: Figure S9), many genes that were up-regulated, such as NOTCH1, NOTCH3, TIMP3, VWF, ADAM15, GAS6, IGFBP1, and TM4SF4, also belong to the GO categories wound healing and tissue regeneration. Cell adhesion-related GO categories were found to be significantly up-regulated in B4 relative to PreG (Table 5; Additional file 1: Figure S10).

## Sub-Population B5: Forty Go/Kegg Categories Found To Differ From Preg

Sub-population B5 was found to be the most distinctive fraction relative to PreG at the gene expression level. Forty GO/KEGG categories were significantly different from PreG (Table 6). Up-regulated categories include cell adhesion, angiogenesis (Additional file 1: Figure S11), branching morphogenesis of a tube (Additional file 1: Figure S12), cell morphogenesis, regulation of epithelial cell proliferation (Additional file 1: Figure S13), blood vessel development, blood vessel morphogenesis, vasculature development, cellular component morphogenesis, cell morphogenesis, cell morphogenesis involved in differentiation, tube morphogenesis, cell projection organization and morphogenesis, cell part morphogenesis, morphogenesis of a branching structure, developmental maturation and response to nutrient. Most of these significantly up-regulated GO/KEGG categories are clearly involved in catalyzing reparative, restorative, or regenerative outcomes.Table 6GO categories and KEGG pathways that were significantly up-regulated in B5 by comparing to PreGCategoryTermP valueBonferroni adjusted PGOTERM_BP_FATGO:0022610 ~ biological adhesion1.36E−144.62E−11GO:0007155 ~ cell adhesion1.36E−144.62E−11GO:0035295 ~ tube development7.52E−102.57E−06GO:0001525 ~ angiogenesis2.88E−099.84E−06GO:0001568 ~ blood vessel development9.91E−093.38E−05GO:0048754 ~ branching morphogenesis of a tube1.46E−085.00E−05GO:0048514 ~ blood vessel morphogenesis1.57E−085.37E−05GO:0001944 ~ vasculature development2.29E−087.82E−05GO:0032989 ~ cellular component morphogenesis3.12E−081.06E−04GO:0000902 ~ cell morphogenesis6.13E−082.09E−04GO:0000904 ~ cell morphogenesis involved in differentiation1.24E−074.24E−04GO:0035239 ~ tube morphogenesis1.31E−074.46E−04GO:0030030 ~ cell projection organization1.49E−075.07E−04GO:0048858 ~ cell projection morphogenesis2.42E−078.25E−04GO:0016337 ~ cell–cell adhesion2.79E−079.52E−04GO:0048812 ~ neuron projection morphogenesis5.46E−070.001862GO:0032990 ~ cell part morphogenesis6.75E−070.0023GO:0048666 ~ neuron development7.48E−070.002549GO:0033273 ~ response to vitamin7.72E−070.002632GO:0001763 ~ morphogenesis of a branching structure1.06E−060.003597GO:0021700 ~ developmental maturation1.20E−060.004074GO:0031175 ~ neuron projection development2.56E−060.008704GO:0050678 ~ regulation of epithelial cell proliferation6.75E−060.022778GO:0007584 ~ response to nutrient1.49E−050.049576KEGGrno04514:Cell adhesion molecules (CAMs)5.89E−050.009201

The canonical WNT/β-catenin signaling pathway is significantly down-regulated relative to PreG (Table 7; Additional file 1: Figure S14). As observed with B4, the key pro-angiogenic receptors CXCR4, TEK, and KDR are prominently up-regulated in B5. In addition, the pro-angiogenic ligands PGF, ANGPT2, ANGPT4, VEGFA, and PDGFA are significantly up-regulated in B5 relative to PreG. Activation of the regulation of GO category epithelial cell proliferation through the TGF-β1 signaling pathway is consistent with a potential role for SRC in the promotion of host tubular epithelial cell proliferation (see Additional file 1: Figure S13).Table 7GO categories and KEGG pathways that were significantly down-regulated in B5 by comparing to PreGCategoryTermRaw-PBonferroni adjusted-PGOTERM_BP_FATGO:0044265 ~ cellular macromolecule catabolic process5.44E−081.79E−04GO:0007242 ~ intracellular signaling cascade2.07E−076.83E−04GO:0051603 ~ proteolysis involved in cellular protein catabolic process5.90E−070.001943GO:0044257 ~ cellular protein catabolic process7.90E−070.002602GO:0043632 ~ modification-dependent macromolecule catabolic process1.48E−060.004857GO:0019941 ~ modification-dependent protein catabolic process1.48E−060.004857GO:0015031 ~ protein transport1.99E−060.006541GO:0030163 ~ protein catabolic process2.23E−060.007313GO:0045184 ~ establishment of protein localization2.78E−060.009125GO:0009057 ~ macromolecule catabolic process3.47E−060.011365KEGGrno04120:Ubiquitin-mediated proteolysis1.17E−072.02E−05rno04310:Wnt signaling pathway8.10E−060.001392rno05200:Pathways in cancer8.19E−060.001408rno05222:Small cell lung cancer2.02E−050.003468rno04530:Tight junction2.11E−040.035682

# Conclusions

SRC component characterization Transcriptomic profiling of B1–B5 confirms functional analysis of distinctive reparative, restorative, or regenerative outcomes associated with cell transplantation of each of B1–B5 in rodent models of chronic kidney disease. Iodixanol gradient density centrifugation of PreG renal cells generates 5 unique subpopulations with distinctive transcriptional signatures.Sub-fraction B1 GO/KEGG categories associated with cell cycle and transcriptional control were down-regulated relative to PreG. GO/KEGG categories associated with inflammation were up-regulated relative to PreGSub-fraction B2 Most similar to PreG. Only GO/KEGG categories associated with ECM–receptor interaction are significantly down-regulated relative to PreG.Sub-fraction B3 GO/KEGG categories associated with immune response were significantly up-regulated relative to PreG. GO/KEGG categories associated with regulation of actin polymerization were significantly down-regulated relative to PreGSub-fraction B4 GO/KEGG categories associated with repair, restoration, or regeneration and cell adhesion were significantly up-regulated relative to PreG.Sub-population B5 The most distinctive sub-population relative to PreG. Forty GO/KEGG categories found to be significantly different from PreG including multiple categories unambiguously associated with repair, restoration, or regeneration.

# Discussion

SRC are manufactured through iodixanol gradient centrifugation of a PreG population principally composed of renal tubular epithelial cells. The standard for SRC separation and reconstitution is entirely biophysical, based on the identification, isolation, and recombination of bands of defined density following gradient centrifugation. No attempt is made to achieve a predefined cellular composition expressing certain cell surface or other markers. In this, SRC are distinct from other cell-based therapeutics currently under development. As SRC is an autologous product, each patient biopsy results in an unique product, also composed principally of tubular epithelial cells. Although manufacturing conditions including media composition may well bias for or against certain cell sub-populations, no attempt is made here to control for selection bias secondary to media formulation. Following gradient centrifugation, B1 is selectively eliminated and B2–B5 collapsed together to create the final product.

Transcriptomic profiling is consistent with SRC representing a pro-regenerative cell population capable of catalyzing functional rescue of aspects of the disease condition in rodent models of chronic kidney disease. Specifically, the transcriptomic analysis of SRC component cell populations presented in this manuscript suggests that B1 subpopulations present significantly less proliferative and regenerative potential, demonstrated by the down-regulation of genetic networks associated with cell cycle and transcriptional control. These data are consistent with the observed absence of functional impact by B1 cells on renal pathophysiology in rodent models of CKD [6, 7, 10]. The transcriptomic profile of B2 was found to be similar to PreG at the pathway level. No GO/KEGG categories were identified as significantly up-regulated. Only one pathway, ECM–receptor interaction (rno04512), was found to be significantly down-regulated. It is well known that excessive deposition of extracellular matrix (ECM) components causes tissue fibrosis [18–20]. Renal fibrosis is the hallmark of disease progression from chronic kidney disease (CKD) to end-stage kidney disease. The lower expression level of ECM proteins in B2 may contribute to the lower level of fibrosis in B2 treated CKD rats, which may in turn explain the therapeutic benefits previously observed in B2 when compared to PreG [6, 7, 10].

In contrast, RHAMM expression was specifically up-regulated in B2/PreG. Expression of the hyaluronic acid (HA) receptor CD168/RHAMM has been directly linked to regenerative bioactivity in a number of in vitro and in vivo model systems. In addition, RHAMM has been shown to promote angiogenesis and cell motility; the increased expression level of RHAMM could be beneficial for tissue regeneration [21–26]. For B3, KEGG and GO analysis found a significant increase in the humoral immune response. It is well established that multiple components of both cellular and humoral immune systems contribute to reparative, restorative, or regenerative outcomes in multiple organs and tissues, including limbs, skeletal muscle, heart, and nervous system [27].

Consistent with its established therapeutic bioactivity in vivo, multiple regeneration-related GO categories, including vasculature development, blood vessel development, blood vessel morphogenesis, angiogenesis, and response to wounding, were found to be significantly up-regulated in B4 over PreG. Angiogenesis (Additional file 1: Figure S8) is central to regeneration, in the kidney, heart, and multiple other organs and tissues [28–31]. Sub-population B5 was found to be the most distinctive fraction relative to PreG at the gene expression level. Up-regulated categories include cell adhesion, angiogenesis, branching morphogenesis of a tube, cell morphogenesis, regulation of epithelial cell proliferation, blood vessel development, blood vessel morphogenesis, vasculature development, cellular component morphogenesis, cell morphogenesis, cell morphogenesis involved in differentiation, tube morphogenesis, cell projection organization and morphogenesis, cell part morphogenesis, morphogenesis of a branching structure, developmental maturation, and response to nutrients, etc. Most of these significantly up-regulated GO/KEGG categories are clearly involved in catalyzing regenerative outcomes.

SRC manufacture applies a 24-h period of exposure to hypoxia (2% oxygen) to the expanded cell population prior to gradient centrifugation. Hypoxic preconditioning has been used to promote the pro-regenerative characteristics of mesenchymal stem cells in preclinical rodent models of renal disease [32]. Consistent with the results presented in this manuscript, genome-wide transcriptional profiling of the effect of hypoxic preconditioning on human mesenchymal stem cells identified altered expression of genes involved in inflammation, immune function, cell survival, migration and proliferation as well as vasculogenesis and angiogenesis [33]. More broadly, genome-wide transcriptional profiling has been leveraged as a tool to monitor product quality and consistency during the manufacture of cell-based therapeutics including mesenchymal stem cells, induced pluripotent stem cells, macrophages, and cardiac progenitors [34–38]. A similar approach may be applied during SRC manufacture, with the expression of key regenerative gene networks serving as metrics of product character and potency [39, 40].

Taken together, these data suggest that SRC catalyze regenerative outcomes in the kidney in part by activating multiple transcriptomic networks associated with the promotion of regeneration while simultaneously suppressing alternate transcriptional pathways that may promote the continued development of renal disease and pathophysiology, see Additional file 1: Figure S17.

## Future Perspectives

SRC is currently being directly evaluated in Phase II clinical trials in patients with type 2 diabetes and chronic kidney disease (www.prokidney.com) This trial is built on the success of a Phase I clinical trial of 7 type II diabetic patients with CKD3/4. SRC formulated and delivered in a hydrogel carrier (“REACT”) was shown to be safe and well-tolerated. As SRC transitions through the clinical translation pathway, a clear understanding of how observed clinical outcomes are mediated through cell bioactivity is essential. The current manuscript establishes a framework for a putative mechanism of action for SRC in rodent preclinical models. In future studies, these results will be extended and further developed using human SRC (Basu et al., manuscript in preparation). As described in [39, 40], mechanistic pathways identified through transcriptomic and other related methodologies may be leveraged for the development and application of specific potency assays for quantitation of SRC regenerative bioactivity.

# Supplementary Information

## 



