# DESCRIPTION

## BACKGROUND

Lipids are indispensable ingredients in the food and cosmetics industries, and they are important precursors in the biodiesel and biochemical industries. Many oleaginous microorganisms produce lipids, including the well-characterized yeast Yarrowia lipolytica.

The lipid yield of oleaginous organisms can be increased by the up-regulation, down-regulation, or deletion of genes implicated in a lipid pathway. Recent data suggests that the activity of the diacylglycerol acyltransferase protein DGA1 may be a significant factor for accumulating high levels of lipids in oleaginous organisms. For example, it was reported that the up-regulation of the native Y. lipolytica diacylglycerol acyltransferase protein DGA1 in Y. lipolytica significantly increases its lipid yield and productivity (ME15:1-9 (2013)).

The Y. lipolytica DGA1 protein is a type 2 diacylglyecrol acyltransferase encoded by the Y. lipolytica diacylgylcerol acyltransferase gene DGAT2. DGA1 is one of the key enzymes in the lipid pathway, involved in the final step of triacylglycerol (“TAG”) synthesis. Triacylglycerols are the major form of storage lipids in Y. lipolytica. Yeast also contain a type 1 diacylglycerol acyltransferase gene DGAT1, which encodes the DGA2 protein.

Diacylglycerol acyltransferase genes can be introduced into a host genome to affect lipid production and composition, including the DGA1 and DGA2 genes from other organisms. For example, other oleaginous yeasts, such as Rhodosporidium toruloides and Lipomyces starkeyi, are able to accumulate significantly more lipids than wild type Y. lipolytica strains, and the expression of DGA1 proteins from organisms with higher native lipid production levels has a greater effect on Y. lipolytica lipid production than the overexpression of native Y. lipolytica DGA1 (U.S. Ser. No. 61/943,664 and PCT Patent Application No. PCT/US15/017227; hereby incorporated by reference).

Additionally, genes involved in the breakdown of lipids or in pathways that draw flux away from lipid biosynthesis have been deleted to increase a cell's lipid content. For example, Dulermo et al. demonstrated (feat the deletion of the triacylglycerol lipase gene TGL3 nearly doubled the total lipid content accumulated by Y. lipolytica (BB1831:1486-95 (2013)).

The successful upregulation of functional enzymes, however, is unpredictable at best. For example, other experiments have shown that expressing DGA1 from Mortierella alpine has no significant effect on Y. lipolytica lipid content (U.S. Pat. No. 7,198,937; hereby incorporated by reference). Similarly, expressing DGA2 has been shown to have no significant effect on the lipid content of yeast in the absence of other genetic modifications.

## SUMMARY

In some embodiments, the invention relates to a transformed cell, comprising a first genetic modification and second genetic modification, wherein said first genetic modification increases the activity of a native type 1 diacylgylcerol acyltransferase or encodes at least one copy of a type 1 diacylgylcerol acyltransferase gene native to the cell or frora a different species, and said second genetic modification increases the activity of a native type 2 diacylgylcerol acyltransferase or encodes at least one copy of a type 2 diacylgylcerol acyltransferase gene native to the cell or from a different species. In some embodiments, the transformed cell comprises a third genetic modification, wherein said third genetic modification decreases the activity of a triacylglycerol lipase in the cell.

In some embodiments, the invention relates to a transformed cell, comprising a genetic modification, wherein said genetic modification increases the activity of a native type 1 diacylgylcerol acyltransferase or encodes at least one copy of a type 1 diacylgylcerol acyltransferase gene native to the cell or from a different species.

In some embodiments, the invention relates to a transformed cell, comprising a genetic modification, wherein said genetic modification increases the activity of a native type 3 diacylglycerol acyltransferase or encodes at least one copy of a type 3 diacylglycerol acyltransferase gene native to the cell or from a different species.

In some aspects, the invention provides a method of increasing the lipid content of a cell, comprising transforming a parent cell with a first nucleotide sequence and a second nucleotide sequence, wherein said first nucleotide sequence increases the activity of a native type 1 diacylglycerol acyltransferase or encodes at least one copy of a type 1 diacylglycerol acyltransferase gene, and said second nucleotide sequence increases the activity of a native type 2 diacylglycerol acyltransferase or encodes at least one copy of a type 2 diacylglycerol acyltransferase gene. In some embodiments, the method comprises transforming the parent cell with a third nucleotide sequence, wherein said third nucleotide sequence decreases the activity of a triacylglycerol lipase. The aforementioned method may also be used to modify the lipid composition of a cell.

In some aspects, the invention provides a method of increasing the lipid content of a cell, comprising transforming a parent cell with a nucleotide sequence, wherein said nucleotide sequence increase the activity of a native type 1 diacylglycerol acyltransferase or encodes at least one copy of a type 1 diacylglycerol acyltransferase gene. The aforementioned method may also be used to modify the lipid composition of a cell.

In some aspects, the invention provides a method of increasing the lipid content of a cell, comprising transforming a parent cell with a nucleotide sequence, wherein said nucleotide sequence increase the activity of a native type 3 diacylglycerol acyltransferase or encodes at least one copy of a type 3 diacylglycerol acyltransferase gene. The aforementioned method may also be used to modify the lipid composition of a cell.

In some aspects, the invention provides a method of increasing the triacylglycerol content of a cell, comprising; (a) providing a cell, comprising (i) a first genetic modification, wherein said first genetic modification increases the activity of a native type 1 diacylglycerol acyltransferase or encodes at least one copy of a type 1 diacylglycerol acyltransferase gene native to the cell or from a different species; and (ii) a second genetic modification, wherein said second genetic modification increases the activity of a native type 2 diacylglycerol acyltransferase or encodes at least one copy of a type 2 diacylglycerol acyltransferase gene native to the cell or from a different species; (b) growing said cell under conditions whereby the first and second genetic modifications are expressed, thereby producing a triacylglycerol; and (c) optionally recovering the triacylglycerol. In some embodiments, the cell comprises a third genetic modification, wherein said third genetic modification decreases the activity of a triacylglycerol lipase in the cell. The aforementioned method may also be used to modify the lipid composition of a cell.

In some aspects, the invention provides a method of increasing the triacylglycerol content of a cell, comprising: (a) providing a cell comprising a genetic modification, wherein said genetic modification increases the activity of a native type 1 diacylglycerol acyltransferase or encodes at least oae copy of a type 1 diacylglycerol acyltransferase gene native to the cell or from a different species; (b) growing said cell under conditions whereby the genetic modification is expressed, thereby producing a triacylglycerol; and (c) optionally recovering the triacylglycerol.

In some aspects, the invention provides a method of increasing the triacylglycerol content of a cell, comprising: (a) providing a cell comprising a genetic modification, wherein said genetic modification increases the activity of a native type 3 diacylglycerol aeyltransfcrasc or encodes at least oae copy of a type 3 diacylglycerol acyltransferase gene native to the cell or from a different species; (b) growing said cell under conditions whereby the genetic modification is expressed, thereby producing a triacylglycerol; and (c) optionally recovering the triacylglycerol.

## DETAILED DESCRIPTION

### Overview

Disclosed are methods and compositions for creating transformed cells with increased triacylglycerol content. Expressing the type 2 diacylgylcerol acyltransferase DGA1 increases the amount of protein that can synthesize triacylglycerol, and expressing the DGA1 protein from Rhodosporidium toruloides in a Yarrowia lipolytica cell is effective at increasing the triacylglycerol content of the cell (U.S. Ser. No. 61/943,664 and PCT Patent Application No. PCT/US15/017227; hereby incorporated by reference). The type 1 diacylgylcerol acyltransferase DGA2 can also catalyze the synthesis of triacylglycerol, and the expression of carefully selected DGA1 and DGA2 transgenes may further increase the lipid content of an oleaginous cell relative to DGA1 alone. Specifically, Yarrowia lipolytica that expresses DGA1 from Rhodasporidmm toruloides and DGA2 from Claviceps purpurea produces high triacylglycerol yields. Finally, triacylglycerol lipases catalyze the degradation of triacylglycerols, and thus, the down-regulation of triacylglycerol lipases can increase the triacylglycerol content of a cell. Specifically, Yarrowia lipolytica cells that contain a TGL3 knockout and express DGA1 from Rhodosporidium toruloides produce higher triacylglycerol yields than DGA1-expressing controls (U.S. Ser. No. 61/987,098 and PCT Patent Application No. PCT/US15/28760; hereby incorporated by reference). The combination of DGA1 and DGA2 expression with TGL3 down-regulation may further increase triacylglycerol yields.

The simultaneous expression of DGA1 and DGA2 and concomitant down-regulation of TGL3 could be an attractive approach to increase the triacylglycerol content of a cell; however, the manipulation of proteins that affect a metabolic pathway is unpredictable at best. For example, the overexpression of native DGA2 alone in Y. lipolytica does not increase the cell's lipid production efficiency, whereas DGA1 increases lipid production. DGA2 localizes to the ER and synthesizes triacylglycerol in newly formed lipid bodies. In contrast, DGA1 localizes to lipid body membranes and synthesizes triacylglycerols within these lipid bodies. Whether this distinction or some other difference affects the cell's ability to suppress the effects of a genetic modification is not well understood. Thus, the combination of DGA1 and DGA2 expression with a TGL3 knockout would not be expected to produce cells with a higher lipid content than those cells containing one or two of the genetic modifications.

Disclosed is the successful combination of DGA1 and DGA2 expression and TGL3 down-regulation to increase the triacylglycerol content of a cell. For example, a Yarrowia lipolytica strain that contains a TGL3 knockout and expresses DGA1 from Rhodosporidium toruloides and DGA2 from Claviceps purpurea produced high triacylglycerol yields.

### Definitions

The articles “a” and “an” are used herein to refer to one or to more than one (i.e., to at least one) of the grammatical object of the article. By way of example, “an element” means one element or more than one element.

The term “activity” refers to the total capacity of a cell to perform a function. For example, a genetic modification that decreases the activity of a triacylglycerol lipase in a cell may reduce the amount of triacylglycerol lipase in a cell or reduce the efficiency of triacylglycerol lipase. A triacylglycerol lipase knockout reduces the amount of triacylglycerol lipase in the cell. Alternatively, a mutation to a triacylglycerol lipase gene may reduce the efficiency of its triacylglycerol lipase protein product with little effect on the amount of cellular triacylglycerol lipase. Mutations that reduce the efficiency of triacylglycerol lipase may affect the active site, for example, by changing one or more active site residues; they may impair the enzyme's kinetics, for example, by sterically blocking substrates or products; they may affect protein folding or dynamics, for example, by reducing the proportion of properly-folded enzymes; they may affect protein localization, for example, by preventing the lipase from localizing to lipid particles; or they may affect protein degradation, for example, by adding one or more protein cleavage sites or by adding one or more residues or amino acid sequences that target the protein for proteolysis. These mutations affect coding regions. Mutations that decrease triacylglycerol lipase activity may instead affect the transcription or translation of the gene. For example, imitation to a triacylglycerol lipase enhancer or promoter can reduce triacylglycerol lipase activity by .reducing its expression. Mutating or deleting the non-coding portions of a triacylglycerol lipase gene, such as its introns, may also reduce transcription or translation. Additionally, mutations to the upstream regulators of a triacylglycerol lipase may affect triacylglycerol lipase activity; for example, the over-expression of one or more repressors may decrease triacylglycerol lipase activity, and a knockout or mutation of one or more activators may similarly decrease triacylglycerol lipase activity.

A genetic modification that increases the activity of a diacylglycerol acyltransferase in a cell may increase the amount of triacylglycerol acyltransferase in a cell or increase the efficiency of diacylglycerol acyltransferase. For example, the genetic modification may simply insert an additional copy of diacylglycerol acyltransferase into the cell such that the additional copy is transcribed and translated into additional functional diacylglycerol acyltransferase. The added diacylglycerol acyltransferase gene can be native to the host organism or from a different organism. Alternatively, mutating or deleting the non-coding portions of a native diacylglycerol acyltransferase gene, such as its introns, may also increase translation. A native diacylglycerol acyltransferase gene can be altered by adding a new promoter that causes more transcription. Similarly, enhancers may be added to the diacylglycerol acyltransferase gene that increase transcription, or silencers may be mutated, or deleted from the diacylglycerol acyltransferase gene to increase transcription. Mutations to a native gene's coding region might also increase diacylglycerol acyltransferase activity, for example, by producing a protein variant that does not interact with inhibitory proteins or molecules. The over-expression of one or more activators may increase diacylglycerol acyltransferase activity by increasing the expression of a diacylglycerol acyltransferase protein, and a knockout or mutation of one or more repressors may similarly increase diacylglycerol acyltransferase activity.

The term “biologically-active portion” refers to an amino acid sequence that is less than a full-length amino acid sequence, but exhibits at least one activity of the full length sequence. For example, a biologic ally-active portion of a diacylglycerol acyltransferase may refer to one or more domains of DGA1 or DGA2 having biological activity for converting acyl-CoA and diacylglycerol to triacylglycerol. Biologically-active portions of a DGA1 protein include peptides or polypeptides comprising amino acid sequences sufficiently similar to or derived from the amino acid sequence of the DGA1 protein, e.g., the amino acid sequence as set forth in SEQ ID NOS: 15, 17, 19, 21, 23, 25, 27, 29, 31, 33, 51, 53, 35, 57, 59, 61, 63, 65, 67, and 69, which include fewer amino acids than the full length DGA1, and exhibit at least one activity of a DGA1 protein. Similarly, biologically-active portions of a DGA2 protein include peptides or polypeptides comprising amino acid sequences sufficiently similar to or derived from the amino acid sequence of the DGA2 protein, e.g., the amino acid sequence as set forth in SEQ ID NOs: 1, 3, 5, 7, 9, 11, 13, 71, 73, 75, 77, 79, 81, and 83, which include fewer amino acids than the full length DGA2, and exhibit at least one activity of a DGA2 protein. Similarly, biologically-active portions of a DGA3 protein include peptides or polypeptides comprising amino acid sequences sufficiently similar to or derived from the amino acid sequence of the DGA3 protein, e.g., the amino acid sequence as set forth in SEQ ID NOS: 87 and 89, which, include fewer amino acids than the full length DGA3, and exhibit at least one activity of a DGA3 protein. A biologically-active portion of a diacylglycerol acyltransferase may comprise, for example, 100, 101, 102, 103, 104, 105, 106, 107, 108, 109, 110, 111, 112, 113, 114, 115, 116, 117, 118, 119, 120, 121, 122, 123, 124, 125, 126, 127, 128, 129, 130, 131, 132, 133, 134, 135, 136, 137, 138, 139, 140, 141, 142, 143, 144, 145, 146, 147, 148, 149, 150, 151, 152, 153, 154, 155, 156, 157, 158, 159, 160, 161, 162, 163, 164, 165, 166, 167, 168, 169, 170, 171, 172, 173, 174, 175, 176, 177, 178, 179, 180, 181, 182, 183, 184, 185, 186, 187, 188, 189, 190, 191, 192, 193, 194, 195, 196, 197, 198, 199, 200, 201, 202, 203. 204, 205, 206, 207, 208, 209, 210, 211, 212, 213, 214, 215, 216, 217, 218, 219, 220, 221, 222, 223, 224, 225, 226, 227, 228, 229, 230, 231, 232, 233, 234, 235, 236, 237, 238, 239, 240, 241, 242, 243, 244, 245, 246, 247, 248, 249, 250, 251, 252, 253, 254, 255, 256, 257, 258, 259, 260, 261, 262, 263, 264, 265, 266, 267, 268, 269, 270, 271, 272, 273, 274, 275, 276, 277, 278, 279, 280, 281, 282, 283, 284, 285, 286, 287, 288, 289, 290, 291, 292, 293, 294, 295, 296, 297, 298, 299, 300, 301, 302, 303, 304, 305, 306, 307, 308, 309, 310, 311, 312, 313, 314, 315, 316, 317, 318, 319, 320, 321, 322, 323, 324, 325, 326, 327, 328, 329, 330, 331, 332, 333, 334, 335, 336, 337, 338, 339, 340, 341, 342, 343, 344, 345, 346, 347, 348, 349, 350, 351, 352, 353, 354, 355, 356, 357, 358, 359, 360, 361, 362, 363, 364, 365, 366, 367, 368, 369, 370, 371, 372, 373, 374, 375, 376, 377, 378, 379, 380, 381, 382, 383, 384, 385, 386, 387, 388, 389, 390, 391, 392, 393, 394, 395, 396, 397, 398, 399, 400, 401, 402, 403, 404, 405, 406, 407, 408. 409, 410, 411, 412, 413, 414, 415, 416, 417, 418, 419, 420, 421, 422, 423, 424, 425, 426, 427, 428, 429, 430, 431, 432, 433, 434, 435, 436, 437, 438, 439, 440, 441, 442, 443, 444, 445, 446, 447, 448, 449, 450, 451, 452, 453, 454, 455, 456, 457, 458, 459, 460, 461, 462, 463, 464, 465, 466, 467, 468, 469, 470, 471, 472, 473, 474, 475, 476, 477, 478, 479, 480, 481, 482, 483, 484, 485, 486, 487, 488, 489, 490, 491, 492, 493, 494, 495, 496, 497, 498, 499, 500, 501, 502, 503, 504, 505, 506, 507, 508, 509, 510, 511, 512, 513, 514, 515, 516, 517, 518, 519, 520, 521, 522, 523, 524, 525, 526, 527, 528, 529, 530, 531, 532, 533, 534, 535, 536, 537, 538, 539, 540, 541, 542, 543, 544, 545, 546, 547, 548, 549, 550, 551, 552, 553, 554, 555, 556, 557, 558, 559, 560, 561, 562, 563, 564, 565, 566, 567, 568, 569, 570, 571, 572, 573, 574, 575, 576, 577, 578, 579, 580, 581, 582, 583, 584, 585, 586, 587, 588, 589, 590, 591, 592, 593, 594, 595, 596, 597, 598, 599, 600, 601, 602, 603, 604, 605, 606, 607, 608, 609, 610, 611, 612, 613, 614, 615, 616, 617, 618, 619, 620, 621, 622, 623, 624, 625, 626, 627, 628, 629, 630, 631, 632, 633, 634, 635, 636, 637, 638, 639, 640, 641, 642, 643, 644, 645, 646, 647, 648, 649, 650, 651, 652, 653, 654, 655, 656, 657, 658, 659, 660, 661, 662, 663, 664, 665, 666, 667, 668, 669, 670, 671, 672, 673, 674, 675, 676, 677, 678, 679, 680, 681, 682, 683, 684, 685, 686, 687, 688, 689, 690, 691, 692, 693, 694, 695, or 696 amino acids. Typically, biologically active portions comprise a domain or motif having the catalytic activity of converting acyl-CoA and diacylgylcerol to triacylglycerol. A biologically active portion of a DGA1 protein can be a polypeptide which is, for example, 262 amino acids in length.

The term “DGAT1” refers to a gene that encodes a type 1 diacylgylcerol acyltransferase protein, such as a gene that encodes a DGA2 protein.

The term “DGAT2” refers to a gene that encodes a type 2 diacylgylcerol acyltransferase protein, such as a gene that encodes a DGA1 protein.

The term “DGAT3” refers to a gene that encodes a type 3 diacylglyeerol acyltransferase protein, such as a gene that encodes a DGA3 protein.

“Diacylglyceride,” “diacylgylcerol,” and “diglyceride” are esters comprised of glycerol and two fatty acids.

The terms “diacylglycerol acyltransferase” and “DGA” refer to any protein that catalyzes the formation of triaeylglycerides from diacylgylcerol. Diacylglycerol acyltransferases include type 1 diacylglycerol acyltransferases (DGA2), type 2 diacylgylcerol acyltransferases (DGA1), and type 3 diacylgylcerol acyltransferases (DGA3) and all homologs that catalyze the above-mentioned reaction.

The terms “diacylglycerol acyltransferase, type 1” and “type 1 diacylglycerol acyltransferases” refer to DGA2 and DGA2 orthologs.

The terms “diacylglycerol acyltransferase, type 2” and “type 2 diacylglycerol acyltransferases” refer to DGA1 and DGA1 orthologs.

The terms “diacylglycerol acyltransferase, type 3” and “type 3 diacylglycerol acyltransferases” refer to DGA1 and DGA1 orthologs.

The term “domain” refers to a part of the amino acid sequence of a protein that is able to fold in to a stable three-dimensional structure independent of the rest of the protein.

The term “drug” refers to any molecule that inhibits cell growth or proliferation, thereby providing a selective advantage to cells that contain a gene that confers resistance to the drug. Drugs include antibiotics, antimicrobials, toxins, and pesticides.

“Dry weight” and “dry cell weight” mean weight determined in the relati ve absence of water. For example, reference to oleaginous cells as comprising a specified percentage of a particular component by dry weight means that the percentage is calculated based on the weight of the cell after substantially all water has been removed.

The term “encode” refers to nucleic acids that comprise a coding region, portion of a coding region, or compliments thereof. Both DNA and RNA may encode a gene. Both DNA and RNA may encode a protein.

The term “exogenous” refers to anything that is introduced into a cell. An “exogenous nucleic acid” is a nucleic acid that entered a cell through the cell membrane. An exogenous nucleic acid may contain a nucleotide sequence that exists in the native genome of a cell and/or nucleotide sequences that did not previously exist in the cell's genome. Exogenous nucleic acids include exogenous genes. An “exogenous gene” is a nucleic acid that codes for the expression of an RNA and/or protein that has been introduced into a cell (e.g., by transformation/transfection), and is also referred to as a “transgene.” A cell comprising an exogenous gene may be referred to as a recombinant cell, into which additional exogenous gene(s) may be introduced. The exogenous gene may be from the same or different species relative to the cell being transformed. Thus, an exogenous gene can include a native gene that occupies a different location in the genome of the cell or is under different control, relative to the endogenous copy of the gene. An exogenous gene may be present in more than one copy in the cell. An exogenous gene may be maintained in a cell as an insertion into the genome (nuclear or plastid) or as an episomal molecule.

The term “expression” refers to the amount of a nucleic acid or amino acid sequence (e.g., peptide, polypeptide, or protein) in a cell. The increased expression of a gene refers to the increased transcription of that gene. The increased expression of an amino acid sequence, peptide, polypeptide, or protein refers to the increased translation of a nucleic acid encoding the amino acid sequence, peptide, polypeptide, or protein.

The term “gene,” as used herein, may encompass genomic sequences that contain exons, particularly polynucleotide sequences encoding polypeptide sequences involved in a specific activity. The term further encompasses synthetic nucleic acids that did not derive from genomic sequence. In certain embodiments, the genes lack introns, as they are synthesized based on the known DNA sequence of cDNA and protein sequence. In other embodiments, the genes are synthesized, non-native cDNA wherein the codons have been optimized for expression in Y. lipolytica based on eodon usage. The term can further include nucleic acid molecules comprising upstream, downstream, and/or intron nucleotide sequences.

The term “genetic modification” refers to the result of a transformation. Every transformation causes a genetic modification by definition.

The term “homolog” as used herein, refers to (a) peptides, oligopeptides, polypeptides, proteins, and enzymes having amino acid substitutions, deletions and/or insertions relative to the unmodified protein in question and having similar biological arid functional activity as the unmodified protein from which they are derived, and (b) nucleic, acids which encode peptides, oligopeptides, polypeptides, proteins, and enzymes with the same characteristics described in (a).

“Inducible promoter” is a promoter that mediates the transcription of an operably linked gene in response to a particular stimulus.

The term “integrated” refers to a nucleic acid that is maintained in a cell as an insertion into the cell's genome, such as insertion into a chromosome, including insertions into a plastid genome.

“In operable linkage” refers to a functional linkage between two nucleic acid sequences, such a control sequence (typically a promoter) and the linked sequence (typically a sequence that encodes a protein, also called a coding sequence). A promoter is in operable linkage with a gene if it can mediate transcription of the gene.

The term “knockout mutation” or “knockout” refers to a genetic modification that prevents a native gene from being transcribed and translated into a functional protein.

The term “native” refers to the composition of a cell or parent cell prior to a transformation event. A “native gene” refers to a nucleotide sequence that encodes a protein that has not been introduced into a cell by a transformation event. A “native protein” refers to an amino acid sequence that is encoded by a native gene.

The terms “nucleic acid” refers to a polymeric form of nucleotides of any length, either deoxyribonucleotides or ribonucleotides, or analogs thereof. Polynucleotides may have any three-dimensional structure, and may perform any function. The following are non-limiting examples of polynucleotides; coding or non-coding regions of a gene or gene fragment, loci (locus) defined from linkage analysis, exons, introns, messenger RNA (mRNA), transfer RNA, ribosomal RNA, ribozymes, cDNA, recombinant polynucleotides, branched polynucleotides, plasmids, vectors, isolated DNA of any sequence, isolated RNA of any sequence, nucleic acid probes, and primers. A polynucleotide may comprise modified nucleotides, such as methylated nucleotides and nucleotide analogs. If present, modifications to the nucleotide structure may be imparted, before or after assembly of the polymer. A polynucleotide may be further modified, such as by conjugation with a labeling component. In all nucleic acid sequences provided herein, U nucleotides are interchangeable with T nucleotides.

The term “parent cell” refers to every cell from which a cell descended. The genome of a cell is comprised of the parent cell's genome and any subsequent genetic modifications to the parent cell's genome.

As used herein, the term “plasmid” refers to a circular DNA molecule that is physically separate from an organism's genomic DNA. Plasmids may be linearized before being introduced into a host cell (referred to herein as a linearized plasmid). Linearized plasmids may not be self-replicating, but may integrate into and be replicated with the genomic DNA of an organism.

The term “portion” refers to peptides, oligopeptides, polypeptides, protein domains, and proteins. A nucleotide sequence encoding a “portion of a protein” includes both nucleotide sequences that can be transcribed and/or translated and nucleotide sequences that must undergo one or more recombination events to be transcribed and/or translated. For example, a nucleic acid may comprise a nucleotide sequence encoding one or more amino acids of a selectable marker protein. This nucleic acid can be engineered to recombine with one or more different nucleotide sequences that encode the remaining portion of the protein. Such nucleic adds are use fill for generating knockout mutations because only recombination with the target sequence is likely to reconstitute the full-length selectable marker gene whereas random-integration events are unlikely to result in a nucleotide sequence that can produce a functional marker protein. A “biologically-active portion” of a polypeptide is any amino acid sequence found in the polypeptide's amino acid sequence that is less than the full amino acid sequence but can perform the same function as the full-length polypeptide. A biologically-active portion of a diacylgylcerol acyltransferase includes any amino acid sequence found in a full-length, diacylgylcerol acyltransferase that, can catalyze the formation of triacylglycerol from diacylgylcerol and acyl-CoA. A biologically-active portion, of a polypeptide includes portions of the polypeptide that have the same activity as the full-length peptide arid every portion that has more activity than background. For example, a biologically-active portion of a diacylgylcerol acyltransferase may have 0.1, 0.5, 1, 2, 3, 4, 5, 10, 25, 30, 35, 40, 45, 50, 55, 60, 65, 70, 75, 80, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99, 99.5, 99.6, 99.7, 99.8, 99.9, 100, 100.1, 100.2, 100.3, 100.4, 100.5, 100.6, 100.7, 100.8, 100.9, 101, 105, 110, 115, 120, 125, 130, 135, 140, 145, 150, 160, 170, 180, 190, 200, 220, 240, 260, 280, 300, 320, 340, 360, 380, 400 percent activity relative to the full-length polypeptide or higher. A biologically-active portion of a polypeptide may include portions of a peptide that lack a domain that targets the polypeptide to a cellular compartment.

A “promoter” is a nucleic acid control sequence that directs the transcription of a nucleic acid. As used herein, a promoter includes the necessary nucleic acid sequences near the start site of transcription. A promoter also optionally includes distal enhancer or repressor elements, which can be located as much as several thousand base pairs from the start site of transcription.

“Recombinant” refers to a cell, nucleic acid, protein, or vector, which has been modified due to the introduction of an exogenous nucleic acid or the alteration of a native nucleic acid. Thus, e.g., recombinant cells can express genes that are not found within the native (non-recombinant) form of the cell or express native genes differently than those genes are expressed by a non-recombinant cell. Recombinant cells can, without limitation, include recombinant nucleic acids that encode for a gene product or for suppression elements such as mutations, knockouts, antisense, interfering RNA (RNAi), or dsRNA that reduce the levels of active gene product in a cell. A “recombinant nucleic acid” is a nucleic acid originally formed in vitro, in general, by the manipulation of nucleic acid, using polymerases, ligases, exonucleases, and endonucleases, or otherwise is in a form not normally found in nature. Recombinant nucleic acids may be produced, for example, to place two or more nucleic acids in operable linkage. Thus, an isolated nucleic acid or an expression vector formed in vitro by ligating DNA molecules that are not normally joined in nature, are both considered recombinant for the purposes of this invention. Once a recombinant nucleic acid is made and introduced into a host cell or organism, it may replicate using the in vivo cellular machinery of the host cell; however, such nucleic acids, once produced recombinantly, although subsequently replicated intracellularly, are still considered recombinant for purposes of this invention. Similarly, a “recombinant protein” is a protein made using recombinant techniques, i.e., through the expression of a recombinant nucleic acid.

The term “regulatory region” refers to nucleotide sequences that affect the transcription or translation of a gene but do not encode an amino acid sequence. Regulatory regions include promoters, operators, enhancers, and silencers.

“Transformation” refers to the transfer of a nucleic acid into a host organism or the genome of a host organism, resulting in genetically stable inheritance. Host organisms containing the transformed nucleic acid fragments are referred to as “recombinant”, “transgenic” or “transformed” organisms. Thus, isolated polynucleotides of the present invention can be incorporated into recombinant constructs, typically DNA constructs, capable of introduction into and replication in a host cell. Such a construct can be a vector that includes a replication system and sequences that are capable of transcription and translation of a polypeptide-encoding sequence in a given host cell. Typically, expression vectors include, for example, one or more cloned genes under the transcriptional control of 5′ and 3′ regulatory sequences and a selectable marker. Such vectors also can contain a promoter regulatory region (e.g., a regulatory region controlling inducible or constitutive, environmentally- or developmentally-regulated, or location-specific expression), a transcription Initiation start site, a ribosome binding site, a transcription termination site, and/or a polyadenylation signal.

The term “transformed cell” refers to a cell that has undergone a transformation. Thus, a transformed cell comprises the parent's genome and an inheritable genetic modification.

The terms “triacylglyceride,” “triacylglycerol,” “triglyceride,” and “TAG” are esters comprised of glycerol and three fatty acids.

The term “triacylglycerol lipase” refers to any protein that can catalyze the removal of a fatty acid chain from a triacylglycerol. Triacylglycerol lipases include TGL3, TGL4, and TGL3/4.

The term “vector” refers to the means by which a nucleic acid can be propagated and/or transferred between organisms, cells, or cellular components. Vectors include plasmids, linear DNA fragments, viruses, bacteriophage, pro-viruses, phagemids, transposons, and artificial chromosomes, and the like, that may or may not be able to replicate autonomously or integrate into a chromosome of a host cell.

### Microbe Engineering

A. Overview

In certain embodiments, the invention relates to a microorganism genetically modified to increase its triacylglycerol content or modify its lipid profile.

Genes and gene products may be introduced into microbial host cells. Suitable host cells for expression of the genes and nucleic acid molecules are microbial hosts that can be found broadly within the fungal or bacterial families. Examples of suitable host strains include but are not limited to fungal or yeast species, such as Arxula, Aspergillus, Aurantiochytrium, Candida, Claviceps, Cryptococcus, Cumnnghamella, Geotrichum, Hansemda, Kluyveromyces, Kodamaea, Leucosporidiella, Lipomyces, Mortierella, Ogataea, Pichia, Prototheca, Rhizopus, Rhodosporidium, Rhodotorula, Saccharomyces, Schizosaccharomyces, Tremella, Trichosporon, Wickerhamomyces, Yarrowia, or bacterial species, such as members of proteobacteria and actinomyeetes, as well as the genera Acinetobacter, Arthrobacter, Brevibacterium, Acidovorax, Bacillus, Clostridia, Streptomyces, Escherichia, Salmonella, Pseudomonas, and Cornyebacterium. Yarrowia lipolytica and Arxula adeninivorans are well-suited for use as the host microorganism because they can accumulate a large percentage of their weight as triacylglycerols.

Microbial expression systems and expression vectors containing regulatory sequences that direct high level expression of foreign proteins are known to those skilled in the art. Any of these could be used to construct chimeric genes to produce any one of the gene products of the instant sequences. These chimeric genes could then be introduced into appropriate microorganisms via transformation techniques to provide high-level expression of the enzymes.

For example, a gene encoding an enzyme can be cloned in a suitable plasmid, and an aforementioned starting parent strain as a host can be transformed with the resulting plasmid. Th s approach, can increase the copy number of each of the genes encoding the enzymes and, as a result, the activities of the enzymes can be increased. The plasmid is not particularly limited so long as it renders a desired genetic modification inheritable to the microorganism's progeny.

Vectors or cassettes useful for the transformation of suitable host cells are well known in the art. Typically the vector or cassette contains sequences that direct the transcription and translation of the relevant gene, a selectable marker, and sequences that allow autonomous replication or chromosomal integration. Suitable vectors comprise a region 5′ of the gene harboring transcriptional initiation controls and a region 3′ of the DNA fragment which controls transcriptional termination. Both control regions may be derived from genes homologous to the transformed host cell, although it is to be understood that such control regions need not be derived from the genes native to the specific species chosen as a production host.

Promoters, cDNAs, and 3′UTRs, as well, as other elements of the vectors, can be generated through cloning techniques using fragments isolated from native sources (Green & Sambrook, Molecular Cloning: A Laboratory Manual (4th ed., 2012); U.S. Pat. No. 4,683,202; hereby incorporated by reference). Alternatively, elements can be generated synthetically using known methods (Gene 164:49-53 (1995)).

B. Homologous Recombination

Homologous recombination is the ability of complementary DNA sequences to align and exchange regions of homology. Transgenic DNA (“donor”) containing sequences homologous to the genomic sequences being targeted (“template”) is introduced into the organism and then undergoes recombina tion into the genome at the site of the corresponding homologous genomic sequences.

The ability to carry out homologous recombination in a host organism has many practical implications for what can be carried out at the molecular genetic level and is useful in the generation of a microbe that can produce a desired product. By its very nature homologous recombination is a precise gene targeting event, and hence, most transgenic lines generated with the same targeting sequence will be essentially identical in terms of phenotype, necessitating the screening of far fewer transformation events. Homologous recombination also targets gene insertion events into the host chromosome, potentially resulting in excellent genetic stability, even in the absence of genetic selection. Because different chromosomal loci will likely impact gene expression, even from exogenous promoters/UTRs, homologous recombination can be a method of querying loci in an unfamiliar genome environment and to assess the impact of these environments on gene expression.

A particularly useful genetic engineering approach using homologous recombination is to co-opt specific host regulatory elements such as promoters/UTRs to drive heterologous gene expression in a highly specific fashion.

Because homologous recombination is a precise gene targeting event, it can be used to precisely modify any nucleotide(s) within a gene or region of interest, so long as sufficient flanking regions have been identified. Therefore, homologous recombination can be used as a means to modify regulatory sequences impacting gene expression of RNA and/or proteins. It can also be used to modify protein coding regions in an effort to modify enzyme activities such as substrate specificity, affinities and Km, thereby affecting a desired change in the metabolism of the host cell. Homologous recombination provides a powerful means to manipulate the host genome resulting in gene targeting, gene conversion, gene deletion, gene duplication, gene inversion, and exchanging gene expression regulatory elements such as promoters, enhancers and 3′UTRs.

Homologous recombination can be achieved by using targeting constructs containing pieces of endogenous sequences to “target” the gene or region of interest within the endogenous host cell genome. Such targeting sequences can either be located 5′ of the gene or region of interest, 3′ of the gene/region of interest or even flank, the gene/region of interest. Such targeting constructs can be transformed into the host cell either as a superceded plasmid DNA with additional vector backbone, a PGR product with no vector backbone, or as a linearized molecule. In some cases, it may be advantageous to first expose the homologous sequences within the transgenic DNA (donor DNA) by cutting the transgenic DNA with a restriction enzyme. This step can increase the recombination efficiency and decrease the occurrence of undesired events. Other methods of increasing recombination efficiency include using PCR to generate transforming transgenic DNA containing linear ends homologous to the genomic sequences being targeted.

C. Vectors and Vector Components

Vectors for transforming microorganisms in accordance with the present invention can be prepared by known techniques familiar to those skilled in the art in view of the disclosure herein. A vector typically contains one or more genes, in which each gene codes for the expression of a desired product (the gene product) and is operably linked to one or more control sequences that regulate gene expression or target the gene product to a particular location in the recombinant cell.

1. Control Sequences

Control sequences are nucleic acids that regulate the expression of a coding sequence or direct a gene product to a particular location inside or outside a cell. Control sequences that regulate expression include, for example, promoters that regulate transcription of a coding sequence and terminators that terminate transcription of a coding sequence. Another control sequence is a 3′ untranslated sequence located at the end of a coding sequence that encodes a polyadenylation signal. Control sequences that direct gene products to particular locations include those that encode signal peptides, which direct the protein to which they are attached to a particular location inside or outside the cell.

Thus, an exemplary vector design for expression of a gene in a microbe contains a coding sequence for a desired gene product (for example, a selectable marker, or an enzyme) in operable linkage with a promoter active in yeast. Alternatively, if the vector does not contain a promoter in operable linkage with the coding sequence of interest, the coding sequence can be transformed into the cells such that it becomes operably linked to an endogenous promoter at the point of vector integration.

The promoter used to express a gene can be the promoter naturally linked to that gene or a different promoter.

A promoter can generally be characterized as constitutive or inducible. Constitutive promoters are generally active or function to drive expression at all times (or at certain times in the cell life cycle) at the same level inducible promoters, conversely, are active (or rendered inactive) or are significantly up- or down-regulated only in response to a stimulus. Both types of promoters find application in the methods of the invention. Inducible promoters useful in the invention include those that mediate transcription of an operably linked gene in response to a stimulus, such as an exogenously provided small molecule, temperature (heat or cold), lack of nitrogen in culture media, etc. Suitable promoters can activate transcription of an essentially silent gene or upregulate, preferably substantially, transcription of an operably linked gene that is transcribed at a low level.

Inclusion of termination region control sequence is optional, and if employed, then the choice is primarily one of convenience, as the termination region is relatively interchangeable. The termination region may be native to the transcriptional initiation region (the promoter), may be native to the DNA sequence of interest, or may be obtainable from another source (See, e.g., Chen & Orozco, Nucleic Acids Research 16:8411 (1988)).

2. Genes and Codan Optimization

Typically, a gene includes a promoter, coding sequence, and termination control sequences. When assembled by recombinant DNA technology, a gene may be termed an expression cassette and may be flanked by restriction sites for convenient insertion into a vector that is used to introduce the recombinant gene into a host cell. The expression cassette can be flanked by DNA sequences from the genome or other nucleic acid target to facilitate stable integration of the expression cassette into the genome by homologous recombination. Alternatively, the vector and its expression cassette may reman unintegrated (e.g., an episome), in which case, the vector typically includes an origin of replication, which is capable of providing for replication of the vector DNA.

A common gene present on a vector is a gene that codes for a protein, the expression of which allows the recombinant cell containing the protein to be differentiated from cells that do not express the protein. Such a gene, and its corresponding gene product, is called a selectable marker or selection marker. Any of a wide variety of selec table markers can be employed in a transgene construct useful for transforming the organisms of the invention.

For optimal expression of a recombinant protein, it is beneficial to employ coding sequences that produce mRNA with codons optimally used by the host cell to be transformed. Thus, proper expression of transgencs can require that the codon usage of the transgene matches the specific codon bias of the organism in which the transgene is being expressed. The precise mechanisms underlying this effect are many, but include the proper balancing of available aminoacylated tRNA pools with proteins being synthesized in the cell, coupled with more efficient translation of the transgenic messenger RNA (mRNA) when this need is met. When codon usage in the transgene is not optimized, available tRNA pools are not sufficient to allow for efficient translation of the transgenic mRNA resulting in ribosomal stalling and termination and possible instability of the transgenic mRNA.

C. Transformation

Cells can be transformed bv any suitable technique including, e.g., biolistics, eleetroporatkm, glass bead transformation, and silicon carbide whisker transformation. Any convenient technique for introducing a transgenic into a microorganism can be employed in the present invention. Transformation can be achieved by, for example, the method of D. M. Morrison (Methods in Enzymology 68:326 (1979)), the method by increasing permeability of recipient cells for DNA with calcium chloride (Mandel & Higa, J. Molecular Biology, 53:159 (1970)), or the like.

Examples of expression of transgenes in oleaginous yeast (e.g., Yarrowia lipolytica) can be found in the literature (Bordes et al., J. Microbiological Methods, 70:493 (2007); Chen et al., Applied Microbiology & Biotechnology 48:232 (1997)). Examples of expression of exogenous genes in bacteria such as E. coli are well known (Green & Sambrook, Molecular Cloning: A Laboratory Manual, (4th ed., 2012)).

Vectors for transformation of microorganisms in accordance with the present invention can be prepared by known techniques familiar to those skilled in the art. In one embodiment, an exemplary vector design for expression of a gene in a microorganism contains a gene encoding an enzyme in operable linkage with a promoter active in the microorganism. Alternatively, if the vector does not contain a promoter in operable linkage with the gene of interest, the gene can be transformed into the cells such that it becomes operably linked to a native promoter at the point of vector integration. The vector can also contain a second gene that encodes a protein. Optionally , one or both gcne(s) is/are followed by a 3′ untranslated sequence containing a polyadenyiation signal. Expression cassettes encoding the two genes can be physically linked in the vector or on separate vectors. Co-transformation of microbes can also be used, in which distinct vector molecules are simultaneously used to transform cells (Protist 155:381-93 (2004)). The transformed cells can be optionally selected based upon the ability to grow in the presence of the antibiotic or other selectable marker under conditions in which cells lacking the resistance cassette would not grow.

### Exemplary Nucleic Acids, Cells, and Methods

A. Diacylglycerol Acyltransferase Nucleic Aacid Molecules and Vectors

In some embodiments, the type 2 diacylgylcerol acyltransferase is DGA1. For example, the diacylgylcerol acyltransferase may be a DGA1 protein, encoded by a DGAT2 gene selected from the group consisting of Arxula adeninivorans, Aspergillus terreus, Aurantiochytrium limacinum, Claviceps purpurea, Gloeophyllum trabeum, Lipomyces starkeyi, Microbotryum violaceum, Pichia guilliermondii, Phaeodactylum tricornutum, Puccinia gramimis, Rhodosporidium diobovatum, Rhodosporidnm toruloides, Rhodotorula graminis, and Yarrowia lipolytica.

The DGAT2 gene may have a nucleotide sequence set forth in SEQ ID NO: 16, 18, 20, 22, 24, 26, 28, 30, 32, 34, 52, 54, 56, 58, 60, 62, 64, 66, 68, or 70. In other embodiments, the DGAT2 gene is substantially identical to SEQ ID NO: 16, 18, 20, 22, 24, 26, 28, 30, 32, 34, 52, 54, 56, 58, 60, 62, 64, 66, 68, or 70, and the nucleotide sequence encodes a protein that retains the functional acti vity of a protein encoded by SEQ ID NO: 16, 18, 20, 22, 24, 26, 28, 30, 32, 34, 52, 54, 56, 58, 60, 62, 64, 66, 68, or 70, yet differs in nucleotide sequence due to natural allelic variation or mutagenesis. In another embodiment, the DGAT2 gene comprises a nucleotide sequence at least about 70%, 71%, 72%, 73%, 74%, 73%, 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, 99.1%, 99,2%, 99.3%, 99.4%, 99.5%, 99.6%, 99.7%, 99.8%, 99.9%, or more identical to SEQ ID NO: 16, 18, 20, 22, 24, 26, 28, 30, 32, 34, 52, 54 56, 58, 60, 62, 64, 66, 68, or 70.

The DGA1 protein may have an amino acid sequence set forth in SEQ ID NO: 15, 17, 19, 21, 23, 25, 27, 29, 31, 33, 51, 53, 55, 57, 59, 61, 63, 65, 67, or 69. In other embodiments, the DGA1 protein is substantially identical to SEQ ID NO: 13, 17, 19, 21, 23, 25, 27, 29, 31, 33, 51, 53, 55, 57, 59, 61, 63, 65, 67, or 69, and retains the functional activity of the protein of SEQ ID NO: 15, 17, 19, 21, 23, 25, 27, 29, 31, 33, 51, 53, 55, 57, 59, 61, 63, 65, 67, or 69, yet differs in amino acid sequence due to natural allelic variation or mutagenesis. In another embodiment, the DGA1 protein comprises an amino acid sequence at least about 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, 99.1%, 99.2%, 99.3%, 99.4%, 99.5%, 99.6%, 99.7%, 99.8%, 99.9% or more identical to SEQ ID NO: 15, 17, 19, 21,23, 25, 27, 29, 31, 33, 51, 53, 55, 57, 59, 61, 63, 65, 67, or 69.

In some embodiments, the type 1 diacylglycerol acyltransferase is DGA2. For example, the diacylglycerol acyltransferase may be a DGA2 protein encoded by a DGAT1 gene found in an organism selected from the group consisting of Arxula adeninivorans, Aspergillus terreus, Chaetomium globosum, Claviceps purpurea, Lipomyces starkeyi, Metarhizium acridim, Ophiocordyceps sinensis, Phaeodactylum tricornutum, Pichia guilliermondii, Rhodosporidium tortuloides, Rhodotorula graminis, Trichoderma virens, and Yarrowia lipolytica.

The DGA1 gene may have a nucleotide sequence set forth in SEQ ID NO: 2, 4, 6, 8, 10, 12, 14, 72, 74, 76, 78, 80, 82, or 84. In other embodiments, the DGAT1 gene is substantially identical to SEQ ID NO: 2, 4, 6, 8, 10, 12, 14, 72, 74, 76, 78, 80, 82, or 84, and the nucleotide sequence encodes a protein that retains the functional activity of a protein encoded by SEQ ID NO: 2, 4, 6, 8, 10, 32, 14, 72, 74, 76, 78, 80, 82, or 84, yet differs in nucleotide sequence due to natural allelic variation or mutagenesis. In another embodiment, theDGAT1 gene comprises a nucleotide sequence at least about 70%, 71%, 72%, 73%, 74%, 75%, 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 95 %, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, 99.1 %, 99.2%, 99.3%, 99.4%, 99.5%, 99.6%, 99.7%. 99.8%, 99.9% or more identical to SEQ ID NO: 2, 4, 6, 8, 10, 12, 14, 72, 74, 76, 78, 80, 82, or 84 .

The DGA2 protein may have an amino acid sequence set forth in SEQ ID NO: 1, 3, 5, 7, 9, 11, 13, 71, 73, 75, 77, 79, 81, or 83. In other embodiments, the DGA2 protein is substantially identical to SEQ ID NO: 1, 3, 5, 7, 9, 11, 33, 71, 73, 75, 77, 79, 81, or 83, and retains the functional activity of the protein of SEQ ID NO: 1, 3, 5, 7, 9, 11, 13, 71, 73, 75, 77, 79, 81, or 83, yet differs in amino acid sequence due to natural allelic variation or mutagenesis. In another embodiment, the DGA2 protein comprises an amino acid sequence at least about 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, 99.1 %, 99.2%, 99.3%, 99.4%, 99.5%, 99.6%, 99,7%, 99.8%,, 99.9% or more identical to SEQ ID NO: 1, 3, 5,7,9, 11, 13, 71, 73, 75, 77, 79, 81, or 83.

In some embodiments, the type 3 diacylglycerol acyltransferase is DGA3. For example, the diacylgylcerol acyltransferase may be a DGA3 protein encoded by a DGAT3 gene found in an organism selected from the group consisting of Ricimus communis and Arachis hypogaea.

The DGAT3 gene may have a nucleotide sequence set forth in SEQ ID NO: 88 or 90. In other embodiments, the DGAT3 gene is substantially identical to SEQ ID NO: 88 or 90, and the nucleotide sequence encodes a protein that retains the functional activity of a protein encoded by SEQ ID NO: 87 or 89, yet differs in nucleotide sequence due to natural allelic variation or mutagenesis. In another embodiment, the DGAT3 gene comprises a nucleotide sequence at least about 70%, 71%, 72%, 73%, 74%, 75%, 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, 99.1%, 99.2%, 99.3%, 99.4%, 99.5%, 99.6%, 99.7%, 99.8%, 99.9% or more identical to SEQ ID NO: 88 or 90.

The DGA3 protein may have an amino acid sequence set forth in SEQ ID NO: 87 or 89. In other embodiments, the DGA3 protein is substantially identical to SEQ ID NO: 87 or 89, and retains the functional activity of the protein of SEQ ID NO: 87 or 89, yet differs in amino acid sequence due to natural allelic variation or mutagenesis. In another embodiment, the DGA3 protein comprises an amino acid sequence at least about 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, 99.1%, 99,2%, 99.3%, 99,4%, 99.5%, 99.6%, 99.7%, 99,8%, 99.9% or more identical to SEQ ID NO: 87 or 89.

The DGAT1, DGAT2, or DGA3 genes may comprise conservative substitutions, deletions, and/or insertions while still encoding a protein that has functional diacylgylcerol acyltransferase activity. For example, the DGAT1, DGAT2, or DGA3 codons may be optimized for a particular host cell, different codons may be substituted for convenience, such as to introduce a restriction site or create optimal PGR primers, or codons may be substituted for another purpose. Similarly, the nucleotide sequence may be altered to create conservative amino acid substitutions, deletions, and/or insertions.

The DGA1, DGA2, and DGA3 polypeptides may comprise conservative substitutions, deletions, and/or insertions while still maintaining functional diacylgylcerol acyltransferase acti vity. Conservati ve substitution tables are well known in the art (Creighton, Proteins (2d, ed., 1992)).

Amino acid substitutions, deletions and/or insertions may readily be made using recombinant DNA manipulation techniques. Methods for the manipulation of DNA sequences to produce substitution, insertion or deletion variants of a protein are well known in the art. These methods include M13 mutagenesis, T7-Gen in vitro mutagenesis (USB, Cleveland, Ohio), Quick Change Site Directed mutagenesis (Stratagene, San Diego, Calif.), PCR-mediated site-directed mutagenesis, and other site-directed mutagenesis protocols.

To determine the percent identity of two amino acid sequences or of two nucleic acid sequences, the sequences can be aligned for optimal comparison purposes (e.g., gaps can be introduced in one or both of a first and a second amino acid or nucleic acid sequence for optimal alignment and non-identical sequences can be disregarded for comparison purposes). The length of a reference sequence aligned for comparison purposes can be at least 95% of the length of the reference sequence. The amino acid residues or nucleotides at corresponding amino acid positions or nucleotide positions can then be compared. When a position in the first sequence is occupied by the same amino acid residue or nucleotide as the corresponding position in the second sequence, then the molecules are identical at that position (as used herein amino acid or nucleic acid “identity” is equivalent to amino acid or nucleic acid “homology”). The percent identity between the two sequences is a function of the number of identical positions shared by the sequences, taking into account the number of gaps, and the length of each gap, which need to be introduced for optimal alignment of the two sequences.

The comparison of sequences and determination of percent identity between two sequences can be accomplished using a mathematical algorithm. In one embodiment, the percent identity between two amino acid sequences can be determined using the Needleman and Wunsch (J. Molecular Biology 48:444-453 (1970)) algorithm which has been incorporated into the GAP program in the GCG software package (available at http://www.gcg.com), using either a Blosum 62 matrix or a PAM250 matrix, and a gap weight of 16, 14, 12, 10, 8, 6, or 4 and a length weight of 1, 2, 3, 4, 5, or 6. In yet another embodiment, the percent identity between two nucleotide sequences can be determined using the GAP program in the GCG software package (available at http://www.gcg.com), using a NWSgapdna.CMP matrix and a gap weight of 40, 50, 60, 70, or 80 and a length weight of 1, 2, 3, 4, 5, or 6. In another embodiment, the percent identity between two amino acid or nucleotide sequences can be determined using the algorithm of E. Meyers and W. Miller (Computer Applications in the Biosciences 4:11-17 (1988)) which has been incorporated into the ALIGN program (version 2.0 or 2.0U), using a PAM120 weight residue table, a gap length penalty of 12 and a gap penalty of 4.

Exemplary computer programs which can be used to determine identity between two sequences include, but are not limited to, the suite of BLAST programs, e.g., BLASTN, MEGABLAST, BLASTX, TBLASTN, TBLASTX, and BLASTP, and Clustal programs, e.g., ClustalW, ClustalX, and Clustal Omega.

Sequence searches are typically carried out using the BLASTN program, when evaluating a given nucleic acid sequence relative to nucleic acid sequences in the GenBank DNA Sequences and other public databases. The BLASTX program is effective for searching nucleic acid sequences that have been translated in all reading frames against amino acid sequences in the GenBank Protein Sequences and other public databases.

An alignment of selected sequences in order to determine “% identity” between two or more sequences is performed using for example, the CLUSTAL-W program.

A “coding sequence” or “coding region” refers to a nucleic acid molecule having sequence information necessary to produce a protein product, such as an amino acid or polypeptide, when the sequence is expressed. The coding sequence may comprise and/or consist of untranslated sequences (including introns or 5′ or 3′ untranslated regi ons) within translated regions, or may lack such intervening untranslated sequences (e.g., as in cDNA).

The abbreviation used throughout the specification to refer to nucleic acids comprising and/or consisting of nucleotide sequences are the conventional one-letter abbreviations. Thus when included in a nucleic acid, the naturally occurring encoding nucleotides are abbreviated as follows: adenine (A), guanine (G), cytosine (C), thymine (T) and uracil (U). Also, unless otherwise specified, the nucleic acid sequences presented herein is the 5′→3′direction.

As used herein, the term “complementary” and derivatives thereof are used in reference to pairing of nucleic acids by the well-known rules that A pairs with T or U and C pairs with G. Complement can be “partial” or “complete”. In partial complement, only some of the nucleic acid bases are matched according to the base pairing rules; while in complete or total complement, all the bases are matched according to the pairing rule. The degree of complement between the nucleic acid strands may have significant effects on the efficiency and strength of hy bridization between nucleic acid strands as well known in the art. The efficiency and strength of said hybridization depends upon the detection method.

As used herein, “DGA1” means a diacylglycerol acyltransferase type 2 (DGAT2). DGA1 is an integral membrane protein that catalyzes the final enzymatic step in oil biosynthesis and the production of triaeyIglycerols in plants, fungi, and mammals. The DGA1 may play a key role in altering the quantity of long-chain polyunsaturated fatty acids produced in oils of oleaginous organisms. DGA1 is related to the aeyl-coenzyme A:cholesterol acyltransferase (“ACAT”). This enzyme is responsible for transferring an aeyl group from acyl-coenzyme-A to the sn-3 position of 1,2-diacylglycerol (“DAG”) to form triacyiglycerol (“TAG”) (thereby involved in the terminal step of TAG biosynthesis). DGA1 is associated with membrane and lipid body fractions in plants and fungi, particularly, in oilseeds where it contributes to the storage of carbon used as energy reserves. TAG is believed to be an important chemical for storage of energy in cells. DGA1 is known to regulate TAG structure and direct TAG synthesis.

The DGA1 polynucleotide and polypeptide sequences may be derived from highly oleaginous organisms having very high, native levels of lipid accumulation. (Bioresource Technology 144:360-69 (2013); Progress Lipid Research 52:395-408 (2013); Applied Microbiology & Biotechnology 90:1219-27 (2011); European Journal Lipid Science & Technology 113:1031-51 (2011); Food Technology & Biotechnology 47:215-20 (2009); Advances Applied Microbiology 51:1-51 (2002); Lipids 11:837-44 (1976)). A list of organisms with a reported lipid content of about 50% and higher is shown in Table 1. R. torulaides and L. starkeyi have the highest lipid content. Among the organisms in Table 1, five have publicly accessible sequence for DGA1, R. toruloides, L. starkeyi, A. limacinum, A. terreus, and C. purpurea (bolded in Table 1).

Nucleic acid constructs for increasing the activity of DGA1 were described in U.S. Ser. No. 61/943,664, and PCT Patent Application No. PCT/US15/017227 (hereby incorporated by reference). FIG. 1 shows expression construct pNC243 used for expression of the Rhodosporidhm toruloides DGA1 gene NG66 (SEQ ID NO:20) in Y. lipolytica. DGA1 expression constructs were linearized before transformation by a PacI/NotI restriction digest. The linear expression constructs each included the expression cassette for DGA1 and for the NatI gene, used as marker for selection with nourseothricin (NAT).

Nucleic acid constructs for increasing the acti vity of DGA2 and/or other diacylglycerol acyltransferases may be created using the methods described above and/or other methods known in the art. FIG. 3 shows expression construct pNC327 used for expression of the Claviceps purpurea DGA2 gene NG112 (SEQ ID NO:9) in Y. lipolytica. DGA2 expression constructs were linearized before transformation by a PacI/AscI restriction digest. The linear expression constructs each included the expression cassette for DGA2 and for the BLE gene, used as marker for selection with Zeocin.

Nucleic acid constructs for increasing the activity of DGA3 and/or other diacylglycerol aeyltxansferases may be created using the methods described above and/or other methods known in the art.

B. Triacylglycerol Lipase Nucleic Acid Molecules and Vectors

Triacylglycerol lipase depletes a cell's triacylglycerol by removing one or more fatty acid chains. Thus, decreasing the net triacylglycerol lipase activity of a cell may increase the cell's triacylglycerol. This decrease may be accomplished by reducing the efficiency of the enzyme, e.g., by mutating amino acids in its active site, or by reducing the expression of the enzyme. For example, a TGL3 knockout mutation will decrease the activity of a triacylglycerol lipase because it prevents the cell from transcribing TGL3. Triacylglycerol lipase knockouts are described in U.S. Ser. No. 61/987,098 and PCT Patent Application No. PCT/US15/28760 (hereby incorporated by reference).

In some embodiments, foe triacylglycerol lipase is TGL3. In other embodiments, the triacylglycerol lipase is TGL3/4 or TGL4.

The TGL3 gene in Y. lipolytica encodes the triacylglycerol lipase protein TGL3 (SEQ ID NO:41), and the TGL4 gene in Y. lipolytica encodes the triacylglycerol lipase protein TGL4 (SEQ ID NO:85), SEQ ID NO:42 contains the TGL3 nucleotide sequence, 100 upstream nucleotides, and 100 downstream nucleotides. Thus, the SEQ ID NO:42 nucleotide sequence may be used to design a nucleic acid capable of recombining with a nucleic acid sequence in a native Y. lipolytica triacylglycerol lipase gene. Similarly, SEQ ID NO:86 contains the TGL4 nucleotide sequence. Thus, the SEQ ID NO:86 nucleotide sequence may be used to design a nucleic acid capable of recombining with a nucleic acid sequence in a native Y. lipolytica triacylglycerol lipase gene.

Knockout, cassettes SEQ ID NOs: 49 and 50 are capable of recombining with the native TGL3 gene in Y. lipolytica. Thus, in some embodiments, the nucleic acids encoded by SEQ ID NOs: 49 and 50 may be used to generate a triacylglycerol lipase knockout mutation in Y. lipolytica. SEQ ID NOs: 49 and 50 each contain portions of a hygroniycin resistance gene hph. Neither isolated sequence encodes a functional protein, but the two sequences are capable of encoding a functional kinase that confers hygromycin resistance upon successful recombination. Further, neither SEQ ID NO:49 nor SEQ ID NO:50 contains a promoter or terminator, and thus, they rely on homologous recombination with the Y. lipolytica TGL3 gene in order for the hph gene to be transcribed and translated. In this way, successfully transformed oleaginous cells may be selected by growing the cells on medium containing hygromycin.

Knockout cassette SEQ ID NO:49 may be prepared by amplifying a hygromycin resistance gene hph (SEQ ID NO:44) with primer NP1798 (SEQ ID NO:47) and primer NP656 (SEQ ID NO:46). Knockout cassette SEQ ID NO:50 may be prepared by amplifying a hygroniycin resistance gene hph (SEQ ID NO:44) with primer NP655 (SEQ ID NO:45 ) and primer NP1799 (SEQ ID NO:48).

Different approaches may be used to design nucleic acids that reduce the activity of TGL3 in Y. lipolytica (Biochimica Biophysica Acta 1831:1486-95 (2013)). The methods disclosed herein and other methods known in the art may be used to reduce triacylglycerol lipase activity in other species. For example, these methods may be used to reduce the activity of the TGL3 gene of Arxula adeninivorans (SEQ ID NO:36), the TGL3/4 gene of Arxula adeninivorans (SEQ ID NO:38), or the TGL4 gene of Arxula adeninivorans (SEQ ID NO:40).

C. Transformed Cell

In some embodiments, the transformed cell is a prokaryotic cell, such as a bacterial cell. In some embodiments, the cell is a eukaryotic cell, such as a mammalian cell, a yeast cell, a filamentous fungi cell, a protist cell, an algae cell, an avian cell, a plant cell, or an insect cell.

The cell may be selected from the group consisting of Arxula, Aspergillus, Aurantiochytrium, Candida, Claviceps, Cryptococcus, Cunninghamella, Geotrichum, Hansenula, Kluyveromyces, Kodamaea, Leucosporidiella, Lipomyces, Mortierella, Ogataea, Pichia, Prototheca, Rhizopus, Rhodosporidium, Rhodotorula, Saccharomyces, Schizosaccharomyces, Tremella, Trichosporon, Wickerhamomyces, and Yarrowia.

In some embodiments, the cell is selected from the group consisting of Arxula adeninivorans, Aspergillus niger, Aspergillus orzyae, Aspergillus terreus, Aurantiochytrium limacinam, Candida utilis, Claviceps purpurea, Cryptococcus albidus, Cryptococcus curvatus, Cryptococcus ramirezgomezianus, Cryptococcus terreus, Cryptococcus wieringae, Cunninghamella echinulata, Cunninghamella japonica, Geotrichum fermentans, Hansemda polymorpha, Kluyveromyces lactis, Kluyveromyces marxiamus, Kodamaea ohmeri, Leucosporidiella creatinivora, Lipomyces lipofer, Lipomyces starkeyi, Lipomyces tetrasporus, Mortierella isabellina, Mortierella alpina, Ogataea polymorpha, Pichia ciferrii, Pichia guilliermondii, Pichia pastoris, Pichia stipites, Prototheca zopfii, Rhizopus arrhizus, Rhodosporidium babjevae, Rhodosporidium toruloides, Rhodosporidium paludigenum, Rhodotorula glutinis, Rhodotorula mucilaginosa, Saccharomyces cerevisiae, Schizosaccharomyces pombe, Tremella enhepala, Trichosporon cutaneum, Trichosporon fermentans, Wickerhamomyces ciferrii, and Yarrowia lipolytica.

In certain embodiments, the transformed cell is a high-temperature tolerant yeast cell. In some embodiments the transformed cell is Kluyveromyces marxianus.

In certain embodiments, the cell is Yarrowia lipolytica or Axula adeninivorans.

D. Increasing the Activity of a Diacylgylcerol Acyltransferase in a Cell

A protein's activity may be increased by overexpressing the protein. Proteins may be overexpressed in a cell, using a variety of genetic modifications. In some embodiments, the genetic modification increases the expression of a native diacylgylcerol acyltransferase. A native diacylgylcerol acyltransferase may be overexpressed by modifying the upstream transcription regulators of a native diacylgylcerol acyltransferase gene, for example, by increasing the expression of a transcription activator or decreasing the expression of a transcription repressor. Alternatively, the promoter of a native diacylglycerol acyltransferase gene may be substituted with a ccmstitutively active or inducible promoter by recombination with an exogenous nucleic acid.

In some embodiments, the genetic modification encodes at least one copy of a type 1 diacylglycerol acyltransferase gene. The type 1 diacylglycerol acyltransferase gene may be a gene native to the cell or from a different species. In certain embodiments, the gene is inheritable to the progeny of a transformed cell. In some embodiments, the gene is inheritable because it resides on a plasmid. In certain embodiments, the gene is inheritable because it is integrated into the genome of the transformed cell.

In certain embodiments, the DGAT1 gene is the type 1 diacylglycerol acyltransferase gene from Arxula adeninivorans, Aspergillus terreus, Chaetomium globosum, Claviceps purpurea, Lipomyces starkeyi, Metarhizium acridum, Ophiocordyceps sinensis, Phaeodactylum tricornutum, Pichia guilliermondii, Rhodosporidium toruloides, Rhodotorula graminis, Trichoderma virens, or Yarrowia lipolytica. In certain embodiments, diacylglycerol acyltransferase is expressed by transforming a cell with a gene encoding a diacylglycerol acyltransferase gene. The genetic modification may encode one or more than one copy of a diacylglycerol acyltransferase gene. In certain embodiments, the genetic modification encodes at least one copy of the DGA2 protein from Chaetomium globosum, Claviceps purpurea, Ophiocordyceps sinensis, or Yarrowia lipolytica. In some embodiments, the genetic modification encodes at least one copy of the DGA2 protein from Chaetomium globosum, Claviceps purpurea, or Ophiocordyceps sinensis and the transformed cell is Y. lipolytica. In some embodiments, the genetic modification encodes at least one copy of the DGA2 protein from Chaetomium globosum, Claviceps purpurea, or Yarrowia lipolytica, and the transformed cell is Arxula adeninivorans.

In some embodiments, the genetic, modification encodes at least one copy of a type 2 diacylglycerol acyltransferase gene. The type 2 diacylglycerol acyltransferase gene may be a gene native to the cell or from a different species. In certain embodiments, the gene is inheritable to the progeny of a transformed cell. In some embodiments, the gene is inheritable because it resides on a plasmid. In certain embodiments, the gene is inheritable because it is integrated into the genome of the transformed cell.

In certain embodiments, the DGAT2 gene is the type 2 diacylglycerol acyltransferase gene from Arxula adeninivorans, Aspergillus terreus, Aurantiochytrium limacinum, Claviceps purpurea, Gloeophyllum trabeum, Lipomyces starkeyi, Microbotryum violaceum, Pichia guilliermondii, Phaeodactylum tricornutum, Puccinia graminis, Rhodosporidium diobovatum, Rhodosporidium toruloides, Rhodotorula graminis, or Yarrowia lipolytica. In certain embodiments, diacylglycerol acyltransferase is expressed by transforming a cell with a gene encoding a diacylglycerol acyltransferase gene. The genetic modification may encode one or more than one copy of a diacylglycerol acyltransferase gene. In certain embodiments, the genetic modification encodes at least one copy of the DGA1 protein from R. toruloides. In some embodiments, the genetic modification encodes at least one copy of the DGA1 protein from R. toruloides and the transformed cell is Y. lipolytica. In some embodiments, the genetic modification encodes at least one copy of the DGA1 protein from R. toruloides, and the transformed cell is Arxula adeninivorans.

In some embodiments, the DGA 1 protein is from R. toruloides and the DGA2 protein is from Chaetomium globosum, Claviceps purpurea, Ophiocordyceps sinensis, or Yarrowia lipolytica. In some embodiments, the DGA1 protein is from R. toruloides, the DGA2 protein is from Claviceps purpurea, Chaetomium globosum, or Ophiocordyceps sinensis, and the transformed cell is Y. lipolytica. In some embodiments, the DGA1 protein is from R. toruloides, the DGA2 protein is from Claviceps purpurea, Chaetomium globosum, or Yarrowia lipolytica, and the transformed cell is Arxula adeninivorans.

In some embodiments, the genetic modification encodes at least one copy of a type 3 diacylglycerol acyltransferase gene. The type 3 diacylglycerol acyltransferase gene may be a gene native to foe cell or from a different species. In certain embodiments, the gene is inheritable to the progeny of a transformed cell. In some embodiments, the gene is inheritable because it resides on a plasmid. In certain embodiments, the gene is inheritable because it is integrated into the genome of the transformed cell.

In certain embodiments, the DGAT3 gene is the type 3 diacylglycerol acyltransferase gene from Ricinus communis or Arachis hypogaea. In certain embodiments, diacylglycerol acyltransferase is expressed by transforming a cell with a gene encoding a diacylglycerol acyltransferase gene. The genetic modification may encode one or more than one copy of a diacylglycerol acyltransferase gene. In certain embodiments, the genetic modification encodes at least one copy of the DGA3 protein from Ricinus communis or Arachis hypogaea. In some embodiments, the genetic modification encodes at least one copy of the DGA3 protein from Ricinus communis or Arachis hypogaea, and the transformed cell is Y. lipolytica. In some embodiments, the genetic modification encodes at least one copy of the DGA1 protein from Ricinus communis or Arachis hypogaea, and the transformed cell is Arxula adeninivorans.

In certain embodiments, the diacylgylcerol acyltransferase gene is inheritable to the progeny of a transformed cell. In some embodiments, the diacylgylcerol acyltransferase gene is inheritable because it resides on a plasmid. In certain embodiments, the diacylgylcerol acyltransferase gene is inheritable because it is integrated into the genome of the transformed cell.

E. Decreasing Triacylglycerol Lipase Activity in a Cell

In some embodiments, the transformed oleaginous cell comprises a genetic modification that decreases the activity of a native triacylglycerol lipase. Such genetic modifications may affect a protein that regulates the transcription of a triacylglycerol lipase gene, including modifications that decrease the expression of a transcription activator and/or increase the expression of a transcription repressor. Modifications that affect a regulator protein may both decrease the expression of triacylglycerol lipase and alter other gene expression profiles that shift the cellular equilibrium toward increased lipid accumulation or modified lipid composition. Alternatively, the genetic modification may be the introduction of a small interfering RNA, or a nucleic acid that encodes a small interfering RNA. In other embodiments, the genetic modification consists of the homologous recombination of a nucleic acid and the regulatory region of a native triacylglycerol lipase gene, including an operator, promoter, sequences upstream from: the promoter, enhancers, and sequences downstream of the gene.

In some embodiments the transformed oleaginous cell comprises a genetic modification consisting of a homologous recombination event. In certain embodiments, the transformed cell comprises a genetic modification consisting of a homologous recombination event between a nati ve triacylglycerol lipase gene and a nucleic acid. Thus, the genetic modification deletes the tnacylglycerol lipase gene, prevents its transcription, or prevents the transcription of a gene that can be transcribed into a fully-active protein. A homologous recombination event may mutate or delete a portion, of a native triacylglycerol lipase gene. For example, the homologous recombination event may mutate one or more residues in the active site of a native triacylglycerol lipase, thereby reducing the efficiency of the lipase or rendering it inactive. Alternatively, the homologous recombination event may affect post-translational modification, folding, stability, or localization within the cell. In some embodiments, the homologous recombination event replaces the promoter with a promoter that drives less transcription. In other embodiments, the homologous recombination event mutates the promoter to impair its ability to drive transcription. In certain embodiments, the genetic modification is a triacylglycerol lipase knockout mutation. Knockout mutations are preferable because they eliminate a pathway that depletes a cell's triacylglycerol content, thereby increasing the triacylglycerol content of a cell.

A knockout mutation may delete one or more triacylglycerol lipase genes. Additionally, the knockout mutation may substitute a triacylglycerol lipase gene with a gene that encodes a different protein. The gene may be operably linked to an exogenous promoter. In certain embodiments, the gene is not linked to an exogenous promoter, and instead, the gene is configured to recombine with the triacylglycerol lipase gene such that the triacylglycerol lipase gene's promoter drives transcription of the gene. Thus, the gene is less likely to be expressed if it randomly integra tes into the cell's genome. Methods for creating knockouts are well-known in the art (See, e.g., Fickers et al., J. Microbiological Methods 55:727 (2003)).

In certain embodiments, the genetic modification comprises two homologous recombination events. In the first event, a nucleic acid encoding a portion of a gene recombines with the triacylglycerol lipase gene, and in the second event, a nucleic acid encoding the remaining portion of the gene recombines with the triacylglycerol lipase gene. The two portions of the gene are designed such that neither portion is functional unless they recombine with each other. These two events further reduce the likelihood that the gene can be expressed following random integration events.

In certain embodiments, the gene encodes a dominant selectable marker. Thus, knockout cells may be selected by screening for the marker. In some embodiments, the dominant selectable marker is a drug resistance marker. A drug resistance marker is a dominant selectable marker that, when expressed by a cell, allows the cell to grow and/or survive in the presence of a drug that would normally inhibit cellular growth and/or survival. Cells expressing a drug resistance marker can be selected by growing the cells in the presence of the drug. In some embodiments, the drug resistance marker is an antibiotic resistance marker. In some embodiments, the drug resistance marker confers resistance to a drug selected from the group consisting of Amphotericin B, Candicidin, Filipin, Hamycin, Natamycin, Nystatin, Rimocidin, Bifonazole, Butoconazole, Clotrimazole, Ecouazole, Fenticonazole, Isoconazole, Ketoconazole, Luliconazole, Miconazole, Omoconazole, Oxiconazole, Sertaconazole, Sulconazole, Tioconazole, Albaconazole, Fluconazole, Isavuconazole, Itraconazole, Posaconazole, Ravueonazole, Terconazole, Voriconazole, Abafungin, Amorolfin, Butenafine, Naftifine, Terbinafine, Anidulafungin, Caspofungin, Micafungin, Benzoic acid, Ciclopirox, Flucytosine, 5-fluorocytosine, Griseofulvin, Haloprogin, Polygodial, Tolnaftate, Crystal violet, Amikacin, Gentamicin, Kanamycin, Neomycin, Netilmicin, Tobramycin, Paromomycin, Spectinomycin, Geldanamycin, Herbimyctn, Rifaximin, Streptomycin, Loracarbef, Ertapenem, Doripenem, Imipenem, Meropenem, Cefadroxil, Cefazolin, Cefalotin, Cefalexin, Cefaclor, Cefamandole, Cefoxitin, Cefprozii, Cefuroxime, Cefixime, Cefdinir, Cefditoren, Cefoperazone, Cefotaxime, Cefpodoxime, Ceftazidime, Ceftibuten, Ceftizoxime, Ceftriaxone, Cefepime, Ceftaroline fosamil, Ceftobiprole, Teicoplanin, Vancomycin, Telavancin, Clindamycin, Lincomycin, Daptomycin, Azithromycin, Clarithromycin, Dirithromycin, Erythromycin, Roxithromycin, Troleandomycin, Telithromycin, Spiramycin, Aztreonam, Furazolidone, Nitrofurantoin, Linezolid, Posizolid, Radezolid, Torezolid, Amoxicillin, Ampicillin, Azlocillin, Carbenicillin, Cloxacillin, Dicloxacillin, Flucloxacillin, Mezlocillin, Methicillin, Nafcillin, Oxacillin, Penicillin G, Penicillin V, Piperacillin, Penicillin G, Temocillin, Ticarcillin, clavulanate, sulbactam, tazobactam, clavulanate, Bacitracin, Colistin, Polymyxin B, Ciprofloxacin, Enoxacin, Gatifloxacin, Levofloxacin, Lomefloxacin, Moxifloxacin, Nalidixic acid, Norfloxacin, Ofloxacin, Trovafloxacin, Grepafloxacin, Sparfloxacin, Temafloxacin, Mafenide, Sulfacetamide, Sulfadiazine, Silver sulfadiazine, Sulfadimethoxine, Sulfamethizole, Sulfamethoxazole, Sulfanilamide, Sulfasalazine, Sulfisoxazole, Trimethoprim-Sulfamethoxazole, Co-trimoxazole, Sulfonamidochrysoidine, Demeclocycline, Doxycycline, Minocycline, Oxytetracycline, Tetracycline, Clofazimine, Dapsone, Capreomycin, Cycloserine, Ethambutol, Ethionamide, Isoniazid, Pyrazinamide, Rifampicin, Rifabutin, Rifapentine, Streptomycin, Arsphenamine, Chloramphenicol, Fosfomycin, Fusidic acid, Metronidazole, Mupirocin, Platensimycin, Quinupristin, Dalfopristin, Thiamphenicol, Tigecyciine, Tinidazole, Trimethoprim, Geneticin, Nourseothricin, Hygromycin, Bleomycin, and Puromycin.

In some embodiments, the dominant selectable marker is a nutritional marker. A nutritional marker is a dominant selectable marker that, when expressed by the cell, enables the cell to grow or survive using one or more particular nutrient sources. Cells expressing a nutritional marker can be selected by growing the cells under limiting nutrient conditions in which cells expressing the nutritional marker can survive and/or grow, but cells lacking the nutrient marker cannot. In some embodiments, the nutritional marker is selected from the group consisting of Orotidine 5-phosphate decarboxylase, Phosphite specific oxidoreductase, Alpha-ketoglutarate-dependent hypophosphite dioxygenase, Alkaline phosphatase, Cyanamide hydratase, Melamine deaminase, Cyanurate amidohydrolase, Biuret hydrolyase, Urea amidolyase, Ammelide aminohydrolase, Guanine deaminase, Phosphodiesterase, Phosphodiesterase, Phosphite hydrogenase, Glycerophosphodiesterase, Parathion hydrolyase, Phosphite dehydrogenase, Dibenzothiophene desulfurization enzyme, Aromatic desulfinase, NADH-dependent FMN reductase, Aminopurine transporter, Hydroxylamine oxidoreductase, Invertase, Beta-glucosidase, Alpha-glucosidase, Beta-galactosidase, Alpha-galactosidase, Amylase, Cellulose, and Pullulonase.

Different approaches may be used to knockout the TGL3 gene in Y. lipolytica (See, e.g., Dulermo et al., Biochimica Biophysics Acta 1831:1486 (2013)). The methods disclosed herein and other methods known in the art may be used to knockout different triacylglycerol lipase genes in other species. For example, these methods may be used to knockout the TGL3 gene of Arxula adeninivorans (SEQ ID NO:36), the TGL3/4 gene of Arxula adeninivorans (SEQ ID NO:38), or the TGL4 gene of Arxula adeninivorans (SEQ ID NO:40).

In some embodiments, a genetic modification decreases the expression of a native triacylglycerol lipase gene by 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 25, 30, 35, 40, 45, 50, 55, 60, 65, 70, 75, 80, 85, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99, 99.1, 99.2, 99.3, 99.4, 99.5. 99.6, 99.7, 99.8, 99.9, or 100 percent.

In some embodiments, a genetic modification decreases the efficiency of a native triacylglycerol lipase gene by 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 25, 30, 35, 40, 45, 50, 55, 60, 65, 70, 75, 80, 85, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99, 99.1, 99.2, 993, 99.4, 99,5, 99.6, 99.7, 99.8, 99.9, or 100 percent.

In some embodiments, a genetic modification decreases the activity of a native triacylglycerol lipase gene by 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 25, 30, 35, 40, 45, 50, 55, 60, 65, 70, 75, 80, 85, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99, 99.1, 99.2, 99.3, 99.4, 99.5, 99.6, 99.7, 99,8, 99.9, or 100 percent.

F. Decreasing Triagylglyccrol Lipase Activity in a Cell with Concomitant Expression of Diacylglycerol Acyltransferase

In some embodiments, the transformed oleaginous cell comprises a triacylglycerol lipase knockout mutation and a genetic modification that increase the expression of a native diacylglycerol acyltransferase. In certain embodiments, the transformed oleaginous cell comprises a triacylglycerol lipase knockout mutation and a genetic modification that encodes at least one copy of a diacylglycerol acyltransferase gene that is either native to the cell or from a different species of cell. In some embodiments, a triacylglycerol acyltransferase gene is disrupted and DGA1 and DGA2 proteins are expressed.

In some embodiments, one nucleic acid increases the expression of a native diacylglycerol acyltransferase or encodes at least one copy of a diacylglycerol acyltransferase gene and a second nucleic acid decreases the activity of a triacylglycerol lipase in the cell. In some embodiments the same nucleic acid encodes at least one copy of a diacylglycerol acyltransferase gene and decreases the activity of a triacylglycerol lipase in the cell. For example, the nucleic acid designed to knock out a triacylglycerol lipase gene may also contain a copy of a diacylglycerol acyltransferase gene.

G. Triacylglycerol Production

In certain embodiments, the transformed cells are grown in the presence of exogenous fatty acids, glucose, ethanol, xylose, stserose, starch, starch dextrin, glycerol, cellulose, and/or acetic acid. These substrates may be added during cultivation to increase lipid production. The exogenous fatty acids may include stearate, oleic acid, linoleic acid, γ-linolenic acid, dihomo-γ-linoleuic acid, arachsdonic acid, α-linolenic acid, stearidonic acid, eicosatetraenoic acid, eicosapenteaenoic acid, docosapentaenoic acid, eicosadienoic acid, and/or eicosatrienoic acid.

In certain embodiments, the present invention relates to a product produced by a modified host cell described herein. In certain embodiments, the product is an oil, lipid, or triacylglycerol. In some embodiments, the product is palmitic acid, pabmtoleie acid, stearic acid, oleic acid, or linoleie acid. In certain embodiments, the product is a saturated fatty acid. Thus, the product may be caprylic acid, capric acid, lauric acid, myristic acid, palmitic acid, stearic acid, arachidic acid, behenic acid, lignoceric acid, or cerotic acid. In some embodiments, the product is an unsaturated fatty acid. Thus, the product may be myristoleic acid, palmitoleic acid, sapienic acid, oleic acid, elaidic acid, vaccenic acid, linoleic acid, linoelaidic acid, α-linolenic acid, arachidonic acid, eicosapentcaenoic acid, erucic acid, or docosahexaenoic acid.

The product may be selected from the group consisting of lipids, triacylglycerides, fatty alcohols, fatty acids, alkanes, alkenes, isoprenoids, isoprene, squalene, farnasene, alcohols, isopropanol, n-propanol, n-butanol, isobutanol, 2-butanol, butadiene, diols, 1,3 propanediol, 1,4 propanediol, succinic acid, adipic acid, nylon precursors, citric acid, malic acid, polyols, and erythritol.

**Genetic Modifications Related to DGA1 and DGA2**

In some embodiments, the invention relates to a transformed cell, comprising a first genetic modification and second genetic modification, wherein said first genetic modification increases the activity of a native type 1 diacylglycerol acyltransferase or encodes at least one copy of a type 1 diacylglycerol acyltransferase gene native to the cell or from a different species, and said second genetic modification increases the activity of a native type 2 diacylglycerol acyltransferase or encodes at least one copy of a type 2 diacylglycerol acyltransferase gene native to the cell or from a different species. In some embodiments, the cell comprises a third genetic modification, wherein said third genetic modification decreases the activity of a triacylglycerol lipase in the cell.

In some embodiments, the invention relates to a transformed cell, wherein a first genetic modification encodes at least one copy of a type 1 diacylglycerol acyltransferase gene native to the cell or from a different species. In some embodiments, at least one copy of a type 1 diacylglycerol acyltransferase gene is integrated into the genome of said cell

The type 1 diacylglycerol acyltransferase gene may be a type 1 diacylglycerol acyltransferase gene from Arxula adeninivorans, Aspergillus terreus, Chaetomium globosum, Claviceps purpurea, Lipomyces starkeyi, Metarhizium acridum, Ophiocordyceps sinensis, Phaeodactylum tricornutum, Pichia guilliermondii, Rhodosporidium toruloides, Rhodotorula graminis, Trichoderma virens, or Yarrowia lipolytica. In some embodiments, the type 1 diacylglycerol acyltransferase gene is a type 1 diacylglycerol acyltransferase gene from Claviceps purpurea, Chaetomium globosum, Ophiocordyceps sinensis, or Yarrowia lipolytica.

In some embodiments, the invention relates to a transformed cell, wherein a second genetic modification encodes at least one copy of a type 2 diacylglycerol acyltransferase gene native to the cell or from a different species. In some embodiments, at least one copy of a type 2 diacylglycerol acyltransferase gene is integrated into the genome of said cell.

The type 2 diacylglycerol acyltransferase gene may be a type 2 diacylglycerol acyltransferase gene from Arxula adeninivorans, Aspergillus terreus, Aurantiochytrium limacinum, Claviceps purpurea, Gloeophyllum trabeum, Lipomyces starkeyi, Miccobotryum violaceum, Pichia guilliermondii, Phaeodactylum tricormutum, Puccinia graminis, Rhodosporidium diobovatum, Rhodosporidium toruloides, Rhodotorula graminis, or Yarrowia lipolytica. In some embodiments, the type 2 diacylglycerol acyltransferase gene is a type 2 diacylglycerol acyltransferase gene from Lipomyces starkeyi or Rhodosporidium toruloides.

In some embodiments, the invention relates to a transformed cell, wherein a third genetic modification is a triacylglycerol lipase knockout mutation. The triacylglycerol lipase may be encoded by a TGL3, TGL3/4, or TGL4 gene.

In some embodiments, the cell is Arxula adeninivorans; and said triacylglycerol lipase comprises the amino acid sequence set forth in SEQ ID NO:35, SEQ ID NO:37, or SEQ ID NO:39. In some embodiments, the cell is Arxula adeninivorans; and said triacylglycerol lipase is encoded by the nucleotide sequence set forth in SEQ ID NO:36, SEQ ID NO:38, or SEQ ID NO:40. In other embodiments, the cell is Yarrowia lipolytica; and said triacylglycerol lipase comprises the amino acid sequence set forth in SEQ ID NO:41 or SEQ ID NO:85. In some embodiments, the cell is Yarrowia lipolytica; and said triacylglycerol lipase is encoded by the nucleotide sequence set forth in SEQ ID NO:42 or SEQ ID NO:86.

**Genetic Modifications Related to DGA2**

In some embodiments, the invention relates to a transformed cell, comprising a genetic modification, wherein, said genetic modification increases the activity of a native type 1 diacylglycerol acyltransferase or encodes at least one copy of a type 1 diacylglycerol acyltransferase gene native to the cell or from a different species. For example, the genetic modification may encode at least one copy of a type 1 diacylglycerol acyltransferase gene native to the cell or from a different species. In some embodiments, at least one copy of a type 1 diacylglycerol acyltransferase gene is integrated into the genome of said cell.

**Genetic Modifications Related to DGA3**

In some embodiments, the invention relates to a transformed cell, comprising a genetic modification, wherein said genetic modification increases the activity of a native type 3 diacylglycerol acyltransferase or encodes at least one copy of a type 3 diacylglycerol acyltransferase gene native to the cell or from a different species. For example, the genetic modification may encode at least one copy of a type 3 diacylglycerol acyltransferase gene native to the cell or from a different species. In some embodiments, at least one copy of a type 3 diacylglycerol acyltransferase gene is integrated into the genome of said cell. The type 3 diacylglycerol acyltransferase gene may be a type 3 diacylglycerol acyltransferase gene from Ricinus communis or Arachis hypogaea.

### Species of Transformed Cell

In some embodiments, the invention relates to a transformed cell, wherein said cell is selected from the group consisting of algae, bacteria, molds, fungi, plants, and yeasts. The cell may be a yeast. The cell may be selected from the group consisting of Arxula, Aspergillus, Aurantiochytrium, Candida, Claviceps, Cryptococcus, Cuminghamella, Geotrichum, Hansenula, Kluyvermyces, Kodamaea, Leucosporidiella, Lipomyces, Mortierella, Ogataea, Pichia, Prototheca, Rhizopus, Rhodosporidium, Rhodotorula, Saccharomyces, Schizosaccharomyces, Tremella, Trichosporon, Wickerhamomyces, and Yarrowia. For example, the cell may be selected from the group consisting of Arxula adenimvoraans, Aspergillus niger, Aspergillus orzyae, Aspergillus terreus, Aurantiochytrium limacinum, Candida utilis, Claviceps purpurea, Cryptococcus albidus, Cryptococcus curvatus, Cryptococcus ramiresgomezianus, Cryptococcus terreus, Cryptococcus wieringae, Cunninghamella echinulata, Cunninghamella japonica, Geotrichum fermentans, Hansenula polymorpha, Kluyveromyces lactis, Kluyveromyces marxianus, Kodamaea ohmeri, Leucosporidiella creatinivora, Lipomyces lipofer, Lipomyces starkeyi, Lipomyces tetrasporus, Mortierella isabellina, Mortierella alpina, Ogataea polymorpha, Pichia ciferrii, Pichia guilliermondii, Pichia pasioris, Pichia stipites, Prototheca zopfi, Rhizopus arrhizus, Rhodosporidium babjevae, Rhodosporidium toruloides, Rhodosporidium paludigenum, Rhodotorula glutinis, Rhodotorula mucilaginosa, Saccharomyces cerevisiae, Schizosaccharomyces pombe, Tremella enchepala, Trichosporon cutaneum, Trichosporon fermentam, Wickerhamomyces ciferrii, and Yarrowia lipolytica.

### Products

In some aspects, the invention relates to a product derived from an aforementioned cell. In some embodiments the product is an oil, lipid, or triacylglycerol. The product may be palmitic acid, palmitoleic acid, stearic acid, oleic acid, or linoleic acid.

### Methods Related to DGA1 and DGA2

In some aspects, the invention provides a method of increasing the triacylglycerol content of a cell, comprising: (a) providing a cell comprising (i) a first genetic modification, wherein said first genetic modification increases the activity of a native type 1 diacylglycerol acyltransferase or encodes at least one copy of a type 1 diacylglycerol acyltransferase gene native to the cell or from a different species; and (ii) a second genetic modification, wherein said second genetic modification increases the activity of a native type 2 diacylglycerol acyltransferase or encodes at least one copy of a type 2 diacylglycerol acyltransferase gene native to the cell or from a different species; (b) growing said cell under conditions whereby the first and second genetic modifications are expressed, thereby producing a triacylglycerol; and (c) optionally recovering the triacylglycerol. The aforementioned method may also be used to modify the lipid composition of a cell. The cell may further comprise a third genetic modification, wherein said third genetic modification decreases the activity of a triacylglycerol lipase in the cell.

In some aspects, the invention, provides a method of increasing the lipid content of a cell, comprising transforming a parent cell with a first nucleotide sequence and second nucleotide sequence, wherein said first nucleotide sequence increases the activity of a native type 1 diacylgylcerol acyltransferase or encodes at least one copy of a type 1 diacylgylcerol acyltransferase gene and said second nucleotide sequence increases the activity of a native type 2 diacylgylcerol acyltransferase or encodes at least one copy of a type 2 diacylgylcerol acyltransferase gene. The method may further comprise transforming said cell with a third nucleotide sequence, wherein said third nucleotide sequence decreases the activity of a triacylglycerol lipase.

The first nucleotide sequence may comprise a type 1 diacylgylcerol acyltransferase gene. In certain embodiments, the first nucleotide sequence encodes an amino acid sequence having at least 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, 99.1%, 99.2%, 99.3%, 99.4%, 99.5%, 99.6%, 99.7%, 99.8%, or 99.9% sequence homology with the amino acid sequence set forth in SEQ ID NO:1, SEQ ID NG:3, SEQ ID NO:5, SEQ ID NO:7, SEQ ID NO:9, SEQ ID NO:11, SEQ ID NO:13, SEQ ID NO:7, SEQ ID NO:73, SEQ ID NO:75, SEQ ID NO:77, SEQ ID NO:79, SEQ ID NOrSl, or SEQ ID NO:83, or a biologically-active portion of any one of them. In some embodiments, the first nucleotide sequence encodes an amino acid sequence having at least 80% sequence homology with the amino acid sequence set forth in SEQ ID NO: 1, SEQ ID NO:3, SEQ ID NO:5, SEQ IDNO:7, SEQ ID NO:9, SEQ ID NO: 11, SEQ ID NO: 13, SEQ ID NO:71, SEQ ID NO:73, SEQ IDNO:75, SEQ ID NO:77, SEQ ID NO:79, SEQ ID NO:81, or SEQ ID NO:83, or a biologically-active portion of any one of them. In some embodiments, the first nucleotide sequence encodes an amino acid sequence having at least 95% sequence homology with the amino acid sequence set forth in SEQ ID NO:1, SEQ ID NO:3, SEQ ID NO:5, SEQ ID NO:7, SEQ ID NO:9, SEQ ID NO: 11, SEQ ID NO: 13, SEQ ID NO:71, SEQ ID NO:73, SEQ ID NO:75, SEQ ID NO:77, SEQ ID NO:79, SEQ ID NO:81, or SEQ ID NO:83, or a biologically-active portion of any one of them. In some embodiments, the first nucleotide sequence encodes the amino acid sequence set forth in SEQ ID NO: 1, SEQ ID NO:3, SEQ ID NO:5, SEQ ID NO:7, SEQ ID NO:9, SEQ ID NO: 11, SEQ ID NO: 13, SEQ ID NO:71, SEQ ID NO:73, SEQ ID NO:75, SEQ ID NO:77, SEQ ID NO:79, SEQ ID MQ:81, or SEQ ID NO:83, or a biologically-active portion of any one of them. In some embodiments, the first nucleotide sequence encodes the amino acid sequence set forth in SEQ ID NO:E SEQ ID NO:9, SEQ ID NO: 11, or SEQ ID NO:81, or a biologically-active portion of any one of them.

In some embodiments, the first nucleotide sequence has at least 70%, 71%, 72%, 73%, 74%, 75%, 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, 99.1 %, 99,2%, 99.3%, 99.4%, 99.5%, 99.6%, 99.7%, 99.8%, or 99.9% sequence homology with the nucleotide sequence set forth in SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO: 10, SEQ ID NO: 12, SEQ ID NO: 14, SEQ ID NO:72, SEQ ID NO:74, SEQ ID NO:76, SEQ ID NO:78, SEQ ID NO:80, SEQ ID NO:82, or SEQ ID NO: 84. In some embodiments, the first nucleotide sequence has at least 70% sequence homology with the nucleotide sequence set forth in SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO: 10, SEQ ID NO: 12, SEQ ID NO: 14, SEQ ID NO:72, SEQ ID NO:74, SEQ ID NO:76, SEQ ID NO:78, SEQ ID NO:80, SEQ ID NO:82, or SEQ ID NO: 84. In some embodiments, the first nucleotide sequence has at least 95% sequence homology with the nucleotide sequence set forth in SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO: 10, SEQ ID NO: 12, SEQ ID NO: 14, SEQ ID NO:72, SEQ ID NO:74, SEQ ID NO:76, SEQ ID NO:78, SEQ ID NO:8D, SEQ ID NO:82, or SEQ ID NO: 84. In some embodiments, the first nucleotide sequence comprises the nucleotide sequence set forth In SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO: 10, SEQ ID NO: 12, SEQ ID NO: 14, SEQ ID NO:72, SEQ ID NO:74, SEQ ID NQ:76, SEQ ID NO:78, SEQ ID NG:80, SEQ ID NO:82, or SEQ ID NO: 84. In some embodiments, the first nucleotide sequence comprises the nucleotide sequence set forth in SEQ ID NO:2, SEQ ID NO: 10, SEQ ID NO: 12, or SEQ ID NO:82.

The second nucleotide sequence may comprise a type 2 diacylgylcerol acyltransferase gene. In some embodiments, the first nucleotide sequence has at least 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, 99.1%, 99.2%, 99,3%, 99.4%, 99.5%, 99,6%, 99.7%, 99.8%, or 99.9% sequence homology with the nucleotide sequence set forth in SEQ ID NO:15, SEQ ID NO:17, SEQ ID NO:19, SEQ ID NO:21, SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID NO:31, SEQ ID NO:33, SEQ ID NO:51, SEQ ID NO:53, SEQ ID NO:55, SEQ ID NO:57, SEQ ID NO:59, SEQ ID NO:61, SEQ ID NO:63, SEQ ID NO:65, SEQ ID NO:67, or SEQ ID NO:69 or a biologically-active portion of any one of them. In some embodiments, the second nucleotide sequence encodes an amino acid sequence having at least 80% sequence homology with the amino acid sequence set forth in SEQ ID NO: 15, SEQ ID NO: 17, SEQ ID NO: 19, SEQ ID NG:21, SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID NO:31, SEQ ID NO:33, SEQ ID NO:51, SEQ ID NO:53, SEQ ID NO:55, SEQ ID NO:57, SEQ ID NO:59, SEQ ID NO:61, SEQ ID NO:63, SEQ ID NO:65, SEQ ID NO:67, or SEQ ID NO:69 or a biologically-active portion of any one of them. In some embodiments, the second nucleotide sequence encodes an amino acid sequence having at least 95% sequence homology with the amino acid sequence set forth in SEQ ID NO: 15, SEQ ID NO: 17, SEQ ID NO: 19, SEQ ID NO:21, SEQ ID NQ:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID NO:31, SEQ ID NO:33, SEQ ID NO:51, SEQ ID NO:53, SEQ ID NO:55, SEQ ID NO:57, SEQ ID NO:59, SEQ ID NO:61, SEQ ID NO:63, SEQ ID NO:65, SEQ ID NO:67, or SEQ ID NO:69 or a biologically-active portion of any one of them. In some embodiments, the second nucleotide sequence encodes the amino acid sequence set forth in SEQ ID NO: 15, SEQ ID NO:17, SEQ ID NO:19, SEQ ID NO:21, SEQ ID NO:23, SEQ IDNO:25, SEQ ID NO:27,SEQ ID NO:29, SEQ ID NO:31. SEQ ID NO:33, SEQ ID NO:51, SEQ ID NO:53, SEQ ID NO:55, SEQ ID NO:57, SEQ ID NO:59, SEQ ID NO:61, SEQ IDNO:63, SEQ ID NO:65, SEQ ID NO: 67, or SEQ ID NO :69 or a biologically-active portion of any one of them. In some embodiments, the second nucleotide sequence encodes the amino acid sequence set forth in SEQ ID NO:19, SEQ ID NO:21, or SEQ ID NO:23, or a biologically-active portion of any one of them. In some embodiments, the second nucleotide sequence encodes the amino acid sequence set forth in SEQ ID NO: 19, or a biologically-acti ve portion thereof.

In some embodiments, the second nucleotide sequence has at least 70%, 71%, 72%, 73%, 74%, 75%, 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, 99,1%, 99.2%, 99.3%, 99.4%, 99.5%, 99.6%, 99.7%, 99.8%, or 99.9% sequence homology with foe nucleotide sequence set forth in SEQ ID NO: 16, SEQ ID NO: 18, SEQ ID NO:20, SEQ ID NO:22, SEQ ID NO:24, SEQ ID NO:26, SEQ ID NO:28, SEQ ID NO:30, SEQ ID NO:32, SEQ ID NO:34, SEQ ID NO:52. SEQ ID NO:54, SEQ ID NO:56, SEQ ID NO:58, SEQ ID NO:60, SEQ ID NO:62, SEQ ID NO:64, SEQ ID NO:66, SEQ ID NO:68, or SEQ ID NO:70. In some embodiments, the second nucleotide sequence has at least 70% sequence homology with the nucleotide sequence set forth in SEQ ID NO:16, SEQ ID NO:18, SEQ ID NO:20, SEQ ID NO:22, SEQ ID NO:24. SEQ ID NO:26, SEQ ID NO:28, SEQ ID NO:30, SEQ ID NO:32, SEQ ID NO:34, SEQ ID NO:52, SEQ ID NO:54, SEQ ID NO:56, SEQ ID NO:58, SEQ ID NO:60, SEQ ID NO:62, SEQ ID NO:64, SEQ ID NO:66, SEQ ID NO:68, or SEQ ID NO:70. In some embodiments, the second nucleotide sequence has at least 95% sequence homology with the nucleotide sequence set forth in SEQ ID NO:16, SEQ ID NO:18, SEQ ID NO:20, SEQ ID NO:22, SEQ ID NO:24, SEQ ID NO:26, SEQ ID NO:28, SEQ ID NO:30, SEQ ID NO:32, SEQ ID NO:34, SEQ ID NO:52, SEQ ID NO:54, SEQ ID NO:56, SEQ ID NO:58, SEQ ID NO:60, SEQ ID NO:62, SEQ ID NO:64, SEQ ID NO:66, SEQ ID NO:68, or SEQ ID NO:70. In some embodiments, the second nucleotide sequence comprises the nucleotide sequence set forth in SEQ ID NO:16, SEQ ID NO:18, SEQ ID NO:20, SEQ ID NO:22, SEQ ID NO:24, SEQ ID NO:26, SEQ ID NO:28, SEQ ID NO:30, SEQ ID NO:32, SEQ ID NO:34, SEQ ID NO:52, SEQ ID NO:54, SEQ ID NO:56, SEQ ID NO:58, SEQ ID NO:60, SEQ IDNO:62, SEQ ID NO:64, SEQ ID NO:66, SEQ ID NO:68, or SEQ ID NO:70. In some embodiments, the second nucleotide sequence comprises the nucleotide sequence set forth in SEQ ID NO:20, SEQ ID NO:22, or SEQ ID NO:24. In some embodiments, the second nucleotide sequence comprises the nucleotide sequence set forth in SEQ ID NO:20.

In certain embodiments, the third nucleotide sequence is capable of recombintng with a nucleotide sequence in a triacylglycerol lipase gene and/or a nucleotide sequence in the regulatory region of a triacylglycerol lipase gene. For example, the triacylglycerol lipase may be encoded by a TGL3, TGL3/4, or TGL4 gene. In some embodiments, the cell is Arxula adminivorans; and said triacylglycerol lipase comprises the amino acid sequence set forth in SEQ ID NO:35, SEQ ID NO:37, or SEQ ID NO:39. In some embodiments, the cell is Arxula adenininivorans; and said triacylglycerol lipase is encoded by the nucleotide sequence set forth in SEQ ID NO:36, SEQ ID NO:38, or SEQ ID NO:40. In other embodiments, the cell is Yarrowia lipolytica; and said triacylglycerol lipase comprises the amino acid sequence set forth in SEQ ID NO:41 or SEQ ID NO:85. In some embodiments, the cell is Yarrowia lipolytica; and said triacylglycerol lipase is encoded by the nucleotide sequence set forth in SEQ ID NO:42 or SEQ I D NO:86.

In some embodiments, the third nucleotide sequence comprises a gene encoding a protein or a portion of a protein. The protein may confer resistance to a drug. The protein may enable the cell to grow or proliferate more quickly on a nutrient source than a cell of the same species that does not express the protein.

In some embodiments, the parent cell is transformed with a first nucleic acid that encodes the first nucleotide sequence and a second nucleic acid that encodes the second nucleotide sequence. In other embodiments, the parent cell is transformed with a first nucleic acid that encodes the first nucleotide sequence and the second nucleotide sequence. The cell may be transformed with a third nucleic acid that encodes the third nucleotide sequence, or either the first nucleic acid or second nucleic acid may encode the third nucleotide sequence. Still, in other embodiments, the parent cell is transformed with a nucleic acid that encodes the first nucleotide sequence, second nucleotide sequence, and third nucleotide sequence.

### Methods Related to DGA2

In some aspects, the invention provides a method of increasing the triacylglycerol content of a cell, comprising: (a) providing a cell comprising a genetic modification, wherein said genetic modification increases the activity of a native type 1 diacylgylcerol acyltransferase or encodes at least one copy of a type 1 diacylgylcerol acyltransferase gene native to the cell or from a different species; (b) growing said cell under conditions whereby the genetic modification is expressed, thereby producing a triacylglycerol; and (c) optionally recovering the triacylglycerol. The aforementioned method may also be used to modify the lipid composition of a cell.

In some aspects, the invention provides a method of increasing the lipid content of a cell, comprising transforming a parent cell with a nucleotide sequence, wherein said nucleotide sequence increase the activ ity of a native type 1 diacylgylcerol acyltransferase or encodes at least one copy of a type 1 diacylgylcerol acyltransferase gene.

The nucleotide sequence may comprise a type I diacylgylcerol acyltransferase gene. In certain embodiments, the nucleotide sequence encodes an amino acid sequence having at least 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, 99.1%, 99.2%, 99.3%, 99.4%, 99.5%, 99.6%, 99.7%, 99,8%, or 99.9% sequence homology with the amino acid sequence set forth in SEQ ID NO:1, SEQ ID NO:3, SEQ ID NO:5, SEQ ID NO:7, SEQ ID NO:9, SEQ ID NO:11, SEQ ID NO:13, SEQ ID NO:71, SEQ ID NO:73, SEQ ID NO:75, SEQ ID NO:77, SEQ ID NO:79, SEQ ID NO:81, or SEQ ID NO:83, or a biologically-active portion of any one of them. In some embodiments, the nucleotide sequence encodes an amino acid sequence having at least 80% sequence homology with the amino acid sequence set forth in SEQ ID NO:1, SEQ ID NO:3, SEQ ID NO:5, SEQ ID NO:7, SEQ ID NO:9, SEQ ID NO:11, SEQ ID NO:13, SEQ ID NO:71, SEQ ID NO:73, SEQ ID NO:75, SEQ ID NO:77, SEQ ID NO:79, SEQ ID NO:81, or SEQ ID NO:83, or a biologically-active portion of any one of them. In some embodiments, the nucleotide sequence encodes an amino acid sequence having at least 95% sequence homology with the amino acid sequence set forth in SEQ ID NO:1, SEQ ID NO:3, SEQ ID NO:5, SEQ ID NO:7, SEQ ID NO:9, SEQ ID NO:11, SEQ ID NO:13, SEQ ID NO:71, SEQ ID NO:73, SEQ ID NO:75, SEQ ID NO:77, SEQ ID NO:79, SEQ ID NO:81, or SEQ ID NO:83, or a biologically-active portion of any one of them. In some embodiments, the nucleotide sequence encodes the amino acid sequence set forth in SEQ ID NO:1, SEQ ID NO:3, SEQ ID NO:5, SEQ ID NO:7, SEQ ID NO:9, SEQ ID NO:11, SEQ ID NO:13, SEQ ID NO:7I, SEQ ID NO:73, SEQ ID NO:75, SEQ ID NO:77, SEQ ID NO:79, SEQ ID NO:81, or SEQ ID NO:83, or a biologically-active portion of any one of them. In some embodiments, the nucleotide sequence encodes the amino acid sequence set forth In SEQ ID NO:1, SEQ ID NO:9, SEQ ID NO:11, or SEQ ID NO:81, or a biologically-active portion of any one of them.

In some embodiments, the nucleotide sequence has at least 70%, 71%, 72%, 73%, 74%, 75%, 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, 99.1%, 99.2%, 99,3%, 99.4%, 99.5%, 99.6%, 99,7%, 99.8%, or 99.9% sequence homology with the nucleotide sequence set forth in SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO:10, SEQ ID NO:12, SEQ ID NO:14, SEQ ID NO:72, SEQ ID NO:74, SEQ ID NO:76, SEQ ID NO:78, SEQ ID NO:80, SEQ ID NO:82, or SEQ ID NO:84. In some embodiments, the first nucleotide sequence has at least 70% sequence homology with the nucleotide sequence set forth in SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO:10, SEQ ID NO:12, SEQ ID NO:14, SEQ ID NO:72, SEQ ID NO:74, SEQ ID NO:76, SEQ ID NO:78, SEQ ID NO:80, SEQ ID NO:82, or SEQ ID NO:84. In some embodiments, the nucleotide sequence has at least 95% sequence homology with the nucleotide sequence set forth in SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO:10, SEQ ID NO:12, SEQ ID NO:14, SEQ ID NO:72, SEQ ID NO:74, SEQ ID NO:76, SEQ ID NO:78, SEQ ID NO:80, SEQ ID NO:82, or SEQ ID NO:84. In some embodiments, the nucleotide sequence comprises the nucleotide sequence set forth in SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NOrlO, SEQ ID NO:12, SEQ ID NO:14, SEQ ID NO:72, SEQ ID NO:74, SEQ ID NO:76, SEQ ID NO:78, SEQ ID NO:80, SEQ ID NO:82, or SEQ ID NO:84. In some embodiments, the nucleotide sequence comprises the nucleotide sequence set forth in SEQ ID NO:2, SEQ ID NO:10, SEQ ID NO:12, or SEQ ID NO:82.

### Methods Related to DGA3

In some aspects, the invention provides a method of increasing the triacylglycerol content of a cell, comprising: (a) providing a cell comprising a genetic modification, wherein said genetic modification increases the activity of a native type 3 diacylgylcerol acyltransferase or encodes at least one copy of a type 3 diacylgylcerol acyltransferase gene native to the cell or from a different species; (b) growing said cell under conditions whereby the genetic modification is expressed, thereby producing a triacylglycerol; and (c) optionally recovering the triacylglycerol. The aforementioned method may also be used to modify the lipid composition of a cell.

In some aspects, the in vention provides a method of increasing the lipid content of a cell comprising transforming a parent cell with a nucleotide sequence, wherein said nucleotide sequence increase the activity of a native type 3 diacylgylcerol acyltransferase or encodes at least one copy of a type 3 diacylgylcerol acyltransferase gene.

The nucleotide sequence may comprise a type 3 diacylgylcerol acyltransferase gene. In certain embodiments, the nucleotide sequence encodes an amino acid sequence having at least 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, 99.1%, 99.2%, 99.3%, 99.4%, 99.5%, 99.6%, 99.7%, 99.8%, or 99.9% sequence homology with the amino acid sequence set forth in SEQ ID NO:87 or SEQ ID NO:89, or a biologically-active portion of either one of them. In some embodiments, the nucleotide sequence encodes an amino acid sequence having at least 80% sequence homology with the amino acid sequence set forth in SEQ ID NO:87 or SEQ ID NO:89, or a biologically-active portion of either one of them. In some embodiments, the nucleotide sequence encodes an amino acid sequence having at least 95% sequence homology with the amino acid sequence set forth in SEQ ID NO:87 or SEQ ID NO:89, or a biologically-active portion of any one of them. In some embodiments, the nucleotide sequence encodes the amino acid sequence set forth in SEQ ID NO:87 or SEQ ID NO:89, or a biologically-active portion of either one of them.

In some embodiments, the nucleotide sequence has at least 70%, 71%, 72%, 73%, 74%, 75%, 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, 99,1%, 99.2%, 99.3%, 99.4%, 99.5%, 99.6%, 99.7%, 99.8%, or 99.9% sequence homology with the nucleotide sequence set forth in SEQ ID NO:88 or SEQ ID NO:90. In some embodiments, the first nucleotide sequence has at least 70% sequence homology with the nucleotide sequence set forth in SEQ ID NO:S88 or SEQ ID NO:90. In some embodiments, the nucleotide sequence has at least 95% sequence homology with the nucleotide sequence set forth in SEQ ID NO:88 or SEQ ID NO:90. In some embodiments, the nucleotide sequence comprises the nucleotide sequence set forth in SEQ ID NO:88 or SEQ ID NO:90.

One skilled in the art will readily appreciate that the present invention is well adapted to carry out the objects and obtain the ends and advantages mentioned, as well as those inherent therein. The embodiments described herein are not intended as limitations on the scope of the invention. These and other features, aspects, and advantages of the present invention will become better understood with reference to the following description, drawings, and claims.

The present description is further illustrated by the following examples, which should not be construed as limiting in any way. The contents of all cited references (including literature references, issued patents, published patent applications and GenBank Accession numbers as cited throughout this application) are hereby expressly incorporated by reference. When definitions of terms in documents that are incorporated by reference herein conflict with those used herein, the definitions used herein govern.

## EXEMPLIFICATION

### Example 1: Method to Increase the Activity of a DGA1 Protein (DGAT2 Gene)

Nucleic acid constructs for expressing DGA1 were described in U.S. Ser. No. 61/943,664 and PCT Patent Application No. PCT/US15/017227 (hereby incorporated by reference). FIG. 1 shows expression, construct pNC243 used for expression of the R. toruloides DGA1 gene NG66 (SEQ ID NO:20) in Y. lipolytica. DGA1 expression constructs were linearized before transformation by a PacI/NotI restriction digest. The linear expression constructs each included an expression cassette for the DGAT2 gene and for the NatI gene, used as a marker for selection with nourseothricin (NAT).

DGA1 expression constructs were randomly integrated into the genome of Y. lipolytica strain NS18 (obtained from ARS Culture Collection, NRRL#YB 392) using a transformation protocol as described in Chen (Applied Microbiology & Biotechnology 48:232-35 (1997)). Transformants were selected on YPD plates with 500 μg/mL NAT and screened for the ability to accumulate lipids by a fluorescent staining lipid assay as described in Example 2 below. For each expression construct, eight transformants were analyzed.

For most constructs, there was significant colony variation between the transformants, likely due to the lack of a functional DGA1 expression cassette in cells that only obtained a functional NatI cassette, or due to a negative effect of the site of DGA1 integration on DGA1 expression. Nevertheless, all transformants had a significant increase in lipid content.

Overexpression of native Y. lipolytica DGA1 (NG15) under a strong promoter increased the transformant's lipid content by about 2-fold compared to the parental strain NS18 as measured by the fluorescence assay described in Example 2. Transformants that demonstrated the highest fluorescence (about 3-fold higher compared to NS18) were generated by the expression of R. toruloides DGA1 (NG66, NG67) and L. starkeyi DGA1 (NG68).

In certain experiments, the effect of native R. toruloides DGA1 (NG49) expression on lipid production in Y. lipolytica was not as high as the effect of synthetic versions of R. toruloides DGAT2 genes that did not contain introns. This result may indicate that the gene splicing of the R. toruloides DGAT2 gene in Y. lipolytica was not very efficient. In certain experiments, codon optimization of the R. toruloides DGA1 gene for expression in Y. lipolytica did not have a positive effect on lipid production.

### Example 2: Lipid Assay

Each well of an autoclaved, multi-well plate was filled with filter-sterilized media containing 0.5 g/L urea, 1.5 g/L yeast extract, 0.85 g/L casamino acids, 1.7 g/L YNB (without amino acids and ammonium sulfate), 100 g/L glucose, and 5.11 g/L potassium hydrogen phthalate (25 mM). Yeast strains that had been incubated for 1-2 days on YPD-agar plates at 30° C. were used to inoculate each well of the mnltiwall plate. 1.5 ml, of media was used per well for 24-well plates and 300 μL of media was used per well for 96-well plates. Alternatively, the yeast cultures were used to inoculate 50 mL of sterilized media in an autoclaved 250 mL flask.

Multi-well plates were covered with a porous cover and incubated at 30° C., 70-90% humidity, and 900 rpm in an Infors Multitron ATR shaker. Alternatively, flasks were covered with aluminum foil and incubated at 30° C., 70-90% humidity, and 900 rpm in a New Brunswick Scientific shaker. After 96 hours, 20 μL of 100% ethanol was added to 20 μl of cells in an analytical microplate and incubated at 4° C. for 30 minutes. 20 μl of cell/ethanol mix was then added to 80 μl of a pre-mixed solution containing 50 μL 1 M potassium iodide, 1 mM μL Bodipy 493/503, 0.5 μL 100% DMSO, 1.5 μL 60% PEG 4000, and 27 μL water in a Costar 96-well, black, clear-bottom plate and covered with a transparent seal. Bodipy fluorescence was monitored with a SpectraMax M2 spectrophotometer (Molecular Devices) kinetic assay at 30° C., and normalized by dividing fluorescence by absorbance at 600 nm. Data was averaged in triplicate growth experiments.

### Example 3: Analysis and Screening of Y. lipolytica Strains that Express DGA1

In order to select strains with the highest lipid production level, Y. lipolytica strain NS18 transformants expressing NG15 (Y. lipolytica DGA1 ) or NG66 (R. toruloides DGA1 ) were screened. For NG15, about 50 colonies were screened by the lipid assay described in Example 2 for foe highest lipid accumulation, and the best transformant was named NS249, For NG66, 80 colonies were screened, and the 8 best colonies were selected for further analysis.

Strain MS249 and the 8 selected NG66 transformants were grown in shake flasks and analyzed by the lipid assay for lipid content and by HPLC for glucose consumption. Y. lipolytica strains expressing R. toruloides DGA1 had significantly higher lipid contents than Y. lipolytica strains with a native Y. lipolytica DGAT2 gene expressed under the same promoter as R. toruldoides DGAT2. At the same time, NG66 transformants used significantly more glucose than NS249, demonstrating that NG66 was more efficient in converting glucose to lipids. The difference in efficiency between the two DGAT2 genes may be attributed to either a higher level of expression of R. toruloides DGA1 in Y. lipolytica or a higher level of R. toruloides DGA1 specific activity, or both.

Strain NS125 is a derivative of Y. lipolytica strain NS18 (obtained from ARS Culture Collection, NRRL# YB 392) that was transformed with a Y. lipolytica DGA1 expression cassette from the pNC104 vector (FIG. 2). The pNC104 construct was linearized by a PacI/NotI restriction digest prior to transformation. The linear expression construct included the expression cassette for the Y. lipolytica DGAT1 gene and for the NatI gene used as a marker for selection with nourseothricin (NAT). The expression construct was randomly integrated into the genome of Y. lipolytica strain NS18 using the transformation protocol as described in Chen (Applied Microbiology & Biotechnology 48:232-35 (1997)). Transformants were selected on YPD plates with 500 μg/mL NAT and screened for the ability to accumulate lipids by a fluorescent staining lipid assay as described in Example 2. The best transformant out of about 100 transformants screened was named NS125.

The NS281 strain was obtained using a similar process as strain NS125, except that the pNC243 construct was used for the transformation of the NS18 strain (FIG. 1). The pNC243 construct contained the Rhodosporidium tondotdes DGAT2 gene (NG66) instead of Y. lipolytica DGAT2 gene used to make NS125. The NS281 strain contains a Rhodosporidium toruloides DGAT2 gene that is integrated into the Y. lipolytica genome.

### Example 4: Method to Knockout Triacylglycerol Lipase Knockout Gene in Y. lipolytica

Nucleic acid constructs for knocking out the Y. lipolytica TGL3 gene while expressing the DGAT2 gene were described in U.S. Ser. No. 61/987,098 and PCT Patent Application No. PCT/US15/28760 (both hereby incorporated by reference). The TGL3 gene was knocked out of Y. lipolytica wild-type strain NS18 (obtained from NRLL#YB-392) and its DGA1 expressing derivative NS281. NS281 expresses the DGA1 gene from Rhodosporidium torulouides as described above. The Y. lipolytica TGL3 gene (YALI0D17534g, SEQ ID NO:42) was deleted as follows: A two-fragment deletion cassette was amplified by PCR from a plasmid containing the hygromycin resistance gene (“hph,” SEQ ID NO:44) using primer pairs NP1798-NP656 and NP655-NP1799 (SEQ ID NOS: 45-48). The resulting PCR fragments (SEQ ID NOs: 49 & 50) were co-transformed into NS18 and NS281 according to the protocol developed in U.S. Ser. No. 61/819,746 and PCT Patent Application Publication No. WO 14/182657 (both hereby incorporated by reference). The omission of a promoter and terminator in the hph cassette and the splitting of the hph coding sequence into two PCR fragments reduce the probability that random integration of these pieces will confer hygromycin resistance. The hph gene should only be expressed if it integrates at the TGL3 locus by homologous recombination so that the TGL3 promoter and terminator can direct its transcription. Hygromycin resistant colonies were screened by PCR to confirm the absence of TGL3 and the presence of a tgl3::hyg specific product. Deletion of TGL3 in NS18 resulted in strain NS421. Deletion of TGL3 in NS281 resulted in strain NS377.

### Example 5: Cells that Overexpress Both DGA1 and DGA2 and That Contain a TGl3 Deletion Accumulate More TAGs Than Cells That Do Not Overexpress DGA2

In order to test the idea that combining DGA1 and DGA2 expression with TGL3 deletion leads to higher lipid accumulation in Y. lipolytica, DGA2 from Claviceps purpurea was expressed in strain NS377. Strain NS377 contains a deletion of TGL3 and expresses DGA1 from Rhodosporidium toruloides as described in Example 4 and U.S. Ser. No. 61/987,098 and PCT Patent Application No. PCT/US15/28760 (both hereby incorporated by reference). DGA2 from Claviceps purpurea was selected based on experiments that demonstrate that this gene increases the lipid content of Y. lipolytica in combination with DGA1 from Rhodosporidium toruloides.

FIG. 3 shows the map of pNC327, the expression construct used to express C. purpurea DGA2 in NS377. The construct was linearized prior to transformation with a PacI/AscI restriction digest. The linear expression construct included an expression cassette for the C. purpurea DGA2 gene and for the BLE gene used as a marker for selection with Zeocin (ZEO). Transformants were analyzed by the fluorescent lipid assay described in Example 2, and the top lipid producer was designated NS432.

The lipid production of strains NS297, NS281, NS412, NS450, NS377, and NS432 were compared. A subset of these strains were either grown using a batch glucose process (in 48-well plates or 50-ml flasks) or using a high cell density fed-batch glucose process (in 1-L bioreactors). Lipid content was analyzed by fluorescence assay or gas chromatography, and strain NS432 was found to have a higher lipid content than its parent strain NS377 and the strains without the TGL3 knockout (FIG. 4). These results demonstrate the advantage of DGA1 and DGA2 expression in a TGL3 knockout.

### Example 6: Increasing the Activity of DGA1, DGA2, or DGA3 in Arxula adeninivorans

Twenty nine genes encoding for diacylglycerol acyltransferase (DGA) type 1 (DGA2), type 2 (DGA1) and type 3 (DGA3) from various donors were selected for expression in Arxula adeninivorans strain NS252 (Table 2). The map of the expression construct used to express the DGAs in A. adeninivorans is shown in the FIG. 5, with NG167 target as an example. The constructs for all other DGAs were foe same except for the target open reading frames (ORFs). The negative control comprised the E.coli hph gene, which encodes for a phosphotransferase that confers resistance to Hygromycin B, in place of a DGA.

The expression constructs were assembled by yeast mediated ligation. Next, the full-length constructs were linearized by a PmeI/AscI restriction digest before transformation. The linear expression constructs included an expression cassette for a DGA and an expression cassette for the Streptomyces noursei NatI gene, used as marker for selection with nourseothricin (NAT). The expression contracts were randomly integrated into the genome of A. ademnivorans NS252 (ATCC #76597) using a protocol specifically adapted to A. adeninivorans. Briefly, 2 mL of YPD was inoculated with the parent A. adeninivorans culture and grown overnight at 37° C. in a rotary shaker. 0.5 mL of the overnight culture was then used to inoculate 25 mL of fresh YPD in a 250 flask, which was then grown at 37° C. for 3.5 to 4 hours. The ceils were pelleted at 3000 rpm for 2 minutes, and the supernatant was discarded. The cells were washed in sterile water, and pelleted again at 3000 rpm for 2 minutes. The pellet was suspended in 2 mL of 100 mM lithium acetate comprising 40 gM dithiothreitol and transferred into a microcentrifuge tube. The suspension was incubated for one hour at 37° C. on a rotary shaker. The cells were pelleted at 10,000 rpm for 10 seconds and the supernatant was discarded. The cells were resuspended in 1 mL of water with gentle pipetting, centrifuged again at 10,000 rpm for 10 seconds, and the water was discarded. The cells were washed by pipetting with 1 M cold sorbitol, then centrifuged again at 10,000 rpm for 10 seconds, and the supernatant discarded. 2 mL of cold 1 M sorbitol was added to the pellet, and the tube was placed on ice. 40 μL of the cells were then added to pre-chilled 0.2 cm electroporation cuvettes along with 5 μL of DNA. The cells were eleetroporated at 25 μF, 200 ohms, 1.5 kV, with a ˜4.9-5.0 ms time constant. The cells were added to 1 mL of YPD, incubated at 37° C. overnight, and 100 μL to 500 μL of cells were plated onto YPD agar.

A. adeninivorans transformants were selected on YPD plates with 50 μg/mL NAT. The transformants were screened for an ability to accumulate lipids by the fluorescent staining lipid assay described in Example 2. The results of the lipid assays are shown in the FIG. 6. For each expression construct, eight transformants were analyzed. Transformants expressing phosphotransferase instead of DGA were used as negative controls (“Neg Cont”). The assays were carried out in 4 plates with the same negative control and NG168, used as a positive control, in each plate to discount for assay variability between plates. FIG. 6 demonstrates that most of the DGAs tested displayed a positive effect on lipid content in A. adeninivorans. DGA2s displayed more significant positive effects on lipid content in A. adeninivorans relative to DGA1s and DGA3s. DGA2s from Y. lipolytica (NG16) and Chaetomium globosum (NG113) displayed the most significant effect on lipid content in A. adeninivoram.

### Example 7: Increasing the Activity of DGA1, DGA2, or DGA3 in Yarrowia lipolytica

Eighteen genes encoding diacylglycerol acyltransferase type 1, type 2 and type 3 from various donors, listed in Table 2, were expressed in Yarrowia lipolytica strain NS598. Strain NS598 is a derivative of Y. lipolytica strain NS18 (obtained from ARS Culture Collection, NRRL#YB 392) with two genetic modifications: 1) the native Δ9 desaturase gene was replaced with a Δ9 desaturase gene from A. adeninivorans; and 2) the native Δ12dcsaturase gene was deleted and replaced with expression cassettes for E. coli phosphotransferase, which confers resistance to Hygromycin B, and Herpes Simplex Virus thymidine kinase, which confers sensitivity to 5-Fluoro-2′-deoxyuridine (see U.S. Provisional Patent Application No. 62/090,169, hereby incorporated by reference).

The map of the expression construct used to express the DGAs in Y. lipolytica is shown in FIG. 7, with the NG288 target as example. The constructs for all other DGAs (except for NG112) were the same, except for the target open reading frames. The expression construct used to express NG112 in NS589 is shown in FIG. 3. The expression constructs were assembled by yeast mediated ligation as described below, except for the NG112 gene, which is described in Example 3, supra.

The constructs were linearized by PmeI/AscI restriction digest before transformation. Each linear expression construct included an expression cassette for a DGA gene and for the S. cerevisiae SUC2 gene encoding invertase, used as marker for selection with sucrose. The expression constructs were randomly integrated into the genome of NS598 using foe transformation protocol described by Chen el al. (Applied Microbiology & Biotechnology 45:232-35 (1997)). The Y. lipolytica transformants were selected on YNB plates with 2% sucrose. The transformants were screened for an ability to accumulate lipids by the fluorescent staining lipid assay described in Example 2. Results for the lipid assays are shown in the FIG. 8. For each expression construct, eight transformants were analyzed. The parental strain NS598 was used as negative control. The assays were carried out in 3 plates. The data in FIG. 8 demonstrates that each DGA1 and DGA3 tested displayed a significant positive effect on the lipid content of Y. lipolytica. Some DGA2s also displayed a positive effect on the lipid content of Y. lipolytica, with Claviceps purpurea DGA2 (NG112) showing the largest increase. These results vary from the DGA screen described for A. adminivorans in Example 6, supra, which suggests that the effect of different DGAs on lipid content is host organism specific.

## EQUIVALENTS

Those skilled in the art will recognize, or be able to ascertain using no more than routine experimentation, many equivalents to the specific embodiments of the invention described herein. Such equivalents are intended to be encompassed by the following claims.

