# DESCRIPTION

## FEDERALLY SPONSORED RESEARCH OR DEVELOPMENT

This invention was made with Government support under Grant Number 3U54-A1057159-0651 awarded by the National Institutes of Health. The Government has certain rights in the invention.

## TECHNICAL FIELD

The invention relates, inter alia, to methods of detecting, diagnosing, and/or identifying pathogens, e.g., infectious disease pathogens, and determining their sensitivity to known or potential treatments.

## BACKGROUND

The development of molecular diagnostics has revolutionized care in most medical disciplines except infectious disease, where they have failed to play a widespread, transforming role. The reliance on slow culture methods is particularly frustrating in the current crisis of antibiotic resistance as the development of molecular tools to rapidly diagnose the inciting pathogen and its drug resistance profile would transform the management of bacterial, fungal, viral, and parasitic infections, guiding rapid, informed drug treatment in an effort to decrease mortality, control health care costs, and improve public health control of escalating resistance among pathogens. In U.S. hospitals alone, 1.7 million people acquire nosocomial bacterial infection and 99,000 die every year, with 70% of these infections due to bacteria resistant to at least one drug and an estimated annual cost of $45 billion (Klevens et al., 2002. Public Health Rep. 2007; 122(2):160-6; Klevens et al., Clin Infect Dis. 2008; 47(7):927-30; Scott, The Direct Medical Costs of Healthcare-Associated Infection in U.S. Hospitals and the Benefits of Prevention. In: Division of Healthcare Quality Promotion NCfP, Detection and Control of Infectious Diseases, editor. Atlanta: CDC, 2009). However, the problem is not limited to the U.S. and microbial resistance now impacts the majority of common bacterial infections globally. Global spread of methicillin-resistant S. aureus (MRSA), multi-drug resistant tuberculosis (MDR-TB), and increasingly drug resistant Gram-negative organisms prompted the formulation of an action plan focusing on surveillance, prevention and control, research and product development (US action plan to combat antimicrobial resistance. Infect Control Hosp Epidemiol. 2001; 22(3):183-4). However, minimal progress has been made on any of these fronts.

Prompt administration of the appropriate antibiotic has repeatedly been shown to minimize mortality in patients with severe bacterial infections, whether within the hospital setting with nosocomial pathogens such as E. faecium, S. aureus, K. pneumoniae, A. baumanii, P. aeruginosa, and Enterobacter species, or in resource-poor settings with pathogens such as tuberculosis (TB) (Harbarth et al., Am J. Med. 2003; 115(7):529-35; Harries et al., Lancet. 2001; 357(9267):1519-23; Lawn et al., Int J Tuberc Lung Dis. 1997; 1(5):485-6). However, because current diagnostic methods involving culture and sub-culture of organisms can take several days or more to correctly identify both the organism and its drug susceptibility pattern, physicians have resorted to increasing use of empiric broad-spectrum antibiotics, adding to the selective pressure for resistance and increasing the associated health-care costs. A point of care diagnostic to rapidly (e.g., less than 1 hour) detect pathogens and their resistance profiles is urgently needed and could dramatically change the practice of medicine. Some effort into designing DNA- or PCR-based tests has resulted in tools that are able to identify pathogens rapidly with low detection limits. However, global use of these tools is currently limited due to cost and demand for laboratory infrastructure and to the inherent insensitivity of PCR-based methods in the setting of crude samples that are not easily amenable to the required enzymology. Molecular approaches to determining drug resistance have been even more limited, available for some organisms (e.g., MRSA, TB) in very limited ways, based on defining the genotype of the infecting bacteria relative to known resistance conferring mutations. This method however, requires fairly comprehensive identification of all resistance conferring single nucleotide polymorphisms (SNPs) for the test to have high sensitivity (Carroll et al., Mol Diagn Ther. 2008; 12(1):15-24).

## SUMMARY

The present invention is based, at least in part, on the discovery of new methods of diagnosing disease, identifying pathogens, and optimizing treatment based on detection of mRNA, e.g., in crude, non-purified samples. The methods described herein provide rapid and accurate identification of pathogens in samples, e.g., clinical samples, and allow for the selection of optimal treatments based on drug sensitivity determinations.

In one aspect, the invention features methods of determining the drug sensitivity of a pathogen, e.g., a disease-causing organism such as a bacterium, fungus, virus, or parasite. The methods include providing a sample comprising a pathogen and contacting the sample with one or more test compounds, e.g., for less than four hours, to provide a test sample. The test sample can be treated under conditions that release mRNA from the pathogen into the test sample and the test sample is exposed to a plurality of nucleic acid probes, comprising a plurality of subsets of probes, wherein each subset comprises one or more probes that bind specifically to a target mRNA that is differentially expressed in organisms that are sensitive to a test compound as compared to organisms that are resistant, wherein the exposure occurs for a time and under conditions in which binding between the probe and target mRNA can occur. The method comprises determining a level of binding between the probe and target mRNA, thereby determining a level of the target mRNA; and comparing the level of the target mRNA in the presence of the test compound to a reference level, e.g., the level of the target mRNA in the absence of the test compound, wherein a difference in the level of target mRNA relative to the reference level of target mRNA indicates whether the pathogen is sensitive or resistant to the test compound.

In one embodiment, the pathogen is known, e.g., an identified pathogen. In some embodiments, the methods determine the drug sensitivity of an unknown pathogen, e.g., a yet to be identified pathogen.

In some embodiments, the sample comprising the pathogen is contacted with two or more test compounds, e.g., simultaneously or in the same sample, e.g., contacted with known or potential treatment compounds, e.g., antibiotics, antifungals, antivirals, and antiparasitics. A number of these compounds are known in the art, e.g., isoniazid, rifampicin, pyrazinamide, ethambutol streptomycin, amikacin, kanamycin, capreomycin, viomycin, enviomycin, ciprofloxacin, levofloxacin, moxifloxacin, ethionamide, prothionamide, cycloserine, p-aminosalicylic acid, rifabutin, clarithromycin, linezolid, thioacetazone, thioridazine, arginine, vitamin D, R207910, ofloxacin, novobiocin, tetracycline, merepenem, gentamicin, neomycin, netilmicin, streptomycin, tobramycin, paromomycin, geldanamycin, herbimycin, loracarbef, ertapenem, doripenem, imipenem/cilastatin, meropenem, cefadroxil, cefazolin, cefalotin, cefalexin, cefaclor, cefamandole, cefoxitin, cefprozil, cefuroxime, cefixime, cefdinir, cefditoren, cefoperazone, cefotaxime, cefpodoxime, ceftazidime, ceftibuten, ceftizoxime, ceftriaxone, cefepime, ceftobiprole, teicoplanin, vancomycin, azithromycin, dirithromycin, erythromycin, roxithromycin, troleandomycin, telithromycin, spectinomycin, aztreonam, amoxicillin, ampicillin, azlocillin, carbenicillin, cloxacillin, dicloxacillin, flucloxacillin, mezlocillin, methicillin, nafcillin, oxacillin, penicillin, piperacillin, ticarcillin, bacitracin, colistin, polymyxin B, enoxacin, gatifloxacin, lomefloxacin, norfloxacin, trovafloxacin, grepafloxacin, sparfloxacin, mafenide, prontosil, sulfacetamide, sulfamethizole, sulfanilimide, sulfasalazine, sulfisoxazole, trimethoprim, trimethoprim-sulfamethoxazole (co-trimoxazole), demeclocycline, doxycycline, minocycline, oxytetracycline, arsphenamine, chloramphenicol, clindamycin, lincomycin, ethambutol, fosfomycin, fusidic acid, furazolidone, metronidazole, mupirocin, nitrofurantoin, platensimycin, quinupristin/dalfopristin, rifampin, thiamphenicol, tinidazole, cephalosporin, teicoplatin, augmentin, cephalexin, rifamycin, rifaximin, cephamandole, ketoconazole, latamoxef, or cefmenoxime.

In some embodiments, the sample is contacted with the compound for less than four hours, e.g., less than three hours, less than two hours, less than one hour, less than 30 minutes, less than 20 minutes, less than 10 minutes, less than five minutes, less than two minutes, less than one minute.

In another aspect, the invention features methods of identifying an infectious disease pathogen, e.g., a bacterium, fungus, virus, or parasite, e.g., Mycobacterium tuberculosis, e.g., detecting the presence of the pathogen in a sample, e.g., a clinical sample. The methods include:


- - providing a test sample from a subject suspected of being infected
    with a pathogen;
  - treating the test sample under conditions that release messenger
    ribonucleic acid (mRNA);
  - exposing the test sample to a plurality of nucleic acid probes,
    comprising a plurality of subsets of probes, wherein each subset
    comprises one or more probes that bind specifically to a target mRNA
    that uniquely identifies a pathogen, wherein the exposure occurs for
    a time and under conditions in which binding between the probe and
    the target mRNA can occur; and
  - determining a level of binding between the probe and target mRNA,
    thereby determining a level of target mRNA. An increase in the
    target mRNA of the test sample, relative to a reference sample,
    indicates the identity of the pathogen in the test sample.

In some embodiments, the methods identify an infectious disease pathogen in or from a sample that is or comprises sputum, blood, urine, stool, joint fluid, cerebrospinal fluid, and cervical/vaginal swab. Such samples may include a plurality of other organisms (e.g., one or more non-disease causing bacteria, fungi, viruses, or parasites) or pathogens. In some embodiments, the sample is a clinical sample, e.g., a sample from a patient or person who is or may be undergoing a medical treatment by a health care provider.

In some embodiments of the invention, the one or more nucleic acid probes are selected from Table 2.

In some embodiments, the mRNA is crude, e.g., not purified, before contact with the probes and/or does not include amplifying the mRNA, e.g., to produce cDNA.

In some embodiments, the methods comprise lysing the cells enzymatically, chemically, and/or mechanically.

In some embodiments, the methods comprise use of a microfluidic device.

In some embodiments, the methods are used to monitor pathogen infection, e.g., incidence, prevalence, for public health surveillance of an outbreak of a pathogen, e.g., a sudden rise in numbers of a pathogen within a particular area.

The methods described herein are effective wherein the pathogen is in a sample from a subject, including humans and animals, such as laboratory animals, e.g., mice, rats, rabbits, or monkeys, or domesticated and farm animals, e.g., cats, dogs, goats, sheep, pigs, cows, horses, and birds, e.g., chickens.

In some embodiments, the methods further feature determining and/or selecting a treatment for the subject and optionally administering the treatment to the subject, based on the outcome of an assay as described herein.

In another general aspect, the invention features methods of selecting a treatment for a subject. The methods include:


- - optionally identifying an infectious disease pathogen (e.g.,
    detecting the presence and/or identity of a specific pathogen in a
    sample), e.g., using a method described herein;
  - determining the drug sensitivity of the pathogen using the methods
    described herein; and
  - selecting a drug to which the pathogen is sensitive for use in
    treating the subject.

In yet another aspect, the invention provides methods for monitoring an infection with a pathogen in a subject. The methods include:


- - obtaining a first sample comprising the pathogen at a first time;
  - determining the drug sensitivity of the pathogen in the first sample
    using the method described herein;
  - optionally selecting a treatment to which the pathogen is sensitive
    and administering the selected treatment to the subject;
  - obtaining a second sample comprising the pathogen at a second time;
  - determining the drug sensitivity of the pathogen in the second
    sample using the method described herein; and
  - comparing the drug sensitivity of the pathogen in the first sample
    and the second sample, thereby monitoring the infection in the
    subject.

In some embodiments of the methods described herein, the subject is immune compromised.

In some embodiments of the methods described herein, the methods include selecting a treatment to which the pathogen is sensitive and administering the selected treatment to the subject, and a change in the drug sensitivity of the pathogen indicates that the pathogen is or is becoming resistant to the treatment, e.g., the methods include determining the drug sensitivity of the pathogen to the treatment being administered.

In some embodiments, a change in the drug sensitivity of the pathogen indicates that the pathogen is or is becoming resistant to the treatment, and the method further comprises administering a different treatment to the subject.

In yet another aspect, the invention features methods of monitoring an infection with a pathogen in a population of subjects. The methods include:


- - obtaining a first plurality of samples from subjects in the
    population at a first time;
  - determining the drug sensitivity of pathogens in the first plurality
    of samples using the method described herein, and optionally
    identifying an infectious disease pathogen in the first plurality of
    samples using the method described herein;
  - optionally administering a treatment to the subjects;
  - obtaining a second plurality of samples from subjects in the
    population at a second time;
  - determining the drug sensitivity of pathogens in the second
    plurality of samples using the method described herein, and
    optionally identifying an infectious disease pathogen in the first
    plurality of samples using the method described herein;
  - comparing the drug sensitivity of the pathogens, and optionally the
    identity of the pathogens, in the first plurality of samples and the
    second plurality of samples, thereby monitoring the infection in the
    population of subject.

In yet another aspect, a plurality of polynucleotides bound to a solid support are provided. Each polynucleotide of the plurality selectively hybridizes to one or more genes from Table 2. In some embodiments, the plurality of polynucleotides comprise SEQ ID NOs:1-227, and any combination thereof.

“Infectious diseases” also known as communicable diseases or transmissible diseases, comprise clinically evident illness (i.e., characteristic medical signs and/or symptoms of disease) resulting from the infection, presence, and growth of pathogenic biological agents in a subject (Ryan and Ray (eds.) (2004). Sherris Medical Microbiology (4th ed.). McGraw Hill). A diagnosis of an infectious disease can confirmed by a physician through, e.g., diagnostic tests (e.g., blood tests), chart review, and a review of clinical history. In certain cases, infectious diseases may be asymptomatic for some or all of their course. Infectious pathogens can include viruses, bacteria, fungi, protozoa, multicellular parasites, and prions. One of skill in the art would recognize that transmission of a pathogen can occur through different routes, including without exception physical contact, contaminated food, body fluids, objects, airborne inhalation, and through vector organisms. Infectious diseases that are especially infective are sometimes referred to as contagious and can be transmitted by contact with an ill person or their secretions.

As used herein, the term “gene” refers to a DNA sequence in a chromosome that codes for a product (either RNA or its translation product, a polypeptide). A gene contains a coding region and includes regions preceding and following the coding region (termed respectively “leader” and “trailer”). The coding region is comprised of a plurality of coding segments (“exons”) and intervening sequences (“introns”) between individual coding segments.

The term “probe” as used herein refers to an oligonucleotide that binds specifically to a target mRNA. A probe can be single stranded at the time of hybridization to a target.

Unless otherwise defined, all technical and scientific terms used herein have the same meaning as commonly understood by one of ordinary skill in the art to which this invention belongs. Methods and materials are described herein for use in the present invention; other, suitable methods and materials known in the art can also be used. The materials, methods, and examples are illustrative only and not intended to be limiting. All publications, patent applications, patents, sequences, database entries, and other references mentioned herein are incorporated by reference in their entirety. In case of conflict, the present specification, including definitions, will control.

Other features and advantages of the invention will be apparent from the following detailed description and figures, and from the claims.

## DETAILED DESCRIPTION

Described herein are rapid, highly sensitive, phenotypic-based methods for both identifying a pathogen, e.g., bacterium, fungus, virus, and parasite, and its drug resistance pattern based on transcriptional expression profile signatures. Sensitive and resistant pathogens respond very differently to drug exposure with one of the earliest, most rapid responses reflected in alterations in their respective expression profiles. Digital gene expression with molecular barcodes can be used to detect these early transcriptional responses to drug exposure to distinguish drug sensitive and resistant pathogens in a rapid manner that requires no enzymology or molecular biology. The invention is applicable to a broad range of microbial pathogens in a variety of clinical samples and can be used in conjunction with current diagnostic tools or independently. The methods will be described primarily for use with tuberculosis (“TB;” Mycobacterium tuberculosis), although it will be understood by skilled practitioners that they may be adapted for use with other pathogens and their associated clinical syndromes (e.g., as listed in Table 1).

The diagnosis and the identification of drug resistance is especially challenging regarding TB due to the extremely slow growth of TB that is required for culture testing even using the more rapid “microscopic-observation drug-susceptibility” (MODS) culture method, phage-delivered reporters, or colorimetric indicators. An alternative approach to determining drug resistance is based on defining the genotype of the infecting pathogen relative to known resistance conferring mutations, however, this approach requires a fairly comprehensive identification of all resistance-conferring single nucleotide polymorphisms (SNPs) in order for the test to have high sensitivity.

The methods described herein can be used, e.g., for identifying a pathogen in a sample, e.g., a clinical sample, as well as determining the drug sensitivity of a pathogen based on expression profile signatures of the pathogen. One of the earliest, most rapid responses that can be used to distinguish drug sensitive and resistant pathogens is their respective transcriptional profile upon exposure to a drug of interest. Pathogens respond very differently to drug exposure depending on whether they are sensitive or resistant to that particular drug. For example, in some cases drug sensitive or drug resistant bacteria will respond within minutes to hours to drug exposure by up- and down-regulating genes, perhaps attempting to overcome the drug as well as the more non-specific stresses that follow while resistant bacteria have no such response. This rapid response is in contrast to the longer time that is required by a compound to kill or inhibit growth of a pathogen. Detecting death or growth inhibition of a pathogen in an efficient manner from clinical samples represents an even greater challenge. Digital gene expression can be used, e.g., with molecular barcodes, to detect these early transcriptional responses to drug exposure to distinguish drug sensitive and resistant pathogens in a rapid manner that requires no enzymology or molecular biology, and thus can be performed directly on crude clinical samples collected from patients. This readout is phenotypic and thus requires no comprehensive definition of SNPs accounting for, e.g., TB drug resistance. Described herein are a set of genes that will provide high specificity for a pathogen, e.g., TB bacillus, and for distinguishing sensitive and resistant pathogens. Based on the selection of genes that constitute the expression signature distinguishing sensitive and resistant pathogens, the sensitivity of the detection limit is optimized by choosing transcripts that are abundantly induced, and thus not limited solely by the number of pathogens within a clinical sample. The size of this set is determined to minimize the numbers of genes required. Thus, the current invention can be used as a highly sensitive, phenotypic test to diagnose a pathogen with its accompanying resistance pattern that is rapid (e.g., a few hours), sensitive, and specific. This test can transform the care of patients infected with a pathogen and is a cost-effective, point-of-care diagnostic for, e.g., TB endemic regions of the world.

The present methods allow the detection of nucleic acid signatures, specifically RNA levels, directly from crude cellular samples with a high degree of sensitivity and specificity. This technology can be used to identify TB and determine drug sensitivity patterns through measurement of distinct expression signatures with a high degree of sensitivity and with rapid, simple processing directly from clinical samples, e.g. sputum, urine, blood, or feces; the technology is also applicable in other tissues such as lymph nodes. High sensitivity can be attained by detecting mRNA rather than DNA, since a single cell can carry many more copies of mRNA per cell (>103) compared to a single genomic DNA copy (which typically requires amplification for detection), and by the high inherent sensitivity of the technology (detects <2000 copies mRNA). The rapid, simple sample processing is possible due to the lack of enzymology and molecular biology required for detection of mRNA molecules; instead, in some embodiments, the methods make use of hybridization of bar-coded probes to the mRNA molecules in crude lysates followed by direct visualization (e.g., as illustrated in FIG. 1). Because hybridization is used in these embodiments, mRNA can be detected directly without any purification step from crude cell lysates, fixed tissue samples, and samples containing guanidinium isothiocyanate, polyacrylamide, and Trizol®. Crude mRNA samples can be obtained from biological fluids or solids, e.g., sputum, blood, urine, stool, joint fluid, cerebrospinal fluid, cervical/vaginal swab, biliary fluid, pleural fluid, peritoneal fluid, or pericardial fluid; or tissue biopsy samples, e.g., from bone biopsy, liver biopsy, lung biopsy, brain biopsy, lymph node biopsy, esophageal biopsy, colonic biopsy, gastric biopsy, small bowel biopsy, myocardial biopsy, skin biopsy, and sinus biopsy can also be used.

### RNA Extraction

RNA can be extracted from cells in a sample, e.g., a pathogen cell or clinical sample, by treating the sample enzymatically, chemically, or mechanically to lyse cells in the sample and release mRNA. It will be understood by skilled practitioners that other disruption methods may be used in the process.

The use of enzymatic methods to remove cell walls is well-established in the art. The enzymes are generally commercially available and, in most cases, were originally isolated from biological sources. Enzymes commonly used include lysozyme, lysostaphin, zymolase, mutanolysin, glycanases, proteases, and mannose.

Chemicals, e.g., detergents, disrupt the lipid barrier surrounding cells by disrupting lipid-lipid, lipid-protein and protein-protein interactions. The ideal detergent for cell lysis depends on cell type and source. Bacteria and yeast have differing requirements for optimal lysis due to the nature of their cell wall. In general, nonionic and zwitterionic detergents are milder. The Triton X series of nonionic detergents and 3-[(3-Cholamidopropyl)dimethylammonio]-1-propanesulfonate (CHAPS), a zwitterionic detergent, are commonly used for these purposes. In contrast, ionic detergents are strong solubilizing agents and tend to denature proteins, thereby destroying protein activity and function. SDS, an ionic detergent that binds to and denatures proteins, is used extensively in the art to disrupt cells.

Physical disruption of cells may entail sonication, French press, electroporation, or a microfluidic device comprising fabricated structures can be used to mechanically disrupt a cell. These methods are known in the art.

Digital Gene Expression with Molecular Barcodes

A flow diagram is shown in FIG. 1 of an exemplary procedure to identify a pathogen based on its gene expression profile. Oligonucleotide probes to identify each pathogen of interest were selected by comparing the coding sequences from the pathogen of interest to all gene sequences in other organisms by BLAST software. Only probes of about 50 nucleotides, e.g., 80 nucleotides, 70 nucleotides, 60 nucleotides, 40 nucleotides, 30 nucleotides, and 20 nucleotides, with a perfect match to the pathogen of interest, but no match of >50% to any other organism were selected. Two probes corresponding to each mRNA of interest and within 100 base pairs of each other are selected.

Two molecular probes are added to a crude sample lysate containing mRNA molecules. A capture probe comprises 50 nucleotides complementary to a given mRNA molecule, and can be conjugated to biotin. A reporter probe comprises a different 50 nucleotides complementary to a different part of the same mRNA molecule, and can be conjugated to a reporter molecule, e.g., a fluorescent tag or quantum dot. Each reporter molecule uniquely identifies a given mRNA molecule. The capture and reporter probes hybridize to their corresponding mRNA molecules within the lysate. Excess reporter is removed by bead purification that hybridizes to a handle on each oligomer, leaving only the hybridized mRNA complexes. The mRNA complexes can be captured and immobilized on a surface, e.g., a streptavidin-coated surface. An electric field can be applied to align the complexes all in the same direction on the surface before the surface is microscopically imaged.

The reporter molecules can be counted to provide a quantitative measure of mRNA molecules. A commercially available nCounter™ Analysis System (NanoString, Seattle, Wash.) can be used in the procedure. However, it will be understood by skilled practitioners that other systems may be used in the process. For example, rather than bar codes the probes can be labeled with quantum dots; see, e.g., Sapsford et al., “Biosensing with luminescent semiconductor quantum dots.” Sensors 6(8): 925-953 (2006); Stavis et al., “Single molecule studies of quantum dot conjugates in a submicrometer fluidic channel.” Lab on a Chip 5(3): 337-343 (2005); and Liang et al., “An oligonucleotide microarray for microRNA expression analysis based on labeling RNA with quantum dot and nanogold probe.” Nucleic Acids Research 33(2): e17 (2005).

In some embodiments, microfluidic (e.g., “lab-on-a-chip”) devices can be used in the present methods for detection and quantification of mRNA in a sample. Such devices have been successfully used for microfluidic flow cytometry, continuous size-based separation, and chromatographic separation. In particular, such devices can be used for the detection of specific target mRNA in crude samples as described herein. A variety of approaches may be used to detect changes in levels of specific mRNAs. Accordingly, such microfluidic chip technology may be used in diagnostic and prognostic devices for use in the methods described herein. For examples, see, e.g., Stavis et al., Lab on a Chip 5(3): 337-343 (2005); Hong et al., Nat. Biotechnol. 22(4):435-439 (2004); Wang et al., Biosensors and Bioelectronics 22(5): 582-588 (2006); Carlo et al., Lab on a Chip 3(4):287-291 (2003); Lion et al., Electrophoresis 24 21 3533-3562 (2003); Fortier et al., Anal. Chem., 77(6):1631-1640 (2005); U.S. Patent Publication No. 2009/0082552; and U.S. Pat. No. 7,611,834. Also included in the present application are microfluidic devices comprising binding moieties, e.g., antibodies or antigen-binding fragments thereof that bind specifically to the pathogens as described herein.

These microfluidic devices can incorporate laser excitation of labeled quantum dots and other reporter molecules. The devices can also incorporate the detection of the resulting emission through a variety of detection mechanisms including visible light and a variety of digital imaging sensor methods including charge-coupled device based cameras. These devices can also incorporate image processing and analysis capabilities to translate the resulting raw signals and data into diagnostic information.

### Rapid, Phenotypic Diagnosis of Pathogen Identity and Pathogen Drug Resistance Using Expression Signatures

This technology can be applied to obtain a rapid determination of identity or drug resistance of a pathogen.

A pathogen can be identified in a sample based on detection of unique genes. Thus, for example, a sputum sample may be obtained from a subject who has symptoms associated with a respiratory disease such as pneumonia or bronchitis, and an assay is performed to determine which disease is present and what pathogen is the cause of that disease (see, e.g., Table 1). Urine samples may be obtained to diagnose cystitis, pyelonephritis, or prostatitis (see, e.g., Table 1). A skilled practitioner will appreciate that a particular type of sample can be obtained and assayed depending on the nature of the symptoms exhibited by the patient and the differential diagnosis thereof. Specific genes for identifying each organism can be identified by methods described herein; exemplary genes for identifying certain pathogens are included in Table 2.

The principle for greatly accelerated resistance testing is based on detecting the differences in transcriptional response that occur between drug sensitive and resistant strains of a pathogen upon exposure to a particular drug of interest. These transcriptional profiles are the earliest phenotypic response to drug exposure that can be measured and they can be detected long before bacillary death upon drug exposure. This transcription-based approach also carries the distinct advantage over genotype-based approaches in that it measures direct response of the pathogen to drug exposure rather than a surrogate SNP.

In some embodiments, the test can be performed as described in FIG. 2. A sample, e.g., a sputum sample from a patient with TB, is partitioned into several smaller sub-samples. The different sub-samples are exposed to either no drug or different, known or potential drugs (e.g., in the case of a TB sample, isoniazid, rifampin, ethambutol, moxifloxacin, streptomycin) for a determined period of time (e.g., less than four hours, less than three hours, less than two hours, less than one hour, less than 30 minutes, less than 20 minutes, less than 10 minutes, less than five minutes, less than two minutes, less than one minute), during which an expression profile is induced in drug sensitive strains that distinguishes it from drug resistant strains. The TB bacilli in the sub-samples are then lysed, the bar-coded molecular probes added for hybridization, and the sub-samples analyzed after immobilization and imaging. The set of transcriptional data is then analyzed to determine resistance to a panel of drugs based on expression responses for drug sensitive and drug resistant strains of TB. Thus, an expression signature to uniquely identify TB and its response to individual antibiotics can be determined, a probe set for the application of digital gene expression created, and sample processing and collection methods optimized.

Two issues that should be taken into account in defining the expression signatures and optimizing the transcriptional signal are: 1. the currently undefined metabolic state of the bacilli in sputum since the cells may be in either a replicating or non-replicating state, and 2. the possibility that the TB bacilli in collected sputum have been pre-exposed to antibiotics (i.e., the patient has already been treated empirically with antibiotics).

In some embodiments, the methods of identifying a pathogen and the methods of determining drug sensitivity are performed concurrently, e.g., on the same sample, in the same microarray or microfluidic device, or subsequently, e.g., once the identity of the pathogen has been determined, the appropriate assay for drug sensitivity is selected and performed.

An exemplary set of genes and probes useful in the methods described herein is provided in Table 2 submitted herewith.

### Methods of Treatment

The methods described herein include, without limitation, methods for the treatment of disorders, e.g., disorders listed in Table 1. Generally, the methods include using a method described herein to identify a pathogen in a sample from a subject, or identify a drug (or drugs) to which a pathogen in a subject is sensitive, and administering a therapeutically effective amount of therapeutic compound that neutralizes the pathogen to a subject who is in need of, or who has been determined to be in need of, such treatment. As used in this context, to “treat” means to ameliorate at least one symptom of the disorder associated with one of the disorders listed in Table 1. For example, the methods include the treatment of TB, which often results in a cough, chest pain, fever, fatigue, unintended weight loss, loss of appetite, chills and night sweats, thus, a treatment can result in a reduction of these symptoms. Clinical symptoms of the other diseases are well known in the art.

An “effective amount” is an amount sufficient to effect beneficial or desired results. For example, a therapeutic amount is one that achieves the desired therapeutic effect. This amount can be the same or different from a prophylactically effective amount, which is an amount necessary to prevent onset of disease or disease symptoms. An effective amount can be administered in one or more administrations, applications or dosages. A therapeutically effective amount of a composition depends on the composition selected. The compositions can be administered from one or more times per day to one or more times per week, including once every other day. The compositions can also be administered from one or more times per month to one or more times per year. The skilled artisan will appreciate that certain factors may influence the dosage and timing required to effectively treat a subject, including but not limited to the severity of the disease or disorder, previous treatments, the general health and/or age of the subject, and other diseases present. Moreover, treatment of a subject with a therapeutically effective amount of the compositions described herein can include a single treatment or a series of treatments.

### Methods of Diagnosis

Included herein are methods for identifying a pathogen and/or determining its drug sensitivity. The methods include obtaining a sample from a subject, and evaluating the presence and/or drug sensitivity of a pathogen in the sample, and comparing the presence and/or drug sensitivity with one or more references, e.g., a level in an unaffected subject or a wild type pathogen. The presence and/or level of a mRNA can be evaluated using methods described herein and are known in the art, e.g., using quantitative immunoassay methods. In some embodiments, high throughput methods, e.g., gene chips as are known in the art (see, e.g., Ch. 12, Genomics, in Griffiths et al., Eds. Modern Genetic Analysis, 1999, W.H. Freeman and Company; Ekins and Chu, Trends in Biotechnology, 1999, 17:217-218; MacBeath and Schreiber, Science 2000, 289(5485):1760-1763; Simpson, Proteins and Proteomics: A Laboratory Manual, Cold Spring Harbor Laboratory Press; 2002; Hardiman, Microarrays Methods and Applications: Nuts & Bolts, DNA Press, 2003), can be used to detect the presence and/or level of mRNA.

In some embodiments, the sample includes biological fluids or solids, e.g., sputum, blood, urine, stool, joint fluid, cerebrospinal fluid, cervical/vaginal swab, biliary fluid, pleural fluid, peritoneal fluid, or pericardial fluid; or tissue biopsy samples, e.g., from a bone biopsy, liver biopsy, lung biopsy, brain biopsy, lymph node biopsy, esophageal biopsy, colonic biopsy, gastric biopsy, small bowel biopsy, myocardial biopsy, skin biopsy, and sinus biopsy. In some embodiments, once it has been determined that a person has a pathogen, e.g., a pathogen listed in Table 1, or has a drug-resistant pathogen, then a treatment, e.g., as known in the art or as described herein, can be administered.

### Kits

Also within the scope of the invention are kits comprising a probe that hybridizes with a region of gene as described herein and can be used to detect a pathogen described herein. The kit can include one or more other elements including: instructions for use; and other reagents, e.g., a label, or an agent useful for attaching a label to the probe. Instructions for use can include instructions for diagnostic applications of the probe for predicting response to treatment in a method described herein. Other instructions can include instructions for attaching a label to the probe, instructions for performing analysis with the probe, and/or instructions for obtaining a sample to be analyzed from a subject. As discussed above, the kit can include a label, e.g., a fluorophore, biotin, digoxygenin, and radioactive isotopes such as 32P and 3H. In some embodiments, the kit includes a labeled probe that hybridizes to a region of gene as described herein, e.g., a labeled probe as described herein.

The kit can also include one or more additional probes that hybridize to the same gene or another gene or portion thereof that is associated with a pathogen. A kit that includes additional probes can further include labels, e.g., one or more of the same or different labels for the probes. In other embodiments, the additional probe or probes provided with the kit can be a labeled probe or probes. When the kit further includes one or more additional probe or probes, the kit can further provide instructions for the use of the additional probe or probes.

Kits for use in self-testing can also be provided. For example, such test kits can include devices and instructions that a subject can use to obtain a sample, e.g., of sputum, buccal cells, or blood, without the aid of a health care provider. For example, buccal cells can be obtained using a buccal swab or brush, or using mouthwash.

Kits as provided herein can also include a mailer, e.g., a postage paid envelope or mailing pack, that can be used to return the sample for analysis, e.g., to a laboratory. The kit can include one or more containers for the sample, or the sample can be in a standard blood collection vial. The kit can also include one or more of an informed consent form, a test requisition form, and instructions on how to use the kit in a method described herein. Methods for using such kits are also included herein. One or more of the forms, e.g., the test requisition form, and the container holding the sample, can be coded, e.g., with a bar code, for identifying the subject who provided the sample.

In some embodiments, the kits can include one or more reagents for processing a sample. For example, a kit can include reagents for isolating mRNA from a sample. The kits can also, optionally, contain one or more reagents for detectably-labeling an mRNA or mRNA amplicon, which reagents can include, e.g., an enzyme such as a Klenow fragment of DNA polymerase, T4 polynucleotide kinase, one or more detectably-labeled dNTPs, or detectably-labeled gamma phosphate ATP (e.g., 33P-ATP).

In some embodiments, the kits can include a software package for analyzing the results of, e.g., a microarray analysis or expression profile.

The invention is further described in the following examples, which do not limit the scope of the invention described in the claims.

## EXAMPLES

### Example 1

**Pathogen Identification**

Escherichia coli, Pseudomonas aeruginosa, Klebsiella pneumoniae, and Staphylococcus aureus.

Unique coding sequences in Escherichia coli, Pseudomonas aeruginosa, Klebsiella pneumoniae, Staphylococcus aureus, and Enterococcus faecalis were identified (Table 2) and used to positively identify these organisms (FIGS. 3-6). Clinical isolates were grown in LB media at 37° C. to log phase. Five microliters of each culture were then added to 100 microliters of guanidinium isothiocyanate lysis buffer (RLT buffer, Qiagen) and vortexed for 5 seconds. Four microliters of each lysate preparation were then used in the nCounter™ System assay according to the manufacturer's standard protocol for lysates. Criteria for identification were counts for all five (for P. aeruginosa or K. pneumoniae) or six (for E. coli) organism identification probes at least two-fold above the average background (the average of counts for all organism identification probes for the other two organisms). To compare between replicates, counts were normalized to counts of proC. Using the organism identification probes described in Table 2, four E. coli clinical isolates were correctly identified using probes designed to six E. coli genes (ftsQ, murC, opgG, putP, secA, and uup) (FIG. 3). Two clinical isolates were correctly identified as P. aeruginosa using probes designed to five P. aeruginosa genes (proA, sltB1, nadD, dacC, and lipB) as shown in FIG. 4. As shown in FIG. 5, probes designed to five K. pneumoniae genes (lrp, ycbK, clpS, ihfB, and mraW) positively identified a K. pneumoniae clinical isolate. Using probes designed to three S. aureus genes (proC, rpoB, and fabD), four clinical isolates were positively identified (FIG. 6). Cut-off criteria for identification were that counts for rpoB and fabD are at least two-fold above the average background (the average of counts for all organism identification probes for E. coli, P. aeruginosa, and K. pneumoniae).

On average, 4-5 sequences for each organism were included in the larger pool to obtain desired levels of specificity. Using this technology, each of these three organisms were detected, identified, and distinguished in axenic culture and in a complex mixture including eight additional Gram-negative pathogens by directly probing crude lysates (FIGS. 22A and 22B).

TB. Probes to Rv1641.1 and Rv3583c.1 detect highly abundant transcripts in M. tuberculosis (reference 8) and will detect orthologous transcripts in M. avium, and M. avium subsp. paratuberculosis, thus can be used for detection of any of these three species. Further, probes to three TB genes (Rv1980c.1, Rv1398c.1, and Rv2031c.1) can be used to differentially identify M. tuberculosis, i.e., they will not detect M. avium or M. avium subsp. paratuberculosis. Probes to MAP—2121c.1, MAV—3252.1, MAV—3239.1, and MAV—1600.1 can be used to detect M. avium or M. avium subsp. paratuberculosis, but will not detect M. tuberculosis. Thus, maximum sensitivity is achieved with the Rv1980c and Rv3853 probes, while the probes to Rv1980c.1, Rv1398c.1, and Rv2031c.1, and MAP—2121c.1, MAV—3252.1, MAV—3239.1, and MAV—1600.1, enable the distinction between M. tuberculosis infection and M. avium or M. avium subsp. paratuberculosis infection.

Probes were designed to genes both conserved throughout the mycobacterium genus and specific only to Mycobacterium tuberculosis. The pan-mycobacterial probes recognized multiple species, while the M. tuberculosis probes were highly specific (FIG. 22C).

Staphylococcus aureus and Stenotrophomonas maltophilia

Using the organism identification probes described Table 2, three S. aureus isolates were correctly identified using probes designed to five S. aureus genes (ileS, ppnK, pyrB, rocD, and uvrC) (FIG. 30). Similarly, three Stenotrophomonas maltophilia isolates were correctly identified using probes designed to six S. maltophilia genes (clpP, dnaK, purC, purF, sdhA, and secD) (Table2; and FIG. 31).

### Example 2

**Sensitivity of the Methods**

As shown in FIGS. 7-10, the present methods are specific for each pathogen of interest and sensitive to detect less than 100 cells in clinical samples, e.g., blood and urine.

RNA isolated from each of the three pathogens (1 ng) was probed with a 24 gene probe set (FIG. 7). E. coli genes, left; K. pneumoniae genes, middle; and P. aeruginosa genes, right. E. coli RNA, top. K. pneumoniae, middle; and P. aeruginosa, bottom. The y-axis shows number of counts for each gene as detected by using digital gene expression technology. RNA from each of the organisms shows distinct expression signatures that allow facile identification of each of the pathogens.

This 24 gene probe set was used to probe crude E. coli lysates from 10,000 cells, 1000 cells, and 100 cells (FIG. 8). The distinct E. coli expression signature could be distinguished for down to 100 cells.

Clinical samples were simulated in spiked urine and blood samples. In the spiked urine sample, a urine sample was spiked with 105 E. coli bacteria/mL of urine. The sample was refrigerated overnight at 4° C. and then the crude bacterial sample was lysed and probed with the 24-gene probe set used for the Gram negative bacteria to identify E. coli (FIGS. 9A, top panel, and 9B). Blood was spiked with 1000 cfu/ml and also detected with the 24-gene probe set (FIG. 9A, bottom panel).

Two clinical isolates of P. aeruginosa (obtained from Brigham and Women's clinical microbiology lab) were probed with the 24-gene probe set used for the Gram negative bacteria to demonstrate that the gene set is able to identify clinical diverse strains of the same bacterial genus (FIG. 10).

Identification of Escherichia coli Directly in Urine Samples.

E. coli strain K12 was grown in LB media at 37° C. to late log phase culture. Bacteria were then added to urine specimens from healthy donors to a final concentration of 100,000 cfu/ml (as estimated by OD600). Urine samples were then left at room temperature for 0 hours, 4 hours, 24 hours, or 48 hours or placed at 4° C. for 24 hours. 1 ml of spiked urine was centrifuged at 13,000×g for 1 minute. The supernatant was removed; pellets were resuspended in 100 microliters of LB media. Bacteria were treated with Bacteria RNase Protect (Qiagen), and then lysed in guianidinium isothiocyanate lysis buffer (RLT buffer, Qiagen). Lysates were used in the nCounter™ System assays per manufacturer's protocol.

Aliquots of patient urine specimens were directly assayed to detect E. coli transcripts in urinary tract infections (FIG. 22D). To condense the signals from multiple transcripts into a single metric that assesses the presence or absence of an organism, the raw counts from each probe were log transformed and summed. When applied to a set of 17 clinical E. coli isolates, every isolate was easily differentiated from a set of 13 non-E. coli samples (Z score>6.5 relative to non-E. coli controls, FIG. 22E).

### Example 3

**Drug Sensitivity of a Pathogen**

Identification of Fluoroquinolone and Aminoglycoside Resistance in Escherichia coli.

Using published expression array data for E. coli upon exposure to fluoroquinolones and aminoglycosides (Sangurdekar D P, Srienc F, Khodursky A B. A classification based framework for quantitative description of large-scale microarray data. Genome Biol 2006; 7(4):R32) sets of genes expected to be significantly down- or up-regulated upon exposure to fluoroquinolones and aminoglycosides were chosen. The pan-sensitive lab strain (K12), fluoroquinolone-resistant clinical isolates 1 and 2, and gentamicin-resistant clinical isolates (E2729181 and EB894940) were grown in LB media to log phase at 37° C. A 2 ml aliquot of each culture was taken, and antibiotics were added to those aliquots at a final concentration of 8 μg/ml ciprofloxacin or 128 μg/ml gentamicin. Cultures were incubated at 37° C. for 10 minutes. Five microliters of each culture was added to 100 microliters of guanidinium isothiocyanate lysis buffer and vortexed for 5 seconds. Lysates were used in the nCounter™ System assays per manufacturer's protocol. Counts were normalized to counts of proC; again proC appeared to be most comparable between experiments; fold induction for each gene was determined by comparing counts in the presence and absence of antibiotic exposure. There were clear signals from 9 probes (carA, deoC, flgF, htrL, recA, uvrA, ybhK, uup, and fabD) that show induction or repression in the drug sensitive K12 strain that distinguishes it from the two resistant clinical isolates (FIG. 11). A tenth probe, wbbK, was neither induced nor repressed, offering a useful comparison for genes with changes expression. Similarly, probes to eight genes show that these genes are either repressed (flgF, cysD, glnA, opgG) induced (ftsQ, b1649, recA, dinD) in the drug sensitive K12 strain that distinguishes it from the two resistant clinical isolates (FIG. 12)

Identification of Methicillin Resistance in Staphylococcus.

Six S. aureus clinical isolates were grown to log phase at 37° C. in LB media. A 2 ml aliquot of each culture was then taken; cloxacillin was added to a final concentration of 25 μg/ml. Cultures were incubated at 37° C. for 30 minutes. Five microliters of each culture was added to 100 microliters of guanidinium isothiocyanate lysis buffer and vortexed for 5 seconds. Lysates were used in the nCounter™ System assays per manufacturer's protocol. Using two independent probes (Table 2), expression of mecA was identified in the four isolates known to be methicillin-resistant. In contrast, there was no detectable mecA expression in the two isolates known to be methicillin-sensitive and minimal mecA expression in the absence of cloxacillin (FIG. 13).

Identification of Vancomycin Resistance in Enterococcus.

Four Enterococcus clinical isolates were grown in LB media to log phase at 37° C. A 2 ml aliquots were taken; vancomycin was added to a final concentration of 128 μg/ml. Cultures were incubated at 37° C. for 30 minutes. Five microliters of each culture was added to 100 microliters of guanidinium isothiocyanate lysis buffer and vortexed for 5 seconds. Lysates were used in the nCounter™ System assays per manufacturer's protocol. Using two independent probes (Table 2), expression of vanA was identified in the two isolates known to be vancomycin resistant. In contrast, there was no detectable vanA expression in the two isolates known to be vancomycin sensitive and minimal expression of vanA in the absence of vancomycin (FIG. 14). There was no detectable vanB expression in any of the four isolates.

Beyond the detection of transcripts for organism identification, detection of genes encoded on mobile genetic elements can provide greater genomic detail about a particular isolate. For example, bacterial isolates were probed for mecA mRNA, which confers resistance to methicillin in Staphylococci, and vanA mRNA, which confers resistance to vancomycin in Enterococci. In both cases, relevant transcripts were detected that allowed for rapid identification of MRSA and vancomycin-resistant Enterococcus (VRE) (FIG. 22F). Thus, direct detection of RNA is able to detect known resistance elements. In addition, this approach is able to discriminate isolates by other genetic factors, such as virulence factors acquired through horizontal genetic exchange in food-borne pathogens, i.e., Shiga toxin in Enterohemorrhagic or Shigatoxigenic E. coli.

Identification of Drug Resistance in TB.

A 24 gene probe set was identified from published gene expression data to identify an expression signature that would allow identification of expression changes of drug sensitive TB upon exposure to different antibiotics, including isoniaid, rifampin, streptomycin, and fluoroquinolones (FIGS. 15-18). The magnitude of induction or repression after drug exposure is shown in Table 3.

Log phase M. tuberculosis cells at A600 0.3 were grown in inkwell bottles (10 ml volume, parallel cultures) in the presence of one of four different drugs (isoniazid, 0.4 μg/ml; streptomycin, 2 μg/ml; ofloxacin, 5 μg/ml; rifampicin 0.5 μg/ml). At the indicated time after the initiation of drug treatment (FIG. 15), cultures were harvested by centrifugation (3000×g, 5 minutes), resuspended in 1 ml Trizol, and bead beaten (100 nm glass beads, max speed, two one-minute pulses). Chloroform (0.2 ml) was added to the samples, and following a five minute centrifugation at 6000×g, the aqueous phase was collected for analysis.

Samples were diluted 1:10 and analyzed using NanoString™ probeset described in Table 2 per the manufacturer's protocol. The relative abundance of each transcript is first calculated by normalizing to the average counts of three housekeeping genes (sigA, rpoB, and mpt64), and then the data is plotted as a fold change relative to samples from untreated controls. The boxes indicate probes that were selected based on previous evidence of drug-specific induction (Boshoff et al., J Biol. Chem. 2004, 279(38):40174-84.)

The drug resistant TB strain shows no expression signature induction upon exposure to isoniazid, in contrast to a drug sensitive strain, which clearly shows induction of an expression signature upon isoniazid exposure (FIG. 16). Three scatter plots comparing isoniazid sensitive and resistant TB strains are shown in FIG. 16, with each dot representing one of the 24 gene probes. The axes report number of transcripts as measured by digital gene expression technology (NanoString™). Left—Comparison of expression in isoniazid resistant and isoniazid sensitive strains in the absence of drug treatment. Middle—Comparison of expression in drug treated vs. drug untreated isoniazid sensitive strain. Right—Comparison of expression in drug treated vs. drug untreated isoniazid resistant strain.

Different sets of genes are induced in drug-sensitive M. tuberculosis depending on the drug as seen in FIG. 17. The transcriptional responses of drug-sensitive and drug-resistant M. tuberculosis (A) Strain A50 (INH-R) treated with INH (0.4 μg/ml) as described herein. (B) The SM-R clone S10 was treated with 2 μg/ml streptomycin. Differential gene induction can be measured by digital gene expression of the TB 24 gene probe set to reveal a clear signature and allow identification of drug sensitivity (FIG. 18).

Three housekeeping genes, mpt64, rpoB, and sigA, were used for normalization. For each experimental sample, the raw counts for the experimental genes were normalized to the average of the raw counts of these three housekeeping genes, providing a measure of the abundance of the test genes relative to the control genes. Induction or repression is defined as a change in these normalized counts in drug-exposed samples as compared to non-drug-exposed samples. Using this methodology, the following genes were found to be induced or repressed in drug-sensitive TB after exposure to isoniazid, rifampin, fluoroquinolones, and streptomycin.

Isoniazid: For drug-dependent induction: kasA, fadD32, accD6, efpA, and Rv3675.1.

Rifampin: For drug-dependent induction: bioD, hisI, era, and Rv2296.

Fluoroquinolones: For drug-dependent induction: rpsR, alkA, recA, ltp1, and lhr; for drug-dependent repression: kasA and accD6.

Streptomycin: For drug-dependent induction: CHP, bcpB, gcvB, and groEL.

### Example 4

**A Phenotypic Expression Signature-Based Test to Identify Drug Sensitive and Resistant TB Using Digital Gene Expression with Molecular Barcodes**

This example describes a phenotypic expression-signature-based test for the diagnosis of TB in sputum and rapid determination of resistance profile. The method is based on detection of genes whose expression profiles will uniquely detect TB and distinguish drug resistant and sensitive strains, with the creation of a probe set of bar-coded, paired molecular probes. The choice of genes was determined through bioinformatic analysis of expression profile data obtained using microarrays under a variety of growth conditions, including TB in axenic culture (both replicating and non-replicating states), TB in cell cultured macrophages, and TB spiked in sputum.

A. Define Signature for Identification of TB

A set of molecular probes have been identified that will specifically hybridize to mRNA from both replicating and non-replicating TB. The probes are specific for mRNA that is highly abundant under all growth conditions and is conserved across all TB strains. While unique DNA sequences have been previously defined to identify TB recognizing 16S rRNA (Amplicor, Roche) or the IS6110 region (Gen-probe), these defined regions do not have the optimal characteristics required for signatures in digital gene expression. The 16S rRNA is not sufficiently divergent among mycobacterial species that could distinguish between the different species using 50-base oligomer gene probes, which can tolerate low levels of genetic variability due to their length. The IS6110 region of the genome is not expressed at high enough levels under all growth conditions that would allow it to be used it as a robust signal to identify TB. Thus, an expression signature that will allow identification of TB from other mycobacterial species is described.

i. Bioinformatic Gene Analysis for Conserved TB Genes.

Unique expression signatures for the detection of TB over other mycobacteria species have been defined. In general, the optimal genes for inclusion in a signature will fulfill the criteria of 1. having high expression levels (high mRNA copy number) to increase sensitivity, 2. being highly conserved across all TB strains as well as having highly conserved sequence, and 3. being highly specific for TB genome over all other mycobacteria species. Such genes were identified using a bioinformatic analysis of conserved genes in the available TB genomes that are not present in all other sequenced mycobacteria species (i.e., M. marinum, M. avium-intracellulaire, M. kansaii, M. fortuitum, M. abscessus). Over 40 TB genomes from clinically isolated strains that have been sequenced at the Broad Institute are available for analysis.

ii. Expression Profile Analysis of mRNA Levels of Candidate Genes.

A second criterion for selection of molecular probes for the detection of TB bacilli in sputum is that they hybridize to highly abundant, stable mRNAs to allow maximum sensitivity. Such mRNAs are anticipated to correspond to essential housekeeping genes. Genes have been selected using a combination of bioinformatic analysis of existing, publicly available expression data in a database created at the Broad Institute and Stanford University (tbdb.org) and experimental expression profiles on TB strain H37Rv using expression profiling to confirm a high level of expression of candidate genes under conditions permissive for replication (logarithmic growth) and non-replication induced by carbon starvation, stationary phase, and hypoxia. Expression profiling experiments on H37Rv are performed using a carbon starvation model of TB that has been established (starvation for 5 weeks in 7H9/tyloxapol), stationary phase growth, and the Wayne model for anaerobic growth (slowly agitated cultures in sealed tubes). Solexa/Illumina sequencing is used to determine expression profiles by converting mRNA to cDNA and using sequencing to count cDNA molecules. This quantitative method for identifying expression levels is more likely to reflect levels obtained using digital gene expression than microarray data and is a method that has been established with the Broad Institute Sequencing Platform. It is possible to multiplex 12 samples per sequencing lane given 75 bp reads and 10 million reads per lane.

iii. Probe Selection of Expression Signature Identifying TB.

Because the digital gene expression technology is based on the hybridization of two 50 nucleotide probes to the mRNA of interest, two 50 base pair regions in the genes are identified from (Ai) and (Aii) that are unique within the genome to minimize non-specific hybridization and that contain minimal polymorphisms as evidenced from sequenced TB genomes. The probes are selected bioinformatically to fit within a 5 degree melting temperature window and with minimal mRNA secondary structure. The probes are tested against mRNA isolated from replicating and non-replicating TB (including multiple strains i.e., H37Rv, CDC1551, F11, Erdman), M. marinum, M. avium-intracellulaire, M. kansaii, and M. fortuitum to confirm the specificity of the entire probe set using available technology. Probes may be selected for these other mycobacterial species, which will allow for identification of these pathogens from sputum as well. The ability to identify intracellular bacilli is tested in a macrophage model of infection, to demonstrate the ability to detect TB mRNA in the presence of host mRNA. Finally, the sensitivity of the assay was determined by titrating down the number of TB bacilli (and thus mRNA present in cell lysates) in the sample tested. All experiments using digital gene expression is confirmed using quantitative RT-PCR against the same gene set. Improvement and refinement of the set will occur in an iterative manner.

B. Define Signature to Distinguish Sensitive and Resistant TB

A set of molecular probes that hybridizes to mRNAs that are specifically induced upon exposure to each individual TB drug has been identified, allowing a profile to be obtained that distinguishes drug sensitive and resistant strains. Signatures have been determined for exposure to isoniazid, rifampin, ethambutol, streptomycin, and moxifloxacin.

In addition to the above characteristics for ideal genes to be included in the signature (i.e., conserved across TB strains, specific for TB genome), several other characteristics are prioritized in gene selection for signatures of drug resistance. Because drug resistance will be determined by the difference between transcript induction in drug sensitive and drug resistant strains, ideal gene candidates will be highly induced in drug sensitive strains upon exposure to a given drug. Ideally, these genes are induced early and quickly, as this time period will determine to a large extent, the rapidity of the overall diagnostic test. Based on data using qRT-PCR, a transcriptional response to drug exposure is observed in as little as 1-2 hours (FIG. 19). Given the half-lives of mRNA molecules, exploiting gene induction rather than gene repression provides a more rapid and detectable response.

For all the described experiments involving isoniazid and streptomycin, TB strain H37Rv was used in a BSL3 setting in which a set of singly resistant strains has been generated to be used to compare to the wild-type, fully drug sensitive H37Rv. To ensure that rifampin remains a treatment option in the unlikely event of a laboratory-acquired infection, rifampin resistant mutants will be generated in an auxotrophic strain of TB (lysA, panC) that requires the addition of lysine and pantothenate for growth.

Finally, signatures that are unique to each antibiotic have been identified rather than a general stress response to any or all antibiotics. The rationale for this specificity is that in a clinical setting, many patients will have already been empirically treated with different antibiotics and thus some general stress response may have already been activated in the bacilli within a sputum sample. However, drug specific responses are preserved for testing and analysis.

i. Expression Profiling in Response to Antibiotic Exposure.

Expression profiling to identify candidate genes that distinguish transcriptional responses in drug sensitive and resistant strains of H37Rv has been performed. Because the replication state or transcriptional activity of the bacilli in sputum is unknown, additional experiments are performed on non-replicating (induced through a 5 week carbon starvation model) bacilli. It will determined if the non-replicating bacilli require a short period (t1) of “growth stimulation” in rich media (7H9/OADC) in order to stimulate some basal transcription that can then be responsive to drug exposure (FIG. 20). The optimal period of time (t2) that is required for drug exposure in order to obtain robust signature to distinguish drug sensitive and resistant strains and the optimal drug concentration is also determined to obtain a robust, reproducible response. These experiments will be performed for each of the individual antibiotics.

A completely non-replicating state is the “worst case scenario” (i.e., the longest period that would be required) if bacilli in sputum is in a non-replicating, dormant state. In fact, based on published work examining expression profiles from bacilli in patient sputum, this period will be extremely short if necessary at all, given that expression profiles were obtained directly from sputum bacilli. (Of note, this published data will also be incorporated into the analysis to provide initial insight into possible gene candidates in bacteria in sputum.) A matrix of profiling experiments are performed, varying the time of exposure to rich 7H9/OADC media (t1) from 0, 1, and 2 hours; for each t1, and the time of exposure to each antibiotic (t2). For each set of t1 and t2, the antibiotic concentration is varied from 1×, 3×, and 5× the minimum inhibitory concentration (MIC) for each antibiotic, for both sensitive and resistant H37Rv strains to determine the optimal parameters. Expression profiling will be used to identify optimal conditions for producing robust, reproducible profiles.

Based on the optimized conditions (t1 and t2), expression profiling is performed on drug sensitive and resistant H37Rv strains under these conditions. Bioinformatic analysis is performed to identify genes for each drug in which the level of induction is high in drug sensitive strains relative to drug resistant strains (with the exception of rifampin in which the level of repression is high in drug sensitive strains relative to drug resistant strains). The levels of expression will be compared between drug sensitive and drug resistant strains and confirmed by quantitative RT-PCR.

ii. Develop Analysis Algorithm to Identify Drug Resistance.

An optimal algorithm is determined to analyze expression ratios for sets of genes that distinguish sensitive and resistant strains (as defined by standard MIC measurements). One of the strengths of this method is that for the majority of cases (i.e., those cases which have not been pre-exposed to TB antibiotics), a comparison can be done between the gene expression levels of the same strain not exposed and exposed to a given antibiotic. Quantitative RT-PCR is used to measure mRNA levels from H37Rv under conditions that include 1. exposure to no antibiotic, 2. exposure to isoniazid, 3. exposure to rifampin, 4. exposure to ethambutol, 5. exposure to streptomycin, and 6. exposure to moxifloxacin. The level of expression from a given gene after exposure to antibiotic will be normalized to the level of expression from a set of steady-state, housekeeping genes (i.e., sigA, which encodes the principal sigma factor that stimulates the transcription of housekeeping genes, and rpoB, which encodes a synthetic subunit of RNA polymerase) and compared to the same normalized level of expression of the same gene in the absence of antibiotic exposure. Comparisons will also be made to standard sensitive and resistant control strains (FIG. 21). Ideally, exposure to a particular drug will induce gene expression in drug sensitive strains to high levels, A>>B while for drug resistant strains, which are insensitive to the drug exposure, A=B. (The exception will be for rifampin, in which gene repression of the mRNAs with shortest half-life is detected, given the mechanism of rifampin, i.e., A=C<<B=D.) Because of the large dynamic range of transcription levels, genes are selected for which the ratio of C/D is maximal, thus allowing for clear robust differentiation of sensitive and resistant strains. In addition, optimal, unique set of genes have been selected for each individual antibiotic so that there is no overlap in induced responses with other antibiotics.

iii. Analysis of Impact of Pre-Antibiotic Exposure on TB Bacilli Signatures.

To determine the efficacy of these signatures to identify resistance patterns even in the event that a patient has been pre-treated with antibiotics, drug sensitive and resistant TB bacilli (replicating, non-replicating, within macrophages) are pre-expose to amoxicillin, cephalosporins, trimethoprim-sulfamethoxazole, and erythromycin which are common antibiotics to which a patient may be exposed in TB endemic settings, prior to application of this test. Pre-exposure of TB bacilli to different combinations of current TB drugs is also performed to determine if such pre-exposure also interferes with the transcriptional response our ability to detect such a response. Unique signatures should be preserved, thus not impairing our ability to determine resistance. Gene expression levels of a set of genes of interest will be determined using quantitative RT-PCR.

iv. Probe Selection of Expression Signature to Identify Resistance Profile.

Based on the data obtained in Sub-aim Bi, a set of candidate genes have been selected that will create a signature for transcriptional response to antibiotic exposure. Two 50 base-pair regions for each gene are selected within regions that are highly conserved across TB genomes. The probes are selected bioinformatically to fit within a 5 degree melting temperature window and with minimal mRNA secondary structure. These probes will be used to compare drug sensitive and resistant strains using available technology under conditions described above, including bacilli in axenic culture that are initially replicating or nonreplicating, intracellular bacilli in a cell culture macrophage infection model that we have currently in our laboratory, and bacteria pre-exposed to different antibiotic combinations. All results will be compared to data obtained by quantitative RT-PCR. Improvement and refinement of the set will occur in an iterative manner.

C. Optimization of Sample Processing for Digital Gene Expression with Molecular Bar Codes.

In addition to defining probe sets for identification of expression signatures, the second major challenge is to optimize processing of samples in order to measure digital gene expression from bacilli present within the sample. Because the majority of TB cases is pulmonary in origin and the majority of samples to be processed is patient sputum, processing of sputum samples to obtain mRNA measurements from infecting TB bacilli is optimized. A spiked sputum model is used in which sputum collected from healthy, uninfected patients (who have not been treated with antibiotics) was spiked with TB bacilli that are either in a replicating or non-replicating (carbon starved) state. Issues that will be addressed include dealing with the variable viscosity of sputum and efficiently lysing the TB bacilli within a sputum sample. One of the major advantages of digital gene expression is the ability to hybridize the mRNAs to their respective probes in extremely crude samples, including crude cell lysates, fixed tissue samples, cells in whole blood and urine, cells from crude lysates of ticks, and samples containing 400 mM guanidinium isothiocyanate (GITC), polyacrylamide, and trizol. Thus, initial indications suggest that no purification step will be required after lysing the bacteria within the sputum, as no purification has been required from whole blood, urine, or fixed tissue samples. The only requirement is sufficient mixing to allow contact between the probes and the mRNAs.

For these experiments, uninfected sputum is obtained from the Brigham and Women's Hospital (BWH) specimen bank. The specimen bank is an IRB regulated unit directed by Lyn Bry, M D, PhD of the BWH pathology department. Discarded sputum will be obtained after all processing is completed in the laboratory (generally within 12-24 hours of collection). Sputum is only collected from subjects who have not received any antibiotics in the previous 48 hours. All samples will be de-identified and no protected health information is collected. Based on the current load of the specimen lab, the necessary amount of sputum (25-50 mL) is obtained within a matter of weeks.

i. Sputum Processing.

Sputum samples vary in bacterial load, consistency, and viscosity. Several approaches are tested to maximize the rapidity with which the bacteria come into contact with bactericidal levels of antibiotic in media conditions and exposure to oligomer probes for hybridization. Several methods of processing sputum, including no processing, passage of sputum through a syringe needle, treatment with lysozyme and/or DNase, Sputalysin (Calbiochem; 0.1% DTT) which is standardly used to treat sputum from cystic fibrosis patients, or simple dilution of the sample into some minimal denaturant (i.e., GITC) are used. Sputum spiked with H37Rv and processed by a variety of methods to alter its viscosity are performed to determine if any of these methods interferes with the technology.

ii. Bacterial Lysis in Sputum Spiked with TB bacilli.

Several approaches to efficiently lyse bacterial cells, arrest transcription and enzyme-based mRNA degradation, and make mRNA accessible to the probes are used in the assay. Previous studies examining the transcriptional responses of bacteria in sputum have first added GTC or similar reagents to the samples to arrest the transcriptional response. Centrifugation can then be used to concentrate bacteria from sputum samples after GTC treatment. Lysis of mycobacteria is generally accomplished through physical means, i.e. homogenization with 0.1 ml glass or zirconium beads. Such physical means are explored to disrupt the bacteria within processed sputum to analyze bacilli that has been spiked into uninfected human sputum using the designed probe set from 1A to detect TB bacilli.

Alternative methods are used for lysis that may be more amenable to field-based considerations, including phage lysis. Addition of phage, or more optimally, purified phage lysin(s), may provide a low-cost, simple, and non-electrical option for bacterial lysis. The Fischetti lab (Rockefeller University) has recently demonstrated the rapid and thorough lysis of several Gram-positive species using purified bacteriophage lysins, which enzymatically hydrolyse peptidoglycan, leading to osmotic lysis. The Hatfull lab (University of Pittsburgh) is currently working to characterize the activity and optimize the performance of LysA enzymes from several lytic mycobacteriophages. In the absence of purified lysins, investigations are performed to determine whether high MOI-infection of TB with a lytic bacteriophage such as D29 can efficiently lyse TB in sputum. It is currently unclear how this approach will affect the transcriptional profile of the bacteria, since it will likely need to occur in the absence of denaturants that would impair the binding, entry, and subsequent lytic properties of the phage. The mycobacteriophage TM4 also expresses a structural protein, Tmp, with peptidoglycan hydrolase activity, which may allow it to be used as a rapid means of cell lysis at high MOI. Once lysed, the mRNA is stabilized with GITC, RNAlater, or other reagents that will inactivate endogenous RNAse activity.

### Example 5

**Bacterial and Fungal Culture**

E. coli, K. pneumoniae, P. aeruginosa, Providencia stuartii, P. mirabilis, S. marcescens, E. aerogenes, E. cloacae, M. morganii, K. oxytoca, C. freundii, or C. albicans were grown to an OD600 of ˜1 in Luria-Bertani medium (LB). For mixing experiments, equal numbers of bacteria as determined by OD600 were combined prior to lysis for NanoString™ analysis. Mycobacterium isolates were grown in Middlebrook 7H9 medium to mid-log phase prior to harvest or antibiotic exposure as described below.

Derivation of Resistant Laboratory Bacterial Strains:

E. coli laboratory strain J53 with defined fluoroquinolone-resistant chromosomal mutations in gyrA (gyrA1-G81D; gyrA2-S83L) were obtained from the Hooper lab, Massachusetts General Hospital, Boston, Mass. Plasma-mediated quinolone resistance determinants (oqxAB, qnrB, aac6-Ib) were purified from clinical isolates previously determined to contain these plasmids. E. coli parent strain J53 was transformed with these plasmids, and their presence was confirmed with PCR.

Viral and Plasmodium Infections:

HeLa cells (1×106), 293T cells (2×105), and human peripheral blood monocytes (5×105), were infected with HSV-1 strain KOS and HSV-1 strain 186 Syn+, influenza A P R8, or HIV-1 NL-ADA, respectively, at the noted MOIs. Primary red blood cells (5×109) were infected with P. falciparum strain 3D7 until they reached the noted levels of parasitemia. At the indicated times, the cells were washed once with PBS and harvested.

Antibiotic Exposure:

Cultures of E. coli or P. aeruginosa were grown to an OD600 of ˜1 in LB. Cultures were then divided into two samples, one of which was treated with antibiotic (E. coli for 10 minutes: ciprofloxacin 4-8 μg/ml or 300 ng/ml, gentamicin 64 or 128 μg/ml, or ampicillin 500 μg/ml; P. aeruginosa for 30 minutes:

ciprofloxacin 16 μg/ml). Both treated and untreated portions were maintained at 37° C. with shaking at 200 rpm. Cultures of S. aureus or E. faecium were grown to an OD600 of ˜1 in LB. Cultures were then exposed to cloxacillin (25 μg/mL) or vancomycin (128 μg/mL), respectively, for 30 minutes.

Cultures of M. tuberculosis were grown to mid-log phase then normalized to OD600 of 0.2. 2 ml of each culture were treated with either no antibiotic or one of the following (final concentration): isoniazid 0.2-1.0 μg/ml; streptomycin 5 μg/ml, rifampicin 0.5 μg/ml, or ciprofloxacin 5 μg/ml. The plates were sealed and incubated without shaking for 3 or 6 hours. Lysates were then made and analyzed as described above, using probes listed in Table 6.

Sample Processing:

For Gram negative isolates, 5-10 μl of culture was added directly to 100 μl RLT buffer and vortexed. For clinical specimens, 20 μl of urine from patients determined by a clinical laboratory to have E. coli urinary tract infection was added directly to 100 μl of RLT buffer. For mycobacteria, 1.5 ml of culture was centrifuged, then resuspended in Trizol (Gibco) with or without mechanical disruption by bead beating, and the initial aqueous phase was collected for analysis. Viral and parasite RNA were similarly prepared using Trizol and chloroform. For all lysates, 3-5 μl were used directly in hybridizations according to standard NanoString™ protocols. Raw counts were normalized to the mean of all probes for a sample, and fold induction for each gene was determined by comparing antibiotic-treated to untreated samples.

Selection of Organism Identification Probes:

To select NanoString™ probes for differential detection of organisms, all publically available sequenced genomes for relevant organisms were compared. Genes conserved within each species were identified by selecting coding sequences (CDS) having at least 50% identity over at least 70% of the CDS length for all sequenced genomes for that species. The CDS was broken into overlapping 50-mers and retained only those 50-mers perfectly conserved within a species and having no greater than 50% identity to a CDS in any other species in the study. Available published expression data in Gene Expression Omnibus was reviewed, and genes with good expression under most conditions were selected. To identify unique M. tuberculosis probes, published microarray data was used to identify highly expressed genes falling into one of two classes: those unique to the M. tuberculosis complex (>70% identity to any other gene in the non-redundant database using BLASTN and conserved across all available M. tuberculosis and M. bovis genomes), as well as those with >85% identity across a set of clinically relevant mycobacteria including M. tuberculosis, M. avium, and M. paratuberculosis. C. albicans probes were designed against 50-mer segments of C. albicans genome unique in comparison with the complete genomes of ten additional pathogenic organisms that were included in its probe set. Viral probes were designed against highly conserved genes within a virus (i.e. all HSV-2 or HIV-1 isolates) that were less conserved among viruses within the same family, (i.e between HSV-1 and HSV-2). Plasmodium falciparum probes were designed against genes expressed abundantly in each of the blood stages of the parasite life cycle. All probes were screened to avoid cross hybridization with human RNA.

Probe Sets:

For Gram-negative organism identification, a pooled probe-set containing probes for E. coli, K. pneumoniae, and P. aeruginosa were used. For mycobacterial organism identification, species-specific probes for M. tuberculosis and broader mycobacterial genus probes were among a larger set of probes against microbial pathogens.

Probes were designed for genes that are differentially regulated upon exposure to various antimicrobial agents to measure the presence or absence of a response (Sangurdekar et al., Genome Biology 7, R32 (2006); Anderson et al., Infect. Immun. 76, 1423-1433, (2008); Brazas and Hancock, Antimicrob. Agents Chemother. 49, 3222-3227, (2005)). Following 10-30 minute exposures of wild-type E. coli K-12 to ciprofloxacin, gentamicin, or ampicillin, the expected changes in transcript levels that together define the drug-susceptible expression signature for each antibiotic were observed (FIGS. 23A and 23B, Table 7). These signatures were not elicited in the corresponding resistant strains (FIGS. 23A and 23B).

Rapid phenotypic drug-susceptibility testing would make a particularly profound impact in tuberculosis, as established methods for phenotypic testing take weeks to months (Minion et al., Lancet Infect Dis 10, 688-698, (2010)). Expression signatures in response to anti-tubercular agents isoniazid, ciprofloxacin, and streptomycin were able to distinguish susceptible and resistant isolates after a 3 to 6 hour antibiotic exposure (FIG. 23C). Some genes in the transcriptional profiles are mechanism-specific (i.e., recA, alkA, and 1 hr for ciprofloxacin; groEL for streptomycin; and kasA and accD6 for isoniazid). Other genes, particularly those involved in mycolic acid synthesis or intermediary metabolism, are down-regulated in response to multiple antibiotics, indicating a shift away from growth towards damage control.

To condense these complex responses into a single, quantitative metric to distinguish susceptible and resistant strains, the metric of the mean-squared distance (MSD) of the expression response was utilized from each experimental sample from the centroid of control, antibiotic-susceptible samples. Antibiotic-susceptible strains cluster closely, thus possessing small MSDs. Conversely, antibiotic-resistant strains have larger values, the result of numerous genes failing to respond to antibiotic in a manner similar to the average susceptible strain. MSD is reported as dimensionless Z-scores, signifying the number of standard deviations a sample lies from the average of sensitive isolates of E. coli (FIGS. 24A, 24B, 27, and 28) or M. tuberculosis (FIGS. 24C and 29).

Because expression profiles reflect phenotype rather than genotype, resistance mediated by a variety of mechanisms can be measured using a single, integrated expression signature. The transcriptional responses of ciprofloxacin-susceptible E. coli strain J53 were compared with a series of isogenic mutants with different mechanisms of resistance: two with single mutations in the fluoroquinolone-target gene topoisomerase gyrA (G81D or S83L) and three carrying episomal quinolone resistance genes including aac(6)-Ib (an acetylating, inactivating enzyme), qnrB (which blocks the active site of gyrA), and oqxAB (an efflux pump). In comparison with the parent strain, all J53 derivatives had large Z-scores, reflective of resistance (FIG. 24B).

Response to isoniazid was compared in a series of sensitive clinical and laboratory isolates and two isoniazid resistant strains, including an H37Rv-derived laboratory strain carrying a mutation in katG (S315T), a catalase necessary for pro-drug activation, and a clinical isolate with a mutation in the promoter of inhA (C-15T), the target of isoniazid. Due to their disparate resistance mechanisms, these two strains have differing levels of resistance to isoniazid, with the katG mutant possessing high level resistance (>100-fold increase in minimal inhibitory concentration (MIC) to >6.4 μg/mL), while the inhA promoter mutation confers only an 8-fold increase in the MIC to 0.4 μg/mL. Exposure to low isoniazid concentrations (0.2 μg/mL) failed to elicit a transcriptional response in either resistant strain, but at higher isoniazid concentrations (1 μg/mL), the inhA mutant responds in a susceptible manner in contrast to the katG mutant (FIG. 24C). Thus, this method is not only mechanism-independent, but can also provide a relative measure to distinguish high and low-level resistance.

Finally, because RNA is almost universal in pathogens ranging from bacteria, viruses, fungi, to parasites, RNA detection can be integrated into a single diagnostic platform applicable across a broad range of infectious agents. Using a large pool of mixed pathogen probes, we were able to directly and specifically detect signals to identify the fungal pathogen Candida albicans (FIG. 25A); human immunodeficiency virus (HIV), influenza virus, and herpes simplex virus-2 (HSV-2) in cell culture in a dose dependent manner (FIGS. 25B-D); and the different stages of the Plasmodium falciparum life cycle in infected erythrocytes (FIG. 25E).

NanoString™ Data Analysis and Calculation of Distance Metric Mean Squared Distance for Drug-Sensitivity:

For all drug-treated samples, raw NanoString™ counts for each probe were first normalized to the mean of all relevant (i.e., species-appropriate) probes for each sample. Fold-change in transcript levels was determined by comparing the normalized counts for each probe in the antibiotic-treated samples with the corresponding counts in the untreated baseline sample for each test condition.

To transform qualitative expression signatures into a binary outcome of sensitive or resistant, an algorithm was developed to calculate mean squared distance (MSD) of a sample's transcriptional profile from that of sensitive strains exposed to the same drug. The MSD metric in drug-sensitivity experiments was calculated as follows:

1. Variation in sample amount is corrected for by normalizing raw values to the average number of counts for all relevant probes in a sample.

2. A panel of NanoString™ probes, which we denote Pj, is selected. The subscript j runs from 1 to Nprobes, the total number of selected probes. The analysis is restricted to probes that changed differentially between drug-sensitive and drug-resistant isolates.

3. Replicates of the drug-sensitive strain are defined as Nsamp. For each replicate, normalized counts for each probe Pj before or after drug treatment were denoted Ci,Pbefore or Ci,Pafter, with i signifying the sample index.

4. “Log induction ratio” is next computed:

Si,P≡ln [Ci,Pbefore/Ci,Pafter]

Log transforming the ratio in this way prevents any single probe from dominating the calculated MSD.

5. The average induction ratio of the drug sensitive samples, , is calculated by summing over the different biological replicates and normalizing by the number of samples:

\(\overset{\_}{S_{j}} = \frac{\text{?}}{\text{?}}\)
\(\text{?}\text{indicates text missing or illegible when filed}\)

6. MSD is next calculated for the each of the replicates of the drug sensitive strain (of index i), a number that reflects how different a sample is from the average behavior of all drug sensitive samples:

\(\text{?} = \frac{\text{?}\left( {S_{i,P_{j}} - \text{?}} \right)}{\text{?}}\)
\(\text{?}\text{indicates text missing or illegible when filed}\)

7. Induction ratios for resistant strains, Ri,P, are calculated similarly to those of sensitive strains:

Ri,P≡ln [Ci,Pbefore/Ci,Pafter]

8. The MSD for the drug resistant strains is calculated relative to the centroid of the drug-sensitive population:

\({MSD}_{j}^{R} = \frac{\text{?}\left( {R_{i,P_{j}} - \overset{\_}{S_{j}}} \right)^{2}}{\text{?}}\)
\(\text{?}\text{indicates text missing or illegible when filed}\)

Because most sensitive strains behave similarly to the average sensitive strain the typical value for MSDjS is small compared to the typical value for a resistant strain, MSDjS.

Finally, statistical significance of the measured MSD values were assigned. Because the MSDjS values are the sum of a number of random deviations from a mean, they closely resemble a normal distribution, a consequence of the Central Limit Theorem. Therefore, z-scores, which reflect the number of standard deviations away a given sample is relative to the drug sensitive population, were computed for each sample:

\(z_{i} = \frac{{MSD}_{i} = \text{?}}{\text{?}}\)
\(\text{?}\text{indicates text missing or illegible when filed}\)

where the standard deviations and means are defined as:

\(\text{?} = \sqrt{\frac{1}{\text{?}}\text{?}\left( {{MSD}_{i}^{S} - \text{?}} \right)^{2}}\)
\({and}\text{:}\)
\(\overset{\_}{{MSD}^{S}} = {\text{?}{MSD}_{i}^{S}}\)
\(\text{?}\text{indicates text missing or illegible when filed}\)

This metric was applied to the analysis of numerous laboratory and clinical isolates that were tested against different antibiotics and the data are shown in FIGS. 24, 27, 28, and 29.

Calculation of Distance Metric for Organism Identification:

To transform the information from multiple probes into a binary outcome, raw counts for each probe were log-transformed. Log transforming the ratio in this way prevents any single probe from dominating the analysis. These log-transformed counts were then averaged between technical replicates.

A panel of NanoString™ probes, which are denoted Pj, is selected as described. The subscript j runs from 1 to Nprobes, the total number of selected probes.

Si,P≡ln [Ci,P]

Because organism identification depends on an ability to detect transcripts relative to mocks or different organisms, background level of NanoString™ counts in samples prepared without the organism of interest was thus used to define a control centroid. The centroid of these control samples, , is calculated by summing over the different biological replicates and normalizing by the number of samples:

\(\overset{\_}{S_{j}} = \frac{\text{?}}{\text{?}}\)
\(\text{?}\text{indicates text missing or illegible when filed}\)

MSD is next calculated for the averaged technical replicates of the experimental samples (of index i), a number that reflects how different a sample is from the average behavior of all control samples:

\({MSD}_{i}^{S} = \frac{\text{?}}{N_{probes}}\)
\(\text{?}\text{indicates text missing or illegible when filed}\)

Finally, statistical significance was assigned to the measured MSD values. Because the MSDjS values are the sum of a number of random deviations from a mean, they closely resemble a normal distribution, a consequence of the Central Limit Theorem. We therefore computed z-scores for each sample, which reflect the number of standard deviations away a given sample is relative to the control population:

\(z_{i} = \frac{{MSD}_{i} - \text{?}}{\text{?}}\)
\(\text{?}\text{indicates text missing or illegible when filed}\)

where the standard deviations and means are defined as:

\(\text{?} = \sqrt{\frac{1}{\text{?}}\text{?}\left( {{MSD}_{i}^{S} - \text{?}} \right)^{2}}\)
\({and}\text{:}\)
\(\overset{\_}{{MSD}^{S}} = {\text{?}{MSD}_{i}^{S}}\)
\(\text{?}\text{indicates text missing or illegible when filed}\)

This metric was applied to the analysis of numerous laboratory strains and clinical isolates that were tested for the relevant bacterial species as shown in FIG. 26 and Table 4. A strain was identified as a particular organism if the MSD>2 for that organism.

The direct measurement of RNA expression signatures described herein can provide rapid identification of a range of pathogens in culture and directly from patient specimens. Significantly, phenotypic responses to antibiotic exposure can distinguish susceptible and resistant strains, thus providing an extremely early and rapid determination of susceptibility that integrates varying resistance mechanisms into a common response. This principle represents a paradigm shift in which pathogen RNA forms the basis for a single diagnostic platform that could be applicable in a spectrum of clinical settings and infectious diseases, simultaneously providing pathogen identification and rapid phenotypic antimicrobial susceptibility testing.

## Other Embodiments

It is to be understood that while the invention has been described in conjunction with the detailed description thereof, the foregoing description is intended to illustrate and not limit the scope of the invention, which is defined by the scope of the appended claims. Other aspects, advantages, and modifications are within the scope of the following claims.

