# DESCRIPTION

## TECHNICAL FIELD OF INVENTION

The present invention describes methods for early noninvasive or minimally invasive detection of pathological changes in organ systems, particular organs, tissues, and/or cells by quantifying organ system/organ/tissue/cell type-enriched miRNA in bodily fluids.

## BACKGROUND OF THE INVENTION

It is well accepted that the treatment of any disease is easier and more effective if an underlying pathology is diagnosed as early as possible. For some diseases early diagnosis (preferably before the appearance of clear clinical symptoms) is critically important because of pathology transition into a more advanced, sometimes irreversible, stage. For example, one of the major problems for drug development and treatment of Alzheimer's Disease and other neurodegenerative diseases is their late clinical manifestation and diagnosis due to high compensatory potentials of the brain. As a result, these diseases are usually diagnosed when many neurons are already dead and currently the best case scenario is the prevention of pathology worsening, not a real recovery. Cancer is another example, since treatment of the metastatic stage of the disease is much more problematic than the treatment of the primary tumor. There are many other pathologies of this kind, but again the treatment of any disease is more effective if it is diagnosed earlier.

There are several basic types of clinical tests: (i) genetic tests that help to predict predisposition to a particular disease; (ii) screening tests, which are applied to a large population for early detection of a disease, preferably prior to its clinical manifestation; (iii) diagnostic tests, which are applied when a person has clinical symptoms of a disease or when the pathology has been detected by a screening test; (iv) molecular predictive tests that should predict the disease outcome and drug sensitivity.

Screening tests are most important for the early detection of a disease. It is true not only for spontaneous diseases but also for genetically linked pathologies, for which high chances of getting a disease are predicted by genetic testing. Ideally, everybody should undergo regular screening for all possible life threatening and many other diseases. There have been numerous attempts to develop tests for early detection of various diseases and different screening tests are currently performed for specific risk groups. For example, periodic colonoscopy is recommended to people over 50 years old, Pap smears are recommended to women and the PSA test to men for early detection of the cervical and prostate cancer, respectively. A main advantage of these tests is their disease-specificity, but at the same time it is their serious disadvantage because each test addresses only one particular pathology. However, there are many hundreds of human diseases, and it is difficult to imagine that such specific screening tests would be developed for all of those pathologies. Moreover, even if specific tests for early detection of all human diseases have been developed, it is highly unlikely that such tests would be used for screening purposes, especially for relatively rare diseases due to economic factors. Because screening tests for each particular disease address large populations their specificity and positive predictive value (PPV) are very important. For example, if a screening test for a relatively common disease (1:10,000) is 100% sensitive and 99% specific, which is almost impossible to achieve, and 1 million people are screened, 100 cases would be detected correctly but about 10,000 people would receive false positive results. Obviously, such outcome would cause emotional distress for these people as well as significant financial consequences for additional tests.

## SUMMARY OF THE INVENTION

As specified in the Background Section, above, there is a great need in the art for new screening tests. According to the current paradigm for screening test development and its clinical application, one of the main features of such test should be its high disease specificity. However, as mentioned above there are many hundreds of diseases and screening for each particular disease is principally impossible. The present invention proposes a significant paradigm shift: development and implementation of one or a small number of universal screening tests (USTs), which specifically detect a pathology of any particular organ system, organ, tissue and/or cell type, without diagnosing a particular disease. Such USTs should be performed periodically for any given subject for an early, preferably at clinically asymptomatic stage, and then more specific tests can be used for a more specific diagnosis. The USTs as described herein would improve disease diagnosis and treatment, and significantly decrease medical expenses. Also, the USTs as described herein would make tests for rare diseases more targeted since they will be applied to much smaller populations preselected by UST and, therefore, will be more economically practical.

Because UST will address a large population (preferably everyone), its first important feature is minimal invasiveness. The present invention proposes to register various physiological and pathological processes in particular organs, tissues and even cell types by analysis of respective biomarkers in bodily fluids that can be obtained by non-invasive or minimally invasive methods, such as, e.g., plasma/serum, urine, or saliva. Secondly, UST cannot be based on the inducing factor or pathogenesis of diseases, since there are too many of them. Thirdly, to be widely used, UST should not be very expensive, which, in particular, means that it has to utilize a limited number of biomarker types that can be analyzed by the same technique.

Biomarkers used for UST should have a set of parameters that make them suitable for such type of a test:


- - 1. Cell/tissue/organ-specificity or significant enrichment (e.g., at
    least 5 times higher concentration as compared to other
    cells/tissues/organs).
  - 2. Ability to be secreted into extracellular space and to pass
    various barriers within the body.
  - 3. Presence in detectable amounts in bodily fluids that can be
    obtained with minimal invasiveness.
  - 4. Stability.
  - 5. Detectability with high sensitivity and specificity at a
    relatively low cost.

The following classes of molecules can be of potential use for UST:


- - 1. Proteins.
  - 2. mRNA and mRNA fragments.
  - 3. miRNA.
  - 4. DNA fragments
  - 5. DNA methylation.
  - 6. Lipids.
  - 7. Sugars.

However, some of these potential tissue-specific biomarkers have serious disadvantages, which make their use impractical or even impossible. For example, DNA methylation, lipids, and sugars are not sufficiently specific to differentiate between various tissue and cell types. Besides, DNA fragments appear in the extracellular space and in the bloodstream mainly from dying cells (Lichtenstein et al., Ann. New York Acad. Sci. 2001, 945:239-249) and, thus, circulating cell-free DNA cannot be used to detect pathology stages, which are not accompanied by cell death. Proteins and mRNA are better candidates for UST due to their higher tissue-specificity/enrichment (Diez-Roux et al., PLOS Biology. 2011, 9:e1000582). However, mRNA are large molecules, they are easily degraded by nucleases, and so only short fragments of these molecules appear in the bloodstream. This does not exclude their use as biomarkers for UST but makes test development more difficult. Proteins are good candidates but current methods for their detection are significantly less sensitive than nucleic acid detection techniques. Also, many proteins are large molecules and are not able to cross cellular membrane and other barriers.

The present invention proposes to use miRNA biomarkers in the UST for the following reasons: miRNA are small molecules, they can appear in the extracellular space, cross brain, kidney, and placenta barriers, appear in various bodily fluids, are stable, and existing methods for their analysis are very specific and sensitive. Most importantly, miRNA constitute a large class of diverse molecules with at least some miRNA being enriched in some organ systems, organs, tissues and/or cells, thus providing molecular markers for all parts of the body.

MicroRNA (miRNA) are a class of non-coding RNAs whose final product (mature miRNA) is an approximately 22 nt functional RNA molecule. They play important roles in the regulation of target genes by binding to complementary regions of messenger transcripts to repress their translation or regulate degradation (Griffiths-Jones Nucleic Acids Research, 2006, 34, Database issue: D140-D144). Frequently, one miRNA can target multiple mRNAs and one mRNA can be regulated by multiple miRNA targeting different regions of the 3′ UTR. Once bound to an mRNA, miRNA can modulate gene expression and protein production by affecting mRNA translation and stability (e.g., Baek et al., Nature. 2008, 455:64; Selbach et al., Nature. 2008, 455:58; Ambros, Nature. 2004, 431:350-355; Bartel, Cell. 2004, 116:281-297; Cullen, Virus Research. 2004, 102:3-9; He et al. Nat. Rev. Genet. 2004, 5:522-531; and Ying et al., Gene. 2004, 342:25-28). There are other classes of less characterized small RNAs (reviewed in Kim, Mol. Cells, 2005, 19: 1-15). Many of miRNA are specific to or over-expressed in certain organs/tissues/cells (See, e.g., Hua et al., BMC Genomics 2009, 10:214; Liang et al., BMC Genomics. 2007, 8:166; Landgraf et al., Cell. 2007, 129:1401-1414; Lee et al., RNA. 2008, 14:35-42). Due to their small size, miRNA can cross the blood-brain, kidney, and placental barriers into bodily fluids where they are sufficiently stable (Rosenfeld et al., Nature Biotech. 2008, 26:462-469; Mitchell et al., Proc. Natl. Acad. Sci. USA. 2008, 105: 10513-10518; Chen et al., Cell Res. 2008, 18:997-1006; Chim et al., Clin. Chem. 2008, 54:482-490; De Smaele et al., Brain Res. 2010, 1338:100-111; Fichtlscherer et al., Circ. Res. 2010, 107:677-684; Scholer et al., Exp. Hematol. 2010, 38:1126-1130). Analysis of cell/tissue-specific miRNA in bodily fluids was proposed for detection of in vivo cell death (U.S. Patent Pub. No 20090081640; Laterza et al., Clin Chem. 2009, 55:1977-1983). The increase of concentrations of circulating cell-free liver-enriched miRNA in the bloodstream has been demonstrated in several studies (Zhang et al., Clin Chem. 2010, 56:1830-1838; Xu et al., Mol Carcinogenesis. 2011, 50:136-142). For example, the levels of liver-enriched miRNA-122a go up in serum of patients with hepatitis and hepatocellular carcinomas, and the authors come to a conclusion that due to this non-specificity for a particular disease these miRNA cannot be used as biomarkers for HCC (Zhang et al., Clin Chem. 2010, 56:1830-1838; Li et al., Cancer Res. 2010, 70:9798-9807). On the contrary, the present invention demonstrates that such disease non-specificity of the organ/tissue/cell-enriched miRNA is a significant advantage when they are used as biomarkers for UST development.

Expression and concentrations of miRNA are regulated by various physiological and pathological signals. Some of miRNA are characteristic of a particular pathology, such as hypoxia (Loscalzo, J. Clin. Invest. 2010, 120: 3815-3817; Pocock, Pflugers Arch. 2011, 461:307-315), inflammation (Tili et al., Int. Rev. Immunol. 2009, 28:264-284; Davidson-Moncada et al., Ann. NY Acad. Sci. 2010, 1183:183-194; Roy and Sen, Physiol. Genomics. 2011, 43:557-565), or carcinogenesis (Budhu et al., J. Hematol. Oncol. 2010, 3:37; Zen and Zhang, Med. Res. Rev. 2012, 32:326-348). This phenomenon makes it reasonable to include such miRNA as biomarkers into the USTs of the invention. An increase of these miRNA concentrations in bodily fluids will indicate the presence of a respective general pathology in the body without localizing it to a particular organ, tissue or cell type. Generally speaking, this would be the same approach as proposed above for organ/tissue-enriched miRNA: organ/tissue-enriched miRNA will help to detect a pathology in a particular organ or tissue; on the other hand, miRNA characteristic of a particular general pathology will help to detect the presence of this pathology somewhere in the body without indicating a specific organ/tissue involved. A combined use of miRNA biomarkers which are enriched in an organ system, organ, tissue, and/or cell type, and miRNA biomarkers characteristic of a particular general pathology will provide a more precise diagnosis, namely the presence of a particular pathology in a particular organ or tissue or cell type. For example, increased concentrations of miRNA characteristic of hypoxia in plasma, combined with increased concentrations of miRNA enriched in the heart, will provide more specific diagnosis of heart ischemia obtained by the UST.

Thus, in one aspect, the present invention provides a method for detecting a pathology in any organ system in a subject, which method comprises:

a. measuring levels of miRNAs enriched in various organ systems in a bodily fluid sample collected from the subject;

b. measuring levels of preselected normalizer miRNAs in the same bodily fluid sample collected from the subject;

c. calculating the ratios of the levels of the miRNAs measured in steps (a) and (b);

d. comparing the ratios of the levels of the miRNAs calculated in step (c) with the corresponding control ratios, and

e. (i) identifying the subject as being afflicted with a pathology of a particular organ system when the ratios of the levels of the miRNAs enriched in said organ system to their respective miRNA normalizers calculated in step (c) are higher than the corresponding control ratios or (ii) identifying the subject as not being afflicted with a pathology of said organ system when the ratios of the levels of the miRNAs enriched in said organ system to their respective miRNA normalizers calculated in step (c) are not higher than the corresponding control ratios.

The invention also provides a method for detecting a pathology in an organ system in a subject, which method comprises:

a. measuring the level of at least one miRNA enriched in said organ system in a bodily fluid sample collected from the subject;

b. measuring the level of a normalizer miRNA in the same bodily fluid sample collected from the subject;

c. calculating the ratio of the levels of the miRNAs measured in steps (a) and (b);

d. comparing the ratio of the levels of the miRNAs calculated in step (c) with a corresponding control ratio, and

e. (i) identifying the subject as being afflicted with a pathology of said organ system when the ratio of the levels of the miRNAs calculated in step (c) is higher than the corresponding control ratio or (ii) identifying the subject as not being afflicted with a pathology of said organ system when the ratio of the levels of the miRNAs calculated in step (c) is not higher than the corresponding control ratio.

The normalizer miRNA useful in the above two methods can be, for example, ubiquitous miRNA, miRNA expressed in many organs but under-expressed in said organ system, or experimentally selected miRNA enriched in said organ system.

The control ratio of the levels of the miRNAs used in the above two methods can be a predetermined standard (e.g., determined using a population of control subjects [e.g., age-matched to the diagnosed subject] without pathologies of the respective organ system) or the ratio of the levels of the same mRNAs in a similarly processed bodily fluid sample from the same subject collected in the past.

In one embodiment, the above methods involve determining two or more miRNA ratios.

In one embodiment of the above methods, the miRNA enriched in an organ system is selected from miRNAs listed in Table 2, below.

In another embodiment of the above methods, the organ system is central nervous system and the miRNA/normalizer pairs are selected from those listed in Table 4, below.

In yet another embodiment of the above methods, the organ system is respiratory system and the miRNA/normalizer pairs are selected from those listed in Table 5, below.

In a further embodiment of the above methods, the organ system is gastrointestinal (GI) system and the miRNA/normalizer pairs are selected from those listed in Table 7, below.

In another aspect, the invention provides a method for detecting a pathology in any organ in a subject, which method comprises:

a. measuring levels of miRNAs enriched in various organs in a bodily fluid sample collected from the subject;

b. measuring levels of preselected normalizer miRNAs in the same bodily fluid sample collected from the subject;

c. calculating the ratios of the levels of the miRNAs measured in steps (a) and (b);

d. comparing the ratios of the levels of the miRNAs calculated in step (c) with the corresponding control ratios, and

e. (i) identifying the subject as being afflicted with a pathology of a particular organ when the ratios of the levels of the miRNAs enriched in said organ to their respective miRNA normalizers calculated in step (c) are higher than the corresponding control ratios or (ii) identifying the subject as not being afflicted with a pathology of said organ when the ratios of the levels of the miRNAs enriched in said organ to their respective miRNA normalizers calculated in step (c) are not higher than the corresponding control ratios.

The invention also provides a method for detecting a pathology in an organ in a subject, which method comprises:

a. measuring the level of at least one miRNA enriched in said organ in a bodily fluid sample collected from the subject;

b. measuring the level of a normalizer miRNA in the same bodily fluid sample collected from the subject;

c. calculating the ratio of the levels of the miRNAs measured in steps (a) and (b);

d. comparing the ratio of the levels of the miRNAs calculated in step (c) with a corresponding control ratio, and

e. (i) identifying the subject as being afflicted with a pathology of said organ when the ratio of the levels of the miRNAs calculated in step (c) is higher than the corresponding control ratio or (ii) identifying the subject as not being afflicted with a pathology of said organ when the ratio of the levels of the miRNAs calculated in step (c) is not higher than the corresponding control ratio.

The normalizer miRNA useful in the above two methods can be, for example, ubiquitous miRNA, miRNA expressed in many organs but under-expressed in said organ, or experimentally selected miRNA enriched in said organ.

The control ratio of the levels of the miRNAs used in the above two methods can be a predetermined standard (e.g., determined using a population of control subjects [e.g., age-matched to the diagnosed subject] without pathologies of the respective organ) or the ratio of the levels of the same mRNAs in a similarly processed bodily fluid sample from the same subject collected in the past.

In one embodiment, the above two methods involve determining two or more miRNA ratios.

In one embodiment of the above two methods, the miRNA enriched in an organ is selected from miRNAs listed in Tables 1 and 2, below.

In another embodiment of the above two methods, the organ is a gastrointestinal (GI) organ and the miRNA/normalizer pairs are selected from those listed in Table 9, below.

In another aspect, the invention provides a method for detecting a pathology in any tissue in a subject, which method comprises:

a. measuring levels of miRNAs enriched in various tissues in a bodily fluid sample collected from the subject;

b. measuring levels of normalizer miRNAs in the same bodily fluid sample collected from the subject;

c. calculating the ratios of the levels of the miRNAs measured in steps (a) and (b);

d. comparing the ratios of the levels of the miRNAs calculated in step (c) with the corresponding control ratios, and

e. (i) identifying the subject as being afflicted with a pathology of a particular tissue when the ratios of the levels of the miRNAs enriched in said tissue to their respective miRNA normalizers calculated in step (c) are higher than the corresponding control ratios or (ii) identifying the subject as not being afflicted with a pathology of said tissue when the ratios of the levels of the miRNAs enriched in said tissue to their respective miRNA normalizers calculated in step (c) are not higher than the corresponding control ratios.

The invention also provides a method for detecting a pathology in a tissue in a subject, which method comprises:

a. measuring the level of at least one miRNA enriched in said tissue in a bodily fluid sample collected from the subject;

b. measuring the level of a normalizer miRNA in the same bodily fluid sample collected from the subject;

c. calculating the ratio of the levels of the miRNAs measured in steps (a) and (b);

d. comparing the ratio of the levels of the miRNAs calculated in step (c) with a corresponding control ratio, and

e. (i) identifying the subject as being afflicted with a pathology of said tissue when the ratio of the levels of the miRNAs calculated in step (c) is higher than the corresponding control ratio or (ii) identifying the subject as not being afflicted with a pathology of said tissue when the ratio of the levels of the miRNAs calculated in step (c) is not higher than the corresponding control ratio.

The normalizer miRNA useful in the above two methods can be, for example, ubiquitous miRNA, miRNA expressed in many tissues but under-expressed in said tissue, or experimentally selected miRNA enriched in said tissue.

The control ratio of the levels of the miRNAs used in the above two methods can be a predetermined standard (e.g., determined using a population of control subjects [e.g., age-matched to the diagnosed subject] without pathologies of the respective tissue) or the ratio of the levels of the same mRNAs in a similarly processed bodily fluid sample from the same subject collected in the past.

In one embodiment, the above two methods involve determining two or more miRNA ratios.

In one embodiment of the above two methods, the miRNA enriched in a tissue is selected from miRNAs listed in Tables 1 and 2, below.

In another aspect, the invention provides a method for detecting a pathology in any cell type in a subject, which method comprises:

a. measuring levels of miRNAs enriched in various cell types in a bodily fluid sample collected from the subject;

b. measuring levels of normalizer miRNAs in the same bodily fluid sample collected from the subject;

c. calculating the ratios of the levels of the miRNAs measured in steps (a) and (b);

d. comparing the ratios of the levels of the miRNAs calculated in step (c) with the corresponding control ratios, and

e. (i) identifying the subject as being afflicted with a pathology of a particular cell type when the ratios of the levels of the miRNAs enriched in said cell type to their respective miRNA normalizers calculated in step (c) are higher than the corresponding control ratios or (ii) identifying the subject as not being afflicted with a pathology of said cell type when the ratios of the levels of the miRNAs enriched in said cell type to their respective miRNA normalizers calculated in step (c) are not higher than the corresponding control ratios.

The invention also provides a method for detecting a pathology in a cell type in a subject, which method comprises:

a. measuring the level of at least one miRNA enriched in said cell type in a bodily fluid sample collected from the subject;

b. measuring the level of a normalizer miRNA in the same bodily fluid sample collected from the subject;

c. calculating the ratio of the levels of the miRNAs measured in steps (a) and (b);

d. comparing the ratio of the levels of the miRNAs calculated in step (c) with a corresponding control ratio, and

e. (i) identifying the subject as being afflicted with a pathology of said cell type when the ratio of the levels of the miRNAs calculated in step (c) is higher than the corresponding control ratio or (ii) identifying the subject as not being afflicted with a pathology of said cell type when the ratio of the levels of the miRNAs calculated in step (c) is not higher than the corresponding control ratio.

The normalizer miRNA useful in the above two methods can be, for example, ubiquitous miRNA, miRNA expressed in many cell types but under-expressed in said cell type, or experimentally selected miRNA enriched in said cell type.

The control ratio of the levels of the miRNAs used in the above two methods can be a predetermined standard (e.g., determined using a population of control subjects [e.g., age-matched to the diagnosed subject] without pathologies of the respective cell type) or the ratio of the levels of the same mRNAs in a similarly processed bodily fluid sample from the same subject collected in the past.

In one embodiment, the above two methods involve determining two or more miRNA ratios.

In one embodiment of the above two methods, the miRNA enriched in a cell type is selected from miRNAs listed in Tables 1 and 2, below.

The above-described methods can be combined. For example, detecting a pathology of an organ system can be followed by determining the affected organ and/or tissue and/or cell type, etc.

Any of the above-described methods can also further comprise identifying whether the pathology is cancer or inflammation, which method comprises:

a. measuring the level of at least one miRNA associated with cancer in a bodily fluid sample collected from the subject;

b. measuring the level of at least one miRNA associated with inflammation in the same bodily fluid sample collected from the subject;

c. measuring the level of at least one miRNA enriched in the involved organ system, organ, tissue or cell type in the same bodily fluid sample collected from the subject;

d. measuring the level of at least one normalizer miRNA in the same bodily fluid sample collected from the subject;

e. calculating pair-wise ratios of the levels of the miRNAs measured in steps (a), (b), (c), and (d) (e.g., ratios a/b, a/c, a/d, b/c, b/d, and c/d);

f. comparing the ratios of the levels of the miRNAs calculated in step (e) with the corresponding predetermined ratios characteristic of cancer and inflammation, and

g. (i) identifying that the pathology is cancer when the ratios of the levels of the miRNAs calculated in step (e) are in the predetermined range characteristic of cancer, or (ii) identifying that the pathology is inflammation when the ratios of the levels of the miRNAs calculated in step (e) are in the predetermined range characteristic of inflammation.

In one embodiment of the above method, the miRNAs associated with cancer and inflammation are selected from miRNAs listed in Table 3, below.

In one embodiment of the above method, the pathology relates to lung and the miRNA pairs are selected from those listed in Table 6, below.

In one embodiment of the above method, the pathology relates to gastrointestinal (GI) system and the miRNA pairs are selected from those listed in Table 8, below.

In one embodiment of the above method, the pathology relates to respiratory system or gastrointestinal (GI) system and the miRNA pairs are selected from those listed in Table 11, below.

A similar method can be applied to distinguishing cancer or inflammation from hypoxia.

Any of the above methods can be followed by administering a disease-specific diagnostic test to the subject.

Any of the above methods can be followed by administering a therapeutic treatment to the subject that has been diagnosed as having a pathology.

Any of the above methods can be followed by recruiting the subject in a clinical trial (this can apply to both subjects diagnosed as having a pathology and to subjects diagnosed as not having a pathology).

The above-described methods of the invention can be applied for detecting pathologies in any subject, including subjects who have no clinical symptoms indicative of a pathology of said organ system or organ or tissue or cell type.

In a separate aspect, the invention provides a method for identifying a compound useful for slowing down the progression or treating a pathology of an organ system or organ or tissue or cell type, which method comprises:

a. measuring the level of at least one miRNA enriched in said organ system or organ or tissue or cell type in one or more bodily fluid samples collected from a subject having said pathology of said organ system or organ or tissue or cell type, wherein said bodily fluid sample(s) is collected prior to administration of a test compound;

b. measuring the level of a normalizer miRNA in the same bodily fluid sample(s) from the subject collected prior to administration of the test compound;

c. calculating the ratio of the levels of the miRNAs measured in steps (a) and (b) for each of the bodily fluid samples collected from the subject prior to administration of the test compound;

d. measuring the level of the same miRNA as in step (a) in one or more bodily fluid samples collected from the subject following administration of the test compound;

e. measuring the level of the same normalizer miRNA as in step (b) in the same bodily fluid sample(s) collected from the subject following administration of the test compound;

f. calculating the ratio of the levels of the miRNAs measured in steps (d) and (e) for each of the bodily fluid samples collected from the subject following administration of the test compound;

g. comparing the ratio of the levels of the miRNAs calculated in steps (c) and (f), and

h. (i) identifying that the test compound is useful for slowing down the progression or treating said pathology of said organ system or organ or tissue or cell type if the ratio of the levels of the miRNAs calculated in step (f) is lower than the ratio of the levels of the miRNAs calculated in step (c); (ii) identifying that the test compound is not useful for slowing down the progression or treating said pathology of said organ system or organ or tissue or cell type if the ratio of the levels of the miRNAs calculated in step (f) is not lower than the ratio of the levels of the miRNAs calculated in step (c).

In one embodiment of the above method, the pathology is cancer or inflammation or hypoxia, and miRNA is selected from miRNAs listed in Table 3, below.

In another aspect, the invention provides a method for determining toxicity of a compound (e.g., a compound being tested in a clinical trial) or environmental factor (e.g., allergen, smoking, UV, radiation, asbestos, etc.) to an organ system or organ or tissue or cell type in a subject free of pathologies of said organ system or organ or tissue or cell type, which method comprises:

a. measuring the level of at least one miRNA enriched in said organ system or organ or tissue or cell type in one or more bodily fluid samples collected from the subject before the subject has been exposed to the compound or environmental factor;

b. measuring the level of a normalizer miRNA in the same bodily fluid sample(s) collected from the subject before the subject has been exposed to the compound or environmental factor;

c. calculating the ratio of the levels of the miRNAs measured in steps (a) and (b) for each of the bodily fluid samples collected from the subject before the subject has been exposed to the compound or environmental factor;

d. measuring the level of the same miRNA enriched in said organ system or organ or tissue or cell type in one or more bodily fluid samples collected from the subject after the subject had been exposed to the compound or environmental factor;

e. measuring the level of the same normalizer miRNA in the same bodily fluid sample(s) collected from the subject after the subject had been exposed to the compound or environmental factor;

f. calculating the ratio of the levels of the miRNAs measured in steps (d) and (e) for each of the bodily fluid samples;

g. comparing the ratio of the levels of the miRNAs calculated in steps (c) and (f), and

h. (i) identifying that the compound or environmental factor is not toxic to said organ system or organ or tissue or cell type if the ratio of the levels of the miRNAs calculated in step (f) is not higher than the ratio of the levels of the miRNAs calculated in step (c); (ii) identifying that the compound or environmental factor is toxic to said organ system or organ or tissue or cell type if the ratio of the levels of the miRNAs calculated in step (f) is higher than the ratio of the levels of the miRNAs calculated in step (c).

In one embodiment, the above two methods involve determining two or more miRNA ratios.

In one embodiment of the above two methods, the miRNA enriched in an organ system or organ or tissue or cell type is selected from miRNAs listed in Tables 1 and 2, below.

In one embodiment of the above two methods, the organ system is central nervous system and the miRNA/normalizer pairs are selected from those listed in Table 4, below.

In one embodiment of the above two methods, the organ system is respiratory system and the miRNA/normalizer pairs are selected from those listed in Table 5, below.

In one embodiment of the above two methods, the organ system is gastrointestinal (GI) system and the miRNA/normalizer pairs are selected from those listed in Table 7, below.

Each of the measuring steps in the above-described methods does not have to be performed in the specific order listed above.

Subjects used in the methods of the present invention include, e.g., humans, veterinary animals and experimental animal models of diseases. Non-limiting examples of biomarker miRNAs and normalizer miRNAs useful in the above-described methods of the present invention are provided, e.g., in Tables 1-11, below.

Non-limiting examples of bodily fluid samples which can be used in the methods of the invention include, e.g., urine, blood plasma and serum. If urine is used, it is preferred that the urine sample has been held in the bladder for less than 4 hours.

Non-limiting examples of methods for determining the level of miRNA in the methods of the invention include, e.g., hybridization, RT-PCR, and direct sequencing.

In some embodiments, the methods of the invention comprise (e.g., as an initial step) the step of collecting a bodily fluid sample from the subject.

In one embodiment (applicable to any of the above methods of the invention), the method also includes a step of reducing or eliminating degradation of miRNA. Non-limiting examples of useful methods for reducing or eliminating miRNA degradation include, e.g., adding an RNase inhibitor, treatment with guanidine chloride, treatment with guanidine isothiocyanate, treatment with N-lauroylsarcosine, treatment with sodium dodecyl sulphate (SDS), and a combination thereof.

In conjunction with the above diagnostic and screening methods, the present invention also provides various kits comprising one or more primer and/or probe sets specific for the detection of target miRNA. Such kits can further include primer and/or probe sets specific for the detection of normalizer miRNA. Non-limiting examples of primer or probe combinations in kits are as follows:

1. primers or probes specific for at least two miRNAs selected from the group consisting of miR-1, 22, 30a-3p, 30e-3p, 133a, 133b, 197, 208a, 208b, 221, 222, 302a, 302c, 367, 378, 499-5p, and 30e*.

2. primers or probes specific for at least two miRNAs selected from the group consisting of miR-1, 22, 95, 133a, 133b, 140, and 206.

3. primers or probes specific for at least two miRNAs selected from the group consisting of miR-15b, 18b, 21, 34b, 126, 135b, 142-3p, 142-5p, 146, 146b-5p, 155, 199b-5p, 200c, 205, 211, 223, 224, 302b, 375, 449a, 449b, 450b-5p, 486, 492, 522, 566, 574-3p, 620, 650, 766, and 886-5p.

4. primers or probes specific for at least two miRNAs selected from the group consisting of miR-34b, 135b, 146, 146b, 147b, 155, 199b-5p, 200b, 200c, 205, 219-5p, 223, 302b, and 375.

5. primers or probes specific for at least two miRNAs selected from the group consisting of miR-30e-3p, 122a, 130b, 136, 148a, 194, 376c, 455-3p, 518b, 616, 801, 885-5p, 17*, 30d*, and 194*.

6. primers or probes specific for at least two miRNAs selected from the group consisting of miR-10a, 10b, 30a-3p, 30c, 107, 135a, 135b, 184, 187, 190, 194, 196b, 200a, 204, 211, 324-5p, 489, 500, 501-5p, 502-3p, 502-5p, 503, 506, 508-3p, 508-5p, 509-3p, 509-5p, 510, 532-5p, 768-3p, 886-3p, 886-5p, 891a, 10b*, 30a*, 30c-2*, 30e*, 200a*, 200b*, 424*, and 500*.

7. primers or probes specific for at least two miRNAs selected from the group consisting of miR-Let-7g, 18, 23b, 26a, 26b, 27b, 28, 106b, 143, 145, 152, 218, 221, 223, 296, 374, 422b, and 451.

6. primers or probes specific for at least two miRNAs selected from the group consisting of miR-10b, 30, 99a, 139-3p, 139-5p, 193a-5p, 196a, 224, 335, 365, 378/378*, 422b, 494, 518d-3p, 642a-3p, 708, 10b*, and 335*.

7. primers or probes specific for at least two miRNAs selected from the group consisting of miR-let-7a, 10b, 26a, 30a-3p, 30a-5p, 125b, 126, 145, 146, 195, 196a-2, 196b, 205, 206, 335, 339-5p, 378, 516-5p, 517c, 519c, 520g, 520h, 525, and 1246.

8. primers or probes specific for at least two miRNAs selected from the group consisting of miR-Let-7a, let-7b, let-7c, 10b, 17-3p, 26a, 100, 125a, 125b, 127, 195, 199a-5p, 202, 214, 298, 382, 503, 672, 741, 742, 883-3p, 199a*, and 202*.

9. primers or probes specific for at least two miRNAs selected from the group consisting of miR-10a, 10b, 31, 34b, 34c, 135a, 135b, 424, and 449.

10. primers or probes specific for at least two miRNAs selected from the group consisting of miR-Let-7c, 10b, 26a, 99a, 100, 125a-5p, 125b, 130a, 140, 143, 145, 195, 196b, 199b, 204, 214, 222, 939, and 199*.

11. primers or probes specific for at least two miRNAs selected from the group consisting of miR-Let-7a, let-7c, let-7 g, 10b, 100, 101, 125a-5p, 125b, 130a, 134, 140, 143, 145, 186, 195, 196b, 197, 199a, 199b, 204, 214, 218, 222, 320, 424, 497, 154*, and 199a*.

12. primers or probes specific for at least two miRNAs selected from the group consisting of miR-Let-7c, 1, 23b, 24, 27b, 28, 34a, 99a, 100, 125b, 130a, 143, 145, 147b, 187, 188-3p, 199b-5p, 205, 214, 222, 328, 373, 410, 455-5p, and 490-3p.

13. primers or probes specific for at least two miRNAs selected from the group consisting of miR-15b, 34a, 34b, 34c, 127, 134, 135a, 135b, 187, 202, 204, 370, 372, 376a, 382, 424, 449, 465a-5p, 465b-5p, 506, 508, 509, 510, 514, 517a, 517c, 871-5p, 871-3p, 888, 202*, and 888*.

14. primers or probes specific for at least two miRNAs selected from the group consisting of miR-Let-7 family, 10a, 17, 18a, 19a, 19b, 20a, 92, 21, 22, 23a, 24, 27a, 27b, 29a, 31, 34a, 98, 100, 106a, 126, 130a, 133a, 143, 145, 146a, 199a-3p, 210, 221, 222, 345, 365, 382, 409-3p, 431, 484, 495, 532-5p, 939, 27a*, 30a*, 30e*, 93*, 126*, 130b*, and 222*.

15. primers or probes specific for at least two miRNAs selected from the group consisting of miR-15a, 15b, 126, 139, 142-3p, 142-5p, 146, 150, 155, 181a, 181b, 181d, 223, 302b, and 342.

16. primers or probes specific for at least two miRNAs selected from the group consisting of miR-15a, 15b, 17-5p, 20b, 106a, 106b, 142-3p, 142-5p, 146, 149, 150, 155, 181a, 181b, 181c, 182, 183, 205, 213, and 342.

17. primers or probes specific for at least two miRNAs selected from the group consisting of miR-Let-7g, 15a, 20b, 21, 106b, 140, 142-3p, 146, 146b, 150, 181b, 181d, 342, and 431.

18. primers or probes specific for at least two miRNAs selected from the group consisting of miR-Let-7g, 9, 15a, 15b, 17, 19b, 20a, 31, 106a, 124a, 124b, 128a, 137, 142-3p, 146b-5p, 150, 186, 191, 197, 222, 223, 328, 342-3p, 423, 431, 454, 484, 766, 27*, and 223*.

19. primers or probes specific for at least two miRNAs selected from the group consisting of miR-142-3p, 146a, 155, 181a, 205, 223, and 424.

20. primers or probes specific for at least two miRNAs selected from the group consisting of miR-142, 150, and 342.

21. primers or probes specific for at least two miRNAs selected from the group consisting of miR-Let-7i, 1, 7, 135a, 135b, 206, and 345.

22. primers or probes specific for at least two miRNAs selected from the group consisting of miR-Let-7g, 7, 15a, 26b, 27a, 99b, 124, 127, 132, 134, 137, 139, 152, 181a, 187, 195, 192, 202, 299, 302b, 323, 324-3p, 324-5p, 328, 330-3p, 331, 335, 340, 365, 369-3p, 375, 379, 382, 409-5p, 429, 431, 432, 455-5p, 483-5p, 514, 126*, 182*, and 202*.

23. primers or probes specific for at least two miRNAs selected from the group consisting of miR-7, 18a, 21,29a, 34a, 103, 127-3p, 129-3p, 130b, 134, 135a, 135b, 136, 141, 148a, 182, 183, 184, 192, 193a-3p, 193a-5p, 195, 199a-3p, 199a-5p, 200b, 200c, 204, 216a, 216b, 217, 224, 340, 365, 367, 374a, 374b, 375, 376a, 376c, 379, 382, 383, 429, 432, 451, 455-5p, 485-5p, 487b, 497, 539, 543, 642, 758, 939, 130b*, 136*, 183*, 200b*, and 493*.

24. primers or probes specific for at least two miRNAs selected from the group consisting of miR-7, 9, 21, 127-3p, 130b, 184, 195, 216a, 216b, 217, 376a, 376c, 497, 939, and 493*.

25. primers or probes specific for at least two miRNAs selected from the group consisting of miR-31, 141, 143, 145, 147b, 192, 194, 200a, 200b, 200bN, 200c, 200cN, 215, 219-2-3p, 321, 375, 378, 422a, 429, 450b-5p, 487a, 490-3p, 492, 504, 565, 574-3p, 622, 650, 801, 143*, and 200b*.

26. primers or probes specific for at least two miRNAs selected from the group consisting of miR-31, 141, 143, 192, 194, 200a, 200b, 200bN, 200c, 200cN, 215, 321, 375, and 429.

27. primers or probes specific for at least two miRNAs selected from the group consisting of miR-31, 106a, 106b, 143, 145, 148a, 203, 205, 210, 211, and 221.

28. primers or probes specific for at least two miRNAs selected from the group consisting of miR-7, 26a, 26b, 29c, 31, 106a, 106b, 124b, 130b, 141, 145, 148a, 182, 188, 192, 197, 203, 375, and 650.

29. primers or probes specific for at least two miRNAs selected from the group consisting of miR-Let-7a, 7, 9, 96, 98, 99a, 103, 107, 124a, 125a, 125b, 127, 128a, 132, 134, 135a, 137, 138, 149, 153, 154, 181a, 181b, 181c, 182, 183, 184, 204, 211, 212, 213, 218, 219, 221, 222, 299-3p, 299-5p, 323-3p, 324-5p, 328, 329, 330, 331, 335, 337, 338, 342, 346, 369-3p, 369-5p, 370, 379, 381, 382, 383, 409-3p, 411, 425, 432, 433-5p, 485-3p, 485-5p, 487b, 488, 491-5p, 494, 495, 496, 504, 539, 541, 543, 584, 656, 668, 758, 874, 889, 935, 939, 1193, 1197, and 9*.

30. primers or probes specific for at least two miRNAs selected from the group consisting of miR-7, 9, 98, 124a, 125a, 125b, 128a, 132, 134, 135a, 137, 138, 154, 182, 183, 213, 218, 323-3p, 329, 337, 369-3p, 369-5p, 370, 381, 382, 409-3p, 425, 433-5p, 483-3p, 485-5p, 487b, 494, 495, 496, 541, 543, 656, 668, 874, 889, 935, 939, and 9*.

31. primers or probes specific for at least two miRNAs selected from the group consisting of miR-9, 124a, 125a, 125b, 128a, 132, 134, 181c, 212, 213, 222, 330-3p, 338-5p, 342, 381, 382, 425, 433, and 491-5p.

32. primers or probes specific for at least two miRNAs selected from the group consisting of miR-9, 96, 99a, 103, 124a, 125b, 128a, 132, 134, 137, 138, 181a, 181b, 212, 219, 221, 222, 324-5p, 328, 330, 331, 335-5p, 338, 369-3p, 381, 382, 383, 425, 433-5p, 485-5p, and 491-5p.

33. primers or probes specific for at least two miRNAs selected from the group consisting of miR-7, 124a, 128a, 132, and 212.

34. primers or probes specific for at least two miRNAs selected from the group consisting of miR-9, 103, 124a, 125b, 128, 132, 134, 137, 138, 181a, 181b, 181c, 204, 212, 213, 218, 338, 381, 382, 425, 432, and 489.

35. primers or probes specific for at least two miRNAs selected from the group consisting of miR-103, 134, 138, 182, 183, 222, 323-3p, 369, 381, and 382.

36. primers or probes specific for at least two miRNAs selected from the group consisting of miR-218, 219, 338, 451, and 486.

37. primers or probes specific for at least two miRNAs selected from the group consisting of miR-7, 132, 212, 213, and 328.

38. primers or probes specific for at least two miRNAs selected from the group consisting of miR-34b, 135b, 146, 146b-5p, 155, 199b-5p, 200c, 205, 223, 302b, and 375.

39. primers or probes specific for at least two miRNAs selected from the group consisting of miR-15b, 18b, 21, 126, 142-3p, 142-5p, 224, 449a, 449b, 450b-5p, 486, 492, 522, 566, 5′74-3p, 650, 766, and 886-5p.

40. primers or probes specific for at least two miRNAs selected from the group consisting of miR-147b, 200b, and 219-5p.

41. primers or probes specific for at least two miRNAs selected from the group consisting of miR-31, 130b, 136, 141, 143, 145, 148a, 192, 203, 215, 375, 376c, 429, 455-5p, and 650.

42. primers or probes specific for at least two miRNAs selected from the group consisting of miR-106a, 106b, 205, and 210, and 221.

43. primers or probes specific for at least two miRNAs selected from the group consisting of miR-7, 26a, 26b, 26c, 106a, 106b, 124b, 182, 188, and 197.

44. primers or probes specific for at least two miRNAs selected from the group consisting of miR-194, 200a, 200b, 200c, and 321.

45. primers or probes specific for at least two miRNAs selected from the group consisting of miR-147b, 194, 200a, 200b, 200c, 219-3p, 378, 450-5p, 487a, 490-3p, 492, 504, 565, 5′74-3p, 622, 801, 143*, and 200b*.

46. primers or probes specific for at least two miRNAs selected from the group consisting of miR-122a, 194, 518b, 616, 801, 885-5p, 17*, 30d*, and 194*.

47. primers or probes specific for at least two miRNAs selected from the group consisting of miR-7, 18a, 21,29a, 34a, 103, 127-3p, 129-3p, 134, 135a, 135b, 182, 183, 184, 193a-3p, 193a-5p, 195, 199a-3p, 199a-5p, 200b, 200c, 204, 216a, 216b, 217, 224, 340, 365, 367, 374a, 374b, 376a, 379, 382, 383, 432, 451, 485-5p, 487b, 497, 539, 543, 642, 758, 939, 130b*, 136*, 183*, 200b*, and 493*.

48. primers or probes specific for at least two miRNAs selected from the group consisting of miR-1, 22, 95, 133a, 133b, 140, and 206.

49. primers or probes specific for at least two miRNAs selected from the group consisting of miR-Let-7a, 7, 9, 124a, 125a, 125b, 128a, 132, 134, 135a, 137, 138, 181a, 181c, 182, 184, 211, 212, 213, 218, 219, 222, 323-3p, 338-5p, 369, 381, 382, 425, 433-5p, 485-5p, 491-5p, 539, 541, 543, 656, 874, 935, and 9*.

50. primers or probes specific for at least two miRNAs selected from the group consisting of miR-7, 9, 98, 124a, 125a, 125b, 128a, 132, 134, 135a, 137, 138, 154, 182, 183, 213, 218, 323-3p, 329, 337, 369-3p, 369-5p, 370, 381, 382, 409-3p, 425, 433-5p, 483-3p, 485-5p, 487b, 494, 495, 496, 541, 543, 656, 668, 874, 889, 935, 939, and 9*.

51. primers or probes specific for miR-330-3p and miR-342.

52. primers or probes specific for at least two miRNAs selected from the group consisting of miR-96, 99a, 103, 181b, 221, 324-5p, 328, 330, 331, 335-5p, and 383.

53. primers or probes specific for at least two miRNAs selected from the group consisting of miR-103, 181b, 204, 432, and 489.

54. primers or probes specific for miR-103 and miR-183.

55. primers or probes specific for miR-451 and miR-486.

56. primers or probes specific for at least two miRNAs selected from the group consisting of miR-22, 133a, 221, 222, and 30e*.

57. primers or probes specific for at least two miRNAs selected from the group consisting of miR-1, 30a-3p, 30e-3p, 133b, 197, 208a, 208b, 302a, 302c, 367, 378, and 499-5p.

58. primers or probes specific for at least two miRNAs selected from the group consisting of miR-Let-7, 10a, 17, 18a, 19a, 19b, 20a, 21, 23a, 24, 27a, 27b, 29a, 31, 34a, 92, 98, 100, 106a, 126, 130a, 143, 145, 146a, 199a-3p, 210, 345, 365, 382, 409-3p, 431, 484, 495, 532-5p, 939, 27a*, 30a*, 93*, 126*, 130b*, and 222*.

59. primers or probes specific for at least two miRNAs selected from the group consisting of miR-Let-7a, Let-7c, 10b, 26a, 100, 125a, 125b, 130a, 140, 143, 145, 195, 196b, 199a, 199b, 204, 214, 222, 424, 517c, and 199a*.

60. primers or probes specific for at least two miRNAs selected from the group consisting of miR-10a, 31, 34b, 34c, 135a, 135b, and 449.

61. primers or probes specific for at least two miRNAs selected from the group consisting of miR-Let-7b, 127, 202, 298, 382, 503, 672, 741, 742, 883-3p, and 202*.

62. primers or probes specific for miR-99a and miR-939.

63. primers or probes specific for at least two miRNAs selected from the group consisting of miR-Let-7g, 101, 134, 186, 197, 218, 320, 497, and 154*.

64. primers or probes specific for at least two miRNAs selected from the group consisting of miR-126, 146, 205, 206, 335, 339-5p, 378, 516-5p, 519c, 520g, 520h, 525, and 1246.

65. primers or probes specific for at least two miRNAs selected from the group consisting of miR-7, 127, and 493*.

66. primers or probes specific for at least two miRNAs selected from the group consisting of miR-Let-7i, 1, 135a, 135b, 206, and 345.

67. primers or probes specific for at least two miRNAs selected from the group consisting of miR-Let-7g, 15a, 26b, 27a, 99b, 124, 132, 134, 137, 139, 152, 181a, 187, 195, 192, 202, 299, 302b, 323, 324-3p, 324-5p, 328, 330-3p, 331, 335, 340, 365, 369-3p, 375, 379, 382, 409-5p, 429, 431, 432, 455-5p, 483-5p, 514, 126*, 182*, and 202*.

68. primers or probes specific for at least two miRNAs selected from the group consisting of miR-9, 21, 130b, 184, 195, 216a, 216b, 217, 376a, 376c, 497, and 939.

69. primers or probes specific for at least two miRNAs selected from the group consisting of miR-15a, 15b, 142-3p, 142-5p, 146, 150, 181a, 181b, 181d, 205, 342, and 423.

70. primers or probes specific for at least two miRNAs selected from the group consisting of miR-126, 139, 155, 223, and 302b.

71. primers or probes specific for at least two miRNAs selected from the group consisting of miR-17-5p, 20b, 106a, 106b, 149, 155, 181c, 182, 183, and 213.

72. primers or probes specific for at least two miRNAs selected from the group consisting of miR-Let-7g, 20b, 21, 106b, 140, 146b, and 431.

73. primers or probes specific for at least two miRNAs selected from the group consisting of miR-Let-7g, 9, 17, 19b, 20a, 31, 106a, 124a, 124b, 128a, 137, 186, 191, 197, 222, 223, 328, 431, 454, 484, 766, 27*, and 223*.

74. primers or probes specific for at least two miRNAs selected from the group consisting of miR-155, 223, and 424.

75. primers or probes specific for at least two miRNAs selected from the group consisting of miR-Let-7 family, 10b, 17-92 family, 21, 29a, 31, 34a, 106a,b, 126, 146a,b, 155, 184, 195, 200/141 family, 210, 373, 375, 423-5p, 451, and 486.

76. primers or probes specific for at least two miRNAs selected from the group consisting of miR-21, 31, 34a, 125-5p, 125b, 126, 146a,b, 150, 155, 221, 222, and 223.

77. primers or probes specific for at least two miRNAs selected from the group consisting of miR-270, 373, and 424.

78. (i) primers or probes specific for at least one miRNA selected from the group consisting of miR-128, miR-132, and miR-874, and (ii) primers or probes specific for at least one miRNA selected from the group consisting of miR-9, miR-181a, miR-491-5p, and miR-141.

79. (i) primers or probes specific for at least one miRNA selected from the group consisting of miR-134, miR-323-3p, and miR-382, and (ii) primers or probes specific for at least one miRNA selected from the group consisting of mir-127 and miR-370.

80. (i) primers or probes specific for at least one miRNA selected from the group consisting of miR-34b, miR-486-5p, and miR-192, and (ii) primers or probes specific for at least one miRNA selected from the group consisting of miR-142-5p, miR-146b-5p, miR-155, miR-223, and miR-409-3p.

81. primers or probes specific for miR-34b and miR-155.

82. primers or probes specific for miR-146b-5p and at least one of miR-486b-5p and miR-192.

83. (i) primers or probes specific for at least one miRNA selected from the group consisting of miR-192, miR-194, miR-203, and miR-215, and (ii) primers or probes specific for at least one miRNA selected from the group consisting of miR-30e-3p, miR-145, and miR-148a.

84. (i) primers or probes specific for miR-215 and (ii) primers or probes specific for at least one miRNA selected from the group consisting of miR-30e-3p, miR-194, and miR-203.

85. (i) primers or probes specific for miR-203 and (ii) primers or probes specific for at least one of miR-148a and miR-192.

86. (i) primers or probes specific for miR-194 and (ii) primers or probes specific for at least one of miR-148a and miR192.

87. primers or probes specific for at least one miRNA pair selected from the group consisting of miR-194/miR-145, miR-194/miR148a, miR-194/miR-30e-3p, miR-215/miR-203, miR-203/miR-30e-3p, miR-203/miR148a, miR-192/miR-145, miR-192/miR148a, and miR-192/miR-30e-3p.

88. primers or probes specific for at least one miRNA pair selected from the group consisting of miR-192/miR-126, miR-155/miR-126, miR-145/miR-126, miR-155/miR-30e-3p, miR-192/miR-30e-3p, miR-155/miR-409-3p, miR-486-5p/miR-17-5p, miR-155/miR-17-5p, miR-192/miR-17-5p, miR-146b-5p/miR-31, miR-155/miR-31, miR-192/miR-31, miR-486-5p/miR-155, miR-192/miR-155, miR-145/miR-155, miR-146b-5p/miR-155, miR-486-5p/miR-203, miR-192/miR-203, miR-145/miR-203, miR-192/miR-215, and miR-155/miR-215.

89. primers or probes specific for at least one miRNA pair selected from the group consisting of miR-17-5p/miR-155, miR-192/miR-155, miR-215/miR-155, miR-192/miR-30e-3p, miR-155/miR-30e-3p, and miR-146b-5p/miR-30e-3p.

Kits of the invention can further comprise miRNA isolation or purification means.

Kits of the invention can further comprise instructions for use.

## DETAILED DESCRIPTION OF THE INVENTION

The present invention is based on the idea of shifting the paradigm in the area of clinical screening and diagnostics from disease-specific screening tests to the Universal Screening Test(s) (UST), which will detect pathology of a particular organ system, such as gastrointestinal, nervous, hematological, etc., or of a particular organ or tissue or cell type, but will be disease-non-specific. Additionally such test will be capable to differentiate some broad types of pathologic processes, e.g. inflammatory diseases and tumors. After the pathology is detected, disease-specific tests can be applied for a more specific diagnosis. Such a test(s) can also be used for drug screening as well as for evaluating toxicity of various compounds and environmental factors (e.g. during drug development or clinical trials). The instant invention is based on the use of organ system-, organ-, tissue-, and/or cell type-enriched miRNA in bodily fluids as biomarkers of organ and/or tissue and/or cell pathology, description of the basis for such miRNA selection, and the methods of the UST interpretation. The UST of the invention can also include miRNA biomarkers for some general pathological processes, such as, e.g., hypoxia, inflammation, carcinogenesis, etc.

The present invention provides a novel noninvasive or minimally invasive method for early, preferably before clinical manifestation, detection of pathological changes (without defining a specific disease) in an organ system or in a particular organ/tissue/cell type in a subject, said method comprising determination of the levels of organ system/organ/tissue-enriched miRNA in a bodily fluid (e.g. plasma, serum, urine, saliva, or other bodily fluid) of said subject as compared to a control. Specifically, the method comprises:

a. measuring levels of miRNAs enriched in various organ systems/organs/tissues/cell types in a bodily fluid sample collected from the subject;

b. measuring levels of preselected normalizer miRNAs in the same bodily fluid sample collected from the subject;

c. calculating the ratios of the levels of the miRNAs measured in steps (a) and (b);

d. comparing the ratios of the levels of the miRNAs calculated in step (c) with the corresponding control ratios, and

e. (i) identifying the subject as being afflicted with a pathology of a particular organ system/organ/tissue/cell type when the ratios of the levels of the miRNAs enriched in said organ system to their respective miRNA normalizers calculated in step (c) are higher than the corresponding control ratios or (ii) identifying the subject as not being afflicted with a pathology of said organ system/organ/tissue/cell type when the ratios of the levels of the miRNAs enriched in said organ system to their respective miRNA normalizers calculated in step (c) are not higher than the corresponding control ratios.

If positive for a pathology, such UST should be followed by tests specific for various known pathologies of the organ system/organ/tissue/cell type identified by UST.

The present invention also provides methods for selecting potential miRNA biomarkers. To reflect pathological changes in a particular organ system, organ, tissue, or cell type such biomarkers should be, first, enriched in one of those organ systems, organs, or tissues, and second, their concentrations in bodily fluids should be high enough to be detectable. Although not all miRNA have currently been identified and organ/tissue expression profiles for many of them are not known, published data (See, e.g., Hua et al., BMC Genomics 2009, 10:214; Liang et al., BMC Genomics. 2007, 8:166; Landgraf et al., Cell. 2007, 129:1401-1414; Lee et al., RNA. 2008, 14:35-42; http://ferrolab.dmi.unict.it/miro/; http://mips.helmholtz-muenchen.de/phenomir/) are sufficient to formulate major principles for selecting potential biomarkers:


- - 1, organ system/organ/tissue/cell type-enrichment. Although some
    miRNA are highly enriched in a particular organ or tissue, e.g.
    miR-122 in the liver and miR-124 in the brain, there is no known
    miRNA, which is 100% specific for one organ or tissue. Of course,
    the higher is miRNA enrichment in a given organ
    system/organ/tissue/cell type as compared to all other organ
    systems/organs/tissues/cell types, the better potential as a
    biomarker it has. Practically, if miRNA concentration in one organ
    is at least 4-5 times higher than in others, they can be selected as
    potential biomarkers for UST. For many organs, such miRNA can be
    found in the literature (See, e.g., Hua et al., BMC Genomics 2009,
    10:214; Liang et al., BMC Genomics. 2007, 8:166; Landgraf et al.,
    Cell. 2007, 129:1401-1414; Lee et al., RNA 2008, 14:35-42;
    http://ferrolab.dmi.unict.it/miro/;
    http://mips.helmholtz-muenchen.de/phenomir/). Table 1 represents
    miRNA enriched in various organs according to numerous published
    data. One can see that some miRNA are enriched in different organs
    of the same organ system. This is especially characteristic of
    gastrointestinal, nervous, genital, and hematological organs (Table
    2). Using these miRNA allows to design a test that will detect
    pathology in those systems but not in a particular organ. At the
    same time, there are miRNA enriched in one-two organs of a system,
    and these miRNA can be used for tests that can define the location
    of the pathology more precisely. The list of these miRNA is
    presented in the third column of Table 2. There are also miRNA
    enriched in two or even more organs, which usually developed during
    embryogenesis from the same cell types. In such cases the increase
    of the miRNA concentration in bodily fluid cannot be interpreted
    unambiguously. However, combination of several miRNA enriched in
    different sets of organs resolves this problem, and the present
    invention also includes a computer implemented method for analysis
    of data obtained with such several miRNA. It is also important to
    note that miRNA enrichment in the second organ will not
    significantly affect the test results, if chances of those miRNA to
    appear in bodily fluids are low. For example, many miRNA are
    enriched (in addition to other organs) in the skin, but if those
    miRNA are located in epidermis their chances of appearing in the
    bloodstream from the skin are very low, and the method of the
    invention uses them.
  - 2. miRNA expression level. The test development is easier and its
    sensitivity is higher if the concentration of a potential miRNA
    biomarker(s) in the target organ system/organ/tissue/cell type is
    sufficiently high, since one can expect that more miRNA biomarker
    molecules appear in bodily fluids. Thus, if there is a choice
    between many potential miRNA biomarkers, those miRNA should be
    selected, which not only are enriched in the target but are also
    highly expressed (e.g., ≧1000 copies per cell). This is especially
    important for small organs or their parts or specific cell types
    within organs, such as pancreatic α-cell islets. It does not mean
    that miRNA which are not highly expressed cannot be used for the UST
    development but the detection of miRNA expressed at low levels may
    need larger volumes of bodily fluids and more sensitive techniques
    for miRNA quantitation. At the same time, since miRNA concentration
    in a bodily fluid also depends on the effectiveness of its secretion
    from cells into extracellular space and transport to the bodily
    fluid (see next section), as many as possible organ
    system/organ/tissue/cell type-enriched miRNA should be analyzed for
    the experimental selection of the most promising biomarkers.
  - 3. miRNA secretion. There are many ways for cell-free miRNA to
    appear in bodily fluids (Hunter et al., PLoS ONE. 2008, 3:e3694;
    Wang et al., Nucleic Acids Res. 2010, 38:7248-7259; Pigati et al.,
    PLoS ONE. 2010, 5:e13515; Gupta et al., Circ. Cardiovasc. Genet.
    2010, 3:484-488; Iguchi et al., Commun Integr. Biol. 2010,
    3:478-481; Kosaka et al., J. Biol. Chem. 2010, 285:17442-17452).
    miRNA can appear in extracellular space and then in bodily fluids as
    a result of: (i) cell death and cellular membrane
    permeabilization; (ii) destruction of some cellular compartments,
    such as axons, dendrites, and spines in neurons; (iii) exocytosis
    (Skog et al. Nat Cell Biol., 2008, 10:1470-1476); (iv) blebbing
    (Charras et al., Biophys. J. 2008, 94:1836-1853; Fackler, Grosse, J.
    Cell Biol. 2008, 181:879-884); (v) secretion of free or
    protein-bound miRNA (Wang et al., Nucleic Acids Res. 2010,
    38:7248-7259). The latter mechanism provides for more than 50% of
    cell-free miRNA into extracellular medium from live cells. The
    secretion of miRNA is selective and the ratios of various miRNA
    concentrations in cells and extracellular medium are different. The
    selectivity of the miRNA secretion is very important for selecting
    potential biomarkers. Since the mechanisms of miRNA secretion from
    normal cells and during pathology development (Rabinowits et al.
    Clin Lung Cancer, 2009, 10:42-46) has not been investigated, it is
    necessary to analyze more miRNA, keeping in mind that some of them,
    which look promising due to high expression and enrichment in a
    target organ/tissue can be secreted at a low level and vice versa.
    Also it was recently demonstrated that some miRNA, e.g., miR-451 and
    miR-1246, which are secreted at a very low level from normal cells,
    can be secreted much more effectively from pathological cells
    (Pigati et al. PLoS ONE, 2010, 5:e13515). These miRNA can also be
    analyzed as potential biomarkers even if they are not highly
    enriched in a particular organ system, organ or tissue, since their
    combination with more organ/tissue-specific miRNA will provide
    additional information useful for detecting a pathology.

It is important to remember that many miRNA have not been discovered yet and the expression profiles of many recently discovered miRNA in various tissues have not been analyzed. In addition, many organs, tissues, and especially cell types were not tested for expression of already known miRNA. Thus, although UST development can be initiated on the basis of already published data for many organs and tissues, additional search for new biomarkers will increase the informative value of UST. First, the miRNA expression profile of all known miRNA in various organ systems/organs/tissues/cell types should be analyzed to define new organ system-/organ-/tissue-enriched miRNA (e.g., using RT-PCR, which currently is the most sensitive and least variable technique for miRNA quantitative measurement). Second, the expression profiles of all newly discovered miRNA should be analyzed as described above. Third, since all organs are composed of various cell types, the expression of miRNA enriched in particular organs should be additionally analyzed to find out the cell type, in which these miRNA are enriched. Currently, the best technique for such a study is in situ hybridization (ISH). Such information, as currently available for various miRNA, has been included in Tables 1 and 2. For example, information on pancreatic β-cells-enriched miRNA is included in addition to pancreas-enriched miRNA, as well as enrichment of some miRNA in neurons located in various brain areas.

Several levels or generations of UST are possible. The first one can be developed for the systems of the human body (gastrointestinal, genitourinary, brain, cardiovascular, etc.). Next level of testing can focus on organs, tissues, cell types, and so on. Depending on clinical demands and economic advantages, there are at least two possible versions of UST and their practical applications: (i) first, the UST for human body systems can be performed and then, if a pathology in one or several systems has been detected, tests for organs/tissues of those systems will be performed; (ii) second, organ/tissue/cell type UST can be directly applied for screening purposes.

All organs and tissues consist of several cell types with different origin, function, and potential pathologies. Although the present invention is mainly focused on the level of organs and tissues, and their systems, the same approach can be used for developing UST covering various cell types, when sufficient information on miRNA expression profiles in various cell types becomes available. Currently, such information has been obtained for pancreatic β-cells (Tables 1 and 2), which makes it feasible to include the β-cell-enriched miRNA in UST, for example, for early detection of type 1 diabetes. miRNA markers for B- and T-lymphocyte are also available and their inclusion into UST will be helpful for early detection of pathologies that involve these cell types.

Although the present invention is focused on the UST development and its use for early detection of pathologies independent on their nature but specific to a particular organ system, organ, tissue, and/or cell type, such a test(s) can also include miRNA biomarkers, whose expression increase is characteristic of most common general pathologies, such as hypoxia, inflammation, and cancer (Table 3). Many more potential miRNA biomarkers for these pathologies are expected to be described soon.

Various useful miRNA biomarkers and normalizers described in the Examples, below, can be summarized in the following Tables:

Examples of useful methods for measuring miRNA level in bodily fluids include hybridization with selective probes (e.g., using Northern blotting, bead-based flow-cytometry, oligonucleotide microchip [microarray], or solution hybridization assays such as Ambion mirVana mirna Detection Kit), polymerase chain reaction(PCR)-based detection (e.g., stem-loop reverse transcription-polymerase chain reaction [RT-PCR], quantitative RT-PCR based array method [qPCR-array]), or direct sequencing by one of the next generation sequencing technologies (e.g., Helicos small RNA sequencing, miRNA BeadArray (Illumina), Roche 454 (FLX-Titanium), and ABI SOLiD). For review of additional applicable techniques see, e.g., Chen et al., BMC Genomics, 2009, 10:407; Kong et al., J Cell Physiol. 2009; 218:22-25. Because many tissue/organ-specific miRNA are present in bodily fluids in much lower concentrations than ubiquitous miRNA, and a screening test should be capable of detecting early disease-associated changes, which can be relatively low, it is important to use, at least at the proof-of-principle stage, the most sensitive and minimally variable technique for measuring miRNA levels. In the present invention RT-PCR, which detects larger numbers of plasma miRNA and is significantly more robust than various array techniques, was used throughout the study. This does not exclude use of other techniques of miRNA analysis in a resulting clinical test.

In some embodiments, miRNA are purified prior to quantification. miRNA can be isolated and purified from bodily fluids by various methods, including the use of commercial kits (e.g., miRNeasy kit [Qiagen], MirVana RNA isolation kit [Ambion/ABI], miRACLE [Agilent], High Pure miRNA isolation kit [Roche], and miRNA Purification kit [Norgen Biotek Corp.]), Trizol extraction(see Example 1, below), concentration and purification on anion-exchangers, magnetic beads covered by RNA-binding substances, or adsorption of certain miRNA on complementary oligonucleotides.

In some embodiments, miRNA degradation in bodily fluid samples and/or during small RNA purification is reduced or eliminated. Useful methods for reducing or eliminating miRNA degradation include, without limitation, adding RNase inhibitors (e.g., RNasin Plus [Promega], SUPERase-In [ABI], etc.), use of guanidine chloride, guanidine isothiocyanate, N-lauroylsarcosine, sodium dodecyl sulphate (SDS), or a combination thereof. Also, when working with urine samples, lower risk of miRNA degradation can be achieved when the sample has been held in the bladder for a shorter time (e.g., less than 4 hours). Reducing miRNA degradation in bodily fluid samples is particularly important when sample storage and transportation is required prior to miRNA quantification.

The present invention also provides several approaches for normalization of miRNA concentrations detected in bodily fluids. To account for possible losses of a given miRNA during purification, potential RT-PCR inhibition, miRNA contaminants derived from dying or damaged blood or urine cells during sample isolation and treatment, variations in kidney filtration, etc., various methods of experimental data normalization can be employed. For example, one or more of the following normalization methods can be used in the present invention:


- - a) Synthetic miRNA (e.g., miRNA absent in human cells)
    oligonucleotides can be synthesized and used as controls for losses
    during purification and RT-PCR inhibition (by adding those to bodily
    fluid samples before miRNA purification).
  - b) Concentration of a target miRNA can be normalized to one of
    ubiquitous miRNA (e.g., miR-16, miR-30e, miR-103 and others), small
    nucleolar RNAs (snoRNAs), U6 small nuclear RNA (U6 RNA).
  - c) Another approach is based on normalization of target miRNA
    concentrations to miRNA that is expressed in numerous tissues but
    not in a target one. For example miR-10a and miR-141 are expressed
    in the brain at much lower level than in other organs, and
    miR-409-3p is expressed in the lung at much lower level than in
    other tissues. This approach decreases chances a normalizer miRNA
    being changed due to a target pathology.
  - d) Concentration of a target miRNA can also be normalized to miRNA
    in other organ(s) (e.g. miRNA enriched in heart can be normalized to
    miRNA enriched in colon or brain and vice versa).
  - e) Normalization of a target miRNA per other miRNA enriched in the
    same organ, tissue, or organ system can be effective if biomarker
    and normalizer miRNA are expressed in different cell types or
    different organs of the same system.
  - f) Normalization of a target miRNA per another miRNA from the same
    organ and tissue if their expression and/or secretion are effected
    differently by a pathology.
  - g) Normalization of a target miRNA per a tissue-enriched miRNA is
    useful when, for example, changes in the expression and/or secretion
    of a target miRNA are characteristic of a particular pathology
    (cancer, inflammation, etc.) but the miRNA is not enriched in the
    organ/tissue of interest. In such case normalization per a
    tissue-enriched miRNA will be helpful for connecting a pathology to
    a particular organ or organ system.
  - h) Normalization per the average of several miRNA normalizers or, if
    many, e.g. \>15, miRNA are analyzed normalization per the average of
    all miRNA tested.
  - i) To account for variations in kidney filtration (when working with
    urine samples), miRNA concentration in urine can be normalized on
    creatinine and/or albumin level.

Normalization calculus along with further data processing can be done by UST software, consisting of three parts:


- - Data Base Management System (DBMS) used for access and maintenance
    of K-base and I-base. It can be one of the industry standard
    systems, like, but not limited to, SQL server, oracle, MySQL, etc.
  - Application for Screening Test(s) development used on the research
    stage. Functions of this application include selection of proper
    miRNA and construction of D-sets, creation of pathology sets, etc.
    The application is desktop-type and includes:(i) user interface for
    entering/inspecting data in K-base and I-base and to control
    algorithm execution; (ii) Data processing for Learning and
    Classification on the developmental stage; (iii) Service programs
    for D-set consistency check, etc.; (iv)Scripts for table
    creation/verification/modification in K-base and I-base. Algorithm
    for data processing (Algorithm1) includes Learning and
    Classification parts.
  - Application for Screening Test processing in the clinical use of
    UST. Functions of this application include Training using large
    amount of data and Classification on actual Screening Test data. The
    application can be desktop-type, or Web application and
    includes: (i) user interface for entering/inspecting data in I-base,
    and to control algorithm execution; (ii) data processing for
    Learning and/or Classification of subject data; (iii) Scripts for
    table creation/verification/modification in K-base and I-base; these
    tables are different from those in application for Screening Test
    development. Algorithm for data processing (Algorithm2), as in
    Algorithm 1 includes Learning and Classification and can differ from
    those in Algorithm1.

As discussed in details below in the Examples, the proposed approach was validated by analysis of plasma miRNA from patients with different diseases of several organs.

1. Nervous system:


- - a) MCI (Mild Cognitive Impairment);
  - b) Alzheimer's disease (AD).

2. Gastrointestinal system:


- - a) Esophageal cancer;
  - b) Gastric cancer;
  - c) Colon cancer;
  - d) Crohn's disease.

3. Respiratory system:


- - a) Asthma;
  - b) Pneumonia;
  - c) COPD (Chronic Obstructive Pulmonary Disease);
  - d) Lung cancer.

For selection of biomarkers and normalizer miRNA, plasma concentrations of many miRNA enriched in respective organs or organ systems were analyzed by RT-PCR. Ubiquitous miRNA and miRNA expressed in numerous organs but under-expressed in organs of interest were also analyzed as potential normalizers. All miRNA analyzed for respective organs or organ systems were tested as potential biomarkers and normalizers, and combinations, which provided statistically significant differentiation between a pathology and respective controls, were selected as most promising.

Nervous System

Early detection of Mild Cognitive Imparment (MCI) and Alzheimer's disease (AD) was used for validating the proposed approach to the development of a screening test for the nervous system. AD is the most common neurodegenerative disease, which comprise a large group of pathologies caused by metabolic changes in brain cells, loss of synapses and other compartments of neurons, and finally neuronal death (for review see Neurodegenerative diseases: From Molecular Concepts to Therapeutic Targets. Editors: R. von Bernhardi, N. C. Inestrosa, Nova Publishers, 2008). AD is characterized by neurite retraction, axonal transport defects, synaptic dysfunction, synaptic loss, and finally by neuronal death in several disease-specific areas of the brain, such as hippocampus and cortex (See, e.g., Crews, Masliah, Human Mol Gen., 2010, 19:R12-R20; Bredesen, Molecular Neurodegeneration 2009, 4:27; Nimmrich and Ebert, Rev Neurosci. 2009, 20:1-12; Yoshiyama et al., Neuron. 2007, 53:337-351; Wishart et al., J Neuropathol Exp Neurol. 2006, 65:733-739; Gylys et al., Neurochem Int. 2004; 44:125-131; Conforti et al., Trends Neurosci. 2007, 30:159-166; Revuelta, et al. Am J Alzheimers Dis Other Demen 2008, 23: 97-102).

The first symptomatic stage of AD that is manifested by mild clinical symptoms is MCI, which is usually defined as an intermediate state between normal aging and dementia (DeCarli, Lancet Neurol., 2003, 2:15-21; Stephan et al., Alzheimer's Res Therapy, 2009, 1:1-9; Apostolova et al., Human Brain Mapping, 2010, 31:786-797). MCI is a heterogeneous syndrome that may lead to different outcomes. Up to 40% of MCI patients revert to normal status (Larrieu et al., Neurology, 2002, 59:1594-1599; Brooks, Loewenstein, Alzheimer's Res Therapy, 2010, 2:28-36), and autopsy studies demonstrate that a substantial percentage of MCI patients do not have evidence of AD pathology (Jicha et al., Arch Neurol, 2006, 63:674-681; Khan, Alkon, Neurobiol. Aging, 2010, 31:889-900). About 60% of MCI patients convert to dementia at a rate of 10-15% annually (Petersen et al., Arch Neurol. 2001, 58:1985-1992; Apostolova et al., Human Brain Mapping, 2010, 31:786-797). Although AD is the most common cause of dementia, about 20% of MCI patients who progress to dementia are diagnosed not with AD but other neurodegenerative diseases, such as vascular, Lewy body, Huntington, Parkinson, and other dementias (Jicha et al., Arch Neurol, 2006, 63:674-681; Stephan et al., Alzheimer's Res Therapy, 2009, 1:1-9).

Thus, detection of MCI and AD by analysis of circulating miRNA supports the idea of developing an organ/system-specific test for various pathologies. As discussed in detail in the Examples for selection of biomarker and normalizer miRNA, concentrations of many brain-enriched miRNA, including neurite/synapse-enriched ones, in plasma of MCI and AD patients and age-matched control group were analyzed by RT-PCR. All selected miRNA were tested as potential biomarkers and normalizers and combinations, which provided statistically significant differentiation between MCI patients and age-matched controls, were identified as most promising. The data have demonstrated that the most effective potential biomarkers are neurite/synapse miRNA and the most effective normalizers are other brain-enriched miRNA, although other miRNA can also be used for normalization. Two families of biomarkers, miR-132 family and miR-134 family, and several normalizers have demonstrated the highest sensitivity (84%-92%) and specificity (84%-90%) in MCI detection. High correlation between members of miR-134 family can be easily explained by the fact that all members of this family, namely miR-134, miR-323-3p and miR-382, belong to the same cluster and are expressed in the same cell types. Close relationships between members of miR-132 family, namely miR-128, miR-132 and miR-874, have not been described before. It is also interesting that miR-132 and miR-134 biomarker families yield better results with different normalizers. miR-132 family works better than miR-134 family with normalizers miR-491-5p, miR-181a, miR-9, and miR-141. On the other hand, miR-134 family demonstrates better results than miR-132 family with normalizers miR-370 and miR-127.

Thus, the heterogeneous MCI syndrome and AD can be detected by analysis of cell-free circulating brain-enriched miRNA in the plasma.

Since different brain areas are involved in various neurodegenerative diseases leading to the development of dementia (Geldmacher & Whitehouse, Neurology. 1997, 48:S2-9; Levy & Chelune, J Geriatr Psychiatry Neurol. 2007 20:227-238; Gong & Lippa, Am J Alzheimer's Dis Other Demen, 2010, 25:547-555) and due to diverse miRNA expression profile in various brain areas (Landgraf et al., Cell. 2007, 129:1401-1414; The miR-Ontology Data Base: http://ferrolab.dmi.unict.it/miro/), analysis of other brain-enriched miRNA would be helpful for distinguishing changes and processes caused by distinct neuronal pathologies in the brain.

Gastrointestinal System

Plasma samples of patients with stages 1 and 2 cancers of three gastrointestinal (GI) organs (esophagus. stomach, and colon) and with the Crohn's disease (an inflammatory bowel disease that may affect any part of the GI tract) were used for validating the proposed approach to the development of a screening test for the GI system. Plasma concentrations of miRNA enriched in GI system or in particular GI organs, e.g. miR-215 highly enriched in colon and small intestine and miR-203, enriched in esophagus and to a lesser degree in stomach, as well as of ubiquitous miR-30e-3p were measured. Ratios of all possible miRNA pairs were calculated to find the most promising biomarker/normalizer combinations. The obtained data have demonstrated that miR-192, miR-194, miR-203 and miR-215 are the most effective biomarkers, and miR-145, miR-148a and miR-30e-3p are the most effective normalizers. Biomarker/normalizer ratios effectively differentiate patients with all studied diseases from controls. miR-203, highly enriched in esophagus and stomach, is especially effective in detecting cancers of these organs, and miR-215, highly enriched in column is most effective in differentiating patients with colon cancer and Crohn's disease from controls. Combination of two or more biomarker/normalizer ratios can be used to increase specificity and sensitivity. For example, miR-192 and miR-203 normalized per miR-30e-3p effectively distinguished patients with all pathologies of GI system from controls with 94% sensitivity and 100% specificity. It is important that all tested tumors were stage 1 or 2, which means that the proposed approach can be effectively used for screening and early diagnosis. Then pairs of different cancers as well as Crohn's disease versus all cancers of the GI system were compared in greater details. As a result the following biomarker/normalizer ratios capable of distinguishing particular pathologies have been found:


- - 1. Crohn's disease versus esophageal, gastric and colorectal
    cancers: miR-194/miR-148; miR-215/miR-30e-3p; miR-215/miR-194;
    miR-203/miR-148a; miR-192/miR-203; miR-215/miR-203 and
    miR-194/miR-192.
  - 2. Esophageal cancer versus gastric cancer: miR-194/miR-145;
    miR194/miR-148a; miR194/miR-30e-3p.
  - 3. Esophageal cancer versus colorectal cancer: miR-192/miR-145;
    miR192/miR-148a; miR192/miR-30e-3p.
  - 4. Gastric cancer versus colorectal cancer: miR-203/30e-3p;
    miR-203/miR-148a; miR-215/miR-203.
  - Thus, analysis of plasma concentrations of miRNA enriched in organs
    of the GI system is effective for: (i) detection of tumors and
    inflammatory conditions, such as Crohn's disease, in esophagus,
    stomach and colon; (ii) differentiation of an inflammatory disease
    from cancers; (iii) differentiation of cancers located in various
    organs of the GI system.

Respiratory System

Plasma samples of patients with four diseases, namely asthma, pneumonia, Chronic Obstructive Pulmonary Disease (COPD), and Non-Small Cell Lung Cancer (NSCLC, stages 1-4), were used for validating the proposed approach to the development of a screening test for the respiratory system. Since enrolled patients with asthma and pneumonia were non-smokers and patients with COPD and NSCLC were smokers, plasma samples were also collected from two distinct control groups—smokers and non-smokers. Plasma concentrations of lung-enriched miRNA and of miR-409-3p, which is present in many organs but is under-expressed in the lung, were measured. Again, as described above for the GI system, ratios of all possible miRNA pairs were calculated to find the most promising biomarker/normalizer combinations.

miR-34b and miR-486-5p, which are highly enriched in the lung, have been found to be effective biomarkers that normalized per miR-409-3p differentiated patients with asthma and pneumonia from non-smoking controls and patients with COPD and NSCLC from smoking controls. Other effective normalizers are lung-enriched miR-142-5p, miR-146b-5p, miR-155 and miR-223, potentially due to their downregulation in lung pathologies (Liu X. et al. Clin. Cancer Res. 2009, 15:1177-1183; Miko E. et al., Exp. Lung Res. 2009, 35:646-664; Halappanavar S. et al. Toxicology 2011, 285: 133-141; Heegaard N H et al. Int. J. Cancer, 2012, 130:1378-1386). Unexpectedly, miR-192 also behaved as an effective biomarker for pulmonary pathologies. Since this miRNA was also shown to be an effective biomarker for diseases of the GI system, it is reasonable to suggest that expression or/and secretion of miR-192 is increased due to inflammation or tumor development processes (Benjamin H. et al., J. Mol. Diagn. 2010, 12:771-779; Lan H Y. Clin. Exp. Pharmacol. Physiol. 2011 Dec. 28 [Epub ahead of print]; Luzna P. et al. Diagn. Pathol. 2011, 6:114; Wu Q. et al. J. Biomed. Biotechnol. 2011, Epub May 26; Zhou J. et al., J. Clin. Oncol. 2011, 29:4781-4788).

Since miRNA biomarker/miRNA normalizer ratios were not different for smoking and non-smoking controls, the four pathologies were compared with the combined controls (smoking and non-smoking subjects). Obtained data demonstrated that patients with all four analyzed pathologies can be effectively distinguished from such combined controls by various sets of miRNA biomarkers and normalizers, e.g. miR-34b normalized per miR-409-3p, miR-486-5p normalized per miR-223, or miR-192 normalized per miR-155. There were also other effective sets of miRNA biomarkers and normalizers.

The ability of various combinations of miRNA biomarkers and normalizers to distinguish NSCLC from such inflammatory diseases as asthma, pneumonia and COPD was also analyzed. The results demonstrated that patients with NSCLC were effectively differentiated from patients with inflammatory diseases using ratios of miR-34b to miR-155, miR-486-5p to miR-146b-5p or to miR-142-5p, miR-192 to miR-146b-5p. There were other effective combinations of miRNA biomarkers and normalizers.

Thus, pathologies of the pulmonary system can be effectively detected by analysis cell-free circulating miRNA in plasma, if lung-enriched miRNA are used as biomarkers, or normalizers, or both. Some biomarker/normalizer combinations can also effectively differentiate cancer patients from patients with inflammatory pulmonary diseases.

Distinction of Pathologies in Different organ Systems

To validate the idea that a test based on analysis of organ-enriched miRNA in bodily fluids is capable of detecting subjects with a pathology of a particular organ system, it is necessary to demonstrate that there are miRNA combinations capable to distinguish pathologies of distinct organ systems. As described in details in Examples, miRNA were purified from the plasma samples obtained from patients with diseases of the GI system (Crohn's disease and esophageal, gastric and colorectal cancers) and of the pulmonary system (asthma, pneumonia, COPD and NSCLC). Concentrations of lung-enriched and GI system-enriched miRNA as well as, several miRNA, involved in pathological processes of various organs, were analyzed. Additionally, ubiquitous miR-30e-3p and miR-409-3p were included in the study as potential normalizers. Concentrations of each miRNA were normalized per miR-30e-3p and miR-409-3p, as well as on each other, converted into Relative Quantity (RQ) of miRNA according the ABI protocol (2−ΔCt), and miRNA profiles characteristic of patients with the diseases of pulmonary and GI systems were compared. The data demonstrated that many miRNA pairs effectively distinguish patients with diseases of pulmonary and GI systems: miR-192/miR-126; miR-155/miR-126; miR-145/miR-126; miR-155/miR-30e-3p; miR-192/miR-30e-3p; miR-155/miR-409-3p; miR-486-5p/miR-17-5p; miR-155/miR-17-5p; miR-192/miR-17-5p; miR-146b-5p/miR-31; miR-155/miR-31; miR-192/miR-31; miR-486-5p/miR-155; miR-192/miR-155; miR-145/miR-155; miR-146b-5p/miR-155; 486-5p/miR-203; miR-192/miR-203; miR-145/miR-203; miR-192/miR-215; miR-155/miR-215. Combination of two miRNA pairs increases the test accuracy. For example, the combination of miR-145/miR-155 and miR-486-5p/miR-155 ratios distinguished patients with all pathologies of the GI system from patients with pulmonary diseases with 95% sensitivity, 90% specificity, and 93% accuracy.

Distinction of Different Pathologies in Various organ Systems

Due to characteristic changes in expression and secretion of some miRNA during inflammatory diseases and cancer development in various organs, it was hypothesized that analysis of their concentrations in bodily fluids could be used for distinguishing among these pathologies. The same plasma samples were used for miRNA purification and the same miRNA that were described in the previous section were analyzed. In this study the ability of various miRNA combinations to differentiate patients with inflammatory diseases (asthma, pneumonia, COPD and Crohn's disease) from patients with various cancers (esophageal, gastric, colorectal and non-small cell lung cancers) was investigated. The data demonstrate that several miRNA pairs effectively distinguish patients with inflammatory diseases from cancer patients: miR-17-5p/miR-155; mir-192/miR-155; miR-215/miR-155; miR146b-5p/miR155; miR192/miR-30e; miR-146b-5p/miR-30e-3p; miR155/miR-30e-3p. There are less miRNA pairs that differentiate inflammation diseases from cancers than miRNA pairs capable to differentiate diseases of the pulmonary system from diseases of GI system. First, changes in expression of many miRNA are characteristic to both pathology types. Second, in many cases carcinogenesis is accompanied by relatively prominent inflammation. Combination of two miRNA pairs increases the test accuracy. For example the combination of miR-146b-5p/miR-155 and miR-146b-5p/miR-30e-3p ratios distinguished all patients with inflammatory diseases from cancer patients with 80% sensitivity, 98% specificity, and 89% accuracy.

Thus, the results of the experiments presented in present invention support its main ideas. The analysis of plasma concentration of miRNA, enriched in a particular organ system or in an organ, differentiates: (i) organ system diseases from controls; (ii) pathologies of three organs of the gastrointestinal system; (iii) diseases of pulmonary and gastrointestinal system; (iv) cancers and inflammatory diseases.

Kits

In conjunction with the above diagnostic and screening methods, the present invention provides various kits comprising one or more primer and/or probe sets specific for the detection of target miRNA. Such kits can further include primer and/or probe sets specific for the detection of normalizer miRNA. Primer or probe combinations in kits can be based, for example, on various combinations of the molecules listed in Tables 1-11.

Such kits can be useful for direct miRNA detection in bodily fluid samples isolated from patients or can be used on purified RNA samples.

A kit of the invention can also provide reagents for primer extension and amplification reactions. For example, in some embodiments, the kit may further include one or more of the following components: a reverse transcriptase enzyme, a DNA polymerase enzyme (such as, e.g., a thermostable DNA polymerase), a polymerase chain reaction buffer, a reverse transcription buffer, and deoxynucleoside triphosphates (dNTPs). Alternatively (or in addition), a kit can include reagents for performing a hybridization assay. The detecting agents can include nucleotide analogs and/or a labeling moiety, e.g., directly detectable moiety such as a fluorophore (fluorochrome) or a radioactive isotope, or indirectly detectable moiety, such as a member of a binding pair, such as biotin, or an enzyme capable of catalyzing a non-soluble colorimetric or luminometric reaction. In addition, the kit may further include at least one container containing reagents for detection of electrophoresed nucleic acids. Such reagents include those which directly detect nucleic acids, such as fluorescent intercalating agent or silver staining reagents, or those reagents directed at detecting labeled nucleic acids, such as, but not limited to, ECL reagents. A kit can further include miRNA isolation or purification means as well as positive and negative controls. A kit can also include a notice associated therewith in a form prescribed by a governmental agency regulating the manufacture, use or sale of diagnostic kits. Detailed instructions for use, storage and trouble shooting may also be provided with the kit. A kit can also be optionally provided in a suitable housing that is preferably useful for robotic handling in a high throughput setting.

The components of the kit may be provided as dried powder(s). When reagents and/or components are provided as a dry powder, the powder can be reconstituted by the addition of a suitable solvent. It is envisioned that the solvent may also be provided in another container. The container will generally include at least one vial, test tube, flask, bottle, syringe, and/or other container means, into which the solvent is placed, optionally aliquoted. The kits may also comprise a second container means for containing a sterile, pharmaceutically acceptable buffer and/or other solvent.

Where there is more than one component in the kit, the kit also will generally contain a second, third, or other additional container into which the additional components may be separately placed. However, various combinations of components may be comprised in a container.

Such kits may also include components that preserve or maintain DNA or RNA, such as reagents that protect against nucleic acid degradation. Such components may be nuclease or RNase-free or protect against RNases, for example. Any of the compositions or reagents described herein may be components in a kit.

### DEFINITIONS

The term “screening test” is used herein to refer to a test, which is used for early detection of a disease, preferably prior to its clinical manifestation. Two types of screening tests are mainly described herein: (i) the screening tests that detect a pathology in a particular organ system/organ/tissue/cell type, and (ii) the screening tests that detect particular general pathological changes, such as, e.g., hypoxia, inflammation, or carcinogenesis, but do not localize this pathology to a particular organ system/organ/tissue/cell type. The term “universal screening test (UST)” refers to one or both of the above screening tests.

The term “organ system” refers to a group of related organs that work together to perform a certain task. For example, esophagus, stomach, duodenum, small and large intestines are organs of the digestive system. Salivary glands, pancreas, and liver also are components of the digestive system. At the same time, pancreatic β-islets, which secrete hormones, are also related to the endocrinal system.

Within the meaning of the present invention, the term “organ/tissue/cell type-enriched miRNA” refers to miRNA, which is present in increased amounts (e.g., at least 5-times higher concentrations) in a respective organ, tissue, or cell type, as compared to other organs, tissues or cell types, and can be a source of detectable amounts of miRNA in a bodily fluid being tested. The term “organ system-enriched miRNA” refers to miRNA, which is present in increased amounts (e.g., at least 5-times higher concentrations) in all or at least several organs of the respective organ system, as compared to other organ systems, organs, tissues or cell types, and can be a source of detectable amounts of miRNA in a bodily fluid being tested. To be useful in the diagnostic methods of the present invention, such organ system-/organ-/tissue-/cell type-enriched miRNA should be detectable in bodily fluids as a result of its release from cells and transport to said bodily fluids.

The term “a pathology” is used herein to refer to a non-specified pathology involving metabolic and/or structural changes in a respective organ, tissue or cell type associated with their dysfunction and/or partial destruction and/or loss. The term “associated with” is used to encompass any correlation, co-occurrence and any cause-and-effect relationship.

The terms “microRNA” or “miRNA” as used herein refer to a class of small approximately 22 nt long non-coding mature RNA molecules. They play important roles in the regulation of target genes by binding to complementary regions of messenger transcripts (mRNA) to repress their translation or regulate degradation(Griffiths-Jones Nucleic Acids Research, 2006, 34, Database issue: D140-D144). Frequently, one miRNA can target multiple mRNAs and one mRNA can be regulated by multiple miRNA targeting different regions of the 3′ UTR. Once bound to an mRNA, miRNA can modulate gene expression and protein production by affecting, e.g., mRNA translation and stability (Baek et al., Nature 455(7209):64 (2008); Selbach et al., Nature 455(7209):58 (2008); Ambros, 2004, Nature, 431, 350-355; Bartel, 2004, Cell, 116, 281-297; Cullen, 2004, Virus Research., 102, 3-9; He et al., 2004, Nat. Rev. Genet., 5, 522-531; and Ying et al., 2004, Gene, 342, 25-28). Examples of organ/tissue/cell-enriched miRNA useful in the methods of the present invention include, without limitation, miRNA listed in Table 1. Examples of organ system-enriched miRNA useful in the methods of the present invention include, without limitation, miRNA listed in Table 2, column 2 Examples of organ/tissue/cell type-enriched miRNA useful for more precise localization of a pathology in the methods of the present invention include, without limitation, miRNA enlisted in Table 2, column 3

Information on most currently known miRNA can be found in the miRNA database miRBase (available at the world wide web at mirbase.org). See also Burside et al., BMC Genomics 9:185 (2008); Williams et al., BMC Genomics 8:172 (2007); Landgraf et al., Cell 129:1401 (2007).

The term “miRNA array” as used herein refers to a multiplex technology used in molecular biology and in medicine. It consists of an arrayed series of multiple (e.g., thousands) microscopic spots of oligonucleotides, each containing a specific sequence (probe)complementary to a particular target miRNA. After probe-target hybridization under high-stringency conditions the resulting hybrids are usually detected and quantified by quantifying fluorophore-, silver-, or chemiluminescence-labeled targets to determine relative abundance of miRNA. In the methods of the present invention, both custom made and commercially available miRNA arrays can be used. Non-limiting examples of useful commercially available miRNA arrays (based on various methods of target labeling, hybrid detection and analysis) include arrays produced by Agilent, Illumina, Exiqon, Invitrogen, Febit, and LC Sciences.

The term “next generation sequencing technologies” as used herein broadly refers to sequencing methods which generate multiple sequencing reactions in parallel. This allows vastly increased throughput and yield of data. Non-limiting examples of commonly used next generation sequencing platforms include Helicos small RNA sequencing, miRNA BeadArray (Illumina), Roche 454 (FLX-Titanium), and ABI SOLiD.

An “individual” or “subject” or “animal”, as used herein, refers to humans, veterinary animals (e.g., cats, dogs, cows, horses, sheep, pigs, etc.) and experimental animal models of neurodegenerative diseases or other neuronal pathologies (see Examples, below). In a preferred embodiment, the subject is a human.

The term “urinary tract” refers to the organs and ducts, which participate in the secretion and elimination of urine from the body.

The term “purified” as used herein refers to material that has been isolated under conditions that reduce or eliminate the presence of unrelated materials, i.e., contaminants, including native materials from which the material is obtained. For example, RNA purification includes elimination of proteins, lipids, salts and other unrelated compounds present in bodily fluids.

As used herein, the term “substantially free” is used operationally, in the context of analytical testing of the material. Preferably, purified material substantially free of contaminants is at least 50% pure; more preferably, at least 90% pure, and still more preferably at least 99% pure. Purity can be evaluated by chromatography, gel electrophoresis, composition analysis, biological assay, and other methods known in the art.

As used herein, the term “similarly processed” refers to samples (e.g., bodily fluid samples or purified RNAs) which have been obtained using the same protocol.

The term “a control level” as used herein encompasses predetermined standards (e.g., a published value in a reference) as well as levels determined experimentally in similarly processed samples from control subjects (e.g., age and gender matched healthy subjects). Since the present invention describes screening tests that can be performed for the same patient periodically, prior data from the same individual can be used as “a control level”.

For differentiation between two pathologies, such as cancer versus inflammation or esophageal cancer versus colorectal cancer, the ratios of levels of miRNA in plasma will be compared not with predetermined control ratios but with predetermined ranges of ratios of miRNA levels in plasma, which is characteristic of the respective pathologies. To define these ratios, several hundred patients will be enrolled and levels of miRNAs of interest will be measured in their plasma samples. Then ratios of levels of various miRNA pairs will be calculated, which will provide the range of ratios characteristic of the respective pathology and covering, for example, 100%, 99% or 95% of pathology cases. The predetermined range used in the real clinical setting will be determined by demands to test sensitivity and specificity.

The term “about” or “approximately” means within a statistically meaningful range of a value. Such a range can be within an order of magnitude, preferably within 50%, more preferably within 20%, still more preferably within 10%, and even more preferably within 5% of a given value or range. The allowable variation encompassed by the term “about” or “approximately” depends on the particular system under study, and can be readily appreciated by one of ordinary skill in the art.

In accordance with the present invention there may be employed conventional molecular biology, microbiology, and recombinant DNA techniques within the skill of the art. Such techniques are explained fully in the literature. See, e.g., Sambrook, Fritsch & Maniatis, Molecular Cloning: A Laboratory Manual, Second Edition. Cold Spring Harbor, N.Y.: Cold Spring Harbor Laboratory Press, 1989 (herein “Sambrook et al., 1989”); DNA Cloning: A Practical Approach, Volumes I and II (D. N. Glover ed. 1985); Oligonucleotide Synthesis (M. J. Gait ed. 1984); Nucleic Acid Hybridization [B. D. Hames & S. J. Higgins eds. (1985)]; Transcription And Translation [B. D. Hames & S. J. Higgins, eds. (1984)]; Animal Cell Culture [R. I. Freshney, ed. (1986)]; Immobilized Cells And Enzymes [IRL Press, (1986)]; B. Perbal, A Practical Guide To Molecular Cloning (1984); Ausubel, F. M. et al. (eds.). Current Protocols in Molecular Biology. John Wiley & Sons, Inc., 1994. These techniques include site directed mutagenesis as described in Kunkel, Proc. Natl. Acad. Sci. USA 82: 488-492 (1985), U.S. Pat. No. 5,071,743, Fukuoka et al., Biochem. Biophys. Res. Commun. 263: 357-360 (1999); Kim and Maas, BioTech. 28: 196-198(2000); Parikh and Guengerich, BioTech. 24: 4 28-431 (1998); Ray and Nickoloff, BioTech. 13: 342-346 (1992); Wang et al., BioTech. 19: 556-559 (1995); Wang and Malcolm, BioTech. 26: 680-682 (1999); Xu and Gong, BioTech. 26: 639-641 (1999), U.S. Pat. Nos. 5,789,166 and 5,932,419, Hogrefe, Strategies 14. 3: 74-75 (2001),U.S. Pat. Nos. 5,702,931, 5,780,270, and 6,242,222, Angag and Schutz, Biotech. 30: 486-488 (2001), Wang and Wilkinson, Biotech. 29: 976-978 (2000), Kang et al., Biotech. 20: 44-46 (1996), Ogel and McPherson, Protein Engineer. 5: 467-468 (1992), Kirsch and Joly, Nucl. Acids. Res. 26: 1848-1850 (1998), Rhem and Hancock, J. Bacteriol. 178: 3346-3349 (1996), Boles and Miogsa, Curr. Genet. 28: 197-198(1995), Barrenttino et al., Nucl. Acids. Res. 22: 541-542 (1993), Tessier and Thomas, Meths. Molec. Biol. 57: 229-237, and Pons et al., Meth. Molec. Biol. 67: 209-218.

The term “Labeled Data” used for Training Data Set refers to results of analysis obtained on clinical samples from persons with known diagnosis. For example, for Labeled Data for miRNA targeting Liver, data should be collected from people with liver pathologies (label “Liver” and, e.g., “hepatitis” or “hepatocellular carcinoma”), and without any liver pathology (label “Control”).

As used herein, the term “D-Set” refers to a set of miRNA (biomarkers) selected for detection of a pathology in a particular organ system/organ/tissue/cell type. Each D-set is comprised of at least one but typically more than one biomarker, and some biomarkers are members of more than one D-set.

The term “K-base” refers to a database, containing knowledge about all Screening Test types and their components. The information is grouped in the set of different tables, such as:


- - list of all Screening tests developed up to date;
  - list of all pathologies, covered by those tests;
  - list of used (and supposed for using) miRNA;
  - list of D-Sets used in all Screening Tests;
  - relationships Screening Tests type—D-sets—miRNA;
  - tables containing constants for each miRNA used in corresponding
    D-sets.

These tables have to be populated on the Learning step of the algorithms, and used for calculations in the Classification part of the algorithm.

This database is modified and expanded with arrival of new verified research data.

The term “I-base” refers to a database, containing actual data of Screening Tests on individuals, including lists of subjects and pertinent information, history of Screening Tests, their raw data and processed results, etc.

Both databases, along with programs, implementing algorithms for Learning/Classification, are part of the present invention.

The term “Iteration” is used at algorithm description. It refers to the body of the program loop, which is cyclically executed.

### EXAMPLES

The present invention is also described and demonstrated by way of the following examples. However, the use of these and other examples anywhere in the specification is illustrative only and in no way limits the scope and meaning of the invention or of any exemplified term. Likewise, the invention is not limited to any particular preferred embodiments described here. Indeed, many modifications and variations of the invention may be apparent to those skilled in the art upon reading this specification, and such variations can be made without departing from the invention in spirit or in scope. The invention is therefore to be limited only by the terms of the appended claims along with the full scope of equivalents to which those claims are entitled.

**Comparison of Different Methods Used for miRNA Purification from Serum and Plasma**

There are many commercial kits for miRNA isolation, including the miRNeasy kit (Qiagen), the MirVana RNA isolation kit (Ambion/ABI), miRACLE (Agilent), High Pure miRNA isolation kit (Roche), and miRNA Purification kit (Norgen Biotek Corp.). In addition, the in-house techniques based on the use of Trizol (Invitrogen) are commonly used. After Trizol deproteinization, RNA is precipitated with isopropyl alcohol or additionally purified on silica columns. In some experiments, purified RNA is treated with RNAse-free DNAse (Qiagen, ABI, Invitrogen or other). miRNA preparations obtained by different methods are compared using RT PCR.

miRNA was purified from plasma and serum samples obtained from the same 5 healthy donors. 107 copies of Arabidopsis thaliana miR-159a (ath-miR-159a) were spiked per 1 ml plasma or serum after addition of guanidine-containing solution for evaluation of miRNA yield. Two techniques, one based on MirVana Paris kit (Ambion/ABI), and another based on Trizol (Invitrogen) deproteinization, and subsequent purification on silica columns, were compared. After RNA purification, concentrations of spiked miRNA and human endogenous miR-9, miR-16, and miR-134 were measured in final preps by RT-PCR. MirVana Paris kit was more effective in miRNA isolation then the Trizol-based technique and was selected for future experiments. Although all analyzed miRNA were detectable in serum and plasma and both sample types are suitable for miRNA testing, the final PCR Ct values were about 2 cycles lower for plasma, and the latter was used in subsequent experiments. Based on the quantitative measurement of spiked ath-miR-159a, average yield of miRNA isolated from plasma with MirVana kit was 71.4%.

**Selection of miRNA for Testing**

Potential miRNA biomarkers (Table 1) were initially selected based on literature data on their enrichment in various organs and tissues (See, e.g., Hua et al., BMC Genomics 2009, 10:214; Liang et al., BMC Genomics. 2007, 8:166; Landgraf et al., Cell. 2007, 129:1401-1414; Lee et al., RNA. 2008, 14:35-42; http://ferrolab.dmi.unict.it/miro/; http://mips.helmholtz-muenchen.de/phenomir/). Then miRNA common for several organs of the same organ system were selected as potential biomarkers for the respective system (Table 2, column 2). Those miRNA that are enriched in one organ but not in other organs of the system were identified as potential biomarkers for more precise pathology localization (Table 2, column 3). For normalization, in addition to spiked synthetic non-human miRNA, e.g., ath-mir-159a, and ubiquitous miRNA, such as miR-16 and miR-30e-3p, miRNA expressed in numerous tissues but not in a target tissue were selected. For example, miR-10b and miR-141 can be used for normalizing brain biomarkers, and miR-409-3p for pulmonary system biomarkers, etc. Other promising normalizers, which are enriched in the analyzed tissues, organs and systems analyzed, were selected experimentally.

All these biomarkers have to be included as a corresponding D-set to K-base per Example 8, below.

**Detection of an Increase in Levels of Brain-Enriched miRNA in Serum/Plasma of Patients Diagnosed with Neurological Diseases**

Plasma samples from amnestic MCI and AD patients and AMC, 20 in each group, were used in the study. RNA was isolated from two 200 μl aliquots of plasma samples by the Trizol-silica method according to an Asuragen procedure. Single target qRT-PCR was performed using the TaqMan® Reverse Transcription Kit and miRNA-specific stem-loop primers (Applied Biosystems). RT step was performed in triplicate and 2 μl plasma equivalents were present in final PCR.

Data presented in FIGS. 1-6 demonstrate the 2-5 times increase in median concentrations of neurite/synapse miRNAs (miR-7, miR-125b, miR-128, miR-132, miR-134, miR-323-3p, miR-382, miR-874) in plasma of MCI and AD patients when compared to age-matched controls. The effect is more prominent when normalization is performed per brain-enriched miRNA, such as miR-9, miR-127, miR-181a, miR-370, and miR-491-5p.

Two families of biomarkers, miR-132 and miR-134 families, and several normalizers have demonstrated the highest sensitivity and specificity. Receiver-Operating Characteristic (ROC) curves for these combinations of biomarkers and a normalizer are presented in FIGS. 7A-C and 8A-C. Biomarkers miR-128, miR-132 and mir-874 (“miR-132 family”) demonstrated 84%-92% sensitivity and 84%-90% specificity when normalized per miR-491-5p. The area under the ROC curve (AUC) for miR-128, miR-132 and miR-874 is 0.95, 0.93 and 0.95, respectively. The second promising set of biomarkers consists of miR-134, miR-323-3p and miR-382 (“miR-134 family”) and demonstrates 78%-91% sensitivity and 85-87% specificity when normalized per miR-370. AUC for miR-134, miR-323-3p and miR-382 are 091, 0.94 and 0.92, respectively.

Correlation analysis shown in FIG. 9A-F demonstrates that miR-128, miR-132 and miR-874 form one family of biomarkers (Spearman test r values in the pair comparison are in the 0.93-0.95 range) and miR-134, miR-323-p and miR-382 form another family of biomarkers (Spearman test r values in the pair comparison are in the 0.87-0.93 range). High correlation between members of miR-134 family can be easily explained by the fact that all members of this family, namely miR-134, miR-323-3p and miR-382, belong to the same cluster and are expressed in the same cell types (http://www.diana.pcbi.upenn.edu/cgi-bin/miRGen/v3/Cluster.cgi). Close relationships between members of miR-132 family, namely miR-128, miR-132 and miR-874, have not been described before. It is also interesting that biomarker families miR-132 and miR-134 produce better results with different normalizers. miR-132 family works better than miR-134 family with normalizers miR-491-5p, miR-181a, miR-9, and miR-141. On the other hand, miR-134 family demonstrates better results than miR-132 family with normalizers miR-370 and miR-127. Correlation between miR-132 and miR-134 biomarker families is relatively low (r values in the pair comparison Spearman test are in the 0.56-0.79 range) indicating that they either reflect distinct pathological processes or are located in different brain areas.

**Detection of an Increase in Levels of Lung-Enriched miRNA in Serum/Plasma of Patients with Lung Diseases**

Plasma samples were obtained from patients diagnosed with various lung diseases, such as asthma, pneumonia, Chronic Pulmonary Obstructive Disease (COPD), and non-small cell lung cancer (NSCLC), 10 in each group. Since enrolled patients with asthma and pneumonia were non-smokers and patients with COPD and NSCLC were smokers, plasma samples were also collected from two groups of controls, smokers and non-smokers, 10 in each group. RNA was isolated from two 200 μl aliquots of plasma samples by the Trizol-silica method according to an Asuragen procedure. Single target qRT-PCR was performed using the TaqMan® Reverse Transcription Kit and miRNA-specific stem-loop primers (Applied Biosystems). RT step was performed in triplicate and 2 μl plasma equivalents were present in final PCR to measure concentration of miR-34b, miR-142-5p, miR-146-5p, miR-155, miR-223, miR-486-5p, enriched in the lung, as well as the levels of ubiquitous and enriched in gastrointestinal system miR-192 and miR-409-3p. The latter is essentially ubiquitous but under-expressed in the lung. Concentrations of each lung-enriched miRNA were normalized per miR-409-3p and miR-192, as well as on each other, converted into Relative Quantity (RQ) of miRNA according the ABI protocol (2−ΔCt), and compared with miRNA profiles from controls.

As expected miR-34b and miR-486-5p, which are highly enriched in the lung, were found to be effective biomarkers and miR-409-3p is an effective normalizer (FIG. 10A-D). Other effective normalizers are lung-enriched miR-142-5p, miR-146b-5p, miR-155 and miR-223 (FIGS. 11A-H and 12A-H), which, at least in some cases, could be explained by their downregulation in lung pathologies (Liu X. et al. Clin. Cancer Res. 2009, 15:1177-1183; Miko E. et al., Exp. Lung Res. 2009, 35:646-664; Halappanavar S. et al. Toxicology 2011, 285: 133-141; Heegaard N H et al. Int. J. Cancer, 2012, 130:1378-1386). Unexpectedly, miR-192 also behaved as an effective biomarker (FIG. 13A-J). Since this miRNA was also shown to be a biomarker for diseases of the GI system, it is reasonable to suggest that expression and/or secretion of this miRNA is increased due to inflammation or tumor development (Benjamin H. et al., J. Mol. Diagn. 2010, 12:771-779; Lan H Y. Clin. Exp. Pharmacol. Physiol. 2011 Dec. 28 [Epub ahead of print]; Luzna P. et al. Diagn. Pathol. 2011, 6:114; Wu Q. et al. J. Biomed. Biotechnol. 2011, Epub May 26; Zhou J. et al., J. Clin. Oncol. 2011, 29:4781-4788).

miRNA biomarker miRNA/miRNA normalizer ratios were not different for smoking and non-smoking controls. Thus, various pathologies could be compared to the combined controls (smoking and non-smoking subjects). FIG. 14A-C demonstrates that the four analyzed pathologies are effectively distinguished from such combined controls by various sets of miRNA biomarkers and normalizers, e.g. miR-34b normalized per miR-409-3p, miR-486-5p normalized per miR-223, or miR-192 normalized per miR-155. There were also other effective sets of miRNA biomarkers and normalizers.

The ability of various combinations of miRNA biomarkers and normalizers to distinguish NSCLC from such inflammatory diseases as asthma, pneumonia and COPD was also analyzed. FIG. 15A-C shows that patients with NSCLC are differentiated from patients with inflammatory diseases using ratios of miR-34b to miR-155, miR-486-5p to miR-146b-5p or miR-192 to miR-146b-5p. There were other effective combinations of miRNA biomarkers and normalizers.

**Example 5**

**Detection of an Increase in Levels of Gastrointestinal System-Enriched miRNA in Serum/Plasma of Patients with Diseases of Gastrointestinal System**

Plasma samples were obtained from patients diagnosed with various diseases of the GI system, such as esophageal, gastric and colon cancers, and an inflammatory condition, Crohn's disease, 10 in each group. RNA was isolated from two 200 μl aliquots of plasma samples by the Trizol-silica method according to an Asuragen procedure. Single target qRT-PCR was performed using the TaqMan® Reverse Transcription Kit and miRNA-specific stem-loop primers (Applied Biosystems). RT step was performed in triplicate and 2 μl plasma equivalents were present in final PCR to measure concentration of miR-145, miR-148a, miR-192, miR-194, miR-203, miR-215, enriched in organs of GI system, as well as the level of ubiquitous miR-30e-3p. Concentrations of each GI system-enriched miRNA were normalized per miR-30e-3p, as well as on each other, converted into Relative Quantity (RQ) of miRNA according the ABI protocol (2−ΔCt), and compared with miRNA profiles from controls. FIG. 16A-L clearly shows miR-192, miR-194, miR-203 and miR-215 as effective biomarkers, and miR-145, miR-148a and miR-30e-3p as effective normalizers. Biomarker/normalizer ratios effectively differentiate patients with all studied diseases from controls. miR-203, highly enriched in esophagus and stomach, is especially effective in detecting cancers of these organs, and miR-215, highly enriched in column is most effective in differentiating patients with colon cancer and Crohn's disease from controls. Combination of miR-192 and miR-203 normalized per miR-30e-3p effectively distinguishes patients with all pathologies of the GI system from controls (FIG. 16M) with 94% sensitivity and 100% specificity calculated as described in Example 8. It is important that all tumors were stage 1 or 2 cancers, which means that the proposed approach can be effectively used for screening and early diagnosis.

Further, various cancers were compared with each other, and Crohn's disease was compared with all cancers of the GI system. As a result the following biomarker/normalizer ratios capable of distinguishing particular pathologies have been found:


- - 1. Crohn's disease versus esophageal, gastric and colorectal
    cancers: miR-194/miR-148a; miR-215/miR-30e-3p; miR-215/miR-194;
    miR-203/miR-148a; miR-192/miR-203; miR-215/miR-203 and
    miR-194/miR-192 (FIG. 17A-G).
  - 2. Esophageal cancer versus gastric cancer: miR-194/miR-145;
    miR194/miR-148a; miR194/miR-30e-3p (FIG. 18A-C).
  - 3. Gastric cancer versus colorectal cancer: miR-203/30e-3p;
    miR-203/miR-148a; miR-215/miR-203 (FIG. 18D-F).
  - 4. Esophageal cancer versus colorectal cancer: miR-192/miR-145;
    miR192/miR-148a; miR192/miR-30e-3p (FIG. 18G-I).
  - Thus, analysis of plasma concentrations of miRNA enriched in organs
    of the GI system is effective for: (i) detection of Crohn's disease
    and tumors in esophagus, stomach and colon; (ii) differentiation of
    an inflammatory disease from cancers; (iii) differentiation of
    cancers located in various organs of the GI system.

**Differentiation of Pathologies of Various organ Systems**

miRNA preparations purified from the plasma samples obtained from patients described in Examples 4 and 5 were used in the study. The ability of various miRNA combinations to differentiate patients with diseases of the GI system (Crohn's disease and esophageal, gastric and colorectal cancers) from patients with diseases of the pulmonary system (asthma, pneumonia, COPD and NSCLC) was investigated. Lung-enriched miR-126, miR-146b-5p, miR-155, and miR-486-5p, and GI-enriched miR-145, miR-192, miR-203 and miR-215 were included in the study. Expression of some of these miRNA (Table 3) is known to be deregulated in pathologies of various organs. Additionally, miR-17-5p and mir-31, involved in pathological processes of various organs, as well as miR-30e-3p and miR-409-3p, which were used as normalizers in Experiments 4 and 5, were analyzed. RNA was isolated from two 200 μl aliquots of plasma samples by the Trizol-silica method according to an Asuragen procedure. Single target TaqMan® miRNA qRT-PCR assays (Applied Biosystems) were run using 2 μl plasma equivalents in triplicate in a reaction volume of 10 μl for final PCR. Concentrations of each miRNA were normalized per miR-30e-3p and miR-409-3p, as well as on each other, converted into Relative Quantity (RQ) of miRNA according the ABI protocol (2−ΔCt), and miRNA profiles characteristic of patients with the diseases of the pulmonary and GI systems were compared. FIGS. 19A-U demonstrate that many miRNA pairs effectively distinguish patients with diseases of pulmonary and GI systems: miR-192/miR-126; miR-155/miR-126; miR-145/miR-126; miR-155/miR-30e-3p; miR-192/miR-30e-3p; miR-155/miR-409-3p; miR-486-5p/miR-17-5p; miR-155/miR-17-5p; miR-192/miR-17-5p; miR-146b-5p/miR-31; miR-155/miR-31; miR-192/miR-31; miR-486-5p/miR-155; miR-192/miR-155; miR-145/miR-155; miR-146b-5p/miR-155; 486-5p/miR-203; miR-192/miR-203; miR-145/miR-203; miR-192/miR-215; miR-155/miR-215.

Combination of two miRNA pairs increases the test accuracy. FIGS. 19V and 19W provide an example with the combination of miR-145/miR-155 and miR-486-5p/miR-155 ratios, which distinguishes patients with all pathologies of the GI system from patients with pulmonary diseases with 95% sensitivity, 90% specificity, and 93% accuracy.

**Differentiation of Cancers from Inflammatory Diseases**

The same plasma samples were used for RNA purification and the same miRNA were analyzed that were studied in the Example 6. In this study the ability of various miRNA combinations to differentiate patients with inflammatory diseases (asthma, pneumonia, COPD and Crohn's disease) from patients with various cancers (esophageal, gastric, colorectal and non-small cell lung cancers) was investigated. FIGS. 20A-F demonstrate that several miRNA pairs effectively distinguish patients with inflammatory diseases from patients with various cancers: miR-17-5p/miR-155; mir-192/miR-155; miR-215/miR-155; miR192/miR-30e; miR-146b-5p/miR-30e-3p; miR155/miR-30e-3p. There are less miRNA pairs differentiating inflammation diseases from cancers than miRNA pairs capable of differentiating diseases of the pulmonary system from diseases of the GI system. First, changes in expression of many miRNA are characteristic of both pathology types. Second, in many cases carcinogenesis is accompanied by relatively prominent inflammation.

Combination of two miRNA pairs increases the test accuracy. FIGS. 20G and 20H provide an example with the combination of miR-146b-5p/miR-155 and miR-146b-5p/miR-30e-3p ratios, which distinguishes all patients with inflammatory diseases from cancer patients with 80% sensitivity, 98% specificity, and 89% accuracy.

Thus, the results of the experiments presented above support the main ideas of the present invention. The analysis of concentration of miRNA, enriched in a particular organ system or in an organ, in plasma differentiates: (i) organ system diseases from controls; (ii) pathologies of three organs of the GI system; (iii) diseases of the pulmonary and GI system; (iv) cancers and inflammatory diseases.

**A Method for Multiple miRNA Analysis and its Use for Biomarker Selection and Detection of an organ System or Particular organ with Pathological Changes**

Two different applications are used for the UST development (research stage) and its clinical usage (Screening Data Processing). Algorithms in both applications contain Training and Classification parts, but they are significantly different.

### The Algorithm Used for Screening Test Development

In the following algorithm description and related figures, the term “Biomarker” defines a miRNA pair, Marker and Normalizer, used for pathology diagnosis and more generally for distinguishing various groups of subjects. The term “ROC” stands for Receiver-Operating Characteristic, the statistics used in Classification.

The proposed algorithm is based on the following simplified assumptions:


- - For any biomarker or combinations of biomarkers, a limited number,
    e.g. ≦20, of experiments are available;
  - Response of any biomarker to a particular pathology can be explored
    independently and should not be linked to the “final” combination of
    biomarkers assigned to a particular D-set;
  - Algorithms should be probabilistic, which allows better result
    estimate.

The first part of the algorithm is related to Training using Labeled data, for a pathology of a respective organ system, organ, tissue, or cell type. FIG. 21A outlines operations in this part of the algorithm.

This part assumes intensive dialog, practically on every step, with the person who performs experiments. The very first step always includes some manual operation—or typing sample values, or performing operations to import data—partially or completely—from some standard documents. The type of the document can be, but is not limited to, Excel spreadsheet, or text delimited file. All other steps can be performed automatically, with only manual confirmation like “Next”, or can have some manual corrections. For example, on the Normalization stage one should provide the best type of Normalization, or make all processing for several types of it. Some samples can have values outside of certain statistically reasonable range. As a rule, this range is multiple of standard deviation, for example, but not limited to, two or three times standard deviation, depending on value distribution (how close is it to the Normal one, etc.), a decision can be made as to whether to exclude these samples from statistics, and so on. Statistical analysis includes calculation of P-level of Target—Control separation and ROC (Receiver-Operating Characteristic) curve parameters. The last step of the algorithm—making a decision to add the marker—can include parts (internal loop) of the Classification algorithm, described below. In this step, calculated parameters are applied to the Training set. The decision to use this marker is based on the level of Classification success. This decision can be revisited at the Classification stage.

Classification part of the algorithm is shown in FIG. 21B.

The second part of the algorithm, Classification, is used mostly for Validation data processing. Also, part of it, internal loop, can be used in the last step of Training algorithm. In general, at the research stage, these parts—Training Procedure and Classification procedure—are used in iterative manner, i.e., are repeated several times, to reach clinically acceptable testing accuracy.

The Classification procedure contains two nested iteration loops, through D-sets (outer loop) and Biomarkers in every D-set (internal loop).

Iteration steps through internal Biomarker loop include:

**Step 1. Normalization of D-Set Biomarkers**

Normalization can include one general Screening test normalization (one per test), based on spiked miRNA, and normalization specific for a particular D-set—type b), c) or d) (see the Detailed Description of the Invention section, above), or more than one of them. The version of specific normalization used for each Biomarker is included into K-base as a description of the D-set. Thus, if one Biomarker is a member of more than one D-sets, it should be normalized accordingly, in every set.

**Steps 2. Biomarker Probability Calculation**

Biomarker concentration for a particular person has to be mapped to two probabilities: of having pathology or not having it. The mapping operation uses two probability functions, based on processing Biomarker data of two populations: with pathology and without it (Control). Each probability function is a linearly approximated curve, which goes from 1 to 0 for Control population, and from 0 to 1 for population with the Target pathology. Description of every curve is stored, retrieved and updated in K-base. Step 3. Using history (time-spaced points)

Significant parts of Target and Control curves are overlapping. If an actual value is within this overlapping area, it is necessary to make a decision how to interpret the result. This is based on, but not limited to, comparison of probabilities of having the pathology and being free of it; analyzing existing Biomarker data for the same person if such data have been taken and stored in I-base.

This is the end of Biomarker iterations.

**Iteration Steps for the D-Set Loop (Outer D-Steps)**

**Step 1. Target Probability Calculation.**

In most cases, Biomarkers, which are included in a particular D-set, are statistically independent, i.e. probabilities of the particular concentration value to belong to the Control or Target Group do not significantly depend on the value(s) of other Biomarkers in the D-set. In this case, the weighted sum of probabilities for Biomarkers, comprising this D-set, has been calculated. By default, all weights are equal, e.g., for three biomarkers each weight is ⅓. However, individual weights can be stored in K-base, for every biomarker within this D-set. They can be based on some Biomarker ROC parameters, like sensitivity, specificity or AUC (area under curve). The sum of weights for all biomarkers in any D-set should be 1.

In the case of significant interdependency of some Biomarkers in a particular D-set, K-base contains a description of multidimensional probability surface for this group, where dimension equals to the number of dependent Biomarkers in the group. Using this description, each combination of Biomarker values is mapped to two probabilities. If the D-set contains also independent Biomarkers, probabilities of the D-set are calculated using weighted sum of probabilities for the group(s) of dependent Biomarkers, and individual independent Biomarkers.

**D-Step 2. Making a Decision.**

As a rule, the result should be presented as POSITIVE or NEGATIVE towards Target pathology. In special cases, the result UNDETERMINED with percentage attached can be also used. Decision making parameters must be previously defined and stored in K-base. They can be Screening-wide, or target (organ system/organ/tissue) specific and applied to probabilities, calculated in the previous step (D-Step1). These parameters can be applied to each of two probabilities (PP—pathology probability, PC—control probability), or, more common, to the difference of two. Examples: (i) difference parameter is 0, i.e. PP>PC means POSITIVE, PP<PC means NEGATIVE; (ii) PP−PC≦0.25 is NEGATIVE, PP−PC≧0.35 POSITIVE and UNDETERMINED in between; (iii) PP>0.6 is POSITIVE, PC>0.7 is NEGATIVE, everything else is UNDETERMINED.

**D-Step 3. Registration**

The results are displayed and saved to I-base, together with identification data, such as test#, date/time stamps, patient identification, etc.

### The Algorithm for Clinical Trial and Usage

When research stage is complete, K-base contains all components of UST and its versions: biomarkers, D-sets, and constants for Classification. In the next stage larger clinical studies are conducted using these tools.

Based on obtained result, verification and evaluation of the biomarkers and D-sets as well as some modifications of the algorithm or/and constants are performed. For example, different classification algorithm, like Multinomial Logistic Regression (Hosmer D W, Lemeshow S. Applied logistic regression. Wiley, 2000; Allison P D. Logistic regression using the SAS system: theory and application. Wiley, 2008) or Support Vector Machines (Cristianini N, Shawe-Taylor J. An Introduction to Support Vector Machines and Other Kernel-based Learning Methods. Cambridge Press, 2000; Abe S. Support Vector Machines for Pattern Classification. Springer, 2010) can be used. K-base gets some changes accordingly. After such changes, clinical trials will be performed.

In the clinical usage, Classification algorithm uses constants and parameters from K-base, and actual test data are stored to I-base. K-base remains unchanged until next revision, i.e. arrival of new research verified data.

In general, any of the algorithms, functional operations, or subject matter described in this specification can be implemented in digital electronic circuitry, or in computer software, firmware, or hardware, including the structures disclosed in this specification and their structural equivalents, or in combinations of one or more of them. For example, in some embodiments, one or more data processing apparatuses can be part of a module that can be installed in a computer and configured to perform algorithms to select and/or detect biomarkers for pathologies as well as perform algorithms for classification of one or more pathologies. In some embodiments, the one or more data processing apparatuses can be part of a module that can be installed in a computer and configured to perform the Classification algorithm set forth in Example 8 including the nested iteration loops (e.g., Biomarker loop and the D-set loop). For example, the one or more data processing apparatuses can be configured to perform one or more of the operations shown in FIGS. 21A and 21B. In some embodiments, the algorithms can be implemented as one or more computer program products, i.e., one or more modules of computer program instructions encoded on a computer readable medium for execution by, or to control the operation of, data processing apparatus.

The computer readable medium can be a machine-readable storage device, a machine-readable storage substrate, a memory device, or a combination of one or more of them. The term “data processing apparatus” encompasses all apparatus, devices, and machines for processing data, including by way of example a programmable processor, a computer, or multiple processors or computers. The apparatus can include, in addition to hardware, code that creates an execution environment for the computer program in question, e.g., code that constitutes processor firmware, a protocol stack, a database management system, an operating system, a runtime environment or a combination of one or more of them. The apparatus includes code for creating, verifying, and/or modifying the K-base or I-base tables. In some embodiments, the K-base and I-base can be maintained on a database management system such as, SQL server, oracle, MySQL, among others.

A computer program (also known as a program, software, software application, script, or code) can be written in any form of programming language, including compiled or interpreted languages, and it can be deployed in any commonly used deployment form, presenting a unit suitable for use in a computing environment. A computer program does not necessarily correspond to a file in a file system. A program can be stored in a portion of a file that holds other programs or data (e.g., one or more scripts stored in a markup language document), in a single file dedicated to the program in question, or in multiple coordinated files (e.g., files that store one or more modules, sub programs, or portions of code). A computer program can be deployed to be executed on one computer or on multiple computers that are located at one site or distributed across multiple sites and interconnected by a communication network.

The processes described in this specification can be performed by one or more programmable processors executing one or more computer programs to perform functions by operating on input data and generating output. Processors suitable for the execution of a computer program include, by way of example, both general and special purpose microprocessors, and any one or more processors of any kind of digital computer.

In some embodiments, a user manually enters, through the one or more devices, data obtained from screening tests in the research stage or clinical stage. In some embodiments, a user can observe or receive, from one or more devices, output data such as, but not limited to, biomarker classification data and data relating to detections of biomarker concentration changes. Other kinds of devices can be used to provide for interaction with a user as well; for example, feedback provided to the user can be any form of sensory feedback, e.g., visual feedback or auditory feedback; and input from the user can be received in any form, including acoustic, speech, or tactile input.

Embodiments of the subject matter described in this specification can be implemented in a computing system that includes a back end component, e.g., as a data server, or that includes a middleware component, e.g., an application server, or that includes a front end component, e.g., a client computer having a graphical user interface or a Web browser through which a user can interact with an implementation of the subject matter described is this specification, or any combination of one or more such back end, middleware, or front end components. For example, in some embodiments, the system can include a graphical user interface/Web interface that allows a user to enter and/or inspect data in the K-base and I-base. Alternatively, or in addition, the interface can allow the user to control, modify, or manipulate an algorithm's execution. In some embodiments, the system can retrieve or output data from other systems and/or components coupled to the network.

The present invention is not to be limited in scope by the specific embodiments described herein. Indeed, various modifications of the invention in addition to those described herein will become apparent to those skilled in the art from the foregoing description. Such modifications are intended to fall within the scope of the appended claims.

All patents, applications, publications, test methods, literature, and other materials cited herein are hereby incorporated by reference in their entirety as if physically present in this specification.

