# DESCRIPTION

## TECHNICAL FIELD OF THE INVENTION

The present invention relates to a method of purifying whole virus particles. In particular, the present invention relates to an improved downstream process for purification of cell culture-derived viruses to obtain a whole virus vaccine candidate stock.

## BACKGROUND OF THE INVENTION

The hepatitis C virus (HCV) is a small enveloped virus, 30 to 80 nm in diameter, with a single positive stranded RNA genome, belonging to the Flaviviridae family. The RNA genome encodes three structural proteins, the capsid protein Core, and the envelope proteins E1 and E2, which are incorporated into the viral particle, as well as seven non-structural proteins (p7, NS2, NS3, NS4A, NS4B, NS5A, and NS5B). There are eight different major genotypes, differing in ˜30% of their nucleotide and amino acid sequence, with genotype 1 being most frequent worldwide.

Each year, there are at least 2 million new HCV infections, of which ˜80% result in chronic infections. There are at least 71 million chronically infected individuals worldwide with an increased risk for liver cirrhosis and hepatocellular carcinoma, resulting in 400.000 deaths per year.

Only a minor fraction of HCV-infected individuals is treated with recently licensed efficient direct-acting antivirals (DAA). The main reasons for this are that most individuals are not aware of their infection status, as the infection is typically asymptomatic until a severe and often irreversible liver disease has developed, and because of the lack of screening programs and the high cost of DAA. Furthermore, resistance to DAA is increasing and might compromise future treatment efficacy. Thus, a vaccine is urgently needed to control HCV on a global scale.

Most antiviral vaccines are based on whole viral particles as vaccine antigens and protect by their induction of neutralizing antibodies. The proof-of-concept for the immunogenicity of cell culture-derived inactivated HCV has been obtained in animal models. However, in these studies, ultracentrifugation-based downstream processes (DSP) were employed for concentration and purification of cell culture-derived HCV. This approach is in general characterized by a relatively low recovery, a limited scalability, and a limited impurity depletion. Thus, as for most other vaccines, the development of an efficient DSP, compatible with industrial requirements, is a major bottleneck for the manufacturing of a whole virus HCV vaccine for human use.

Hence, an improved downstream process providing an efficient, scalable and GMP-compatible DSP for purification of cell culture-derived HCV to facilitate an industrial production of a human HCV vaccine would be advantageous.

## SUMMARY OF THE INVENTION

The present inventors have now surprisingly found that virus particles from cell cultures may be efficiently purified by a DSP comprising specific purification steps, resulting in purified viruses with only a minimal amount of impurities being ready for use for immunizing. These virus particle stocks may be used in facilitating virological studies, and for vaccine development.

Thus, it is an object of the present invention to provide such purified virus particles for example as a whole virus vaccine candidate stock.

Thus, one aspect of the invention relates to a method of purifying whole HCV particles, the method comprising the steps of


- - a) providing a cell culture supernatant comprising virus particles,
  - b) purification and/or concentration of the cell culture
    supernatant,
  - c) purification and/or concentration of the product of above step b)
    using steric exclusion chromatography (SXC) at alkaline pH in the
    range of 8-10,
  - d) purification and/or concentration of the product of above step c)
    using sulphated cellulose membrane absorbers (SCMA),
  - e) obtaining purified whole virus particles.

The present invention will now be described in more detail in the following.

## DETAILED DESCRIPTION OF THE INVENTION

### Definitions

Prior to discussing the present invention in further details, the following terms and conventions will first be defined:

The term “clarification” is to be understood as a step of clarification or a step of clarifying a sample i.e. the cell culture supernatant is crudely purified from large impurities such as whole cells and cell debris.

The term “ultrafiltration” is to be understood as a step of ultrafiltration as known to the persons skilled in the art, where a sample may be concentrated. In addition, the ultrafiltration further purifies the sample by removing impurities such as smaller host cell-derived proteins and DNA, which are below the molecular weight cut-off of the used devices.

The term “chromatography” is to be understood as a step performed by a chromatographic technique as known to the persons skilled in the art including anion-or cation exchange chromatography, hydrophilic interaction chromatography, SXC or SCMA as well as flow through methods such as Capto Core 700 chromatography or size exclusion chromatography.

The term “SXC” is to be understood as steric exclusion chromatography as commonly known to the persons skilled in the art. Steric exclusion chromatography is run in a membrane-containing column, where the sample is loaded via a loading buffer to the column for binding of the product to the membranes, the column is then washed to elute impurities from the column prior to eluting the product from the column by means of an elution buffer.

The term “SCMA” is to be understood as sulphated cellulose membrane absorbers as commonly known to the persons skilled in the art. SCMA is also a chromatography-based method, which is run in a membrane-containing column. The sample is loaded via a loading buffer to the column for binding of the product to the membranes, the column is then washed to elute impurities from the column prior to eluting the product from the column by means of an elution buffer.

The term “dynamic binding capacity (DBC)” is to be understood as commonly known by the persons skilled in the art i.e. it describes the maximum amount of product that may be loaded onto the column without causing unnecessary loss, measured under realistic experimental conditions (default flow-rate, real sample).

The term “virus particles” is to be understood as infectious viruses, inactive viruses as well as virus-like particles.

The term “whole virus particles” is to be understood as virus particles, which consist of RNA or DNA surrounded by a protein shell, optionally surrounded by an envelope.

The terms “isolate” and “strain” are used herein interchangeably.

As commonly defined “identity” is here defined as sequence identity between genes or proteins at the nucleotide or amino acid level, respectively. Thus, in the present context “sequence identity” is a measure of identity between proteins at the amino acid level and a measure of identity between nucleic acids at nucleotide level. The protein sequence identity may be determined by comparing the amino acid sequence in a given position in each sequence when the sequences are aligned. Similarly, the nucleic acid sequence identity may be determined by comparing the nucleotide sequence in a given position in each sequence when the sequences are aligned. To determine the percent identity of two amino acid sequences or of two nucleic acid sequences, the sequences are aligned for optimal comparison purposes (e.g., gaps may be introduced in the sequence of a first amino acid or nucleic acid sequence for optimal alignment with a second amino or nucleic acid sequence). The amino acid residues or nucleotides at corresponding amino acid positions or nucleotide positions are then compared. When a position in the first sequence is occupied by the same amino acid residue or nucleotide as the corresponding position in the second sequence, then the molecules are identical at that position. The percent identity between the two sequences is a function of the number of identical positions shared by the sequences (i.e., % identity=# of identical positions/total # of positions (e.g., overlapping positions)×100).

One may manually align the sequences and count the number of identical amino acids. Alternatively, alignment of two sequences for the determination of percent identity may be accomplished using a mathematical algorithm. Such an algorithm is incorporated into the NBLAST and XBLAST programs (Altschul et al. 1997; Altschul et al. 2005). BLAST nucleotide searches may be performed with the NBLAST program, score=100, wordlength=12, to obtain nucleotide sequences homologous to a nucleic acid molecules of the invention. BLAST protein searches may be performed with the XBLAST program, score=50, wordlength=3 to obtain amino acid sequences homologous to a protein molecule of the invention. To obtain gapped alignments for comparison purposes, Gapped BLAST may be utilised. Alternatively, PSI-Blast may be used to perform an iterated search, which detects distant relationships between molecules. When utilising the NBLAST, XBLAST, and Gapped BLAST programs, the default parameters of the respective programs may be used.

See http://www.ncbi.nlm.nih.gov. Alternatively, sequence identity may be calculated after the sequences have been aligned e.g. by the BLAST program in the EMBL database (www.ncbi.nlm.gov/cgi-bin/BLAST). Generally, the default settings with respect to e.g. “scoring matrix” and “gap penalty” may be used for alignment. In the context of the present invention, the BLASTN and PSI BLAST default settings may be advantageous.

The percent identity between two sequences may be determined using techniques similar to those described above, with or without allowing gaps. In calculating percent identity, only exact matches are counted.

It should be noted that while several of the sequences in the present application are DNA sequences, the present invention contemplates the corresponding RNA sequence, and DNA and RNA complementary sequences as well.

Thus, in cases where a DNA sequence is mentioned refers such DNA sequence also to the RNA equivalent i.e. with Ts exchanged with Us as well as their complimentary sequences.

The term “adaptive mutation” is meant to cover mutations identified in passaged viruses that provide the original and any other HCV sequence the ability to grow efficiently in culture. Furthermore, all introductions of mutations into the sequences described, whether or not yielding better growth abilities, and the introduction of these mutations into any HCV sequence should be considered. Throughout the description the adaptive mutations as described herein is to be interpreted as XZY being amino acid/nucleic acid “X” at position “Z” being changed to amino acid/nucleic acid “Y” e.g. as an example A1226G is to be understood as the amino acid alanine (A) at position 1226 being changed to the amino acid glycine (G).

The term “composition” refers to a pharmaceutical composition suitable for administration to a subject, and such compositions may comprise a pharmaceutically acceptable carrier or diluent, for any of the indications or modes of administration as described. The active materials in the compositions of this invention can be administered by any appropriate route, for example, orally, parenterally, intravenously, intradermally, subcutaneously, or topically, in liquid or solid form.

The term “pharmaceutically acceptable” refers to molecular entities and compositions that are physiologically tolerable and do not typically produce an allergic or similar untoward reaction, such as gastric upset, dizziness and the like, when administered to a human. Preferably, as used herein, the term “pharmaceutically acceptable” means approved by a regulatory agency of the Federal or a state government or listed in the U.S. Pharmacopoeia or other generally recognized pharmacopoeia for use in animals, and more particularly in humans.

The term “excipient” refers to a diluent, adjuvant, carrier, or vehicle with which the compound is administered. Such pharmaceutical carriers can be sterile liquids, such as water and oils, including those of petroleum, animal, vegetable or synthetic origin, such as peanut oil, soybean oil, mineral oil, sesame oil and the like. Water or aqueous solution saline solutions and aqueous dextrose and glycerol solutions are preferably employed as carriers, particularly for injectable solutions. Suitable pharmaceutical carriers are described in “Remington's Pharmaceutical Sciences” by E. W. Martin.

The term “adjuvant” refers to a compound or mixture that enhances the immune response to an antigen. An adjuvant can serve as a tissue depot that slowly releases the antigen and also as a lymphoid system activator that non-specifically enhances the immune response. Often, a primary challenge with an antigen alone, in the absence of an adjuvant, will fail to elicit a humoral or cellular immune responses.

The terms “cell” and/or “cell systems” are to be understood comprising primary cultures or other, also non hepatic cell lines. “Primary cultures” refers in the present context to a culture of cells that is directly derived from cells or tissues from an individual, as well as cells derived by passage from these cells, or immortalized cells.

The term “cell line” refers to a population of cells capable of continuous or prolonged growth and division in vitro. The term “cell lines” also includes immortalized cells. Often, cell lines are clonal populations derived from a single progenitor cell. Such cell lines are also termed “cell clones”.

Method

For development of vaccines targeting different viruses such as e.g. HCV, an inactivated whole virus approach is attractive, given the intricate conformation of the envelope proteins, which is difficult to mimic in subunit envelope vaccines. Also considering the higher immunogenicity of whole particles compared to envelope proteins, and the historic success of this strategy in antiviral vaccine development, whole virus particles are preferable. For HCV, this approach has only become feasible due to the relatively recent development of cell culture systems for the production of HCV. However, for a success of this strategy, it is important that the virus particles are purified from the cell cultures in an efficient manner and in a manner leaving only traces of impurities in the purified virus stock.

Here, we evaluated clarification and ultrafiltration in combination with two chromatography technologies, such as the membrane-based chromatography technologies (1) SXC, and (2) SCMA, for the development of a cost-efficient, scalable DSP, compatible with good manufacturing practices (GMP).

In one aspect, the present invention relates to a method of purifying whole virus particles, the method comprising the steps of


- - a) providing a cell culture supernatant comprising virus particles,
  - b) purification and/or concentration of the cell culture
    supernatant,
  - c) purification and/or concentration of the product of above step b)
    using chromatography,
  - d) purification and/or concentration of the product of above step c)
    using chromatography,
  - e) obtaining purified whole virus particles.

In one embodiment, the chromatography used in step c) is steric exclusion chromatography (SXC) and the chromatography used in step d) is sulphated cellulose membrane absorbers (SCMA).

The examples show experiments using such methods and the resultant high recovery rates and low amount of impurities. Example 7 also demonstrates how this method is advantageous as compared to an alternative method known in the art.

Purification and/or Concentration of the Cell Culture Supernatant

The first step of purification and/or concentration of the cell culture supernatant is performed on the cell culture supernatant in step b) in order to remove vast amounts of impurities obtained when harvesting the cell culture supernatant comprising the virus particles and/or to concentrate the supernatant.

In one embodiment, the purification and/or concentration is performed using a method selected from the list consisting of centrifugation, ultracentrifugation, density gradient ultracentrifugation, iodixanol cushion centrifugation, sucrose cushion centrifugation, nycodenz cushion centrifugation, cesium chloride cushion, iodixanol gradient centrifugation, sucrose gradient centrifugation, nycodenz gradient centrifugation, cesium chloride gradient centrifugation, ultracentrifugation pelleting, filtration, clarification, microfiltration, nanofiltration, direct filtration, cross-flow filtration, ultrafiltration, precipitation, polyethylene glycol precipitation, polymer precipitation and polyelectrolyte precipitation.

In a further embodiment, the purification and/or concentration is performed using filtration.

In a further embodiment, the filtration is selected from the list consisting of conventional direct or dead-end filtration, depth filtration, cut-off filtration, microfiltration, nanofiltration, ultrafiltration, small-scale cross-filtration, and cross-flow filtration.

In a further embodiment, the precipitation is polyethylene glycol (PEG) precipitation.

In one embodiment, the purification in step b) comprises at least one step of clarification. In a further embodiment, two steps of clarification are performed. The step of clarification may beneficially be a step of filtration that covers clarification and endotoxin removal. This can be done using a 3M Purification Inc.(TM) filtration system. The 3M Purification Inc.(TM) filtration system includes but is not limited to depth filters “Zeta Plus”, “Zeta Plus EXT Series” and “Betapure NT-P™” which can be used for clarification, adsorption-based separation systems “Zeta Plus ZA”, which can be used for endotoxin removal, and surface filters “LifeASSURE SP”, which can be used for endotoxin removal. Alternatively, the clarification is performed using capsule filters such as Sartopur® PP3 capsule filters.

Alternatively, the purification may be performed using 3 or 4 cushions. Cushions can have different densities, for example, iodixanol cushions could be 10%, 20%, 28%, 30%, 60% or 70%.

In a further embodiment, the ultracentrifugation is selected from the group consisting of iodixanol gradient ultracentrifugation, sucrose gradient ultracentrifugation, and ultracentrifugation pelleting. The gradient ultracentrifugations can be done at different ranges, for example 1-80%, 5-60%, 10-40%, 20-60%, or 20-70%.

In a further embodiment, step b) further comprises at least one step of ultrafiltration.

In a still further embodiment, the purification is performed using two steps of ultrafiltration. The ultrafiltration may advantageously be performed by two subsequent steps of ultrafiltrations preferably using different parameter settings e.g. different filter sizes, where the largest filter is used in the first ultrafiltration. The ultrafiltration is preferably performed using hollow fiber filters.

In a further embodiment, the hollow-fiber filters may be filters such as but not limited to MicroKros® Filter Modules, MidiKros® Filter Modules, MidiKros® TC Filter Modules, MiniKros® Sampler Filter Modules, MiniKros® Filter Modules, KrosFlo® Filter Modules, KrosFlo® Max Filter Modules and Vivaflow. Different molecular weight cut-offs such as 500 kDa, 300 kDA, 200 kDa, 100 kDa, 70 kDa, 50 kDa, 30 kDa, 10 kDa, 5 kDa, 3 kDa, 1 kDa might be used. Filters with different surface areas may be used.

Different filters can be applied, and they may have different pore size, greater surface or have a higher molecular weight cut-off to allow purification from bigger proteins.

In a preferred embodiment, the first step of purification and/or concentration i.e. step b) comprises two steps; step b1) a step of clarification performed by filtration, optionally using capsule filters, for purification of the cell culture supernatant and step b2) a step of ultrafiltration, optionally using hollow fiber filters, for concentrating the product obtained from step b1).

Steps b1) and b2) are performed at least once. Thus, the step of clarification and/or ultrafiltration may be performed one time, two times, three times, four times or so forth.

In a further embodiment, step b1) comprises two steps of clarification.

In a still further embodiment, step b2) comprises two steps of ultrafiltration.

In a more preferred embodiment, step b1) and step b2) comprises two subsequently following steps of clarification or ultrafiltration, respectively.

Chromatographic Purifications and/or Concentrations

In the steps c) and d), the product is further purified and/or concentrated by means of chromatographic techniques. These techniques include anion- or cation exchange chromatography, hydrophilic interaction chromatography, SXC or SCMA, which can be performed either by binding the virus particle and washing out the contaminants or vice versa. The chromatographic techniques could also be flow through methods such as Capto Core 700 chromatography or size exclusion chromatography.

In one embodiment, the chromatographic purifications and/or concentrations are a combination of SXC and SCMA. In a further embodiment, the chromatographic purifications and/or concentrations are a SXC in step c) and a SCMA in step d). Alternatively, the chromatographic purification and/or concentrations are a SCMA in step c) and a SCMA in step d).

SXC

SXC is a membrane-based chromatography technology known as steric exclusion chromatography. The method is based on the steric exclusion of particles in a solution of an inert polymer, e.g. polyethylene glycol (PEG). This exclusion leads to the formation of polymer-rich and polymer-deficient zones, resulting in a thermodynamic instability, which is resolved by an association of the excluded particles with each other, and with the hydrophilic stationary phase, thus, retaining target molecules from the mobile phase. Adjustment of the PEG concentration and the molecular weight, with regard to the size of the product and expected process impurities, allows a selective product retention. Retained particles are eluted by the removal of the inert polymer from the mobile phase as known to the persons skilled in the art.

In one embodiment, step c) is performed using SXC.

The chromatographic process may be monitored continuously by techniques common to the persons skilled in the art e.g. by light-scattering techniques, system-integrated UV, measurements of conductivity and by using systems such as a Nano DLS Particle Size Analyzer.

The SXC may be run using membranes as known to the persons skilled in the art such as cellulose membranes like regenerated cellulose membranes, or other hydrophilic membranes such as polyamide, glass fibre membranes may be used. In one embodiment, the SXC is performed using cellulose membranes.

The number of membranes used for each separate run would depend on the product to be purified. However, in one embodiment, the SXC is performed using 4-20 membranes, such as 5-15 membranes, like 8-12 membranes, such as 10 membranes per column.

In a further embodiment, the membranes have a pore size in the range of 0.1-10 μm, such as in the range of 0.2-9 μm, like in the range of 0.3-8 μm, such as in the range of 0.4-7 μm, like in the range of 0.5-6 μm such as in the range of 0.6-5 μm, like in the range of 0.5-4 μm, such as in the range of 0.6-3 μm, like in the range of 0.7-2 μm, such as in the range of 0.8-1.5 μm, like in the range of 0.9-1.2 μm, such as around 1 μm. The purification ability depends on the size of the product to be purified why it is important to adjust the pore size to the size of the product to obtain optimal purification.

In an even further embodiment, the SXC is performed at a flow rate in the range of 0.1-20 mL/min, such as in the range of 0.2-18 mL/min, like in the range of 0.3-16 mL/min, such as in the range of 0.4-14 mL/min, like in the range of 0.5-12 mL/min, such as in the range of 0.6-10 mL/min, like in the range of 0.7-8 mL/min, such as in the range of 0.8-6 mL/min, like in the range of 1.0-4 mL/min, such as in the range of 1.5-3 mL/min, like around 2 mL/min.

The pH of the SXC may be adjusted depending on the target virus particle for it to be adjusted around the isoelectric point of the target particle. In one embodiment, purifying HCV, the SXC is performed at a pH in the range of 8-10, such as 8.5-9.5, like around 9. In a further embodiment, the pH is 8 or above. In an even further embodiment, the pH is below 11. In a further embodiment, the pH is adjusted to a level at which the virus particle would remain stable.

In a further embodiment, the elution buffer comprises a conductivity in the range of 0-300 mS/cm2, such as in the range of 25-250 mS/cm2, like in the range of 35-200 mS/cm2, such as in the range of 50-150 mS/cm2.

The elution buffer may be chosen from different salt combinations of anions like F−, SO42−, HPO42−, CH3COO−, NO3−, Br−, ClO3−, I−,ClO4−, SCN−, Cl3CCOO− and cations like NH4+, K+, Na+, Li+, Mg2+, Ca2+. In one embodiment, the elution buffer comprises NaCl. In a further embodiment, the concentration of NaCl is 0-2 M, such as 0.1-2 M.

In a further embodiment, the inert buffer used in the system is polyethylene glycol, which is advantageously mixed with the product prior to loading onto the SXC column.

Different PEG sized may be used depending on the virus particles to be purified such as PEG 4000 to PEG 20000. In one embodiment, PEG6000 is used.

The total concentration of PEG in the column may differ depending on the virus particles to be purified. In one embodiment, the final concentration in the column during loading and washing is in the range of 1-20%, such as 2-18%, like 3-16%, such as 4-14%, like 5-12%, such as 6-10%, like 7-9%, such as around 8%.

SCMA

SCMA is a pseudo affinity-based orthogonal technique. The method utilizes the heparin-mimicking effect of sulphated cellulose.

In one embodiment, step d) is performed using SCMA for subsequent polishing.

The chromatographic process may be monitored continuously by techniques common to the persons skilled in the art e.g. by light-scattering techniques, system-integrated UV, measurements of conductivity and by using systems such as a Nano DLS Particle Size Analyzer.

The SCMA may be run using membranes as known to the persons skilled in the art such as sulphated cellulose membranes. In one embodiment, the SCMA is performed using Sartobind® sulphated cellulose membranes.

The number of membranes used for each separate run would depend on the product to be purified. However, in one embodiment, the SCMA is performed using 1-20 membranes, such as 5-15 membranes, like 8-12 membranes, such as 10 membranes per column.

In a further embodiment, the membranes have a nominal pore size in the range of 0.1-5 μm, such as in the range of 0.2-4 μm, like in the range of 0.3-3 μm, such as in the range of 0.4-2 μm, like in the range of 0.5-1.5 μm, such as in the range of 0.6-1.2 μm, like around 0.8 μm. The purification ability depends on the size of the product to be purified why it is important to adjust the pore size to the size of the product to obtain optimal purification.

In an even further embodiment, the SCMA is performed at a flow rate in the range of 0.1-5 mL/min, such as in the range of 0.2-4 mL/min, like in the range of 0.3-3 mL/min, such as in the range of 0.4-2 mL/min, like in the range of 0.5-1 mL/min, such as in the range of 0.6-1 mL/min, like in the range of 0.7-0.9 mL/min, such as around 0.8 mL/min.

In a further embodiment, the elution buffer comprises a conductivity in the range of 10-300 mS/cm2, such as in the range of 25-250 mS/cm2, like in the range of 35-200 mS/cm2, such as in the range of 50-150 mS/cm2, like in the range of 80-110 mS/cm2, such as around 100 mS/cm2.

The elution buffer may be chosen from different salt combinations of anions like F−, SO42−−, HPO42−, CH3COO−, NO3−, Br−, ClO3−, I−, ClO4−, SCN−, Cl3CCOO− and cations like NH4+, K+, Na+, Li+, Mg2+, Ca2+. In one embodiment, the elution buffer comprises NaCl. In a further embodiment, the concentration of NaCl is 0.1-2 M, such as 0.5-0.7 M NaCl.

In embodiment, the present invention relates to a method of purifying whole virus particles, the method comprising the steps of


- - a) providing a cell culture supernatant comprising virus particles,
  - b) purification and/or concentration of the cell culture
    supernatant,
  - c) purification and/or concentration of the product of above step b)
    using SXC,
  - d) purification and/or concentration of the product of above step c)
    using SCMA,
  - e) obtaining purified whole virus particles.

In one particular embodiment, the method comprises the steps of:


- - a) providing a cell culture supernatant comprising virus particles,
  - b1) at least one clarification of the cell culture supernatant,
  - b2) at least one ultrafiltration of the product of step b1)
  - c) purification and/or concentration of the product of above step b)
    using SXC, optionally at alkaline pH in the range of 8-10, such as
    8.5-9.5, like around 9,
  - d) purification and/or concentration of the product of above step c)
    using SCMA,
  - e) obtaining purified whole virus particles.

In one particular embodiment, the method comprises the steps of:


- - a) providing a cell culture supernatant comprising virus particles,
  - b1a) first clarification of the cell culture supernatant,
  - b1b) second clarification of the product of step b1a)
  - b2a) first ultrafiltration of the product of step b1b)
  - b2b) second ultrafiltration of the product of step b2a)
  - c) purification and/or concentration of the product of above step b)
    using SXC, optionally at alkaline pH in the range of 8-10, such as
    8.5-9.5, like around 9,
  - d) purification and/or concentration of the product of above step c)
    using SCMA,
  - e) obtaining purified whole virus particles.

Inactivation, Nuclease Treatment and Freezing

In one embodiment, the method may further comprise a step of inactivating the virus particles. In principle, the virus particles may be inactivated before or after any of the steps b)-e). However, in a further embodiment, the virus particles are inactivated prior to step c). By prior to step c), is to be understood that the virus particles may be inactivated at any time during the process before step c) is initiated e.g. after step a), and/or after step b). Inactivation of the virus particles results in inactivated whole virus particles.

In another embodiment, the inactivation is performed using UV irradiation, UV combined with photosensitizer, paraformaldehyde, or betapropiolactone, binary ethylenimine, or gamma-irradiation. For certain chemicals and in case the inactivation is performed after step d) or e) an additional process step may be introduced to remove the chemical, used for the inactivation, such as diafiltration. In one embodiment, the inactivation is performed using UV irradiation.

In a further embodiment, the method further comprises a step of nuclease treatment. In a still further embodiment, the nuclease treatment is performed during step b). In an even further embodiment, the nuclease treatment is performed following the step(s) of ultrafiltration and optionally, after the step of clarification. In one embodiment, the nuclease treatment is performed prior to step c). The nuclease treatment may be performed using commercially available nucleases and according to the instructions of the manufacturer. Preferably, the nuclease activity is following blocked e.g. by addition of EDTA.

In a still further embodiment, the method comprises at least one step of nuclease treatment. In an even further embodiment, the method comprises two or more steps of nuclease treatment. As an example, the sample may be treated with nuclease during step b) and between step c) and d). In another embodiment, the sample is treated with nuclease during step b) and after step d). In an even further embodiment, the sample is treated with nuclease prior to step c) and between step c) and d). In a still further embodiment, the sample is treated with nuclease prior to step c) and after step d).

If required, an additional DNA reduction may be achieved by optimizing the nuclease treatment regarding the enzyme amount and the incubation time.

In one embodiment, the method comprises nuclease treatment during step b) and inactivation of the virus particles after step b).

In a further embodiment, the eluate does not comprise a step of freezing between step c) and step d). In a still further embodiment, the method does not comprise a step of freezing. As demonstrated in Example 5, better recovery rates are obtained if the samples are not frozen between these purification steps using SXC and SCMA.

Virus Particles

The method according to the present invention may be used for the purification of different virus particles obtaining purified whole virus particles. In one embodiment, the virus particle is non-enveloped or enveloped.

In a further embodiment, the virus particle belongs to the Flaviviridae family. In another embodiment, the virus particle belongs to the Coronaviridae family. In a still further embodiment, the virus particle is selected from the genus flavivirus, hepacivirus, pegivirus or pestivirus.

In a further embodiment, the virus particle is a virus selected from the group consisting of yellow fever virus, West Nile virus, dengue fever virus, tick borne encephalitis virus, Zika virus, Usutu virus, GB virus C, bovine viral diarrhea virus, classical swine fever, border disease virus coronaviridae, SARS-CoV-2, SARS-CoV, MERS-CoV, as well as human coronaviruses 229E, NL63, 0C43, and HKU1, HAV, HBV and HCV.

In one embodiment, the virus particles are HCV particles. In a further embodiment, the HCV is of a genotype selected from the group consisting of genotype 1, 2, 3, 4, 5, 6, 7 and 8 as well as their subtypes such as a, b, c, d, e, and f.

In a further embodiment, the HCV particles are of genotype 1a and/or genotype 5a. In a still further embodiment, the HCV particles are HCV genotype 1a such as strain TNcc (GenBank accession no. JX993348), strain H77 (GenBank accession no. KP098533.1) and strain HCV1 (GenBank accession no. KP098532). In an even further embodiment, the HCV particles are HCV genotype 5a such as strain SA13 (GenBank accession no. F3393024.1).

The hepatitis C virus can be either full length or an intra- or intergenotypic recombinant.

In a still further embodiment, the HCV comprises adaptive mutations. The adaptive mutations attenuate the virus in vivo. Cell cultures comprising HCV having adaptive mutations are known to be able to replicate and form viral particles in cell culture with high efficiency. These genomes have the complete functions of HCV and in consequence, they are able to produce infectious viruses.

The adaptive mutation is a mutation that can be observed by clonal or direct sequencing. One or more adaptive mutations may be present in Core, E1, E2, p7, NS2, NS3, NS4A, NS4B, NSSA or NSSB singly or in combination.

As an example, the virus particle may comprise at least one amino acid mutation selected from the group consisting of A1226G, F1464L, A1672S, Q1773H, N1927T, D2979G, Y2981F, F2994S according to the H77 reference sequence with GenBank accession number AF009606. In a further example, the virus particle may be TNcc (GenBank accession no. JX993348) as set forth in SEQ ID NO. 1 and SEQ ID NO. 2. In addition, TNcc may comprise additional mutations selected from the group detailed in Table 1.

As another example, the virus particle may encode a human hepatitis C virus wherein the hepatitis C virus is derived from genotype 5a, comprising at least one amino acid mutation selected from the group consisting of R114W, V187A, V235L, T385P, L782V, Y900C, A1021G, K1118R, N2034D, E2238G, V2252A, L2266P, 12340T, A2500S, V2841A according to the H77 reference sequence with GenBank accession number AF009606. In a further example, the virus particle may be SA13 (GenBank accession no. F3393024.1). In an even further example, the virus particle may be as set forth in SEQ ID NO. 3 and SEQ ID NO. 4.

In a further embodiment, the virus particles are obtained from a high-titre cell culture. This may be determined in IU/ml (international units/ml) with Taq-Man Real-Time-PCR and infectious titers are determined with a focus forming unit assay.

The infectious titers are determined as TCID50/ml (median tissue culture infectious dose/ml) or FFU/ml (focus forming unites/ml); in such method, infectivity titers are determined by infection of cell culture replicates with serial dilutions of virus containing supernatants and, following immunostainings for HCV antigens, counting of HCV-antigen positive cell foci.

HCV RNA titers and infectivity titers can be determined extracellularly, in cell culture supernatant (given as IU and TCID50 or FFU per ml, respectively) or intracellularly, in lysates of pelleted cells (given as IU and TCID50 or FFU related to a the given cell number or culture plate wells, which was lysed).

In another embodiment, the high titer would be an HCV infectivity titer of at least 102 FFU/ml or above following transfection and/or subsequent viral passage, such as a titer of at least 103 FFU/ml, such as a titer of at least 104 FFU/ml, such as a titer of at least 105 FFU/ml, such as a titer of at least 106 FFU/ml, such as a titer of at least 107 FFU/ml, such as a titer of at least 108 FFU/ml, such as a titer of at least 109 FFU/ml or such as a titer of at least 1010 FFU/ml.

The virus particles may be obtained from cell cultures that are cultured under different culture conditions and in different culture settings in order to provide cell culture supernatants comprising the virus particles for step a) of the method.

In one embodiment, the cell culture is grown on optimized surfaces, in suspension, on beads, on microcarriers, on macrocarriers, in cell factories or bioreactors. Virus particles grown in serum free medium may have favorable density profiles. Thus, in one such embodiment, the cell culture is grown in a serum free medium.

In a further embodiment, the cell culture is grown in an adenovirus expression medium, optionally supplemented with penicillin 100 U/ml and streptomycin 100 μg/ml.

The cell cultures used for culturing the virus particles are commonly known by the persons skilled in the art. In one embodiment, the cell line is a hepatocyte cell line such as Huh7 or derived cell lines e.g. Huh7.5 or Huh7.5.1. In one embodiment, the cells in the cell culture are Huh7.5 cells.

In a further embodiment, the cell culture is any cell expressing the genes necessary for HCV infection and replication, such as but not limited to CD81, SR-BI, Claudin-1, −4, −6 or −9, occludin, and the low-density lipoprotein receptor.

Various methods for producing e.g. HCV particles are commonly known to the persons skilled in the art and as e.g. described in WO2013/139339, WO2015/058772, WO2016/066171 and WO2019/154472, the content of which is hereby incorporated by reference. These include culturing a host expression cell line transfected with HCV RNA under conditions that permit expression of HCV particle proteins; and isolating HCV particles or particle proteins from the cell culture.

The replication level of a virus may be determined using techniques known in the art, and in other embodiments, as exemplified herein. For example, the genome level can be determined using RT-PCR, and northern blot. To determine the level of a viral protein, one can use techniques including ELISA, immunoprecipitation, immunofluorescence, EIA, RIA, and Western blotting analysis.

Pharmaceutical Compositions/Use of Purified Virus Particles

The purified virus particles such as HCV particles from cell cultures may be used for the development or production of therapeutics and vaccines as well as for diagnostic purposes after being purified using the method according to the present invention.

A further aspect of the present invention relates to purified whole virus particles obtained from the methods as herein described. These purified whole virus particles may form a whole virus vaccine candidate stock, such as a whole virus vaccine inactivated candidate stock.

A still further aspect of the present invention relates to a pharmaceutical composition comprising the whole virus vaccine inactivated candidate stock as herein described formulated with one or more adjuvant(s), excipients and/or carriers.

Such pharmaceutical compositions are ideal for use in immunizing and vaccination. They will also be key in facilitating virological studies, and for vaccine development.

Adjuvants include, but are not limited to, complete Freund's adjuvant, incomplete Freund's adjuvant, saponin, mineral gels such as aluminum hydroxide, surface active substances such as lysolecithin, pluronicpolyols, polyanions, peptides, oil or hydrocarbon emulsions, keyhole limpet hemocyanins, dinitrophenol, and potentially useful human adjuvants such as BCG (bacilleCalmette-Guerin), Corynebacteriumparvmm, aluminum hydroxide, aluminum hydroxide+MPL, Addavax, MF59, CAF01, CAF04, CAF05 and CAF09, AS03, CpG, CpG 1018, Toll-like receptor agonists such as, but not limited to poly:IC, and Sigma adjuvant system.

Preferably, the adjuvant is pharmaceutically acceptable.

In a further aspect, the present invention relates to use of whole virus particles obtained by a method as described herein for preparation of a whole virus vaccine candidate stock and/or a pharmaceutical composition.

It should be noted that embodiments and features described in the context of one of the aspects of the present invention also apply to the other aspects of the invention.

All patent and non-patent references cited in the present application, are hereby incorporated by reference in their entirety.

The invention will now be described in further details in the following non-limiting examples.

### EXAMPLES

**Example 1—Materials and Methods**

Huh7.5 Cell Culture

Huh7.5 cells were maintained in DMEM (Gibco™) with 10% fetal bovine serum (Sigma) and penicillin (100 U/mL)/streptomycin (100 μg/mL) (Sigma) and were incubated at 37° C. and 5% CO2. Adenovirus Expression Medium (AEM) (Gibco™), supplemented with penicillin (100 U/mL) and streptomycin (100 μg/mL), was used for HCV production under serum-free conditions.

The percentage of HCV infected cells was evaluated by immunostainings (Scheel, T. K. H. et al. 2011). In brief, cells were seeded in a chamber slide (Thermo Fisher Scientific) for a confluent cell layer, fixed with acetone (Merck) the next day, and stained with primary antibody 9E10 diluted 1:3000 (Lindenbach, B. D. et al., 2005), followed by secondary antibody Alexa Flour 488 goat anti-mouse IgG diluted 1:500 (Invitrogen), and Hoechst 33342 (Molecular Probes) diluted 1:1000.

HCV-infectivity titres were determined with three technical replicates as FFU/mL in a cell-based assay in 96-well plates as described (Scheel, T. K. H. et al. 2011). The immunostaining of 96-well plates was carried out with primary antibody 9E10 diluted 1:5000, secondary antibody ECL Anti-mouse IgG Horseradish Peroxidase linked from sheep (Amersham Biosciences) diluted 1:500, and visualized with Pierce™ DAB Substrate Kit (Thermo Scientific). 96-well plates were imaged and automatically counted for FFU quantification.

Serial Passage for Generation of High-Titre Genotype La HCV

For the production of genotype 1a HCV, the cell culture infectious recombinant TNcc (Li, Y.-P. et al. 2012) (GenBank accession no.: JX993348) was further adapted to cell culture by serial passage in Huh7.5 cells. Following a transfection of HCV RNA transcripts, 18 viral passages to naïve cells were carried out in T80 cell culture flasks (Nunc). Naïve cells were inoculated with cell culture supernatant derived from the previous culture at the peak of infection according to immunostainings as described (Mathiesen, C. K. et al. 2015). A passage 19, virus seed stock was prepared in T500 triple-layer-flasks (NUNC™ TripleFlask™ Treated Cell Culture Flask); supernatants from two time points at the peak of infection were pooled.

Production of Genotype 1a and 5a HCV for DSP Development

For genotype 1a HCV, T175 flasks, seeded with 6×106 cells in DMEM on the previous day, were inoculated at a multiplicity of infection 0.003 with the passage 19 virus seed stock. Cultures were expanded to T500 triple-layer-flasks. When 80% of cells were estimated to be infected by immunostaining in a replicate T25 culture, the cultures were washed with PBS (Sigma) and subsequently maintained in AEM under serum-free conditions. The supernatant was harvested five times every 2-3 days, yielding 10.5 L, which was stored at ˜80° C. until further processing.

For genotype 5a HCV (amino acid sequence—SEQ ID NO. 3; nucleotide sequence-SEQ ID NO. 4), 18×106 cells, seeded the previous day in DMEM, were infected at a multiplicity of infection 0.003 in T500 triple layer cell culture flasks with a 3rd passage seed stock of the further adapted SA13/JFH1 recombinant (Mathiesen, C. K. et al. 2015). The following day, cells were transferred to cell factories (NUNC™ Cell Factory™). When 80% of cells were expected to be infected as indicated by immunostaining in a replicate T25 culture, the cells were washed with PBS and cultured in AEM. The harvesting of the supernatant was carried out five times every 2-3 days, yielding 20.4 L total.

Sequence Analysis

NGS (next generation sequencing) of the virus populations was carried out as described (Jensen, S. B. et al. 2019). Briefly, RNA was extracted with Trizol LS and the RNA Clean & Concentrator™-5 (Zymo research) kit. The reverse transcription was carried out with Maxima H Minus Reverse Transcriptase (ThermoScientific), the whole open reading frame was amplified with PCR Q5® Hot start High-Fidelity DNA Polymerase (New England Biolabs), and the PCR product was purified (DNA Clean & Concentrator™-25 and Zymoclean™ Large Fragment DNA Recovery Kit, Zymo research). The NEBNext ultra II FS DNA Library Prep Kit was used for library preparation, and sequencing was performed with an Illumina Miseq platform.

The alignment of amino acid sequences of structural proteins (Core, envelope proteins E1 and E2) of 1a and 5a HCV was done in BLAST® (Altschul, S. F. et al. 1997, Altschul, S. F. et al. 2005).

Evaluation of Infectious HCV Stability at Alkaline pH Values

The HCV genotype 5a seed stock described above, was concentrated using Ultra-15 Centrifugal Filter Unit-100K (Amicon) and diluted by a factor 17 in PBS (for pH 7.4 reference), DMEM (standard cell culture medium, pH 8.5) or phosphate buffer (KH2PO4 (Sigma) and K2HPO4 (Sigma), adjusted with NaOH for pH values of 9.5, 10 and 11) in triplicate Eppendorf tubes, and incubated for 90 minutes at room temperature. After incubation, the virus/buffer solutions were diluted 1:40 in DMEM containing 20 mM HEPES (HEPES solution 1 M, Sigma), and added to triplicate wells seeded with 7×103 cells/well in 96-well poly-D lysine plates (Thermo Scientific) the previous day. The infected cell plates were incubated for six hours at 37° C. and 5% CO2 before the medium was exchanged to DMEM without HEPES. The cell plates were fixed, stained, and evaluated as described above, in order to quantify HCV infected single cells. The cell viability was evaluated after the experimental read-out had been obtained in a replicate experiment with the CellTiter 96® AQueous One Solution Cell Proliferation Assay (Promega) according to the manufacturer's protocol. The pH effect on virus stability was evaluated as the number of HCV-positive cells in a well, relative to the average number of HCV-positive cells obtained from the virus incubated at a pH value of 7.4.

Virus Clarification, Ultrafiltration and Inactivation

The genotype 1a and 5a virus material was passed over 5 μm and 0.65 μm Sartopure® PP3 (Sartorius) capsule filters for a two-step clarification by peristaltic pumping (Masterflex L/S 7554-95 Cole-Parmer, Masterflex L/S Easy Load pump head, and Extended Lifetime Silicone Tubing size 17 (Repligen)). Subsequently, the clarified 1a and 5a viruses were concentrated 249 and 325 times, respectively, in two sequential ultrafiltration steps with hollow fibre filters (MINIKROS SAMPLER 65CM 500K MPES 0.5MM ¾TC×¾TC STERILE followed by MINIKROS SAMPLER 20CM 500K MPES 0.5MM ¾TC×¾TC STERILE, both Repligen). The HCV infectivity titres and RNA titres were determined for samples from each step of clarification and concentration. The virus recovery was calculated from the HCV RNA titres.

The virus was inactivated by UV exposure (UVP Handheld UV lamp, UVG-54 254 nm in lamp stand) in 6-well plates (Nunc™) with 1.5-2.5 mL per well for eight hours. The E-well plate was kept on ice with frequent agitation. To confirm inactivation, naïve Huh7.5 cells were seeded in triplicate T25 flasks (1×106 cells/flask) the previous day, and inoculated with 20 μL of UV-treated material. Inoculated cultures were passaged for 21 days and monitored for HCV positive cells by immunostaining as described above. In replicate samples, it was confirmed that a similar incubation without UV irradiation did not inactivate HCV.

Nuclease Treatment

The nuclease treatment was performed in triplicates for both genotypes. The clarified virus was subjected to 250 U/mL Benzonase® nuclease (Merck) at a final concentration of 2 mM MgCl2. The incubation was done overnight at 4° C., and the nuclease activity was blocked afterwards, using a final concentration of 5 mM EDTA. Subsequently, the chromatography was performed, using nuclease—digested, clarified HCV.

Chromatographic Purification

The chromatographic experiments were done with an Akta Pure 25 system, operated by the Unicorn 7 software (GE Healthcare Life Sciences). Online monitoring was done by system-integrated UV (280 nm) and conductivity detectors, and additionally light-scattering was detected with a Nano DLS Particle Size Analyzer (Brookhaven Instruments). All chromatographic experiments were done in technical triplicates, unless stated otherwise.

Virus Capture Using SXC

SXC was performed using regenerated cellulose membranes with 1 μm pore size (Whatman), as previously reported (Lothert, K. et al. 2019). In brief, for preparing the column, 10 membranes were punched and stacked into a 13 mm filter holder (Pall), yielding a total membrane area of 13.3 cm2. All steps were performed at a flow rate of 2 mL/min. The stack was equilibrated using 5-10 mL of 20 mM Tris at the specified pH value, supplemented with 180 mM NaCl, and 8% PEG 6000. Clarified, concentrated, and inactivated HCV was mixed 1:4 with the above stated buffer and supplemented with 32% PEG to yield final concentrations of 8% PEG to match the equilibration conditions. After sample application, the stack was washed with equilibration buffer until the detector signals decreased to baseline (>5 mL). Elution was achieved using 20 mM Tris at pH 7.4 without PEG, but supplemented with 0.4 M NaCl. Initial screening SXC runs were tested at pH 7.4 to pH 11 for genotype 1a HCV, while final process conditions were at pH 9, and tested for robustness at pH 8.5 and 9.5. Following optimization, the SXC performance was verified for the genotype 5a HCV at pH 9 with a preceding nuclease treatment.

Virus Polishing Using SCMA

Sartobind® Sulphated Cellulose membranes with a nominal pore size of 0.8 μm (Sartorius Stedim Biotech GmbH) were punched to disks of 13 mm diameter. As for SXC, the disks were stacked to layers of 10 membranes (13.3 cm2 total membrane area). All steps were performed at a flow rate of 0.8 mL/min. Membranes were equilibrated using 20 mM Tris pH 7.4 prior to sample application. For the purification, SXC elution fractions were diluted 1:10 with equilibration buffer in order to reduce the conductivity of the solution below 5 mS/cm. After complete sample loading, the membranes were washed with equilibration buffer until UV- and light scattering signals returned to baseline. Bound components were subsequently eluted in three fractions, using increasing NaCl concentrations (0.6, 1.2 and 2 M). The SCMA was evaluated for genotype 1a HCV, using SXC elutions with and without an interim storage at ˜80° C., as well as with and without additional nuclease treatment before SXC. Finally, the SCMA performance was confirmed, using a nuclease-treated and SXC-purified genotype 5a HCV.

Determination of Dynamic Binding Capacities

For SXC and SCMA, the DBC (dynamic binding capacities) was determined in order to optimize the virus load on the membrane stacks. Stationary and mobile phase compositions were the same as described above. A clarified, concentrated, and inactivated virus feed of a known concentration was prepared and applied to the column, until detected breakthrough of 10% and 100% of the particles, based on the evaluation of the light-scattering detector signal. Depending on the loaded volume, the total amount of virus particles, at which breakthrough rates of 10% or 100% (DBC10 and DBC100) occurred, was calculated and related to the area of the membrane. All process runs were performed at or below DBC10.

HCV Quantification

The virus amount was evaluated using an in-house quantitative polymerase chain reaction as described previously, with minor modifications (Mathiesen, C. K. et al. 2014). Briefly, viral RNA was extracted from 200 μL sample and eluted in 50 μL water, using the High Pure Viral Nucleic Acid Kit (Roche) according to the manufacturer's instructions. Afterwards, a mixture comprising TaqMan® Fast Virus 1-step Mastermix (Thermo Fisher Scientific), nuclease-free water, probe (5′ FAM-CCTTGTGGTACTGCCTGA-MGB 3′; SEQ ID NO. 5 containing a FAM dye and an MGB quencher) and primers (Forward: 5′ AGYGTTGGGTYGCGAAAG 3′ [SEQ ID NO. 6]; Reverse: 5′ CACTCGCAAGCRCCCT 3′ [SEQ ID NO. 7]; Sigma-Aldrich) was prepared. 12 μL of that mixture were added to 8 μL of the extracted RNA in a 96-well PCR plate (twin.tec®, Eppendorf) preparing duplicates for each sample. The amplification was done using a Mastercycler Ep gradient S realplex (Eppendorf) after a pre-incubation period at 50° C. for 300 s. A total of 53 cycles of 95° C. for 20 s, followed by 62° C. for 60 s, were performed. An HCV standard panel, containing 102 to 106 IU/mL in 1-log increments, was prepared and included in each run, in addition to negative control samples. HCV RNA titres (IU/mL) were calculated using a standard curve generated from values obtained for the standard panel and corresponding cycle threshold values. The standard deviation of triplicate measurements was below 20%.

Protein Determination

For a quantification of the total protein amount contained in the chromatographic samples, the Pierce™ BCA Protein Assay Kit (Thermo Fisher Scientific) was applied according to the manufacturer's instructions. In brief, 25 μL of sample were transferred into a clear 96-well plate; duplicates were prepared for each sample. The standard panel (in the range of 25 to 2000 μg/mL) was prepared from gamma globulin according to the manufacturer's instructions. To each well, 200 μL of the reaction mix were added, and absorbance at 562 nm was measured after 30 min of incubation at 37° C. using the Cytation 3 plate reader (BioTek). The values obtained from a blank sample (buffer) were subtracted before interpolating the sample concentrations. Results given are from duplicate measurements with less than 10% standard deviation.

DNA Determination

The total amount of double stranded DNA (referred to as “DNA” in this work) was determined, using the Quant-iT™ PicoGreen® dsDNA Kit according to the manufacturers' instructions. The assay was performed in a 96-well format, using black microtiter plates (Nunc). Chromatographic samples, including the feed, were mixed 1:4 (S×C samples) or 1:2 (SCMA samples) with the assay's 1× TE buffer to a final volume of 100 μL. For each plate, blank samples (buffer) and two standard panels were prepared from kit-contained lambda-DNA in the range of 1 to 1000 ng/mL and 0.025 to 25 ng/mL, using a tenfold dilution series. After adding 100 μL of the reaction dye, the plate was incubated for 5 min in the dark, and a fluorescence emission at 520 nm (excitation: 485 nm) was subsequently determined, using the Cytation 3 plate reader (BioTek). All measurements were done in duplicates with a general standard deviation of less than 10%.

Ultracentrifugation-Based Downstream Process

This process is performed as described in U.S. Pat. No. 10,258,687 B2, which is hereby incorporated by reference, with a slight modification of introducing one additional filtration step upstream of the ultracentrifugation. In general, ˜16 L cell factory supernatants were clarified on Sartopure® PP3 filters with a pore size of 5 μM and 0.65 μM (Sartorius). Pooled supernatants were subjected to two ultrafiltration steps using mPES hollow fiber filters with a surface area (SA) of 2600 cm2 and molecular weight cut off (MWCO) of 500 kD followed by a fiber with SA of 790 cm2 and MWCO of 500 kD (Repligen) to yield a total volume of ˜30 mL.

For the subsequent 3-cushion ultracentrifugation (UC), 1 mL of undiluted Optiprep™ Density Gradient Medium (60%) (Sigma) was layered in a Beckman centrifuge tube (Beckman Coulter). Optiprep™ dilutions of 28% and 12% were made in phosphate buffered saline (PBS) (Sigma) and 1 mL of each was added to the tube sequentially.

Approximately 6 mL clarified and concentrated HCV-containing supernatant and PBS to a total volume of 11 mL were layered on top of the dilutions in each of 6 tubes and centrifuged in a Beckman XL-70 ultracentrifuge at 4° C. for 2 h at 40,000 rpm in a SW-41 rotor (Beckman Coulter).

Three fractions (F) of ˜8 mL (F1), ˜1.5 mL (F2) and ˜1.5 mL (F3) were collected from the top of each tube. Fraction 2 from 6 tubes were pooled to ˜9.5 mL, diluted in PBS to a total volume of 20 mL and subjected to cross flow filtration on a fiber with SA of 20 cm2 and MWCO of 500 kD (Repligen), resulting in a volume of ˜1.5 mL. For gradient UC, Optiprep™ dilutions of 40%, 30%, 20% and 10% in PBS were made and a semi-continuous gradient was prepared by sequentially adding 2.5 mL of each dilution to a tube followed by upright incubation overnight at 4° C.

Approximately 1.5 mL of HCV-containing material resulting from the previous step was loaded on top of the gradient in one tube and centrifuged for 6 h as described above. Eighteen fractions of ˜550 μL were eluted from the bottom of the tube and ˜400 μL of each fraction were weighed to determine fraction density. Three fractions with densities of ˜1.1 g/mL were selected and pooled.

Sephadex G-100 (Sigma Aldrich), swelled with sterile water three days earlier was added to a chromatography column (PD-10 reservoirs, GE Healthcare Life Sciences). HCV-containing material from gradient UC was added to the top of the column and allowed to enter before 12 fractions of 1 mL were eluted from the bottom of the column in NaCl (9 mg/mL), of which 5 fractions were selected based on OD (230 nM) measurement using a NanoDrop (Thermo Scientific) and pooled. The 5 mL pool was UV-irradiated by exposing ˜1.5 mL per well of a 6-well plate to a UVG-54 Handheld UV lamp (254 nm UV, 6 watt) (Analytik Jena) for 25 min.

**Example 2—Production of High-Titre Genotype La Virus Stock for DSP Development**

Aim

To produce a high-titre cell culture with infectious viruses in order to facilitate DSP development.

Results

The previously reported recombinant genotype 1a virus TNcc (Li, Y.-P. et al. 2012) was serially passaged in naïve human hepatoma cell line 7.5 (Huh7.5) cells for a further adaptation to the cell culture, until HCV infectivity titres of ˜6 log10, focus forming units (FFU)/mL, were observed for several passages.

A passage 19 stock was prepared, serving as the seed for the genotype 1a passage 20 virus production in triple-layer culture flasks, which was then used in the DSP development.

Next generation sequencing (NGS) revealed that, in addition to the eight cell-culture adaptive substitutions in the original 1a virus, passage 20 viruses had acquired 3 substitutions present in >50% of the virus population, G1909A in NS4B as well as N2651H and H2986R in NSSB (Table 1).

Conclusion

High-titre infectious genotype 1a viruses were successfully produced in cell culture at least partly due to acquisition of further adaptive mutations resulting in a heterogeneous population of viruses. The largest heterogeneity was observed in the non-structural proteins, which are not believed to be incorporated in the HCV particle.

**Example 3—HCV Clarification and Ultrafiltration**

Aim

To establish clarification and ultrafiltration as well as inactivation and to monitor virus recovery.

Results

A two-step filtration, with cut-offs of 5 μm and 0.65 μm, was selected for the clarification of genotype 1a HCV. Subsequently, the material was concentrated in two sequential ultrafiltration steps with a cut-off of 500 kDa from a starting volume of 10.5 L to volumes of 600 mL and 42 mL, respectively.

We observed a virtually complete virus recovery for the clarification and the first ultrafiltration step, whereas a 45% recovery was observed for the second ultrafiltration step (data not shown).

For inactivation, the resulting material was treated by UV irradiation, and naïve cell cultures were inoculated and followed for three weeks by regular immunostainings for the HCV NS5A antigen to confirm inactivation.

Conclusion

The two steps of clarification along with the first step of ultrafiltration resulted in an almost complete virus recovery. However, the second step of ultrafiltration resulted in depletion of 55% of the viruses from the sample. It should be noted that 55% is in the lower range of recoveries observed using ultrafiltration in other experiments. Thus, mean recovery of seven ultrafiltrations with the larger fibre was 86% and the mean recovery of eight ultrafiltrations with the smaller fibre was 77%.

Inactivation of the virus using UV irradiation resulted in virtually complete inactivation of the HCV viruses.

**Example 4—HCV Capture by SXC**

Aim

To study the ability of SXC to capture HCV including studying factors influencing the purity of the obtained sample as well as the recovery rate.

Results

In initial SXC experiments, using standard conditions with 8% PEG and physiological pH values, the majority of the viruses were found in the flow-through fraction, and an increase in the PEG concentration to up to 12% did not result in improvements.

We hypothesized that the isoelectric point (pI) of HCV was alkaline. Thus, prior to testing alkaline SXC conditions, we investigated HCV stability at different pH values. HCV was incubated in phosphate-buffered saline (PBS, pH 7.4), Dulbecco's Modified Eagle Medium (DMEM, standard cell culture medium, pH 8.5), and phosphate buffers for final pH values of ˜9.5, 10 and 11, prior to inoculation of naïve Huh7.5 cells for the determination of HCV infectivity.

Of note, the tested conditions did not result in an impairment of cell viability. HCV was stable when subjected to pH values of up to 10 for 90 minutes (FIG. 1), which equals the approximate duration of the SXC.

Testing alkaline SXC conditions revealed a large virus breakthrough at pH 8 and 10 during loading (FIG. 2A,C) based on light-scattering detection. At pH 11 a strongly increasing back pressure was observed with increasing loading volume during sample application and wash (FIG. 2D). This resulted in a reduced virus breakthrough, a decreased possible loading volume, and nearly no virus recovery in any of the fractions. In contrast, at pH 9, the virus breakthrough was minimized (FIG. 2B).

In an additional experiment, the qualitative data, based on the light-scattering signal, was verified by quantitative polymerase chain reaction (qPCR) analytics of the recovered viral RNA. Here, at pH 8.5, 9, and 9.5, a virtually full virus retention and recovery in the elution fraction could be achieved, with a product yield in the range of 90% to 105% (FIG. 3, Table 2). An additional nuclease treatment did not affect the SXC and resulted in similar yields of 99% (FIG. 3, Table 2) with minor amounts of virus found in the flow-through (2%) and wash (<1%) fractions.

The dynamic binding capacity (DBC) of the membranes was determined using 3.9E+08 international units (IU)/cm2 until a pressure limitation occurred and was approximately 2.1E+08 IU/cm2 until a 10% breakthrough was observed (DBC10%). However, due to an excessive pressure increase, it was not possible to load the virus until a 100% breakthrough occurred, thus DBC100% could not be determined.

The impurity removal did not depend on the pH value. For the runs without a preceding nuclease treatment as shown in FIG. 3 (pH 8.5, 9 and 9.5), the protein depletion was above 99% and the DNA depletion was at 84% (data for pH 9 shown in Table 2, other data not shown). The additional nuclease digestion, followed by SXC at pH 9, did not affect the protein depletion, which was above 99% (FIG. 4A, Table 2), but resulted in an increased DNA depletion of 94%, and DNA concentrations of 9 ng/mL at viral RNA titres of 9.3E+07 IU/mL after SXC (FIG. 4B, Table 2).

Conclusion

HCV may be purified by SXC having a high degree of recovery. In particular, a pH in the range of 8-10 appeared to result in a good recovery rate. Importantly the HCV virus particle was able to remain stable for the duration of the SXC under these alkaline conditions.

For SXC, the initial application of published process conditions (Marichal-Gallardo, P. et al. 2017) did not result in a successful virus retention. It was previously described, that the SXC performance is optimal at pH conditions near the pI (Lee, J. et al. 2012). For HCV, no characterization of the pI of the complete virus has been published so far. Testing alkaline pH conditions, we defined a small operating window for an optimal SXC performance at pH 9±0.5.

For HCV, the intense pressure increase observed during SXC at pH 11 suggested severe membrane fouling, resulting in a nearly complete virus retention, which hampers elution. This might be caused by the precipitation of proteins, medium components, or virus particles under these conditions. The pressure increase during DBC determination may be due to the possible aggregation potential of the virus, leading to increased membrane fouling for higher loading volumes. It is not likely, that an increased pore blockage is caused by protein impurities, as these are mostly washed out during sample application, with mainly virus particles remaining on the column. On a process scale, the pressure limitations might be additionally reduced as well as the binding capacity increased by using a different type of membrane housing, offering an altered angle of the incident flow.

Treatment with nuclease prior to the SXC run did not affect the protein depletion but increased the DNA depletion.

Accordingly, SXC may be used for purification of virus particles such as HCV with high recovery rates.

**Example 5—HCV Polishing by SCMA Chromatography**

Aim

To study the effect of further purifying the SXC elution obtained in Example 4 by means of SCMA chromatography.

Results

The SXC elutions, resulting from the SXC experiments done at pH 9 without and with preceding nuclease treatment (FIG. 3), were further processed using SCMA.

Following SXC without a preceding nuclease treatment, the HCV recovery was 63% in the 0.6 M NaCl elution fraction, if the SXC elution was directly processed without an additional freeze-thaw cycle (FIG. 5).

A storage at ˜80° C. in between the SXC and SCMA led to a reduction of retained and eluted viruses to 15%, with the majority of viruses found in the flow-through and wash fractions.

The implementation of an additional nuclease treatment prior to SXC resulted in a virus recovery of 50% in the 0.6 M NaCl elution fraction. Minor amounts of virus eluted at higher salt concentrations of 1.2 and 2M NaCl (FIG. 5), whereas 42% of the loaded virus was found in the flow-through and wash fractions.

Due to a breakthrough of about 40-50% of the virus, DBC10% and DBC100% could not be reached. However, the sample application was not limited by binding capacity, but by pressure, as the pressure after an application of ˜5.9E+09 IU/cm2 exceeded the limits.

Considering the removal of impurities, no proteins could be detected in any of the SCMA fractions (flow-through, wash, elution/-s) obtained following SXC (pH 9) without or with preceding nuclease treatment, indicating a protein content below the assays' limit of detection (<25 μg/mL) and hence a virtually complete protein depletion (FIG. 4A for nuclease treatment+SXC (pH 9)+SCMA; Table 2 for both datasets). The overall DNA depletion was 90% following SXC (pH 9) and SCMA without a preceding nuclease treatment compared to the initial feed concentration before SXC (Table 2). The introduction of a nuclease digestion followed by SXC (pH 9) and SCMA resulted in an increased DNA depletion of above 99% compared to the initial feed concentration before nuclease treatment and SXC, leading to DNA concentrations of about 2 ng/mL at viral RNA titres of 3.5E+07 IU/mL (FIG. 4B, Table 2).

Conclusion

The DSP developed in this study consisted of clarification, ultrafiltration, nuclease treatment, and SXC and SCMA steps. While filtration-based clarification and concentration are commonly used initial process steps for the production of viral vaccines, a major rationale for using SXC for the virus capture was the predominant dependency on the size of the target species (Lee, J. et al. 2012). This promised an independence with regard to the specific HCV genotype and the robust depletion of smaller impurities. SCMA was selected for virus polishing, based on the heparin affinity of HCV (Fortuna, A. R. et al. 2019). Another benefit of the chosen methodology is the possibility to directly load the SXC eluent to the SCMA—if necessary by an inline dilution.

Our data highlights the importance of avoiding a freeze-thaw cycle in a SXC elution buffer preceding SCMA, which resulted in a large decrease in recovery, possibly due to a degradation of HCV particles, or changes in the surface protein composition or structure. Although in general storage times using freeze-thaw cycles are unusual during a production process, this information may support similar trials in other laboratories.

With regard to virus recovery, no significant differences (according to a students' T-test, data not shown) were observed for samples that had been subjected to a nuclease treatment+SXC prior to SCMA (50% for 1a HCV) compared to samples that had been processed by SXC only prior to SCMA (63% for 1a HCV), with respect to the analytical error. Thus, SCMA appeared to be unaffected by a preceding nuclease treatment and independent of the virus genotype.

We observed a highly efficient protein depletion. Within the analytical error, a virtually full protein depletion could be achieved by SXC, with a protein removal of >99% for 1a HCV. Following SCMA for 5a HCV, protein levels were below the detection limit. The absence of protein in the SCMA flow-through and wash fractions may be caused by the sample dilution preceding the SCMA.

A comparable DNA depletion of at least 98% was achieved following a nuclease treatment, SXC, and SCMA. Of this overall DNA depletion, 5-12% were achieved by SCMA, whereas the nuclease treatment allowed an additional removal of about 10% of the total DNA. Most likely, the remaining DNA represents fragments attached to the virus as described above, or DNA being co-eluted with the virus particles using 0.6 M NaCl as SCMA elution buffer. For the latter, a further optimization of the SCMA procedure, including the evaluation of the virus elution using buffers with lower conductivity, is conceivable.

Accordingly, it is shown that this method of purification results in a high recovery of virus particles together with a low level of impurities.

**Example 6—the Developed DSP was Equally Efficient for Different HCV Genotypes**

Aim

To investigate the applicability of the developed DSP for different HCV isolates, we applied this strategy to a high-titre cell culture-derived genotype 5a virus.

Results

The 5a virus differs with structural proteins differing in ˜20% and envelope proteins differing in ˜26% from the genotype 1a virus on the amino acid level.

The 5a virus was produced in cell factories; NGS showed that, in comparison to the published sequence, no additional substitutions were present in >2% of the viral population.

Clarification and ultrafiltration were carried out as for the 1a virus, with a volume reduction from 20.4 L to 420 mL and 63 mL in the first and second ultrafiltration, respectively. During clarification and the first ultrafiltration, we observed a virtually complete virus recovery, whereas an 87% recovery was observed for the second ultrafiltration step. The resulting 5a material was UV-irradiated and the inactivation was confirmed as described for the 1a material.

With a preceding nuclease digestion, the SXC virus recovery was 97% (FIG. 6A, Table 2). During the SXC, the DBC10% was determined with 9.8E+07 IU/cm2 and a sample application was pressure-limited at about 2.7E+08 IU/cm2.

SCMA was carried out directly after SXC and resulted in a virus recovery of 49% in the 0.6 M NaCl elution fraction, whereas 47% of the applied virus was lost in the flow-through (FIG. 6B, Table 2). As for the 1a virus, DBC10% or DBC100% could not be determined during SCMA.

The whole process led to a complete protein removal with a protein depletion of 97% after SXC as well as undetectable protein levels in the SCMA fractions (FIG. 6A,C, Table 2). The DNA depletion was 86% after SXC and 98% after SCMA compared to the initial feed concentration before nuclease treatment and SXC (FIG. 6A,B,D, Table 2). In the SCMA eluate, the DNA concentration was 3 ng/mL at viral RNA titres of 3.2E+07 IU/mL (FIG. 6B,D, Table 2).

Conclusion

Importantly, the described DSP showed a similar performance for two major HCV genotypes (1a and 5a), facilitating the development of vaccines targeting different HCV genotypes.

With regard to virus recovery, no significant differences (according to a students' T-test, data not shown) were observed for samples that had been subjected to a nuclease treatment+SXC prior to SCMA (49% for 5a HCV) compared to samples that had been processed by SXC only prior to SCMA, with respect to the analytical error. Thus, SCMA appeared to be unaffected by a preceding nuclease treatment and independent of the virus genotype.

We observed a highly efficient protein depletion. Within the analytical error, a virtually full protein depletion could be achieved by SXC, with a protein removal of 3% of the remaining proteins in the SXC elution for 5a HCV. Following SCMA for 5a HCV, protein levels were below the detection limit. The absence of protein in the SCMA flow-through and wash fractions may be caused by the sample dilution preceding the SCMA. Since the SCMA elution resulted in a sample concentration, the absence of proteins in the final 5a HCV product suggested a successful depletion of the remaining proteins during SCMA.

Additionally, it should be mentioned, that not the entire DNA amount in the SCMA feed could be recovered in the subsequent fractions. This might be caused by the remaining DNA on the column, and by an inhomogeneous error distribution between the varying salt concentrations.

The DSP may not only be used for successful recovery of genotype 1a but also results in successful recovery of HCV genotype 5a.

**Example 7— Comparison of a Chromatography-Based DSP to an Ultracentrifugation-Based Downstream Process**

Aim

To study the efficiency of the method according to the present invention as compared to ultracentrifugation-based downstream processes (DSP) as commonly used in the technical area.

Results

The method according to the present invention (chromatography-based) was performed as described in the materials and methods section as described in Example 1 for HCV genotype 5a.

The results of several experiments demonstrated that the chromatography-based techniques according to the present invention in general showed the following recovery percentage:

Chromatography-Based Technique According to the Present Invention:


- - i. Clarification ˜100% recovery
  - ii. Ultrafiltration ˜50-100% recovery (mean recovery of 7
    experiments employing large followed by small hollow fiber was 70%)
  - iii. SXC ˜100% recovery
  - iv. SCMA ˜50-60% recovery

Steps downstream of clarification and ultrafiltration (step iii and iv) resulted in a ˜50% total recovery.

The ultracentrifugation-based downstream process was performed on HCV genotype 5a as described in Example 1.

FIG. 7 shows a representative example of the results of HCV infectivity titers and sample volumes from specified process steps of an ultracentrifugation-based technique for comparison to the present invention.

The ultracentrifugation-based technique includes clarification (FIG. 7: Clarification) and concentration by ultrafiltration with a large and smaller hollow fiber filter (FIG. 7: Ultrafiltration 2600 cm2 and Ultrafiltration 720 cm2) using a starting volume of 9.6 L HCV containing cell culture supernatant. 37 mL concentrated virus was passed to the first ultracentrifugation step with 3 cushions of different iodixanol concentrations (60, 28 and 12%) and three fractions were collected. Fraction 2 (FIG. 7: 3-cushion UC) consisting of 9.9 ml with an HCV infectivity titer of 9 log10 FFU/mL was further processed in a buffer exchange step by ultrafiltration using a 20 cm2 hollow fiber filter (FIG. 7: Ultrafiltration 20 cm2). The second ultracentrifugation using a 10-40% iodixanol gradient yielded 18 fractions, 3 fractions were selected based on buoyant density and pooled; this ˜1.6 mL pool had a titer of 9.5 log10 FFU/mL (FIG. 7: Gradient UC pool). Iodixanol was removed by sephadex chromatography resulting in 12 fractions. Five fractions were selected based on optical density and pooled; this pool hada titer of 8.9 log10 FFU/mL (FIG. 7: Purified HCV preparation). This purified HCV preparation was UV-inactivated for a final ˜4 mL preparation of purified inactivated HCV.

The results of several experiments demonstrated that the ultracentrifugation-based technique in general showed the following recovery percentage:

Ultracentrifugation-Based Technique:


- - i. Clarification ˜100% recovery
  - ii. Ultrafiltration filtration ˜50-100% recovery (mean recovery of 7
    experiments employing large followed by small hollow fiber was 70%)
  - iii. Ultracentrifugation ˜31-64% recovery (mean recovery of 3
    experiments was 49%)
  - iv. Small-scale ultrafiltration ˜71-100% recovery (mean recovery of
    3 experiments was 94%)
  - v. Ultracentrifugation ˜21-100% recovery (mean recovery of 3
    experiments was 75%)
  - vi. Sephadex chromatography ˜45-100% recovery (mean recovery of 3
    experiments was 78%)

Steps downstream of clarification and ultrafiltration (steps iii to vi) resulted in a ˜6-39% total recovery (mean recovery of 3 experiments was 22%).

Furthermore, the chromatography-based technique allows a scalable set-up, an improved recovery and real-time monitoring during process steps. In contrast, the ultracentrifugation-based technique is not easily scalable, shows large variation in recovery and has a delayed monitoring read-out.

Additionally, the contamination of the products obtained by the two processes were measured as described under Example 1. These results show that for the 5a HCV the ultracentrifugation-based technique resulted in residual contaminant levels of 400-500 μg (protein) and 150-250 ng (DNA) compared to 1E+08 viruses (RNA titer). The chromatography-based technique resulted for the 5a HCV in a complete protein removal (<20 μg/ml, being the lower limit of detection) and remaining DNA levels of 9 ng compared to 1E+08 viruses.

Conclusion

This comparison between the techniques demonstrates the advantages in recovery rates of the new chromatography-based technique as compared to the ultracentrifugation-based technique. Thus, the chromatography based technique is superior the ultracentrifugation-based technique.

In addition, the ultracentrifugation-based technique showed high variability.

Table 1 shows a next generation sequencing analysis of the open reading frame of the 1a HCV virus production. The sequence of 1a virus produced for DSP development (passage 20) was analyzed. Nucleotide and protein positions are stated relative to the TNcc sequence (GenBank accession no. JX993348) and are equivalent to positions in the 1a H77 sequence (GenBank accession no. AF009606). Positions with changes of at least 20% population prevalence are included in the table.

Table 2 shows an overview on viral recovery and impurity depletion for the two chromatography-based process steps and cell culture-derived genotype 1a and 5a HCV. Shown are the values for SXC capture at pH 9 and the SCMA polishing using a TRIS buffer at pH 7.4. Recoveries are step recoveries comparing feed and product fractions of the respective step, and depletions are overall values, related to the initial feed concentrations before nuclease treatment and SXC. For a better overview, normalized DNA contents are given for each step, calculated for virus titres of 1.0E+08. While stated values for protein and DNA concentrations are rounded, values for % protein and DNA depletion as well as DNA per 1.0E+08 IU/ml were calculated using non-rounded values.

