# DESCRIPTION

## FIELD OF THE INVENTION

This invention relates to methods for the detection of a bovine that is affected by or carrier of brachyspina, an inherited defect with autosomal recessive inheritance. The present invention provides methods for determining whether a bovine is affected by or carrier of brachyspina by analyzing its genomic DNA or its RNA. The methods include obtaining a sample of material containing genomic DNA or RNA from the bovine, and genotyping said nucleic acid for the presence of a 3.3 Kb deletion (nucleotide positions 20537017 to 20540346 in the bTau4.0 genome build) eliminating exons 25, 26 and 27 of the bovine FANCI gene, the mutation causing the brachyspina syndrome.

## DESCRIPTION OF THE BACKGROUND ART

### Marker Assisted Selection Against Genetic Defects in Livestock.

Intense selection for desired characteristics in livestock often results in increased inbreeding which contributes to the emergence of novel recessive defects. Examples of such outburst in Holstein-Friesian cattle include bovine leucocyte adhesion deficiency (BLAD)(1) and complex vertebral malformation (CVM)(2). Calf mortality resulting from such defects causes important economic losses and raises welfare concerns.

Most inherited defects are autosomal recessive, and are typically due to loss-of-function mutations (symbol “d”) in essential genes. Matings between animals that are healthy but carry one copy of the mutation (genotype “+/d”, i.e. “carriers”) will yield 25% of homozygous mutant animals (genotype “d/d”) that will be affected. A diagnostic test that allows the identification of +/d carrier animals, can be used either to cull carrier animals thereby eliminating the mutation and hence the defect from the population, or to avoid “at risk” matings between carrier sires and dams. The recent development of highly effective genomic tools, now allows for the rapid identification of the causative “d” mutations at the molecular level (3). Once identified, effective diagnostic tests can be developed using a range of generic DNA-based technologies that are well known by the people skilled in the art.

### The Brachyspina Syndrome and Locus.

Recently (4) a new genetic defect, referred to as the brachyspina syndrome, was identified in Holstein-Friesian dairy cattle. Affected animals are characterized by severely reduced body weight, growth retardation, severe vertebral malformations associated with a significant shortening of the spine (brachyspina) and long and slender limbs. In addition, affected animals exhibit inferior brachygnatism as well as malformation of the inner organs, in particular the heart, kidneys and testis. All reported cases traced back on both sire and dam side to a common ancestor, suggesting autosomal recessive transmission.

We previously positioned the brachyspina locus in a 2.46 Mb genomic segment on bovine chromosome 21 (5) using recently developed 50K SNP arrays and a statistical approach called “autozygosity mapping” (3). Based on these findings, we developed an “indirect” diagnostic test on the basis of a panel of SNP markers spanning the brachyspina locus. Such an indirect test, often referred to as haplotype-based test, can and has already been used to detect +/d carrier animals. However, because the association between the disease causing “d” allele and the SNP alleles is not perfect, such indirect test suffer from a lack of senstivity and specificity. Some homozygous +/+ animals may erroneously be called carriers, while some +/d carrier animals may be missed. Improved diagnostic tests, ideally based on the detection of the causative mutation hence having near-perfect sensibility and specificity, are thus needed.

### Effect of the Brachyspina Syndrome on Fertility.

We previously (5) reported that cows inseminated with sperm from sires that carry the brachyspina mutation show a decrease in non-return rate (the fact of not returning into heat as a result of successful pregnancy), an increase in stillbirth, and an increased culling rate. All these features are thought to result from embryonic and fetal mortality of ˜4% of conceptuses. In addition to causing the brachyspina syndrome, the brachyspina mutation or mutations thus has/have an important effect on male and female fertility, two of the most important economic traits in dairy cattle breeding. Being able to detect the brachyspina mutation(s) via an appropriate diagnostic test would thus have an important impact on improving fertility in Holstein-Friesian dairy cattle.

## SUMMARY OF THE INVENTION

In view of the above, the technical problem underlying the present invention was to provide means and methods that allow for a selective and convenient diagnosis of brachyspina or of carrier-status for this disease in cattle. The solution to said technical problem is achieved by the embodiments characterized in the claims. The present invention provides for the first time the identity of the mutation causing the brachyspina syndrome in cattle.

Thus, the present invention relates in a first embodiment to a method of detecting brachyspina syndrome from a bovine biological sample comprising genotyping a polynucleotide, DNA or RNA, for a deletion causing Brachyspina. In particular the present invention provides a method for determining whether a bovine is affected by or a carrier of brachyspina (BS) by analyzing its genomic DNA, the method comprising the steps of:


- - a) extracting the DNA from a sample of biological material
    containing said genomic DNA obtained from the bovine,
  - b) genotyping said DNA for a deletion in the interval between
    nucleotide positions 20537017 to 20540346 on bovine chromosome 21
    (bTau4.0), and
  - c) determining whether said animal carries the brachyspina
    mutation.  
    Equally preferred is:

A method for determining whether a bovine is affected by brachyspina (BS) or a carrier of brachyspina (BS) by analyzing its genomic DNA, the method comprising the steps of:


- - a) obtaining a sample of material containing said genomic DNA from
    the bovine,
  - b) extracting the DNA from said sample,
  - c) genotyping said DNA for a deletion in the interval between
    nucleotide positions 20537017 to 20540346 on bovine chromosome 21
    (bTau4.0), and
  - d) determining whether said animal carries the brachyspina mutation.

The term “bovine” in accordance with the present invention encompasses all cattle or cattle breeds from the species bos taurus. In a preferred embodiment of the methods of the present invention the bovine is selected from the group consisting of Holstein, Friesian and Holstein-Friesian Cross breeds, British and/or Dutch Friesian.

The term “carrier of brachyspina” refers to a bovine that carries the mutation causing the brachyspina defect on one of its chromosomes (whether inherited from sire or dam), and a wild-type allele on the other chromosome.

The term “sample” or “biological sample” according to the present invention refers to any material containing nuclear DNA from said bovine to be tested. In a preferred embodiment the biological sample to be used in the methods of the present invention is selected from the group consisting of blood, sperm, hair roots, milk, as well as body fluids including nucleated cells. Even more preferred as a biological sample is a tissue or tissues including nucleated cells.

Thus, in a further embodiment a method is provided for determining in a biological sample whether a bovine is a) unaffected b) has brachyspina or c) is a carrier of the disease.

DNA extraction/isolation and purification methods are well-known in the art and can be applied in the present invention. Standard protocols for the isolation of genomic DNA are inter alia referred to in Sambrook, J., Russell, D. W., Molecular Cloning: A Laboratory Manual, the third edition, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., 1.31-1.38, 2001 and Sharma, R. C., et al., A rapid procedure for isolation of RNA-free genomic DNA from mammalian cells, BioTechniques, 14, 176-178, 1993.

The term “brachyspina mutation” or “brachyspina deletion” or “deletion” in accordance with the present invention refers to a deletion in the interval between nucleotide positions 20537017 to 20540346 (bTau4.0) on bovine chromosome 21. Thus, in a preferred aspect the deletion encompasses 3,329 base pairs (also referred to herein as the 3.3 Kb deletion) spanning nucleotide positions 20537017 to 20540346 on bovine chromosome 21 (bTau4.0) in the bovine FANCI gene. Preferred deletions remove exons 25, 26 and 27 from the bovine FANCI gene. The brachyspina mutation/deletion is predicted to cause a frameshift at amino-acid position 877 substituting the 451 carboxyterminal amino-acids with a 26-residue long illegitimate peptide. Moreover, the ensuing stop codon in exon 28 is expected to cause nonsense mediated RNA decay.

The term “genotyping said DNA for the brachyspina mutation” or “genotyping said DNA for the brachyspina deletion” in accordance with the present invention refers to a method for determining or identifying whether a particular nucleotide sequence is present in a DNA sample. There are several methods known by those skilled in the art, e.g. (6) for determining whether such nucleotide sequence is present in a DNA sample. These include the amplification of a DNA segment encompassing the mutation by means of the polymerase chain reaction (PCR) or any other amplification method, and interrogate the amplicons by means of allele specific hybridization, or the 3′exonuclease assay (Taqman assay), or fluorescent dye and quenching agent-based PCR assay, or the use of allele-specific restriction enzymes (RFLP-based techniques), or direct sequencing, or the oligonucleotide ligation assay (OLA), or pyrosequencing, or the invader assay, or minisequencing, or DHPLC-based techniques, or single strand conformational polymorphism (SSCP), or allele-specific PCR, or denaturating gradient gel electrophoresis (DGGE), temperature gradient gel electrophoresis (TGGE), chemical mismatch cleavage (CMC), heteroduplex analysis based system, techniques based on mass spectroscopy, invasive cleavage assay, polymorphism ratio sequencing (PRS), microarrays, a rolling circle extension assay, HPLC-based techniques, extension based assays, ARMS (Amplification Refractory Mutation System), ALEX (Amplification Refractory Mutation Linear Extension), SBCE (Single base chain extension), a molecular beacon assay, invader (Third wave technologies), a ligase chain reaction assay, 5′-nuclease assay-based techniques, hybridization capillary array electrophoresis (CAE), protein truncation assay (PTT), immunoassays and solid phase hybridization (dot blot, reverse dot blot, chips). This list of methods is not meant to be exclusive, but just to illustrate the diversity of available methods. Some of these methods can be performed in accordance with the methods of the present invention in microarray format (microchips) or on beads.

The invention thus also relates to the use of primers or primer pairs, wherein the primers or primer pairs hybridize(s) under stringent conditions to the DNA corresponding the brachyspina deletion (nucleotide positions 20537017 to 20540346 (bTau4.0) or flanking it (i.e. for instance nucleotide positions 20527017 to 20537017 and 20540346 to 20550346 (bTau4.0), or to the complementary strand thereof.

Preferably, the primers of the invention have a length of at least 14 nucleotides such as 17 or 21 nucleotides.

In one embodiment of the diagnostic test, two primer sets are simultaneously used to respectively amplify the wild-type and the mutant allele. The corresponding amplicons are respectively detected using a 5′exonuclease assay using internal primers that respectively recognize the wild-type and mutant allele under stringent hybridization conditions. “Stringent or highly stringent conditions” of hybridization are well known to or can be established by the person skilled in the art according to conventional protocols. Appropriate stringent conditions for each sequence may be established on the basis of well-known parameters such as temperature, composition of the nucleic acid molecules, salt conditions etc.: see, for example, Sambrook et al., “Molecular Cloning, A Laboratory Manual”; CSH Press, Cold Spring Harbor, 1989 or Higgins and Hames (eds.), “Nucleic acid hybridization, a practical approach”, IRL Press, Oxford 1985, see in particular the chapter “Hybridization Strategy” by Britten & Davidson, 3 to 15. Typical (highly stringent) conditions comprise hybridization at 65° C. in 0.5×SSC and 0.1% SDS or hybridization at 42° C. in 50% formamide, 4×SSC and 0.1% SDS. Hybridization is usually followed by washing to remove unspecific signals. Washing conditions include conditions such as 65° C., 0.2×SSC and 0.1% SDS or 2×SSC and 0.1% SDS or 0,3×SSC and 0.1% SDS at 25° C.-65° C.

The term “base positions 20537017 to 20540346 on bovine chromosome 21” refers to the Bos taurus reference sequence (bTau4.0) which can be retrieved from e.g. the UCSC, Ensembl, and NCBI genome browsers. The Btau—4.0 was generated by the Atlas genome assembly system at Baylor College of Medicine Human Genome Sequencing Center. The sequencing strategy combined BAC shotgun reads with whole genome shotgun reads from small insert libraries as well as BAC end sequences. The nucleotide reference sequence of the present invention spanning base positions 20156961 to 22499122 of bovine chromosome 21 (bTau—4.0) as referred to herein is included within the scope of the present invention and shown in SEQ ID NO.: 1 of PCT/EP2009/058190.

The wild type allelic sequence of the bovine FANCI gene is provided herewith as SEQ ID NO: 1. This sequence is located in the interval between nucleotide positions 20,485,327 to 20,551,026 of the reference sequence bTau4.0 on bovine chromosome 21.

An exemplary and particular preferred mutated allelic sequence of the bovine “brachyspina mutation” or “brachyspina deletion” in accordance with the present invention in the FANCI gene is provided herewith as SEQ ID NO: 2. This sequence is located in the interval between nucleotide positions 20,485,327 to 20,551,026 of the reference sequence bTau4.0 on bovine chromosome 21.

In a further embodiment of the present invention a method is provided for determining whether a bovine is affected by or a carrier of brachyspina (BS) by analyzing its genomic RNA, the method comprising the steps of:


- - a) extracting the RNA from a sample of biological material
    containing said genomic RNA obtained from the bovine,
  - b) genotyping said for a deletion in the interval between nucleotide
    positions 20537017 to 20540346 on bovine chromosome 21 (bTau4.0),
    and
  - c) determining whether said animal carries the brachyspina mutation.

In a preferred aspect the deletion encompasses 3,329 base pairs spanning nucleotide positions 20537017 to 20540346 on bovine chromosome 21 (bTau4.0) in the bovine FANCI gene.

Equally preferred is:

A method for determining whether a bovine is affected by brachyspina (BS) or a carrier of brachyspina (BS) by analyzing its genomic RNA, the method comprising the steps of:


- - a) obtaining a sample of material containing said genomic RNA from
    the bovine,
  - b) extracting the RNA from said sample,
  - c) genotyping said RNA for a deletion in the interval between
    nucleotide positions 20537017 to 20540346 on bovine chromosome 21
    (bTau4.0); and
  - d) determining whether said animal carries the brachyspina mutation.

Again, in a preferred aspect the deletion encompasses 3,329 base pairs spanning nucleotide positions 20537017 to 20540346 on bovine chromosome 21 (bTau4.0) in the bovine FANCI gene.

“RNA” as referred to herein encompasses all types of RNA. Techniques well known in the art can be used for the isolation of total RNA, mitochondrial RNA, or messenger RNA. The person skilled in the art can select a suitable extraction method without further ado depending on the nature of the sample to be tested.

If a sample containing RNA is to be used in accordance with the present invention as a template for an amplification reaction, it will be necessary to transcribe said RNA in cDNA before amplification can be carried out. Again, techniques for doing so are well known to the person skilled in the art. As an example the RNA may be purified with RNeasy™ Mini Kit (Qiagen). The RNA will then be reversely transcribed to cDNA using, e.g. the SuperScript™ Choice System (Invitrogen).

In another embodiment of the present invention a method is provided for determining whether a bovine is affected by or a carrier of brachyspina syndrome (BS) by analyzing its genomic DNA or RNA, the method comprising the steps of:


- - a) extracting the DNA or RNA from a sample of biological material
    containing said genomic DNA or RNA obtained from the bovine,
  - b) genotyping said DNA or RNA for a deletion in the interval between
    nucleotide positions 20537017 to 20540346 on bovine chromosome 21
    (bTau4.0), further comprising genetic markers that are linked to the
    brachyspina locus; and
  - c) determining whether said animal carries the brachyspina mutation.

In a further embodiment of the present invention a method is provided for determining whether a bovine is affected with the brachyspina syndrome or carrier of brachyspina by analyzing its DNA or RNA, the method comprising the steps of:


- - a. obtaining a sample of material containing said genomic DNA or RNA
    from the bovine,
  - b. extracting the DNA or RNA from said sample,
  - c. genotyping said DNA or RNA for genetic markers that are linked to
    the brachyspina locus; and
  - d. ascertaining whether said animal carries the brachyspina mutation
    by linkage analysis.

Thus, in a further preferred aspect of the present invention the genotyping step of the claimed method further utilises genetic markers that are linked to the brachyspina locus.

The term “brachyspina locus” as defined in the present invention means a polynucleotide sequence in the bovine FANCI gene on chromosome 21 which when mutated or deleted is causative for brachyspina or leading to a brachyspina carrier status. In a preferred embodiment, “brachyspina locus” is the region encompasses 3,329 base pairs spanning nucleotide positions 20537017 to 20540346 on bovine chromosome 21 (bTau4.0) in the bovine FANCI gene.

The term “genetic markers linked to the brachyspina locus” refers to DNA sequence variants such as microsatellite markers or Single Nucleotide Markers (SNPs) that are located on bovine chromosomes 21 at less than 50% genetic recombination units from the brachyspina locus and which can be used in accordance with the present invention. In the bovine, 50% genetic recombination units correspond to approximately 50 million base pairs. Preferred genetic marker molecules of the present invention are selected from the group consisting of SNP markers located within 1 million base pairs from the FANCI gene.

The term “ascertaining” or “determining” “whether said animal carries the brachyspina mutation by linkage analysis”, or “ascertaining” or “determining” whether said animal carries the brachyspina deletion by linkage analysis” refers to the determination of which allele at any of the genetic markers linked to the brachyspina locus is associated with the brachyspina mutation in a known carrier parent (which can be either the sire, the dam or both), and determining whether such linked marker allele is transmitted to the tested individual using standard linkage analysis procedures which are well known by those skilled in the art. Standard linkage analysis procedures are inter alia referred to in (7).

Further provided is a method for determining whether a bovine is affected by or a carrier of brachyspina syndrome (BS) by analyzing its genomic DNA or RNA, the method comprising the steps of:


- - a) extracting the DNA or RNA from a sample of biological material
    containing said genomic DNA or RNA obtained from the bovine,
  - b) genotyping said DNA or RNA for a deletion in the interval between
    nucleotide positions 20537017 to 20540346 on bovine chromosome 21
    (bTau4.0), further comprising genetic markers that are in linkage
    disequilibrium to the brachyspina locus; and
  - c) determining whether said animal carries the brachyspina mutation
    comprising linkage analysis or association analysis.

Equally provided is a method is provided for determining whether a bovine is affected with the brachyspina syndrome or a carrier of brachyspina by analyzing its DNA or RNA, the method comprising the steps of:


- - a. obtaining a sample of material containing said genomic DNA or RNA
    from the bovine,
  - b. extracting the DNA or RNA from said sample,
  - c. genotyping said DNA or RNA for genetic markers that are in
    linkage disequilibrium with the brachyspina locus, and
  - d. ascertaining whether said animal carries the brachyspina mutation
    by association analysis or linkage analysis.

Thus, in a further preferred aspect of the present invention the determining step of the claimed method further comprises linkage analysis or association analysis.

The term “genetic markers that are in linkage disequilibrium with the brachyspina locus” refers to DNA sequence variants such as microsatellite markers or Single Nucleotide Markers (SNPs) that are in linkage disequilibrium with the brachypsina locus in cattle populations. Linkage disequilibrium, also referred to as gametic association or association, refers to the non-random assortment of alleles at distinct genetic loci in the general population. In the present case, these are DNA sequence variants for which one allele is more often associated with the brachyspina mutation in the general population than expected only by chance. In the bovine these include genetic markers, whether microsatellites or SNPs, that are located between nucleotide positions 20 million and 22.5 million on bovine chromosome 21 (bTau4.0).

The term “ascertaining whether said animal carries the brachyspina mutation by association analysis” or “ascertaining whether said animal carries the brachyspina deletion by association analysis” indicates that one will determine whether said animal is carrier of the brachyspina mutation from the analysis of its genotype at DNA sequence variants that are in linkage disequilibrium with the brachypsina mutation. The association analysis can be performed by extracting linkage disequilibrium information from DNA sequence variants considered individually (“single point analyses”), or by considering the DNA sequence variants jointly (“multipoint analyses” including “haplotype-based analyses”). The principles of association studies are known by those skilled in the art and are for instance described in (8).

Furthermore, the possibility to detect animals that are carriers of the brachyspina mutation can be utilized for marker assisted selection to enhance fertility. We have indeed demonstrated that carrier-status for brachyspina is strongly correlated with fertility, one of the most important economic traits in cattle. We demonstrate in this invention that the brachyspina mutation is present in ˜7.5% of Holstein-Friesian animals, which is more common than might be expected from the apparently low incidence of the disease. Thus, brachyspina is a much more important issue in cattle than reflected by the incidence of calves born affected. Detecting brachyspina carriers can thus be used for marker assisted selection to enhance fertility.

As a result of this invention, it is now possible to detect carrier animals for brachyspina by means of simple genetic tests performed on a nucleic acid extracted from biological samples originating from said animals and use the information obtained by the methods of the present invention for marker assisted selection for increased fertility.

The term “marker assisted selection for increased fertility” in accordance with the present invention refers to the use of DNA sequence variant information, corresponding either to the direct detection of the brachyspina mutation, or its indirect detection by means of DNA sequence variants that are either linked or in linkage disequilibrium with the brachyspina locus, to identify animals that are carriers of brachyspina following the procedures described above, and thereby obtain information about their breeding value for phenotypes related to male or female fertility. It is noteworthy that a novel form of marker assisted selection has been recently introduced referred to as “genomic selection” (see for instance reference 9). Information about the presence or absence of the brachyspina mutation can be utilized if the genomic selection procedure were to be applied for traits related to both male and female fertility. Genomic selection for such traits would thus utilize information that is disclosed in the present invention.

Thus, in a further embodiment the present inventions provides for the use of the methods of the present to perform marker assisted selection or genomic selection for increased fertility in said bovine.

In accordance with the present invention a method for increasing fertility in a bovine or bovine population is provided comprising


- a) obtaining a sample of material containing said genomic DNA from the
  bovine,
- b) extracting the DNA from said sample,
- c) genotyping said DNA for a deletion causing brachyspina as described
  herein; and
- d) identifying a bovine that is a carrier for brachyspina.

Another aspect of the present invention focuses on a method for the detection of the above identified brachyspina mutation/deletion comprising amplifying by techniques well established in the art, e.g., and encompassed within the ambit of the present invention, polymerase chain reaction (PCR), the isolated DNA obtained from the bovine with specific primers for said mutation. As a non-limiting example, the nucleotide sequences as set forth in SEQ ID NOs: 5 and 6 can be applied as a as mutant PCR primer pair for the detection of brachyspina. In accordance with the present invention, the nucleotide sequences as provided in SEQ ID NOs: 3 and 4 can be used as a wild-type control PCR primer pair. Preferably, the genotyping step is carried out simultaneously to detect the mutant and the wild-type allele. It is contemplated to design further specific primers or primer pairs for the detection of brachyspina. Thus, primers are within the scope of the present invention which are directed to the specific sequences adjacent to or flanking the above defined mutation. Preferably, primers are included binding specifically to a region within 1 to 500 nucleotides, preferably 1 to 100 nucleotides or even more preferred 1 to 50 nucleotides surrounding the mutation as disclosed herein.

In a further embodiment of the present invention a method for the detection of the disclosed brachyspina mutation is provided comprising amplifying the isolated DNA obtained from the bovine, e.g. by PCR and further utilising specific probes directed to the brachyspina locus as referred to herein. As a non-limiting example, probes like 5′HEX-AGT CCC AGT GTG GCT AAG GAG TGA-3′IABkFQ (wild-type) (SEQ ID NO: 7) and 5′FAM-CCA TTC CAC/ZEN/CTT TCT ATC CGT GTC CT-3′IABkFQ (mutant) (SEQ ID NO: 8) can be used in accordance with the present invention. Again, in a further embodiment of the present invention it is envisaged to design further specific probes directed to nucleotide sequences flanking the above defined mutation. Preferably, probes are included binding specifically to a region within 1 to 1000 nucleotides, preferably 1 to 500 nucleotides, more preferably 1 to 100 nucleotides or even more preferred 1 to 50 nucleotides surrounding the mutation as disclosed herein.

In a further aspect of the present invention the probe is labeled with a fluorophore. Fluorophores are well-known in the art. Preferably, applied in the methods and uses of the present invention are: 6-carboxyfluorescein (FAM), hexachlorofluorescein (HEX), or Fluorescein isothiocyanate (FITC). It is also envisaged by the present invention that the probe or probes applied in methods and uses provided herein further comprise a quencher. Even more preferred is an internal quencher, having a distance between the fluorophore and the quencher of 20 to 30 bases. Most preferred is a ZEN™ quencher which decreases the length to only around 9 bases.

## DETAILED DESCRIPTION OF THE INVENTION

### Autozygosity-Mapping Positions the Brachyspina Locus in a 2.5 Mb BTA21 Interval.

Between January 2008 and December 2009, we obtained biological material from six Holstein-Friesian calves diagnosed with brachyspina. As the previously reported cases (f.i. 4), the six affected animals traced back, on sire and dam side, to Sweet Haven Tradition, a once popular artificial insemination (AI) Holstein-Friesian bull. Genomic DNA was extracted using standard procedures and genotyped using a previously described bovine 50K SNP array (3). Assuming that brachyspina is indeed inherited as a autosomal recessive defect and genetically homogeneous in Holstein-Friesian (as suggested from pedigree analysis), the six cases are predicted to be homozygous for a common haplotype encompassing the causative mutation. We performed autozygosity mapping using the ASSIST program (3) and 15 healthy Holstein-Friesian bulls as controls, and identified a single genome-wide significant peak (p<0.001) on chromosome 21 (BTA21). The shared haplotype spans 2.46 Mb (bTau4.0: 20,132,767-22,588,403) encompassing 56 annotated genes (FIG. 1).

### Targeted and Genome-Wide Resequencing Identifies the Causative 3.3 Kb Brachyspina Deletion in the FANCI Gene.

Several of the 56 genes in the interval are known to cause embryonic lethality when knocked out in the mouse. We amplified the corresponding open reading frames (ORF) from genomic DNA of cases and controls but did not find any obvious disruptive DNA sequence variant (DSV). We then performed targeted sequencing of the entire 2.46 Mb interval. A custom sequence capture array (Roche Nimblegen) was designed based on the bovine bTau4.0 build, and used to enrich the corresponding sequences from total genomic DNA of two affected individuals prior to paired-end sequencing (2×36 bp) on an Illumina GAIIx instrument. Resulting sequence traces were mapped to the bTau4.0 build using Mosaik (http://bioinformatics.bc.edu/marthlab). In the targeted region, the coverage of non-repetitive bases averaged 90.45 (range: 0-336) for the first sample, and 61.28 (range: 0-189) for the second, to be compared with 0.01 (range: 0-24) for the first and 0.01 (range: 0-104) for the second sample outside the targeted region. The proportion of targeted non-repetitive bases with coverage <10 was 0.12 for both samples. We used the GigaBayes software (Gabor T. Marth, Boston College, http://bioinformatics.bc.edu/marthlab) to identify DSV and detected 2,368 SNPs and 572 indels for a total of 2,940 DSVs. One thousand thirty two of these corresponded to DSV previously reported in breeds other than Holstein-Friesian, and were therefore eliminated as candidate causative mutations. Of the remaining 1,908 DSV only one was coding, causing a serine to glycine substitution in the L00516866 gene encoding a myosin light chain kinase-like protein. This DSV was not considered to be a credible candidate mutation underlying brachyspina.

We then generated mate-pair libraries from self-ligated 4.8 Kb (±0.35 Kb) fragments of one brachyspina case and three unrelated, healthy controls and generated <3.7 Gb of sequence on a Illumina GAIIx instrument for each animal. Resulting traces were mapped to the bTau4.0 build using the Burrows-Wheeler Aligner (BWA)(10), and alignments visualized with the Integrative Genomics Viewer (IGV)(11). Analysis of the reads mapping to the 2.46 Mb interval readily revealed a 3.3 Kb deletion removing exons 25-27 of the 37 composing the FANCI (Fanconi anemia complentation-group I) gene. The deletion was apparent from a cluster of 27 mate-pairs mapping ˜8 Kb apart on the bTau4.0 build, and from the complete absence of reads mapping to the deleted segment for the brachyspina case, contrary to the three controls showing normal, uniform coverage in the region. Retrospective analysis of the sequence traces captured from affected individuals confirmed the abrupt coverage drop at the exact same location. We designed a primer pair spanning the presumed deletion, allowing productive amplification of a 409 by product from genomic DNA of affected and carrier animals but not of unrelated healthy controls from the same or other breeds. Sequencing this amplicon defined the deletion breakpoints, confirming the 3,329 bp deletion (FIG. 2A). Retrospective analysis of the sequence traces captured from affected individuals identified several reads bridging and confirming the breakpoint. Conversely, primer pairs designed within the deletion did not yield any amplification from DNA of affected individuals compared to healthy ones.

Assuming that the deletion of exons 25 to 27 results in the juxtaposition of exons 24 and 28 in the mRNA, the 3.3 Kb deletion is predicted to cause a frameshift at amino-acid position 877 substituting the 451 carboxyterminal amino-acids with a 26-residue long illegitimate peptide. Moreover, the ensuing stop codon in exon 28 is expected to cause nonsense mediated RNA decay (FIG. 2B).

With its homologue FANCD2, the FANCI protein forms the ID complex that localizes to damage-induced chromatin foci. FANCI is essential for DNA interstrand crosslink repair. Like FANCD2, FANCI is monoubiquitinated by the ubiquitin ligase FA core complex, and phosphorylated by the ATM/ATR kinase (f.i. 12). Missense, nonsense and splice-site variants in the FANCI gene underlie ˜2% of cases of Fanconi anemia (FA) in human (12,13). FA patients exhibit heterogenous symptoms, including growth retardation, skeletal abnormalities, renal, cardiac, gastrointestinal and reproductive malformations (reminiscent of bovine brachyspina), as well as bone marrow failure, early onset of cancer and mortality at a young age.

Development of a Diagnostic Test Directly Interrogating the 3.3 Kb FANCI Deletion and Confirmation its Causality.

We developed a 5′ exonuclease genotyping assay that simultaneously interrogates the mutant and wild-type allele. The assay uses 5′-TGT TAG CCC AGC AGA GGA-3′ (SEQ ID NO: 3) and 5′-ATT CTG AAT CCA CTA GAT GTC-3′ (SEQ ID NO: 4) as wild-type PCR primer pair combined with 5′-GCA CAC ACC TAT CTT ACG GTA C-3′ (SEQ ID NO: 5) and 5′-GGG AGA AGA ACT GAA CAG ATG G-3′ (SEQ ID NO: 6) as mutant PCR primer pair, and 5′HEX-AGT CCC AGT GTG GCT AAG GAG TGA-3′IABkFQ (wild-type) (SEQ ID NO: 7) and 5′FAM-CCA TTC CAC/ZEN/CTT TCT ATC CGT GTC CT-3′IABkFQ (mutant) (SEQ ID NO: 8) as probes (Integrated DNA Technologies, Leuven, Belgium). Allelic discrimination reactions were carried out on an ABI7900HT instrument (Applied Biosystems, Fosters City, Calif.) for 40 cycles in 2.5 μl volume with a final concentration of 250 nM for each probe, 500 nM for wild-type primers, 350 nM for mutant primers, Taqman Universal PCR Master Mix 1X (Applied Biosystems, Fosters City, Calif.) and 10 ng of genomic DNA. Typical results are illustrated in FIG. 3.

As expected, all available brachyspina cases were shown by these test to be homozygous for the deletion. The deletion proved to be absent in a sample of 131 sires healthy animals representing ten breeds other than Holstein. We then genotyped a random sample of 3,038 healthy Holstein-Friesian animals Carriers of the deletions accounted for 7.4% of the sample, while no animals were found to be homozygous. Assuming Hardy-Weinberg equilibrium, the absence of homozygous animals in a sample of 3,038 individuals has probability <5%. This strongly suggests that homozygosity for the mutation is not compatible with normal health, i.e. that the 3.3 Kb FANCI deletion is causal.

## BRIEF DESCRIPTION OF THE FIGURES

FIG. 1: Schematic representation of the brachyspina locus. Shown are: (A) the results of autozygosity mapping positioning the brachyspina locus on bovine chromosome 21; (B) the genotypes of six brachyspina cases and three healthy controls for 1,269 SNP markers on chromosome 21, showing the 2.46 Mb region of autozygosity in black & white; (C) the gene content of the 2.46 Mb region; (D) the structure of the FANCI gene with indication of the position of the 3.3 Kb brachyspina deletion.

FIG. 2: Schematic representation of the brachyspina mutation. (A) Detailed view of the 3.3 Kb brachyspina deletion deleting exons 25, 26 and 27 of the bovine FANCI gene. The sequences flanking the deletion breakpoints are given. (B) Predicted effect of the 3.3 Kb brachyspina deletion on the structure of the bovine FANCI protein.

FIG. 3: Example of results obtained with the brachyspina 5′ exonuclease test. Each animal is represented by a dot and the three clusters correspond to +/+, +/d, and d/d animals respectively; non template controls (NTC) are visualized as x.

