# DESCRIPTION

## BACKGROUND

Cancer cells exhibit elevated de novo intracellular lipogenesis, resulting in increased levels of fatty acids, membrane phospholipids, and cholesterol (Menendez, J. A. & Lupu, R. Nat Rev Cancer 7:763-777 (2007); Santos, C. R. & Schulze, A. Febs J 279:2610-2623 (2012); Mishra, P. & Ambs, S. Mol Cell Oncol 2 (2015); Migita, T. et al. J Natl Cancer Inst 101:519-532 (2009)). Notably, de novo lipogenesis only contributes minimally to the overall lipid content of normal non-proliferating cells, which generally rely on the uptake of lipids from the circulation. Increased de novo lipogenesis in cancer cells is required to supply lipids for the synthesis of membrane and signaling molecules during rapid cell proliferation and tumor growth, due to limited availability of lipids from the circulation in the tumor microenvironment (Santos, C. R. & Schulze, A. Febs J 279:2610-2623 (2012); Beloribi-Djefaflia, S., et al. Oncogenesis 5:e189 (2016); Lewis, C. A. et al. Oncogene 34:5128-5140 (2015); Baumann, J., et al. Biochim Biophys Acta 1831:1509-1517 (2013); Rios-Esteves, J., et al. Cell reports 4:1072-1081 (2013); Pavlova, N. N. & Thompson, C. B. Cell metabolism 23:27-47 (2016); Boroughs, L. K. & DeBerardinis, R. J. Nat Cell Biol 17:351-359 (2015)). Because of heightened intracellular lipogenesis in cancer, it has been widely reported that drugs that inhibit lipogenesis may have anticancer potential (Mishra, P. & Ambs, S. Mol Cell Oncol 2 (2015); Mudduluru, G., et al. Drug Resist Updat 26:10-27 (2016); Kuzu, O. F., et al. Cancer Research 76:2063-2070 (2016); Kambach, D. M. et al. Oncotarget 8:14860-14875 (2017)). Indeed, numerous completed and active clinical trials have tested statins, which inhibit HMG-CoA reductase and lower blood cholesterol levels, for cancer prevention and treatment (Mudduluru, G., et al. Drug Resist Updat 26:10-27 (2016)). However, it remains controversial as to whether statins provide benefits for cancer prevention or treatment (Mudduluru, G., et al. Drug Resist Updat 26:10-27 (2016); Clendening, J. W. & Penn, L. Z. Oncogene 31:4967-4978 (2012); Ravnskov, U., et al. J Clin Oncol 33:810-811 (2015)). The lack of clear effects of statins on cancer in clinical studies highlights the need to clearly understand the mechanisms that control de novo lipogenesis in cancer cells and to identify biomarkers that can predict treatment responses to statins and the development of potential new agents targeting the lipogenesis pathway for effective cancer therapy.

Lipogenesis is controlled by sterol regulatory element-binding proteins, SREBP1 and SREBP2, that have also been shown to play a critically important role in maintaining lipid synthesis in cancer (Griffiths, B. et al. Cancer Metab 1:3 (2013)). SREBP1/2 precursors are sequestered in endoplasmic reticulum (ER). When sterol supply is low, SREBP1/2 are transported to the Golgi apparatus where they are cleaved by proteases, and the N-terminal domain of SREBPs are then released and imported into the nucleus to activate genes that contain the sterol regulatory element (SRE) required for lipogenesis. Oncogenic drivers such as KRAS, P13K and mTOR signaling have been shown to promote de novo lipogenesis in breast and other cancer types (Ricoult, S. J. et al. Oncogene 35:1250-1260 (2016); Ru, P. et al. Cell reports 16:1527-1535 (2016); Guo, D. et al. Science signaling 2:ra82 (2009)). mTOR signaling apparently promotes lipogenesis through stabilizing SREBP1 and SREBP2 by opposing phosphorylation-dependent poly-ubiquitination of SREBP1/2 by the E3 ubiquitin ligase and tumor suppressor FBXW7 and subsequent proteasomal degradation (Sundqvist, A. et al. Cell metabolism 1:379-391 (2005); Welcker, M., et al. Genes Dev 27:2531-2536 (2013); Li, S., et al. Oncogene 35:642-650 (2016)). Notably, tumors efficiently convert acetate to acetyl-CoA (Comerford, S. A. et al. Cell 159:1591-1602 (2014); Schug, Z. T. et al. Cancer Cell 27:57-71 (2015)), which is predominantly used for lipid synthesis (Bulusu, V. et al. Cell reports 18:647-658 (2017)), highlighting the need for cancer cells to produce lipogenic enzymes (Gao, X., et al. Nature communications 7:11960 (2016)). While the dependence on de novo lipogenesis in cancer is well documented, the mechanisms that directly control SREBP-mediated transcription underlying de novo lipogenesis in cancer cells remain largely unknown.

## SUMMARY

Genetic depletion of DAXX-encoding death domain-associated protein is shown herein to markedly downregulate the expression of key lipogenic regulators and impair de novo lipogenesis. DAXX interacts with sterol regulatory element-binding proteins (SREBP1 and SREBP2) and activates SREBP-mediated transcription and is associated with chromatins containing SREBP-binding cis-acting elements. SREBP2 knockdown markedly attenuates DAXX-mediated effects on lipogenesis, indicating that DAXX directly promotes the SREBP2 lipogenic pathway. Notably, the DAXX/SREBP2 axis also activates MYC expression. In vivo, DAXX depletion attenuates, while DAXX overexpression enhances, the growth of xenograft tumors. Strikingly, a DAXX mutant deficient of small ubiquitin-related modifier (SUMO)-binding fails to activate de novo lipogenesis, and this mutant is completely defective for stimulating tumor growth. Thus, DAXX's SUMO-binding property is critical for cancer lipogenesis program. Remarkably, a peptide derived from the C-terminal SUMO-interacting motif of DAXX (termed SIM2) is spontaneously cell-membrane permeable, inhibits de novo lipogenesis in cells and tumor growth in vivo, highlighting therapeutic potential of SIM2. These results establish DAXX as a key regulator of de novo lipogenesis and a therapeutic target for cancer therapy. Therefore, disclosed herein is a method for treating cancer in a subject that involves administering to the subject a therapeutically effective amount of a composition comprising a polypeptide that comprises an amino acid sequence corresponding to at the C-terminal SUMO-interacting motif (SIM2) of a DAXX protein. The disclosed polypeptide is not a functional DAXX protein but competes with endogenous DAXX for binding to SUMO.

DAXX has two SUMO interacting motifs (SIMs) that interact with SUMO. SIM1 is at the N-terminus, and SIM2 is at the C-terminus (aa 729-740 of human DAXX). Therefore, in some embodiments, the polypeptide comprises at least amino acids 729 to 740 of human DAXX protein, or a homologue or variant thereof that binds SUMO. In some embodiments, the polypeptide lacks amino acids 1 to 728 of the DAXX protein. In some embodiments, the polypeptide comprises or consists of the amino acid sequence DPEEIIVLSDSD (SEQ ID NO:1, SIM2), or a variant thereof, e.g. having one or two conservative amino acid substitutions, that binds SUMO-1. Therefore, in some embodiments, the polypeptide comprises or consists of an amino acid sequence selected from the group comprising DPDDIIVLSDSD (SEQ ID NO:2, SIM008), DPEEIIVLSESE (SEQ ID NO:3, SIM009), DPEEIIVLDDDD (SEQ ID NO:4, SIM010), DPEEKIVLSDSD (SEQ ID NO:5, SIM011), DPEEIIDLSDSD (SEQ ID NO:6, SIM012), EPEEIIVLSDSD (SEQ ID NO:7, SIM013), and IIVLSDS (SEQ ID NO:8, SIM014). In some embodiments, the polypeptide comprises the amino acid sequence X1PX2X3X4X5X6X7X8X9X10X11 (SEQ ID NO:9), wherein X1 is D or E, X2 is E or D, X3 is E or D, X4 is I, V or L, X5 is I, V, or L, X6 is V, I, or L, X7 is L, I, or V, X8 is S, T, E or D, X9 is D or E, X10 is S, T, E or D, and X11 is D or E.

In some embodiments, the polypeptide comprises the amino acid sequence X1X2X3X4X5X6X7X8 (SEQ ID NO:10), wherein X1 is S, T, D or E, X2 is S, T, D, or E, X3 is I, V or L, X4 is I, V, or L, X5 is V, I, or L, X6 is L, I, or V, X7 is S, T, E or D, X8 is S, T, D or E.

In some embodiments, the polypeptide comprises the amino acid sequence XXIIVLXXXX (SEQ ID NO:11), wherein X is D, E, S, or T.

The cancer of the disclosed methods can be any cell in a subject undergoing unregulated growth, invasion, or metastasis. In some aspects, the cancer is breast cancer, prostate cancer, or colon cancer. In some aspects, the cancer can be any neoplasm or tumor for which radiotherapy is currently used. Alternatively, the cancer can be a neoplasm or tumor that is not sufficiently sensitive to radiotherapy using standard methods. Thus, the cancer can be a sarcoma, lymphoma, leukemia, carcinoma, blastoma, or germ cell tumor. A representative but non-limiting list of cancers that the disclosed compositions can be used to treat include lymphoma, B cell lymphoma, T cell lymphoma, mycosis fungoides, Hodgkin's Disease, myeloid leukemia, bladder cancer, brain cancer, nervous system cancer, head and neck cancer, squamous cell carcinoma of head and neck, kidney cancer, lung cancers such as small cell lung cancer and non-small cell lung cancer, neuroblastoma/glioblastoma, ovarian cancer, skin cancer, liver cancer, melanoma, squamous cell carcinomas of the mouth, throat, larynx, and lung, endometrial cancer, cervical cancer, cervical carcinoma, breast cancer, epithelial cancer, renal cancer, genitourinary cancer, pulmonary cancer, esophageal carcinoma, head and neck carcinoma, large bowel cancer, hematopoietic cancers; testicular cancer; colon and rectal cancers, prostatic cancer, and pancreatic cancer.

The disclosed polypeptides can be used in combination with any compound, moiety or group which has a cytotoxic or cytostatic effect. Drug moieties include chemotherapeutic agents, which may function as microtubule inhibitors, mitosis inhibitors, topoisomerase inhibitors, or DNA intercalators, and particularly those which are used for cancer therapy. The disclosed polypeptides can be used in combination with a checkpoint inhibitor. The two known inhibitory checkpoint pathways involve signaling through the cytotoxic T-lymphocyte antigen-4 (CTLA-4) and programmed-death 1 (PD-1) receptors. These proteins are members of the CD28-B7 family of cosignaling molecules that play important roles throughout all stages of T cell function. The PD-1 receptor (also known as CD279) is expressed on the surface of activated T cells. Its ligands, PD-L1 (B7-H1; CD274) and PD-L2 (B7-DC; CD273), are expressed on the surface of APCs such as dendritic cells or macrophages. PD-L1 is the predominant ligand, while PD-L2 has a much more restricted expression pattern. When the ligands bind to PD-1, an inhibitory signal is transmitted into the T cell, which reduces cytokine production and suppresses T-cell proliferation. Checkpoint inhibitors include, but are not limited to antibodies that block PD-1 (Nivolumab (BMS-936558 or MDX1106), CT-011, MK-3475), PD-L1 (MDX-1105 (BMS-936559), MPDL3280A, MSB0010718C), PD-L2 (rHIgM12B7), CTLA-4 (Ipilimumab (M DX-010), Tremelimumab (CP-675,206)), IDO, B7-H3 (MGA271), B7-H4, TIM3, LAG-3 (BMS-986016).

Also disclosed is a composition comprising the disclosed polypeptide in a pharmaceutically acceptable excipient. For example, in some embodiments, the polypeptide is encapsulated in a biocompatible nanoparticle. In particular embodiments, the polypeptide is formulated in a vehicle containing a mixture of 2-Hydroxypropyl)-β-cyclodextrin (HPBCD) water solution and polyethylene glycols (e.g., PEG400).

The details of one or more embodiments of the invention are set forth in the accompanying drawings and the description below. Other features, objects, and advantages of the invention will be apparent from the description and drawings, and from the claims.

## DETAILED DESCRIPTION

As disclosed herein, DAXX is as a key regulator of cancer lipogenesis pathway. DAXX interacts with SREBPs and is recruited to SREBP-binding chromatin sites. While DAXX depletion impairs, DAXX overexpression promotes lipogenesis and tumor growth. DAXX expression is induced in the absence of exogenous lipid supplies; under which DAXX activates MYC expression through SREBP2. Strikingly, a SUMO-binding defective DAXX mutant could not promote lipogenesis and tumor growth, and a DAXX SIM-derived peptide (SIM2) blocks gene expression, de novo lipogenesis and tumor growth. Thus, the DAXX SIM/SUMO interface represents a tractable therapeutic target with the SIM2 peptide. These studies define a previously unknown oncogenic mechanism underlying cancer biology and a therapeutic approach with translational potential for cancer therapy.

Definitions

The term “subject” refers to any individual who is the target of administration or treatment. The subject can be a vertebrate, for example, a mammal. Thus, the subject can be a human or veterinary patient. The term “patient” refers to a subject under the treatment of a clinician, e.g., physician.

The term “therapeutically effective” refers to the amount of the composition used is of sufficient quantity to ameliorate one or more causes or symptoms of a disease or disorder. Such amelioration only requires a reduction or alteration, not necessarily elimination.

The term “pharmaceutically acceptable” refers to those compounds, materials, compositions, and/or dosage forms which are, within the scope of sound medical judgment, suitable for use in contact with the tissues of human beings and animals without excessive toxicity, irritation, allergic response, or other problems or complications commensurate with a reasonable benefit/risk ratio.

The term “carrier” means a compound, composition, substance, or structure that, when in combination with a compound or composition, aids or facilitates preparation, storage, administration, delivery, effectiveness, selectivity, or any other feature of the compound or composition for its intended use or purpose. For example, a carrier can be selected to minimize any degradation of the active ingredient and to minimize any adverse side effects in the subject.

The term “treatment” refers to the medical management of a patient with the intent to cure, ameliorate, stabilize, or prevent a disease, pathological condition, or disorder. This term includes active treatment, that is, treatment directed specifically toward the improvement of a disease, pathological condition, or disorder, and also includes causal treatment, that is, treatment directed toward removal of the cause of the associated disease, pathological condition, or disorder. In addition, this term includes palliative treatment, that is, treatment designed for the relief of symptoms rather than the curing of the disease, pathological condition, or disorder; preventative treatment, that is, treatment directed to minimizing or partially or completely inhibiting the development of the associated disease, pathological condition, or disorder; and supportive treatment, that is, treatment employed to supplement another specific therapy directed toward the improvement of the associated disease, pathological condition, or disorder.

“Polypeptide” as used herein refers to any peptide, oligopeptide, polypeptide, gene product, expression product, or protein. A polypeptide is comprised of consecutive amino acids. The term “polypeptide” encompasses naturally occurring or synthetic molecules.

SIM2 Polypeptide

Disclosed herein is a method for treating cancer in a subject that involves administering to the subject a therapeutically effective amount of a composition comprising a polypeptide that comprises an amino acid sequence corresponding to at the C-terminal SUMO-interacting motif (SIM2) of a DAXX protein. The polypeptide is not a functional DAXX protein. Therefore, in some embodiments, the polypeptide lacks all of part of one or more of the SIM domain, DAXX helical bundle (DHB) domain; histone binding domain (HBD); or PEST domain of the DAXX protein. Therefore, in some embodiments, the polypeptide lacks one or more of amino acids 1-17, 55-144, 180-384, or 500-728. In some embodiments, the polypeptide lacks all of amino acids 1 to 728 of the DAXX protein.

Therefore, in some embodiments, the polypeptide lacks at least 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 25, 30, 35, 40, 45, 50, 60, 70, 80, 90, 100, 110, 120, 130, 140, 5 141, 142, 143, 144, 145, 156, 147, 148, 149, 150, 151, 152, 153, 154, 155, 156, 157, 158, 159, 160, 161, 162, 163, 164, 165, 166, 167, 168, 169, 170, 171, 172, 173, 174, 175, 176, 177, 178, 179, 180, 181, 182, 183, 184, 185, 186, 187, 188, 189, 190, 191, 192, 193, 194, 195, 196, 197, 198, 199, 200, 201, 202, 203, 204, 205, 206, 207, 208, 209, 210, 211, 212, 213, 214, 215, 216, 217, 218, 219, 220, 221, 222, 223, 224, 225, 10 226, 227, 228, 229, 230, 231, 232, 233, 234, 235, 236, 237, 238, 239, 240, 241, 242, 243, 244, 245, 246, 247, 248, 249, 250, 251, 252, 253, 254, 255, 256, 257, 258, 259, 260, 261, 262, 263, 264, 265, 266, 267, 268, 269, 270, 271, 272, 273, 274, 275, 276, 277, 278, 279, 280, 281, 282, 283, 284, 285, 286, 287, 288, 289, 290, 300, 301, 302, 303, 304, 305, 306, 307, 308, 309, 310, 311, 312, 313, 314, 315, 316, 317, 318, 319, 15 320, 321, 322, 323, 324, 325, 326, 327, 328, 329, 330, 331, 332, 333, 334, 335, 336, 337, 338, 339, 340, 341, 342, 343, 344, 345, 346, 347, 348, 349, 350, 351, 352, 353, 354, 355, 356, 357, 358, 359, 360, 361, 362, 363, 364, 365, 366, 367, 368, 369, 370, 371, 372, 373, 374, 375, 376, 377, 378, 379, 380, 381, 382, 383, 384, 385, 386, 387, 388, 389, 390, 391, 392, 393, 394, 395, 396, 397, 398, 399, 400, 401, 402, 403, 404, 20 405, 406, 407, 408, 409, 410, 420, 430, 440, 441, 442, 443, 444, 445, 456, 447, 448, 449, 450, 451, 452, 453, 454, 455, 456, 457, 458, 459, 460, 461, 462, 463, 464, 465, 466, 467, 468, 469, 470, 471, 472, 473, 474, 475, 476, 477, 478, 479, 480, 481, 482, 483, 484, 485, 486, 487, 488, 489, 490, 491, 492, 493, 494, 495, 496, 497, 498, 499, 500, 501, 502, 503, 504, 505, 506, 507, 508, 509, 510, 511, 512, 513, 514, 515, 516, 25 517, 518, 519, 520, 521, 522, 523, 524, 525, 526, 527, 528, 529, 530, 531, 532, 533, 534, 535, 536, 537, 538, 539, 540, 541, 542, 543, 544, 545, 546, 547, 548, 549, 550, 551, 552, 553, 554, 555, 556, 557, 558, 559, 560, 561, 562, 563, 564, 565, 566, 567, 568, 569, 570, 571, 572, 573, 574, 575, 576, 577, 578, 579, 580, 581, 582, 583, 584, 585, 586, 587, 588, 589, 590, 591, 592, 593, 594, 595, 596, 597, 598, 599, 600, 601, 30 602, 603, 604, 605, 606, 607, 608, 609, 610, 611, 612, 613, 614, 615, 616, 617, 618, 619, 620, 621, 622, 623, 624, 525, 626, 527, 628, 629, 630, 631, 632, 633, 634, 635, 636, 637, 638, 639, 640, 641, 642, 643, 644, 645, 646, 647, 648, 649, 660, 661, 662, 663, 664, 665, 666, 667, 668, 669, 660, 661, 662, 663, 664, 665, 666, 667, 668, 669, 670, 671, 672, 673, 674, 675, 676, 677, 678, 679, 680, 681, 682, 683, 684, 685, 686, 687, 688, 689, 690, 691, 692, 693, 694, 695, 696, 697, 698, 699, 700, 701, 702, 703, 704, 705, 706, 707, 708, 709, 710, 711, 712, 713, 714, 715, 716, 717, 718, 719, 720, 721, 722, 723, 724, 725, 726, 727, or 728 consecutive amino acids of amino acids 1 to 728 of the DAXX protein.

In some embodiments, the polypeptide comprises or consists of the amino acid sequence DPEEIIVLSDSD (SEQ ID NO:1, SIM2), or a variant thereof, e.g. having one or two conservative amino acid substitutions, that binds SUMO-1. Therefore, in some embodiments, the polypeptide comprises or consists of an amino acid sequence selected from the group comprising DPDDIIVLSDSD (SEQ ID NO:2, SIM008), DPEEIIVLSESE (SEQ ID NO:3, SIM009), DPEEIIVLDDDD (SEQ ID NO:4, SIM010), DPEEKIVLSDSD (SEQ ID NO:5, SIM011), DPEEIIDLSDSD (SEQ ID NO:6, SIM012), EPEEIIVLSDSD (SEQ ID NO:7, SIM013), and IIVLSDS (SEQ ID NO:8, SIM014). In some embodiments, the polypeptide comprises the amino acid sequence X1PX2X3X4X5X6X7X8X9X10X11 (SEQ ID NO:9), wherein X1 is D or E, X2 is E or D, X3 is E or D, X4 is I, V or L, X5 I, V, or L, X6 is V, I, or L, X7 is L, I, or V, X8 is S, T, E or D, X9 is D or E, X10 is S, T, E or D, AND X11 is D or E.

In some embodiments, the disclosed polypeptide is a peptidomimetic. As used herein, “peptidomimetic” means a mimetic of a peptide which includes some alteration of the normal peptide chemistry. Peptidomimetics typically enhance some property of the 20 original peptide, such as increase stability, increased efficacy, enhanced delivery, increased half life, etc. Methods of making peptidomimetics based upon a known polypeptide sequence is described, for example, in U.S. Pat. Nos. 5,631,280; 5,612,895; and 5,579,250. Use of peptidomimetics can involve the incorporation of a non-amino acid residue with non-amide linkages at a given position. One embodiment of the present invention is a peptidomimetic wherein the compound has a bond, a peptide backbone or an amino acid component replaced with a suitable mimic. Some non-limiting examples of unnatural amino acids which may be suitable amino acid mimics include β-alanine, L-α-amino butyric acid, L-γ-amino butyric acid, L-α-amino isobutyric acid, L-ϵ-amino caproic acid, 7-amino heptanoic acid, L-aspartic acid, L-glutamic acid, N-ϵ-Boc-N-α-CBZ-L-lysine, N-ϵ-Boc-N-α-Fmoc-L-lysine, L-methionine sulfone, L-norleucine, L-norvaline, N-α-Boc-N-δCBZ-L-ornithine, N-δ-Boc-N-α-CBZ-L-ornithine, Boc-p-nitro-L-phenylalanine, Boc-hydroxyproline, and Boc-L-thioproline.

The disclosed composition can be linked to an internalization sequence or a protein transduction domain to effectively enter the cell. Recent studies have identified several cell penetrating peptides, including the TAT transactivation domain of the HIV virus, antennapedia, and transportan that can readily transport molecules and small peptides across the plasma membrane (Schwarze et al., Science. 1999 285(5433):1569-72; Derossi et al. J Biol Chem. 1996 271(30):18188-93; Yuan et al., Cancer Res. 2002 62(15):4186-90). More recently, polyarginine has shown an even greater efficiency of transporting peptides and proteins across the plasma, membrane making it an attractive tool for peptide mediated transport (Fuchs and Raines, Biochemistry. 2004 43(9):2438-44). Nonaarginine has been described as one of the most efficient polyarginine based protein transduction domains, with maximal uptake of significantly greater than TAT or antennapeadia. Peptide mediated cytotoxicity has also been shown to be less with polyarginine-based internalization sequences. R9 mediated membrane transport is facilitated through heparan sulfate proteoglycan binding and endocytic packaging. Once internalized, heparan is degraded by heparanases, releasing R9 which leaks into the cytoplasm (Deshayes et al., Cell Mol Life Sci. 2005 62(16):1839-49). Studies have recently shown that derivatives of polyarginine can deliver a full length p53 protein to oral cancer cells, suppressing their growth and metastasis, defining polyarginine as a potent cell penetrating peptide (Takenobu et al., Mol Cancer Ther. 2002 1(12):1043-9). Thus, the provided polypeptide can comprise a cellular internalization transporter or sequence. The cellular internalization sequence can be any internalization sequence known or newly discovered in the art, or conservative variants thereof. Non-limiting examples of cellular internalization transporters and sequences include Polyarginine (e.g., R9), Antennapedia sequences, TAT, HIV-Tat, Penetratin, Antp-3A (Antp mutant), Buforin II, Transportan, MAP (model amphipathic peptide), K-FGF, Ku70, Prion, pVEC, Pep-1, SynB1, Pep-7, HN-1, BGSC (Bis-Guanidinium-Spermidine-Cholesterol, and BGTC (Bis-Guanidinium-Tren-Cholesterol).

In some embodiments, the polypeptide is a chimeric molecule comprising a “targeting molecule” or “targeting moiety.” A targeting molecule is a molecule such as a ligand or an antibody that specifically binds to its corresponding target, for example a receptor on a cell surface. Thus, for example, where the targeting molecule is a ligand, the chimeric molecule will specifically bind (target) cells and tissues bearing expressing the receptor for that ligand.

Pharmaceutical Compositions

Also disclosed is a composition comprising the disclosed polypeptide in a pharmaceutically acceptable excipient. Pharmaceutical carriers suitable for administration of the compounds provided herein include any such carriers known to those skilled in the art to be suitable for the particular mode of administration. The polypeptides can be formulated into suitable pharmaceutical preparations such as solutions, suspensions, tablets, dispersible tablets, pills, capsules, powders, sustained release formulations or elixirs, for oral administration or in sterile solutions or suspensions for parenteral administration, as well as transdermal patch preparation and dry powder inhalers. In one embodiment, the polypeptides described above are formulated into pharmaceutical compositions using techniques and procedures well known in the art (See, e.g., Ansel, Introduction to Pharmaceutical Dosage Forms, 4th Edition, 1985, 126).

In addition, the compounds may be formulated as the sole pharmaceutically active ingredient in the composition or may be combined with other active ingredients. For example, the compounds may be formulated or combined with known NSAIDs, anti-inflammatory compounds, steroids, and/or antibiotics.

In one embodiment, the compositions are formulated for single dosage administration. To formulate a composition, the polypeptide is dissolved, suspended, dispersed or otherwise mixed in a selected carrier at an effective concentration such that the treated condition is relieved or one or more symptoms are ameliorated.

In some embodiments, the peptide is formulated in a suitable peptide-delivery nanoparticle, such as encapsulated within nanoparticles of poly(lactide-co-glycolide) copolymer, cyclodextrin nanoparticles, or cetyl alcoholipolysorbate.

Peptides may be chemically modified so that oral delivery of the derivative is efficacious. Generally, the chemical modification contemplated is the attachment of at least one moiety to the component molecule itself, where said moiety permits (a) inhibition of proteolysis; and (b) uptake into the blood stream from the stomach or intestine. Also desired is the increase in overall stability of the component or components and increase in circulation time in the body. For example, PEGylation is a preferred chemical modification for pharmaceutical usage. Other moieties that may be used include: propylene glycol, copolymers of ethylene glycol and propylene glycol, carboxymethyl cellulose, dextran, polyvinyl alcohol, polyvinyl pyrrolidone, polyproline, poly-1,3-dioxolane and poly-1,3,6-tioxocane.

To ensure full gastric resistance a coating can be impermeable to at least pH 5.0. Examples of the more common inert ingredients that are used as enteric coatings are cellulose acetate trimellitate (CAT), hydroxypropylmethylcellulose phthalate (HPMCP), HPMCP 50, HPMCP 55, polyvinyl acetate phthalate (PVAP), Eudragit L30D, Aquateric, cellulose acetate phthalate (CAP), Eudragit L, Eudragit S, and Shellac. These coatings may be used as mixed films.

A coating or mixture of coatings can also be used on tablets, which are not intended for protection against the stomach. This can include sugar coatings, or coatings which make the tablet easier to swallow. Capsules may consist of a hard shell (such as gelatin) for delivery of dry therapeutic (i.e. powder), for liquid forms a soft gelatin shell may be used. The shell material of cachets could be thick starch or other edible paper. For pills, lozenges, molded tablets or tablet triturates, moist massing techniques can be used. The peptides could also be given in a film coated tablet and the materials used in this instance are divided into 2 groups. The first are the nonenteric materials and include methyl cellulose, ethyl cellulose, hydroxyethyl cellulose, methylhydroxy-ethyl cellulose, hydroxypropyl cellulose, hydroxypropyl-methyl cellulose, sodium carboxy-methyl cellulose, providone and the polyethylene glycols. The second group consists of the enteric materials that are commonly esters of phthalic acid.

To aid dissolution of peptides into the aqueous environment a surfactant might be added as a wetting agent. Surfactants may include anionic detergents such as sodium lauryl sulfate, dioctyl sodium sulfosuccinate and dioctyl sodium sulfonate. Cationic detergents might be used and could include benzalkonium chloride or benzethomium chloride. The list of potential nonionic detergents that could be included in the formulation as surfactants are lauromacrogol 400, polyoxyl 40 stearate, polyoxyethylene hydrogenated castor oil 10, 50 and 60, glycerol monostearate, polysorbate 20, 40, 60, 65 and 80, sucrose fatty acid ester, methyl cellulose and carboxymethyl cellulose. These surfactants could be present in the formulation of the protein or derivative either alone or as a mixture in different ratios.

In some embodiments, the polypeptide is encapsulated in a biocompatible nanoparticle. In particular embodiments, the polypeptide is formulated in a vehicle containing about 33% 2-Hydroxypropyl)-8-cyclodextrin (HPBCD) in PBS and about 45% polyethylene glycol 400.

Cancer Treatment

The cancer of the disclosed methods can be any cell in a subject undergoing unregulated growth, invasion, or metastasis. In some aspects, the cancer is breast cancer, prostate cancer, or colon cancer. In some aspects, the cancer can be any neoplasm or tumor for which radiotherapy is currently used. Alternatively, the cancer can be a neoplasm or tumor that is not sufficiently sensitive to radiotherapy using standard methods. Thus, the cancer can be a sarcoma, lymphoma, leukemia, carcinoma, blastoma, or germ cell tumor. A representative but non-limiting list of cancers that the disclosed compositions can be used to treat include lymphoma, B cell lymphoma, T cell lymphoma, mycosis fungoides, Hodgkin's Disease, myeloid leukemia, bladder cancer, brain cancer, nervous system cancer, head and neck cancer, squamous cell carcinoma of head and neck, kidney cancer, lung cancers such as small cell lung cancer and non-small cell lung cancer, neuroblastoma/glioblastoma, ovarian cancer, pancreatic cancer, prostate cancer, skin cancer, liver cancer, melanoma, squamous cell carcinomas of the mouth, throat, larynx, and lung, endometrial cancer, cervical cancer, cervical carcinoma, breast cancer, epithelial cancer, renal cancer, genitourinary cancer, pulmonary cancer, esophageal carcinoma, head and neck carcinoma, large bowel cancer, hematopoietic cancers; testicular cancer; colon and rectal cancers, prostatic cancer, and pancreatic cancer.

The disclosed polypeptides can be used in combination with any compound, moiety or group which has a cytotoxic or cytostatic effect. Drug moieties include chemotherapeutic agents, which may function as microtubulin inhibitors, mitosis inhibitors, topoisomerase inhibitors, or DNA intercalators, and particularly those which are used for cancer therapy.

The disclosed polypeptides can be used in combination with a checkpoint inhibitor. The two known inhibitory checkpoint pathways involve signaling through the cytotoxic T-lymphocyte antigen-4 (CTLA-4) and programmed-death 1 (PD-1) receptors. These proteins are members of the CD28-B7 family of cosignaling molecules that play important roles throughout all stages of T cell function. The PD-1 receptor (also known as CD279) is expressed on the surface of activated T cells. Its ligands, PD-L1 (B7-H1; CD274) and PD-L2 (B7-DC; CD273), are expressed on the surface of APCs such as dendritic cells or macrophages. PD-L1 is the predominant ligand, while PD-L2 has a much more restricted expression pattern. When the ligands bind to PD-1, an inhibitory signal is transmitted into the T cell, which reduces cytokine production and suppresses T-cell proliferation. Checkpoint inhibitors include, but are not limited to antibodies that block PD-1 (Nivolumab (BMS-936558 or MDX1106), CT-011, MK-3475), PD-L1 (MDX-1105 (BMS-936559), MPDL3280A, MSB0010718C), PD-L2 (rHIgM12B7), CTLA-4 (Ipilimumab (MDX-010), Tremelimumab (CP-675,206)), IDO, B7-H3 (MGA271), B7-H4, TIM3, LAG-3 (BMS-986016).

The herein disclosed compositions, including pharmaceutical composition, may be administered in a number of ways depending on whether local or systemic treatment is desired, and on the area to be treated. For example, the disclosed compositions can be administered intravenously, intraperitoneally, intramuscularly, subcutaneously, intracavity, or transdermally. The compositions may be administered orally, parenterally (e.g., intravenously), by intramuscular injection, by intraperitoneal injection, transdermally, extracorporeally, ophthalmically, vaginally, rectally, intranasally, topically or the like, including topical intranasal administration or administration by inhalant.

In one embodiment, the disclosed polypeptide compositions are administered in a dose equivalent to parenteral administration of about 0.1 ng to about 100 g per kg of body weight, about 10 ng to about 50 g per kg of body weight, about 100 ng to about 1 g per kg of body weight, from about 1 μg to about 100 mg per kg of body weight, from about 1 μg to about 50 mg per kg of body weight, from about 1 mg to about 500 mg per kg of body weight; and from about 1 mg to about 50 mg per kg of body weight. Alternatively, the amount of polypeptide administered to achieve a therapeutic effective dose is about 0.1 ng, 1 ng, 10 ng, 100 ng, 1 μg, 10 μg, 100 μg, 1 mg, 2 mg, 3 mg, 4 mg, 5 mg, 6 mg, 7 mg, 8 mg, 9 mg, 10 mg, 11 mg, 12 mg, 13 mg, 14 mg, 15 mg, 16 mg, 17 mg, 18 mg, 19 mg, 20 mg, 30 mg, 40 mg, 50 mg, 60 mg, 70 mg, 80 mg, 90 mg, 100 mg, 500 mg per kg of body weight or greater.

Although the polypeptide compositions may be administered once or several times a day, and the duration of the treatment may be once per day for a period of about 1, 2, 3, 4, 5, 6, 7 days or more, it can be more preferably to administer either a single dose in the form of an individual dosage unit or several smaller dosage units or by multiple administration of subdivided dosages at certain intervals. For instance, a dosage unit can be administered from about 0 hours to about 1 hr, about 1 hr to about 24 hr, about 1 to about 72 hours, about 1 to about 120 hours, or about 24 hours to at least about 120 hours. Alternatively, the dosage unit can be administered from about 0.5, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 30, 40, 48, 72, 96, 120 hours. Subsequent dosage units can be administered any time following the initial administration such that a therapeutic effect is achieved.

A number of embodiments of the invention have been described. Nevertheless, it will be understood that various modifications may be made without departing from the spirit and scope of the invention. Accordingly, other embodiments are within the scope of the following claims.

## EXAMPLES

### Example 1

**DAXX Drives De Novo Lipogenesis and Represents a Novel Therapeutic Target for Cancer Therapy**

Introduction

Cancer cells exhibit elevated de novo intracellular lipogenesis, resulting in increased levels of fatty acids, membrane phospholipids, and cholesterol (Rohrig F, et al. Nat Rev Cancer 2016 16(11):732-49). Obesity and hypercholesterolemia are known cancer risk factors (Boyd N F, et al. J Natl Cancer Inst 1990 82(6):460-8). Notably, de novo lipogenesis contributes minimally to the overall lipid content of normal non-proliferating cells, which generally rely on the uptake of lipids from the circulation. Increased de novo lipogenesis in cancer cells is thought to supply lipids for the synthesis of membranes and signaling molecules during rapid cell proliferation and tumor growth, due to limited availability of lipids from the circulation in the tumor microenvironment ((Rohrig F, et al. Nat Rev Cancer 2016 16(11):732-49; Lewis C A, et al. Oncogene 2015 34(40):5128-40). Thus, targeting de novo lipogenesis may be an effective strategy for cancer therapy. Indeed, epidemiological studies have shown that the use of statins, which inhibit 3-hydroxy-3-methylglutaryl-CoA reductase (HMGCR)—the rate-limiting enzyme in cholesterol synthesis via the mevalonate pathway—appears to provide survival benefits for some breast cancer (BC) patients (Shaitelman S F, et al. J Cancer 2017 8(11):2026-32). Statins have antiproliferative and apoptotic effects in BC cells in tissue cultures and tumor specimens obtained from patients treated with statins (Freed-Pastor W A, et al. Cell 2012 148(1-2):244-58; Ricoult S J, et al. Oncogene 2016 35(10):1250-60; Pandyra A A, et al. Oncotarget 2015 6(29):26909-21; Wang T, et al. Oncotarget 2016 7(3):2532-44), suggesting that this classic metabolic pathway is intrinsically active within tumor cells for cell survival, proliferation and tumor growth (Mullen P J, et al. Nat Rev Cancer. 2016 16(11):718-731). Significantly, heightened cholesterol production is linked to treatment resistance and poor prognosis (Simigdala N, et al. Breast Cancer Res 2016 18(1):58). Certain cholesterol metabolites such as 25-and 27-hydroxycholesterol (25-HC and 27-HC) are partial ligands for estrogen receptor-a (ER) and have been shown to promote ER-mediated transcription, enhance tumor growth and confer treatment resistance (Simigdala N, et al. Breast Cancer Res 2016 18(1):58; Nelson ER, et al. Science 2013 342(6162):1094-8). Recent works also show that 27-HC appears to promote BC metastatic progression (Baek A E, et al. Nat Commun 2017 8(1):864).

Lipogenesis is controlled by several transcription factors, such as the sterol regulatory element-binding proteins, SREBP1 and SREBP2 (collectively referred to as SREBP1/2), that have also been shown to play an important role in maintaining lipid synthesis in cancer (Griffiths B, et al. Cancer Metab 2013 1(1):3). SREBP1/2 precursors are sequestered in endoplasmic reticulum. When sterol supply is low, SREBP1/2 are transported to the Golgi apparatus where they are cleaved by proteases, and the N-terminal domain of SREBPs are then released and imported into the nucleus to promote transcription of genes that contain the sterol regulatory elements (SREs) required for lipogenesis. Oncogenic drivers such as KRAS, PI3K and mTOR have been shown to promote de novo lipogenesis in breast and other cancers (Rohrig F, et al. Nat Rev Cancer 2016 16(11):732-49; Ricoult S J, et al. Oncogene 2016 35(10):1250-60; Guo D, et al. Sci Signal 2009 2(101):ra82). For example, mTOR promotes lipogenesis through stabilizing SREBP1/2 by opposing phosphorylation-dependent poly-ubiquitination by the E3 ubiquitin ligase FBXW7 and subsequent proteasomal degradation (Sundqvist A, et al. Cell Metab 2005 1(6):379-91). Notably, tumors efficiently convert acetate to acetyl-CoA (Comerford SA, et al. Cell 2014 159(7):1591-602), which is predominantly used for lipid synthesis (Bulusu V, et al. Cell Rep 2017 18(3):647-58), highlighting the need for cancer cells to produce lipogenic enzymes (Gao X, et al. Nat Commun 2016 7:11960). While the dependence on de novo lipogenesis in cancer is well documented, the mechanisms that control SREBP-mediated transcription underlying oncogenic de novo lipogenesis remain poorly understood.

DAXX, originally discovered as a context-dependent regulator of cell death or survival (Yang X L, et al. Cell 1997 89(7):1067-76; Michaelson J S, et al. Genes Dev 1999 13(15):1918-23), has an extensively documented role in transcriptional regulation through interacting with transcription factors including p53 (Zhao L Y, et al. J Biol Chem 2004 279(48):50566-79) and NF-KB (Puto L A, et al. Genes Dev 2008 22(8):998-1010). DAXX is a small ubiquitin-related modifier (SUMO)-binding protein via two SUMO-interacting motifs (SIMs) and that the SUMO-binding property of DAXX appears critical for it to regulate transcription (Santiago A, et al. Cell Cycle 2009 8(1):76-87; Chang C C, et al. Mol Cell 2011 42(1):62-74; Lin D Y, et al. Mol Cell 2006 24(3):341-54). More recent studies have defined DAXX as a specific chaperone for the histone variant H3.3 (Lewis P W, et al. Proc Natl Acad Sci USA 2010 107(32):14075-80; Goldberg A D, et al. Cell 2010 140(5):678-91; Drane P, et al. Genes Dev 2010 24(12):1253-65). DAXX binds specifically to the H3.3/H4 dimer and deposits it onto chromatin (Elsasser S J, et al. Nature 2012 491(7425):560-5; Liu C P, et al. Nat Struct Mol Biol 2012 19(12):1287-92). Emerging evidence suggests that DAXX has an oncogenic role in diverse cancer types (Pan W W, et al. J Biol Chem 2013 288(19):13620-30; Puto L A, et al. J Biol Chem 2015 290(25):15406-20), which appears to be linked to its functions in gene regulation (Puto L A, et al. J Biol Chem 2015 290(25):15406-20; Benitez J A, et al. Nat Commun 2017 8:15223). Whereas the levels of DAXX expression directly correlate with its ability to promote tumor growth (Pan W W, et al. J Biol Chem 2013 288(19):13620-30; Puto L A, et al. J Biol Chem 2015 290(25):15406-20; Benitez J A, et al. Nat Commun 2017 8:15223), the molecular mechanisms underlying DAXX's oncogenic function remain to be defined. Of critical importance is to understand whether DAXX is a tractable therapeutic target for cancer therapy.

In this Example, DAXX was identified as a key regulator of cancer lipogenesis pathway, through interacting with SREBPs, thereby activating lipogenic gene expression programs mediated by SREBPs and promoting cancer cell proliferation in vitro and tumor growth in vivo. Strikingly, a SUMO-binding defective DAXX mutant could not stimulate lipogenesis and tumor growth, and a DAXX SIM-derived peptide (SIM2) blocks DAXX-SREBP interactions, lipogenic gene expression, de novo lipogenesis and tumor growth. Thus, the DAXX SIM/SUMO interface represents a tractable therapeutic target with the SIM2 peptide. Disclosed herein is a previously unknown oncogenic mechanism underlying cancer biology and a novel therapeutic approach with translational potential for cancer therapy.

Results

DAXX Expression Levels Dictate Lipogenic Gene Expression and De Novo Lipogenesis

Bioinformatic analyses of clinical BC samples revealed that DAXX mRNA levels are elevated in all four major BC subtypes and correlate with poor prognosis (FIG. 8). Consistently, the levels of cholesterols and other lipids are higher in BC samples compared to normal controls (FIG. 8D). To understand potential oncogenic function of DAXX, DAXX was genetically depleted and overexpressed in the BC cell lines MDA-MB-231 and MDA-MB-468 (FIG. 1A). Microarray-based RNA profiling and Ingenuity Pathway Analysis (IPA) of differentially expressed genes between cells with DAXX depletion and control cells revealed a marked downregulation of the de novo lipogenesis pathway (FIG. 1B-1E). Lipogenesis regulators (SREBF1/2 and SOAP) were among the most highly inhibited “upstream regulators.” Likewise, the cholesterol, mevalonate and other metabolic pathways related to lipogenesis were also markedly downregulated (FIG. 1B and 1C). Effects of DAXX overexpression on lipogenesis and observed marked upregulation of lipogenic genes was then assessed (FIG. 1B, 1D and 1E). Gene Set Enrichment Analysis (GSEA) of mRNA microarray data confirmed suppression and activation of the de novo lipogenesis pathway by genetic DAXX knockdown (KD) and wt cDNA overexpression (wt OE), respectively (FIG. 1D). The impact of DAXX knockdown or overexpression on lipogenic gene expression was further validated by immunofluorescence microscopy (FIG. 9A), immunoblotting (FIG. 9B), and qRT-PCR (FIG. 9D). Using a tetracycline-inducible gene expression system, it was found that DAXX induction increased lipogenic gene expression (FIG. 9C), suggesting that DAXX directly activates lipogenic gene expression.

Two SIMs in DAXX have been identified and both SIMs are critical for DAXX to activate transcription (Santiago A, et al. Cell Cycle 2009 8(1):76-87; Chang C C, et al. Mol Cell 2011 42(1):62-74; Lin D Y, et al. Mol Cell 2006 24(3):341-54). Remarkably, the overexpression of the DAXX mutant with the 17K/1733K double mutation that prevents DAXX from binding to SUMO (DAXX double-SIM mutant or DSM) (Santiago A, et al. Cell Cycle 2009 8(1):76-87) was impaired to activate the lipogenic genes (FIG. 1E). An analysis of gene expression data of MDA-MB-231 wt DAXX and DSM OE cells using IPA revealed that the DSM mutant could not promote lipogenesis and activate SREBPs (FIG. 1F; FIG. 9E).

De novo lipogenesis assays using [14C]-acetate metabolic labeling confirmed that DAXX expression levels correlated with levels of intracellular lipid synthesis, with reduced or increased de novo lipid synthesis in DAXX KD or wt OE cells, respectively (FIG. 1G). However, the DSM mutant (DSM OE) failed to enhance de novo lipogenesis (FIG. 1G). Consistently, mass spectrometry (MS)-based lipidomic profiling revealed that DAXX KD reduced but wt OE increased levels of specific lipid molecules, respectively, and again, the DSM mutant could not increase the levels of these lipid molecules (FIG. 1H and 1I; FIG. 10A and 10B). DAXX knockdown through CRISPR/Cas9 also impaired lipid production in MDA-MB-231 cells, providing an independent validation (FIG. 10C). Diminished de novo lipogenesis upon DAXX depletion was also observed in BC cell lines of luminal subtypes (MCF7 and T47D) and the colon cancer cell line HCT116 (FIG. 10D). Furthermore, reduced expression of lipogenic genes was also observed in the prostate cancer cell line PC3 with DAXX knockdown (Puto L A, et al. J Biol Chem 2015 290(25):15406-20), and mouse embryonic stem cells with Daxx knockout (Hoelper D, et al. Nat Commun 2017 8(1):1193) (FIG. 11A-1C). These data together indicate that DAXX is a general activator of lipogenesis.

DAXX Interacts with SREBP1 and SREBP2

SREBP112 promote lipid production when the intracellular levels of lipid/sterols are low (Griffiths B, et al. Cancer Metab 2013 1(1):3). Because DAXX depletion markedly downregulated the SREBP/Iipid biosynthesis pathway (FIG. 1), it was reasoned that DAXX could regulate lipid biosynthesis through interacting with SREBPs. Immunoprecipitation (IP) of total cell extracts with anti-DAXX antibody evidenced co-precipitation of two SREBP2 proteins (FIG. 2A) that correspond to the precursor (P) and the mature (M) forms of SREBP2. Note that the apparent molecular weight of the co-precipitated putative precursor appears larger than the predicted full-length SREBP2 (˜125 kDa) in multiple cell lines under various experimental conditions, likely due to differential posttranslational modifications (FIGS. 12 and 13).

In MDA-MB-231 cells, DAXX interacted with both precursor and mature forms of SREBP1 (FIG. 13A). Notably, whereas the mature SREBP1 was predominantly detected in the nucleus, the precursor was seen in both nuclear and cytoplasmic fractions (FIG. 13A; similar patterns were seen for SREBP2, FIG. 12B). However, the mature SREBP1 was clearly enriched in the DAXX immunoprecipitates of both fractions, indicating that DAXX has a high affinity to the mature SREBP1 (FIG. 13A). Consistently, the interactions between DAXX and the precursor (full-length or FL) as well as the mature form of SREBP1/2 (both endogenous and ectopically expressed) were observed in 293T cells (FIG. 13B). Notably, the endogenous SREBP1 (precursor and mature) were highly enriched in the DAXX immunoprecipitates, as they were undetectable in the total 293T cell lysates (Input, in which the transfected SREBP1 constructs were clearly visible), but readily detected in the IP samples (FIG. 13B). Collectively, these data establish that DAXX binds to both precursor and mature SREBP1/2.

Using various DAXX deletion constructs in transfected 293T cells, it was found that the mature SREBP2 interacted with two separate regions of DAXX, the N-terminal part encompassing the well-folded helical bundle domain termed DHB (DAXX helical bundle) (Escobar-Cabrera E, et al. Structure 2010 18(12):1642-53) and a part of the central histone-binding domain (HBD) (Elsasser S J, et al. Nature 2012 491(7425):560-5) (FIG. 2). Interestingly, although DHB and HBD individually bound robustly to SREBP2 (FIG. 2B lanes 11 and 12; FIG. 2C lanes 1 and 3), the integrity of both binding sites appeared critical when both were present in a DAXX construct. Indeed, mutations within DHB or HBD (FIG. 2B lanes 13-15) attenuated the DAXX-SREBP2 interaction. Interestingly, the DAXX construct (aa 1-437) lacking the sequence from the acidic domain to the C-terminus seemed to show higher affinity to SERBP2 (FIG. 2B lane 16).

The C-terminal region of DAXX spanning aa 574-740 did not bind to SREBP2 (FIG. 2D lane 3; note that the co-IP of endogenous DAXX with FLAG-SREBP2 was detectable). The deletion of the amino acid (aa) sequence between the DHB and HBD (aa 128-187), the N-terminal part of HBD (aa 191-241), the acidic domain (aa 434-496), and the Pro/Glu/Ser/Thr-rich (PEST) sequence immediately after the acidic domain (aa 503-572) did not affect the SREBP2-DAXX binding (FIG. 13C). Point mutations in the DHB (Y124F/Y126F) and C-terminal region (S561 and K630A/631A) also did not influence the DAXX-SREBP2 interaction (FIG. 2D). The mature SREBP1a bound to DAXX in a similar fashion (FIG. 13D and 13E). Thus, SREBP1/2 interact with two independent sites in DAXX (FIG. 2E).

SREBP-Binding Sites are Enriched in DAXX-Associated Chromatins

DAXX knockdown reduced the expression of lipogenic genes (FIG. 1) and DAXX physically interacted with SREBP112 (FIG. 2 and FIG. 13). It was reasoned that DAXX promotes SREBP-mediated transcription on chromatins. To test this, luciferase reporter assays were first conducted. As shown in FIG. 3A, enforced expression of SREBP2, SREBP1a and SREBP1c increased the activity of luciferase reporter that was under the control of the SREBF2 promoter containing sterol-responsive elements (SREs). Co-expression of DAXX further increased the luciferase activity, while DAXX alone had only minimal effects. In contrast, the DAXX DSM mutant was largely unable to coactivate SREBP1/2-mediated transcription (FIG. 3A).

Genome-wide occupancy of DAXX was surveyed using the ChIP-seq technology. Overexpression of wt DAXX but not the DSM mutant increased DAXX's chromatin association (FIG. 3B and 3C). Consistent with other studies (Puto L A, et al. Oncoscience 2015 2(4):362-72), DAXX primarily bound to sites in introns and intergenic regions with less frequent association with promoters (FIG. 3D). A de novo motif analysis revealed that SREBP-binding elements were significantly enriched in DAXX-associated sites (FIG. 3E).

Notably, the DAXX DSM mutant exhibits reduced overall chromatin association (FIG. 3B and C) as well as diminished occupancy at individual lipogenic genes compared to wt DAXX (FIG. 3F and FIG. 14). De novo motif analysis indicates that the SREBP1 motif was less significantly enriched, while the enrichment of the SREBP2 motif was insignificant (FIG. 14A) in DAXX-precipitated chromatins from MDA-MB-231 DSM OE cells. The reduced chromatin-recruitment of the DSM mutant was not due to its reduced affinity to the antibody used for ChIP, as this antibody was found to bind the DSM mutant as efficiently as the wt DAXX. These ChIP-seq data along with the luciferase reporter assays show that the SUMO-binding activity of DAXX may be critical for efficient recruitment of DAXX to chromatins, which provides a probable molecular explanation for the inability of the DSM mutant to activate lipogenic gene expression and de novo lipogenesis (FIG. 1 and FIG. 10).

Notably, the binding motifs of other known DAXX-binding transcription factors such as NF-KB (Puto L A, et al. Genes Dev 2008 22(8):998-1010) were highly enriched in DAXX ChIP-seq peaks (FIG. 3E). These data also implicate the chromatin recruitment of DAXX by other transcription factors such as RUNX1, RUNX2 and USF1 that were not previously shown to interact with DAXX (FIGS. 3E and 14). Furthermore, DAXX might interact with the core-transcriptional machinery, as the TATA-box and DCE (downstream core element) were enriched in DAXX-binding chromatins (FIG. 3E and FIG. 14).

DAXX is Critical for Tumor Growth In Vivo

To assess roles of DAXX in cell proliferation and tumor growth, the effects of DAXX expression levels on the proliferation of MDA-MB-231 cells was first tested. DAXX expression levels did not exert significant effects on cell proliferation under conventional two-dimensional cell culture condition, although DAXX knockdown moderately slowed the proliferation of MDA-MB-231 cells (FIG. 15A). The SREBP/lipogenesis pathway has been linked to the control of cell size/volume (Ricoult S J, et al. Oncogene 2016 35(10):1250-60; Porstmann T, et al. Cell Metab 2008 8(3):224-36). Indeed, the cell morphology and size were markedly different for cells with variable levels of DAXX expression (FIG. 15B). Cells with reduced DAXX levels appeared smaller with spindle-like morphology, while cells with wt DAXX OE were larger when compared to control cells (FIG. 15B and FIG. 9A). In contrast, the DAXX DSM mutant OE did not appear to affect cell size or morphology (FIG. 15B). The impact of DAXX expression levels on cell proliferation was then assessed in three-dimensional cell culture models. DAXX knockdown reduced the number and size of colonies when compared to control, while wt DAXX OE had the opposite effects. Again, the DSM mutant had no effect on either colony size or number (FIG. 15C). Similar phenotypes were observed for a panel of HCT116 cells with different levels of DAXX expression (FIG. 15D-H).

Effects of DAXX expression levels on tumor growth was examined in vivo. In orthotopic BC xenograft models using female mice, DAXX knockdown reduced while wt DAXX OE significantly increased xenograft tumor growth rate of both MDA-MB-231 and MDA-MB-468, two triple-negative BC (TNBC) cell lines (FIG. 4A and 4B). Notably, the MDA-MB-231 xenograft tumors grew much more aggressively than MDA-MB-468 tumors (FIG. 4A and 4B), but the effects of DAXX expression levels on tumor growth were observed regardless of tumor growth kinetics. The DSM mutant OE did not significant affect tumor growth in both MDA-MB-231 and MDA-MB-468 xenograft models. Immunoblotting analysis of tumor extracts shows that the levels for both the wt DAXX and DSM mutant were similar (FIG. 4C), indicating that the inability of the DSM mutant to promote tumor growth was not due to instability of the mutant protein. Lipidome in xenograft tumors derived from cells with different levels of DAXX expression was profiled, demonstrating that DAXX expression levels positively correlated with levels of lipids (FIG. 4D).

In a prostate cancer xenograft model (male mice) and a colon cancer xenograft model (both female and male mice), similar tumor growth phenotypes were observed: DAXX depletion slowed while wt DAXX OE accelerated the tumor growth. Consistently, DSM mutant OE was either completely unable to promote tumor growth, or moderately hindered tumor growth in all xenograft models (FIG. 4 and FIG. 16). In mouse allografts, Daxx depletion also markedly impaired tumor growth in immunocompetent (C57BLJ6) and immunodeficient (NSG) hosts (FIG. 16C and 16D). These results demonstrate that DAXX exerts a strong oncogenic property and that the SUMO-binding activity is critical for DAXX's oncogenic function.

The DA SREBP2 Axis Underpins Lipogenesis and Tumor Growth

SREBP1/2 drive lipid biosynthesis to promote tumorigenesis (Griffiths B, et al. Cancer Metab 2013 1(1):3; Moon S-H, et al. Cell. 2019 176(3):564-580.e19). In MDA-MB-231 cells, SREBP2 knockdown reduced de novo lipogenesis from acetate and tumor growth in vivo. By contrast, SREBP2 overexpression increased lipogenesis and tumor growth (FIG. 5A and 5B). Likewise, SREBP1 knockdown (or overexpression) impaired (or promoted) de novo lipogenesis and tumor growth respectively. To link SREBP2 to DAXX-mediated tumorigenesis, SREBP2 was depleted in MDA-MB-231 DAXX OE cells. SREBP2 knockdown in the DAXX OE cells significantly attenuated the levels of lipogenic enzymes and tumor growth (FIG. 5C), indicating that SREBP2 is a critical effector of DAXX's oncogenic function.

DAXX is a robustly phosphorylated protein on a number of frequently phosphorylated serine and threonine residues (Ecsedy J A, et al. Mol Cell Biol 2003 23(3):950-60) (FIG. 17). The interaction of DAXX isoforms containing mutations at frequently phosphorylated residues was tested with SREBP2. While wt DAXX and the S668A mutant coprecipitated with SREBP2, the S671A, S668A/S671A, S495A and S690A DAXX mutants failed to coprecipitate (FIG. 6A). Of note, the DAXX S668A mutant migrated faster than wt DAXX and other mutants in the SDS-PAGE gel in 293T and MDA-MB-231 cells (FIG. 6A and FIG. 17B). Both DAXX S671A and S495A mutants were capable of binding ATRX (FIG. 17C), which interacts directly with the DAXX DHB domain (Hoelper D, et al. Nat Commun 2017 8(1):1193). In luciferase reporter assays, both DAXX S495A and S671A mutants failed to enhance SREBP2-mediated transcription (FIG. 6B). Experiments using ChIP followed by qPCR indicate that wt DAXX bound to the promoters of FASN, ACACA, and SREBF2, but the S495A, S671A and DSM mutants were not recruited to these promoters in MDA-MB-231 cells cultured in serum-free medium (FIG. 6C). Neither wt DAXX nor the DSM mutant were enriched in the 3′ UTR region of FASN or ACACA gene (FIG. 6C), indicating that DAXX is recruited specifically to the promoter regions of lipogenic genes. qRT-PCR assays show that lipogenic genes were downregulated in MDA-MB-231 cells expressing the DAXX S495A or S671A mutant (FIG. 6D). Immunoblotting analysis confirmed the reduced expression of ACC1 and FAS, and to a lesser extent ACSS2 (FIG. 6E). Because of the documented role for mTOR signaling in lipogenesis (Ricoult S J, et al. Oncogene 2016 35(10):1250-60), effects of the mTOR kinase inhibitors AZD8055 (inhibits both mTORC1 and mTORC2) and temsirolimus (inhibits mTORC1) were tested. Neither of these inhibitors had obvious effects on the protein levels of ACC1, FAS and ACSS2 in the control or S671A-expressing cells, although temsirolimus reduced levels of both ACC1 and FAS in cells expressing the S495A mutant (FIG. 6E lane 12). Together, these results suggest that DAXX is recruited to the promoters of lipogenic genes through interacting with SREBP2, thereby enhancing lipogenic gene expression.

Unexpectedly, the DAXX S671A mutant protein failed to accumulate in transduced MDA-MB-231 cells, although the S495A mutant protein expressed at a high level (FIG. 6E). Although why the S671A mutant is less stable remains to be determined, its inability to bind to H3.3 might provide an explanation (FIG. 17C), as the interaction of DAXX with the H3.3/H4 dimer appears to stabilize DAXX (Hoelper D, et al. Nat Commun 2017 8(1):1193). Regardless, these data indicate that the loss-of-function phenotypes of the DAXX S671A and S495A mutations were unrelated to the steady-state protein levels detected in immunoblotting.

Remarkably, the S671A mutant was profoundly defective to stimulate tumor growth in vivo (FIG. 6F). The S495A mutant was also impaired to promote tumor growth compared to wt DAXX (FIG. 6F). Interestingly, the ΔCT mutant (lacking aa 737-740) and the Y222P mutant were as potent as the wt DAXX to enhance tumor growth (FIG. 6F). The Y222P mutant is expected to interrupt the interactions of DAXX with H3.3 (Elsasser S J, et al. Nature 2012 491(7425):560-5) (FIG. 17D). Indeed, the Y222P mutant did not bind H3.3 (FIG. 7C). Together, these data suggest that DAXX's oncogenic property is distinct from its histone chaperone function and its interaction with ATRX.

Lipidomic profiling of the xenograft tumors revealed that lipid production was impaired in tumors expressing the DAXX S671A or S495A mutant (FIG. 6G). Gene expression profiling of MDA-MB-231 cells expressing the DAXX S671A or the S495A mutant revealed that the lipogenesis pathways, especially the cholesterol biosynthesis pathway, were markedly suppressed. Significantly, repressed glycolysis was also observed in cells expressing the DAXX S671A mutant. Among the repressed transcriptional activators, SREBP1/2 and MYC ranked among the top inhibited upstream regulators (FIG. 6H). Interestingly, transcription factors that promote cell cycle progression such as E2Fs and FOXM1 also emerged as highly repressed upstream regulators. Conversely, upstream regulators that suppress lipogenesis (INSIG1 and INSIG2) and cell cycling (e.g., TP53, CDKN2A, and RB) were activated in the IPA analysis. Collectively, these results establish a critical role for the DAXX-SREBP interaction in driving lipogenesis and tumor growth.

The S1M2 Peptide Blocks De Novo Lipogenesis and Inhibits Tumor Growth

The data presented above indicate the DAXX's SUMO-binding property is critical to de novo lipogenesis and in vivo tumor growth. It was hypothesized that the interface between DAXX SIMs and SUMOs can be targeted for cancer therapy. To test this hypothesis, a peptide corresponding to the C-terminal 12 amino acids of DAXX (amino acids 729-740) was synthesized. This synthetic peptide encompasses the C-terminal SUMO-interacting motif of DAXX (termed SIM2). Remarkably, SIM2 was spontaneously and rapidly internalized into cells (FIG. 18A). Thus, SIM2 is a naturally cell membrane permeable peptide that does not require specific modification such as hydrocarbon staple or attaching to a cell-penetrating sequence (Fosgerau K, et al. Drug Discov Today 2015 20(1):122-8). SIM2 bound to SUMO1 in vitro (FIG. 18B) and blocked de novo lipogenesis in diverse types of cancer cells including breast (MDA-MB-231, MDA-MB-468 and Hs578T), prostate (R1-AD1 and R1-D567), colon (HCT116), and mouse tumor cells (4T1 and CT26.CL25) (FIG. 18C-18H). SIM2 also effectively inhibited de novo lipogenesis in MDA-MB-231 cells overexpressing DAXX (FIG. 18C). Further, SIM2 blocked the expression of lipogenic genes, and the inhibition appeared more potent when DAXX expression level was high (FIG. 18I).

Mechanistically, SIM2 markedly inhibited the interaction of DAXX with full-length and mature SREBP1/2, while DAXX self-association and interaction with ATRX were only slightly affected (FIG. 7A), suggesting that SIM2 preferentially blocks the DAXX-SREBP interaction. To test in vivo efficacy of SIM2 for inhibiting tumor growth, orthotopic human MDA-MB-231 xenograft tumors were generated in the mammary fat pads of female NSG mice. As shown in FIG. 7B, SIM2 significantly impeded tumor growth, and at the experimental endpoint, the tumor masses were much smaller in the SIM2 treatment group compared to the vehicle group. In a syngeneic mouse mammary tumor model, the mouse 4T1 cell line (derived from a spontaneous mouse mammary tumor with human basal/TNBC characteristics (Kaur P, et al. BMC Cancer 2012 12:120)) was transplanted 20 into mammary fat pads of immunocompetent BALB/c mice. 4T1 tumor-bearing mice were dosed with vehicle or SIM2. Consistent with the MDA-MB-231 xenograft results (FIG. 7B), SIM2 was highly potent to inhibit the 4T1 tumor growth (FIG. 7D). To understand potential mechanism of action for SIM2, MDA-MB-231 tumor extracts were subjected to immunoblotting. Interestingly, the protein levels of DAXX, SREBP2, and FASN were lower in the SIM2 treatment group compared to the control group (FIG. 7C), suggesting that SIM2 acts to downregulate the DAXX-driven lipogenesis pathway. Lipidomic profiling of MDA-MB-231 xenograft tumor samples from each treatment group revealed a marked suppression of lipid production by SIM2 (FIG. 7E). Overall, these data indicate that the SIM/SUMO interface represents a tractable therapeutic target to inhibit de novo lipogenesis for cancer therapy. These data also highlight the therapeutic potential of SIM2 as a class of cancer therapeutics.

Discussion

Lipid availability for proliferating cells determines the activity of intracellular lipid biosynthesis pathway. In a nutrient-poor tumor microenvironment, limited supplies of lipids necessitate the activation of intracellular lipid production in tumor cells for sustained tumor growth. An elaborate sterol sensing mechanism controls the nuclear translocation of SREBP1/2, which promote the expression of enzymes required for de novo lipogenesis (Rohrig F, et al. Nat Rev Cancer 2016 16(11):732-49; Bengoechea-Alonso M T, et al. Curr Opin Cell Biol 2007 19(2):215-22). SREBP1/2 in conjunction with several transcription factors, such as the E-box-binding basic helix-loop-helix (bHLH) transcription factor USF1, activate the expression of lipogenic enzymes and regulators (Wang Y, et al. Nat Rev Mol Cell Biol 2015 16(11):678-89). Other coregulators of gene expression such as acetyltransferases (e.g., p300 and PCAF) as well as oncogenic signaling pathways (e.g., KRAS and mTOR) also play important roles in stimulating de novo lipogenesis (Ricoult S J, et al. Oncogene 2016 35(10):1250-60). DAXX is shown here to be critical for de novo lipogenesis. Mechanistically, DAXX interacts with SREBP1/2 and is enriched in chromatins containing SRE motifs. Importantly, DAXX mutants that cannot bind SREBP2 are unable to promote lipogenesis and tumor growth. Thus, DAXX enhances lipogenesis through interacting with SREBP1/2 to promote lipogenic gene expression. Most significantly, the SUMO-binding property of DAXX is critical for de novo lipogenesis and tumor growth, and the DAXX/SUMO interface could be targeted to inhibit lipogenesis and tumor growth.

Consistent with a critical role for the DAXX SUMO-binding activity in DAXX's chromatin recruitment, lipogenic gene expression and hence tumor growth, the SIM2 peptide corresponding to the C-terminal SIM of DAXX blocks de novo lipogenesis and potently inhibits in vivo tumor growth (FIG. 7 and FIG. 18). Mechanistically, SIM2 blocks DAXX-SREBP1/2 interaction, and SIM2 treatment results in reduced lipid levels and the downregulation of DAXX, SREBP2, and FAS in vivo (FIG. 7). Thus, the inhibition of the DAXX/SREBP pathway by SIM2 blocks de novo lipogenesis and potentially other critical mechanisms underpinning aggressive tumor growth, which may underlie SIM2-mediated antitumor effects. The SIM2 peptide has a unique sequence and displays a high affinity to SUMO1 (Santiago A, et al. Cell Cycle 2009 8(1):76-87; Chang C C, et al. Mol Cell 2011 42(1):62-74) (FIG. 18B). In general, peptides are not permeable through cell membranes. However, SIM2 is spontaneously and rapidly internalized into cells (FIG. 18A). Importantly, the SIM2 peptide was highly tolerated in vivo (FIG. 7). The discovery of SIM2 as a class of therapeutics has immediate implications for developing peptide drugs. Overall, these findings indicate that SIM2, through a new anticancer mechanism of action, has an exciting potential for translation into the clinic for treating cancer.

These results imply that the interaction between DAXX and SREBP2 is regulated through site-specific phosphorylation. Specially, converting S495 and S671, the two most frequently phosphorylated sites (FIG. 17A), to un-phosphorylatable alanine, abolished the DAXX-SREBP2 interaction (FIG. 6A). Remarkably, both DAXX S495A and S671A mutants are defective in promoting lipogenic gene expression, lipid production and tumor growth (FIG. 6). These results suggest that the DAXX-SREBP2 axis is subject to regulation by protein phosphorylation. Although how DAXX phosphorylation regulates the DAXX-SREBP interaction remains to be determined, it has been well documented that phosphorylation alters protein structure and protein-protein interaction (Dick F A, et al. Nat Rev Mol Cell Biol 2013 14(5):297-306). Notably, a previous study indicates that phosphorylation at the serine in mouse Daxx equivalent to S495 of the human DAXX is regulated (Ecsedy J A, et al. Mol Cell Biol 2003 23(3):950-60). Currently, the kinases that phosphorylate S495 and S471 remain unknown. The identification of such kinases may define signaling mechanisms that govern the DAXX-SREBP axis, and more importantly, may have translational potential, as kinase inhibitors that block DAXX phosphorylation may effectively inhibit oncogenic lipogenesis and tumor growth.

Biochemically, DAXX's oncogenic function through promoting lipogenesis could be separated from its well-defined histone H3.3 chaperone function (Lewis P W, et al. Proc Natl Acad Sci USA 2010 107(32):14075-80; Goldberg A D, et al. Cell 2010 20 140(5):678-91; Drane P, et al. Genes Dev 2010 24(12):1253-65). Indeed, S495A mutant, despite being capable of H3.3 binding (FIG. 17C), fails to promote lipogenic gene expression and tumor growth (FIG. 6). In contrast, the H3.3-binding defective DAXX Y222P mutant retains the ability to promote tumor growth (FIG. 6). Notably, DAXX's histone binding activity stabilizes DAXX protein (Hoelper D, et al. Nat Commun 2017 8(1):1193). In keeping with this observation, the DAXX Y222P mutant was expressed at a low level (FIG. 17C). Furthermore, DAXX S671A mutant did not bind H3.3 and this mutant accumulated to a much lower level than wt DAXX and the S495A mutant (FIG. 17C). Interestingly, cells expressing DAXX S671A mutant are more profoundly impaired in lipogenic and cell cycle gene expression, lipogenesis, and tumor growth (FIG. 6). Indeed, these data indicate that the DAXX S671A mutant appears to exhibit a dominant negative effect on tumor growth compared to the S495A mutant. Thus, although the histone chaperone function of DAXX per se is not required for DAXX's oncogenic activity, stabilization of DAXX through histone binding may contribute indirectly to oncogenesis.

These data also suggest that the DAXX-ATRX interaction is not involved in DAXX's oncogenic function. Indeed, DAXX mutants S495A and S671A that could not enhance lipogenesis or tumorigenesis retain the ability to bind ATRX (FIG. 17C), but these mutants are defective in promoting lipid production and tumor growth. The DAXX-5 ATRX interaction plays an important role in depositing the histone variant H3.3 to telomeric heterochromatin regions and gene repression (Lewis PW, et al. Proc Natl Acad Sci USA 2010 107(32):14075-80; Goldberg A D, et al. Cell 2010 140(5):678-91; Drane P, et al. Genes Dev 2010 24(12):1253-65; Hoelper D, et al. Nat Commun 2017 8(1):1193). In mouse embryonic stem cells, Daxx and Atrx co-occupy and transcriptionally silence tandem repetitive elements including telomeres through promoting histone H3K9 trimethylation (He Q, et al. Cell Stem Cell 2015 17(3):273-86). More recent studies have shown that the DAXX-ATRX complex represses endogenous retroviruses in an H3.3-dependent fashion and through engaging the SETDB1, KAP1 and HDAC co-repressors (Hoelper D, et al. Nat Commun 2017 8(1):1193; Elsasser S J, et al. Nature 2015 522(7555):240-4). Notably, pancreatic neuroendocrine tumors and other tumor types exhibit frequent loss-of-function mutations of both DAXX and ATRX (Jiao Y, et al. Science 2011 331(6021):1199-203). Interestingly, tumor-derived missense mutations in DAXX occur most frequently in the two structurally defined and evolutionally conserved DHB and HBD domains in DAXX, which bind ATRX and H3.3/H4 dimer, respectively (Hoelper D, et al. Nat Commun 2017 8(1):1193). It was shown that Men1/Daxx/Atrx complex-mediated gene repression underlies tumor suppression in murine neuroendocrine tumor models (Feng Z, et al. Cancer Res 2017 77(2):401-11). Thus, the putative tumor suppressor activity of DAXX may be linked to ATRX binding, H3.3 deposition, and gene silencing. However, DAXX is rarely mutated in breast cancer and other epidemiologically prevalent cancer types. Of note, ATRX-independent oncogenic function of DAXX was also implicated in glioblastoma (Benitez J A, et al. Nat Commun 2017 8:15223). Taken together, DAXX's oncogenic function may derive from its activity to promote gene expression required for increased lipogenesis and perhaps also for cell cycle progression, which is independent of its association with ATRX.

In summary, this study establishes DAXX as a key regulator of oncogenic lipogenesis. Molecularly, DAXX interacts with SREBPs to stimulate lipogenic gene expression, thereby promoting tumor growth. The DAXX-SREBP2 interaction may be regulated through phosphorylation. Importantly, DAXX's SUMO-binding property is critical for lipogenesis and the SIM2 peptide targeting the SUMO-SIM interface has potent anti-tumor efficacy.

Methods

Cell culture.

Cells were cultured in Dulbecco's Modified Eagle's Medium (DMEM with 4.5 g/L glucose, L-glutamine and sodium pyruvate; Corning) with 10% bovine calf serum (HyClone, GE Healthcare Bio-Sciences, Pittsburgh, PA), penicillin (10 units/m1), and streptomycin (10 μg/ml) (the complete DM EM medium). The T47D cell line was cultured in DMEM plus 10% fetal bovine serum (Atlanta Biologics, Atlanta, GA), penicillin (10 units/m1), and streptomycin (10 μg/ml). To culture cells in serum starvation condition, serum-containing medium was removed from cell cultures after overnight culture and the culture was washed once with phosphate-buffered saline (PBS, without calcium and magnesium, Corning). Cells were then cultured in serum-free DMEM. For culturing cells in suspension (3D culture), plates were coated with a 1:1 mixture of Matrigel (Corning, Tewksbury, MA) and complete DMEM medium. A desirable number of cells were suspended in the Matrigel and medium mixture and layered on the top of the solidified Matrigel. Complete DM EM medium was added after the Matrigel was solidified. Medium was replaced with fresh complete medium every three days. Colonies were imaged under a microscope; colony numbers and sizes were quantified. Human cell lines were obtained from ATCC (Manassas, VA) and authenticated by Genetica DNA Laboratories (Burlington, NC). The mouse cancer cell lines 4T1, CT26.CL25, and TRAMP-C2 were from ATCC. The mouse breast cancer cell line E0771 was from CH3 BioSystems (Amherst, NY).

Microarray, RNA-seq and qRT-PCR.

For microarray experiments, cells were cultured in the complete DMEM or serum-free DMEM, and total RNAs were isolated using the RNeasy kit (Qiagen). The RNAs were then processed for microarray hybridization to the Affymetrix GeneChip Human Transcriptome Array 2.0 as described previously (Wang Y, et al. Chem Biol 2015 22(2):273-84). RNA-seq was done with 20M raw reads/sample using Illumina Platform PE150 at Novogene Corporation Inc. (Sacramento, CA).

Following the RNAseq data analysis pipeline reported previously (Pertea M, et al. Nat Protoc 2016 11(9):1650-67), fastq files were aligned to Genome Reference Consortium Human Build 38 (GRCh38) using HISAT2 (Kim D, et al. Nat Methods 2015 12(4):357-60); the transcripts assembling was performed using StringTie (Pertea M, et al. Nat Biotechnol 2015 33(3):290-5) with RefSeq as transcripts ID; and the normalized counts (by FPKM) was called using Ballgown (Frazee A C, et al. Nat Biotechnol 2015 33(3):243-6). The differential expression analysis was performed using R package limma (Smyth G K. Stat Appl Genet Mol Biol 2004 3:Article3); and the pathway enrichment analysis was performed using ingenuity pathway analysis.

For quantitative real-time PCR (qRT-PCR), the isolated RNAs were reverse transcribed with random hexamers using 2 μg of total RNA, an RNase inhibitor, and reagents in the Multiscribe reverse transcriptase kit (Life Technologies). The resulting cDNAs were diluted and used as input for qPCR using the SYBR green detection method. The relative levels of gene expression were determined using the ΔΔCt method with the Ct values of ACTB expression as the common normalizer. The primers for qPCR and other applications are provided below.

In microarray experiments, probe set files (.cel file) were normalized by RMA algorithm and analyzed using both R statistical package as well as Affymetrix expression and transcriptome console software from ThermoFisher Scientific.

DNA Constructs.

cDNAs for wild-type (wt) DAXX and mutants (17K/1733K, DSM), Y124F/Y126F, Y222P, S495A, S561A, K630A/K631A, S668A, S671A, S668A/S671A, S690A, and deICT) with a 5′ coding sequence for the FLAG epitope tag and a 3′ coding sequences for the MYC and 6x His tags were cloned into a lentiviral vector under the control of the cytomegalovirus immediate early (CMV IE) promoter. GFP-DAXX constructs were cloned in the pEGFP-C2 vector. A short hairpin RNA (shRNA) target the DAXX coding sequence (nucleotide 624-642, 5′-GGAGTTGGATCTCTCAGAA-3′, SEQ ID NO:64) was cloned into a lentiviral vector under the control of the human U6 promoter. An shRNA construct with a scrambled sequence (Plasmid # 36311) was from Addgene. Expression vectors for mature SREBP1a (Plasmid # 26801), mature SREBP1c (Plasmid # 26802), mature SREBP2 (Plasmid # 26807), full-length SREBP1 (Plasmid # 32017) and SREBP2 (Plasmid # 32018) were purchased from Addgene. The shRNA clones for SREBF1 (TRCN0000020607 and TRCN0000020605), and SREBF2 (TRCN0000020667 and TRCN0000020668) were from the human pLKO.1 TRC Library collection at the University of Florida. The SREBF2 shRNA vector TRCN0000020667 was used to knockdown SREBF2 expression in MDA-MB-231 cells with DAXX OE. A SREBP2 promoter fragment was PCR amplified from the genomic DNA isolated from MDA-MB-231 cell line, and cloned at sites upstream of the firefly luciferase reporter by the Gibson assembly method. The DNA sequence was confirmed by Sanger sequencing. The PCR primers are shown in Table 1.

Stable expression of cDNA and shRNA was established through lentiviral transduction of cell lines and puromycin (2 μg/ml) selection. The derived cell lines were cultured with DM EM without puromycin.

Immunoprecipitation (IP) and Immunoblotting.

Cell pellets were resuspended in the IP lysis buffer (50 mM Tris-HCl, pH 7.5, 0.5% Igepal-CA630, 5% glycerol, 150 mM NaCl, 1.5 mM MgCl2, and 25 mM NaF) containing 100-fold diluted protease inhibitor cocktail (Millipore-Sigma P8340). The cell suspension was subjected to two freezing/thawing cycle. The cell lysates were then centrifuged at 15,000 rpm at 4° C. for 20 min. The supernatant was used for IP with a control or an antibody to a specific protein at 2 μg per IP in the presence of protein A-agarose beads. For IPs in the presence of the SIM2 peptide, SIM2 in PBS was added to a desirable concentration. IP mixtures were rotated at 4° C. overnight. The beads were washed four times with the IP lysis buffer and once with the RIPA buffer (50 mM Tris-HCl, pH 7.4, 150 mM NaCl, 1% NP-40, 0.5% sodium deoxycholate). The beads were resuspended in the IP lysis buffer along with one fifth of the volume of the 6× SDS sample buffer (0.375 M Tris-HCl, pH 6.8, 12% SDS, 60% glycerol, 0.6 M DTT, and 0.06% bromophenol blue). Samples were heated at 95° C. for 5 min and chilled on ice for 2 min. After brief centrifugation, the samples were loaded on a 4-20% gradient gel (Novex Tris-Glycine Mini Gels, ThermoFisher). Proteins were then electrotransferred to an Immobilon®-P polyvinylidene fluoride (PVDF) membrane (Millipore). Membrane was blocked with 5% non-fat milk, incubated with a primary antibody and a proper secondary antibody. The proteins were detected using a chemiluminescent detection kit (Millipore) and the Fuji Super RX-N X-ray films.

For immunoblotting analyses of cell lysates of monolayer cultures, medium was removed from culture plates and lx Passive Lysis buffer (Promega) was added. The plates were frozen at −80° C. overnight and then thawed at room temperature. The lysates were transferred to a centrifuge tube. To prepare tumor lysates, xenograft tumor tissues were fragmented in the presence of liquid nitrogen, approximately 50 mg of tumor fragment was homogenized in 1 mL of 1× RIPA lysis buffer on ice using a micro-homogenizer. After brief sonication at a low power output for 5 sec on ice, the lysates were cleared by centrifugation at 13,000 rpm for 15 min at 4° C. Protein contents were quantified using a Qubit protein assay kit. Protein extracts from cell culture or tumor lysates were subjected to SDS-PAGE and electro-blotting as above. The antibodies used for this study are listed in Table 2.

De Novo Lipogenesis Assays

Cells (0.5 million per well) were plated in a 6-well plate in complete DMEM medium in triplicate. At 24h after seeding, cells were washed once with PBS and cultured in serum-free DMEM for 16 to 18 h; 5 μCi of [1-14C] acetate (NEC084H001MC, Perkin Elmer, Waltham, MA, USA) per ml was added and the cells were cultured for four more hours. Cells were then washed twice with PBS and trypsinized. Cells were pelleted and resuspended in 0.5 ml of 0.5% Triton X-100. The protein concentration of the lysates was determined for normalization. The lysates were extracted with ice cold chloroform/methanol (2:1 v/v). After centrifugation at 1,000 rpm for 20 min, the organic phase was collected and air dried. The radioactivity was determined with a liquid scintillation counter (Beckman LS 5000TD). The radioactivity was normalized against protein concentration.

Liquid chromatography (LC)-mass spectrometry (MS) experiments. For lipid analysis, these internal lipid standards were used: triglyceride (TG 15:0/15:0/15:0 and TG 17:0/17:0/17:0, Sigma-Aldrich), lysophosphatidylcholines (LPC, 17:0 and 19:0), phosphatidylcholines (PC, 17:0/17:0 and 19:0/19:0), phosphatidylethanolamines (PE, 15:0/15:0 and 17:0/17:0), phosphatidylserines (PS, 14:0/14:0 and 17:0/17:0), and phosphatidylglycerols (PG, 14:0/14:0 and 17:0/17:0) (Avanti Polar Lipids, Alabaster, AI). The lipid standards were dissolved in 2:1 (v/v) chloroform:methanol to make a 1000 ppm stock solution and a working 100 ppm standard mix was then prepared by diluting the stock solution with the same solvent mixture. For sample normalization, total protein concentration in each sample was determined using a Qubit 3.0 Fluorometer.

Cell lines with shControl, shDAXX, wt DAXX overexpression (wt OE), and the DSM mutant overexpression (DSM OE) were cultured with the complete DMEM. When cells grew to approximately 80% confluency, they were washed twice with PBS and cells were detached using a cell lifter.

Cell pellets were washed twice with 40 mM ammonium formate (AF). The cell pellets were resuspended in 50 μL of AF with vortex in a glass vial and subjected to high efficient bead beater cell disruption to release intracellular lipids. A small amount of the homogenized cell pellet was taken for Qubit protein concentration determination. Lipids were extracted by adding ice-cold chloroform (2 mL) and methanol (1 mL) along with 20 μL of internal standard mixtures. The extraction mixture was incubated on ice for 1 h with occasional vortex mixing. Finally, 1 mL H2O was added to the mixture, which was incubated for 10 min with occasional vortex mixing. Samples were then centrifuged at 2,000 rpm for 5 minutes. The lower phase (organic layer) was collected in a separate glass vial and subjected to dry under nitrogen gas at 30 ° C. using a dryer (MultiVap,

Organomation Associates). Dried samples were reconstituted by adding 50 μL isopropyl alcohol and transferred to a glass LC vial with insert. Samples were loaded to an auto-sampler at 5° C.

For analyzing lipids, samples were run for quality control (QC) in each instrument run. A pooled QC sample (a 25 μL aliquot) for each extraction was injected after analyzing every five samples. The pooled QC sample was run to assess system reproducibility, and a blank (solvent mixture only) was used to flush the column. No changes were observed regarding the number of background ions, which always corresponded to the specific solvent used for lipid extraction. Also, no effects on reproducibility of ion source was observed regardless of solvents used for extraction.

The stability and repeatability of the instruments were evaluated using identical neat QC samples (a mixture of all internal standards in deuterated form) throughout the process of sample injection. Principal component analysis (PCA) was performed to evaluate the variation of QC samples. All neat QC samples clustered together, confirming the stability and reproducibility of our experimental lipid analysis system.

For data collection, processing, and analysis, a Dionex Ultimate 3000 UHPLC system coupled to a Q Exactive™ hybrid quadrupole-orbitrap mass spectrometer operated in HESI-positive and negative ion mode was used. A Supelco Analytical Titan reverse-phase column (RPC) C18 (2.1×75 mm with 1.9 pm monodisperse silica) equilibrated at 30° C. with solvents A (acetonitrile and water 60:40, v/v) and B (isopropyl alcohol, acetonitrile, and water 90:8:2, v/v/v) as mobile phases was used for data collection. The flow rate was 0.5 ml/min, and the injection volume was 5 μL. The total run time was 22 min, including a 2-min equilibration. The MS conditions for positive and negative ion modes were spray voltage at 3.5 kV, sheath gas at 30 arbitrary units, sweep gas at 1 arbitrary units, auxiliary nitrogen pressure at 5 arbitrary units, capillary temperature at 300° C., HESI auxiliary gas heater temperature at 350° C., and S-lens RF at 35 arbitrary units. The instrument was set to acquire in the mass range of most expected cellular lipids and therefore m/z 100-1500 was chosen with a mass resolution of 70,000 (defined at m/z 200). Global lipid profiling was performed using full scan and ddMS2 (data dependent MS-MS).

Data were recorded from 0.0 to 17 min as total ion chromatography (TIC) and then corresponding MS data were extracted using Thermo Xcalibur (version 2.2.44). After data collection, raw data files were converted to mzXML format using the Proteowizard MSConvert software. MZmine 2.15 (freeware) was used for mass detection with mass detector centroid noise set at 1.0E5 using only MS level 1 data; chromatogram building and deconvolution were then applied (m/z tolerance, 0.005 or 10 ppm; retention time tolerance, 0.2 min; minimum time span, 0.1 min; and minimum height, 5.0E5) followed by isotope grouping, alignment (m/z tolerance, 0.005 or 10 ppm; retention time tolerance, 0.2 min), and gap filling (m/z tolerance, 0.005 or 10 ppm; retention time tolerance, 0.2 min, and intensity tolerance 25%). MZmine-based online metabolite search engine KEGG, MMCD database, XCMS online database, Metaboanalyst 3.0, R program, and internal retention time library were used for the identification and analysis of metabolites.

Peptides

The SIM2 (DPEEIIVLSDSD, SEQ ID NO:1) and the TAM RA (5-carboxytetramethylrhodamine)-SIM2 peptides were synthesized at >95% purity by GenScript (Piscataway, NJ).

In Vivo Tumor Growth and Treatment Experiments

All mice were maintained under pathogen-free conditions. Female NSG (NOD.Cg-Prkdcscid∥2rgtm1Wjl/SzJ) mice, between the ages of 4-6 weeks, were injected subcutaneously in a mammary fat-pad area with one million cells in 100 μl of complete DM EM (MDA-MB-231-derived cell lines) or in a suspension of 50 ρl of Matrigel and 50 ρl of cell suspension (MDA-MB-468-derived cell lines). The prostate cancer R1-AD1-derived cells (Matrigel suspension) and the colon cancer HCT116-derived cells (medium suspension) were injected subcutaneously in a flank of male NSG mice (the R1-AD1 model) or that of both female and male NSG mice (the HCT116 model). Tumor growth was monitored by measuring tumor dimensions using a digital caliper once a week until endpoint. Tumor volume was calculated with the formula ½×length×width2. At the endpoint, mice were euthanized, tumors were excised, weighted, and photographed.

For the in vivo SIM2 treatment study, MDA-MB-231 xenograft tumors were established as above and the mouse mammary tumor cell line 4T1 syngeneic tumors were established by transplanting one million cells in 100 μl of DMEM into mammary fat pads of female BALB/c mice. When tumors grew to a palpable size, tumor-bearing mice were randomized into vehicle and SIM2 treatment arms, so that each group has similar distributions of tumor volumes. The vehicle consisted of 33% (2-HydroxypropyI)-8-cyclodextrin (HPBCD, RND Center Inc., La Jolla) in PBS and 45% polyethylene glycol 400 (Alfa Aesar, Tewksbury, MA), which was filtered through a 0.22 pm filter. The SIM2 peptide was formulated at 5 mg/ml in the vehicle. Tumor-bearing mice were injected intraperitoneally with 100 μl of vehicle or the formulated SIM2 peptide once daily every weekday until a predefined endpoint. During the treatment, tumor dimensions and mouse body weights were recorded once weekly. At the endpoint, tumors were excised, weights and photographed. Tumors were flash frozen in liquid nitrogen, and then stored at −80° C. Tumor lysates were prepared for immunoblotting analysis. Animal use has been approved for this project by the University of Florida IACUC.

Ingenuity Pathway Analysis (IPA)

Genes that were differentially expressed (fold-change over±1.3 and p-value<0.05) were used for the Ingenuity Pathway Analysis (Ingenuity Systems, Qiagen Bioinformatics).

Gene Set Enrichment Analysis (GSEA)

GSEA was performed using the Java desktop software, as described previously (Subramanian A, et al. Proc Natl Acad Sci USA 2005 102(43):15545-50). The GSEA tool was used in pre-ranked mode with all default parameters.

ChIP-seq Analysis.

The panel of MDA-MB-231-derived cell lines (control, shDAXX, wt DAXX and DSM mutant OE) were cultured in complete DMEM. At about 90% confluency, the cells were crosslinked by adding 37% formaldehyde to the final concentration of 1% for 10 min at room temperature. Crosslinking was stopped by adding glycine to the final concentration of 125 mM. Cells were lifted, washed with cold PBS, and pelleted by centrifugation. The cells were resuspended in a swelling buffer in the presence of the protease inhibitor cocktail (Sigma) and then pelleted and resuspended in the SDS lysis buffer. The lysates were transferred to a Covaris microTUBE and sonicated with an E220 Covaris Ultrasonicator. Chromatin fragmentation (˜500 bps) was verified through agarose gel electrophoresis. The fragmented chromatins were diluted and incubated with a control IgG and the DAXX mAb (5G11) along with protein A/G magnetic beads. The beads were washed sequentially with a low salt buffer, high salt buffer, LiCl buffer, and TE buffer (twice). The immunoprecipitated chromatins were eluted at 65° C. for 15 min, and the eluted chromatins were subjected to proteinase K digestion at 65° C. for 3 h. The DNAs were recovered through a Qiagen mini-prep column.

The immunoprecipitated DNAs were used for qPCR and library construction and high throughput sequencing using an Illumina Hi-Seq 2500 sequencer. ChIP-seq sequencing reads (Fastq files) were mapped to the human genome (GRCh37/hg19) using Bowtie2 (Langmead B, et al. Nat Methods 2012 9(4):357-9), where option -local was specified to trim or clip unaligned reads from one or both ends of the alignment. Genome browser BedGraph tracks and read density histograms were generated using SeqMINER. Peak finding and annotation to the nearest Refseq gene promoter was performed and de novo motif discovery was carried out using HOMER (Heinz S, et al. Mol Cell 2010 38(4):576-89).

Bioinformatics Analysis

The copy number and gene expression were analyzed based on publicly available datasets. Gene expression data for normal, benign, primary, and metastatic tumor samples were included for our analysis. Normalized expression levels for specific genes were compared between different sample types. Computations were conducted in R statistical package and in GraphPad Prism 7.0.

Statistical Analysis

Gene expression assays were conducted in two to three biological replicates. Metabolic profiling assays were performed in >six replicates. Data are presented as the mean along with standard error of the mean (SEM). Student's t-test was used to compare two groups of independent samples. For all data analysis, p<0.05 was considered statistically significant.

Unless defined otherwise, all technical and scientific terms used herein have the same meanings as commonly understood by one of skill in the art to which the disclosed invention belongs. Publications cited herein and the materials for which they are cited are specifically incorporated by reference.

Those skilled in the art will recognize, or be able to ascertain using no more than routine experimentation, many equivalents to the specific embodiments of the invention described herein. Such equivalents are intended to be encompassed by the following claims.

