# DESCRIPTION

## BACKGROUND

Nanopore-based sensors are advancing the sensitivity and selectivity of single-molecule detection in molecular medicine and biotechnology. Conventional electrical and electro-optical sensing devices are based on either membrane protein pores supported in planar lipid bilayers or solid-state pores drilled into thin metallic membranes. While both types of nanosensors have been used in a variety of applications, each has inherent disadvantages that limit their use.

## SUMMARY

Hybrid nanopores in accordance with an embodiment of the invention, comprising a protein pore supported within a solid-state membrane, combine the robust nature of solid-state membranes with the easily tunable and precise engineering of protein nanopores. A lipid-free hybrid nanopore comprises a water soluble and stable, modified portal protein of the Thermus thermophilus bacteriophage G20c, electrokinetically inserted into a larger nanopore in a solid-state membrane. The hybrid pore is stable and easy to fabricate, and exhibits low peripheral leakage, allowing sensing and discrimination among different types of biomolecules.

In one embodiment according to the invention, there is provided a sensor. The sensor comprises: a solid-state matrix comprising a solid-state pore opening; and a hydrophilic protein channel in a stable insertion fit within the solid-state pore opening, the hydrophilic protein channel comprising a protein nanopore channel.

In further, related embodiments, a protein forming at least part of the hydrophilic protein channel may comprise a modification of an amino acid sequence comprising SEQ ID NO: 1, the modification comprising one or more of: (i) a modification of an upper internal surface residue or a lower internal surface residue of the protein of the hydrophilic protein channel, the modification comprising an alteration of an electrostatic surface potential of an upper internal surface or a lower internal surface of SEQ ID NO: 1; (ii) a modification comprising an insertion of a cysteine residue into SEQ ID NO: 1 or a replacement of an amino acid residue of SEQ ID NO: 1 with a cysteine residue; (iii) a modification of a tunnel loop residue of the protein of the hydrophilic protein channel, the modification comprising an expansion of a narrowest constriction of a tunnel loop of SEQ ID NO: 1, a restriction of the narrowest constriction of the tunnel loop of SEQ ID NO: 1, or a removal of the narrowest constriction of the tunnel loop of SEQ ID NO: 1; (iv) a modification of a tunnel loop residue of the protein of the hydrophilic protein channel, the modification resulting in an alteration of an electrostatic charge property of a tunnel loop of SEQ ID NO: 1; (v) a modification which alters an external charge of the protein of the hydrophilic protein channel; (vi) a modification which promotes binding of the protein of the hydrophilic protein channel to the solid-state matrix; (vii) a modification which extends an N-terminus or a C-terminus of SEQ ID NO: 1 or an N-terminus or a C-terminus of a cleaved portion of SEQ ID NO: 1; and (viii) a deletion of an amino acid residue of at least one of a C-terminal region and an N-terminal region of SEQ ID NO: 1. The modification of SEQ ID NO: 1 may comprise a modification of a lower internal surface residue of the protein of the hydrophilic protein channel, the modification comprising the alteration of the electrostatic surface potential of the lower internal surface of SEQ ID NO: 1, and the protein of the hydrophilic protein channel may comprise SEQ ID NO: 27. The modification of SEQ ID NO: 1 may comprises a replacement of an amino acid residue of SEQ ID NO: 1 with a cysteine residue, and the protein of the hydrophilic protein channel may comprise SEQ ID NO: 11 or SEQ ID NO: 37. The modification of SEQ ID NO: 1 may comprise a modification of the tunnel loop residue of the protein of the hydrophilic protein channel, the modification comprising the expansion of the narrowest constriction of the tunnel loop of SEQ ID NO: 1, the restriction of the narrowest constriction of the tunnel loop of SEQ ID NO: 1, or the removal of the narrowest constriction of the tunnel loop of SEQ ID NO: 1, and the protein of the hydrophilic protein channel may comprise one of: SEQ ID NO: 15, SEQ ID NO: 17, SEQ ID NO: 21, SEQ ID NO: 23 and SEQ ID NO: 25. The modification of SEQ ID NO: 1 may comprise the modification in a tunnel loop residue of the protein of the hydrophilic protein channel, the modification resulting in the alteration of the electrostatic charge property of the tunnel loop of SEQ ID NO: 1, and the protein of the hydrophilic protein channel may comprise SEQ ID NO: 19. The modification of SEQ ID NO: 1 may comprise the modification which alters an external charge of the hydrophilic protein channel, and the protein of the hydrophilic protein channel may comprise one of: SEQ ID NO: 9 and SEQ ID NO: 13. The modification of SEQ ID NO: 1 may comprise the modification which promotes binding of the hydrophilic protein channel to the solid-state matrix, and the protein of the hydrophilic protein channel may comprise one of: SEQ ID NO: 29, SEQ ID NO: 31 and SEQ ID NO: 35. The modification of SEQ ID NO: 1 may comprise the modification which extends the N-terminus of the cleaved portion of SEQ ID NO: 1, and the protein of the hydrophilic protein channel may comprise SEQ ID NO: 33. The modification of SEQ ID NO: 1 may comprise the deletion of an amino acid residue of at least one of the C-terminal region and the N-terminal region of SEQ ID NO: 1, and the modification may comprise a modification of one of: SEQ ID NO: 3, SEQ ID NO: 5 and SEQ ID NO: 7. The protein of the hydrophilic protein channel may comprise a modification of SEQ ID NO: 1, the modification comprising a modification in one or more of: a wing loop residue, a tunnel loop residue, an upper internal surface residue, a lower internal surface residue, an external surface residue, a C-terminal residue, and an N-terminal residue.

In other related embodiments, the sensor may further comprise a voltage source configured to apply a voltage to an electrolyte solution on both sides of the solid-state matrix. The solid-state matrix may comprise at least one of: silicon, hafnium and nickel. The solid-state matrix may comprise at least one of: a silicon containing nitride, a silicon containing carbide and a silicon containing oxide. The solid-state matrix may comprise a thickness of less than about 30 nm. The solid-state pore opening may comprise a diameter of between about 5.4 nm and about 6 nm. The sensor may further comprise a coating on the solid-state matrix to promote binding of the solid-state matrix to a protein forming at least part of the hydrophilic protein channel. The coating may comprise a thiol-coupling compound; and may comprise a maleimide compound.

In other related embodiments, a protein forming at least part of the hydrophilic protein channel may comprise a modification of an amino acid sequence comprising SEQ ID NO: 1, the modification comprising one or more of: a modification of residue 328 of SEQ ID NO: 1, a modification of residue 189 of SEQ ID NO: 1, and a modification of residue 367 of SEQ ID NO: 1. A protein forming at least part of the hydrophilic protein channel may comprise a modification of SEQ ID NO: 1, the modification comprising one of: SEQ ID NO: 27; SEQ ID NO: 11; SEQ ID NO: 37; SEQ ID NO: 15, SEQ ID NO: 17, SEQ ID NO: 21, SEQ ID NO: 23, SEQ ID NO: 25, SEQ ID NO: 19, SEQ ID NO: 9, SEQ ID NO: 13, SEQ ID NO: 29, SEQ ID NO: 31, SEQ ID NO: 35, SEQ ID NO: 33, SEQ ID NO: 3, SEQ ID NO: 5, and SEQ ID NO: 7. A protein forming at least part of the hydrophilic protein channel may comprise a modification of SEQ ID NO: 1, the modification comprising one of: SEQ ID NO: 27; SEQ ID NO: 37; SEQ ID NO: 15, SEQ ID NO: 17, SEQ ID NO: 23, SEQ ID NO: 25, SEQ ID NO: 19, SEQ ID NO: 9, SEQ ID NO: 13, SEQ ID NO: 29, SEQ ID NO: 31, SEQ ID NO: 35, SEQ ID NO: 33, SEQ ID NO: 3, SEQ ID NO: 5, and SEQ ID NO: 7.

In some embodiments, where reference is made to modifications of SEQ ID NO: 1, it will be appreciated that a modification of a portal protein of the Thermus thermophilus bacteriophage G20c, can be used, or a modification of a portal protein from other bacteriophages (including other bacteriophages of Thermus thermophilus, and other bacteriophages) can be used. In one example, the portal protein of the Thermus thermophilus bacteriophage P23-45, or modifications of that portal protein, can be used. Likewise, in some embodiments, where reference is made to modification of SEQ ID NO: 2, it will be appreciated that other modifications of nucleic acids encoding for a portal protein of the Thermus thermophilus bacteriophage G20c, can be used, or those encoding for a modification of a portal protein from other bacteriophages (including other bacteriophages of Thermus thermophilus, and other bacteriophages) can be used. In one example, modifications of nucleic acids encoding for the portal protein of the Thermus thermophilus bacteriophage P23-45, or modifications of that portal protein, can be used.

In another embodiment according to the invention, there is provided a protein variant, the protein variant comprising a modification of an amino acid sequence comprising SEQ ID NO: 1, the modification comprising one or more of: (i) a modification of an upper internal surface residue or a lower internal surface residue of the protein variant, the modification comprising an alteration of an electrostatic surface potential of an upper internal surface or a lower internal surface of SEQ ID NO: 1; (ii) a modification comprising an insertion of a cysteine residue into SEQ ID NO: 1 or a replacement of an amino acid residue of SEQ ID NO: 1 with a cysteine residue; (iii) a modification of a tunnel loop residue of the protein variant, the modification comprising an expansion of a narrowest constriction of a tunnel loop of SEQ ID NO: 1, a restriction of the narrowest constriction of the tunnel loop of SEQ ID NO: 1, or a removal of the narrowest constriction of the tunnel loop of SEQ ID NO: 1; (iv) a modification of a tunnel loop residue of the protein variant, the modification resulting in an alteration of an electrostatic charge property of a tunnel loop of SEQ ID NO: 1; (v) a modification which alters an external charge of the protein variant; (vi) a modification which promotes binding of the protein variant to a solid-state matrix; (vii) a modification which extends an N-terminus or a C-terminus of SEQ ID NO: 1 or an N-terminus or a C-terminus of a cleaved portion of SEQ ID NO: 1; and (viii) a deletion of an amino acid residue of at least one of a C-terminal region and an N-terminal region of SEQ ID NO: 1.

In further, related embodiments, the modification of SEQ ID NO: 1 may comprise a modification of a lower internal surface residue of the protein variant, the modification comprising the alteration of the electrostatic surface potential of the lower internal surface of SEQ ID NO: 1, and the protein variant may comprise SEQ ID NO: 27. The modification of SEQ ID NO: 1 may comprise a replacement of an amino acid residue of SEQ ID NO: 1 with a cysteine residue, and the protein variant may comprise SEQ ID NO: 11 or SEQ ID NO: 37. The modification of SEQ ID NO: 1 may comprise a modification of the tunnel loop residue of the protein variant, the modification comprising the expansion of a narrowest constriction of a tunnel loop of SEQ ID NO: 1, the restriction of the narrowest constriction of the tunnel loop of SEQ ID NO: 1, or the removal of the narrowest constriction of the tunnel loop of SEQ ID NO: 1, and the protein variant may comprise one of: SEQ ID NO: 15, SEQ ID NO: 17, SEQ ID NO: 21, SEQ ID NO: 23, and SEQ ID NO: 25. The modification of SEQ ID NO: 1 may comprise the modification of a tunnel loop residue of the protein variant, the modification resulting in the alteration of the electrostatic charge property of the tunnel loop of SEQ ID NO: 1, and the protein variant may comprise SEQ ID NO: 19. The modification of SEQ ID NO: 1 may comprise the modification which alters the external charge of the protein variant, and the protein variant may comprise one of: SEQ ID NO: 9 and SEQ ID NO: 13. The modification of SEQ ID NO: 1 may comprise the modification which promotes binding of the protein variant to a solid-state matrix, and the protein variant may comprise one of: SEQ ID NO: 29, SEQ ID NO: 31 and SEQ ID NO: 35. The modification of SEQ ID NO: 1 may comprise the modification which extends the N-terminus of the cleaved portion of SEQ ID NO: 1, and the protein variant may comprise SEQ ID NO: 33. The modification of SEQ ID NO: 1 may comprise the deletion of an amino acid residue of at least one of the C-terminal region and the N-terminal region of SEQ ID NO: 1, and the modification may comprise a modification of one of: SEQ ID NO: 3, SEQ ID NO: 5 and SEQ ID NO: 7. The modification of SEQ ID NO: 1 may comprise a modification in one or more of: a wing loop residue, a tunnel loop residue, an upper internal surface residue, a lower internal surface residue, an external surface residue, a C-terminal residue, and an N-terminal residue.

In another embodiment according to the invention, there is provided a protein variant, the protein variant comprising a modification of an amino acid sequence comprising SEQ ID NO: 1, the modification comprising one of: SEQ ID NO: 27; SEQ ID NO: 11; SEQ ID NO: 37; SEQ ID NO: 15, SEQ ID NO: 17, SEQ ID NO: 21, SEQ ID NO: 23, SEQ ID NO: 25, SEQ ID NO: 19, SEQ ID NO: 9, SEQ ID NO: 13, SEQ ID NO: 29, SEQ ID NO: 31, SEQ ID NO: 35, SEQ ID NO: 33, SEQ ID NO: 3, SEQ ID NO: 5, and SEQ ID NO: 7.

In another embodiment according to the invention, there is provided a protein variant, the protein variant comprising a modification of an amino acid sequence comprising SEQ ID NO: 1, the modification comprising one of: SEQ ID NO: 27; SEQ ID NO: 37; SEQ ID NO: 15, SEQ ID NO: 17, SEQ ID NO: 23, SEQ ID NO: 25, SEQ ID NO: 19, SEQ ID NO: 9, SEQ ID NO: 13, SEQ ID NO: 29, SEQ ID NO: 31, SEQ ID NO: 35, SEQ ID NO: 33, SEQ ID NO: 3, SEQ ID NO: 5, and SEQ ID NO: 7.

In another embodiment according to the invention, there is provided a nucleic acid molecule comprising a modification of a nucleic acid sequence of SEQ ID NO: 2, the modification comprising one or more of: (i) a modification in a portion of the nucleic acid sequence that encodes an upper internal surface residue or a lower internal surface residue of a protein encoded by the nucleic acid sequence, the modification producing an alteration of an electrostatic surface potential of an upper internal surface or a lower internal surface of the protein encoded by the nucleic acid sequence; (ii) a modification comprising an insertion of a cysteine residue into a protein encoded by the nucleic acid sequence or a replacement of an amino acid residue of a protein encoded by the nucleic acid sequence with a cysteine residue; (iii) a modification to produce a modification of a tunnel loop residue of a protein encoded by the nucleic acid sequence, the modification in the tunnel loop residue comprising an expansion of a narrowest constriction of a tunnel loop of the protein encoded by the nucleic acid sequence, a restriction of the narrowest constriction of the tunnel loop of the protein encoded by the nucleic acid sequence, or a removal of the narrowest constriction of the tunnel loop of the protein encoded by the nucleic acid sequence; (iv) a modification to produce a modification of a tunnel loop residue of a protein encoded by the nucleic acid sequence, the modification resulting in an alteration of an electrostatic charge property of a tunnel loop of the protein encoded by the nucleic acid sequence; (v) a modification which alters an external charge of a protein encoded by the nucleic acid sequence; (vi) a modification which promotes binding of a protein encoded by the nucleic acid sequence to a solid-state matrix; (vii) a modification which extends an N-terminus or a C-terminus of a protein encoded by the nucleic acid sequence or an N-terminus or a C-terminus of a cleaved portion of the protein encoded by the nucleic acid sequence; and (viii) a deletion of an amino acid residue of at least one of a C-terminal region and an N-terminal region of a protein encoded by the nucleic acid sequence.

In further, related embodiments, the modification of SEQ ID NO: 2 may comprise a modification in the portion of the nucleic acid sequence that encodes the lower internal surface residue of the protein encoded by the nucleic acid sequence, the modification producing the alteration of the electrostatic surface potential of the lower internal surface of the protein encoded by the nucleic acid sequence, and the nucleic acid molecule may comprise SEQ ID NO: 28. The modification of SEQ ID NO: 2 may comprise the modification comprising an insertion of a cysteine residue into a protein encoded by the nucleic acid sequence or a replacement of an amino acid residue of a protein encoded by the nucleic acid sequence with a cysteine residue, and the nucleic acid molecule may comprise SEQ ID NO: 12 or SEQ ID NO: 38. The modification of SEQ ID NO: 2 may comprise the modification to produce a modification of a tunnel loop residue of the protein encoded by the nucleic acid sequence, the modification in the tunnel loop residue comprising the expansion of the narrowest constriction of the tunnel loop of the protein encoded by the nucleic acid sequence, the restriction of the narrowest constriction of the tunnel loop of the protein encoded by the nucleic acid sequence, or the removal of the narrowest constriction of the tunnel loop of the protein encoded by the nucleic acid sequence, and the nucleic acid molecule may comprise one of: SEQ ID NO: 16, SEQ ID NO: 18, SEQ ID NO: 22, SEQ ID NO: 24 and SEQ ID NO: 26. The modification of SEQ ID NO: 2 may comprise a modification to produce a modification of a tunnel loop residue of the protein encoded by the nucleic acid sequence, the modification resulting in the alteration of the electrostatic charge property of the tunnel loop of the protein encoded by the nucleic acid sequence, and the nucleic acid molecule may comprise SEQ ID NO: 20. The modification of SEQ ID NO: 2 may comprise the modification which alters the external charge of the protein encoded by the nucleic acid sequence, and the nucleic acid molecule may comprise one of: SEQ ID NO: 10 and SEQ ID NO: 14. The modification of SEQ ID NO: 2 may comprise the modification which promotes binding of the protein encoded by the nucleic acid sequence to the solid-state matrix, and the nucleic acid molecule may comprise one of: SEQ ID NO: 30, SEQ ID NO: 32 and SEQ ID NO: 36. The modification of SEQ ID NO: 2 may comprise the modification which extends the N-terminus of the cleaved portion of the protein encoded by the nucleic acid sequence, and the nucleic acid molecule may comprise SEQ ID NO: 34. The modification of SEQ ID NO: 2 may comprise a deletion of an amino acid residue of at least one of the C-terminal region and the N-terminal region of the protein encoded by the nucleic acid sequence, and the nucleic acid molecule may comprise one of: SEQ ID NO: 4, SEQ ID NO: 6 and SEQ ID NO: 8. The modification of SEQ ID NO: 2 may comprise a modification to produce a modification in one or more of: a wing loop residue, a tunnel loop residue, an upper internal surface residue, a lower internal surface residue, an external surface residue, a C-terminal end residue, and an N-terminal end residue; of a protein encoded by the nucleic acid sequence.

In another embodiment according to the invention, there is provided a nucleic acid molecule comprising a modification of a nucleic acid sequence of SEQ ID NO: 2, the nucleic acid molecule comprising one of: SEQ ID NO: 28; SEQ ID NO: 12; SEQ ID NO: 38; SEQ ID NO: 16, SEQ ID NO: 18, SEQ ID NO: 22, SEQ ID NO: 24, SEQ ID NO: 26, SEQ ID NO: 20, SEQ ID NO: 10, SEQ ID NO: 14, SEQ ID NO: 30, SEQ ID NO: 32, SEQ ID NO: 36, SEQ ID NO: 34, SEQ ID NO: 4, SEQ ID NO: 6, and SEQ ID NO: 8.

In another embodiment according to the invention, there is provided a nucleic acid molecule comprising a modification of a nucleic acid sequence of SEQ ID NO: 2, the nucleic acid molecule comprising one of: SEQ ID NO: 28; SEQ ID NO: 38; SEQ ID NO: 16, SEQ ID NO: 18, SEQ ID NO: 24, SEQ ID NO: 26, SEQ ID NO: 20, SEQ ID NO: 10, SEQ ID NO: 14, SEQ ID NO: 30, SEQ ID NO: 32, SEQ ID NO: 36, SEQ ID NO: 34, SEQ ID NO: 4, SEQ ID NO: 6, and SEQ ID NO: 8.

In another embodiment according to the invention, there is provided a method of sensing a biomolecule, the method comprising: applying a voltage to an electrolyte on both sides of a solid-state matrix, the solid-state matrix comprising a solid-state pore opening, and a hydrophilic protein channel in a stable insertion fit within the solid-state pore opening, the hydrophilic protein channel comprising a protein nanopore channel; and measuring a voltage change produced by passage of the biomolecule through the protein nanopore channel.

In further, related embodiments, the biomolecule may comprise one or more of: a protein, a nucleic acid, a biopolymer and an organic molecule. The biomolecule may comprise single-stranded DNA, double-stranded DNA or RNA. A protein forming at least part of the hydrophilic protein channel may comprise a modification of an amino acid sequence comprising SEQ ID NO: 1, the modification comprising one or more of: (i) a modification of an upper internal surface residue or a lower internal surface residue of the protein of the hydrophilic protein channel, the modification comprising an alteration of an electrostatic surface potential of an upper internal surface or a lower internal surface of SEQ ID NO: 1; (ii) a modification comprising an insertion of a cysteine residue into SEQ ID NO: 1 or a replacement of an amino acid residue of SEQ ID NO: 1 with a cysteine residue; (iii) a modification of a tunnel loop residue of the protein of the hydrophilic protein channel, the modification comprising an expansion of a narrowest constriction of a tunnel loop of SEQ ID NO: 1, a restriction of the narrowest constriction of the tunnel loop of SEQ ID NO: 1, or a removal of the narrowest constriction of the tunnel loop of SEQ ID NO: 1; (iv) a modification of a tunnel loop residue of the protein of the hydrophilic protein channel, the modification resulting in an alteration of an electrostatic charge property of a tunnel loop of SEQ ID NO: 1; (v) a modification which alters an external charge of the protein of the hydrophilic protein channel; (vi) a modification which promotes binding of the protein of the hydrophilic protein channel to the solid-state matrix; (vii) a modification which extends an N-terminus or a C-terminus of SEQ ID NO: 1 or an N-terminus or a C-terminus of a cleaved portion of SEQ ID NO: 1; and (viii) a deletion of an amino acid residue of at least one of a C-terminal region and an N-terminal region of SEQ ID NO: 1. The modification of SEQ ID NO: 1 may comprise a modification of a lower internal surface residue of the protein of the hydrophilic protein channel, the modification comprising the alteration of the electrostatic surface potential of the lower internal surface of SEQ ID NO: 1, and the protein of the hydrophilic protein channel may comprise SEQ ID NO: 27. The modification of SEQ ID NO: 1 may comprise a replacement of an amino acid residue of SEQ ID NO: 1 with a cysteine residue, and the protein of the hydrophilic protein channel may comprise SEQ ID NO: 11 or SEQ ID NO: 37. The modification of SEQ ID NO: 1 may comprise a modification of the tunnel loop residue of the protein of the hydrophilic protein channel, the modification comprising the expansion of the narrowest constriction of the tunnel loop of SEQ ID NO: 1, the restriction of the narrowest constriction of the tunnel loop of SEQ ID NO: 1, or the removal of the narrowest constriction of the tunnel loop of SEQ ID NO: 1, and the protein of the hydrophilic protein channel may comprise one of: SEQ ID NO: 15, SEQ ID NO: 17, SEQ ID NO: 21, SEQ ID NO: 23, and SEQ ID NO: 25. The modification of SEQ ID NO: 1 may comprise the modification in a tunnel loop residue of the protein of the hydrophilic protein channel, the modification resulting in the alteration of the electrostatic charge property of the tunnel loop of SEQ ID NO: 1, and the protein of the hydrophilic protein channel may comprise SEQ ID NO: 19. The modification of SEQ ID NO: 1 may comprise the modification which alters an external charge of the hydrophilic protein channel, and the protein of the hydrophilic protein channel may comprise one of: SEQ ID NO: 9 and SEQ ID NO: 13. The modification of SEQ ID NO: 1 may comprise the modification which promotes binding of the protein of the hydrophilic protein channel to the solid-state matrix, and the protein of the hydrophilic protein channel may comprises one of: SEQ ID NO: 29, SEQ ID NO: 31 and SEQ ID NO: 35. The modification of SEQ ID NO: 1 may comprise the modification to extend the N-terminal of the cleaved portion of SEQ ID NO: 1, and the protein of the hydrophilic protein channel may comprise SEQ ID NO: 33. The modification of SEQ ID NO: 1 may comprise the deletion of an amino acid residue of at least one of the C-terminal region and the N-terminal region of SEQ ID NO: 1, and the modification may comprise a modification of one of: SEQ ID NO: 3, SEQ ID NO: 5 and SEQ ID NO: 7. The protein of the hydrophilic protein channel may comprise a modification of SEQ ID NO: 1, the modification comprising a modification in one or more of: a wing loop residue, a tunnel loop residue, an upper internal surface residue, a lower internal surface residue, an external surface residue, a C-terminal residue, and an N-terminal residue. A protein forming at least part of the hydrophilic protein channel may comprise a modification of SEQ ID NO: 1, the modification comprising one or more of: a modification of residue 328 of SEQ ID NO: 1, a modification of residue 189 of SEQ ID NO: 1, and a modification of residue 367 of SEQ ID NO: 1. A protein forming at least part of the hydrophilic protein channel may comprise a modification of SEQ ID NO: 1, the modification comprising one of: SEQ ID NO: 27; SEQ ID NO: 11; SEQ ID NO: 37; SEQ ID NO: 15, SEQ ID NO: 17, SEQ ID NO: 21, SEQ ID NO: 23, SEQ ID NO: 25, SEQ ID NO: 19, SEQ ID NO: 9, SEQ ID NO: 13, SEQ ID NO: 29, SEQ ID NO: 31, SEQ ID NO: 35, SEQ ID NO: 33, SEQ ID NO: 3, SEQ ID NO: 5, and SEQ ID NO: 7. A protein forming at least part of the hydrophilic protein channel may comprise a modification of SEQ ID NO: 1, the modification comprising one of: SEQ ID NO: 27; SEQ ID NO: 37; SEQ ID NO: 15, SEQ ID NO: 17, SEQ ID NO: 23, SEQ ID NO: 25, SEQ ID NO: 19, SEQ ID NO: 9, SEQ ID NO: 13, SEQ ID NO: 29, SEQ ID NO: 31, SEQ ID NO: 35, SEQ ID NO: 33, SEQ ID NO: 3, SEQ ID NO: 5, and SEQ ID NO: 7.

In other, related embodiments, the solid-state matrix may comprise at least one of: silicon, hafnium and nickel. The solid-state matrix may comprise at least one of: a silicon containing nitride, a silicon containing carbide and a silicon containing oxide. The solid-state matrix may comprise a thickness of less than about 30 nm. The solid-state pore opening may comprise a diameter of between about 5.4 nm and about 6 nm. The solid-state matrix may comprise a coating to promote binding of the solid-state matrix to a protein forming at least part of the hydrophilic protein channel. The coating may comprise a thiol-coupling compound. The coating may comprise a maleimide compound.

In another embodiment according to the invention, there is provided a method of manufacturing a sensor, the method comprising: applying at least one of a voltage and a pressure to an electrolyte solution on both sides of a solid-state matrix comprising a solid-state pore opening, the electrolyte solution comprising a hydrophilic protein; and as a result of the applying of the at least one of the voltage and the pressure, forming a stable insertion fit of a hydrophilic protein channel comprising the hydrophilic protein within the solid-state pore opening, the hydrophilic protein channel comprising a protein nanopore channel.

In further, related embodiments, the hydrophilic protein of the hydrophilic protein channel may comprise a modification of an amino acid sequence comprising SEQ ID NO: 1, the modification comprising one or more of: (i) a modification of an upper internal surface residue or a lower internal surface residue of the protein of the hydrophilic protein channel, the modification comprising an alteration of an electrostatic surface potential of an upper internal surface or a lower internal surface of SEQ ID NO: 1; (ii) a modification comprising an insertion of a cysteine residue into SEQ ID NO: 1 or a replacement of an amino acid residue of SEQ ID NO: 1 with a cysteine residue; (iii) a modification of a tunnel loop residue of the protein of the hydrophilic protein channel, the modification comprising an expansion of a narrowest constriction of a tunnel loop of SEQ ID NO: 1, a restriction of the narrowest constriction of the tunnel loop of SEQ ID NO: 1, or a removal of the narrowest constriction of the tunnel loop of SEQ ID NO: 1; (iv) a modification of a tunnel loop residue of the protein of the hydrophilic protein channel, the modification resulting in an alteration of an electrostatic charge property of a tunnel loop of SEQ ID NO: 1; (v) a modification which alters an external charge of the protein of the hydrophilic protein channel; (vi) a modification which promotes binding of the protein of the hydrophilic protein channel to the solid-state matrix; (vii) a modification which extends an N-terminus or a C-terminus of SEQ ID NO: 1 or an N-terminus or a C-terminus of a cleaved portion of SEQ ID NO: 1; and (viii) a deletion of an amino acid residue of at least one of a C-terminal region and an N-terminal region of SEQ ID NO: 1. The modification of SEQ ID NO: 1 may comprise a modification of the lower internal surface residue of the protein of the hydrophilic protein channel, the modification comprising the alteration of the electrostatic surface potential of the lower internal surface of SEQ ID NO: 1, and the protein of the hydrophilic protein channel may comprise SEQ ID NO: 27. The modification of SEQ ID NO: 1 may comprise a replacement of an amino acid residue of SEQ ID NO: 1 with a cysteine residue, and the protein of the hydrophilic protein channel may comprise SEQ ID NO: 11 or SEQ ID NO: 37. The modification of SEQ ID NO: 1 may comprise a modification of the tunnel loop residue of the protein of the hydrophilic protein channel, the modification comprising the expansion of the narrowest constriction of the tunnel loop of SEQ ID NO: 1, the restriction of the narrowest constriction of the tunnel loop of SEQ ID NO: 1, or the removal of the narrowest constriction of the tunnel loop of SEQ ID NO: 1, and the protein of the hydrophilic protein channel may comprise one of: SEQ ID NO: 15, SEQ ID NO: 17, SEQ ID NO: 21, SEQ ID NO: 23, and SEQ ID NO: 25. The modification of SEQ ID NO: 1 may comprise the modification in a tunnel loop residue of the protein of the hydrophilic protein channel, the modification resulting in the alteration of the electrostatic charge property of the tunnel loop of SEQ ID NO: 1, and the protein of the hydrophilic protein channel may comprise SEQ ID NO: 19. The modification of SEQ ID NO: 1 may comprise the modification which alters the external charge of the hydrophilic protein channel, and the protein of the hydrophilic protein channel may comprise one of: SEQ ID NO: 9 and SEQ ID NO: 13. The modification of SEQ ID NO: 1 may comprise the modification which promotes binding of the protein of the hydrophilic protein channel to the solid-state matrix, and the protein of the hydrophilic protein channel may comprise one of: SEQ ID NO: 29, SEQ ID NO: 31 and SEQ ID NO: 35. The modification of SEQ ID NO: 1 may comprise the modification which extends the N-terminus of the cleaved portion of SEQ ID NO: 1, and the protein of the hydrophilic protein channel may comprise SEQ ID NO: 33. The modification of SEQ ID NO: 1 may comprise the deletion of an amino acid residue of at least one of the C-terminal region and the N-terminal region of SEQ ID NO: 1, and the modification may comprise a modification of one of: SEQ ID NO: 3, SEQ ID NO: 5 and SEQ ID NO: 7. The protein of the hydrophilic protein channel may comprise a modification of SEQ ID NO: 1, the modification comprising a modification in one or more of: a wing loop residue, a tunnel loop residue, an upper internal surface residue, a lower internal surface residue, an external surface residue, a C-terminal residue, and an N-terminal residue. The protein of the hydrophilic protein channel may comprise a modification of SEQ ID NO: 1, the modification comprising one or more of: a modification of residue 328 of SEQ ID NO: 1, a modification of residue 189 of SEQ ID NO: 1, and a modification of residue 367 of SEQ ID NO: 1. A protein forming at least part of the hydrophilic protein channel may comprise a modification of SEQ ID NO: 1, the modification comprising one of: SEQ ID NO: 27; SEQ ID NO: 11; SEQ ID NO: 37; SEQ ID NO: 15, SEQ ID NO: 17, SEQ ID NO: 21, SEQ ID NO: 23, SEQ ID NO: 25, SEQ ID NO: 19, SEQ ID NO: 9, SEQ ID NO: 13, SEQ ID NO: 29, SEQ ID NO: 31, SEQ ID NO: 35, SEQ ID NO: 33, SEQ ID NO: 3, SEQ ID NO: 5, and SEQ ID NO: 7. A protein forming at least part of the hydrophilic protein channel may comprise a modification of SEQ ID NO: 1, the modification comprising one of: SEQ ID NO: 27; SEQ ID NO: 37; SEQ ID NO: 15, SEQ ID NO: 17, SEQ ID NO: 23, SEQ ID NO: 25, SEQ ID NO: 19, SEQ ID NO: 9, SEQ ID NO: 13, SEQ ID NO: 29, SEQ ID NO: 31, SEQ ID NO: 35, SEQ ID NO: 33, SEQ ID NO: 3, SEQ ID NO: 5, and SEQ ID NO: 7.

In other related embodiments, the solid-state matrix may comprise at least one of: silicon, hafnium and nickel. The solid-state matrix may comprise at least one of: a silicon containing nitride, a silicon containing carbide and a silicon containing oxide. The solid-state matrix may comprise a thickness of less than about 30 nm. The solid-state pore opening may comprise a diameter of between about 5.4 nm and about 6 nm. The method may further comprise coating the solid-state matrix to promote binding of the solid-state matrix to a protein forming at least part of the hydrophilic protein channel. Coating the solid-state matrix may comprise applying a thiol-coupling compound to the solid-state matrix. Coating the solid-state matrix may comprise applying a coating comprising a maleimide compound to the solid-state matrix.

In another embodiment according to the invention, there is provided a protein variant encoded by a modification of a nucleic acid sequence of SEQ ID NO: 2, the modification comprising one or more of: (i) a modification in a portion of the nucleic acid sequence that encodes an upper internal surface residue or a lower internal surface residue of the protein variant encoded by the nucleic acid sequence, the modification producing an alteration of an electrostatic surface potential of an upper internal surface or a lower internal surface of the protein variant encoded by the nucleic acid sequence; (ii) a modification comprising an insertion of a cysteine residue into the protein variant encoded by the nucleic acid sequence or a replacement of an amino acid residue of the protein variant encoded by the nucleic acid sequence with a cysteine residue; (iii) a modification to produce a modification of a tunnel loop residue of the protein variant encoded by the nucleic acid sequence, the modification in the tunnel loop residue comprising an expansion of a narrowest constriction of a tunnel loop of the protein variant encoded by the nucleic acid sequence, a restriction of the narrowest constriction of the tunnel loop of the protein variant encoded by the nucleic acid sequence, or a removal of the narrowest constriction of the tunnel loop of the protein variant encoded by the nucleic acid sequence; (iv) a modification to produce a modification of a tunnel loop residue of the protein variant encoded by the nucleic acid sequence, the modification resulting in an alteration of an electrostatic charge property of a tunnel loop of the protein variant encoded by the nucleic acid sequence; (v) a modification which alters an external charge of the protein variant encoded by the nucleic acid sequence; (vi) a modification which promotes binding of the protein variant encoded by the nucleic acid sequence to a solid-state matrix; (vii) a modification which extends an N-terminus or a C-terminus of the protein variant encoded by the nucleic acid sequence or an N-terminus or a C-terminus of a cleaved portion of the protein variant encoded by the nucleic acid sequence; and (viii) a deletion of an amino acid residue of at least one of a C-terminal region and an N-terminal region of the protein variant encoded by the nucleic acid sequence.

In further, related embodiments, the modification of SEQ ID NO: 2 may comprise a modification in a portion of the nucleic acid sequence that encodes the lower internal surface residue of the protein variant encoded by the nucleic acid sequence, the modification producing the alteration of the electrostatic surface potential of the lower internal surface of the protein variant encoded by the nucleic acid sequence, and the modification of the nucleic acid sequence may comprise SEQ ID NO: 28. The modification of SEQ ID NO: 2 may comprise a replacement of an amino acid residue of the protein variant encoded by the nucleic acid sequence with the cysteine residue, and the modification of the nucleic acid sequence may comprise SEQ ID NO: 12 or SEQ ID NO: 38. The modification of SEQ ID NO: 2 may comprise the modification to produce a modification of a tunnel loop residue of the protein variant encoded by the nucleic acid sequence, the modification in the tunnel loop residue comprising the expansion of the narrowest constriction of the tunnel loop of the protein variant encoded by the nucleic acid sequence, the restriction of the narrowest constriction of the tunnel loop of the protein variant encoded by the nucleic acid sequence, or the removal of the narrowest constriction of the tunnel loop of the protein variant encoded by the nucleic acid sequence, and the modification of the nucleic acid sequence may comprise one of: SEQ ID NO: 16, SEQ ID NO: 18, SEQ ID NO: 22, SEQ ID NO: 24, and SEQ ID NO: 26. The modification of SEQ ID NO: 2 may comprise the modification to produce a modification of a tunnel loop residue of the protein variant encoded by the nucleic acid sequence, the modification resulting in an alteration of an electrostatic charge property of a tunnel loop of the protein variant encoded by the nucleic acid sequence, and the modification of the nucleic acid sequence may comprise SEQ ID NO: 20. The modification of SEQ ID NO: 2 may comprise the modification which alters the external charge of the protein variant encoded by the nucleic acid sequence, and the modification of the nucleic acid sequence may comprise one of: SEQ ID NO: 10 and SEQ ID NO: 14. The modification of SEQ ID NO: 2 may comprise the modification which promotes binding of the protein variant encoded by the nucleic acid sequence to the solid-state matrix, and the modification of the nucleic acid sequence may comprise one of: SEQ ID NO: 30, SEQ ID NO: 32 and SEQ ID NO: 36. The modification of SEQ ID NO: 2 may comprise the modification which extends the N-terminus of the cleaved portion of the protein variant encoded by the nucleic acid sequence, and the modification of the nucleic acid sequence may comprise SEQ ID NO: 34. The modification of SEQ ID NO: 2 may comprise a deletion of an amino acid residue of at least one of a C-terminal region and an N-terminal region of the protein variant encoded by the nucleic acid sequence, and the modification of the nucleic acid sequence may comprise one of: SEQ ID NO: 4, SEQ ID NO: 6 and SEQ ID NO: 8. The modification of SEQ ID NO: 2 may comprise a modification to produce a modification in one or more of: a wing loop residue, a tunnel loop residue, an upper internal surface residue, a lower internal surface residue, an external surface residue, a C-terminal residue, and an N-terminal residue; of the protein variant encoded by the modification of the nucleic acid sequence.

In another embodiment according to the invention, there is provided a protein variant encoded by a modification of a nucleic acid sequence of SEQ ID NO: 2, the modification comprising one or more of: SEQ ID NO: 28; SEQ ID NO: 12; SEQ ID NO: 38; SEQ ID NO: 16, SEQ ID NO: 18, SEQ ID NO: 22, SEQ ID NO: 24, SEQ ID NO: 26, SEQ ID NO: 20, SEQ ID NO: 10, SEQ ID NO: 14, SEQ ID NO: 30, SEQ ID NO: 32, SEQ ID NO: 36, SEQ ID NO: 34, SEQ ID NO: 4, SEQ ID NO: 6, and SEQ ID NO: 8.

In another embodiment according to the invention, there is provided a protein variant encoded by a modification of a nucleic acid sequence of SEQ ID NO: 2, the modification comprising one or more of: SEQ ID NO: 28; SEQ ID NO: 38; SEQ ID NO: 16, SEQ ID NO: 18, SEQ ID NO: 24, SEQ ID NO: 26, SEQ ID NO: 20, SEQ ID NO: 10, SEQ ID NO: 14, SEQ ID NO: 30, SEQ ID NO: 32, SEQ ID NO: 36, SEQ ID NO: 34, SEQ ID NO: 4, SEQ ID NO: 6, and SEQ ID NO: 8.

In another embodiment according to the invention, there is provided a vector comprising a modification of a nucleic acid sequence of SEQ ID NO: 2, the modification comprising one or more of: (i) a modification in a portion of the nucleic acid sequence that encodes an upper internal surface residue or a lower internal surface residue of a protein encoded by the nucleic acid sequence, the modification producing an alteration of an electrostatic surface potential of an upper internal surface or a lower internal surface of the protein encoded by the nucleic acid sequence; (ii) a modification comprising an insertion of a cysteine residue into a protein encoded by the nucleic acid sequence or a replacement of an amino acid residue of a protein encoded by the nucleic acid sequence with a cysteine residue; (iii) a modification to produce a modification of a tunnel loop residue of a protein encoded by the nucleic acid sequence, the modification of the tunnel loop residue comprising an expansion of a narrowest constriction of a tunnel loop of the protein encoded by the nucleic acid sequence, a restriction of the narrowest constriction of the tunnel loop of the protein encoded by the nucleic acid sequence, or a removal of the narrowest constriction of the tunnel loop of the protein encoded by the nucleic acid sequence; (iv) a modification to produce a modification of a tunnel loop residue of a protein encoded by the nucleic acid sequence, the modification resulting in an alteration of an electrostatic charge property of a tunnel loop of the protein encoded by the nucleic acid sequence; (v) a modification which alters an external charge of a protein encoded by the nucleic acid sequence; (vi) a modification which promotes binding of a protein encoded by the nucleic acid sequence to a solid-state matrix; (vii) a modification which extends an N-terminus or a C-terminus of a protein encoded by the nucleic acid sequence or an N-terminus or a C-terminus of a cleaved portion of the protein encoded by the nucleic acid sequence; and (viii) a deletion of an amino acid residue of at least one of a C-terminal region and an N-terminal region of a protein encoded by the nucleic acid sequence.

In further related embodiments, the modification of SEQ ID NO: 2 may comprise a modification in the portion of the nucleic acid sequence that encodes the lower internal surface residue of the protein encoded by the nucleic acid sequence, the modification producing the alteration of the electrostatic surface potential of the lower internal surface of the protein encoded by the nucleic acid sequence, and the modification of the nucleic acid sequence may comprise SEQ ID NO: 28. The modification of SEQ ID NO: 2 may comprise a modification comprising an insertion of a cysteine residue into the protein encoded by the nucleic acid sequence or a replacement of an amino acid residue of a protein encoded by the nucleic acid sequence with the cysteine residue, and the modification of the nucleic acid sequence may comprise one of: SEQ ID NO: 12 and SEQ ID NO: 38. The modification of SEQ ID NO: 2 may comprise a modification to produce a modification of a tunnel loop residue of the protein encoded by the nucleic acid sequence, the modification in the tunnel loop residue comprising the expansion of the narrowest constriction of the tunnel loop of the protein encoded by the nucleic acid sequence, the restriction of the narrowest constriction of the tunnel loop of the protein encoded by the nucleic acid sequence, or the removal of the narrowest constriction of the tunnel loop of the protein encoded by the nucleic acid sequence, and the modification of the nucleic acid sequence may comprise one of: SEQ ID NO: 16, SEQ ID NO: 18, SEQ ID NO: 22, SEQ ID NO: 24, and SEQ ID NO: 26. The modification of SEQ ID NO: 2 may comprise a modification to produce a modification of the tunnel loop residue of the protein encoded by the nucleic acid sequence, the modification resulting in the alteration of the electrostatic charge property of the tunnel loop of the protein encoded by the nucleic acid sequence, and the modification of the nucleic acid sequence may comprise SEQ ID NO: 20. The modification of SEQ ID NO: 2 may comprise the modification which alters the external charge of the protein encoded by the nucleic acid sequence, and the modification of the nucleic acid sequence may comprise one of: SEQ ID NO: 10 and SEQ ID NO: 14. The modification of SEQ ID NO: 2 may comprise the modification which promotes binding of the protein encoded by the nucleic acid sequence to the solid-state matrix, and the modification of the nucleic acid sequence may comprise one of: SEQ ID NO: 30, SEQ ID NO: 32 and SEQ ID NO: 36. The modification of SEQ ID NO: 2 may comprise the modification which extends the N-terminus of the cleaved portion of the protein encoded by the nucleic acid sequence, and the modification of the nucleic acid sequence may comprise SEQ ID NO: 34. The modification of SEQ ID NO: 2 may comprise the deletion of the amino acid residue of at least one of the C-terminal region and the N-terminal region of the protein encoded by the nucleic acid sequence, and the modification of the nucleic acid sequence may comprise one of: SEQ ID NO: 4, SEQ ID NO: 6 and SEQ ID NO: 8. The modification of SEQ ID NO: 2 may comprise a modification to produce a modification in one or more of: a wing loop residue, a tunnel loop residue, an upper internal surface residue, a lower internal surface residue, an external surface residue, a C-terminal residue, and an N-terminal residue; of a protein encoded by the modification of the nucleic acid sequence.

In another embodiment according to the invention, there is provided a vector comprising a modification of a nucleic acid sequence of SEQ ID NO: 2, the modification comprising one or more of: SEQ ID NO: 28; SEQ ID NO: 12; SEQ ID NO: 38; SEQ ID NO: 16, SEQ ID NO: 18, SEQ ID NO: 22, SEQ ID NO: 24, SEQ ID NO: 26, SEQ ID NO: 20, SEQ ID NO: 10, SEQ ID NO: 14, SEQ ID NO: 30, SEQ ID NO: 32, SEQ ID NO: 36, SEQ ID NO: 34, SEQ ID NO: 4, SEQ ID NO: 6, and SEQ ID NO: 8.

In another embodiment according to the invention, there is provided a vector comprising a modification of a nucleic acid sequence of SEQ ID NO: 2, the modification comprising one or more of: SEQ ID NO: 28; SEQ ID NO: 38; SEQ ID NO: 16, SEQ ID NO: 18, SEQ ID NO: 24, SEQ ID NO: 26, SEQ ID NO: 20, SEQ ID NO: 10, SEQ ID NO: 14, SEQ ID NO: 30, SEQ ID NO: 32, SEQ ID NO: 36, SEQ ID NO: 34, SEQ ID NO: 4, SEQ ID NO: 6, and SEQ ID NO: 8.

In other embodiments, there is provided a cell comprising any of the vectors taught herein.

In another embodiment according to the invention, there is provided a nucleic acid sequence encoding any of the protein variants taught herein.

In further, related embodiments, the cell may be any of the vectors taught in the foregoing paragraphs.

## DETAILED DESCRIPTION

A description of example embodiments follows.

Hybrid nanopores in accordance with an embodiment of the invention, comprising a protein pore supported within a solid-state membrane, combine the robust nature of solid-state membranes with the easily tunable and precise engineering of protein nanopores. A lipid-free hybrid nanopore comprises a water soluble and stable, modified portal protein of the Thermus thermophilus bacteriophage G20c, electrokinetically inserted into a larger nanopore in a solid-state membrane. The hybrid pore is stable and easy to fabricate, and exhibits low peripheral leakage, allowing sensing and discrimination among different types of biomolecules.

The protein channel in accordance with embodiments of the invention can either voltage- or pressure-insert into the solid-state nanopore matrix to form the hybrid nanopore sensor device. The signal for sensing using this device can be either electrical or optical, the latter offering high-density parallelized readout from multiple adjacent pores. Embodiments include mechanisms to obtain the hybrid structure, to stabilize it, and to modify it so that different types of biomolecules can be sensed.

In embodiments, the hybrid sensor does not require any lipid support, which is typically fragile and not durable; it allows atomic-precision engineering to chemically define the pore sensor properties; and chemical methods of stabilizing the portal-to-solid-state interface are controlled by biomolecular engineering and materials science approaches. The hybrid sensor can, for example, provide the advantages of: rapid and stable insertion of a protein into a solid-state nanopore; mutations of the protein can be used for sensing improvement; and translocation of biopolymers (such as nucleic acids and polypeptides) through the hybrid sensor can be performed for sensing applications. Example potential merits of such a device are in applications that include: 1) high-resolution mapping of DNA, RNA sequencing, DNA sequencing; 2) protein identification, protein conformational change monitoring; 3) polypeptide sequencing; 4) small-molecule detection, biomolecular complex detection, and enzyme-ligand binding. The broad range of uses could potentially impact many areas of the human health, biotechnology and agri-food sectors.

The advent of single-molecule detection is having an unparalleled impact on the speed with which structural and dynamic aspects of molecules can be probed (1). In this regard, nanopores have shown much promise as electrical (2-7) and electro-optical sensors (8-10) and several nanopore-based systems are now being adopted as primary tools for DNA (11-13) and RNA (14) sequencing.

Despite recent progress, identification and quantification of molecular species in solution (15-28) requires a reproducible nanopore platform that affords physical stability, structural precision, and often, a spatially-defined pore position (for example, in electro-optical sensing). While synthetic nanopores fabricated in solid-state (SS) membranes offer physical robustness (29-31), pore-to-pore variability often limits the reproducibility of experiments, necessitating additional control checks and validation. On the contrary, protein channels embedded in organic thin membranes (e.g., a lipid-bilayer) offer the highest reproducibility due to the precise folding and repetitive nature of the constituting multi-subunit protein oligomers (32,33), but their supporting membrane is typically less chemically and physically robust, and further, the pore position is not well-defined due to in-plane diffusion of the protein channel (34). Hybrid nanopore devices, in which channel-containing proteins are embedded in larger pores made in a SS matrix, have been proposed as a strategic solution for combining the benefits—while overcoming the limitations—of existing nanopores (35). Although initial experiments based on inserting pore-containing proteins with lipophilic regions into a SS pore looked promising (35), challenges in inserting such proteins into a SS pore and in controlling the protein orientation have remained major obstacles in the applicability of hybrid nanopores to nanotechnology.

An embodiment according to the present invention provides a hybrid nanopore based on the hydrophilic portal protein derived from a thermostable virus, the Thermus thermophilus bacteriophage G20c (36). In double-stranded DNA viruses, the portal protein is incorporated into the capsid shell (see, e.g., FIG. 1), thereby serving as a natural pore through which DNA is moved in and out (37). FIG. 1 is a schematic diagram of the DNA packaging machine of a dsDNA virus. Viral genomic DNA 100 is translocated into the preformed virus capsid by the packaging ATPase 102 through the portal protein 104 embedded in the viral capsid 106. The protein contains a tight tunnel constriction with a repetitive chemical character, being made up by a circle of identical “tunnel loops”, contributed by 12 subunits (38).

In an embodiment according to the invention, this protein is engineered to reprogram its physico-chemical and electrostatic properties. For example, in one protein version, CGG, (33) a portal with a larger minimum aperture of ˜2.3 nm is defined by two residues in the tunnel loops substituted to glycines; and in another protein version, CD/N, the internal surface charges are electrostatically engineered by replacing aspartic acid (D) residues with asparagines (N). The latter CD/N mutation had a major impact on the charge of the internal tunnel's surface, (see FIG. 2) and permitted electrical sensing of biomolecules. FIG. 2 is a schematic diagram showing electrostatic properties of the tunnel in wild-type (left) and mutant (right) portal proteins, where the mutant portal protein is the CD/N mutant of the G20c bacteriophage in accordance with an embodiment of the invention. FIG. 2 shows a slice through the middle of molecular surface colored according to charge from red (−1 kT/e) to blue (+1 kT/e). In another example of this portal system, a cysteine substitution is made in an externally facing residue 49 (designated “C”) which allows chemical labeling and surface immobilization of the portal protein. (33)

An embodiment uses this structurally programmable portal protein as a nanoscale adapter by electrokinetically embedding it snugly inside a larger pore made in a freestanding silicon nitride (SiN), or other solid-state, membrane (see FIGS. 4 and 5). Electrokinetic “corking” occurs when the force on the protein, induced by applied voltage, is sufficient to “squeeze” the portal into the SS pore. It is found that, for stable insertion, a diameter of the solid-state nanopore of from 5.4 to 6 nm and a nominal membrane thickness of 30 nm, work well. Given the dimensions of the portal assembly (33) (see FIG. 3), the geometric constraints set by the SS pore restrict the range of possible orientations of the portal pore in it, such that the stem is inserted within the SS nanopore constriction, and the wider “cap” self-orients towards the top of the trans chamber (see FIG. 5). FIG. 3 is a schematic diagram showing dimensions of the portal protein (left) and the SS nanopore (right), in accordance with an embodiment of the invention. The portal protein is, for example, about 14.5 nm wide at its top, “cap” end, and about 8.5 nm wide at its narrower base end. The solid-state nanopore is, for example, between about 5.5 and about 6 nm wide, and about 30 nm in thickness (across the membrane). The larger size of the “cap”, as compared with the SS pore diameter, prevents the entire protein from moving through the SS nanopore. Remarkably, interactions between the portal protein squeezed into the SS pore and the SS-pore surface contribute to a stable, self-inserting and self-aligning hybrid (see FIG. 5) that exhibits tolerable peripheral ion leakage, probed using cyclodextrin as a pore current modulator. FIG. 4 is a schematic diagram illustrating insertion of the purified portal protein, which assembles its dodecameric units to form a hydrophilic protein channel 408, into a nanopore solid-state pore opening 410 drilled into a thin solid-state (SS) matrix membrane 412, in accordance with an embodiment of the invention. Portal protein is applied to the trans chamber 415 of a SS nanopore device containing an electrolyte solution of 20 mM Tris pH 7.5, 0.5 M NaCl. The protein electrokinetically inserts into the SS pore during application of a positive voltage by voltage source 414. FIG. 5 is a schematic diagram of the hybrid nanopore, in accordance with an embodiment of the invention, in which application of voltage results in ion current 516 through the pore, as well as leakage current 518 that is peripheral to the pore. The hybrid nanopore sensor includes, with reference to both FIGS. 4 and 5, the voltage source 414, the electrolyte (not shown) in both the trans chamber 415 and cis chamber 417 (see FIG. 4) on both sides of the solid-state matrix 412. With reference to FIG. 5, the hybrid nanopore sensor includes: the hydrophilic protein channel 508, formed from the assembled dodecameric units of hydrophilic portal protein monomers; the solid-state matrix 512 with the solid-state pore opening 510 formed therein, where the solid-state pore opening 510 is a nanopore; and the protein nanopore channel 520 through the middle of the hydrophilic protein channel 508. The hydrophilic protein channel 508 forms a stable insertion fit 522 within the solid-state pore opening 510.

FIG. 13 is a schematic diagram showing component amino acid residue regions of the portal protein of the Thermus thermophilus bacteriophage G20c, which can be modified, in embodiments according to the invention, to promote assembly and operation of, and sensing with, the hybrid sensor. In FIG. 13, two monomers 1324a and 1324b are shown (left and right are reflected version of each other), but it will be appreciated that twelve such monomers assemble to form the full dodecameric assembly that can create a protein channel in accordance with an embodiment of the invention. As shown in FIG. 13, the regions include: the N-terminal region 1326, which includes amino acid residue 29; the wing loop region 1328, which includes amino acid residues 36-41 and 46-54; the external surface residues 1330, which include amino acid residue 230; the internal surface residues 1332 of the lower tunnel region, which include amino acid residues 273-303; the tunnel loop region 1334, which includes amino acid residues 316-335, and forms a narrowest constriction 1340 where the tunnel loops are closest to each other; the internal surface residue 1336 of the upper tunnel region, which include amino acid residues 381-400; and the C-terminal region 1338, which includes amino acid residues 435-436.

FIG. 14 is a schematic diagram showing an example of residues that can be mutated in one example mutant, the CD/N mutant, in accordance with an embodiment of the invention. Here, residues ASP 281, ASP 286, ASP 289 and ASP 296 are mutated from aspartic acid (D) to asparagine (N).

The G20c portal protein is a stable circular dodecameric assembly of 12 monomers with a central tunnel of defined geometry and physicochemical properties. In accordance with some embodiments, a base scaffold of the protein is used that is trimmed to the core structure comprising residues 25-438 of the portal protein's amino acid sequence, where the unstructured N- and C-terminal regions (residues 1-24 and 439-448 of the wild type residues 1-448 of the protein) have been removed to improve stability. A major advantage of this scaffold is the absence of cysteine residues, which allows flexible design of cysteine-containing mutants that can be chemically derivatized for different applications, such as attachment to surfaces or insertion into membranes.

In accordance with embodiments of the invention, the properties of the portal protein can be engineered for specific sensing and device integration applications by mutating the surface residues lining the tunnel, those on the outer surface of the portal protein assembly and residues that stabilize or otherwise alter either intramonomer or intermonomer contacts. Mutations can, for example, include, but are not limited to, the segments containing internal tunnel lining residues 273-303 of the lower tunnel region and 381-400 of the upper tunnel region; tunnel loop residues 316-335; and surface residues such as the wing loops 36-41 and 46-54 that are part of a range of residues 36-54.

Further modifications to the protein can, for example, include fusion of peptide sequences, protein domains or proteins to the N or C-terminus of the protein or into external loops that will confer properties for attachment or sensing of ligand binding events to different biomolecules, bacteria, cells viruses and/or chemical assemblies.

In some embodiments, portal protein variants comprise specifically placed cysteine residues for chemical attachment to surfaces, insertion into lipid bilayers and/or linking to additional sensor components, such as ligand binding aptamers. These include the 49C or C mutant where a cysteine has been introduced into one of the wing loops at position 49; and the D400C variant where a cysteine has been placed at the top of the upper tunnel helix. Mutant proteins are referred to herein by the “Protein Version” names given in Tables 1-7, below. The 49C version has the amino acid sequence given in SEQ ID NO: 11 and the DNA sequence given in SEQ ID NO: 12. The D400C version has the amino acid sequence given in SEQ ID NO: 37 and the DNA sequence given in SEQ ID NO: 38. The 49C version can be used, either alone or in combination with other mutations, such as CGG or CD/N. D400C has been found to form dodecamer sized assemblies when purified.

In other embodiments, the tunnel properties of the portal assembly can be engineered, for example by expanding the narrowest constriction of the tunnel loops of residues 316-335 (see FIG. 13) and by altering the electrostatic surface potential of the tunnel, as in CD/N. Additional examples of other variants in the tunnel loop geometry that assemble into the characteristic circular protein complexes include: V325M designed to further restrict the narrowest aperture of the tunnel; and the Loop2GG and CLoop3G mutants that entirely remove the central constriction in the tunnel. Further charge alterations to the tunnel include the placement of a flexible positive lysine residue at the narrowest point of the tunnel loops (1328K) and proposed additional substitution of aspartic acid residues 383 and 400 in the upper tunnel helix (residues 381-400) with asparagine residues to reduce the negative charge in the upper tunnel surface.

In further embodiments, the external surface charge properties of the protein can be altered by introducing charged amino acids in place of uncharged surface residues, such as with the V40E and L230E, which are, respectively, in the wing loop region (for V40E) and the external surface residue region (for L230E). Similar alterations in the external surface residues can be made to create a more hydrophobic external surface. Peptide sequences have been successfully introduced into the wing loop 46-54 to promote binding to silicon surfaces (in the SIN1, SIN 2 and SIN 4 mutations). SIN1 and SIN2 form circular assemblies.

In other embodiments, extensions to the N- and C-terminal can be used to introduce longer peptide sequences for targeted surface (metal or lipid) interaction or to introduce ligand binding domains for specific sensing applications. Such extensions include the hexahistidine affinity tag used for nickel binding and purification. Additional N-terminal extensions include the SIN3 silicon binding sequence.

In one embodiment, the wild type full length portal protein of G20c bacteriophage, comprising SEQ ID NO: 1, can be modified in the lower internal surface residue of the hydrophilic protein channel, to alter the electrostatic surface potential of the lower internal surface of SEQ ID NO: 1. For example, for such a purpose, the hydrophilic protein channel can comprise the CD/N variant with amino acid SEQ ID NO: 27, which has a corresponding modified DNA sequence of SEQ ID NO: 28.

In another embodiment, the wild type portal protein, comprising SEQ ID NO: 1 can be modified to replace the residue of SEQ ID NO: 1 with a cysteine residue. For example, for such a purpose, the hydrophilic protein channel can comprise the 49C variant with amino acid SEQ ID NO: 11, which has a corresponding modified DNA sequence of SEQ ID NO: 12, or can comprise the D400C variant with amino acid SEQ ID NO: 37, which has a corresponding modified DNA sequence of SEQ ID NO: 38.

In another embodiment, the Wild Type portal protein of SEQ ID NO: 1 can be modified in the tunnel loop residue of the hydrophilic protein channel, the modification comprising an expansion of the narrowest constriction of the tunnel loop of SEQ ID NO: 1, a restriction of the narrowest constriction of the tunnel loop of SEQ ID NO: 1, or a removal of the narrowest constriction of the tunnel loop of SEQ ID NO: 1. For such purposes, the hydrophilic protein channel can, for example, comprise one of the G, M, CGG, Loop2GG and 49CLoop3G variants, which have, respectively, the amino acid sequences of SEQ ID NO: 15, SEQ ID NO: 17, SEQ ID NO: 21, SEQ ID NO: 23, and SEQ ID NO: 25, and the corresponding modified DNA sequences of SEQ ID NO: 16, SEQ ID NO: 18, SEQ ID NO: 22, SEQ ID NO: 24, and SEQ ID NO: 26.

In another embodiment, the Wild Type portal protein of SEQ ID NO: 1 can be modified in the tunnel loop residue of the hydrophilic protein channel, the modification comprising an alteration of an electrostatic charge property of the tunnel loop of SEQ ID NO: 1. For this purpose, the hydrophilic protein channel can, for example, comprise the K variant, which has amino acid SEQ ID NO: 19 and corresponding modified DNA sequence of SEQ ID NO: 20.

In another embodiment, the Wild Type portal protein of SEQ ID NO: 1 can be modified to alter an external charge of the hydrophilic protein channel. For example, for such a purpose, the hydrophilic protein channel can comprise one of the L230E and 40E variants, which respectively have amino acid SEQ ID NO: 9 and SEQ ID NO: 13, and corresponding modified DNA sequences SEQ ID NO: 10 and SEQ ID NO: 14.

In another embodiment, the Wild Type portal protein of SEQ ID NO: 1 can be modified to promote binding of the hydrophilic protein channel to the solid-state matrix. For example, for such a purpose, the hydrophilic protein channel can comprise one of the SIN1, SIN2 and SIN 4 variants, which respectively have amino acid SEQ ID NO: 29, SEQ ID NO: 31 and SEQ ID NO: 35, which have corresponding modified DNA sequences of SEQ ID NO: 30, SEQ ID NO: 32 and SEQ ID NO: 36.

In another embodiment, the Wild Type portal protein of SEQ ID NO: 1 can be modified to extend the N-terminal of a cleaved portion of SEQ ID NO: 1. For example, for such a purpose, the hydrophilic protein channel can comprise the SIN3 variant, which has amino acid SEQ ID NO: 33, and corresponding modified DNA sequence SEQ ID NO: 34.

In another embodiment, the Wild Type portal protein of SEQ ID NO: 1 can be modified to cleave off the amino acid residue of at least one of the C-terminal region and the N-terminal region of SEQ ID NO: 1. For example, for such a purpose, the modification can comprise a modification of one of the WT 1-438 C-term, WT 1-438 3C prot and WT Nanopore variants, for which the amino acid sequences are respectively given by SEQ ID NO: 3, SEQ ID NO: 5 and SEQ ID NO: 7, and for which the corresponding modified DNA sequences are SEQ ID NO: 4, SEQ ID NO: 6 and SEQ ID NO: 8.

In another embodiment, the Wild Type portal protein of SEQ ID NO: 1 can be modified in one or more of: a wing loop residue (such as residues in the range 36-41 or 46-54), a tunnel loop residue (such as residues in the range 316-335), an upper internal surface residue (such as residues in the range 381-400), a lower internal surface residue (such as residues in the range 273-303), an external surface residue (for example, residue 230), a C-terminal end residue (for example, residues 435-436), and an N-terminal end residue (for example, residue 29).

In addition, attachment of the portal protein to the solid-state surface can, for example, be assisted using vapor deposition of maleimide/thiol-silane compounds; by silicon binding peptides, such as binders to either, or both, SiN or SiOx; or by thiocholesterol or other maleimide-lipid conjugated to L49C by thiol chemistry. The solid-state matrix may comprise a coating to promote binding of the solid-state matrix to the hydrophilic protein channel. For example, the coating may comprise a thiol-coupling compound; and may, for example, comprise a maleimide compound or other thiol-coupling compound.

Although some embodiments herein are discussed based on use of a modified portal protein of the Thermus thermophilus bacteriophage G20c, it should be appreciated that portal proteins from other bacteriophages (including bacteriophages of Thermus thermophilus, and other bacteriophages) can be used, including any hydrophilic portal protein that achieves performance of a similar function to the hydrophilic protein channel in a stable insertion fit within a solid-state pore opening, that is taught herein. In one example, the portal protein of the Thermus thermophilus bacteriophage P23-45, or modifications of that portal protein, can be used. In that regard, SEQ ID NO: 40, or a modified protein based on SEQ ID NO: 40, can be used; and a protein encoded by its DNA sequence, which is SEQ ID NO: 41, or a modified protein encoded by a modified DNA of SEQ ID NO: 41, can be used. In particular, one or more of three amino acid variations of SEQ ID NO:1 can be made, which results in SEQ ID NO: 40, or a modification thereof: at residue 328 of SEQ ID NO: 1, there is an I amino acid residue, which, if changed to V, becomes the amino acid residue 328 of SEQ ID NO: 40; at residue 189 of SEQ ID NO: 1, there is an S amino acid residue, which, if changed to an N amino acid residue, becomes the amino acid residue 189 of SEQ ID NO: 40; and at amino acid residue 367 of SEQ ID NO: 1, there is an S residue, which, if changed to a G amino acid residue, becomes amino acid residue 367 of SEQ ID NO: 40. Thus, a mutation at one or more of amino acid residue locations 328, 189 and/or 367 of SEQ ID NO: 1 can be used.

In addition, it should be noted that monomer protein units of proteins taught herein can be assembled to form the full portal protein that functions to form the stable insertion fit within a solid-state pore opening that is taught herein. For example, the portal protein of the Thermus thermophilus bacteriophage G20c forms a dodecameric structure, made of 12 monomer protein units, which together assemble to form the full “plug” protein (see FIG. 4) that forms a stable insertion fit within the solid-state pore opening. Thus, a “hydrophilic protein channel,” as used herein, can include more than one monomer of a protein, such as 12 monomer protein units assembled together to form the hydrophilic protein channel through the dodecameric combined protein structure assembled from the monomers of the protein.

### EXAMPLES

There will now be described a set of example experiments, conducted in accordance with an embodiment of the invention.

**Example 1**

FIGS. 6-10 are graphs showing characterization of hybrid pore formation, in experiments in accordance with an embodiment of the invention. FIG. 6 is a graph of a typical current profile over time recorded through a 5.5 nm SS pore at +100 mV. After injection of 0.1 nmol of portal protein, short current drops are detected, interpreted as portal collisions with the solid-state nanopore. FIG. 7 is a graph of a representative current vs time trace recorded for a 5.4 nm SS nanopore at +80 mV, showing stable insertion of a portal protein. FIG. 8 is a graph of current as a function of the applied voltage for a 5.5 nm SS pore recorded before (red markers, with a higher slope) and after insertion of a portal protein (purple, with a lower slope). FIG. 9 is a graph of current noise analysis of a 5.5 nm diameter solid-state nanopore before (red, top curve) and after insertion of a portal protein (purple, bottom curve). FIG. 10 is a graph of conductance of solid-state nanopore vs conductance of portal hybrid pore (n=32 for CD/N hybrids and n=15 for CGG hybrids). Experiments were performed in 0.5 M NaCl, 20 mM Tris pH 7.5.

In experiments in accordance with an embodiment of the invention, the hybrid nanopores exhibit lifetimes of hours, and similar ion current noise values to a lipid bilayer-supported portal protein nanopore (33) (see FIGS. 6-10). The electrical properties of the hybrid pore were characterized and applied to electrically detect different biomolecules. Using the hybrid portal with engineered internal pore properties (CD/N, see FIG. 2, right), it was demonstrated that a folded protein larger than the pore interior does not enter the hybrid portal, whereas homopolymeric single-stranded DNA (ssDNA), double-stranded DNA (dsDNA) that contains a single-stranded tail, and a peptide predicted to have a random coil conformation with a 10-amino acid α-helix at the C-terminus, can all be discriminated based on their distinct signal amplitudes, in a way that is commensurate with their molecular cross-section. The results indicate that the hybrid portal is a versatile sensor of various biopolymer types, which may, with further development, find uses in genomic mapping as well as polypeptide and oligonucleotide sequencing.

After confirming the base current of stable SS nanopores of the desired diameter, addition of the portal protein to the trans chamber results in reversible partial blockades of the ionic current (FIGS. 6 and 7). These short-lived events are interpreted as portal protein collisions with the SS nanopore without stable insertion, where the ion current is partially blocked as the protein approaches the SS pore, prior to movement away. These short-lived events were usually followed by long-lived events (FIG. 7), of comparable current blockade levels, events that were only observed in SS pores with diameters of 5.4 to 6 nm. The long-lived events are interpreted as stable insertion of a portal protein into the SS nanopore to form a hybrid nanopore. The average conductance of these hybrid pores was calculated (FIG. 10) to be 1.50±0.48 nS and 1.33±0.42 nS for the CD/N (from 32 hybrid nanopores) and the CGG (from 15 hybrid nanopores) variants, respectively. Such hybrid pores remain stable at both positive and negative voltages up to +120 mV and −80 mV (FIG. 8), however, application of an electric field with strengths greater than −80 mV generally resulted in release of the protein from the SS nanopore. These data are consistent with the protein insertion being electrokinetically driven.

Obtaining a sufficient increase in the signal-to-noise ratio is a major challenge for properly identifying transport events by nanopore sensing. Power spectral densities of the current noise for a SS nanopore before, and after, insertion of a portal protein (FIG. 9) showed that the 1/f noise at low frequencies decreased upon formation of the hybrid pore. This 1/f noise reduction is consistent with a reduced pore conductance, as well as an indicator of the reduced surface charge fluctuations that are hallmarks of silicon nitride surfaces (39). This, along with the observation that capacitance-dominated noise at high frequencies was comparable for both pores, suggests that no new source was introduced during hybrid pore formation. It was deduced that the observed variation in the open pore current for different hybrid nanopores was likely to be caused by differences in SS geometry and the associated leakage currents around the portal protein. It was attempted to measure the extent of ion leakage from the pore by measuring β-cyclodextrin interactions with the hybrid pore for the CGG mutant, a mutant that was previously embedded into a lipid membrane (33). The results show that β-cyclodextrin does not translocate the pore, in contrast for the same experiment conducted on the lipid-embedded version of the same portal protein. While this precludes an accurate measurement of the leakage, these results suggest that “corking” the protein into a snug SS nanopore slightly reduces the innermost pore constriction. The reproducible signals obtained from biomolecules, as well as the steady baselines of the hybrid, allow current blockades as low as ˜20 pA to be accurately measured. These data demonstrate that despite a low level of constant peripheral leakage, these hybrid pores are unique lipid-free protein-based pore sensors.

**Example 2**

The sensing capabilities of these hybrid nanopores were then investigated by analyzing the transport of a peptide, comprising residues 1-43 of the human TPX2 protein, as a function of applied voltage (FIGS. 11A-11C).

FIGS. 11A-11C are graphs showing dynamics of TPX2 peptide transport, in experiments in accordance with an embodiment of the invention. FIG. 11A is a graph showing a current vs time trace recorded through a hybrid pore at +30, +40 and +55 mV in the presence of 10.3 μM TPX2 peptide. FIG. 11B is a semi-log plot of the event frequency as a function of the applied voltage. The line is an exponential fit to the equation. FIG. 11C is a semi-log plot of the peptide dwell time as a function of the applied voltage. The lines in FIGS. 11B and 11C are exponential fits. Experiments were performed in 0.5 M NaCl, 20 mM Tris pH 7.5.

The TPX2 peptide is negatively charged at pH 7.5 (pI=3.7) and was added to the cis chamber, on the opposite side of the membrane to which the portal protein was introduced (see inset to FIG. 11B). Adjusting the applied voltage from +30 to +60 mV resulted in an increased baseline ion current through the hybrid pore, as well as the frequency of observed current blockades (FIG. 11A). Two kinds of current blockades associated with two different events were detected: bumping events, characterized by brief, low-level current blockades, arising from diffusion of the peptide close to the hybrid pore entrance; and translocation events, characterized by larger current blockades of longer duration. These two types of events are typically seen during translocation of DNA (40-42) and proteins (43-45) through protein channels. The inter-event time distribution is well fit by a single exponential equation. The entry frequency (FIG. 11B) of the peptide into the hybrid pore is described by a Van′t Hoff Arrhenius relationship (44,46), f=f0exp(V/V0), consistent with both translocation of DNA (40-42), proteins (43,47) and peptides (46, 48-51) through either α-hemolysin or aerolysin; and a significant entropic barrier for peptide entry into the pore. The dwell time distributions were well fit by a double-exponential equation, which are typically due to two types of processes, normally associated with short bumping and longer translocation events (16). It was found that the average frequency for both types of events increases exponentially (FIG. 11B), while the average dwell time for the long events decreased exponentially with the applied voltage (FIG. 11C). Based on prior work that employed the α-hemolysin and aerolysin nanopores, (43, 45) it was concluded that the long events represent transport of the peptide through the hybrid pore to the trans chamber.

**Example 3**

In order to further demonstrate the sensing capabilities of this hybrid pore, the transport of other biopolymers was further investigated: dsDNA that contains a ssDNA tail, ssDNA, a folded protein as well as the TPX2 peptide (FIGS. 12A-12E).

FIGS. 12A-12E are graphs illustrating sensing of different biopolymers using a hybrid nanopore, in experiments in accordance with an embodiment of the invention. Current vs time trace recorded through the hybrid pore at +60 mV in the presence of (FIG. 12A) 36.0 μM insulin, (FIG. 12B) 7.7 μM DNA hairpin, (FIG. 12C) 10.3 μM TPX2 peptide and (FIG. 12D) 16.6 μM ssDNA. The data in (FIG. 12A) were filtered at 10 kHz (grey) or 0.5 kHz (green). FIG. 12E is a scatter plot of ΔI vs dwell time for the DNA hairpin (red), the peptide (purple) and the ssDNA (orange). Experiments were performed in 0.5 M NaCl, 20 mM Tris pH 7.5.

Since all of these polymers are negatively charged at pH 7.5, following their addition to the cis chamber (the opposite side of the SS membrane to portal insertion), electrophoresis allows molecular capture into the base of the portal protein. After addition of each biopolymer: 36.0 μM Insulin (FIG. 12A); 7.7 μM hairpin-polydT50 (FIG. 12B); 10.3 μM TPX2 peptide (FIGS. 11A-11C and 12C); 6.9 μM 60 bp-polydT30; and 16.6 μM ssDNA polydA20dC20dA20 (FIG. 12D); reversible partial blockades of the ionic current are observed at +60 mV. Similar short-lived bumping events as well as longer events were observed for each biopolymer, as described above for the TPX2 peptide (FIGS. 11A-11C). These types of blockades were also observed at several different voltages for DNA molecules, with voltage dependent changes in event frequency and duration for ssDNA polydA20dC20dA20 consistent with translocation occurring, as noted for the peptide above (FIGS. 11A-11C). Conversely, the folded, globular molecule of insulin, with a smallest dimension of ˜3 nm (PDB code: lzeh) (52, 53) is too large for the ˜2 nm constriction of the hybrid pore and therefore does not translocate. It is however possible that insulin explores the cavity at the portal tunnel's entrance (˜5 nm) without being transported to the trans chamber, producing structured events that are long-lived and have a low current blockade level. Such events have been previously observed for nanoreactors, where biomolecules are “captured” or “tethered” within ClyA and FracC nanopores (20, 54).

Lastly, the event characteristics for different biopolymers were compared at the same applied voltage of +60 mV by overlaying their scatter plots of ΔI vs. dwell times, as shown in FIG. 12E. The level of current blockade, ΔI, appeared to be biopolymer dependent. Current blockades were found for the dsDNA (FIG. 12E) of ΔI=34.6±4.2 pA, while in contrast, ΔI=18.1±3.2 pA was found for ssDNA. This is nearly two times less than for the partially dsDNA, and is consistent with values found for dsDNA and ssDNA in SS nanopores (55), where the difference in conductance was found to be ˜2.75 fold. For the peptide, ΔI=30.1±5.5 pA was found. Since it was shown that the peptide is transported through the pore (FIGS. 11A-11C) and the peptide is predicted to contain an α-helix of ˜1.4 nm in diameter as seen in the structure of the TPX2 peptide bound to its partner kinase, Aurora A (PDB: 1OL5), the data are compatible with translocation through the narrowest constriction of the hybrid pore (˜2 nm diameter). The ΔI value found for the peptide is consistent with the α-helical region being the main cause of the blockade, and with its diameter being intermediate between that of dsDNA and ssDNA. These data suggest that the predicted α-helix is present in the isolated peptide under these experimental conditions. While transport of structured biopolymers has been reported for nucleic acids (56, 57), it is believed that only a single report presents transport of an α-helical peptide through a protein nanopore (45).

Hybrid nanopores, supported by SS membranes could offer superior properties to both the planar lipid bilayer based pores (that are sensitive to temperature, osmotic pressure, and applied electric field strength and not geometrically controllable) and SS nanopores (that are prone to edge erosion and difficult to reproducibly fabricate with diameters <5 nm). However, despite having been the subject of industrial and academic research, development of a device that can be easily fabricated, has proven difficult. For example, producing a hybrid pore based on the α-hemolysin, a membrane protein, noted relatively short hybrid pore lifetime and required complex protein modifications (35). In contrast, the hybrid nanopore described here is based on a soluble, stable and relatively hydrophilic viral portal protein, whose chemical properties, including those inside tunnel, can be easily tuned.

An embodiment demonstrates biomolecule sensing capabilities of a novel lipid-free hybrid nanopore comprising the G20c portal protein inserted into a thin SS SiN membrane. This hybrid pore is easy to assemble, with the portal protein readily electrokinetically inserting into the SS-pores and typically remaining stable for hours of experimental time. The electrical sensing data clearly show characteristic readout for ds and ssDNA, as well as a peptide and a globular protein. The hybrid pore demonstrates utility as a nanosensor.

Materials and Methods

1. Preparation of CGG and CD/N

To produce the mutant portal protein, CGG, the following procedure was followed. An analogous procedure was used to produce a different mutant portal protein, CD/N. The following method of cloning, expression and purification of G20C portal proteins was used (here described for CGG and 49C mutants). The DNA encoding for G20c portal protein (residues 25-438) was amplified by PCR using Phusion high fidelity DNA polymerase (New England Biolabs, Ipswich, Mass., U.S.A.), and cloned into the YSBL-Lic+(Bonsor, D.; Butz, S. F.; Solomons, J.; Grant, S.; Fairlamb, I. J. S.; Fogg, M. J.; Grogan, G. Ligation Independent Cloning (LIC) as a Rapid Route to Families of Recombinant Biocatalysts From Sequenced Prokaryotic Genomes. Org. Biomol. Chem. 2006, 4, 1252-1260) expression plasmid encoding an N-terminal 3C protease cleavable hexahistidine tag using the HiFi DNA assembly master mix (New England Biolabs, Ipswich, Mass., U.S.A.). All mutant variations of the wild-type (WT) protein were produced using a variation of the linear exponential PCR and ligase-dependent production of closed circular plasmid DNA using Phusion DNA polymerase (New England Biolabs, Ipswich, Mass., U.S.A.). Briefly, primer sets were designed to introduce the mutation(s) and amplify the entire plasmid by PCR, after which the DNA product was purified using a PCR cleanup kit (Thermofisher). Phosphorylation and ligation of the amplified DNA ends and Dpn I digest of the template plasmid was achieved in a 3 h reaction at 37° C., containing 1× Cutsmart buffer (New England Biolabs, Ipswich, Mass., U.S.A.), 1 mM ATP, 10 mM DTT, and 1 unit each of Dpn I, T4 polynucleotide kinase, T4 ligase (New England Biolabs, Ipswich, Mass., U.S.A.). Ligated closed circular plasmid DNA was transformed into competent DH5α cells. Mutants were screened by colony PCR and confirmed by DNA sequencing in both directions.

Wild type and mutant proteins G20c WT (WT) and G20c V325G_I328G (GG)) were expressed and purified from E. coli BL21 (DE3) pLys S cells. Proteins containing cysteine mutants, G20c-L49C (49C) and G20c-L49C_V325G_I328G (CGG) were expressed and purified from the SHuffle (New England Biolabs, Ipswich, Mass., U.S.A.) expression strain. Protein expression and purification was conducted as described (Williams, L. S.; Levdikov, V. M.; Minakhin, L.; Severinov, K.; Antson, A. A. 12-Fold Symmetry of the Putative Portal Protein From the Thermus Thermophilus Bacteriophage G20C Determined by XRay Analysis. Acta Crystallogr., Sect. F: Struct. Biol. Cryst. Commun. 2013, 69, 1239-1241.) in LB (Melford) containing 35 μg/mL kanamycin and 50 μg/mL chloramphenicol. Briefly, 10 mL of an overnight culture was inoculated into 1 L of LB (containing antibiotics) and incubated at 37° C. until the OD600 reached 0.8, followed by induction overnight at 16° C. with 0.5 mM IPTG when the cells were harvested by centrifugation at 4000 rpm for 30 min and the pellets snap frozen in liquid nitrogen and stored at −80° C. until use. Proteins (49C and CGG) expressed in Shuffle cells were incubated at 30° C. before and after induction. Cell pellets were thawed and resuspended in 5 mL/g of lysis buffer (50 mM Tris pH 8, 1 M NaCl, 10 mM imidazole, 100 mM AEBSF, 10 mg/mL lysozyme) and lysed by sonication on ice. The lysate was clarified by centrifugation at 15000 rpm for 30 min, filtered through a 0.22 μm membrane before loading on a HisTrap FF 5 mL (GE Healthcare Life Sciences). The His-tagged G20c protein was eluted using a gradient to 100% Buffer B (1 M NaCl, 50 mM Tris pH 7.5, 500 mM imidazole) over 10 column volumes. Fractions of the purified protein were pooled, buffer exchanged into 50 mM Tris pH 8, 500 mM NaCl 50 mM potassium glutamate, and the histidine tag removed by 3C protease digestion at rt overnight. The cleaved protein was then further purified over a HisTrap FF 5 mL. Fractions containing cleaved G20c protein were pooled, concentrated, and finally purified on a 16/600 Superose 6 (GE healthcare Life Sciences) gel filtration column in 20 mM Tris pH8, 1 M NaCl, 50 mM potassium glutamate, concentrated by ultrafiltration using a 100 kDa cutoff filter (Vivaspin) to ˜4 mg/mL for biophysical experiments or ˜10 mg/mL for crystallographic studies, snap frozen on liquid nitrogen and stored at −80° C. Proteins containing the L49C mutation were purified in buffers containing 2 mM DTT.

Likewise, other mutant proteins taught herein were expressed and purified in a similar fashion as described in the preceding paragraph.

2. Protein Engineering and Purification:

Mutant portal proteins, CGG (33) was produced as described in Section 1 (“Preparation of CGG and CD/N”), above, in E. coli, and CD/N was prepared in an analogous fashion, with the exception that two buffer exchange steps over a desalting column (GE Healthcare) were used to improve 3C cleavage of the histidine affinity tag (50 mM Tris pH 8, 500 mM NaCl, 50 mM K Glutamate, 1 mM DTT) prior to a second IMAC step and purification to homogeneity in 20 mM Tris pH 8, 1 M NaCl, 1 mM DTT, before freezing in liquid nitrogen and storage at −80° C. Protein was exchanged into 20 mM Tris pH 7.5, 0.5 M NaCl buffer (Zeba Spin Columns, Thermofisher) for use in hybrid nanopore formation. CD/N mutant proteins where characterized for stability and assembly state by nanoDSF and negative stained TEM.

3. Experimental Set-Up:

Nanopores were fabricated in 30 nm thick SiN membranes using previously reported methods (58,59). The pore diameters ranged between 5.4 and 6 nm in order to squeeze properly the portal protein. Nanopores were cleaned with hot piranha (3:1 H2SO4/H2O2), followed by hot deionized water before each experiment. After being dried under vacuum, nanopore chips were assembled in a custom cell equipped with Ag/AgCl electrodes, and quick-curing silicone elastomer was applied between the chip and the cell to seal the device and thereby reduce the noise by minimizing the chip capacitance. We introduced 0.5 M NaCl, 20 mM Tris pH 7.5 as an electrolyte solution onto both sides of the chip. Portal protein was always added to the trans chamber and the biopolymers to the cis chamber. All experiments were carried out at ambient temperature. Human insulin was purchased from Alfa Aesar (Thermofisher), dsDNA Hairpin (5′-GCTGTCTGTTGCTCTCTCGCAACAGACAGC T50-3′), ssDNA (5′-dA20dC20dA20-3′), 60 bp-polydT30 ((5′-TCAGGGTTTTTTTACT)4 T30-3′) and its complementary strand ((3′-AGTAAAAAAACCCTGA-5′)4) were synthesized by Integrated DNA Technology.

4. Electrical Detection and Data Acquisition:

The ionic current through SS nanopores and portal hybrid protein was measured using an Axopatch 200B amplifier (Molecular Devices). Data were filtered at 10 kHz and acquired at 250 kHz using the DigiData 1200 digitizer with a custom National Instruments LabVIEW program. Data was processed and events were detected using Pythion software (https://github.com/rhenley/Pyth-Ion/). The values for the open pore current (I0) and the standard deviation of the noise (a) was extracted. (The threshold (Th) applied in Pythion to separate events from the noise is given by Th=I0−4σ. The average duration of blockades is deduced from the distribution of blockade duration, τt. The two blockade time distributions of independent events are adjusted with a double exponential function, y=A1exp(t/τ1)+A2exp(t/τ2). All statistical analyses were performed using Igor Pro software (WaveMetrics Inc.).

### Definitions and Discussion of Terminology

As used herein, a “protein” is a biological molecule consisting of one or more chains of amino acids. Proteins differ from one another primarily in their sequence of amino acids, which is dictated by the nucleotide sequence of the encoding gene. A peptide is a single linear polymer chain of two or more amino acids bonded together by peptide bonds between the carboxyl and amino groups of adjacent amino acid residues; multiple peptides in a chain can be referred to as a polypeptide. Proteins can be made of one or more polypeptides. Shortly after or even during synthesis, the residues in a protein are often chemically modified by posttranslational modification, which alters the physical and chemical properties, folding, stability, activity, and ultimately, the function of the proteins. Sometimes proteins have non-peptide groups attached, which can be called prosthetic groups or cofactors.

As used herein, a “protein variant” refers to a protein that differs from a reference peptide by one or more modifications, for example, substitutions, insertions or deletions, and is not naturally occurring. A protein variant can include an isolated protein, which is not naturally occurring, and is free from the cell, or other proteins in a medium, in which it was produced. An isolated protein is a protein or fragment thereof that is substantially free of other proteins, and encompasses proteins that are isolated to a higher purity, such as proteins that are 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99% or 100% pure. The protein can, for example, be isolated from the extracellular medium in which the microorganism to be assayed is growing, or from the cell membrane of the microorganism, using standard protein purification techniques, described, for example, in (See, e.g., Ausubel, F. M. et al. (“Current Protocols in Molecular Biology”, John Wiley & Sons, (1998) the entire teachings of which are incorporated herein by reference).

The proteins of embodiments also encompass fragments and sequence variants of the proteins described herein. Variants include a substantially homologous protein encoded by the same genetic locus in an organism, i.e., an allelic variant, as well as other variants. Variants also encompass proteins derived from other genetic loci in an organism. Variants also include proteins substantially homologous or identical to these proteins but derived from another organism and/or d and 1 isomers (i.e., an ortholog), produced by chemical synthesis, or produced by recombinant methods.

In some embodiments, the protein variant comprises an amino acid sequence, such as one of the sequences listed herein or a sequence having at least 50%, 60%, 70%, 80%, 85%, 90%, 95%, or 99% sequence identity to one of the sequences listed herein, as determined using a sequence comparison program and parameters described herein.

The percent identity of two amino acid sequences can be determined by aligning the sequences for optimal comparison purposes (e.g., gaps can be introduced in the sequence of a first sequence). The amino acids at corresponding positions are then compared, and the percent identity between the two sequences is a function of the number of identical positions shared by the sequences (i.e., % identity=# of identical positions/total # of positions×100). In certain embodiments, the length of the amino acid sequence aligned for comparison purposes is at least 30%, preferably, at least 40%, more preferably, at least 60%, and even more preferably, at least 70%, 80%, 90%, or 100% of the length of the reference sequence. The actual comparison of the two sequences can be accomplished by well-known methods, for example, using a mathematical algorithm. A preferred, non-limiting example of such a mathematical algorithm is described in Karlin et al., 90 PROC. NAT'L ACAD. SCI. USA 5873-77 (1993), which is incorporated herein by reference. Such an algorithm is incorporated into the BLAST programs (version 2.2) as described by Schaffer et al., 29 NUCLEIC ACIDS RES. 2994-3005 (2001), which is incorporated herein by reference. When utilizing BLAST and Gapped BLAST programs, the default parameters of the respective programs can be used. In one embodiment, the database searched is a non-redundant database, and parameters for sequence comparison can be set at: no filters; Expect value of 10; Word Size of 3; the Matrix is BLOSUM62; and Gap Costs have an Existence of 11 and an Extension of 1.

In another embodiment, the percent identity between two amino acid sequences can be determined by using the GAP program in the GCG software package (available from Accelrys, Inc. of San Diego, Calif.) using either a Blossom 63 matrix or a PAM250 matrix, and a gap weight of 12, 10, 8, 6, or 4 and a length weight of 2, 3, or 4. In yet another embodiment, the percent identity between two nucleic acid sequences can be determined using a gap weight of 50 and a length weight of 3. Other preferred sequence comparison methods are described herein.

The invention also encompasses proteins having a lower degree of identity but having sufficient similarity so as to perform one or more of the same functions performed by a protein encoded by a nucleic acid molecule of the invention (e.g., the ability to provide the hydrophilic protein channel in a stable insertion fit within a solid-state pore opening). Similarity is determined by conserved amino acid substitution. Such substitutions are those that substitute a given amino acid in a peptide by another amino acid of like characteristics. Conservative substitutions are likely to be phenotypically silent. Typically seen as conservative substitutions are the replacements, one for another, among the aliphatic amino acids Ala, Val, Leu, and Ile; interchange of the hydroxyl residues Ser and Thr; exchange of the acidic residues Asp and Glu; substitution between the amide residues Asn and Gln; exchange of the basic residues Lys and Arg; and replacements among the aromatic residues Phe and Tyr. Guidance concerning which amino acid changes are likely to be phenotypically silent are found in Bowie et al., SCIENCE 247:1306-10 (1990), which is incorporated herein by reference.

Functional variants can also contain substitution of similar amino acids that result in no change or an insignificant change in function. Alternatively, such substitutions may positively or negatively affect function to some degree. Non-functional variants typically contain one or more non-conservative amino acid substitutions, deletions, insertions, inversions, or truncations or a substitution, insertion, inversion, or deletion in a critical residue or critical region.

The invention also includes protein and peptide fragments of the amino acid sequences of the various mentioned proteins or variants (e.g., functional variants) thereof.

Fragments can be discrete (not fused to other amino acids or peptides) or can be within a larger peptide. Further, several fragments can be comprised within a single larger peptide. The peptides can, for example, be produced using standard recombinant protein techniques (See, e.g., Ausubel, F. M. et al. (“Current Protocols in Molecular Biology”, John Wiley & Sons, (1998) the entire teachings of which are incorporated herein by reference). In addition, the proteins of the present invention can also be generated using recombinant techniques.

In some embodiments, protein variants are produced by mutation of amino acid sequences, but protein variants in embodiments can also be produced by removing or adding one or more amino acid residues through other well-known means, including chemical synthesis.

As used herein, a “modification” of a protein refers to a substitution, insertion or deletion of one or more amino acids.

As used herein, a “modification of an amino acid sequence” refers to a mutant amino acid sequence that is not naturally occurring, and that has a mutation relative to a reference amino acid sequence, that is, by an alteration of the amino acid sequence of the reference amino acid sequence, such as by substitution, insertion or deletion of one or more amino acid residues.

As used herein, “nucleic acid” refers to a macromolecule composed of chains (a polymer or an oligomer) of monomeric nucleotide. The most common nucleic acids are deoxyribonucleic acid (DNA) and ribonucleic acid (RNA). It should be further understood that the present invention can be used for sensing biomolecules containing artificial nucleic acids such as peptide nucleic acid (PNA), morpholino, locked nucleic acid (LNA), glycol nucleic acid (GNA) and threose nucleic acid (TNA), among others. In various embodiments of the present invention, nucleic acids can be derived from a variety of sources such as bacteria, virus, humans, and animals, as well as sources such as plants and fungi, among others. The source can be a pathogen. Alternatively, the source can be a synthetic organism. Nucleic acids can, for example, be genomic, extrachromosomal or synthetic. In addition, the term “nucleic acid,” is used herein to include a polymeric form of nucleotides of any length, including, but not limited to, ribonucleotides or deoxyribonucleotides. Further, the term refers only to the primary structure of the molecule. Thus, in certain embodiments the term can include triple-, double- and single-stranded DNA, PNA, complementary DNA (cDNA), as well as triple-, double- and single-stranded RNA. It can also include modifications, such as by methylation and/or by capping, and unmodified forms of a polynucleotide. More particularly, the term “nucleic acid,” includes polydeoxyribonucleotides (containing 2-deoxy-D-ribose), polyribonucleotides (containing D-ribose), any other type of polynucleotide which is an N- or C-glycoside of a purine or pyrimidine base, and other polymers containing nonnucleotidic backbones, for example, polyamide (e.g., peptide nucleic acids (PNAs)) and polymorpholino (commercially available from Anti-Virals, Inc., Corvallis, Oreg., U.S.A., as Neugene) polymers, and other synthetic sequence-specific nucleic acid polymers providing that the polymers contain nucleobases in a configuration which allows for base pairing and base stacking, such as is found in DNA and RNA. In addition, a “nucleic acid” can include a plasmid DNA (pDNA), such as a plasmid DNA vector.

As used herein, a “modification of a nucleic acid sequence” refers to a mutant nucleic acid (e.g., DNA) that is not naturally occurring, and that has a mutation relative to a reference nucleic acid, that is, by an alteration of the nucleotide sequence of the reference nucleic acid sequence, such as by substitution, insertion or deletion of one or more nucleotides. In some embodiments, the mutation can be a missense mutation, which is a type of nonsynonymous substitution that is a point mutation in which a single nucleotide change results in a codon that codes for a different amino acid. In some embodiments, modifications of a nucleic acid sequence produce modified proteins and peptides described herein.

In some embodiments, a nucleic acid molecule comprising a modification of a nucleic acid sequence can be isolated or recombinant. In addition, such a modification of a nucleic acid sequence can be produced using techniques of cell-free protein synthesis, which produce protein using biological machinery in a cell-free system, without the use of living cells. Cell free expression systems can, for example, be used, that use linear DNA sequences propagated by polymerase chain reaction (PCR) reactions.

As used herein, a “vector” is a molecule, e.g., a plasmid or virus designed for gene expression in cells. The vector is used to introduce a specific gene into a target cell, and can commandeer the cell's mechanism for protein synthesis to produce the protein encoded by the gene. In addition, in some embodiments, a “vector” comprising a modification of a nucleic acid sequence can refer to any DNA plasmid containing the sequence for propagation of the DNA plasmid and/or for expression of the peptide encoded by the modification of a nucleic acid sequence in any cellular system (such as a bacteria, yeast or eukaryotic cell system).

Modifications of a nucleic acid sequence taught herein can, for example, be produced using the techniques taught in “Improved Methods for Site-directed Mutagenesis using NEBuilder® HiFi DNA Assembly Master Mix,” New England BioLabs® Inc., Ipswich, Mass., U.S.A., found at the URL https://www.neb.com/applications/cloning-and-synthetic-biology/dna-assembly-and-cloning/-/media/nebus/files/application-notes/improved-methods-for-site-directed-mutagenesis-using-nebuilder-hifi- dna-assembly-master-mix.pdf. where “..” replaces a hyperlink. Other means of mutating or modifying nucleic acids are well known in the art.

As used herein, a “biomolecule” refers to a nucleic acid, a protein, a biopolymer, or any other biological molecule, or an organic molecule, or fragment or variant thereof, or any combination of such nucleic acids, proteins, biopolymers, other biological molecules, or organic molecules, or any combination thereof. For example, the biomolecule can, in some embodiments, be or include single-stranded DNA, double-stranded DNA or RNA. In addition, a “biomolecule” can include (1) an antibody, such as a monoclonal antibody, or another ligand specific molecule, and (2) other molecules that may have or could affect biologic and/or cellular activity.

In some embodiments, a protein included in a hydrophilic protein channel comprises a modification of SEQ ID NO: 1, the modification comprising one of: SEQ ID NO: 27; SEQ ID NO: 11; SEQ ID NO: 37; SEQ ID NO: 15, SEQ ID NO: 17, SEQ ID NO: 21, SEQ ID NO: 23, SEQ ID NO: 25, SEQ ID NO: 19, SEQ ID NO: 9, SEQ ID NO: 13, SEQ ID NO: 29, SEQ ID NO: 31, SEQ ID NO: 35, SEQ ID NO: 33, SEQ ID NO: 3, SEQ ID NO: 5, and SEQ ID NO: 7.

In another embodiment, a protein included in a hydrophilic protein channel comprises a modification of SEQ ID NO: 1, the modification comprising one of: SEQ ID NO: 27; SEQ ID NO: 37; SEQ ID NO: 15, SEQ ID NO: 17, SEQ ID NO: 23, SEQ ID NO: 25, SEQ ID NO: 19, SEQ ID NO: 9, SEQ ID NO: 13, SEQ ID NO: 29, SEQ ID NO: 31, SEQ ID NO: 35, SEQ ID NO: 33, SEQ ID NO: 3, SEQ ID NO: 5, and SEQ ID NO: 7.

In other embodiments, a protein variant comprises a modification of SEQ ID NO: 1, the modification comprising one of: SEQ ID NO: 27; SEQ ID NO: 11; SEQ ID NO: 37; SEQ ID NO: 15, SEQ ID NO: 17, SEQ ID NO: 21, SEQ ID NO: 23, SEQ ID NO: 25, SEQ ID NO: 19, SEQ ID NO: 9, SEQ ID NO: 13, SEQ ID NO: 29, SEQ ID NO: 31, SEQ ID NO: 35, SEQ ID NO: 33, SEQ ID NO: 3, SEQ ID NO: 5, and SEQ ID NO: 7.

In another embodiment, a protein variant comprises a modification of SEQ ID NO: 1, the modification comprising one of: SEQ ID NO: 27; SEQ ID NO: 37; SEQ ID NO: 15, SEQ ID NO: 17, SEQ ID NO: 23, SEQ ID NO: 25, SEQ ID NO: 19, SEQ ID NO: 9, SEQ ID NO: 13, SEQ ID NO: 29, SEQ ID NO: 31, SEQ ID NO: 35, SEQ ID NO: 33, SEQ ID NO: 3, SEQ ID NO: 5, and SEQ ID NO: 7.

In other embodiments, a nucleic acid molecule comprises a modification of a nucleic acid sequence of SEQ ID NO: 2, the modification of the nucleic acid sequence comprising one of: SEQ ID NO: 28; SEQ ID NO: 12; SEQ ID NO: 38; SEQ ID NO: 16, SEQ ID NO: 18, SEQ ID NO: 22, SEQ ID NO: 24, SEQ ID NO: 26, SEQ ID NO: 20, SEQ ID NO: 10, SEQ ID NO: 14, SEQ ID NO: 30, SEQ ID NO: 32, SEQ ID NO: 36, SEQ ID NO: 34, SEQ ID NO: 4, SEQ ID NO: 6, and SEQ ID NO: 8.

In another embodiment, a nucleic acid molecule comprises a modification of a nucleic acid sequence of SEQ ID NO: 2, the modification of the nucleic acid sequence comprising one of: SEQ ID NO: 28; SEQ ID NO: 38; SEQ ID NO: 16, SEQ ID NO: 18, SEQ ID NO: 24, SEQ ID NO: 26, SEQ ID NO: 20, SEQ ID NO: 10, SEQ ID NO: 14, SEQ ID NO: 30, SEQ ID NO: 32, SEQ ID NO: 36, SEQ ID NO: 34, SEQ ID NO: 4, SEQ ID NO: 6, and SEQ ID NO: 8.

In other embodiments, a protein variant is encoded by a modification of a nucleic acid sequence of SEQ ID NO: 2, the modification of the nucleic acid sequence comprising one of: SEQ ID NO: 28; SEQ ID NO: 12; SEQ ID NO: 38; SEQ ID NO: 16, SEQ ID NO: 18, SEQ ID NO: 22, SEQ ID NO: 24, SEQ ID NO: 26, SEQ ID NO: 20, SEQ ID NO: 10, SEQ ID NO: 14, SEQ ID NO: 30, SEQ ID NO: 32, SEQ ID NO: 36, SEQ ID NO: 34, SEQ ID NO: 4, SEQ ID NO: 6, and SEQ ID NO: 8.

In another embodiment, a protein variant is encoded by a modification of a nucleic acid sequence of SEQ ID NO: 2, the modification of the nucleic acid sequence comprising one of: SEQ ID NO: 28; SEQ ID NO: 38; SEQ ID NO: 16, SEQ ID NO: 18, SEQ ID NO: 24, SEQ ID NO: 26, SEQ ID NO: 20, SEQ ID NO: 10, SEQ ID NO: 14, SEQ ID NO: 30, SEQ ID NO: 32, SEQ ID NO: 36, SEQ ID NO: 34, SEQ ID NO: 4, SEQ ID NO: 6, and SEQ ID NO: 8.

In other embodiments, a vector comprises a modification of a nucleic acid sequence of SEQ ID NO: 2, the modification of the nucleic acid sequence comprising one of: SEQ ID NO: 28; SEQ ID NO: 12; SEQ ID NO: 38; SEQ ID NO: 16, SEQ ID NO: 18, SEQ ID NO: 22, SEQ ID NO: 24, SEQ ID NO: 26, SEQ ID NO: 20, SEQ ID NO: 10, SEQ ID NO: 14, SEQ ID NO: 30, SEQ ID NO: 32, SEQ ID NO: 36, SEQ ID NO: 34, SEQ ID NO: 4, SEQ ID NO: 6, and SEQ ID NO: 8.

In another embodiment, a vector comprises a modification of a nucleic acid sequence of SEQ ID NO: 2, the modification of the nucleic acid sequence comprising one of: SEQ ID NO: 28; SEQ ID NO: 38; SEQ ID NO: 16, SEQ ID NO: 18, SEQ ID NO: 24, SEQ ID NO: 26, SEQ ID NO: 20, SEQ ID NO: 10, SEQ ID NO: 14, SEQ ID NO: 30, SEQ ID NO: 32, SEQ ID NO: 36, SEQ ID NO: 34, SEQ ID NO: 4, SEQ ID NO: 6, and SEQ ID NO: 8.

In other embodiments, a cell comprises a vector that comprises a modification of a nucleic acid sequence of SEQ ID NO: 2, the modification of the nucleic acid sequence comprising one of: SEQ ID NO: 28; SEQ ID NO: 12; SEQ ID NO: 38; SEQ ID NO: 16, SEQ ID NO: 18, SEQ ID NO: 22, SEQ ID NO: 24, SEQ ID NO: 26, SEQ ID NO: 20, SEQ ID NO: 10, SEQ ID NO: 14, SEQ ID NO: 30, SEQ ID NO: 32, SEQ ID NO: 36, SEQ ID NO: 34, SEQ ID NO: 4, SEQ ID NO: 6, and SEQ ID NO: 8.

In another embodiment, a cell comprises a vector that comprises a modification of a nucleic acid sequence of SEQ ID NO: 2, the modification of the nucleic acid sequence comprising one of: SEQ ID NO: 28; SEQ ID NO: 38; SEQ ID NO: 16, SEQ ID NO: 18, SEQ ID NO: 24, SEQ ID NO: 26, SEQ ID NO: 20, SEQ ID NO: 10, SEQ ID NO: 14, SEQ ID NO: 30, SEQ ID NO: 32, SEQ ID NO: 36, SEQ ID NO: 34, SEQ ID NO: 4, SEQ ID NO: 6, and SEQ ID NO: 8.

As used herein, a “nanopore” refers to a pore with a maximum pore dimension of less than about 20 nanometers, such as less than about 10 nanometers, or less than about 5 nanometers, or about 2 nanometers or less.

As used herein, a “solid-state matrix” refers to matrix made of a solid-state material, which is non-molecular solid material. For example, the solid-state matrix can be made of materials such as a silicon-containing nitride (e.g., silicon nitride), a silicon-containing carbide (e.g., silicon carbon), a silicon-containing oxide (e.g. silicon oxide), nickel, silicon, hafnium, or other solid-state materials.

As used herein, a “solid-state pore opening” refers to a pore opening formed in a solid-state matrix.

As used herein, a “hydrophilic protein” refers to a protein that includes residues that are soluble in water, at least over a substantial portion of a surface that is in contact with the water, such as an external surface of the hydrophilic protein.

As used herein, a “hydrophilic protein channel” refers to a protein structure comprising a channel opening therethrough, the structure being formed by one or more hydrophilic proteins.

As used herein a “stable insertion fit” of a hydrophilic protein channel within a solid-state pore opening refers to a substantially snug fit of the hydrophilic protein channel's external surface within the solid-state pore opening, such that the hydrophilic protein channel is stable for at least 2 hours, for example more than 3 hours, 4 hours, 5 hours or more, including substantially permanently stable, in the solid-state pore opening, while in the presence of a water solvent.

As used herein, a “protein nanopore channel” refers to a channel opening formed by and through a protein, the channel having a diameter less than about 20 nanometers, such as less than about 10 nanometers, or less than about 5 nanometers, or about 2 nanometers or less.

As used herein, a “wing loop residue” refers to an amino acid residue in a portion of a protein that is positioned in a substantially wing-shaped external portion of the protein. For example, on the portal protein of the Thermus thermophilus bacteriophage G20c, the wing loop residue can be one of the residues 36-41 or 46-54.

As used herein, a “tunnel loop residue” refers to an amino acid residue in a portion of a protein that forms an intruding loop within a channel that is formed by the protein, such as by multiple monomers of the protein or within a single monomer of the protein. For example, on the portal protein of the Thermus thermophilus bacteriophage G20c, the tunnel loop residue can be one of the residues 316-335.

As used herein, an “upper internal surface residue” refers to an amino acid residue in a portion of a protein that forms an internal surface of a tunnel portion of a channel that is formed by the protein, such as by multiple monomers of the protein or within a single monomer of the protein, and that is positioned “upwards” as determined by an accepted orientation of the protein, for example one that is in a “cap” portion of a cork-shaped protein that becomes inserted in a pore with its cap portion upwards and its “stem” portion embedded further into the pore. For example, on the portal protein of the Thermus thermophilus bacteriophage G20c, the upper internal surface residue can be one of the residues 381-400.

As used herein, a “lower internal surface residue” refers to an amino acid residue in a portion of a protein that forms an internal surface of a tunnel portion of a channel that is formed by the protein, such as by multiple monomers of the protein or within a single monomer of the protein, and that is positioned “downwards” or in a lower portion as determined by an accepted orientation of the protein, for example one that is in a “stem” portion of a cork-shaped protein that becomes inserted in a pore with its cap portion upwards and its “stem” portion embedded further into the pore. For example, on the portal protein of the Thermus thermophilus bacteriophage G20c, the lower internal surface residue can be one of the residues 273-303.

As used herein, an “external surface residue” refers to a residue on an outer surface of the protein, such as one that is exposed to a solid-state pore. For example, on the portal protein of the Thermus thermophilus bacteriophage G20c, the external surface residue can be residue 230, although is not limited to that residue and can include other external surface residues.

Tables and Sequences on Protein Versions and the Associated DNA

The below tables, Tables 1-7, provide properties of protein versions taught herein, and the associated DNA. In the left column of each table, the protein version is provided, corresponding to the same protein version listed in the other tables.

In the below sequences, the affinity tags in the protein sequence are included for those constructs where they are not removed post purification, for example, SEQ ID NO: 3 and SEQ ID NO: 37. However, since histidine tags can perturb nucleic acid interaction with proteins, particularly a circular ring of twelve such tags as is formed in a dodecameric assembly of proteins, the final purified protein (with, for example, a hexahistadine tag) can be cleaved with 3C protease to remove the hexahistidine tag, leaving GPA as the remaining part of the tag at the N-terminus of the protein. Hence, GPA is the sequence for the N-terminal 3 amino acids for the following sequence listings, which reflect those proteins after having been cleaved: SEQ ID NO: 5, SEQ ID NO: 7, SEQ ID NO: 9, SEQ ID NO: 11, SEQ ID NO: 13, SEQ ID NO: 15, SEQ ID NO: 17, SEQ ID NO: 19, SEQ ID NO: 21, SEQ ID NO: 23, SEQ ID NO: 25, SEQ ID NO: 27, SEQ ID NO: 29, SEQ ID NO: 31, SEQ ID NO: 33 and SEQ ID NO: 35. However, the full length tag sequence present prior to cleavage is included in the DNA sequence of the full open reading frame used to express the protein in E. coli. For the protein version WT Nanopore, below, the full length amino acid sequence prior to cleavage is given in SEQ ID NO: 39. The cleaved amino acid sequence of this protein is given in SEQ ID NO: 7.

Residue position number in full length wild type (WT) amino acid sequence.

Nucleotide position number in the full length wild type DNA sequence of the open reading frame coding for the protein.

Sequences marked with an “*” in Table 4, above, were sequenced in one direction (across the new introduced mutation) and the complete sequence was assembled from the data available from the template used to produce the new mutant construct. All other sequences assembled from original DNA sequencing data.

In the below sequences, a letter “X” signifies “any amino acid,” and a letter “n” signifies any nucleotide (n=A, T, C or G).

